# A review of SARS-CoV-2 drug repurposing: databases and machine learning models

CorpusID: 260671013 - [https://www.semanticscholar.org/paper/08de4ebae463c00e3ab12c65a3b8ab5955f1aa0d](https://www.semanticscholar.org/paper/08de4ebae463c00e3ab12c65a3b8ab5955f1aa0d)

Fields: Computer Science, Medicine

## (s1) Databases for drug repurposing studies
(p1.0) There have been onerous efforts to collate, archive, and digitize life sciences research data worldwide, leading to a rise of hundreds of online databases providing various features for specific purposes. There are numerous repository classifications depending on the classifier perspective. Bagherian et al (2021) categorized the databases as drug-target interaction (DTI), drug/target-centered, drug-target binding affinity, and supporting databases. Wu et al (2022) classified the databases as drug combination databases and databases listed in the other related databases section. Tanoli et al (2021) grouped the databases as chemical, genomics, threedimensional protein structures, protein classifications and interactions, reaction pathways, molecular omics, DTI, and disease databases. Nevertheless, it is clear that various databases fit under multiple categories. Given these observations, most databases have different data types, therefore, they can be referred to as heterogeneous databases. In this section, 20 frequently used repositories in drug repurposing studies have been considered. An overview of the data types provided by each database is rendered in Table 1. Within these databases, some of them provide specific services considering relevant SARS-CoV-2 data and are jointly represented in Table 2, while being elaborated beneath each database section below.
## (s6) DrugCentral
(p6.0) DrugCentral (https://drugcentral.org/) is a drug-centred database that was first released in 2016 to quantify the existing pharmaceutical drugs and their targets (Avram et al, 2021). The Division of Translational Informatics currently maintains it at the University of New Mexico (UNM) in collaboration with Illuminating the Druggable Genome (IDG) Consortium. This database provides information on active ingredients, general drugs, FDA-approved drugs, European Medicines Agency (EMA)-approved drugs, and Pharmaceuticals and Medical Devices Agency (PMDA)-approved drugs. Drug annotation includes drug dosage, absorption, distribution, metabolism, excretion, and toxicity (ADMET), adverse events for both males and females, pharmacologic action, drug use, pharmaceutical products, bioactivity, and new drug applications. Also, this repository offers target search to allocate the drugs acting on them. DrugCentral compiles 4,927 active ingredients and 112,359 FDA drug labels. The last updated database release comprises 226 veterinary drugs along with 804 targets.   (Kanehisa et al, 2017). This resource provides numerous rich databases for multiple purposes. KEGG-Drugs database gathers information about approved drugs in Japan, the US, and Europe. Two additional databases are associated with the KEGG-Drug database, KEGG-pathway database, and KEGG-Medicus database. The former database provides information about the drug pathway, whilst the latter database delivers reported drug-target interaction. Furthermore, the known drug-target interaction pairs can be extracted from the KEGG-Brite database. KEGG-COMPOUND database collects the most relevant compounds in the biological system. The database is integrated with other KEGG databases and provides information about the chemical properties, reactions, pathways, modules, and enzymes associated with the compound of interest. Moreover, KEGG-Genes are a collection of gene and protein sequences of cellular organisms and viruses from public databases, mainly NCBI RefSeq and GenBank. As per 2023 statistics, 46,086,588 and 648,811 genes expressing proteins are appended for organisms and viruses, respectively. The counts for proteins are 312 for viruses and 4,125 for other organisms. The similarity among the sequences is generated through KEGG (Sequence Similarity DataBase) SSDB. Furthermore, the molecular function of functional orthologs in many biological pathways are provided in KEGG-KO database. Each functional ortholog is given a KO entry which is cross-referenced with experimental evidence. Additionally, the KEGG-Genome database collects detailed genomic information about many cellular organisms and viruses. Besides, there is the KEGG-Enzymes database that collects information about enzymatic reactions.
## (s36) Machine learning models
(p36.0) Machine learning (ML) is a distinct division of artificial intelligence (AI) that gained attention in drug repurposing field for its appealing advantages. It was proven to have an efficient intervention to accelerate the prediction of potential SARS-CoV-2 inhibitors and to prioritize drugs for in vitro testing (Yang et al, 2022). A significant feature of ML models is their ability to learn and explore functional relationships in the given data set that humans could hardly investigate. Usually, ML workflow comprises four steps: 1) data curation and preprocessing, 2) feature extraction, 3) model fitting, and 4) interpretation (Angermueller et al, 2016). Upon the human intervention intensity in each step, the ML can be classified into conventional ML models and deep learning (DL) models. In the former models, significant human intervention is required compared to the latter models. Specifically, in DL models, the feature extraction step is automated, unlike in the conventional ML models where it is done manually (Sarker, 2021). Both models can be implemented for classification, regression, clustering, or pattern recognition problems (Carracedo-Reboredo et al, 2021).
## (s46) Application in drug repurposing
(p46.0) Ivanov et al (2020) developed an SVM (radial kernel) model based on QSAR methodology to predict inhibitors of SARS-CoV-2 RdRp protein. The model screened three datasets, FDA-approved drugs dataset, molecules from the COVID-19 Antiviral dataset, and molecules from published research studies on SARS, MERS, and SARS-CoV-2. Among the total predictions, 92 FDA-approved drugs were suggested as RdRp inhibitors. Some of which are Thalomid, Grazoprevir, Sildenafil, Ruxolitinib, Duvelisib, Moxifloxacin, Acalabrutinib, and Telmisartan. Kowalewski and Ray (2020) aimed to mine potential drugs for COVID-19. Instead of considering a particular target, the authors gathered a compendium of 65 human proteins which were proven to interact with SARS-CoV-2 proteins as targets from bioassay data. Three RBF-SVM models for classification and regression were combined (ensemble model) and implemented for all the targets. For only one target, namely, EIF4H, regularized random forest (RRF) was aggregated along with the SVM ensemble model. After training, the ML models were utilized to predict inhibitors against the targets from a set of approved and registered drugs. Predictions were categorized and curated based on the estimated mammalian toxicity and vapor pressure. The team constructed a network of the predicted drugs and targets. The targets with few drug candidates were excluded. The multi-target drug predictions comprise Phenazopyridine, Abemaciclib, Promazine, Tyverb, Pirenzepine, and Ebastine. Rajput et al (2021b) implemented SVM to predict anti-SARS-CoV-2 drugs. The model predicted 12 drugs: Verteporfin, Argatroban, Reboxetine, Guanfacine, Telotristat ethyl, Betrixaban, Leuprolide, Trovafloxacin, Peramivir, Salmeterol, Oxybuprocaine, and Warfarin. These predictions were further validated by molecular docking to investigate the binding affinity of the drugs against the complex of spike protein and ACE2 receptor. The authors decided that the most potential candidates are those having binding affinities ranging from âˆ’9.5 kcal/mol to -8 kcal/mol. Accordingly, seven molecules were prioritized: Verteporfin, Alatrofloxacin, Metergoline, Rescinnamine, Goserelin, Leuprolide, and Telotristat ethyl.
