# A Survey of Pyridoxal 5 ′ -Phosphate-Dependent Proteins in the Gram-Positive Model Bacterium Bacillus subtilis

CorpusID: 148571692
 
tags: #Medicine, #Environmental_Science, #Chemistry, #Biology

URL: [https://www.semanticscholar.org/paper/7b99572d1e04b3fc034d5bc00b0f4f0c25f00ff4](https://www.semanticscholar.org/paper/7b99572d1e04b3fc034d5bc00b0f4f0c25f00ff4)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

A Survey of Pyridoxal 5 ′ -Phosphate-Dependent Proteins in the Gram-Positive Model Bacterium Bacillus subtilis
May 2019

Roberto Contestabile 
Angela Tramonti 
CNR), ItalyDelle Ricerche 
Riccardo Percudani 
Fabian M Commichau 
Björn Richts 
Jonathan Rosenberg 
Fabian M Commichau 

Istituto di Biologia e Patologia Molecolari (IBPM)
Sapienza University of Rome
Consiglio NazionaleItaly


Department of General Microbiology
University of Parma
Italy


University of Goettingen
GöttingenGermany

A Survey of Pyridoxal 5 ′ -Phosphate-Dependent Proteins in the Gram-Positive Model Bacterium Bacillus subtilis

Frontiers in Molecular Biosciences | www.frontiersin.org
132May 201910.3389/fmolb.2019.00032Specialty section: This article was submitted to Protein Chemistry and Enzymology, a section of the journal Frontiers in Molecular Biosciences Received: 13 March 2019 Accepted: 18 April 2019REVIEW Edited by: Reviewed by: *Correspondence: Citation: Richts B, Rosenberg J and Commichau FM (2019) A Survey of Pyridoxal 5 ′ -Phosphate-Dependent Proteins in the Gram-Positive Model Bacterium Bacillus subtilis. Front. Mol. Biosci. 6:32.vitamin B6PLP-omeamino transferasemetabolic engineeringtoxicity
The B6 vitamer pyridoxal 5 ′ -phosphate (PLP) is a co-factor for proteins and enzymes that are involved in diverse cellular processes. Therefore, PLP is essential for organisms from all kingdoms of life. Here we provide an overview about the PLP-dependent proteins from the Gram-positive soil bacterium Bacillus subtilis. Since B. subtilis serves as a model system in basic research and as a production host in industry, knowledge about the PLP-dependent proteins could facilitate engineering the bacteria for biotechnological applications. The survey revealed that the majority of the PLP-dependent proteins are involved in metabolic pathways like amino acid biosynthesis and degradation, biosynthesis of antibacterial compounds, utilization of nucleotides as well as in iron and carbon metabolism. Many PLP-dependent proteins participate in de novo synthesis of the co-factors biotin, folate, heme, and NAD + as well as in cell wall metabolism, tRNA modification, regulation of gene expression, sporulation, and biofilm formation. A surprisingly large group of PLP-dependent proteins (29%) belong to the group of poorly characterized proteins. This review underpins the need to characterize the PLP-dependent proteins of unknown function to fully understand the "PLP-ome" of B. subtilis.

## INTRODUCTION

The term "vitamin B6" collectively designates the vitamers pyridoxal (PL), pyridoxine (PN), and pyridoxamine (PM), and the respective phosphate esters pyridoxal 5 ′ -phosphate (PLP), pyridoxine 5 ′ -phosphate (PNP), and pyridoxamine 5 ′ -phosphate (PMP) (György, 1956;Rosenberg, 2012) ( Figure 1A). Since vitamin B6 is an essential micronutrient component in the diet of mammals, it is of commercial interest for the pharmaceutical and the food industry (Domke et al., 2005;Fitzpatrick et al., 2007Fitzpatrick et al., , 2010Eggersdorfer et al., 2012;Kraemer et al., 2012;Rosenberg et al., 2017;Acevedo-Rocha et al., 2019). As yet, the B6 vitamers are chemically synthesized via different routes (Pauling and Weimann, 1996;Kleemann et al., 2008;Eggersdorfer et al., 2012). Since chemical synthesis requires the usage of expensive and/or toxic chemicals, the shift from chemical synthesis to sustainable fermentation technologies using microorganisms is of great interest (Rosenberg et al., 2017;Acevedo-Rocha et al., 2019). So far, the microbial vitamin B6 production processes could not replace chemical production processes (Commichau et al., 2014(Commichau et al., , 2015Rosenberg et al., 2017Rosenberg et al., , 2018Acevedo-Rocha et al., 2019).

PLP is a co-factor for many proteins and enzymes (Jansonius, 1998;Christen and Mehta, 2001;Eliot and Kirsch, 2004;Phillips, 2015). About 1.5% of the genes of free-living prokaryotes encode PLP-dependent proteins and over 160 enzymes with different catalytic activities require vitamin B6 as a co-factor (about 4% of all described catalytic activities) Peracchi, 2003, 2009). Certainly, novel PLP-dependent proteins and enzymes will be identified and characterized in the future because the number of sequenced genomes is increasing (https://www.ncbi. nlm.nih.gov/genome/browse/#!/overview/). The majority of the PLP-dependent enzymes are involved in amino acid metabolism (John, 1995;Eliot and Kirsch, 2004). Some enzymes catalyzing decarboxylation and racemization reactions, cleavage of C a -C b bonds, α-elimination and replacement as well as β-and γ -elimination or replacement reactions also require PLP as a co-factor. Moreover, PMP and PM serves as co-factors for enzymes of deoxysugar and amino acid biosynthetic pathways, respectively (Burns et al., 1996;Mehta and Christen, 2000;Yoshikane et al., 2006;Romo and Liu, 2011). PLP also modulates the activity of DNA-binding transcription factors in eukaryotes and prokaryotes (Oka et al., 2001;Belitsky, 2004aBelitsky, , 2014Huq et al., 2007;El Qaidi et al., 2013;Tramonti et al., 2015Tramonti et al., , 2017Suvorova and Rodionov, 2016). Moreover, vitamin B6 is implicated in oxidative stress responses (Bilski et al., 2000;Mooney et al., 2009;Mooney and Hellmann, 2010;Vanderschuren et al., 2013;Moccand et al., 2014). Thus, vitamin B6 fulfills a variety of vital functions in different cellular processes (Parra et al., 2018).


## DE NOVO SYNTHESIS OF VITAMIN B6

Two pathways for de novo PLP synthesis are currently known ( Figure 1B) (Mittenhuber, 2001;Tanaka et al., 2005;Fitzpatrick et al., 2007Fitzpatrick et al., , 2010Rosenberg et al., 2017). The deoxyxylulose-5-phosphate (DXP)-dependent vitamin B6 biosynthesis pathway was identified in the Gram-negative model bacterium Escherichia coli and consists of two branches and seven enzymatic steps. The first three enzymes Epd, PdxB, and SerC of the longer branch convert a pentose phosphate pathway intermediate to 4-phosphohydroxy-L-threonine (4HTP) ( Figure 1B) Drewke et al., 1996;Boschi-Muller et al., 1997;Tazoe et al., 2006;Rudolph et al., 2010). Next, PdxA converts 4HTP to 2-amino-3-oxo-4-(phosphohydroxy)butyric acid, which undergoes spontaneous decarboxylation to 3phosphohydroxy-1-aminoacetone (Cane et al., 1998;Laber et al., 1999;Sivaraman et al., 2003). The PNP synthase PdxJ produces the B6 vitamer PNP from 3-phosphohydroxy-1-aminoacetone and DXP, of which the latter substrate is generated by the DXP synthase Dxs from glyceraldehyde 3-phosphate and pyruvate in the short branch of the DXP-dependent vitamin B6 pathway ( Figure 1B) (Takiff et al., 1992;Sprenger et al., 1997;Cane et al., 1999;Laber et al., 1999). The PNP oxidase PdxH catalyzes the final step yielding in the biologically most-relevant B6 vitamer PLP . The DXP-dependent vitamin B6 pathway is present in α-and γ-proteobacteria (Mittenhuber, 2001;Tanaka et al., 2005). Recently, it has been shown that bacteria possess promiscuous enzymes that may feed into the DXP-dependent pathway and bypass a block in pyridoxal-5 ′phosphate synthesis ( Figure 1B) Kim and Copley, 2012;Smirnov et al., 2012;Oberhardt et al., 2016;Thiaville et al., 2016;Zhang et al., 2016;Rosenberg et al., 2018). The hybrid pathways consisting of enzymes of native and nonnative vitamin B6 pathways and of promiscuous enzymes may be improved by metabolic engineering to enhance production of B6 vitamers (Rosenberg and Commichau, 2019).

The DXP-independent vitamin B6 biosynthetic pathway involves only the PdxST enzyme complex (Ehrenshaft and Daub, 2001;Belitsky, 2004b;Burns et al., 2005;Raschle et al., 2005;Strohmeier et al., 2006). PdxT is a glutaminase that hydrolyses glutamine into glutamate and ammonium, of which the latter serves as a substrate to the PLP synthase PdxS (Belitsky, 2004b). The PdxS subunit generates PLP from ammonium together with either ribulose 5-phosphate or ribose 5-phosphate and with either G3P or dihydroxyacetone phosphate ( Figure 1B). Many organisms possess a salvage pathway for the interconversion of the B6 vitamers ( Figure 1B) (Fitzpatrick et al., 2007;di Salvo et al., 2011). For instance, specific B6-vitamer kinases can phosphorylate PN, PM and PL into their respective phosphate esters (White and Dempsey, 1970;Yang et al., 1996Yang et al., , 1998di Salvo et al., 2004;Park et al., 2004). Those organisms carrying only salvage pathways have to take up B6 vitamers. So far, only few vitamin B6 transporters have been in described in eukaryotes (Stolz and Vielreicher, 2003;Szydlowski et al., 2013). The accumulation of PLP to toxic levels can be prevented by dephosphorylation and export of PL. Indeed, bacteria like E. coli and Sinorhizobium meliloti synthesize a phosphatase for dephosphorylation of PNP and PLP (Tazoe et al., 2005;Nagahashi et al., 2008;Sugimoto et al., 2017).


## VITAMIN B6 METABOLISM IN BACILLUS SUBTILIS

The Gram-positive model bacterium Bacillus subtilis relies on the PdxST enzyme complex to synthesize the B6 vitamer PLP (Pflug and Lingens, 1978;Sakai et al., 2002b;Belitsky, 2004b;Burns et al., 2005). The B. subtilis PdxST enzyme complex has been biochemically and structurally studied (Raschle et al., 2005;Zhu et al., 2005;Strohmeier et al., 2006;Wallner et al., 2009;Smith et al., 2015). In many Gram-positive bacteria, the expression of the pdxST genes and the genes encoding kinases that phosphorylate the B6 vitamers PM, PN, and PL, are regulated by MocR-like DNA-binding transcription factors (Jochmann et al., 2011;El Qaidi et al., 2013;Belitsky, 2014;Liao et al., 2015;Tramonti et al., 2015;Suvorova and Rodionov, 2016). In B. subtilis the pdxST genes are not subject to transcriptional regulation (Nicolas et al., 2012).

B. subtilis must possess an uptake system and a kinase for the B6 vitamer PL because exogenously supplied PL relieves PLP auxotrophy of a pdxST mutant (Belitsky, 2004b;Commichau et al., 2014). The uptake system and the kinase are not specific for PL because the overexpression of the E. coli pdxH PN(P) oxidase gene in a B. subtilis pdxST mutant enabled the FIGURE 1 | (A) The B6 vitamers: pyridoxal (PL), pyridoxal 5 ′ -phosphate (PLP), pyridoxine (PN), pyridoxine 5 ′ -phosphate (PNP), pyridoxamine (PM), and pyridoxamine 5 ′ -phosphate (PMP). (B) The deoxyxylulose 5-phosphate (DXP)-dependent and DXP-independent vitamin B6 biosynthetic routes and the salvage pathway for the interconversion of the B6 vitamers. Epd, erythrose 4-phosphate dehydrogenase; PdxB (PdxR), 4-phosphoerythronate dehydrogenase; SerC, 3-phosphoserine aminotransferase; PdxA, 4-phosphohydroxy-L-threonine dehydrogenase; PdxJ, PNP synthase; Dxs, 1-deoxyxylulose 5-phosphate synthase; PdxH, PNP oxidase; PdxS (PLP synthase subunit), and PdxT (glutaminase subunit) form the PLP synthase complex; PdxK, PL kinase present in B. subtilis and E. coli; PdxY, PL kinase present in E. coli. PdxK from B. subtilis has PN, PL, and PM kinase activity (see text). E4P, erythrose 4-phosphate; 4PE, 4-phosphoerythronate; OHPB, 2-oxo-3-hydroxy-4-phosphobutanoate; 4HTP, 4-hydroxy-threonine; AOPB, 2-amino-3-oxo-4-(phosphohydroxyl)-butyrate; PHA, 3-phosphohydroxy-1-aminoacetone; DXP, deoxyxylulose-5-phosphate; G3P, glyceraldehyde-3-phosphate; DHAP, dihydroxyacetonephosphate; R5P, Ribose-5-phosphate. Red arrows indicate the steps where promiscuous enzymes may feed into the DXP-dependent and DXP-independent vitamin B6 biosynthetic pathways Oberhardt et al., 2016;Thiaville et al., 2016;Rosenberg et al., 2018). bacteria to synthesize PLP from exogenous PN (Commichau et al., 2014). While the PL and PN uptake system remains to be identified in B. subtilis, the kinase phosphorylating the B6 vitamers PL, PM, and PN is known (Park et al., 2004;Newman et al., 2006a,b). It is interesting to note that the B. subtilis PL kinase PdxK is phylogenetically related to HMPP ribokinases converting 4-amino-5-hydroxymethyl-2-methylpyrimidine (HMP) to 4-amino-5-hydroxymethyl-2methylpyrimidine phosphate (HMPP) and HMPP to HMPP phosphate, a precursor of thiamine biosynthesis (Mizote et al., 1999;Newman et al., 2006b). In the future, it will be interesting to elucidate whether exogenously supplied PL controls the activity of the PdxST enzyme complex in B. subtilis to prevent the accumulation of PLP to toxic levels (see below).

In contrast to the enzymes of the DXP-dependent vitamin B6 pathway from E. coli, the PdxST enzyme complex from B. subtilis is rather slow (Rosenberg et al., 2017). Therefore, a heterologous DXP-dependent vitamin B6 pathway has been introduced into B. subtilis for producing the B6 vitamer PN (Commichau et al., 2014(Commichau et al., , 2015. The fact that the engineered B. subtilis strains synthesized significant amounts of PN, which was detectable in the culture supernatant, suggests that the bacteria might possess a PNP phosphatase and an export system for PN. Recently, the PLP phosphatase YbhA has been identified in E. coli (Sugimoto et al., 2017). YbhA shows about 31% overall sequence identity with the YitU protein from B. subtilis. Even though it has been shown that YitU is a HAD phosphatase having a minor activity in dephosphorylating the riboflavin precursor 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione 5 ′ -phosphate (Sarge et al., 2015), it will be interesting to test whether the protein may act as a PNP/PLP phosphatase.


## TOXICITY OF VITAMIN B6 AND PATHWAY INTERMEDIATES

PLP can be toxic for the cell because the reactive 4 ′ -aldehyde moiety of the B6 vitamer forms covalent adducts with other compounds and PLP-independent proteins containing thiol or amino groups. For instance, PLP was shown to inhibit enzymes that are involved in DNA metabolism and in central carbon metabolism in eukaryotes (Mizushina et al., 2003;Vermeersch et al., 2004;Lee et al., 2005). Moreover, the modification of the E. coli initiation factor 3, the adenylsuccinate synthetase and the PL kinase by PLP results in activity loss (Ohsawa and Gualerzi, 1981;Dong and Fromm, 1990;Ghatge et al., 2012). Recently, it has been shown that the addition of vitamin B6 to the E. coli wild type strain BW25113 and an E. coli mutant strain lacking the ZipA cell division protein affects multiple metabolic pathways, which are involved in amino acid biosynthesis (Vega and Margolin, 2017). Thus, excess of PLP affects different cellular processes. PLP is also prone to damage either due to side reactions that are catalyzed by promiscuous enzymes or due to spontaneous chemical reactions (Linster et al., 2013). In fact, the B6 vitamers PLP and PMP were identified as members of the 30 most damage-prone metabolites (Lerma-Ortiz et al., 2016). However, given the fact that PLP is required for optimal growth in little amounts, the essential cofactor can be synthesized at a minimal necessary rate (Hartl et al., 2017). The low requirement of PLP and its low cellular concentration prevent perturbation of other essential processes in the cell.

In the past years, several attempts have been made to engineer bacteria for the overproduction of the B6 vitamers PL and PN (Rosenberg et al., 2017). In contrast to PL(P), PN(P) is less toxic for living cells (see above; Commichau et al., 2014). Therefore, the DXP-dependent seems to be a promising pathway for engineering bacteria for vitamin B6 overproduction. However, it has also been shown that intermediates of the DXP-dependent pathway can be highly toxic for bacteria. For instance, the erythrose-4-phosphate dehydrogenase Epd generates the DXP-dependent vitamin B6 pathway intermediate 4-phosphoerythronate (4PE), which is required in low amounts for PLP biosynthesis, is toxic for the cells when overproduced (Sachla and Helmann, 2019). Eukaryotic cells do have a phosphatase that hydrolyzes and detoxifies 4PE that is also mistakenly generated by the glycolytic enzyme glyceraldehyde-3-phosphate dehydrogenase (Beaudoin and Hanson, 2016;Collard et al., 2016). 4PE inihibits the 6-phosphogluconate dehydrogenase from the pentosephosphate pathway (Collard et al., 2016). Recently, it has been demonstrated that 4PE also inhibits the B. subtilis 6-phosphogluconate dehydrogenase GndA (Sachla and Helmann, 2019). In this organism, 4PE is detoxified by the GTPase CpgA, which is a checkpoint protein known to be involved in ribosome assembly (Campbell et al., 2005). It will be interesting to assess whether 4PE also inhibits the 6-phosphogluconate dehydrogenase in E. coli because the bacterium does not possess a CpgA homolog. However, the accumulation of 4PE to toxic levels does not seem to be problematic in E. coli because 4PE can be produced in only small amounts that are sufficient for de novo synthesis of PLP. The intermediate 4HTP from the DXP-dependent vitamin B6 pathway is also inhibits bacterial growth. 4HTP interferes with biosynthesis of threonine and isoleucine in E. coli and B. subtilis (Drewke et al., 1993;Farrington et al., 1993;Commichau et al., 2014Commichau et al., , 2015Rosenberg et al., 2016). The toxicity of the pathway intermediates may explain why it is difficult to engineer bacteria that stably express the genes of the DXP-dependent vitamin B6 pathway (Commichau et al., 2014). The understanding of the metabolite toxicity is crucial for the rational design and engineering of bacteria overproducing PN at commercially attractive levels. Moreover, the knowledge about the functions of target proteins of PLP is very important to understand how the B6 vitamer affects cellular metabolism upon overproduction.


## PLP-DEPENDENT PROTEINS AND ENZYMES INVOLVED IN VITAMIN B6 METABOLISM IN BACILLUS SUBTILIS

To identify the B. subtilis proteins and enzymes that require PLP for activity and are involved in vitamin B6 metabolism, we compared the Enzyme Commission (E.C.) numbers of the proteins from the B. subtilis 168 laboratory strain found in the SubtiWiki database (http://subtiwiki.uni-goettingen.de/v3/) (Zhu and Stülke, 2017) with the E.C. numbers that are deposited in the B6 database (Table 1) (http://bioinformatics.unipr.it/cgi-bin/ bioinformatics/B6db/home.pl) (Percudani and Peracchi, 2009). We also describe proteins from the SubtiWiki database that are specific for B. subtilis and are therefore not present in the B6 database. Publications describing proteins involved in vitamin B6 metabolism in B. subtilis were also added to the Table. A recent mass spectrometry approach in combination with modified pyridoxal analogs identified proteins in the Grampositive pathogen Staphylococcus aureus that probably depend on the B6 vitamer PLP (Hoegl et al., 2018). The study confirmed the binding of PLP to proteins of known and unknown function and identified 4 additional PLP-binding proteins (HemH, HemQ, YtoP, and YwlG) (see below). In total we ended up with 65 PLP-dependent proteins in B. subtilis, of which 61 proteins are bona fide PLP-dependent proteins. The PLP-dependency of four proteins remains to be experimentally validated. Table 1 also contains the PDB identifiers of structures that are available in the PDB database for the B. subtilis proteins. In case the structural information was not available, we have added the PDB identifiers from PLP-dependent homologs showing more than 27% overall sequence identity. We have also included information about the physiological functions of the proteins and their paralogs, the transcription factors that are involved in synthesis of the proteins and information about the sequence similarities with other proteins from the UniProt database (https://www.uniprot. org). The list of proteins involved in vitamin B6 metabolism in B. subtilis will certainly be extended in the future because PLPdependent enzymes are ubiquitous and evolutionary diverse, making their classification based on sequence homology difficult.


## FUNCTIONAL ASSIGNMENT OF KNOWN PLP-DEPENDENT PROTEINS IN B. SUBTILIS

Most of the proteins that require PLP in B. subtilis are metabolic enzymes, of which the majority is involved in anabolism and catabolism of proteinogenic and non-proteinogenic amino acids (Table 1; Figure 2). The enzymes can be assigned to known protein families of PLP-dependent enzymes and for most of them it has been shown that they are indeed active in amino acid metabolism (Mehta et al., 1993;Mehta and Christen, 2000) (Table 1). B. subtilis also possesses a PLPdependent 2-amino-3-ketobutyrate CoA ligase (Kbl), which could be involved in threonine utilization together with the Lthreonine dehydrogenase Tdh (Schmidt et al., 2001;Reitzer, 2005). Both enzymes are encoded in the bicistronic tdh-kbl operon (Nicolas et al., 2012). The regulation of the tdh and kbl genes and the catalytic activities of the Tdh and Kbl enzymes remain to be studied. Two PLP-dependent enzymes BacF and NtdA are involved in the synthesis of bacilysin and kanosamine in B. subtilis. Bacilysin is a non-ribosomally synthesized peptide that is active against various bacteria and some fungi (Inaoka et al., 2003(Inaoka et al., , 2009Karatas et al., 2003;Köroglu et al., 2011). Kanosamine is an antibiotic, which is produced by Bacillus A large-scale mass spectrometry-based screen revealed that the proteins probably bind PLP in S. aureus (Hoegl et al., 2018). It has to be experimentally validated that HemH, HemQ, YtoP and YwlG are functional PLP-dependent proteins.

Frontiers in Molecular Biosciences | www.frontiersin.org  Table 1 for more information.

and Streptomyces species and inhibits cell wall synthesis in microorganisms (Dolak et al., 1980;Milner et al., 1996;Inaoka et al., 2004;Inaoka and Ochi, 2011;Vetter et al., 2013). The PLP-dependent enzymes SufS, GlgP, and PucG are involved in iron-sulfur cluster formation, glycogen biosynthesis, and purine utilization, respectively ( Table 1). Homologs of SufS and GlgP are also present in E. coli (48 and 44% overall sequence identity, respectively). However, in B. subtilis the glycogen phosphorylase seems to be involved in a sporulation-specific process because the glgP gene is expressed early during sporulation in the mother cell (Kiel et al., 1994). Glycogen biosynthesis exclusively occurs in the presence of carbon sources allowing efficient sporulation (Kiel et al., 1994). Eight PLP-dependent enzymes are involved in the biosynthesis of the co-factors biotin, folate, heme and NAD (Table 1). While the biochemical and structural characterization of BioA, BioF, PabC, GsaB, HemL, and NifS revealed that the proteins require PLP for enzyme activity, it has to be investigated whether HemH and HemQ are bona fide PLP-dependent proteins. HemH and HemQ were recently identified in a mass spectrometry approach in the Gram-positive pathogen Staphylococcus aureus (Hoegl et al., 2018). B. subtilis possesses in total five PLP-dependent enzymes (Alr, Dat, PatA, NifZ, and YrvO) that are involved in cell wall metabolism and in information processing ( Table 1). The alanine racemase Alr, the D-alanine aminotransferase Dat and the N-acetyl-L,Ldiaminopimelate aminotransferase PatA generate precursors for the peptidoglycan of the cell wall. The tRNA-modifing enzymes NifZ and YrvO are both cysteine desulfurases that are active in biosynthesis of 4-thiouridine and 2-thiouridine, respectively, for the formation of modified tRNA molecules. YrvO transfers sulfur to the TrmU tRNA methyltransferase, which is essential for 2thiouridine biosynthesis (Black and Dos Santos, 2015). Finally, four PLP-dependent enzymes play a role in sporulation and biofilm formation in B. subtilis (Table 1). While it has been shown that Sps proteins such as SpsC are required for spore germination (Cangiano et al., 2014), the role of the alanine racemase YncD in sporulation is currently unkown. The two biofilm-related enzymes, the arginine decarboxylases SpeA and the UDP-2,6dideoxy-2-acetamido-4-keto-glucose aminotransferase EpsN are important for biosynthesis of polyamines such as spermidine and extracellular polysaccharides (Burrell et al., 2010;Marvasi et al., 2010). Indeed, B. subtilis strains lacking either SpeA or EpsN are defective in biofilm formation (Burrell et al., 2010;Pozsgai et al., 2012). SpeA also possesses a paralog (YaaO, 34% overall sequence identity), However, this proteins does not seem to be involved in biofilm formation (see below). To conclude, B. subtilis possesses several PLP-dependent enzymes that are involved in different cellular processes. Moreover, many PLP-dependent enzymes do have paralogs that have similar activities or fulfill specific functions in the cell, probably due to specialization during evolution (Table 1).


## PLP-DEPENDENT TRANSCRIPTION FACTORS IN BACILLUS SUBTILIS

B. subtilis possesses seven PLP-dependent DNA-binding transcription factors of which only one has been intensively characterized (see below; Table 1). The PLP-dependent transcription factors belong to the MocR-subfamily and contain a GntR-family DNA-binding domain at the N-terminus and an aminotransferase-like sensory domain at the C-terminus (Bramucci et al., 2011;Milano et al., 2015;Tramonti et al., 2015Tramonti et al., , 2017Suvorova and Rodionov, 2016). The MocR-familytype PLP-dependent transcription factors that have been characterized are involved in controlling the expression of genes involved in PLP, γ-aminobutyrate, ecotoine, and taurine metabolism (Suvorova and Rodionov, 2016;Schulz et al., 2017;Tramonti et al., 2018). B. subtilis can utilize γ-aminobutyrate (GABA) as a source of nitrogen (Belitsky and Sonenshein, 2002). The catabolism of GABA requires the activities of the GABA aminotransferase GabT and the succinic semi-aldehyde dehydrogenase GabD that are encoded in the bicistronic gabT-gabD operon. The MocR-family-type regulator GabR activates the transcription of the gabT and gabD genes in a PLPand GABA-dependent manner (Belitsky and Sonenshein, 2002;Belitsky, 2004a). The protein also binds to the gabR promoter and represses gabR transcription in the absence of GABA (Belitsky and Sonenshein, 2002;Edayathumangalam et al., 2013). Structual analysis of GabR revealed that the aminotransferase-like activity at the C-terminus of the protein is not essential for its function as a transcription regulator (Edayathumangalam et al., 2013). It has also been shown that the formation of an external aldimine between GABA and PLP causes a conformational transition from the open form to a closed form in the aminotransferase domain of GabR, triggering transcription activation of the gabT-gabD operon (Okuda et al., 2015a;Milano et al., 2017;Park et al., 2017;Wu et al., 2017). Moreover, the dimerization of the C-terminal domain of GabR enables the N-terminal domain to bind to DNA and the N-terminal domain controls the binding specificity of the effector domain (Okuda et al., 2015b). The analysis of the interaction between GabR and DNA revealed that two cognate binding sites and the bendability of the interjacent DNA sequence are important for transcription factor binding (Al-Zyoud et al., 2016;Amidani et al., 2017). To conclude, the biochemical and structural characterization of the B. subtilis GabR transcriptional regulator has uncovered mechanistic insights into the MocRfamily-type PLP-dependent transcription factors and provides the basis for the characterization of the remaining six putative PLP-dependent DNA-binding transcription factors YcxD, YdeF, YdeL, YdfD, YhdI, and YisV from B. subtilis. Their characterization will presumably uncover novel metabolic pathways and transport systems (Suvorova and Rodionov, 2016).


## POORLY CHARACTERIZED PLP-DEPENDENT ENZYMES

In addition to the 6 transcription factors whose DNA-binding activities depend on PLP and additional unknown effectors (see above), B. subtilis possesses several poorly characterized PLPdependent enzymes (Table 1; Figure 2). The DsdA protein from B. subtilis shows about 58% overalls sequence identity with the E. coli D-serine deaminase (D-serine ammonia lyase) DsdA, which catalyzes the deamination of D-serine to form pyruvate and ammonia (Gale and Stephenson, 1938;McFall, 1964). The PLP-dependency and the structure of the enzyme from E. coli have been determined (Schnackerz et al., 1999;Urusova et al., 2012). Phylogenetic analyses suggest that the E. coli and B. subtilis D-serine deaminases and threonine synthases with similarities in the catalytic mechanisms may have evolved from a common ancestor (Parsot, 1986). The primary function of DsdA seems to be the detoxification of D-serine, which inhibits bacterial growth because it is a competitive antagonist of β-alanine in the pantothenate (vitamin B5) biosynthetic pathway, generating the precursor for coenzyme A biosynthesis (Cosloy and McFall, 1973). Previously, it has also been shown that E. coli mutants constitutively expressing DsdA are able to use D-serine as the sole source of carbon and nitrogen (Bloom and McFall, 1975). In B. subtlilis the dsdA gene is located in the yqjP-yqjQ-dsdA-coaA-yqjT operon, containing three genes of unknown function as well as the dsdA and coaA genes, of which the latter encodes the major pantothenate kinase CoaA (Ogata et al., 2014). The genetic context of the dsdA gene strongly suggests that the D-serine deaminase may be involved in the detoxification of D-serine that probably also interferes with pantothenate synthesis in B. subtilis. The presence of a D-serine deaminase may be explained by the fact that L-serine is more rapidly racemized than most other amino acids (Reitzer, 2005). It will be interesting to elucidate whether overproduction of DsdA allows B. subtilis to grow with D-serine as the sole source of carbon and nitrogen and whether DsdA is involved in the detoxification of D-serine. The B. subtilis KamA enzyme shows about 60% overalls sequence identity with the PLP-, S-adenosyl-L-methionine and [4Fe-4S]-dependent lysine-2,3-aminomutase from the obligate anaerobe bacterium Clostridium subterminale ( Table 1) (Lepore et al., 2005). The lysine-2,3-aminomutase catalyzes the converstion of L-lysine to L-β-lysine, which is the first step in the anaerobic degradation of lysine in clostridia (Chirpich et al., 1970). In vitro characterization of KamA from B. subtilis revealed that enzyme also catalyzes the conversion of L-lysine to L-βlysine under anaerobic conditions (Chen et al., 2000). The KamA enzyme is only produced during sporulation of B. subtilis (Feucht et al., 2003). Therefore, the enzyme does not seem to play a role during vegetative growth. The lysine-2,3-aminomutase EpmB from E. coli, which shows about 31% overalls sequence identity with KamA from B. subtilis, has low lysine-2,3-aminomutase activity, indicating that L-lysine does not seem to be the natural substrate (Chen et al., 2000;Yanagisawa et al., 2010). Recently, it has been shown that the E. coli lysine-2,3-aminomutase EpmB enhances the lysylation of the elongation factor EF-P by the aminoacyl-tRNA synthetase GenX (Yanagisawa et al., 2010). The lysylation of EF-P is a post-translational modification that is essential for cell survival (Yanagisawa et al., 2010;Park et al., 2012). However, the physiological function of the lysine-2,3-aminomutase KamA from B. subtilis remains to be determined.

The B. subtilis YaaO enzyme, which belongs to class I Orn/Lys/Arg decarboxylases, encodes a putative arginine decarboxylase ( Table 1). Arginine decarboxylases are important for biosynthesis of polyamines such as spermidine, substances that are crucial for biofilm formation (Burrell et al., 2010). The B. subtilis arginine decarboxylase SpeA, which can be considered as a paralog of YaaO (34% overall sequence identity), is indeed essential for the synthesis of polyamines and thus biofilm formation (see above) (Burrell et al., 2010). However, no biofilmrelated phenotype has been reported so far for a B. subtilis mutant lacking YaaO. Recently, it has been reported that YaaO and two other proteins of unknown function (YfkA and YwlG, see below) influence the level of the post-translational aminopentanol modification of the elongation factor EF-P (Witzky et al., 2018). However, the precise role of YaaO in the modification of EF-P in B. subtilis remains to be elucidated.

The uncharacterized PLP-dependent proteins YcbU, AlaT, YhxA, and its paralog YodT are probably PLP-dependent amino acid transferases (Table 1). YcbU might be a cysteine desulfurase that is involved in co-factor biosynthesis (Mihara and Esaki, 2002). However, it remains to be elucidated whether YcbU is functional in B. subtilis because a mutant lacking YcbU shows no obvious phenotype (Koo et al., 2017). AlaT is similar to PLP-dependent methionine amino acid transferases and the protein shares about 47% overall sequence identity with an amino acid transferase from Pyrococcus horikoshii that acts on aromatic amino acids (Matsui et al., 2000). However, not experimental evidence supporting the annotation of AlaT is available. Both, YhxA and YodT are annotated as putative adenosylmethionine-8-amino-7-oxononanoate aminotransferases, enzymes that were shown to be involved in biotin biosynthesis in bacteria (Izumi et al., 1975). YhxA shares about 35 and 33% sequence identity with YodT and BioA, respectively. The PLP-dependent lysine-8-amino-7-oxononanoate aminotransferase BioA is required for biotin biosynthesis in B. subtilis (Table 1). Therefore, it is tempting to speculate that YhxA and YodT are also involved in biotin metabolism in this organism. The expression of the yhxA and yodT genes depends on SigA and on the sporulationspecific sigma factor SigE, respectively. Therefore, these enzymes seem to be active in different cellular differentiation processes of B. subtilis.

The B. subtilis YlmE protein of unknown function shows about 33% overall sequence identity with the YggS protein from E. coli. Recently, it has been shown that YggS is a PLP-binding protein, which belongs to a highly conserved COG0325 protein family and exists in almost all kingdoms of life, including bacteria, fungi and animals (Ito et al., 2013). The high conservation of YggS indicates that the protein fulfills an important function in bacteria. Indeed, the lack of YggS in E. coli affects balance of PLP homeostasis, sensitivity toward the B6 vitamer PN and perturbation of biosynthesis of branched-chain amino acids . Similar phenotypes have been associated to a mutant strain of Synechococcus elongatus PCC 7942 lacking the pipY gene, which encodes a COG0325 homolog (Labella et al., 2017;Tremiño et al., 2017). It will be very interesting to elucidate the precise function of COG0325 homologs in controlling vitamin B6 homeostasis.

The proteins YhdR, YnbB, and YwlG cannot be assigned to a specific protein family (Table 1). YhdR shares 34% overall sequence identity with an amino acid transferase from Eubacterium rectale but its role in amino acid metabolism is unknown (Table 1). Interestingly, like YaaO, YnbB, and YwlG are involved in the post-translational aminopentanol modification of the elongation factor EF-P (Witzky et al., 2018). While YwlG influences the level of the post-translational modification, YnbB seems to be required for the modification. The precise function of the proteins in controlling the activity of the elongation factor EF-P in B. subtilis needs further investigation. Moreover, it remains to be experimentally determined whether the function of YwlG depends on PLP.

The YtkP protein is a putative cysteine synthase that shares sequence similarity with the bifunctional cysteine synthase CysK and the O-acetylserine-thiol lyase MccA (Table 1). Since a cysK mccA double mutant is auxotrophic for cysteine, YtkP does not seem to be involved in cysteine biosynthesis. Thus, the function of YtkP remains elusive. The YtoP protein has been annotated as a putative glutamyl aminopeptidase because it shares about 44% overall sequence identity with PepA, a protease from Streptococcus pneumoniae that has been structurally and biochemically analyzed . Interestingly, YsdC, the paralog of YtoP (44% overall sequence identity), is annotated as an endo-1,4-β-glucanase (Table 1). Therefore, it is tempting to speculate whether YtoP is indeed involved in protein turnover. The binding of PLP to YtoP has to be experimentally validated. To conclude, B. subtilis contains several poorly characterized PLP-dependent proteins, which need to be studied in the future.


## CONCLUSIONS AND FUTURE PERSPECTIVES

For a complete understanding of the vitamin B6 metabolism of B. subtilis it is crucial to identify and characterize all the proteins that require the essential co-factor to fulfill their function. However, even for well-studied model bacteria like B. subtilis the complete set of the enzymes involved in vitamin B6 metabolism and the PLP-dependent proteins remains to be identified. Several bioinformatics-driven approaches have been performed to identify and classify PLP-dependent enzymes Peracchi, 2003, 2009). Even though the PLP-dependent proteins often show low sequence similarities, using sensitive Hidden Markov Model-base sequence similarity searches PLP-dependent proteins can be identified (Yoon, 2009). However, in case the protein has a fold that is different from the known PLPdependent fold it is difficult to identify novel PLP-dependent proteins by sequence comparison. Even though structural similarity searches allowed assigning the PLP-dependent proteins to five distinct fold types (Mehta et al., 1993;Mehta and Christen, 2000;Catazaro et al., 2014), other approaches have to be pursued to uncover the full repertoire of PLP-dependent enzymes in a given organism. Indeed, mass spectrometry and biochemical approaches have been performed to identify proteins that were modified by PLP (Simon and Allison, 2009;Whittaker et al., 2015;Wu et al., 2018). As described above, a recent mass spectrometry approach identified proteins in the Gram-positive pathogen S. aureus that might depend on the B6 vitamer PLP (Hoegl et al., 2018). It will be interesting to evaluate whether the same approach will lead to the identification of novel PLPdependent proteins in B. subtilis and related bacteria. Moreover, the transport systems for the uptake and export of the B6 vitamers PN and PL have to be identified by B. subtilis. The phosphatase involved in the dephosphorylation of PNP is so far unknown (Commichau et al., 2014(Commichau et al., , 2015. Furthermore, the function of the conserved YlmE protein (YggS in E. coli) in vitamin B6 homeostasis has to be studied. Finally, it has to be elucidated how the PLP molecules are delivered to their target proteins.


## SPECIALITY SECTION

PLP-Dependent Enzymes: Extraordinary Versatile Catalysts and Ideal Biotechnological Tools for the Production of Unnatural Amino Acids and Related Compounds, in Process and Industrial Biotechnology, a section of the journal Frontiers in Bioengineering and Biotechnology.


## AUTHOR CONTRIBUTIONS

BR, JR, and FC performed the database search. FC coordinated the work and wrote the manuscript with input from all authors.


## FUNDING

This work was supported by the Fonds der Chemischen Industrie (to FC), the Max-Buchner-Forschungsstiftung (MBFSSt 3381 to FC), and the Deutsche Forschungsgemeinschaft (DFG grants CO 1139/1-2 and GSC 226/2 to FC and JR, respectively). This project has received funding from the European Union's Horizon 2020 research and innovation programme under grant agreement No 720776.


## ACKNOWLEDGMENTS

We are grateful to members of the Commichau laboratory for fruitful comments and suggestions. We also acknowledge the foundational work on the metabolism of vitamin B6 in bacteria and other organisms that was published beforehand and cannot be fully covered because of the limit of references.

## FIGURE 2 |
2Functional distribution of PLP-dependent proteins in B. subtilis. See

## TABLE 1 |
1PLP-dependent proteins in B. subtilis. Protein BSU no.Kalcheva et al., 1997;Steil et al., 2005 Frontiers in Molecular Biosciences | www.frontiersin.orgThe overall amino acid sequence identity to other proteins from B. subtilis is shown in brackets.Essential a 
E.C. no. 

PDB no. b 

Function 

Pathway 

Regulation 
Paralogs/Protein familiy c 

References 

METABOLISM 

Amino acid biosynthesis 
ArgD 

BSU11220 

No 

2.6.1.11 
2EH6 (Aquifex aeolicus, 
45%) 

Acetylornithine 
aminotransferase 

Arginine biosynthesis 

SigA, 
AhrC (-), 
CodY (-), 
YlxR (-) 
PLP AAT class III family AAT, 
ArgD subfamily 

Czaplewski et al., 

1992; Brinsmade et al., 
2014; Ogura and 
Kanesaki, 2018 

AspB 

BSU22370 

No 

2.6.1.1 
1J32 (Phosmidium 
lapideum, 

45%) 
Aspartate aminotransferase 

Aspartate biosynthesis 

Unknown 
AlaT (41%), PatA (41%)/PLP 
AAT class I family 

Zhao et al., 2018 

CysK 

BSU00730 

No 

2.5.1.47 
1Y7L (Haemophilus 
influenzae, 

39%) 
Cysteine synthase, control 

of CymR activity 

Cysteine biosynthesis 

SigA, SigM, 
Spx (+) 
YtkP (59%), MccA 
(45%)/cysteine synthase, 
cystathione β-synthase 

family 

Tanous et al., 2008 

GlyA 

BSU36900 

No 

2.1.2.1 
2VI8 (Geobacillus 
stearothermophilus, 

80%) 

Serine hydroxymethyl-
transferase 

Glycine biosynthesis 

SigA, T-Box, 
PurR (-) 

SHMT family 

Gutiérrez-Preciado 

et al., 2009 

HisC 

BSU22620 

No 

2.6.1.9 
3FFH (Listeria innocua, 
52%) 

Histidinol-phosphate 
aminotransferase 

Aromatic amino acids 
Biosynthesis 

MtrB (-) 
PLP AAT class II family 

Nester and Montoya, 
1976; Babitzke et al., 
1992 

IlvA 

BSU21770 

No 

4.3.1.19 
1TDJ (E. coli, 

39%) 
Threonine dehydratase 

Branched-chain amino acid 

biosynthesis 

CodY (-) 
Ser/Thr dehydratase familiy 

Molle et al., 2003a; 
Rosenberg et al., 2016 

LysA 

BSU23380 

No 

4.1.1.20 
1HKW (Mycobacterium 
tuberculosis, 

42%) 

Diaminopimelate 
decarboxylase 

Lysine biosynthesis 

SigG, SpoVT 
(+) 
Orn/Lys/Arg decarboxylase 

class II family 

MccA 

BSU27260 

No 

-

4QL4 (Bacillus 
anthracis, 

63%) 

O-Acetylserine-thiol-lyase 

Methionine/cysteine 

conversion 

SigA, Spx (+), 
CymR (-) 
CysK (45%), YtkP 
(42%)/cysteine synthase, 
cystathione β-synthase 

family 

Nakano et al., 2003; 
Choi et al., 2006; Even 
et al., 2006 

MccB 

BSU27250 

No 

4.4.1.1 
4L0O (Helicobacter 
pylori, 

61%) 

Cystathionine lyase/ 
homocysteine -lyase 

Methionine/ cysteine 
conversion 

SigA, Spx (+), 
CymR (-) 
MetC (52%), MetI 
(48%)/Trans-sulfuration 

enzyme family 

Nakano et al., 2003; 
Choi et al., 2006; Even 
et al., 2006 

MetC 

BSU11880 

No 

4.4.1.8 
4L0O (H. pylori, 

51%) 
Cystathionine β-lyase 

Methionine biosynthesis 

SigA, S-box 
MccB (52%), MetI 
(43%)/trans-sulfuration 

enzyme family 

Grundy and Henkin, 
1998; Auger et al., 
2002; Tomsic et al., 
2008 

MetI 

BSU11870 

No 

-

4L0O (H. pylori, 

48%) 

O-Succinyl-homoserine 

lyase 

Methionine biosynthesis 

SigA, S-box 
MccB (48%), MetC 
(43%)/trans-sulfuration 

enzyme family 

Grundy and Henkin, 
1998; Auger et al., 
2002; Tomsic et al., 
2008 

MtnE 

BSU13580 

No 

-
2O1B (Staphylococcus 
aureus, 

42%) 
Glutamine transaminase 

Methionine salvage 

SigA, 
unknown 
BacF (50%)/ PLP AAT class 

I family 

Sekowska and 
Danchin, 2002; Berger 

et al., 2003 

PatB 

BSU31440 

No 

4.4.1.8 
3T32 (B. anthracis, 
46%) 

Cystathionine β-lyase 

Methionine biosynthesis 

Unknown 
PLP AAT class II family 

Auger et al., 2005 

SerC 

BSU10020 

No 

2.6.1.52 
1W23 (Bacillus 
alcalophilus, 

59%) 

3-Phosphoserine 
aminotransferase 

Serine biosynthesis 

Unknown 
PLP AAT class V family 

Sakai et al., 2002a 

(Continued) 

Frontiers in Molecular Biosciences | www.frontiersin.org 
May 2019 | Volume 6 | Article 32
Frontiers in Molecular Biosciences | www.frontiersin.org
Conflict of Interest Statement:The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.Copyright © 2019 Richts,Rosenberg and Commichau.This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.
Microbial cell factories for the sustainable manufacturing of B vitaminas. C G Acevedo-Rocha, L Gronenberg, M Mack, F M Commichau, H J Genee, 10.1016/j.copbio.2018.07.006Curr. Opin. Biotechnol. 56Acevedo-Rocha, C. G., Gronenberg, L., Mack, M., Commichau, F. M., and Genee, H. J. (2019). Microbial cell factories for the sustainable manufacturing of B vitaminas. Curr. Opin. Biotechnol. 56, 18-29. doi: 10.1016/j.copbio.2018.07.006

SufU is an essential iron-sulfur cluster scaffold protein in Bacillus subtilis. A G Albrecht, D J Netz, M Miethke, A J Pierik, F Burghaus, R Lill, 10.1128/JB.01536-09J. Bacteriol. 192Albrecht, A. G., Netz, D. J., Miethke, M., Pierik, A. J., Burghaus, F., Lill, R., et al. (2010). SufU is an essential iron-sulfur cluster scaffold protein in Bacillus subtilis. J. Bacteriol. 192, 1643-1651. doi: 10.1128/JB. 01536-09

Binding of transcription factor GabR to DNA requires recognition of DNA shape at a location distinct from its cognate binding site. W A Al-Zyoud, R M Hynson, L A Ganuelas, A C Coster, A P Duff, M A Baker, 10.1093/nar/gkv1466Nucleic Acids Res. 44Al-Zyoud, W. A., Hynson, R. M., Ganuelas, L. A., Coster, A. C., Duff, A. P., Baker, M. A., et al. (2016). Binding of transcription factor GabR to DNA requires recognition of DNA shape at a location distinct from its cognate binding site. Nucleic Acids Res. 44, 1411-1420. doi: 10.1093/nar/ gkv1466

Study of DNA binding and bending by Bacillus subtilis GabR, a PLP-dependent transcription factor. D Amidani, A Tramonti, A V Canosa, B Campanini, S Maggi, T Milano, 10.1016/j.bbagen.2016.09.013Biochim. Biophys. Acta Gen. Subj. 1861Amidani, D., Tramonti, A., Canosa, A. V., Campanini, B., Maggi, S., Milano, T., et al. (2017). Study of DNA binding and bending by Bacillus subtilis GabR, a PLP-dependent transcription factor. Biochim. Biophys. Acta Gen. Subj. 1861, 3474-3489. doi: 10.1016/j.bbagen.2016.09.013

An experimentally supported model of the Bacillus subtilis global transcriptional regulatory network. M L Arrieta-Ortiz, C Hafemeister, A R Bate, T Chu, A Greenfield, B Shuster, 10.15252/msb.20156236Mol. Syst. Biol. 11839Arrieta-Ortiz, M. L., Hafemeister, C., Bate, A. R., Chu, T., Greenfield, A., Shuster, B., et al. (2015). An experimentally supported model of the Bacillus subtilis global transcriptional regulatory network. Mol. Syst. Biol. 11:839. doi: 10.15252/msb.20156236

The PatB protein of Bacillus subtilis is a C-S-lyase. S Auger, M P Gomez, A Danchin, I Martin-Verstraete, 10.1016/j.biochi.2004.09.007Biochimie. 87Auger, S., Gomez, M. P., Danchin, A., and Martin-Verstraete, I. (2005). The PatB protein of Bacillus subtilis is a C-S-lyase. Biochimie 87, 231-238. doi: 10.1016/j.biochi.2004.09.007

The metIC operon involved in methionine biosynthesis in Bacillus subtilis is controlled by transcription antitermination. S Auger, W H Yuen, A Danchin, I Martin-Verstraete, 10.1099/00221287-148-2-507Microbiology. 148Auger, S., Yuen, W. H., Danchin, A., and Martin-Verstraete, I. (2002). The metIC operon involved in methionine biosynthesis in Bacillus subtilis is controlled by transcription antitermination. Microbiology 148, 507-518. doi: 10.1099/00221287-148-2-507

The mtrAB operon of Bacillus subtilis encodes GTP cyclohydrolase I (MtrA), an enzyme involved in folic acid biosynthesis, and MtrB, a regulator of tryptophan biosynthesis. P Babitzke, P Gollnick, Yanofsky , C , 10.1128/jb.174.7.2059-2064.1992J. Bacteriol. 174Babitzke, P., Gollnick, P., and Yanofsky, C. (1992). The mtrAB operon of Bacillus subtilis encodes GTP cyclohydrolase I (MtrA), an enzyme involved in folic acid biosynthesis, and MtrB, a regulator of tryptophan biosynthesis. J. Bacteriol. 174, 2059-2064. doi: 10.1128/jb.174.7.2059-2064.1992

A guardian angel phosphatase for mainline carbon metabolism. G A Beaudoin, A D Hanson, 10.1016/j.tibs.2016.08.005Trends Biochem. Sci. 41Beaudoin, G. A., and Hanson, A. D. (2016). A guardian angel phosphatase for mainline carbon metabolism. Trends Biochem. Sci. 41, 893-894. doi: 10.1016/j.tibs.2016.08.005

Transcription analysis of the Bacillus subtilis PucR regulon and identification of a cis-acting sequence for PucR-regulated expression of genes involved in purine catabolism. L Beier, P Nygaard, H Jarmer, H H Saxild, 10.1128/JB.184.12.3232-3241.2002J. Bacteriol. 184Beier, L., Nygaard, P., Jarmer, H., and Saxild, H. H. (2002). Transcription analysis of the Bacillus subtilis PucR regulon and identification of a cis-acting sequence for PucR-regulated expression of genes involved in purine catabolism. J. Bacteriol. 184, 3232-3241. doi: 10.1128/JB.184.12.3232-3241.2002

Bacillus subtilis GabR, a protein with DNA-binding and aminotransferase domains, is a PLP-dependent transcriptional regulator. B R Belitsky, 10.1016/j.jmb.2004.05.020J. Mol. Biol. 340Belitsky, B. R. (2004a). Bacillus subtilis GabR, a protein with DNA-binding and aminotransferase domains, is a PLP-dependent transcriptional regulator. J. Mol. Biol. 340, 655-664. doi: 10.1016/j.jmb.2004.05.020

Physiological and enzymological interaction of Bacillus subtilis proteins required for de novo pyridoxal 5 ′ -phosphate biosynthesis. B R Belitsky, 10.1128/JB.186.4.1191-1196.2004J. Bacteriol. 186Belitsky, B. R. (2004b). Physiological and enzymological interaction of Bacillus subtilis proteins required for de novo pyridoxal 5 ′ -phosphate biosynthesis. J. Bacteriol. 186, 1191-1196. doi: 10.1128/JB.186.4.1191-1196.2004

Role of PdxR in the activation of vitamin B6 biosynthesis in Listeria monocytogenes. B R Belitsky, 10.1111/mmi.12618Mol. Microbiol. 92Belitsky, B. R. (2014). Role of PdxR in the activation of vitamin B6 biosynthesis in Listeria monocytogenes. Mol. Microbiol. 92, 1113-1128. doi: 10.1111/mmi.12618

GabR, a member of a novel protein family, regulates the utilization of gamma-aminobutyrate in Bacillus subtilis. B R Belitsky, A L Sonenshein, 10.1046/j.1365-2958.2002.03036.xMol. Microbiol. 45Belitsky, B. R., and Sonenshein, A. L. (2002). GabR, a member of a novel protein family, regulates the utilization of gamma-aminobutyrate in Bacillus subtilis. Mol. Microbiol. 45, 569-583. doi: 10.1046/j.1365-2958.2002. 03036.x

Genetic and biochemical analysis of CodY-binding sites in Bacillus subtilis. B R Belitsky, A L Sonenshein, 10.1128/JB.01780-07J. Bacteriol. 190Belitsky, B. R., and Sonenshein, A. L. (2008). Genetic and biochemical analysis of CodY-binding sites in Bacillus subtilis. J. Bacteriol. 190, 1224-1236. doi: 10.1128/JB.01780-07

Roadblock repression of transcription by Bacillus subtilis CodY. B R Belitsky, A L Sonenshein, 10.1016/j.jmb.2011.06.012J. Mol. Biol. 411Belitsky, B. R., and Sonenshein, A. L. (2011). Roadblock repression of transcription by Bacillus subtilis CodY. J. Mol. Biol. 411, 729-743. doi: 10.1016/j.jmb.2011.06.012

. B J Berger, S English, G Chan, M H Knodel, Berger, B. J., English, S., Chan, G., and Knodel, M. H. (2003).

Methionine regeneration and aminotransferases in Bacillus subtilis, Bacillus cereus, and Bacillus anthracis. 10.1128/JB.185.8.2418-2431.2003J. Bacteriol. 185Methionine regeneration and aminotransferases in Bacillus subtilis, Bacillus cereus, and Bacillus anthracis. J. Bacteriol. 185, 2418-2431. doi: 10.1128/JB.185.8.2418-2431.2003

Vitamin B6 (pyridoxine) and its derivatives are efficient singlet oxygen quenchers and potential fungal antioxidants. P Bilski, M Y Li, M Ehrenshaft, M E Daub, C F Chingell, 10.1562/0031-8655(2000)071$<$0129:SIPVBP$>$2.0.CO;2doi: 10.1562/ 0031-8655(2000)071<0129:SIPVBP>2.0.CO;2Photochem. Photobiol. 71Bilski, P., Li, M. Y., Ehrenshaft, M., Daub, M. E., and Chingell, C. F. (2000). Vitamin B6 (pyridoxine) and its derivatives are efficient singlet oxygen quenchers and potential fungal antioxidants. Photochem. Photobiol. 71, 129-134. doi: 10.1562/ 0031-8655(2000)071<0129:SIPVBP>2.0.CO;2

Abbreviated pathway for biosynthesis of 2-thiouridine in Bacillus subtilis. K A Black, Dos Santos, P C , 10.1128/JB.02625-14J. Bacteriol. 197Black, K. A., and Dos Santos, P. C. (2015). Abbreviated pathway for biosynthesis of 2-thiouridine in Bacillus subtilis. J. Bacteriol. 197, 1952-1962. doi: 10.1128/JB.02625-14

Isolation and characterization of D-serine deaminase constitutive mutants by utilization of D-serine as sole carbon or nitrogen source. F R Bloom, E Mcfall, J. Bacteriol. 121Bloom, F. R., and McFall, E. (1975). Isolation and characterization of D-serine deaminase constitutive mutants by utilization of D-serine as sole carbon or nitrogen source. J. Bacteriol. 121, 1078-1084.

Comparative enzymatic properties of GapB-encoded erythrose-4-P dehydrogenase of Escherichia coli and phosphorylating glyceraldehyde-3-phosphate dehydrogenase. S Boschi-Muller, S Azza, D Pollastro, C Corbier, G Branlant, 10.1074/jbc.272.24.15106J. Biol. Chem. 272Boschi-Muller, S., Azza, S., Pollastro, D., Corbier, C., and Branlant, G. (1997). Comparative enzymatic properties of GapB-encoded erythrose-4-P dehydrogenase of Escherichia coli and phosphorylating glyceraldehyde-3-phosphate dehydrogenase. J. Biol. Chem. 272, 15106-15112. doi: 10.1074/jbc.272.24.15106

Cloning, sequencing, and characterization of the Bacillus subtilis biotin biosynthetic operon. S Bower, J B Perkins, R R Yocum, C L Howitt, P Rahaim, J Pero, 10.1128/jb.178.14.4122-4130.1996J. Bacteriol. 178Bower, S., Perkins, J. B., Yocum, R. R., Howitt, C. L., Rahaim, P., and Pero, J. (1996). Cloning, sequencing, and characterization of the Bacillus subtilis biotin biosynthetic operon. J. Bacteriol. 178, 4122-4130. doi: 10.1128/jb.178.14.4122-4130.1996

Genomic distribution and heterogeneity of MocR-like transcriptional factors containing a domain belonging to the superfamily of the pyridoxal-5 ′ -phosphate dependent enzymes of fold type I. E Bramucci, T Milano, S Pascarella, 10.1016/j.bbrc.2011.10.017Biochem. Biophys. Res. Commun. 415Bramucci, E., Milano, T., and Pascarella, S. (2011). Genomic distribution and heterogeneity of MocR-like transcriptional factors containing a domain belonging to the superfamily of the pyridoxal-5 ′ -phosphate dependent enzymes of fold type I. Biochem. Biophys. Res. Commun. 415, 88-93. doi: 10.1016/j.bbrc.2011.10.017

Hierarchical expression of genes controlled by the Bacillus subtilis global regulatory protein CodY. S R Brinsmade, E L Alexander, J Livny, A J Stettner, D Segrè, K Y Rhee, 10.1073/pnas.1321308111Proc. Natl. Acad. Sci. U.S.A. 111Brinsmade, S. R., Alexander, E. L., Livny, J., Stettner, A. J., Segrè, D., Rhee, K. Y., et al. (2014). Hierarchical expression of genes controlled by the Bacillus subtilis global regulatory protein CodY. Proc. Natl. Acad. Sci. U.S.A. 111, 8227-8232. doi: 10.1073/pnas.1321308111

Studies on the redox properties of CDP-6-deoxy-L-threo-D-glycero-4-hexulose-3-dehydrase (E1) and CDP-6-deoxy-L-threo-D-glycero-4-hexulose-3-dehydrase reductase (E3): two important enzymes involved in the biosynthesis of ascarylose. K D Burns, P A Pieper, H W Liu, M T Stankovich, 10.1021/bi960284tBiochemistry. 35Burns, K. D., Pieper, P. A., Liu, H. W., and Stankovich, M. T. (1996). Studies on the redox properties of CDP-6-deoxy-L-threo-D-glycero-4-hexulose-3-dehydrase (E1) and CDP-6-deoxy-L-threo-D-glycero-4-hexulose-3-dehydrase reductase (E3): two important enzymes involved in the biosynthesis of ascarylose. Biochemistry 35, 7879-7889. doi: 10.1021/bi960284t

Reconstitution and biochemical characterization of a new pyridoxal-5 ′ -phosphate biosynthetic pathway. K E Burns, Y Xiang, C L Kinsland, F W Mclafferty, T P Begley, 10.1021/ja042792tJ. Am. Chem. Soc. 127Burns, K. E., Xiang, Y., Kinsland, C. L., McLafferty, F. W., and Begley, T. P. (2005). Reconstitution and biochemical characterization of a new pyridoxal- 5 ′ -phosphate biosynthetic pathway. J. Am. Chem. Soc. 127, 3682-3683. doi: 10.1021/ja042792t

Evolution and multiplicity of arginine decarboxylases in polyamine biosynthesis and essential role in Bacillus subtilis biofilm formation. M Burrell, C C Hanfrey, E J Murray, N R Stanley-Wall, A J Michael, 10.1074/jbc.M110.163154J. Biol. Chem. 285Burrell, M., Hanfrey, C. C., Murray, E. J., Stanley-Wall, N. R., and Michael, A. J. (2010). Evolution and multiplicity of arginine decarboxylases in polyamine biosynthesis and essential role in Bacillus subtilis biofilm formation. J. Biol. Chem. 285, 39224-39238. doi: 10.1074/jbc.M110. 163154

Characterization of the Bacillus subtilis GTPase YloQ and its role in ribosome function. T L Campbell, D M Daigle, E D Brown, 10.1042/BJ20041873Biochem. J. 389Campbell, T. L., Daigle, D. M., and Brown, E. D. (2005). Characterization of the Bacillus subtilis GTPase YloQ and its role in ribosome function. Biochem. J. 389, 843-852. doi: 10.1042/BJ20041873

Biosynthesis of vitamin B6: enzymatic conversion of 1-deoxy-D-xylulose-5-phosphate to pyridoxol phosphate. D E Cane, S Du, J K Robinson, Y Hsiung, I D Spenser, 10.1021/ja9914947J. Am. Chem. Soc. 121Cane, D. E., Du, S., Robinson, J. K., Hsiung, Y., and Spenser, I. D. (1999). Biosynthesis of vitamin B6: enzymatic conversion of 1-deoxy-D-xylulose- 5-phosphate to pyridoxol phosphate. J. Am. Chem. Soc. 121, 7722-7723. doi: 10.1021/ja9914947

Biosynthesis of vitamin B6: the oxidation of 4-(phosphohydroxy)-L-threonine by PdxA. D E Cane, Y Hsiung, J A Cornish, J K Robinson, I D Spenser, 10.1021/ja9742085doi: 10.1021/ ja9742085J. Am. Chem. Soc. 120Cane, D. E., Hsiung, Y., Cornish, J. A., Robinson, J. K., and Spenser, I. D. (1998). Biosynthesis of vitamin B6: the oxidation of 4-(phosphohydroxy)- L-threonine by PdxA. J. Am. Chem. Soc. 120, 1936-1937. doi: 10.1021/ ja9742085

The sps gene products affect the germination, hydrophobicity, and protein adsorption of Bacillus subtilis spores. G Cangiano, T Sirec, C Panarella, R Isticato, L Baccigalupi, M De Felice, 10.1128/AEM.02893-14Appl. Environ. Microbiol. 80Cangiano, G., Sirec, T., Panarella, C., Isticato, R., Baccigalupi, L., De Felice, M., et al. (2014). The sps gene products affect the germination, hydrophobicity, and protein adsorption of Bacillus subtilis spores. Appl. Environ. Microbiol. 80, 7293-7302. doi: 10.1128/AEM.02893-14

Funtional evolution of PLP-dependent enzymes based on active-site structural similarities. J Catazaro, A Caprez, A Guru, D Swanson, R Powers, 10.1002/prot.24624Proteins. 82Catazaro, J., Caprez, A., Guru, A., Swanson, D., and Powers, R. (2014). Funtional evolution of PLP-dependent enzymes based on active-site structural similarities. Proteins 82, 2597-2608. doi: 10.1002/prot.24624

A novel lysine 2,3-aminomutase encoded by the yodO gene of Bacillus subtilis: characterization and the observation of organic radical intermediates. D Chen, F J Ruzicka, P A Frey, 10.1042/bj3480539Biochem. J. 348Chen, D., Ruzicka, F. J., and Frey, P. A. (2000). A novel lysine 2,3-aminomutase encoded by the yodO gene of Bacillus subtilis: characterization and the observation of organic radical intermediates. Biochem. J. 348, 539-549. doi: 10.1042/bj3480539

Lysine 2,3-aminomutase. Purification and properties of a pyridoxal phosphate and S-adenosylmethionine-activated enzyme. T P Chirpich, V Zappia, R N Costilow, H A Barker, J. Biol. Chem. 245Chirpich, T. P., Zappia, V., Costilow, R. N., and Barker, H. A. (1970). Lysine 2,3-aminomutase. Purification and properties of a pyridoxal phosphate and S-adenosylmethionine-activated enzyme. J. Biol. Chem. 245, 1778-1789.

The global regulator Spx functions in the control of organosulfur metabolism in B. subtilis. S Y Choi, D Reyes, M Leelakriangsak, P Zuber, 10.1128/JB.00443-06J. Bacteriol. 188Choi, S. Y., Reyes, D., Leelakriangsak, M., and Zuber, P. (2006). The global regulator Spx functions in the control of organosulfur metabolism in B. subtilis. J. Bacteriol. 188, 5741-5751. doi: 10.1128/JB.00443-06

From cofactor to enzymes. The molecular evolution of pyridoxal-5 ′ -phosphate-dependent enzymes. P Christen, P K Mehta, 10.1002/tcr.10005Chem. Rec. 1Christen, P., and Mehta, P. K. (2001). From cofactor to enzymes. The molecular evolution of pyridoxal-5 ′ -phosphate-dependent enzymes. Chem. Rec. 1, 436-447. doi: 10.1002/tcr.10005

Genome-wide binding profiles of the Bacillus subtilis transition state regulator AbrB and its homolog Abh reveals their interactive role in transcription regulateion. O Chumsakul, H Takahashi, T Oshima, T Hishimoto, S Kanaya, N Ogasawara, 10.1093/nar/gkq780Nucleic Acids Res. 39Chumsakul, O., Takahashi, H., Oshima, T., Hishimoto, T., Kanaya, S., Ogasawara, N., et al. (2011). Genome-wide binding profiles of the Bacillus subtilis transition state regulator AbrB and its homolog Abh reveals their interactive role in transcription regulateion. Nucleic Acids Res. 39, 414-428. doi: 10.1093/nar/gkq780

A conserved phosphatase destroys toxic glycolytic side products in mammals and yeast. F Collard, F Baldin, I Gerin, J Bolsée, G Noel, J Graff, 10.1038/nchembio.2104Nat. Chem. Biol. 12Collard, F., Baldin, F., Gerin, I., Bolsée, J., Noel, G., Graff, J., et al. (2016). A conserved phosphatase destroys toxic glycolytic side products in mammals and yeast. Nat. Chem. Biol. 12, 601-601. doi: 10.1038/nchembi o.2104

Overexpression of a non-native deoxyxylulose-dependent vitamin B6 pathway in Bacillus subtilis for the production of pyridoxine. F M Commichau, A Alzinger, R Sande, W Bretzel, F M Meyer, B Chevreux, 10.1016/j.ymben.2014.06.007Metab. Eng. 25Commichau, F. M., Alzinger, A., Sande, R., Bretzel, W., Meyer, F. M., Chevreux, B., et al. (2014). Overexpression of a non-native deoxyxylulose-dependent vitamin B6 pathway in Bacillus subtilis for the production of pyridoxine. Metab. Eng. 25, 38-49. doi: 10.1016/j.ymben.2014.06.007

Overexpression of a non-native deoxyxylulose-dependent vitamin B6 pathway in Bacillus subtilis for the production of pyridoxine. F M Commichau, A Alzinger, R Sande, W Bretzel, D R Reuß, M Dormeyer, 10.1016/j.ymben.2015.03.007Metab. Eng. 29Commichau, F. M., Alzinger, A., Sande, R., Bretzel, W., Reuß, D. R., Dormeyer, M., et al. (2015). Overexpression of a non-native deoxyxylulose-dependent vitamin B6 pathway in Bacillus subtilis for the production of pyridoxine. Metab. Eng. 29, 196-207. doi: 10.1016/j.ymben.2015.03.007

Metabolism of D-serine in Escherichia coli K-12: mechanism of growth inhibition. S D Cosloy, E Mcfall, J. Bacteriol. 114Cosloy, S. D., and McFall, E. (1973). Metabolism of D-serine in Escherichia coli K-12: mechanism of growth inhibition. J. Bacteriol. 114, 685-694.

Purification and initial characterization of AhrC: the regulator of arginine metabolism genes in Bacillus subtilis. L G Czaplewski, A K North, M C Smith, S Baumberg, P G Stockley, 10.1111/j.1365-2958.1992.tb02008.xMol. Microbiol. 6Czaplewski, L. G., North, A. K., Smith, M. C., Baumberg, S., and Stockley, P. G. (1992). Purification and initial characterization of AhrC: the regulator of arginine metabolism genes in Bacillus subtilis. Mol. Microbiol. 6, 267-275. doi: 10.1111/j.1365-2958.1992.tb02008.x

Prokaryotic heme biosynthesis: multiple pathways to a common essential product. Microbiol. H A Dailey, T A Dailey, S Gerdes, D Jahn, M Jahn, M R O&apos;brian, 10.1128/MMBR.00048-16Mol. Biol. Rev. 81Dailey, H. A., Dailey, T. A., Gerdes, S., Jahn, D., Jahn, M., O'Brian, M. R., et al. (2017). Prokaryotic heme biosynthesis: multiple pathways to a common essential product. Microbiol. Mol. Biol. Rev. 81, e00048-e00016. doi: 10.1128/MMBR.00048-16

The trp RNA-binding attenuation protein (TRAP) regulates the steady-state levels of transcripts of the Bacillus subtilis folate operon. A De Saizieu, P Vankan, C Vockler, A P Van Loon, 10.1099/00221287-143-3-979Microbiology. 143de Saizieu, A., Vankan, P., Vockler, C., and van Loon, A. P. (1997). The trp RNA-binding attenuation protein (TRAP) regulates the steady-state levels of transcripts of the Bacillus subtilis folate operon. Microbiology 143, 979-989. doi: 10.1099/00221287-143-3-979

Vitamin B6 salvage enzymes: mechanism, structure and regulation. M L Di Salvo, R Contestabile, M K Safo, 10.1016/j.bbapap.2010.12.006Biochim. Biophys. Acta. 1854di Salvo, M. L., Contestabile, R., and Safo, M. K. (2011). Vitamin B6 salvage enzymes: mechanism, structure and regulation. Biochim. Biophys. Acta. 1854, 1597-1608. doi: 10.1016/j.bbapap.2010.12.006

Expression, purification, and kinetic constants for human and Escherichia coli pyridoxal kinases. M L Di Salvo, S Hunt, V Schich, 10.1016/j.pep.2004.04.021Protein Expr. Purif. 36di Salvo, M. L., Hunt, S., and Schich, V. (2004). Expression, purification, and kinetic constants for human and Escherichia coli pyridoxal kinases. Protein Expr. Purif. 36, 300-306. doi: 10.1016/j.pep.2004.04.021

3-Amino-3-deoxyglucose produced by a Streptomyces sp. L A Dolak, T M Castle, A Dietz, A L Laborde, 10.7164/antibiotics.33.900J. Antibiot. 33Dolak, L. A., Castle, T. M., Dietz, A., and Laborde, A. L. (1980). 3-Amino- 3-deoxyglucose produced by a Streptomyces sp. J. Antibiot. 33, 900-901. doi: 10.7164/antibiotics.33.900

Use of Vitamins in Foods -Toxicological and Nutritional-Physiological Aspects. A Domke, R Großklaus, B Niemann, H Przyrembel, K Richter, E Schmidt, BfR-WissenschaftBerlinDomke, A., Großklaus, R., Niemann, B., Przyrembel, H., Richter, K., Schmidt, E., et al. (2005). Use of Vitamins in Foods -Toxicological and Nutritional- Physiological Aspects. Berlin: BfR-Wissenschaft.

Chemical modification of adenylsuccinate synthetase from Escherichia coli by pyridoxal 5 ′ -phosphate. Identification of an active site lysyl residue. Q Dong, H J Fromm, J. Biol. Chem. 265Dong, Q., and Fromm, H. J. (1990). Chemical modification of adenylsuccinate synthetase from Escherichia coli by pyridoxal 5 ′ -phosphate. Identification of an active site lysyl residue. J. Biol. Chem. 265, 6235-6240.

4-O-phosphoryl-L-threonine, a substrate of the pdxC(serC) gene product involved in vitamin B6 biosynthesis. C Drewke, M Klein, D Clade, A Arenz, A Müller, E Leistner, 10.1016/0014-5793(96)00652-7FEBS Lett. 390Drewke, C., Klein, M., Clade, D., Arenz, A., Müller, A., and Leistner, E. (1996). 4-O-phosphoryl-L-threonine, a substrate of the pdxC(serC) gene product involved in vitamin B6 biosynthesis. FEBS Lett. 390, 179-182. doi: 10.1016/0014-5793(96)00652-7

Growth response to 4-hydroxy-L-threonine of Escherichia coli mutants blocked in vitamin B6 biosynthesis. C Drewke, C Notheis, U Hansen, E Leistner, T Hemscheidt, R E Hill, 10.1016/0014-5793(93)80005-FFEBS Lett. 3189380005Drewke, C., Notheis, C., Hansen, U., Leistner, E., Hemscheidt, T., Hill, R. E., et al. (1993). Growth response to 4-hydroxy-L-threonine of Escherichia coli mutants blocked in vitamin B6 biosynthesis. FEBS Lett. 318, 125-128. doi: 10.1016/0014-5793(93)80005-F

Crystal structure of Bacillus subtilis GabR, an autorepressor and transcriptional activator of gabT. R Edayathumangalam, R Wu, R Garcia, Y Wang, W Wang, C A Kreinbring, 10.1073/pnas.1315887110Proc. Natl. Acad. Sci. U.S.A. 110Edayathumangalam, R., Wu, R., Garcia, R., Wang, Y., Wang, W., Kreinbring, C. A., et al. (2013). Crystal structure of Bacillus subtilis GabR, an autorepressor and transcriptional activator of gabT. Proc. Natl. Acad. Sci. U.S.A. 110, 17820-17825. doi: 10.1073/pnas.1315887110

One hundred years of vitamins -a success story of natural sciences. M Eggersdorfer, D Laudert, U Létinois, T Mcclymont, J Medlock, T Netscher, 10.1002/anie.201205886Angew. Chem. Int. Ed. Engl. 51Eggersdorfer, M., Laudert, D., Létinois, U., McClymont, T., Medlock, J., Netscher, T., et al. (2012). One hundred years of vitamins -a success story of natural sciences. Angew. Chem. Int. Ed. Engl. 51, 12960-12990. doi: 10.1002/anie.201205886

Isolation of PDX2, a second novel gene in the pyridoxine biosynthesis pathway of eukaryotes, archaebacteria, and a subset of eubacteria. M Ehrenshaft, M E Daub, 10.1128/JB.183.11.3383-3390.2001J. Bacteriol. 183Ehrenshaft, M., and Daub, M. E. (2001). Isolation of PDX2, a second novel gene in the pyridoxine biosynthesis pathway of eukaryotes, archaebacteria, and a subset of eubacteria. J. Bacteriol. 183, 3383-3390. doi: 10.1128/JB.183.11.3383-3390.2001

The program of gene transcription for a single differentiating cell type during sporulation in Bacillus subtilis. P Eichenberger, M Fujita, S T Jensen, E M Conlon, D Z Rudner, S T Wang, 10.1371/journal.pbio.0020328PLoS Biol. 2328Eichenberger, P., Fujita, M., Jensen, S. T., Conlon, E. M., Rudner, D. Z., Wang, S. T., et al. (2004). The program of gene transcription for a single differentiating cell type during sporulation in Bacillus subtilis. PLoS Biol. 2:e328. doi: 10.1371/journal.pbio.0020328

The vitamin B6 pathway in Streptococcus pneumoniae is controlled by pyridoxal 5 ′ -phosphate and the transcription factor PdxR and has an impact on ear infection. S El Qaidi, J Yang, J R Zhang, D W Metzger, G Bai, 10.1128/JB.00041-13J. Bacteriol. 195El Qaidi, S., Yang, J., Zhang, J. R., Metzger, D. W., and Bai, G. (2013). The vitamin B6 pathway in Streptococcus pneumoniae is controlled by pyridoxal 5 ′ -phosphate and the transcription factor PdxR and has an impact on ear infection. J. Bacteriol. 195, 2187-2196. doi: 10.1128/JB.00041-13

Pyridoxal phosphate enzymes: mechanistic, structural, and evolutionary considerations. A C Eliot, J F Kirsch, 10.1146/annurev.biochem.73.011303.074021Annu. Rev. Biochem. 73Eliot, A. C., and Kirsch, J. F. (2004). Pyridoxal phosphate enzymes: mechanistic, structural, and evolutionary considerations. Annu. Rev. Biochem. 73, 383-415. doi: 10.1146/annurev.biochem.73.011303.074021

Global control of cysteine metabolism by CymR in Bacillus subtilis. S Even, P Burguière, S Auger, O Soutourina, A Danchin, I Martin-Verstraete, 10.1128/JB.188.6.2184-2197.2006J. Bacteriol. 188Even, S., Burguière, P., Auger, S., Soutourina, O., Danchin, A., and Martin- Verstraete, I. (2006). Global control of cysteine metabolism by CymR in Bacillus subtilis. J. Bacteriol. 188, 2184-2197. doi: 10.1128/JB.188.6.2184-2197.2006

Threonine synthase of Escherichia coli: inhibition by classical and slow-binding analogues of homoserine phosphate. G K Farrington, A Kumar, S L Shames, J I Ewaskiewicz, D E Ash, F C Wedler, 10.1006/abbi.1993.1575Arch. Biochem. Biophys. 307Farrington, G. K., Kumar, A., Shames, S. L., Ewaskiewicz, J. I., Ash, D. E., and Wedler, F. C. (1993). Threonine synthase of Escherichia coli: inhibition by classical and slow-binding analogues of homoserine phosphate. Arch. Biochem. Biophys. 307, 165-174. doi: 10.1006/abbi.1993.1575

Identification of sporulation genes by genome-wide analysis of the sigmaE regulon of Bacillus subtilis. A Feucht, J Evans, J Errington, 10.1099/mic.0.26413-0Microbiology. 149Feucht, A., Evans, J., and Errington, J. (2003). Identification of sporulation genes by genome-wide analysis of the sigmaE regulon of Bacillus subtilis. Microbiology 149, 3023-3034. doi: 10.1099/mic.0.26413-0

Two independent routes for de novo vitamin B6 biosynthesis: not that different after all. T B Fitzpatrick, N Amrhein, B Kappes, P Macheroux, I Tews, Raschle , T , 10.1042/BJ20070765Biochem. J. 407Fitzpatrick, T. B., Amrhein, N., Kappes, B., Macheroux, P., Tews, I., and Raschle, T. (2007). Two independent routes for de novo vitamin B6 biosynthesis: not that different after all. Biochem. J. 407, 1-13. doi: 10.1042/BJ20070765

Vitamin B6 biosynthesis: charting the mechanistic landscape. T B Fitzpatrick, C Moccand, C Roux, 10.1002/cbic.201000084ChemBioChem. 11Fitzpatrick, T. B., Moccand, C., and Roux, C. (2010). Vitamin B6 biosynthesis: charting the mechanistic landscape. ChemBioChem 11, 1185-1193. doi: 10.1002/cbic.201000084

Factors influencing bacterial deamination: factors influencing the activity of dl-serine deaminase in Bacterium coli. E F Gale, M Stephenson, 10.1042/bj0320392Biochem. J. 32Gale, E. F., and Stephenson, M. (1938). Factors influencing bacterial deamination: factors influencing the activity of dl-serine deaminase in Bacterium coli. Biochem. J. 32, 392-404. doi: 10.1042/bj0320392

Expression of the rocDEF operon involved in arginine catabolism in Bacillus subtilis. R Gardan, G Rapoport, M Débarbouillé, 10.1006/jmbi.1995.0342J. Mol. Biol. 249Gardan, R., Rapoport, G., and Débarbouillé, M. (1995). Expression of the rocDEF operon involved in arginine catabolism in Bacillus subtilis. J. Mol. Biol. 249, 843-856. doi: 10.1006/jmbi.1995.0342

Crystal structure of glutamate-1-semialdehyde aminotransferase from Bacillus subtilis with bound pyridoxamine-5 ′ -phosphate. H Ge, X Ly, J Fan, Y Gao, M Teng, L Niu, 10.1016/j.bbrc.2010.10.033Biochem. Biophys. Res. Commun. 402Ge, H., Ly, X., Fan, J., Gao, Y., Teng, M., and Niu, L. (2010). Crystal structure of glutamate-1-semialdehyde aminotransferase from Bacillus subtilis with bound pyridoxamine-5 ′ -phosphate. Biochem. Biophys. Res. Commun. 402, 356-360. doi: 10.1016/j.bbrc.2010.10.033

Pyridoxal 5 ′ -phosphate is a slow tight binding inhibitor of E. coli pyridoxal kinase. M S Ghatge, R Contestabile, M L Di Salvo, J V Desai, A K Gandhi, C M Camara, 10.1371/journal.pone.0041680PLoS ONE. 741680Ghatge, M. S., Contestabile, R., di Salvo, M. L., Desai, J. V., Gandhi, A. K., Camara, C. M., et al. (2012). Pyridoxal 5 ′ -phosphate is a slow tight binding inhibitor of E. coli pyridoxal kinase. PLoS ONE. 7:e41680. doi: 10.1371/journal.pone. 0041680

The S box regulon: a new global transcription termination control system for methionine and cysteine biosyngthesis genes in Gram-positive bacteria. F Grundy, T Henkin, 10.1046/j.1365-2958.1998.01105.xMol. Microbiol. 30Grundy, F., and Henkin, T. (1998). The S box regulon: a new global transcription termination control system for methionine and cysteine biosyngthesis genes in Gram-positive bacteria. Mol. Microbiol. 30, 737-749. doi: 10.1046/j.1365-2958.1998.01105.x

Biochemical features and functional implications of the RNAbased T-box regulatory mechanism. Microbiol. A Gutiérrez-Preciado, T M Henkin, F J Grundy, C Yanofsky, E Merino, 10.1128/MMBR.00026-08Mol. Biol. Rev. 73Gutiérrez-Preciado, A., Henkin, T. M., Grundy, F. J., Yanofsky, C., and Merino, E. (2009). Biochemical features and functional implications of the RNA- based T-box regulatory mechanism. Microbiol. Mol. Biol. Rev. 73, 36-61. doi: 10.1128/MMBR.00026-08

The history of vitamin B6. P György, 10.1093/ajcn/4.4.313Am. J. Clin. Nutr. 4György, P. (1956). The history of vitamin B6. Am. J. Clin. Nutr. 4, 313-317. doi: 10.1093/ajcn/4.4.313

The Bacillus subtilis hemAXCDBL gene cluster, which encodes enzymes of the biosynthetic pathway from glutamate to uroporphyrinogen III. M Hansson, L Rutberg, I Schröder, L Hederstedt, 10.1128/jb.173.8.2590-2599.1991J. Bacteriol. 173Hansson, M., Rutberg, L., Schröder, I., and Hederstedt, L. (1991). The Bacillus subtilis hemAXCDBL gene cluster, which encodes enzymes of the biosynthetic pathway from glutamate to uroporphyrinogen III. J. Bacteriol. 173, 2590-2599. doi: 10.1128/jb.173.8.2590-2599.1991

Longevity of major coenzymes allows minimal de novo synthesis in microorganisms. J Hartl, P Kiefer, F Meyer, J A Vorholt, 10.1038/nmicrobiol.2017.73Nat. Microbiol. 217073Hartl, J., Kiefer, P., Meyer, F., and Vorholt, J. A. (2017). Longevity of major coenzymes allows minimal de novo synthesis in microorganisms. Nat. Microbiol. 2:17073. doi: 10.1038/nmicrobiol.2017.73

Controlled lysis of bacterial cells utilizing mutants with defective synthesis of D-alanine. M P Heaton, R B Johnston, T L Thompson, 10.1139/m88-047Can. J. Microbiol. 34Heaton, M. P., Johnston, R. B., and Thompson, T. L. (1988). Controlled lysis of bacterial cells utilizing mutants with defective synthesis of D-alanine. Can. J. Microbiol. 34, 256-261. doi: 10.1139/m88-047

Roles of metal ions and hydrogen peroxide in modulating the interaction of the Bacillus subtilis PerR peroxide regulon repressor with operator DNA. A F Herbig, J D Helmann, 10.1046/j.1365-2958.2001.02543.xMol. Microbiol. 41Herbig, A. F., and Helmann, J. D. (2001). Roles of metal ions and hydrogen peroxide in modulating the interaction of the Bacillus subtilis PerR peroxide regulon repressor with operator DNA. Mol. Microbiol. 41, 849-859. doi: 10.1046/j.1365-2958.2001.02543.x

Mining the cellular inventory of pyridoxal phosphate-dependent enzymes with functionalized cofactor mimics. A Hoegl, M B Nodwell, V C Kirsch, N C Bach, M Pfanzelt, M Stahl, 10.1038/s41557-018-0144-2Nat. Chem. 10Hoegl, A., Nodwell, M. B., Kirsch, V. C., Bach, N. C., Pfanzelt, M., Stahl, M., et al. (2018). Mining the cellular inventory of pyridoxal phosphate-dependent enzymes with functionalized cofactor mimics. Nat. Chem. 10, 1234-1245. doi: 10.1038/s41557-018-0144-2

Glycerol catabolism in Bacillus subtilis: nucleotide sequence of the genes encoding glycerol kinase (glpK) and glycerol-3-phosphate dehydrogenase (glpD). C Holmberg, L Beijer, B Rutberg, L Rutberg, 10.1099/00221287-136-12-2367J. Gen. Microbiol. 136Holmberg, C., Beijer, L., Rutberg, B., and Rutberg, L. (1990). Glycerol catabolism in Bacillus subtilis: nucleotide sequence of the genes encoding glycerol kinase (glpK) and glycerol-3-phosphate dehydrogenase (glpD). J. Gen. Microbiol. 136, 2367-2375. doi: 10.1099/00221287-136-12-2367

Identification of target promoters for the Bacillus subtilis extracytoplasmic function sigma factor, sigma W. X Huang, A Gaballa, M Cao, J D Helmann, 10.1046/j.1365-2958.1999.01180.xMol. Microbiol. 31Huang, X., Gaballa, A., Cao, M., and Helmann, J. D. (1999). Identification of target promoters for the Bacillus subtilis extracytoplasmic function sigma factor, sigma W. Mol. Microbiol. 31, 361-371. doi: 10.1046/j.1365-2958.1999.01180.x

Vitamin B6 conjugation to nuclear corepressor RIP140 and its role in gene regulation. M D Huq, N P Tsai, Y P Lin, L Higgins, L N Wei, 10.1038/nchembio861Nat. Chem. Biol. 3Huq, M. D., Tsai, N. P., Lin, Y. P., Higgins, L., and Wei, L. N. (2007). Vitamin B6 conjugation to nuclear corepressor RIP140 and its role in gene regulation. Nat. Chem. Biol. 3, 161-165. doi: 10.1038/nchembio861

Activation of dormant secondary metabolism neotrehalosadiamine synthesis by an RNA polymerase mutation in Bacillus subtilis. T Inaoka, K Ochi, 10.1271/bbb.100854Biosci. Biotechnol. Biochem. 75Inaoka, T., and Ochi, K. (2011). Activation of dormant secondary metabolism neotrehalosadiamine synthesis by an RNA polymerase mutation in Bacillus subtilis. Biosci. Biotechnol. Biochem. 75, 618-623. doi: 10.1271/bbb.100854

Guanine nucleosides guanosine 5 ′ -diphosphate 3 ′ -diphosphate and GTP co-operatively regulate the production of an antibiotic bacilysin in Bacillus subtilis. T Inaoka, K Takahashi, M Ohnishi-Kameyama, M Yoshida, K Ochi, 10.1074/jbc.M208722200J. Biol. Chem. 278Inaoka, T., Takahashi, K., Ohnishi-Kameyama, M., Yoshida, M., and Ochi, K. (2003). Guanine nucleosides guanosine 5 ′ -diphosphate 3 ′ -diphosphate and GTP co-operatively regulate the production of an antibiotic bacilysin in Bacillus subtilis. J. Biol. Chem. 278, 2169-2176. doi: 10.1074/jbc.M208722200

RNA polymerase mutation activates the production of a dormant antibiotic 3,3 ′ -neotrehalosamine via an autoinduction mechanism in Bacillus subtilis. T Inaoka, K Takahashi, H Yada, M Yoshida, K Ochi, 10.1074/jbc.M309925200J. Biol. Chem. 279Inaoka, T., Takahashi, K., Yada, H., Yoshida, M., and Ochi, K. (2004). RNA polymerase mutation activates the production of a dormant antibiotic 3,3 ′ - neotrehalosamine via an autoinduction mechanism in Bacillus subtilis. J. Biol. Chem. 279, 3885-3892. doi: 10.1074/jbc.M309925200

ScoC regulates bacilysin production at the transcription level in Bacillus subtilis. T Inaoka, G Wang, K Ochi, 10.1128/JB.01081-09J. Bacteriol. 191Inaoka, T., Wang, G., and Ochi, K. (2009). ScoC regulates bacilysin production at the transcription level in Bacillus subtilis. J. Bacteriol. 191, 7367-7371. doi: 10.1128/JB.01081-09

A regulatory RNA required for antitermination of biofilm and capsular polysaccharide operons in Bacillus subtilis. I Irnov, W C Winkler, 10.1111/j.1365-2958.2010.07131.xMol. Microbiol. 76Irnov, I., and Winkler, W. C. (2010). A regulatory RNA required for antitermination of biofilm and capsular polysaccharide operons in Bacillus subtilis. Mol. Microbiol. 76, 559-575. doi: 10.1111/j.1365-2958.2010.07131.x

Conserved pyridoxal protein that regulates Ile and Val metabolism. T Ito, J Limori, S Takayama, A Moriyama, Y Yamauchi, H Hemmi, 10.1128/JB.00593-13J. Bacteriol. 195Ito, T., Limori, J., Takayama, S., Moriyama, A., Yamauchi, Y., Hemmi, H., et al. (2013). Conserved pyridoxal protein that regulates Ile and Val metabolism. J. Bacteriol. 195, 5439-5449. doi: 10.1128/JB.00593-13

7,8-diaminopelargonic acid aminotransferase, an enzyme involved in biotin synthesis by microorganisms. Y Izumi, K Sato, Y Tani, K Ogata, 10.1080/00021369.1975.10861584Agric. Biol. Chem. 39Izumi, Y., Sato, K., Tani, Y., and Ogata, K. (1975). 7,8-diaminopelargonic acid aminotransferase, an enzyme involved in biotin synthesis by microorganisms. Agric. Biol. Chem. 39, 175-181. doi: 10.1080/00021369.1975.10861584

Structure, evolution and action of vitamin B6-dependent enzymes. J N Jansonius, 10.1016/S0959-440X(98)80096-1Curr. Opin. Struct. Biol. 8Jansonius, J. N. (1998). Structure, evolution and action of vitamin B6-dependent enzymes. Curr. Opin. Struct. Biol. 8, 759-769. doi: 10.1016/S0959-440X(98)80096-1

Positive transcriptional control of the pyridoxal phosphate biosynthesis genes pdxST by the MocR-type regulatore PdxR of Corynebacterium glutamicum ATCC 13032. N Jochmann, S Götker, A Tauch, 10.1099/mic.0.044818-0Microbiology. 157Jochmann, N., Götker, S., and Tauch, A. (2011). Positive transcriptional control of the pyridoxal phosphate biosynthesis genes pdxST by the MocR-type regulatore PdxR of Corynebacterium glutamicum ATCC 13032. Microbiology 157, 77-88. doi: 10.1099/mic.0.044818-0

Pyridoxal phosphate dependent enzymes. R A John, 10.1016/0167-4838(95)00025-PBiochim. Biophys. Acta. 1248John, R. A. (1995). Pyridoxal phosphate dependent enzymes. Biochim. Biophys. Acta. 1248, 81-96. doi: 10.1016/0167-4838(95)00025-P

Isolation and comparative analysis of diaminopimelate decarboxylase from Streptococcus bovis and Bacillus subtilis. E O Kalcheva, M M Faiziev, S S Malyuta, Mol. Gen. Mikrobiol. Virosol. 1Kalcheva, E. O., Faiziev, M. M., and Malyuta, S. S. (1997). Isolation and comparative analysis of diaminopimelate decarboxylase from Streptococcus bovis and Bacillus subtilis. Mol. Gen. Mikrobiol. Virosol. 1, 34-37.

The effects of insertional mutations in comQ, comP, srfA, spo0H, spo0A and abrB genes on bycilysin biosynthesis in Bacillus subtilis. A Y Karatas, S Cetin, G Ozcengiz, 10.1016/S0167-4781(03)00037-XBiochim. Biophys. Acta. 1626Karatas, A. Y., Cetin, S., and Ozcengiz, G. (2003). The effects of insertional mutations in comQ, comP, srfA, spo0H, spo0A and abrB genes on bycilysin biosynthesis in Bacillus subtilis. Biochim. Biophys. Acta. 1626, 51-56. doi: 10.1016/S0167-4781(03)00037-X

A master regulator for biofilm formation by Bacillus subtilis. D B Kearns, F Chu, S S Branda, R Kolter, R Losick, 10.1111/j.1365-2958.2004.04440.xMol. Microbiol. 55Kearns, D. B., Chu, F., Branda, S. S., Kolter, R., and Losick, R. (2005). A master regulator for biofilm formation by Bacillus subtilis. Mol. Microbiol. 55, 739-749. doi: 10.1111/j.1365-2958.2004.04440.x

Glycogen in Bacillus subtilis: molecular characterization of an operon encoding enzymes involved in glycogen biosynthesis and degradation. J A Kiel, J M Boels, G Beldman, G Venema, 10.1111/j.1365-2958.1994.tb00301.xMol. Microbiol. 11Kiel, J. A., Boels, J. M., Beldman, G., and Venema, G. (1994). Glycogen in Bacillus subtilis: molecular characterization of an operon encoding enzymes involved in glycogen biosynthesis and degradation. Mol. Microbiol. 11, 203-218. doi: 10.1111/j.1365-2958.1994.tb00301.x

Structural basis for the substrate specificity of PepA from Streptococcus pneumoniae, a dodecameric tetrahedral protease. D Kim, B H San, S H Moh, H Park, D Y Kim, S Lee, 10.1016/j.bbrc.2009.11.075Biochem. Biophys. Res. Commun. 391Kim, D., San, B. H., Moh, S. H., Park, H., Kim, D. Y., Lee, S., et al. (2010). Structural basis for the substrate specificity of PepA from Streptococcus pneumoniae, a dodecameric tetrahedral protease. Biochem. Biophys. Res. Commun. 391, 431-436. doi: 10.1016/j.bbrc.2009.11.075

Inhibitory cross-talk upon introduction of a new metabolic pathway into an existing metabolic network. J Kim, S D Copley, 10.1073/pnas.1208509109Proc. Natl. Acad. Sci. U.S.A. 109Kim, J., and Copley, S. D. (2012). Inhibitory cross-talk upon introduction of a new metabolic pathway into an existing metabolic network. Proc. Natl. Acad. Sci. U.S.A. 109, E2856-E2864. doi: 10.1073/pnas.1208509109

Three serendipitous pathways in E. coli can bypass a block in pyridoxal-5 ′ -phosphate synthesis. J Kim, J P Kershner, Y Novikov, R K Shoemaker, S D Copley, 10.1038/msb.2010.88Mol. Sys. Biol. 6436Kim, J., Kershner, J. P., Novikov, Y., Shoemaker, R. K., and Copley, S. D. (2010). Three serendipitous pathways in E. coli can bypass a block in pyridoxal-5 ′ - phosphate synthesis. Mol. Sys. Biol. 6:436. doi: 10.1038/msb.2010.88

A Kleemann, B Engel, D Kutscher, D Reichert, Pharmaceutical Substances. Syntheses, Patents, Applications of the Most Relevant APIs. StuttgartThiemeKleemann, A., Engel, B., Kutscher, D., and Reichert, D. (2008). Pharmaceutical Substances. Syntheses, Patents, Applications of the Most Relevant APIs. Stuttgart: Thieme.

Construction and analysis of two genme-scale deletion libraries for Bacillus subtilis. B M Koo, G Kritikos, J D Farelli, H Todor, K Tong, H Kimsey, 10.1016/j.cels.2016.12.013Cell Syst. 4Koo, B. M., Kritikos, G., Farelli, J. D., Todor, H., Tong, K., Kimsey, H., et al. (2017). Construction and analysis of two genme-scale deletion libraries for Bacillus subtilis. Cell Syst. 4, 291-305.e7. doi: 10.1016/j.cels.2016.12.013

Global regulatory systems operating in bacilysin biosynthesis in Bacillus subtilis. T E Köroglu, I Ogülür, A Mutlu, A Yazgan-Karatas, G Ozcengiz, 10.1159/000328639J. Mol. Microbiol. Biotechnol. 20Köroglu, T. E., Ogülür, I., Mutlu, A., Yazgan-Karatas, A., and Ozcengiz, G. (2011). Global regulatory systems operating in bacilysin biosynthesis in Bacillus subtilis. J. Mol. Microbiol. Biotechnol. 20, 144-155. doi: 10.1159/000328639

Introduction: the diverse and essential biological functions of vitamins. K Kraemer, R D Semba, M Eggersdorfer, D A Schaumberg, 10.1159/000343103Ann. Nutr. Metab. 61Kraemer, K., Semba, R. D., Eggersdorfer, M., and Schaumberg, D. A. (2012). Introduction: the diverse and essential biological functions of vitamins. Ann. Nutr. Metab. 61, 185-191. doi: 10.1159/000343103

GTP dysregulation in Bacillus subtilis cells lacking (p)ppGpp results in phenotypic amino acid auxotrophy and failure to adapt to nutrient downshift and regulate biosyntiesis genes. A Kriel, S R Brinsmade, J L Tse, A K Tehranchi, A N Bittner, A L Sonenshein, 10.1128/JB.00918-13J. Bacteriol. 196Kriel, A., Brinsmade, S. R., Tse, J. L., Tehranchi, A. K., Bittner, A. N., Sonenshein, A. L., et al. (2014). GTP dysregulation in Bacillus subtilis cells lacking (p)ppGpp results in phenotypic amino acid auxotrophy and failure to adapt to nutrient downshift and regulate biosyntiesis genes. J. Bacteriol. 196, 189-201. doi: 10.1128/JB.00918-13

PipY, a member of the conserved COG0325 family of PLP-binding proteins, expands the cyanobacterial nitrogen regulatory network. J I Labella, R Cantos, J Espinosa, A Forcada-Nadal, V Rubio, A Contreras, 10.3389/fmicb.2017.01244Front. Microbiol. 81244Labella, J. I., Cantos, R., Espinosa, J., Forcada-Nadal, A., Rubio, V., and Contreras, A. (2017). PipY, a member of the conserved COG0325 family of PLP-binding proteins, expands the cyanobacterial nitrogen regulatory network. Front. Microbiol. 8:1244. doi: 10.3389/fmicb.2017.01244

Vitamin B6 biosynthesis: formation of pyridoxine 5 ′ -phosphate from 4-(phosphohydroxy)-L-threonine and 1-deoxy-D-xylulose-5-phosphate by PdxA and PdxJ protein. B Laber, W Maurer, S Scharf, K Stepusin, F S Schmidt, 10.1016/S0014-5793(99)00393-2FEBS Lett. 449Laber, B., Maurer, W., Scharf, S., Stepusin, K., and Schmidt, F. S. (1999). Vitamin B6 biosynthesis: formation of pyridoxine 5 ′ -phosphate from 4- (phosphohydroxy)-L-threonine and 1-deoxy-D-xylulose-5-phosphate by PdxA and PdxJ protein. FEBS Lett. 449, 45-48. doi: 10.1016/S0014-5793(99)0 0393-2

Inhibition of acetyl-CoA carboxylase isoforms by pyridoxal phosphate. W M Lee, J E Elliot, R W Brownsey, 10.1074/jbc.M510728200J. Biol. Chem. 280Lee, W. M., Elliot, J. E., and Brownsey, R. W. (2005). Inhibition of acetyl-CoA carboxylase isoforms by pyridoxal phosphate. J. Biol. Chem. 280, 41385-41843. doi: 10.1074/jbc.M510728200

The x-ray crystal structure of lysine-2,3-aminomutase from Clostridium subterminale. B W Lepore, F J Ruzicka, P A Frey, Ringe , D , 10.1073/pnas.0505726102Proc. Natl. Acad. Sci. U.S.A. 102Lepore, B. W., Ruzicka, F. J., Frey, P. A., and Ringe, D. (2005). The x-ray crystal structure of lysine-2,3-aminomutase from Clostridium subterminale. Proc. Natl. Acad. Sci. U.S.A. 102, 13829-13824. doi: 10.1073/pnas.0505726102

Nothing of chemistry disappears in biology": the top 30 damage-prone endogenous metabolites. C Lerma-Ortiz, J G Jeffryes, A J Cooper, T D Niehaus, A M Thamm, O Frelin, 10.1042/BST20160073Biochem. Soc. Trans. 44Lerma-Ortiz, C., Jeffryes, J. G., Cooper, A. J., Niehaus, T. D., Thamm, A. M., Frelin, O., et al. (2016). "Nothing of chemistry disappears in biology": the top 30 damage-prone endogenous metabolites. Biochem. Soc. Trans. 44, 961-971. doi: 10.1042/BST20160073

Deficiency of PdxR in Streptococcus mutans affects vitamin B6 metabolism, acid tolerance response and biofilm formation. S Liao, J P Bitoun, A H Nguyen, D Bozner, X Yao, Z T Wen, 10.1111/omi.12090Mol. Oral Microbiol. 30Liao, S., Bitoun, J. P., Nguyen, A. H., Bozner, D., Yao, X., and Wen, Z. T. (2015). Deficiency of PdxR in Streptococcus mutans affects vitamin B6 metabolism, acid tolerance response and biofilm formation. Mol. Oral Microbiol. 30, 255-268. doi: 10.1111/omi.12090

Metabolite damage and its repair of pre-emption. C L Linster, E Van Schaftingen, A D Hanson, 10.1038/nchembio.1141Nat. Chem. Biol. 9Linster, C. L., Van Schaftingen, E., and Hanson, A. D. (2013). Metabolite damage and its repair of pre-emption. Nat. Chem. Biol. 9, 72-80. doi: 10.1038/nchembio.1141

A glycine-dependent riboswitch that uses cooperative binding to control gene expression. M Mandal, M Lee, J E Barrick, Z Weinberg, G M Emilsson, W L Ruzzo, 10.1126/science.1100829Science. 306Mandal, M., Lee, M., Barrick, J. E., Weinberg, Z., Emilsson, G. M., Ruzzo, W. L., et al. (2004). A glycine-dependent riboswitch that uses cooperative binding to control gene expression. Science 306, 275-279. doi: 10.1126/science.1100829

Exopolymeric substances (EPS) from Bacillus subtilis: polymers and genes encoding their synthesis. M Marvasi, P T Visscher, Casillas Martinez, L , 10.1111/j.1574-6968.2010.02085.xFEMS Microbiol. Lett. 313Marvasi, M., Visscher, P. T., and Casillas Martinez, L. (2010). Exopolymeric substances (EPS) from Bacillus subtilis: polymers and genes encoding their synthesis. FEMS Microbiol. Lett. 313, 1-9. doi: 10.1111/j.1574-6968.2010.02085.x

The molecular structure of hyperthermostabole aromatic aminotransferase with novel substrate specificity from Pyrococcus horikoshii. I Matsui, E Matsui, Y Sakai, H Kikuchi, Y Kawarabayasi, H Ura, 10.1074/jbc.275.7.4871J. Biol. Chem. 275Matsui, I., Matsui, E., Sakai, Y., Kikuchi, H., Kawarabayasi, Y., Ura, H., et al. (2000). The molecular structure of hyperthermostabole aromatic aminotransferase with novel substrate specificity from Pyrococcus horikoshii. J. Biol. Chem. 275, 4871-4879. doi: 10.1074/jbc.275.7.4871

Genetic structure of the D-serine deaminase system of Escherichia coli. E Mcfall, 10.1016/S0022-2836(64)80179-0J. Mol. Biol. 9McFall, E. (1964). Genetic structure of the D-serine deaminase system of Escherichia coli. J. Mol. Biol. 9, 746-753. doi: 10.1016/S0022-2836(64)80179-0

The molecular evolution of pyridoxal-5 ′ -phosphate-dependent enzymes. P K Mehta, Christen , P , 10.1002/9780470123201.ch4Adv. Enzymol. Relat. Areas Mol. Biol. 74Mehta, P. K., and Christen, P. (2000). The molecular evolution of pyridoxal- 5 ′ -phosphate-dependent enzymes. Adv. Enzymol. Relat. Areas Mol. Biol. 74, 129-184. doi: 10.1002/9780470123201.ch4

Aminotransferase: demonstration of homology and division into evolutionary subgroups. P K Mehta, T I Hale, Christen , P , 10.1111/j.1432-1033.1993.tb17953.xEur. J. Biochem. 214Mehta, P. K., Hale, T. I., and Christen, P. (1993). Aminotransferase: demonstration of homology and division into evolutionary subgroups. Eur. J. Biochem. 214, 549-561. doi: 10.1111/j.1432-1033.1993.tb17953.x

Bacterial cysteine desulfurases: theis function and mechanisms. H Mihara, N Esaki, 10.1007/s00253-002-1107-4Appl. Microbiol. Biotechnol. 60Mihara, H., and Esaki, N. (2002). Bacterial cysteine desulfurases: theis function and mechanisms. Appl. Microbiol. Biotechnol. 60, 12-23. doi: 10.1007/s00253-002-1107-4

The aspartate aminotransferase-like domain of Firmicutes MocR transcriptional retulators. T Milano, R Contestabile, A Lo Presti, M Ciccozzi, S Pascarella, 10.1016/j.compbiolchem.2015.05.003Comput. Biol. Chem. 58Milano, T., Contestabile, R., Lo Presti, A., Ciccozzi, M., and Pascarella, S. (2015). The aspartate aminotransferase-like domain of Firmicutes MocR transcriptional retulators. Comput. Biol. Chem. 58, 55-61. doi: 10.1016/j.compbiolchem.2015.05.003

Molecular dynamics simulation unveils the conformational flexibility of the interdomain linker in the bacterial transcriptional regulator GabR from Bacillus subtilis bound to pyridoxal 5 ′ -phosphate. T Milano, A Gulzar, D Narzi, L Guidoni, S Pascarella, 10.1371/journal.pone.0189270PLoS ONE. 12189270Milano, T., Gulzar, A., Narzi, D., Guidoni, L., and Pascarella, S. (2017). Molecular dynamics simulation unveils the conformational flexibility of the interdomain linker in the bacterial transcriptional regulator GabR from Bacillus subtilis bound to pyridoxal 5 ′ -phosphate. PLoS ONE 12:e0189270. doi: 10.1371/journal.pone.0189270

Production of kanosamine by Bacillus cereus UW85. J L Milner, L Silo-Suh, J C Lee, H He, J Clardy, J Handelsman, Appl. Environ. Microbiol. 62Milner, J. L., Silo-Suh, L., Lee, J. C., He, H., Clardy, J., and Handelsman, J. (1996). Production of kanosamine by Bacillus cereus UW85. Appl. Environ. Microbiol. 62, 3061-3065.

. N Mirouze, E Bidnenko, P Noirot, S Auger, Mirouze, N., Bidnenko, E., Noirot, P., and Auger, S. (2015).

Genome-wide mapping of TnrA-binding sites provides new insights into the TnrA regulon in Bacillus subtilis. 10.1002/mbo3.249doi: 10.1002/ mbo3.2494Genome-wide mapping of TnrA-binding sites provides new insights into the TnrA regulon in Bacillus subtilis. Microbiologyopen 4, 423-435. doi: 10.1002/ mbo3.249

Phylogenetic analyses and comparative genomics of vitamin B6 (pyridoxine) and pyridoxal phosphate biosynthesis pathways. G Mittenhuber, J. Mol. Microbiol. Biotechnol. 3Mittenhuber, G. (2001). Phylogenetic analyses and comparative genomics of vitamin B6 (pyridoxine) and pyridoxal phosphate biosynthesis pathways. J. Mol. Microbiol. Biotechnol. 3, 1-20.

Cloning and characterization of the thiD/J gene of Escherichia coli encoding a thiamin-synthesizing bifunctional enzyme, hydroxymethylpyrimidine kinase/phosphomethylpyrimidine kinase. T Mizote, M Tsuda, D D Smith, H Nakayama, T Nakazawa, 10.1099/13500872-145-2-495Microbiology. 145Mizote, T., Tsuda, M., Smith, D. D., Nakayama, H., and Nakazawa, T. (1999). Cloning and characterization of the thiD/J gene of Escherichia coli encoding a thiamin-synthesizing bifunctional enzyme, hydroxymethylpyrimidine kinase/phosphomethylpyrimidine kinase. Microbiology 145, 495-501. doi: 10.1099/13500872-145-2-495

Pyridoxal 5 ′ -phosphate is a selective inhibitor in vivo of DNA polymerase alpha and epsilon. Y Mizushina, X Xu, K Matsubara, C Murakami, I Kuriyama, M Oshige, 10.1016/j.bbrc.2003.11.027Biochem. Biophys. Res. Commun. 312Mizushina, Y., Xu, X., Matsubara, K., Murakami, C., Kuriyama, I., Oshige, M., et al. (2003). Pyridoxal 5 ′ -phosphate is a selective inhibitor in vivo of DNA polymerase alpha and epsilon. Biochem. Biophys. Res. Commun. 312, 1025-1032. doi: 10.1016/j.bbrc.2003.11.027

The pseudoenzyme PDX1.2 boosts vitamin B6 biosynthesis under heat and oxidative stress in Arabidopsis. C Moccand, S Boycheva, P Surriabre, M Tambasco-Studart, M Raschke, M Kaufmann, 10.1074/jbc.M113.540526J. Biol. Chem. 289Moccand, C., Boycheva, S., Surriabre, P., Tambasco-Studart, M., Raschke, M., Kaufmann, M., et al. (2014). The pseudoenzyme PDX1.2 boosts vitamin B6 biosynthesis under heat and oxidative stress in Arabidopsis. J. Biol. Chem. 289, 8203-8216. doi: 10.1074/jbc.M113.540526

The Spo0A regulon of Bacillus subtilis. V Molle, Y Fujita, S T Jensen, P Eichenberger, J E González-Pastor, J S Liu, 10.1046/j.1365-2958.2003.03818.xMol. Microbiol. 50Molle, V., Fujita, Y., Jensen, S. T., Eichenberger, P., González-Pastor, J. E., Liu, J. S., et al. (2003b). The Spo0A regulon of Bacillus subtilis. Mol. Microbiol. 50, 1683-1701. doi: 10.1046/j.1365-2958.2003. 03818.x

Additional targets of the Bacillus subtilis global regulator CodY identified by chromatin immunoprecipitation and genome-wide transcript analysis. V Molle, Y Nakaura, R P Shivers, H Yamaguchi, R Losick, Y Fujita, 10.1128/JB.185.6.1911-1922.2003J. Bacteriol. 185Molle, V., Nakaura, Y., Shivers, R. P., Yamaguchi, H., Losick, R., Fujita, Y., et al. (2003a). Additional targets of the Bacillus subtilis global regulator CodY identified by chromatin immunoprecipitation and genome-wide transcript analysis. J. Bacteriol. 185, 1911-1922. doi: 10.1128/JB.185.6.1911- 1922.2003

Vitamin B6: killing two birds with one stone. S Mooney, H Hellmann, 10.1016/j.phytochem.2009.12.015Phytochemistry. 71Mooney, S., and Hellmann, H. (2010). Vitamin B6: killing two birds with one stone. Phytochemistry 71, 495-501. doi: 10.1016/j.phytochem.2009.12.015

Vitamin B6: a long known compound of surprising complexity. S Mooney, J E Leuendorf, C Hendrickson, H Hellmann, 10.3390/molecules14010329Molecules. 14Mooney, S., Leuendorf, J. E., Hendrickson, C., and Hellmann, H. (2009). Vitamin B6: a long known compound of surprising complexity. Molecules 14, 329-351. doi: 10.3390/molecules14010329

Cloning of the pyridoxine 5 ′ -phosphatase gene (pdxP) and vitamin B6 production in pdxP recombinant Sinorhizobium meliloti. Y Nagahashi, M Tazoe, T Hoshino, 10.1271/bbb.70539Biosci. Biotechnol. Biochem. 72Nagahashi, Y., Tazoe, M., and Hoshino, T. (2008). Cloning of the pyridoxine 5 ′ -phosphatase gene (pdxP) and vitamin B6 production in pdxP recombinant Sinorhizobium meliloti. Biosci. Biotechnol. Biochem. 72, 421-427. doi: 10.1271/bbb.70539

A regulatory protein that interferes with activator-stimulated transcription in bacteria. S Nakano, M M Nakano, Y Zhang, M Lellakriangsak, P Zuber, 10.1073/pnas.0637648100Proc. Natl. Acad. Sci. U.S.A. 100Nakano, S., Nakano, M. M., Zhang, Y., Lellakriangsak, M., and Zuber, P. (2003). A regulatory protein that interferes with activator-stimulated transcription in bacteria. Proc. Natl. Acad. Sci. U.S.A. 100, 4233-4238. doi: 10.1073/pnas.0637648100

An enzyme common to histidine and aromatic amino acid biosynthesis in Bacillus subtilis. E W Nester, A L Montoya, J. Bacteriol. 126Nester, E. W., and Montoya, A. L. (1976). An enzyme common to histidine and aromatic amino acid biosynthesis in Bacillus subtilis. J. Bacteriol. 126, 699-705.

Cloning, purification and preliminary crystallographic analysis of a putative pyridoxal kinase from Bacillus subtilis. J A Newman, S K Das, S E Sedelnikova, D W Rice, 10.1107/S1744309106035779Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun. 62Newman, J. A., Das, S. K., Sedelnikova, S. E., and Rice, D. W. (2006a). Cloning, purification and preliminary crystallographic analysis of a putative pyridoxal kinase from Bacillus subtilis. Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun. 62, 1006-1009. doi: 10.1107/S1744309106035779

The crystal structure of an ADP complex of Bacillus subtilis pyridoxal kinase provides evidence for the parallel emergence of enzyme activity during evolution. J A Newman, S K Das, S E Sedelnikova, D W Rice, 10.1016/j.jmb.2006.08.013J. Mol. Biol. 363Newman, J. A., Das, S. K., Sedelnikova, S. E., and Rice, D. W. (2006b). The crystal structure of an ADP complex of Bacillus subtilis pyridoxal kinase provides evidence for the parallel emergence of enzyme activity during evolution. J. Mol. Biol. 363, 520-530. doi: 10.1016/j.jmb.2006.08.013

Condition-dependent transcriptome reveals highlevel regulatory architecture in Bacillus subtilis. P Nicolas, U Mäder, E Dervxn, T Rochat, A Leduc, N Pigeonneau, 10.1126/science.1206848Science. 335Nicolas, P., Mäder, U., Dervxn, E., Rochat, T., Leduc, A., Pigeonneau, N., et al. (2012). Condition-dependent transcriptome reveals high- level regulatory architecture in Bacillus subtilis. Science 335, 1103-1106. doi: 10.1126/science.1206848

Systems-wide prediction of enzyme promiscuity reveals a new underground alternative route for pyridoxal 5 ′ -phosphate production in E. coli. M A Oberhardt, R Zarecki, L Reshef, F Xia, M Druan-Frigola, R Schreiber, 10.1371/journal.pcbi.1004705PLoS Comput. Biol. 121004705Oberhardt, M. A., Zarecki, R., Reshef, L., Xia, F., Druan-Frigola, M., Schreiber, R., et al. (2016). Systems-wide prediction of enzyme promiscuity reveals a new underground alternative route for pyridoxal 5 ′ -phosphate production in E. coli. PLoS Comput. Biol. 12:e1004705. doi: 10.1371/journal.pcbi.1004705

Role of prokaryotic type I and III pantothenate kinases in the coenzyme A biosynthesis pathway. Y Ogata, H Katoh, M Asayama, S Chohnan, 10.1139/cjm-2013-0793Can. J. Microbiol. 60Ogata, Y., Katoh, H., Asayama, M., and Chohnan, S. (2014). Role of prokaryotic type I and III pantothenate kinases in the coenzyme A biosynthesis pathway. Can. J. Microbiol. 60, 297-305. doi: 10.1139/cjm-2013-0793

Newly identified nucleoid-associated-like protein YlxR regulates metabolic gene expression in Bacillus subtilis. M Ogura, Y Kanesaki, 10.1128/mSphere.00501-183Ogura, M., and Kanesaki, Y. (2018). Newly identified nucleoid-associated-like protein YlxR regulates metabolic gene expression in Bacillus subtilis. mSphere 3, e00501-e00518. doi: 10.1128/mSphere.00501-18

Structure-function relationship in Escherichia coli initiation factors. Identification of a lysine residue in the ribosomal binding site of initiation factor by site-specific chemical modification with pyridoxal phosphate. H Ohsawa, C Gualerzi, J. Biol. Chem. 256Ohsawa, H., and Gualerzi, C. (1981). Structure-function relationship in Escherichia coli initiation factors. Identification of a lysine residue in the ribosomal binding site of initiation factor by site-specific chemical modification with pyridoxal phosphate. J. Biol. Chem. 256, 4905-4912.

Pyridoxal 5 ′ -phosphate inhibits DNA binding of HNF1. T Oka, H Sugitatsu, H Nordin, M K Thakur, M Aoyama, T Sasagawa, 10.1016/S0304-4165(01)00221-5Biochim. Biophys. Acta. 1568Oka, T., Sugitatsu, H., Nordin, H., Thakur, M. K., Aoyama, M., Sasagawa, T., et al. (2001). Pyridoxal 5 ′ -phosphate inhibits DNA binding of HNF1. Biochim. Biophys. Acta 1568, 189-196. doi: 10.1016/S0304-4165(01)00221-5

Domain characterization of Bacillus subtilis GabR, a pyridoxal 5 ′ -phosphate-dependent transcriptional regulator. K Okuda, T Ito, M Goto, T Takenaka, H Hemmi, Yoshimura , T , 10.1093/jb/mvv040J. Biochem. 158Okuda, K., Ito, T., Goto, M., Takenaka, T., Hemmi, H., and Yoshimura, T. (2015b). Domain characterization of Bacillus subtilis GabR, a pyridoxal 5 ′ - phosphate-dependent transcriptional regulator. J. Biochem. 158, 225-234. doi: 10.1093/jb/mvv040

Role of the aminotransferase domain in Bacillus subtilis GabR, a pyridoxal 5 ′ -phosphate-dependent transcriptional regulator. K Okuda, S Kato, T Ito, S Shiraki, Y Kawase, K Goto, 10.1111/mmi.12861Mol. Microbiol. 95Okuda, K., Kato, S., Ito, T., Shiraki, S., Kawase, Y., Goto, K., et al. (2015a). Role of the aminotransferase domain in Bacillus subtilis GabR, a pyridoxal 5 ′ - phosphate-dependent transcriptional regulator. Mol. Microbiol. 95, 245-257. doi: 10.1111/mmi.12861

Characterization of two kinases involved in thiamine pyrophosphate and pyridoxal phosphate biosynthesis in Bacillus subtilis: 4-amino-5-hydroxymethyl-2-methylpyrimidine kinase and pyridoxal kinase. J H Park, K Burns, C Kinsland, T P Begley, 10.1128/JB.186.5.1571-1573.2004J. Bacteriol. 186Park, J. H., Burns, K., Kinsland, C., and Begley, T. P. (2004). Characterization of two kinases involved in thiamine pyrophosphate and pyridoxal phosphate biosynthesis in Bacillus subtilis: 4-amino-5-hydroxymethyl-2- methylpyrimidine kinase and pyridoxal kinase. J. Bacteriol. 186, 1571-1573. doi: 10.1128/JB.186.5.1571-1573.2004

Post-translational modification by β-lysylation is required for activity of Escherichia coli elongation factor P (EF-P). J H Park, H E Johansson, H Aoki, B X Huang, H Y Kim, M C Ganoza, 10.1074/jbc.M111.309633J. Biol. Chem. 287Park, J. H., Johansson, H. E., Aoki, H., Huang, B. X., Kim, H. Y., Ganoza, M. C., et al. (2012). Post-translational modification by β-lysylation is required for activity of Escherichia coli elongation factor P (EF-P). J. Biol. Chem. 287, 2579-2590. doi: 10.1074/jbc.M111.309633

Crystal structure of the Cterminal domain of Bacillus subtilis GabR reveals a closed conformation by γ-aminobutyric acid binding, inducing transcriptional activation. S A Park, Y S Park, K S Lee, 10.1016/j.bbrc.2017.04.052Biochem. Biophys. Res. Commun. 487Park, S. A., Park, Y. S., and Lee, K. S. (2017). Crystal structure of the C- terminal domain of Bacillus subtilis GabR reveals a closed conformation by γ-aminobutyric acid binding, inducing transcriptional activation. Biochem. Biophys. Res. Commun. 487, 287-291. doi: 10.1016/j.bbrc.2017.04.052

Vitamin B6 and its role in cell metabolism and physiology. M Parra, S Stahl, H Hellmann, 10.3390/cells7070084Cells. 784Parra, M., Stahl, S., and Hellmann, H. (2018). Vitamin B6 and its role in cell metabolism and physiology. Cells 7:84. doi: 10.3390/cells7070084

Evolution of biosynthetic pathways: a common ancestor for threonine synthase, threonine dehydratase and D-serine dehydratase. C Parsot, 10.1002/j.1460-2075.1986.tb04600.xEMBO J. 5Parsot, C. (1986). Evolution of biosynthetic pathways: a common ancestor for threonine synthase, threonine dehydratase and D-serine dehydratase. EMBO J. 5, 3013-3019. doi: 10.1002/j.1460-2075.1986.tb04600.x

Vitamin B6. H Pauling, B J. ; H J Weimann, E Arpe, H T Biekert, W Davis, H Gerhartz, W Gerrens, J L Keim, A Mcguire, H Mitsutani, C Pilat, H E Reece, E Simmons, R Weise, H R Wirtz, Wüthrich, Ullmann's Encyclopedia of Industrial Chemistry. WeinheimWiley-VCH27Pauling, H., and Weimann, B. J. (1996). "Vitamin B6, " in Ullmann's Encyclopedia of Industrial Chemistry, Vol. A27, eds H. J. Arpe, E. Biekert, H. T. Davis, W. Gerhartz, H. Gerrens, W. Keim, J. L. McGuire, A. Mitsutani, H. Pilat, C. Reece, H. E. Simmons, E. Weise, R. Wirtz, and H. R. Wüthrich (Weinheim: Wiley-VCH), 530-540.

A genomic overview of pyridoxal-phosphate-dependent enzymes. R Percudani, A Peracchi, 10.1038/sj.embor.embor914EMBO Rep. 4Percudani, R., and Peracchi, A. (2003). A genomic overview of pyridoxal-phosphate-dependent enzymes. EMBO Rep. 4, 850-854. doi: 10.1038/sj.embor.embor914

The B6 database: a tool for the description and classification of vitamin B6-dependent enzymatic activities and of the corresponding protein families. R Percudani, A Peracchi, 10.1186/1471-2105-10-273BMC Bioinformatics. 10273Percudani, R., and Peracchi, A. (2009). The B6 database: a tool for the description and classification of vitamin B6-dependent enzymatic activities and of the corresponding protein families. BMC Bioinformatics 10:273. doi: 10.1186/1471-2105-10-273

Identification and characterization of transcripts from the biotin biosynthetic operon of Bacillus subtilis. J B Perkins, S Bower, C L Howitt, R R Yocum, J Pero, 10.1128/jb.178.21.6361-6365.1996J. Bacteriol. 178Perkins, J. B., Bower, S., Howitt, C. L., Yocum, R. R., and Pero, J. (1996). Identification and characterization of transcripts from the biotin biosynthetic operon of Bacillus subtilis. J. Bacteriol. 178, 6361-6365. doi: 10.1128/jb.178.21.6361-6365.1996

Vitamin B6 biosynthesis in Bacillus subtilis. W Pflug, F Lingens, 10.1515/bchm.1978.359.1.559Hoppe Seylers Z. Physiol. Chem. 359Pflug, W., and Lingens, F. (1978). Vitamin B6 biosynthesis in Bacillus subtilis. Hoppe Seylers Z. Physiol. Chem. 359, 559-570. doi: 10.1515/bchm.1978.359.1.559

Chemistry and diversity of pyridoxal-5 ′ -phosphate dependent enzymes. R S Phillips, 10.1016/j.bbapap.2014.12.028Biochim. Biophys. Acta. 1854Phillips, R. S. (2015). Chemistry and diversity of pyridoxal-5 ′ -phosphate dependent enzymes. Biochim. Biophys. Acta 1854, 1167-1174. doi: 10.1016/j.bbapap.2014.12.028

Gene cloning and characterization of a second alanine racemase from Bacillus subtilis encoded by yncD. K J Pierce, S P Salifu, M Tangney, 10.1111/j.1574-6968.2008.01151.xFEMS Microbiol. Lett. 283Pierce, K. J., Salifu, S. P., and Tangney, M. (2008). Gene cloning and characterization of a second alanine racemase from Bacillus subtilis encoded by yncD. FEMS Microbiol. Lett. 283, 69-74. doi: 10.1111/j.1574-6968.2008.01151.x

Evidence that COG0325 proteins are involved in PLP homeostasis. L Prunetti, B El Yacoubi, C R Schiavon, E Kirkpatrick, L Huang, M Bailly, 10.1099/mic.0.000255Microbiology. 162Prunetti, L., El Yacoubi, B., Schiavon, C. R., Kirkpatrick, E., Huang, L., Bailly, M., et al. (2016). Evidence that COG0325 proteins are involved in PLP homeostasis. Microbiology 162, 694-706. doi: 10.1099/mic.0.000255

Modified mariner transposons for random inducible-expression insertions and transcriptional reporter fusion insertions in Bacillus subtilis. E R Pozsgai, K M Blair, D B Kearns, 10.1128/AEM.07098-11Appl. Environ. Microbiol. 78Pozsgai, E. R., Blair, K. M., and Kearns, D. B. (2012). Modified mariner transposons for random inducible-expression insertions and transcriptional reporter fusion insertions in Bacillus subtilis. Appl. Environ. Microbiol. 78, 778-785. doi: 10.1128/AEM.07098-11

Functional analysis of Bacillus subtilis genes involved in the biosynthesis of 4-thiouridine in tRNA. L J Rajakovich, J Tomlinson, Dos Santos, P C , 10.1128/JB.00842-12J. Bacteriol. 194Rajakovich, L. J., Tomlinson, J., and Dos Santos, P. C. (2012). Functional analysis of Bacillus subtilis genes involved in the biosynthesis of 4-thiouridine in tRNA. J. Bacteriol. 194, 4933-4940. doi: 10.1128/JB.00842-12

An aminotransferase branch point connects purine catabolism to amino acid recycling. I Ramazzina, R Costa, L Cendron, R Berni, A Peracchi, G Zanotti, 10.1038/nchembio.445Nat. Chem. Biol. 6Ramazzina, I., Costa, R., Cendron, L., Berni, R., Peracchi, A., Zanotti, G., et al. (2010). An aminotransferase branch point connects purine catabolism to amino acid recycling. Nat. Chem. Biol. 6, 801-806. doi: 10.1038/nchembio.445

On the two components of pyridoxal 5 ′ -phosphate synthase from Bacillus subtilis. T Raschle, N Amrhein, T B Fitzpatrick, 10.1074/jbc.M501356200J. Biol. Chem. 280Raschle, T., Amrhein, N., and Fitzpatrick, T. B. (2005). On the two components of pyridoxal 5 ′ -phosphate synthase from Bacillus subtilis. J. Biol. Chem. 280, 32291-32300. doi: 10.1074/jbc.M501356200

Catabolism of amino acids and related compounds. L Reitzer, 10.1128/ecosalplus.3.4.7EcoSal Plus. 1Reitzer, L. (2005). Catabolism of amino acids and related compounds. EcoSal Plus 1, 1-56. doi: 10.1128/ecosalplus.3.4.7

The blueprint of a minimal cell: miniBacillus. Microbiol. D R Reuss, F M Commichau, J Gundlach, B Zhu, J Stülke, 10.1128/MMBR.00029-16Mol. Biol. Rev. 80Reuss, D. R., Commichau, F. M., Gundlach, J., Zhu, B., and Stülke, J. (2016). The blueprint of a minimal cell: miniBacillus. Microbiol. Mol. Biol. Rev. 80, 955-987. doi: 10.1128/MMBR.00029-16

An RNA pyrophosphohydrolase triggers 5 ′ exonucleolytic degradation of mRNA in B. subtilis. J Richards, Q Liu, O Pellegrini, H Celesnik, S Yao, D H Bechhofer, 10.1016/j.molcel.2011.07.023Mol. Cell. 43Richards, J., Liu, Q., Pellegrini, O., Celesnik, H., Yao, S., Bechhofer, D. H., et al. (2011). An RNA pyrophosphohydrolase triggers 5 ′ exonucleolytic degradation of mRNA in B. subtilis. Mol. Cell 43, 940-949. doi: 10.1016/j.molcel.201 1.07.023

Mechanisms and structures of vitamin B6-dependent enzymes involved in deoxy sugar biosynthesis. A J Romo, H W Liu, 10.1016/j.bbapap.2011.02.003Biochim. Biophys. Acta. 1814Romo, A. J., and Liu, H. W. (2011). Mechanisms and structures of vitamin B6- dependent enzymes involved in deoxy sugar biosynthesis. Biochim. Biophys. Acta 1814, 1534-1547. doi: 10.1016/j.bbapap.2011.02.003

A history of the isolation and identification of vitamin B6. I H Rosenberg, 10.1159/000343113Ann. Nutr. Metab. 61Rosenberg, I. H. (2012). A history of the isolation and identification of vitamin B6. Ann. Nutr. Metab. 61, 236-238. doi: 10.1159/000343113

Harnessing underground metabolism for pathway development. J Rosenberg, F M Commichau, 10.1016/j.tibtech.2018.08.001Trends Biotechnol. 37Rosenberg, J., and Commichau, F. M. (2019). Harnessing underground metabolism for pathway development. Trends Biotechnol. 37, 29-37. doi: 10.1016/j.tibtech.2018.08.001

Vitamin B6 metabolism in microbes and approaches for fermentative production. J Rosenberg, T Ischebeck, F M Commichau, 10.1016/j.biotechadv.2016.11.004Biotechnol. Adv. 35Rosenberg, J., Ischebeck, T., and Commichau, F. M. (2017). Vitamin B6 metabolism in microbes and approaches for fermentative production. Biotechnol. Adv. 35, 31-40. doi: 10.1016/j.biotechadv.2016.11.004

ThrR, a DNA-binding transcription factor involved in controlling threonine biosynthesis in Bacillus subtilis. J Rosenberg, P Müller, S Lentes, M J Thiele, D R Zeigler, D Tödter, 10.1111/mmi.13429Mol. Microbiol. 101Rosenberg, J., Müller, P., Lentes, S., Thiele, M. J., Zeigler, D. R., Tödter, D., et al. (2016). ThrR, a DNA-binding transcription factor involved in controlling threonine biosynthesis in Bacillus subtilis. Mol. Microbiol. 101, 879-893. doi: 10.1111/mmi.13429

A two-step evolutionary process establishes a non-native vitamin B6 pathway in Bacillus subtilis. J Rosenberg, K C Yeak, F M Commichau, 10.1111/1462-2920.13950Environ. Microbiol. 20Rosenberg, J., Yeak, K. C., and Commichau, F. M. (2018). A two-step evolutionary process establishes a non-native vitamin B6 pathway in Bacillus subtilis. Environ. Microbiol. 20, 156-168. doi: 10.1111/1462-2920.13950

YrxA is the transcriptional regulator that represses de novo NAD biosynthesis in Bacillus subtilis. P Rossolillo, I Marinoni, E Galli, A Colosimo, Albertini , A M , 10.1128/JB.187.20.7155-7160.2005J. Bacteriol. 187Rossolillo, P., Marinoni, I., Galli, E., Colosimo, A., and Albertini, A. M. (2005). YrxA is the transcriptional regulator that represses de novo NAD biosynthesis in Bacillus subtilis. J. Bacteriol. 187, 7155-7160. doi: 10.1128/JB.187.20.7155-7160.2005

Multiple turnovers of the nicotinoenzyme PdxB require α-keto acids as cosubstrates. J Rudolph, J Kim, S D Copley, 10.1021/bi101291dBiochemistry. 49Rudolph, J., Kim, J., and Copley, S. D. (2010). Multiple turnovers of the nicotino- enzyme PdxB require α-keto acids as cosubstrates. Biochemistry 49, 9249-9255. doi: 10.1021/bi101291d

An early cytoplasmatic step of peptidoglycan synthesis is associated to MreB in Bacillus subtilis. A S Rueff, A Chastanet, J Domínguez-Escobar, Z Yao, J Yates, M V Prejean, 10.1111/mmi.12467Mol. Microbiol. 91Rueff, A. S., Chastanet, A., Domínguez-Escobar, J., Yao, Z., Yates, J., Prejean, M. V., et al. (2014). An early cytoplasmatic step of peptidoglycan synthesis is associated to MreB in Bacillus subtilis. Mol. Microbiol. 91, 348-362. doi: 10.1111/mmi.12467

A bacterial checkpoint protein for ribosome assembly moonlights as an essential metabolite proofreading enzyme. A J Sachla, J D Helmann, 10.1038/s41467-019-09508-zNat. Commun. 101526Sachla, A. J., and Helmann, J. D. (2019). A bacterial checkpoint protein for ribosome assembly moonlights as an essential metabolite proofreading enzyme. Nat. Commun. 10:1526. doi: 10.1038/s41467-019-09508-z

yaaD and yaaE are involved in vitamin B6 biosynthesis in Bacillus subtilis. A Sakai, M Kita, T Katsuragi, N Ogasawara, Tani , Y , 10.1263/jbb.93.309J. Biosci. Bioeng. 93Sakai, A., Kita, M., Katsuragi, T., Ogasawara, N., and Tani, Y. (2002b). yaaD and yaaE are involved in vitamin B6 biosynthesis in Bacillus subtilis. J. Biosci. Bioeng. 93, 309-312. doi: 10.1263/jbb.93.309

serC is involved in vitamin B6 biosynthesis in Escherichia coli but not in Bacillus subtilis. A Sakai, M Kita, T Katsuragi, Tani , Y , 10.1263/jbb.93.334J. Biosci. Bioeng. 93Sakai, A., Kita, M., Katsuragi, T., and Tani, Y. (2002a). serC is involved in vitamin B6 biosynthesis in Escherichia coli but not in Bacillus subtilis. J. Biosci. Bioeng. 93, 334-337. doi: 10.1263/jbb.93.334

Catalysis of an essential step in vitamin B2 biosynthesis by a consortium of broad spectrum hydrolases. S Sarge, I Haase, B Illarionov, D Laudert, H P Hohmann, A Bacher, 10.1002/cbic.201500352ChemBioChem. 16Sarge, S., Haase, I., Illarionov, B., Laudert, D., Hohmann, H. P., Bacher, A., et al. (2015). Catalysis of an essential step in vitamin B2 biosynthesis by a consortium of broad spectrum hydrolases. ChemBioChem 16, 2466-2469. doi: 10.1002/cbic.201500352

Three-dimensional structure of 2-amino-3-ketobutyrate CoA ligase from Escherichia coli complexed with a PLP-substrate intermediate: inferred reation mechanism. A Schmidt, J Sivaraman, Y Li, R Larocue, J A Barbosa, C Smith, 10.1021/bi002204yBiochemistry. 40Schmidt, A., Sivaraman, J., Li, Y., Larocue, R., Barbosa, J. A., Smith, C., et al. (2001). Three-dimensional structure of 2-amino-3-ketobutyrate CoA ligase from Escherichia coli complexed with a PLP-substrate intermediate: inferred reation mechanism. Biochemistry 40, 5151-5160. doi: 10.1021/bi002204y

Substitution of pyridoxal 5 ′ -phosphate in D-serine dehydratase from Escherichia coli by cofactor analogues provides information on cofactor binding and catalysis. K D Schnackerz, C H Tai, R K Pötsch, P F Cook, 10.1074/jbc.274.52.36935J. Biol. Chem. 274Schnackerz, K. D., Tai, C. H., Pötsch, R. K., and Cook, P. F. (1999). Substitution of pyridoxal 5 ′ -phosphate in D-serine dehydratase from Escherichia coli by cofactor analogues provides information on cofactor binding and catalysis. J. Biol. Chem. 274, 36935-36943. doi: 10.1074/jbc.274.52.36935

Functional anaylsis of 14 genes that constitute the purine catabolic pathway in Bacillus subtilis and evidence for a novel regulon controlled by the PucR transcription activator. A C Schultz, P Nygaard, H H Saxlid, 10.1128/JB.183.11.3293-3302.2001J. Bacteriol. 183Schultz, A. C., Nygaard, P., and Saxlid, H. H. (2001). Functional anaylsis of 14 genes that constitute the purine catabolic pathway in Bacillus subtilis and evidence for a novel regulon controlled by the PucR transcription activator. J. Bacteriol. 183, 3293-3302. doi: 10.1128/JB.183.11.3293-3302.2001

Feeding on compatible solutes: a substrate-induced pathway for uptake and catabolism of ectoines and its genetic control by EnuR. A Schulz, N Stöveken, I M Binzen, T Hoffmann, J Heider, E Bremer, 10.1111/1462-2920.13414Environ. Microbiol. 19Schulz, A., Stöveken, N., Binzen, I. M., Hoffmann, T., Heider, J., and Bremer, E. (2017). Feeding on compatible solutes: a substrate-induced pathway for uptake and catabolism of ectoines and its genetic control by EnuR. Environ. Microbiol. 19, 926-946. doi: 10.1111/1462-2920.13414

Characterization of polyamine synthesis pathway in Bacillus subtilis 168. A Sekowska, P Bertin, A Danchin, 10.1046/j.1365-2958.1998.00979.xMol. Microbiol. 29Sekowska, A., Bertin, P., and Danchin, A. (1998). Characterization of polyamine synthesis pathway in Bacillus subtilis 168. Mol. Microbiol. 29, 851-858. doi: 10.1046/j.1365-2958.1998.00979.x

The methionine salvage pathway in Bacillus subtilis. A Sekowska, A Danchin, 10.1186/1471-2180-2-8BMC Microbiol. 28Sekowska, A., and Danchin, A. (2002). The methionine salvage pathway in Bacillus subtilis. BMC Microbiol. 2:8. doi: 10.1186/1471-2180-2-8

Novel form of transcription attenuation regulates expression of the Bacillus subtilis tryptophan operon. H Shimotsu, M I Kuroda, C Yanofsky, D J Henner, 10.1128/jb.166.2.461-471.1986J. Bacteriol. 166Shimotsu, H., Kuroda, M. I., Yanofsky, C., and Henner, D. J. (1986). Novel form of transcription attenuation regulates expression of the Bacillus subtilis tryptophan operon. J. Bacteriol. 166, 461-471. doi: 10.1128/jb.166.2.461-471.1986

Determination of pyridoxal-5 ′ -phosphate (PLP)-bonding sites in proteins: a peptide mass fingerprinting approach based on diagnostic tandem mass spectral features of PLP-modified peptides. E S Simon, Allison , J , 10.1002/rcm.4270Rapid Commun. Mass Spectrom. 23Simon, E. S., and Allison, J. (2009). Determination of pyridoxal-5 ′ -phosphate (PLP)-bonding sites in proteins: a peptide mass fingerprinting approach based on diagnostic tandem mass spectral features of PLP-modified peptides. Rapid Commun. Mass Spectrom. 23, 3401-3408. doi: 10.1002/rcm.4270

Crystal structure of Escherichia coli PdxA, an enzyme involved in the pyridoxal phosphate biosynthesis pathway. J Sivaraman, Y Li, J Banks, D E Cane, A Matte, M Cygler, 10.1074/jbc.M306344200J. Biol. Chem. 278Sivaraman, J., Li, Y., Banks, J., Cane, D. E., Matte, A., and Cygler, M. (2003). Crystal structure of Escherichia coli PdxA, an enzyme involved in the pyridoxal phosphate biosynthesis pathway. J. Biol. Chem. 278, 43682-43690. doi: 10.1074/jbc.M306344200

A novel family of bacterial dioxygenases that catalyse the hydroxylation of free L-amino acids. S V Smirnov, P M Sokolov, T Kodera, M Sugiyama, M Hibi, S Shimizu, 10.1111/j.1574-6968.2012.02558.xFEMS Microbiol. Lett. 331Smirnov, S. V., Sokolov, P. M., Kodera, T., Sugiyama, M., Hibi, M., Shimizu, S., et al. (2012). A novel family of bacterial dioxygenases that catalyse the hydroxylation of free L-amino acids. FEMS Microbiol. Lett. 331, 97-104. doi: 10.1111/j.1574-6968.2012.02558.x

Crystal structures capture three states in the catalytic cycle of a pyridoxal phosphate (PLP) synthase. A M Smith, W C Brown, E Harms, J L Smith, 10.1074/jbc.M114.626382J. Biol. Chem. 290Smith, A. M., Brown, W. C., Harms, E., and Smith, J. L. (2015). Crystal structures capture three states in the catalytic cycle of a pyridoxal phosphate (PLP) synthase. J. Biol. Chem. 290, 5226-5239. doi: 10.1074/jbc.M114. 626382

Identification of a thiamin-dependent synthase in Escherichia coli required for the formation of the 1-deoxy-D-xylulose 5-phosphate precursor to isoprenoids, thiamin, and pyridoxol. G A Sprenger, U Schorken, T Wiegert, S Grolle, A A De Graaf, S V Taylor, 10.1073/pnas.94.24.12857Proc. Natl. Acad. Sci. U.S.A. 94Sprenger, G. A., Schorken, U., Wiegert, T., Grolle, S., de Graaf, A. A., Taylor, S. V., et al. (1997). Identification of a thiamin-dependent synthase in Escherichia coli required for the formation of the 1-deoxy-D-xylulose 5-phosphate precursor to isoprenoids, thiamin, and pyridoxol. Proc. Natl. Acad. Sci. U.S.A. 94, 12857-12862. doi: 10.1073/pnas.94.24.12857

Genomewide analysis of temporally regulated and compartment-specific gene expression in sporulating cells of Bacillus subtilis. L Steil, M Serrano, A O Henriques, U Völker, 10.1099/mic.0.27493-0Microbiology. 151Steil, L., Serrano, M., Henriques, A. O., and Völker, U. (2005). Genome- wide analysis of temporally regulated and compartment-specific gene expression in sporulating cells of Bacillus subtilis. Microbiology 151, 399-420. doi: 10.1099/mic.0.27493-0

Tpn1p, the plasma membrane vitamin B6 transporter of Saccharomyces cerevisiae. J Stolz, M Vielreicher, 10.1074/jbc.M300949200J. Biol. Chem. 21Stolz, J., and Vielreicher, M. (2003). Tpn1p, the plasma membrane vitamin B6 transporter of Saccharomyces cerevisiae. J. Biol. Chem. 21, 18990-18996. doi: 10.1074/jbc.M300949200

Structure of a bacterial pyridoxal 5 ′ -phosphate synthase complex. M Strohmeier, T Raschle, J Mazurkiewicz, K Rippe, I Sinning, T B Fitzpatrick, 10.1073/pnas.0604950103Proc. Natl. Acad. Sci. U.S.A. 103Strohmeier, M., Raschle, T., Mazurkiewicz, J., Rippe, K., Sinning, I., Fitzpatrick, T. B., et al. (2006). Structure of a bacterial pyridoxal 5 ′ -phosphate synthase complex. Proc. Natl. Acad. Sci. U.S.A. 103, 19284-19289. doi: 10.1073/pnas.0604950103

Identification of YbhA as the pyridoxal 5 ′ -phosphate (PLP) phosphatase in Escherichia coli: importance of PLP homeostasis on the bacterial growth. R Sugimoto, N Saito, T Shimada, K Tanaka, 10.2323/jgam.2017.02.008J. Gen. Appl. Microbiol. 63Sugimoto, R., Saito, N., Shimada, T., and Tanaka, K. (2017). Identification of YbhA as the pyridoxal 5 ′ -phosphate (PLP) phosphatase in Escherichia coli: importance of PLP homeostasis on the bacterial growth. J. Gen. Appl. Microbiol. 63, 362-368. doi: 10.2323/jgam.2017.02.008

Cloning, nucleotide sequence, and regulation of the Bacillus subtilis nadB gene and a nifS-like gene, both of which are essential for NAD biosynthesis. D Sun, P Setlow, 10.1128/jb.175.5.1423-1432.1993J. Bacteriol. 175Sun, D., and Setlow, P. (1993). Cloning, nucleotide sequence, and regulation of the Bacillus subtilis nadB gene and a nifS-like gene, both of which are essential for NAD biosynthesis. J. Bacteriol. 175, 1423-1432. doi: 10.1128/jb.175.5.1423-1432.1993

Comparative genomics of pyridoxal 5 ′ -phosphate-dependent transcription factor regulons in bacteria. I A Suvorova, D A Rodionov, 10.1099/mgen.0.000047Microb. Genom. 247Suvorova, I. A., and Rodionov, D. A. (2016). Comparative genomics of pyridoxal 5 ′ -phosphate-dependent transcription factor regulons in bacteria. Microb. Genom. 2:e000047. doi: 10.1099/mgen.0.000047

Recycling of pyridoxine (vitamin B6) by PUP1 in Arabidopsis. N Szydlowski, L Bürkle, L Pourcel, M Moulin, J Stolz, T B Fitzpatrick, 10.1111/tpj.12195Plant J. 75Szydlowski, N., Bürkle, L., Pourcel, L., Moulin, M., Stolz, J., and Fitzpatrick, T. B. (2013). Recycling of pyridoxine (vitamin B6) by PUP1 in Arabidopsis. Plant J. 75, 40-52. doi: 10.1111/tpj.12195

Locating essential Escherichia coli genes by using mini-Tn10 transposons: the pdxJ operon. H E Takiff, T Baker, T Copeland, S M Chen, D L Court, 10.1128/jb.174.5.1544-1553.1992J. Bacteriol. 174Takiff, H. E., Baker, T., Copeland, T., Chen, S. M., and Court, D. L. (1992). Locating essential Escherichia coli genes by using mini-Tn10 transposons: the pdxJ operon. J. Bacteriol. 174, 1544-1553. doi: 10.1128/jb.174.5.1544-1553.1992

Evolution of vitamin B6 (pyridoxine) metabolism by gain and loss of genes. T Tanaka, Y Tateno, T Gojobori, 10.1093/molbev/msi011Mol. Biol. Evol. 22Tanaka, T., Tateno, Y., and Gojobori, T. (2005). Evolution of vitamin B6 (pyridoxine) metabolism by gain and loss of genes. Mol. Biol. Evol. 22, 243-250. doi: 10.1093/molbev/msi011

The CymR regulator in complex with the enzyme CysK controls cysteine metabolism in Bacillus subtilis. C Tanous, O Soutourina, B Raynal, M F Hullo, P Mervelet, A M Gilles, 10.1074/jbc.M805951200J. Biol. Chem. 283Tanous, C., Soutourina, O., Raynal, B., Hullo, M. F., Mervelet, P., Gilles, A. M., et al. (2008). The CymR regulator in complex with the enzyme CysK controls cysteine metabolism in Bacillus subtilis. J. Biol. Chem. 283, 35551-35560. doi: 10.1074/jbc.M805951200

Purification and characterization of pyridoxine 5 ′ -phosphate phosphatase from Sinorhizobium meliloti. M Tazoe, K Ichikawa, Z Hoshino, 10.1271/bbb.69.2277Biosci. Biotechnol. Biochem. 69Tazoe, M., Ichikawa, K., and Hoshino, Z. (2005). Purification and characterization of pyridoxine 5 ′ -phosphate phosphatase from Sinorhizobium meliloti. Biosci. Biotechnol. Biochem. 69, 2277-2284. doi: 10.1271/bbb.69.2277

Flavin adenine dinucleotidedependent 4-phospho-D-erythronate dehydrogenase is responsible for the 4-phosphohydroxy-L-threonine pathway in vitamin B6 biosynthesis in Sinorhizobium meliloti. M Tazoe, K Ichikawa, Z Hoshino, 10.1128/JB.01999-05J. Bacteriol. 188Tazoe, M., Ichikawa, K., and Hoshino, Z. (2006). Flavin adenine dinucleotide- dependent 4-phospho-D-erythronate dehydrogenase is responsible for the 4-phosphohydroxy-L-threonine pathway in vitamin B6 biosynthesis in Sinorhizobium meliloti. J. Bacteriol. 188, 4635-4645. doi: 10.1128/JB.01999-05

Members of a novel kinase family (DUF1537) can recycle toxic intermediates into an essential metabolite. J J Thiaville, J Flood, S Yurgel, L Prunetti, M Elbadawi-Sidhu, G Hutinet, 10.1021/acschembio.6b00279ACS Chem. Biol. 19Thiaville, J. J., Flood, J., Yurgel, S., Prunetti, L., Elbadawi-Sidhu, M., Hutinet, G., et al. (2016). Members of a novel kinase family (DUF1537) can recycle toxic intermediates into an essential metabolite. ACS Chem. Biol. 19, 2304-2311. doi: 10.1021/acschembio.6b00279

Natural variability in S-adenosylmethionine (SAM)-dependent riboswitches: S-box elements in Bacillus subtilis exhibit differential sensitivity to SAM in vivo and in vitro. J Tomsic, B A Mcdaniel, F Grundy, T M Henkin, 10.1128/JB.01034-07J. Bacteriol. 190Tomsic, J., McDaniel, B. A., Grundy, F., and Henkin, T. M. (2008). Natural variability in S-adenosylmethionine (SAM)-dependent riboswitches: S-box elements in Bacillus subtilis exhibit differential sensitivity to SAM in vivo and in vitro. J. Bacteriol. 190, 823-833. doi: 10.1128/JB.01034-07

Molecular mechanism of PdxR -a transcriptional activator involved in the regulation of vitamin B6 biosynthesis in the probiotic bacterium Bacillus clausii. A Tramonti, A Fiascarelli, T Milano, M L Di Salvo, I Nogués, S Pascarella, 10.1111/febs.13338FEBS J. 282Tramonti, A., Fiascarelli, A., Milano, T., di Salvo, M. L., Nogués, I., Pascarella, S., et al. (2015). Molecular mechanism of PdxR -a transcriptional activator involved in the regulation of vitamin B6 biosynthesis in the probiotic bacterium Bacillus clausii. FEBS J. 282, 2966-2984. doi: 10.1111/febs.13338

Salmonalle typhimurium PtsJ is a novel MocR-like transcriptional repressor involved in regulating the vitamin B6 salvage pathway. A Tramonti, T Milano, C Nardella, M L Di Salvo, S Pascarella, Contestabile , R , 10.1111/febs.13994FEBS J. 284Tramonti, A., Milano, T., Nardella, C., di Salvo, M. L., Pascarella, S., and Contestabile, R. (2017). Salmonalle typhimurium PtsJ is a novel MocR-like transcriptional repressor involved in regulating the vitamin B6 salvage pathway. FEBS J. 284, 466-484. doi: 10.1111/febs.13994

The MocR-like transcription factors: pyridoxal 5 ′ -phosphatedepdendent regulators of bacterial metabolism. A Tramonti, C Nardella, M L Di Salvo, S Pascarella, Contestabile , R , 10.1111/febs.14599FEBS J. 285Tramonti, A., Nardella, C., di Salvo, M. L., Pascarella, S., and Contestabile, R. (2018). The MocR-like transcription factors: pyridoxal 5 ′ -phosphate- depdendent regulators of bacterial metabolism. FEBS J. 285, 3925-3944. doi: 10.1111/febs.14599

Studies on cyanobacterial protein PipY shed light on structure, potential functions, and vitamin B6-dependent epilepsy. L Tremiño, A Forcada-Nadal, A Contreras, V Rubio, 10.1002/1873-3468.12841FEBS Lett. 591Tremiño, L., Forcada-Nadal, A., Contreras, A., and Rubio, V. (2017). Studies on cyanobacterial protein PipY shed light on structure, potential functions, and vitamin B6-dependent epilepsy. FEBS Lett. 591, 3431-3442. doi: 10.1002/1873-3468.12841

Crystal structure of D-serine dehydratase from Escherichia coli. D V Urusova, M N Isupov, S Antonyuk, G S Kachalova, G Obmolova, A A Vagin, 10.1016/j.bbapap.2011.10.017Biochim. Biophys. Acta. 1824Urusova, D. V., Isupov, M. N., Antonyuk, S., Kachalova, G. S., Obmolova, G., Vagin, A. A., et al. (2012). Crystal structure of D-serine dehydratase from Escherichia coli. Biochim. Biophys. Acta 1824, 422-432. doi: 10.1016/j.bbapap.2011.10.017

Strategies for vitamin B6 biofortification of plants: a dual role as a micronutrient and a stress protectant. H Vanderschuren, S Boycheva, K T Li, N Szydlowski, W Gruissem, T B Fitzpatrick, 10.3389/fpls.2013.00143Front. Plant. Sci. 4143Vanderschuren, H., Boycheva, S., Li, K. T., Szydlowski, N., Gruissem, W., and Fitzpatrick, T. B. (2013). Strategies for vitamin B6 biofortification of plants: a dual role as a micronutrient and a stress protectant. Front. Plant. Sci. 4:143. doi: 10.3389/fpls.2013.00143

Suppression of a thermosensitive zipA cell division mutant by altering amino acid metabolism. D E Vega, W Margolin, 10.1128/JB.00535-17J. Bacteriol. 200Vega, D. E., and Margolin, W. (2017). Suppression of a thermosensitive zipA cell division mutant by altering amino acid metabolism. J. Bacteriol. 200, e00535-e00517. doi: 10.1128/JB.00535-17

Pyridoxal 5 ′ -phosphate inactivates DNA topoisomerase IB by modifying the lysine general acid. J J Vermeersch, S Christmann-Franck, L V Karabashyan, S Fermandjian, G Mirambeau, Arsène Der Garabedian, P , 10.1093/nar/gkh897Nucleic Acids Res. 32Vermeersch, J. J., Christmann-Franck, S., Karabashyan, L. V., Fermandjian, S., Mirambeau, G., and Arsène Der Garabedian, P. (2004). Pyridoxal 5 ′ -phosphate inactivates DNA topoisomerase IB by modifying the lysine general acid. Nucleic Acids Res. 32, 5649-5657. doi: 10.1093/nar/gkh897

A previously unrecognized kanosamine biosynthesis pathway in Bacillus subtilis. N D Vetter, D M Langill, S Anjum, J Boisvert-Martel, R C Jagdhane, E Omene, 10.1021/ja4010255J. Am. Chem. Soc. 135Vetter, N. D., Langill, D. M., Anjum, S., Boisvert-Martel, J., Jagdhane, R. C., Omene, E., et al. (2013). A previously unrecognized kanosamine biosynthesis pathway in Bacillus subtilis. J. Am. Chem. Soc. 135, 5970-5973. doi: 10.1021/ja40 10255

Dissection of contributions from invariant amino acids to complex formation and catalysis in the heteromeric pyridoxal 5-phosphate synthase complex from Bacillus subtilis. S Wallner, M Neuwirth, K Flicker, I Tews, P Macheroux, 10.1021/bi801887rBiochemistry. 48Wallner, S., Neuwirth, M., Flicker, K., Tews, I., and Macheroux, P. (2009). Dissection of contributions from invariant amino acids to complex formation and catalysis in the heteromeric pyridoxal 5-phosphate synthase complex from Bacillus subtilis. Biochemistry 48, 1928-1935. doi: 10.1021/bi80 1887r

Purification and properties of vitamin B6 kinase from Escherichia coli B. R S White, W B Dempsey, 10.1021/bi00823a005Biochemistry. 9White, R. S., and Dempsey, W. B. (1970). Purification and properties of vitamin B6 kinase from Escherichia coli B. Biochemistry 9, 4057-4064. doi: 10.1021/bi00823a005

The Mtm1p carrier and pyridoxal 5 ′ -cofactor trafficking in yeast mitochondria. M M Whittaker, A Penmatsa, J W Whittaker, 10.1016/j.abb.2015.01.021Arch. Biochem. Biophys. 568Whittaker, M. M., Penmatsa, A., and Whittaker, J. W. (2015). The Mtm1p carrier and pyridoxal 5 ′ -cofactor trafficking in yeast mitochondria. Arch. Biochem. Biophys. 568, 64-70. doi: 10.1016/j.abb.2015.01.021

RemA is a DNA-binding protein that activates biofilm matrix gene expression in Bacillus subtilis. J T Winkelman, A C Bree, A R Bate, P Eichenberger, R L Gourse, D B Kearns, 10.1111/mmi.12235Mol. Microbiol. 88Winkelman, J. T., Bree, A. C., Bate, A. R., Eichenberger, P., Gourse, R. L., and Kearns, D. B. (2013). RemA is a DNA-binding protein that activates biofilm matrix gene expression in Bacillus subtilis. Mol. Microbiol. 88, 984-997. doi: 10.1111/mmi.12235

EF-P posttranslational modification has variable impact on polyproline translation in Bacillus subtilis. A Witzky, K R Hummels, R Tollerson, A Rajkovic, L A Jones, D B Kearns, 10.1128/mBio.00306-18MBio. 9Witzky, A., Hummels, K. R., Tollerson, R., Rajkovic, A., Jones, L. A., Kearns, D. B., et al. (2018). EF-P posttranslational modification has variable impact on polyproline translation in Bacillus subtilis. MBio 9, e00306-e00318. doi: 10.1128/mBio.00306-18

PLP and GABA trigger GabR-mediated transcription regulation in Bacillus subtilis via external aldimine formation. R Wu, R Sanishvili, B R Belitsky, J I Juncosa, H V Le, H J Lehrer, 10.1073/pnas.1703019114Proc. Natl. Acad. Sci. U.S.A. 114Wu, R., Sanishvili, R., Belitsky, B. R., Juncosa, J. I., Le, H. V., Lehrer, H. J., et al. (2017). PLP and GABA trigger GabR-mediated transcription regulation in Bacillus subtilis via external aldimine formation. Proc. Natl. Acad. Sci. U.S.A. 114, 3891-3896. doi: 10.1073/pnas.1703019114

Identification of pyridoxalmodified proteins using mass spectrometry. Y Wu, J Chen, Z Liu, Wang , F , 10.1002/rcm.8030Rapid Commun. Mass Spectrom. 32Wu, Y., Chen, J., Liu, Z., and Wang, F. (2018). Identification of pyridoxal- modified proteins using mass spectrometry. Rapid Commun. Mass Spectrom. 32, 195-200. doi: 10.1002/rcm.8030

A paralog of lysyl-tRNA synthetase aminoacylates a conserved lysine residue in translation elongation factor P. T Yanagisawa, T Sumida, R Ishii, C Takemoto, Yokoyama , S , 10.1038/nsmb.1889Nat. Struct. Mol. Biol. 17Yanagisawa, T., Sumida, T., Ishii, R., Takemoto, C., and Yokoyama, S. (2010). A paralog of lysyl-tRNA synthetase aminoacylates a conserved lysine residue in translation elongation factor P. Nat. Struct. Mol. Biol. 17, 1136-1143. doi: 10.1038/nsmb.1889

Identification and function of the pdxY gene, which encodes a novel pyridoxal kinase involved in the salvage pathway of pyridoxal 5 ′ -phosphate biosynthesis in Escherichia coli K-12. Y Yang, H C Tsui, T K Man, M E Winkler, J. Bacteriol. 180Yang, Y., Tsui, H. C., Man, T. K., and Winkler, M. E. (1998). Identification and function of the pdxY gene, which encodes a novel pyridoxal kinase involved in the salvage pathway of pyridoxal 5 ′ -phosphate biosynthesis in Escherichia coli K-12. J. Bacteriol. 180, 1814-1821.

Identification of the pdxK gene that encodes pyridoxine (vitamin B6) kinase in Escherichia coli K-12. Y Yang, G Zhao, M E Winkler, 10.1111/j.1574-6968.1996.tb08368.xFEMS Microbiol. Lett. 141Yang, Y., Zhao, G., and Winkler, M. E. (1996). Identification of the pdxK gene that encodes pyridoxine (vitamin B6) kinase in Escherichia coli K-12. FEMS Microbiol. Lett. 141, 89-95. doi: 10.1111/j.1574-6968.1996. tb08368.x

Hidden markov models and their applications in biological sequence analysis. B J Yoon, 10.2174/138920209789177575Curr. Genomics. 10Yoon, B. J. (2009). Hidden markov models and their applications in biological sequence analysis. Curr. Genomics 10, 402-415. doi: 10.2174/138920209789177575

Molecular cloning, expression and characterization of pyridoxamine-pyruvate aminotransferase. Y Yoshikane, N Yokochi, K Ohnishi, H Hayashi, Yagi , T , 10.1042/BJ20060078Biochem. J. 396Yoshikane, Y., Yokochi, N., Ohnishi, K., Hayashi, H., and Yagi, T. (2006). Molecular cloning, expression and characterization of pyridoxamine-pyruvate aminotransferase. Biochem. J. 396, 499-507. doi: 10.1042/BJ20060078

Assigment of function to a domain of unknown function: DUF1537 is a new kinase family in catabolic pathways for acid sugars. X Zhang, M S Carter, M W Vetting, B S Francisco, S Zhao, N F Al-Obaidi, 10.1073/pnas.1605546113Proc. Natl. Acad. Sci. U.S.A. 113Zhang, X., Carter, M. S., Vetting, M. W., Francisco, B. S., Zhao, S., Al- Obaidi, N. F., et al. (2016). Assigment of function to a domain of unknown function: DUF1537 is a new kinase family in catabolic pathways for acid sugars. Proc. Natl. Acad. Sci. U.S.A. 113, E4261-E4269. doi: 10.1073/pnas. 1605546113

Biochemical characterization of gapB-encoded erythrose 4-phosphate dehydrogenase of Escherichia coli K-12 and its possible role in pyridoxal 5 ′ -phosphate biosynthesis. G Zhao, A J Pease, N Bharani, M E Winkler, 10.1128/jb.177.10.2804-2812.1995J. Bacteriol. 177Zhao, G., Pease, A. J., Bharani, N., and Winkler, M. E. (1995). Biochemical characterization of gapB-encoded erythrose 4-phosphate dehydrogenase of Escherichia coli K-12 and its possible role in pyridoxal 5 ′ -phosphate biosynthesis. J. Bacteriol. 177, 2804-2812. doi: 10.1128/jb.177.10.2804-2812.1995

Kinetic limitation and cellular amount of pyridoxine (pyridoxamine) 5 ′ -phosphate oxidase of Escherichia coli K-12. G Zhao, M E Winkler, 10.1128/jb.177.4.883-891.1995J. Bacteriol. 177Zhao, G., and Winkler, M. E. (1995). Kinetic limitation and cellular amount of pyridoxine (pyridoxamine) 5 ′ -phosphate oxidase of Escherichia coli K-12. J. Bacteriol. 177, 883-891. doi: 10.1128/jb.177.4.883-891.1995

Aspartate deficiency limits peptidoglycan synthesis and sensitizes cells to antibiotics targeting cell wall synthesis in Bacillus subtilis. H Zhao, D M Roistacher, J D Helmann, 10.1111/mmi.14078Mol. Microbiol. 109Zhao, H., Roistacher, D. M., and Helmann, J. D. (2018). Aspartate deficiency limits peptidoglycan synthesis and sensitizes cells to antibiotics targeting cell wall synthesis in Bacillus subtilis. Mol. Microbiol. 109, 826-844. doi: 10.1111/mmi.14078

SubtiWiki in 2018: from genes and proteins to functional network annotation of the model organism Bacillus subtilis. B Zhu, J Stülke, 10.1093/nar/gkx908Nucleic Acids Res. 46Zhu, B., and Stülke, J. (2017). SubtiWiki in 2018: from genes and proteins to functional network annotation of the model organism Bacillus subtilis. Nucleic Acids Res. 46, D743-D748. doi: 10.1093/nar/gkx908

A new arrangement of (beta/alpha)8 barrels in the synthase subunit of PLP synthase. J Zhu, J W Burgner, E Harms, B R Belitsky, J L Smith, 10.1074/jbc.M503642200J. Biol. Chem. 280Zhu, J., Burgner, J. W., Harms, E., Belitsky, B. R., and Smith, J. L. (2005). A new arrangement of (beta/alpha)8 barrels in the synthase subunit of PLP synthase. J. Biol. Chem. 280, 27914-27923. doi: 10.1074/jbc.M503642200