# A survey of the small RNA population during far-red light-induced apical hook opening

CorpusID: 18603468
 
tags: #Medicine, #Environmental_Science, #Biology

URL: [https://www.semanticscholar.org/paper/a07406b9309bc059d7a6ef974c77c776c06ead60](https://www.semanticscholar.org/paper/a07406b9309bc059d7a6ef974c77c776c06ead60)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

A survey of the small RNA population during far-red light-induced apical hook opening
published: 29 April 2014

Ying Li 
Department of Crop Sciences
University of Illinois at Urbana-Champaign
UrbanaILUSA

Kranthi Varala 
Department of Crop Sciences
University of Illinois at Urbana-Champaign
UrbanaILUSA

Matthew E Hudson mhudson@illinois.edu 
Department of Crop Sciences
University of Illinois at Urbana-Champaign
UrbanaILUSA

Energy Biosciences Institute
University of Illinois at Urbana-Champaign
UrbanaILUSA

CINVESTAV-IPN
Department of Crop Sciences
University of Padova
Italy Stefan De FolterMexico

34 National Soybean Res Ctr
University of Illinois at Urbana-Champaign
1101 West Peabody Drive61801UrbanaILUSA

Serena Varotto 
Matthew E Hudson 
A survey of the small RNA population during far-red light-induced apical hook opening
published: 29 April 201410.3389/fpls.2014.00156ORIGINAL RESEARCH ARTICLE Edited by: Tohru Hashimoto, Uozaki Life Science Laboratory, Japan Reviewed by: *Correspondence:de-etiolationphotomorphogenesismiRNAfunctional genomicssoybeansapical hook
Photomorphogenesis is a mechanism employed by plants to regulate their architecture and developmental program in response to light conditions. As they emerge into light for the first time, dark-grown seedlings employ a rapid and finely-controlled photomorphogenic signaling network. Small RNAs have increasingly been revealed to play an important role in regulating multiple aspects of plant development, by modulating the stability of mRNAs. The rapid alteration of the mRNA transcriptome is a known hallmark of the de-etiolation response, thus we investigated the small RNA transcriptome during this process in specific seedling tissues. Here we describe a survey of the small RNA expression profile in four tissues of etiolated soybean seedlings, the cotyledons, hypocotyl and the convex and concave sides of the apical hook. We also investigate how this profile responds to a 1-h far-red light treatment. Our data suggests that miRNAs show a different global profile between these tissues and treatments, suggesting a possible role for tissue-and treatment-specific expression in the differential morphology of the seedling on de-etiolation. Further evidence for the role of miRNA in light-regulated development is given by the de-etiolation responses of a hypomorphic ago1 mutant, which displays reduced and delayed photomorphogenic responses in apical hook and cotyledon angle to far-red light. A survey of the small RNA population during far-red light-induced apical hook opening. Front. Plant Sci. 5:156.

## INTRODUCTION

Light is an essential energy source for plants. Plants have therefore evolved a sophisticated system to sense the intensity, wavelength, direction, duration, and diurnal span of environmental light and accordingly adjust their developmental plan and architecture. The response of plants to light, in addition to photosynthesis, can be divided into three categories: photomorphogenesis, phototropism, and photoperiodism (Smith, 1974). Photomorphogenesis refers to the phenomenon wherein a plant developmentally regulates its body architecture in response to a light stimulus in a non-period-sensitive, non-directional-manner (Smith, 1974;Quail, 2002). De-etiolation, also known as seedling photomorphogenesis, is a well-studied example of photomorphogenesis. A dark-grown dicotyledon seedling displays an elongated hypocotyl, unexpanded, closed cotyledons, closed apical hook, and undifferentiated chloroplasts. De-etiolation occurs after it is exposed to light, and involves an inhibition of hypocotyl elongation, opening of cotyledons and apical hook, and chloroplast maturation. This response would typically be activated at the time that a germinated seedling emerges from soil or leaf litter into sunlight. The timing of de-etiolation is thus critical to the survival of plants (Quail, 2002), and this process is influenced both by light signals and the plant's developmental program.

Photomorphogenesis is mediated by signaling pathways that are downstream of photoreceptors, among which phytochrome was the first to be characterized (Borthwick et al., 1952;Butler et al., 1959;Smith, 2000;Nagy and Schafer, 2002;Quail, 2002Quail, , 2007Franklin and Quail, 2010). In Arabidopsis, there are five phytochromes (phyA-phyE) collectively mediating far-red light and red-light induced de-etiolation. PhyA is responsible for the de-etiolation induced by continuous far-red light (FRc), while phyB-E are mainly responsible for red-light induced de-etiolation (Sharrock and Quail, 1989;Clack et al., 1994;Devlin et al., 1998;Quail, 2002). While red light has sufficient energy to trigger photosynthesis, far-red light does not. Therefore, FR-induced and phyA-mediated de-etiolation is an ideal model system to study light signaling pathways, because a single wavelength stimulus and a single photoreceptor are involved in initializing the cascade.

A few different yet overlapping signaling mechanisms downstream of the light-activated phytochrome have been reported (Chen and Chory, 2011). Among them, one important mechanism is that phytochrome interacts with transcription factors to regulate target gene expression. Upon photoactivation, the biologically active phytochrome translocates from the cytoplasm to the nucleus and interacts with transcription factors to regulate downstream gene expression and to trigger morphological changes (Fankhauser and Chen, 2008). The mRNAs of many genes have been identified as regulated by light downstream of phytochrome using high-throughput gene expression profiling methods, for example in Arabidopsis (Quail, 2007) and in soybean . However, another important component of the transcriptome, the small RNA, has not been studied in the context of light.

Small RNA is a class of short, regulatory non-coding RNAs of 20-30 nt (Lee et al., 1993;Hamilton and Baulcombe, 1999), which has been shown to play an essential role in regulating plant development (Chen, 2010;Axtell, 2013). A key player in small RNA function is the ARGONAUTE family protein (Carmell et al., 2002). AGO1 was the first well characterized ARGONAUTE protein (Baumberger and Baulcombe, 2005) and is known to bind with 21 nt small RNAs as part of the RNA intermediated silencing complex (RISC) and target mRNAs sequence specifically for cleavage. In a loss-of-function mutant of ago1, the functions of miRNA and siRNA are greatly impaired, and the mutant therefore displays severe developmental defects (Morel et al., 2002). These defects include some aspects of deetiolation such as altered adventitious rooting and a relatively short hypocotyl in light-grown seedlings (Sorin et al., 2005), indicating a role for small RNA in light controlled developmental processes. It has also been reported that a global regulator of light responsive transcription, LONG HYPOCOTYL 5 (HY5), regulates the transcription of eight miRNA genes (Zhang et al., 2011) in response to light, in addition to many mRNAs. The global profile of small RNA expression in specific seedling tissues during de-etiolation, however, has never been systematically examined to our knowledge. We therefore investigated whether small RNAs are regulated by light during seedling photomorphogenesis by profiling the expression of small RNA in both etiolated seedlings and de-etiolating seedlings.

During seedling photomorphogenesis, distinct morphological changes and photomorphogenic gene regulation have been observed in different organs, e.g., cotyledons, hook, and hypocotyl . Specifically, within the apical hook, the hook concave region (the half of the tissue on the inward side of the hook, see Figure 1A) and hook convex region (the outward facing half, see Figure 1A) display opposite morphological changes during light induced hook opening. Indeed, the light-induced changes in gene expression in the apical hook are in some cases limited to the concave region or convex region (Peck et al., 1998;Li et al., 2011). We hence examined the light regulation of small RNA in four different tissues (cotyledons, hook concave region, hook convex region, and hypocotyl) with Illumina short read sequencing. We used Glycine max cv. Williams 82 for this study, as soybean is an important crop in the US and, the soybean genome is completely sequenced and well annotated (Schmutz et al., 2010). The small RNA transcriptome of soybean has been recently revealed to mediate signaling and developmental events (Tuteja et al., 2008;Joshi et al., 2010;Li et al., 2012a;Hu et al., 2013). However, photomorphogenesis and the apical hook of soybean have not yet been investigated. Soybean therefore offers a good system to study the apical hook, since the miRNA transcriptome is well characterized, as is the genome, but unlike Arabidopsis, the apical hook is sufficiently large to dissect and extract sufficient small RNA for this tissue-specific type of study. Our analysis suggests that the population of small RNAs, especially miR-NAs, may respond differentially in the hook to a 1-h FRc light stimulus. FIGURE 1 | Angle of cotyledon and hook in ago1-27 mutants compared to background accession during growth under continuous far-red light (FRc). Three-day old etiolated seedlings ago1-27 seedlings and the corresponding wild-type background accession Columbia (WT) were transferred to FRc. The angles of hook (A) and cotyledon (B) of both mutants and WT were measured from photographs taken at 3 h intervals. The error bars represent the standard error of the mean.


## MATERIALS AND METHODS


## PLANT MATERIAL

Glycine max cv. Williams 82 seeds were grown in Sunshine Mix LC1 at 25 • C in darkness for 7 days. Seedlings were then irradiated with far-red light (FRc) (peak 733 nm; 20 µmol m −2 s −2 ) for 1 h while the control seedlings were kept in darkness. Both types of seedlings were then harvested in liquid nitrogen under green safelight. For the phenotypic study of ago1, ago1-27 (Morel et al., 2002) was used. The mutant genotype was verified with PCR amplification with gene-specific primers GAGGCTGTTTGCTCAGAACC and TCTACCAGCCATTCCACCTC, and then sequenced through SNP region for confirmation. The phenotyping of ago1-27 and its background control, Columbia, were performed as previously described .


## ILLUMINA SMALL RNA SEQUENCING LIBRARY PREPARATION AND DATA ANALYSIS

Cotyledons, hook and hypocotyls of the FRc treated soybean seedlings and dark grown seedlings were flash frozen in liquid nitrogen. The apical hooks were then transferred to RNAlater ICE (Ambion), which preserves the RNA while making the tissue more pliable for further dissection into the hook concave and hook convex without RNA change or loss. The dissection of hook concave and convex parts was done under white light with a dissection microscope, by cutting along the axis of the curved hook. For each light condition, 32 hooks were dissected this way and four pools of eight were created to make four biological replicates, so that the possible contamination caused by dissection error was minimized through the pooling process. Then, four replicate total RNA samples, each extracted from a pool of eight individual replicate seedlings, were prepared by the pine tree method (Chang et al., 1993) for each combination of tissue type and light condition. The four RNA samples of the same tissue type and light condition were then pooled in equi-molar amounts for small RNA profiling. Eight RNA samples representing the transcriptomes of cotyledons, hook convex, hook concave, and hypocotyl in seedlings after 1 h FRc treatment and dark grown seedlings were submitted to the Keck center (UIUC) for small RNA library construction and Illumina sequencing. The raw reads from Illumina sequencing were first preprocessed using FreClu with default parameters (Qu et al., 2009) to remove low-quality reads, trim adaptors, select reads with 17-35 nt size and correct for sequencing errors. In this process the read set was reduced to unique sequences with an associated abundance (counts). The preprocessed reads were then aligned to soybean tRNA and rDNA databases using novoalign while allowing up to three mismatches (www.novocraft.com). The soybean tRNA sequences were downloaded from GtRNAdb (Chan and Lowe, 2009). The soybean rDNA sequences were combined from a few resources: the NOR of soybean genome (phytozome.org, Gm13:14785374-15674994), Rfam (Gardner et al., 2008) and Gmax.rDNA.scaffolds file from phytozome V5.0. Reads mapped to tRNA or rDNA were removed from downstream analysis. Remaining reads were then mapped to the JGI soybean genome assembly (Schmutz et al., 2010) using novoalign allowing only perfect matches (t = 0). Reads aligning to the genome were used in the differential analysis. The unmapped reads from the last alignment (perfect matches) were then aligned to the soybean genome allowing three mismatches (t = 30). The reads that were still unmapped were then first aligned to the chloroplast genome and then to the mitochondrial genome (phytozome V5.0).


## SMALL RNA BLOCK BUILDING AND COUNTS CALCULATION

The proximity-based small RNA block was generated as previously described (Lu et al., 2005;Li et al., 2012b). Perfectly matching reads from the eight libraries were combined to build blocks on the 20 chromosomes. The small RNA expression level of each block in a library was calculated as the sum of the abundances of small RNAs belonging to the block. A weighing method was used for calculating block counts from repetitively mapped small RNA as described in Li et al. (2012b) to prevent overcounting of the expression level of repeat-mapping small RNAs.

As a result, the block counts generated with the weighted counts displayed a high degree of linearity in a pairwise comparison. By contrast, block counts generated with un-weighted small RNA counts showed a skewed x-y scatter plot and were not used for the analysis presented.


## PREDICTION OF NOVEL miRNA

Seventy five known Glycine max miRNAs and their precursors were downloaded from miRBase (as of Feb 2010). Those loci were folded with a range of parameters to determine the combination with the greatest sensitivity. By visually comparing the predicted folds from the program to the submitted folds in miRBase and scoring them individually for similarity, the following parameters were found to be optimal for finding the best secondary structure of the pri-miRNA: Folding temperature = 25 • C, Threshold dG = − 40 kCal/mole, Window = default, Length = 170 bp.

Goodness of the fold was determined based on the following criterion: (i) Over 75% of the bases in the small RNA need to be paired; (ii) the length of the complementary sequence (predicted miRNA * ) should not be more than 1.5 times the length of the small RNA; (iii) no bases in the small RNA or within 10 bases of its end can be in the complementary strand of the stem-loop i.e., the distance between an miRNA and its complement should be at least 20 bases. Loci passing these filters were screened against known repeat sequences (GFF from http://www.phytozome.net) to remove the small number of repeat sequences that pass all these filters. These putative miRNAs were then aligned to known miRNA sequences from miRBase to identify the known miRNA and novel miRNA.


## DIFFERENTIAL EXPRESSION ANALYSIS

The following comparisons were performed: (1) For light response: FR cotyledon vs. dark cotyledons, FR hypocotyl vs. dark hypocotyl, FR hook convex vs. dark hook convex, and FR hook concave vs. dark hook concave; (2) For tissue specificity in hook: FR hook convex vs. FR hook concave and dark hook convex vs. dark hook concave. In each pairwise comparison, the Difference in Proportion method (DIP; Kal et al., 1999) was applied to select the blocks with significant difference between the two libraries. The p-values generated by DIP were corrected for multi-testing error by applying a Bonferroni correction, and any corrected p-value lower than 0.05 was reported to be significant. A 2-fold cutoff was also applied to select significantly regulated blocks. A low expression filter of 20 reads per million (RPM) was performed prior to the differential analysis, so that only sRNA blocks with an expression level greater than 20 RPM in at least one of the conditions in comparison are included. The DIP method features an internal normalization and, therefore a RPM normalization was not performed prior to the differential analysis.


## ANNOTATION OF FR RESPONSIVE SMALL RNA BLOCKS

The following approaches were used to annotate the differentially expressed (DE) blocks: (i) the abundance of small RNA with different sizes in each block was computed by an in-house Perl script. (ii) the highest expressed small RNA from each block (referred to as the "key sequence" hereafter) was used to search against the miRBase mature miRNA database (Kozomara and Griffiths-Jones) using SSEARCH35 (Pearson and Lipman, 1988), to identify blocks potentially producing known miRNA (p-value cutoff 0.01). (iii) A strand-specific genomic sequence (170 bp) around the key sequence was tested for folding into a hairpin structure using UNAfold (Version 3.6, http://mfold.rna.albany.edu/) using parameters described above. (iv) the small RNA expression data generated in the study was visualized in Gbrowse2.0 (http:// gmod.org/) with phytozome V5.0 genome annotation (http:// phytozome.org/). DE blocks were classified to determine into siRNA-block-like or miRNA-block-like, by determining whether the small RNAs mapped to both strands or one strand, whether the small RNAs within the blocks mapped mostly to transposons or genes and whether it could be a likely case of cis-nat siRNA generation.

The target prediction of putative novel miRNA was done using psRNAtarget (http://plantgrn.noble.org/psRNATarget/) against Glycine max (soybean) gene models phytozome v8.0 using default parameters.


## RESULTS


## THE ago1 MUTANT IS IMPAIRED IN PHOTOMORPHOGENESIS

To test whether a functional miRNA system is important for photomorphogenesis, we first investigated the de-etiolation process in an Arabidopsis mutant that is defective in small RNA function. We used the hypomorphic mutant allele ago1-27 (Morel et al., 2002), because the stronger mutant alleles (e.g., ago1-8 and ago1-24) are severely defective in development and sterile, and hence are not suitable for phenotyping at the seedling stage-this also applies to other mutants we are aware of that affect the small RNA machinery. By contrast, the hypomorphogenic mutant ago1-27, which carries an Ala to Val mutation at amino acid 992, is nearly normal in development, and fertile. The ago1-27 mutant and WT were grown in dark for 3 days and then given 24 h FRc treatment, during which the angles of hook opening and cotyledon opening were measured in a time series. The ago1-27 mutants show a slower opening hook ( Figure 1A) and impaired cotyledon opening ( Figure 1B) in continuous FR light. This indicates that AGO1, and thus a functioning miRNA system, is required in order for Arabidopsis to de-etiolate normally. We thus conducted a profiling experiment to uncover the small RNA transcriptome during de-etiolation.


## OVERVIEW OF THE SMALL RNA PROFILE OF SOYBEAN SEEDLINGS

Total RNA samples extracted from cotyledons, the apical hook concave region, apical hook convex region and hypocotyl of 7day-old dark-grown seedlings and seedlings treated with 1 h FRc were submitted for small RNA sequencing using the Illumina GAIIx (Keck center, University of Illinois). Eight to fifteen million reads were generated from each library (Figure 2A; Table  S1). The raw reads were preprocessed and first aligned to the soybean tRNA and rDNA databases using Novoalign (www. novocraft.com). Twenty five to Fifty five percent of total small RNA reads from these libraries mapped to tRNA or rDNA databases (Figure 2A; Table S1). The two cotyledon libraries have a relatively higher percentage (approximately 55% of the total reads) of small RNA reads mapped to rDNA (Figure 2A). This higher proportion might be a result of the cotyledon samples including actively dividing young tissues, which were previously reported to produce more rDNA-mapped small RNA compared to mature tissue (Lu et al., 2005). These rDNA and tRNA mapped reads were considered unlikely to be relevant to light-responsive small RNA regulation and were hence removed from further analysis. The remaining reads were aligned to the JGI soybean genome assembly (v5.0) using Novoalign, allowing only perfect matches (t = 0). From 68 to 91% of the reads mapped to the soybean genome perfectly (Figure 2B). Allowing two mismatches in each read increased the percentage of mapped reads by 2.3-3.6% (Figure 2B). This is likely caused by sequencing errors, or by polymorphisms within the Williams 82 population (Varala et al., 2011). A small portion of the remaining non-mapped reads aligned to the soybean chloroplast and mitochondrial sequences ( Figure 2B). Finally, 6.0-25.4% of reads remained unmapped to either the nuclear genome or the plastid genome even with relaxed stringency (Figure 2B). In summary, most reads mapped to the soybean nuclear genome assembly perfectly, resulting in 1.7-9 million nuclear genome-mapped reads representing 0.5-2.5 million unique sequences for each library. Those reads were used for further analysis (Table S1).

Different functional categories of small RNA are known to have specific sizes (Chen, 2009(Chen, , 2010Axtell, 2013), thus we analyzed the size distribution of the genome-mapped small RNAs. The 21 and 24 nt small RNAs are the two most abundant classes in all libraries (Figure 3). Interestingly, a distinctive 21:24 nt ratio was observed across different tissue types. The 21 nt small RNAs represent the most abundant class in the cotyledons, with a 21:24 nt ratio close to 2:1 in both libraries, the highest of the tissues examined. In contrast, the 24 nt class is the most abundant in both hypocotyl libraries, with 21:24 nt ratios of 0.81:1 and 0.67:1, respectively. Among the four hook libraries, the dark treated plants resembled the hypocotyl libraries, with the 24 nt class as the most abundant class (21:24 nt ratio of 0.71:1 and 0.75:1 for hook concave and hook convex, respectively), however, surprisingly, the FR libraries had almost equal amounts of 21 and 24 nt small RNA (21:24 nt ratio as 0.98:1 and 1.07:1). This may indicate either an induction of 21 nt class or a repression of 24 nt sRNA in response to FR. Most miRNAs belong to the 21 nt class, while the 24 nt class mainly consists of heterochromatic siRNA (Chen, 2010). Therefore, the change in ratio of 21:24 nt may indicate that miRNAs are relatively more abundant in response to FR treatment in the hook within 1 h. The proportion of 22 nt small RNA, reported to be ta-siRNA triggers (Chen, 2010), is relatively constant across the eight libraries (Figure 3).


## IDENTIFICATION OF NOVEL miRNAs IN DE-ETIOLATING SOYBEAN SEEDLINGS

To test whether there are novel miRNAs in our dataset, a pipeline was developed to identify putative novel miRNAs based on the secondary structure of the putative pri-miRNA as well as all other standards considered necessary for description of a new miRNA (Meyers et al., 2008). We included small RNA data from a global profile of small RNAs in soybean to broaden the scope of this analysis to cover a wide range of organs and developmental stages. Specifically, data from our experiment was combined with data from Zabala et al. (2012) to create a broad profile of endogenous small RNA populations in soybean. Appropriate parameters for maximizing the sensitivity of the prediction process were determined using a training set of known miRNA generating loci in Glycine max in miRBase (mirbase.org; Kozomara and Griffiths-Jones, 2011). Combinations of parameters, including folding temperature, threshold dG, and the length of flanking sequence were tested using UNAfold (Markham and Zuker, 2008). The learned set of parameters was then used to select small RNA producing loci, from the combined dataset, that show a favorable secondary structure.

In soybean, identical miRNAs are generated by multiple loci distributed across the genome. For example, the highly conserved miRNA miR156 is encoded by 12 independent loci. Accordingly, small RNAs from the soybean libraries that mapped to less than or equal to 20 genomic loci were submitted to the prediction pipeline. Approximately 200,000 loci were tested within this set and after a few filtering steps (see Methods), a total of 188 putative novel miRNA sequences were identified (Table S10). These predicted miRNAs include 21, 22, and 24 nt long representative sequences. For further analysis, only the predicted miRNAs from the 20-22 nt classes (137 out of the 188) were chosen for further studies (Table S2).

An investigation of the role these miRNAs might play in soybean transcriptional control was performed by computationally identifying the potential targets of these miRNAs. psRNATarget (Dai and Zhao, 2011) was used to identify the potential targets of the predicted novel miRNAs. With the exception of two miRNA sequences, all novel miRNAs were predicted to target multiple genes in the soybean genome (Table S3). A total of 1094 gene models in the soybean genome were identified as potential targets with a large majority predicted to be targeted for cleavage, as opposed to translational regulation (Table S3). Gene Ontology (GO) terms associated with these gene models were processed to identify over-represented GO terms, using AgriGO (bioinfo.cau.edu.cn/agriGO). Significantly over-represented GO terms were determined by a t-test, assuming a hypergeometric distribution, and an FDR correction at alpha of 0.05 (Figure 4; Table S4).


## FRc-RESPONSIVE SMALL RNA CLUSTERS

When mapping small RNAs to the genome, siRNAs sharing the same targets and/or biogenesis are usually found in a dense cluster on both strands of a siRNA-generating genomic locus (e.g., a transposon or a targeted protein coding gene); in contrast, a miRNA usually co-localizes with an miRNA * on the transcribed strand of the miRNA gene. When comparing between two libraries, individual siRNA may be different between two libraries, while the overall counts of small RNA from an siRNA generating genomic locus (which is more biologically relevant) do not change. To capture the biologically relevant small RNA regulation, a proximity-based clustering algorithm was used to build 309,645 blocks on the genome from a pool of small RNA reads from all libraries. The mean length of the blocks is 2621 bp and the median 575 bp. Each block represents a genomic region where potentially functionally related small RNAs are generated. The abundance of small RNAs in a block was summed to represent the small RNA expression level of the block in a library (hereafter referred to as block count).

The block counts from the dark-grown libraries were compared with the FRc-treated libraries using the DIP statistical test to identify FRc-responsive DE blocks in the cotyledons, hook convex region, hook concave region, and hypocotyl. The p-value computed by DIP was corrected with Bonferroni multi-testing control (Kal et al., 1999). We counted DE blocks that had a corrected p-value smaller than 0.05, and we also imposed a fold change cutoff of FR/dark or dark/FR greater than 2-fold. With these criteria, 11 blocks were identified as being regulated by FRc in the cotyledons, 57 in the hook concave side, 139 in hook convex side, and 25 in hypocotyl (Table S5). The small RNA expression profiles of the hook concave region were also compared to those of the hook convex region under both light conditions, leading to the identification of 4 blocks DE between the hook convex and hook concave in the darkness, and 26 blocks after 1 h FRc treatment (Table S5).


## FUNCTIONAL ANNOTATION OF THE FRc RESPONSIVE SMALL RNA BLOCKS

To identify miRNA or siRNA associated with the FR-responsive small RNA blocks, we investigated the DE blocks with multiple approaches to characterize their biological functions. First, we examined the length distribution of small RNAs in each DE block, since different categories of small RNAs have different sizes (Axtell, 2013). For example, most microRNAs are 21 nt in length, and the 23/24 nt small RNAs usually represent heterochromatic siRNAs. Therefore, the abundance of small RNA from 20 to 24 nt in each DE block was plotted by length ( Figure 5). Most of the DE blocks identified as DE in hypocotyl in response to FR are dominated by 23/24 nt small RNA (Figure 5A), while DE blocks identified from the comparison of the hook concave region and hook convex region after 1 h FRc are dominated by 20/21/22 nt small RNA (Figure 5B). This result again suggested that miRNAs are actively regulated in response to FR treatment in the hook within 1 h, supporting our previous result. DE blocks identified from the other comparisons have comparable numbers of blocks dominated by different size classes (Figure 5).

To identify known miRNA genes in the DE blocks, the most highly expressed small RNA from each block (hereafter termed the "key sequence") was used for a sequence similarity search in miRBase (mirbase.org; Kozomara and Griffiths-Jones, 2011). Meanwhile, to identify both known and novel miRNA genes in the DE block, the genomic sequence surrounding the key sequence was extracted to test whether it could fold into a canonical stemloop structure commonly observed in pri-miRNA. Additionally, the small RNA expression profile across the block was visualized using Gbrowse v.2 to determine whether the block resembled the expected features of a miRNA generating locus (i.e., production of 21/22 nt RNA predominantly from two sites: the miRNA and miRNA * ). If a block fails to satisfy the above criteria, it is likely to be a siRNA generating block, in which case it was recorded whether most of the siRNAs in the block map to a proteincoding gene or a transposon according to the phytozome soybean genome annotation (v.5). Using the pipelines described above, the DE blocks were annotated as miRNA loci, siRNA blocks matching protein-coding genes or siRNA blocks matching transposable elements (TE). The results for miRNA blocks and siRNA blocks are discussed separately below.


## TISSUE-SPECIFIC FRc-RESPONSIVE miRNA

If (1) a foldback structure meeting our criteria could be identified from a DE block, (2) the key sequence matched a known miRNA, (3) the predominant small RNA class is 20/21/22 nt, and (4) the small RNA expression profiles of the block displays the features of a miRNA gene locus described previously (sparse cluster dominated by a 20-22 nt sequence miRNA and miRNA * ), the block was identified as a known miRNA gene.

Using these criteria, we found several miRNAs that met our definition of being potentially DE in response to FRc in cotyledons, the hook concave region, hook convex region or hypocotyl. miR167 was down-regulated by FRc in the cotyledons ( Figure 5A). miR394, miR396, miR530, miR1509, and miR2218 were up-regulated by FRc in the hook concave region (Figure 5A), while miR166, miR394, miR396, miR1508, miR1509, and miR2218 were up-regulated by FRc in the hook convex region ( Figure 5A). Collectively, this suggests that miR394, miR396, miR1509, and miR2218 were up-regulated in the apical hook by FRc; In the hypocotyl, miR168, miR166, and miR1507 were down regulated and miR167 was up-regulated by FRc (Figure 5A; Table S6). Interestingly, the comparison between the hook convex region and hook concave region of FRc treated seedlings showed that miR166 and miR1508 were expressed higher in the hook convex region than the hook concave region under FRc (Figure 5B; FIGURE 4 | Over-represented Gene Ontology terms associated with predicted mRNA targets of the putative novel miRNAs found in this study. Gene Ontology (GO) terms associated with these gene models were processed to identify over-represented GO terms, using AgriGO (bioinfo.cau.edu.cn/agriGO). The graph indicates the percentage of predicted target mRNAs associated with a particular term that is significantly over-represented, and the percentage of all mRNAs in the genome associated with the same term. Significantly over-represented GO terms were determined by hypergeometric test with an FDR correction (alpha = 0.05). Table S7). Therefore, miR166 and miR1508 are likely induced by FRc only in the hook convex ( Figure 6A). For some of the microRNAs, e.g., miR166, miR167, and miR396, their mRNA targets have been well characterized to regulate plant growth and development (Williams et al., 2005;Wu et al., 2006;Jung and Park, 2007;Liu et al., 2009;Rodriguez et al., 2010). For the other identified FR-responsive miRNA, little is known about their mRNA targets. We used psRNAtarget (Dai and Zhao, 2011) to predict the potential targets of those miRNAs (Table S8). None of the predicted novel miRNAs that we identified in this study (Table  S2) was detected to be regulated by FRc.


## siRNA BLOCKS

If a DE block does not have a key sequence similar to a known miRNA and does not form a stem-loop foldback structure, it likely represents a siRNA-generating locus. We visualized the small RNA expression of those blocks in Gbrowse v2.0 to confirm that they showed a similar pattern to that expected for siRNA generating loci (i.e., generation of many different small RNA species across the block). We then investigated if the majority of siRNAs in those blocks map to any gene or TE, based on the phytozome (www.phytozome.org) annotation (v5.0) of the soybean genome. As a result, 7 of the 57 FR responsive blocks in the hook concave sample and 36 of the 139 FR responsive blocks in the hook convex sample featured siRNA mapping to TEs. Interestingly, all of these are down-regulated by FRc. These small RNAs are mostly 24 nt, and are likely heterochromatic siRNAs repressing transponson activity to maintain genome integrity. No DE block with TE-mapping small RNA was identified among the FR responsive blocks identified in the cotyledons and hypocotyl. In 24 DE blocks from various tissues, the majority of small RNAs mapped to a high confidence gene, suggesting that many of the small RNAs identified may regulate gene expression, either in response to light or differentially in the hook convex region compared to the hook concave region (Table S9). The functions of those genes were annotated using a homology search against the model organism Arabidopsis (Table S9). One such gene Glyma13g36840, which encodes a TCP family transcription factor, has more mapped siRNA reads in the hook convex compared to hook concave, and was shown as example in Figure 6B. 


## DISCUSSION

The ability to respond to environmental light allows plants to adapt rapidly to environmental changes. It has been known for a long time that the plant light responses involve induction and repression of protein-coding gene expression at the mRNA level. Recently, small RNA has been implicated in plant development and abiotic stress responses (Lewis et al., 2009;Chen et al., 2010). However, the role of small RNA in light-related plant development has not been investigated to our knowledge. In our study, we showed that a mutant with defective miRNA function displayed impaired de-etiolation. We further performed a survey sequencing of the small RNAs in the cotyledons, hook convex region, hook concave region, and hypocotyl, both from etiolated seedlings and seedlings treated with 1-h FRc light. We showed that potentially novel miRNAs may exist in these little-characterized tissues, and that the level of some small RNAs met our statistical criteria for differential expression after a 1-h light treatment in the specific regions of the seedling investigated. Together these results expand our knowledge of the role of small RNA in plant development, as well as add to our current understanding of light signaling.

First, by developing a novel miRNA prediction pipeline when the data was first available, we identified 137 putative miRNA that are novel to the miRBase at that time (2010). To compare our predicted miRNAs with more recent studies on miRNA identification, we compared the 137 novel miRNAs with a current miRBase database at the time of writing (March 2014), and found that 73 of our previously predicted novel miR-NAs have been reported in other studies as novel miRNAs (Table S2). This result underscores the consistency of our novel miRNA prediction pipeline with those used by other researchers. Interestingly, one of the miRNAs recently added to miRBase, miR395 (CUGAAGUGUUUGGGGGAACU), is predicted by our analysis to target Glyma11g36210, which is rapidly reduced in expression in response to FRc in the soybean apical hook as reported in Li et al. (2011). Glyma11g36210 encodes a sulfate transporter, SULTR2;1. Thus not only is a sulfate transporter rapidly down-regulated by light in the apical hook, but this rapid down-regulation is likely facilitated by miRNA-mediated turnover of the transcript.

A major advantage of using soybean as the experimental system to study hook opening is that, compared to the model organism Arabidopsis, the soybean hook is large and relatively robust, and thus relatively easy to dissect into the oppositely-responding convex and concave regions. In our study, the dissection of concave vs. convex regions was facilitated by a novel method of using a fixative on flash-frozen apical hook tissue (RNAlater ICE), which preserves the RNA and prevents biochemical responses to dissection or light, while making the tissue soft for dissection at low temperatures. Therefore, by dissecting along the middle axis of the curved hook under a binocular microscope under white light, the RNA expression patterns can be attributed to the responses that occurred before freezing and fixation. This process both addresses concerns that light or wounding would cause a response in RNA expression, and that the dissection process may cause variable RNA degradation, for example by means of releasing degradative enzymes from lytic compartments. For each light condition examined experimentally, 32 hooks were dissected in this manner and pooled for each RNA sample used for library generation. The possible contamination caused by errors in the dissection boundary is minimized by this pooling process.

In our study, the proportion of 21-nt small RNA in the hook convex and hook concave regions increased greatly after a 1 h FRc treatment (Figure 3). Since most identified miRNAs in plants belong to the 20/21/22 nt class, this likely suggests that the overall FIGURE 6 | Graphs showing the small RNA reads mapping to a miRNA block (A) and siRNA block (B). (A) A hook-convex specific FRc-responsive miRNA miR1508 was shown as an example. The mature miR1508 reads was mapped to a genomic region in Gm09. Each sample was represented as a track, while the Y-axis of the track represents the RPM normalized weighted counts of the sRNA reads. (B) A siRNA block with higher expression level in hook convex than hook concave co-localizes with a protein-coding gene Glyma13g36840. Each sample was represented as a track, while the Y-axis of the track represents the RPM normalized weighted counts of the sRNA reads (negative value means mapping to reverse strand). Blue bar represents sRNA mapping to the forward strand, and red bar represents sRNA mapping to the reverse strand. The graph was prepared with Integrative Genomics Viewer (Thorvaldsdóttir et al., 2012). level of miRNA is up regulated by light in the hook region. While it is also true that siRNA can be generated in a 21 nt form, the identification of up-regulated miRNAs in the hook provides evidence to support this ( Figure 5A). Combined with the fact that the hypomorphic ago1 mutant, expected to slow but not eliminate miRNA directed cleavage, shows a slower-opening hook and cotyledons during de-etiolation, we propose that light induces expression of miRNAs in the hook region, and this triggers mRNA degradation required for normal hook opening and subsequent cotyledon opening.

A few known miRNAs identified here as light responsive have well characterized functions. miR167, for example, is known to target Auxin Response Factors ARF6 and ARF8 to correct the patterning of ARF6 and ARF8 expression domains during ovule and anther development (Wu et al., 2006). Recently, soybean miR167 was also implicated in Nematode defense (Li et al., 2012a). In our study, miR167 was found to be DE. While this expression pattern remains to be confirmed by another technique, such as qPCR or a Northern blot, our data indicates it is down regulated in the cotyledons and up regulated in the hypocotyl in response to a 1 h FR stimulus. This is consistent with changes in cellular expansion in these organs, with an active expansion of the cotyledons and an inhibition of growth in the hypocotyl during de-etiolation. miR396 is also a well characterized and well conserved miRNA. It targets Growth Regulating Factor (GRF) transcription factors, which regulate leaf growth (Liu et al., 2009;Rodriguez et al., 2010). The miR396 level was at higher levels in response to FRc in both sides of the hook in our study, again requiring confirmation, but possibly part of a mechanism to alter growth regulation. Our data also suggested, without confirmation, that two miRNAs, miR1508, and miR166, are up-regulated by FRc specifically in the hook convex region. In Arabidopsis, miR166 targets HD-ZIPIII genes which regulate many aspects of plant development, including shoot apical meristem architecture (Williams et al., 2005;Jung and Park, 2007), vascular patterning (Zhong and Ye, 2007), leaf polarity (Juarez et al., 2004;Nogueiral et al., 2007), and floral development (Jung and Park, 2007). Recently, miR166 was implicated in soybean Nematode defense (Li et al., 2012a). An induction of miR166 in response to FRc in the hook convex in our study may suggest miR166 and its target HD-ZIPIII genes are involved in the opening of apical hook. Since the global mRNA response to FRc in cotyledon, hook and hypocotyl in soybean was examined in Li et al. (2011), we crosschecked the targets of FR-responsive miRNA and siRNA identified in this study with the results of Li et al. (2011), but no overlap was found. However, since both studies focus on changes in expression within 1 h of exposure to light, it is unlikely an miRNA change in expression will have a two-fold effect on a mRNA within this time period. Therefore, replicated time series data will likely be necessary to capture the negative correlation between FR-responsive sRNA and FR-responsive mRNA.

In conclusion, the miRNA mechanism (specifically the AGO1 dependent miRNA function) is needed for the normal, rapid responses of seedlings to FR. In conjunction with this result, the profile of miRNA expression is noticeably different between seedling tissues, between the concave and convex sides of the apical hook, and between dark-grown and FR-treated seedlings. These changes include miRNAs known to regulate cell expansion and organ growth. While further confirmatory experiments are necessary to verify the responses of individual miRNAs described here, at least one of the profile changes fits with changes in expression of a potential miRNA target described in another study. Together these findings suggest that the miRNA system is implicated in regulating the rapid morphological responses to light in the apical region of dicot seedlings. 


## AUTHOR CONTRIBUTIONS

## FIGURE 2 |
2Abundance of small RNA mapping to soybean genome. Small RNA reads were first aligned to soybean tRNA and rDNA to filter these types of noncoding RNA (A). Reads which passed this filter were then aligned to the soybean nuclear and plastid genomes (B).

## FIGURE 3 |
3Abundance of size classes of small RNA (17-36 nt). The percentages of the abundance of genome-mapped small RNA of each size class are plotted for each library.

## FIGURE 5 |
5(A) Size distribution of sRNAs and expression level of miRNAs in FR responsive sRNA blocks and (B) size distribution of sRNAs and expression level of miRNAs for sRNA blocks differentially expressed in the concave vs. convex side of the apical hook. The percentages of the total abundance of each small RNA species with lengths 20-24 nt in differentially expressed blocks are shown in the white-blue heatmaps. The log10 expression levels of previously characterized miRNAs are shown in the white-red heatmaps.


Matthew E. Hudson and Ying Li designed the experiment. Ying Li performed the phenotypic analysis, tissue culture, and RNA extraction. Ying Li and Kranthi Varala performed the data analysis. Ying Li, Kranthi Varala, and Matthew E. Hudson prepared the manuscript. Ying Li and Kranthi Varala prepared the figures and tables.
www.frontiersin.orgApril 2014 | Volume 5 | Article 156 | 3
www.frontiersin.orgApril 2014 | Volume 5 | Article 156 | 7
www.frontiersin.orgApril 2014 | Volume 5 | Article 156 | 9
ACKNOWLEDGMENTSWe would like to thank Dr. Hervé Vaucheret for kindly sending us the ago1-27 seeds, thank Dr. Kankshita Swaminathan for helpful discussion, and thank Wendy Win for discussion on the small RNA block analysis. We would also like to thank Dr. Alvaro Hernandez and Keck Center of University of Illinois for constructing and sequencing the small RNA libraries. This work was supported by the National Science Foundation (grant no. 0641639), by the U.S. Department of Energy (grant no. DE-SC00400879), and by startup funds from the University of Illinois awarded to Matthew E. Hudson. The Illumina sequencing data was submitted to NCBI SRA with accession number SRX493829.SUPPLEMENTARY MATERIALThe Supplementary Material for this article can be found online at: http://www.frontiersin.org/journal/10.3389/fpls.2014. 00156/abstract
Classification and comparison of small RNAs from plants. M J Axtell, 10.1146/annurev-arplant-050312-120043Annu. Rev. Plant Biol. 64Axtell, M. J. (2013). Classification and comparison of small RNAs from plants. Annu. Rev. Plant Biol. 64, 137-159. doi: 10.1146/annurev-arplant-050312- 120043

Arabidopsis ARGONAUTE 1 is an RNA slicer that selectively recruits microRNAs and short interfering RNAs. N Baumberger, D C Baulcombe, 10.1073/pnas.0505461102Proc. Natl. Acad. Sci. U.S.A. 102Baumberger, N., and Baulcombe, D. C. (2005). Arabidopsis ARGONAUTE 1 is an RNA slicer that selectively recruits microRNAs and short interfering RNAs. Proc. Natl. Acad. Sci. U.S.A. 102, 11928-11933. doi: 10.1073/pnas.0505461102

A reversible photoreaction controlling seed germination. H A Borthwick, S B Hendricks, M W Parker, E H Toole, V K Toole, 10.1073/pnas.38.8.662Proc. Natl. Acad. Sci. U.S.A. 38Borthwick, H. A., Hendricks, S. B., Parker, M. W., Toole, E. H., and Toole, V. K. (1952). A reversible photoreaction controlling seed germination. Proc. Natl. Acad. Sci. U.S.A. 38, 662-666. doi: 10.1073/pnas.38.8.662

. W L Butler, K H Norris, H W Siegelman, S B Hendricks, Butler, W. L., Norris, K. H., Siegelman, H. W., and Hendricks, S. B. (1959).

Detection, assay, and preliminary purification of the pigment controlling photoresponsive development of plants. 10.1073/pnas.45.12.1703Proc. Natl. Acad. Sci. U.S.A. 45Detection, assay, and preliminary purification of the pigment controlling pho- toresponsive development of plants. Proc. Natl. Acad. Sci. U.S.A. 45, 1703-1708. doi: 10.1073/pnas.45.12.1703

the argonaute family: tentacles that reach into RNAi, developmental control, stem cell maintenance, and tumorigenesis. M A Carmell, Z Xuan, M Q Zhang, G J Hannon, 10.1101/gad.1026102Genes Dev. 16Carmell, M. A., Xuan, Z., Zhang, M. Q., and Hannon, G. J. (2002). the argonaute family: tentacles that reach into RNAi, developmental control, stem cell mainte- nance, and tumorigenesis. Genes Dev. 16, 2733-2742. doi: 10.1101/gad.1026102

GtRNAdb: A database of transfer RNA genes detected in genomic sequence. P P Chan, T M Lowe, 10.1093/nar/gkn787Nucl. Acids Res. 37Chan, P. P., and Lowe, T. M. (2009). GtRNAdb: A database of transfer RNA genes detected in genomic sequence. Nucl. Acids Res. 37, 93-97. doi: 10.1093/nar/gkn787

A simple and efficient method for isolating RNA from pine trees. S Chang, J Puryear, J Cairney, 10.1007/BF02670468Plant Mol. Biol. Rep. 11Chang, S., Puryear, J., and Cairney, J. (1993). A simple and efficient method for isolating RNA from pine trees. Plant Mol. Biol. Rep. 11, 113-116. doi: 10.1007/BF02670468

22-nucleotide RNAs trigger secondary siRNA biogenesis in plants. H M Chen, L T Chen, K Patel, Y H Li, D C Baulcombe, S H Wu, 10.1073/pnas.1001738107Proc. Natl. Acad. Sci. U.S.A. 107Chen, H. M., Chen, L. T., Patel, K., Li, Y. H., Baulcombe, D. C., and Wu, S. H. (2010). 22-nucleotide RNAs trigger secondary siRNA biogenesis in plants. Proc. Natl. Acad. Sci. U.S.A. 107, 15269-15274. doi: 10.1073/pnas.1001738107

Phytochrome signaling mechanisms and the control of plant development. M Chen, J Chory, 10.1016/j.tcb.2011.07.002Trends Cell Biol. 21Chen, M., and Chory, J. (2011). Phytochrome signaling mechanisms and the control of plant development. Trends Cell Biol. 21, 664-671. doi: 10.1016/j.tcb.2011.07.002

Small RNAs and their roles in plant development. X Chen, 10.1146/annurev.cellbio.042308.113417Annu. Rev. Cell Dev. Biol. 25Chen, X. (2009). Small RNAs and their roles in plant development. Annu. Rev. Cell Dev. Biol 25, 21-44. doi: 10.1146/annurev.cellbio.042308.113417

Small RNA-secrets and surprises of the genome. X Chen, 10.1111/j.1365-313X.2009.04089.xPlant J. 61Chen, X. (2010). Small RNA-secrets and surprises of the genome. Plant J. 61, 941-958. doi: 10.1111/j.1365-313X.2009.04089.x

The phytochrome apoprotein family in Arabidopsis is encoded by five genes: the sequences and expression of PHYD and PHYE. T Clack, S Mathews, R A Sharrock, 10.1007/BF00043870Plant Mol. Biol. 25Clack, T., Mathews, S., and Sharrock, R. A. (1994). The phytochrome apoprotein family in Arabidopsis is encoded by five genes: the sequences and expression of PHYD and PHYE. Plant Mol. Biol. 25, 413-427. doi: 10.1007/BF00043870

psRNAtarget: a plant small RNA target analysis server. X Dai, P X Zhao, 10.1093/nar/gkr319Nucleic Acids Res. 39Dai, X., and Zhao, P. X. (2011). psRNAtarget: a plant small RNA target analysis server. Nucleic Acids Res. 39, W115-W159. doi: 10.1093/nar/gkr319

Phytochrome E influences internode elongation and flowering time in Arabidopsis. P F Devlin, S R Patel, G C Whitelam, 10.1105/tpc.10.9.1479Plant Cell. 19Devlin, P. F., Patel, S. R., and Whitelam, G. C. (1998). Phytochrome E influ- ences internode elongation and flowering time in Arabidopsis. Plant Cell 19, 1479-1487. doi: 10.1105/tpc.10. 9.1479

Transposing phytochrome into the nucleus. C Fankhauser, Chen , M , 10.1016/j.tplants.2008.08.007Trends Plant Sci. 13Fankhauser, C., and Chen, M. (2008). Transposing phytochrome into the nucleus. Trends Plant Sci. 13, 596-601. doi: 10.1016/j.tplants.2008.08.007

Phytochrome functions in Arabidopsis development. K Franklin, P H Quail, 10.1093/jxb/erp304J. Exp. Bot. 61Franklin, K., and Quail, P. H. (2010). Phytochrome functions in Arabidopsis development. J. Exp. Bot. 61, 11-14. doi: 10.1093/jxb/erp304

Rfam: updates to the RNA families database. P P Gardner, J Daub, J G Tate, E P Nawrocki, D L Kolbe, S Lindgreen, 10.1093/nar/gkn766Nucleic Acids Res. 37Gardner, P. P., Daub, J., Tate, J. G., Nawrocki, E. P., Kolbe, D. L., Lindgreen, S., et al. (2008). Rfam: updates to the RNA families database. Nucleic Acids Res. 37, 136-140. doi: 10.1093/nar/gkn766

A species of small antisense RNA in posttranscriptional gene silencing in plants. A J Hamilton, D C Baulcombe, 10.1126/science.286.5441.950Science. 286Hamilton, A. J., and Baulcombe, D. C. (1999). A species of small antisense RNA in posttranscriptional gene silencing in plants. Science 286, 950-952. doi: 10.1126/science.286.5441.950

Prediction and identification of natural antisense transcripts and their small RNAs in soybean (Glycine max). Z Hu, Q Jiang, Z Ni, H Zhang, 10.1186/1471-2164-14-280BMC Genomics. 14280Hu, Z., Jiang, Q., Ni, Z., and Zhang, H. (2013). Prediction and identification of natural antisense transcripts and their small RNAs in soybean (Glycine max). BMC Genomics 14:280. doi: 10.1186/1471-2164-14-280

Prediction of novel miRNAs and associated target genes in Glycine max. T Joshi, Z Yan, M Libault, D H Jeong, S Park, P J Green, 10.1186/1471-2105-11-S1-S14BMC Bioinformatics. 1114Joshi, T., Yan, Z., Libault, M., Jeong, D. H., Park, S., Green, P. J., et al. (2010). Prediction of novel miRNAs and associated target genes in Glycine max. BMC Bioinformatics 11:S14 doi: 10.1186/1471-2105-11-S1-S14

microRNA-mediated repression of rolled leaf1 specifies maize leaf polarity. M T Juarez, J S Kui, J Thomas, B A Heller, M C P Timmermans, 10.1038/nature02363428Juarez, M. T., Kui, J. S., Thomas, J., Heller, B. A., and Timmermans, M. C. P. (2004). microRNA-mediated repression of rolled leaf1 specifies maize leaf polarity. 428, 84-88. doi: 10.1038/nature02363

MIR166/165 genes exhibit dynamic expression patterns in regulating shoot apical meristem and floral development in Arabidopsis. J H Jung, C M Park, 10.1007/s00425-006-0439-1Planta. 225Jung, J. H., and Park, C. M. (2007). MIR166/165 genes exhibit dynamic expres- sion patterns in regulating shoot apical meristem and floral development in Arabidopsis. Planta 225, 1327-1338. doi: 10.1007/s00425-006-0439-1

Dynamics of Gene expression revealed by comparison of serial analysis of gene expression transcript profiles from yeast grown on two different carbon sources. A J Kal, A J Zonneveld, V Benes, M Berg, M G Koerkamp, K Albermann, 10.1091/mbc.10.6.1859Mol. Biol. Cell. 10Kal, A. J., Zonneveld, A. J., Benes, V., Berg, M., Koerkamp, M. G., Albermann, K., et al. (1999). Dynamics of Gene expression revealed by comparison of serial analysis of gene expression transcript profiles from yeast grown on two different carbon sources. Mol. Biol. Cell 10, 1859-1872. doi: 10.1091/mbc.10. 6.1859

miRBase: integrating microRNA annotation and deep-sequencing data. A Kozomara, S Griffiths-Jones, 10.1093/nar/gkq1027Nucleic Acids Res. 39Kozomara, A., and Griffiths-Jones, S. (2011). miRBase: integrating microRNA annotation and deep-sequencing data. Nucleic Acids Res. 39, D152-D157. doi: 10.1093/nar/gkq1027

The C.elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14. R C Lee, R L Feinhaum, Ambros , V , 10.1016/0092-8674Cell. 759390529Lee, R. C., Feinhaum, R. L., and Ambros, V. (1993). The C.elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14. Cell 75, 842-854. doi: 10.1016/0092-8674(93)90529-Y

Chapter 15: roles of microRNAs in plant abiotic stress. R Lewis, V Mendu, D Mcnear, G Tang, 10.1007/978-90-481-2967-6Molecular Techniques in Crop Improvement. S. M. Jain and D. S. BrarNew York, NYSpringerLewis, R., Mendu, V., McNear, D., and Tang, G. (2009). "Chapter 15: roles of microRNAs in plant abiotic stress," in Molecular Techniques in Crop Improvement, eds S. M. Jain and D. S. Brar (New York, NY: Springer), 357-372. doi: 10.1007/978-90-481-2967-6

Identification of soybean MicroRNAs involved in soybean cyst nematode infection by deep sequencing. X Li, X Wang, S Zhang, D Liu, Y Duan, Dong , W , 10.1371/journal.pone.0039650PLoS ONE. 739650Li, X., Wang, X., Zhang, S., Liu, D., Duan, Y., and Dong, W. (2012a). Identification of soybean MicroRNAs involved in soybean cyst nematode infection by deep sequencing. PLoS ONE 7:e39650. doi: 10.1371/journal.pone.0039650

Rapid, organ-specific transcriptional responses to light regulate photomorphogenic development in dicot seedlings. Y Li, K Swaminathan, M E Hudson, 10.1104/pp.111.179416Plant Physiol. 156Li, Y., Swaminathan, K., and Hudson, M. E. (2011). Rapid, organ-specific tran- scriptional responses to light regulate photomorphogenic development in dicot seedlings. Plant Physiol 156, 2124-2140. doi: 10.1104/pp.111.179416

The inheritance pattern of 24 nt siRNA clusters in Arabidopsis hybrids is influenced by proximity to transposable elements. Y Li, K Varala, S P Moose, M E Hudson, 10.1371/journal.pone.0047043PLoS ONE. 747043Li, Y., Varala, K., Moose, S. P., and Hudson, M. E. (2012b). The inheritance pattern of 24 nt siRNA clusters in Arabidopsis hybrids is influenced by proximity to transposable elements. PLoS ONE 7:e47043. doi: 10.1371/journal.pone.0047043

Ectopic expression of miR396 suppresses GRF target gene expression and alters leaf growth in Arabidopsis. D Liu, Y Song, Z Chen, Yu , D , 10.1111/j.1399-3054.2009.01229.xPhysiol Plant. 136Liu, D., Song, Y., Chen, Z., and Yu, D. (2009). Ectopic expression of miR396 sup- presses GRF target gene expression and alters leaf growth in Arabidopsis. Physiol Plant. 136: 223-236. doi: 10.1111/j.1399-3054.2009.01229.x

Elucidation of the small RNA component of the transcriptome. C Lu, S S Tej, S Luo, C D Haudenschild, B C Meyers, P Green, 10.1126/science.1114112Science. 2Lu, C., Tej, S. S., Luo, S., Haudenschild, C. D., Meyers, B. C., and Green, P. (2005). Elucidation of the small RNA component of the transcriptome. Science 2, 1567-1569. doi: 10.1126/science.1114112

UNAFold: software for nucleic acid folding and hybridization. N R Markham, M Zuker, 10.1007/978-1-60327-429-6_1Methods Mol. Biol. 453Markham, N. R., and Zuker, M. (2008). UNAFold: software for nucleic acid fold- ing and hybridization. Methods Mol. Biol. 453, 3-31. doi: 10.1007/978-1-60327- 429-6_1

Criteria for annotation of plant MicroRNAs. B C Meyers, M J Axtell, B Bartel, D P Bartel, D Baulcombe, J L Bowman, 10.1105/tpc.108.064311Plant Cell. 20Meyers, B. C., Axtell, M. J.,Bartel, B., Bartel, D. P., Baulcombe, D., Bowman, J. L., et al. (2008). Criteria for annotation of plant MicroRNAs. Plant Cell 20, 3186-3190. doi: 10.1105/tpc.108.064311

Fertile hypomorphic ARGONAUTE (ago1) mutants impaired in posttranscriptional gene silencing and virus resistance. J B Morel, C Godon, P Mourrain, C Beclin, S Bouter, F Feuerbach, 10.1105/tpc.010358Plant Cell. 14Morel, J. B., Godon, C., Mourrain, P., Beclin, C., Bouter, S., Feuerbach, F., et al. (2002). Fertile hypomorphic ARGONAUTE (ago1) mutants impaired in post- transcriptional gene silencing and virus resistance. Plant Cell 14, 629-639. doi: 10.1105/tpc.010358

Phytochromes control photomorphogenesis by differentially regulated, interacting signaling pathways in higher plants. F Nagy, E Schafer, Nagy, F., and Schafer, E. (2002). Phytochromes control photomorphogenesis by differentially regulated, interacting signaling pathways in higher plants.

. 10.1146/annurev.arplant.53.100301.135302Annu. Rev. Plant Biol. 53Annu. Rev. Plant Biol. 53, 329-355. doi: 10.1146/annurev.arplant.53.100301. 135302

Two small regulatory RNAs establish opposing fates of a developmental axis. F T S Nogueiral, S Madi, D H Chitwood, M T Juarez, M C P Timmermans, 10.1101/gad.1528607Genes Dev. 21Nogueiral, F. T. S., Madi, S., Chitwood, D. H., Juarez, M. T., and Timmermans, M. C. P. (2007). Two small regulatory RNAs establish opposing fates of a developmental axis. Genes Dev. 21, 750-755. doi: 10.1101/gad.1528607

Improved tools for biological sequence comparison. W R Pearson, D J Lipman, 10.1073/pnas.85.8.2444Proc. Natl. Acad. Sci. U.S.A. 85Pearson, W. R., and Lipman, D. J. (1988). Improved tools for biological sequence comparison. Proc. Natl. Acad. Sci. U.S.A. 85, 2444-2448. doi: 10.1073/pnas.85.8.2444

Asymmetric Responsiveness to Ethylene mediates cell elongation in the apical hook of peas. S C Peck, K Pawlowski, H Kende, 10.1105/tpc.10.5.713Plant Cell. 10Peck, S. C., Pawlowski, K., and Kende, H. (1998). Asymmetric Responsiveness to Ethylene mediates cell elongation in the apical hook of peas. Plant Cell 10, 713-719. doi: 10.1105/tpc.10.5.713

Efficient frequency-based de novo short-read clustering for error trimming in next-generation sequencing. W Qu, S Hashimoto, S Morishita, Qu, W., Hashimoto, S., and Morishita, S. (2009). Efficient frequency-based de novo short-read clustering for error trimming in next-generation sequencing.

. 10.1101/gr.089151.108Genome Res. 19Genome Res. 19, 1309-1315. doi: 10.1101/gr.089151.108

Phytochrome photosensory signalling networks. P H Quail, 10.1038/nrm728Nature Rev. Mol. Cell Biol. 3Quail, P. H. (2002). Phytochrome photosensory signalling networks. Nature Rev. Mol. Cell Biol. 3, 85-93. doi: 10.1038/nrm728

Phytochrome-regulated gene expression. P H Quail, 10.1111/j.1744-7909.2006.00422.xJ. Integ. Plant Biol. 49Quail, P. H. (2007). Phytochrome-regulated gene expression. J. Integ. Plant Biol. 49, 11-20. doi: 10.1111/j.1744-7909.2006.00422.x

Control of cell proliferation in Arabidopsis thaliana by microRNA miR396. R E Rodriguez, M A Mecchia, J M Debernardi, C Schommer, D Weigel, J F Palatnik, 10.1242/dev.043067Development. 137Rodriguez, R. E., Mecchia, M. A., Debernardi, J. M., Schommer, C., Weigel, D., and Palatnik, J. F. (2010). Control of cell proliferation in Arabidopsis thaliana by microRNA miR396. Development 137, 103-112. doi: 10.1242/dev.043067

. J Schmutz, S B Cannon, J Schlueter, J Ma, T Mitros, W Nelson, Schmutz, J., Cannon, S. B., Schlueter, J., Ma, J., Mitros, T., Nelson, W., et al. (2010).

Genome sequence of the palaeopolyploid soybean. 10.1038/nature08670Nature. 463Genome sequence of the palaeopolyploid soybean. Nature 463, 178-183. doi: 10.1038/nature08670

Novel phytochrome sequences in Arabidopsis thaliana: structure, evolution, and differential expression of a plant regulatory photoreceptor family. R A Sharrock, P H Quail, 10.1101/gad.3.11.1745Genes Dev. 3Sharrock, R. A., and Quail, P. H. (1989). Novel phytochrome sequences in Arabidopsis thaliana: structure, evolution, and differential expression of a plant regulatory photoreceptor family. Genes Dev. 3 :1745-1757. doi: 10.1101/gad.3.11.1745

Phytochrome and Photomorphogenesis. London: MrGraw-Hill. H Smith, 10.1042/bst0040396Smith, H. (1974). Phytochrome and Photomorphogenesis. London: MrGraw-Hill. doi: 10.1042/bst0040396

Phytochromes and light signal perception by plants-an emerging synthesis. H Smith, 10.1038/35036500Nature. 407Smith, H. (2000). Phytochromes and light signal perception by plants-an emerg- ing synthesis. Nature 407, 585-591. doi: 10.1038/35036500

Auxin and light control of adventitious rooting in Arabidopsis require ARGONAUTE1. C Sorin, J D Bussell, I Camus, K Ljung, M Kowalczyk, G Geiss, 10.1105/tpc.105.031625Plant Cell. 17Sorin, C., Bussell, J. D., Camus, I., Ljung, K., Kowalczyk, M., Geiss, G., et al. (2005). Auxin and light control of adventitious rooting in Arabidopsis require ARGONAUTE1. Plant Cell 17, 1343-1359. doi: 10.1105/tpc.105.031625

Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. H Thorvaldsdóttir, J T Robinson, J P Mesirov, 10.1093/bib/bbs017Brief Bioinform. 14Thorvaldsdóttir, H., Robinson, J. T., and Mesirov, J. P. (2012). Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. Brief Bioinform. 14, 178-192. doi: 10.1093/bib/bbs017

. J H Tuteja, G Zabala, K Varala, M E Hudson, L O Vodkin, Tuteja, J. H., Zabala, G., Varala, K., Hudson, M. E., and Vodkin, L. O. (2008).

tissue-specific short interfering RNAs silence the chalcone synthase gene family in Glycine Max seed coats. Endogenous, 10.1105/tpc.109.069856Plant Cell. 21Endogenous, tissue-specific short interfering RNAs silence the chalcone syn- thase gene family in Glycine Max seed coats. Plant Cell 21, 3063-3077. doi: 10.1105/tpc.109.069856

Rapid genotyping of soybean cultivars using high throughput sequencing. K Varala, K Swaminathan, Y Li, M E Hudson, 10.1371/journal.pone.0024811PLoS ONE. 624811Varala, K., Swaminathan, K., Li, Y., and Hudson, M. E. (2011). Rapid genotyping of soybean cultivars using high throughput sequencing. PLoS ONE 6: e24811. doi: 10.1371/journal.pone.0024811

Regulation of Arabidopsis shoot apical meristem and lateral organ formation by microRNA miR166g and its AtHD-ZIP target genes. L Williams, S P Grigg, M Xie, S Christensen, J C Fletcher, 10.1242/dev.01942Development. 132Williams, L., Grigg, S. P., Xie, M., Christensen, S., and Fletcher, J. C. (2005). Regulation of Arabidopsis shoot apical meristem and lateral organ forma- tion by microRNA miR166g and its AtHD-ZIP target genes. Development 132, 3654-3668. doi: 10.1242/dev.01942

Arabidopsis microRNA167 controls patterns of ARF6 and ARF8 expression, and regulates both female and male reproduction. M F Wu, Q Tian, J W Reed, 10.1242/dev.02602Development. 133Wu, M. F., Tian, Q., and Reed, J. W. (2006). Arabidopsis microRNA167 controls patterns of ARF6 and ARF8 expression, and regulates both female and male reproduction. Development 133, 4211-4218. doi: 10.1242/dev.02602

Divergent patterns of endogenous small RNA populations from seed and vegetative tissues of Glycine max. G Zabala, E Campos, K K Varala, S Bloomfield, S I Jones, H Win, 10.1186/1471-2229-12-177BMC Plant Biol. 12177Zabala, G., Campos, E., Varala, K. K., Bloomfield, S., Jones, S. I., Win, H., et al. (2012). Divergent patterns of endogenous small RNA populations from seed and vegetative tissues of Glycine max. BMC Plant Biol. 12:177. doi: 10.1186/1471-2229-12-177

Genome-wide mapping of the HY5-mediated gene networks in Arabidopsis that involve both transcriptional and post-transcriptional regulation. H Zhang, H He, X Wang, X Wang, X Yang, L Li, 10.1111/j.1365-313X.2010.04426.xPlant J. 65Zhang, H., He, H., Wang, X., Wang, X., Yang, X., Li, L., et al. (2011). Genome-wide mapping of the HY5-mediated gene networks in Arabidopsis that involve both transcriptional and post-transcriptional regulation. Plant J. 65, 346-358. doi: 10.1111/j.1365-313X.2010.04426.x

Regulation of HD-ZIPIII genes by MicroRNA 165. R Zhong, Ye , Z H , 10.4161/psb.2.5.4119Plant Signal. Behav. 2Zhong, R., and Ye, Z. H. (2007). Regulation of HD-ZIPIII genes by MicroRNA 165. Plant Signal. Behav. 2, 351-353. doi: 10.4161/psb.2.5.4119

Conflict of Interest Statement: The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest. Conflict of Interest Statement: The authors declare that the research was con- ducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.