# Overview of noncoding RNAs involved in the osteogenic differentiation of periodontal ligament stem cells

CorpusID: 218618253
 
tags: #Biology, #Medicine

URL: [https://www.semanticscholar.org/paper/e3f1eee365cbe3629b169e812e307b0207d2c957](https://www.semanticscholar.org/paper/e3f1eee365cbe3629b169e812e307b0207d2c957)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Overview of noncoding RNAs involved in the osteogenic differentiation of periodontal ligament stem cells


Wei Qiu 
Bu-Ling Wu 
Fu-Chun 

Department of Stomatology
Department of Stomatology
Nanfang Hospital
Southern Medical University
510515Guangzhou, Guangdong ProvinceChina


Guangdong Province
Nanfang Hospital
Southern Medical University
1838 Guangzhou Avenue North510515GuangzhouChina

Overview of noncoding RNAs involved in the osteogenic differentiation of periodontal ligament stem cells
10.4252/wjsc.v12.i4.251W J S C World Journal of Stem Cells Submit a Manuscript: https://www.f6publishing.com World J Stem Cells 2020 April 26; 12(4): 251-265 REVIEW Author contributions: Fang FC and Wu BL contributed to the conception and design of the review; Qiu W contributed to the writing and drafting of the manuscript; Fang FC and Qiu W contributed to the critical revision of the manuscript for important intellectual content; all the authors have given approval of the final version to be published and agree to be accountable for all aspects of the work. Conflict-of-interest statement: The authors declare that they have no conflict of interest.


## INTRODUCTION

Periodontal diseases are infectious diseases characterized by progressive destruction of the periodontium (tooth-supporting tissue), which includes the periodontal ligament (PDL), cementum, alveolar bone, and gingiva [1] . Tooth loss mainly results from periodontal diseases in adults, which adds a substantial burden to public health worldwide [2,3] . Periodontal treatment is not as easy as only controlling inflammation and preventing disease development; the reconstruction of a healthy periodontium destroyed by diseases deserves equal attention [4,5] . Current therapies for periodontal diseases in the clinic, including conservative approaches, radicular conditioning, bioactive bone grafting/substitution, and guided tissue regeneration (GTR), encounter difficulty in regenerating the periodontium completely [6] . Therefore, the stem cell-based tissue regeneration approach involving transplantation of stem cells to enhance periodontal tissue regeneration has gradually taken the place of guided bone/tissue regeneration [7][8][9] .

PDL is a specialized soft connective tissue that connects the cementum and alveolar bone; it shows the function of maintaining and supporting teeth in situ, preserving tissue homoeostasis and repairing damaged periodontal tissue [1] . In the 1980s, Bordin et al [10] reported that PDL tissue possessed periodontal regenerative properties due to its resident cells, which were considered to be seed cells and a reliable source for periodontium regeneration. In 2004, Seo et al [11] first identified and characterized multipotent stem cells in human PDL and termed them periodontal ligament stem cells (PDLSCs). PDLSCs show similar features to other postnatal mesenchymal stem cells (MSCs): Multilineage differentiation potential and potent self-renewal ability. PDLSCs can further differentiate into cementoblasts/osteoblasts, chondrocytes and adipocytes in vitro and regenerate cementum/PDL-like tissues in vivo [12] . As a consequence, PDLSC-mediated periodontium tissue regeneration is likely to be a practical cellular-based treatment for periodontal diseases [13] . However, what determines and regulates the multilineage differentiation potential of PDLSCs warrants further research.

Instead of the potential to encode proteins or peptides, noncoding RNAs (ncRNAs) are a category of unique RNAs that are widely present in eukaryotic cells [14][15][16] . Following the development of this field, scientists have determined that ncRNAs play a significant role in the regulation of gene expression by controlling the expression levels of protein-coding RNAs and are involved in diverse cellular processes, including cell proliferation, cell differentiation, and ontogenesis, and are thus closely related to embryonic development and disease pathogenesis [17][18][19] . However, there are currently no uniform criteria for ncRNA classification. ncRNAs can be divided into cytoplasmic and nuclear ncRNAs based on their subcellular localization. In addition, ncRNAs are generally categorized into structural and regulatory ncRNAs, as well as regarding their function in cellular processes [20] . Structural ncRNAs include ribosomal RNAs (rRNAs) and transfer RNAs (tRNAs), whereas regulatory RNAs can be further divided into categories based on their length, such as long noncoding RNAs (lncRNAs) (size from > 200 nt to 100 kb) and several types of small RNAs, which include small interfering siRNAs (18-30 nt), piwiRNAs (24-30 nt) and microRNAs (miRNAs,(20)(21)(22)(23)(24) [21] . Circular RNAs (circRNAs) are covalently linked to the end of RNA molecules, in which the 3′ and 5′ ends are connected in a non-collinear way through the back-splicing process [22] . CircRNAs, which are a type of competing endogenous RNA (ceRNA), can act as miRNA sponges. Recently, growing research has indicated that circRNAs are involved in embryonic development, cellular activities, and many other human diseases [23][24][25] . In summary, the cell differentiation of PDLSCs is collectively or individually regulated by ncRNAs. This review focuses on the three most important ncRNAs, namely, miRNAs, lncRNAs and circRNAs, currently identified to play a role in osteogenic differentiation (Tables 1 and 2).  


## HISTORY OF PDLSCS

The progenitor cells residing within the PDL (periodontal ligament progenitor, PDLPs) were first described in seminal studies by Melcher in 1994 [26] . Seo et al [11] described the identification and characterization of multipotent stem cells in human PDL in 2004, although these cells had been suspected to be present in the PDL for a long time. Nevertheless, there is no uniform standard for defining the features of PDLSCs. Often, reports suggest that the isolation of particular subsets of cells from bulk explant cultures is far less rigorous and was too liberal for the use of the term PDLSC [27] . Prateeptongkum et al [28] reported that the isolation methods of PDLPs and PDLSCs from PDL tissues are different and demonstrated that PDLPs could be isolated using outgrowth methods, while PDLSCs need single-cell isolation methods for isolation. PDLSCs can be further characterized by their cell surface expression of CD29, CD44, STRO-1, STRO-4, CD146, CD73, CD90, CD105 and CD166 and the lack of expression of endothelial (CD31), haematopoietic (CD14, CD34, CD45, and CD79a), and helper immune antigens (HLA-DR, CD40, CD54, CD80, and CD86) [10,29] . Functionally, PDLSCs have been determined to fulfil all of the criteria of identifiable MSC-like properties, including self-renewal capacity, multipotency in vitro, tissue regenerative capacity in vivo, and immunomodulation [30,31] . These processes are illustrated in Figure 1.


## MICRORNAS IN PDLSCS: ORCHESTRATING CELLULAR OSTEOGENIC DIFFERENTIATION

miRNAs are endogenous, single-stranded noncoding RNAs derived from genomic sequences [32] . The lengths of mature miRNAs are typically 20~24 nucleotides, 8 of which are identified as the "seed sequences" (with nt positions 2 to 7 that were 99% conserved) [33,34] . miRNAs have been extensively investigated in the past two decades, and the underlying mechanism is relatively clear. Mature miRNAs are targeted to a sequence in the 3′ UTR (untranslatedregion) of mRNAs matching the seed sequence and further influence the stability of mRNAs or inhibit their translation to eventually downregulate protein expression [35] . miRNAs are a leading representative of small ncRNAs, and they are closely associated with diverse biological and pathological processes. miRNA microarrays are a widely accepted high-throughput method and are very effective in analysing miRNA expression levels during osteogenic differentiation of PDLSCs [36] . Our team used a miRNA microarray to detect the different expression profiles of miRNAs in PDLSCs during the osteogenic differentiation process in vitro [37] . The results showed a significant change in the expression level of 116 miRNAs, 30 of which were increased, while 86 miRNAs were downregulated in PDLSCs after 14 d of osteogenic induction. The results probably suggested an important regulatory role that miRNAs might play in the osteogenic differentiation of PDLSCs.

Similarly, a microarray was used in the study of Li et al [38] to detect the expression level of miRNAs in differentiated and undifferentiated PDLSCs and demonstrated that the expression level of miR-24-3p was significantly downregulated in  osteogenically differentiated PDLSCs. Furthermore, double luciferase reporter assays and genetic engineering experiments demonstrated that miR-24-3p directly bound to the 3'-UTR of transduction protein 5 (SMAD family member 5, Smad5) and inhibited the transcription of the target gene.

The inhibition of the osteogenic differentiation of PDLSCs was the result of miR-21 downregulating the expression of Smad5 in the research of Wei et al [39] . Yang et al [40] found that the inhibition was attributed to the regulation of the miR-21/palmitate phosphoprotein Sprouty1 axis by tissue tumour necrosis factor-α. In addition, miR-203, miR-1305, and miR-218 have all been confirmed to target runt-related transcriptionfactor 2 (Runx2) and play important inhibitory roles in the osteogenic differentiation of PDLSCs [41][42][43] . miR-214 not only targeted activated transcription factor 4 [44] but also bound with catenin beta 1 to modulate the Wnt/β-catenin signalling pathway, which is involved in osteogenic differentiation of PDLSCs [45] . Liu et al [46] found that miR-17 regulated the osteogenic differentiation of PDLSCs by reducing the expression of transcriptional factor 3 and inhibiting the Wnt signalling pathway. In contrast, Liu et al [47] demonstrated that miR-17 promoted differentiation by binding to the Smad ubiquitin regulatory factor one 3'-UTR in PDLSCs isolated from PDL tissue from periodontitis patients. miR-31 plays a regulatory role by targeting special ATrich sequence-binding protein 2 in osteogenic differentiation mediated by a high dose of glucose in PDLSCs [48] . All of the abovementioned miRNAs exerted inhibitory effects on osteogenic differentiation by targeting osteogenesis-related transcription factors through the classic miRNA regulatory mechanism.

Although several miRNAs suppress the osteogenic differentiation of PDLSCs, recent research has revealed that miRNAs promote the osteogenic differentiation of PDLSCs, including miR-200c, miR-543 and miR-22. Hong et al [49] demonstrated that miR-200c decreased the levels of interleukin-6, interleukin-8 and chemokines-5 and increased the osteogenic differentiation of PDLSCs and BMSCs. miR-200c is a potentially effective means of preventing periodontitis-associated bone loss by arresting inflammation and osteoclast/osteogenesis and regenerating bone tissue. Previous research by our team found that miR-543 directly interacted with the 3'-UTR of transducer of ERBB2 and promoted osteogenesis in PDLSCs [50] . Yan et al [51] claimed that miR-22 promoted the osteogenic differentiation of PDLSCs by inhibiting the expression of histone deacetylase 6.

According to previous work, one of the factors affecting osteogenic differentiation of PDLSCs is mechanical stretch [52][53][54] . To investigate miRNA expression specifically in stretched PDLSCs, a microarray assay was utilized by Wei et al [55] to describe the differential expression of miRNAs in normal and stretched PDLSCs by using a tension system to achieve external mechanical stimulation. The results showed that 53 miRNAs were differentially expressed in stretched PDLSCs, and 26 of the miRNAs were upregulated, while 27 were downregulated. Noticeably, miR-21 directly targeted the 3'-UTR of activin A receptor type 2B (ACVR2B), thereby reducing the expression of ACVR2B and repressing the osteogenic differentiation of stretched PDLSCs [56] .

The main regulatory mechanism of microRNAs is the posttranscriptional repression of target genes. However, several studies have reported that miRNAs function in other unconventional ways, including pri-miRNAs coding for short peptides and miRNAs interacting with non-AGO proteins, activating toll-like receptors, upregulating protein expression, targeting mitochondrial transcripts, directly activating transcription, and targeting nuclear ncRNAs [57][58][59] . To date, research has mainly focused on the classic regulatory mechanism of miRNAs in PDLSCs, and other regulatory mechanisms require further in-depth exploration.


## LNCRNAS INVOLVED IN OSTEOGENIC DIFFERENTIATION OF PDLSCs

LncRNAs are a family of RNA molecules with transcript lengths of 200 nt to 20000 nt. These RNAs are unable to encode proteins or are only translated into small peptides at a very low level. Initially, lncRNAs were identified as a by-product of RNA polymerase II transcription and thought to be the "noise" of genomic transcription (referred to as "the dark matter" of the genome) with no biological function [36,60,61] . However, research on lncRNAs has rapidly developed in recent years. According to their position relative to protein-coding genes in the genome, lncRNAs can be divided into five types: Sense, antisense, bidirectional, intronic, and intergenic lncRNAs [62] . Recent studies show that lncRNAs act as novel and important regulators of numerous biological, developmental, and numerous cellular processes, including chromatin modification, X-chromosome silencing, genomic imprinting, transcriptional activation or interference, and intranuclear transport [63] , and act through such mechanisms as transcriptional regulation, posttranscriptional regulation, and epigenetic regulation [64][65][66] .

Gu et al [67] compared the lncRNA profiles of PDLSCs on the 7th day with or without osteogenic differentiation medium using RNA sequencing. The results showed that 17 lncRNAs were upregulated and 31 were downregulated during osteogenic differentiation in PDLSCs. Zhang et al [68] also used RNA sequencing to detect the different expression profiles of lncRNAs in PDLSCs at different time points during osteogenic differentiation. These results indicated that 48 lncRNAs had significant changes on days 3, 7 and 14, of which 17 lncRNAs were upregulated and 31 were downregulated in PDLSCs. Our team used a lncRNA microarray to determine the expression levels of lncRNAs in osteoblast-induced and noninduced PDLSCs, and the results showed that 994 lncRNAs were upregulated and 1177 lncRNAs were downregulated at 14 d of osteogenic differentiation in PDLSCs. Further GO analysis and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analysis revealed that a total of 83 signalling pathways were involved in the osteogenic differentiation of PDLSCs, including the mitogen-activated protein kinase (MAPK) and transforming growthfactor-β signalling pathways. In addition, coding-noncodinggenecoexpression analysis indicated a potential regulatory relationship between the differentially expressed lncRNAs and the osteogenesis-related mRNAs, of which 131 pairs of lncRNAs and mRNAs had negative correlations and 262 pairs had positive correlations [69] . Huang et al [70] used RNA sequencing to describe the lncRNA landscape during osteogenic differentiation in PDLSCs subjected to compressive force. The results indicated that 90 lncRNAs and 519 mRNAs were differentially expressed in PDLSCs under compressive stress. Of the lncRNAs, 72 were upregulated, and 18 were downregulated. To summarize, investigations of the regulatory mechanisms of lncRNAs in the osteogenic differentiation of PDLSCs are mainly focused on transcriptional regulation and posttranscriptional regulation.


## Transcriptional regulation by lncRNAs in PDLSCs

Studies of the underlying mechanisms indicated that lncRNAs can specifically bind with a specific family of proteins called RNA binding proteins (RBPs) to regulate the biological functions of these proteins and then affect the transcriptional activation of downstream genes [71] . As a novel protein type, RBPs have been demonstrated to interact with lncRNAs and have attracted increasing attention from researchers. He et al [72] identified the novel lncRNA-taurine upregulated gene 1 (TUG1), which was significantly upregulated in osteogenically induced PDLSCs. Gain/loss-of-function experiments showed that lncRNA-TUG1 was positively correlated with the osteogenic differentiation of PDLSCs following induction. Meanwhile, bioinformatic analysis demonstrated that lin-28 homologue A (Lin28A, a member of the RBP family) containing multiple binding sites of lncRNA-TUG1 was a potential target in the process of osteogenic differentiation of PDLSCs. Further investigation demonstrated that suppression of Lin28A inhibited the expression of several osteogenic-related gene markers, such as alkaline phosphatase, osteocalcin, and Runx2, in PDLSCs. These results suggested that lncRNA-TUG1 might interact with Lin28A to form a positive regulatory network of osteogenic-related genes to promote osteogenic differentiation of PDLSCs. However, there is insufficient evidence about the direct binding between lncRNA-TUG1 and Lin28A in this study. In a study by Liu et al [73] , lncRNA maternally expressed gene 3 (MEG3) was markedly downregulated in osteogenically differentiated human PDLSCs compared with undifferentiated cells, as determined by microarray analysis. Overexpression of lncRNA MEG3 inhibited the activation of bone morphogenetic protein-2 (BMP2) and reversed osteogenic differentiation induced by mineralizing solution in PDLSCs. Furthermore, RNA-binding protein immunoprecipitation (RIP) assays verified that lncRNA MEG3 suppressed BMP2 through direct interaction with heterogeneous nuclear ribonucleoprotein I (hnRNPI) during osteogenic differentiation in hPDLCs. These results indicated that lncRNA MEG3 could modulate the osteogenic differentiation of hPDLCs by interacting with hnRNPI and thus inhibiting the transcriptional activity of BMP2. Moreover, lncRNAs can also interfere with the transcriptional activation of mRNAs or other ncRNAs. Jia et al [74] found an inhibitory effect of ANCR (anti-differentiation noncoding RNA) on the gene expression of glycogen synthase kinase 3β and Runx2 and the classic Wnt/βcatenin signalling pathway, thereby suppressing osteogenic differentiation of PDLSCs.


## Posttranscriptional regulation of lncRNAs in PDLSCs

Recent studies revealed that lncRNAs might act as miRNA sponges to compete with target genes for miRNA binding sites, thereby affecting the activity of the targeted genes in various biological processes [75] . These regulatory mechanisms involving lncRNAs, miRNAs and mRNAs are one type of ceRNA [76] mechanism. Wang et al [77] successfully identified the novel lncRNA-POIR (osteogenesis impairment-related lncRNA of PDLSCs), which is an osteogenesis impairment-related lncRNA of PDLSCs, and the gradual reduction in the expression of this lncRNA in PDLSCs was recorded among periodontitis patients, as demonstrated by lncRNA microarray analysis. Overexpression or knockdown of lncRNA-POIR was performed, and lncRNA-POIR was shown to positively regulate osteogenic differentiation of PDLSCs both in vitro and in vivo. Further luciferase reporter assays and quantitative real-time PCRs demonstrated that lncRNA-POIR was likely to act as a ceRNA for miR-182, thereby leading to derepression of the target gene Forkhead box O1 (FoxO1). Activated FoxO1 could compete with transcription factor 4 for β-catenin binding and inhibit the classic Wnt signalling pathway, thereby promoting osteogenic differentiation of inflammatory PDLSCs. On the other hand, Peng et al [78] suggested that lncRNA-ANCR targeted miR-758 directly as a molecular sponge via RNA immunoprecipitation, and a dual luciferase reporter assay was also performed to demonstrate that miR-758 modulates the transcript expression of neurogenic locus notch homolog protein 2 (Notch2) by targeting the 3'-UTR of Notch2. These findings suggest that the lncRNA-ANCR/miR-758/Notch2 axis plays an essential role in regulating the osteogenic differentiation of PDLSCs. In addition, prostate cancerassociated ncRNA transcript-1 (lncPCAT1) was significantly increased in osteogenically induced PDLSCs and could positively regulate osteogenic differentiation both in vitro and in vivo according to the study of Jia et al [79] . Thereafter, these researchers inferred a predicted interaction and then confirmed the direct binding sites of miR-106a-5p on lncPCAT1. In conclusion, lncRNA-PCAT1 promoted osteogenic differentiation of PDLSCs by sponging miR-106a-5p to upregulate the miR-106a-5p-targeted gene BMP2. Interestingly, the authors also found that another target of miR-106a-5p, E2F5, could bind to the promoter of lncPCAT1 and then form a feed-forward regulatory network targeting BMP2. Chen et al [80] studied two lncRNAs, HIF1A antisense RNA 1 and HIF1A antisense RNA 2, that regulated the mRNA expression of HIF1α. The results showed that HIF1A-AS2 exerted a remarkable negative regulatory function on hypoxia-inducible factor-1α (HIF-1α) through complementary base pairing with HIF-1α mRNA in PDLCs under hypoxia.


## Epigenetic regulation of lncRNAs in PDLSCs

Epigenetics plays a central role in regulating many critical cellular processes. From the perspective of epigenetics, several lncRNAs can interact with protein complexes and modulate the levels of DNA methylation as coordinators of chromatin modification, thereby regulating the expression of related genes at the epigenetic level [81] . Studies have claimed that lncRNAs affect the osteogenic differentiation of PDLSCs by epigenetic regulation. Zhu et al [82] found that lncRNA ANCR posed a physical interaction with enhancer of zeste homologue 2, and Runx2 expression was suppressed due to the catalysis of H3K27me3 in the Runx2 gene promoter, leading to the inhibiting effect on osteoblast differentiation of hFOB1.19 cells. At present, further exploration is needed to explore the function of lncRNAs in regulating the osteogenic differentiation of PDLSCs at the epigenetic level.

The discovery of lncRNAs greatly broadened the understanding of molecular regulatory mechanisms. LncRNAs regulate the expression of related genes via three different regulatory mechanisms at the levels of transcription, posttranscription, and epigenetics and participate in many important cellular processes. Current studies on lncRNAs regulating the osteogenic differentiation of PDLSCs have mainly focused on the transcriptional and posttranscriptional levels. However, most studies have not determined the subcellular localization of lncRNAs. Generally, the localization of an lncRNA in the cytoplasm is important when considering its functions, especially as a ceRNA sponge [83] . Due to the numerous lncRNAs and their complexity, the mechanisms of lncRNAs regulating osteogenic differentiation in PDLSCs need to be explored further.


## CIRCRNAS ASSOCIATED WITH OSTEOGENIC DIFFERENTIATION OF PDLSCS

circRNAs are a novel class of endogenous lncRNAs that are characterized by a structure of covalently closed continuous loops lacking 5' or 3' polarities and are widely present in eukaryotic cells [84] . The circRNAs are more stable than linear RNAs because of their covalently circular structure, making them more resistant to RNase R digestion. The majority of circRNAs are conserved across species and often exhibit cell type-specific, tissue-specific or developmental stage-specific expression [85] . Evidence is increasing that circRNAs might act as miRNA sponges and play critical roles in signal transduction in a posttranscriptional manner. The circRNA-miRNA axis is involved in several cellular processes, such as proliferation, differentiation and apoptosis [86] . Gu et al [67] conducted high-throughput sequencing to detect the different expression profiles of circRNAs in PDLSCs after 7 d of osteogenic differentiation. These researchers found that 766 circRNAs were significantly upregulated and 690 circRNAs were downregulated in PDLSCs. Furthermore, the authors predicted the potential functions of circRNAs as ceRNAs based on miRanda analysis and further investigated them using GO and KEGG analysis. The results showed that a total of 1382 circRNAs (including circRNA PTPRG, EXOC4, PRKCA, and SETBP1) were predicted to be able to interact with 148 miRNAs and compete for miRNA binding sites with 744 mRNAs, which were predicted to be significantly associated with osteoblast differentiation and the MAPK and Wnt signalling pathways regulating pluripotency of mesenchymal stem cells. Among these circRNAs, one circRNA could bind with multiple miRNAs, and the same miRNA could also interact with multiple circRNAs. Li et al [87] revealed that circRNA antisense to the cerebellar degenerationrelated protein 1 transcript (CDR1as) could act as a miR-7 adsorption sponge and then induce the upregulation of growth differentiation factor 5 and activate the Smad1/5/8 and p38MAPK signalling pathways, thereby promoting osteogenesis in PDLSCs. Wang et al [88] found that circRNA expression patterns were responsive to mechanical force in PDLSCs. Bioinformatic analysis showed that one circRNA could modulate one or several miRNA/miRNAs and vice versa. Importantly, the authors found that circRNA3140 was widely and highly related to microRNA-21, which played a key role in mechanical force-induced osteogenic differentiation of PDLSCs. These findings revealed that mechanical force induced the differential expression of circRNAs in PDLSCs, which might regulate the orthodontic tooth movement process and alveolar bone remodelling.

Overall, as a novel class of endogenous lncRNAs, circRNAs may modulate many pathophysiological processes, serve as diagnostic or predictive biomarkers for several diseases, and represent a novel and useful therapeutic method [89] . Compared with research on miRNAs and lncRNAs, research on circRNAs is in its infancy, and the potential functions and regulatory mechanisms of circRNAs are diverse. Investigating the regulatory mechanisms and functions of circRNAs in the osteogenic differentiation of PDLSCs may provide exciting potential therapies in periodontal regeneration.


## CONCLUSION

Numerous ncRNAs are associated with the osteogenic differentiation of PDLSCs ( Figure 2). ncRNAs offer an additional and promising possibility of osteogenesisrelated gene regulation that has not been fully elucidated to date. With increasing numbers of miRNAs, lncRNAs and circRNAs discovered in this process, it has become possible to use these ncRNA-related therapeutic methods in the field of periodontium repair and regeneration.

To date, ncRNA-related research on the osteogenic differentiation of PDLSCs has mainly focused on miRNAs. The demonstrated regulatory mechanism of miRNAs (miR-24-3p, miR-21, miR-203, miR-1305, miR-218, miR-214, miR-17, miR-31, miR-200c, miR-543 and miR-22) is to inhibit the mRNA levels or protein expression of targets. Other unconventional mechanisms could impact osteogenic differentiation. Currently, miRNAs are considered to be potential therapeutic targets based on their defined regulatory mechanism and clear functioning mode. miRNA-based therapeutic methods could become valuable in promoting periodontium repair and regeneration. Similarly, there are several studies on the role of lncRNAs during this process. Among these lncRNAs, lncRNA TUG1, MEG3 and ANCR regulate the osteogenic differentiation of PDLSCs in a transcriptional manner. The lncRNAs POIR, PACT1, HIF1A-AS2 and ANCR act as miRNA sponges and play critical roles during osteogenic differentiation of PDLSCs in a posttranscriptional manner ( Figure 3). However, lncRNA-ANCR has been demonstrated to suppress osteogenic differentiation of PDLSCs in an epigenetic regulatory manner. To date, few studies have investigated circRNAs during osteogenic differentiation in PDLSCs. The demonstrated mechanism of circRNAs is to act as miRNA sponges to inhibit the mRNA levels of target genes (circRNA CDR1as and circRNA3140). Other types of ncRNAs involved in the osteogenic differentiation of PDLSCs warrant further exploration.

With the development of sequencing and microarray technologies, numerous novel ncRNAs have been screened out and identified in the past few years, and their regulatory mechanisms have also been predicted and explored, benefiting from the advancement of related bioinformatics databases. Subsequently, standard molecular biology experiments and genetic engineering methods, such as quantitative real-time PCR, western blotting, dual luciferase reporter assays, RNAi and overexpression plasmid transfections, have been used to characterize ncRNA functions and explore their regulatory mechanisms. In addition, some new experimental methods have emerged, such as RIP, RNA pull-down, chromatin isolation by RNA purification, cross-linking immunoprecipitation, cross-linking, ligation, and sequencing of hybrids, and capture hybridization analysis of RNA targets, which provide an ideal research platform for elucidating the signalling transduction mechanisms of ncRNAs. In addition, the regulatory mechanisms of ncRNAs, especially lncRNAs and circRNAs, in cellular processes and diseases are highly complex. However, there are few studies  Red indicates miRNAs, green indicates mRNAs, blue indicates long noncoding RNAs and purple indicates circular RNAs. Collectively, the regulatory mechanism of miRNAs is to directly bind to the 3'-UTR of target genes and inhibit the mRNA levels or protein expression. Long noncoding RNAs regulate the osteogenic differentiation of periodontal ligament stem cells at the transcriptional, posttranscriptional or epigenetic level. And the demonstrated mechanism of circular RNAs is to act as miRNA sponges to inhibit the mRNA levels of target genes. ATF4: Activated transcription factor 4; CTNNB1: Catenin beta 1; TCF3: Transcriptional factor 3; Smurf1: Smad ubiquitin regulatory factor 1; Spry1: Palmitate phosphoprotein Sprouty1; ACVR2B: Activin A receptor type 2B; Smad5: SMAD family member 5; Runx2: Runt-relatedtranscriptionfactor 2; TOB2: Transducer of ERBB2; Satb2: Special AT-rich sequence-binding protein 2; IL-6: Interleukin-6; CDR1as: Antisense to the cerebellar degenerationrelated protein 1 transcript; GDF5: Growth differentiation factor 5; HDAC6: Histone deacetylase 6; HIF-1α: Hypoxiainducible factor-1α; HIF1AAS1/2: HIF1A antisense RNA 1/2; PCAT1: Prostate cancer-associated ncRNA transcript-1; E2F5: E2F transcription factor 5; POIR: Osteogenesis impairment-related long noncoding RNA of periodontal ligament stem cells; FoxO1: Forkhead box O1; ANCR: Anti-differentiation noncoding RNA; Notch2: Neurogenic locus notch homolog protein 2; TUG1: Taurine upregulated gene 1; Lin28A: Lin-28 homolog A; MEG3: Maternally expressed gene 3; hnRNP I: Heterogeneous nuclear ribonucleoprotein I; EZH2: Enhancer of zeste homolog 2; BMP2: Bone morphogenetic protein-2; ALP: Alkaline phosphatase, OCN: Osteocalcin; OSX: Osterix; BSP: Bone sialoprotein; miR: MicroRNA; ncRNAs: Non-coding RNAs.

concerning the nonconventional mechanisms of ncRNAs during the osteogenic differentiation of PDLSCs.

Due to their osteogenic differentiation capability, PDLSCs show effective potential in the clinical application of periodontium repair and regeneration ( Figure 4). However, reports have appeared that are less rigorous in the isolation and identification of PDLSCs. In addition, most current studies of ncRNAs involved in osteogenic differentiation in PDLSCs have focused on the cell level in vitro; therefore, in vivo experiments in this field warrant further in-depth exploration. Therefore, whereas interest and investigation in the contribution of ncRNAs to the osteogenesis of PDLSCs have increased considerably, the field is still a long way from understanding the full extent of the contribution of ncRNAs and the mechanisms by which ncRNAs exert their potential effects in this field.    

## Figure 1
1Figure 1

## Figure 1
1Isolation and characterization of periodontal ligament stem cells. A: Diagram of the isolation of periodontal ligament stem cells (PDLPs) and PDLSCs from human PDL tissue; B: Flow cytometric analysis to assess the immunophenotype of PDLSCs. Markers of mesenchymal stem cells (CD29, CD90) and nonmesenchymal stem cells (CD34); C: Assessment of the differentiation potential of PDLSCs in vitro. PDL: Periodontal ligament; PDLPs: Periodontal ligament progenitor; PDLSCs: Periodontal ligament stem cells.

## Figure 2
2Figure 2

## Figure 2
2Regulatory mechanisms of noncoding RNAs associated with the osteogenic differentiation of periodontal ligament stem cells.

## Figure 3
3Figure 3

## Figure 3
3Overview of the role of long noncoding RNAs and circular RNAs during osteogenic differentiation of periodontal ligament stem cells. Red frame indicates long noncoding RNAs (lncRNAs) or circular RNAs (circRNAs), yellow frame indicates miRNAs interacted by lncRNAs and circRNAs, blue frame indicates target mRNAs or osteogenesis-related biomarkers, green frame indicates signaling pathways associated with osteogenic differentiation. These lncRNAs and circRNAs affect downstream factors to trigger related biomarkers or signaling pathways and then regulate the osteogenic differentiation of periodontal ligament stem cells.PCAT1: Prostate cancer-associated ncRNA transcript-1; E2F5: E2F transcription factor 5; MEG3: Maternally expressed gene 3; hnRNP I: Heterogeneous nuclear ribonucleoprotein I; BMP2: Bone morphogenetic protein-2; TUG1: Taurine upregulated gene 1; Lin28A: Lin-28 homolog A; Runx2: Runt-relatedtranscriptionfactor 2; ANCR: Anti-differentiation noncoding RNA; EZH2: Enhancer of zeste homolog 2; Notch2: Neurogenic locus notch homolog protein 2; FoxO1: Forkhead box O1; POIR: Osteogenesis impairment-related lncRNA of periodontal ligament stem cells; TCF4: Transcription factor 4; CDR1as: Antisense to the cerebellar degeneration-related protein 1 transcript; GDF5: Growth differentiation factor 5; Smad1/5/8: SMAD family member 1/5/8; MAPK: Mitogen-activated protein kinase; HIF-1α: Hypoxiainducible factor-1α; HIF1AAS1/2: HIF1A antisense RNA 1/2; ACVR2B: Activin A receptor type 2B; ALP: Alkaline phosphatase, OCN: Osteocalcin; OSX: Osterix; BSP: Bone Sialoprotein; DSP: Desmoplakin; miR: MicroRNA.

## Figure 4
4Figure 4

## Figure 4
4Schematic diagram of noncoding RNAs genetic modification-based periodontal ligament stem cells transplantation therapy applications for periodontium regeneration of periodontal disease. Genetic modification of noncoding RNAs in periodontal ligament stem cells can regulate the capability of osteogenic differentiation. periodontal ligament stem cells sheet with powerful osteogenic differentiation capability be injected or transplanted into the location of bone defects to regenerate the periodontium in periodontal diseases. miRNAs: MicroRNAs; lncRNAs: Long non-coding RNAs; circRNAs: Circular RNAs; PDLSCs: Periodontal ligament stem cells.

## Table 1
1Expression profile of ncRNAs involved in the osteogenic differentiation of periodontal ligament stem cellsncRNAs 
Method 
Subjects 
Results 
Ref. 

mi-RNA 
Microarray 
3 hPDLSCs from normal 
healthy premolars without 
periodontitis or caries 
cultured in mineralized 
medium and 3 in non-
mineralized medium 

30 upregulated and 86 
downregulated miRNAs (P < 
0.05, FC ≥ 2) 

Hao et al [37] 2017 

mi-RNA 
Microarray 
6 samples from untreated 
hPDLSCs and 6 hPDLSCs 
under osteogenic induction 

miR-24-3p with the minimum 
fold change was significantly 
downregulated (P < 0.05, FC 
≥ 2) 

Li et al [38] 2019 

mi-RNA 
Microarray 
3 hPDLSCs cultured with 5.5 
mmol/L glucose or 3 
cultured with 25 mmol/L 
glucose mineralized medium 

Analyzed 700 miRNAs and 
found miR-31 was the most 
upregulated in hPDLSCs 
cultured with 25 mmol/L 
glucose (P < 0.01, FC ≥ 2) 

Zhen et al [48] 2017 

mi-RNA 
Microarray 
3 hPDLSCs from 3 volunteers 
not subjected to stretch and 3 
hPDLSCs from 3 volunteers 
subjected to mechanical 
stretch 

26 miRNAs were up-
regulated while 27 miRNAs 
were down-regulated with 
stretching (P < 0.01, FC ≥ 2) 

Wei et al [55] 2014 

lncRNA 
RNA sequencing 
3 PDLSCs every group 
cultured in an osteogenic 
medium for 0, 3, 7, or 14 d 

lncRNAs showed stage-
specific expression, and 17 
lncRNAs were up-regulated 
while31 were down-
regulated in PDLSCs in an 
osteogenic medium for 3, 7, 
or 14 d (P < 0.05, FC ≥ 2) 

Zheng et al [68] 2018 

lncRNA 
Microarray 
3 hPDLSCs both in 
osteoblast-induced group and 
non-induced group 

994 lncRNAs were up-
regulated and 1177 lncRNAs 
were down-regulated during 
osteogenic differentiation in 
PDLSCs at 14 d (P < 0.05, FC 
≥ 2) 

Qu et al [69] 2016 

lncRNA 
RNA sequencing 
3 hPDLSCs subjected to static 
compressive stress (2 g/cm 2 ) 
for 12 h and 3 normal 
hPDLSCs 

72 lncRNAs were 
upregulated and 18 
downregulated by 
compressive stress (P < 0.05, 
FC ≥ 1.5) 

Huang et al [70] 2019 

lncRNA 
Microarray 
3 noninduced and 3 
osteogenically induced 
hPDLSCs 

12 upregulated and 8 
downregulated lncRNAs and 
MEG3 belonging to 
significant downregulation 
genes in induced cells (P < 
0.05, FC ≥ 2) 

Liu et al [73] 2019 

lncRNA 
Microarray 
3 hPDLSCs from 3 normal 
persons and 3 pPDLSCs from 
3 periodontitis patients with 
osteogenic differentiation 

89 lncRNAs were 
differentially expressed 
between the two groups of 
cells and lncRNA-POIR was 
the most significantly altered 
between the non-induced 
group and osteogenic-
induced group (P < 0.05, FC > 
2) 

Wang et al [77] 2016 

lncRNA and circRNA 
RNA sequencing 
3 samples from untreated 
hPDLSCs and 3 hPDLSCs 
under osteogenic induction 

A total of 960 lncRNAs and 
1456 circRNAs were found to 
be differentially expressed (P 
< 0.05, FC ≥ 2) 

Gu et al [67] 2017 

circRNA 
RNA sequencing 
3 hPDLSCs were subjected to 
mechanical force and 3 
hPDLSCs were not subjected 
to force 

identified 2970 and 2788 
circRNAs, respectively, in the 
control group and the force 
group, and 1191 circRNAs 
were significantly 
upregulated and 1,487 were 
downregulated in the force 
group (P < 0.05, FC > 2) 

Wang et al [89] 2019 

miRNAs: MicroRNAs; lncRNAs: Long non-coding RNAs; circRNAs: Circular RNAs; hPDLSCs: Human periodontal ligament stem cells; PDLSCs: 
Periodontal ligament stem cells; MEG3: Maternally expressed gene 3; POIR: Osteogenesis impairment-related lncRNA of PDLSCs. 


## Table 2
2Overview of ncRNAs involved in the osteogenic differentiation of periodontal ligament stem cells 

ncRNAs 
Regulatory levels 
Modes of action 
Associated signaling 
pathways or biomarkers 
Ref. 

miR-24-3p 
Posttranscriptional regulation Interacting with Smad5 3′-
UTR 

Inhibits Smad5 and Runx2, 
BMP2, OCN biomarkers 

Li et al [38] 2019 

miR-21 
Posttranscriptional regulation Interacting with Smad5 3′-
UTR 

Inhibits Smad5 and Runx2, 
ALP, BSP, OSX biomarkers 

Wei et al [39] 2017 

Interacting with Spry1 3′-UTR Inhibits Spry1 and Runx2, 
OSX biomarkers 

Yang et al [40] 2017 

Interacting with ACVR2B 3′-
UTR 

Inhibits ACVR2B and 
enhances Runx2, OCN, ALP 
biomarkers 

Wei et al [56] 2015 

miR-203 
Posttranscriptional regulation Interacting with Runx2 3′-
UTR 

Inhibits Runx2 and ALP, 
OCN, OPN biomarkers 

Feng et al [41] 2019 

miR-1305 
Posttranscriptional regulation Interacting with Runx2 3′-
UTR 

Inhibits Runx2 and ALP, 
OCN, OPN biomarkers 

Chen et al [42] 2017 

miR-218 
Posttranscriptional regulation Interacting with Runx2 3′-
UTR 

Inhibits Runx2 and OCT4, 
NANOG cytokines 

Gay et al [43] 2014 

miR-214 
Posttranscriptional regulation Interacting with ATF4 3′-UTR Inhibits ATF4 and Runx2, 
ALP, OCN biomarkers 

Yao et al [44] 2017 

Interacting with CTNNB1 3′-
UTR 

Activates Wnt/β-catenin 
signaling pathway and 
inhibits ALP, OCN, BSP 
biomarkers 

Cao et al [45] 2017 

miR-17 
Posttranscriptional regulation Interacting with TCF3 3′-UTR Inhibits Runx2, ALP 
biomarkers 

Liu et al [46] 2013 

Interacting with Smurf1 3′-
UTR 

Activates Smad family 
proteins and Runx2, ALP, 
OCN biomarkers 

Liu et al [47] 2011 

miR-31 
Posttranscriptional regulation Interacting with Stab2 3′-UTR Inhibits Stab2 and Runx2, 
OSX, OCN biomarkers 

Zhen et al [48] 2017 

miR-200c 
Posttranscriptional regulation Interacting with IL-6, IL-1β 
and CCL-5 3′-UTRs 

Inhibits IL-6, IL-1β, CCL-5 
and enhances Runx2, ALP, 
OCN, OPG biomarkers 

Hong et al [49] 2016 

miR-543 
Posttranscriptional regulation Interacting with TOB2 3′-UTR Inhibits TOB2 and enhances 
Runx2, ALP, BSP, COL1A1 
biomarkers 

Ge et al [50] 2018 

miR-22 
Posttranscriptional regulation Interacting with HDAC6 3′-
UTR 

Inhibits HDAC6 and 
enhances Runx2, OCN 
biomarkers 

Yan et al [51] 2017 

lncRNA TUG1 
Transcriptional regulation 
Binding with Lin28A protein Promotes Lin28A and Runx2, 
ALP, OCN biomarkers 

He et al [72] 2018 

lncRNA MEG3 
Transcriptional regulation 
Suppresses BMP2 through 
binding with hnRNPI protein 

Inhibits BMP2 and Runx2, 
ALP, OCN biomarkers 

Liu et al [73] 2019 

lncRNA ANCR 
Transcriptional regulation 
Activates Wnt/β-catenin 
signaling pathway indirectly 

Activates Wnt/β-catenin 
signaling pathway and 
inhibits ALP, BSP, DSPP, 
OCN, Runx2, Gsk3-β 
biomarkers 

Jia et al [74] 2015 

Posttranscriptional regulation lncRNA ANCR/miR-
758/Notch2 axis "ceRNA" 

Activates Notch2-Wnt/β-
catenin signaling pathway 
and inhibits ALP, Runx2, 
OSX biomarkers 

Peng et al [78] 2018 

Epigenetic regulation 
Interacting with EZH2 and 
catalysis H3K27me3 of Runx2 

Inhibits Runx2 and ALP, 
OCN biomarkers 

Zhu et al [82] 2013 

lncRNA POIR 
Posttranscriptional regulation LncRNA POIR/miR 
182/FoxO1 axis "ceRNA" 

Inhibits TCF-4-Wnt/β-
catenin signaling pathway 
and promotes ALP, Runx2, 
COL1 biomarkers 

Wang et al [77] 2016 

lncRNA PCAT1 
Posttranscriptional regulation A feed-forward regulatory 
loop of lncPCAT1/miR-106a-
5p/E2F5/ BMP2 axis 
"ceRNA" 

Activates BMP2 and ALP, 
Runx2, OSX biomarkers 

Jia et al [79] 2019 

lncRNA HIF1AAS2 
Posttranscriptional regulation Base complementary pairing 
with the mRNA of HIF-1α 

WJSC https://www.wjgnet.com April 26, 2020 Volume 12 Issue 4 Qiu W et al. ncRNAs in osteogenesis of PDLSCs
© 2020 Baishideng Publishing Group Inc. All rights reserved.

The intricate anatomy of the periodontal ligament and its development: Lessons for periodontal regeneration. T De Jong, A D Bakker, V Everts, T H Smit, 10.1111/jre.12477J Periodontal Res. 52de Jong T, Bakker AD, Everts V, Smit TH. The intricate anatomy of the periodontal ligament and its development: Lessons for periodontal regeneration. J Periodontal Res 2017; 52: 965-974 [PMID: 28635007 DOI: 10.1111/jre.12477]

Impact of the global burden of periodontal diseases on health, nutrition and wellbeing of mankind: A call for global action. M S Tonetti, S Jepsen, Jin L Otomo-Corgel, J , 10.1111/jcpe.12732J Clin Periodontol. 44Tonetti MS, Jepsen S, Jin L, Otomo-Corgel J. Impact of the global burden of periodontal diseases on health, nutrition and wellbeing of mankind: A call for global action. J Clin Periodontol 2017; 44: 456-462 [PMID: 28419559 DOI: 10.1111/jcpe.12732]

Periodontal diseases. B L Pihlstrom, B S Michalowicz, N W Johnson, 10.1016/S0140-6736(05)67728-8Lancet. 366Pihlstrom BL, Michalowicz BS, Johnson NW. Periodontal diseases. Lancet 2005; 366: 1809-1820 [PMID: 16298220 DOI: 10.1016/S0140-6736(05)67728-8]

Surgical treatment of the residual periodontal pocket. F Graziani, D Karapetsa, N Mardas, N Leow, N Donos, 10.1111/prd.12156Periodontol. 76Graziani F, Karapetsa D, Mardas N, Leow N, Donos N. Surgical treatment of the residual periodontal pocket. Periodontol 2000 2018; 76: 150-163 [PMID: 29193404 DOI: 10.1111/prd.12156]

Nonsurgical and surgical treatment of periodontitis: how many options for one disease?. F Graziani, D Karapetsa, B Alonso, D Herrera, 10.1111/prd.12201Periodontol. 75Graziani F, Karapetsa D, Alonso B, Herrera D. Nonsurgical and surgical treatment of periodontitis: how many options for one disease? Periodontol 2000 2017; 75: 152-188 [PMID: 28758300 DOI: 10.1111/prd.12201]

Tissue engineered constructs for periodontal regeneration: Current status and future perspectives. C Vaquette, S P Pilipchuk, P M Bartold, D W Hutmacher, W V Giannobile, S Ivanovski, 10.1002/adhm.201800457Adv Healthc Mater. 71800457Vaquette C, Pilipchuk SP, Bartold PM, Hutmacher DW, Giannobile WV, Ivanovski S. Tissue engineered constructs for periodontal regeneration: Current status and future perspectives. Adv Healthc Mater 2018; 7: e1800457 [PMID: 30146758 DOI: 10.1002/adhm.201800457]

Efficacy of stem cells on periodontal regeneration: Systematic review of pre-clinical studies. S A Tassi, N Z Sergio, Myo Misawa, C C Villar, 10.1111/jre.12455J Periodontal Res. 52Tassi SA, Sergio NZ, Misawa MYO, Villar CC. Efficacy of stem cells on periodontal regeneration: Systematic review of pre-clinical studies. J Periodontal Res 2017; 52: 793-812 [PMID: 28394043 DOI: 10.1111/jre.12455]

Conditioned Medium from Periodontal Ligament Stem Cells Enhances Periodontal Regeneration. M Nagata, K Iwasaki, K Akazawa, M Komaki, N Yokoyama, Y Izumi, I Morita, 10.1089/ten.TEA.2016.0274Tissue Eng Part A. 23Nagata M, Iwasaki K, Akazawa K, Komaki M, Yokoyama N, Izumi Y, Morita I. Conditioned Medium from Periodontal Ligament Stem Cells Enhances Periodontal Regeneration. Tissue Eng Part A 2017; 23: 367-377 [PMID: 28027709 DOI: 10.1089/ten.TEA.2016.0274]

Stem cells, tissue engineering and periodontal regeneration. J Han, D Menicanin, S Gronthos, P M Bartold, 10.1111/adj.12100Aust Dent J. 591SupplHan J, Menicanin D, Gronthos S, Bartold PM. Stem cells, tissue engineering and periodontal regeneration. Aust Dent J 2014; 59 Suppl 1: 117-130 [PMID: 24111843 DOI: 10.1111/adj.12100] 10

Heterogeneity of normal human diploid fibroblasts: isolation and characterization of one phenotype. S Bordin, R C Page, A S Narayanan, 10.1126/science.6691142Science. 223Bordin S, Page RC, Narayanan AS. Heterogeneity of normal human diploid fibroblasts: isolation and characterization of one phenotype. Science 1984; 223: 171-173 [PMID: 6691142 DOI: 10.1126/science.6691142]

Investigation of multipotent postnatal stem cells from human periodontal ligament. B M Seo, M Miura, S Gronthos, P M Bartold, S Batouli, J Brahim, M Young, P G Robey, C Y Wang, S Shi, 10.1016/S0140-6736(04)16627-0Lancet. 364Seo BM, Miura M, Gronthos S, Bartold PM, Batouli S, Brahim J, Young M, Robey PG, Wang CY, Shi S. Investigation of multipotent postnatal stem cells from human periodontal ligament. Lancet 2004; 364: 149- 155 [PMID: 15246727 DOI: 10.1016/S0140-6736(04)16627-0]

Stem cells in the periodontal ligament differentiated into osteogenic, fibrogenic and cementogenic lineages for the regeneration of the periodontal complex. J Liu, Z Zhao, J Ruan, M D Weir, T Ma, K Ren, A Schneider, T W Oates, A Li, L Zhao, Hhk Xu, 10.1016/j.jdent.2019.103259J Dent. 92103259Liu J, Zhao Z, Ruan J, Weir MD, Ma T, Ren K, Schneider A, Oates TW, Li A, Zhao L, Xu HHK. Stem cells in the periodontal ligament differentiated into osteogenic, fibrogenic and cementogenic lineages for the regeneration of the periodontal complex. J Dent 2020; 92: 103259 [PMID: 31809792 DOI: 10.1016/j.jdent.2019.103259]

Macrophage polarization in periodontal ligament stem cells enhanced periodontal regeneration. J Liu, B Chen, J Bao, Y Zhang, L Lei, F Yan, 10.1186/s13287-019-1409-4Stem Cell Res Ther. 10320Liu J, Chen B, Bao J, Zhang Y, Lei L, Yan F. Macrophage polarization in periodontal ligament stem cells enhanced periodontal regeneration. Stem Cell Res Ther 2019; 10: 320 [PMID: 31730019 DOI: 10.1186/s13287-019-1409-4]

Long Noncoding RNAs: From Clinical Genetics to Therapeutic Targets?. R A Boon, N Jaé, L Holdt, S Dimmeler, 10.1016/j.jacc.2015.12.051J Am Coll Cardiol. 67Boon RA, Jaé N, Holdt L, Dimmeler S. Long Noncoding RNAs: From Clinical Genetics to Therapeutic Targets? J Am Coll Cardiol 2016; 67: 1214-1226 [PMID: 26965544 DOI: 10.1016/j.jacc.2015.12.051] 15

Noncoding RNAs in Heart Failure. S Dangwal, K Schimmel, A Foinquinos, K Xiao, T Thum, 10.1007/164_2016_99Handb Exp Pharmacol. 243Dangwal S, Schimmel K, Foinquinos A, Xiao K, Thum T. Noncoding RNAs in Heart Failure. Handb Exp Pharmacol 2017; 243: 423-445 [PMID: 27995387 DOI: 10.1007/164_2016_99] 16

The emergence of noncoding RNAs as Heracles in autophagy. J Zhang, P Wang, L Wan, S Xu, D Pang, 10.1080/15548627.2017.1312041Autophagy. 13Zhang J, Wang P, Wan L, Xu S, Pang D. The emergence of noncoding RNAs as Heracles in autophagy. Autophagy 2017; 13: 1004-1024 [PMID: 28441084 DOI: 10.1080/15548627.2017.1312041] 17

miR-1792 family clusters control iNKT cell ontogenesis via modulation of TGF-β signaling. M Fedeli, M Riba, Garcia Manteiga, J M Tian, L Viganò, V Rossetti, G Pagani, M Xiao, C Liston, A Stupka, E Cittaro, D Abrignani, S Provero, P Dellabona, P Casorati, G , 10.1073/pnas.1612024114Proc Natl Acad Sci. 113Fedeli M, Riba M, Garcia Manteiga JM, Tian L, Viganò V, Rossetti G, Pagani M, Xiao C, Liston A, Stupka E, Cittaro D, Abrignani S, Provero P, Dellabona P, Casorati G. miR-1792 family clusters control iNKT cell ontogenesis via modulation of TGF-β signaling. Proc Natl Acad Sci USA 2016; 113: E8286- E8295 [PMID: 27930306 DOI: 10.1073/pnas.1612024114] 18

miR200-regulated CXCL12β promotes fibroblast heterogeneity and immunosuppression in ovarian cancers. A M Givel, Y Kieffer, A Scholer-Dahirel, P Sirven, M Cardon, F Pelon, I Magagna, G Gentric, A Costa, C Bonneau, V Mieulet, A Vincent-Salomon, F Mechta-Grigoriou, 10.1038/s41467-018-03348-zNat Commun. 91056Givel AM, Kieffer Y, Scholer-Dahirel A, Sirven P, Cardon M, Pelon F, Magagna I, Gentric G, Costa A, Bonneau C, Mieulet V, Vincent-Salomon A, Mechta-Grigoriou F. miR200-regulated CXCL12β promotes fibroblast heterogeneity and immunosuppression in ovarian cancers. Nat Commun 2018; 9: 1056 [PMID: 29535360 DOI: 10.1038/s41467-018-03348-z]

Regulation of the U3-, U8-, and U13snoRNA Expression by the DEAD Box Proteins Ddx5/Ddx17 with Consequences for Cell Proliferation and Survival. H Ismael, S Altmeyer, H Stahl, 10.3390/ncrna2040011Noncoding RNA. 2Ismael H, Altmeyer S, Stahl H. Regulation of the U3-, U8-, and U13snoRNA Expression by the DEAD Box Proteins Ddx5/Ddx17 with Consequences for Cell Proliferation and Survival. Noncoding RNA 2016; 2 [PMID: 29657268 DOI: 10.3390/ncrna2040011] 20

Non-coding RNAs: Classification, Biology and Functioning. S Hombach, M Kretz, 10.1007/978-3-319-42059-2_1Adv Exp Med Biol. 937Hombach S, Kretz M. Non-coding RNAs: Classification, Biology and Functioning. Adv Exp Med Biol 2016; 937: 3-17 [PMID: 27573892 DOI: 10.1007/978-3-319-42059-2_1]

Long Noncoding RNAs, and Their Functions in Human Disease. M Xue, Y Zhuo, Shan B Micrornas, 10.1007/978-1-4939-7046-9_1Methods Mol Biol. 1617Xue M, Zhuo Y, Shan B. MicroRNAs, Long Noncoding RNAs, and Their Functions in Human Disease. Methods Mol Biol 2017; 1617: 1-25 [PMID: 28540673 DOI: 10.1007/978-1-4939-7046-9_1]

Translation of noncoding RNAs: Focus on lncRNAs, pri-miRNAs, and circRNAs. L J Li, R X Leng, Y G Fan, H F Pan, D Q Ye, 10.1016/j.yexcr.2017.10.010Exp Cell Res. 361Li LJ, Leng RX, Fan YG, Pan HF, Ye DQ. Translation of noncoding RNAs: Focus on lncRNAs, pri- miRNAs, and circRNAs. Exp Cell Res 2017; 361: 1-8 [PMID: 29031633 DOI: 10.1016/j.yexcr.2017.10.010]

CircRNAs in hematopoiesis and hematological malignancies. A Bonizzato, E Gaffo, Te Kronnie, G Bortoluzzi, S , 10.1038/bcj.2016.81Blood Cancer J. 6483Bonizzato A, Gaffo E, Te Kronnie G, Bortoluzzi S. CircRNAs in hematopoiesis and hematological malignancies. Blood Cancer J 2016; 6: e483 [PMID: 27740630 DOI: 10.1038/bcj.2016.81] 24

CircRNAs in the brain. M Hanan, H Soreq, S Kadener, 10.1080/15476286.2016.1255398RNA Biol. 14Hanan M, Soreq H, Kadener S. CircRNAs in the brain. RNA Biol 2017; 14: 1028-1034 [PMID: 27892769 DOI: 10.1080/15476286.2016.1255398] 25

circRNAs in Cancer. I L Patop, S Kadener, 10.1016/j.gde.2017.11.007Curr Opin Genet Dev. 48Patop IL, Kadener S. circRNAs in Cancer. Curr Opin Genet Dev 2018; 48: 121-127 [PMID: 29245064 DOI: 10.1016/j.gde.2017.11.007] 26

Cellular origins and differentiation control mechanisms during periodontal development and wound healing. S Pitaru, C A Mcculloch, S A Narayanan, 10.1111/j.1600-0765.1994.tb01095.xJ Periodontal Res. 29Pitaru S, McCulloch CA, Narayanan SA. Cellular origins and differentiation control mechanisms during periodontal development and wound healing. J Periodontal Res 1994; 29: 81-94 [PMID: 8158503 DOI: 10.1111/j.1600-0765.1994.tb01095.x] 27

Standardization of Criteria Defining Periodontal Ligament Stem Cells. P M Bartold, S Gronthos, 10.1177/0022034517697653J Dent Res. 96Bartold PM, Gronthos S. Standardization of Criteria Defining Periodontal Ligament Stem Cells. J Dent Res 2017; 96: 487-490 [PMID: 28425840 DOI: 10.1177/0022034517697653] 28

Characterization of progenitor cells and stem cells from the periodontal ligament tissue derived from a single person. E Prateeptongkum, C Klingelhöffer, S Müller, T Ettl, C Morsczeck, 10.1111/jre.12306J Periodontal Res. 51Prateeptongkum E, Klingelhöffer C, Müller S, Ettl T, Morsczeck C. Characterization of progenitor cells and stem cells from the periodontal ligament tissue derived from a single person. J Periodontal Res 2016; 51: 265-272 [PMID: 26108342 DOI: 10.1111/jre.12306] 29

A Method to Isolate, Purify, and Characterize Human Periodontal Ligament Stem Cells. K Mrozik, S Gronthos, S Shi, P M Bartold, 10.1007/978-1-4939-6685-1_24Methods Mol Biol. 1537Mrozik K, Gronthos S, Shi S, Bartold PM. A Method to Isolate, Purify, and Characterize Human Periodontal Ligament Stem Cells. Methods Mol Biol 2017; 1537: 413-427 [PMID: 27924608 DOI: 10.1007/978-1-4939-6685-1_24]

The periodontal stem/progenitor cell inflammatory-regenerative cross talk: A new perspective. Fawzy El-Sayed, K M Elahmady, M Adawi, Z Aboushadi, N Elnaggar, A Eid, M Hamdy, N Sanaa, D Dörfer, C E , 10.1111/jre.12616J Periodontal Res. 54Fawzy El-Sayed KM, Elahmady M, Adawi Z, Aboushadi N, Elnaggar A, Eid M, Hamdy N, Sanaa D, Dörfer CE. The periodontal stem/progenitor cell inflammatory-regenerative cross talk: A new perspective. J Periodontal Res 2019; 54: 81-94 [PMID: 30295324 DOI: 10.1111/jre.12616] 31

Mesenchymal stem cells derived from dental tissues vs. those from other sources: their biology and role in regenerative medicine. G T Huang, S Gronthos, S Shi, 10.1177/0022034509340867J Dent Res. 88Huang GT, Gronthos S, Shi S. Mesenchymal stem cells derived from dental tissues vs. those from other sources: their biology and role in regenerative medicine. J Dent Res 2009; 88: 792-806 [PMID: 19767575 DOI: 10.1177/0022034509340867]

Nutrition, microRNAs, and Human Health. J Cui, B Zhou, S A Ross, J Zempleni, 10.3945/an.116.013839Adv Nutr. 8Cui J, Zhou B, Ross SA, Zempleni J. Nutrition, microRNAs, and Human Health. Adv Nutr 2017; 8: 105- 112 [PMID: 28096131 DOI: 10.3945/an.116.013839] 33

Functional relevance of "seed" and "non-seed" sequences in microRNAmediated promotion of C. elegans developmental progression. H Zhang, K L Artiles, A Z Fire, 10.1261/rna.053793.115RNA. 21Zhang H, Artiles KL, Fire AZ. Functional relevance of "seed" and "non-seed" sequences in microRNA- mediated promotion of C. elegans developmental progression. RNA 2015; 21: 1980-1992 [PMID: 26385508 DOI: 10.1261/rna.053793.115]

. D Pineda-Gómez, E Garrido, P Chávez, M Salcedo, Detection of microRNAs seed sequences within human papillomavirus genomesPineda-Gómez D, Garrido E, Chávez P, Salcedo M. [Detection of microRNAs seed sequences within human papillomavirus genomes].

. 26462509Rev Med Inst Mex Seguro Soc. 532SupplRev Med Inst Mex Seguro Soc 2015; 53 Suppl 2: S140-S153 [PMID: 26462509] 35

The Non-Canonical Aspects of MicroRNAs: Many Roads to Gene Regulation. C J Stavast, S J Erkeland, 10.3390/cells8111465Cells. 8Stavast CJ, Erkeland SJ. The Non-Canonical Aspects of MicroRNAs: Many Roads to Gene Regulation. Cells 2019; 8 [PMID: 31752361 DOI: 10.3390/cells8111465] 36

Noncoding RNAs: new insights into the odontogenic differentiation of dental tissue-derived mesenchymal stem cells. F Fang, K Zhang, Z Chen, B Wu, 10.1186/s13287-019-1411-xStem Cell Res Ther. 10297Fang F, Zhang K, Chen Z, Wu B. Noncoding RNAs: new insights into the odontogenic differentiation of dental tissue-derived mesenchymal stem cells. Stem Cell Res Ther 2019; 10: 297 [PMID: 31547871 DOI: 10.1186/s13287-019-1411-x]

Identification of MicroRNAs by Microarray Analysis and Prediction of Target Genes Involved in Osteogenic Differentiation of Human Periodontal Ligament Stem Cells. Y Hao, Y Ge, J Li, Y Hu, B Wu, F Fang, 10.1902/jop.2017.170079J Periodontol. 88Hao Y, Ge Y, Li J, Hu Y, Wu B, Fang F. Identification of MicroRNAs by Microarray Analysis and Prediction of Target Genes Involved in Osteogenic Differentiation of Human Periodontal Ligament Stem Cells. J Periodontol 2017; 88: 1105-1113 [PMID: 28598283 DOI: 10.1902/jop.2017.170079] 38

Downregulation of miR-24-3p promotes osteogenic differentiation of human periodontal ligament stem cells by targeting SMAD family member 5. Z Li, Y Sun, S Cao, J Zhang, J Wei, 10.1002/jcp.27499J Cell Physiol. 234Li Z, Sun Y, Cao S, Zhang J, Wei J. Downregulation of miR-24-3p promotes osteogenic differentiation of human periodontal ligament stem cells by targeting SMAD family member 5. J Cell Physiol 2019; 234: 7411-7419 [PMID: 30378100 DOI: 10.1002/jcp.27499] 39

MicroRNA-21 regulates Osteogenic Differentiation of Periodontal Ligament Stem Cells by targeting Smad5. F Wei, S Yang, Q Guo, X Zhang, D Ren, T Lv, X Xu, 10.1038/s41598-017-16720-8Sci Rep. 716608Wei F, Yang S, Guo Q, Zhang X, Ren D, Lv T, Xu X. MicroRNA-21 regulates Osteogenic Differentiation of Periodontal Ligament Stem Cells by targeting Smad5. Sci Rep 2017; 7: 16608 [PMID: 29192241 DOI: 10.1038/s41598-017-16720-8]

Tumor necrosis factor-α suppresses adipogenic and osteogenic differentiation of human periodontal ligament stem cell by inhibiting miR-21/Spry1 functional axis. N Yang, Y Li, G Wang, Y Ding, Y Jin, Y Xu, 10.1016/j.diff.2017.08.004Differentiation. 97Yang N, Li Y, Wang G, Ding Y, Jin Y, Xu Y. Tumor necrosis factor-α suppresses adipogenic and osteogenic differentiation of human periodontal ligament stem cell by inhibiting miR-21/Spry1 functional axis. Differentiation 2017; 97: 33-43 [PMID: 28946056 DOI: 10.1016/j.diff.2017.08.004] 41

Study on effect of miR-203 for targeting RUNX2 on osteogenic differentiation of human periodontal ligament stem cells. B J Feng, X B Zhao, Z H Hao, Q M Yuan, X G Wang, 10.19748/j.cn.kqxf.1009-3761.2019.01.010Kouqiang Hemian Xiufuxue Zazhi. 20Feng BJ, Zhao XB, Hao ZH, Yuan QM, Wang XG. Study on effect of miR-203 for targeting RUNX2 on osteogenic differentiation of human periodontal ligament stem cells. Kouqiang Hemian Xiufuxue Zazhi 2019; 20: 44-48 [DOI: 10.19748/j.cn.kqxf.1009-3761.2019.01.010] 42

Restoration of miR-1305 relieves the inhibitory effect of nicotine on periodontal ligament-derived stem cell proliferation, migration, and osteogenic differentiation. Z Chen, H L Liu, 10.1111/jop.12492J Oral Pathol Med. 46Chen Z, Liu HL. Restoration of miR-1305 relieves the inhibitory effect of nicotine on periodontal ligament-derived stem cell proliferation, migration, and osteogenic differentiation. J Oral Pathol Med 2017; 46: 313-320 [PMID: 27604968 DOI: 10.1111/jop.12492] 43

Differentiation of human dental stem cells reveals a role for microRNA-218. I Gay, A Cavender, D Peto, Z Sun, A Speer, H Cao, B A Amendt, 10.1111/jre.12086J Periodontal Res. 49Gay I, Cavender A, Peto D, Sun Z, Speer A, Cao H, Amendt BA. Differentiation of human dental stem cells reveals a role for microRNA-218. J Periodontal Res 2014; 49: 110-120 [PMID: 23662917 DOI: 10.1111/jre.12086] 44

MicroRNA-214 Suppresses Osteogenic Differentiation of Human Periodontal Ligament Stem Cells by Targeting ATF4. S Yao, W Zhao, Q Ou, L Liang, X Lin, Y Wang, 10.1155/2017/3028647Stem Cells Int. 3028647Yao S, Zhao W, Ou Q, Liang L, Lin X, Wang Y. MicroRNA-214 Suppresses Osteogenic Differentiation of Human Periodontal Ligament Stem Cells by Targeting ATF4. Stem Cells Int 2017; 2017: 3028647 [PMID: 29213288 DOI: 10.1155/2017/3028647] 45

miR-214 promotes periodontal ligament stem cell osteoblastic differentiation by modulating Wnt/βcatenin signaling. F Cao, J Zhan, X Chen, K Zhang, R Lai, Z Feng, 10.3892/mmr.2017.7821Mol Med Rep. 16Cao F, Zhan J, Chen X, Zhang K, Lai R, Feng Z. miR-214 promotes periodontal ligament stem cell osteoblastic differentiation by modulating Wnt/βcatenin signaling. Mol Med Rep 2017; 16: 9301-9308 [PMID: 29152645 DOI: 10.3892/mmr.2017.7821] 46

TCF3, a novel positive regulator of osteogenesis, plays a crucial role in miR-17 modulating the diverse effect of canonical Wnt signaling in different microenvironments. W Liu, Y Liu, T Guo, C Hu, H Luo, L Zhang, S Shi, T Cai, Y Ding, Y Jin, 10.1038/cddis.2013.65Cell Death Dis. 4539Liu W, Liu Y, Guo T, Hu C, Luo H, Zhang L, Shi S, Cai T, Ding Y, Jin Y. TCF3, a novel positive regulator of osteogenesis, plays a crucial role in miR-17 modulating the diverse effect of canonical Wnt signaling in different microenvironments. Cell Death Dis 2013; 4: e539 [PMID: 23492770 DOI: 10.1038/cddis.2013.65]

. Y Liu, W Liu, C Hu, Z Xue, G Wang, B Ding, H Luo, L Tang, X Kong, X Chen, N Liu, Y Ding, Y Jin, Liu Y, Liu W, Hu C, Xue Z, Wang G, Ding B, Luo H, Tang L, Kong X, Chen X, Liu N, Ding Y, Jin Y.

MiR-17 modulates osteogenic differentiation through a coherent feed-forward loop in mesenchymal stem cells isolated from periodontal ligaments of patients with periodontitis. 10.1002/stem.728Stem Cells. 29MiR-17 modulates osteogenic differentiation through a coherent feed-forward loop in mesenchymal stem cells isolated from periodontal ligaments of patients with periodontitis. Stem Cells 2011; 29: 1804-1816 [PMID: 21898695 DOI: 10.1002/stem.728] 48

MiR-31 is involved in the high glucose-suppressed osteogenic differentiation of human periodontal ligament stem cells by targeting Satb2. L Zhen, X Jiang, Y Chen, D Fan, 28559988Am J Transl Res. 9Zhen L, Jiang X, Chen Y, Fan D. MiR-31 is involved in the high glucose-suppressed osteogenic differentiation of human periodontal ligament stem cells by targeting Satb2. Am J Transl Res 2017; 9: 2384-2393 [PMID: 28559988]

. L Hong, T Sharp, B Khorsand, C Fischer, S Eliason, A Salem, A Akkouch, K Brogden, B A Amendt, Hong L, Sharp T, Khorsand B, Fischer C, Eliason S, Salem A, Akkouch A, Brogden K, Amendt BA.

MicroRNA-200c Represses IL-6, IL-8, and CCL-5 Expression and Enhances Osteogenic Differentiation. 10.1371/journal.pone.0160915PLoS One. 11160915MicroRNA-200c Represses IL-6, IL-8, and CCL-5 Expression and Enhances Osteogenic Differentiation. PLoS One 2016; 11: e0160915 [PMID: 27529418 DOI: 10.1371/journal.pone.0160915] 50

MicroRNA-543 functions as an osteogenesis promoter in human periodontal ligament-derived stem cells by inhibiting transducer of ERBB2, 2. Y Ge, J Li, Y Hao, Y Hu, D Chen, B Wu, F Fang, 10.1111/jre.12572J Periodontal Res. 53Ge Y, Li J, Hao Y, Hu Y, Chen D, Wu B, Fang F. MicroRNA-543 functions as an osteogenesis promoter in human periodontal ligament-derived stem cells by inhibiting transducer of ERBB2, 2. J Periodontal Res 2018; 53: 832-841 [PMID: 29851072 DOI: 10.1111/jre.12572] 51

MicroRNA-22 Promoted Osteogenic Differentiation of Human Periodontal Ligament Stem Cells by Targeting HDAC6. G Q Yan, X Wang, F Yang, M L Yang, G R Zhang, G K Wang, Q Zhou, 10.1002/jcb.25931J Cell Biochem. 118Yan GQ, Wang X, Yang F, Yang ML, Zhang GR, Wang GK, Zhou Q. MicroRNA-22 Promoted Osteogenic Differentiation of Human Periodontal Ligament Stem Cells by Targeting HDAC6. J Cell Biochem 2017; 118: 1653-1658 [PMID: 28195408 DOI: 10.1002/jcb.25931] 52

Force-Induced H 2 S by PDLSCs Modifies Osteoclastic Activity during Tooth Movement. F Liu, F Wen, D He, D Liu, R Yang, X Wang, Y Yan, Y Liu, X Kou, Y Zhou, 10.1177/0022034517690388J Dent Res. 96Liu F, Wen F, He D, Liu D, Yang R, Wang X, Yan Y, Liu Y, Kou X, Zhou Y. Force-Induced H 2 S by PDLSCs Modifies Osteoclastic Activity during Tooth Movement. J Dent Res 2017; 96: 694-702 [PMID: 28165889 DOI: 10.1177/0022034517690388]

Mechanical stress potentiates the differentiation of periodontal ligament stem cells into keratocytes. J Chen, W Zhang, L J Backman, P Kelk, P Danielson, 10.1136/bjophthalmol-2017-311150Br J Ophthalmol. 102Chen J, Zhang W, Backman LJ, Kelk P, Danielson P. Mechanical stress potentiates the differentiation of periodontal ligament stem cells into keratocytes. Br J Ophthalmol 2018; 102: 562-569 [PMID: 29306866 DOI: 10.1136/bjophthalmol-2017-311150]

Mechanical stress regulates osteogenic differentiation and RANKL/OPG ratio in periodontal ligament stem cells by the Wnt/β-catenin pathway. L Zhang, W Liu, J Zhao, X Ma, L Shen, Y Zhang, Jin F Jin, Y , 10.1016/j.bbagen.2016.05.003Biochim Biophys Acta. 1860Zhang L, Liu W, Zhao J, Ma X, Shen L, Zhang Y, Jin F, Jin Y. Mechanical stress regulates osteogenic differentiation and RANKL/OPG ratio in periodontal ligament stem cells by the Wnt/β-catenin pathway. Biochim Biophys Acta 2016; 1860: 2211-2219 [PMID: 27154288 DOI: 10.1016/j.bbagen.2016.05.003] 55

Mechanical force-induced specific MicroRNA expression in human periodontal ligament stem cells. F L Wei, J H Wang, G Ding, S Y Yang, Y Li, Y J Hu, S L Wang, 10.1159/000369613Cells Tissues Organs. 199Wei FL, Wang JH, Ding G, Yang SY, Li Y, Hu YJ, Wang SL. Mechanical force-induced specific MicroRNA expression in human periodontal ligament stem cells. Cells Tissues Organs 2014; 199: 353- 363 [PMID: 25823370 DOI: 10.1159/000369613] 56

microRNA-21 mediates stretch-induced osteogenic differentiation in human periodontal ligament stem cells. F Wei, D Liu, C Feng, F Zhang, S Yang, Y Hu, G Ding, S Wang, 10.1089/scd.2014.0191Stem Cells Dev. 24Wei F, Liu D, Feng C, Zhang F, Yang S, Hu Y, Ding G, Wang S. microRNA-21 mediates stretch-induced osteogenic differentiation in human periodontal ligament stem cells. Stem Cells Dev 2015; 24: 312-319 [PMID: 25203845 DOI: 10.1089/scd.2014.0191] 57

SnapShot: Unconventional miRNA Functions. M P Dragomir, E Knutsen, G A Calin, 10.1016/j.cell.2018.07.040Cell. 174Dragomir MP, Knutsen E, Calin GA. SnapShot: Unconventional miRNA Functions. Cell 2018; 174: 1038-1038.e1 [PMID: 30096304 DOI: 10.1016/j.cell.2018.07.040] 58

miRNA-encoded peptides (miPEPs): A new tool to analyze the roles of miRNAs in plant biology. J M Couzigou, D Lauressergues, G Bécard, J P Combier, 10.1080/15476286.2015.1094601RNA Biol. 12Couzigou JM, Lauressergues D, Bécard G, Combier JP. miRNA-encoded peptides (miPEPs): A new tool to analyze the roles of miRNAs in plant biology. RNA Biol 2015; 12: 1178-1180 [PMID: 26400469 DOI: 10.1080/15476286.2015.1094601]

MicroRNA in Control of Gene Expression: An Overview of Nuclear Functions. C Catalanotto, C Cogoni, G Zardo, 10.3390/ijms17101712Int J Mol Sci. 17Catalanotto C, Cogoni C, Zardo G. MicroRNA in Control of Gene Expression: An Overview of Nuclear Functions. Int J Mol Sci 2016; 17 [PMID: 27754357 DOI: 10.3390/ijms17101712] 60

Technological Developments in lncRNA Biology. S Jathar, V Kumar, J Srivastava, V Tripathi, 10.1007/978-981-10-5203-3_10Adv Exp Med Biol. 1008Jathar S, Kumar V, Srivastava J, Tripathi V. Technological Developments in lncRNA Biology. Adv Exp Med Biol 2017; 1008: 283-323 [PMID: 28815544 DOI: 10.1007/978-981-10-5203-3_10]

The bright side of dark matter: lncRNAs in cancer. J R Evans, F Y Feng, A M Chinnaiyan, 10.1172/JCI84421J Clin Invest. 126Evans JR, Feng FY, Chinnaiyan AM. The bright side of dark matter: lncRNAs in cancer. J Clin Invest 2016; 126: 2775-2782 [PMID: 27479746 DOI: 10.1172/JCI84421] 62

Mechanisms of long noncoding RNA function in development and disease. S U Schmitz, P Grote, B G Herrmann, 10.1007/s00018-016-2174-5Cell Mol Life Sci. 73Schmitz SU, Grote P, Herrmann BG. Mechanisms of long noncoding RNA function in development and disease. Cell Mol Life Sci 2016; 73: 2491-2509 [PMID: 27007508 DOI: 10.1007/s00018-016-2174-5] 63

Gene regulation of mammalian long non-coding RNA. H Bunch, 10.1007/s00438-017-1370-9Mol Genet Genomics. 293Bunch H. Gene regulation of mammalian long non-coding RNA. Mol Genet Genomics 2018; 293: 1-15 [PMID: 28894972 DOI: 10.1007/s00438-017-1370-9]

The Landscape of long noncoding RNA classification. G St Laurent, C Wahlestedt, P Kapranov, 10.1016/j.tig.2015.03.007Trends Genet. 31St Laurent G, Wahlestedt C, Kapranov P. The Landscape of long noncoding RNA classification. Trends Genet 2015; 31: 239-251 [PMID: 25869999 DOI: 10.1016/j.tig.2015.03.007] 65

The multidimensional mechanisms of long noncoding RNA function. F P Marchese, I Raimondi, M Huarte, 10.1186/s13059-017-1348-2Genome Biol. 18206Marchese FP, Raimondi I, Huarte M. The multidimensional mechanisms of long noncoding RNA function. Genome Biol 2017; 18: 206 [PMID: 29084573 DOI: 10.1186/s13059-017-1348-2]

Local regulation of gene expression by lncRNA promoters, transcription and splicing. J M Engreitz, J E Haines, E M Perez, G Munson, J Chen, M Kane, P E Mcdonel, M Guttman, E S Lander, 10.1038/nature20149Nature. 539Engreitz JM, Haines JE, Perez EM, Munson G, Chen J, Kane M, McDonel PE, Guttman M, Lander ES. Local regulation of gene expression by lncRNA promoters, transcription and splicing. Nature 2016; 539: 452-455 [PMID: 27783602 DOI: 10.1038/nature20149] 67

Identification and integrated analysis of differentially expressed lncRNAs and circRNAs reveal the potential ceRNA networks during PDLSC osteogenic differentiation. X Gu, M Li, Y Jin, D Liu, F Wei, 10.1186/s12863-017-0569-4BMC Genet. 18100Gu X, Li M, Jin Y, Liu D, Wei F. Identification and integrated analysis of differentially expressed lncRNAs and circRNAs reveal the potential ceRNA networks during PDLSC osteogenic differentiation. BMC Genet 2017; 18: 100 [PMID: 29197342 DOI: 10.1186/s12863-017-0569-4]

Time series clustering of mRNA and lncRNA expression during osteogenic differentiation of periodontal ligament stem cells. Y Zheng, X Li, Y Huang, L Jia, W Li, 10.7717/peerj.5214PeerJ. 65214Zheng Y, Li X, Huang Y, Jia L, Li W. Time series clustering of mRNA and lncRNA expression during osteogenic differentiation of periodontal ligament stem cells. PeerJ 2018; 6: e5214 [PMID: 30038865 DOI: 10.7717/peerj.5214]

Potential Role of Long Non-Coding RNA in Osteogenic Differentiation of Human Periodontal Ligament Stem Cells. Q Qu, F Fang, B Wu, Y Hu, M Chen, Z Deng, D Ma, T Chen, Y Hao, Y Ge, 10.1902/jop.2016.150592J Periodontol. 87Qu Q, Fang F, Wu B, Hu Y, Chen M, Deng Z, Ma D, Chen T, Hao Y, Ge Y. Potential Role of Long Non- Coding RNA in Osteogenic Differentiation of Human Periodontal Ligament Stem Cells. J Periodontol 2016; 87: e127-e137 [PMID: 26991483 DOI: 10.1902/jop.2016.150592] 70

The long non-coding RNA landscape of periodontal ligament stem cells subjected to compressive force. Y Huang, Y Zhang, X Li, H Liu, Q Yang, L Jia, Y Zheng, W Li, 10.1093/ejo/cjy057Eur J Orthod. 41Huang Y, Zhang Y, Li X, Liu H, Yang Q, Jia L, Zheng Y, Li W. The long non-coding RNA landscape of periodontal ligament stem cells subjected to compressive force. Eur J Orthod 2019; 41: 333-342 [PMID: 30169774 DOI: 10.1093/ejo/cjy057] 71

Revealing protein-lncRNA interaction. F Ferrè, A Colantoni, M Helmer-Citterich, 10.1093/bib/bbv031Brief Bioinform. 17Ferrè F, Colantoni A, Helmer-Citterich M. Revealing protein-lncRNA interaction. Brief Bioinform 2016; 17: 106-116 [PMID: 26041786 DOI: 10.1093/bib/bbv031] 72

Long noncoding RNA TUG1 facilitates osteogenic differentiation of periodontal ligament stem cells via interacting with Lin28A. Q He, S Yang, X Gu, M Li, C Wang, F Wei, 10.1038/s41419-018-0484-2Cell Death Dis. 9455He Q, Yang S, Gu X, Li M, Wang C, Wei F. Long noncoding RNA TUG1 facilitates osteogenic differentiation of periodontal ligament stem cells via interacting with Lin28A. Cell Death Dis 2018; 9: 455 [PMID: 29674645 DOI: 10.1038/s41419-018-0484-2]

Upregulation of long noncoding RNA MEG3 inhibits the osteogenic differentiation of periodontal ligament cells. Y Liu, X Zeng, J Miao, C Liu, F Wei, D Liu, Z Zheng, K Ting, C Wang, J Guo, 10.1002/jcp.27248J Cell Physiol. 234Liu Y, Zeng X, Miao J, Liu C, Wei F, Liu D, Zheng Z, Ting K, Wang C, Guo J. Upregulation of long noncoding RNA MEG3 inhibits the osteogenic differentiation of periodontal ligament cells. J Cell Physiol 2019; 234: 4617-4626 [PMID: 30256394 DOI: 10.1002/jcp.27248] 74

Down-regulated non-coding RNA (lncRNA-ANCR) promotes osteogenic differentiation of periodontal ligament stem cells. Q Jia, W Jiang, L Ni, 10.1016/j.archoralbio.2014.10.007Arch Oral Biol. 60Jia Q, Jiang W, Ni L. Down-regulated non-coding RNA (lncRNA-ANCR) promotes osteogenic differentiation of periodontal ligament stem cells. Arch Oral Biol 2015; 60: 234-241 [PMID: 25463901 DOI: 10.1016/j.archoralbio.2014.10.007] 75

Analyzing MiRNA-LncRNA Interactions. M D Paraskevopoulou, A G Hatzigeorgiou, 10.1007/978-1-4939-3378-5_21Methods Mol Biol. 1402Paraskevopoulou MD, Hatzigeorgiou AG. Analyzing MiRNA-LncRNA Interactions. Methods Mol Biol 2016; 1402: 271-286 [PMID: 26721498 DOI: 10.1007/978-1-4939-3378-5_21]

Comprehensive characterization of lncRNA-mRNA related ceRNA network across 12 major cancers. Y Zhang, Y Xu, L Feng, F Li, Z Sun, T Wu, X Shi, J Li, X Li, 10.18632/oncotarget.11637Oncotarget. 7Zhang Y, Xu Y, Feng L, Li F, Sun Z, Wu T, Shi X, Li J, Li X. Comprehensive characterization of lncRNA-mRNA related ceRNA network across 12 major cancers. Oncotarget 2016; 7: 64148-64167 [PMID: 27580177 DOI: 10.18632/oncotarget.11637] 77

Long noncoding RNA related to periodontitis interacts with miR-182 to upregulate osteogenic differentiation in periodontal mesenchymal stem cells of periodontitis patients. L Wang, F Wu, Y Song, X Li, Q Wu, Y Duan, Jin Z , 10.1038/cddis.2016.125Cell Death Dis. 72327Wang L, Wu F, Song Y, Li X, Wu Q, Duan Y, Jin Z. Long noncoding RNA related to periodontitis interacts with miR-182 to upregulate osteogenic differentiation in periodontal mesenchymal stem cells of periodontitis patients. Cell Death Dis 2016; 7: e2327 [PMID: 27512949 DOI: 10.1038/cddis.2016.125] 78

Long noncoding RNA ANCR suppresses bone formation of periodontal ligament stem cells via sponging miRNA-758. W Peng, W Deng, J Zhang, G Pei, Q Rong, S Zhu, 10.1016/j.bbrc.2018.06.081Biochem Biophys Res Commun. 503Peng W, Deng W, Zhang J, Pei G, Rong Q, Zhu S. Long noncoding RNA ANCR suppresses bone formation of periodontal ligament stem cells via sponging miRNA-758. Biochem Biophys Res Commun 2018; 503: 815-821 [PMID: 29913147 DOI: 10.1016/j.bbrc.2018.06.081] 79

A feed-forward regulatory network lncPCAT1/miR-106a-5p/E2F5 regulates the osteogenic differentiation of periodontal ligament stem cells. B Jia, X Qiu, J Chen, X Sun, X Zheng, J Zhao, Q Li, Z Wang, 10.1002/jcp.28550J Cell Physiol. 234Jia B, Qiu X, Chen J, Sun X, Zheng X, Zhao J, Li Q, Wang Z. A feed-forward regulatory network lncPCAT1/miR-106a-5p/E2F5 regulates the osteogenic differentiation of periodontal ligament stem cells. J Cell Physiol 2019; 234: 19523-19538 [PMID: 30997692 DOI: 10.1002/jcp.28550] 80

Comparison of HIF1AAS1 and HIF1AAS2 in regulating HIF1α and the osteogenic differentiation of PDLCs under hypoxia. D Chen, L Wu, L Liu, Q Gong, J Zheng, C Peng, J Deng, 10.3892/ijmm.2017.3138Int J Mol Med. 40Chen D, Wu L, Liu L, Gong Q, Zheng J, Peng C, Deng J. Comparison of HIF1AAS1 and HIF1AAS2 in regulating HIF1α and the osteogenic differentiation of PDLCs under hypoxia. Int J Mol Med 2017; 40: 1529-1536 [PMID: 28949371 DOI: 10.3892/ijmm.2017.3138] 81

Non-coding RNAs: Epigenetic regulators of bone development and homeostasis. M Q Hassan, C E Tye, G S Stein, J B Lian, 10.1016/j.bone.2015.05.026Bone. 81Hassan MQ, Tye CE, Stein GS, Lian JB. Non-coding RNAs: Epigenetic regulators of bone development and homeostasis. Bone 2015; 81: 746-756 [PMID: 26039869 DOI: 10.1016/j.bone.2015.05.026] 82

Downregulated LncRNA-ANCR promotes osteoblast differentiation by targeting EZH2 and regulating Runx2 expression. L Zhu, P C Xu, 10.1016/j.bbrc.2013.02.036Biochem Biophys Res Commun. 432Zhu L, Xu PC. Downregulated LncRNA-ANCR promotes osteoblast differentiation by targeting EZH2 and regulating Runx2 expression. Biochem Biophys Res Commun 2013; 432: 612-617 [PMID: 23438432 DOI: 10.1016/j.bbrc.2013.02.036]

Long non-coding RNA in osteogenesis: A new world to be explored. J Zhang, X Hao, M Yin, T Xu, F Guo, 10.1302/2046-3758.82.BJR-2018-0074.R1Bone Joint Res. 8Zhang J, Hao X, Yin M, Xu T, Guo F. Long non-coding RNA in osteogenesis: A new world to be explored. Bone Joint Res 2019; 8: 73-80 [PMID: 30915213 DOI: 10.1302/2046-3758.82.BJR-2018-0074.R1]

Circular RNA: A new star of noncoding RNAs. S Qu, X Yang, X Li, J Wang, Y Gao, R Shang, W Sun, K Dou, H Li, 10.1016/j.canlet.2015.06.003Cancer Lett. 365Qu S, Yang X, Li X, Wang J, Gao Y, Shang R, Sun W, Dou K, Li H. Circular RNA: A new star of noncoding RNAs. Cancer Lett 2015; 365: 141-148 [PMID: 26052092 DOI: 10.1016/j.canlet.2015.06.003] 85

Increased complexity of circRNA expression during species evolution. R Dong, X K Ma, L L Chen, L Yang, 10.1080/15476286.2016.1269999RNA Biol. 14Dong R, Ma XK, Chen LL, Yang L. Increased complexity of circRNA expression during species evolution. RNA Biol 2017; 14: 1064-1074 [PMID: 27982734 DOI: 10.1080/15476286.2016.1269999] 86

An emerging function of circRNA-miRNAs-mRNA axis in human diseases. D Rong, H Sun, Z Li, S Liu, C Dong, K Fu, W Tang, H Cao, 10.18632/oncotarget.19154Oncotarget. 8Rong D, Sun H, Li Z, Liu S, Dong C, Fu K, Tang W, Cao H. An emerging function of circRNA-miRNAs- mRNA axis in human diseases. Oncotarget 2017; 8: 73271-73281 [PMID: 29069868 DOI: 10.18632/onco- target.19154] 87

Circular RNA CDR1as regulates osteoblastic differentiation of periodontal ligament stem cells via the miR-7/GDF5/SMAD and p38 MAPK signaling pathway. X Li, Y Zheng, Y Zheng, Y Huang, Y Zhang, L Jia, W Li, 10.1186/s13287-018-0976-0Stem Cell Res Ther. 9232Li X, Zheng Y, Zheng Y, Huang Y, Zhang Y, Jia L, Li W. Circular RNA CDR1as regulates osteoblastic differentiation of periodontal ligament stem cells via the miR-7/GDF5/SMAD and p38 MAPK signaling pathway. Stem Cell Res Ther 2018; 9: 232 [PMID: 30170617 DOI: 10.1186/s13287-018-0976-0] 88

Identification and characterization of circular RNAs involved in mechanical force-induced periodontal ligament stem cells. H Wang, C Feng, Jin Y Tan, W Wei, F , 10.1002/jcp.27686J Cell Physiol. 234Wang H, Feng C, Jin Y, Tan W, Wei F. Identification and characterization of circular RNAs involved in mechanical force-induced periodontal ligament stem cells. J Cell Physiol 2019; 234: 10166-10177 [PMID: 30422310 DOI: 10.1002/jcp.27686] 89

Circular RNAs: Identification, biogenesis and function. K K Ebbesen, J Kjems, T B Hansen, 10.1016/j.bbagrm.2015.07.007Biochim Biophys Acta. 1859Ebbesen KK, Kjems J, Hansen TB. Circular RNAs: Identification, biogenesis and function. Biochim Biophys Acta 2016; 1859: 163-168 [PMID: 26171810 DOI: 10.1016/j.bbagrm.2015.07.007]