# Rapid genotyping protocol to improve dengue virus serotype 2 survey in Lao PDR

CorpusID: 221075969
 
tags: #Biology, #Medicine, #Environmental_Science

URL: [https://www.semanticscholar.org/paper/91c70cf51d202ca25dbc38b3f73851203642569a](https://www.semanticscholar.org/paper/91c70cf51d202ca25dbc38b3f73851203642569a)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Rapid genotyping protocol to improve dengue virus serotype 2 survey in Lao PDR


Elodie Calvezid 
Institut Pasteur du Laos
Vientiane

Lao People's Democratic Republic


Somphavanh Somlor 
Institut Pasteur du Laos
Vientiane

Lao People's Democratic Republic


Souksakhone Viengphouthong 
Institut Pasteur du Laos
Vientiane

Lao People's Democratic Republic


Charlotte Balière 
Institut Pasteur
ParisFrance

Phaithong Bounmany 
Institut Pasteur du Laos
Vientiane

Lao People's Democratic Republic


Sitsana Keosenhom 
Institut Pasteur du Laos
Vientiane

Lao People's Democratic Republic


Valé Rie Caro 
Institut Pasteur
ParisFrance

Marc Grandadam 
Institut Pasteur du Laos
Vientiane

Lao People's Democratic Republic


Institut de Recherche Biomé dicale des Armé es
Bré tigny-sur-OrgeFrance

Rapid genotyping protocol to improve dengue virus serotype 2 survey in Lao PDR
RESEARCH ARTICLE
Dengue fever is one of the major public health problems in Lao PDR. Over the last decade, dengue virus (DENV) epidemics were characterized by a novel predominant serotype accompanied by at least two other serotypes. Since 2008, DENV-2 circulated at a low level in Lao PDR but its epidemiologic profile changed at the end of 2018. Indeed, the number of confirmed DENV-2 cases suddenly increased in October 2018 and DENV-2 became predominant at the country level in early 2019. We developed a Genotype Screening Protocol (GSP) to determine the origin(s) of the Lao DENV-2 and study their genetic polymorphism. With a good correlation with full envelope gene sequencing data, this molecular epidemiology tool evidence the co-circulation of two highly polymorphic DENV-2 genotypes, i.e. Asian I and Cosmopolitan genotypes, over the last five years, suggesting multiple introductions of DENV-2 in the country. GSP approach provides relevant first line information that may help countries with limited laboratory resources to reinforce their capabilities to DENV-2 and to follow the epidemics progresses and assess situations at the regional level.

## Introduction

Dengue fever is the most prevalent arboviral disease in the world. It represents a threat for 390 million people in 128 countries primarily in Latin America, Western Pacific and in South East Asia [1,2]. The World Health Organization (WHO) estimated that at least 500,000 people, mostly children, are hospitalized annually; of whom 2.5% have fatal outcomes.

Lao People's Democratic Republic (PDR) is a low-middle income country located at a central position of the Indochinese peninsula. Lao PDR has a population of 6.76 million with nearly 40% under the age of 20 (sources: WHO; www.lsb.gov.la). The first dengue virus (DENV) outbreak in Lao PDR was reported in 1979 [3]. Since then, several epidemics have been recorded in the country that raise dengue to the rank of a major public health concern for their national authorities [3]. Previous studies have highlighted the dramatic increase in commercial and tourism exchanges between Lao PDR and its neighboring countries as the driving force behind the entry of multiple DENV serotypes and/or genotypes into the country PLOS ONE PLOS ONE | https://doi.org/10.1371/journal.pone.0237384 August 7, 2020 1 / 14 a1111111111 a1111111111 a1111111111 a1111111111 a1111111111 [4,5]. The emergence of a new DENV serotype or genotype has been reported to enhance the number of cases and the diseases's severity [6][7][8].

In 2012, an arbovirus laboratory surveillance system was set up in the capital city of Vientiane to improve arboviral diagnosis and prevention in the country [5,9,10]. Since 2012, the Vientiane Capital Province has experienced two major DENV epidemics, first with a predominance of DENV-3 in 2012-2013 and then DENV-4 from the end of 2014 to end of 2018 [4,5,9,11]. Over this period, DENV-2 was only the cause of 3 to 20% of confirmed cases [5] (https://www.pasteur.la/). However, in October 2018 DENV-2 burden progressively increased and reached 43% of the confirmed dengue infections in Vientiane capital the following December [12]. Since the beginning of 2019, DENV-2 has become the predominant serotype; representing 68.9% of the samples serotyped in the capital [13]. Of the 21 fatal cases recorded in 2019 in Vientiane Capital by the arbovirus surveillance system, six could be serotyped, of which four were assigned to DENV-2. At the country level, DENV-2 has been reported in 12 of the 18 provinces (Fig 1).

Little is known concerning DENV-2 genotype circulation and phylogeny in Lao PDR. Here we report on the development of a Genotype Screening Protocol (GSP) based on partial E gene sequencing for the rapid identification of DENV-2 genotyping and its origin. Performance of the GSP was compared to full E gene sequencing. GSP was also retrospectively applied to a panel of DENV-2 positive samples or isolates collected over the last eight years in Vientiane Capital as well as other Lao provinces.


## Materials and methods


## Sample collection

A panel of 53 DENV-2 isolates, screened by the arbovirus surveillance network as previously described [5], of autochthonous (51 samples) and imported cases (2 samples) across six Lao provinces between 2012 to 2019, was analyzed (Fig 1; Table 1).


## Ethic statement

The National Ethic Committee for Health Research of the Ministry of Health of Lao PDR approved the protocol used in this study. This protocol was also approved by all public hospitals' management committees and the agreement of the Ministry of health was obtained.

For all the patient samples included in this study, a written informed consent was provided by all the adult volunteers or by a parent or legal guardian on behalf of their children.


## Screening of dengue suspected cases and serotype identification

RNA was extracted from the plasma of suspected dengue cases using Nucleospin RNA virus kit (Macherey-Nagel) following the manufacturer's instructions. Sample RNA was first screened by mean of a pan-dengue virus one-step RT-PCR as described previously [5,14]. DENV serotype identification was tentatively determined using primer/probe sets described by Ito et al adapted to a multiplex one-step RT-PCR format [15].


## Gene E sequencing analysis

The DENV genotypes and their geographic origin(s) were initially determined by the rapid Genotype Screening Protocol (GSP) based on partial envelope gene sequencing. Viral genome RNA was extracted from human plasma when the Cq value of the screening RT-PCR was below 30. For samples which displayed a Cq value above 30, and when the remaining volume was sufficient, DENV was isolated on C6/36 monolayers to improve the chance of sequencing.

Such additional viral amplification has been documented to have no impact on viral genome stability [16]. In these cases, the above RNA extraction protocol was applied to viral culture supernatants. For sequencing purposes, first strand cDNA was generated using random hexamers and the Maxima H Minus First Strand cDNA Synthesis kit (Thermo Scientific). PCR was performed using Phusion Flash High-Fidelity PCR Master Kit (New England Biolabs 1 Inc). GSP (552 nt) was performed using FG4 primers ( Table 2). Complete envelope gene sequences (1485 nt) were established for a panel of samples (Table 2) selected within the clusters obtained by GSP using primers FR2 or FGT2 (respectively adapted for Cosmopolitan and Asian I genotypes), FGT3 and FG4 (Table 2). Amplified fragments were purified using ExoSA-P-IT TM PCR Product Cleanup Reagent (Thermo Fisher Scientific) or by purification of PCR products from agarose gel using the Cleanup Gel extraction kit (Macherey-Nagel) when  


## Phylogenetic analysis

Partial (53 sequences; GenBank MN444556 to MN444604) and complete (22 sequences; Gen-Bank MN444605 to MN444622) envelope sequences of Lao isolates (Table 1) were compared with envelope gene sequences retrieved from GenBank (S1 Table) [4,. Maximum-likelihood trees were constructed using MEGA version 7 (www.megasoftware.net) as previously described [5,6,[45][46][47], with the Kimura-2-parameter method and a bootstrap of 1000 replications [48].


## Results


## Co-circulation of two DENV-2 genotypes in Lao PDR

Of this panel of 53 isolates, GSP verified the co-circulation of two DENV-2 genotypes. Asian I and Cosmopolitan genotypes were both detected in different Lao PDR provinces (Fig 2). The co-circulation of these genotypes was detected in 2017, 2018 and 2019 in the country.


## Persistence of DENV-2 Asian I genotype in Lao PDR

Lao DENV-2 samples isolated between 2008 and 2010 have been assigned to the Asian I genotype [4]. In our series, 19 isolates (36%) obtained after 2012 shared strong identity (>97.2%) with this genotype and demonstrates its persistence in the country for over eleven years. Lao Asian I isolates since 2008 were classed into seven clusters, displaying distances of 1.0 to 2.8% from each other at the nucleotide level (Fig 2). Three of these clusters were described (1) Genotype Screening Protocol.

(2) Cq were obtained from plasma or culture supernatant with pan-dengue RT-PCR from Warrilow et al [14]. ND: not done. F: Failed. Imported cases are indicated in bold.

https://doi.org/10.1371/journal.pone.0237384.t001 


## PLOS ONE

previously and one was found specifically from the Attapeu province [4]. All the Lao isolates showed strong links with strains from Vietnam, Thailand, China, Taiwan and Myanmar.


## Co-circulation of different lineages and clusters within DENV-2 cosmopolitan genotype

A total of 34 isolates (64%) belonged to the Cosmopolitan genotype, and analysis showed that it may has been circulating in Lao PDR since at least 2015. Interestingly, the Cosmopolitan Lao isolates split into two distinct lineages (A; B), displaying 4.9-5.7% divergence from each other (bootstrap 87%). These lineages have coexisted at least as early as 2017. Within Lineage A, all Lao isolates grouped together sharing more than 99.45% identity (Cluster 1). Strong links (>98.7% identity; bootstrap 79%) were found between Lineage A with DENV-2 strains circulating in China and India in 2015 and 2011/2013 respectively. All four fatal cases included in this series belonged to Lineage B (Fig 2). Three of the four fatal cases grouped with the rest of the autochthonous isolates in a major cluster (Cluster 2, Fig 2). Surprisingly, the last fatal case (LaoPDR-Vientiane Capital-2019-9080) displayed between 1.1 and 2.1% nucleotide distance with the rest of Cluster 2. Autochthonous Lao isolates belonging to Lineage B strongly linked with DENV-2 strains from Thailand/Malaysia/Singapore/Sri-Lanka. One of the two imported cases (ex Thailand), fell in Cluster 2 whereas the second (ex Vietnam) grouped with other regional isolates detected in China, India, Bangladesh, Sri Lanka, and Southeast Asia.


## Validation of a Genotype Screening Protocol

To assess the accuracy of the GSP, the genotype/lineage/cluster assignments of 8 Asian I and 14 Cosmopolitan isolates were checked by full length envelope gene sequencing (Fig 3). The distribution of reference strains and Lao isolates established by the full envelope genotyping globally matched with the GSP results including at the levels of lineages and/or clusters. However, differences in the topology of the trees were observed for the Asian II genotype, for which some strains were previously found more closely related to the Asian/American genotype by GSP. At a higher level, the Asian II and the Asian/American genotypes were linked to the Cosmopolitan genotype by GSP rather than to the Asian I genotype as with full envelope sequencing (Fig 3).

In the subset selected for the complete envelope gene sequencing, sequences could not be established for some samples (plasma or culture supernatants) with a Cq>30 when tested with the screening real time RT-PCR (Table 1). For instance, from the 4 investigated fatal DENV-2 cases, the genotype of one sample (LaoPDR-Vientiane Capital-2019-9060) with a Cq = 36 could only be determined by GSP whereas the full envelope gene sequencing failed.


## Discussion

Dengue molecular epidemiology in Lao PDR is complex and the dynamic over time depends in part on virus serotypes and geographic origins [4,5]. Indeed, Lao PDR is localized in a hyperendemic DENV region in Southeast Asia in the middle of the Indochinese peninsula [49]. Since the first dengue outbreak in 1979, the four DENV serotypes are frequently detected 


## PLOS ONE

Dengue virus serotype 2 phylogeny in Vientiane, Lao PDR in the country [3-5, 9, 11]. DENV genotype co-circulation has already been depicted in Lao PDR for DENV-3 [5]. For DENV-2, the co-circulation of Asian I and Asian/American or Cosmopolitan genotypes has been documented in Thailand, Cambodia or Vietnam [43,50].

In most of these situations, the genotypes coexisted over several years [50]. In Lao PDR, Asian I and Cosmopolitan genotypes have been co-circulating since at least 2017 and was still ongoing in 2019. Compared to a recent study in Thailand, our series identified different lineages within the DENV-2 Cosmopolitan genotype [43]. Indeed, in Thailand only the Cosmopolitan genotype/Lineage B was observed between 2015 and 2016. As seen elsewhere, since 2008 the Lao clusters within the Asian I genotype have had different geographic origins and evolved over time as independent topotypes as is seen on other countries [4,43,50]. In a panel tested previously and in this study, the co-circulation of two or three clusters were observed in 2009-2010 (cluster 1, 2 and 3), in 2012 (clusters 3 and 5), in 2017 (clusters 4 and 6) and in 2018 (clusters 4 and 7). Interestingly, some clusters seemed to be localized only in some part of Lao PDR such as clusters 2 and 6 in Vientiane, and cluster 4 in Attapeu. Though, this observation could be due to the limited size of the series. However, the close relation between cluster 4 isolates and Vietnamese prototype strains supports another hypothesis. It can be assumed that this cluster originates from a direct introduction from Vietnam as the Eastern limit of the province materialize part of the Lao-Vietnam border. A main road, rubber plantations held by Vietnamese companies, are factors that may at least in part facilitate the traffic of DENV virus in southern Lao PDR. Further investigation is needed to better document the cluster-specific circulation of DENV-2 Asian I genotype in Lao PDR and to determine factors which could impact the selection of specific DENV strains. For instance, it has already been demonstrated that specific adaptations between the vector population and DENV may lead to an enhancement of mosquitos' ability to transmit DENV [51,52].

GSP results showed independent introduction events of DENV-2 between 2012 and 2019 in Lao PDR. Genetic links and the geographic origin of the imported cases suggest a possible role of human regional mobility as a factor of DENV-2 polymorphism through the introduction of novel genotype, lineage, or clusters into the country. Observations of the presence of the Cosmopolitan genotype with the co-circulation of two different lineages detected as early as 2016, along with the circulation of several Asian I genotype clusters since 2008 support this hypothesis. Lao PDR shares a border with five countries where DENV circulation is highly active [1,43,49]. Exchanges between Lao PDR and its neighboring countries is increasing constantly especially with regards to commerce [4,5]. In this context, the identification of potential DENV introduction routes into Lao PDR and their impact on dengue epidemiology in the country will be challenging in the future.

Here, GSP demonstrated rapid preliminary information on DENV-2 polymorphisms that were in most cases confirmed by full length envelope sequencing. Tree topology differences could be due to the limited number of nucleotides used for GSP (552nt versus 1,485nt for full length envelope). However, GSP analytic sensitivity seemed to be higher compared to full length envelope sequencing. In this study, samples with high Cq (above 30) could be only sequenced using GSP, which can help the investigation of samples with volume limitations or for whom viral culture is not possible.

Identification of DENV serotype(s)/genotype(s) during an outbreak can help improve dengue prevention, especially in endemic countries where the disease severity is still challenging PLOS ONE [53]. Improvement of rapid RT-PCR diagnostic tool already help the identification of DENV serotype [5,15].

As previously described, DENV genotype switch or co-circulation could impact DENV emergence and spread as well as modify its epidemiology profile by increasing the number of infected patients and fatal cases [6][7][8]. DENV genotypes may display differences in their fitness and virulence. Some studies already suggested a possible link between a sudden switch of DENV genotypes and an increase of disease severity for DENV-1, DENV-2 and DENV-3 in Asia or South America [54][55][56][57]. In our series, the four DENV-2 isolates from fatal cases in 2018 and 2019 belonged to the Cosmopolitan genotype/Lineage B which was detected in 2017, but has not had an increase in cases by the end of 2018. Parallel studies are needed to investigate the respective roles of virus pathogenicity and indirect factors such as socio-economic factors governing access to care to determine the real impact of the emergence of DENV genotypes in endemic countries like in Lao PDR [58].

In Lao PDR, herd immunity against the different dengue virus serotypes remains cryptic and it can be assumed to be low for DENV-2, as seen by the background circulation and the various introductions seen throughout this study period.

Virologic investigations are crucial for DENV epidemic prevention and follow up. Direct diagnosis by RT-PCR is at present the gold standard for confirming dengue virus infection during the acute phase of the disease. When the geographic coverage is sufficient across the country, the serotype determination by real time RT-PCR can be used to estimate the proportions of each DENV serotype and generate data that could be used as a proxy to estimate the herd immunity against each. Molecular epidemiology by partial or complete envelope gene sequencing has been used for decades to better understand DENV circulation at regional or inter-continental levels. Genotype determination reinforces the capacity to identify transborder routes of circulation and establish a more precise timepoint of emergence of a specific cluster of DENV isolates. From that perspective, GSP is a valuable frontline molecular epidemiological tool for countries with limited laboratory resources that are confronted with dengue. Moreover, GSP could be useful to follow the circulation of DENV-2 on both a country and regional scale.

Supporting information S1 

## Fig 1 .
1Map of Lao PDR. Provinces with DENV-2 reported cases are in yellow. Triangles indicate the localization of DENV-2 isolates which were sequenced in this study. This map was generated by IPL staff using Inkscape software free of copyright.https://doi.org/10.1371/journal.pone.0237384.g001PLOS ONE

## Table 2 .
2List and positions of primers used for partial or complete envelope gene RT-PCR and sequencing. TCA-CCA-TAA-TGG-CAG-CAA-TC 3' 836-855 5' TGC-ACC-AGC-CAA-GCT-TTA-TGT-GAA-GGA-AAT-AGC-AGA 3' 1860-1877 5' AGT-TCT-TTG-TTT-TTC-CAG-CT 3' 2440-2459 The genome positions are given according to the dengue virus serotype 2 reference genome (GenBank: U87411). a indicates the primers used for complete envelope gene sequencing and b for the Genotype Screening Protocol. https://doi.org/10.1371/journal.pone.0237384.t002

## Fig 2 .
2Maximum-likelihood phylogenetic tree of DENV-2 sequences from Lao PDR. The tree was constructed on a 552nt segment of a partial envelope protein gene (Genotype Screening Protocol). Bootstrap values >70 are shown next the node. The Lao PDR strains analyzed in this study are indicated in red. The Lao PDR strains previously described are in blue. Cluster 1-3 were previously described by Castonguay-Vanier et al). Scale bar indicates nucleotide substitution per site. The triangles indicate discordance between Genotype Screening Protocol and full envelope protein gene sequencing at the genotype and lineage level.https://doi.org/10.1371/journal.pone.0237384.g002

## Fig 3 .
3Maximum-likelihood phylogenetic tree of DENV-2 sequences from Lao PDR. The tree was constructed on a 1485nt segment of the envelope protein gene. Only the bootstrap values >70 are shown. Scale bar indicates nucleotide substitution per site. The Lao PDR strains analyzed in this study are indicated in red. The Lao PDR strains previously described are in blue. Cluster 1-3 were previously described by Castonguay-Vanier et al).https://doi.org/10.1371/journal.pone.0237384.g003

## Table 1 .
1References of Lao DENV-2 isolates.nonspecific amplification was observed by agarose gel electrophoresis. Forward and reverse strands were independently sequenced using BigDye Terminator v3.1 Cycle sequencing kit (Applied Biosystem) and loaded on a Genetic Analyzer 3500xL (Applied Biosystem).Sample identification 
Years of collection 
RNA source 
GenBank accession number 
Cq (2) 

GSP (1) 
Full Envelopee protein 

LaoPDR-Vientiane Capital -2015-3231 
2015 
Culture supernatant 
MN444556 
MN444605 
27 

LaoPDR-Vientiane Capital -2016-3550 
2016 
Culture supernatant 
MN444557 
MN444606 
24 

LaoPDR-Vientiane Capital -2016-3771 
2016 
Culture supernatant 
MN444558 
MN444607 
26 

LaoPDR-Vientiane Capital -2016-3831 
2016 
Plasma 
MN444559 
MN444608 
25 

LaoPDR-Vientiane Capital -2017-5207 
2017 
Culture supernatant 
MN444560 
MN444609 
20 

LaoPDR-Vientiane Capital -2017-5208 
2017 
Culture supernatant 
MN444561 
MN444610 
21 

LaoPDR-Vientiane Capital -2017-5443 
2017 
Culture supernatant 
MN444562 
MN444611 
24 

LaoPDR-Vientiane Capital -2017-5902 
2017 
Plasma 
MN444563 
F 
20 

LaoPDR-Vientiane Capital -2017-5955 
2017 
Culture supernatant 
MN444564 
F 
25 

LaoPDR-Vientiane Capital -2017-6086 
2017 
Plasma 
MN444565 
F 
21 

LaoPDR-Vientiane Capital -2017-6177 
2017 
Plasma 
MN444566 
F 
25 

LaoPDR-Vientiane Capital -2017-6446 
2017 
Culture supernatant 
MN444567 
MN444612 
22 

LaoPDR-Vientiane Capital -2018-7406 
2018 
Culture supernatant 
MN444568 
MN444613 
22 

LaoPDR-Vientiane Capital -2018-7692 
2018 
Plasma 
MN444569 
ND 
31 

LaoPDR-Vientiane Capital -2018-7740 
2018 
Plasma 
MN444570 
F 
22 

LaoPDR-Vientiane Capital -2018-7800 
2018 
Plasma 
MN444571 
ND 
23 

LaoPDR-Vientiane Capital -2018-7944 
2018 
Plasma 
MN444572 
F 
23 

LaoPDR-Vientiane Capital -2018-8075 
2018 
Plasma 
MN444573 
ND 
21 

LaoPDR-Vientiane Capital -2018-8173 
2018 
Plasma 
MN444574 
MN444614 
22 

LaoPDR-Vientiane Capital -2018-8359 
2018 
Plasma 
MN444575 
ND 
25 

LaoPDR-Vientiane Capital -2018-8366 
2018 
Plasma 
MN444576 
ND 
26 

LaoPDR-Vientiane Capital (fatal case)-2018-8372 
2018 
Culture supernatant 
MN444577 
MN444615 
18 

LaoPDR-Vientiane Capital -2018-8509 
2018 
Plasma 
MN444578 
ND 
29 

LaoPDR-Vientiane Capital -2018-8518 
2018 
Plasma 
MN444579 
ND 
24 

LaoPDR-Vientiane Capital -2018-8541 
2018 
Plasma 
MN444580 
ND 
31 

LaoPDR-Vientiane Capital (fatal case)-2018-8920 
2018 
Plasma 
MN444581 
MN444616 
23 

LaoPDR-Vientiane Capital (fatal case)-2019-9060 
2019 
Plasma 
MN444582 
F 
36 

LaoPDR-Vientiane Capital (fatal case)-2019-9080 
2019 
Culture supernatant 
MN444583 
MN444617 
18 

LaoPDR-Vientiane Capital -2019-9128 
2019 
Plasma 
MN444584 
ND 
20 

LaoPDR-Vientiane Province-2017-5545 
2017 
Plasma 
MN444585 
F 
29 

LaoPDR-Vientiane Province-2017-5725 
2017 
Plasma 
MN444586 
F 
25 

LaoPDR-Vientiane Province-2017-5942 
2017 
Culture supernatant 
MN444587 
ND 
24 

LaoPDR-Luangprabang-2017-5705 
2017 
Plasma 
MN444588 
MN444618 
24 

LaoPDR-Luangprabang-2018-8737 
2018 
Plasma 
MN444589 
F 
26 

LaoPDR-Luangprabang-2018-8741 
2018 
Plasma 
MN444590 
F 
25 

LaoPDR-Savannakhet-2018-8014 
2018 
Plasma 
MN444591 
MN444619 
25 

LaoPDR-Savannakhet-2018-8135 
2018 
Plasma 
MN444592 
ND 
20 

LaoPDR-Savannakhet-2018-8358 
2018 
Plasma 
MN444593 
F 
29 

LaoPDR-Saravane-2018-7670 
2018 
Plasma 
MN444594 
F 
26 

LaoPDR-Saravane-2018-7936 
2018 
Plasma 
MN444595 
F 
21 

LaoPDR-Saravane-2019-9444 
2019 
Plasma 
MN444596 
ND 
25 

LaoPDR-Saravane-2019-9474 
2019 
Plasma 
MN444597 
ND 
21 

LaoPDR-Attapeu-2017-5601 
2017 
Culture supernatant 
MN444598 
MN444620 
23 

LaoPDR-Attapeu-2017-6144 
2017 
Plasma 
MN444599 
F 
31 

LaoPDR-Attapeu-2018-7507 
2018 
Plasma 
MN444600 
F 
23 

LaoPDR-Attapeu-2018-7511 
2018 
Plasma 
MN444601 
MN444621 
24 

(Continued ) 


## Table 1 .
1(Continued)    Sample identification 
Years of collection 
RNA source 
GenBank accession number 
Cq (2) 

GSP (1) 
Full Envelopee protein 

LaoPDR-Attapeu-2019-9494 
2019 
Plasma 
MN444602 
ND 
22 

LaoPDR-Vientiane Capital (ex Thailand)-2017-5642 
2017 
Plasma 
MN444603 
MN444622 
27 

LaoPDR-Vientiane Capital (ex Vietnam)-2018-8746 
2018 
Plasma 
MN444604 
F 
30 



## Table .
.References of DENV-2 envelope gene sequences from GenBank used in this study. (DOCX) Resources: Somphavanh Somlor, Souksakhone Viengphouthong, Phaithong Bounmany, Sitsana Keosenhom. 18. Leitmeyer KC, Vaughn DW, Watts DM, Salas R, Villalobos I, de Chacon null, et al. Dengue virus structural differences that correlate with pathogenesis. J Virol. 1999 Jun; 73(6):4738-47. https://doi.org/10. 1128/JVI.73.6.4738-4747.1999 PMID: 10233934 19. Vasilakis N, Fokam EB, Hanson CT, Weinberg E, Sall AA, Whitehead SS, et al. Genetic and phenotypic characterization of sylvatic dengue virus type 2 strains. Virology. 2008 Aug; 377(2):296-307. https://doi. org/10.1016/j.virol.2008.04.044 PMID: 18570968 20. Phommanivong V, Kanda S, Shimono T, Lamaningao P, Darcy AW, Mishima N, et al. Co-circulation of the dengue with chikungunya virus during the 2013 outbreak in the southern part of Lao PDR. Trop Med Health. 2016 Dec; 44(1). 21. Kyaw AK, Ngwe Tun MM, Moi ML, Nabeshima T, Soe KT, Thwe SM, et al. Clinical, virological and epidemiological characterization of dengue outbreak in Myanmar, 2015. Epidemiol Infect. 2017 Jul; 145 (9):1886-97. https://doi.org/10.1017/S0950268817000735 PMID: 28414004 22. Warrilow D, Northill JA, Pyke AT. Sources of Dengue Viruses Imported into Queensland, Australia, 2002-2010. Emerg Infect Dis. 2012 Nov; 18(11):1850-7. https://doi.org/10.3201/eid1811.120014 PMID: 23092682 23. Yang C-F, Chang S-F, Hsu T-C, Su C-L, Wang T-C, Lin S-H, et al. Molecular characterization and phylogenetic analysis of dengue viruses imported into Taiwan during 2011-2016. Blacksell SD, editor. PLoS Negl Trop Dis. 2018 Sep 20; 12(9):e0006773. https://doi.org/10.1371/journal.pntd.0006773 PMID: 30235208 24. Vasilakis N, Holmes EC, Fokam EB, Faye O, Diallo M, Sall AA, et al. Evolutionary Processes among Sylvatic Dengue Type 2 Viruses. J Virol. 2007 Sep 1; 81(17):9591-5. https://doi.org/10.1128/JVI. 02776-06 PMID: 17553878 25. Phadungsombat J, Lin MY-C, Srimark N, Yamanaka A, Nakayama EE, Moolasart V, et al. Emergence of genotype Cosmopolitan of dengue virus type 2 and genotype III of dengue virus type 3 in Thailand. Samy AM, editor. PLOS ONE. 2018 Nov 12; 13(11):e0207220. https://doi.org/10.1371/journal.pone. 0207220 PMID: 30419004 26. Salje H, Lessler J, Maljkovic Berry I, Melendrez MC, Endy T, Kalayanarooj S, et al. Dengue diversity across spatial and temporal scales: Local structure and the effect of host population size. Science. 2017 Mar 24; 355(6331):1302-6. https://doi.org/10.1126/science.aaj9384 PMID: 28336667 27. Christenbury JG, Aw PPK, Ong SH, Schreiber MJ, Chow A, Gubler DJ, et al. A method for full genome sequencing of all four serotypes of the dengue virus. J Virol Methods. 2010 Oct; 169(1):202-6. https:// doi.org/10.1016/j.jviromet.2010.06.013 PMID: 20600330 28. Gruenberg A, Woo WS, Biedrzycka A, Wright PJ. Partial Nucleotide Sequence and Deduced Amino Acid Sequence of the Structural Proteins of Dengue Virus Type 2, New Guinea C and PUO-218 Strains. J Gen Virol. 1988 Jun 1; 69(6):1391-8. 29. Lewis JA, Chang G-J, Lanciotti RS, Kinney RM, Mayer LW, Trent DW. Phylogenetic Relationships of Dengue-2 Viruses. Virology. 1993 Nov; 197(1):216-24. https://doi.org/10.1006/viro.1993.1582 PMID: 8212556 30. Anzai S, Fukuda M, Otsuka Y, Eshita Y. Nucleotide Sequence and Phylogenetic Analyses of Dengue Type 2 Virus Isolated in the Dominican Republic. Virus Genes. 2004 Oct; 29(2):219-27. https://doi.org/ 10.1023/B:VIRU.0000036382.77987.84 PMID: 15284482 31. Añez G, Morales-Betoulle ME, Rios M. Circulation of Different Lineages of Dengue Virus Type 2 in Central America, Their Evolutionary Time-Scale and Selection Pressure Analysis. Coffey LL, editor. PLoS ONE. 2011 Nov 4; 6(11):e27459. https://doi.org/10.1371/journal.pone.0027459 PMID: 22076162 32. Williams M, Mayer SV, Johnson WL, Chen R, Volkova E, Vilcarromero S, et al. Lineage II of Southeast Asian/American DENV-2 is associated with a severe dengue outbreak in the Peruvian Amazon. Am J Trop Med Hyg. 2014 Sep; 91(3):611-20. https://doi.org/10.4269/ajtmh.13-0600 PMID: 25002298 33. Wu W, Bai Z, Zhou H, Tu Z, Fang M, Tang B, et al. Molecular epidemiology of dengue viruses in southern China from 1978 to 2006. Virol J. 2011 Jun 26; 8:322. 34. Zhao H, Zhang F-C, Zhu Q, Wang J, Hong W-X, Zhao L-Z, et al. Epidemiological and Virological Characterizations of the 2014 Dengue Outbreak in Guangzhou, China. PloS One. 2016; 11(6):e0156548. https://doi.org/10.1371/journal.pone.0156548 PMID: 27257804 35. Ma X, Zhen W, Yang P, Sun X, Nie W, Zhang L, et al. First confirmation of imported dengue virus serotype 2 complete genome in urine from a Chinese traveler returning from India. Virol J. 2014 Mar 25; 11:56. https://doi.org/10.1186/1743-422X-11-56 PMID: 24666930 36. Dash PK, Sharma S, Soni M, Agarwal A, Parida M, Rao PVL. Complete genome sequencing and evolutionary analysis of Indian isolates of Dengue virus type 2. Biochem Biophys Res Commun. 2013 Jul 5; 436(3):478-85. https://doi.org/10.1016/j.bbrc.2013.05.130 PMID: 23756811Supervision: Marc Grandadam. 

Visualization: Elodie Calvez. 

Writing -original draft: Elodie Calvez, Marc Grandadam. 

Writing -review & editing: Elodie Calvez, Valérie Caro, Marc Grandadam. 

PLOS ONE | https://doi.org/10.1371/journal.pone.0237384 August 7, 2020
PLOS ONE | https://doi.org/10.1371/journal.pone.0237384 August 7, 2020 14 / 14
AcknowledgmentsWe thank Joseph Kononchik Jr and Antony Black for helpful manuscript revision.
The global distribution and burden of dengue. S Bhatt, P W Gething, O J Brady, J P Messina, A W Farlow, C L Moyes, 10.1038/nature1206023563266Nature. 4967446Bhatt S, Gething PW, Brady OJ, Messina JP, Farlow AW, Moyes CL, et al. The global distribution and burden of dengue. Nature. 2013; 496(7446):504-7. https://doi.org/10.1038/nature12060 PMID: 23563266

Refining the global spatial limits of dengue virus transmission by evidence-based consensus. O J Brady, P W Gething, S Bhatt, J P Messina, J S Brownstein, A G Hoen, 10.1371/journal.pntd.000176022880140PLoS Negl Trop Dis. 681760Brady OJ, Gething PW, Bhatt S, Messina JP, Brownstein JS, Hoen AG, et al. Refining the global spatial limits of dengue virus transmission by evidence-based consensus. PLoS Negl Trop Dis. 2012; 6(8): e1760. https://doi.org/10.1371/journal.pntd.0001760 PMID: 22880140

Epidemiological situation of dengue infection in Lao P. T Fukunaga, B Phommasack, K Bounlu, M Saito, M Tadano, Y Makino, D.R. Trop Med. 354Fukunaga T, Phommasack B, Bounlu K, Saito M, Tadano M, Makino Y, et al. Epidemiological situation of dengue infection in Lao P.D.R. Trop Med. 1994; 35(4):219-27.

Molecular epidemiology of dengue viruses in three provinces of Lao PDR. J Castonguay-Vanier, R Klitting, O Sengvilaipaseuth, G Piorkowski, C Baronti, B Sibounheuang, 10.1371/journal.pntd.000620329377886PLoS Negl Trop Dis. 1216203Aguilar PV, editorCastonguay-Vanier J, Klitting R, Sengvilaipaseuth O, Piorkowski G, Baronti C, Sibounheuang B, et al. Molecular epidemiology of dengue viruses in three provinces of Lao PDR, 2006-2010. Aguilar PV, edi- tor. PLoS Negl Trop Dis. 2018 Jan 29; 12(1):e0006203. https://doi.org/10.1371/journal.pntd.0006203 PMID: 29377886

Co-Circulation of Dengue Virus Type 3 Genotypes in Vientiane Capital. M Lao, V Caro, J-M Thiberges, P Bounmany, K Vongpayloth, P Buchy, PLoS ONE. Lao PDR. Coffey LL912115569Lao M, Caro V, Thiberges J-M, Bounmany P, Vongpayloth K, Buchy P, et al. Co-Circulation of Dengue Virus Type 3 Genotypes in Vientiane Capital, Lao PDR. Coffey LL, editor. PLoS ONE. 2014 Dec 31; 9 (12):e115569.

Epidemiological and molecular features of dengue virus type-1 in. M Dupont-Rouzeyrol, Aubry M O&apos;connor, O Roche, C Gourinat, A C Guigon, A , 10.1186/1743-422X-11-6124684835Virol J. 1161Dupont-Rouzeyrol M, Aubry M, O'Connor O, Roche C, Gourinat AC, Guigon A, et al. Epidemiological and molecular features of dengue virus type-1 in New Caledonia, South Pacific, 2001-2013. Virol J. 2014; 11:61. https://doi.org/10.1186/1743-422X-11-61 PMID: 24684835

Emergence of new genotypes and lineages of dengue viruses during the 2012-15 epidemics in southern India. S F Ahamed, V Rosario, C Britto, M Dias, K Nayak, A Chandele, Int J Infect Dis. 84Ahamed SF, Rosario V, Britto C, Dias M, Nayak K, Chandele A, et al. Emergence of new genotypes and lineages of dengue viruses during the 2012-15 epidemics in southern India. Int J Infect Dis. 2019 Jul; 84:S34-43.

Molecular Characterization of Dengue Virus Serotype 2 Cosmospolitan Genotype From 2015 Dengue Outbreak in Yunnan. L Jiang, D Ma, C Ye, L Li, X Li, J Yang, China. Front Cell Infect Microbiol. 8Jiang L, Ma D, Ye C, Li L, Li X, Yang J, et al. Molecular Characterization of Dengue Virus Serotype 2 Cosmospolitan Genotype From 2015 Dengue Outbreak in Yunnan, China. Front Cell Infect Microbiol. 2018 Jun 27; 8.

epidemiological and laboratory findings. B Khampapongpane, H C Lewis, P Ketmayoon, D Phonekeo, V Somoulay, A Khamsing, West Pac Surveill Response J WPSAR. 51National dengue surveillance in theKhampapongpane B, Lewis HC, Ketmayoon P, Phonekeo D, Somoulay V, Khamsing A, et al. National dengue surveillance in the Lao People's Democratic Republic, 2006-2012: epidemiological and labora- tory findings. West Pac Surveill Response J WPSAR. 2014; 5(1):7-13.

Chikungunya virus emergence in the Lao PDR. S Somlor, K Vongpayloth, L Diancourt, P Buchy, V Duong, D Phonekeo, 10.1371/journal.pone.0189879292840122013. LFP12189879Somlor S, Vongpayloth K, Diancourt L, Buchy P, Duong V, Phonekeo D, et al. Chikungunya virus emer- gence in the Lao PDR, 2012-2013. LFP, editor. PLOS ONE. 2017 Dec 28; 12(12):e0189879. https:// doi.org/10.1371/journal.pone.0189879 PMID: 29284012

Dengue infection in Lao PDR. D Soukaloun, Southeast Asian J Trop Med Public Health. 451SupplSoukaloun D. Dengue infection in Lao PDR. Southeast Asian J Trop Med Public Health. 2014; 45 Suppl 1:113-9.

. Institut Pasteur du Laos. Institut Pasteur du Laos Activities Report. Institut Pasteur du Laos. Institut Pasteur du Laos Activities Report 2018-2019. 2019.

. Institut Pasteur du Laos. Institut Pasteur du Laos Activities Report. Institut Pasteur du Laos. Institut Pasteur du Laos Activities Report 2019-2020. 2020.

Single rapid TaqMan fluorogenic probe based PCR assay that detects all four dengue serotypes. D Warrilow, J A Northill, A Pyke, G A Smith, J Med Entomol. 664Warrilow D, Northill JA, Pyke A, Smith GA. Single rapid TaqMan fluorogenic probe based PCR assay that detects all four dengue serotypes. J Med Entomol. 2002; 66(4):524-8.

Development and Evaluation of Fluorogenic TaqMan Reverse Transcriptase PCR Assays for Detection of Dengue Virus Types 1 to 4. M Ito, T Takasaki, K-I Yamada, R Nerome, S Tajima, I Kurane, 10.1128/JCM.42.12.5935-5937.200415583346J Clin Microbiol. 4212Ito M, Takasaki T, Yamada K-I, Nerome R, Tajima S, Kurane I. Development and Evaluation of Fluoro- genic TaqMan Reverse Transcriptase PCR Assays for Detection of Dengue Virus Types 1 to 4. J Clin Microbiol. 2004 Dec 1; 42(12):5935-7. https://doi.org/10.1128/JCM.42.12.5935-5937.2004 PMID: 15583346

E/NS1 Modifications of Dengue 2 Virus after Serial Passages in Mammalian and/or Mosquito Cells. W-J Chen, H-R Wu, S-S Chiou, 10.1159/00007320814555849Intervirology. 465Chen W-J, Wu H-R, Chiou S-S. E/NS1 Modifications of Dengue 2 Virus after Serial Passages in Mam- malian and/or Mosquito Cells. Intervirology. 2003; 46(5):289-95. https://doi.org/10.1159/000073208 PMID: 14555849

Natural attenuation of dengue virus type-2 after a series of island outbreaks: A retrospective phylogenetic study of events in the South Pacific three decades ago. A Steel, D J Gubler, S N Bennett, 10.1016/j.virol.2010.05.03320663532Virology. 4052Steel A, Gubler DJ, Bennett SN. Natural attenuation of dengue virus type-2 after a series of island out- breaks: A retrospective phylogenetic study of events in the South Pacific three decades ago. Virology. 2010 Sep; 405(2):505-12. https://doi.org/10.1016/j.virol.2010.05.033 PMID: 20663532

dengue outbreaks in Singapore and Malaysia caused by different viral strains. L-C Ng, Y-K Chem, Koo C Mudin, Rnb Amin, F M Lee, K-S , 10.4269/ajtmh.14-058825846296Am J Trop Med Hyg. 926Ng L-C, Chem Y-K, Koo C, Mudin RNB, Amin FM, Lee K-S, et al. 2013 dengue outbreaks in Singapore and Malaysia caused by different viral strains. Am J Trop Med Hyg. 2015 Jun; 92(6):1150-5. https://doi. org/10.4269/ajtmh.14-0588 PMID: 25846296

Dengue viruses in Papua New Guinea: evidence of endemicity and phylogenetic variation, including the evolution of new genetic lineages. Emerg Microbes Infect. P R Moore, A F Van Den Hurk, J S Mackenzie, A T Pyke, 10.1038/emi.2017.103292593296114Moore PR, van den Hurk AF, Mackenzie JS, Pyke AT. Dengue viruses in Papua New Guinea: evidence of endemicity and phylogenetic variation, including the evolution of new genetic lineages. Emerg Microbes Infect. 2017 Dec 20; 6(12):e114. https://doi.org/10.1038/emi.2017.103 PMID: 29259329

Introduction and evolution of dengue virus type 2 in Pakistan: a phylogeographic analysis. M Akram, Z Fatima, M A Purdy, A Sue, S Saleem, I Amin, 10.1186/s12985-015-0371-8Virol J. 1226395339Akram M, Fatima Z, Purdy MA, Sue A, Saleem S, Amin I, et al. Introduction and evolution of dengue virus type 2 in Pakistan: a phylogeographic analysis. Virol J. 2015 Sep 22; 12:148. https://doi.org/10. 1186/s12985-015-0371-8 PMID: 26395339

A single amino acid in nonstructural protein NS4B confers virulence to dengue virus in AG129 mice through enhancement of viral RNA synthesis. D Grant, G K Tan, M Qing, Jkw Ng, A Yip, G Zou, 10.1128/JVI.00665-1121632767J Virol. 8515Grant D, Tan GK, Qing M, Ng JKW, Yip A, Zou G, et al. A single amino acid in nonstructural protein NS4B confers virulence to dengue virus in AG129 mice through enhancement of viral RNA synthesis. J Virol. 2011 Aug; 85(15):7775-87. https://doi.org/10.1128/JVI.00665-11 PMID: 21632767

Epidemic resurgence of dengue fever in Singapore in 2013-2014: A virological and entomological perspective. H C Hapuarachchi, C Koo, J Rajarethinam, C-S Chong, C Lin, G Yap, BMC Infect Dis. 161Hapuarachchi HC, Koo C, Rajarethinam J, Chong C-S, Lin C, Yap G, et al. Epidemic resurgence of den- gue fever in Singapore in 2013-2014: A virological and entomological perspective. BMC Infect Dis. 2016 Dec; 16(1).

Evolution of dengue virus type 2 during two consecutive outbreaks with an increase in severity in southern Taiwan in 2001-2002. H-L Chen, S-R Lin, H-F Liu, C-C King, S-C Hsieh, W-K Wang, 18840735Am J Trop Med Hyg. 794Chen H-L, Lin S-R, Liu H-F, King C-C, Hsieh S-C, Wang W-K. Evolution of dengue virus type 2 during two consecutive outbreaks with an increase in severity in southern Taiwan in 2001-2002. Am J Trop Med Hyg. 2008 Oct; 79(4):495-505. PMID: 18840735

Phylogenetic analysis revealed the co-circulation of four dengue virus serotypes in Southern Thailand. R Hamel, P Surasombatpattana, S Wichit, A Dauvé, C Donato, J Pompon, 10.1371/journal.pone.022117931415663Roques P14221179Hamel R, Surasombatpattana P, Wichit S, Dauvé A, Donato C, Pompon J, et al. Phylogenetic analysis revealed the co-circulation of four dengue virus serotypes in Southern Thailand. Roques P, editor. PLOS ONE. 2019 Aug 15; 14(8):e0221179. https://doi.org/10.1371/journal.pone.0221179 PMID: 31415663

Molecular characterization and phylogenetic analysis of dengue viruses imported into Taiwan during 2008-2010. J-H Huang, C-L Su, Yang C-F Liao, T-L Hsu, T-C Chang, S-F , 10.4269/ajtmh.2012.11-066622855770Am J Trop Med Hyg. 872Huang J-H, Su C-L, Yang C-F, Liao T-L, Hsu T-C, Chang S-F, et al. Molecular characterization and phy- logenetic analysis of dengue viruses imported into Taiwan during 2008-2010. Am J Trop Med Hyg. 2012 Aug; 87(2):349-58. https://doi.org/10.4269/ajtmh.2012.11-0666 PMID: 22855770

Evolutionary Analysis of Dengue Serotype 2 Viruses Using Phylogenetic and Bayesian Methods from New Delhi, India. Scarpino SV, editor. N Afreen, I H Naqvi, S Broor, A Ahmed, S N Kazim, R Dohare, 10.1371/journal.pntd.000451126977703PLoS Negl Trop Dis. 1034511Afreen N, Naqvi IH, Broor S, Ahmed A, Kazim SN, Dohare R, et al. Evolutionary Analysis of Dengue Serotype 2 Viruses Using Phylogenetic and Bayesian Methods from New Delhi, India. Scarpino SV, edi- tor. PLoS Negl Trop Dis. 2016 Mar 15; 10(3):e0004511. https://doi.org/10.1371/journal.pntd.0004511 PMID: 26977703

Dengue Virus Surveillance for Early Warning. K-S Lee, Y-L Lai, S Lo, T Barkham, P Aw, P-L Ooi, 10.3201/eid1605.0910062040938116SingaporeEmerg Infect DisLee K-S, Lai Y-L, Lo S, Barkham T, Aw P, Ooi P-L, et al. Dengue Virus Surveillance for Early Warning, Singapore. Emerg Infect Dis. 2010 May; 16(5):847-9. https://doi.org/10.3201/eid1605.091006 PMID: 20409381

The Complete Genome Phylogeny of Geographically Distinct Dengue Virus Serotype 2 Isolates (1944-2013) Supports Further Groupings within the Cosmopolitan Genotype. A Ali, I Ali, 10.1371/journal.pone.013890026414178Alvisi G10138900Ali A, Ali I. The Complete Genome Phylogeny of Geographically Distinct Dengue Virus Serotype 2 Iso- lates (1944-2013) Supports Further Groupings within the Cosmopolitan Genotype. Alvisi G, editor. PLOS ONE. 2015 Sep 28; 10(9):e0138900. https://doi.org/10.1371/journal.pone.0138900 PMID: 26414178

MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets. S Kumar, G Stecher, K Tamura, 10.1093/molbev/msw05427004904Mol Biol Evol. 337Kumar S, Stecher G, Tamura K. MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Big- ger Datasets. Mol Biol Evol. 2016; 33(7):1870-4. https://doi.org/10.1093/molbev/msw054 PMID: 27004904

Global spread of dengue virus types: mapping the 70 year history. J P Messina, O J Brady, T W Scott, C Zou, D M Pigott, K A Duda, 10.1016/j.tim.2013.12.01124468533Trends Microbiol. 223Messina JP, Brady OJ, Scott TW, Zou C, Pigott DM, Duda KA, et al. Global spread of dengue virus types: mapping the 70 year history. Trends Microbiol. 2014; 22(3):138-46. https://doi.org/10.1016/j.tim. 2013.12.011 PMID: 24468533

Emergence of the Asian 1 genotype of dengue virus serotype 2 in viet nam: in vivo fitness advantage and lineage replacement in South-East Asia. Ty Hang, T T Holmes, E C Duong, V Nguyen, T Q Tran, T H Quail, M , PLoS Negl Trop Dis. 4Ty Hang TT, Holmes EC, Duong V, Nguyen TQ, Tran TH, Quail M, et al. Emergence of the Asian 1 genotype of dengue virus serotype 2 in viet nam: in vivo fitness advantage and lineage replacement in South-East Asia. PLoS Negl Trop Dis. 2010; 4.

Quantitative genetics of Aedes aegypti vector competence for dengue viruses: towards a new paradigm?. L Lambrechts, 10.1016/j.pt.2010.12.00121215699Trends Parasitol. 273Lambrechts L. Quantitative genetics of Aedes aegypti vector competence for dengue viruses: towards a new paradigm? Trends Parasitol. 2011; 27(3):111-4. https://doi.org/10.1016/j.pt.2010.12.001 PMID: 21215699

Genetic specificity and potential for local adaptation between dengue viruses and mosquito vectors. L Lambrechts, C Chevillon, R G Albright, B Thaisomboonsuk, J H Richardson, R G Jarman, BMC Evol Biol. 9160Lambrechts L, Chevillon C, Albright RG, Thaisomboonsuk B, Richardson JH, Jarman RG, et al. Genetic specificity and potential for local adaptation between dengue viruses and mosquito vectors. BMC Evol Biol. 2009; 9(160):1471-2148.

Molecular evolution of dengue viruses: contributions of phylogenetics to understanding the history and epidemiology of the preeminent arboviral disease. S C Weaver, N Vasilakis, 10.1016/j.meegid.2009.02.00319460319Infect Genet Evol. 94Weaver SC, Vasilakis N. Molecular evolution of dengue viruses: contributions of phylogenetics to understanding the history and epidemiology of the preeminent arboviral disease. Infect Genet Evol. 2009; 9(4):523-40. https://doi.org/10.1016/j.meegid.2009.02.003 PMID: 19460319

Emergence and global spread of a dengue serotype 3, subtype III virus. W B Messer, D J Gubler, E Harris, K Sivananthan, A M Silva, Emerg Infect Dis. 9Messer WB, Gubler DJ, Harris E, Sivananthan K, Silva AM. Emergence and global spread of a dengue serotype 3, subtype III virus. Emerg Infect Dis. 2003;9.

Co-circulation of all the four dengue virus serotypes and detection of a novel clade of DENV-4 (genotype I) virus in Pune, India during 2016 season. S Shrivastava, D Tiraki, A Diwan, S K Lalwani, M Modak, A C Mishra, 10.1371/journal.pone.019267229470509PLOS ONE. Ansari AA132192672Shrivastava S, Tiraki D, Diwan A, Lalwani SK, Modak M, Mishra AC, et al. Co-circulation of all the four dengue virus serotypes and detection of a novel clade of DENV-4 (genotype I) virus in Pune, India dur- ing 2016 season. Ansari AA, editor. PLOS ONE. 2018 Feb 22; 13(2):e0192672. https://doi.org/10.1371/ journal.pone.0192672 PMID: 29470509

Reemergence of dengue virus type-3 (subtype-III) in India: implications for increased incidence of DHF & DSS. P K Dash, M M Parida, P Saxena, A Abhyankar, C P Singh, K N Tewari, 10.1186/1743-422X-3-5516824209Virol J. 355Dash PK, Parida MM, Saxena P, Abhyankar A, Singh CP, Tewari KN, et al. Reemergence of dengue virus type-3 (subtype-III) in India: implications for increased incidence of DHF & DSS. Virol J. 2006 Jul 6; 3:55. https://doi.org/10.1186/1743-422X-3-55 PMID: 16824209

Origins of Dengue Type 2 Viruses Associated with Increased Pathogenicity in the Americas. R Rico-Hesse, L M Harrison, R A Salas, D Tovar, A Nisalak, C Ramos, 10.1006/viro.1997.85049143280Virology. 2302Rico-Hesse R, Harrison LM, Salas RA, Tovar D, Nisalak A, Ramos C, et al. Origins of Dengue Type 2 Viruses Associated with Increased Pathogenicity in the Americas. Virology. 1997 Apr; 230(2):244-51. https://doi.org/10.1006/viro.1997.8504 PMID: 9143280

Microevolution and virulence of dengue viruses. R Rico-Hesse, Adv Exp Med Biol. 59Rico-Hesse R. Microevolution and virulence of dengue viruses. Adv Exp Med Biol. 2003; 59:315-41.