# Using environmental DNA methods to survey for rare groundwater fauna: Detection of an endangered endemic cave crayfish in northern Alabama

CorpusID: 228100640
 
tags: #Biology, #Medicine, #Geography, #Environmental_Science

URL: [https://www.semanticscholar.org/paper/0e3f95ff30dcbeb0f8d03fa1005fbad22d9faa28](https://www.semanticscholar.org/paper/0e3f95ff30dcbeb0f8d03fa1005fbad22d9faa28)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Using environmental DNA methods to survey for rare groundwater fauna: Detection of an endangered endemic cave crayfish in northern Alabama


Spencer H Boyd 
Department of Biological Sciences
The University of Alabama in Huntsville
HuntsvilleALUnited States of America

K Denise 
Department of Biological Sciences
The University of Alabama in Huntsville
HuntsvilleALUnited States of America

Kendall Niemiller 
Department of Biological Sciences
The University of Alabama in Huntsville
HuntsvilleALUnited States of America

Katherine E Dooley Id 
Department of Biological Sciences
The University of Alabama in Huntsville
HuntsvilleALUnited States of America

Jennifer Nix 
Department of Biological Sciences
The University of Alabama in Huntsville
HuntsvilleALUnited States of America

Matthew L Niemiller Id *matthew.niemiller@uah.edu 
Department of Biological Sciences
The University of Alabama in Huntsville
HuntsvilleALUnited States of America

Using environmental DNA methods to survey for rare groundwater fauna: Detection of an endangered endemic cave crayfish in northern Alabama
RESEARCH ARTICLE
The conservation and management of subterranean biodiversity is hindered by a lack of knowledge on the true distributions for many species, e.g., the Wallacean shortfall. In recent years, several studies have demonstrated the potential of environmental DNA (eDNA) as an effective approach to detect and monitor biodiversity, including rare, threatened, and endangered taxa. However, there are few eDNA studies of groundwater fauna. Here we report the results of the development and implementation of an eDNA assay targeting a short fragment of the mitochondrial CO1 locus of a critically imperiled cave crayfish, the Sweet Home Alabama Cave Crayfish (Cambarus speleocoopi), known from just four cave systems in the Interior Plateau karst region of northern Alabama. We detected C. speleocoopi DNA from water samples collected at 5 of 16 sites sampled (caves and springs), including two historical sites as well as three additional and potentially new sites in Marshall County, Alabama. All three of these sites were within 2 km of historical sites. Our study is the first to detect a groundwater crustacean in the Interior Plateau karst region. Additionally, our study contributes to the growing literature that eDNA is a viable complementary tool for detection and monitoring of a fauna that is difficult to survey and study using traditional approaches.OPEN ACCESSCitation: Boyd SH, Niemiller KDK, Dooley KE, Nix J, Niemiller ML (2020) Using environmental DNA methods to survey for rare groundwater fauna: Detection of an endangered endemic cave crayfish in northern Alabama. PLoS ONE 15(12): e0242741.

## Introduction

Effective conservation and management of biodiversity is limited by a lack of knowledge on the distributions of species. This biodiversity knowledge gap known as the Wallacean shortfall [1] is particularly prominent for fauna that live in groundwater and other subterranean ecosystems [2,3]. This is, in part, because subterranean habitats are extremely challenging to access and survey using traditional approaches, such as visual surveys and trapping (i.e., the Racovitzan shortfall; [4]). Most stygofauna-obligate groundwater species-are thought to have small, restricted distributions (i.e., short-range endemics; sensu [5]) and limited dispersal ability [6] and, consequently, are of high conservation concern. Thus, the development of sound management strategies and measurable conservation priorities for most stygofauna is exceedingly difficult.


## PLOS ONE

PLOS ONE | https://doi.org/10.1371/journal.pone.0242741 December 10, 2020 1 / 13 a1111111111 a1111111111 a1111111111 a1111111111 a1111111111

Few cave systems in the Tennessee-Alabama-Georgia (TAG) region have been comprehensively surveyed biologically [29,30], and there is great potential that species such as C. speleocoopi occur at additional undocumented sites. In fact, cavers regularly report observations of "white and blind crayfishes" from undocumented cave systems in northern Alabama. However, species identification requires specimen collection and genetic analysis, as morphology alone cannot easily distinguish C. speleocoopi, C. laconensis, and C. jonesi [26]. In fact, populations of C. speleocoopi from Beech Spring and Keller's caves in Marshall County were identified as C. jonesi previously [31]. Consequently, eDNA may be an appealing, nondestructive alternative to rapidly determine species occupancy and identification at a spring, cave, or well for rare, threatened, or endangered groundwater species without the need for specimen collection and expert identification.

In this study, we developed, tested, and validated an eDNA assay for C. speleocoopi and screened water samples collected from springs and cave systems (including historical sites) within and near its distribution to test the applicability of an eDNA approach to detect a karst groundwater crustacean and identify possible new sites of this imperiled crayfish. Our study demonstrates the potential utility of eDNA as an effective surveying and monitoring tool for groundwater biodiversity but also highlights some challenges of this approach when applied to groundwater ecosystems.


## Methods


## Ethics statement

This research was authorized under Alabama Department of Conservation and Natural Resources scientific collection permit nos. 2018061776268680 and 2019060224868680 and Alabama State Parks scientific permit no. 192213. Cave location data has been intentionally omitted to protect these sensitive ecosystems and their biodiversity. Cave location data are maintained by the Alabama Cave Survey (ACS; http://www.alabamacavesurvey.org) and can be requested from ACS or the corresponding author.


## Sampling sites

We collected water samples from 13 sites between March 2018 and April 2019 (Table 1), including two historical sites-Cherry Hollow Cave (and the associated spring run) and Beech Spring Cave. Cambarus speleocoopi has been confirmed at these sites within the past three years (Niemiller, unpublished data). We could not obtain permission to access Keller's Cave (site 17) (but collected water at the nearby spring; site 12) and Porches Spring Cave (site 18). We also sampled 10 additional sites (three caves and seven springs) located within a 20-km radius of historical sites in Jackson and Marshall counties, Alabama (Table 1; Fig 2). Three sites (sites 14-16 in Table 1) outside of the suspected range of C. speleocoopi were included to serve as negative field controls.


## Water sampling, filtering, and eDNA filter extraction

We collected 1-2 L of water in total (2-4 500 mL samples) at each site by submerging sterile Nalgene bottles beneath the surface. For spring sites, we collected water samples as close as possible to the point where water emerged from underground. Water samples were collected, placed on ice in a cooler, then transported back to the laboratory at The University of Alabama in Huntsville for filtering.

Water samples were vacuum-filtered within 24 hours of collection in the laboratory using 0.45-μm cellulose-nitrate filters (Thermo Scientific TM Nalgene TM ) following Niemiller et al. [16]. After filtration, the filters were folded, transferred to 8-mL sterile tubes, then stored at -20˚C until DNA extraction. Multiple filters were needed for a few water samples with substantial amounts of suspended silt and organic matter. For each set of water samples filtered, we also filtered a water sample comprised of distilled water to serve as a negative control.

We extracted DNA from filters in a dedicated laminar flow hood using a modified Qia-gen1 DNeasy Blood and Tissue Kit (Qiagen Inc., Valencia, CA, USA) protocol. One half of each filter was cut into small pieces using flame-sterilized scissors and transferred into a 2-mL microcentrifuge tube. The remaining half of each filter was stored long-term to serve as a back-up. For environmental water samples that required multiple filters, we extracted DNA from one-half of each filter and then pooled elutions. We added 360 μL ATL Buffer and 40 μL of Proteinase K (Qiagen, Inc.) then incubated filter pieces overnight at 55˚C. Samples were then vortexed for 15 s and centrifuged for 1 min (8000 g). The resulting supernatant was transferred into a new 2-mL microcentrifuge and processed following the manufacturer's protocol, with the exceptions of using 400 μL of AL Buffer and 400 μL of ethanol. The final elution volume for all samples was 125 μL of buffer AE preheated to 70˚C. In addition, elutions were treated with the OneStep™ PCR Inhibitor Removal Kit (Zymo Research) to remove potential PCR inhibitors that may be present. Elutions were stored at -20˚C until qPCR.


## qPCR assay design

Candidate species-specific qPCR assays that included forward and reverse primers and an intervening hydrolysis probe were designed using Integrated DNA Technologies'(IDT) Pri-merQuest tool [32] available online at https://www.idtdna.com/PrimerQuest/Home/Index. The default settings for qPCR assay design were used except for modifying the optimal primer temperature (60˚C), optimal probe temperature (70˚C), and amplicon length (200 bp max). CO1 sequences available on GenBank for C. speleocoopi (accession nos. DQ411780-DQ411781; [33]) as well as other crayfish species that may occur in the study area were aligned using MUSCLE [34] in the program Jalview [35]. A consensus sequence from this alignment for C. speleocoopi was used as input into PrimerQuest.


## In silico and in vitro assay validation

We examined specificity of candidate assays both in silico and in vitro. For in silico validation, we used NCBI Primer-BLAST to query forward and reverse primers and probes against the nr/nt database [36]. Candidate assays then were synthesized by IDT; the internal probe was a PrimeTime1 double-quenched ZEN™/IOWA Black™ FQ probe labeled with 6-FAM and validated in vitro. We tested specificity in vitro by qPCR on a 678-bp gBlock gene fragment synthesized by IDT based on GenBank accession no. DQ411780 for C. speleocoopi and tissue-derived genomic DNA for C. speleocoopi and other crayfish species in the family Cambaridae, that may potentially occur in caves and springs in the study area, including C. hamulatus, C. jonesi, C. laconensis, C. pecki, C. tenebrosus, and Orconectes australis. We evaluated performance of our candidate assays after optimizing primer-probe concentrations and qPCR reagents and settings for our field samples (see below).


## Primer-probe concentration optimization

We optimized concentrations of primers and probe by screening varying concentrations of forward and reverse primers (600 nM, 900 nM, and 1200 nM) and probe (125 nM, 250 nM, and 500 nM) in triplicate with 0.5 ng of target-species synthetic DNA per 20-μL reaction (see next section). Primer-probe concentrations with the greatest peak fluorescence and lower Ct values were employed in screening field-collected water samples.


## Field eDNA sample screening

Real-time qPCR amplification of each field-collected water sample was conducted with six replicates in a final volume of 20 μL, using 10.0 μL of TaqMan1 Environmental Master Mix 2.0 (Applied Biosystems), 4.7 μL of ddH20, 0.9 μL of forward primer (20 μM), 0.9 μL of reverse primer (20 μM), 0.5 μL of probe (20 μM), and 3.0 μL of template DNA. Final concentrations of primers and probe in each 20-μL reaction were 900 and 250 nM, respectively. Samples were run in 96-well optical plates on a QuantStudio1 3 Real-Time PCR System (Applied Biosystems) under the following conditions: an initial 10-min incubation at 95˚C to activate the AmpliTaq Gold1 enzyme followed by 50 cycles of denaturation at 95˚C for 15 s and annealing/extension at 60˚C for 1 min. A dilution series of C. speleocoopi synthesized gBlock was used as a positive control standard, ranging from 10 −1 to 10 −9 ng/μL in concentration. This standard also was used to determine the limit of detection (LOD)-the concentration in which there was at least one positive amplification among technical replicates, and limit of quantification (LOQ)-the concentration in which all technical replicates amplified. Negative controls with all PCR reagents but no template (six replicates) were included on each plate to assess potential contamination. We also screened negative filter controls. Positive amplifications were purified using ExoSAP-IT and specificity confirmed via Sanger sequencing at Eurofins Genomics (Louisville, Kentucky, USA).


## Contamination precautions

False positives can result from contamination during every step in the sampling and quantification pipeline from field collection and filtering of water samples to DNA extraction and qPCR amplification. To minimize the potential for contamination, we employed several procedures in addition to those already outlined. Prior to water sample collection in the field, all bottles and collection supplies were sterilized with 10% bleach solution and autoclaved. Filtering occurred in a lab space dedicated for such purposes. All eDNA filter extractions and qPCR preparations were conducted in dedicated laminar flow hoods. Lab space surfaces and equipment (e.g., pipettes, forceps, tubes, and other consumables etc.) were decontaminated before and after use with 10% bleach solution and/or 30-min ultraviolet light (UV) treatment. Filtered tips were used during all protocols that required pipetting. Finally, disposable gloves were worn during field collection, filtering, DNA extraction, and qPCR setup and cleanup. We included negative controls during the filtering, DNA extracting, and qPCR amplification stages to check for potential contamination.


## Results


## Assay design and validation

The species-specific qPCR assay developed for C. speleocoopi targeted a 163-bp fragment of the mitochondrial CO1 locus ( Table 2). In silico assay validation demonstrated that this assay was not likely to amplify non-target crayfish species, particularly other cave-dwelling crayfishes in the genus Cambarus that also occur in the Tennessee River Valley of northern Alabama-C. In vitro validation also showed that the designed assay was specific to C. speleocoopi with a LOD of 1-2 copies/μL and LOQ of 13.5 copies/ μL (R 2 = 0.995).


## Field surveys

We detected C. speleocoopi eDNA at 5 of 16 sites sampled, including two historical sites-Beech Spring Cave at the spring (site 3) and Cherry Hollow Cave (both in the cave and at the spring; sites 6 and 7). We also detected C.  16) in Jackson County. These sites are far removed from the range of C. speleocoopi, but populations of the closely related C. hamulatus occur in these two caves. We found no evidence of contamination in either our field or laboratory controls across qPCR runs. All positive detections were confirmed by Sanger sequencing.


## Discussion

The use of eDNA as an effective approach in the detection and monitoring of groundwater fauna is still in its infancy. However, our study contributes to a growing literature that indicates eDNA is a viable monitoring tool for occupancy of a fauna that is otherwise difficult to survey and study using traditional approaches. To date, single-species eDNA assay approaches have been applied successfully in the detection of two groundwater salamanders-Olm (Proteus anguinus) from caves, springs, and wells in Bosnia and Herzegovina, Croatia, Montenegro, and Slovenia [17][18][19] and Barton Springs Salamander (Eurcyea sosorum) from a spring in the Edwards Aquifer region of Texas, USA [20], four groundwater fishes-Mexican Blindcat (Prietella phreatophila) from a cave in the Edwards Aquifer region of Texas, USA [20], Blind Cave Eel (Ophisternon candidum) from boreholes in northwestern Australia [22], and Ozark Cavefish (Troglichthys rosae) and Eigenmann's Cavefish (Typhlichthys eigenmanni) from caves, springs, and wells in the Ozarks region of Arkansas, Missouri, and Oklahoma, USA [21], two amphipods-Hay's Spring Amphipod (Stygobromus hayi) and Potomac Groundwater Amphipod (S. tenuis potomacus) from hypotelminorheic springs in the District of Columbia, USA [16], and two crayfishes-Oklahoma Cave Crayfish (Cambarus tartarus) from caves in the Ozarks region of Oklahoma, USA [21] and Caney Mountain Cave Crayfish (Oroconectes stygocaneyi) from a cave in the Ozarks region of Missouri, USA [21,23]. This study is the first to successfully detect eDNA of crayfishes from groundwater of the Interior Plateau karst region and demonstrates that eDNA can detect groundwater crustaceans in karst groundwater habitats, such as springs and cave streams. We designed a species-specific assay that was lab and field validated to discriminate C. speleocoopi from several closely related, morphologically cryptic congeners. Our assay detected C. speleocoopi at two historical sites where occupancy has been confirmed by visual surveys within the last five years. However, we also detected C. speleocoopi eDNA at three potentially new sites in Marshall County, suggesting that an eDNA approach is a rapid and cost-effective method to detect rare subterranean fauna of conservation concern inhabiting karst groundwater. The three positive detections of C. speleocoopi eDNA from Cushion, Davis, and McGehee springs (sites 8, 9, and 12, respectively) appear reasonable given the close proximity to historical sites north of the Tennessee River on the escarpments of Grassy Mountain (Fig 2). Cushion Spring is located 1. Creek drainage south of the Tennessee River did not detect C. speleocoopi eDNA, despite being just 1.2 km west of Cherry Hollow Cave (site 6). Cambarus speleocoopi has never been observed at Kings Spring Cave, including several recent surveys since 2018. Hydrological connectivity between these cave and spring sites is not well understood, as dye tracing investigations have yet to be conducted for these particular karst cave systems.

It is possible that C. speleocoopi is present but rare (i.e., in low abundance) at non-detection sites in Marshall County, but our assay was not sensitive enough to detect extremely low abundance. Conducting eDNA studies on groundwater crayfishes (and other groundwater fauna) in an occupancy-modelling framework [37,38] would benefit future research efforts [e.g., 19]. In addition, we did not assess the effects of water level/flow on detectability. Water samples were collected primarily in late winter and early spring when local water tables were higher, as some spring sites have little to no flow in summer and autumn. However, we only sampled when water conditions were considered normal for the time of the year, and avoided collecting water samples after recent heavy rainfall events that resulted in higher water levels in caves, increased discharge from karst springs, and increased sediment transport, as water volume, flow, and sediment load all can influence eDNA detection [39][40][41][42]. Regardless, eDNA concentrations may vary seasonally in caves and springs due to variation in environmental factors, as observed in other aquatic systems [e.g., 43] and suggested for cave systems [21], such as water volume, flow rate, concentrations of potential inhibitors, etc., as well as biological factors, such as population dynamics and seasonality of reproduction. For example, seasonal variation in crayfish life history and behavior may influence detectability using eDNA [44]; however, little is known about the life history of C. speleocoopi. These and other factors can affect the production rate of eDNA. We encourage future researchers to examine seasonal and annual variation in eDNA concentration and detectability.

Several abiotic and biotic factors affect the degradation and persistence time of eDNA in water, such as UV radiation, temperature, pH, and microbial activity [45][46][47][48][49][50][51][52]. A wide range of degradation rates of eDNA have been reported in the literature for aquatic species in surface ecosystems [12,48]; however, estimates from groundwater habitats are unknown. In general, degradation rates are greater at warmer temperatures [46,48], higher UV radiation exposure [45,46,50, but see 51], and higher levels of microbial activity [45,46,49,52]. Groundwater ecosystems are characterized by lack of light (and UV radiation), cooler and more stable temperatures, lower nutritional resources, and lower microbial activity compared to most surface aquatic ecosystems [53,54]. Consequently, eDNA is thought to be comparatively stable in groundwater habitats and capable of being detected [18]. However, the persistence time of eDNA in groundwater is unknown but likely is considerably longer than the timescale of hours to days reported for many surface species and habitats. Macro-organismal eDNA has been recovered from dry cave sediments dating back thousands to hundreds of thousands of years before present [e.g., [55][56][57], demonstrating the potential for subterranean habitats to act as incubators for eDNA.

Determining degradation rates and persistence time of eDNA in various groundwater habitats should be priority of future research, as inaccurate estimates may have implications for conservation and management. Many groundwater and cave-dwelling species have small, restricted distributions (i.e., short-range endemics) and are of conservation concern [58, 59]. An underlying assumption of eDNA approaches for detection and monitoring is that a positive detection represents contemporary occupancy. However, if eDNA persists for weeks to months or even years in groundwater habitats, then we could overestimate the distribution of a species-e.g., a "false positive" in which eDNA is detected at a site where a population has long been extirpated-or fail to detect declines in range size when eDNA is employed as the sole approach in a monitoring program.

The use of eDNA in the detection and monitoring of groundwater fauna is still in its early stages. Here we report the first application of eDNA to detect a stygobitic crayfish in karst groundwater habitats of the Interior Plateau karst region and provide a demonstration for how eDNA can be applied in groundwater biodiversity monitoring of rare and endangered taxa. Our study along with the encouraging results from the few other recent studies to date have shown that eDNA can be a valid and effective complement to traditional sampling approaches for determining occupancy of groundwater species. We envision a quick transition from proof-of-concept studies to experimental approaches examining rates of degradation and persistence in groundwater to development of best practices for long-term monitoring programs of groundwater fauna. 


hamulatus, C. jonesi, C. laconensis, C. pecki, and C. tenebrosus. The forward primer had one mismatch with published sequences of C. jonesi (GenBank accession nos. DQ411777-DQ411779), two mismatches with C. laconensis (accession no. DQ411782) and C. pecki (accession no. JX514434), 1-2 mismatches with C. hamulatus (accession nos. DQ411760-DQ411776) and C. tenebrosus (accession no. EU583576 and JX514444), and 1-3 mismatches with O. australis (accession nos. EF207161-EF207162, EU583506-EU583551, EU583577-EU583580, EU583583-EU583604, EU583607-EU583620, EU583622-EU583626, EU583628). The reverse primer had 2-3 mismatches with published sequences of C. hamulatus, C. laconensis, C. pecki, C. tenebrosus, and O. australis, and one mismatch with published sequences of C. jonesi. Finally, the probe had 4-5 mismatches with published sequences of C. hamulatus, C. jonesi, C. laconensis, C. pecki, and C. tenebrosus, and 3-5 mismatches with published sequences of O. australis.


2 km northwest of Porches Spring Cave (site 18) on the south side of Grassy Mountain. McGehee Spring is located just 55 m north and is the main resurgence of the stream in Keller's Cave (site 17), which is the type locality of C. speleocoopi located on the north side of Grassy Mountain. Davis Spring is located 1.9 km east of Keller's Cave. All of these springs and caves drain into the Paint Rock River. However, eDNA samples from Kings Spring Cave (site 11) located on the south side of Kings Hollow on Brindley Mountain in the Little Cane

## 58 .
58Zagmajster M, Eme D, Fišer C, Galassi D, Marmonier P, Stoch F, et al. Geographic variation in range size and beta diversity of groundwater crustaceans: insights from habitats with low thermal seasonality. Glob Ecol Biogeogr. 2014; 23: 1135-1145. 59. Eme D, Malard F, Colson-Proch C, Jean P, Calvignac S, Konecny-Dupré L, et al. Integrating phylogeography, physiology and habitat modelling to explore species range determinants. J Biogeogr. 2014; 41: 687-699.

## Table 1 .
1Sampling sites where water samples were collected in Jackson and Marshall counties, Alabama, USA and results of screening of an eDNA assay for Cambarus speleocoopi.Historical sites for C. speleocoopi. b Sites used for negative field controls. Two to four water samples were collected from a site, each with six PCR technical replicates. Sites 17 and 18 are historical sites but we could not gain permission to sample.https://doi.org/10.1371/journal.pone.0242741.t001Site no. 
County 
Site name 
Collection date 
Technical replicates 

1 
Marshall 
Ashburn Spring 
11 Jan 2019 
0/12 

2 
Marshall 
Babe Wright Spring 
07 Feb 2019 
0/24 

3 
Marshall 
Beech Spring Cave (spring) a 
29 Apr 2019 
6/12 

4 
Marshall 
Cathedral Caverns 
07 Feb 2019 
0/12 

5 
Marshall 
Cathedral Caverns (spring) 
07 Feb 2019 
0/12 

6 
Marshall 
Cherry Hollow Cave a 
18 Jan 2019 
5/12 

7 
Marshall 
Cherry Hollow Cave (spring) a 
18 Jan 2019 
3/12 

8 
Marshall 
Cushion Spring 
11 Jan 2019 
2/12 

9 
Marshall 
Davis Spring 
11 Jan 2019 
1/18 

10 
Marshall 
Guffey Cave 
22 Feb 2019 
0/12 

11 
Marshall 
Kings Spring Cave 
01 Feb 2019 
0/18 

12 
Marshall 
McGehee Spring 
11 Jan 2019 
1/12 

13 
Marshall 
New Hope Spring 
31 Jan 2019 
0/12 

14 
Jackson 
Bluff River Cave b 
18 Aug 2018 
0/12 

15 
Jackson 
Tumbling Rock Cave b 
04 Mar 2018 
0/12 

16 
Jackson 
Tumbling Rock Cave (spring) b 
04 Mar 2018 
0/12 

17 
Marshall 
Keller's Cave a 

18 
Marshall 
Porches Spring Cave a 

a 




Fig 2. Distribution of the Sweet Home Alabama Cave Crayfish (Cambarus speleocoopi) (blue dots) and eDNA sampling sites (white dots and numbered blue dots) in northern Alabama, USA.Cambarus speleocoopi eDNA was detected (black triangle) at two historical sites and three new sites. Site numbers correspond to those listed inTable 1. Sites 17 and 18 are historical sites but we could not gain permission to sample. Karst and cave-bearing carbonate strata is shown in gray.https://doi.org/10.1371/journal.pone.0242741.g002


speleocoopi at three additional and potentially new sites in Marshall County-Cushion Spring (site 8), McGehee Spring (site 12), and Davis Spring (site 9). All three of these sites were within 2 km of historical sites for the species.Positive detections 
from Cherry Hollow Cave (in cave and at the spring) were identical to the corresponding 
region of a CO1 haplotype from the cave sampled previously (accession no. D411780). Positive 
detections from Beech Spring Cave, Cushion Spring, McGehee Spring, and Davis Spring 
matched a CO1 haplotype from Keller's and Porches Spring Cave (accession no. D411781). 
We did not detect C. speleocoopi eDNA at Bluff River Cave (site 14) and Tumbling Rock Cave 
(site 15) and associated spring (site 

## Table 2 .
2Primers and probe developed and used in the current study to amplify a 163-bp fragment of CO1 for Cambarus speleocoopi.Oligo 
Sequence (5' to 3') 
Direction 
Length (bp) 
Tm (˚C) 

Forward 
TGGGATAGTTGGGACTTCA 
Sense 
19 
60 

Reverse 
ATTRCCAAACCCTCCAATTA 
Antisense 
20 
60 

Probe 
TCCGAGTTGAATTGGGTCAGGTAGGAAGG 
Sense 
29 
70 

https://doi.org/10.1371/journal.pone.0242741.t002 

PLOS ONE | https://doi.org/10.1371/journal.pone.0242741 December 10, 2020
PLOS ONE | https://doi.org/10.1371/journal.pone.0242741 December 10, 2020 13 / 13
AcknowledgmentsWe thank the many private landowners, Randall Blackwood (Cathedral Caverns State Park), and the Southeastern Cave Conservancy, Inc. for access to springs and caves on their properties. We thank members of the Niemiller Lab at UAH who assisted with field collection of water samples, filtering, and molecular procedures, including Jashen Bailey, Joseph Benito, Kayla Wilson, and Carson Woodward. We also thank Amata Hinkle, and Joe Lamb for assistance in the field.
Frontiers of biogeography: new directions in the geography of nature. M V Lomolino, Lomolino M.V. & Heaney L.R.Sinauer AssociatesSunderland, MassachusettsConservation biogeographyLomolino MV. Conservation biogeography. In Lomolino M.V. & Heaney L.R. (Eds.). Frontiers of bioge- ography: new directions in the geography of nature. Sunderland, Massachusetts: Sinauer Associates; 2004. pp. 293-296.

Conservation of cave fauna in Europe and the Americas. M L Niemiller, E Bichuette, S J Taylor, 10.1002/ar.2404430537183Ecological studies: cave ecology. Moldovan O.T., Kovac L., & Halse S.DordrechtSpringerNiemiller ML, Bichuette E, Taylor SJ. Conservation of cave fauna in Europe and the Americas. In Moldo- van O.T., Kovac L., & Halse S.(Eds.), Ecological studies: cave ecology. Dordrecht: Springer; 2018. pp. 451-478. https://doi.org/10.1002/ar.24044 PMID: 30537183

Scientists' warning on the conservation of subterranean ecosystems. S Mammola, P Cardoso, D C Culver, L Deharveng, R L Ferreira, C Fiser, BioScience. 69Mammola S, Cardoso P, Culver DC, Deharveng L, Ferreira RL, Fiser C, et al. Scientists' warning on the conservation of subterranean ecosystems. BioScience. 2019; 69: 641-650.

The Racovitzan impediment and the hidden biodiversity of unexplored environments. G F Ficetola, C Canedoli, F Stoch, 10.1111/cobi.1317929962096Conservation Biology. 33Ficetola GF, Canedoli C, Stoch F. The Racovitzan impediment and the hidden biodiversity of unex- plored environments. Conservation Biology. 2019; 33: 214-216. https://doi.org/10.1111/cobi.13179 PMID: 29962096

Protecting the innocent: studying short-range endemic taxa enhances conservation outcomes. M S Harvey, M G Rix, V W Framenau, Z R Hamilton, M S Johnson, R J Teale, Invertebr Syst. 25Harvey MS, Rix MG, Framenau VW, Hamilton ZR, Johnson MS, Teale RJ, et al. Protecting the inno- cent: studying short-range endemic taxa enhances conservation outcomes. Invertebr Syst. 2011; 25: 1-10.

Subterranean ecosystems: A truncated functional biodiversity. J Gibert, L Deharveng, BioScience. 52Gibert J, Deharveng L. Subterranean ecosystems: A truncated functional biodiversity. BioScience. 2002; 52: 473-481.

Species detection using environmental DNA from water samples. G F Ficetola, C Miaud, F Pompanon, P Taberlet, 10.1098/rsbl.2008.011818400683Biol Lett. 23Ficetola GF, Miaud C, Pompanon F, Taberlet P. Species detection using environmental DNA from water samples. Biol Lett. 2008; 23: 423-425. https://doi.org/10.1098/rsbl.2008.0118 PMID: 18400683

Moving environmental DNA methods from concept to practice for monitoring aquatic macroorganisms. C S Goldberg, K M Strickler, D S Pilliod, Biol Conserv. 183Goldberg CS, Strickler KM, Pilliod DS. Moving environmental DNA methods from concept to practice for monitoring aquatic macroorganisms. Biol Conserv. 2015; 183: 1-3.

Environmental DNA-an emerging tool in conservation for monitoring past and present biodiversity. P F Thomsen, E Willerslev, Biol Conserv. 183Thomsen PF, Willerslev E. Environmental DNA-an emerging tool in conservation for monitoring past and present biodiversity. Biol Conserv. 2015; 183: 4-18.

The ecology of environmental DNA and implications for conservation genetics. M A Barnes, C R Turner, Conserv Genet. 17Barnes MA, Turner CR. The ecology of environmental DNA and implications for conservation genetics. Conserv Genet. 2016; 17: 1-17.

Environmental DNA for wildlife biology and biodiversity monitoring. K Bohmann, A Evans, Mtp Gilbert, G R Carvalho, S Creer, M Knapp, 10.1016/j.tree.2014.04.00324821515Trends Ecol Evol. 29Bohmann K, Evans A, Gilbert MTP, Carvalho GR, Creer S, Knapp M, et al. Environmental DNA for wild- life biology and biodiversity monitoring. Trends Ecol Evol. 2014; 29: 358-367. https://doi.org/10.1016/j. tree.2014.04.003 PMID: 24821515

Applications of environmental DNA (eDNA) in ecology and conservation: opportunities, challenges and prospects. K C Beng, R T Corlett, Biodiversity and Conservation. 29Beng KC, Corlett RT. Applications of environmental DNA (eDNA) in ecology and conservation: opportu- nities, challenges and prospects. Biodiversity and Conservation. 2020; 29: 2089-2121.

Sight unseen" detection of rare aquatic species using environmental DNA. C L Jerde, A R Mahon, W L Chadderton, D M Lodge, Conserv Lett. 4Jerde CL, Mahon AR, Chadderton WL, Lodge DM. "Sight unseen" detection of rare aquatic species using environmental DNA. Conserv Lett. 2011; 4: 150-157.

Monitoring endangered freshwater biodiversity using environmental DNA. P F Thomsen, J Kielgast, L L Iversen, C Wiuf, M Rasmussen, Mtp Gilbert, 10.1111/j.1365-294X.2011.05418.xMol Ecol. 2122151771Thomsen PF, Kielgast J, Iversen LL, Wiuf C, Rasmussen M, Gilbert MTP, et al. Monitoring endangered freshwater biodiversity using environmental DNA. Mol Ecol. 2012; 21: 2565-2573. https://doi.org/10. 1111/j.1365-294X.2011.05418.x PMID: 22151771

Characterizing the distribution of an endangered salmonid using environmental DNA analysis. M B Laramie, D S Pilliod, C S Goldberg, Biol Conserv. 183Laramie MB, Pilliod DS, Goldberg CS. Characterizing the distribution of an endangered salmonid using environmental DNA analysis. Biol Conserv. 2015; 183: 29-37.

Evaluation of eDNA for groundwater invertebrate detection and monitoring: a case study with endangered Stygobromus (Amphipoda: Crangonyctidae). M L Niemiller, M L Porter, J Keany, H Gilbert, D W Fong, D C Culver, Conserv Genet Resour. 10Niemiller ML, Porter ML, Keany J, Gilbert H, Fong DW, Culver DC, et al. Evaluation of eDNA for ground- water invertebrate detection and monitoring: a case study with endangered Stygobromus (Amphipoda: Crangonyctidae). Conserv Genet Resour. 2018; 10: 247-257.

Searching for black Proteus with the help of eDNA. Š Gorički, D Stanković, M Aljančič, A Snoj, M Kuntner, T Gredar, Nat Sloveniae. 18Gorički Š , Stanković D, Aljančič M, Snoj A, Kuntner M, Gredar T, et al. Searching for black Proteus with the help of eDNA. Nat Sloveniae. 2016; 18: 57-58.

Environmental DNA in subterranean biology: range extension and taxonomic implications for. Š Gorički, D Stanković, A Snoj, M Kuntner, W R Jeffery, P Trontelj, 10.1038/srep4505428345609Proteus. Sci Rep. 7Gorički Š , Stanković D, Snoj A, Kuntner M, Jeffery WR, Trontelj P, et al. Environmental DNA in subterra- nean biology: range extension and taxonomic implications for Proteus. Sci Rep. 2017; 7: 45054. https:// doi.org/10.1038/srep45054 PMID: 28345609

Surveying Europe's only cave-dwelling chordate species (Proteus anguinus) using environmental DNA. J Vö Rös, O Marton, B R Schmidt, J T Gal, D Jelic, 10.1371/journal.pone.017094528129383PLoS ONE. 12170945Vö rös J, Marton O, Schmidt BR, Gal JT, Jelic D. Surveying Europe's only cave-dwelling chordate spe- cies (Proteus anguinus) using environmental DNA. PLoS ONE. 2017; 12: e0170945. https://doi.org/10. 1371/journal.pone.0170945 PMID: 28129383

Exploring the distribution of groundwater salamanders and catfish with environmental DNA. K M Lyons, 32Austin, TexasThe University of Texas at AustinM.Sc. Thesis.Lyons KM. Exploring the distribution of groundwater salamanders and catfish with environmental DNA. M.Sc. Thesis. The University of Texas at Austin, Austin, Texas. 2019, pp. 32.

Examining occurrence, life history, and ecology of cavefishes and cave crayfishes using both traditional and novel approaches. J Mouser, 122Stillwater, OklahomaOklahoma State UniversityM.Sc. Thesis.Mouser J. Examining occurrence, life history, and ecology of cavefishes and cave crayfishes using both traditional and novel approaches. M.Sc. Thesis. Oklahoma State University, Stillwater, Oklahoma. 2019, pp. 122.

Detection of rare Australian endemic blind cave eel (Ophisternon candidum) with environmental DNA: implications for threatened species management in subterranean environments. N E White, M T Guzik, A D Austin, G I Moore, W F Humphreys, Alexander J , 10.1007/s10750-020-04304-zHydrobiologica. White NE, Guzik MT, Austin AD, Moore GI Humphreys WF, Alexander J, et al. Detection of rare Austra- lian endemic blind cave eel (Ophisternon candidum) with environmental DNA: implications for threat- ened species management in subterranean environments. Hydrobiologica. 2020. https://doi.org/10. 1007/s10750-020-04304-z

Preliminary investigation of the critically imperiled Caney Mountain Cave Crayfish Orconectes stygocaneyi Hobbs III, 2001 (Decapoda: Cambaridae) in Missouri. R J Distefano, D Ashley, S K Brewer, J B Mouser, M L Niemiller, USA. Freshwater Crayfish. 25DiStefano RJ, Ashley D, Brewer SK, Mouser JB, Niemiller ML. Preliminary investigation of the critically imperiled Caney Mountain Cave Crayfish Orconectes stygocaneyi Hobbs III, 2001 (Decapoda: Cam- baridae) in Missouri, USA. Freshwater Crayfish. 2020; 25: 47-57.

Wells provide a distorted view of life in the aquifer: implications for sampling, monitoring and assessment of groundwater ecosystems. K Korbel, A Chariton, S Stephenson, P Greenfield, G C Hose, 10.1038/srep4070228102290Sci Rep. 740702Korbel K, Chariton A, Stephenson S, Greenfield P, Hose GC. Wells provide a distorted view of life in the aquifer: implications for sampling, monitoring and assessment of groundwater ecosystems. Sci Rep. 2017; 7: 40702. https://doi.org/10.1038/srep40702 PMID: 28102290

Taxonomic revision of cave crayfish in the genus Cambarus, subgenus Aviticambarus (Decapoda: Cambaridae) with descriptions of two new species, C. speleocoopi and C. laconensis, endemic to Alabama. J E Buhay, K A Crandall, A. J. Crustacean Biology. 29Buhay JE, Crandall KA. Taxonomic revision of cave crayfish in the genus Cambarus, subgenus Aviti- cambarus (Decapoda: Cambaridae) with descriptions of two new species, C. speleocoopi and C. laco- nensis, endemic to Alabama, U.S.A. J. Crustacean Biology. 2009; 29: 121-134.

Sweet Home Alabama Cave Crayfish Cambarus speleocoopi Buhay and Crandall. A Huryn, Shelton-Nix E, editor. Alabama WildlifeThe University of Alabama Press113Tuscaloosa, AlabamaHuryn A. Sweet Home Alabama Cave Crayfish Cambarus speleocoopi Buhay and Crandall. In: Shel- ton-Nix E, editor. Alabama Wildlife. Tuscaloosa, Alabama: The University of Alabama Press; 2017. p. 113.

. Natureserve, 28NatureServe. 2020 [cited 28 May 2020].

. NatureServe Explorer. web applicationNatureServe Explorer [web application].

. Arlington Natureserve, Virginia , NatureServe, Arling- ton, Virginia. Available from: https://explorer.natureserve.org/.

. J Buhay, K A Crandall, J Cordeiro, 28Buhay J, Crandall KA, Cordeiro J. 2010 [cited 28 May 2020].

Cambarus Speleocoopi, 10.2305/IUCN.UK.2010-3.RLTS.T164915A5938003.enThe IUCN Red List of Threatened Species 2010: e.T164915A5938003. Available from. Cambarus speleocoopi. The IUCN Red List of Threatened Species 2010: e.T164915A5938003. Available from: https://dx.doi.org/10.2305/ IUCN.UK.2010-3.RLTS.T164915A5938003.en.

Patterns of cave biodiversity and endemism in the Appalachians and Interior Plateau of Tennessee, USA. M L Niemiller, K S Ziger, 10.1371/journal.pone.006417723717562PLoS One. 8Niemiller ML, Ziger KS. Patterns of cave biodiversity and endemism in the Appalachians and Interior Plateau of Tennessee, USA. PLoS One. 2013; 8, e64177. https://doi.org/10.1371/journal.pone. 0064177 PMID: 23717562

Biodiversity from caves and other subterranean habitats of Georgia, USA. K S Zigler, M L Niemiller, Cdr Stephen, B N Ayala, M A Milne, N S Gladstone, J Cave Karst Stud. 82Zigler KS, Niemiller ML, Stephen CDR, Ayala BN, Milne MA, Gladstone NS, et al. Biodiversity from caves and other subterranean habitats of Georgia, USA. J Cave Karst Stud. 2020; 82, 125-167.

A review of the troglobitic crustaceans of the Americas. Smithson. I I Hobbs, Hh, Iii Hobbs, Hh, M A Daniel, Contrib Zool. 244Hobbs II HH, Hobbs III HH, Daniel MA. A review of the troglobitic crustaceans of the Americas. Smith- son Contrib Zool. 1977; 244: 1-183.

IDT SciTools: a suite for analysis and design of nucleic acid oligomers. R Owczarzy, A V Tataurov, Y Wu, J A Manthey, K A Mcquisten, H G Almabrazi, 10.1093/nar/gkn19818440976Nucleic Acids Res. 36Owczarzy R, Tataurov AV, Wu Y, Manthey JA, McQuisten KA, Almabrazi HG, et al. IDT SciTools: a suite for analysis and design of nucleic acid oligomers. Nucleic Acids Res. 2008; 36: W163-W169. https://doi.org/10.1093/nar/gkn198 PMID: 18440976

Molecular taxonomy in the dark: Evolutionary history, phylogeography, and diversity of the cave crayfish in the subgenus Aviticambarus, genus Cambarus. M E Buhay, G Moni, N Mann, K A Crandall, 10.1016/j.ympev.2006.07.01416971141Mol Phylogenet Evol. 42Buhay ME, Moni G, Mann N, Crandall KA. Molecular taxonomy in the dark: Evolutionary history, phylo- geography, and diversity of the cave crayfish in the subgenus Aviticambarus, genus Cambarus. Mol Phylogenet Evol. 2007; 42: 435-448. https://doi.org/10.1016/j.ympev.2006.07.014 PMID: 16971141

MUSCLE: multiple sequence alignment with high accuracy and high throughput. R C Edgar, 10.1093/nar/gkh34015034147Nucleic Acids Res. 32Edgar RC. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 2004; 32: 1792-1797. https://doi.org/10.1093/nar/gkh340 PMID: 15034147

Jalview Version 2-a multiple sequence alignment editor and analysis workbench. A M Waterhouse, J B Procter, Dma Martin, M Clamp, G J Barton, 10.1093/bioinformatics/btp03319151095Bioinformatics. 25Waterhouse AM, Procter JB, Martin DMA, Clamp M, Barton GJ. Jalview Version 2-a multiple sequence alignment editor and analysis workbench. Bioinformatics. 2009; 25: 1189-1191. https://doi. org/10.1093/bioinformatics/btp033 PMID: 19151095

Primer-BLAST: A tool to design target-specific primers for polymerase chain reaction. J Ye, G Coulouris, I Zaretskaya, I Cutcutache, S Rozen, T L Madden, 10.1186/1471-2105-13-13422708584BMC Bioinformatics. 13134Ye J, Coulouris G, Zaretskaya I, Cutcutache I, Rozen S, Madden TL. Primer-BLAST: A tool to design target-specific primers for polymerase chain reaction. BMC Bioinformatics. 2012; 13: 134. https://doi. org/10.1186/1471-2105-13-134 PMID: 22708584

Site occupancy models in the analysis of environmental DNA presence/absence surveys: a case study of an emerging amphibian pathogen. B R Schmidt, M Kery, S Ursenbacher, O J Hyman, J P Collins, Meth Ecol Evol. 4Schmidt BR, Kery M, Ursenbacher S, Hyman OJ, Collins JP. Site occupancy models in the analysis of environmental DNA presence/absence surveys: a case study of an emerging amphibian pathogen. Meth Ecol Evol. 2013; 4: 646-653.

An r package for multiscale occupancy modelling of environmental DNA data. R M Dorazio, R A Erickson, Mol Ecol Res. 18Dorazio RM, Erickson RA. An r package for multiscale occupancy modelling of environmental DNA data. Mol Ecol Res. 2018; 18, 368-380.

Fish environmental DNA is more concentrated in aquatic sediments than surface water. C R Turner, K L Uy, R C Everhart, Biol Conserv. 183Turner CR, Uy KL, Everhart RC. Fish environmental DNA is more concentrated in aquatic sediments than surface water. Biol Conserv. 2015; 183: 93-102.

Distance, flow and PCR inhibition: eDNA dynamics in two headwater streams. S F Jane, T M Wilcox, K S Mckelvey, M K Young, M K Schwartz, W H Lowe, 10.1111/1755-0998.1228524890199Mol Ecol Resour. 15Jane SF, Wilcox TM, McKelvey KS, Young MK, Schwartz MK, Lowe WH, et al. Distance, flow and PCR inhibition: eDNA dynamics in two headwater streams. Mol Ecol Resour. 2015; 15: 216-227. https://doi. org/10.1111/1755-0998.12285 PMID: 24890199

Influence of stream bottom substrate on retention and transport of vertebrate environmental DNA. C L Jerde, B P Olds, A J Shogren, E A Andruszkiewicz, A R Mahon, D Bolster, 10.1021/acs.est.6b0176127409250Environ Sci Technol. 50Jerde CL, Olds BP, Shogren AJ, Andruszkiewicz EA, Mahon AR, Bolster D, et al. Influence of stream bottom substrate on retention and transport of vertebrate environmental DNA. Environ Sci Technol. 2016; 50: 8770-8779. https://doi.org/10.1021/acs.est.6b01761 PMID: 27409250

A systematic approach to evaluate the influence of environmental conditions on eDNA detection success in aquatic ecosystems. B C Stoeckle, S Beggel, A F Cerwenka, E Motivans, J Geist, 10.1371/journal.pone.018911929220394PLoS One. 12189119Stoeckle BC, Beggel S, Cerwenka AF, Motivans E, Geist J. A systematic approach to evaluate the influ- ence of environmental conditions on eDNA detection success in aquatic ecosystems. PLoS One. 2017; 12: e0189119. https://doi.org/10.1371/journal.pone.0189119 PMID: 29220394

Seasonal variation in environmental DNA in relation to population size and environmental factors. A S Buxton, J J Groombridge, N B Zakaria, R A Griffiths, 10.1038/srep4629428393885Sci Rep. 746294Buxton AS, Groombridge JJ, Zakaria NB, Griffiths RA. Seasonal variation in environmental DNA in rela- tion to population size and environmental factors. Sci Rep. 2017; 7: 46294. https://doi.org/10.1038/ srep46294 PMID: 28393885

Behavior and season affect crayfish detection and density inference using environmental DNA. N Dunn, V Priestley, A Herraiz, R Arnold, V Savolainen, 10.1002/ece3.331629043033Ecol Evol. 7Dunn N, Priestley V, Herraiz A, Arnold R, Savolainen V. Behavior and season affect crayfish detection and density inference using environmental DNA. Ecol Evol. 2017; 7: 7777-7785. https://doi.org/10. 1002/ece3.3316 PMID: 29043033

Factors influencing detection of eDNA from a streamdwelling amphibian. D S Pilliod, C S Goldberg, R S Arkle, L P Waits, 10.1111/1755-0998.1215924034561Mol Ecol Resour. 14Pilliod DS, Goldberg CS, Arkle RS, Waits LP. Factors influencing detection of eDNA from a stream- dwelling amphibian. Mol Ecol Resour. 2014; 14: 109-116. https://doi.org/10.1111/1755-0998.12159 PMID: 24034561

Quantifying effects of UV-B, temperature, and pH on eDNA degradation in aquatic microcosms. K M Strickler, A K Fremier, C S Goldberg, Biol Conserv. 183Strickler K M, Fremier AK, Goldberg CS. Quantifying effects of UV-B, temperature, and pH on eDNA degradation in aquatic microcosms. Biol Conserv. 2015; 183: 85-92.

Degradation and dispersion limit environmental DNA detection of rare amphibians in wetlands: increasing efficacy of sampling designs. C S Goldberg, K M Strickler, A K Fremier, 10.1016/j.scitotenv.2018.02.29529602110Sci Total Environ. 633Goldberg CS, Strickler KM, Fremier AK. Degradation and dispersion limit environmental DNA detection of rare amphibians in wetlands: increasing efficacy of sampling designs. Sci Total Environ. 2018; 633: 695-703. https://doi.org/10.1016/j.scitotenv.2018.02.295 PMID: 29602110

Predicting the fate of eDNA in the environment and implications for studying biodiversity. J B Harrison, J M Sunday, S M Rogers, 10.1098/rspb.2019.140931744434Proceedings of the Royal Society B. 286Harrison JB, Sunday JM, Rogers SM. Predicting the fate of eDNA in the environment and implications for studying biodiversity. Proceedings of the Royal Society B. 2019; 286: 20191409. https://doi.org/10. 1098/rspb.2019.1409 PMID: 31744434

Effects of temperature and trophic state on degradation of environmental DNA in lake water. J J Eichmiller, S E Best, P W Sorensen, 10.1021/acs.est.5b0567226771292Environ Sci Technol. 50Eichmiller JJ, Best SE, Sorensen PW. Effects of temperature and trophic state on degradation of envi- ronmental DNA in lake water. Environ Sci Technol. 2016; 50: 1859-1867. https://doi.org/10.1021/acs. est.5b05672 PMID: 26771292

Persistence of marine fish environmental DNA and the influence of sunlight. E A Andruszkiewicz, L M Sassoubre, A B Boehm, 10.1371/journal.pone.018504328915253PLoS ONE. 12185043Andruszkiewicz EA, Sassoubre LM, Boehm AB. Persistence of marine fish environmental DNA and the influence of sunlight. PLoS ONE. 2017; 12: e0185043. https://doi.org/10.1371/journal.pone.0185043 PMID: 28915253

Shedding light on eDNA: neither natural levels of UV radiation nor the presence of a filter feeder affect eDNA-based detection of aquatic organisms. E Mächler, M Osathanunkul, F Altermatt, 10.1371/journal.pone.019552929624607PLoS ONE. 13195529Mächler E, Osathanunkul M, Altermatt F. Shedding light on eDNA: neither natural levels of UV radiation nor the presence of a filter feeder affect eDNA-based detection of aquatic organisms. PLoS ONE. 2018; 13: e0195529. https://doi.org/10.1371/journal.pone.0195529 PMID: 29624607

Effects of microbial activity and environmental parameters on the degradation of extracellular environmental DNA from a eutrophic lake. N S Zulkefi, K H Kim, S J Hwang, Int J Environ Res Public Health. 163339Zulkefi NS, Kim KH, Hwang SJ. Effects of microbial activity and environmental parameters on the deg- radation of extracellular environmental DNA from a eutrophic lake. Int J Environ Res Public Health. 2019; 16: 3339.

Groundwater Ecology. J Gibert, D Danielopol, J A Stanford, Academic PressGibert J, Danielopol D, Stanford JA. Groundwater Ecology. Academic Press; 1994.

The vertical extent of groundwater metazoans: an ecological and evolutionary perspective. C Fiser, T Pipan, D C Culver, BioScience. 64Fiser C, Pipan T, Culver DC. The vertical extent of groundwater metazoans: an ecological and evolu- tionary perspective. BioScience. 2014; 64: 971-979.

Diverse plant and animal genetic records from Holocene and Pleistocene sediments. E Willerslev, A J Hansen, J Binladen, T B Brand, Mtp Gilbert, B Shapiro, 10.1126/science.108411412702808Science. 300Willerslev E, Hansen AJ, Binladen J, Brand TB, Gilbert MTP, Shapiro B, et al. Diverse plant and animal genetic records from Holocene and Pleistocene sediments. Science. 2003; 300: 791-795. https://doi. org/10.1126/science.1084114 PMID: 12702808

Molecular caving. M Hofreiter, J I Mead, P Martin, H N Poinar, 10.1016/j.cub.2003.08.03913678604Current Biology. 13Hofreiter M, Mead JI, Martin P, Poinar HN. Molecular caving. Current Biology. 2003; 13: R693-R695. https://doi.org/10.1016/j.cub.2003.08.039 PMID: 13678604

Ancient DNA chronology within sediment deposits: are paleobiological reconstructions possible and is DNA leaching a factor?. J Haile, R Holdaway, K Oliver, M Bunce, Mtp Gilbert, R Nielsen, 10.1093/molbev/msm01617255121Mol Biol Evol. 24Haile J, Holdaway R, Oliver K, Bunce M, Gilbert MTP, Nielsen R, et al. Ancient DNA chronology within sediment deposits: are paleobiological reconstructions possible and is DNA leaching a factor? Mol Biol Evol. 2007; 24: 982-989. https://doi.org/10.1093/molbev/msm016 PMID: 17255121