# SURVEY AND SUMMARY Regulatory roles of RNA modifications in breast cancer

CorpusID: 237555138
 
tags: #Medicine, #Biology

URL: [https://www.semanticscholar.org/paper/266a902ceb71a7637608844159df34a9f8cfb99b](https://www.semanticscholar.org/paper/266a902ceb71a7637608844159df34a9f8cfb99b)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

SURVEY AND SUMMARY Regulatory roles of RNA modifications in breast cancer
September 2021 NAR Cancer, 2021

Kanchan Kumari 
Department of Molecular Biology
Umeå University
SE-901 85UmeåSweden

Wallenberg Centre for Molecular Medicine
Umeå University
SE-901 85UmeåSweden

Paula Groza 
Department of Molecular Biology
Umeå University
SE-901 85UmeåSweden

Wallenberg Centre for Molecular Medicine
Umeå University
SE-901 85UmeåSweden

Francesca Aguilo 
Department of Molecular Biology
Umeå University
SE-901 85UmeåSweden

Wallenberg Centre for Molecular Medicine
Umeå University
SE-901 85UmeåSweden

SURVEY AND SUMMARY Regulatory roles of RNA modifications in breast cancer
33September 2021 NAR Cancer, 202110.1093/narcan/zcab036Received July 05, 2021; Revised August 07, 2021; Editorial Decision August 11, 2021; Accepted August 25, 2021https://doi.org/10.1093/narcan/zcab036
Collectively referred to as the epitranscriptome, RNA modifications play important roles in gene expression control regulating relevant cellular processes. In the last few decades, growing numbers of RNA modifications have been identified not only in abundant ribosomal (rRNA) and transfer RNA (tRNA) but also in messenger RNA (mRNA). In addition, many writers, erasers and readers that dynamically regulate the chemical marks have also been characterized. Correct deposition of RNA modifications is prerequisite for cellular homeostasis, and its alteration results in aberrant transcriptional programs that dictate human disease, including breast cancer, the most frequent female malignancy, and the leading cause of cancerrelated death in women. In this review, we emphasize the major RNA modifications that are present in tRNA, rRNA and mRNA. We have categorized breast cancer-associated chemical marks and summarize their contribution to breast tumorigenesis. In addition, we describe less abundant tRNA modifications with related pathways implicated in breast cancer. Finally, we discuss current limitations and perspectives on epitranscriptomics for use in therapeutic strategies against breast and other cancers.GRAPHICAL ABSTRACT

## INTRODUCTION

RNA modifications, collectively termed the epitranscriptome, are crucial regulators of temporal and spatial gene expression programs. Currently, over 170 RNA modifications decorating all RNA species and in all three kingdoms of life have been described (1,2). Although modification to RNA has been documented for over 50 years, the functions of most of these modifications are largely unknown. Recent development of more sensitive and specific technologies, such as high-throughput sequencing and improved mass spectrometry, shed light on to the exciting new field of RNA epitranscriptomics.

In general, the fate of modified RNAs is determined by the coordinated actions of writers, erasers and readers that impose, remove and recognize the chemical mark. Some of the writers are stand-alone enzymes, whereas others act as multiprotein writer complexes that also comprise accessory subunits. In addition, some of the RNA modifications are reversible i.e. removed by the erasers, whereas others are irreversible. Most of the known RNA modifications map to abundant RNAs such as transfer RNA (tRNA) and ribosomal RNA (rRNA), tRNA being the most extensively modified RNA type in the cell (3) (Figure 1).

In eukaryotes, the nuclear-encoded tRNAs contain on average 11-13 modifications per molecule whereas mitochondrial tRNAs are modified to a lesser extent, with an average of 5 modifications per molecule (4). The molecular consequence of the modification depends on the type of chemical modification and the location within a tRNA. Hence, modifications occurring at the wobble position (position 34) and to the residue adjacent to the anticodon loop (position 37) are highly conserved and lead to the strongest effects in optimization of codon usage, regulating translational efficiency and fidelity (5)(6)(7). However, modifications along the whole L-shape affect tRNA stability, localization and functional folding (8,9). Such modifications include but are not limited to 5methylcytosine (m 5 C), N 1 -methyladenosine (m 1 A), pseudouridine ( ), 5-methyluridine (m 5 U), 1-methylguanosine and 7-methylguanosine (m 1 G and m 7 G, respectively), and inosine (I), and complex multistep chemical modifications, such as N 6 -threonylcarbamoyladenosine (t 6 A) and 5-methoxycarbonylmethyl-2-thiouridine (mcm5s 2 U) (10) (Figure 2).

Human ribosomes contain four rRNA types, i.e. 28S, 5S and 5.8S rRNAs in the 60S subunit, and 18S rRNA in the 40S subunit. In each subunit, the rRNAs form the core of the translation machinery whereas ribosomal proteins stabilize the structure and fine-tune the function of the ribosome (11,12). During ribosome biogenesis, rRNAs are extensively modified, expanding the topological properties of RNAs and optimizing the ribosome functionality (13). Recently, >130 individual rRNA modifications have been visualized in the three-dimensional structure of the human ribosome, being several of the modifications associated with degenerate states in cancer (14). In eukaryotic ribosomes, the most abundant rRNA modifications are ribose 2 -O-methylation (2 O-Me or Nm) and catalyzed by box C/D and box H/ACA ribonucleoprotein (RNP) enzymes, respectively, using small nucleolar RNAs (snoR-NAs) for the recognition of specific rRNA target sites (15)(16)(17). Conversely, base modifications, such as methylations and acetylations, are catalyzed by conventional protein enzymes, most of which have only recently been identified (18)(19)(20)(21), and thereby, their function is largely unknown (22).

Shortly after the discovery of the 5 cap and 3 polyadenylation, N 6 -methyladenosine (m 6 A) was identified in mRNA (18)(19)(20)(21). m 6 A is the most abundant internal mRNA modification with on average 3-5 adenines methylated per mRNA (23,24). Other less abundant modifications within eukaryotic mRNA include m 1 A, N 6 ,2 -O-dimethyladenosine (m 6 A m ), m 5 C, 5-hydroxymethylcytosine (hm 5 C) and ( Figure 3) (3). These modifications are not randomly distributed in the mRNA, and depending on the modification type and the deposition site, they can virtually affect all aspects of RNA fate including RNA processing, RNA export, mRNA translation and degradation. All these molecular events shape the transcriptome in a spatiotemporal manner to tightly regulate gene expression programs (25).

The deposition of chemical modifications into RNA is dynamic allowing the rapid adaptation to changing envi-ronmental cues and to various stresses. Such adaptation is crucial for cellular homeostasis. Hence, alterations in the expression levels of RNA modifiers and thereby, dysregulated RNA modification pathways, have been linked to tumorigenesis as well as other human diseases (26). In this review, we describe the current understanding on how these epitranscriptomic marks are implicated in breast tumorigenesis, the most prevalent cancer among women worldwide. Specifically, we highlight several critical modifications, namely m 6 A, m 1 A, m 5 C, 2 O-Me, RNA editing and (Table 1). We describe which RNA species they have been identified in, their molecular and cellular functions (where known) and the evidence linking them to the development, maintenance and progression of breast cancer.


## Breast cancer

According to a global cancer statistics study, breast cancer is the most commonly diagnosed cancer in women and among the leading cause of cancer-related death in females (27) (Figure 4). Although there is an evident increase in breast cancer incidence and mortality among pre-and postmenopausal women, females of younger age are also at risk every year, with higher emphasis in developing countries (28). Hence, although advanced therapies and early detection have improved the survival rate (29), the causal factors of breast carcinoma still remain elusive.

The clinical behavior and the treatment outcomes in breast cancer are highly influenced by tumor heterogeneity consisting in increased morphological variability and fluctuating therapy response (30). Based on the expression of the hormone receptors (estrogen (ER) and progesterone (PR)) and the human epidermal growth factor receptor 2 (HER2), there are four molecular subtypes of breast cancer, namely luminal A, luminal B, HER2 positive and basal-like or triple-negative (TNBC) (31). Although both luminal A and B are ER and PR positive, the later one has a worst prognosis and is either HER positive or negative with high levels of the proliferation marker Ki-67 (30,32). The HER2 subtype is ER and PR negative but positive for HER2 and the TNBC presents a triple negative immunophenotype (ER, PR and HER2 negative), increased proliferation rate and the highest incidence of relapse (30,33,34). Hence, in addition to providing prognostic information, the molecular subtypes can be used to evaluate clinical behaviors and response to treatments. However, the intrinsic heterogeneity of breast cancer impedes the full characterization based on the aforementioned histopathologic parameters. Thus, recent evidence supports the presence of multiple subtypes within a tumor (32) and the existence of even more than four subtypes (30), manifesting the complex molecular landscape of breast cancer cells. In addition, it has been shown that the microenvironment can dictate plasticity of breast cancer cells (35). For instance, the mammary stroma can induce basal differentiation in MCF7 cells, a luminal cellular model, and the site of injection of these cells into mice models can confer different tumor phenotypes, with injection in the milk ducts leading to an increased mimicry of the original tumor compared with the injection into the fat pads (35). Therefore, finding an effective cure for this het-  erogeneous and multifactorial disease is still a major challenge. Such complexity and heterogeneity of breast cancer cells can be better understood through integration of multiomics approaches that provide resourceful information of the different layers of gene expression regulation. Although genomic, epigenomic and transcriptomic datasets have added new insights into the true biological landscape of breast cancer, epitranscriptomic analysis are still in their infancy. Given that many of the RNA modifications are dysregulated in human cancers, the epitranscriptome represents a hot-spot of interest in the quest of elucidating the transition from a normal physiological to a pathological state, hoping that its study will enable the development of more efficient and effective therapies against cancer. For breast cancer, recognition of the function of RNA modifications in its development may represent the missing piece of the puzzle in deciphering the complex pathogenesis of these tumors.


## N6-methyladenosine

m 6 A is the most studied and abundant internal modification on eukaryotic mRNA. It also appears on tRNAs and other non-coding RNAs (ncRNAs) and more recently, it has been found on rRNA (2,36,37). Due to its large impact on RNA metabolism at multiple levels i.e. splicing, nuclear export, mRNA stability, translation and even RNA and protein interactions, it plays a critical role not only in proper functioning of all sorts of biological processes but also in acquired pathologies such as cancer (26,38,39). m 6 A is co-transcriptionally deposited on mRNA by a stable methyltransferase complex consisting of two core components, i.e. methyltransferase-like 3 (METTL3) and 14 (METTL14), and other proteins that ensure m 6 A specificity (40,41). The demethylases include fat mass and obesityassociated protein (FTO) and ␣-ketoglutarate dependent dioxygenase alk B homolog 5 (ALKBH5) (42,43). The most representative readers are members of the YTH domaincontaining protein family (YTHDF1/2/3 and YTHDC1) (44,45).

In recent years, the expression of m 6 A regulators has been correlated with hallmarks of cancer and it was shown that breast cancer cells exhibited higher m 6 A methylation levels compared to healthy mammary epithelial cells (Table  1) (46,47). However, most of the studies seem controversial as a defined gene expression pattern of writers, erasers and readers is missing, reflecting the complexity and heterogeneity of breast pathogenesis. For instance the writer METTL14 was downregulated in TNBC, with METTL14 expression being correlated with favorable outcomes (48). Similarly, Gong et al. reported that low expression levels of METTL14 led to poor prognosis and tumor progression in TNBC (49). On the contrary, overexpression of METTL14 was observed in studies that compared breast cancer patients with healthy controls, METTL14 acting as an oncogene that promoted invading and migrating capacities of the cells (46,50). Dong et al. have attributed METTL14 overexpression to luminal A and B subtypes, being negatively correlated with tumor grade and not exerting influence over prognosis (51). In addition, the expression of the erasers of m 6 A has also been shown to be dysregulated in breast cancer patients, being the increased expression of ALKBH5 correlated with poor prognosis in TNBC (47), whereas the expression of the demethylase FTO appeared to be decreased in breast cancer patients compared to normal controls (47). However, Xu et al. observed upregulation of FTO in HER2+ subtype and a strong connection with unfavorable prognosis in this subtype (52). The expression of readers correlated as well with different outcomes. For instance, increased expression of YTHDF3 was linked with unfavorable survival, nodal metastasis and poor prognosis in breast cancer patients (53,54). Other readers such as YTHDF1 and YTHDF1 displayed increased expression in various analyses of breast cancer patient samples (48,54,55), while YTHDC1 and YTHDC2 were shown to be downregulated in TNBC (48). These studies reflect that the same writer protein, for example METTL14, can display distinct expression patters depending on the breast cancer subtype. They also show how the expression of distinct erasers or readers can follow opposite patterns of gene expression. Therefore, dysregulated expression of m 6 A writer, eraser and reader, either high or low, might lead to aberrant gene expression programs that promote breast tumorigenesis. Indeed, it has been shown that the interplay among m 6 A regulators determines the m 6 A levels and, consequently, the stability of several transcripts that are known to play a critical role in cell cycle, epithelial-mesenchymal transition (EMT) and angiogenesis in breast cancer (56).

Several mechanisms involving METTL3 in breast cancer progression have been described ( Figure 5). METTL3 was found to participate in a feedback loop with Hepatitis B virus X-interacting protein (HBXIP) and the mi-croRNA (miRNA) let-7g, in which METTL3 expression was positively regulated by HBXIP through inhibiting let-7g miRNA, while HBXIP expression was increased by METTL3. This positive feedback loop resulted in increased cell proliferation and, ultimately, in cancer progression (57). In addition, METTL3-mediated increased methylation lev-  els promoted the translation of Bcl-2, a major regulator of cell death modulating cellular proliferation and apoptosis of cancerous cells (58). Furthermore, it has been shown that m 6 A can induce breast cancer lung metastasis by increasing the stability of a mRNA duplex formed from the keratin 7 (KRT7) transcript, encoding a regulator of EMT, cytoskeleton programming and cellular transformation, and its antisense long noncoding RNA (lncRNA) KRT7-AS (59). On the contrary, in TNBC cell lines, METTL3 halts metastatic progression by hypermethylation of the Collagen Type III Alpha 1 Chain (COL3A1) transcript which trigger its degradation (60). Recently, it was shown that METTL3 is the host gene of a circRNA, so called cir-cMETTL3 whose expression was regulated via a m 6 Adependent mechanism (61). This circRNA was upregulated and impacted cell proliferation, migration and invasion in breast cancer through upregulation of cyclin-dependent kinases (CDK1), recently identified as a breast cancer prognosis indicator (61). Moreover, METTL14 was reported to enhance cell proliferation by facilitating the deposition of m 6 A on the transcripts of the oncogenes CXCR4 and CYP1B1 (62). Mechanisms showing the implications of FTO in cell proliferation, colony formation and metastasis of breast cancer both in vitro and in vivo were also described. Hence, FTO-mediated demethylation of Bcl-2/adenovirus E1B mRNA (BNIP3), a pro-apoptotic tumor suppressor gene, favored tumor progression (63). Recently, the reader of m 6 A YTHDF2 was found to sustain MYC-driven cell growth and survival in TNBC cell lines by facilitating the turnover of mRNAs belonging to MAPK/ERK signaling pathways. Hence, YTHDF2 was limiting the endoplasmic reticulum stress response and therefore contributed to EMT and breast tumorigenesis (64).

Another way by which the m 6 A machinery can contribute to breast cancer formation and development is through its roles in miRNA biogenesis. METTL3 has been shown to deposit m 6 A on primary miRNAs (pri-miRNAs) enabling their recognition by the RNA binding protein DGCR8 and thereby facilitating miRNAs biogenesis (65). Particularly, in breast cancer, METTL14 reshaped the miRNA profile (50). Modulation of the expression of miR-146a-5p by METTL14 promoted cell migration and invasion underlying the predominant control that m 6 A may have over multiple aspects of breast tumorigenicity. In addition, inhibition of miR-181b-3p by FTO resulted in upregulation of ADP ribosylation factor like GTPase 5B (ARL5B) (52). ARL5B is responsible for promoting lysosome motility facilitating cell migration. Therefore, through this mechanism FTO enhanced the invasive and migratory capabilities of the breast cancer cells (52). Considering the data obtained in studying other cancer types, further studies will most certainly reveal an even higher impact of m 6 A on miRNA in breast carcinogenesis. m 6 A plays a significant role in the formation of cancer stem cells (CSCs), tumorigenic cells with stem cell properties, which are known for their capabilities of facilitating the carcinogenic process and the resistance to therapy (66). Hypoxic environments, through the action of hypoxiainducible factors (HIF), have been linked with EMT and thus with acquisition of stem cell-like properties that favor migration and invasion in breast cancer (67,68). Expression levels of ALKBH5 and the oncogenic factor Zinc-Finger Protein 217 (ZNF217) are increased in these environments (69,70). Notably, the mouse ortholog ZFP217 has been shown to interact with METTL3 and restrict the m 6 A deposition on pluripotency factors (71), increasing their mRNA stability. Thus, ZNF217-and ALKBH5mediated demethylation of pluripotency factors, i.e. Nanog and KLF4, induced the acquisition of stem cell-like properties and therefore, BCSC specification (69,70). IGF2BP2, a reader of m 6 A, was also linked to mechanisms enhancing BCSC by stabilization of the mRNA of two oncogenes, namely DROSHA and c-Myc (72,73). All these results point toward a significant role of this mark in elucidating means that can provide new therapeutic strategies in both drug resistant and refractory hypoxic breast tumors.

Recent evidence indicates that m 6 A also plays important roles in treatment outcomes, being either a vector of acquired resistance or a perfect target for more efficient therapies. On one hand, resistance to tamoxifen and doxorubicin were modulated by METTL3 through methylation of adenylate kinase 4 (AK4) transcripts, and miRNA-221-3p, respectively (74,75). AK4 is a mitochondrial matrix protein involved in energy metabolism homeostasis whose expression is linked to the progression of multiple cancers. In tamoxifen resistant MCF-7 cells, m 6 A deposition on AK4 mRNA led to increased expression which in turn resulted in higher reactive oxygen species and p38 levels while depletion of METTL3 and AK4 resensitized the cells to tamoxifen (74). METTL3 regulation of miRNA-221-3p through methylation of pri-miRNA-221-3p resulted in chemoresistance to doxorubicin in breast cancer cells, due to the suppression of the tumor suppressor HIPK2 that led to overexpression of Che-1, a transcription regulator known for its role in development of anticancer drug resistance (75). On the other hand, metformin, a drug used in the treatment of type 2 diabetes mellitus, has been shown to suppress m 6 A through downregulation of METTL3 (76). Metformin inhibited METTL3 and thus the m 6 A deposition on p21 (also known as cyclin-dependent kinase inhibitor 1 (CDKN1A)) exhibiting an antiproliferative effect in breast cancer cells (76).

Lastly, m 6 A on 18S rRNA at position 1832 has also been linked to breast cancer cell growth. This modification is catalyzed by METTL5 which must form a heterodimeric complex with TRMT112 to perform its methyltransferase activity (77). METTL5-mediated 1832 methylation at 18S rRNA fine-tuned the conformation of the ribosome decoding cen-ter, increasing its interaction with mRNAs (78). METTL5 has been shown to be overexpressed in breast cancer samples and its loss led to a reduction in proliferation of different breast cancer cell lines and of the S6K phosphorylation needed for the cells to initiate translation and to undergo growth (42). Similarly, enhanced expression of ZCCHC4, which catalyzes the m 6 A deposition of human 28S rRNA at position A4220 (79), is also evidenced in breast tumor tissues although the function of this modification in breast tumorigenesis is yet to be investigated (47). Taken together, this research topic is still in its infancy and more studies are needed to contour a better picture of the role of m 6 A on rRNA in breast cancer.


## N1-methyladenosine

Methylation of adenosine at position N1 or m 1 A has the capacity to disrupt the Watson-Crick base-pairing specificity affecting RNA structure and protein-RNA interaction (80)(81)(82). It was originally discovered in tRNA and rRNA, and later studies detected its presence in mRNA although its ubiquity is still controversial. Hence, whereas original studies suggested that thousands of mRNAs showed m 1 A enrichment in the 5 UTR, usually in GC-rich regions and that m 1 A modification played an important role in translation and in environmental stress response (80,83), later studies revealed few transcripts with this modification (84). In addition, the original authors themselves conceded that m 1 A distribution showed no 5 UTR bias (85).

Therefore, although there is not enough evidence to suggest that m 1 A is present at appreciable levels on mRNA, one study has explored its role in breast cancer. Cytokine macrophage colony-stimulating factor (CSF-1) is an oncogene that promotes metastatic dispersion in breast and ovarian carcinomas. It has been shown that CSF-1 transcripts decorated with m 1 A were targeted for mRNA decay. Hence, ALKBH3-mediated demethylation increased the stability of CSF-1 mRNA, thereby increasing its expression. Given that ALKBH3 is overexpressed in many types of tumors, it is plausible that this is a general mechanism and not exclusively for breast or ovarian cancers (86).


## 5-methylcytosine

The methylation of the carbon 5 in cytosine (m 5 C) was originally found in rRNA and tRNA; however, recent studies have also detected m 5 C on mRNA (87,88). In eukaryotes, the large ribosomal subunit contains two m 5 C residues (22) that are essential to maintain ribosomal structure and fidelity during translation, playing a significant role in lifespan and stress resistance (89). On tRNAs, m 5 C protected from angiogenin-mediated endonucleolytic cleavage, and thereby avoided the biogenesis of tRNA-derived small RNA fragments (tRFs) (90,91). Such tRFs play a key role in regulating gene expression programs participating in various physiological processes such as cell stress, cell growth and cell differentiation (92). tRFs also play significant roles in various human diseases, including cancer (93). On mRNA, m 5 C promotes export (87), stabilization (88) and translation (94,95).

Depending on the RNA species, distinct m 5 C methyltransferase from the NOL1/NOP2/SUN domain (NSUN) family of proteins (NSUN1 to NSUN7) and the DNA methyltransferase member 2 (DNMT2) catalyze the m 5 C modification (95). Hence, NOP2 (NSUN1) and NSUN5 methylate 28S rRNA, while NSUN4 modify mitochondrial rRNA (96)(97)(98)(99). DNMT2, NSUN2, NSUN3 and NSUN6 all methylate cytoplasmic tRNAs, with different specificity and at different residues, and NSUN3 targets mitochondrial tRNA (100)(101)(102)(103)(104). Moreover, NSUN6 also targets sitespecific deposition of m 5 C in mRNA (105), NSUN2 in ncRNAs and mRNA (87,106), and NSUN7 targets enhancer RNAs (eRNAs) (107).

Several m 5 C methyltransferases, including NOP2, NSUN2 and NSUN4, are up-regulated in breast cancer although the molecular consequences of this overexpression are not well characterized (108)(109)(110). In addition, the genomic region containing NSUN2 (5p15.31-33) was associated with a strong risk for the development of breast cancer (111). Hence, both amplification and overexpression of NSUN2 has therapeutic potential as a drug target (109). Additionally, overexpression of NSUN2 by DNA hypomethylation has been associated with proliferation, migration and invasion while NSUN2 knockdown inhibited these processes in vitro and in vivo (112). These results were corroborated in another pan-cancer study showing that NSUN2, among other RNA methyltransferases, was amplified or mutated in breast cancer, and its expression was associated with poor prognosis in these patients (113). Notably, as NSUN2 can catalyze the deposition of m 5 C on distinct RNA species, i.e. of tRNAs, mRNAs and ncRNAs (87,106,114) additional work will be required to address the molecular mechanism that leads to breast carcinogenesis. Recently, Huang et al. reported that all eleven m 5 C regulators (NSUN2-NSUN7, DNMT1, DNMT3A, DNMT3B, ALYREF and TET2) were differentially expressed in TNBC and can potentially predict clinical prognostic risk in patients. Whereas upregulated expression of NSUN2 was found to be closely associated with cell cycle signaling pathways, RNA degradation and RNA polymerase, reduced expression of NSUN6 was linked with cell adhesion, metabolism and extracellular matrix receptor interaction (115).

Y-box binding protein 1 (YBX1) is a specific reader of m 5 C in mRNA which stabilizes the oncogene HDGF in urothelial carcinoma of the bladder (88). YBX1 is a multifunctional protein that is frequently overexpressed in breast cancer regardless of the subtype and its expression was correlated with poor survival, drug resistance and relapse (116,117). Therefore, these studies suggest that dysregulated YBX1-mediated decoding of m 5 C can lead to not only urothelial carcinoma but also to other cancers, including breast cancer.


# -O-methylation

2 O-Me is the addition of a methyl group at the 2 hydroxyl of the ribose moiety of all four nucleosides. This mark can be catalyzed by stand-alone enzymes (118) or by the methyltransferase fibrillarin in association with the C/D box family snoRNAs and the conserved proteins Nop56p, Nop58p and 15.5K or NHP2L1, collectively known as box C/D RNP complex (119). 2 O-Me is a predominant mark in rRNA, where >100 sites exist (120), but it has also been found in tRNA, mRNA (121)(122)(123)(124)(125)(126) and other small RNAs (127).

Internal modification of 2 O-Me at 3 -terminal protects terminal ribose, inhibits the function of the RNA ligase and negatively impacts the efficiency of the polyA-polymerase (128,129), directly reducing the translational capacities of ribosomes (130). A Pan-cancer analysis highlighted that several RNA methyltransferases are either amplified or mutated in different cancers (68). Among these, FtsJ RNA 2 -O-methyltransferase 3 (FTSJ3), that modifies both rRNA and mRNA is among the stand-alone enzymes that has been correlated with cell growth and survival of breast cancer cells (113).

2 O-Me is linked to ribosome biogenesis and deregulated ribosome biogenesis is found to be associated with breast cancer cell progression (131). Therefore, studying the signature of 2 O-Me sites in both patient samples or cell lines is of therapeutic importance. This can be performed by Ri-boMethSeq which maps 2 O-Me sites based on the principle that methylation of ribose 2 OH makes 3 -adjacent phosphodiester bond resistance to alkaline hydrolysis and nuclease cleavage (132). Indeed, RiboMethSeq in 195 primary breast tumor samples showed that rRNA 2 O-Me levels differed between breast cancer subtypes and tumor grades (133). Particularly, TNBC patients displayed a signature where 2 O-Me levels at 18S-Am576 and at 18S-Gm1447 sites were increased and decreased, respectively. In addition, modification at the 18S-Gm1447 site was found to be guided by SNORD127 at 18S of rRNA, suggesting that studies evaluating the mechanism of SNORD127 in TNBC might be of therapeutic importance.

Recently, a distinct 2 O-Me rRNA pattern was observed upon exposing normal breast epithelial and breast cancer cell lines, i.e. MCF10A and T47D cells, to hypoxic conditions (134). In hypoxic conditions both cell lines displayed high 2 O-Me levels at 1858 and 4436 sites and reduced methylation at 390, 1612, 2848, 4806 and 1803 sites, compared to normoxia. Noteworthy, in low oxygen conditions genes such as vascular endothelial growth factor (VEGF-C) are translated in a cap-independent manner through internal ribosome entry site (IRES). Hence, a specific pool of ribosomes with a distinct 2 O-Me pattern facilitated the IRES recognition enhancing the translation of VEGF-C and most probably, other oncogenes (135). Although mechanistic evaluation of 2 O-Me in breast tumorigenesis is lacking, several groups have shown that deregulated expression of components of 2 O-Me RNP complex, such as C/D box snoRNAs and fibrillarin, led to breast tumorigenesis (136,137). For instance, increased fibrillarin expression was associated with aberrant 2 O-Me rRNA pattern and thus impaired translational fidelity. In addition, Marcel et al. showed that p53 acted as a safeguard of protein synthesis by repressing the expression of fibrillarin in breast cancer (138). Moreover, 58 differentially expressed snoRNAs were identified in 26 TNBC cell lines (139) being SNORD78, SNORD93, SNORD62A, SNORD2 and SNORD57 among the most highly upregulated snoRNAs in the invasive breast cancer MDA-MB-231 cell line. Small RNA sequencing in normal and primary breast tumor tissues discovered thirteen snoRNAs which were associated with overall survival and relapse free survival of breast cancer patients (140). In another study, Kothari et al. performed a transcriptome array in patient samples from different breast cancer subtypes and found SNORD114 and SNORD115 as important regulators of breast tumorigenesis (141).

The snoRNA host genes can be of great therapeutic importance as they mostly show altered expression in multiple cancers, and are known to regulate cell growth, tumor progression, metastasis and chemoresistance (142)(143)(144). ZFas1, host RNA for SNORD12, SNORD12b and SNORD12c (ZNFX1 Antisense RNA 1) regulated alveolar development and epithelial cell differentiation during normal mammary development and thus is a potential biomarker acting as a tumor suppressor in breast cancer (145). However, not all snoRNAs are involved in 2 O-Me. A combined analysis of high-throughput sequencing protocols and computational methods (146) suggested alternative roles for snoR-NAs beyond 2 O-Me. Therefore, for a better understanding of correlation between snoRNAs expression and 2'O-Me, more research is needed in this area.


## Pseudouridylation

Pseudouridylation is the isomerization of 1-ribosyluracil (uridine) to 5-ribosyluracil (pseudouridine, ) within the ribonucleoside. Known to be present in all kinds of RNA species, pseudouridylation is the second most common RNA modification of rRNA and has important roles in protein translation and pre-mRNA splicing (37,147). The mechanism of isomerization of uridine to pseudouridine can be either RNA independent or RNA dependent. RNA independent pseudouridylation is performed by enzymes having both substrate recognition and catalytic properties known as pseudouridine synthases (148). However, RNA dependent pseudouridylation is carried out by the RNP complex that consists of catalytic protein dyskerin (DKC1), box H/ACA snoRNAs (SNORAs) (substrate recognition), non-histone protein 2 (Nhp2), nucleolar protein 10 (Nop10) and glycine-arginine-rich protein-1 (Gar1) (149).

In breast cancer patients and cell lines, reduced expression of DKC1 was found to be associated with reduced telomerase activity, rRNA pseudouridylation and better clinical outcome (150,151). Similarly, depletion of DKC1 led to p53-inactivation and increased IRESmediated VEGF mRNA translation in breast cancer cell lines (152,153). Moreover, enhanced DKC1 expression increased ribosome efficiency and stimulated the aggressive phenotype in the normal breast epithelial MCF10A cell line (154). In addition, expression of Nop10 was found to be associated with aggressive breast cancer (155) and coding missense variants of Nhp2 gene were detected in hereditary breast cancer patients although no pathogenic role was elucidated (156). Levels of small nucleolar RNAs (SNOR-NAs) in cancer are deregulated (157), being SNORA3, SNORA18, SNORA7B, SNORA13 SNORA2A among others highly expressed in metastatic breast cancer patients and cell lines (139,158,159). Inhibition of oncogenic SNORA7B impaired cell growth, migration and apoptosis in breast cancer cells (160). Relative pattern of pseudouridylation in normal and breast cancerous cases is yet to be evaluated and hence needs more research to elucidate its direct role in breast tumor initiation and progression.


## RNA editing

RNA editing refers to modifications of specific nucleotides in RNA sequence differentiating it from its corresponding DNA sequence that may lead to pathological consequences. The most common form of RNA editing is from adenosine (A) to inosine (I) (161) and is catalyzed by specific enzymes namely adenosine deaminases acting on RNA (ADARs). In mammals, three ADAR genes encode for four different isoforms known as ADAR1p150, ADAR1p110, ADAR2 and ADAR3 (162). A-to-I editing within the protein-coding region of an mRNA can result in an amino acid change in the encoded protein as the resulting inosine is interpreted as guanosine by the translational machinery leading to an amino acid substitution in the protein product (163). In addition, A-to-I can alter regulatory motifs, binding of RNA-binding proteins, or RNA secondary structures influencing pre-mRNA alternative splicing (164)(165)(166). Furthermore, RNA editing within 3 UTRs of target transcripts can alter miRNA targeting an perturb miRNA-mediated regulation of oncogenes and tumor-supressors (167). The enzymes involved in A-I editing on tRNAs are known as Adenosine deaminases that act on tRNAs (ADATs). Deregulated ADATs-mediated RNA editing is associated with diseases such as type 2 diabetes, neurological disorders, mitochondria-related disorders and cancer (168). Less frequent cytidine (C) to uridine (U) mRNA editing is carried out by cytidine deaminases belonging to the family of enzymes known as activation induced cytidine deaminase/apolipoprotein B editing complex (AID/APOBEC) or AADs (169), eleven of which are found in humans (170).


## A-to-I editing in breast cancer

A-to-I editing is known to be the major source of mRNA variability in breast cancer (171), and it is controlled by type-I interferon response and high ADAR DNA copy number (172). Hence, >76 000 RNA editing sites were identified by using a bioluminescent reporter system and upon interferon treatment of the breast cancer MCF7 cell line (173). Breast tumors harbor amplified levels of ADAR (174) and inhibiting its activity has been shown to induce apoptosis. Indeed, A-to-I mRNA editing regulated breast cancer cell proliferation by modulating the stability and expression of genes involved in cell cycle and DNA damage response being ATM, MDM2, MDM4, CENPN and XPO1 among the affected transcripts (175). In another study, increased A-to-I mRNA editing was found on the mRNA of ATM, GINS4 and POLH affecting their stability and/ or expression in primary breast tumors when compared to the noncancerous tissues (171). In addition, Gumireddy et al. reported that A-to-I mRNA editing of ␥ -amino butyric acid receptor alpha 3 (GABRA3), one of the subunits of GABA (A) receptor, changed its role from a metastasis promoter to a tumor suppressor by suppressing the ERK pathway activation (176). In an attempt to assess the role of A-to-I RNA editing on proteomic diversity, Peng et al. showed that A-to-I RNA editing contributed to protein heterogeneity in breast cancer (177). Hence, nine unique RNA editing sites with variant peptide evidence were identified such as COPA I164V and IGFBP R78G which were detected across 11 out of 36 BRCA mass spectrometry datasets (177). These peptide variants were shown to be presented as self-antigens by human leukocyte antigen (HLA) molecules, and thus recognized by the immune system. For instance, CD8 + effector T cells were evidenced in tumors in response to peptide variants generated from cyclin I (178). A-to-I RNA editing has also been linked to resistance to chemotherapy. Specifically, RNA editing at 26 sites of 3 UTR modulated the expression of dihydrofolate reductase resulting in highly proliferative breast cancer cells which were resistant to methotrexate (179). To sum up, existing data suggests an important role of A-to-I RNA editing in genes associated with breast cancer-relevant pathways and treatment outcomes, suggesting an important role of ADAR1 function in breast tumorigenesis.


## C-to-U editing in breast cancer

APOBEC-dependent somatic mutation confers increased susceptibility for breast cancer (180) and is found to be enriched in the HER2 subtype (181,182). Overexpression of cytidine deaminase APOBEC3A in HEK293T cells caused mRNA alterations in several tumor associated genes including PTEN, KMT2A, ATM, BRCA1 and BRCA2 (183); however, their significance in breast tumorigenesis is yet to be evaluated. mRNA expression of APOBEC3B was evaluated in ductal carcinoma in situ (DCIS), invasive breast cancer (IBC) and normal breast cells (184). Normal breast cells expressed reduced levels of APOBEC3B, while increased expression was evident in DCIS and IBC. Recently, APOBEC3-mediated C-to-U RNA editing in breast cancer was shown to be associated with improved patient survival and enhanced immune activity (185). In another study, using a bioinformatics approach, C-to-U RNA editing levels were estimated in 1040 primary breast tumor tissues and 93 adjacent normal tissues. More than 5000 APOBEC3-mediated RNA editing sites were identified using TCGA sequencing data. For 440 sites, editing was found on 411 transcripts among which most prevalent were on GATA Zinc Finger Domain Containing 2B (GATAD2B), Serpin Family A Member 1 (SERPINA1) and Adenosine Monophosphate Deaminase 3 (AMPD3) (185). Moreover, Apobec-1 complementation factor (A1CF), another mRNA editing enzyme, promoted proliferation of basal-like breast cancer cells by targeting interleukin-6 (186). Overall, the existing data on C-to-U editing suggests important roles in breast cancer progression and further research may identify targets with therapeutic importance.


## Other RNA modifications related to breast cancer

Growing evidence in recent years has pointed out that tRNAs and their derivatives are dysregulated in breast and other cancers (187). However, the functional role of tRNA modifications in tumorigenesis is still elusive. As mentioned before, several modifications are shared by the distinct RNA species but others are unique to tRNAs, as is the case of cm5U34 and mcm5U34 at the wobble position. These marks have been shown to influence the translational efficiency and the accuracy of the reading frame (188). In addition, the catalytic enzymes for mcm5s2 deposition are upregulated in breast cancer sustaining metastasis (189). They enhanced the translation of DEK and LEF1, two oncogenes, enabling breast cancer cells to migrate and invade other tissues (189). Another modification that occurs at the wobble anticodon position is queuosine which is dependent on the gut microbiome (190). Thus, the microbiome product queuine is the substrate for the enzymes tRNA guanine transglycosylases (TGTs). In eukaryotes, the active TGT is a heterodimer formed of a catalytically active queuine tRNA-ribosyltransferase subunit 1 (QTRT1) and a catalytically inactive QTRT2 subunit (191). Knockout of QTRT1 in MCF7 and MDA-MB-231 cells produced changes in the functions of genes involved in cell proliferation, junction formation and migration (190). These results have been validated in mouse models, thus enforcing the significant role that this modification is playing in breast cancer development (190).

TRMT12, a tRNA methyltransferase that mediates posttranscriptional modifications on tRNAPhe has been reported to be consistently amplified and overexpressed in cell lines and breast cancer patient samples (192). In yeast, it catalyses the formation of wybutosine (yW) on the 37th residue of tRNAPhe, modification that helps in stabilization of the codon-anticodon interaction and in maintenance of the reading frame (192). Thus, disruption at the level of this modification can lead to translational errors and is therefore of great interest to establish the role of these enzymes, together with its effect on tRNAs, in tumorigenesis of breast cells.

It is noteworthy that many of the methyltransferases that catalyze the methylation of different types of RNAs have been shown to be deregulated in breast cancer. Although the lack of any association between the deposited marks and the expression of these proteins made us filter out these studies, there are some that are worth mentioning. For instance, tRNA methyltransferase 2 homolog A (TRMT2A) which catalyzes the methylation of the U5 of tRNAs and regulates cell cycle has been associated with increased recurrence risk in HER2+ breast cancer patients (193). Human tRNA methyltransferase 9-like (hTRM9L) protein, an another tRNA methyltransferase, is downregulated in breast cancer and was shown to suppress tumor growth in vivo by decreasing proliferation, cell cycle arrest in G0/G1, upregulation of LIN9, and blocking the hypoxia response (194) in colorectal cancer. Whether this mechanism is the same in breast cancer and if it is mediated though methylation remains to be further investigated.

Despite being less explored than other RNA marks, many of these modifications convey to be promising clues in deciphering the molecular enigma behind the development of breast cancer. Thus, it will be of increased interest to see how extensive studies of these marks on the various species of RNA can contribute to a more precise picture of breast cancer tumorigenic process.


## Perspectives

In the last decade, and as a result of the advent of more reliable methodology, the number of epitranscriptomic studies has dramatically increased. The boost and excitement of this emerging field has led to a tremendous progress in identifying the machinery and the role of RNA modifications. Hence, RNA marks have emerged as important regulators of a variety of cellular processes and their roles in human disease, including cancer, have extensively been described. It has been shown that writers, erasers and readers can act as tumor suppressors or as oncogenes depending on the cellular context. Thus, a single enzyme can have opposing roles in distinct cancer types. However, as a consequence of this fast-growing discipline, many erroneous conclusions have been drawn, and a large number of controversial studies has emerged. In addition, most of the works have just focused on few RNA modifications, and thereby the functions of most RNA modifications remain to be characterized.

During these years, RNA modifications have been comprehensively mapped transcriptome-wide by coupling antibody immunoprecipitation or chemical probing with nextgeneration sequencing. Yet, tools for the simultaneous identification of distinct chemical marks in the same RNA are currently not available. Hence, it is unknown how the different RNA modifications interplay to influence cancer development. Aberrant expression of such RNA-modifying machinery has been acknowledged for most aspects of breast and other cancer types. One particular challenge is to attribute a phenotype to a particular chemical mark as there is a crosstalk between rRNA, tRNA and mRNA modifications which involves common modification factors. For example, NSUN2 can impose m 5 C on both mRNA and tRNA species (195). In addition, several previous conclusions might have to be revised as some of these key players are moonlighting proteins which perform multiple autonomous and often unrelated functions. Therefore, it is also challenging to distinguish between the canonical, i.e. directly related to RNA modifications and secondary protein functions. For instance, knockouts of IME4, the yeast homolog of METTL3, display more severe phenotypes than those derived by a catalytic mutant, suggesting that IME4 has methylation-independent functions (196,197). Indeed, it has been shown that in lung cancer, cytoplasmic METTL3 promotes the translation of oncogenes by a mechanism independent of its methyltransferase activity (198).

Because of the inherent reversibility of some of the RNA modifications, inhibitors targeting the RNA-modification machinery are promising therapeutic targets for breast and other tumors. As m 6 A is the most studied RNA modification, it is not surprising that lead biotech companies have developed drugs to target the writer complex, specifically METTL3 (199). Hence, STORM Therapeutics has developed the first catalytic inhibitor of METTL3. Pharmacological inhibition of METTL3 led to strong antitumor effects in vitro and in relevant mouse models of acute myeloid leukemia (AML) (200).

Overall, as highlighted in this review, considerable advance has been made in recent years in the research field of epitranscriptomics which has provided a link between RNA modifications and breast cancer. Yet, the epitranscrip-tome remains vastly unexplored. Thus, it appears that we just perceive the tip of the iceberg and that the number of RNA modifications is much higher than previously estimated. The exhaustive characterization of RNA modifications and the molecular mechanism in which they are involved holds a new avenue for novel cancer therapies.

## Figure 1 .
1Chemical structures of common RNA modifications known to have a role in breast cancer. The modified groups are highlighted in red. The type of RNA in which the modification is detected is indicated using the symbols of rRNA, tRNA and mRNA.

## Figure 2 .
2tRNA modifications. Clover leaf model representing the structure of a human tRNA. Nucleotides that undergo modification are shown in pink. Distinct chemical modifications are represented in different colors. The enzymes known to mediate tRNA modifications linked to breast cancer are capitalized and highlighted in blue.

## Figure 3 .
3Internal mRNA modifications. Schematic representation of internal mRNA modifications and the predicted location on the mRNA are shown. The main writers of the respective modifications and their functions in breast cancer are depicted.

## Figure 4 .
4Global cancer statistics. Pie chart depicting the percentage of new cases in each tumor type in 2020 for women globally. The data used to prepare the chart were taken from GLOBOCAN 2020(27).

## Figure 5 .
5Functions of m6 A in breast cancer. m 6 A modification is dynamically regulated by its writers (METTL3, METTL14, WTAP, HAKAI and VIRMA) erasers (FTO and ALKBH5) and readers (YTHDFC3 and IGF2BP2). Only enzymes with reported function in breast cancer are depicted for simplicity. Deposition of m 6 A on distinct transcripts can enhance (HBXIP, p21, CXCR4, CDK1, CYP1B1, KRT7, Bcl2; red) or diminish (COL3A1; green) breast cancer initiation and progression and affect therapy outcome (AK4; blue). Demethylation of BNIP3 also enhances breast cancer (red) whereas demethylation of Nanog, KLF4 and Sox2 promotes the breast cancer stem cell phenotype. Similarly, recognition of m 6 A-modified DROSHA and c-Myc mRNAs leads to the acquisition of breast cancer stem cell characteristics. YTHDFC3 binds to ST6GALNAC5, EGFR and GJA1 to induce breast cancer metastasis. YTHDF2 recognizes mRNAs involved in MAPK/ERK signaling. Deposition of m 6 A on pri-miRNA allows DGCR8 recognition and further processing. Increased expression of METTL14, FTO and METTL3 alters the expression of miRNA, enhancing the breast cancer phenotype (red) or induces drug resistance (blue) in breast cancer cells. Enhanced METTL5 mediated methylation at adenosine 1832 of mammalian 18S rRNA promotes p70-S6K activation and an increased translation initiation thus stimulating breast cancer cell growth.

## Table 1 .
1List of RNA modifications in breast cancer. RNA modifications, enzymes and associated effect in breast cancerType of RNA 
Enzymes 
Function in breast cancer 
Ref 

m 6 A 
mRNA 
ALKBH5, IGF2BP1 
Promoted BCSC phenotype 
(69,70,73) 
mRNA 
METTL3 
Promoted BCSC phenotype, induces metastasis 
(201) 
mRNA 
IGF2BP2 
Promoted BCSC phenotype 
(72) 
mRNA 
METTL3, METTL14, WTAP, FTO, 
ALKBH5 

Inhibited colony formation and migration 
(202) 

mRNA 
YTHDF3 
Promoted breast cancer, induces brain metastasis 
and angiogenesis 

(203) 

mRNA 
METTL14, ALKBH5, YTHDF3 
Promoted growth, proliferation and angiogenesis 
(56) 
mRNA 
METTL14, ZC3H13 
Correlation with unfavorable prognosis in breast 
cancer patients 

(49) 

mRNA 
METTL3 
Increased cell proliferation and tumor progression 
(57) 
mRNA 
METTL3 
Increased proliferation and decreased apoptosis in 
breast cancer cells in vitro and in vivo 

(58) 

mRNA 
METTL3, FTO, IGF2BP1, YTHDF1 
Promoted lung metastasis and clinical progression in 
breast cancer, induction of EMT 

(59) 

mRNA 
METTL3 
Inhibited migration and invasive capacities of the 
cells in TNBC 

(60) 

mRNA 
METTL14 
Promoted breast cancer initiation and progression 
(62) 
mRNA 
FTO 
Promoted cell proliferation, colony formation, tumor 
growth and metastasis 

(63) 

mRNA 
METTL3 
Enhanced expression of oncogenes, induces acquired 
chemoresistance 

(74) 

mRNA 
METTL3 
Induced breast cancer cell proliferation, associated 
with drug sensitivity 

(76) 

mRNA 
YTHDF2 
Promoted breast cancer progression 
(204) 
mRNA 
ALKBH3 
Increased breast cancer cell invasiveness 
(83) 
miRNA 
METTL14 
Promoted migration and invasion in breast cancer 
cells 

(50) 

miRNA 
FTO 
Promoted cell invasion and migration 
(52) 
miRNA 
METTL3 
Associated with acquired chemoresistance 
(75) 
circRNA 
METTL3 
Promoted cell proliferation 
(61) 
rRNA 
METTL5 
p70-S6K activation and translation initiation, 
increased breast cancer cell growth 

(77) 

m 5 C 
Type of RNA 
Enzymes 
Function in breast cancer 
Ref 
mRNA 
NSUN2-NSUN7, DNMT1, DNMT3A, 
DNMT3B, ALYREF TET2 

Affected tumor development, tumor immune 
microenvironment and potential markers for TNBC 
patients 

(110) 

2'-O-Me 
Type of RNA 
Enzymes 
Function in breast cancer 
Ref 
rRNA 
Fibrillarin 
Promoted BCSC phenotype 
(128) 
rRNA 
Undefined 
Associated with breast cancer subtypes and tumor 
grade, linked with overall patient survival 

(127) 

Type of RNA 
Enzymes 
Function in breast cancer 
Ref 
rRNA 
DKC1 
Telomerase activity, linked with better clinical 
outcome 

(143) 

A-to-I 
Type of RNA 
Enzymes 
Function in breast cancer 
Ref 
mRNA 
ADAR 
Resistance to methotrexate 
(167) 
mRNA 
ADAR 
Cell viability, drug sensitivity clinically relevant 
editing events in breast tumors than normal tissues 

(161) 

mRNA 
ADAR1 
Cell cycle control, DNA damage response, increased 
breast cancer cell progression 

(163) 

mRNA 
ADAR1 
DNA damage, immunity, DNA replication, 
increased breast cancer cell progression 

(159) 

mRNA 
ADAR1p110 
Increased proliferation and breast cancer metastasis 
(164) 
C-to-U 
Type of RNA 
Enzymes 
Function in breast cancer 
Ref 
mRNA 
APOBEC3 
Improved breast cancer survival 
(173) 
mRNA 
Apobec-1 complementation factor 
(A1CF) 

Increased breast cancer progression 
(174) 

 NAR Cancer, 2021, Vol. 3, No. 3   
 NAR Cancer, 2021, Vol. 3, No. 3   
 NAR Cancer, 2021, Vol. 3, No. 3   
 NAR Cancer, 2021, Vol. 3, No. 3   
ACKNOWLEDGEMENTSWe sincerely apologize to authors whose work could not be included due to space limitations.
Role of modified nucleosides of yeast tRNA(Phe) in ribosomal binding. S S Ashraf, R H Guenther, G Ansari, A Malkiewicz, E Sochacka, P F Agris, Cell Biochem. Biophys. 33Ashraf,S.S., Guenther,R.H., Ansari,G., Malkiewicz,A., Sochacka,E. and Agris,P.F. (2000) Role of modified nucleosides of yeast tRNA(Phe) in ribosomal binding. Cell Biochem. Biophys., 33, 241-252.

MODOMICS: a database of RNA modification pathways. 2017 update. P Boccaletto, M A Machnicka, E Purta, P Piatkowski, B Baginski, T K Wirecki, V De Crecy-Lagard, R Ross, P A Limbach, A Kotter, Nucleic Acids Res. 46Boccaletto,P., Machnicka,M.A., Purta,E., Piatkowski,P., Baginski,B., Wirecki,T.K., de Crecy-Lagard,V., Ross,R., Limbach,P.A., Kotter,A. et al. (2018) MODOMICS: a database of RNA modification pathways. 2017 update. Nucleic Acids Res., 46, D303-D307.

Dynamic RNA Modifications in Gene Expression Regulation. I A Roundtree, M E Evans, T Pan, C He, Cell. 169Roundtree,I.A., Evans,M.E., Pan,T. and He,C. (2017) Dynamic RNA Modifications in Gene Expression Regulation. Cell, 169, 1187-1200.

Modifications and functional genomics of human transfer RNA. T Pan, Cell Res. 28Pan,T. (2018) Modifications and functional genomics of human transfer RNA. Cell Res., 28, 395-404.

tRNA wobble modifications and protein homeostasis. N Ranjan, M V Rodnina, Translation (Austin). 41143076Ranjan,N. and Rodnina,M.V. (2016) tRNA wobble modifications and protein homeostasis. Translation (Austin), 4, e1143076.

Codon optimality, bias and usage in translation and mRNA decay. G Hanson, J Coller, Nat. Rev. Mol. Cell Biol. 19Hanson,G. and Coller,J. (2018) Codon optimality, bias and usage in translation and mRNA decay. Nat. Rev. Mol. Cell Biol., 19, 20-30.

tRNA tKUUU, tQUUG, and tEUUC wobble position modifications fine-tune protein translation by promoting ribosome A-site binding. V A Rezgui, K Tyagi, N Ranjan, A L Konevega, J Mittelstaet, M V Rodnina, M Peter, P G Pedrioli, Proc. Natl. Acad. Sci. U.S.A. 110Rezgui,V.A., Tyagi,K., Ranjan,N., Konevega,A.L., Mittelstaet,J., Rodnina,M.V., Peter,M. and Pedrioli,P.G. (2013) tRNA tKUUU, tQUUG, and tEUUC wobble position modifications fine-tune protein translation by promoting ribosome A-site binding. Proc. Natl. Acad. Sci. U.S.A., 110, 12289-12294.

C Lorenz, C E Lunse, M Morl, tRNA Modifications: Impact on Structure and Thermal Adaptation. Biomolecules. 735Lorenz,C., Lunse,C.E. and Morl,M. (2017) tRNA Modifications: Impact on Structure and Thermal Adaptation. Biomolecules, 7, 35.

Chemical and Conformational Diversity of Modified Nucleosides Affects tRNA Structure and Function. V Y Vare, E R Eruysal, A Narendran, K L Sarachan, P F Agris, Biomolecules. 729Vare,V.Y., Eruysal,E.R., Narendran,A., Sarachan,K.L. and Agris,P.F. (2017) Chemical and Conformational Diversity of Modified Nucleosides Affects tRNA Structure and Function. Biomolecules, 7, 29.

The expanding world of tRNA modifications and their disease relevance. T Suzuki, Nat. Rev. Mol. Cell Biol. 22Suzuki,T. (2021) The expanding world of tRNA modifications and their disease relevance. Nat. Rev. Mol. Cell Biol., 22, 375-392.

The structure and function of the eukaryotic ribosome. D N Wilson, J H Cate, Cold Spring Harb. Perspect. Biol. 411536Wilson,D.N. and Doudna Cate,J.H. (2012) The structure and function of the eukaryotic ribosome. Cold Spring Harb. Perspect. Biol., 4, a011536.

Structural insights into eukaryotic ribosomes and the initiation of translation. F Voigts-Hoffmann, S Klinge, N Ban, Curr. Opin. Struct. Biol. 22Voigts-Hoffmann,F., Klinge,S. and Ban,N. (2012) Structural insights into eukaryotic ribosomes and the initiation of translation. Curr. Opin. Struct. Biol., 22, 768-777.

Tuning the ribosome: the influence of rRNA modification on eukaryotic ribosome biogenesis and function. K E Sloan, A S Warda, S Sharma, K D Entian, D L J Lafontaine, M T Bohnsack, RNA Biol. 14Sloan,K.E., Warda,A.S., Sharma,S., Entian,K.D., Lafontaine,D.L.J. and Bohnsack,M.T. (2017) Tuning the ribosome: the influence of rRNA modification on eukaryotic ribosome biogenesis and function. RNA Biol, 14, 1138-1152.

Visualization of chemical modifications in the human 80S ribosome structure. S K Natchiar, A G Myasnikov, H Kratzat, I Hazemann, B P Klaholz, Nature. 551Natchiar,S.K., Myasnikov,A.G., Kratzat,H., Hazemann,I. and Klaholz,B.P. (2017) Visualization of chemical modifications in the human 80S ribosome structure. Nature, 551, 472-477.

Site-specific ribose methylation of preribosomal RNA: a novel function for small nucleolar RNAs. Z Kiss-Laszlo, Y Henry, J P Bachellerie, M Caizergues-Ferrer, T Kiss, Cell. 85Kiss-Laszlo,Z., Henry,Y., Bachellerie,J.P., Caizergues-Ferrer,M. and Kiss,T. (1996) Site-specific ribose methylation of preribosomal RNA: a novel function for small nucleolar RNAs. Cell, 85, 1077-1088.

Site-specific pseudouridine formation in preribosomal RNA is guided by small nucleolar RNAs. P Ganot, M L Bortolin, T Kiss, Cell. 89Ganot,P., Bortolin,M.L. and Kiss,T. (1997) Site-specific pseudouridine formation in preribosomal RNA is guided by small nucleolar RNAs. Cell, 89, 799-809.

The structure of the box C/D enzyme reveals regulation of RNA methylation. A Lapinaite, B Simon, L Skjaerven, M Rakwalska-Bange, F Gabel, T Carlomagno, Nature. 502Lapinaite,A., Simon,B., Skjaerven,L., Rakwalska-Bange,M., Gabel,F. and Carlomagno,T. (2013) The structure of the box C/D enzyme reveals regulation of RNA methylation. Nature, 502, 519-523.

Modified nucleosides and bizarre 5 -termini in mouse myeloma mRNA. J M Adams, S Cory, Nature. 255Adams,J.M. and Cory,S. (1975) Modified nucleosides and bizarre 5 -termini in mouse myeloma mRNA. Nature, 255, 28-33.

Identification of methylated nucleosides in messenger RNA from Novikoff hepatoma cells. R Desrosiers, K Friderici, F Rottman, Proc. Natl. Acad. Sci. U.S.A. 71Desrosiers,R., Friderici,K. and Rottman,F. (1974) Identification of methylated nucleosides in messenger RNA from Novikoff hepatoma cells. Proc. Natl. Acad. Sci. U.S.A., 71, 3971-3975.

The methylation state of poly A-containing messenger RNA from cultured hamster cells. D T Dubin, R H Taylor, Nucleic Acids Res. 2Dubin,D.T. and Taylor,R.H. (1975) The methylation state of poly A-containing messenger RNA from cultured hamster cells. Nucleic Acids Res., 2, 1653-1668.

The methylated constituents of L cell messenger RNA: evidence for an unusual cluster at the 5 terminus. R P Perry, D E Kelley, K Friderici, F Rottman, Cell. 4Perry,R.P., Kelley,D.E., Friderici,K. and Rottman,F. (1975) The methylated constituents of L cell messenger RNA: evidence for an unusual cluster at the 5 terminus. Cell, 4, 387-394.

View From A Bridge': a new perspective on eukaryotic rRNA base modification. S Sharma, D L J Lafontaine, Trends Biochem. Sci. 40Sharma,S. and Lafontaine,D.L.J. (2015) 'View From A Bridge': a new perspective on eukaryotic rRNA base modification. Trends Biochem. Sci., 40, 560-575.

Comprehensive analysis of mRNA methylation reveals enrichment in 3 UTRs and near stop codons. K D Meyer, Y Saletore, P Zumbo, O Elemento, C E Mason, S R Jaffrey, Cell. 149Meyer,K.D., Saletore,Y., Zumbo,P., Elemento,O., Mason,C.E. and Jaffrey,S.R. (2012) Comprehensive analysis of mRNA methylation reveals enrichment in 3 UTRs and near stop codons. Cell, 149, 1635-1646.

Topology of the human and mouse m6A RNA methylomes revealed by m6A-seq. D Dominissini, S Moshitch-Moshkovitz, S Schwartz, M Salmon-Divon, L Ungar, S Osenberg, K Cesarkas, J Jacob-Hirsch, N Amariglio, M Kupiec, Nature. 485Dominissini,D., Moshitch-Moshkovitz,S., Schwartz,S., Salmon-Divon,M., Ungar,L., Osenberg,S., Cesarkas,K., Jacob-Hirsch,J., Amariglio,N., Kupiec,M. et al. (2012) Topology of the human and mouse m6A RNA methylomes revealed by m6A-seq. Nature, 485, 201-206.

Chemical modifications in the life of an mRNA transcript. S Nachtergaele, C He, Annu. Rev. Genet. 52Nachtergaele,S. and He,C. (2018) Chemical modifications in the life of an mRNA transcript. Annu. Rev. Genet., 52, 349-372.

A mark of disease: how mRNA modifications shape genetic and acquired pathologies. E Destefanis, G Avsar, P Groza, A Romitelli, S Torrini, P Pir, S G Conticello, F Aguilo, E Dassi, RNA. 27Destefanis,E., Avsar,G., Groza,P., Romitelli,A., Torrini,S., Pir,P., Conticello,S.G., Aguilo,F. and Dassi,E. (2021) A mark of disease: how mRNA modifications shape genetic and acquired pathologies. RNA, 27, 367-389.

Global Cancer Statistics 2020: GLOBOCAN Estimates of Incidence and Mortality Worldwide for 36 Cancers in 185 Countries. H Sung, J Ferlay, R L Siegel, M Laversanne, I Soerjomataram, A Jemal, F Bray, CA Cancer J. Clin. 71Sung,H., Ferlay,J., Siegel,R.L., Laversanne,M., Soerjomataram,I., Jemal,A. and Bray,F. (2021) Global Cancer Statistics 2020: GLOBOCAN Estimates of Incidence and Mortality Worldwide for 36 Cancers in 185 Countries. CA Cancer J. Clin., 71, 209-249.

A review on methods for diagnosis of breast cancer cells and tissues. Z He, Z Chen, M Tan, S Elingarami, Y Liu, T Li, Y Deng, N He, S Li, J Fu, Cell Prolif. 5312822He,Z., Chen,Z., Tan,M., Elingarami,S., Liu,Y., Li,T., Deng,Y., He,N., Li,S., Fu,J. et al. (2020) A review on methods for diagnosis of breast cancer cells and tissues. Cell Prolif., 53, e12822.

Cancer Statistics, 2021. R L Siegel, K D Miller, H E Fuchs, A Jemal, CA Cancer J. Clin. 71Siegel,R.L., Miller,K.D., Fuchs,H.E. and Jemal,A. (2021) Cancer Statistics, 2021. CA Cancer J. Clin., 71, 7-33.

Molecular classification of breast cancer. J Y S Tsang, G M Tse, Adv. Anat. Pathol. 27Tsang,J.Y.S. and Tse,G.M. (2020) Molecular classification of breast cancer. Adv. Anat. Pathol., 27, 27-35.

Molecular portraits of human breast tumours. C M Perou, T Sorlie, M B Eisen, M Van De Rijn, S S Jeffrey, C A Rees, J R Pollack, D T Ross, H Johnsen, L A Akslen, Nature. 406Perou,C.M., Sorlie,T., Eisen,M.B., van de Rijn,M., Jeffrey,S.S., Rees,C.A., Pollack,J.R., Ross,D.T., Johnsen,H., Akslen,L.A. et al. (2000) Molecular portraits of human breast tumours. Nature, 406, 747-752.

Breast cancer: multiple subtypes within a tumor? Trends Cancer. S K Yeo, J L Guan, 3Yeo,S.K. and Guan,J.L. (2017) Breast cancer: multiple subtypes within a tumor? Trends Cancer, 3, 753-760.

Basal-like and triple-negative breast cancers: a critical review with an emphasis on the implications for pathologists and oncologists. S Badve, D J Dabbs, S J Schnitt, F L Baehner, T Decker, V Eusebi, S B Fox, S Ichihara, J Jacquemier, S R Lakhani, Mod. Pathol. 24Badve,S., Dabbs,D.J., Schnitt,S.J., Baehner,F.L., Decker,T., Eusebi,V., Fox,S.B., Ichihara,S., Jacquemier,J., Lakhani,S.R. et al. (2011) Basal-like and triple-negative breast cancers: a critical review with an emphasis on the implications for pathologists and oncologists. Mod. Pathol., 24, 157-167.

Differences in risk factors for breast cancer molecular subtypes in a population-based study. X R Yang, M E Sherman, D L Rimm, J Lissowska, L A Brinton, B Peplonska, S M Hewitt, W F Anderson, N Szeszenia-Dabrowska, A Bardin-Mikolajczak, Cancer Epidemiol. Biomarkers Prev. 16Yang,X.R., Sherman,M.E., Rimm,D.L., Lissowska,J., Brinton,L.A., Peplonska,B., Hewitt,S.M., Anderson,W.F., Szeszenia-Dabrowska,N., Bardin-Mikolajczak,A. et al. (2007) Differences in risk factors for breast cancer molecular subtypes in a population-based study. Cancer Epidemiol. Biomarkers Prev., 16, 439-443.

A preclinical model for eralpha-positive breast cancer points to the epithelial microenvironment as determinant of luminal phenotype and hormone response. G Sflomos, V Dormoy, T Metsalu, R Jeitziner, L Battista, V Scabia, W Raffoul, J F Delaloye, A Treboux, M Fiche, Cancer Cell. 29Sflomos,G., Dormoy,V., Metsalu,T., Jeitziner,R., Battista,L., Scabia,V., Raffoul,W., Delaloye,J.F., Treboux,A., Fiche,M. et al. (2016) A preclinical model for eralpha-positive breast cancer points to the epithelial microenvironment as determinant of luminal phenotype and hormone response. Cancer Cell, 29, 407-422.

RNA N6-methyladenosine methylation in post-transcriptional gene expression regulation. Y Yue, J Liu, C He, Genes Dev. 29Yue,Y., Liu,J. and He,C. (2015) RNA N6-methyladenosine methylation in post-transcriptional gene expression regulation. Genes Dev., 29, 1343-1355.

. M Taoka, Y Nobe, Y Yamaki, K Sato, H Ishikawa, K Izumikawa, Y Yamauchi, K Hirota, H Nakayama, 14 NAR Cancer. 33Taoka,M., Nobe,Y., Yamaki,Y., Sato,K., Ishikawa,H., Izumikawa,K., Yamauchi,Y., Hirota,K., Nakayama,H., 14 NAR Cancer, 2021, Vol. 3, No. 3

Landscape of the complete RNA chemical modifications in the human 80S ribosome. N Takahashi, Nucleic Acids Res. 46Takahashi,N. et al. (2018) Landscape of the complete RNA chemical modifications in the human 80S ribosome. Nucleic Acids Res., 46, 9289-9298.

Steering pluripotency and differentiation with N(6)-methyladenosine RNA modification. S Malla, D Melguizo-Sanchis, F Aguilo, Biochim. Biophys. Acta Gene Regul. Mech. 1862Malla,S., Melguizo-Sanchis,D. and Aguilo,F. (2019) Steering pluripotency and differentiation with N(6)-methyladenosine RNA modification. Biochim. Biophys. Acta Gene Regul. Mech., 1862, 394-402.

The N(6)-Methyladenosine RNA modification in pluripotency and reprogramming. F Aguilo, M J Walsh, Curr. Opin. Genet. Dev. 46Aguilo,F. and Walsh,M.J. (2017) The N(6)-Methyladenosine RNA modification in pluripotency and reprogramming. Curr. Opin. Genet. Dev., 46, 77-82.

Mechanistic insights into m(6)A RNA enzymes. T Lence, C Paolantoni, L Worpenberg, J Y Roignant, Biochim. Biophys. Acta Gene Regul. Mech. 1862Lence,T., Paolantoni,C., Worpenberg,L. and Roignant,J.Y. (2019) Mechanistic insights into m(6)A RNA enzymes. Biochim. Biophys. Acta Gene Regul. Mech, 1862, 222-229.

Where, when, and how: context-dependent functions of RNA methylation writers, readers, and erasers. H Shi, J Wei, C He, Mol. Cell. 74Shi,H., Wei,J. and He,C. (2019) Where, when, and how: context-dependent functions of RNA methylation writers, readers, and erasers. Mol. Cell, 74, 640-650.

N6-methyladenosine in nuclear RNA is a major substrate of the obesity-associated FTO. G Jia, Y Fu, X Zhao, Q Dai, G Zheng, Y Yang, C Yi, T Lindahl, T Pan, Y G Yang, Nat. Chem. Biol. 7Jia,G., Fu,Y., Zhao,X., Dai,Q., Zheng,G., Yang,Y., Yi,C., Lindahl,T., Pan,T., Yang,Y.G. et al. (2011) N6-methyladenosine in nuclear RNA is a major substrate of the obesity-associated FTO. Nat. Chem. Biol., 7, 885-887.

ALKBH5 is a mammalian RNA demethylase that impacts RNA metabolism and mouse fertility. G Zheng, J A Dahl, Y Niu, P Fedorcsak, C M Huang, C J Li, C B Vagbo, Y Shi, W L Wang, S H Song, Mol. Cell. 49Zheng,G., Dahl,J.A., Niu,Y., Fedorcsak,P., Huang,C.M., Li,C.J., Vagbo,C.B., Shi,Y., Wang,W.L., Song,S.H. et al. (2013) ALKBH5 is a mammalian RNA demethylase that impacts RNA metabolism and mouse fertility. Mol. Cell, 49, 18-29.

Reading m(6)A in the Transcriptome: m(6)A-Binding Proteins. D P Patil, B F Pickering, S R Jaffrey, Trends Cell Biol. 28Patil,D.P., Pickering,B.F. and Jaffrey,S.R. (2018) Reading m(6)A in the Transcriptome: m(6)A-Binding Proteins. Trends Cell Biol., 28, 113-127.

Reading, writing and erasing mRNA methylation. S Zaccara, R J Ries, S R Jaffrey, Nat. Rev. Mol. Cell Biol. 20Zaccara,S., Ries,R.J. and Jaffrey,S.R. (2019) Reading, writing and erasing mRNA methylation. Nat. Rev. Mol. Cell Biol., 20, 608-624.

Upregulated N6-methyladenosine RNA in peripheral blood: potential diagnostic biomarker for breast cancer. H Xiao, X Fan, R Zhang, G Wu, Cancer Res. Treat. 53Xiao,H., Fan,X., Zhang,R. and Wu,G. (2021) Upregulated N6-methyladenosine RNA in peripheral blood: potential diagnostic biomarker for breast cancer. Cancer Res. Treat., 53, 399-408.

Expression pattern of m(6)A regulators is significantly correlated with malignancy and antitumor immune response of breast cancer. X He, L Tan, J Ni, G Shen, Cancer Gene Ther. 28He,X., Tan,L., Ni,J. and Shen,G. (2021) Expression pattern of m(6)A regulators is significantly correlated with malignancy and antitumor immune response of breast cancer. Cancer Gene Ther., 28, 188-196.

Effect of N6-methyladenosine regulators on progression and prognosis of triple-negative breast cancer. S Wang, X Zou, Y Chen, W C Cho, X Zhou, Front Genet. 11580036Wang,S., Zou,X., Chen,Y., Cho,W.C. and Zhou,X. (2020) Effect of N6-methyladenosine regulators on progression and prognosis of triple-negative breast cancer. Front Genet, 11, 580036.

Analysis of N6-methyladenosine methyltransferase reveals METTL14 and ZC3H13 as tumor suppressor genes in breast cancer. P J Gong, Y C Shao, Y Yang, W J Song, X He, Y F Zeng, S R Huang, L Wei, J W Zhang, Front. Oncol. 10578963Gong,P.J., Shao,Y.C., Yang,Y., Song,W.J., He,X., Zeng,Y.F., Huang,S.R., Wei,L. and Zhang,J.W. (2020) Analysis of N6-methyladenosine methyltransferase reveals METTL14 and ZC3H13 as tumor suppressor genes in breast cancer. Front. Oncol., 10, 578963.

2020) METTL14 promotes the migration and invasion of breast cancer cells by modulating N6methyladenosine and hsamiR146a5p expression. D Yi, R Wang, X Shi, L Xu, Y Yilihamu, J Sang, Oncol. Rep. 43Yi,D., Wang,R., Shi,X., Xu,L., Yilihamu,Y. and Sang,J. (2020) METTL14 promotes the migration and invasion of breast cancer cells by modulating N6methyladenosine and hsamiR146a5p expression. Oncol. Rep., 43, 1375-1386.

Downregulated METTL14 expression correlates with breast cancer tumor grade and molecular classification. X F Dong, Y Wang, B F Huang, G N Hu, J K Shao, Q Wang, C H Tang, C Q Wang, Biomed. Res. Int. 8823270Dong,X.F., Wang,Y., Huang,B.F., Hu,G.N., Shao,J.K., Wang,Q., Tang,C.H. and Wang,C.Q. (2020) Downregulated METTL14 expression correlates with breast cancer tumor grade and molecular classification. Biomed. Res. Int., 2020, 8823270.

The FTO/miR-181b-3p/ARL5B signaling pathway regulates cell migration and invasion in breast cancer. Y Xu, S Ye, N Zhang, S Zheng, H Liu, K Zhou, L Wang, Y Cao, P Sun, T Wang, Cancer Commun. 40Xu,Y., Ye,S., Zhang,N., Zheng,S., Liu,H., Zhou,K., Wang,L., Cao,Y., Sun,P. and Wang,T. (2020) The FTO/miR-181b-3p/ARL5B signaling pathway regulates cell migration and invasion in breast cancer. Cancer Commun (Lond), 40, 484-500.

Expression and prognostic characteristics of m(6) A RNA methylation regulators in breast cancer. B Zhang, Y Gu, G Jiang, Front Genet. 11604597Zhang,B., Gu,Y. and Jiang,G. (2020) Expression and prognostic characteristics of m(6) A RNA methylation regulators in breast cancer. Front Genet, 11, 604597.

N6-methyladenosine-related genomic targets are altered in breast cancer tissue and associated with poor survival. L Liu, X Liu, Z Dong, J Li, Y Yu, X Chen, F Ren, G Cui, R Sun, J. Cancer. 10Liu,L., Liu,X., Dong,Z., Li,J., Yu,Y., Chen,X., Ren,F., Cui,G. and Sun,R. (2019) N6-methyladenosine-related genomic targets are altered in breast cancer tissue and associated with poor survival. J. Cancer, 10, 5447-5459.

The m6A readers YTHDF1 and YTHDF3 aberrations associated with metastasis and predict poor prognosis in breast cancer patients. R Anita, A Paramasivam, J V Priyadharsini, S Chitra, Am J Cancer Res. 10Anita,R., Paramasivam,A., Priyadharsini,J.V. and Chitra,S. (2020) The m6A readers YTHDF1 and YTHDF3 aberrations associated with metastasis and predict poor prognosis in breast cancer patients. Am J Cancer Res, 10, 2546-2554.

Cross-talk among writers, readers, and erasers of m(6)A regulates cancer growth and progression. S Panneerdoss, V K Eedunuri, P Yadav, S Timilsina, S Rajamanickam, S Viswanadhapalli, N Abdelfattah, B C Onyeagucha, X Cui, Z Lai, Sci. Adv. 48263Panneerdoss,S., Eedunuri,V.K., Yadav,P., Timilsina,S., Rajamanickam,S., Viswanadhapalli,S., Abdelfattah,N., Onyeagucha,B.C., Cui,X., Lai,Z. et al. (2018) Cross-talk among writers, readers, and erasers of m(6)A regulates cancer growth and progression. Sci. Adv., 4, eaar8263.

HBXIP-elevated methyltransferase METTL3 promotes the progression of breast cancer via inhibiting tumor suppressor let-7g. X Cai, X Wang, C Cao, Y Gao, S Zhang, Z Yang, Y Liu, X Zhang, W Zhang, L Ye, Cancer Lett. 415Cai,X., Wang,X., Cao,C., Gao,Y., Zhang,S., Yang,Z., Liu,Y., Zhang,X., Zhang,W. and Ye,L. (2018) HBXIP-elevated methyltransferase METTL3 promotes the progression of breast cancer via inhibiting tumor suppressor let-7g. Cancer Lett., 415, 11-19.

N6-methyladenosine METTL3 promotes the breast cancer progression via targeting Bcl-2. H Wang, B Xu, J Shi, Gene. 722144076Wang,H., Xu,B. and Shi,J. (2020) N6-methyladenosine METTL3 promotes the breast cancer progression via targeting Bcl-2. Gene, 722, 144076.

N(6) -methyladenosine regulates mRNA stability and translation efficiency of KRT7 to promote breast cancer lung metastasis. F Chen, Z Chen, T Guan, Y Zhou, L Ge, H Zhang, Y Wu, G M Jiang, W He, J Li, Cancer Res. 81Chen,F., Chen,Z., Guan,T., Zhou,Y., Ge,L., Zhang,H., Wu,Y., Jiang,G.M., He,W., Li,J. et al. (2021) N(6) -methyladenosine regulates mRNA stability and translation efficiency of KRT7 to promote breast cancer lung metastasis. Cancer Res., 81, 2847-2860.

Reduced expression of METTL3 promotes metastasis of triple-negative breast cancer by m6a methylation-mediated COL3A1 up-regulation. Y Shi, C Zheng, Y Jin, B Bao, D Wang, K Hou, J Feng, S Tang, X Qu, Y Liu, Front. Oncol. 101126Shi,Y., Zheng,C., Jin,Y., Bao,B., Wang,D., Hou,K., Feng,J., Tang,S., Qu,X., Liu,Y. et al. (2020) Reduced expression of METTL3 promotes metastasis of triple-negative breast cancer by m6a methylation-mediated COL3A1 up-regulation. Front. Oncol., 10, 1126.

2021) CircMETTL3, upregulated in a m6A-dependent manner, promotes breast cancer progression. Z Li, H Y Yang, X Y Dai, X Zhang, Y Z Huang, L Shi, J F Wei, Q Ding, Int. J. Biol. Sci. 17Li,Z., Yang,H.Y., Dai,X.Y., Zhang,X., Huang,Y.Z., Shi,L., Wei,J.F. and Ding,Q. (2021) CircMETTL3, upregulated in a m6A-dependent manner, promotes breast cancer progression. Int. J. Biol. Sci., 17, 1178-1190.

LNC942 promoting METTL14-mediated m(6)A methylation in breast cancer cell proliferation and progression. T Sun, Z Wu, X Wang, Y Wang, X Hu, W Qin, S Lu, D Xu, Y Wu, Q Chen, Oncogene. 39Sun,T., Wu,Z., Wang,X., Wang,Y., Hu,X., Qin,W., Lu,S., Xu,D., Wu,Y., Chen,Q. et al. (2020) LNC942 promoting METTL14-mediated m(6)A methylation in breast cancer cell proliferation and progression. Oncogene, 39, 5358-5372.

RNA N6-methyladenosine demethylase FTO promotes breast tumor progression through inhibiting BNIP3. Y Niu, Z Lin, A Wan, H Chen, H Liang, L Sun, Y Wang, X Li, X F Xiong, B Wei, Mol. Cancer. 1846Niu,Y., Lin,Z., Wan,A., Chen,H., Liang,H., Sun,L., Wang,Y., Li,X., Xiong,X.F., Wei,B. et al. (2019) RNA N6-methyladenosine demethylase FTO promotes breast tumor progression through inhibiting BNIP3. Mol. Cancer, 18, 46.

Inhibition of YTHDF2 triggers proteotoxic cell death in MYC-driven breast cancer. J M Einstein, M Perelis, I A Chaim, J K Meena, J K Nussbacher, A T Tankka, B A Yee, H Li, A A Madrigal, N J Neill, Mol. Cell. 81Einstein,J.M., Perelis,M., Chaim,I.A., Meena,J.K., Nussbacher,J.K., Tankka,A.T., Yee,B.A., Li,H., Madrigal,A.A., Neill,N.J. et al. (2021) Inhibition of YTHDF2 triggers proteotoxic cell death in MYC-driven breast cancer. Mol. Cell, 81, 3048-3064.

N6-methyladenosine marks primary microRNAs for processing. C R Alarcon, H Lee, H Goodarzi, N Halberg, S F Tavazoie, Nature. 519Alarcon,C.R., Lee,H., Goodarzi,H., Halberg,N. and Tavazoie,S.F. (2015) N6-methyladenosine marks primary microRNAs for processing. Nature, 519, 482-485.

Cancer stem cells and epithelial-to-mesenchymal transition in cancer metastasis. T Celia-Terrassa, M K Jolly, Cold Spring Harb. Perspect. Med. 1036905Celia-Terrassa,T. and Jolly,M.K. (2020) Cancer stem cells and epithelial-to-mesenchymal transition in cancer metastasis. Cold Spring Harb. Perspect. Med., 10, a036905.

. M Vivanco Mdel, Mammary Stem Cells. Preface. Methods Mol. Biol. 1293Vivanco Mdel,M. (2015) Mammary Stem Cells. Preface. Methods Mol. Biol., 1293, v-vi.

Direct regulation of TWIST by HIF-1alpha promotes metastasis. M H Yang, M Z Wu, S H Chiou, P M Chen, S Y Chang, C J Liu, S C Teng, K J Wu, Nat. Cell Biol. 10Yang,M.H., Wu,M.Z., Chiou,S.H., Chen,P.M., Chang,S.Y., Liu,C.J., Teng,S.C. and Wu,K.J. (2008) Direct regulation of TWIST by HIF-1alpha promotes metastasis. Nat. Cell Biol., 10, 295-305.

Hypoxia induces the breast cancer stem cell phenotype by HIF-dependent and ALKBH5-mediated m(6)A-demethylation of NANOG mRNA. C Zhang, D Samanta, H Lu, J W Bullen, H Zhang, I Chen, X He, G L Semenza, Proc. Natl. Acad. Sci. U.S.A. 113Zhang,C., Samanta,D., Lu,H., Bullen,J.W., Zhang,H., Chen,I., He,X. and Semenza,G.L. (2016) Hypoxia induces the breast cancer stem cell phenotype by HIF-dependent and ALKBH5-mediated m(6)A-demethylation of NANOG mRNA. Proc. Natl. Acad. Sci. U.S.A., 113, E2047-E2056.

Hypoxia-inducible factors regulate pluripotency factor expression by ZNF217-and ALKBH5-mediated modulation of RNA methylation in breast cancer cells. C Zhang, W I Zhi, H Lu, D Samanta, I Chen, E Gabrielson, G L Semenza, Oncotarget. 7Zhang,C., Zhi,W.I., Lu,H., Samanta,D., Chen,I., Gabrielson,E. and Semenza,G.L. (2016) Hypoxia-inducible factors regulate pluripotency factor expression by ZNF217-and ALKBH5-mediated modulation of RNA methylation in breast cancer cells. Oncotarget, 7, 64527-64542.

Coordination of m(6)A mRNA methylation and gene transcription by ZFP217 regulates pluripotency and reprogramming. F Aguilo, F Zhang, A Sancho, M Fidalgo, S Di Cecilia, A Vashisht, D F Lee, C H Chen, M Rengasamy, B Andino, Cell Stem Cell. 17Aguilo,F., Zhang,F., Sancho,A., Fidalgo,M., Di Cecilia,S., Vashisht,A., Lee,D.F., Chen,C.H., Rengasamy,M., Andino,B. et al. (2015) Coordination of m(6)A mRNA methylation and gene transcription by ZFP217 regulates pluripotency and reprogramming. Cell Stem Cell, 17, 689-704.

Oncogenic AURKA-enhanced N(6)-methyladenosine modification increases DROSHA mRNA stability to transactivate STC1 in breast cancer stem-like cells. F Peng, J Xu, B Cui, Q Liang, S Zeng, B He, H Zou, M Li, H Zhao, Y Meng, Cell Res. 31Peng,F., Xu,J., Cui,B., Liang,Q., Zeng,S., He,B., Zou,H., Li,M., Zhao,H., Meng,Y. et al. (2021) Oncogenic AURKA-enhanced N(6)-methyladenosine modification increases DROSHA mRNA stability to transactivate STC1 in breast cancer stem-like cells. Cell Res., 31, 345-361.

A novel hypoxic long noncoding RNA KB-1980E6.3 maintains breast cancer stem cell stemness via interacting with IGF2BP1 to facilitate c-Myc mRNA stability. P Zhu, F He, Y Hou, G Tu, Q Li, T Jin, H Zeng, Y Qin, X Wan, Y Qiao, Oncogene. 40Zhu,P., He,F., Hou,Y., Tu,G., Li,Q., Jin,T., Zeng,H., Qin,Y., Wan,X., Qiao,Y. et al. (2021) A novel hypoxic long noncoding RNA KB-1980E6.3 maintains breast cancer stem cell stemness via interacting with IGF2BP1 to facilitate c-Myc mRNA stability. Oncogene, 40, 1609-1627.

Adenylate kinase 4 modulates the resistance of breast cancer cells to tamoxifen through an m(6)A-based epitranscriptomic mechanism. X Liu, G Gonzalez, X Dai, W Miao, J Yuan, M Huang, D Bade, L Li, Y Sun, Y Wang, Mol. Ther. 28Liu,X., Gonzalez,G., Dai,X., Miao,W., Yuan,J., Huang,M., Bade,D., Li,L., Sun,Y. and Wang,Y. (2020) Adenylate kinase 4 modulates the resistance of breast cancer cells to tamoxifen through an m(6)A-based epitranscriptomic mechanism. Mol. Ther., 28, 2593-2604.

X Pan, X Hong, S Li, P Meng, F Xiao, METTL3 promotes adriamycin resistance in MCF-7 breast cancer cells by NAR Cancer. 315Pan,X., Hong,X., Li,S., Meng,P. and Xiao,F. (2021) METTL3 promotes adriamycin resistance in MCF-7 breast cancer cells by NAR Cancer, 2021, Vol. 3, No. 3 15

accelerating pri-microRNA-221-3p maturation in a m6A-dependent manner. Exp. Mol. Med. 53accelerating pri-microRNA-221-3p maturation in a m6A-dependent manner. Exp. Mol. Med., 53, 91-102.

Metformin exhibits antiproliferation activity in breast cancer via miR-483-3p/METTL3/m(6)A. L Cheng, X Zhang, Y Z Huang, Y L Zhu, L Y Xu, Z Li, X Y Dai, L Shi, X J Zhou, J F Wei, p21 pathway. Oncogenesis, 10, 7Cheng,L., Zhang,X., Huang,Y.Z., Zhu,Y.L., Xu,L.Y., Li,Z., Dai,X.Y., Shi,L., Zhou,X.J., Wei,J.F. et al. (2021) Metformin exhibits antiproliferation activity in breast cancer via miR-483-3p/METTL3/m(6)A/p21 pathway. Oncogenesis, 10, 7.

The human 18S rRNA m6A methyltransferase METTL5 is stabilized by TRMT112. N Van Tran, F G M Ernst, B R Hawley, C Zorbas, N Ulryck, P Hackert, K E Bohnsack, M T Bohnsack, S R Jaffrey, M Graille, Nucleic Acids Res. 47van Tran,N., Ernst,F.G.M., Hawley,B.R., Zorbas,C., Ulryck,N., Hackert,P., Bohnsack,K.E., Bohnsack,M.T., Jaffrey,S.R., Graille,M. et al. (2019) The human 18S rRNA m6A methyltransferase METTL5 is stabilized by TRMT112. Nucleic Acids Res., 47, 7719-7733.

Ribosome 18S m(6)A methyltransferase METTL5 promotes translation initiation and breast cancer cell growth. B Rong, Q Zhang, J Wan, S Xing, R Dai, Y Li, J Cai, J Xie, Y Song, J Chen, Cell Rep. 33108544Rong,B., Zhang,Q., Wan,J., Xing,S., Dai,R., Li,Y., Cai,J., Xie,J., Song,Y., Chen,J. et al. (2020) Ribosome 18S m(6)A methyltransferase METTL5 promotes translation initiation and breast cancer cell growth. Cell Rep., 33, 108544.

The human methyltransferase ZCCHC4 catalyses N6-methyladenosine modification of 28S ribosomal RNA. R Pinto, C B Vagbo, M E Jakobsson, Y Kim, M P Baltissen, M F O&apos;donohue, U H Guzman, J M Malecki, J Wu, F Kirpekar, Nucleic Acids Res. 48Pinto,R., Vagbo,C.B., Jakobsson,M.E., Kim,Y., Baltissen,M.P., O'Donohue,M.F., Guzman,U.H., Malecki,J.M., Wu,J., Kirpekar,F. et al. (2020) The human methyltransferase ZCCHC4 catalyses N6-methyladenosine modification of 28S ribosomal RNA. Nucleic Acids Res., 48, 830-846.

Transcriptome-wide mapping reveals reversible and dynamic N(1)-methyladenosine methylome. X Li, X Xiong, K Wang, L Wang, X Shu, S Ma, C Yi, Nat. Chem. Biol. 12Li,X., Xiong,X., Wang,K., Wang,L., Shu,X., Ma,S. and Yi,C. (2016) Transcriptome-wide mapping reveals reversible and dynamic N(1)-methyladenosine methylome. Nat. Chem. Biol., 12, 311-316.

2020) RNA modifications and cancer. P Haruehanroengra, Y Y Zheng, Y Zhou, Y Huang, J Sheng, RNA Biol. 17Haruehanroengra,P., Zheng,Y.Y., Zhou,Y., Huang,Y. and Sheng,J. (2020) RNA modifications and cancer. RNA Biol., 17, 1560-1575.

. S Oerum, C Degut, P Barraud, C Tisne, 1Oerum,S., Degut,C., Barraud,P. and Tisne,C. (2017) m1A

Post-Transcriptional, Modification in tRNAs. Biomolecules, 7. 20Post-Transcriptional Modification in tRNAs. Biomolecules, 7, 20.

The dynamic N(1)-methyladenosine methylome in eukaryotic messenger RNA. D Dominissini, S Nachtergaele, S Moshitch-Moshkovitz, E Peer, N Kol, M S Ben-Haim, Q Dai, A Di Segni, M Salmon-Divon, W C Clark, Nature. 530Dominissini,D., Nachtergaele,S., Moshitch-Moshkovitz,S., Peer,E., Kol,N., Ben-Haim,M.S., Dai,Q., Di Segni,A., Salmon-Divon,M., Clark,W.C. et al. (2016) The dynamic N(1)-methyladenosine methylome in eukaryotic messenger RNA. Nature, 530, 441-446.

The m1A landscape on cytosolic and mitochondrial mRNA at single-base resolution. M Safra, A Sas-Chen, R Nir, R Winkler, A Nachshon, D Bar-Yaacov, M Erlacher, W Rossmanith, N Stern-Ginossar, S Schwartz, Nature. 551Safra,M., Sas-Chen,A., Nir,R., Winkler,R., Nachshon,A., Bar-Yaacov,D., Erlacher,M., Rossmanith,W., Stern-Ginossar,N. and Schwartz,S. (2017) The m1A landscape on cytosolic and mitochondrial mRNA at single-base resolution. Nature, 551, 251-255.

Evolution of a reverse transcriptase to map N(1)-methyladenosine in human messenger RNA. H Zhou, S Rauch, Q Dai, X Cui, Z Zhang, S Nachtergaele, C Sepich, C He, B C Dickinson, Nat. Methods. 16Zhou,H., Rauch,S., Dai,Q., Cui,X., Zhang,Z., Nachtergaele,S., Sepich,C., He,C. and Dickinson,B.C. (2019) Evolution of a reverse transcriptase to map N(1)-methyladenosine in human messenger RNA. Nat. Methods, 16, 1281-1288.

Human ALKBH3-induced m(1)A demethylation increases the CSF-1 mRNA stability in breast and ovarian cancer cells. H H Woo, S K Chambers, Biochim. Biophys. Acta Gene Regul. Mech. 1862Woo,H.H. and Chambers,S.K. (2019) Human ALKBH3-induced m(1)A demethylation increases the CSF-1 mRNA stability in breast and ovarian cancer cells. Biochim. Biophys. Acta Gene Regul. Mech., 1862, 35-46.

) 5-methylcytosine promotes mRNA export -NSUN2 as the methyltransferase and ALYREF as an m(5)C reader. X Yang, Y Yang, B F Sun, Y S Chen, J W Xu, W Y Lai, A Li, X Wang, D P Bhattarai, W Xiao, Cell Res. 27Yang,X., Yang,Y., Sun,B.F., Chen,Y.S., Xu,J.W., Lai,W.Y., Li,A., Wang,X., Bhattarai,D.P., Xiao,W. et al. (2017) 5-methylcytosine promotes mRNA export -NSUN2 as the methyltransferase and ALYREF as an m(5)C reader. Cell Res., 27, 606-625.

) 5-methylcytosine promotes pathogenesis of bladder cancer through stabilizing mRNAs. X Chen, A Li, B F Sun, Y Yang, Y N Han, X Yuan, R X Chen, W S Wei, Y Liu, C C Gao, Nat. Cell Biol. 21Chen,X., Li,A., Sun,B.F., Yang,Y., Han,Y.N., Yuan,X., Chen,R.X., Wei,W.S., Liu,Y., Gao,C.C. et al. (2019) 5-methylcytosine promotes pathogenesis of bladder cancer through stabilizing mRNAs. Nat. Cell Biol., 21, 978-990.

Corrigendum: Methylation of ribosomal RNA by NSUN5 is a conserved mechanism modulating organismal lifespan. M Schosserer, N Minois, T B Angerer, M Amring, H Dellago, E Harreither, A Calle-Perez, A Pircher, M Peter Gerstl, S Pfeifenberger, Nat. Commun. 711530Schosserer,M., Minois,N., Angerer,T.B., Amring,M., Dellago,H., Harreither,E., Calle-Perez,A., Pircher,A., Peter Gerstl,M., Pfeifenberger,S. et al. (2016) Corrigendum: Methylation of ribosomal RNA by NSUN5 is a conserved mechanism modulating organismal lifespan. Nat. Commun., 7, 11530.

Aberrant methylation of tRNAs links cellular stress to neuro-developmental disorders. S Blanco, S Dietmann, J V Flores, S Hussain, C Kutter, P Humphreys, M Lukk, P Lombard, L Treps, M Popis, EMBO J. 33Blanco,S., Dietmann,S., Flores,J.V., Hussain,S., Kutter,C., Humphreys,P., Lukk,M., Lombard,P., Treps,L., Popis,M. et al. (2014) Aberrant methylation of tRNAs links cellular stress to neuro-developmental disorders. EMBO J., 33, 2020-2039.

RNA methylation by Dnmt2 protects transfer RNAs against stress-induced cleavage. M Schaefer, T Pollex, K Hanna, F Tuorto, M Meusburger, M Helm, F Lyko, Genes Dev. 24Schaefer,M., Pollex,T., Hanna,K., Tuorto,F., Meusburger,M., Helm,M. and Lyko,F. (2010) RNA methylation by Dnmt2 protects transfer RNAs against stress-induced cleavage. Genes Dev., 24, 1590-1595.

Novel insights into the emerging roles of tRNA-derived fragments in mammalian development. N Guzzi, C Bellodi, RNA Biol. 17Guzzi,N. and Bellodi,C. (2020) Novel insights into the emerging roles of tRNA-derived fragments in mammalian development. RNA Biol, 17, 1214-1222.

Detection of N6methyladenosine modification residues (Review). W Zhu, J Z Wang, Z Xu, M Cao, Q Hu, C Pan, M Guo, J F Wei, H Yang, Int. J. Mol. Med. 43Zhu,W., Wang,J.Z., Xu,Z., Cao,M., Hu,Q., Pan,C., Guo,M., Wei,J.F. and Yang,H. (2019) Detection of N6methyladenosine modification residues (Review). Int. J. Mol. Med., 43, 2267-2278.

NSun2 delays replicative senescence by repressing p27 (KIP1) translation and elevating CDK1 translation. H Tang, X Fan, J Xing, Z Liu, B Jiang, Y Dou, M Gorospe, W Wang, Aging (Albany NY). 7Tang,H., Fan,X., Xing,J., Liu,Z., Jiang,B., Dou,Y., Gorospe,M. and Wang,W. (2015) NSun2 delays replicative senescence by repressing p27 (KIP1) translation and elevating CDK1 translation. Aging (Albany NY), 7, 1143-1158.

Eukaryotic 5-methylcytosine (m(5)C) RNA methyltransferases: mechanisms, cellular functions, and links to disease. K E Bohnsack, C Hobartner, M T Bohnsack, Genes (Basel). 10102Bohnsack,K.E., Hobartner,C. and Bohnsack,M.T. (2019) Eukaryotic 5-methylcytosine (m(5)C) RNA methyltransferases: mechanisms, cellular functions, and links to disease. Genes (Basel), 10, 102.

Epigenetic loss of RNA-methyltransferase NSUN5 in glioma targets ribosomes to drive a stress adaptive translational program. M Janin, V Ortiz-Barahona, M C De Moura, A Martinez-Cardus, P Llinas-Arias, M Soler, D Nachmani, J Pelletier, U Schumann, M E Calleja-Cervantes, Acta Neuropathol. 138Janin,M., Ortiz-Barahona,V., de Moura,M.C., Martinez-Cardus,A., Llinas-Arias,P., Soler,M., Nachmani,D., Pelletier,J., Schumann,U., Calleja-Cervantes,M.E. et al. (2019) Epigenetic loss of RNA-methyltransferase NSUN5 in glioma targets ribosomes to drive a stress adaptive translational program. Acta Neuropathol., 138, 1053-1074.

Loss of the ribosomal RNA methyltransferase NSUN5 impairs global protein synthesis and normal growth. C Heissenberger, L Liendl, F Nagelreiter, Y Gonskikh, G Yang, E M Stelzer, T L Krammer, L Micutkova, S Vogt, D P Kreil, Nucleic Acids Res. 47Heissenberger,C., Liendl,L., Nagelreiter,F., Gonskikh,Y., Yang,G., Stelzer,E.M., Krammer,T.L., Micutkova,L., Vogt,S., Kreil,D.P. et al. (2019) Loss of the ribosomal RNA methyltransferase NSUN5 impairs global protein synthesis and normal growth. Nucleic Acids Res., 47, 11807-11825.

The ribosomal RNA m(5)C methyltransferase NSUN-1 modulates healthspan and oogenesis in Caenorhabditis elegans. C Heissenberger, J A Rollins, T L Krammer, F Nagelreiter, I Stocker, L Wacheul, A Shpylovyi, K Tav, S Snow, J Grillari, 56205Elife, 9Heissenberger,C., Rollins,J.A., Krammer,T.L., Nagelreiter,F., Stocker,I., Wacheul,L., Shpylovyi,A., Tav,K., Snow,S., Grillari,J. et al. (2020) The ribosomal RNA m(5)C methyltransferase NSUN-1 modulates healthspan and oogenesis in Caenorhabditis elegans. Elife, 9, e56205.

NSUN4 is a dual function mitochondrial protein required for both methylation of 12S rRNA and coordination of mitoribosomal assembly. M D Metodiev, H Spahr, P Loguercio Polosa, C Meharg, C Becker, J Altmueller, B Habermann, N G Larsson, B Ruzzenente, PLos Genet. 101004110Metodiev,M.D., Spahr,H., Loguercio Polosa,P., Meharg,C., Becker,C., Altmueller,J., Habermann,B., Larsson,N.G. and Ruzzenente,B. (2014) NSUN4 is a dual function mitochondrial protein required for both methylation of 12S rRNA and coordination of mitoribosomal assembly. PLos Genet., 10, e1004110.

RNA cytosine methylation by Dnmt2 and NSun2 promotes tRNA stability and protein synthesis. F Tuorto, R Liebers, T Musch, M Schaefer, S Hofmann, S Kellner, M Frye, M Helm, G Stoecklin, F Lyko, Nat. Struct. Mol. Biol. 19Tuorto,F., Liebers,R., Musch,T., Schaefer,M., Hofmann,S., Kellner,S., Frye,M., Helm,M., Stoecklin,G. and Lyko,F. (2012) RNA cytosine methylation by Dnmt2 and NSun2 promotes tRNA stability and protein synthesis. Nat. Struct. Mol. Biol., 19, 900-905.

Identification of direct targets and modified bases of RNA cytosine methyltransferases. V Khoddami, B R Cairns, Nat. Biotechnol. 31Khoddami,V. and Cairns,B.R. (2013) Identification of direct targets and modified bases of RNA cytosine methyltransferases. Nat. Biotechnol., 31, 458-464.

Methylation of tRNAAsp by the DNA methyltransferase homolog Dnmt2. M G Goll, F Kirpekar, K A Maggert, J A Yoder, C L Hsieh, X Zhang, K G Golic, S E Jacobsen, T H Bestor, Science. 311Goll,M.G., Kirpekar,F., Maggert,K.A., Yoder,J.A., Hsieh,C.L., Zhang,X., Golic,K.G., Jacobsen,S.E. and Bestor,T.H. (2006) Methylation of tRNAAsp by the DNA methyltransferase homolog Dnmt2. Science, 311, 395-398.

The RNA methyltransferase Misu (NSun2) mediates Myc-induced proliferation and is upregulated in tumors. M Frye, F M Watt, Curr. Biol. 16Frye,M. and Watt,F.M. (2006) The RNA methyltransferase Misu (NSun2) mediates Myc-induced proliferation and is upregulated in tumors. Curr. Biol., 16, 971-981.

NSUN6 is a human RNA methyltransferase that catalyzes formation of m5C72 in specific tRNAs. S Haag, A S Warda, J Kretschmer, M A Gunnigmann, C Hobartner, M T Bohnsack, RNA. 21Haag,S., Warda,A.S., Kretschmer,J., Gunnigmann,M.A., Hobartner,C. and Bohnsack,M.T. (2015) NSUN6 is a human RNA methyltransferase that catalyzes formation of m5C72 in specific tRNAs. RNA, 21, 1532-1543.

Sequence-and structure-specific cytosine-5 mRNA methylation by NSUN6. T Selmi, S Hussain, S Dietmann, M Heiss, K Borland, S Flad, J M Carter, R Dennison, Y L Huang, S Kellner, Nucleic Acids Res. 49Selmi,T., Hussain,S., Dietmann,S., Heiss,M., Borland,K., Flad,S., Carter,J.M., Dennison,R., Huang,Y.L., Kellner,S. et al. (2021) Sequence-and structure-specific cytosine-5 mRNA methylation by NSUN6. Nucleic Acids Res., 49, 1006-1022.

Loss of 5-methylcytosine alters the biogenesis of vault-derived small RNAs to coordinate epidermal differentiation. A A Sajini, N R Choudhury, R E Wagner, S Bornelov, T Selmi, C Spanos, S Dietmann, J Rappsilber, G Michlewski, M Frye, Nat. Commun. 102550Sajini,A.A., Choudhury,N.R., Wagner,R.E., Bornelov,S., Selmi,T., Spanos,C., Dietmann,S., Rappsilber,J., Michlewski,G. and Frye,M. (2019) Loss of 5-methylcytosine alters the biogenesis of vault-derived small RNAs to coordinate epidermal differentiation. Nat. Commun., 10, 2550.

Deposition of 5-methylcytosine on enhancer RNAs enables the coactivator function of PGC-1alpha. F Aguilo, S Li, N Balasubramaniyan, A Sancho, S Benko, F Zhang, A Vashisht, M Rengasamy, B Andino, C H Chen, Cell Rep. 14Aguilo,F., Li,S., Balasubramaniyan,N., Sancho,A., Benko,S., Zhang,F., Vashisht,A., Rengasamy,M., Andino,B., Chen,C.H. et al. (2016) Deposition of 5-methylcytosine on enhancer RNAs enables the coactivator function of PGC-1alpha. Cell Rep., 14, 479-492.

Prognostic significance of proliferation associated nucleolar antigen p120 in human breast carcinoma. J W Freeman, P Mcgrath, V Bondada, N Selliah, H Ownby, T Maloney, R K Busch, H Busch, Cancer Res. 51Freeman,J.W., McGrath,P., Bondada,V., Selliah,N., Ownby,H., Maloney,T., Busch,R. K. and Busch,H. (1991) Prognostic significance of proliferation associated nucleolar antigen p120 in human breast carcinoma. Cancer Res., 51, 1973-1978.

Genomic gain of 5p15 leads to over-expression of Misu (NSUN2) in breast cancer. M Frye, I Dragoni, S F Chin, I Spiteri, A Kurowski, E Provenzano, A Green, I O Ellis, D Grimmer, A Teschendorff, Cancer Lett. 289Frye,M., Dragoni,I., Chin,S.F., Spiteri,I., Kurowski,A., Provenzano,E., Green,A., Ellis,I.O., Grimmer,D., Teschendorff,A. et al. (2010) Genomic gain of 5p15 leads to over-expression of Misu (NSUN2) in breast cancer. Cancer Lett., 289, 71-80.

Genome-wide meta-analyses of breast, ovarian, and prostate cancer association studies identify multiple new susceptibility loci shared by at least two cancer types. S P Kar, J Beesley, A Amin Al Olama, K Michailidou, J Tyrer, Z Kote-Jarai, K Lawrenson, S Lindstrom, S J Ramus, D J Thompson, Cancer Discov. 6Kar,S.P., Beesley,J., Amin Al Olama,A., Michailidou,K., Tyrer,J., Kote-Jarai,Z., Lawrenson,K., Lindstrom,S., Ramus,S.J., Thompson,D.J. et al. (2016) Genome-wide meta-analyses of breast, ovarian, and prostate cancer association studies identify multiple new susceptibility loci shared by at least two cancer types. Cancer Discov., 6, 1052-1067.

Strong evidence of a genetic determinant for mammographic density, a major risk factor for breast cancer. C M Vachon, T A Sellers, E E Carlson, J M Cunningham, C A Hilker, R L Smalley, D J Schaid, L E Kelemen, F J Couch, V S Pankratz, Cancer Res. 67Vachon,C.M., Sellers,T.A., Carlson,E.E., Cunningham,J.M., Hilker,C.A., Smalley,R.L., Schaid,D.J., Kelemen,L.E., Couch,F.J. and Pankratz,V.S. (2007) Strong evidence of a genetic determinant for mammographic density, a major risk factor for breast cancer. Cancer Res., 67, 8412-8418.

Overexpression of NSUN2 by DNA hypomethylation is associated with metastatic progression in human breast cancer. J Yi, R Gao, Y Chen, Z Yang, P Han, H Zhang, Y Dou, W Liu, W Wang, G Du, Oncotarget. 8Yi,J., Gao,R., Chen,Y., Yang,Z., Han,P., Zhang,H., Dou,Y., Liu,W., Wang,W., Du,G. et al. (2017) Overexpression of NSUN2 by DNA hypomethylation is associated with metastatic progression in human breast cancer. Oncotarget, 8, 20751-20765.

Pan-cancer analysis of RNA methyltransferases identifies FTSJ3 as a potential regulator of breast cancer progression. M Manning, Y Jiang, R Wang, L Liu, S Rode, M Bonahoom, S Kim, Z Q Yang, RNA Biol. 17Manning,M., Jiang,Y., Wang,R., Liu,L., Rode,S., Bonahoom,M., Kim,S. and Yang,Z.Q. (2020) Pan-cancer analysis of RNA methyltransferases identifies FTSJ3 as a potential regulator of breast cancer progression. RNA Biol., 17, 474-486.

Mammalian NSUN2 introduces 5-methylcytidines into mitochondrial tRNAs. S Shinoda, S Kitagawa, S Nakagawa, F Y Wei, K Tomizawa, K Araki, M Araki, T Suzuki, T Suzuki, Nucleic Acids Res. 47Shinoda,S., Kitagawa,S., Nakagawa,S., Wei,F.Y., Tomizawa,K., Araki,K., Araki,M., Suzuki,T. and Suzuki,T. (2019) Mammalian NSUN2 introduces 5-methylcytidines into mitochondrial tRNAs. Nucleic Acids Res., 47, 8734-8745.

Prognostic Significance and Tumor Immune Microenvironment Heterogenicity of m5C RNA Methylation Regulators in Triple-Negative Breast Cancer. Z Huang, J Pan, H Wang, X Du, Y Xu, Z Wang, D Chen, Front. Cell Dev. Biol. 9657547Huang,Z., Pan,J., Wang,H., Du,X., Xu,Y., Wang,Z. and Chen,D. (2021) Prognostic Significance and Tumor Immune Microenvironment Heterogenicity of m5C RNA Methylation Regulators in Triple-Negative Breast Cancer. Front. Cell Dev. Biol., 9, 657547.

Y-box binding protein, YB-1, as a marker of tumor aggressiveness and response to adjuvant chemotherapy in breast cancer. J Huang, P H Tan, K B Li, K Matsumoto, M Tsujimoto, B H Bay, Int. J. Oncol. 26Huang,J., Tan,P.H., Li,K.B., Matsumoto,K., Tsujimoto,M. and Bay,B.H. (2005) Y-box binding protein, YB-1, as a marker of tumor aggressiveness and response to adjuvant chemotherapy in breast cancer. Int. J. Oncol., 26, 607-613.

Elevated nuclear YB1 expression is associated with poor survival of patients with early breast cancer. A Maciejczyk, J Szelachowska, M Ekiert, R Matkowski, A Halon, H Lage, P Surowiak, Anticancer Res. 32Maciejczyk,A., Szelachowska,J., Ekiert,M., Matkowski,R., Halon,A., Lage,H. and Surowiak,P. (2012) Elevated nuclear YB1 expression is associated with poor survival of patients with early breast cancer. Anticancer Res., 32, 3177-3184.

Characterization of two homologous 2 -O-methyltransferases showing different specificities for their tRNA substrates. J Somme, B Van Laer, M Roovers, J Steyaert, W Versees, L Droogmans, RNA. 20Somme,J., Van Laer,B., Roovers,M., Steyaert,J., Versees,W. and Droogmans,L. (2014) Characterization of two homologous 2 -O-methyltransferases showing different specificities for their tRNA substrates. RNA, 20, 1257-1271.

Functional organization of box C/D RNA-guided RNA methyltransferase. Z Yang, J Wang, L Huang, D M J Lilley, K Ye, Nucleic Acids Res. 48Yang,Z., Wang,J., Huang,L., Lilley,D.M.J. and Ye,K. (2020) Functional organization of box C/D RNA-guided RNA methyltransferase. Nucleic Acids Res., 48, 5094-5105.

Identification of sites of 2 -O-methylation vulnerability in human ribosomal RNAs by systematic mapping. S Sharma, V Marchand, Y Motorin, D L J Lafontaine, Sci. Rep. 711490Sharma,S., Marchand,V., Motorin,Y. and Lafontaine,D.L.J. (2017) Identification of sites of 2 -O-methylation vulnerability in human ribosomal RNAs by systematic mapping. Sci. Rep., 7, 11490.

RNA ribose methylation (2 -O-methylation): Occurrence, biosynthesis and biological functions. L Ayadi, A Galvanin, F Pichot, V Marchand, Y Motorin, Biochim. Biophys. Acta Gene Regul. Mech. 1862Ayadi,L., Galvanin,A., Pichot,F., Marchand,V. and Motorin,Y. (2019) RNA ribose methylation (2 -O-methylation): Occurrence, biosynthesis and biological functions. Biochim. Biophys. Acta Gene Regul. Mech., 1862, 253-269.

Long noncoding RNA ZFAS1 promoting small nucleolar RNA-mediated 2 -O-methylation via NOP58 recruitment in colorectal cancer. H Wu, W Qin, S Lu, X Wang, J Zhang, T Sun, X Hu, Y Li, Q Chen, Y Wang, Mol. Cancer. 1995Wu,H., Qin,W., Lu,S., Wang,X., Zhang,J., Sun,T., Hu,X., Li,Y., Chen,Q., Wang,Y. et al. (2020) Long noncoding RNA ZFAS1 promoting small nucleolar RNA-mediated 2 -O-methylation via NOP58 recruitment in colorectal cancer. Mol. Cancer, 19, 95.

Nopp140-chaperoned 2 -O-methylation of small nuclear RNAs in Cajal bodies ensures splicing fidelity. J Bizarro, S Deryusheva, L Wacheul, V Gupta, F G M Ernst, D L J Lafontaine, J G Gall, U T Meier, Genes Dev. 35Bizarro,J., Deryusheva,S., Wacheul,L., Gupta,V., Ernst,F.G.M., Lafontaine,D.L.J., Gall,J.G. and Meier,U.T. (2021) Nopp140-chaperoned 2 -O-methylation of small nuclear RNAs in Cajal bodies ensures splicing fidelity. Genes Dev., 35, 1123-1141.

Next-Generation Sequencing-Based RiboMethSeq Protocol for Analysis of tRNA 2 -O-Methylation. V Marchand, F Pichot, K Thuring, L Ayadi, I Freund, A Dalpke, M Helm, Y Motorin, 13Biomolecules, 7Marchand,V., Pichot,F., Thuring,K., Ayadi,L., Freund,I., Dalpke,A., Helm,M. and Motorin,Y. (2017) Next-Generation Sequencing-Based RiboMethSeq Protocol for Analysis of tRNA 2 -O-Methylation. Biomolecules, 7, 13.

2020) tRNA 2 -O-methylation by a duo of TRM7/FTSJ1 proteins modulates small RNA silencing in Drosophila. M T Angelova, D G Dimitrova, B Da Silva, V Marchand, C Jacquier, C Achour, M Brazane, C Goyenvalle, V Bourguignon-Igel, S Shehzada, Nucleic Acids Res. 48Angelova,M.T., Dimitrova,D.G., Da Silva,B., Marchand,V., Jacquier,C., Achour,C., Brazane,M., Goyenvalle,C., Bourguignon-Igel,V., Shehzada,S. et al. (2020) tRNA 2 -O-methylation by a duo of TRM7/FTSJ1 proteins modulates small RNA silencing in Drosophila. Nucleic Acids Res., 48, 2050-2072.

Modification of messenger RNA by 2 -O-methylation regulates gene expression in vivo. B A Elliott, H T Ho, S V Ranganathan, S Vangaveti, O Ilkayeva, H Abou Assi, A K Choi, P F Agris, C L Holley, Nat. Commun. 103401Elliott,B.A., Ho,H.T., Ranganathan,S.V., Vangaveti,S., Ilkayeva,O., Abou Assi,H., Choi,A.K., Agris,P.F. and Holley,C.L. (2019) Modification of messenger RNA by 2 -O-methylation regulates gene expression in vivo. Nat. Commun., 10, 3401.

2 -O-methylation stabilizes Piwi-associated small RNAs and ensures DNA elimination in Tetrahymena. H M Kurth, K Mochizuki, RNA. 15Kurth,H.M. and Mochizuki,K. (2009) 2 -O-methylation stabilizes Piwi-associated small RNAs and ensures DNA elimination in Tetrahymena. RNA, 15, 675-685.

Direct quantification of 3 terminal 2 -O-methylation of small RNAs by RT-qPCR. N Wang, S Qu, W Sun, Z Zeng, H Liang, C Y Zhang, X Chen, K Zen, RNA. 24Wang,N., Qu,S., Sun,W., Zeng,Z., Liang,H., Zhang,C.Y., Chen,X. and Zen,K. (2018) Direct quantification of 3 terminal 2 -O-methylation of small RNAs by RT-qPCR. RNA, 24, 1520-1529.

Optimization of enzymatic reaction conditions for generating representative pools of cDNA from small RNA. D B Munafo, G B Robb, RNA. 16Munafo,D.B. and Robb,G.B. (2010) Optimization of enzymatic reaction conditions for generating representative pools of cDNA from small RNA. RNA, 16, 2537-2552.

Evidence for rRNA 2 -O-methylation plasticity: Control of intrinsic translational capabilities of human ribosomes. J Erales, V Marchand, B Panthu, S Gillot, S Belin, S E Ghayad, M Garcia, F Laforets, V Marcel, A Baudin-Baillieu, Proc. Natl. Acad. Sci. U.S.A. 114Erales,J., Marchand,V., Panthu,B., Gillot,S., Belin,S., Ghayad,S.E., Garcia,M., Laforets,F., Marcel,V., Baudin-Baillieu,A. et al. (2017) Evidence for rRNA 2 -O-methylation plasticity: Control of intrinsic translational capabilities of human ribosomes. Proc. Natl. Acad. Sci. U.S.A., 114, 12934-12939.

Dysregulation of ribosome biogenesis and translational capacity is associated with tumor progression of human breast cancer cells. S Belin, A Beghin, E Solano-Gonzalez, L Bezin, S Brunet-Manquat, J Textoris, A C Prats, H C Mertani, C Dumontet, J J Diaz, PLoS One. 47147Belin,S., Beghin,A., Solano-Gonzalez,E., Bezin,L., Brunet-Manquat,S., Textoris,J., Prats,A.C., Mertani,H.C., Dumontet,C. and Diaz,J.J. (2009) Dysregulation of ribosome biogenesis and translational capacity is associated with tumor progression of human breast cancer cells. PLoS One, 4, e7147.

Illumina-based RiboMethSeq approach for mapping of 2 -O-Me residues in RNA. V Marchand, F Blanloeil-Oillo, M Helm, Y Motorin, Nucleic Acids Res. 44135Marchand,V., Blanloeil-Oillo,F., Helm,M. and Motorin,Y. (2016) Illumina-based RiboMethSeq approach for mapping of 2 -O-Me residues in RNA. Nucleic Acids Res., 44, e135.

. V Marcel, J Kielbassa, V Marchand, K S Natchiar, H Paraqindes, F Nguyen Van Long, L Ayadi, V Bourguignon-Igel, P Lo Monaco, D Monchiet, Marcel,V., Kielbassa,J., Marchand,V., Natchiar,K.S., Paraqindes,H., Nguyen Van Long,F., Ayadi,L., Bourguignon-Igel,V., Lo Monaco,P., Monchiet,D. et al. (2020)

Ribosomal RNA 2 O-methylation as a novel layer of inter-tumour heterogeneity in breast cancer. NAR Cancer. 236Ribosomal RNA 2 O-methylation as a novel layer of inter-tumour heterogeneity in breast cancer. NAR Cancer, 2, zcaa036.

Hypoxia re-programs 2 -O-Me modifications on ribosomal RNA. B J Metge, S C Kammerud, H C Pruitt, L A Shevde, R S Samant, 102010Metge,B.J., Kammerud,S.C., Pruitt,H.C., Shevde,L.A. and Samant,R.S. (2021) Hypoxia re-programs 2 -O-Me modifications on ribosomal RNA. iScience, 24, 102010.

Hypoxia-inducible factors are required for chemotherapy resistance of breast cancer stem cells. D Samanta, D M Gilkes, P Chaturvedi, L Xiang, G L Semenza, Proc. Natl. Acad. Sci. U.S.A. 111Samanta,D., Gilkes,D.M., Chaturvedi,P., Xiang,L. and Semenza,G.L. (2014) Hypoxia-inducible factors are required for chemotherapy resistance of breast cancer stem cells. Proc. Natl. Acad. Sci. U.S.A., 111, E5429-E5438.

A Pan-cancer Analysis of the Expression and Clinical Relevance of Small Nucleolar RNAs in Human Cancer. J Gong, Y Li, C J Liu, Y Xiang, C Li, Y Ye, Z Zhang, D H Hawke, P K Park, L Diao, Cell Rep. 21Gong,J., Li,Y., Liu,C.J., Xiang,Y., Li,C., Ye,Y., Zhang,Z., Hawke,D.H., Park,P.K., Diao,L. et al. (2017) A Pan-cancer Analysis of the Expression and Clinical Relevance of Small Nucleolar RNAs in Human Cancer. Cell Rep., 21, 1968-1981.

Proliferation, cancer, and aging-novel functions of the nucleolar methyltransferase fibrillarin?. M Y Shubina, Y R Musinova, E V Sheval, Cell Biol. Int. 42Shubina,M.Y., Musinova,Y.R. and Sheval,E.V. (2018) Proliferation, cancer, and aging-novel functions of the nucleolar methyltransferase fibrillarin? Cell Biol. Int., 42, 1463-1466.

acts as a safeguard of translational control by regulating fibrillarin and rRNA methylation in cancer. V Marcel, S E Ghayad, S Belin, G Therizols, A P Morel, E Solano-Gonzalez, J A Vendrell, S Hacot, H C Mertani, M A Albaret, Cancer Cell. 24Marcel,V., Ghayad,S.E., Belin,S., Therizols,G., Morel,A.P., Solano-Gonzalez,E., Vendrell,J.A., Hacot,S., Mertani,H.C., Albaret,M.A. et al. (2013) p53 acts as a safeguard of translational control by regulating fibrillarin and rRNA methylation in cancer. Cancer Cell, 24, 318-330.

The landscape of small non-coding RNAs in triple-negative breast cancer. Y Guo, H Yu, J Wang, Q Sheng, S Zhao, Y Y Zhao, B D Lehmann, Genes (Basel). 929Guo,Y., Yu,H., Wang,J., Sheng,Q., Zhao,S., Zhao,Y.Y. and Lehmann,B.D. (2018) The landscape of small non-coding RNAs in triple-negative breast cancer. Genes (Basel), 9, 29.

Profiling of small nucleolar RNAs by next generation sequencing: potential new players for breast cancer prognosis. P Krishnan, S Ghosh, B Wang, M Heyns, K Graham, J R Mackey, O Kovalchuk, S Damaraju, PLoS One. 162622Krishnan,P., Ghosh,S., Wang,B., Heyns,M., Graham,K., Mackey,J.R., Kovalchuk,O. and Damaraju,S. (2016) Profiling of small nucleolar RNAs by next generation sequencing: potential new players for breast cancer prognosis. PLoS One, 11, e0162622.

Identification of a gene signature for different stages of breast cancer development that could be used for early diagnosis and specific therapy. C Kothari, G Ouellette, Y Labrie, S Jacob, C Diorio, F Durocher, Oncotarget. 9Kothari,C., Ouellette,G., Labrie,Y., Jacob,S., Diorio,C. and Durocher,F. (2018) Identification of a gene signature for different stages of breast cancer development that could be used for early diagnosis and specific therapy. Oncotarget, 9, 37407-37420.

GAS5, a non-protein-coding RNA, controls apoptosis and is downregulated in breast cancer. M Mourtada-Maarabouni, M R Pickard, V L Hedge, F Farzaneh, G T Williams, Oncogene. 28Mourtada-Maarabouni,M., Pickard,M.R., Hedge,V.L., Farzaneh,F. and Williams,G.T. (2009) GAS5, a non-protein-coding RNA, controls apoptosis and is downregulated in breast cancer. Oncogene, 28, 195-208.

An Emerging Class of Long Non-coding RNA With Oncogenic Role Arises From the snoRNA Host Genes. A A Zimta, A B Tigu, C Braicu, C Stefan, C Ionescu, I Berindan-Neagoe, Front. Oncol. 10389Zimta,A.A., Tigu,A.B., Braicu,C., Stefan,C., Ionescu,C. and Berindan-Neagoe,I. (2020) An Emerging Class of Long Non-coding RNA With Oncogenic Role Arises From the snoRNA Host Genes. Front. Oncol., 10, 389.

Long non-coding small nucleolar RNA host genes (SNHGs) in endocrine-related cancers. Y Qin, W Sun, Z Wang, W Dong, L He, T Zhang, H Zhang, Onco Targets Ther. 13Qin,Y., Sun,W., Wang,Z., Dong,W., He,L., Zhang,T. and Zhang,H. (2020) Long non-coding small nucleolar RNA host genes (SNHGs) in endocrine-related cancers. Onco Targets Ther, 13, 7699-7717.

SNORD-host RNA Zfas1 is a regulator of mammary development and a potential marker for breast cancer. M E Askarian-Amiri, J Crawford, J D French, C E Smart, M A Smith, M B Clark, K Ru, T R Mercer, E R Thompson, S R Lakhani, RNA. 17Askarian-Amiri,M.E., Crawford,J., French,J.D., Smart,C.E., Smith,M.A., Clark,M.B., Ru,K., Mercer,T.R., Thompson,E.R., Lakhani,S.R. et al. (2011) SNORD-host RNA Zfas1 is a regulator of mammary development and a potential marker for breast cancer. RNA, 17, 878-891.

High-throughput identification of C/D box snoRNA targets with CLIP and RiboMeth-seq. R Gumienny, D J Jedlinski, A Schmidt, F Gypas, G Martin, A Vina-Vilaseca, M Zavolan, Nucleic Acids Res. 45Gumienny,R., Jedlinski,D.J., Schmidt,A., Gypas,F., Martin,G., Vina-Vilaseca,A. and Zavolan,M. (2017) High-throughput identification of C/D box snoRNA targets with CLIP and RiboMeth-seq. Nucleic Acids Res., 45, 2341-2353.

A flexible RNA backbone within the polypyrimidine tract is required for U2AF65 binding and pre-mRNA splicing in vivo. C Chen, X Zhao, R Kierzek, Y T Yu, Mol. Cell. Biol. 30Chen,C., Zhao,X., Kierzek,R. and Yu,Y.T. (2010) A flexible RNA backbone within the polypyrimidine tract is required for U2AF65 binding and pre-mRNA splicing in vivo. Mol. Cell. Biol., 30, 4108-4119.

. M D De Zoysa, Y T Yu, Posttranscriptional RNA pseudouridylation. Enzymes. 41De Zoysa,M.D. and Yu,Y.T. (2017) Posttranscriptional RNA pseudouridylation. Enzymes, 41, 151-167.

The role of m(6)A, m(5)C and Psi RNA modifications in cancer: novel therapeutic opportunities. P Nombela, B Miguel-Lopez, S Blanco, Mol. Cancer. 2018Nombela,P., Miguel-Lopez,B. and Blanco,S. (2021) The role of m(6)A, m(5)C and Psi RNA modifications in cancer: novel therapeutic opportunities. Mol. Cancer, 20, 18.

Dyskerin expression influences the level of ribosomal RNA pseudo-uridylation and telomerase RNA component in human breast cancer. L Montanaro, M Brigotti, J Clohessy, S Barbieri, C Ceccarelli, D Santini, M Taffurelli, M Calienni, J Teruya-Feldstein, D Trere, J. Pathol. 210Montanaro,L., Brigotti,M., Clohessy,J., Barbieri,S., Ceccarelli,C., Santini,D., Taffurelli,M., Calienni,M., Teruya-Feldstein,J., Trere,D. et al. (2006) Dyskerin expression influences the level of ribosomal RNA pseudo-uridylation and telomerase RNA component in human breast cancer. J. Pathol., 210, 10-18.

Inhibition of human dyskerin as a new approach to target ribosome biogenesis. L Rocchi, A J Barbosa, C Onofrillo, A Del Rio, L Montanaro, PLoS One. 9101971Rocchi,L., Barbosa,A.J., Onofrillo,C., Del Rio,A. and Montanaro,L. (2014) Inhibition of human dyskerin as a new approach to target ribosome biogenesis. PLoS One, 9, e101971.

Novel dyskerin-mediated mechanism of p53 inactivation through defective mRNA translation. L Montanaro, M Calienni, S Bertoni, L Rocchi, P Sansone, G Storci, D Santini, C Ceccarelli, M Taffurelli, D Carnicelli, Cancer Res. 70Montanaro,L., Calienni,M., Bertoni,S., Rocchi,L., Sansone,P., Storci,G., Santini,D., Ceccarelli,C., Taffurelli,M., Carnicelli,D. et al. (2010) Novel dyskerin-mediated mechanism of p53 inactivation through defective mRNA translation. Cancer Res., 70, 4767-4777.

Dyskerin depletion increases VEGF mRNA internal ribosome entry site-mediated translation. L Rocchi, A Pacilli, R Sethi, M Penzo, R J Schneider, D Trere, M Brigotti, L Montanaro, Nucleic Acids Res. 41Rocchi,L., Pacilli,A., Sethi,R., Penzo,M., Schneider,R.J., Trere,D., Brigotti,M. and Montanaro,L. (2013) Dyskerin depletion increases VEGF mRNA internal ribosome entry site-mediated translation. Nucleic Acids Res., 41, 8308-8318.

DKC1 overexpression induces a more aggressive cellular behavior and increases intrinsic ribosomal activity in immortalized mammary gland cells. A N Guerrieri, F Zacchini, C Onofrillo, S Di Viggiano, M Penzo, A Ansuini, I Gandin, Y Nobe, M Taoka, T Isobe, Cancers (Basel). 123512Guerrieri,A.N., Zacchini,F., Onofrillo,C., Di Viggiano,S., Penzo,M., Ansuini,A., Gandin,I., Nobe,Y., Taoka,M., Isobe,T. et al. (2020) DKC1 overexpression induces a more aggressive cellular behavior and increases intrinsic ribosomal activity in immortalized mammary gland cells. Cancers (Basel), 12, 3512.

Nucleolar protein 10 (NOP10) predicts poor prognosis in invasive breast cancer. K A Elsharawy, M Althobiti, O J Mohammed, A I Aljohani, M S Toss, A R Green, E A Rakha, Breast Cancer Res. Treat. 185Elsharawy,K.A., Althobiti,M., Mohammed,O.J., Aljohani,A.I., Toss,M.S., Green,A.R. and Rakha,E.A. (2021) Nucleolar protein 10 (NOP10) predicts poor prognosis in invasive breast cancer. Breast Cancer Res. Treat., 185, 615-627.

Mutational analysis of telomere genes in BRCA1/2-negative breast cancer families with very short telomeres. K Yanowsky, A Barroso, A Osorio, M Urioste, J Benitez, B Martinez-Delgado, Breast Cancer Res. Treat. 134Yanowsky,K., Barroso,A., Osorio,A., Urioste,M., Benitez,J. and Martinez-Delgado,B. (2012) Mutational analysis of telomere genes in BRCA1/2-negative breast cancer families with very short telomeres. Breast Cancer Res. Treat., 134, 1337-1343.

Guide snoRNAs: drivers or passengers in human disease?. M Deogharia, M Majumder, Biology. Deogharia,M. and Majumder,M. (2018) Guide snoRNAs: drivers or passengers in human disease? Biology (Basel), 8, 1.

Comprehensive molecular biomarker identification in breast cancer brain metastases. H J Schulten, M Bangash, S Karim, A Dallol, D Hussein, A Merdad, F K Al-Thoubaity, J Al-Maghrabi, A Jamal, F Al-Ghamdi, J. Transl. Med. 15269Schulten,H.J., Bangash,M., Karim,S., Dallol,A., Hussein,D., Merdad,A., Al-Thoubaity,F.K., Al-Maghrabi,J., Jamal,A., Al-Ghamdi,F. et al. (2017) Comprehensive molecular biomarker identification in breast cancer brain metastases. J. Transl. Med., 15, 269.

Molecular biomarkers screened by next-generation RNA sequencing for non-sentinel lymph node status prediction in breast cancer patients with metastatic sentinel lymph nodes. F Liang, H Qu, Q Lin, Y Yang, X Ruan, B Zhang, Y Liu, C Yu, H Zhang, X Fang, World J. Surg. Oncol. 13258Liang,F., Qu,H., Lin,Q., Yang,Y., Ruan,X., Zhang,B., Liu,Y., Yu,C., Zhang,H., Fang,X. et al. (2015) Molecular biomarkers screened by next-generation RNA sequencing for non-sentinel lymph node status prediction in breast cancer patients with metastatic sentinel lymph nodes. World J. Surg. Oncol., 13, 258.

H/ACA box small nucleolar RNA 7B acts as an oncogene and a potential prognostic biomarker in breast cancer. Y Sun, E Chen, Y Li, D Ye, Y Cai, Q Wang, Q Li, X Zhang, Cancer Cell Int. 19125Sun,Y., Chen,E., Li,Y., Ye,D., Cai,Y., Wang,Q., Li,Q. and Zhang,X. (2019) H/ACA box small nucleolar RNA 7B acts as an oncogene and a potential prognostic biomarker in breast cancer. Cancer Cell Int., 19, 125.

A-to-I RNA editing occurs at over a hundred million genomic sites, located in a majority of human genes. L Bazak, A Haviv, M Barak, J Jacob-Hirsch, P Deng, R Zhang, F J Isaacs, G Rechavi, J B Li, E Eisenberg, Genome Res. 24Bazak,L., Haviv,A., Barak,M., Jacob-Hirsch,J., Deng,P., Zhang,R., Isaacs,F.J., Rechavi,G., Li,J.B., Eisenberg,E. et al. (2014) A-to-I RNA editing occurs at over a hundred million genomic sites, located in a majority of human genes. Genome Res., 24, 365-376.

The ADAR protein family. Y A Savva, L E Rieder, R A Reenan, Genome Biol. 13252Savva,Y.A., Rieder,L.E. and Reenan,R.A. (2012) The ADAR protein family. Genome Biol., 13, 252.

RNA editing: a driving force for adaptive evolution?. W M Gommans, S P Mullen, S Maas, Bioessays. 31Gommans,W.M., Mullen,S.P. and Maas,S. (2009) RNA editing: a driving force for adaptive evolution? Bioessays, 31, 1137-1145.

Regulation of alternative splicing by RNA editing. S M Rueter, T R Dawson, R B Emeson, Nature. 399Rueter,S.M., Dawson,T.R. and Emeson,R.B. (1999) Regulation of alternative splicing by RNA editing. Nature, 399, 75-80.

Controlling the Editor: The Many Roles of RNA-Binding Proteins in Regulating A-to-I RNA Editing. M C Washburn, H A Hundley, Adv. Exp. Med. Biol. 907Washburn,M.C. and Hundley,H.A. (2016) Controlling the Editor: The Many Roles of RNA-Binding Proteins in Regulating A-to-I RNA Editing. Adv. Exp. Med. Biol., 907, 189-213.

RNA editing in nascent RNA affects pre-mRNA splicing. Y E Hsiao, J H Bahn, Y Yang, X Lin, S Tran, E W Yang, G Quinones-Valdez, X Xiao, Genome Res. 28Hsiao,Y.E., Bahn,J.H., Yang,Y., Lin,X., Tran,S., Yang,E.W., Quinones-Valdez,G. and Xiao,X. (2018) RNA editing in nascent RNA affects pre-mRNA splicing. Genome Res., 28, 812-823.

Altered RNA editing in 3 UTR perturbs microRNA-mediated regulation of oncogenes and tumor-suppressors. L Zhang, C S Yang, X Varelas, S Monti, Sci. Rep. 623226Zhang,L., Yang,C.S., Varelas,X. and Monti,S. (2016) Altered RNA editing in 3 UTR perturbs microRNA-mediated regulation of oncogenes and tumor-suppressors. Sci. Rep., 6, 23226.

Role of tRNA modifications in human diseases. A G Torres, E Batlle, L Ribas De Pouplana, Trends Mol. Med. 20Torres,A.G., Batlle,E. and Ribas de Pouplana,L. (2014) Role of tRNA modifications in human diseases. Trends Mol. Med., 20, 306-314.

Functions and regulation of the APOBEC family of proteins. H C Smith, R P Bennett, A Kizilyer, W M Mcdougall, K M Prohaska, Semin. Cell Dev. Biol. 23Smith,H.C., Bennett,R.P., Kizilyer,A., McDougall,W.M. and Prohaska,K.M. (2012) Functions and regulation of the APOBEC family of proteins. Semin. Cell Dev. Biol., 23, 258-268.

The AID/APOBEC family of nucleic acid mutators. S G Conticello, Genome Biol. 9229Conticello,S.G. (2008) The AID/APOBEC family of nucleic acid mutators. Genome Biol., 9, 229.

ADAR1-mediated RNA-editing of 3 UTRs in breast cancer. E A Sagredo, A Blanco, A I Sagredo, P Perez, G Sepulveda-Hermosilla, F Morales, B Muller, R Verdugo, K Marcelain, O Harismendy, Biol. Res. 5136Sagredo,E.A., Blanco,A., Sagredo,A.I., Perez,P., Sepulveda-Hermosilla,G., Morales,F., Muller,B., Verdugo,R., Marcelain,K., Harismendy,O. et al. (2018) ADAR1-mediated RNA-editing of 3 UTRs in breast cancer. Biol. Res., 51, 36.

Principles Governing A-to-I RNA Editing in the. D Fumagalli, D Gacquer, F Rothe, A Lefort, F Libert, D Brown, N Kheddoumi, A Shlien, T Konopka, R Salgado, Breast Cancer Transcriptome. Cell Rep. 13Fumagalli,D., Gacquer,D., Rothe,F., Lefort,A., Libert,F., Brown,D., Kheddoumi,N., Shlien,A., Konopka,T., Salgado,R. et al. (2015) Principles Governing A-to-I RNA Editing in the Breast Cancer Transcriptome. Cell Rep., 13, 277-289.

The genomic landscape and clinical relevance of A-to-I RNA editing in human cancers. L Han, L Diao, S Yu, X Xu, J Li, R Zhang, Y Yang, H M J Werner, A K Eterovic, Y Yuan, Cancer Cell. 28Han,L., Diao,L., Yu,S., Xu,X., Li,J., Zhang,R., Yang,Y., Werner,H.M.J., Eterovic,A.K., Yuan,Y. et al. (2015) The genomic landscape and clinical relevance of A-to-I RNA editing in human cancers. Cancer Cell, 28, 515-528.

Elevated RNA Editing Activity Is a Major Contributor to Transcriptomic Diversity in Tumors. N Paz-Yaacov, L Bazak, I Buchumenski, H T Porath, M Danan-Gotthold, B A Knisbacher, E Eisenberg, E Y Levanon, Cell Rep. 13Paz-Yaacov,N., Bazak,L., Buchumenski,I., Porath,H.T., Danan-Gotthold,M., Knisbacher,B.A., Eisenberg,E. and Levanon,E.Y. (2015) Elevated RNA Editing Activity Is a Major Contributor to Transcriptomic Diversity in Tumors. Cell Rep., 13, 267-276.

ADAR1 Transcriptome editing promotes breast cancer progression through the regulation of cell cycle and DNA damage response. E A Sagredo, A I Sagredo, A Blanco, P Rojas De Santiago, S Rivas, R Assar, P Perez, K Marcelain, R Armisen, Biochim. Biophys. Acta Mol. Cell Res. 118716Sagredo,E.A., Sagredo,A.I., Blanco,A., Rojas De Santiago,P., Rivas,S., Assar,R., Perez,P., Marcelain,K. and Armisen,R. (2020) ADAR1 Transcriptome editing promotes breast cancer progression through the regulation of cell cycle and DNA damage response. Biochim. Biophys. Acta Mol. Cell Res., 1867, 118716.

The mRNA-edited form of GABRA3 suppresses GABRA3-mediated Akt activation and breast cancer metastasis. K Gumireddy, A Li, A V Kossenkov, M Sakurai, J Yan, Y Li, H Xu, J Wang, P J Zhang, L Zhang, Nat. Commun. 710715Gumireddy,K., Li,A., Kossenkov,A.V., Sakurai,M., Yan,J., Li,Y., Xu,H., Wang,J., Zhang,P.J., Zhang,L. et al. (2016) The mRNA-edited form of GABRA3 suppresses GABRA3-mediated Akt activation and breast cancer metastasis. Nat. Commun., 7, 10715.

A-to-I RNA editing contributes to proteomic diversity in cancer. X Peng, X Xu, Y Wang, D H Hawke, S Yu, L Han, Z Zhou, K Mojumdar, K J Jeong, M Labrie, Cancer Cell. 33Peng,X., Xu,X., Wang,Y., Hawke,D.H., Yu,S., Han,L., Zhou,Z., Mojumdar,K., Jeong,K.J., Labrie,M. et al. (2018) A-to-I RNA editing contributes to proteomic diversity in cancer. Cancer Cell, 33, 817-828.

RNA editing derived epitopes function as cancer antigens to elicit immune responses. M Zhang, J Fritsche, J Roszik, L J Williams, X Peng, Y Chiu, C C Tsou, F Hoffgaard, V Goldfinger, O Schoor, Nat. Commun. 93919Zhang,M., Fritsche,J., Roszik,J., Williams,L.J., Peng,X., Chiu,Y., Tsou,C.C., Hoffgaard,F., Goldfinger,V., Schoor,O. et al. (2018) RNA editing derived epitopes function as cancer antigens to elicit immune responses. Nat. Commun., 9, 3919.

A-to-I RNA editing up-regulates human dihydrofolate reductase in breast cancer. M Nakano, T Fukami, S Gotoh, M Nakajima, J. Biol. Chem. 292Nakano,M., Fukami,T., Gotoh,S. and Nakajima,M. (2017) A-to-I RNA editing up-regulates human dihydrofolate reductase in breast cancer. J. Biol. Chem., 292, 4873-4884.

Association of a germline copy number polymorphism of APOBEC3A and APOBEC3B with burden of putative APOBEC-dependent mutations in breast cancer. S Nik-Zainal, D C Wedge, L B Alexandrov, M Petljak, A P Butler, N Bolli, H R Davies, S Knappskog, S Martin, E Papaemmanuil, Nat. Genet. 46Nik-Zainal,S., Wedge,D.C., Alexandrov,L.B., Petljak,M., Butler,A.P., Bolli,N., Davies,H.R., Knappskog,S., Martin,S., Papaemmanuil,E. et al. (2014) Association of a germline copy number polymorphism of APOBEC3A and APOBEC3B with burden of putative APOBEC-dependent mutations in breast cancer. Nat. Genet., 46, 487-491.

An APOBEC cytidine deaminase mutagenesis pattern is widespread in human cancers. S A Roberts, M S Lawrence, L J Klimczak, S A Grimm, D Fargo, P Stojanov, A Kiezun, G V Kryukov, S L Carter, G Saksena, Nat. Genet. 45Roberts,S.A., Lawrence,M.S., Klimczak,L.J., Grimm,S.A., Fargo,D., Stojanov,P., Kiezun,A., Kryukov,G.V., Carter,S.L., Saksena,G. et al. (2013) An APOBEC cytidine deaminase mutagenesis pattern is widespread in human cancers. Nat. Genet., 45, 970-976.

Integrative genomic analysis identifies associations of molecular alterations to APOBEC and BRCA1/2 mutational signatures in breast cancer. V Trevino, Mol Genet Genomic Med. 7810Trevino,V. (2019) Integrative genomic analysis identifies associations of molecular alterations to APOBEC and BRCA1/2 mutational signatures in breast cancer. Mol Genet Genomic Med, 7, e810.

Transient overexpression of exogenous APOBEC3A causes C-to-U RNA editing of thousands of genes. S Sharma, S K Patnaik, Z Kemer, B E Baysal, RNA Biol. 14Sharma,S., Patnaik,S.K., Kemer,Z. and Baysal,B.E. (2017) Transient overexpression of exogenous APOBEC3A causes C-to-U RNA editing of thousands of genes. RNA Biol, 14, 603-610.

APOBEC3B gene expression in ductal carcinoma in situ and synchronous invasive breast cancer. A M Sieuwerts, S C Doebar, V De Weerd, E I Verhoef, C M Beauford, M C Agahozo, J W M Martens, C H M Van Deurzen, Cancers. 1062Sieuwerts,A.M., Doebar,S.C., de Weerd,V., Verhoef,E.I., Beauford,C.M., Agahozo,M.C., Martens,J.W.M. and van Deurzen,C.H.M. (2019) APOBEC3B gene expression in ductal carcinoma in situ and synchronous invasive breast cancer. Cancers (Basel), 11, 1062.

APOBEC3-Mediated RNA Editing in Breast Cancer is Associated with Heightened Immune Activity and Improved Survival. M Asaoka, T Ishikawa, K Takabe, S K Patnaik, Int. J. Mol. Sci. 205621Asaoka,M., Ishikawa,T., Takabe,K. and Patnaik,S.K. (2019) APOBEC3-Mediated RNA Editing in Breast Cancer is Associated with Heightened Immune Activity and Improved Survival. Int. J. Mol. Sci., 20, 5621.

L Zhou, J Hao, Y Yuan, R Peng, H Wang, D Ni, Y Gu, L Huang, Z Mao, Z Lyu, Motif is Essential for A1CF Nucleus Localization and A1CF. 8aaZhou,L., Hao,J., Yuan,Y., Peng,R., Wang,H., Ni,D., Gu,Y., Huang,L., Mao,Z., Lyu,Z. et al. (2016) EIYMNVPV Motif is Essential for A1CF Nucleus Localization and A1CF (-8aa)

Promotes Proliferation of MDA-MB-231 Cells via Up-Regulation of IL-6. Int. J. Mol. Sci. 17811Promotes Proliferation of MDA-MB-231 Cells via Up-Regulation of IL-6. Int. J. Mol. Sci., 17, 811.

. NAR Cancer. 33NAR Cancer, 2021, Vol. 3, No. 3

) tRNA over-expression in breast cancer and functional consequences. M Pavon-Eternod, S Gomes, R Geslain, Q Dai, M R Rosner, T Pan, Nucleic Acids Res. 37Pavon-Eternod,M., Gomes,S., Geslain,R., Dai,Q., Rosner,M.R. and Pan,T. (2009) tRNA over-expression in breast cancer and functional consequences. Nucleic Acids Res., 37, 7268-7280.

Novel base-pairing interactions at the tRNA wobble position crucial for accurate reading of the genetic code. A Rozov, N Demeshkina, I Khusainov, E Westhof, M Yusupov, G Yusupova, Nat. Commun. 710457Rozov,A., Demeshkina,N., Khusainov,I., Westhof,E., Yusupov,M. and Yusupova,G. (2016) Novel base-pairing interactions at the tRNA wobble position crucial for accurate reading of the genetic code. Nat. Commun., 7, 10457.

Elp3 links tRNA modification to IRES-dependent translation of LEF1 to sustain metastasis in breast cancer. S Delaunay, F Rapino, L Tharun, Z Zhou, L Heukamp, M Termathe, K Shostak, I Klevernic, A Florin, H Desmecht, J. Exp. Med. 213Delaunay,S., Rapino,F., Tharun,L., Zhou,Z., Heukamp,L., Termathe,M., Shostak,K., Klevernic,I., Florin,A., Desmecht,H. et al. (2016) Elp3 links tRNA modification to IRES-dependent translation of LEF1 to sustain metastasis in breast cancer. J. Exp. Med., 213, 2503-2523.

2020) tRNA queuosine modification enzyme modulates the growth and microbiome recruitment to breast tumors. J Zhang, R Lu, Y Zhang, Z Matuszek, W Zhang, Y Xia, T Pan, J Sun, Cancers (Basel). 12628Zhang,J., Lu,R., Zhang,Y., Matuszek,Z., Zhang,W., Xia,Y., Pan,T. and Sun,J. (2020) tRNA queuosine modification enzyme modulates the growth and microbiome recruitment to breast tumors. Cancers (Basel), 12, 628.

Characterization of the human tRNA-guanine transglycosylase: confirmation of the heterodimeric subunit structure. Y C Chen, V P Kelly, S V Stachura, G A Garcia, RNA. 16Chen,Y.C., Kelly,V.P., Stachura,S.V. and Garcia,G.A. (2010) Characterization of the human tRNA-guanine transglycosylase: confirmation of the heterodimeric subunit structure. RNA, 16, 958-968.

Chromosome 8 BAC array comparative genomic hybridization and expression analysis identify amplification and overexpression of TRMT12 in breast cancer. V Rodriguez, Y Chen, A Elkahloun, A Dutra, E Pak, S Chandrasekharappa, Genes Chromosomes Cancer. 46Rodriguez,V., Chen,Y., Elkahloun,A., Dutra,A., Pak,E. and Chandrasekharappa,S. (2007) Chromosome 8 BAC array comparative genomic hybridization and expression analysis identify amplification and overexpression of TRMT12 in breast cancer. Genes Chromosomes Cancer, 46, 694-707.

The expression of TRMT2A, a novel cell cycle regulated protein, identifies a subset of breast cancer patients with HER2 over-expression that are at an increased risk of recurrence. D G Hicks, B R Janarthanan, R Vardarajan, S A Kulkarni, T Khoury, D Dim, G T Budd, B J Yoder, R Tubbs, M T Schreeder, BMC Cancer. 10108Hicks,D.G., Janarthanan,B.R., Vardarajan,R., Kulkarni,S.A., Khoury,T., Dim,D., Budd,G.T., Yoder,B.J., Tubbs,R., Schreeder,M.T. et al. (2010) The expression of TRMT2A, a novel cell cycle regulated protein, identifies a subset of breast cancer patients with HER2 over-expression that are at an increased risk of recurrence. BMC Cancer, 10, 108.

A human tRNA methyltransferase 9-like protein prevents tumour growth by regulating LIN9 and HIF1-alpha. U Begley, M S Sosa, A Avivar-Valderas, A Patil, L Endres, Y Estrada, C T Chan, D Su, P C Dedon, J A Aguirre-Ghiso, EMBO Mol. Med. 5Begley,U., Sosa,M.S., Avivar-Valderas,A., Patil,A., Endres,L., Estrada,Y., Chan,C.T., Su,D., Dedon,P.C., Aguirre-Ghiso,J.A. et al. (2013) A human tRNA methyltransferase 9-like protein prevents tumour growth by regulating LIN9 and HIF1-alpha. EMBO Mol. Med., 5, 366-383.

Multiple links between 5-methylcytosine content of mRNA and translation. U Schumann, H N Zhang, T Sibbritt, A Pan, A Horvath, S Gross, S J Clark, L Yang, T Preiss, BMC Biol. 1840Schumann,U., Zhang,H.N., Sibbritt,T., Pan,A., Horvath,A., Gross,S., Clark,S.J., Yang,L. and Preiss,T. (2020) Multiple links between 5-methylcytosine content of mRNA and translation. BMC Biol., 18, 40.

Induction of sporulation in Saccharomyces cerevisiae leads to the formation of N6-methyladenosine in mRNA: a potential mechanism for the activity of the IME4 gene. M J Clancy, M E Shambaugh, C S Timpte, J A Bokar, Nucleic Acids Res. 30Clancy,M.J., Shambaugh,M.E., Timpte,C.S. and Bokar,J.A. (2002) Induction of sporulation in Saccharomyces cerevisiae leads to the formation of N6-methyladenosine in mRNA: a potential mechanism for the activity of the IME4 gene. Nucleic Acids Res., 30, 4509-4518.

RNA methylation by the MIS complex regulates a cell fate decision in yeast. S D Agarwala, H G Blitzblau, A Hochwagen, G R Fink, PLos Genet. 81002732Agarwala,S.D., Blitzblau,H.G., Hochwagen,A. and Fink,G.R. (2012) RNA methylation by the MIS complex regulates a cell fate decision in yeast. PLos Genet., 8, e1002732.

The m(6)A methyltransferase METTL3 promotes translation in human cancer cells. S Lin, J Choe, P Du, R Triboulet, R I Gregory, Mol. Cell. 62Lin,S., Choe,J., Du,P., Triboulet,R. and Gregory,R.I. (2016) The m(6)A methyltransferase METTL3 promotes translation in human cancer cells. Mol. Cell, 62, 335-345.

Chemical inhibitors make their RNA epigenetic mark. M Cully, Nat. Rev. Drug Discov. 18Cully,M. (2019) Chemical inhibitors make their RNA epigenetic mark. Nat. Rev. Drug Discov., 18, 892-894.

Small-molecule inhibition of METTL3 as a strategy against myeloid leukaemia. E Yankova, W Blackaby, M Albertella, J Rak, E De Braekeleer, G Tsagkogeorga, E S Pilka, D Aspris, D Leggate, A G Hendrick, Nature. 593Yankova,E., Blackaby,W., Albertella,M., Rak,J., De Braekeleer,E., Tsagkogeorga,G., Pilka,E.S., Aspris,D., Leggate,D., Hendrick,A.G. et al. (2021) Small-molecule inhibition of METTL3 as a strategy against myeloid leukaemia. Nature, 593, 597-601.

The m6A methyltransferase METTL3 promotes the stemness and malignant progression of breast cancer by mediating m6A modification on SOX2. J Xie, J Ba, M Zhang, Y Wan, Z Jin, Y Yao, J. BUON. 26Xie,J., Ba,J., Zhang,M., Wan,Y., Jin,Z. and Y.,Yao. (2021) The m6A methyltransferase METTL3 promotes the stemness and malignant progression of breast cancer by mediating m6A modification on SOX2. J. BUON., 26, 444-449.

Changes of N6-methyladenosine modulators promote breast cancer progression. L Wu, D Wu, J Ning, W Liu, D Zhang, BMC Cancer. 19326Wu,L., Wu,D., Ning,J., Liu,W. and Zhang,D. (2019) Changes of N6-methyladenosine modulators promote breast cancer progression. BMC Cancer, 19, 326.

YTHDF3 Induces the Translation of m6A-Enriched Gene Transcripts to Promote Breast Cancer Brain Metastasis. G Chang, L Shi, Y Ye, H Shi, L Zeng, S Tiwary, J T Huse, L Huo, L Ma, Y Ma, Cancer Cell. 38Chang,G., Shi,L., Ye,Y., Shi,H., Zeng,L., Tiwary,S., Huse,J.T., Huo,L., Ma,L., Ma,Y. et al. (2020) YTHDF3 Induces the Translation of m6A-Enriched Gene Transcripts to Promote Breast Cancer Brain Metastasis. Cancer Cell, 38, 857-871.

Inhibition of YTHDF2 triggers proteotoxic cell death in MYC-driven breast cancer. J M Einstein, M Perelis, I A Chaim, J K Meena, J K Nussbacher, A T Tankka, B A Yee, H Li, A A Madrigal, N J Neill, Mol. Cell. 81Einstein,J.M., Perelis,M., Chaim,I.A., Meena,J.K., Nussbacher,J.K., Tankka,A.T., Yee,B.A., Li,H., Madrigal,A.A., Neill,N.J. et al. (2021) Inhibition of YTHDF2 triggers proteotoxic cell death in MYC-driven breast cancer. Mol. Cell, 81, 3048-3064.