# Metagenomic Survey of Tomato Rhizosphere Microbiome Using the Shotgun Approach

CorpusID: 246487669
 
tags: #Medicine, #Environmental_Science, #Agricultural_And_Food_Sciences, #Biology

URL: [https://www.semanticscholar.org/paper/5d3f94fabf5dc25539a2687cee82dd962088a8c3](https://www.semanticscholar.org/paper/5d3f94fabf5dc25539a2687cee82dd962088a8c3)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Metagenomic Survey of Tomato Rhizosphere Microbiome Using the Shotgun Approach


Olubukola Oluranti Babalola 
Faculty of Natural and Agricultural Sciences
Food Security and Safety Niche
North-West University
MmabathoSouth Africa

Afeez Adesina Adedayo 
Faculty of Natural and Agricultural Sciences
Food Security and Safety Niche
North-West University
MmabathoSouth Africa

Ayomide Emmanuel Fadiji 
Faculty of Natural and Agricultural Sciences
Food Security and Safety Niche
North-West University
MmabathoSouth Africa

Olubukola Oluranti Babalola 
Faculty of Natural and Agricultural Sciences
Food Security and Safety Niche
North-West University
MmabathoSouth Africa

Metagenomic Survey of Tomato Rhizosphere Microbiome Using the Shotgun Approach

Food sustainability, e.g., fruit and vegetables, is a major agricultural problem that requires monitoring. Rhizosphere microbiomes' abundance and functionality are essential in promoting tomato plants' growth and health. We selected farms in South Africa's North West Province and present the metagenomes of their tomato rhizospheres and associated functional potentials.

T he North West Province is a semiarid region with high temperatures. The soil in this region is populated by important microorganisms, with essential characteristics that promote the planting of tomatoes (1). Tomatoes produce annual crops of about 600,000 tonnes and find their way to South Africa through Europe and South American countries like Peru and Ecuador (2). The availability of these fruits in South Africa will enhance food production for human consumption because of their richness in essential vitamins and carotenoids (3). Three main cultivars of tomatoes are predominant in South Africa, namely, round or fresh tomatoes, Roma tomatoes, and cherry tomatoes, contributing about 24% of the total vegetable production in South Africa (4). Tomatoes are cultivated in open fields under irrigation in the following South African provinces: Eastern Cape, Western Cape, northern Kwazulu-Natal, North West, Mpumalanga, and Limpopo.

We collected the soil samples used in this study from the root region of Roma tomato plants at the experimental farm of the North-West University, Mafikeng (26°019936.90S, 26°0 53919.00E; 25°47919.10S, 25°37905.10E; 25°47917.00S, 25°37903.20E; altitude, 159 km). The site has a 450-mm annual rainfall record and regional temperatures ranging from 25°C to 37°C (5). The experimental soil samples were collected from the rhizosphere of healthy and diseased tomato plants and the bulk soil. Bulk soil, which served as the control soil, was collected from a natural grassland with no tomato plantation, 20 m from the tomato plantation field. We took rhizosphere soil from the Roma tomato plants from three different areas of a site. Fifteen samples were put in separate sterile polyethene bags, kept in a cold box at 24°C, and transported to the laboratory. All collected soil samples were then stored at a temperature of 220°C before extraction of DNA for shotgun metagenomic sequencing.

From the stored rhizosphere soil, 5 g from each sample was measured using a calibrated scale. DNA was extracted using the NucleoSpin Soil kit (Macherey-Nagel, Germany). The quality of the extracted DNA was assessed using a NanoDrop spectrophotometer.

The libraries were prepared with 50 ng DNA using a Nextra DNA Flex kit, undergoing fragmentation and the ligation of adapter sequences. The final concentrations of the libraries were measured using the Qubit double-stranded DNA (dsDNA) HS assay kit (Life Technologies), and the mean lengths of the DNA fragments were ascertained using a 2100 Bioanalyzer (Agilent Technologies). The libraries were then monitored, combined at 0.6 nM, and sequenced using a NovaSeq 6000 system (Illumina) with 300 cycles. SolexaQA v1.6 was used to conduct the quality control (QC) of raw data, reduce low-quality reads, and remove replicate data (6). Duplicate read inferred sequencing error estimation (DRISEE) enables us to assess the error of sequenced samples caused by artificial replicated sequenced data (7). We employed the default settings of the MG-RAST v4.0.3 server to perform analytical processing downstream (8,9) ( Table 1). The domain or kingdoms obtained according to the taxonomic system are Bacteria, Eukaryota, and Archaea. The most abundant phyla belonged to the Bacteria domain; Proteobacteria (38.8 to 54%) and Actinobacteria (25.4 to 35.5%) were the most abundant, and others, such as Acidobacteria (2.3 to 5.2%), Bacteroidetes (3.0 to 3.9%), Planctomycetes (2.6 to 3.4%), Verrucomicrobia (2.2 to 2.3%), and Firmicutes (1.7 to 2.4%), were also significant. Moreover, reads for fungi (Ascomycota and Basidiomycota) and archaea (Thaumarchaeota and Euryarchaeota) were also identified but at ,1% relative abundance (Fig. 1).

Functional annotation after mapping with SEED subsystems (10) revealed the presence of the following important attributes: carbohydrates (13.2 to 14.8%), clustering-based systems (12.7 to 12.8%), amino acids and derivatives (10.1 to 10.3%), protein metabolism (8.2 to 8.3%), DNA metabolism (4.3 to 4.6%), cell wall and capsule (3.4 to 3.6%), RNA metabolism (3.3 to 3.5%), and stress response (2.5 to 2.7%).

Data availability. The metagenomes of rhizosphere soil sequence reads were submitted to the NCBI with BioProject accession number PRJNA766489 and Sequence Read Archive (SRA) accession numbers SRX12366062, SRX12366063, and SRX12366064 (healthy), SRX12366065, SRX12366066, and SRX12366067 (diseased), and SRX12366068, SRX12366069, and SRX12366070 (bulk).

## TABLE 1
1Sequence reads for the rhizosphere soil samples analyzedSample a 
Data before QC 


ACKNOWLEDGMENTSO.O.B. acknowledges the National Research Foundation (South Africa) grants (UID123634 and UID132595) that support research in our laboratory.
Shotgun metagenomic survey of maize soil rhizobiome. O O Babalola, S A Akinola, A S Ayangbenro, 10.1128/MRA.00860-20Microbiol Resour Announc. 9Babalola OO, Akinola SA, Ayangbenro AS. 2020. Shotgun metagenomic survey of maize soil rhizobiome. Microbiol Resour Announc 9:e00860-20. https://doi.org/10.1128/MRA.00860-20.

Ethnobotany of food plants (quelites) sold in two traditional markets of Oaxaca, Mexico. G I Manzanero-Medina, M A Vásquez-Dávila, H Lustre-Sánchez, A Pérez-Herrera, 10.1016/j.sajb.2020.01.002S Afr J Bot. 130Manzanero-Medina GI, Vásquez-Dávila MA, Lustre-Sánchez H, Pérez- Herrera A. 2020. Ethnobotany of food plants (quelites) sold in two tradi- tional markets of Oaxaca, Mexico. S Afr J Bot 130:215-223. https://doi .org/10.1016/j.sajb.2020.01.002.

Antioxidants in plants: a valorization potential emphasizing the need for the conservation of plant. Llauradó Maury, G , Méndez Rodríguez, D Hendrix, S , Escalona Arranz, J C , Fung Boix, Y Pacheco, A O , García Díaz, J Morris-Quevedo, H J , Ferrer Dubois, A Aleman, E I Beenaerts, N Méndez-Santos, I E , Orberá Ratón, T Cos, P Cuypers, A , 10.3390/antiox9111048biodiversity in Cuba. Antioxidants. 9Llauradó Maury G, Méndez Rodríguez D, Hendrix S, Escalona Arranz JC, Fung Boix Y, Pacheco AO, García Díaz J, Morris-Quevedo HJ, Ferrer Dubois A, Aleman EI, Beenaerts N, Méndez-Santos IE, Orberá Ratón T, Cos P, Cuypers A. 2020. Antioxidants in plants: a valorization potential empha- sizing the need for the conservation of plant biodiversity in Cuba. Antioxi- dants 9:1048. https://doi.org/10.3390/antiox9111048.

Bulk and compound-specific stable isotope ratio analysis for authenticity testing of organically grown tomatoes. L Bontempo, K A Van Leeuwen, M Paolini, Holst Laursen, K Micheloni, C Prenzler, P D , Ryan D Camin, F , 10.1016/j.foodchem.2020.126426Food Chem. 318126426Bontempo L, van Leeuwen KA, Paolini M, Holst Laursen K, Micheloni C, Prenzler PD, Ryan D, Camin F. 2020. Bulk and compound-specific stable iso- tope ratio analysis for authenticity testing of organically grown tomatoes. Food Chem 318:126426. https://doi.org/10.1016/j.foodchem.2020.126426.

Metagenomes of maize rhizosphere samples after different fertilization treatments at Molelwane Farm. O O Babalola, M C Enebe, 10.1128/MRA.00937-20South Africa. Microbiol Resour Announc. 9Babalola OO, Enebe MC. 2020. Metagenomes of maize rhizosphere sam- ples after different fertilization treatments at Molelwane Farm, located in North-West Province, South Africa. Microbiol Resour Announc 9:e00937- 20. https://doi.org/10.1128/MRA.00937-20.

SolexaQA: at-a-glance quality assessment of Illumina second-generation sequencing data. M P Cox, D A Peterson, P J Biggs, 10.1186/1471-2105-11-485BMC Bioinformatics. 11485Cox MP, Peterson DA, Biggs PJ. 2010. SolexaQA: at-a-glance quality assessment of Illumina second-generation sequencing data. BMC Bioin- formatics 11:485. https://doi.org/10.1186/1471-2105-11-485.

High-throughput amplicon sequencing of the full-FIG 1 Abundant phyla obtained according to the taxonomic system. length 16S rRNA gene with single-nucleotide resolution. B J Callahan, J Wong, C Heiner, S Oh, C M Theriot, A S Gulati, S K Mcgill, M K Dougherty, 10.1093/nar/gkz569Nucleic Acids Res. 47Callahan BJ, Wong J, Heiner C, Oh S, Theriot CM, Gulati AS, McGill SK, Dougherty MK. 2019. High-throughput amplicon sequencing of the full- FIG 1 Abundant phyla obtained according to the taxonomic system. length 16S rRNA gene with single-nucleotide resolution. Nucleic Acids Res 47:e103. https://doi.org/10.1093/nar/gkz569.

The metagenomics RAST server: a public resource for the automatic phylogenetic and functional analysis of metagenomes. F Meyer, D Paarmann, D&apos; Souza, M Olson, R Glass, E Kubal, M Paczian, T Rodriguez, A Stevens, R Wilke, A Wilkening, J Edwards, R , 10.1186/1471-2105-9-386BMC Bioinformatics. 9386Meyer F, Paarmann D, D'Souza M, Olson R, Glass E, Kubal M, Paczian T, Rodriguez A, Stevens R, Wilke A, Wilkening J, Edwards R. 2008. The meta- genomics RAST server: a public resource for the automatic phylogenetic and functional analysis of metagenomes. BMC Bioinformatics 9:386. https://doi.org/10.1186/1471-2105-9-386.

Profiling the functional diversity of termite mound soil bacteria as revealed by shotgun sequencing. B J Enagbonma, B R Aremu, O O Babalola, 10.3390/genes10090637Genes. 10637Enagbonma BJ, Aremu BR, Babalola OO. 2019. Profiling the functional di- versity of termite mound soil bacteria as revealed by shotgun sequencing. Genes 10:637. https://doi.org/10.3390/genes10090637.

Functional analysis of metagenomes and metatranscriptomes using SEED and KEGG. S Mitra, P Rupek, D C Richter, T Urich, J A Gilbert, F Meyer, A Wilke, D H Huson, 10.1186/1471-2105-12-S1-S21BMC Bioinformatics. 12121SupplMitra S, Rupek P, Richter DC, Urich T, Gilbert JA, Meyer F, Wilke A, Huson DH. 2011. Functional analysis of metagenomes and metatranscriptomes using SEED and KEGG. BMC Bioinformatics 12(Suppl 1):S21. https://doi .org/10.1186/1471-2105-12-S1-S21.