# An environmental DNA metabarcoding survey reveals generic-level occurrence of scleractinian corals at reef slopes of Okinawa Island

CorpusID: 257774462
 
tags: #Medicine, #Environmental_Science

URL: [https://www.semanticscholar.org/paper/2ae7872d3cd7095f16604385d9e58e675b59c92a](https://www.semanticscholar.org/paper/2ae7872d3cd7095f16604385d9e58e675b59c92a)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

An environmental DNA metabarcoding survey reveals generic-level occurrence of scleractinian corals at reef slopes of Okinawa Island


Noriyuki Satoh 
Koki Nishitsuji koki.nishitsuji@oist.jp 
Marine Genomics Unit
Okinawa Institute of Science and Technology Graduate University
904-0495OnnaOkinawaJapan

Tomofumi Nagata 
Incorporated Foundation Okinawa Environment Science Center
901-2111UrasoeOkinawaJapan

Haruhi Narisoko 
Marine Genomics Unit
Okinawa Institute of Science and Technology Graduate University
904-0495OnnaOkinawaJapan

Megumi Kanai 
Incorporated Foundation Okinawa Environment Science Center
901-2111UrasoeOkinawaJapan

Kanako Hisata 
Marine Genomics Unit
Okinawa Institute of Science and Technology Graduate University
904-0495OnnaOkinawaJapan

Chuya Shinzato 
Atmosphere and Ocean Research Institute
The University of Tokyo
277-8564KashiwaChibaJapan

Noriyuki Satoh 
Marine Genomics Unit
Okinawa Institute of Science and Technology Graduate University
904-0495OnnaOkinawaJapan

An environmental DNA metabarcoding survey reveals generic-level occurrence of scleractinian corals at reef slopes of Okinawa Island
Received: 10 January 2023 Accepted: 1 March 2023Research Cite this article: Nishitsuji K, Nagata T, Narisoko H, Kanai M, Hisata K, Shinzato C, Satoh N. 2023 An environmental DNA metabarcoding survey reveals generic-level occurrence of scleractinian corals at reef slopes of Okinawa Island. Proc. R. Soc. B 290: 20230026. Authors for correspondence: Koki Nishitsuji † These authors have contributed equally to this work. Electronic supplementary material is available online at https://doi.org/10.6084/m9.figshare. c.6461943.Subject Category: Global change and conservation Subject Areas: ecology Keywords: coral reef surveyenvironmental DNA barcoding methoddirect diving observationOkinawa Island
Coral reefs have the highest biodiversity of all marine ecosystems in tropical and subtropical oceans. However, scleractinian corals, keystone organisms of reef productivity, are facing a crisis due to climate change and anthropogenic activities. A broad survey of reef-building corals is essential for worldwide reef preservation. To this end, direct observations made by coral-specialist divers might be supported by another robust method. We improved a recently devised environmental DNA (eDNA) metabarcoding method to identify more than 43 scleractinian genera by sampling 2 l of surface seawater above reefs. Together with direct observations by divers, we assessed the utility of eDNA at 63 locations spanning approximately 250 km near Okinawa Island. Slopes of these islands are populated by diverse coral genera, whereas shallow 'moats' sustain fewer and less varied coral taxa. Major genera recorded by divers included Acropora, Pocillopora, Porites and Montipora, the presence of which was confirmed by eDNA analyses. In addition, eDNA identified more genera than direct observations and documented the presence of previously unrecorded species. This scleractinian coral-specific eDNA method promises to be a powerful tool to survey coral reefs broadly, deeply and robustly.

## Introduction

Coral reefs in tropical and subtropical waters foster about 30% of all marine life, thus providing one of the most diverse ecosystems on Earth [1]. Reef-building or scleractinian corals (Anthozoa, Cnidaria) create robust reef structure by depositing massive calcium carbonate skeletons. Approximately 1300 scleractinian species, comprising 236 genera and 25 families, have been reported (Integrated Taxonomic Information System or ITIS; https://itis.gov). In the last two and a half decades, however, coral reefs have confronted growing crises due to global climate change and anthropogenic insults. More frequent coral bleaching caused by global warming of seawater, severe storms and predation by crown-of-thorns starfish have degraded and diminished healthy reef ecosystems [2][3][4][5][6]. Since coral reef environments are constantly and dramatically changing, frequent, high-density coral monitoring is essential to assess these changes, in the hope of more effective coral reef preservation.

Conventional coral monitoring depends on direct observation by coralspecialist divers. Such monitoring has greatly contributed to our understanding of corals at various reefs in the Indo-Pacific Ocean, detailing species variation, coverage, health, bleaching, diseases, etc. On the other hand, this method has limited utility for broader surveys. First, it requires coral biologists who are experienced at identifying coral species and are competent divers. In addition, this work is challenging because corals sometimes vary their morphology so drastically as to be unrecognizable [7]. Second, dive time is limited; thus, observations underwater are similarly restricted.

Our previous study demonstrated simultaneous detection of Acropora corals and their symbiotic algae from environmental DNA (eDNA) in seawater [8]. Recently, an eDNA metabarcoding method was developed to enable extensive biodiversity monitoring [9][10][11] including shallow reef corals [12][13][14]. eDNA barcoding relies on primers that amplify specific regions of mitochondrial and/or nuclear genomes, since sequence differences enable identification of coral genera and species. Early coral eDNA studies used general primers that amplified not only corals, but also other marine invertebrates [12][13][14]. Therefore, our second study developed other sets of eDNA amplification primers, more broadly applicable, yet specific to scleractinian corals [15]. We developed a set of primers for 12S-ribosomal-RNA genes (12S) to amplify eDNA by which all 36 genera with reported mitochondrial genome sequences (see https://www.ncbi.nlm.nih.gov/genome/browse#!/ organelles/, accessed 1 May 2022) can theoretically be identified by sequence differences [15]. Sampling of only 1 l surface seawater over selected reefs successfully identified coral genera present [15].

This study sought to answer several questions regarding coral-specific eDNA barcoding techniques and interpretation of results. First, since eDNA is a comparatively new technique, its validity and efficiency should be evaluated by simultaneous examination using other methods, such as direct monitoring by coral specialists. If the two sets of results differ, the eDNA method should be used with caution. Second, in the Pacific Ocean, corals in shallow reefs suffered greatly during the 1998 El Niño followed by severe bleaching in 2016-2017, especially Acropora [3,[16][17][18]. In inner reefedge, shallow moat habitats of the Okinawa Island coast, corals suffered severe damage and their recovery has been very slow, although deeper reef slopes appeared less degraded [3,6,16]. Accordingly, in this study we sought to assess the recovery in moats, and to determine which genera are returning. Third, Okinawa Island is located at approximately 26.270-26.739°N latitude and 127.698-128.266°E longitude. Okinawa's east coast faces the Pacific Ocean while its west coast faces the South China Sea (figure 1), with complex seawater currents in various directions. We attempted to determine whether there are differences in coral genera between sites. With these objectives, we examined the utility of recently developed coral eDNA barcoding method.


## Material and methods (a) Sampling

Reef monitoring was conducted at 63 coral reef locations in the latter half of 2021 by simultaneous diving observations of corals and collection of surface seawater above these reefs. Names of monitored locations, survey dates, geomorphic classification, latitude, longitude, types of coral community and major coral genera observed are summarized in figure 1, and electronic supplementary material, table S1 and figure S1. Most locations were reef slopes, 3-10 m in depth, and some were inner reef edge moats, 1-3 m in depth (figure 1; electronic supplementary  material, table S1).

Two experienced coral reef specialists snorkeled around reefs and recorded coral community types and dominant coral genera (species), according to the conventional Monitoring Sites 1000 Project conducted annually by the Ministry of the Environment of Japan (https://www.biodic.go.jp/moni1000/manual/spot-check_ver5.pdf ). Types of coral community included 'multispecies mixed', 'specific species' and 'not determined', where the first and second categories were distinguished by dominant species and their relative abundances (electronic supplementary material, table S1 and figure S1). Dominant species included tabular Acropora, branching Pocillopora and massive Porites. Monitoring of each site required about 30 min with repeated snorkeling. Each diver recorded coral genera based on his or her taxonomic experience. At the end of each day, the divers discussed their observations to reach a consensus regarding major coral genera observed at each site (electronic supplementary material, table S1). Coral coverage of individual reefs was beyond the scope of the present study and was not assessed.

At the same time that divers were observing corals at a given coral reef slope, 3 × 2 l of surface seawater were collected by eDNA researchers in a boat. At moats, the same number of samples was collected by throwing a bucket into the sea. To avoid contamination of seawater samples, containers and bucket were washed with fresh water prior to every sampling. Two-litre samples were filtered individually through 0.45 μm Sterivex filters (Merck) within 5 min after seawater collection. Then 1 ml of RNAlater (Qiagen) was added to the filtrate to prevent DNA degradation. Filters were maintained at 4°C before transfer to a −20°C freezer in the laboratory. Due to heavy waves, we were unable to collect a seawater eDNA sample at site #42.

(b) eDNA extraction, PCR amplicons and sequencing eDNA in Sterivex filters was extracted following the Environmental DNA Sampling and Experiment Manual v. 2.1 [19]. Extracted eDNA samples were PCR-amplified using primers, Scle_12S_Fw (5 0 -CCAGCMGACGCGGTRANACTTA-3 0 ) and Scle_12S_Rv (5 0 -AAWTTGACGACGGCCATGC-3 0 ), for mitochondrial 12S rRNA genes, as described in Shinzato et al. [15]. These primers were designed to identify 36 scleractinian coral genera, including Acanthastrea, Acropora, Anthemiphyllia, Astreopora, Coscinaraea, Crispatotrochus, Dipsastraea, Echinophyllia, Favites, Fimbriaphyllia, Fungia, Galaxea, Goniastrea, Goniopora, Heliofungia, Herpolitha, Hydnophora, Isopora, Leptoria, Lobophyllia, Montipora, Orbicella, Pachyseris, Pavona, Pectinia, Platygyra, Plesiastrea, Pocillopora, Polycyathus, Polyphyllia, Porites, Psammocora and Turbinaria [20,21]. All genera are common at Okinawa Archipelago (OA) except for Anthemiphyllia (https://www.gbif.org/ species/2260364), Crispatotrochus (https://www.gbif.org/ species/2259048) and Polycyathus (https://www.gbif.org/ species/2259234), though they are reportedly present at OA [20]. Orbicella is an Atlantic Ocean genus, and its closest relatives in the Pacific Ocean are Astrea and Leptastrea, both of which are common in the OA [20].

Genomic DNA isolated from an Acropora tenuis colony was used as a positive PCR control (data not shown). Tks Gflex DNA Polymerase (Takara) was used for PCR amplification. PCR cycling conditions were 1 min at 94°C, followed by 35 cycles of 10 s at 98°C, 15 s at 60°C and 30 s at 68°C, with an extension of 5 min at 68°C in the final cycle. PCR products were extracted and cleaned with a FastGene Gel/PCR Extraction Kit (NIPPON Genetics Co., Ltd. 


## (c) Bioinformatic analysis

Analyses were carried out as described by Shinzato et al. [15]. Briefly, after removal of low-quality bases (Phred quality score less than 20) and Illumina sequencing adapters [22], the remaining sequences were merged using USEARCH, v. 11.0.667 [23].

Then, de-noised (error-corrected) operational taxonomic units, called ZOTUs (zero-radius operational taxonomic units), were prepared for each sample. ZOTU sequences from all samples were concatenated and clustered using CD-HIT-EST v. 4.6 with 100% nucleotide identity [24]. Clustered, unique ZOTU sequences were used for the database for mapping. Merged sequences from each sample were mapped to the clustered ZOTUs and numbers of mapped sequences for each ZOTU were counted using the USEARCH 'otutab' command with 99% percent identity (-id 0.99).

Identification of ZOTUs originating from scleractinians were selected based on criteria described in our previous study [15]. After selecting scleractinian ZOTUs, mapped reads from the  same genera were combined. To remove possible errors and contamination, only genera with more than 0.1% of the total number of mapped reads in a given sample were considered. To infer similarities between samples and sampling locations, hierarchical clustering based on the ward D method was performed using percentages of coral genera. Numbers of ZOTUs at sampling locations are shown in electronic supplementary material, table S2. A rough estimation suggested that 1% of ZOTUs was supported by 28.2 ± 12.3 sequence reads.

During direct observations, divers identified Stylophora (electronic supplementary material, table S1), which was not included in our previous eDNA study [15], since the mt genome sequence of Stylophora pistillata was deposited in the NCBI database only on June 6, 2021. Therefore, we updated the previously reported informatic tools so that Stylophora was also included in the present analysis. Simultaneously, an additional 8 genera were also included (Agaricia, Catalaphyllia, Diploastrea, Micromussa, Oulastrea, Paraechinophyllia, Physogyra and Thalamophyllia). Therefore, this method is probably able to identify 45 genera, and 43 of these were detected in the present study (electronic supplementary material, figure S2).

Numbers of genera detected by direct observation and eDNA metabarcoding method were statistically compared for slopes and moats, respectively, using the Wilcoxon signed-rank test using R v. 4.2.1 [25]. Overlap coefficients between genera detected by direct observation and eDNA method was calculated for each sampling point.


## Results

With a combination of diver observations and eDNA barcoding, we surveyed scleractinian occurrence at 63 coral reef sites around Okinawa Island (figure 1) from early September to late December 2021, although seawater sampling was unsuccessful at one site (electronic supplementary material, table S1).


## (a) Direct observation by diving

Major genera recorded at each location included Acropora (tabulate or branching), Porites (massive), Pocillopora (branching), Stylophora, Isopora, Dipsastraea, Turbinaria, Montipora, Favites and Coelastrea (this is the new generic name of Goniastrea; electronic supplementary material, table S1) and Pavona (electronic supplementary material, table S1) (figure 1). Although divers observed that Heliopora and Palythoa were also abundant, the former is a zoantharian genus and the latter is an octocoral genus; thus, they were not detected by this eDNA method. An overview of the results showed that Acropora was dominant at many reefs around Okinawa Island (electronic supplementary material, table S1, figure S1; figure 1), and it appeared to be more common at reefs in the South China Sea (electronic supplementary material, table S1, figure S1; figure 1). By contrast, reefs along the northern part of the Island facing the Pacific Ocean (called nPC) exhibited mixed coral genera, including Pocillopora, Isopora, and Acropora (electronic supplementary material, table S1; figure S1; figure 1).

Of 63 monitoring sites, 51 were slopes (3-10 m in depth; figure 2a), 8 were moats (1-3 m in depth; figure 2b), and 4 sites were both slopes and moats (#35-37, 62). Direct observations demonstrated clear differences in coral composition between slopes and moats. As mentioned above, Acropora was dominant over most slopes except at nPC. By contrast, moats, e.g. sites #20, 21, 22, 23 and 24, fostered a variety of genera including Porites, Pavona, Montipora and Coelastrea (electronic supplementary material, table S1; figure 1), although fewer Acropora were also found in moats (figure 2b). This feature was more clearly distinguishable when comparing sites #23 (moat), 24 (moat), 29 (slope) and 30 (slope). These four sites are located along 6 km of the Okinawa coast adjacent to Yomitan and Onna villages. Two slopes (#29 and #30) were dominated by Acropora, with Pocillopora second (electronic supplementary material, table S1; figure 1). By contrast, moat #23 was covered with Porites, Coelastrea and Pavona, and #24 with Pavona, Porites, Fungia and Coelastrea, and a smaller number of Acropora (electronic supplementary material, table S1; figure 1).

Further evidence for this tendency came from a comparison of coral genera between moats and adjacent slopes. Sites #35, 36 and 37 constituted such a case (electronic supplementary material, table S1; figure 1). It appeared that slopes were dominated by Acropora and Pocillopora, whereas the moats were most heavily populated with Porites, Montipora, Isopora and Pavona (electronic supplementary material, table S1; figure 1). Location #62 exhibited similar composition (electronic supplementary material, table S1; figure 1).

Local differences in coral composition were also observed. For example, sites #25 and #27-29 are along 20 km of the coast of the Onna Village, at popular diving spots. The royalsocietypublishing.org/journal/rspb Proc. R. Soc. B 290: 20230026 dominant genus was Acropora at all four sites, whereas the next most dominant taxa were Porites at site #25 and Pocillopora at sites #27-29 (electronic supplementary material, table S1; figure 1). Since the geomorphic structures of these coral reef slopes are similar, this difference is interesting. Another local difference was found at sites #53 and 54, reefs between Ie and Sesoko Islands (figure 1). Dipsastraea was dominant at these sites, even though it was not at neighbouring sites (electronic supplementary material, table S1; figure 1).


## (b) eDNA barcoding examination

The eDNA method identified 43 genera, including Stylophora (figure 3), but not the octocoral, Heliopora (electronic supplementary material, figure S2). The first question regarding the use of eDNA was whether sampling only 2 l of seawater over reefs approximately 3-10 m in depth is sufficient for barcoding analysis. Although our previous study showed that 1 l was enough for amplification of eDNA [15], that survey was limited to one area (near sampling site #27) along a calm, 5 km seashore. The validity of the method over coral reefs with different morphologies and currents was not necessarily assured. However, we obtained amplicons from all samples at 62 sites (no sample was collected at site #42), confirming that 2 l of surface seawater are enough for scleractinianspecific eDNA metabarcoding (figure 3; electronic supplementary material, figure S2). The second question was whether only one sample is enough to achieve consistent results. As described in the Material and methods section, this study collected triplicate samples of surface seawater from boats, at the same time as diving observations. Triplicate samples from points (#1-9) were sequenced while duplicate samples from the remaining 53 locations (#10-63, except #42) were sequenced (electronic supplementary material, figure S2). We first examined whether triplicate or duplicate samples identified the same coral genera using ZOTU calls. As a result, for 7 of 9 triplicate samples, all three samples proved similar, whereas at the other two sites, two of the three samples were similar, but the third was different. In these cases, we averaged the data from the two similar samples. Among 53 duplicate samples, we judged that the average of the two samples was usable, although results at 5 sites differed slightly (electronic supplementary material, figure S2). Therefore, analysis of at least duplicate samples is recommended.

The third question was a threshold problem. The presence or absence of coral genera was shown by the percent scores of ZOTUs of selected genera per total ZOTU number (figure 3). Scores greater than 5% seem reasonable to use as documentation of the presence of major coral genera (figure 3; table 1; electronic supplementary material figure S2). On the other hand, it is uncertain how to treat samples with scores of 0.5% and below. For example, at location #3, Acropora, Montipora, Porites and Pocillopora accounted for 54.8%, 15.4%, 9.5% and 6.7% of all ZOTUs, and we concluded that all four genera were present at the reef (figure 3; table 1). In addition, ZOTU percentages were 3.2% for Platygyra, 3.0% for Plesiastrea, 2.3% for Favites and 2.2% for Caulastrea. At this site, the average number of reads/ZOTUs was 2265.4 (electronic supplementary material, table S2). This means that 1% ZOTU corresponds to 22.7 Reads. Therefore, we concluded that these four genera were present at location #3.

In addition, eDNA likely provided evidence for genera not yet known from Okinawa Island. For example, two species in the genus Heliofungia have been reported [3], H. actiniformis (Quoy & Gaimard 1833) and H. fralinae (Nemenzo 1959), the latter being recorded in southern Asia, including Japan. However, this species reportedly reaches its northern limit at Miyako Island, and there have been no reports of its presence at Okinawa or Amami Islands. This eDNA analysis identified Heliofungia at slopes #1 and #11 (figure 3, marked with yellow). Although the scores seemed low (0.08% for #1 and 0.14% for #11), these scores corresponded to 1.3 (1,673.6 × 0.0008 = 1.3) and 4.4 Reads (3,119.6 × 0.0014 = 4.4) (electronic supplementary material, table S2). That is, ZOTU scores suggest a more northward distribution of H. fralinae, reaching at least as far as Okinawa Island.

Another example was the genus Catalaphyllia, which comprises only one species, C. jardinei [26]. C. jardinei has been recorded at non-reef areas of Japan, including Miyazaki, Kochi, and recently, Amami Island, at 30 m depth [27], but not at Okinawa Island. This study recorded it at sites #49 and 52 (0.01% score; figure 3, marked with magenta). The score corresponded to Reads count of 1.7 (2,431.4 × 0.0001 = 1.7) at #49 and 4.0 (3978.6 × 0.0001 = 4.0) at #52 (electronic supplementary material, table S2). Therefore, although these suggestions should be examined further, it is likely that this represents a new record of a small, cryptic genus that had not been reported from Okinawa Island previously.


## (c) Match of results from two methods

To evaluate this scleractinian coral-specific eDNA method, we compared results of eDNA barcoding with direct observations (table 1). As mentioned above, Acropora was the dominant genus at most spots. To simplify comparisons, in table 1, we first listed slope sites facing the South China Sea, those facing the Pacific Ocean, and then moats (table 1). For comparison of each point, we listed up to three major genera called by divers and the three most abundant genera identified by eDNA (table 1).

Numbers of genera detected by eDNA were significantly larger than direct observation in both slopes (Wilcoxon signed-rank test, p < 0.001) and moats (Wilcoxon signedrank test, p < 0.05). We obtained overlap coefficients (1, 0.66, 0.5, 0.33 and 0) between genera detected by direct observation and eDNA method for each sampling point, and we allocated them to four classes based on the overlap coefficient. That is, strong matches included those with an overlap coefficient of 1. Moderate matches had coefficients from 0.5 to 0.66. Partial matches had coefficients of 0.33, and non-matches had coefficients of 0 (table 1). As a result, 41 of 62 points were well matched (67%), 15 were moderately matched (24%), 4 were partially match (6%), and only 2 points showed no match (3%). In other words, at more than 91% (67%+24%) of monitored locations, eDNA results were confirmed by direct observations. At 25 slope sites (#25-34, #35-38 and # 49-59), genera identified by eDNA method were well matched to those identified by divers. At most points, the dominant genus was Acropora, followed by Pocillopora, Porites and/or Montipora. In general, both methods identified these genera as dominant and subdominant genera, respectively (table 1)  direct observations identified Acropora as the dominant genus at all four sites, the next most abundant genus at #25 was Porites while it was Pocillopora at sites #27-29 (table 1). eDNA also identified the same genera as secondary genera (figure 3;  table 1). Since the two groups of coral reef slopes are similar, it is interesting to ask how this difference originated. This will be examined in future studies. At sites #53 and 54, located at reefs between Ie and Sesoko Islands, Dipsastraea was dominant, even though at neighbouring sites, it was not (table 1; figure 1). Both direct observations and eDNA method detected this taxon (table 1). Interestingly, Acropora was more frequently called by direct observation and Montipora was more frequently called by eDNA (table 1). This is partially because branching and tabular Acropora are easily spotted, while many Montipora species are encrusting taxa and not readily observed. Another example of poorly matched assessments was seen at site #37, in a moat, Isopora was identified as the dominant genus by direct obversion, and this site has been recognized as an ideal place to see Isopora. On the other hand, eDNA method identified Astreopora as dominant, and Isopora appeared eighth in abundance (figure 3; table 1). Therefore, the two methods did not always yield the same conclusion.


## Discussion (a) Coral eDNA barcoding method

Coral eDNA barcoding requires cautious interpretation of results, since the method is based on several assumptions. Corals constantly secrete mucus, which probably provides most of the coral eDNA in seawater. Coral cellular debris may be another source of eDNA. One assumption is that all coral species secrete the same amount of mucus (eDNA) or debris per unit of body surface. If this differs among genera, data may require adjustment. This assumption needs to be examined and such a study is now underway in our laboratory.

The velocity of surface currents and the magnitude of waves are also problematic. If currents are quite strong, eDNA may not correspond just to that secreted by the reef below, but from neighbouring reefs as well. In this study, of some triplicate samples, one replicate identified a different genus than the other two (electronic supplementary material, figure S2). Therefore, single samples at strong current sites seem inadequate, and at least two replicates should be examined. In addition, our eDNA method is specific to scleractinian corals and cannot detect soft corals or other cnidarians. Therefore, if a given spot was covered by many soft corals, a large amount of soft coral eDNA may affect estimates of scleractinian coral eDNA.

In addition, this study showed that results obtained from direct observation are not completely same as those detected by eDNA method. For example, Acropora was more frequently called by direct observation and Montipora was more frequently called by eDNA method (table 1). Another example of poorly matched assessments was seen at a moat, site #37. Isopora was identified as the dominant genus by direct obversion, but eDNA method identified Astreopora as dominant, and Isopora appeared eighth in abundance ( figure 3; table 1). Therefore, the two methods do not always yield the same conclusion. This should be considered when interpreting results.

In other words, constant improvement of the eDNA barcoding method is required. The bioinformatic method of our previous study based on mt sequence information of 36 genera detected 23 genera from eDNA collected at sites near #27 [15]. However, those primers could not detect Stylophora. Therefore, this study improved the method and did detect this genus (Material and Methods). Updates of target mt genome sequences may be important for future identification of a greater number of corals by this method.

Another key issue of eDNA method is that the present methodology cannot assess coral coverage at a given reef. This problem should be challenged in the future by incorporation of various new techniques and ideas.


## (b) Scleractinian corals at Okinawa Island

Our survey examined coral diversity at 63 coral reef locations around Okinawa Island. Both diving observations and eDNA method clearly showed that Acropora is the dominant genus at most slopes around Okinawa, followed by Pocillopora, Porites and Montipora as sub-dominant genera. This tendency is more conspicuous at reef slopes facing the South China Sea (table 1; figure 1). The diversity of coral genera is higher on slopes facing the Pacific Ocean than the South China Sea (figure 1) and also among neighbouring locations. This may be caused by seashore-specific ecological and/or geomorphic features, although the basis of this difference need to be investigated in future studies.

In addition to chronic anthropogenic impacts such as soil run-off [3,6], coral reefs around Okinawa Island suffered a mass bleaching event in 1998 and 2016-2017 [3,[16][17][18], and outbreaks of crown-of-thorns starfish in the 2000s [5,28], resulting in low coral coverage up to the 2020s, especially at moats. The recovery of coral coverage from such problems has been studied extensively [3,11,[16][17][18], although their evaluation is not always the same. The quality and quantity of coral genera demonstrated by this study at the 63 locations surround the islands appears higher than we expected, suggesting gradual recovery of scleractinian corals, especially at slopes.

The moats we examined in this study are places that are known to have gradual appearance of corals, while most moats have still fewer corals at Okinawa Island than before. This survey demonstrates remarkable differences in the diversity of coral genera in slopes and moats. By contrast to the dominance of Acropora and Pocillopora on reef slopes, Porites and Montipora appeared dominant in moats. The former are branching and/or tubular in morphology, while the latter are massive, branching and/or foliose. The reason for this difference may be tolerance to strong waves, fluctuations in water temperature, and turbidity-sedimentation caused by terrestrial input. Since foliose and branching species of Montipora are more fragile to waves while branching and massive species of Porites and branching Montipora are more tolerant to turbidity-sedimentation and high watertemperature. Insolation intensity may also explain these differences. Nevertheless, long-term monitoring of scleractinian corals in slopes and moats is essential to understand the recovery of healthy coral reefs at this island.


## (c) Future perspective

As such, the eDNA method involves intrinsic technical problems. Nevertheless, overall results of eDNA method are royalsocietypublishing.org/journal/rspb Proc. R. Soc. B 290: 20230026 comparable with those obtained by diving observations. That is, the two types of results agree at nearly 90% of locations examined in this study (table 1). This provides strong support for eDNA method as a powerful tool to monitor coral reefs more broadly, deeply and robustly than conventional diving surveys. Our previous eDNA study surveyed coral genera in May 2021 at three points along the Onna Village seashore and recorded abundances of Acropora (62%), Pocillopora (15%), Montipora (4%), Dipsastraea (2%), Porites (2%), Favites (2%) and others [15]. Present diving observations at point #27 in October 2021, closest to the previously surveyed area, recognized Acropora as dominant and Pocillopora as sub-dominant (table 1). In addition, the present eDNA barcoding found occurrences of Acropora (73%), Pocillopora (7.5%), Montipora (5%), Porites (3%), Fungia (1%), Platygyra (1.7%), Favites (0.5%) and Dipsastraea (0.5%) ( figure 3). This provides further support for the robustness of coral-specific eDNA metabarcoding.

In addition, only 1 l of surface seawater was sufficient for eDNA surveying [11][12][13][14][15]. Therefore, occurrence of scleractinian corals can be robustly monitored using eDNA, hopefully enabling more efficacious plans for coral reef preservation and restoration. In the last two and half decades, dominant scleractinian coral taxa have confronted growing crises due to global climate change and anthropogenic insults. It is urgent to document the present status of corals in reefs worldwide, including at isolated islands in the Pacific Ocean. More comprehensive monitoring of such reefs can be accomplished using this eDNA method.

Data accessibility. The dataset used in this study is publicly accessible:

## Figure 1 .
1Sixty-three coral reef monitoring locations around Okinawa Island. Collections sites and representative coral genera are shown. Colour codes for genera are shown in the right corner. The order (from left to right) does not always indicate relative dominance of genera. Moats are indicated. Other locations are slopes. At locations #35, 36, 37 and 62, both slopes and moats were examined by diving, whereas only eDNA data were obtained from moats.

## Figure 2 .
2Distribution of various scleractinian corals at a reef slope (#8, Gesashi Uppama East Slope) (a) and a moat (#21, Ohdo East Moat) (b). Dominant genera in (a) are tabular Acropora and branching Pocillopora and (b) massive and branching Porites and tabular Acropora.


Proc. R. Soc. B 290: 20230026


). Amplicon sequencing libraries of cleaned PCR products were prepared using a KAPA Hyper royalsocietypublishing.org/journal/rspb Proc. R. Soc. B 290: 20230026 Prep Kit (NIPPON Genetics) without fragmentation. Libraries were multiplexed and 300-bp paired-end reads were sequenced on a MiSeq platform (Illumina) using a MiSeq Reagent kit v. 3 (600 cycles). The number of sequence reads, total base-pair length, and average and maximum length of reads of each sample are shown in electronic supplementary material, table S2.


. For example, as previously mentioned, sites #25 and #27-#29 are along the coastline of Onna Village, and although royalsocietypublishing.org/journal/rspb Proc. R. Soc. B 290: 202300260.10 
66.21 
0.00 
0.00 
0.12 
0.00 
0.00 
0.00 
0.00 
0.07 
0.00 
0.00 
0.41 
0.00 
0.15 
0.44 
1.11 
0.00 
0.08 
0.40 
0.12 
0.45 
0.00 
0.72 
0.10 
3.57 
0.00 
0.00 
0.04 
0.00 
3.46 
0.00 
0.00 
0.99 
1.77 
0.82 
0.00 
0.16 
17.69 
0.34 
0.00 
0.00 
0.68 

0.00 
28.46 
0.00 
0.00 
1.93 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
1.89 
0.00 
0.36 
1.03 
2.32 
0.00 
0.00 
0.46 
0.28 
0.03 
0.00 
0.00 
0.00 
19.48 
0.00 
0.00 
0.91 
0.00 
0.31 
0.00 
0.00 
1.36 
30.86 
6.90 
0.00 
0.00 
3.40 
0.00 
0.00 
0.00 
0.00 

0.00 
54.75 
0.00 
0.00 
0.08 
0.00 
0.00 
0.00 
0.00 
0.63 
0.00 
0.00 
2.25 
0.00 
0.00 
0.71 
2.20 
0.00 
0.00 
0.26 
0.35 
0.08 
0.00 
0.00 
0.00 
15.43 
0.00 
0.00 
0.00 
0.00 
1.64 
0.00 
0.00 
3.22 
2.20 
6.71 
0.00 
0.00 
9.49 
0.01 
0.00 
0.00 
0.00 

0.00 
37.09 
0.00 
0.00 
1.05 
0.00 
0.00 
0.23 
0.00 
2.78 
0.00 
0.00 
1.08 
0.00 
0.00 
1.34 
1.99 
0.00 
0.00 
0.32 
0.00 
7.01 
0.00 
0.00 
0.00 
13.60 
0.00 
0.00 
0.00 
0.00 
16.22 
0.00 
0.00 
0.85 
1.18 
1.45 
0.00 
0.01 
13.74 
0.05 
0.00 
0.00 
0.00 

0.00 
66.47 
0.01 
0.00 
0.88 
0.00 
0.00 
0.00 
0.00 
0.20 
0.00 
0.01 
0.98 
0.00 
0.00 
0.30 
1.73 
0.00 
0.00 
0.04 
0.31 
0.37 
0.00 
0.01 
0.00 
1.52 
0.00 
0.00 
0.08 
0.00 
16.32 
0.00 
0.00 
0.43 
1.72 
2.08 
0.00 
0.00 
5.85 
0.47 
0.00 
0.00 
0.22 

0.00 
35.45 
0.00 
0.00 
5.20 
0.00 
0.00 
0.00 
0.00 
1.28 
0.00 
0.00 
2.01 
0.00 
0.00 
0.00 
2.60 
0.00 
0.00 
0.00 
3.55 
4.33 
0.00 
0.00 
0.00 
13.01 
0.00 
0.00 
0.00 
0.00 
2.06 
0.00 
0.00 
0.76 
9.19 
10.47 
0.00 
0.00 
8.09 
0.00 
0.00 
0.00 
2.01 

0.00 
55.05 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.20 
0.00 
0.00 
0.60 
0.00 
0.00 
1.74 
2.56 
0.00 
0.00 
0.00 
0.00 
0.02 
0.00 
0.00 
0.00 
18.77 
0.00 
0.00 
0.00 
0.00 
1.42 
0.00 
0.00 
0.44 
3.84 
9.25 
0.09 
0.00 
5.99 
0.01 
0.01 
0.00 
0.00 

0.00 
42.57 
0.00 
0.06 
0.08 
0.00 
0.00 
0.00 
0.00 
0.14 
0.19 
0.00 
1.10 
0.00 
0.00 
0.00 
4.83 
0.00 
0.00 
0.00 
5.49 
1.33 
0.00 
0.00 
0.00 
15.84 
0.00 
0.00 
0.30 
0.00 
2.16 
0.00 
0.00 
1.26 
9.55 
3.54 
0.00 
0.15 
11.22 
0.18 
0.00 
0.00 
0.00 

0.00 
42.03 
0.00 
0.00 
1.33 
0.00 
0.11 
0.00 
0.00 
1.81 
0.00 
0.00 
0.49 
0.00 
0.00 
0.19 
7.47 
0.00 
0.00 
0.11 
0.14 
1.81 
0.01 
0.00 
0.00 
11.17 
0.00 
0.00 
0.00 
0.00 
2.77 
0.00 
0.00 
1.13 
2.64 
1.10 
0.00 
0.06 
25.24 
0.40 
0.00 
0.00 
0.00 

0.00 
18.95 
0.00 
1.76 
0.00 
0.00 
0.00 
0.00 
0.00 
1.49 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.52 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
57.50 
0.00 
0.00 
0.00 
0.00 
2.41 
0.00 
0.00 
0.00 
6.14 
11.23 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 

0.00 
36.90 
0.00 
0.00 
0.04 
0.00 
0.00 
0.00 
0.00 
0.47 
0.97 
0.00 
0.89 
0.00 
0.00 
0.00 
0.96 
0.00 
0.14 
1.65 
0.33 
0.00 
0.00 
0.00 
0.00 
11.24 
0.00 
0.00 
0.00 
0.00 
28.78 
0.00 
0.00 
0.00 
0.84 
13.39 
0.00 
0.00 
3.38 
0.00 
0.00 
0.00 
0.00 

0.00 
10.82 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
3.71 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
76.36 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
8.85 
0.04 
0.00 
0.00 
0.22 
0.00 
0.00 
0.00 
0.00 

0.00 
50.75 
0.00 
0.00 
0.01 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
1.53 
0.00 
0.00 
0.46 
2.49 
0.03 
0.00 
0.00 
0.00 
20.82 
0.00 
0.00 
0.00 
0.00 
0.34 
0.00 
0.00 
0.00 
2.27 
6.74 
0.00 
0.00 
14.56 
0.00 
0.00 
0.00 
0.00 

0.00 
79.90 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.66 
0.00 
0.00 
1.84 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.30 
0.00 
0.00 
0.00 
0.00 
10.24 
0.00 
0.00 
0.00 
0.00 
1.56 
0.00 
0.00 
5.49 
0.00 
0.00 
0.00 
0.00 

0.00 
39.66 
0.00 
0.00 
0.01 
0.00 
0.00 
0.00 
0.00 
7.37 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
1.56 
0.00 
0.00 
0.46 
0.16 
0.01 
0.00 
0.00 
0.00 
10.52 
0.00 
0.00 
0.00 
0.00 
2.13 
0.00 
0.00 
4.09 
4.03 
0.02 
1.24 
0.00 
28.75 
0.00 
0.00 
0.00 
0.00 

0.00 
30.23 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.67 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
3.59 
0.00 
0.00 
1.21 
0.00 
42.54 
0.00 
0.00 
10.53 
0.00 
5.67 
0.52 
0.00 
0.00 
1.92 
3.12 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 

0.00 
64.84 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.09 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
35.07 
0.00 
0.00 
0.00 
0.00 

0.00 
19.30 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
3.70 
0.00 
0.00 
2.21 
9.14 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
42.85 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.31 
12.41 
10.08 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 

0.00 
0.02 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
97.09 
0.00 
0.00 
0.00 
0.00 
1.39 
0.00 
0.00 
0.00 
0.00 
0.61 
0.00 
0.00 
0.89 
0.00 
0.00 
0.00 
0.00 

0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
100.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 

0.00 
16.27 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
4.55 
0.00 
0.00 
2.49 
0.00 
0.00 
0.00 
4.57 
0.00 
0.00 
0.96 
0.00 
0.00 
0.00 
0.00 
0.00 
23.19 
0.00 
0.00 
0.00 
0.00 
40.76 
0.00 
0.00 
0.24 
0.32 
0.74 
0.00 
0.23 
4.54 
1.13 
0.00 
0.00 
0.00 

0.00 
7.50 
0.00 
0.00 
0.01 
0.00 
0.00 
0.00 
0.00 
0.21 
0.00 
0.63 
2.20 
0.00 
0.10 
0.41 
11.53 
0.00 
0.00 
0.78 
1.52 
0.01 
0.02 
0.36 
0.00 
44.73 
0.00 
0.00 
0.00 
0.00 
6.34 
0.00 
0.00 
2.45 
2.23 
0.53 
0.00 
0.11 
9.80 
8.01 
0.00 
0.00 
0.52 

0.00 
2.27 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
14.79 
0.00 
0.00 
0.00 
0.00 
0.00 
0.43 
2.16 
0.00 
0.00 
3.92 
0.00 
0.00 
0.08 
0.00 
0.00 
35.31 
0.00 
0.00 
0.00 
0.00 
8.97 
0.00 
0.00 
1.22 
0.00 
5.45 
0.00 
0.11 
20.51 
4.78 
0.00 
0.00 
0.00 

0.00 
0.00 
0.02 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
52.75 
0.00 
0.00 
4.94 
0.00 
0.00 
0.00 
3.42 
0.00 
0.00 
0.35 
0.00 
0.00 
0.00 
0.00 
0.00 
0.10 
0.00 
0.00 
0.00 
0.00 
28.95 
0.00 
0.00 
0.54 
0.26 
0.20 
0.00 
0.00 
8.46 
0.00 
0.00 
0.00 
0.00 

0.00 
61.47 
0.00 
0.00 
0.02 
0.00 
0.00 
0.00 
0.00 
1.23 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.59 
0.00 
0.00 
0.00 
0.13 
0.00 
0.46 
0.00 
9.69 
0.00 
0.00 
0.00 
0.75 
0.00 
0.00 
0.00 
0.00 
0.00 
10.82 
0.38 
0.00 
13.41 
0.00 
0.00 
0.00 
1.05 

0.00 
48.80 
0.00 
0.35 
0.15 
0.00 
0.00 
0.00 
0.00 
0.83 
0.15 
0.00 
0.32 
0.00 
0.00 
0.00 
0.52 
0.58 
0.00 
0.00 
0.19 
0.55 
0.01 
0.00 
0.00 
0.83 
0.00 
0.00 
1.63 
0.05 
0.91 
0.00 
0.00 
0.07 
1.24 
0.00 
0.00 
0.00 
40.07 
0.46 
0.00 
0.00 
2.30 

0.00 
73.53 
0.00 
0.00 
0.02 
0.00 
0.00 
0.00 
0.00 
0.49 
0.00 
0.30 
0.44 
0.00 
0.00 
1.35 
1.29 
0.00 
0.00 
0.00 
0.17 
0.06 
0.00 
0.89 
0.00 
5.64 
0.00 
0.00 
0.89 
0.00 
0.67 
0.00 
0.00 
1.65 
0.49 
7.02 
0.00 
0.00 
3.34 
1.32 
0.00 
0.00 
0.45 

0.00 
75.43 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
1.14 
0.87 
0.00 
0.60 
0.00 
0.00 
0.00 
0.38 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.22 
0.00 
4.31 
0.00 
0.00 
0.00 
0.00 
10.42 
0.00 
0.00 
0.21 
0.88 
3.90 
0.00 
0.00 
1.64 
0.00 
0.00 
0.00 
0.00 

0.00 
67.49 
0.00 
0.00 
0.01 
0.00 
0.00 
0.00 
0.00 
2.44 
0.00 
0.00 
1.53 
0.00 
0.00 
0.00 
1.38 
0.00 
0.00 
0.00 
0.54 
0.00 
0.00 
1.14 
0.00 
4.65 
0.00 
0.00 
0.51 
0.00 
1.25 
0.00 
0.00 
1.69 
0.97 
11.58 
0.00 
0.25 
3.26 
1.30 
0.00 
0.00 
0.00 

0.00 
32.68 
0.00 
0.00 
0.01 
0.00 
0.00 
0.00 
0.00 
8.24 
0.00 
0.00 
0.00 
0.00 
0.00 
0.95 
5.96 
0.00 
0.00 
0.00 
0.96 
0.00 
0.00 
2.68 
0.00 
5.52 
0.00 
0.00 
0.00 
0.00 
2.48 
0.00 
0.00 
5.29 
3.79 
27.47 
0.00 
0.93 
3.03 
0.00 
0.00 
0.00 
0.00 

0.00 
82.59 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.64 
0.59 
0.00 
0.00 
0.00 
0.00 
0.00 
0.09 
0.00 
0.00 
2.11 
0.30 
0.00 
0.00 
0.17 
0.00 
9.84 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.27 
3.10 
0.00 
0.08 
0.22 
0.00 
0.00 
0.00 
0.00 

1 
2 
3 
4 
5 
6 
7 
8 
9 
1 0 
11 
12 
13 
14 
15 
16 
17 
18 
19 
20 (m) 
21 (m) 

22 (m) 23 (m) 24 (m) 
25 
26 (m) 
27 
28 
29 
30 
31 

0.00 
78.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
5.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.99 
1.73 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
2.51 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.67 
0.00 
0.00 
0.00 
9.04 
0.27 
0.00 
0.00 
1.78 

0.00 
58.75 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
2.70 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
5.18 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
13.95 
0.00 
0.00 
4.79 
1.70 
5.52 
0.00 
0.00 
0.00 
7.15 
0.00 
0.00 
0.00 
0.00 
0.27 
0.00 
0.00 
0.00 

0.00 
77.45 
0.00 
0.00 
0.04 
0.00 
0.00 
0.00 
0.00 
0.24 
0.31 
0.23 
0.00 
0.26 
0.00 
0.44 
2.68 
0.00 
0.00 
0.19 
0.32 
0.01 
0.00 
0.20 
0.23 
1.79 
0.00 
0.00 
0.00 
0.00 
0.34 
0.00 
0.00 
0.50 
1.05 
12.47 
0.00 
0.08 
0.67 
0.14 
0.00 
0.00 
0.36 

0.00 
10.17 
0.00 
0.00 
0.04 
0.00 
0.00 
0.00 
0.00 
9.54 
0.00 
0.00 
0.00 
0.00 
0.00 
0.19 
0.00 
0.00 
0.06 
0.33 
0.24 
0.07 
0.16 
0.00 
0.00 
66.37 
0.00 
0.00 
0.00 
0.00 
1.09 
0.00 
0.00 
0.26 
0.36 
1.25 
0.00 
0.30 
9.24 
0.34 
0.00 
0.00 
0.00 

0.00 
21.62 
0.00 
0.01 
0.14 
0.00 
0.00 
0.00 
0.00 
3.56 
0.00 
0.00 
0.25 
0.00 
0.00 
0.00 
0.42 
0.00 
0.00 
0.17 
0.01 
0.07 
0.15 
0.00 
0.00 
54.76 
0.00 
0.00 
0.00 
0.00 
1.00 
0.00 
0.00 
2.77 
0.57 
1.03 
0.00 
0.52 
10.44 
1.51 
0.00 
0.00 
1.01 

0.02 
1.06 
0.00 
0.01 
58.50 
0.00 
0.08 
0.00 
0.00 
5.84 
0.00 
0.00 
0.50 
0.00 
0.00 
0.00 
0.45 
0.00 
0.00 
0.33 
0.10 
1.99 
0.00 
0.00 
0.00 
2.87 
0.00 
0.01 
0.00 
0.00 
15.78 
0.00 
0.00 
1.09 
2.03 
2.41 
0.11 
0.62 
5.22 
0.95 
0.00 
0.00 
0.00 

0.00 
89.71 
0.00 
0.17 
0.02 
0.00 
0.00 
0.00 
0.00 
1.09 
0.13 
0.20 
0.61 
0.00 
0.00 
0.23 
0.46 
0.00 
0.00 
0.00 
0.11 
0.05 
0.00 
0.10 
0.00 
3.36 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.49 
0.54 
0.91 
0.00 
0.00 
1.82 
0.00 
0.00 
0.00 
0.00 

0.00 
25.79 
0.00 
0.00 
0.17 
0.00 
0.00 
0.00 
0.00 
2.19 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
4.33 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
6.17 
0.00 
43.06 
0.00 
0.00 
0.00 
0.00 
11.71 
0.00 
0.00 
4.04 
0.00 
0.00 
0.00 
0.00 
2.54 
0.00 
0.00 
0.00 
0.00 

0.00 
41.91 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.09 
0.00 
3.95 
0.00 
0.00 
0.00 
11.22 
0.00 
0.00 
0.00 
3.13 
0.00 
0.00 
0.00 
0.00 
21.64 
0.00 
0.00 
8.45 
0.00 
0.00 
0.00 
0.00 
0.00 
1.06 
1.36 
0.00 
0.00 
7.18 
0.00 
0.00 
0.00 
0.00 

0.00 
21.56 
0.00 
0.00 
0.01 
0.00 
0.00 
0.24 
0.30 
0.00 
2.67 
0.00 
1.82 
0.00 
0.00 
0.00 
2.82 
3.19 
0.00 
0.00 
3.65 
0.00 
0.00 
0.00 
0.00 
21.99 
0.00 
0.00 
13.55 
0.00 
0.00 
0.00 
0.00 
3.49 
0.00 
0.71 
0.00 
0.00 
22.45 
1.53 
0.00 
0.00 
0.00 

0.00 
34.72 
0.00 
4.76 
0.00 
0.00 
0.00 
0.00 
0.00 
0.04 
0.00 
0.00 
1.88 
0.00 
0.00 
0.00 
0.30 
0.00 
0.00 
0.00 
4.78 
0.00 
0.00 
2.35 
0.00 
15.06 
0.00 
0.00 
0.00 
0.00 
3.38 
0.00 
0.00 
0.19 
2.33 
23.01 
0.00 
0.00 
7.20 
0.00 
0.00 
0.00 
0.00 

0.00 
93.51 
0.00 
0.00 
0.01 
0.00 
0.00 
0.00 
0.00 
0.41 
0.00 
0.00 
0.06 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.49 
0.12 
0.01 
0.00 
0.00 
0.00 
3.77 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.01 
0.02 
0.22 
1.08 
0.00 
0.07 
0.01 
0.22 
0.00 
0.00 
0.00 

0.00 
30.17 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
26.55 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
5.55 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
3.23 
0.00 
10.16 
0.00 
0.00 
24.34 
0.00 
0.00 
0.00 
0.00 

2.62 
27.23 
0.00 
0.00 
0.01 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
5.54 
0.00 
5.99 
0.00 
0.00 
0.06 
0.00 
0.00 
0.00 
0.00 
0.00 
16.71 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
20.87 
0.00 
0.00 
20.97 
0.00 
0.00 
0.00 
0.00 

0.00 
13.56 
0.00 
0.00 
2.01 
0.00 
0.00 
0.00 
0.00 
4.73 
0.00 
0.00 
12.21 
0.00 
1.33 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
11.95 
0.00 
0.00 
0.00 
0.00 
4.77 
0.00 
0.00 
8.72 
5.61 
28.48 
0.00 
0.00 
6.64 
0.00 
0.00 
0.00 
0.00 

0.04 
73.65 
0.00 
0.00 
0.04 
0.00 
0.00 
0.00 
0.00 
0.51 
0.00 
0.34 
0.57 
0.29 
0.00 
0.52 
1.56 
0.00 
0.07 
0.07 
0.00 
0.47 
0.00 
0.39 
0.00 
4.59 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
1.89 
1.30 
9.86 
0.00 
0.24 
3.58 
0.00 
0.00 
0.00 
0.00 

0.00 
59.24 
0.00 
0.00 
0.08 
0.01 
0.00 
0.00 
0.00 
0.32 
1.24 
0.00 
3.97 
0.00 
0.00 
0.00 
0.01 
0.35 
0.00 
0.00 
1.53 
0.02 
0.00 
0.00 
0.00 
11.43 
0.00 
0.00 
0.00 
0.00 
2.20 
0.00 
0.00 
1.54 
3.75 
10.39 
0.00 
0.00 
3.91 
0.00 
0.00 
0.00 
0.00 

0.00 
81.77 
0.00 
0.00 
0.11 
0.00 
0.00 
0.00 
0.00 
1.25 
0.28 
0.00 
0.00 
0.48 
0.62 
0.00 
2.69 
0.00 
0.00 
0.00 
0.00 
0.10 
0.00 
0.27 
0.00 
1.79 
0.00 
0.00 
0.00 
0.24 
0.95 
0.00 
0.00 
0.16 
0.66 
1.05 
0.00 
0.34 
7.22 
0.00 
0.00 
0.00 
0.00 

0.04 
66.87 
0.00 
0.03 
0.07 
0.01 
0.00 
0.44 
0.00 
4.69 
0.00 
0.00 
1.00 
0.00 
0.00 
1.97 
6.87 
0.00 
0.00 
0.00 
2.30 
0.05 
0.00 
0.47 
0.00 
2.20 
0.00 
0.01 
0.00 
0.00 
0.00 
0.00 
0.00 
0.14 
3.81 
2.08 
0.00 
0.03 
4.29 
1.57 
0.00 
0.00 
1.06 

0.00 
33.74 
0.00 
0.01 
0.06 
0.01 
0.00 
0.00 
0.00 
0.43 
0.00 
0.00 
5.47 
0.00 
0.00 
0.00 
2.78 
0.00 
0.20 
0.58 
0.27 
0.00 
0.00 
0.46 
0.00 
12.24 
0.00 
0.00 
0.59 
0.00 
4.87 
0.00 
0.00 
4.27 
3.43 
27.41 
0.00 
0.15 
2.67 
0.38 
0.00 
0.00 
0.00 

0.00 
56.82 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
2.23 
0.00 
0.00 
0.00 
0.01 
0.00 
0.00 
0.00 
5.10 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
1.52 
0.00 
2.77 
0.00 
0.00 
2.65 
0.00 
21.21 
0.00 
0.00 
7.70 
0.00 
0.00 
0.00 
0.00 

0.00 
46.72 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.02 
0.00 
0.00 
0.00 
0.00 
0.00 
0.04 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
44.76 
0.00 
0.00 
8.47 
0.00 
0.00 
0.00 
0.00 

0.00 
27.75 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
1.34 
8.15 
0.00 
0.00 
0.00 
0.00 
0.00 
8.18 
1.80 
0.00 
0.00 
5.90 
0.01 
0.00 
0.00 
0.00 
2.04 
0.00 
0.00 
3.95 
2.13 
0.00 
4.98 
0.00 
0.98 
0.00 
2.53 
0.00 
0.00 
27.95 
0.00 
0.00 
2.31 
0.00 

0.00 
100.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 

0.00 
82.11 
0.00 
0.15 
0.09 
0.00 
0.00 
0.00 
0.00 
0.45 
0.00 
0.00 
0.51 
0.00 
0.00 
1.17 
0.18 
0.00 
0.00 
0.06 
0.21 
0.10 
0.11 
0.19 
0.00 
6.53 
0.00 
0.00 
0.00 
0.00 
1.26 
0.00 
0.00 
0.37 
1.00 
1.76 
0.00 
0.01 
3.60 
0.08 
0.06 
0.00 
0.00 

0.00 
70.44 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
29.56 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 

0.00 
80.37 
0.00 
0.00 
0.01 
0.00 
0.00 
0.00 
0.00 
0.91 
0.00 
0.57 
0.44 
0.00 
0.00 
0.00 
0.13 
0.00 
0.00 
0.00 
1.03 
0.01 
0.04 
0.00 
0.00 
3.82 
0.00 
0.00 
0.00 
0.00 
0.70 
0.00 
0.00 
0.98 
2.09 
3.99 
0.00 
0.00 
2.67 
0.01 
0.00 
0.00 
2.23 

0.00 
21.47 
0.00 
0.00 
14.26 
0.00 
0.00 
0.00 
0.00 
18.01 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
29.09 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
17.17 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 

0.00 
64.52 
0.00 
0.01 
0.02 
0.00 
0.00 
0.00 
0.00 
6.16 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
6.26 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
3.32 
0.00 
0.00 
0.00 
0.00 
2.04 
0.00 
0.00 
0.00 
0.00 
0.00 
0.00 
0.37 
16.01 
1.27 
0.00 
0.00 
0.00 

0.00 
20.60 
0.00 
0.00 
2.19 
0.00 
0.00 
0.00 
0.00 
0.66 
0.00 
0.00 
0.60 
0.00 
0.00 
0.00 
6.45 
0.00 
0.03 
0.51 
0.00 
0.04 
0.00 
0.00 
0.00 
19.51 
0.00 
0.00 
0.00 
0.00 
6.19 
0.00 
0.00 
7.70 
2.93 
11.01 
0.00 
0.14 
10.74 
2.64 
0.00 
0.00 
8.05 

0.00 
45.99 
0.00 
0.00 
0.33 
0.00 
0.00 
0.47 
0.00 
0.97 
0.00 
0.00 
6.02 
0.00 
0.00 
0.00 
5.36 
0.00 
0.00 
0.00 
0.00 
0.86 
0.00 
0.00 
0.00 
21.76 
0.00 
0.00 
0.00 
0.00 
1.12 
0.00 
0.00 
0.00 
1.09 
6.60 
0.00 
0.39 
9.01 
0.00 
0.02 
0.00 
0.00 

32 
33 
34 
35 (m) 36 (m) 37 (m) 
38 
39 
40 
41 
43 

44 (m) 
45 
46 
47 
48 
49 
50 
51 
52 
53 
54 
55 
56 
57 
58 
59 
60 
61 
62 (m) 63 (m) 

0 

20 

40 

60 

80 
Table 1. Comparison of scleractinian corals called by direct diver obseration and coral-specific eDNA barcoding method. 


Acknowledgements. We thank Yoshie Nishitsuji and Mayuki Suwa for eDNA library preparation and sequencing. Dr Steven D. Aird (sdatechnical-editor.org) is acknowledged for editing the manuscript.Conflict of interest declaration. We declare no competing interests.
Electronic supplementary material is available from Figshare. 10.5061/dryad.5dv41ns9j29https://doi.org/10.5061/dryad.5dv41ns9j [29]. Electronic sup- plementary material is available from Figshare [30].

conceptualization, data curation, formal analysis, investigation, resources, supervision, validation; T.N.: conceptualization, data curation, formal analysis, investigation. K N Authors&apos; Contributions, project administration, writing-original draftAuthors' contributions. K.N.: conceptualization, data curation, formal analysis, investigation, resources, supervision, validation; T.N.: con- ceptualization, data curation, formal analysis, investigation, project administration, writing-original draft;

data curation, investigation. H N , H.N.: data curation, investi- gation;

data curation, formal analysis, investigation. M K , M.K.: data curation, formal analysis, investigation;

data curation, investigation, software; C.S.: data curation, investigation, software, supervision; N.S.: conceptualization, investigation, project administration, validation, writing-original draft. All authors gave final approval for publication and agreed to be held accountable for the work performed therein. K H , K.H.: data curation, investigation, software; C.S.: data curation, investi- gation, software, supervision; N.S.: conceptualization, investigation, project administration, validation, writing-original draft. All authors gave final approval for publication and agreed to be held accountable for the work performed therein.

Coral reef biodiversity. N Knowlton, R E Brainard, R Fisher, M Moews, L Plaisance, M J Caley, Life in the world's oceans: diversity, distribution, and abundance. Chichester, UKWiley-BlackwellKnowlton N, Brainard RE, Fisher R, Moews M, Plaisance L, Caley MJ. 2010 Coral reef biodiversity. In Life in the world's oceans: diversity, distribution, and abundance (ed. AD McIntyre), pp. 65-79. Chichester, UK: Wiley-Blackwell.

The 27-year decline of coral cover on the Great Barrier Reef and its causes. G De&apos;ath, K E Fabricius, H Sweatman, M Puotinen, 10.1073/pnas.1208909109Proc. Natl Acad. Sci. USA. 109De'ath G, Fabricius KE, Sweatman H, Puotinen M. 2012 The 27-year decline of coral cover on the Great Barrier Reef and its causes. Proc. Natl Acad. Sci. USA 109, 17 995-17 999. (doi:10.1073/pnas. 1208909109)

Species-specific responses of corals to bleaching events on anthropogenically turbid reefs on Okinawa Island, Japan, over a 15-year Period. C Hongo, H Yamano, 10.1371/journal.pone.0060952PLoS ONE. 8Hongo C, Yamano H. 2013 Species-specific responses of corals to bleaching events on anthropogenically turbid reefs on Okinawa Island, Japan, over a 15-year Period (1995-2009). PLoS ONE 8, e60952. (doi:10.1371/journal.pone.0060952)

Shifting communities after typhoon damage on an upper mesophotic reef in Okinawa. K N White, D K Weinstein, T Ohara, V Denis, J Montenegro, J D Reimer, 10.7717/peerj.3573Japan. PeerJ. 53573White KN, Weinstein DK, Ohara T, Denis V, Montenegro J, Reimer JD. 2017 Shifting communities after typhoon damage on an upper mesophotic reef in Okinawa, Japan. PeerJ 5, e3573. (doi:10.7717/peerj.3573)

Spatial and temporal population dynamics of the crown-of-thorns starfish, Acanthaster planci, over a 24-year period along the central west coast of Okinawa Island. M Nakamura, K Okaji, H Higa, E Yamakawa, S Mitarai, 10.1007/s00227-014-2524-5Mar. Biol. 161Nakamura M, Okaji K, Higa H, Yamakawa E, Mitarai S. 2014 Spatial and temporal population dynamics of the crown-of-thorns starfish, Acanthaster planci, over a 24-year period along the central west coast of Okinawa Island, Japan. Mar. Biol. 161, 2521-2530. (doi:10.1007/s00227-014-2524-5)

Expanding walls and shrinking beaches: loss of natural coastline in Okinawa Island. G D Masucci, J D Reimer, 10.7717/peerj.7520doi:10. 7717/peerj.7520Japan. PeerJ. 77520Masucci GD, Reimer JD. 2020 Expanding walls and shrinking beaches: loss of natural coastline in Okinawa Island, Japan. PeerJ 7, e7520. (doi:10. 7717/peerj.7520)

Morphological plasticity in scleractinian corals. P A Todd, 10.1111/j.1469-185X.2008.00045.xdoi:10. 1111/j.1469-185X.2008.00045Biol. Rev. 83Todd PA. 2008 Morphological plasticity in scleractinian corals. Biol. Rev. 83, 315-337. (doi:10. 1111/j.1469-185X.2008.00045.x)

Using seawater to document coral-zoothanthella diversity: a new approach to coral reef monitoring using environmental DNA. Front. C Shinzato, Y Zayasu, S Kanda, M Kawamitsu, N Satoh, H Yamashita, G Suzuki, 10.3389/fmars.2018.00028doi:10. 3389/fmars.2018.00028Mar. Sci. 5Shinzato C, Zayasu Y, Kanda S, Kawamitsu M, Satoh N, Yamashita H, Suzuki G. 2018 Using seawater to document coral-zoothanthella diversity: a new approach to coral reef monitoring using environmental DNA. Front. Mar. Sci. 5, 28. (doi:10. 3389/fmars.2018.00028)

. K Bohmann, A Evans, M T Gilbert, G R Carvalho, S Creer, M Knapp, W Y Douglas, M De Bruyn, Bohmann K, Evans A, Gilbert MT, Carvalho GR, Creer S, Knapp M, Douglas WY, De Bruyn M. 2014

Environmental DNA for wildlife biology and biodiversity monitoring. 10.1016/j.tree.2014.04.003Trends Ecol. Evol. 29Environmental DNA for wildlife biology and biodiversity monitoring. Trends Ecol. Evol. 29, 358-367. (doi:10.1016/j.tree.2014.04.003)

Environmental monitoring. Harnessing DNA to improve environmental management. R P Kelly, 10.1126/science.1251156doi:10. 1126/science.1251156Science. 344Kelly RP et al. 2014 Environmental monitoring. Harnessing DNA to improve environmental management. Science 344, 1455-1456. (doi:10. 1126/science.1251156)

Environmental DNA can act as a biodiversity barometer of anthropogenic pressures in coastal ecosystems. J D Dibattista, J D Reimer, M Stat, G D Masucci, P Biondi, De Brauwer, M Wilkinson, S P Chariton, A A Bunce, M , 10.1038/s41598-020-64858-9doi:10.1038/ s41598-020-64858-9Sci. Rep. 10DiBattista JD, Reimer JD, Stat M, Masucci GD, Biondi P, De Brauwer M, Wilkinson SP, Chariton AA, Bunce M. 2020 Environmental DNA can act as a biodiversity barometer of anthropogenic pressures in coastal ecosystems. Sci. Rep. 10, 8635. (doi:10.1038/ s41598-020-64858-9)

Rapid assessment of coral cover from environmental. P K Nichols, P B Marko, 10.1002/edn3.8DNA in Hawai'i. Environ. DNA. 1Nichols PK, Marko PB. 2019 Rapid assessment of coral cover from environmental DNA in Hawai'i. Environ. DNA 1, 40-53. (doi:10.1002/edn3.8)

Development of a multiassay approach for monitoring coral diversity using eDNA metabarcoding. J B Alexander, 10.1007/s00338-019-01875-9Coral Reefs. 39Alexander JB et al. 2020 Development of a multi- assay approach for monitoring coral diversity using eDNA metabarcoding. Coral Reefs 39, 159-171. (doi:10.1007/s00338-019-01875-9)

Coral monitoring in northwest Australia with environmental DNA metabarcoding using a curated reference database for optimized detection. L Dugal, 10.1002/edn3.199doi:10.1002/ edn3.199Environ. DNA. 4Dugal L et al. 2021 Coral monitoring in northwest Australia with environmental DNA metabarcoding using a curated reference database for optimized detection. Environ. DNA 4, 63-76. (doi:10.1002/ edn3.199)

Novel mitochondrial DNA markers for scleractinian corals and generic-level environmental DNA metabarcoding. C Shinzato, H Narisoko, K Nishitsuji, T Nagata, N Satoh, J Inoue, 10.3389/fmars.2021.758207Front. Mar. Sci. 8758207Shinzato C, Narisoko H, Nishitsuji K, Nagata T, Satoh N, Inoue J. 2021 Novel mitochondrial DNA markers for scleractinian corals and generic-level environmental DNA metabarcoding. Front. Mar. Sci. 8, 758207. (doi:10.3389/fmars.2021.758207)

Coral bleaching: the winners and the losers. Y Loya, K Sakai, K Yamazato, Y Nakano, H Sambali, R Van Woesik, Ecol. Lett. 4Loya Y, Sakai K, Yamazato K, Nakano Y, Sambali H, van Woesik R. 2001 Coral bleaching: the winners and the losers. Ecol. Lett. 4, 122-131.

High-resolution modeling of thermal thresholds and environmental influences on coral bleaching for local and regional reef management. N H Kumagai, H Yamano, Committee Sango, 10.7717/peerj.4382PeerJ. 64382Map-ProjectKumagai NH, Yamano H, Committee Sango-Map- Project. 2018 High-resolution modeling of thermal thresholds and environmental influences on coral bleaching for local and regional reef management. PeerJ 6, e4382. (doi:10.7717/peerj.4382)

Bleaching and postbleaching mortality of Acropora corals on a heat-susceptible reef in 2016. K Sakai, T Singh, A Iguchi, 10.7717/peerj.8138PeerJ. 78138Sakai K, Singh T, Iguchi A. 2019 Bleaching and post- bleaching mortality of Acropora corals on a heat-susceptible reef in 2016. PeerJ 7, e8138. (doi:10.7717/peerj.8138)

An illustrated manual for environmental DNA research: water sampling guidelines and experimental protocols. T Minamoto, 10.1002/edn3.121Environ. DNA. 3Minamoto T et al. 2021 An illustrated manual for environmental DNA research: water sampling guidelines and experimental protocols. Environ. DNA 3, 8-13. (doi:10.1002/edn3.121)

Hermatypic corals of Japan. M Nishihira, Jen Veron, KaiyushaTokyo, JapanNishihira M, Veron JEN. 1995 Hermatypic corals of Japan. Tokyo, Japan: Kaiyusha.

Short review: molecular phylogenetic analyses of reef corals. H Fukami, 10.3755/galaxea.10.47Galaxea. 10Fukami H. 2008 Short review: molecular phylogenetic analyses of reef corals. Galaxea 10, 47-55. (doi:10.3755/galaxea.10.47)

Cutadapt removes adapter sequences from high-throughput sequencing reads. M Martin, Martin M. 2011 Cutadapt removes adapter sequences from high-throughput sequencing reads.

. 10.14806/ej.17.1.200EMBnet J. 17EMBnet J. 17, 10. (doi:10.14806/ej.17.1.200)

Search and clustering orders of magnitude faster than BLAST. R C Edgar, 10.1093/bioinformatics/btq461Bioinformatics. 26Edgar RC. 2010 Search and clustering orders of magnitude faster than BLAST. Bioinformatics 26, 2460-2461. (doi:10.1093/bioinformatics/btq461)

Cd-hit: a fast program for clustering and comparing large sets of protein or royalsocietypublishing.org/journal/rspb. W Li, A Godzik, 10.1093/bioinformatics/btl158Proc. R. Soc. B. 290BioinformaticsLi W, Godzik A. 2006 Cd-hit: a fast program for clustering and comparing large sets of protein or royalsocietypublishing.org/journal/rspb Proc. R. Soc. B 290: 20230026 nucleotide sequences. Bioinformatics 22, 1658-1659. (doi:10.1093/bioinformatics/ btl158)

R: A language and environment for statistical computing. R Development Core, Team, R Foundation for Statistical Computing. Vienna, AustriaR Development Core Team. 2021 R: A language and environment for statistical computing. Vienna, Austria: R Foundation for Statistical Computing.

The Great Barrier Reef of Australia: its products and potentialities. W Saville-Kent, WH Allen & CoLondon, UKSaville-Kent W. 1893 The Great Barrier Reef of Australia: its products and potentialities. London, UK: WH Allen & Co.

2020 New Distributional records of three species of Euphylliidae (Cnidaria, Anthozoa, Hexacorallia, Scleractinia) from the Ryukyu Islands. K Fujii, Y F Kitano, H Tachikawa, 10.12782/specdiv.25.275doi:10.12782/ specdiv.25.275Japan Species Diversity. 25Fujii K, Kitano YF, Tachikawa H. 2020 New Distributional records of three species of Euphylliidae (Cnidaria, Anthozoa, Hexacorallia, Scleractinia) from the Ryukyu Islands. Japan Species Diversity 25, 272-282. (doi:10.12782/ specdiv.25.275)

Two hidden mtDNA-clades of crown-of-thorns starfish in the Pacific Ocean. Front. N Yasuda, 10.5061/dryad.5dv41ns9jdoi:10. 5061/dryad.5dv41ns9jDryad Digital Repository. 9831240Mar. Sci.Yasuda N et al. 2022 Two hidden mtDNA-clades of crown-of-thorns starfish in the Pacific Ocean. Front. Mar. Sci. 9, 831240. (doi:10.3389/fmars.2022.831240) 29. Satoh N. 2023 eDNA data set from 63 locations of Okinawa Island. Dryad Digital Repository. (doi:10. 5061/dryad.5dv41ns9j)

An environmental DNA metabarcoding survey reveals generic-level occurrence of scleractinian corals at reef slopes of Okinawa Island. K Nishitsuji, T Nagata, H Narisoko, M Kanai, K Hisata, C Shinzato, N Satoh, 10.6084/m9.figshare.c.6461943Figshare. Nishitsuji K, Nagata T, Narisoko H, Kanai M, Hisata K, Shinzato C, Satoh N. 2023 An environmental DNA metabarcoding survey reveals generic-level occurrence of scleractinian corals at reef slopes of Okinawa Island. Figshare. (doi:10.6084/m9.figshare.c.6461943)

. /journal/rspb Proc. R. Soc. B. 29020230026royalsocietypublishing.org/journal/rspb Proc. R. Soc. B 290: 20230026