# Short Survey on the Protein Modifications in Plasma during SARS-CoV-2 Infection

CorpusID: 261978336
 
tags: #Medicine, #Environmental_Science

URL: [https://www.semanticscholar.org/paper/dde10fddc106f9c7c6275c26c671d9dc73651f0a](https://www.semanticscholar.org/paper/dde10fddc106f9c7c6275c26c671d9dc73651f0a)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Short Survey on the Protein Modifications in Plasma during SARS-CoV-2 Infection
14 September 2023

Manuela Rizzi 
Pier Paolo Sainaghi 
Agnieszka Gęgotek agnieszka.gegotek@umb.edu.pl 0000-0002-5240-1346
Neve Zarkovi zarkovic@irb.hr 0000-0001-5032-0369
Biserka Orehovec biserka.orehovec@gmail.com 0000-0001-5032-0369
Morana Jaganjac morana.jaganjac@irb.hr 0000-0001-5051-1843
Suzana Borovic Sunjic suzana.borovic.sunjic@irb.hr 0000-0001-5051-1843
Elżbieta Skrzydlewska elzbieta.skrzydlewska@umb.edu.pl 0000-0001-5397-7139

1 Department of Analytical Chemistry, Medical University of Bialystok,
Kilinskiego 1, 15-069 Bialystok, Poland


2 Ruder Boskovic Institute, Div. Molecular Medicine Laboratory for Oxidative Stress Bijenicka 54,
HR-10000 Zagreb, Croatia


3 Clinical Hospital Dubrava,
HR-10000 Zagreb, Croatia

Short Survey on the Protein Modifications in Plasma during SARS-CoV-2 Infection
14 September 202383CE1133A53BA86EDDAD1A3C459B53CF10.3390/ijms241814109Received: 1 September 2023 Revised: 7 September 2023 Accepted: 13 September 2023Gęgotek, A.Zarkovic, N.Orehovec, B.Jaganjac, M.Sunjic, S.B.Skrzydlewska, E. Short Survey COVID-19plasma proteomeprotein adducts4-hydroxynonenalmalondialdehyde15-deoxy-12,14-prostaglandin J2
Although the COVID-19 pandemic has ended, it is important to understand the pathology of severe SARS-CoV-2 infection associated with respiratory failure and high mortality.The plasma proteome, including protein modification by lipid peroxidation products in COVID-19 survivors (COVID-19; n = 10) and deceased individuals (CovDeath; n = 10) was compared in samples collected upon admission to the hospital, when there was no difference in their status, with that of healthy individuals (Ctr; n = 10).The obtained results show that COVID-19 development strongly alters the expression of proteins involved in the regulation of exocytosis and platelet degranulation (top 20 altered proteins indicated by analysis of variance; p-value (False Discovery Rate) cutoff at 5%).These changes were most pronounced in the CovDeath group.In addition, the levels of 4-hydroxynonenal (4-HNE) adducts increased 2-and 3-fold, whereas malondialdehyde (MDA) adducts increased 7-and 2.5-fold, respectively, in COVID-19 and CovDeath groups.Kinases and proinflammatory proteins were particularly affected by these modifications.Protein adducts with 15-deoxy-12,14-prostaglandin J2 (15d-PGJ2) were increased 2.5-fold in COVID-19 patients, including modifications of proteins such as p53 and STAT3, whereas CovDeath showed a decrease of approximately 60% compared with Ctr.This study for the first time demonstrates the formation of lipid metabolism products-protein adducts in plasma from survived and deceased COVID-19 patients, significantly distinguishing them, which may be a predictor of the course of SARS-CoV-2 infection.

## Introduction

The pandemics of COVID-19 caused by infection with the SARS-CoV-2 coronavirus have affected the entire world.According to official data from the World Health Organization (accessed on 26 June 2023), about seven million people died from this disease within four years (2019-2023) [1].The early symptoms of SARS-CoV-2 infection resembled those of influenza infection.Many patients suffered from respiratory failure leading to interstitial changes in the lungs.However, in severe cases, renal failure, coagulation disorders, and sepsis occurred, which were the cause of the high mortality rate of the disease [2].

Therefore, the pathophysiology of COVID-19 has been extensively studied.Data from the Web of Science Core Collection (accessed on 26 June 2023) show that more than 120,000 articles have been published addressing the changes caused by SARS-CoV-2 infection (Figure 1) [3].However, less than 1% of these studies used a comprehensive proteomics approach to identify specific biomarkers for this disease or molecules that could predict the course of infection.

Int. J. Mol.Sci.2023, 24, x FOR PEER REVIEW proteomics approach to identify specific biomarkers for this disease or molecule could predict the course of infection.Previously, it has been described that the development of COVID-19 is also a panied with a significant disturbance of redox homeostasis resulting in a high le oxidative stress that is related to the severity of the damage caused by the infection The decrease in the activity of antioxidant enzymes (superoxide dismutases 1 and 2 the decrease in the amount of low-molecular-weight antioxidants (glutathione, vit A and E) observed in the patient's plasma [4] also favors pro-oxidant metabolism patient's body.As a consequence of COVID-19, increased lipid metabolism has bee served, due to both oxidative stress and increased activity of phospholipase A2 (P lipoxygenases (LOXs), and cyclooxygenases (COXs) [5,6], manifested by an incr level of lipid metabolism products, including reactive aldehydes, such as 4-hydrox enal (4-HNE) and malondialdehyde (MDA), and cyclization products like neuro tanes/isoprostanes and prostaglandins [4,5,7].

The above-mentioned lipid peroxidation products have been identified as sign agents in the promotion of proinflammatory pathways, including activation of the factor [8][9][10][11].In addition, lipid-peroxidation-derived reactive aldehydes can bind nucleophilic amino acid side chain of proteins modifying protein structure and fun which is true for both proinflammatory proteins and proteins related to antioxida sponse or apoptosis [12].To date, however, little is known about the modification o teins by lipid peroxidation products that have occurred in the bodies of COVID-1 tients.Of particular importance seem to be data correlating the degree of protein m cation by 4-HNE with the severity and impact of SARS-CoV-2 infection, which may lead to death [7].MDA adducts with the innate immune system protein (protein D also equally important for neutralizing the SARS-CoV-2 virus [13].These data al show how important protein modifications by products of lipid metabolism are f course of infection and how important they can be for the treatment process.Ther the aim of this study was to analyze the proteomic profile of plasma samples derived deceased and surviving COVID-19 patients with particular reference to the rate o mation and biological efficacy of proteins modified by products of lipid metabolism as reactive aldehydes 4-HNE and MDA, as well as 15-deoxy-12,14-prostaglandin J2 Previously, it has been described that the development of COVID-19 is also accompanied with a significant disturbance of redox homeostasis resulting in a high level of oxidative stress that is related to the severity of the damage caused by the infection [4,5].The decrease in the activity of antioxidant enzymes (superoxide dismutases 1 and 2) and the decrease in the amount of low-molecular-weight antioxidants (glutathione, vitamins A and E) observed in the patient's plasma [4] also favors pro-oxidant metabolism in the patient's body.As a consequence of COVID-19, increased lipid metabolism has been observed, due to both oxidative stress and increased activity of phospholipase A2 (PLA2), lipoxygenases (LOXs), and cyclooxygenases (COXs) [5,6], manifested by an increased level of lipid metabolism products, including reactive aldehydes, such as 4-hydroxynonenal (4-HNE) and malondialdehyde (MDA), and cyclization products like neuroprostanes/isoprostanes and prostaglandins [4,5,7].

The above-mentioned lipid peroxidation products have been identified as signaling agents in the promotion of proinflammatory pathways, including activation of the NFκB factor [8][9][10][11].In addition, lipid-peroxidation-derived reactive aldehydes can bind to the nucleophilic amino acid side chain of proteins modifying protein structure and function, which is true for both proinflammatory proteins and proteins related to antioxidant response or apoptosis [12].To date, however, little is known about the modification of proteins by lipid peroxidation products that have occurred in the bodies of COVID-19 patients.Of particular importance seem to be data correlating the degree of protein modification by 4-HNE with the severity and impact of SARS-CoV-2 infection, which may even lead to death [7].MDA adducts with the innate immune system protein (protein D) are also equally important for neutralizing the SARS-CoV-2 virus [13].These data already show how important protein modifications by products of lipid metabolism are for the course of infection and how important they can be for the treatment process.Therefore, the aim of this study was to analyze the proteomic profile of plasma samples derived from deceased and surviving COVID-19 patients with particular reference to the rate of formation and biological efficacy of proteins modified by products of lipid metabolism, such as reactive aldehydes 4-HNE and MDA, as well as 15-deoxy-12,14-prostaglandin J2 (15d-PGJ2), in comparison with healthy individuals.


## Results

Proteomics analyses revealed differences in protein expression among the three study groups: COVID-19 survivors (COVID-19), COVID-19 deceased patients (CovDeath), and healthy donors who were set as the control group (Ctr).Below are described point by point the most important statistically significant results, including those resulting from gel profiling, PCA statistical analysis, as well as levels of individual top-altered proteins and their modifications by lipid peroxidation products.Differences between all studied groups were already observed during sample preparation, as the lower parts of the SDS-PAGE gels varied in the distribution of bands between samples (Figure 2


## Results

Proteomics analyses revealed differences in protein expression among the three study groups: COVID-19 survivors (COVID-19), COVID-19 deceased patients (CovDeath), and healthy donors who were set as the control group (Ctr).Below are described point by point the most important statistically significant results, including those resulting from gel profiling, PCA statistical analysis, as well as levels of individual topaltered proteins and their modifications by lipid peroxidation products.Differences between all studied groups were already observed during sample preparation, as the lower parts of the SDS-PAGE gels varied in the distribution of bands between samples (Figure 2).A comprehensive proteomics workflow allowed the identification and label-free quantification of 1477 proteins in all analyzed samples that met the established cut-off parameters, as described in methodological section (Supplementary file T1).

The expression of 299 of these proteins was found to be significantly altered (Figure 3A).PCA showed distinct proteomic signatures for the Ctr and CovDeath groups whereas the COVID-19 survivor group showed partial overlap with the Ctr or CovDeath groups, despite the strong concentration of samples in the upper-right corner of the plot (Figure 3B).A comprehensive proteomics workflow allowed the identification and label-free quantification of 1477 proteins in all analyzed samples that met the established cut-off parameters, as described in methodological section (Supplementary file T1).

The expression of 299 of these proteins was found to be significantly altered (Figure 3A).PCA showed distinct proteomic signatures for the Ctr and CovDeath groups, whereas the COVID-19 survivor group showed partial overlap with the Ctr or CovDeath groups, despite the strong concentration of samples in the upper-right corner of the plot, (Figure 3B).This distribution was consistent with the clustering of samples in the case of the t twenty modified proteins shown in the heatmap (Figure 4A).For half of these protei the main function was defined as regulation of exocytosis (10/20) and platelet degranu tion (9/20), including fibronectin (FN1, P02751), transforming growth factor β-3 prop tein (TGFB3, P10600), apolipoprotein A-I (APOA1, Q6PGN4), transgelin-2 (TAGLN P37802), fructose-bisphosphate aldolase A (ALDOA, P02647), growth/differentiation f tor 8 (MSTN, O14793), α2-HS-glycoprotein (AHSG, P02765), coagulation factor XIII (F13A1, P00488), complement C3 (C3, P01024), and tetranectin (CLEC3B, P05452) (Figu 4B).Moreover, 70% of them were found to be glycoproteins (KW-0325) (14/20) and 65% them were modified by the formation of a bond between the thiol groups of two peptid cysteine residues (KW-1015) (13/20) (Figure 4C).

The protein expression profile was not the only factor that distinguished the analyz groups.The degree of protein modification by lipid peroxidation also varied considerab among the different study groups.A significant increase in the level of proteins modifi by reactive aldehydes was observed (Figure 5).In plasma from COVID-19 patients, HNE-protein adducts increased more than two-fold and more than three-fold in plasm from patients who did not survive because of infection.

The dominant 4-HNE-modified proteins were serpin A12 (Q8IW75), inhibitor of N κB kinase (O15111), macrophage migration inhibitory factor (P14174), and kinases: ade osine kinase (P55263), MAPK 10 (P53779), and non-specific serine/threonine protein nase (Q9Y6B5).All proteins modified by 4-HNE are shown in Figure 6.For protein a ducts with MDA, the highest level was observed in COVID-19 surviving patients, whi was seven times higher than the Ctr group.In the CovDeath group, the level of MD protein adducts was about 2.5 times higher than the Ctr group.This result was related the increased levels of MDA adducts with kinases (cAMP-dependent protein kina (P22612), ribose-phosphate pyrophosphokinase 3 (P21108), and MAP kinase-activat protein kinase 2 (P49137)), which were mainly observed in COVID-19 surviving patien Histones H2A (Q8IUE6) and H3 (Q5TEC6) were also altered to an increased extent (Figu 7).Interestingly, although a 2.5-fold increase in 15d-PGJ2 protein adducts was observ in the COVID-19 surviving group, a decrease of up to 60% was observed in the COVI This distribution was consistent with the clustering of samples in the case of the top twenty modified proteins shown in the heatmap (Figure 4A).For half of these proteins, the main function was defined as regulation of exocytosis (10/20) and platelet degranulation (9/20), including fibronectin (FN1, P02751), transforming growth factor β-3 proprotein (TGFB3, P10600), apolipoprotein A-I (APOA1, Q6PGN4), transgelin-2 (TAGLN2, P37802), fructose-bisphosphate aldolase A (ALDOA, P02647), growth/differentiation factor 8 (MSTN, O14793), α2-HS-glycoprotein (AHSG, P02765), coagulation factor XIII A1 (F13A1, P00488), complement C3 (C3, P01024), and tetranectin (CLEC3B, P05452) (Figure 4B).Moreover, 70% of them were found to be glycoproteins (KW-0325) (14/20) and 65% of them were modified by the formation of a bond between the thiol groups of two peptidyl-cysteine residues (KW-1015) (13/20) (Figure 4C).

The protein expression profile was not the only factor that distinguished the analyzed groups.The degree of protein modification by lipid peroxidation also varied considerably among the different study groups.A significant increase in the level of proteins modified by reactive aldehydes was observed (Figure 5).In plasma from COVID-19 patients, 4-HNEprotein adducts increased more than two-fold and more than three-fold in plasma from patients who did not survive because of infection.

The dominant 4-HNE-modified proteins were serpin A12 (Q8IW75), inhibitor of NF-κB kinase (O15111), macrophage migration inhibitory factor (P14174), and kinases: adenosine kinase (P55263), MAPK 10 (P53779), and non-specific serine/threonine protein kinase (Q9Y6B5).All proteins modified by 4-HNE are shown in Figure 6.For protein adducts with MDA, the highest level was observed in COVID-19 surviving patients, which was seven times higher than the Ctr group.In the CovDeath group, the level of MDA-protein adducts was about 2.5 times higher than the Ctr group.This result was related to the increased levels of MDA adducts with kinases (cAMP-dependent protein kinase (P22612), ribosephosphate pyrophosphokinase 3 (P21108), and MAP kinase-activated protein kinase 2 (P49137)), which were mainly observed in COVID-19 surviving patients.Histones H2A (Q8IUE6) and H3 (Q5TEC6) were also altered to an increased extent (Figure 7).Interestingly, although a 2.5-fold increase in 15d-PGJ2 protein adducts was observed in the COVID-19 surviving group, a decrease of up to 60% was observed in the COVID-19 deceased group (Figure 5).This was particularly true for proteins such as p53 (S5LQU3), collapsin response mediator protein 1 (CRMP1, X5DNI1), signal transducer and activator of transcription 3 (STAT 3, P40763), and apoptosis inhibitor 5 (API5, Q9BZZ5) (Figure 8).


## Discussion

Since the COVID-19 pandemic has dominated the modern world, the researchers are putting joint efforts to understand the pathogenesis of this disease.The fact that COVID-19 patients can develop pneumonia, severe symptoms of acute respiratory distress syndrome, and multiple-organ failure connected with post-acute conditions, further encourages an in-depth analysis of this disease [14,15].The results obtained in this study show that COVID-19 development strongly alters the expression of proteins involved in the regulation of exocytosis and platelet degranulation, simultaneously increasing 4-HNE/MDA-protein adducts, thus affecting the structure of kinases and proinflammatory proteins.In addition, in the plasma of COVID-19 patients who survived and those who died, different changes (increase and decrease, respectively) were found for the protein adducts with 15d-PGJ2.It has been described previously that infection with COVID-19 triggers a systemic immune response in the human body, manifested by increased expression of proinflammatory cytokines, including interleukins (IL-1, 6, 8) and tumor necrosis factor α (TNFα) [16][17][18][19].However, it remains unclear whether some of the proinflammatory proteins are specific to COVID-19 and how signaling pathways are involved in the progression of infection [20,21].In this study, the whole proteome of plasma from COVID-19 patients was analyzed, with particular attention to metabolic pathways associated with different responses between survivors and deceased patients, as well as differences in the interaction of protein and lipid peroxidation products in the study groups.


### COVID-19-Induced Changes in Protein Profile

The results of this study clearly show that half of the twenty major proteins whose expression is altered in the plasma of COVID-19 patients are involved in the regulation of exocytosis and platelet degranulation.These observations are in agreement with data from the literature, and both processes enable the spread of the virus in the body [22,23].After replication in lysosomes, the virus requires an extracellular egress and simultaneously blocks autophagy [24].This is the mechanism by which SARS-CoV-2 interacts with host proteins to promote its spread.Moreover, the previously described changes in exocytosis, which regulates protein expression, are closely associated with the severity of COVID-19 [23], which is also observed between survivors and deceased patients in our study.At the same time, the activation of platelets stimulates the host immune system to fight SARS-CoV-2 infection [25].Platelets contain multiple receptors that interact with specific ligands, including proteins of pathogens [26].These are receptors such as Tolllike receptors (TLRs) or the family of C-type lectin receptors, but also glycoproteins that allow pathogens to interact with platelets [27].The aforementioned receptors are considered important components in the activation of the innate immune response against SARS-CoV-2 [28].Moreover, platelet activation or degranulation during COVID-19 may promote vascular and coronary thrombosis [28].It was found that platelets in severely ill COVID-19 patients were much more hyperactivated than those in mildly ill patients [29,30].Moreover, the results of this study indicated that in the patients with the worst prognosis, most of the proteins responsible for platelet degranulation are silenced, suggesting that the body capitulates and stops making an effort to eliminate the pathogens.However, this observation was not true for transgelin-2 (TAGLN2, P37802), whose levels were highest in the CovDeath group.The biological roles of transgelin-2 are not fully understood, but it is known that it is involved in apoptosis inhibition, induction of cell proliferation, and promotion of cancer metastasis, corresponding to stimulation of the expression of ABC transporters and PI3K/Akt kinase activity [31].In the lung, transgelin-2 has been found to be involved in reducing pulmonary resistance in asthma by airway muscle cell relaxation [32].Thus, it is possible that transgelin-2 has a significant role in lung function in COVID-19 patients.However, transgelin-2 cannot be a COVID-19 prognostic biomarker because its high levels are also associated with other diseases such as asthma, type 2 diabetes, or cancer [33][34][35].

The function and activity of individual proteins are influenced not only by their concentration but also by all post-translational modifications.It is well known that during the development of the disease, various pathological conditions, including oxidative stress and inflammation, affect the protein structure and function of signaling molecules and enzymatic proteins by modifying their active centers [36,37].The obtained results show that within the top twenty altered proteins, more than half are modified by the formation of a bond between the thiol groups of two peptidyl-cysteine residues.This is certainly related to the oxidative stress found in COVID-19 patients [4,38].The formation of double bonds between cysteine residues within or between proteins is due to the decrease in the amount of reduced thiol group donors, including glutathione and thioredoxin, but also to increased activity of enzymes (glutathione peroxidase and thioredoxin reductase) that also use them as cofactors [4].In addition, the proportion of proteins forming double bonds is lower in the CovDeath group compared to the survivors, which also corresponds to smaller changes in antioxidant system levels/activities compared to the control group [4].This may suggest that the degree of protein modification may be a predictor of disease progression.


### Protein Modifications by Reactive Aldehydes in COVID-19 Patients

Reactive aldehydes such as 4-HNE or MDA are the main known products of lipid peroxidation with the described role of signaling molecules both in health and disease [39].The rate of increase of these molecules determines whether cells will survive by activating their antioxidant system or whether they induce mutation or apoptosis [39].In the plasma of COVID-19 patients, a strong increase in the activity of enzymes responsible for lipid metabolism, such as phospholipase A2, is one of the factors explaining the increase in lipid peroxidation products, including the aforementioned reactive aldehydes [5].As a result, both 4-HNE and MDA can directly interact with proteins, altering their structure and function.In general, such adducts support the antioxidant response through changes in transcriptional activity or stimulate proinflammatory signaling, although interactions with particular proteins can lead to specific effects [12].

The amount of the 4-HNE-modified proteins in the plasma of COVID-19 patients compared with the other protein modifications analyzed confirmed our previous findings suggesting that 4-HNE may be directly involved in the pathogenesis of COVID-19, especially with a lethal outcome [7,11,40].Namely, the 4-HNE-modified proteins were first detected in the kidneys of the deceased COVID-19 patient [38].Further study involving severely ill COVID-19 patients revealed higher levels of the 4-HNE-modified proteins upon admission to the hospital in the plasma of patients who died after five days than were the levels of the 4-HNE-protein adducts in plasma of survivors [7].Moreover, dynamic changes of the 4-HNE-protein adducts in plasma indicating a kind of systemic oxidative stress response to the SARS-CoV-2 infection were observed upon admission only for patients who recovered, being absent in the plasma of patients who eventually passed away [7].Preliminary immunohistochemical findings conducted for the 4-HNE-protein adducts in the lungs of one deceased patient, applying the same monoclonal antibody specific to the 4-HNE-histidine adducts used for the ELISA detection of the 4-HNE-protein adducts in plasma, suggested that 4-HNE might be associated with the fatal outcome.That was later confirmed in another study by the post-mortem immunohistochemical evaluation of the vital organs of the additional nine patients [11].

It is likely that modification of proteins by 4-HNE might occur during the process of death, causing their abundance in the organs of the dead patients, but in spite of that, their distribution was not random and diffuse, as inflammatory cells were immunohistochemically negative, while blood vessels and their content, as well as edematous liquid, were strongly positive for 4-HNE [11,38].It is also certain that 4-HNE-protein adducts were increased at the time of admission to the hospital in the plasma of patients who eventually died when there was no difference in any relevant parameter between these patients and survivors [7].Therefore, we assume that modification of proteins by 4-HNE in patients with COVID-19 might be relevant for the pathogenesis of the disease and its final outcome, at least in severely ill patients.

In the current study of plasma samples collected upon admission to the hospital, when there was no difference between the status of survivors and patients who eventually passed away revealed that the 4-HNE modifications mainly affected proteins involved in immune response, including serpin A12 (Q8IW75), inhibitor of NF-κB kinase (O15111), and macrophage migration inhibitory factor (MIF, P14174).Serpins are potent serine protease inhibitors with potential anti-inflammatory properties [41].It is known that their oxidative modifications lead to their inactivation [42]; therefore, the formation of adducts with 4-HNE should also suppress their anti-inflammatory effect.At the same time, 4-HNE activates the NF-κB-dependent proinflammatory signaling pathway [43], also by interacting with the inhibitor of NF-κB kinase as well as with other kinases important for proinflammatory signaling.Unfortunately, there are no data in the literature on how such modification affects the functioning of MIF.MIF is a cytokine whose activity arrests immune cell movement [44], thereby blocking the inflammatory response.Moreover, the isoform of MIF has been described as an important determinant of COVID-19 symptomatic infection and severity [42,43].Therefore, further attention should be paid to this protein and the modification of its structure depending on the isoform, which would improve the prognosis of disease progression.

To date, little is known about the activity of specific proteins modified by MDA.In the plasma of the COVID-19 patients studied, a large group of modified MDA proteins are kinases (MAP kinase-activated protein kinase 2 (MAPKAPK2, P49137), ribosephosphate pyrophosphokinase 3 (PRPS1L1, P21108), and cAMP-dependent protein kinase (PKA, P22612)).Some kinases are known to be activated when modified with MDA [12].For example, in SARS-CoV-infected Vero E6 cells, MAPKAPK2 is activated resulting in p38/MAPK pathway stimulation leading to the upregulation of Bax expression, which in turn triggers apoptosis and severely damages lung tissue [45,46].Therefore, additional MDA-induced kinase upregulation further worsens the health of patients.Moreover, transcriptomic analysis of SARS-CoV-2 infection highlighted a correlation between the host-associated signaling pathway and the virus based on the PKA-dependent pathway.In cells in which this pathway was knocked down using small interfering RNAs, replication of SARS-CoV-2 genetic material was inhibited by approximately 50% [47].However, in COVID-19 patients, histones H2A (Q8IUE6) and H3 (Q5TEC6) were also modified by MDA to an increased extent.Extracellular (plasma) histones are nuclear proteins that can be released into the extracellular space during apoptosis, necrosis, or netosis formation.Under oxidative stress associated with inflammation and disease development, histones stimulate the release of prototypical sepsis cytokines and decreased cell integrity [48].Therefore, it has been described that plasma histones are significantly increased in COVID-19 patients [49].Moreover, the formation of MDA adducts with histones promotes their activity, e.g., by increasing their binding (DNA or protein) properties [50].Combining these facts with the observed increase in MDA-histone adducts in COVID-19 patients, which was particularly high in those who did not survive the infection, such changes could be an additional prognostic factor in the course of disease development.


### Formation of Prostaglandin-Protein Adducts in Plasma of COVID-19 Patients

Prostaglandins (PGs) are the products of enzymatic PUFA metabolism with very broad signaling functions, including the regulation of inflammation and pain sensation [51].PGJ2, among other PGs, is characterized by a cyclopentenone ring with reactive α,β-unsaturated carbonyl groups that can form adducts with cysteine residues in proteins [52].Some of the PGJ2, including 15d-PGJ2, have anti-inflammatory and antiviral effects, which is being attempted to be used in COVID-19 therapies [53].This may be due to the fact that a significant increase in 15d-PGJ2 levels is observed in the plasma of patients with mild COVID-19, whereas no such observation has been made in severe COVID-19 patients who consequently do not survive SARS-CoV-2 infection [5].The results obtained in this study confirm this, as the amount of 15d-PGJ2-modified proteins is highest in COVID-19 survivors, whereas in deceased COVID-19 patients, the amount of 15d-PGJ2-modified proteins is decreased.This has been demonstrated for p53 and STAT3, which are both factors involved in proliferation, apoptosis, but also inflammation and carcinogenesis [54,55].Moreover, both proteins are inactivated after modification with 15d-PGJ2 [56,57].Earlier studies have found STAT3 to be increased in COVID-19 patients [58]; therefore, its modification by 15d-PGJ2 counteracts this change and possibly could lead to restoring homeostasis in COVID-19 survivors.However, since the interaction between SARS-CoV-2 and p53 is not clear [59], further research is needed to understand the biological relevance of 15d-PGJ2-p53 adducts in plasma of COVID-19 patients.


### Limitations

The results obtained in this study are subject to certain limitations.Due to the use of non-target proteomic analysis, the number of studied samples was limited to a small representative group of patients, who could be at different stages of the disease, which could not be determined by classical diagnostic methods.Moreover, it is not possible to distinguish if proteome changes, notably protein modifications by the lipid peroxidation products, observed in plasma of decreased patients were caused by SARS-CoV-2 infection, the reaction of the organism to it in the period of at least five days before the death, or by the dying process.Additionally, in the case of some unstable lipid peroxidation productprotein adducts, the sample preparation used for proteomic analysis may mean that some of the complexes were not detected in this analysis.


## Materials and Methods


### Samples Collection

Plasma samples were collected upon admission to the hospital from a group of 10 COVID-19 survivors (6 females and 4 males) with a mean age of 62 (54-70) years and from 10 later-deceased patients with COVID-19 (6 females and 4 males) with a mean age of 68 (62-77) years who were treated during the period December 2020 to February 2022 at the Clinical Hospital Dubrava in Zagreb, which served as a national COVID-19 center.Patients were randomly selected from a group of 66 (COVID-19 survivors) and 22 (COVID-19 deceased) patients, whose results of classical analysis have previously been published [4,5].The number of patients in each group was selected as 10 to provide convergent apparatus conditions to all injections and to ensure high-statistical-power analysis [60].The control group consisted of ten healthy donors (6 females and 4 males) with a mean age of 43  years.The study was conducted in accordance with the Declaration of Helsinki, according to the approval of the ethics committee 2020-1012-13 of the Dubrava Clinical Hospital in Zagreb and written informed consent was obtained from all participants.

Blood samples were collected in ethylenediaminetetraacetic acid (EDTA) tubes and centrifuged at 3000 g for 20 min to separate plasma.The antioxidant butylhydroxytoluene was added to the plasma samples, which were stored at −80 • C until analysis.


### Protein Separation and Digestion

Total protein concentration was measured by Bradford assay [61] and 30 µg of proteins was mixed with sample loading buffer (Laemmle buffer containing 5% 2-mercaptoethanol), heated at 95 • C for 7 min, and separated on 12% Tris-Glycine SDS-PAGE gels.Following electrophoresis, gels were fixed in 40% methanol and 10% acetic acid for 1 h and stained with Coomassie Brilliant Blue R-250 for 4 h.Complete lanes were excised from the gel and sliced into eight sections (Figure 2).Each band in the gel was reduced with 10 mM 1,4-dithiothreitol (DTT) and alkylated with 50 mM iodoacetamide (IAA).Samples were then in-gel digested overnight at 37 • C with trypsin (Promega, Madison, WI, USA) at a ratio of 1:50 (trypsin:proteins).Digestion was stopped by the addition of 10% formic acid (FA) in an amount to ensure a final concentration in the samples of 0.1% [62].The obtained peptide mixture was collected, dried under inert gas, and frozen until analysis.


### Proteomic Analysis and Protein Identification

Dried peptides were reconstituted in 5% acetonitrile (ACN) with 0.1% FA and separated using a high-performance liquid chromatography system (Ultimate 3000; Dionex, Idstein, Germany) on a 150 mm × 75 mm PepMap RSLC capillary analytical C18 column with 2 µm particle size (Dionex, LC Packings) at a constant flow rate of 0.300 µL/min.Solvents used for separation were solvent A (5% ACN with 0.1% (v/v) FA) and solvent B (90% ACN with 0.1% (v/v) FA), and the separation gradient was set to increase from 5% to 60% B over 55 min.The Q Exactive HF mass spectrometer with an electrospray ionization source (ESI) (Thermo Fisher Scientific, Bremen, Germany) was used to analyze the eluted peptides.The conditions for the analysis for peptide identification have been described in detail previously [63].

Raw data were searched against the UniProtKB-SwissProt database (taxonomy: Homo sapiens, release February 2023) using Proteome Discoverer 2.0 (Thermo Fisher Scientific, Seattle, WA, USA).Peptide mass tolerance was set to 10 ppm, MS/MS mass tolerance was set to 0.02 Da, and up to two missed cleavages were allowed.Dynamic modifications were set as cysteine carbamidomethylation/carboxymethylation, methionine oxidation, and lipid peroxidation products (4-HNE, MDA, 15d-PGJ2) with cysteine/lysine/histidine adducts [64,65].Label-free quantification of proteins was based on the signal intensities of precursor ions, and the amount of lipid peroxidation product-protein adducts was estimated based on the peak intensity of peptides modified by 4-HNE, MDA, or 15d-PGJ2.Only proteins with at least three identified peptides longer than six amino acid residues and at least two unique peptides were selected for further analysis.


### Statistical Analysis

The label-free quantification results of each protein using the open-source Metabo-Analyst 5.0 software (http://www.metaboanalyst.ca(accessed on 1 April 2023)) [66] were log-transformed, auto-scaled (mean-centered and divided by the standard deviation of each variable), and normalized by the median of the protein intensities obtained for each sample, ensuring normal distribution of the samples [67].All missing values (a total of 12 (<0%) missing values were detected) were replaced by 1/5 of the positive minimum values of the corresponding variables.To improve the results, the data were statistically filtered for variables that were nearly constant throughout the experimental conditions (variables detected using the interquartile range (IQR) at 40%) [68].MetaboAnalyst 5.0 was also used for biostatistical analysis, including analysis of variance (ANOVA with false discovery rate (FDR) <5%), principal-component analysis (PCA), a heatmap, and boxplots.Protein functions were determined using the STRING.11.5 database [69].


## Conclusions

Although the pandemic of aggressive COVID-19 seems to be under control, SARS-CoV-2 virus infection continues to be a significant medical problem affecting people around the world.Understanding the different metabolic responses of patients to the presence of this virus could help develop targeted therapies based on targeted immune response support.This study for the first time demonstrates the formation and biological efficacy of adducts of proteins-products of lipid metabolism, such as reactive aldehydes 4-HNE and MDA, as well as 15d-PGJ2 in plasma from surviving and deceased COVID-19 patients, significantly distinguishing them, which may be a predictor of the course of SARS-CoV-2 infection.


## Supplementary Materials:

The following supporting information can be downloaded at: https:// www.mdpi.com/article/10.3390/ijms241814109/s1.


## Funding:

The biochemical analysis performed on cells was supported by the Ministry of Science and Higher Education (Poland) as part of the scientific activity of the Medical University of Bialystok.


## Institutional Review Board Statement:

The study was conducted in accordance with the Declaration of Helsinki, according to the approval of the ethics committee 2020-1012-13 of the Dubrava Clinical Hospital in Zagreb.

Informed Consent Statement: Informed consent was obtained from all subjects involved in the study.Written informed consent has been obtained from the patient(s) to publish this paper.

Data Availability Statement: Data are available via ProteomeXchange with the identifier PXD044040.

## Figure 1 .
1
Figure 1.The number of published studies on changes in patients' bodies caused by SARSinfection over the past 10 years, with emphasis on proteomics studies and articles containin yses of protein modifications.Data obtained from Web of Science Core Collection (accessed June 2023) [3].


## Figure 1 .
1
Figure 1.The number of published studies on changes in patients' bodies caused by SARS-CoV-2 infection over the past 10 years, with emphasis on proteomics studies and articles containing analyses of protein modifications.Data obtained from Web of Science Core Collection (accessed on 26 June 2023) [3].


## Figure 2 .
2
Figure 2. Electrophoretic separation and staining with Coomassie Brilliant Blue R-250 of plasma proteins from healthy donors (Ctr, n = 10) and COVID-19 patients (COVID-19-surviving patients n = 10, and CovDeath-deceased patients, n = 10).The lines of the black frames indicate how the gels were cut before protein digestion.


## Figure 2 .
2
Figure 2. Electrophoretic separation and staining with Coomassie Brilliant Blue R-250 of plasma proteins from healthy donors (Ctr, n = 10) and COVID-19 patients (COVID-19-surviving patients, n = 10, and CovDeath-deceased patients, n = 10).The lines of the black frames indicate how the gels were cut before protein digestion.


## Figure 3 .
3
Figure 3.The results of statistical analysis of protein level quantified in plasma from healthy don (Ctr, n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID deceased patients, n = 10).(A) One-way ANOVA showing p distribution within all data and results of principal-component analysis (PCA).


## Figure 3 .
3
Figure 3.The results of statistical analysis of protein level quantified in plasma from healthy donors (Ctr, n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID-19 deceased patients, n = 10).(A) One-way ANOVA showing p distribution within all data and (B) results of principal-component analysis (PCA).


## Figure 4 .
4
Figure 4. (A) Heatmap and clustering for the top 20 altered proteins from the plasma from healthy donors (Ctr, n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID-19 deceased patients, n = 10).Protein expression levels (log transformed) were scaled to the row mean.The color key relates the heatmap colors to the standard score (z-score), which is the deviation from the row mean in units of standard deviation above or below the mean.(B) The main biological pathways in which proteins selected in the heatmap participate and (C) structural modifications of these proteins.Data analyzed using STRING.11.5.Keywords used (KW): (KW-0325) protein containing one or more covalently linked carbohydrates of various types, i.e., from monosaccharides to branched polysaccharides, including glycosylphosphatidylinositol-inositol (GPI) and glycosaminoglycans (GAG); (KW-1015) protein which is modified by the formation of a bond between the thiol groups of two peptidyl-cysteine residues.


## Figure 4 .
4
Figure 4. (A) Heatmap and clustering for the top 20 altered proteins from the plasma from healthy donors (Ctr, n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID-19 deceased patients, n = 10).Protein expression levels (log transformed) were scaled to the row mean.The color key relates the heatmap colors to the standard score (z-score), which is the deviation from the row mean in units of standard deviation above or below the mean.(B) The main biological pathways in which proteins selected in the heatmap participate and (C) structural modifications of these proteins.Data analyzed using STRING.11.5.Keywords used (KW): (KW-0325) protein containing one or more covalently linked carbohydrates of various types, i.e., from monosaccharides to branched polysaccharides, including glycosylphosphatidylinositol-inositol (GPI) and glycosaminoglycans (GAG); (KW-1015) protein which is modified by the formation of a bond between the thiol groups of two peptidyl-cysteine residues.


## Figure 5 .
5
Figure 5.Total level of protein modifications by lipid peroxidation products 4-hydroxynonenal (4-HNE), malondialdehyde (MDA), and 15-deoxy-12,14-prostaglandin J2 (15d-PGJ2) in plasma from healthy donors (Ctr, n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID-19 deceased patients, n = 10).The levels of lipid peroxidation product-protein adducts were estimated based on the peak intensity of modified peptides.Mean values ± SD are presented.a statistically significant differences vs. Ctr group, p < 0.05; b statistically significant differences vs. COVID-19 group, p < 0.05.


## Figure 5 . 16 Figure 5 .
5165
Figure 5.Total level of protein modifications by lipid peroxidation products 4-hydroxynonenal (4-HNE), malondialdehyde (MDA), and 15-deoxy-12,14-prostaglandin J2 (15d-PGJ2) in plasma from healthy donors (Ctr, n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID-19 deceased patients, n = 10).The levels of lipid peroxidation product-protein adducts were estimated based on the peak intensity of modified peptides.Mean values ± SD are presented.a statistically significant differences vs. Ctr group, p < 0.05; b statistically significant differences vs. COVID-19 group, p < 0.05.


## Figure 6 .
6
Figure 6.The level of 4-hydroxynonenal (4-HNE)-protein adducts in plasma from healthy donors (Ctr, n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID-19 deceased patients, n = 10).The levels of lipid peroxidation product-protein adducts were estimated based on the peak intensity of modified peptides.a statistically significant differences vs. Ctr group, p < 0.05; b statistically significant differences vs. COVID-19 group, p < 0.05.


## Figure 6 .
6
Figure 6.The level of 4-hydroxynonenal (4-HNE)-protein adducts in plasma from healthy donors (Ctr, n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID-19 deceased patients, n = 10).The levels of lipid peroxidation product-protein adducts were estimated based on the peak intensity of modified peptides.a statistically significant differences vs. Ctr group, p < 0.05; b statistically significant differences vs. COVID-19 group, p < 0.05.


## Figure 7 .
7
Figure 7.The level of malondialdehyde (MDA)-protein adducts in plasma from healthy donors (Ctr, n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID-19 deceased patients, n = 10).The levels of lipid peroxidation product-protein adducts were estimated based on the peak intensity of modified peptides.a statistically significant differences vs. Ctr group, p < 0.05; b statistically significant differences vs. COVID-19 group, p < 0.05.


## Figure 8 .
8
Figure 8.The level of 15-deoxy-12,14-prostaglandin J2 (15d-PGJ2)-protein adducts in plasma from healthy donors (Ctr, n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID-19 deceased patients, n = 10).The levels of lipid peroxidation product-protein adducts were estimated based on the peak intensity of modified peptides.a statistically significant differences vs. Ctr group, p < 0.05; b statistically significant differences vs. COVID-19 group, p < 0.05.


## Figure 7 .
7
Figure 7.The level of malondialdehyde (MDA)-protein adducts in plasma from healthy donors n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID-19 deceased patients, n = 10).The levels of lipid peroxidation product-protein adducts were estimated based on the peak intensity of modified peptides.a statistically significant differences vs. Ctr group, p < 0.05; b statistically significant differences vs. COVID-19 group, p < 0.05.


## Figure 6 .
6
Figure 6.The level of 4-hydroxynonenal (4-HNE)-protein adducts in plasma from healthy donors (Ctr, n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID-19 deceased patients, n = 10).The levels of lipid peroxidation product-protein adducts were estimated based on the peak intensity of modified peptides.a statistically significant differences vs. Ctr group, p < 0.05; b statistically significant differences vs. COVID-19 group, p < 0.05.


## Figure 7 .
7
Figure 7.The level of malondialdehyde (MDA)-protein adducts in plasma from healthy donors (Ctr, n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID-19 deceased patients, n = 10).The levels of lipid peroxidation product-protein adducts were estimated based on the peak intensity of modified peptides.a statistically significant differences vs. Ctr group, p < 0.05; b statistically significant differences vs. COVID-19 group, p < 0.05.


## Figure 8 .
8
Figure 8.The level of 15-deoxy-12,14-prostaglandin J2 (15d-PGJ2)-protein adducts in plasma from healthy donors (Ctr, n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID-19 deceased patients, n = 10).The levels of lipid peroxidation product-protein adducts were estimated based on the peak intensity of modified peptides.a statistically significant differences vs. Ctr group, p < 0.05; b statistically significant differences vs. COVID-19 group, p < 0.05.


## Figure 8 .
8
Figure 8.The level of 15-deoxy-12,14-prostaglandin J2 (15d-PGJ2)-protein adducts in plasma from healthy donors (Ctr, n = 10), COVID-19 patients (COVID-19-surviving patients, n = 10), and CovDeath (COVID-19 deceased patients, n = 10).The levels of lipid peroxidation product-protein adducts were estimated based on the peak intensity of modified peptides.a statistically significant differences vs. Ctr group, p < 0.05; b statistically significant differences vs. COVID-19 group, p < 0.05.


## Author

Contributions: Conceptualization, N.Z.and E.S.; data curation, A.G. and B.O.; formal analysis, M.J.; investigation, A.G. and S.B.S.; methodology, A.G.; resources, B.O. and S.B.S.; software, A.G.; supervision, N.Z.and E.S.; validation, M.J.; visualization, A.G.; writing-original draft, A.G. and M.J.; writing-review and editing, N.Z.and E.S.All authors have read and agreed to the published version of the manuscript.




).
t. J. Mol. Sci. 2023, 24, x FOR PEER REVIEW3 of 16
Int. J. Mol. Sci. 2023, 24, 14109. https://doi.org/10.3390/ijms241814109 https://www.mdpi.com/journal/ijms
Acknowledgments:The authors are thankful to the COST Action EpiLipidNET, CA19105-Pan-European Network in Lipidomics and EpiLipidomics.Conflicts of Interest:The authors declare no conflict of interest.The funders had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript; or in the decision to publish the results.
Who, Coronavirus, COVID-19) Dashboard. Available online. 26 June 2023

Evolution of CT Manifestations in a Patient Recovered from 2019 Novel Coronavirus (2019-nCoV) Pneumonia in Wuhan. 10.1148/radiol.202020026926 June 2023

N Žarković, A Jastrz Ąb, I Jarocka-Karpowicz, B Orehovec, B Baršić, M Tarle, M Kmet, I Lukšić, W Łuczaj, E Skrzydlewska, 10.3390/molecules27165323The Impact of Severe COVID-19 on Plasma Antioxidants. Molecules. 202227

Diversified Effects of COVID-19 as a Consequence of the Differential Metabolism of Phospholipids and Lipid Peroxidation Evaluated in the Plasma of Survivors and Deceased Patients upon Admission to the Hospital. N Žarković, W Łuczaj, I Jarocka-Karpowicz, B Orehovec, B Baršić, M Tarle, M Kmet, I Lukšić, M Biernacki, E Skrzydlewska, 10.3390/ijms231911810Int. J. Mol. Sci. 232022. 11810

Lipidomics Revealed Plasma Phospholipid Profile Differences between Deceased and Recovered COVID-19 Patients. N Žarković, B Orehovec, B Baršić, M Tarle, M Kmet, I Lukšić, F Tatzber, W Wonisch, E Skrzydlewska, W Łuczaj, 10.3390/biom12101488Biomolecules. 122022. 1488

Preliminary Findings on the Association of the Lipid Peroxidation Product 4-Hydroxynonenal with the Lethal Outcome of Aggressive COVID-19. N Žarković, B Orehovec, L Milković, B Baršić, F Tatzber, W Wonisch, M Tarle, M Kmet, A Mataić, A Jakovčević, 10.3390/antiox10091341Antioxidants. 102021. 1341

. A Negre-Salvayre, N Auge, V Ayala, H Basaga, J Boada, R Brenke, S Chapple, G Cohen, J Feher, T Grune, 10.3109/10715762.2010.498478Pathological Aspects of Lipid Peroxidation. Free Radic. Res. 442010

Regulation of NF-kB-Induced Inflammatory Signaling by Lipid Peroxidation-Derived Aldehydes. U C S Yadav, K V Ramana, 10.1155/2013/690545Oxidative Med. Cell. Longev. 2013, 2013, e690545

Plasma Proteomics Unveil Novel Immune Signatures and Biomarkers upon SARS-CoV-2 Infection. V Urbiola-Salvador, S Lima De Souza, P Grešner, T Qureshi, Z Chen, 10.3390/ijms24076276Int. J. Mol. Sci. 242023

Post-Mortem Findings of Inflammatory Cells and the Association of 4-Hydroxynonenal with Systemic Vascular and Oxidative Stress in Lethal COVID-19. N Zarkovic, A Jakovcevic, A Mataic, M Jaganjac, T Vukovic, G Waeg, K Zarkovic, 10.3390/cells11030444202211444

Biological Effect of Protein Modifications by Lipid Peroxidation Products. A Gęgotek, E Skrzydlewska, 10.1016/j.chemphyslip.2019.03.011Chem. Phys. Lipids. 2212019

Malondialdehyde Acetaldehyde Adduction of Surfactant Protein D Attenuates SARS-CoV-2 Spike Protein Binding and Virus Neutralization. A Muralidharan, C Bauer, D M Katafiasz, D Pham, O O Oyewole, M J Morwitzer, E Roy, K L Bailey, S P Reid, T A Wyatt, 10.1111/acer.14974Alcohol. 472023

COVID-19: Immunopathogenesis and Immunotherapeutics. L Yang, S Liu, J Liu, Z Zhang, X Wan, B Huang, Y Chen, Y Zhang, 10.1038/s41392-020-00243-2Signal Transduct. Target. Ther. 52020

Decreased Gas6 and sAxl Plasma Levels Are Associated with Hair Loss in COVID-19 Survivors. D Apostolo, D D'onghia, S Tonello, R Minisini, A Baricich, C Gramaglia, F Patrucco, P Zeppegno, A Acquaviva, P E Balbo, 10.3390/ijms24076257Int. J. Mol. Sci. 242023

COVID-19 Pathophysiology: A Review. K Yuki, M Fujiogi, S Koutsogiannaki, 10.1016/j.clim.2020.108427Clin. Immunol. 2152020. 108427

SARS-CoV-2 Spike Proteins and Cell-Cell Communication Induce P-Selectin and Markers of Endothelial Injury, NETosis, and Inflammation in Human Lung Microvascular Endothelial Cells and Neutrophils: Implications for the Pathogenesis of COVID-19 Coagulopathy. B Bhargavan, G D Kanmogne, 10.3390/ijms241612585Int. J. Mol. Sci. 242023. 12585

Predictive Role of Cytokine and Adipokine Panel in Hospitalized COVID-19 Patients: Evaluation of Disease Severity, Survival and Lung Sequelae. L Bergantini, M ; D'alessandro, S Gangi, F Bianchi, P Cameli, B Perea, M Meocci, G Fabbri, S Marrucci, M Ederbali, 10.3390/ijms241612994Int. J. Mol. Sci. 242023. 12994

Cytokine Profiles as Potential Prognostic and Therapeutic Markers in SARS-CoV-2-Induced ARDS. F Salton, P Confalonieri, G Campisciano, R Cifaldi, C Rizzardi, D Generali, R Pozzan, S Tavano, C Bozzi, G Lapadula, 10.3390/jcm11112951J. Clin. Med. 2022, 11, 2951

Cellular and Plasma Proteomic Determinants of COVID-19 and Non-COVID-19 Pulmonary Diseases Relative to Healthy Aging. L Arthur, E Esaulova, D A Mogilenko, P Tsurinov, S Burdess, A Laha, R Presti, B Goetz, M A Watson, C W Goss, 10.1038/s43587-021-00067-xNat. Aging. 12021

Plasma Proteomics of SARS-CoV-2 Infection and Severity Reveals Impact on Alzheimer and Coronary Disease Pathways. L Wang, D Western, J Timsina, C Repaci, W.-M Song, J Norton, P Kohlfeld, J Budde, S Climer, O H Butt, 2022

Plasma Proteomic and Metabolomic Characterization of COVID-19 Survivors 6 Months after Discharge. H Li, X Li, Q Wu, X Wang, Z Qin, Y Wang, Y He, Q Wu, L Li, H Chen, 10.1038/s41419-022-04674-3Cell Death Dis. 132352022

. T Shu, W Ning, D Wu, J Xu, Q Han, M Huang, X Zou, Q Yang, Y Yuan, Y Bie, 10.1016/j.immuni.2020.10.008COVID-19Plasma Proteomics Identify Biomarkers and Pathogenesis. 532020Immunity. e5. [CrossRef

ORF3a of SARS-CoV-2 Promotes Lysosomal Exocytosis-Mediated Viral Egress. D Chen, Q Zheng, L Sun, M Ji, Y Li, H Deng, H Zhang, 10.1016/j.devcel.2021.10.006Dev. Cell. 562021e5. [CrossRef

Platelets and SARS-CoV-2 During COVID-19: Immunity, Thrombosis, and Beyond. A Sciaudone, H Corkrey, F Humphries, M Koupenova, 10.1161/CIRCRESAHA.122.321930Circ. Res. 1322023

The Interaction of Bacterial Pathogens with Platelets. J R Fitzgerald, T J Foster, D Cox, 10.1038/nrmicro1425Nat. Rev. Microbiol. 42006

Platelets and the Innate Immune System: Mechanisms of Bacterial-induced Platelet Activation. D Cox, S W Kerrigan, S P Watson, 10.1111/j.1538-7836.2011.04264.xJ. Thromb. Haemost. 92011

Thrombosis in COVID-19 Infection: Role of Platelet Activation-Mediated Immunity. M B Fard, S B Fard, S Ramazi, A Atashi, Z Eslamifar, 10.1186/s12959-021-00311-9Thromb. J. 192021

Platelets Can Associate With SARS-CoV-2 RNA and Are Hyperactivated in COVID-19. Y Zaid, F Puhm, I Allaeys, A Naya, M Oudghiri, L Khalki, Y Limami, N Zaid, K Sadki, R Ben El Haj, 10.1161/CIRCRESAHA.120.317703Circ. Res. 1272020

The COVID Complex: A Review of Platelet Activation and Immune Complexes in COVID-19. S D Jevtic, I Nazy, 10.3389/fimmu.2022.807934Front. Immunol. 132022

Transgelin 2 Promotes Paclitaxel Resistance, Migration, and Invasion of Breast Cancer by Directly Interacting with PTEN and Activating PI3K/Akt/GSK-3β Pathway. L Liu, T Meng, X Zheng, Y Liu, R Hao, Y Yan, S Chen, H You, J Xing, Y Dong, 10.1158/1535-7163.MCT-19-0261Mol. Cancer Ther. 182019

Transgelin-2 as a Therapeutic Target for Asthmatic Pulmonary Resistance. L.-M Yin, Y.-D Xu, L.-L Peng, T.-T Duan, J.-Y Liu, Z Xu, W.-Q Wang, N Guan, X.-J Han, H.-Y Li, 10.1126/scitranslmed.aam8604Sci. Transl. Med. 102018eaam8604. [CrossRef

Mapping of Type 2 Diabetes Proteins to COVID-19 Biomarkers: A Proteomic Analysis. A S M Moin, A Al-Qaissi, T Sathyapalan, S L Atkin, A E Butler, 10.1016/j.metop.2020.100074Metab. Open. 92020. 100074

Transgelin-2 in Multiple Myeloma: A New Marker of Renal Impairment? Molecules. K Woziwodzka, J Małyszko, E Koc-Żórawska, M Żórawski, P Dumnicka, A Jurczyszyn, K Batko, P Mazur, M Banaszkiewicz, M Krzanowski, 20222779

Transgelin-2: Biochemical and Clinical Implications in Cancer and Asthma. L.-M Yin, L Ulloa, Y.-Q Yang, 10.1016/j.tibs.2019.05.004Trends Biochem. Sci. 442019

Metal Ions and Oxidative Protein Modification in Neurological Disease. L M Sayre, P I Moreira, M A Smith, G Perry, Ann. Ist. Super. Sanità. 412005

Oxidatively Modified Proteins in Aging and Disease. Free Radic. M F Beal, 10.1016/S0891-5849(02)00780-3Biol. Med. 322002

UPenn COVID Processing Unit. Protein Posttranslational Signatures Identified in COVID-19 Patient Plasma. P Vedula, H.-Y Tang, D W Speicher, A Kashina, 10.3389/fcell.2022.807149Front. Cell Dev. Biol. 102022

Lipid Peroxidation: Production, Metabolism, and Signaling Mechanisms of Malondialdehyde and 4-Hydroxy-2-Nonenal. A Ayala, M F Muñoz, S Argüelles, 10.1155/2014/360438Oxidative Med. Cell. Longev. 3604382014. 2014

Fatal Lymphocytic Cardiac Damage in Coronavirus Disease. W Jacobs, M Lammens, A Kerckhofs, E Voets, E Van San, S Van Coillie, C Peleman, M Mergeay, S Sirimsi, V Matheeussen, 10.1002/ehf2.12958COVID-19Autopsy Reveals a Ferroptosis Signature. 2019. 20207

The Serpin Solution; Targeting Thrombotic and Thrombolytic Serine Proteases in Inflammation. H A Chen, J Davids, D Zheng, M Bryant, I Bot, J C Van Berckel, T Biessen, E Pepine, C Ryman, K Progulski-Fox, A , 10.2174/1871529X11313020003Cardiovasc. Haematol. Disord.-Drug Targets. 132013

Conformational Properties of Serine Proteinase Inhibitors (Serpins) Confer Multiple Pathophysiological Roles. S Janciauskiene, 10.1016/S0925-4439(01)00025-4Biochim. Biophys. Acta (BBA)-Mol. Basis Dis. 15352001

Antioxidants and HNE in Redox Homeostasis. Free Radic. W Łuczaj, A Gęgotek, E Skrzydlewska, Biol. Med. 1112017

Macrophage Migration Inhibitory Factor (MIF): A Promising Biomarker. G Grieb, M Merk, J Bernhagen, R Bucala, 10.1358/dnp.2010.23.4.1453629Drug News Perspect. 232010

MIF Is a Common Genetic Determinant of COVID-19 Symptomatic Infection and Severity. J J Shin, W Fan, J Par-Young, M Piecychna, L Leng, K Israni-Winger, H Qing, J Gu, H Zhao, W L Schulz, 10.1093/qjmed/hcac234QJM Int. J. Med. 1162023

The Roles of Signaling Pathways in SARS-CoV-2 Infection; Lessons Learned from SARS-CoV and MERS-CoV. N Hemmat, Z Asadzadeh, N K Ahangar, H Alemohammad, B Najafzadeh, A Derakhshani, A Baghbanzadeh, H B Baghi, D Javadrashid, S Najafi, 10.1007/s00705-021-04958-7Arch. Virol. 1662021

SARS-CoV-2 Infection Activates CREB/CBP in Cellular Cyclic AMP-Dependent Pathways. Q Yang, J Tang, J Cao, F Liu, M Fu, B Xue, A Zhou, S Chen, J Liu, Y Zhou, 10.1002/jmv.28383J. Med. Virol. 952023. e28383

Impact of Plasma Histones in Human Sepsis and Their Contribution to Cellular Injury and Inflammation. M L Ekaney, G P Otto, M Sossdorf, C Sponholz, M Boehringer, W Loesche, D Rittirsch, A Wilharm, O Kurzai, M Bauer, 10.1186/s13054-014-0543-8Crit. Care. 185432014

The Role of Extracellular Histones in COVID-19. F De Vries, J Huckriede, K Wichapong, C Reutelingsperger, G A Nicolaes, 10.1111/joim.13585J. Intern. Med. 2932023

Oxidative Stress Signaling to Chromatin in Health and Disease. S Kreuz, W Fischle, 10.2217/epi-2016-0002Epigenomics. 82016

Prostaglandins and Inflammation. E Ricciotti, G A Fitzgerald, 10.1161/ATVBAHA.110.207449Arterioscler. Thromb. Vasc. Biol. 312011

Prostaglandin J2: A Potential Target for Halting Inflammation-Induced Neurodegeneration. M E Figueiredo-Pereira, C Corwin, J Babich, 10.1111/nyas.12987Ann. N. Y. Acad. Sci. 13632016

Immuno-Pathogenesis of nCOVID-19 and a Possible Host-Directed Therapy Including Anti-Inflammatory and Anti-Viral Prostaglandin (PG J2) for Effective Treatment and Reduction in the Death Toll. S Shahzad, M Willcox, Med. Hypotheses. 1100802020

Signal Transducer and Activator of Transcription-3, Inflammation, and Cancer. B B Aggarwal, A B Kunnumakkara, K B Harikumar, S R Gupta, S T Tharakan, C Koca, S Dey, B Sung, 10.1111/j.1749-6632.2009.04911.xAnn. N. Y. Acad. Sci. 11712009

The P53 Functional Circuit. S Jin, A J Levine, 10.1242/jcs.114.23.4139J. Cell Sci. 1142001

15-Deoxy-∆12,14-Prostaglandin J2 Binds and Inactivates STAT3 via Covalent Modification of Cysteine 259 in H-Ras-Transformed Human Breast Epithelial Cells. S.-J Kim, N.-C Cho, B Han, K Kim, Y.-I Hahn, K P Kim, Y G Suh, B Y Choi, H.-K Na, Y.-J Surh, 10.1002/1873-3468.14040FEBS Lett. 5952021

15-Deoxy-∆12,14-Prostaglandin J2 Stabilizes, but Functionally Inactivates P53 by Binding to the Cysteine 277 Residue. D.-H Kim, E.-H Kim, H.-K Na, Y Sun, Y.-J Surh, 10.1038/onc.2010.8Oncogene. 292010

Contribution of STAT3 to the Pathogenesis of COVID-19. A Jafarzadeh, M Nemati, S Jafarzadeh, 10.1016/j.micpath.2021.104836Microb. Pathog. 1542021. 104836

D Milani, L Caruso, E Zauli, A M Al Owaifeer, P Secchiero, G Zauli, D Gemmati, V Tisato, 10.3389/fphar.2022.871583P53/NF-kB Balance in SARS-CoV-2 Infection: From OMICs. 202213871583

The Role of Statistical Power Analysis in Quantitative Proteomics. Y Levin, 10.1002/pmic.201100033Proteomics. 112011

A Rapid and Sensitive Method for the Quantitation of Microgram Quantities of Protein Utilizing the Principle of Protein-Dye Binding. M M Bradford, 10.1016/0003-2697(76)90527-3Anal. Biochem. 721976

In-Solution Digestion of Proteins for Mass Spectrometry. K F Medzihradszky, 10.1016/S0076-6879(05)05003-2Methods Enzymol. 4052005

Proteomic Plasma Profile of Psoriatic Patients. A Gęgotek, P Domingues, A Wro Ński, P Wójcik, E Skrzydlewska, 10.1016/j.jpba.2018.03.068J. Pharm. Biomed. Anal. 1552018

Systematic Evaluation of Protein Reduction and Alkylation Reveals Massive Unspecific Side Effects by Iodine-Containing Reagents. T Müller, D Winter, 10.1074/mcp.M116.064048Mol. Cell. Proteom. 162017

Mass Spectrometry Detection of Protein Modification by Cross-Reaction with Lipid Peroxidation Products. React. Oxyg. Species Lipid Peroxidation Protein Oxid. M R Domingues, M Fedorova, P Domingues, 20153

Using MetaboAnalyst 5.0 for LC-HRMS Spectra Processing, Multi-Omics Integration and Covariate Adjustment of Global Metabolomics Data. Z Pang, G Zhou, J Ewald, L Chang, O Hacariz, N Basu, J Xia, 10.1038/s41596-022-00710-wNat. Protoc. 172022

A Systematic Evaluation of Normalization Methods in Quantitative Label-Free Proteomics. T Välikangas, T Suomi, L L Elo, 10.1093/bib/bbw095Brief. Bioinform. 192018

Filtering for Increased Power for Microarray Data Analysis. A J Hackstadt, A M Hess, 10.1186/1471-2105-10-11BMC Bioinform. 10112009

STRING V11: Protein-Protein Association Networks with Increased Coverage, Supporting Functional Discovery in Genome-Wide Experimental Datasets. D Szklarczyk, A L Gable, D Lyon, A Junge, S Wyder, J Huerta-Cepas, M Simonovic, N T Doncheva, J H Morris, P Bork, 10.1093/nar/gky1131Nucleic Acids Res. 472019

Disclaimer/Publisher's Note: The statements, opinions and data contained in all publications are solely those of the individual author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to people or property resulting from any ideas, methods. instructions or products referred to in the content