# Overview of worldwide diversity of Diaphorina citri Kuwayama mitochondrial cytochrome oxidase 1 haplotypes: two Old World lineages and a New World invasion

CorpusID: 795643
 
tags: #Medicine, #Environmental_Science, #Biology

URL: [https://www.semanticscholar.org/paper/0e704f6c5e54962766e732ec6872e7eff429b00d](https://www.semanticscholar.org/paper/0e704f6c5e54962766e732ec6872e7eff429b00d)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Overview of worldwide diversity of Diaphorina citri Kuwayama mitochondrial cytochrome oxidase 1 haplotypes: two Old World lineages and a New World invasion
2012

L M Boykin lboykin@mac.com 
Horticultural Research Laboratory
USDA, ARS
2001 South Rock Road34945Fort PierceFLUSA

P De Barro 
CSIRO Ecosystem Sciences
GPO Box 25834001BrisbaneQLDAustralia

D G Hall 
Horticultural Research Laboratory
USDA, ARS
2001 South Rock Road34945Fort PierceFLUSA

W B Hunter 
Horticultural Research Laboratory
USDA, ARS
2001 South Rock Road34945Fort PierceFLUSA

C L Mckenzie 
Horticultural Research Laboratory
USDA, ARS
2001 South Rock Road34945Fort PierceFLUSA

C A Powell 
Indian River Research and Education Center
University of Florida
34945Fort PierceFLUSA

R G Shatters Jr
Horticultural Research Laboratory
USDA, ARS
2001 South Rock Road34945Fort PierceFLUSA

Overview of worldwide diversity of Diaphorina citri Kuwayama mitochondrial cytochrome oxidase 1 haplotypes: two Old World lineages and a New World invasion

Bulletin of Entomological Research
102201210.1017/S0007485312000181*Author for correspondence Fax: 64 3 325 3864 †Current address: Bio-Protection Research Centre, PO Box 84, Lincoln University, Lincoln 7647, Christchurch, New Zealandgenetic structurestatistical parsimonyAsian citrus psyllidcitruscitrus greening diseaseHuanglongbingnovel COI primers
Relationships among worldwide collections of Diaphorina citri (Asian citrus psyllid) were analyzed using mitochondrial cytochrome oxidase I (mtCOI) haplotypes from novel primers. Sequences were produced from PCR amplicons of an 821 bp portion of the mtCOI gene using D. citri specific primers, derived from an existing EST library. An alignment was constructed using 612 bps of this fragment and consisted of 212 individuals from 52 collections representing 15 countries. There were a total of eight polymorphic sites that separated the sequences into eight different haplotypes (Dcit-1 through Dcit-8). Phylogenetic network analysis using the statistical parsimony software, TCS, suggests two major haplotype groups with preliminary geographic bias between southwestern Asia (SWA) and southeastern Asia (SEA). The recent (within the last 15 to 25 years) invasion into the New World originated from only the SWA group in the northern hemisphere (USA and Mexico) and from both the SEA and SWA groups in the southern hemisphere (Brazil). In only one case, Reunion Island, did haplotypes from both the SEA and SWA group appear in the same location. In Brazil, both groups were present, but in separate locations. The Dcit-1 SWA haplotype was the most frequently encountered, including *50% of the countries sampled and 87% of the total sequences obtained from India, Pakistan and Saudi Arabia. The second most frequently encountered haplotype, Dcit-2, the basis of the SEA group, represented *50% of the countries and contained most of the sequences from Southeast Asia and China. Interestingly, only the Caribbean collections (Puerto Rico and Guadeloupe) represented a unique haplotype not found in other countries, indicating no relationship between the USA (Florida) and Caribbean introductions. There is no evidence for cryptic speciation for D. citri based on the COI region included in this study.The online version of this article is published within an Open Access environment subject to the conditions of the Creative Commons Attribution-NonCommercial-ShareAlike licence < http://creativecommons.org/licenses/by-nc-sa/2.5/ > . The written permission of Cambridge University Press must be obtained for commercial re-use.

## Introduction

The Asian citrus psyllid, Diaphorina citri Kuwayama, Hemiptera: Psyllidae, is a phloem feeding insect native to Asia and the Far East with a host range of only Citrus spp. and its near Rutaceae relatives (Halbert & Núñez, 2004). Diaphorina citri causes little direct feeding damage, but it vectors the phloem limited bacterial citrus pathogens, 'Candidatus Liberibacter asiaticus', 'C. L. africanus', and 'C. L. americanus', which cause 'citrus greening disease' or Huanglongbing (HLB), a disease which is considered one of the most damaging to citrus (Capoor et al., 1967;Martinez & Wallace, 1967;Teixeira et al., 2005). The impact of this disease on world citrus production and concern over its continued spread have lead to research on the vector's plant host range, interactions with parasitoids and host-vector-pathogen interactions (Bové, 2006;Halbert & Manjunath, 2004;Hall et al., 2008). Despite these endeavors, there is virtually no information available regarding D. citri genetic diversity and the implications this may hold for these and other studies.

The Asian citrus psyllid's current distribution extends from the Arabian Peninsula (Saudi Arabia and Yemen) and from Afghanistan through to the Indian Subcontinent, Southeast Asia and the Ryukyu Archipelago and Kyushu in Japan, Taiwan, Hong Kong and the coastal provinces of Guangxi, Guangdong, Fujian and Zhejiang in China, the Philippine Islands and through the Indonesian archipelago to northeastern Papua New Guinea; in the Pacific Islands of Hawaii and Guam; the continental USA (Alabama, California, Florida, Georgia, Louisiana, Mississippi, South Carolina and Texas), as well as the Caribbean (Bahamas, Cayman Islands, Cuba, Jamaica, Dominican Republic, Guadeloupe and the Commonwealth of Puerto Rico), Central (Honduras and Mexico) and South America (Argentina, Brazil, Paraguay, Uruguay and Venezuela) and the Indian Ocean islands of Mauritius and Réunion (Costa Lima, 1942;French et al., 2001;Halbert & Manjunath, 2004;Halbert & Núñez, 2004;Mead, 1977;Pluke et al., 2008;Tsai et al., 2000). Because of the lack of global phylogenetics for this species, hypotheses describing its origin include either southeastern Asia or southwestern Asia (India/Pakistan) (Mead, 1977). Anthropogenic spread via the pathway of infested plant material has been the principal means by which this pest has spread. This was responsible for the New World invasions that have seen its distribution expand to include countries in North, Central and South America and the Caribbean (Núñez, 2004). Diaphorina citri was first reported from the continental USA (Florida) in 1998 (Bové, 2006) and now occurs from Florida to California.

Understanding the genetic diversity underlying an invasive pest is a key element that supports the development of effective management strategies (Rosen, 1978;Schauff et al., 1998). Complicating the analysis of genetic diversity among world populations of many invasive insects is that many contain a cryptically diverse collection of populations that cannot be easily categorized by morphological characteristics (Scheffer, 2000;Clarke et al., 2005). Diversity can be represented as distinct biotypes and may extend to reproductive isolation, suggesting the existence of multiple cryptic species (Bickford et al., 2007;. The emergence of molecular systematic approaches has allowed the more precise and rapid characterization of diversity among populations of invasive species beyond what can be ascertained from morphological evaluations. This information can be used to identify the geographic origin of the invaders, if not directly, then at least from an evolutionary perspective (Xavier et al., 2009;De Barro & Ahmed, 2011). Such knowledge is invaluable when looking for effective biological control agents, and understanding basic biology and ecology.

The purpose of this study was to use mitochondrial cytochrome oxidase I (mtCO1) to assess the genetic relationships between different collections of D. citri from around the world. The questions we set out to answer were: (i) Can mtCOI haplotype analysis provide an insight into phylogeographic relationships of Old World D. citri populations? If so, (ii) is there population structure that can be used to monitor global movement of specific genetic types? And, (iii) is there evidence of cryptic species based on relative genetic relationships of individuals from the different collections?


## Methods

Collection of global D. citri collections and outgroup selection Individuals were collected from Bali, Brazil, China, USA (Florida and Texas), Guadeloupe, India, Java, Mauritius Island, Mexico, Pakistan, Puerto Rico, Réunion Island, Saudia Arabia, Taiwan, Texas-USA, Thailand and Vietnam (table 1). All material was stored in 70% ethanol until DNA was extracted. In most cases, multiple individuals were collected in each location. Truncated sequences (sequences less than 550 base pairs) were deleted to leave a final aligned data set of 212 individual sequences. These sequences were then screened using a similarity matrix, and eight unique haplotypes were identified. An outgroup mtCOI sequence was from Pachypsylla venusta, the hackberry petiole gall psyllid (NC_006157). This was chosen because it was the closest psyllid relative for which the entire 612 bp mtCOI sequence was available and was used to aid in alignment.

Identification of novel COI primers specific for D. citri Amplification of mtCOI was not successful using the universal primers C1-J-2195 and TL2-N-3014 (Simon et al., 1994). New mtCOI primers for D. citri were designed from an EST library generated at the USDA (Hunter et al., 2008) and available in GenBank (http://www.ncbi.nlm.nih.gov/). The primers were: DCITRI COI-L 5′-AGG AGG TGG AGA CCC AAT CT and DCITRI COI-R 5′-TCA ATT GGG GGA GAG TTT TG. These amplify an 821 base pair fragment of the mtCOI coding region.


## DNA extraction, amplification and sequencing

Total DNA was extracted using Cartagen's (http://www. cartagen.com) rapid homogenization kit for plant leaf DNA amplification (catalog number 20700-500, lot number 08180400134). The 30 μl PCR reactions were heated at 94°C for two minutes followed by 35 cycles of 30 s at 94°C denaturation, 30 s at 53°C annealing, 1 min at 72°C extension and a final extension of 72°C for 10 min in a MJ Research PTC-200 Peltier thermal cycler. The PCR reactions were composed of 27 μl Platinum PCR SuperMix (Invitrogen, catalog number 11306-016), 1 μl forward primer (10 ρmol), 1 μl reverse primer (10 ρmol), and 1 μl DNA template. Prior to sequencing, the amplified products were cleaned using the montage PCR filter units from Millipore (catalog number UFC7PCR50). All sequencing was performed using the amplification primers and BigDye® Terminator Cycle Sequencing Kits. Sequence 
1-dcitrico1 FJ190167 Dcit-1 psy1-2-dcitrico1 FJ190168 Dcit-1 psy1-3-dcitrico1 FJ190169 Dcit-1 psy1-4-dcitrico1 FJ190170 Dcit-1 psy1-5-dcitrico1 FJ190171 Dcit-1 psy9-1-dcitrico1 FJ190192 Dcit-1 psy9-2-dcitrico1 FJ190193 Dcit-1 psy9-3-dcitrico1 FJ190194 Dcit-1 psy9-4-dcitrico1 FJ190195 Dcit-1 psy9-5-dcitrico1 FJ190196 Dcit-1 psy10-1-dcitrico1 FJ190197 Dcit-1 psy10-2-dcitrico1 FJ190198 Dcit-1 psy10-3-dcitrico1 FJ190199 Dcit-1 psy10-4-dcitrico1 FJ190200 Dcit-1 psy10-5-dcitrico1 FJ190201 Dcit-1 psy11-1-dcitrico1 FJ190202 Dcit-1 psy11-2-dcitrico1 FJ190203 Dcit-1 psy11-3-dcitrico1 FJ190204 Dcit-1 psy11-4-dcitrico1 FJ190205 Dcit-1 psy11-5-dcitrico1 FJ190206 Dcit-1 psy18-1-dcitrico1 FJ190232 Dcit-1 psy18-2-dcitrico1 FJ190233 Dcit-1 psy18-3-dcitrico1 FJ190234 Dcit-1 psy18-4-dcitrico1 FJ190235 Dcit-1 psy18-5-dcitrico1 FJ190236 Dcit-1 psy30-1-dcitrico1 FJ190277 Dcit-1 psy30-3-dcitrico1 FJ190278 Dcit-1 psy21-1-dcitrico1 FJ190246 Dcit-1 psy21-2-dcitrico1 FJ190247 Dcit-1 psy21-3-dcitrico1 FJ190248 Dcit-7 psy21-4-dcitrico1 FJ190249 Dcit-1 psy21-5-dcitrico1 FJ190250 Dcit-1 Palm Beach, Co. (16) psy2-1-dcitrico1 FJ190172 Dcit-1 psy2-2-dcitrico1 FJ190173 Dcit-1 psy2-3-dcitrico1 FJ190174 Dcit-1 psy2-4-dcitrico1 FJ190175 Dcit-1 psy2-5-dcitrico1 FJ190176 Dcit-1 psy4-1-dcitrico1 FJ190182 Dcit-1 psy4-2-dcitrico1 FJ190183 Dcit-1 psy4-3-dcitrico1 FJ190184 Dcit-1 psy4-4-dcitrico1 FJ190185 Dcit-1 psy4-5-dcitrico1 FJ190186 Dcit-1 psy16-1-dcitrico1 FJ190226 Dcit-1 psy16-5-dcitrico1 FJ190227 Dcit-1 psy20-1-dcitrico1 FJ190242 Dcit-1 psy20-2-dcitrico1 FJ190243 Dcit-1 psy20-4-dcitrico1 FJ190244 Dcit-1 psy20-5-dcitrico1 FJ190245 Dcit-1 Brevard, Co. (10) psy13-1-dcitrico1 FJ190211 Dcit-1 psy13-2-dcitrico1 FJ190212 Dcit-1 psy13-3-dcitrico1 FJ190213 Dcit-1 psy13-4-dcitrico1 FJ190214 Dcit-1 psy13-5-dcitrico1 FJ190215 Dcit-1 psy19-1-dcitrico1 FJ190237 Dcit-1 psy19-2-dcitrico1 FJ190238 Dcit-1 psy19-3-dcitrico1 FJ190239 Dcit-1 psy19-4-dcitrico1 FJ190240 Dcit-1 psy19-5-dcitrico1 FJ190241 Dcit-1 Miami-Dade, Co. (4) psy23-1-dcitrico1 FJ190256 Dcit-1 psy23-3-dcitrico1 FJ190257 Dcit-1 psy23-4-dcitrico1 FJ190258 Dcit-1 psy23-5-dcitrico1 FJ190259 Dcit-1 Broward, Co. (2) psy39-2-dcitrico1 FJ190310 Dcit-1 psy39-5-dcitrico1 FJ190311 Dcit-1
Phylogeography of D. citri  
1-dcitrico1 FJ190187 Dcit-1 psy8-2-dcitrico1 FJ190188 Dcit-1 psy8-3-dcitrico1 FJ190189 Dcit-1 psy8-4-dcitrico1 FJ190190 Dcit-1 psy8-5-dcitrico1 FJ190191 Dcit-1 psy14-1-dcitrico1 FJ190216 Dcit-1 psy14-2-dcitrico1 FJ190217 Dcit-1 psy14-3-dcitrico1 FJ190218 Dcit-1 psy14-4-dcitrico1 FJ190219 Dcit-1 psy14-5-dcitrico1 FJ190220 Dcit-1 Okeechobee, Co. (9) psy12-1-dcitrico1 FJ190207 Dcit-1 psy12-2-dcitrico1 FJ190208 Dcit-1 psy12-4-dcitrico1 FJ190209 Dcit-1 psy12-5-dcitrico1 FJ190210 Dcit-1 psy22-1-dcitrico1 FJ190251 Dcit-1 psy22-2-dcitrico1 FJ190252 Dcit-1 psy22-3-dcitrico1 FJ190183 Dcit-1 psy22-4-dcitrico1 FJ190254 Dcit-1 psy22-5-dcitrico1 FJ190255 Dcit-1 Martin, Co. (5) psy15-1-dcitrico1 FJ190221 Dcit-1 psy15-2-dcitrico1 FJ190222 Dcit-1 psy15-3-dcitrico1 FJ190223 Dcit-1 psy15-4-dcitrico1 FJ190224 Dcit-1 psy15-5-dcitrico1 FJ190225 Dcit-1 Collier, Co. (6) psy55-4-dcitrico1 FJ190372 Dcit-1 psy55-5-dcitrico1 FJ190373 Dcit-1 psy56-1-dcitrico1 FJ190374 Dcit-1 psy56-2-dcitrico1 FJ190375 Dcit-1 psy56-3-dcitrico1 FJ190376 Dcit-1 psy56-5-dcitrico1 FJ190377 Dcit-1 USA, Texas Hidalgo, Co. (5) psy3-1-dcitrico1 FJ190177 Dcit-1 psy3-2-dcitrico1 FJ190178 Dcit-1 psy3-3-dcitrico1 FJ190179 Dcit-1 psy3-4-dcitrico1 FJ190180 Dcit-1 psy3-5-dcitrico1 FJ190181 Dcit-1 Brazil Sao Paulo (17) psy17-1-dcitrico1 FJ190228 Dcit-2 psy17-2-dcitrico1 FJ190229 Dcit-2 psy17-3-dcitrico1 FJ190230 Dcit-2 psy17-4-dcitrico1 FJ190231 Dcit-2 psy42-2-dcitrico1 FJ190321 Dcit-1 psy42-3-dcitrico1 FJ190322 Dcit-1 psy42-4-dcitrico1 FJ190323 Dcit-1 psy42-5-dcitrico1 FJ190324 Dcit-1 psy43-1-dcitrico1 FJ190325 Dcit-2 psy43-2-dcitrico1 FJ190326 Dcit-2 psy43-3-dcitrico1 FJ190327 Dcit-2 psy43-5-dcitrico1 FJ190328 Dcit-2 psy44-1-dcitrico1 FJ190329 Dcit-2 psy44-2-dcitrico1 FJ190330 Dcit-2 psy44-3-dcitrico1 FJ190331 Dcit-2 psy44-4-dcitrico1 FJ190332 Dcit-2 psy44-5-dcitrico1 FJ190333 Dcit-2 Puerto Rico Univ. of PR (3) psy24-3-dcitrico1 FJ190260 Dcit-3 psy24-4-dcitrico1 FJ190261 Dcit-3 psy24-5-dcitrico1 FJ190262 Dcit-3 Indonesia Java (10) psy26-2-dcitrico1 FJ190263 Dcit-2 psy26-4-dcitrico1 FJ190264 Dcit-2 psy26-5-dcitrico1 FJ190265 Dcit-2 psy31-1-dcitrico1 FJ190279 Dcit-2 psy31-2-dcitrico1 FJ190280 Dcit-2 psy31-3-dcitrico1 FJ190281 Dcit-2 psy31-5-dcitrico1 FJ190282 Dcit-2 psy45-1-dcitrico1 FJ190279 Dcit-1 psy45-2-dcitrico1 FJ190280 Dcit-1 psy45-3-dcitrico1 FJ190336 Dcit-1 L.M. Boykin et al.Bali (6) psy27-2-dcitrico1 FJ190266 Dcit-2 psy28-1-dcitrico1 FJ190267 Dcit-2 psy28-2-dcitrico1 FJ190268 Dcit-2 psy28-3-dcitrico1 FJ190269 Dcit-2 psy28-4-dcitrico1 FJ190270 Dcit-2 psy28-5-dcitrico1 FJ190271 Dcit-2 Vietnam Hanoi (10) psy29-1-dcitrico1 FJ190272 Dcit-2 psy29-2-dcitrico1 FJ190273 Dcit-2 psy29-3-dcitrico1 FJ190274 Dcit-2 psy29-4-dcitrico1 FJ190275 Dcit-2 psy29-5-dcitrico1 FJ190276 Dcit-2 psy57-1-dcitrico1 FJ190378 Dcit-2 psy57-2-dcitrico1 FJ190379 Dcit-2 psy57-3-dcitrico1 FJ190380 Dcit-2 psy57-4-dcitrico1 FJ190381 Dcit-2 psy57-5-dcitrico1 FJ190382 Dcit-2 Taiwan Taipei (5) psy32-1-dcitrico1 FJ190283 Dcit-2 psy32-2-dcitrico1 FJ190284 Dcit-2 psy32-3-dcitrico1 FJ190285 Dcit-2 psy32-4-dcitrico1 FJ190286 Dcit-2 psy32-5-dcitrico1 FJ190287 Dcit-2 Pakistan Punjab (5) psy33-1-dcitrico1 FJ190288 Dcit-6 psy33-2-dcitrico1 FJ190289 Dcit-6 psy33-3-dcitrico1 FJ190290 Dcit-6 psy33-4-dcitrico1 FJ190291 Dcit-6 psy33-5-dcitrico1 FJ190292 Dcit-6 Thailand Hat Yai (4) psy34-2-dcitrico1 FJ190293 Dcit-2 psy34-3-dcitrico1 FJ190294 Dcit-2 psy34-4-dcitrico1 FJ190295 Dcit-2 psy34-5-dcitrico1 FJ190296 Dcit-2 China Zhejaing (3) psy35-2-dcitrico1 FJ190297 Dcit-5 psy35-4-dcitrico1 FJ190298 Dcit-5 psy35-5-dcitrico1 FJ190299 Dcit-5 Fuzhou (9) psy51-1-dcitrico1 FJ190357 Dcit-2 psy51-2-dcitrico1 FJ190358 Dcit-2 psy51-3-dcitrico1 FJ190359 Dcit-2 psy51-4-dcitrico1 FJ190360 Dcit-2 psy51-5-dcitrico1 FJ190361 Dcit-2 psy52-1-dcitrico1 FJ190362 Dcit-2 psy52-2-dcitrico1 FJ190363 Dcit-2 psy52-3-dcitrico1 FJ190364 Dcit-2 psy52-4-dcitrico1 FJ190365 Dcit-2 Gangzhou (4) psy54-1-dcitrico1 FJ190366 Dcit-2 psy54-2-dcitrico1 FJ190367 Dcit-8 psy54-4-dcitrico1 FJ190368 Dcit-2 psy54-5-dcitrico1 FJ190369 Dcit-2 Mexico Akil, Yucatan (6) psy36-1-dcitrico1 FJ190300 Dcit-4 psy36-2-dcitrico1 FJ190301 Dcit-4 psy36-3-dcitrico1 FJ190302 Dcit-4 psy37-1-dcitrico1 FJ190303 Dcit-4 psy37-2-dcitrico1 FJ190304 Dcit-4 psy37-3-dcitrico1 FJ190305 Dcit-4 Nuevo Leon (4) psy38-1-dcitrico1 FJ190306 Dcit-1 psy38-3-dcitrico1 FJ190307 Dcit-1 psy38-4-dcitrico1 FJ190308 Dcit-1 psy38-5-dcitrico1 FJ190309 Dcit-1 Mauritius (5) psy40-1-dcitrico1 FJ190312 Dcit-2 psy40-2-dcitrico1 FJ190313 Dcit-2 psy40-3-dcitrico1 FJ190314 Dcit-2 psy40-4-dcitrico1 FJ190315 Dcit-2 psy40-5-dcitrico1 FJ190316 Dcit-2 Reunion (4) psy41-1-dcitrico1 FJ190317 Dcit-2 psy41-2-dcitrico1 FJ190318 Dcit-1 psy41-3-dcitrico1 FJ190319 Dcit-2 psy41-4-dcitrico1 FJ190320 Dcit-2
Phylogeography of D. citri analysis was conducted on an Applied Biosystems 3730 × l DNA Analyzer. Sequence fragments were assembled with Sequencher ® version 4.2 (Gene Codes Corporation, 2004) and aligned using ClustalX (Thompson et al., 1997). Minor alignment issues were corrected using Se-Al (Rambaut, 2000). Sequences generated at the United States Horticultural Research Laboratory (USHRL) were deposited in GenBank (table 1), and voucher specimens are deposited in the USHRL collection in Fort Pierce, Florida.


## Haplotype network construction

A haplotype network was constructed using the statistical parsimony program TCS (Clement et al., 2000). TCS estimates phylogenetics networks from DNA sequences or nucleotide distances using the statistical parsimony algorithm which estimates the 95% parsimony limits and outgroup weights (Templeton et al., 1992). Computation was performed using an alignment of each of the eight haplotypes produced in ClustalX. The program was used to construct a statistical parsimony cladogram from parsimonious connections calculated at P j ≥ 0.95 and a matrix of absolute pairwise differences was calculated. Haplotype frequencies were not used to calculate outgroup probabilities, as there was minimal sharing of haplotypes among locations and our sampling effort was not uniform (e.g. there was more sampling within Florida than elsewhere).


## Results and discussion


## Identification of D. citri mtCOI haplotypes

Alignment of the mtCOI sequences showed the presence of eight polymorphic sites within the 612 bps. Four of the substitutions corresponded to synonymous sites and four to non-synonymous (amino-acid altering) sites (table 2). Six of the eight polymorphisms were the result of transitions while two were from transversions. The eight haplotypes were highly similar with shared identities >99% (table 2). Two of these haplotypes (Dcit-1 and Dcit-2) were the most frequent and represented 87% of the sequences (table 3). Of the remaining six haplotypes, two (Dcit-7 and Dcit-8) were rare variants found among populations predominated by Dcit-1 or Dcit-2. The other four haplotypes (Dcit-3, Dcit-4, Dcit-5 and Dcit-6), without the benefit of larger sample sizes, appear to be fixed in their respective populations.

The rare haplotypes Dcit-7 and Dcit-8 were each recovered from a single individual in Florida and China, respectively. Moreover, each of these rare variants differed from all other sequences by a single non-synonymous (amino acid altering) substitution at an otherwise invariant site. Inspection of sequence chromatograms shows clean sequence reads for both variants, indicating that these were not related to sequence read errors.


## Global relationships of D. citri

Due to the minimal sequence diversity in this intraspecific haplotype comparison, and the resulting poor support for the phylogenetic inference using standard tree construction methods (data not shown), it was determined that a haplotype network analysis would be the most appropriate way to consider the relationship between haplotypes (Hart & Sunday, 2007;Chen et al., 2010;De Barro & Ahmed, 2011). The analysis using TCS is graphically represented in fig. 1.

By considering the two most frequently encountered haplotypes and their singleton derivatives as separate 'groups', Dcit-1 groups with Dcit-4, Dcit-6 and Dcit-7 to form what we refer to as the SWA group. Dcit-1 was the most frequently encountered haplotype in this group. The other haplotype group was composed of Dcit-2 and Dcit-8 and represents the SEA group. Placement of Dcit-3 (all Caribbean collections) and Dcit-5 (a single Chinese collection) was ambiguous, possibly because of the small numbers collected from those areas and because of the greater number  This network analysis shows a diverged haplotype, Dcit-3, distributed in the Caribbean yet absent from all other samples ( fig. 1). This result was surprising since the Caribbean was thought to be the source of D. citri that entered Florida (Halbert & Núñez, 2004). Since Dcit-3 is not found in Florida, either the Florida D. citri have come from some location other than the Caribbean or there is greater diversity in the Caribbean than we observed in the four collections  and 13 individuals obtained from Puerto Rico and Guadaloupe (table 1). The relatively distant relationship between the Florida and Caribbean haplotypes (three substitutions, see fig. 1) indicate that the invasion of Florida did not come from the established Caribbean (Guadeloupe and Puerto Rico) populations, but more sampling is needed to verify these findings. Further, because D. citri is hypothesized to have entered the Caribbean as recently as the 1990s (Hall, 2008) and the Caribbean samples have a mtCOI genotype that was not found anywhere else in the old world, we are left to draw the conclusion that there is more worldwide diversity than what was captured in our sampling.
1-dcitrico1 FJ190337 Dcit-1 psy46-2-dcitrico1 FJ190338 Dcit-1 psy46-3-dcitrico1 FJ190339 Dcit-1 psy46-4-dcitrico1 FJ190340 Dcit-1 psy46-5-dcitrico1 FJ190341 Dcit-1 India (4) psy47-1-dcitrico1 FJ190342 Dcit-1 psy47-2-dcitrico1 FJ190343 Dcit-1 psy47-4-dcitrico1 FJ190344 Dcit-1 psy47-5-dcitrico1 FJ190345 Dcit-1 Guadeloupe (11) psy48-1-dcitrico1 FJ190346 Dcit-3 psy48-2-dcitrico1 FJ190347 Dcit-3 psy48-3-dcitrico1 FJ190348 Dcit-3 psy48-4-dcitrico1 FJ190349 Dcit-3 psy49-1-dcitrico1 FJ190350 Dcit-3 psy49-3-dcitrico1 FJ190351 Dcit-3 psy49-4-dcitrico1 FJ190352 Dcit-3 psy49-5-dcitrico1 FJ190353 Dcit-3 psy50-1-dcitrico1 FJ190354 Dcit-3 psy50-2-dcitrico1 FJ190355 Dcit-3 psy50-5-dcitrico1
Within the Old World, few haplotypes predominate. However, the distribution of these haplotypes suggests a phylogeographic break. Dcit-2 occurs in >90% of all the Southeast Asia individuals and 75% of all Chinese samples. Despite its common occurrence in SE Asia and China, it appears to be absent from the Middle East and the Indian subcontinent. Conversely, Dcit-1 and Dcit-6, just one synonymous mutation away, are the only haplotypes observed in both the Middle East and Indian subcontinent. Considering all the haplotypes present in Old World collections, Dcit-1 and Dcit-6 displayed a southwestern Asia (SWA) bias and Dcit-2, -5 and -8 a southeastern Asia bias.

In the New World, where D. citri was introduced within the last 25 years (Halbert & Núñez, 2004), a geographical bias to haplotype distribution was also observed. With the exception of one individual carrying haplotype Dcit-7, all 23 USA collections (99 individuals) contained Dcit-1. The individual carrying Dcit-7, which differs by a single nonsynonymous substitution, was collected from St Lucie County, Florida alongside Dcit-1. In the four Brazilian collections, both Dcit-1 and Dcit-2 haplotypes were observed, but not in the same collections. In Mexico, three collections were analyzed, one from northern Mexico near the Texas, USA border and the other two from the Yucatan Peninsula. The collection close to Texas contained the same haplotype found in Texas (Dcit-1), but the Yucatan Peninsula collections  
G A G G G A A A Dcit-2 A A G G G A A G Dcit-3 G G G G G A T G Dcit-4 G A G G G C A A Dcit-5 A A G G G C A A Dcit-6 G A G A G A A A Dcit-7 G A G G A A A A Dcit-8 A A A G G A A G Sub. Type* Ns S Ns S Ns Ns S S
* Substitution type is indicated as (S) synonymous or (Ns) nonsynonymous.


## Phylogeography of D. citri

were fixed for a unique haplotype Dcit-4, which differed from Dcit-1 by a single nucleotide. Although D. citri is a pest that has spread worldwide, unlike other pests such as the whitefly Bemisia tabaci (Boykin et al., 2007b;Dinsdale et al., 2010), the low level of sequence diversity indicates that it is most likely a single species rather than a species complex.

A major aim of this study was to show how D. citri in Florida are related to D. citri elsewhere. We found that the Florida D. citri were most closely related to those from Mexico, Pakistan, Brazil, Texas, Saudi Arabia, India, Indonesia and Réunion Island, and includes the SWA sequences. New World populations show three distinct separations based on phylogeographic interpretation of the haplotype network ( fig. 1). North American continental populations (USA: Florida and Texas; and Mexico) belong to the SWA group, whereas individuals from locations in South America were either SWA or SEA in origin; SWA and SEA did not co-occur in any of the collections, suggesting two separate introductions. The only case where a collection from a single location contained individuals from both SWA and SEA was in the Indian Ocean island of Réunion. It is interesting to consider here whether there is some factor that limits the capacity of these two groups to co-occur. The only other New World locations sampled were the two Caribbean islands of Puerto Rico and Guadeloupe, where both contained the same haplotype Dcit-3.


## Geographic origins of D. citri

Network-based genealogies can offer insight into ancestral sequences, as these sequences are expected to be common, widely distributed and highly connected (Chen et al., 2010). While these predictions hold in theory, application is often difficult. Insufficient or non-uniform sampling schemes can bias the frequencies of observed haplotypes. Anthropogenic activity can alter geographical distributions at a scale and rate well beyond the ability of genetic systems to equilibrate. In our study, >60% of our samples are from the New World where D. citri is invasive. Using frequency of occurrence or geographic distribution as a metric for predicting ancestry would bias predictions by frequency of invasion and invasion success.

While both haplotype frequency and geographical distribution are likely not reflective of equilibrium evolutionary processes, patterns of haplotype connectivity may be more robust. Using this criterion, the mitochondrial haplotype network for D. citri suggests a basal and thus ancestral position for Dcit-1 ( fig. 1). Even a conservative analysis, excluding all New World haplotypes that were singleton branches from the main Dcit-1 and Dcit-2 groups leaving only old world haplotypes, predicts Dcit-1 as ancestral based on connectivity.

Although early work suggested a far eastern Asia (Asia region to the east of India) origin of D. citri (Mead, 1977), more recent reconstructions based on plant host origins and historical information suggest a southwestern Asia origin, i.e. India (Hall, 2008). Therefore, ancestral placement of the SWA group supports the more contemporary view of D. citri evolution. However, because of limited sampling in the Old World, we do not have a clear phylogeographic picture of the complete distribution of the SWA and SEA groups. The phylogeographic interpretations of our data identify two geographical redistributions of D. citri. First, some apparent geographical separation between the SWA and SEA groups that requires more intense sampling in the Old World to provide a true characterization of this relationship, and then a more recent radiation of the SWA and SEA groups into the New World most likely within the last 25 years (Halbert & Núñez, 2004). There is one earlier report of D. citri from Brazil in the early 1940s (Costa Lima, 1942); however, no one can substantiate this record, and it is certain that established populations were not reported until much more recently.


## Conclusion

Our data indicate that with respect to the mCOI haplotypes and within the limits of our global sampling, there is very little genetic diversity among the populations worldwide. Despite this limited diversity, enough information was gained to partially answer the three questions on which this research was based.

(i) Can mtCOI haplotype analysis provide an insight into phylogeographic relationships of Old World D. citri populations? Limited sampling became a factor in this interpretation; however, two mtCOI haplotypes predominate, Dcit-1 and Dcit-2. These differed from each other by just two nucleotides and formed the basis of Dcit-1 group individuals (Dcit -1,-4, -6 and -7) and Dcit-2 group individuals (Dcit-2 and -8), as defined by those individuals whose mtCOI sequence diverges from the most common sequence (either Dcit-1 or Dcit-2) by only one nucleotide. Distribution of these two haplotype groups displayed some apparent Old World geographical bias. The Dcit-1 group was more abundant in SWA samples, and the Dcit-2 group was more abundant in SEA samples. Clearly, more sampling was done in SEA (Indonesia, Vietnam, Taiwan, Thailand and China) with 12 different collections, than in SWA (Pakistan, Saudia Arabia and India) with only three collections. However, if collections from Mauritius and Reunion are excluded as anthropogenic introductions because they are not considered a center of D. citri diversity and are isolated islands where both Dcit-1 and-2 are present, 11 of the 12 SEA samples contained individuals belonging to the Dcit-2 group, while all SWA samples contained only Dcit-1 group individuals. Based on these results, there is an appearance of geographically distinct populations; however, due to the limited sequence divergence between the observed groups, more in-depth sampling is needed for this to be verified. The authors suggest that further sampling from these Middle East and Asian countries needs to be conducted to better survey diversity and also utilize the other genetic markers such as microsatellites (Boykin et al., 2007a).

(ii) Is there population structure that can be used to monitor global movement of specific genetic types? Analysis of the New World collections showed that invasions into North America came from the SWA group only, whereas those involving South America came from both the SWA and SEA groups. There was extensive sampling in the USA (22 collections within Florida) and only minimal sampling in South America (four collections), so it is unlikely that SEA populations were missed in the USA. Despite this observed difference in haplotype distribution in the New World, our limited Old World sampling and the small amount of global mtCOI sequence variation precludes the ability to provide a concise determination of origin. However, knowledge from this research may be useful in searching for effective biological control agents and may help explain differences in effectiveness of Tamarixia radiata, a parasitoid introduced in the New World to control invasive populations of D. citri (Hoy & Nguyen, 2001;Hoy et al., 2004;Michaund, 2004;Pluke et al., 2008;Barr et al., 2009). Although, one needs to be cautious here, as ecological factors may be as or more important than genetic variability within the parasitoid and should not be overlooked.

(iii) Is there evidence of cryptic species based on relative genetic relationships of collections? All individuals were closely related, and there is no evidence for complex cryptic species structure within these populations. This may be expected as genetic variation is expected to increase with time since divergence (globalization) and increased host variability (local adaptation). D. citri has a very limited host range (only within the Rutaceae) and has only recently (within the last 15 to 25 years) spread to the New World.


substitutions in relation to the most common sequences.

## Fig. 1 .
1Haplotype network generated with statistical parsimony analysis of D. citri mtCOI DNA sequence. Multicolored markers indicate the presence and identity of multiple haplotypes found within a region. The block dots represent hypothetical haplotypes. See table 3 to evaluate haplotype frequencies.

## Table 1 .
1Accession numbers for each of the 212 sequences from 52 populations of Diaphorina citri used to generatefig. 1.Location 
Locality 
Genebank Code 
Accession Number 
Haplotype Assignment* 

USA, Florida 
St. Lucie, Co. (32) 
psy1-

## Table 1 .
1(Cont.) 

Location 
Locality 
Genebank Code 
Accession Number 
Haplotype Assignment* 

Indian River, Co. (10) 
psy8-

## Table 1 .
1(Cont.) 

Location 
Locality 
Genebank Code 
Accession Number 
Haplotype Assignment* 



## Table 1 .
1(Cont.) 

Location 
Locality 
Genebank Code 
Accession Number 
Haplotype Assignment* 

Saudi Arabia 
(5) 
psy46-

## Table 3 .
3Geographic distribution and abundance of Diaphorina citri haplotypes.Location 
Number of 
individuals 
(collections) 

Abundance by haplotype 

Dcit-01 
Dcit-02 
Dcit-03 
Dcit-04 
Dcit-05 
Dcit-06 
Dcit-07 
Dcit-08 

Florida, USA 
94 (22) 
93 
1 
Texas, USA 
5 (1) 
5 
Mexico 
10 (3) 
4 
6 
Puerto Rico 
3 (1) 
3 
Guadeloupe Island 
11 (3) 
11 
Brazil 
17 (4) 
4 
13 
Saudi Arabia 
5 (1) 
5 
Pakistan 
5 (1) 
5 
India 
4 (1) 
4 
China 
16 (4) 
12 
3 
1 
Taiwan 
5 (1) 
5 
Vietnam 
10 (2) 
10 
Thailand 
4 (1) 
4 
Indonesia 
16 (5) 
3 
17 
Mauritius Island 
5 (1) 
5 
Reunion Island 
4 (1) 
1 
3 

Total 
212 (52) 
122 
69 
14 
6 
3 
5 
1 
1 



## Table 2 .
2Unique haplotypes and polymorphism locations observed from the alignment of the 212 Diaphorina citri mtCOI sequences. 

Haplotype 
Base location relative to start of the FJ190167 accession 

130 
138 
217 
447 
500 
533 
612 
645 
Dcit-1 

AcknowledgementsThe authors would like to thank Ankush Zitsi, Chris Knox, Paula Hall, Kathy Moulton, Matthew Hentz and John Prokop for laboratory assistance of the D. citri collections at the USHRL. Diana Percy provided valuable discussions regarding sequencing of the COI region. W. Evan Braswell provided useful comments to the manuscript, table 3 andfig. 1. We thank Y. S. Ahlawat, Virendra Baranwal, Shahid Nadeem Chohan, Shu-Sheng Liu, Baoli Qui, Ratana Sdoodee and Siti Subandiyah for providing us with samples.
Comparison of laboratory colonies and field populations of Tamarixia radiata, an ectoparasitoid of the Asian citrus psyllid, using internal transcribed spacer and cytochrome oxidase subunit I DNA sequences. N B Barr, D G Hall, Iii Weathersbee, A A Nguyen, R Stansly, P Qureshi, J A Flores, D , Journal of Economic Entomology. 102Barr, N.B., Hall, D.G., Weathersbee III, A.A., Nguyen, R., Stansly, P., Qureshi, J.A. & Flores, D. (2009) Comparison of laboratory colonies and field populations of Tamarixia radiata, an ectoparasitoid of the Asian citrus psyllid, using internal transcribed spacer and cytochrome oxidase subunit I DNA sequences. Journal of Economic Entomology 102, 2325-2332.

Cryptic species as a window on diversity and conservation. D Bickford, D J Lohman, N S Sodhi, P K Ng, R Meier, K Winker, K K Ingram, I Das, Trends in Ecology and Evolution. 22Bickford, D., Lohman, D.J., Sodhi, N.S., Ng, P.K., Meier, R., Winker, K., Ingram, K.K. & Das, I. (2007) Cryptic species as a window on diversity and conservation. Trends in Ecology and Evolution 22, 148-155.

Huanglongbing: a destructive, newly-emerging, century-old disease of citrus. J Bové, Journal of Plant Pathology. 88Bové, J. (2006) Huanglongbing: a destructive, newly-emerging, century-old disease of citrus. Journal of Plant Pathology 88, 7-37.

Twelve polymorphic microsatellite loci from the Asian citrus psyllid, Diaphorina citri Kuwayama, the vector for citrus greening disease, huanglongbing. L M Boykin, R A Bagnall, D R Frohlich, D G Hall, W B Hunter, C S Katsar, C L Mckenzie, R C Rosell, R G Shatters, Molecular Ecology Notes. 7Boykin, L.M., Bagnall, R.A., Frohlich, D.R., Hall, D.G., Hunter, W.B., Katsar, C.S., McKenzie, C.L., Rosell, R.C. & Shatters, R.G. (2007a) Twelve polymorphic microsatellite loci from the Asian citrus psyllid, Diaphorina citri Kuwayama, the vector for citrus greening disease, huanglongbing. Molecular Ecology Notes 7, 1202-1204.

Global relationships of Bemisia tabaci (Hemiptera: Aleyrodidae) revealed using Bayesian analysis of mitochondrial COI Phylogeography of D. citri DNA sequences. L M Boykin, R G Shatters, Jr, R C Rosell, C L Mckenzie, R A Bagnall, P De Barro, D R Frohlich, Molecular Phylogenetics and Evolution. 44Boykin, L.M., Shatters, R.G. Jr, Rosell, R.C., McKenzie, C.L., Bagnall, R.A., De Barro, P. & Frohlich, D.R. (2007b) Global relationships of Bemisia tabaci (Hemiptera: Aleyrodidae) revealed using Bayesian analysis of mitochondrial COI Phylogeography of D. citri DNA sequences. Molecular Phylogenetics and Evolution 44, 1306-1319.

Diaphorina citri Kuway., a vector of greening disease of citrus in India. S Capoor, D G Rao, S M Viswanath, Indian Journal of Agricultural Science. 37Capoor, S., Rao, D.G. & Viswanath, S.M. (1967) Diaphorina citri Kuway., a vector of greening disease of citrus in India. Indian Journal of Agricultural Science 37, 572-576.

Statistical parsimony networks and species assemblages in Cephalotrichid nemerteans (Nemertea). H Chen, M Strand, J L Norenburg, S Sun, H Kajihara, A V Chernyshev, S A Maslakova, P Sundberg, PLoS One. 5Chen, H., Strand, M., Norenburg, J.L., Sun, S., Kajihara, H., Chernyshev, A.V., Maslakova, S.A. & Sundberg, P. (2010) Statistical parsimony networks and species assemblages in Cephalotrichid nemerteans (Nemertea). PLoS One 5, e12885.

Invasive phytophagous pests arising through a recent tropical evolutionary radiation: the Bactrocera dorsalis complex of fruit flies. A R Clarke, K F Armstrong, A E Carmichael, J R Milne, S Raghu, G K Roderick, D K Yeates, Annual Reveiw of Entomology. 50Clarke, A.R., Armstrong, K.F., Carmichael, A.E., Milne, J.R., Raghu, S., Roderick, G.K. & Yeates, D.K. (2005) Invasive phytophagous pests arising through a recent tropical evol- utionary radiation: the Bactrocera dorsalis complex of fruit flies. Annual Reveiw of Entomology 50, 293-319.

TCS: a computer program to estimate gene genealogies. M Clement, D Posada, K A Crandall, Molecular Ecology. 9Clement, M., Posada, D. & Crandall, K.A. (2000) TCS: a computer program to estimate gene genealogies. Molecular Ecology 9, 1657-1659.

. Costa Lima, A , Homopteros. Insetos do Brazil. 3Costa Lima, A. (1942) Homopteros. Insetos do Brazil 3, 1-327.

Genetic Networking of the Bemisia tabaci Cryptic Species Complex Reveals Pattern of Biological Invasions. P De Barro, M Ahmed, PLoS Biol. 6De Barro, P. & Ahmed, M. (2011) Genetic Networking of the Bemisia tabaci Cryptic Species Complex Reveals Pattern of Biological Invasions. PLoS Biol 6, e25579.

Bemisia tabaci: A Statement of Species Status. P J De Barro, S S Liu, L M Boykin, A B Dinsdale, Annual Reveiw of Entomology. 56De Barro, P.J., Liu, S.S., Boykin, L.M. & Dinsdale, A.B. (2011) Bemisia tabaci: A Statement of Species Status. Annual Reveiw of Entomology 56, 1-19.

Refined global analysis of Bemisia tabaci (Gennadius) (Hemiptera: Sternorrhyncha: Aleyroidea) mitochondrial CO1 to identify species level genetic boundries. A Dinsdale, L Cook, C Riginos, Y M Buckley, P De Barro, Annals of the Entomological Society of America. 103Dinsdale, A., Cook, L., Riginos, C., Buckley, Y.M. & De Barro, P. (2010) Refined global analysis of Bemisia tabaci (Gennadius) (Hemiptera: Sternorrhyncha: Aleyroidea) mitochondrial CO1 to identify species level genetic boundries. Annals of the Entomological Society of America 103, 196-208.

First record of the Asian citrus psylla Diaphorina citri Kuwayama (Homoptera:Psyllidae), in Texas. J V French, C J Kahlke, J V De Graca, Subtropical Plant Science. 53French, J.V., Kahlke, C.J. & De Graca, J.V. (2001) First record of the Asian citrus psylla Diaphorina citri Kuwayama (Homoptera:Psyllidae), in Texas. Subtropical Plant Science 53, 14-15.

. Gene Codes Corporation. Gene Codes Corporation, IncSequencher 4.2.Gene Codes Corporation (2004) Sequencher 4.2. Madison, WI, USA, Gene Codes Corporation, Inc.

Asian citrus psyllids (Sternorrhyncha: Psyllidae) and greening disease of citrus: A literature review and assessment of risk in Florida. S E Halbert, K L Manjunath, Florida Entomologist. 87Halbert, S.E. & Manjunath, K.L. (2004) Asian citrus psyllids (Sternorrhyncha: Psyllidae) and greening disease of citrus: A literature review and assessment of risk in Florida. Florida Entomologist 87, 330-353.

Distribution of the Asian citrus psyllid, Diaphorina citri Kuwayama (Rhynchota: Psyllidae), in the Caribbean basin. S E Halbert, C A Núñez, Florida Entomologist. 87Halbert, S.E. & Núñez, C.A. (2004) Distribution of the Asian citrus psyllid, Diaphorina citri Kuwayama (Rhynchota: Psyllidae), in the Caribbean basin. Florida Entomologist 87, 401-402.

Biology, history and world status of Diaphorina citri. D G Hall, Proceedings of the International Workshop on Huanglongbing and Asian Citrus Psyllid. the International Workshop on Huanglongbing and Asian Citrus PsyllidHermosillo, Sonora, MexicoNorth American Plant Protection OrganizationHall, D.G. (2008) Biology, history and world status of Diaphorina citri. pp. 1-11 in Proceedings of the International Workshop on Huanglongbing and Asian Citrus Psyllid, North American Plant Protection Organization, 7-9 May 2008, Hermosillo, Sonora, Mexico.

Population ecology and phenology of Diaphorina citri (Hemiptera: Psyllidae) in two Florida citrus groves. D G Hall, M G Hentz, R C AdairJr, Environmental Entomology. 37Hall, D.G., Hentz, M.G. & Adair, R.C. Jr (2008) Population ecology and phenology of Diaphorina citri (Hemiptera: Psyllidae) in two Florida citrus groves. Environmental Entomology 37, 914-924.

Things fall apart: biological species form unconnected parsimony networks. M W Hart, J Sunday, Biological Letters. 3Hart, M.W. & Sunday, J. (2007) Things fall apart: biological species form unconnected parsimony networks. Biological Letters 3, 509-512.

Classical biological control of Asian citrus psylla. M A Hoy, R Nguyen, Citrus Industry. 81Hoy, M.A. & Nguyen, R. (2001) Classical biological control of Asian citrus psylla. Citrus Industry 81, 48-50.

Classical biological control of Asian citrus psyllid in Florida. Integrated Pest management Florida. M A Hoy, R Nguyen, A Jeyaprakash, 13University of FloridaHoy, M.A., Nguyen, R. & Jeyaprakash, A. (2004) Classical bio- logical control of Asian citrus psyllid in Florida. Integrated Pest management Florida, University of Florida, available online at http://ipm.ifas.ufl.edu/agriculture/citrus/psyllid. shtml (accessed 13 March 2012).

Psyllid biology: Expressed genes in adult Asian citrus psyllids. W Hunter, C S Katsar, R G Shatters, Jr, C L Mckenzie, A A Weathersbee, D G Hall, Diaphorina citri Kuwayama. The Open Entomology Journal. 3Hunter, W., Katsar, C.S., Shatters, R.G. Jr, McKenzie, C.L., Weathersbee, A.A. & Hall, D.G. (2008) Psyllid biology: Expressed genes in adult Asian citrus psyllids, Diaphorina citri Kuwayama. The Open Entomology Journal 3, 18-29.

Citrus leaf-mottle-yellows disease in the Philippines and transmission of the causal virus by a psyllid, Diaphorina citri. A Martinez, J Wallace, Plant Disease Report. 51Martinez, A. & Wallace, J. (1967) Citrus leaf-mottle-yellows disease in the Philippines and transmission of the causal virus by a psyllid, Diaphorina citri. Plant Disease Report 51, 692-695.

Natural mortality of the Asian citrus psyllid (Homoptera: Psyllidae) in central Florida. J P Michaund, Biological Control. 296Michaund, J.P. (2004) Natural mortality of the Asian citrus psyllid (Homoptera: Psyllidae) in central Florida. Biological Control 296, 260-269.

The Asiatic citrus psyllid, Diaphorina citri Kuwayama (Homoptera: Psyllidae). Florida Department of Agriculture Conservation Service, Division of Plant Industry Entomology Circular 180, Florida Division of Plant Industry. F W Mead, Mead, F.W. (1977) The Asiatic citrus psyllid, Diaphorina citri Kuwayama (Homoptera: Psyllidae). Florida Department of Agriculture Conservation Service, Division of Plant Industry Entomology Circular 180, Florida Division of Plant Industry, available online at http://www.freshfromflorida.com/pi/ enpp/ento/entcirc/ent180.pdf.

Distribution of the Asian Citrus Psyllid, Diaphorina citri Kuwayama (Rhynchota: Psyllidae) in the Caribbean basin. C Núñez, Florida Entomologist. 87Núñez, C. (2004) Distribution of the Asian Citrus Psyllid, Diaphorina citri Kuwayama (Rhynchota: Psyllidae) in the Caribbean basin. Florida Entomologist 87, 401-402.

Citrus flushing patterns, Diaphorina citri (Hemiptera: Psyllidae) populations and parasitism by Tamarixia radiata (Hymenoptera: Eulophidae) in Puerto Rico. R W H Pluke, J A Qureshi, P A Stansly, Florida Entomologist. 91Pluke, R.W.H., Qureshi, J.A. & Stansly, P.A. (2008) Citrus flushing patterns, Diaphorina citri (Hemiptera: Psyllidae) populations and parasitism by Tamarixia radiata (Hymenoptera: Eulophidae) in Puerto Rico. Florida Entomologist 91, 36-42.

Se-Al: Sequence Alignment Editor v2.0. Se-Al v2.0a11 carbon edn. A Rambaut, Oxford, UK, University of OxfordRambaut, A. (2000) Se-Al: Sequence Alignment Editor v2.0. Se-Al v2.0a11 carbon edn. Oxford, UK, University of Oxford.

The importance of cryptic species and specific identifications as related to biological control. D Rosen, Biosystematics in Agriculture. Romberger, J.A.Osmun & CoRosen, D. (1978) The importance of cryptic species and specific identifications as related to biological control. pp. 23-35 in Romberger, J.A. (Ed.) Biosystematics in Agriculture. Montclair, NJ, USA, Allanheld, Osmun & Co.

The Genera of Chalcid Parasitoids (Hymenoptera: Chalcidoidea) of. M E Schauff, J Lasalle, G A Wijeskara, Schauff, M.E., LaSalle, J. & Wijeskara, G.A. (1998) The Genera of Chalcid Parasitoids (Hymenoptera: Chalcidoidea) of

Lepidoptera: Gracillariidae). Citrus Leafminer Phyllocnistis citrella Stainton. 32Journal of Natural HistoryCitrus Leafminer Phyllocnistis citrella Stainton (Lepidoptera: Gracillariidae). Journal of Natural History 32, 1001-1056.

Molecular evidence of cryptic species within the Liriomyza huidobrensis (Diptera: Agromyzidae). S J Scheffer, Journal of Economic Entomology. 93Scheffer, S.J. (2000) Molecular evidence of cryptic species within the Liriomyza huidobrensis (Diptera: Agromyzidae). Journal of Economic Entomology 93, 1146-1151.

Evolution, weighting, and phylogenetic utility of mitochondrial gene sequences and a compilation of conserved polymerase chain reaction "primers. C Simon, F Frati, A Beckenbach, B Crespi, H Lui, P Flook, Annals of the Entomological Society of America. 87Simon, C., Frati, F., Beckenbach, A., Crespi, B., Lui, H. & Flook, P. (1994) Evolution, weighting, and phylogenetic utility of mitochondrial gene sequences and a compilation of conserved polymerase chain reaction "primers". Annals of the Entomological Society of America 87, 651-701.

Candidatus Liberibacter americanus', associated with citrus huanglongbing (greening disease) in São Paulo State. D C Teixeira, C Saillard, S Eveillard, J L Danet, P Inácio Da Costa, A J Ayres, J Bové, Brazil. International Journal of Systematic and Evolutionary Microbiology. 55Teixeira, D.C., Saillard, C., Eveillard, S., Danet, J.L., Inácio da Costa, P., Ayres, A.J. & Bové, J. (2005) 'Candidatus Liberibacter americanus', associated with citrus huan- glongbing (greening disease) in São Paulo State, Brazil. International Journal of Systematic and Evolutionary Microbiology 55, 1857-1862.

A cladistic analysis of phenotypic associations with haplotypes inferred from restriction endonuclease mapping and DNA sequence data. III. Cladogram estimation. A R Templeton, K A Crandall, C F Sing, Genetics. 132Templeton, A.R., Crandall, K.A. & Sing, C.F. (1992) A cladistic analysis of phenotypic associations with haplotypes inferred from restriction endonuclease mapping and DNA sequence data. III. Cladogram estimation. Genetics 132, 619-633.

The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. J D Thompson, T J Gibson, F Plewniak, F Jeanmougin, D G Higgins, Nucleic Acids Research. 24Thompson, J.D., Gibson, T.J., Plewniak, F., Jeanmougin, F. & Higgins, D.G. (1997) The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Research 24, 4876-4882.

Sampling of Diaphorina citri (Homoptera: Psyllidae) on orange Jasmine in southern Florida. J H Tsai, J J Wang, Y H Liu, Florida Entomologist. 83Tsai, J.H., Wang, J.J. & Liu, Y.H. (2000) Sampling of Diaphorina citri (Homoptera: Psyllidae) on orange Jasmine in southern Florida. Florida Entomologist 83, 446-459.

Invasion or invisibility: using genetic and distributional data to investigate the alien or indigenous status of the Atlantic populations of the peracarid isopod. R Xavier, A M Santos, F P Lima, M Branco, Stenosoma nadejda. 18Xavier, R., Santos, A.M., Lima, F.P. & Branco, M. (2009) Invasion or invisibility: using genetic and distributional data to in- vestigate the alien or indigenous status of the Atlantic populations of the peracarid isopod, Stenosoma nadejda (Rezig 1989). Molecular Ecology 18, 3283-3290.

. L M Boykin, L.M. Boykin et al.