# Functional genomics tools applied to plant metabolism: a survey on plant respiration, its connections and the annotation of complex gene functions

CorpusID: 9669553
 
tags: #Medicine, #Environmental_Science, #Biology

URL: [https://www.semanticscholar.org/paper/4529b01607f07ef41594d2953eab9d113c506d30](https://www.semanticscholar.org/paper/4529b01607f07ef41594d2953eab9d113c506d30)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | True |
| By Annotator      | (Not Annotated) |

---

Functional genomics tools applied to plant metabolism: a survey on plant respiration, its connections and the annotation of complex gene functions
published: 06 September 2012

Wagner L Araújo wlaraujo@ufv.br 
Departamento de Biologia Vegetal
Universidade Federal de Viçosa
ViçosaBrazil

Adriano Nunes-Nesi 
Departamento de Biologia Vegetal
Universidade Federal de Viçosa
ViçosaBrazil

Departamento de Biologia Vegetal
Max-Planck Partner Group
Universidade Federal de Viçosa
ViçosaBrazil

Max Planck Institute of Molecular Plant Physiology
Departamento de Biologia Vegetal
Purdue University
USA, Germany

Universidade Federal de Viçosa
36570-000Viçosa, Minas GeraisBrazil

Thomas C R Williams 
Departamento de Biologia Vegetal
Universidade Federal de Viçosa
ViçosaBrazil

John A Morgan 
Ronan Sulpice 
Wagner L Araújo 
Functional genomics tools applied to plant metabolism: a survey on plant respiration, its connections and the annotation of complex gene functions
published: 06 September 201210.3389/fpls.2012.00210REVIEW ARTICLE Edited by: Daniel J. Kliebenstein, University of California, Davis, USA Reviewed by: *Correspondence:functional genomicssystems biologyplant respirationomicsdata integrationmodeling approaches
The application of post-genomic techniques in plant respiration studies has greatly improved our ability to assign functions to gene products. In addition it has also revealed previously unappreciated interactions between distal elements of metabolism. Such results have reinforced the need to consider plant respiratory metabolism as part of a complex network and making sense of such interactions will ultimately require the construction of predictive and mechanistic models. Transcriptomics, proteomics, metabolomics, and the quantification of metabolic flux will be of great value in creating such models both by facilitating the annotation of complex gene function, determining their structure and by furnishing the quantitative data required to test them. In this review, we highlight how these experimental approaches have contributed to our current understanding of plant respiratory metabolism and its interplay with associated process (e.g., photosynthesis, photorespiration, and nitrogen metabolism). We also discuss how data from these techniques may be integrated, with the ultimate aim of identifying mechanisms that control and regulate plant respiration and discovering novel gene functions with potential biotechnological implications.

## INTRODUCTION

Metabolism represents one of the most important and most likely the best characterized network within biological systems. From pioneering studies defining key metabolic pathways, subsequent decades of enzymology characterizing the catalytic and regulatory properties of enzymes, through to more recent genetic studies of metabolism, there is an unprecedented density of both mechanistic and descriptive data relating to metabolic behavior. Advances in the understanding of metabolic regulation and control, however, still suffer from insufficient research concerning the mode of operation of metabolic networks, and plant respiratory metabolism provides an illustrative example of this problem. Due to the crucial roles of mitochondria in meeting the energy demands of the cell whilst simultaneously being involved in amino acid metabolism and photorespiration, precise control of mitochondrial metabolism and function is crucial for cellular homeostasis, and unsurprisingly mitochondrial dysfunction leads to diverse metabolic and phenotypic consequences . However, despite the fact that the major respiratory pathways in plants were elucidated decades ago (Beevers, 1961), our current knowledge about their regulation and control, and how this relates to whole plant physiology, is still limited (Fernie et al., 2004). Unsurprisingly then, considerable research effort is being devoted to elucidating the metabolic basis of the regulation of the tricarboxylic acid (TCA) cycle and the mitochondrial electron transport chain Fahnenstich et al., 2008;Fukushima et al., 2009;Araújo et al., 2010;Tomaz et al., 2010;Zell et al., 2010) as well as their interactions with photosynthesis , photorespiration (Bauwe et al., 2010(Bauwe et al., , 2012, and nitrate assimilation (Foyer et al., 2011).

Given the complex patterns of regulation and the interactions between respiratory metabolism and other processes uncovered by such studies, it is likely that only through the use of modelbased approaches we will be able to understand how the individual components of metabolism (e.g., enzymes and metabolites) work together to produce a functioning system, and in this way truly understand the function of individual components of that system. In this context systems biology offers the promise of a better understanding of respiratory metabolism and its role within the metabolic network.

Systems biology aims to understand how populations of molecules, cells, and organisms interact to give rise to complex biological processes, including cell division, growth, development, metabolism, and behavioral and ecological patterns (Karsenti, 2012). Although initially applied exclusively to mathematical modeling strategies (Edwards and Palsson, 1999) systems biology may now incorporate data from the functional genomics tools of transcriptomics, proteomics, and metabolomics as well as genomics (Somerville et al., 2004;Baginsky and Fernie, 2007), and has become a truly interdisciplinary area of research, some basic and applied aspects of which have been expertly reviewed elsewhere (Ideker et al., 2001;Oltvai and Barabási, 2002;Kitano, 2004;Somerville et al., 2004;Breitling, 2010). Whilst an accurate understanding of metabolism as a system requires far more than enumeration of its components (Kleessen et al., 2012), it is nonetheless the case that the development of mathematical models of plant metabolism in general will require accurate gene annotations, functional characterization of enzymes, and the acquisition and statistical analysis of quantitative data.

In the case of plant respiratory metabolism, whilst the core components, including enzymes of the TCA cycle and components of the mitochondrial electron transport chain, have been identified and characterized in numerous species Araújo et al., 2012a), how they are regulated in response to changing environmental conditions and developmental programs remains less clear, and extent to which respiratory metabolism interacts with other parts of the metabolic network is only now beginning to emerge (Balmer et al., 2004;Sweetlove et al., 2006;Timm et al., 2008Timm et al., , 2011Fukushima et al., 2009;Tomaz et al., 2010;Foyer et al., 2011). Moreover, the functions of more peripheral components of plant respiratory metabolism remain to be investigated. Genes for many of the mitochondrial transporters that are hypothesized to exist have yet to be identified, or have only been assigned on the basis of homology with non-plant species (Linka and Weber, 2010;Palmieri et al., 2011), whilst it is only recently that several alternative electron donors to the mitochondrial electron transport chain have been characterized (Ishizaki et al., 2005(Ishizaki et al., , 2006Araújo et al., 2010). Although here we focus mainly on function and interactions of respiratory enzymes, it is important to note that the construction of predictive models of mitochondrial metabolism will also require a better understanding of electron transport processes. Thus, both structural biology and measurements of electron transport rates remain exciting topics for further study.

The availability of comprehensive plant genome information, coupled with the integration of large-scale unbiased molecular profiling technologies such as whole-genome microarrays, quantitative proteomics, and metabolite profiling (Table 1; Figure 1) has increased our ability to annotate gene functions within plant respiratory metabolism, obtain quantitative data and uncover unanticipated relationships between respiration and other cellular processes such as photosynthesis, photorespiration, redox regulation, and signaling (Sweetlove et al., 2002;Heazlewood et al., 2004;Dutilleul et al., 2005;Urbanczyk-Wochniak et al., 2006). This information will ultimately facilitate the application of systems biology to plant respiratory metabolism (Fernie, 2012) and progress in these areas as they relate to research on plant respiration and the annotation of gene function will be discussed in the following sections.


## APPLYING TRANSCRIPTOMICS TO THE STUDY OF MITOCHONDRIAL FUNCTION

Determining the complete transcriptional capability of the cell, including large and small RNAs, novel transcripts from unannotated genes, splicing isoforms, and gene-fusion transcripts, serves as the foundation for a comprehensive study of the transcriptome. Recent advances in DNA sequencing technology have greatly increased both sequencing scale and throughput and led to the development of RNA-Seq (Liu et al., 2011;Martin and Wang,  2011), a technique capable of complementing microarray-based transcriptomics. In recent years transcriptomic techniques have been extensively used to reveal interactions between plant respiratory metabolism and other important processes such as seed germination , stress tolerance (Giraud et al., 2008;Lehmann et al., 2009;Meyer et al., 2009), hypoxia , and the operation of the circadian clock (Fukushima et al., 2009). The combination of a comprehensive list of putatively mitochondrially expressed genes with a large number of publicly available stress-related microarray datasets was used to define the mitochondrial stress response in Arabidopsis (van Aken et al., 2009). This complex analysis indicated that the mitochondrial stress response extends far beyond the alternative electron transport chain components, with, most prominently, mitochondrial substrate carrier proteins and heat shock proteins also showing extensive stress responsiveness. It is important to note, however, that many of the stress-responsive genes discovered by this approach are of unknown function and this indicates an attractive avenue for research aiming to identify new mitochondrial functions and targets for engineering of stress tolerance.

Transcriptomic studies can also provide details of expression patterns during plant development. Recent work indicated that BCS1, a likely AAA-ATPase, displayed similar levels of expression throughout development (Duncan et al., 2011), despite the fact that previous studies have shown that this gene responds to various stresses (van Aken et al., 2009). Based on these patterns of expression the authors propose that the protein encoded by this gene may be involved in protein repair in response to stress (Xu et al., 2011).

This study also indicated that the transcript abundance for genes encoding outer and inner membrane mitochondrial proteins was maximal during seed germination, and identified a subset of genes that were highly expressed in root tissues (Duncan et al., 2011).

Responses to stress conditions, including the effect of absence of alternative oxidase1a (aox1a) expression in Arabidopsis, has also been investigated using transcriptomics. Such plants, when grown in moderate light under drought conditions, displayed 10-fold increases in leaf anthocyanin levels, together with alterations in photosynthetic efficiency, increased superoxide radical production, and reduced root growth (Giraud et al., 2008). Furthermore, microarray and quantitative reverse transcription polymerase chain reaction (qRT-PCR) analysis indicated that even under normal growth conditions genes normally induced under stress conditions were expressed (Giraud et al., 2008), including those involved in defense against reactive oxygen species (ROS) and in stress signaling. This study was incorporated into a recent meta-analysis of transcriptomic studies that aimed to uncover targets of mitochondrial retrograde signaling (Schwarzländer et al., 2012), and construct a model for how mitochondrial dysfunction may affect nuclear gene expression. Transcripts encoding proteins involved in photosynthesis, protein synthesis, and plant-pathogen interactions were revealed as the major targets of retrograde regulation.

A combination of transcript and metabolite profiling methods has been used to investigate the molecular and physiological responses following root hypoxia caused by flooding in gray poplar (Populus × canescens; Kreuzwieser et al., 2009). Interestingly, changes in metabolite levels occurred in both roots and leaves, whilst changes in transcript abundance were restricted to the roots, the actual site of hypoxia. The general pattern of metabolite and transcript abundance suggests that the response to hypoxia comprises both the repression of energy demanding processes such as cell wall biosynthesis, and the reconfiguration of carbohydrate and nitrogen metabolism to ensure sufficient substrate supply during long periods of root hypoxia (Kreuzwieser et al., 2009).


## PROTEOMIC APPROACHES FOR THE DISCOVERY OF GENE FUNCTION IN PLANTS

Proteomic approaches have been used to generate great insights into plant metabolism in general and into mitochondrial respiratory metabolism in particular. Recently, the utilization of multiple fluorescent dyes in single gels (DIGE) has renewed the use of 2-D gels for quantitative comparisons of the mitochondrial proteome. Studies using DIGE have investigated knockout mutants for carbonic anhydrase-like proteins (Perales et al., 2005), complex I (Meyer et al., 2009), and malate dehydrogenase (Tomaz et al., 2010), as well as differences between tissues (Lee et al., 2008), the impact of rotenone (Garmier et al., 2008), and changes associated with the diurnal cycle . Shotgun proteomic techniques utilizing Liquid chromatography coupled tandem mass spectrometry (LCMS/MS) of trypsin-digested samples, without the use of gels, have also been used for in-depth studies of the mitochondrial proteome in Arabidopsis (Brugiere et al., 2004;Heazlewood et al., 2004) and rice (Heazlewood et al., 2003b;Huang et al., 2009). These studies greatly increased the number of proteins identified within plant mitochondrial isolates that are below the www.frontiersin.org level of detection by techniques based on gel separation and staining.

The mitochondrial proteome of Arabidopsis in particular has been extensively analyzed by both gel-based and gel-free strategies (Kruft et al., 2001;Millar et al., 2001;Heazlewood et al., 2004;Lee et al., 2008Lee et al., , 2011, and may contain as many as 2000-3000 different proteins, each of which may be subject to post-translational modification (Millar et al., 2005;Taylor et al., 2011). Recent proteomic studies have been able to identify more than 500 proteins, including subunits of mitochondrial respiratory complexes, supercomplexes, phosphorylated proteins, and oxidized proteins (Millar et al., 2005;. However, even these identified proteins have not all been functionally characterized. Moreover, targeting prediction software tools assign more than 1,500 proteins encoded by the Arabidopsis genome to this subcellular compartment and it therefore seems likely that the function of most mitochondrial proteins, especially those of low abundance and/or high hydrophobicity , remains to be discovered.

In addition to analysis of the mitochondrial proteome as a whole, proteomic approaches have been used extensively to dissect mitochondrial protein complexes, in particular complex I of the respiratory electron transport chain (Leterme and Boutry, 1993;Herz et al., 1994;, and the importance of this characterization using different proteomic approaches has recently been expertly reviewed . These methods have revealed that the complex I in plants has an extremely sophisticated configuration and although some of the postulated functions of the characterized extra subunits remain to be fully investigated, proteomic approaches have already contributed significantly to our current understanding of this complex (Heazlewood et al., 2003a;Cardol et al., 2004;Klodmann et al., 2010. For example, recent work has revealed that plant complex I is especially large and includes 15 extra unique subunits. Five of these subunits represent proteins resembling carbonic anhydrases and one represents a L-galactono-1,4-lactone dehydrogenase (GLDH), introducing side activities to plant complex I, while all the remaining nine subunits are rather small (7-12 kDa) and of so far unknown function (Klodmann et al., 2010;.

Many mitochondrial proteins in fact form part of protein complexes and it has recently been shown that the formation of supercomplexes, "complexes of complexes," could have a regulatory function in guiding electrons through alternative respiratory pathways, particularly under variations in oxygen levels (Ramírez-Aguilar et al., 2011). The extent to which this phenomenon occurs is currently unclear though as only the most abundant complexes have so far been functionally characterized.

Proteomics may also help to provide details of the mechanisms that regulate mitochondrial metabolism, and recent studies have identified potential thioredoxin-linked proteins in mitochondria isolated from autotrophic and heterotrophic plant tissues (Balmer et al., 2004;Martí et al., 2009). Fifty mitochondrial thioredoxinlinked proteins were identified including six TCA cycle enzymes [aconitase, succinyl-CoA ligase, isocitrate, malate, pyruvate, and succinate dehydrogenases (SDH); Balmer et al., 2004]. These findings suggest mitochondrial enzymes as potential targets for redox regulation through thioredoxins. However, despite the fact that these studies, amongst others, indicate that by sensing redox state thioredoxins enable mitochondria to communicate with other organelles in photosynthetic tissues, considerable experimental effort is still required to provide functional details.


## INSIGHTS INTO PLANT RESPIRATORY METABOLISM OBTAINED FROM METABOLITE PROFILING

The term metabolome can be defined as the total small-molecule complement of a cell, and metabolomics is therefore the study of all small molecules or metabolites presents in a cell or organism (Oliver et al., 1998;Tweeddale et al., 1998). Although no single analytical system is ever likely to cover the whole metabolome technological developments have considerably extended our ability to analyze complex biological systems, facilitating the simultaneous detection of different compound classes with diverse chemical properties (Osorio et al., 2012). Gas chromatography coupled to mass spectrometry (GC-MS) is a versatile and widely applied technique in modern metabolomic studies and allows the identification and quantification of a relatively broad range of compounds, including organic and amino acids, sugars, sugar alcohols, phosphorylated intermediates, and lipophilic compounds (Fiehn et al., 2000;Roessner et al., 2001;Weckwerth, 2003;Lisec et al., 2006;, resulting in fairly comprehensive coverage of the central pathways of primary metabolism. Metabolite profiling using LC-MS and capillary electrophoresis-MS can complement the analysis by GC-MS for certain classes of metabolite. Of particular note, tandem MS methods for the quantification of phosphorylated sugars (Lunn et al., 2006;Luo et al., 2007;Hasunuma et al., 2010), Calvin-Benson cycle intermediates (Arrivault et al., 2009) and cell wall precursors (Alonso et al., 2010), which are difficult to analyze by GC-MS, have recently been developed.

Due in part to these technological advances application of metabolite profiling to plant biology is becoming increasingly common, including diagnostic and descriptive analyses of metabolic responses to genetic and/or environmental perturbations, the use in annotation of gene function and in systems biology Guy et al., 2008;Fernie and Schauer, 2009). Recently, metabolite profiling has, in particular, been used to generate significant insight into plant respiratory metabolism and its interactions with photosynthesis. Despite recognition of the importance of mitochondrial metabolism our understanding of the complex pathways through which organic acids are metabolized as well as how these pathways are regulated in vivo remains far from complete (Fernie et al., 2004;Sweetlove et al., 2007;Fait et al., 2008;Møller and Sweetlove, 2010). As part of attempts to remedy this issue, the operation and role of the TCA cycle in illuminated leaves has received particular interest (Krömer, 1995;Raghavendra and Padmasree, 2003;Noguchi and Yoshida, 2008;Nunes-Nesi et al., 2011). The precise mode of operation of the TCA cycle in the light remains somewhat obscure, in part because whilst TCA cycle flux in the light is reduced through inhibition of pyruvate dehydrogenase carbon skeletons deriving from TCA cycle intermediates are still required for nitrogen assimilation. In attempt to address this apparent contradiction, a process of systematic suppression of TCA cycle enzymes through reverse genetics was undertaken. Metabolite profiling of the transgenic lines generated through this process indicated the importance of the TCA cycle in metabolism in illuminated leaves . However, these experiments have also revealed a surprising complexity in the response to these manipulations, with suppression of some enzymes leading to increased photosynthesis (Carrari et al., 2003;Nunes-Nesi et al., 2005;Araújo et al., 2011b), others leading to decreased photosynthesis ) and yet others having no effect (Studart-Guimarães et al., 2007;Sienkiewicz-Porzucek et al., 2008, 2010Sulpice et al., 2010a;Araújo et al., 2012b). Several of the more recent studies from this investigation, highlighting the importance of information obtained from metabolite profiling experiments, are discussed here.

Antisense inhibition of the 2-oxoglutarate dehydrogenase (OGDH) complex in tomato led to alterations in whole plant development that were linked to reductions in total amino acid and nitrate pools despite unaltered photosynthesis (Araújo et al., 2012b). These results clearly imply that OGDH plays an important role in both the TCA cycle and nitrogen assimilation and suggest a novel role for this enzyme in whole plant development and fruit maturation. Furthermore, as was reported for succinyl-CoA ligase antisense plants (Studart-Guimarães et al., 2007), the OGDH antisense plants displayed increased GABA shunt flux, presumably in compensation for decreased succinate production via the TCA cycle (Araújo et al., 2012b). However, it is important to stress that such a compensatory response also caused significant shifts in cellular pools of amino acids and nitrate that were detected by metabolite profiling.

Antisense inhibition of the iron-sulfur subunit of the SDH in tomato plants, on the other hand, resulted in increased photosynthesis and whole plant biomass via an organic acid-mediated effect on stomatal aperture. These results contrast with those obtained for antisense inhibition of fumarase, where decreased photosynthesis and biomass were observed Araújo et al., 2011b). Furthermore, measurement of apoplastic organic acid levels in SDH and fumarase antisense plants, revealed a negative correlation between the levels of fumarate and stomatal conductance, though the influence of fumarate appears to be weaker than that of malate (Araújo et al., 2011b). These results provided strong evidence to support that modulation of malate and fumarate concentration can greatly influence stomatal function. Thus, metabolite profiling has aided in the identification of novel interactions between the TCA cycle and photosynthesis, stomatal function and nitrogen metabolism. Further work is clearly required in order to establish the regulatory mechanisms involved in such responses.

Protein degradation during plant development and substrate deficiency can be an important source of substrate for respiratory metabolism, however, our current understanding of the regulation of the classical and alternative pathways of respiration under these conditions is still limited. The combination of genetic approaches with GC-MS-based metabolite profiling has greatly improved our understanding of the complex metabolic interactions observed during dark-induced senescence (Ishizaki et al., 2005(Ishizaki et al., , 2006Araújo et al., 2010Araújo et al., , 2011a. These studies have demonstrated that during dark-induced senescence there is a significant accumulation of amino acids and TCA cycle intermediates. Moreover, it has also been demonstrated that both isovaleryl-CoA dehydrogenase and D-2-hydroxyglutarate dehydrogenase provide electrons to the plant ubiquinol pool via the electron transfer flavoprotein (ETF)-ETF:ubiquinone oxidoreductase (ETF/ETFQO) complex (Engqvist et al., 2009;Araújo et al., 2010). Given that the chlorophyll breakdown intermediate phytanoyl-CoA accumulates dramatically both in knockout mutants of the ETF/ETFQO complex and of isovaleryl-CoA dehydrogenase following growth in extended dark periods it was suggested that chlorophyll breakdown could be important for the supply of carbon and electrons during this process. However, metabolic analyses of phytanoyl-CoA 2-hydroxylase knockout mutants under the same extended darkness regime as previously used suggest that phytol and phytanoyl-CoA enzyme does not primarily function as a substrate of the ETF/ETFQO pathway (Araújo et al., 2011a). These studies also showed that these mutants were not compromised in their ability to withstand significant extension of the dark period but do accumulate phytanoyl-CoA and to a lesser extent 2-hydroxyglutarate as well as sharing some of the other metabolic features of mutants of the ETF/ETFQO complex following darkinduced senescence treatment. In summary, the results obtained through this work indicated that both isovaleryl-CoA dehydrogenase and 2-hydroxyglutarate dehydrogenase essentially account for the entire electron input via the ETF complex.

Plant metabolism is also reorganized under a range of different stress conditions including salt, cold, drought, and oxidative stress (Kaplan et al., 2004(Kaplan et al., , 2007Gong et al., 2005;Sanchez et al., 2008Sanchez et al., , 2012Alet et al., 2012;Siahpoosh et al., 2012), allowing plants to continue to produce indispensable metabolites whilst preventing the accumulation of ROS. Metabolite profiling has proven a powerful tool to gain an overview of such reorganizations in response to stressful conditions (Shulaev et al., 2008). Menadione, a quinone which causes ROS generation from both mitochondrial and plastidal electron transport chains, has been used to study metabolic and transcriptomic responses to ROS in Arabidopsis roots (Lehmann et al., 2009). Detailed metabolic studies revealed a down-regulation of glycolysis and TCA cycle and the redirection of carbon from glycolysis to the oxidative pentose phosphate pathway in menadione-treated roots suggesting a reorganization of central carbon metabolism under oxidative stress conditions that was undone rapidly after the removal of menadione (Lehmann et al., 2012).


## CONTRIBUTIONS OF METABOLIC FLUX ANALYSIS TO OUR UNDERSTANDING OF PLANT RESPIRATION

Metabolic flux ultimately underpins both plant growth and development and for this reason experiments that provide insight into metabolic flux, as well as its regulation and control, can be valuable tools to both improve our understanding of biological systems and aid in the discovery of gene function. Feeding experiments using both 13 C-and 14 C-labeled precursors have been used extensively to delineate metabolic pathways, but may also be used to investigate and quantify metabolic flux. Labeling experiments with 14 C offer high sensitivity and fractionation of labeled metabolites and biomass components can readily indicate the fate of metabolized radiolabel. Accordingly, feeding experiments with www.frontiersin.org 14 CO 2 demonstrated the effect of reduced malate dehydrogenase activity on photosynthetic carbon assimilation in tomato leaves, and revealed that ascorbate feeding led to increased photosynthesis and altered assimilate partitioning in these transgenic lines (Nunes-Nesi et al., 2005). Quantifying the release of 14 CO 2 from the metabolism of different isotopomers of 14 C-glucose can also provide information about the relative activity of different primary metabolic pathways (Nunes-Nesi et al., 2005). This method has been used extensively to identify changes in metabolic flux in plants with altered TCA cycle enzyme activities; for example, decreased carbon entry into the TCA cycle was identified in tomato plants with decreased SDH and OGDH activity (Araújo et al., 2011b(Araújo et al., , 2012b, which are in good agreement with measurements of respiration and the results of metabolite profiling experiments.

Labeling experiments with stable isotopes can be used for similar ends, however since detection of label incorporation is in these cases carried out by mass spectrometry or Nuclear Magnetic Resonance (NMR) spectroscopy it is possible to determine both enrichment and location of incorporation of label in individual metabolites without laborious chemical cleavage. This method was used to investigate the fate of 13 C-labeled glutamate in leaves of tomato plants with decreased succinyl-CoA ligase, and revealed that TCA cycle flux was maintained in these plants by diversion of carbon through the GABA shunt (Studart-Guimarães et al., 2007). Additionally, in vivo NMR spectroscopy can be used to monitor such labeling experiments in real-time. For example, tracing the metabolism of 13 C-pyruvate by mitochondria isolated from Arabidopsis plants with reduced manganese superoxide dismutase activity revealed decreased TCA cycle flux (Morgan et al., 2008), most likely as a result of decreased TCA cycle enzyme activity caused by oxidative damage.

Whilst the experiments described above may provide valuable qualitative information about metabolic flux, the number of fluxes that can be quantified using these approaches is typically quite limited. Steady state metabolic flux analysis (Wiechert, 2001;Ratcliffe and Shachar-Hill, 2006) on the other hand allows the absolute quantification of metabolic fluxes in medium size networks, permitting the degree to which a particular environmental condition or gene product controls or regulates multiple fluxes to be determined. In this way, steady state flux analysis has been able to provide important insights into the relationship between photosynthesis and respiration in developing seeds. Labeling experiments and metabolic modeling indicated that developing green embryos are able to decrease losses of fixed carbon during oil synthesis by both refixing CO 2 released by respiration (Schwender et al., 2006), and reducing the need for TCA cycle flux through the use of photosynthesis to meet energy demands (Allen et al., 2009). Although these experiments effectively revealed a new function for RuBisCO, steady state analysis can also be used to reveal the effect of alterations in gene expression. This method has been widely applied for the discovery of gene function in microorganisms (Blank et al., 2005;Fischer and Sauer, 2005), and its feasibility in plants was demonstrated by experiments using Arabidopsis embryos (Lonien and Schwender, 2009) that revealed how flux is rerouted in embryos deficient in two plastidic pyruvate kinase isoforms.

The lack of wider adoption of steady state flux analysis as a tool in plant systems biology is in part due to the strict requirements for an experimental system that can obtain metabolic and isotopic steady state, which limits such experiments to tissue culture and isolated organs. Therefore, the so called instationary flux analysis (Wiechert and Nöh, 2005;Young et al., 2008) has the potential to allow quantification of metabolic flux in networks that do not reach isotopic steady state, by incorporating measurements of labeling dynamics and metabolite pool sizes into the modeling process. Recent work has demonstrated the feasibility of this approach for flux quantification in photoautotrophic organisms (Young et al., 2011), whilst the necessary technological approaches for carrying out this work in plants have already been developed (Arrivault et al., 2009;Hasunuma et al., 2010), suggesting that this method may be used in the discovery of gene function in higher plants in the relatively near future.

Whilst empirical determination of fluxes using labeling experiments can generate important biological insight, constraints based modeling (Price et al., 2003) can also be used to investigate the behavior of metabolic networks. The construction of genome scale metabolic models (Durot et al., 2009;Fell et al., 2010) and their analysis using flux balance analysis, provides both a way to exploit the information pertaining to metabolism contained within a sequenced genome and models now exist for several plant species including Arabidopsis (Poolman et al., 2009;Mintz-Oron et al., 2012), maize (Saha et al., 2011), sugarcane (de Oliveira Dal'Molin et al., 2010, and Brassica napus (Hay and Schwender, 2011). Recent work has highlighted the degree to which flux quantification using constraints based modeling agrees with empirical results (Williams et al., 2010;Beurton-Aimar et al., 2011;Chen et al., 2011), but perhaps a more important role of genome scale modeling is as a means to generate hypotheses regarding the capacities of large metabolic networks.

In this regard, flux balance analysis has already provided interesting insights into plant respiratory metabolism. For example, work using a model of Arabidopsis carbon metabolism revealed the large demand for energy made by cell maintenance processes (Poolman et al., 2009), whilst a study of metabolism in barley seeds (Grafahrend-Belau et al., 2009) indicated how fluxes in central carbon metabolism and the TCA cycle may be affected by the hypoxic conditions that can occur within the developing seed. Such models also provide an excellent means for the discovery of new functions for previously annotated genes though the prediction of the consequences of gene knockouts in silico. Such methods are expected to form an important part of methods for metabolic engineering in species with biotechnological importance. Indeed, models have already been created for the model algal species Chlamydomonas reinhardtii (Boyle and Morgan, 2009;Chang et al., 2011;de Oliveira Dal'Molin et al., 2011) that can be further used to explore the potential of microalgae for biofuel production.


## INTEGRATION OF PROFILING DATA: FROM DIAGNOSTICS TO SYSTEMS BIOLOGY

Whilst investigation of the transcriptome, proteome, and metabolome alone may be highly informative, cellular physiology and the regulation of plant respiratory metabolism is the result of complex interactions between transcripts, proteins, and Frontiers in Plant Science | Plant Systems Biology metabolites. Bearing that in mind straightforward relationships between these components of the metabolic network are therefore not to be expected. Thus, despite the apparent easy understanding of simple relationships between changes in transcripts, proteins, metabolite, and downstream biological functions, quite often there is a major discordance. This apparent discrepancy can often be explained by quantitative analysis of the data sets and information about translational regulation and protein turnover. These incoherencies are not only due to technical issues but also related to the complexity and structure of metabolic networks. Accordingly, some priorities in metabolic research to reduce these problems have recently been proposed (Fernie and Stitt, 2012). For this reason recent research into plant metabolism has often included the usage of modeling and correlation based approaches, and interactions between the results obtained using different post-genomic techniques (Figure 1 and Table 1) are expected.

Such studies have the ability to reveal previously unappreciated and complex interactions between transcripts, proteins, metabolites, and fluxes. In this way, a recent study was able to uncover a potential signaling role for leucine; a combination of extensive metabolite profiling and transcript profiling demonstrated a correlation between expression of a subset of transcripts and the levels of this amino acid (Hannah et al., 2010). An ability to simultaneously quantify components of the metabolic network can also provide information about the relationship between different components of central carbon metabolism and resource allocation. For example, by exploiting the natural variation present in different accessions of Arabidopsis (Sulpice et al., , 2010b, it was possible to detect correlations between enzyme activities, metabolite abundances, and biomass. These studies indicate the presence of a strong negative correlation between starch and biomass, whilst the relative proportion of protein invested in enzymes is positively correlated to biomass accumulation. The relationship between transcripts, metabolites, and fluxes in plant respiratory metabolism was investigated in Arabidopsis cell suspension cultures subjected to menadione treatment (Baxter et al., 2007). The combinatory analysis of these parameters identified a transient reconfiguration of metabolism, involving down-regulation of the TCA cycle and amino acid biosynthesis, apparently working to avoid wasting energy under conditions of oxidative stress. Such methods may also have biotechnological applications. For example, metabolite:transcript correlations identified in potato tubers have revealed target genes for metabolic engineering of sucrose metabolism (Urbanczyk-Wochniak et al., 2003).

Whilst the unbiased nature of such approaches makes them excellent discovery tools, their capacity to improve our understanding of the regulation and control of respiratory metabolism could be further improved by the use of a mechanistic framework that can relate the components of respiratory metabolism and their properties to the functions of this system. In this regard, a strategy recently employed in yeast linking transcriptional regulation to metabolic fluxes (Moxley et al., 2009) may facilitate the integration of data from multiple post-genomic platforms to improve our understanding of plant respiratory metabolism. Whilst direct mRNA measurements were only poorly correlated with metabolic flux, the incorporation of a parameter representing regulation of enzyme activity by metabolite abundance greatly improved this correlation and the resulting model could be used successfully to predict flux changes occurring in amino acid metabolism in yeast. Genome scale metabolic models also provide a promising means via which such integration can take place (Durot et al., 2009;Poolman et al., 2009;Fell et al., 2010;Mintz-Oron et al., 2012). Since such models are based on annotated genome information, transcript profiling, and proteomic data which are already linked to specific genes can potentially be incorporated and used to constrain the behavior of the network. Such an approach has been shown to be useful in microorganisms (Covert and Palsson, 2002) and might also be successfully applied in plants.

Integration may also take place through the use of kinetic models of metabolism (Schallau and Junker, 2010;Rohwer, 2012). Such models are inevitably on a smaller scale than genome scale models, but compensate by providing a mechanistic link between the kinetic and regulatory properties of enzymes and the behavior of the system under study. This approach was used to study metabolism of the aspartate family of amino acids in Arabidopsis (Curien et al., 2009), and revealed how the allosteric properties of enzymes in this pathway permit fluxes in different branches to vary independently of one another. An additional advantage of such models is that once constructed they immediately allow the application of metabolic control analysis, and this has been exploited to investigate the distribution of control over sucrose cycling amongst the enzymes of sucrose synthesis in sugarcane (Rohwer and Botha, 2001;Uys et al., 2007). Whilst the mitochondrial TCA cycle in plants has been the subject of structural modeling (Steuer et al., 2007) and included in models of wider scope (Leduc et al., 2006), a detailed kinetic model based on experimental data has yet to be produced. Profiling techniques could aid in the provision of data for the construction of such a model, but more importantly, a kinetic model of the plant TCA cycle would provide a medium through which the results of profiling experiments could be better understood.

Overall, whilst post-genomic technologies have already aided our understanding of respiratory metabolism at a systems level through additions to the "parts list" of the metabolic network , they are poised to become more useful still through the discovery of new interactions and the provision of the quantitative data required for the construction of predictive models of metabolism. The further development and combination of many analytical techniques (Figure 1) will additionally allow a fuller description of the metabolic status of a plant. When this is achieved, global analyses of RNA, protein, and metabolites will allow us to obtain a full picture of the complexity of the system under study. Although the application of these techniques requires substantial financial investment, it is likely to bring returns in the form of an improved ability to carry out rational engineering of the plant metabolic network.


## ACKNOWLEDGMENTS

Financial support was provided by the National Council for Scientific and Technological Development CNPq-Brazil (grant number 478261/2010-1 to Adriano Nunes-Nesi and 472787/2011-0 to Wagner L. Araújo). Thomas C. R. Williams was supported by a Petrobras (Petroleo Brasileiro S.A.) post-doctoral grant.

www.frontiersin.org

## Table 1 |
1Omics fields associated with illustrative and available Arabidopsis resources on web.Layers 
Instances 
Resources 
Web 

Phenome 
Natural variations 
NASC 
http://arabidopsis.org.uk/home.html 

September 2012 | Volume 3 | Article 210 | 3
September 2012 | Volume 3 | Article 210 | 4
September 2012 | Volume 3 | Article 210 | 5
September 2012 | Volume 3 | Article 210 | 6
September 2012 | Volume 3 | Article 210 | 7
September 2012 | Volume 3 | Article 210 | 8
September 2012 | Volume 3 | Article 210 | 9
September 2012 | Volume 3 | Article 210 | 11
Conflict of Interest Statement:The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.Received
New insights into the role of spermine in Arabidopsis thaliana under long-term salt stress. A I Alet, D H Sanchez, J C Cuevas, M Marina, P Carrasco, T Altabella, A F Tiburcio, O A Ruiz, Plant Sci. 182Alet, A. I., Sanchez, D. H., Cuevas, J. C., Marina, M., Carrasco, P., Alta- bella, T., Tiburcio, A. F., and Ruiz, O. A. (2012). New insights into the role of spermine in Arabidopsis thaliana under long-term salt stress. Plant Sci. 182, 94-100.

The role of light in soybean seed filling metabolism. D K Allen, J B Ohlrogge, Y Shachar-Hill, Plant J. 58Allen, D. K., Ohlrogge, J. B., and Shachar-Hill, Y. (2009). The role of light in soybean seed filling metabolism. Plant J. 58, 220-234.

Quantifying the labeling and the levels of plant cell wall precursors using ion chromatography tandem mass spectrometry. A P Alonso, R J Piasecki, Y Wang, R W Laclair, Y Shachar-Hill, Plant Physiol. 153Alonso, A. P., Piasecki, R. J., Wang, Y., Laclair, R. W., and Shachar-Hill, Y. (2010). Quantifying the label- ing and the levels of plant cell wall precursors using ion chromatogra- phy tandem mass spectrometry. Plant Physiol. 153, 915-924.

Identification of the 2-hydroxyglutarate and isovaleryl-CoA dehydrogenases as alternative electron donors linking lysine catabolism to the electron transport chain of Arabidopsis mitochondria. W L Araújo, K Ishizaki, A Nunes-Nesi, T R Larson, T Tohge, I Krahnert, S Witt, T Obata, N Schauer, I A Graham, C J Leaver, A R Fernie, Plant Cell. 22Araújo, W. L., Ishizaki, K., Nunes- Nesi, A., Larson, T. R., Tohge, T., Krahnert, I., Witt, S., Obata, T., Schauer, N., Graham, I. A., Leaver, C. J., and Fernie, A. R. (2010). Iden- tification of the 2-hydroxyglutarate and isovaleryl-CoA dehydrogenases as alternative electron donors link- ing lysine catabolism to the elec- tron transport chain of Arabidop- sis mitochondria. Plant Cell 22, 1549-1563.

Analysis of a range of catabolic mutants provides evidence that phytanoylcoenzyme A does not act as a substrate of the electron-transfer flavoprotein/electron-transfer flavoprotein:ubiquinone oxidoreductase complex in Arabidopsis during darkinduced senescence. W L Araújo, K Ishizaki, A Nunes-Nesi, T Tohge, T R Larson, I Krahnert, I Balbo, S Witt, P Doermann, I A Graham, C J Leaver, A R Fernie, Plant Physiol. 157Araújo, W. L., Ishizaki, K., Nunes- Nesi, A., Tohge, T., Larson, T. R., Krahnert, I., Balbo, I., Witt, S., Doer- mann, P., Graham, I. A., Leaver, C. J., and Fernie, A. R. (2011a). Anal- ysis of a range of catabolic mutants provides evidence that phytanoyl- coenzyme A does not act as a substrate of the electron-transfer flavoprotein/electron-transfer flavo- protein:ubiquinone oxidoreductase complex in Arabidopsis during dark- induced senescence. Plant Physiol. 157, 55-69.

Antisense inhibition of the iron-sulphur subunit of succinate dehydrogenase enhances photosynthesis and growth in tomato via an organic acid-mediated effect on stomatal aperture. W L Araújo, A Nunes-Nesi, S Osorio, B Usadel, D Fuentes, R Nagy, I Balbo, M Lehmann, C Studart-Witkowski, T Tohge, E Martinoia, X Jordana, F M Damatta, A R Fernie, Plant Cell. 23Araújo, W. L., Nunes-Nesi, A., Oso- rio, S., Usadel, B., Fuentes, D., Nagy, R., Balbo, I., Lehmann, M., Studart- Witkowski, C., Tohge, T., Martinoia, E., Jordana, X., Damatta, F. M., and Fernie, A. R. (2011b). Antisense inhi- bition of the iron-sulphur subunit of succinate dehydrogenase enhances photosynthesis and growth in tomato via an organic acid-mediated effect on stomatal aperture. Plant Cell 23, 600-627.

Metabolic control and regulation of the tricarboxylic acid cycle in photosynthetic and heterotrophic plant tissues. W L Araújo, A Nunes-Nesi, Z Nikoloski, L J Sweetlove, A R Fernie, Plant Cell Environ. 35Araújo, W. L., Nunes-Nesi, A., Nikoloski, Z., Sweetlove, L. J., and Fernie, A. R. (2012a). Metabolic control and regulation of the tricar- boxylic acid cycle in photosynthetic and heterotrophic plant tissues. Plant Cell Environ. 35, 1-21.

Antisense inhibition of the 2-oxoglutarate dehydrogenase complex in tomato demonstrates its importance for plant respiration and during leaf senescence and fruit maturation. W L Araújo, T Tohge, S Osorio, M Lohse, I Balbo, I Krahnert, A Sienkiewicz-Porzucek, B Usadel, A Nunes-Nesi, A R Fernie, Plant Cell. 24Araújo, W. L., Tohge, T., Osorio, S., Lohse, M., Balbo, I., Krahnert, I., Sienkiewicz-Porzucek, A., Usadel, B., Nunes-Nesi, A., and Fernie, A. R. (2012b). Antisense inhibition of the 2-oxoglutarate dehydrogenase complex in tomato demonstrates its importance for plant respiration and during leaf senescence and fruit mat- uration. Plant Cell 24, 2328-2351.

Use of reverse-phase liquid chromatography, linked to tandem mass spectrometry, to profile the Calvin cycle and other metabolic intermediates in Arabidopsis rosettes at different carbon dioxide concentrations. S Arrivault, M Guenther, A Ivakov, R Feil, D Vosloh, J T Van Dongen, R Sulpice, M Stitt, Plant J. 59Arrivault, S., Guenther, M., Ivakov, A., Feil, R., Vosloh, D., Van Dongen, J. T., Sulpice, R., and Stitt, M. (2009). Use of reverse-phase liquid chromatog- raphy, linked to tandem mass spec- trometry, to profile the Calvin cycle and other metabolic intermediates in Arabidopsis rosettes at different car- bon dioxide concentrations. Plant J. 59, 824-839.

. S Baginsky, A R Fernie, Plant Systems Biology. BirkhäuserBaginsky, S., and Fernie, A. R. (2007). Plant Systems Biology. Basel: Birkhäuser.

Thioredoxin links redox to the regulation of fundamental processes of plant mitochondria. Y Balmer, W H Vensel, C K Tanaka, W J Hurkman, E Gelhaye, N Rouhier, J.-P Jacquot, W Manieri, P Schürmann, M Droux, B B Buchanan, Proc. Nat. Acad. Sci. U.S.A. 101Balmer, Y., Vensel, W. H., Tanaka, C. K., Hurkman, W. J., Gelhaye, E., Rouhier, N., Jacquot, J.-P., Manieri, W., Schürmann, P., Droux, M., and Buchanan, B. B. (2004). Thioredoxin links redox to the regulation of funda- mental processes of plant mitochon- dria. Proc. Nat. Acad. Sci. U.S.A. 101, 2642-2647.

Photorespiration: players, partners and origin. H Bauwe, M Hagemann, A R Fernie, Trends Plant Sci. 15Bauwe, H., Hagemann, M., and Fernie, A. R. (2010). Photorespiration: play- ers, partners and origin. Trends Plant Sci. 15, 330-336.

Photorespiration has a dual origin and manifold links to central metabolism. H Bauwe, M Hagemann, R Kern, S Timm, Curr. Opin. Plant Biol. 15Bauwe, H., Hagemann, M., Kern, R., and Timm, S. (2012). Photorespira- tion has a dual origin and manifold links to central metabolism. Curr. Opin. Plant Biol. 15, 269-275.

The metabolic response of heterotrophic Arabidopsis cells to oxidative stress. C J Baxter, H Redestig, N Schauer, D Repsilber, K R Patil, J Nielsen, J Selbig, J Liu, A R Fernie, L J Sweetlove, Plant Physiol. 143Baxter, C. J., Redestig, H., Schauer, N., Repsilber, D., Patil, K. R., Nielsen, J., Selbig, J., Liu, J., Fernie, A. R., and Sweetlove, L. J. (2007). The metabolic response of heterotrophic Arabidopsis cells to oxidative stress. Plant Physiol. 143, 312-325.

Comparison between elementary flux modes analysis and 13C-metabolic fluxes measured in bacterial and plant cells. H Beevers, Row Harper, M Beurton-Aimar, B Beauvoit, A Monier, F Vallee, M Dieuaide-Noubhani, Colombie , S , 10.1186/1752-0509-5-95BMC Syst. Biol. 595Respiratory Metabolism in PlantsBeevers, H. (1961). Respiratory Metabolism in Plants. New York: Harper and Row. Beurton-Aimar, M., Beauvoit, B., Monier, A., Vallee, F., Dieuaide- Noubhani, M., and Colombie, S. (2011). Comparison between ele- mentary flux modes analysis and 13C-metabolic fluxes measured in bacterial and plant cells. BMC Syst. Biol. 5, 95. doi: 10.1186/1752-0509- 5-95

Large-scale 13C-flux analysis reveals mechanistic principles of metabolic network robustness to null mutations in yeast. L Blank, L Kuepfer, U Sauer, Genome Biol. 649Blank, L., Kuepfer, L., and Sauer, U. (2005). Large-scale 13C-flux analy- sis reveals mechanistic principles of metabolic network robustness to null mutations in yeast. Genome Biol. 6, R49.

Flux balance analysis of primary metabolism in Chlamydomonas reinhardtii. N Boyle, Morgan , J , 10.1186/1752-0509-3-4BMC Syst. Biol. 34Boyle, N., and Morgan, J. (2009). Flux balance analysis of primary metabolism in Chlamydomonas rein- hardtii. BMC Syst. Biol. 3, 4. doi: 10.1186/1752-0509-3-4

What is systems biology. R Breitling, 10.3389/fphys.2010.00009Front. Physiol. 19Breitling, R. (2010). What is systems biology? Front. Physiol. 1:9. doi: 10.3389/fphys.2010.00009.

The hydrophobic proteome of mitochondrial membranes from Arabidopsis cell suspensions. S Brugiere, S Kowalski, M Ferro, D Seigneurin-Berny, S Miras, D Salvi, S Ravanel, P D&apos;herin, J Garin, J Bourguignon, J Joyard, Rolland , N , Phytochemistry. 65Brugiere, S., Kowalski, S., Ferro, M., Seigneurin-Berny, D., Miras, S., Salvi, D., Ravanel, S., D'Herin, P., Garin, J., Bourguignon, J., Joyard, J., and Rolland, N. (2004). The hydropho- bic proteome of mitochondrial mem- branes from Arabidopsis cell suspen- sions. Phytochemistry 65, 1693-1707.

Higher plant-like subunit composition of mitochondrial complex I from Chlamydomonas reinhardtii: 31 conserved components among eukaryotes. P Cardol, F Vanrobaeys, B Devreese, J Van Beeumen, R F Matagne, Remacle , C , Biochim. Biophys. Acta. 1658Cardol, P., Vanrobaeys, F., Devreese, B., Van Beeumen, J., Matagne, R. F., and Remacle, C. (2004). Higher plant-like subunit composition of mitochon- drial complex I from Chlamydomonas reinhardtii: 31 conserved compo- nents among eukaryotes. Biochim. Biophys. Acta 1658, 212-224.

Reduced expression of aconitase results in an enhanced rate of photosynthesis and marked shifts in carbon partitioning in illuminated leaves of wild species tomato. F Carrari, A Nunes-Nesi, Y Gibon, A Lytovchenko, M E Loureiro, A R Fernie, Plant Physiol. 133Carrari, F., Nunes-Nesi, A., Gibon, Y., Lytovchenko, A., Loureiro, M. E., and Fernie, A. R. (2003). Reduced expression of aconitase results in an enhanced rate of photosynthesis and marked shifts in carbon partition- ing in illuminated leaves of wild species tomato. Plant Physiol. 133, 1322-1335.

Synergy between 13C-metabolic flux analysis and flux balance analysis for understanding metabolic adaption. R L Chang, L Ghamsari, A Manichaikul, E F Y Hom, S Balaji, W Fu, Y Shen, T Hao, B O Palsson, K Salehi-Ashtiani, J A Papin, X Chen, A P Alonso, D K Allen, J L Reed, Y Shachar-Hill, Mol. Syst. Biol. 7Metab. Eng.Chang, R. L., Ghamsari, L., Manichaikul, A., Hom, E. F. Y., Balaji, S., Fu, W., Shen, Y., Hao, T., Palsson, B. O., Salehi-Ashtiani, K., and Papin, J. A. (2011). Metabolic network reconstruction of Chlamydomonas offers insight into light-driven algal metabolism. Mol. Syst. Biol. 7, 518. Chen, X., Alonso, A. P., Allen, D. K., Reed, J. L., and Shachar-Hill, Y. (2011). Synergy between 13C- metabolic flux analysis and flux balance analysis for understanding metabolic adaption to anaerobiosis in E. coli. Metab. Eng. 13, 38-48.

Transcriptional regulation in constraints-based metabolic models of Escherichia coli. M W Covert, B Ø Palsson, J. Biol. Chem. 277Covert, M. W., and Palsson, B. Ø. (2002). Transcriptional regulation in constraints-based metabolic models of Escherichia coli. J. Biol. Chem. 277, 28058-28064.

Understanding the regulation of aspartate metabolism using a model based on measured kinetic parameters. G Curien, O Bastien, M Robert-Genthon, A Cornish-Bowden, M L Cardenas, R Dumas, C G De Oliveira Dal&apos;molin, L.-E Quek, R Palfreyman, L Nielsen, 10.1186/1471-2164-12-S4-S5Mol. Syst. Biol. 5BMC GenomicsCurien, G., Bastien, O., Robert- Genthon, M., Cornish-Bowden, A., Cardenas, M. L., and Dumas, R. (2009). Understanding the regula- tion of aspartate metabolism using a model based on measured kinetic parameters. Mol. Syst. Biol. 5, 271. de Oliveira Dal'Molin, C. G., Quek, L.- E., Palfreyman, R., and Nielsen, L. (2011). AlgaGEM -a genome-scale metabolic reconstruction of algae based on the Chlamydomonas rein- hardtii genome. BMC Genomics 12, S5. doi: 10.1186/1471-2164-12-S4-S5

. C G De Oliveira Dal&apos;molin, L.-E Quek, R W Palfreyman, S M Brumbley, L K Nielsen, de Oliveira Dal'Molin, C. G., Quek, L.-E., Palfreyman, R. W., Brumbley, S. M., and Nielsen, L. K. (2010).

C4GEM, a genome-scale metabolic model to study C4 plant metabolism. Plant Physiol. 154C4GEM, a genome-scale metabolic model to study C4 plant metabolism. Plant Physiol. 154, 1871-1885.

Multiple lines of evidence localize signaling, morphology, and lipid biosynthesis machinery to the mitochondrial outer membrane of Arabidopsis. O Duncan, N L Taylor, C Carrie, H Eubel, S Kubiszewski-Jakubiak, B Zhang, R Narsai, A H Millar, J Whelan, Plant Physiol. 157Duncan, O., Taylor, N. L., Carrie, C., Eubel, H., Kubiszewski-Jakubiak, S., Zhang, B., Narsai, R., Millar, A. H., and Whelan, J. (2011). Multiple lines of evidence localize signaling, morphology, and lipid biosynthe- sis machinery to the mitochondrial outer membrane of Arabidopsis. Plant Physiol. 157, 1093-1113.

Genome-scale models of bacterial metabolism: reconstruction and applications. M Durot, P.-Y Bourguignon, V Schachter, FEMS Microbiol. Rev. 33Durot, M., Bourguignon, P.-Y., and Schachter, V. (2009). Genome-scale models of bacterial metabolism: reconstruction and applications. FEMS Microbiol. Rev. 33, 164-190.

Mitochondriadriven changes in leaf NAD status exert a crucial influence on the control of nitrate assimilation and the integration of carbon and nitrogen metabolism. C Dutilleul, C Lelarge, J.-L Prioul, R De Paepe, C H Foyer, G Noctor, Plant Physiol. 139Dutilleul, C., Lelarge, C., Prioul, J.- L., De Paepe, R., Foyer, C. H., and Noctor, G. (2005). Mitochondria- driven changes in leaf NAD sta- tus exert a crucial influence on the control of nitrate assimilation and the integration of carbon and nitro- gen metabolism. Plant Physiol. 139, 64-78.

Systems properties of the Haemophilus influenzae Rd metabolic genotype. J S Edwards, B O Palsson, J. Biol. Chem. 274Edwards, J. S., and Palsson, B. O. (1999). Systems properties of the Haemophilus influenzae Rd metabolic genotype. J. Biol. Chem. 274, 17410- 17416.

Two D-2-hydroxy-acid dehydrogenases in Arabidopsis thaliana with catalytic capacities to participate in the last reactions of the methylglyoxal and βoxidation pathways. M Engqvist, M F Drincovich, U.-I Flügge, V G Maurino, J. Biol. Chem. 284Engqvist, M., Drincovich, M. F., Flügge, U.-I., and Maurino, V. G. (2009). Two D-2-hydroxy-acid dehydrogenases in Arabidopsis thaliana with catalytic capacities to participate in the last reactions of the methylglyoxal and β- oxidation pathways. J. Biol. Chem. 284, 25026-25037.

Generation of hydrogen peroxide in chloroplasts of Arabidopsis overexpressing glycolate oxidase as an inducible system to study oxidative stress. H Fahnenstich, T E Scarpeci, E M Valle, U.-I Fluegge, V G Maurino, Plant Physiol. 148Fahnenstich, H., Scarpeci, T. E., Valle, E. M., Fluegge, U.-I., and Maurino, V. G. (2008). Generation of hydrogen peroxide in chloroplasts of Arabidop- sis overexpressing glycolate oxidase as an inducible system to study oxidative stress. Plant Physiol. 148, 719-729.

Highway or byway: the metabolic role of the GABA shunt in plants. A Fait, H Fromm, D Walter, G Galili, A R Fernie, Trends Plant Sci. 13Fait, A., Fromm, H., Walter, D., Galili, G., and Fernie, A. R. (2008). Highway or byway: the metabolic role of the GABA shunt in plants. Trends Plant Sci. 13, 14-19.

Building and analysing genome-scale metabolic models. D A Fell, M G Poolman, A Gevorgyan, Biochem. Soc. Trans. 38Fell, D. A., Poolman, M. G., and Gevorgyan, A. (2010). Building and analysing genome-scale metabolic models. Biochem. Soc. Trans. 38, 1197-1201.

Grand challenges in plant systems biology: closing the circle(s). Front. A Fernie, 10.3389/fpls.2012.00035Plant Sci. 335Fernie, A. (2012). Grand challenges in plant systems biology: closing the cir- cle(s). Front. Plant Sci. 3:35. doi: 10.3389/fpls.2012.00035

Respiratory metabolism: glycolysis, the TCA cycle and mitochondrial electron transport. A R Fernie, F Carrari, L J Sweetlove, Curr. Opin. Plant Biol. 7Fernie, A. R., Carrari, F., and Sweetlove, L. J. (2004). Respiratory metabolism: glycolysis, the TCA cycle and mito- chondrial electron transport. Curr. Opin. Plant Biol. 7, 254-261.

Metabolomics-assisted breeding: a viable option for crop improvement?. A R Fernie, N Schauer, Trends Genet. 25Fernie, A. R., and Schauer, N. (2009). Metabolomics-assisted breeding: a viable option for crop improvement? Trends Genet. 25, 39-48.

On the discordance of metabolomics with proteomics and transcriptomics: coping with increasing complexity in logic, chemistry, and network interactions. A R Fernie, M Stitt, Plant Physiol. 158Fernie, A. R., and Stitt, M. (2012). On the discordance of metabolomics with proteomics and transcrip- tomics: coping with increasing com- plexity in logic, chemistry, and net- work interactions. Plant Physiol. 158, 1139-1145.

Metabolite profiling for plant functional genomics. O Fiehn, J Kopka, P Dörmann, T Altmann, R N Trethewey, L Willmitzer, E Fischer, U Sauer, Nat. Biotechnol. 18Fiehn, O., Kopka, J., Dörmann, P., Altmann, T., Trethewey, R. N., and Willmitzer, L. (2000). Metabo- lite profiling for plant functional genomics. Nat. Biotechnol. 18, 1157. Fischer, E., and Sauer, U. (2005).

Large-scale in vivo flux analysis shows rigidity and suboptimal performance of Bacillus subtilis metabolism. Nat. Genet. 37Large-scale in vivo flux analysis shows rigidity and suboptimal performance of Bacillus subtilis metabolism. Nat. Genet. 37, 636-640.

Respiration and nitrogen assimilation: targeting mitochondria-associated metabolism as a means to enhance nitrogen use efficiency. C H Foyer, G Noctor, M Hodges, J. Exp. Bot. 62Foyer, C. H., Noctor, G., and Hodges, M. (2011). Respiration and nitrogen assimilation: tar- geting mitochondria-associated metabolism as a means to enhance nitrogen use efficiency. J. Exp. Bot. 62, 1467-1482.

Impact of clock-associated Arabidopsis pseudo-response regulators in metabolic coordination. A Fukushima, M Kusano, N Nakamichi, M Kobayashi, N Hayashi, H Sakakibara, T Mizuno, K Saito, Proc. Nat. Acad. Sci. U.S.A. 106Fukushima, A., Kusano, M., Nakamichi, N., Kobayashi, M., Hayashi, N., Sakakibara, H., Mizuno, T., and Saito, K. (2009). Impact of clock-associated Arabidopsis pseudo-response regu- lators in metabolic coordination. Proc. Nat. Acad. Sci. U.S.A. 106, 7251-7256.

Complex I dysfunction redirects cellular and mitochondrial metabolism in Arabidopsis. M Garmier, A J Carroll, E Delannoy, C Vallet, D A Day, I D Small, A H Millar, Plant Physiol. 148Garmier, M., Carroll, A. J., Delannoy, E., Vallet, C., Day, D. A., Small, I. D., and Millar, A. H. (2008). Complex I dys- function redirects cellular and mito- chondrial metabolism in Arabidopsis. Plant Physiol. 148, 1324-1341.

The Absence of ALTER-NATIVE OXIDASE1a in Arabidopsis results in acute sensitivity to combined light and drought stress. E Giraud, L H M Ho, R Clifton, A Carroll, G Estavillo, Y.-F Tan, K A Howell, A Ivanova, B J Pogson, A H Millar, J Whelan, Plant Physiol. 147Giraud, E., Ho, L. H. M., Clifton, R., Carroll, A., Estavillo, G., Tan, Y.-F., Howell, K. A., Ivanova, A., Pogson, B. J., Millar, A. H., and Whelan, J. (2008). The Absence of ALTER- NATIVE OXIDASE1a in Arabidopsis results in acute sensitivity to com- bined light and drought stress. Plant Physiol. 147, 595-610.

Salinity stress adaptation competence in the extremophile Thellungiella halophila in comparison with its relative Arabidopsis thaliana. Q Gong, P Li, S Ma, S Indu Rupassara, H J Bohnert, Plant J. 44Gong, Q., Li, P., Ma, S., Indu Rupas- sara, S., and Bohnert, H. J. (2005). Salinity stress adaptation competence in the extremophile Thellungiella halophila in comparison with its rel- ative Arabidopsis thaliana. Plant J. 44, 826-839.

Flux balance analysis of barley seeds: a computational approach to study systemic properties of central metabolism. E Grafahrend-Belau, F Schreiber, D Koschützki, B H Junker, Plant Physiol. 149Grafahrend-Belau, E., Schreiber, F., Koschützki, D., and Junker, B. H. (2009). Flux balance analysis of bar- ley seeds: a computational approach to study systemic properties of cen- tral metabolism. Plant Physiol. 149, 585-598.

Plant metabolomics coming of age. C Guy, J Kopka, T Moritz, Physiol. Plant. 132Guy, C., Kopka, J., and Moritz, T. (2008). Plant metabolomics coming of age. Physiol. Plant. 132, 113-116.

Combined transcript and metabolite profiling of Arabidopsis grown under widely variant growth conditions facilitates the identification of novel metabolitemediated regulation of gene expression. M A Hannah, C Caldana, D Steinhauser, I Balbo, A R Fernie, L Willmitzer, Plant Physiol. 152Hannah, M. A., Caldana, C., Stein- hauser, D., Balbo, I., Fernie, A. R., and Willmitzer, L. (2010). Combined transcript and metabolite profiling of Arabidopsis grown under widely vari- ant growth conditions facilitates the identification of novel metabolite- mediated regulation of gene expres- sion. Plant Physiol. 152, 2120-2129.

Metabolic turnover analysis by a combination of in vivo 13C-labelling from 13CO2 and metabolic profiling with CE-MS/MS reveals rate-limiting steps of the C3 photosynthetic pathway in Nicotiana tabacum leaves. T Hasunuma, K Harada, S.-I Miyazawa, A Kondo, E Fukusaki, C Miyake, J. Exp. Bot. 61Hasunuma, T., Harada, K., Miyazawa, S.-I., Kondo, A., Fukusaki, E., and Miyake, C. (2010). Metabolic turnover analysis by a combination of in vivo 13C-labelling from 13CO2 and metabolic profiling with CE- MS/MS reveals rate-limiting steps of the C3 photosynthetic pathway in Nicotiana tabacum leaves. J. Exp. Bot. 61, 1041-1051.

Metabolic network reconstruction and flux variability analysis of storage synthesis in developing oilseed rape (Brassica napus L.) embryos. J Hay, J Schwender, Plant J. 67Hay, J., and Schwender, J. (2011). Metabolic network reconstruction and flux variability analysis of storage synthesis in developing oilseed rape (Brassica napus L.) embryos. Plant J. 67, 526-541.

Mitochondrial complex I from Arabidopsis and rice: orthologs of mammalian and fungal components coupled with plantspecific subunits. J L Heazlewood, K A Howell, A H Millar, Biochim. Biophys. Acta. 1604Heazlewood, J. L., Howell, K. A., and Millar, A. H. (2003a). Mitochondrial complex I from Arabidopsis and rice: orthologs of mammalian and fun- gal components coupled with plant- specific subunits. Biochim. Biophys. Acta 1604, 159-169.

. J L Heazlewood, K A Howell, J Whelan, A H Millar, Heazlewood, J. L., Howell, K. A., Whe- lan, J., and Millar, A. H. (2003b).

Towards an analysis of the rice mitochondrial proteome. Plant Physiol. 132Towards an analysis of the rice mito- chondrial proteome. Plant Physiol. 132, 230-242.

Experimental analysis of the Arabidopsis mitochondrial proteome highlights signaling and regulatory components, provides assessment of targeting prediction programs, and indicates plant-specific mitochondrial proteins. J L Heazlewood, J S Tonti-Filippini, A M Gout, D A Day, J Whelan, A H Millar, Plant Cell. 16Heazlewood, J. L., Tonti-Filippini, J. S., Gout, A. M., Day, D. A., Whelan, J., and Millar, A. H. (2004). Exper- imental analysis of the Arabidop- sis mitochondrial proteome high- lights signaling and regulatory com- ponents, provides assessment of tar- geting prediction programs, and indicates plant-specific mitochon- drial proteins. Plant Cell 16, 241-256.

Purification of the NADH:ubiquinone oxidoreductase (complex I) of the respiratory chain from the inner mitochondrial membrane of Solanum tuberosum. U Herz, W Schröder, A Liddell, C J Leaver, A Brennicke, L Grohmann, J. Biol. Chem. 269Herz, U., Schröder, W., Liddell, A., Leaver, C. J., Brennicke, A., and Grohmann, L. (1994). Purification of the NADH:ubiquinone oxidoreduc- tase (complex I) of the respiratory chain from the inner mitochondrial membrane of Solanum tuberosum. J. Biol. Chem. 269, 2263-2269.

Mapping metabolic and transcript temporal switches during germination in rice highlights specific transcription factors and the role of RNA instability in the germination process. K A Howell, R Narsai, A Carroll, A Ivanova, M Lohse, B Usadel, A H Millar, J Whelan, Plant Physiol. 149Howell, K. A., Narsai, R., Carroll, A., Ivanova, A., Lohse, M., Usadel, B., Millar, A. H., and Whelan, J. (2009). Mapping metabolic and transcript temporal switches during germina- tion in rice highlights specific tran- scription factors and the role of RNA instability in the germination process. Plant Physiol. 149, 961-980.

Experimental analysis of the rice mitochondrial proteome, its biogenesis, and heterogeneity. S Huang, N L Taylor, R Narsai, H Eubel, J Whelan, A H Millar, Plant Physiol. 149Huang, S., Taylor, N. L., Narsai, R., Eubel, H., Whelan, J., and Millar, A. H. (2009). Experimental analysis of the rice mitochondrial proteome, its biogenesis, and heterogeneity. Plant Physiol. 149, 719-734.

A new approach to decoding life: systems biology. T Ideker, T Galitski, L Hood, Ann. Rev. Genomics Hum. Genet. 2Ideker, T., Galitski, T., and Hood, L. (2001). A new approach to decod- ing life: systems biology. Ann. Rev. Genomics Hum. Genet. 2, 343-372.

The critical role of Arabidopsis electrontransfer flavoprotein:ubiquinone oxidoreductase during darkinduced starvation. K Ishizaki, T R Larson, N Schauer, A R Fernie, I A Graham, C J Leaver, Plant Cell. 17Ishizaki, K., Larson, T. R., Schauer, N., Fernie, A. R., Graham, I. A., and Leaver, C. J. (2005). The critical role of Arabidopsis electron- transfer flavoprotein:ubiquinone oxidoreductase during dark- induced starvation. Plant Cell 17, 2587-2600.

The mitochondrial electron transfer flavoprotein complex is essential for survival of Arabidopsis in extended darkness. K Ishizaki, N Schauer, T R Larson, I A Graham, A R Fernie, C J Leaver, Plant J. 47Ishizaki, K., Schauer, N., Larson, T. R., Graham, I. A., Fernie, A. R., and Leaver, C. J. (2006). The mito- chondrial electron transfer flavopro- tein complex is essential for survival of Arabidopsis in extended darkness. Plant J. 47, 751-760.

Exploring the temperaturestress metabolome of Arabidopsis. F Kaplan, J Kopka, D W Haskell, W Zhao, K C Schiller, N Gatzke, D Y Sung, C L Guy, Plant Physiol. 136Kaplan, F., Kopka, J., Haskell, D. W., Zhao, W., Schiller, K. C., Gatzke, N., Sung, D. Y., and Guy, C. L. (2004). Exploring the temperature- stress metabolome of Arabidopsis. Plant Physiol. 136, 4159-4168.

Transcript and metabolite profiling during cold acclimation of Arabidopsis reveals an intricate relationship of cold-regulated gene expression with modifications in metabolite content. F Kaplan, J Kopka, D Y Sung, W Zhao, M Popp, R Porat, C L Guy, Plant J. 50Kaplan, F., Kopka, J., Sung, D. Y., Zhao, W., Popp, M., Porat, R., and Guy, C. L. (2007). Transcript and metabo- lite profiling during cold acclima- tion of Arabidopsis reveals an intri- cate relationship of cold-regulated gene expression with modifications in metabolite content. Plant J. 50, 967-981.

Towards an 'oceans systems biology. E Karsenti, Mol. Syst. Biol. 8575Karsenti, E. (2012). Towards an 'oceans systems biology.' Mol. Syst. Biol. 8, 575.

Biological robustness. H Kitano, Nat. Rev. Genet. 5Kitano, H. (2004). Biological robust- ness. Nat. Rev. Genet. 5, 826-837.

Modelbased confirmation of alternative substrates of mitochondrial electron transport chain. S Kleessen, W L Araújo, A R Fernie, Z Nikoloski, J. Biol. Chem. 287Kleessen, S., Araújo, W. L., Fernie, A. R., and Nikoloski, Z. (2012). Model- based confirmation of alternative substrates of mitochondrial electron transport chain. J. Biol. Chem. 287, 11122-11131.

Proteomic approach to characterize mitochondrial complex I from plants. J Klodmann, H.-P Braun, Phytochemistry. 72Klodmann, J., and Braun, H.-P. (2011). Proteomic approach to characterize mitochondrial complex I from plants. Phytochemistry 72, 1071-1080.

Defining the protein complex proteome of plant mitochondria. J Klodmann, M Senkler, C Rode, H.-P Braun, Plant Physiol. 157Klodmann, J., Senkler, M., Rode, C., and Braun, H.-P. (2011). Defining the protein complex proteome of plant mitochondria. Plant Physiol. 157, 587-598.

Internal architecture of mitochondrial complex I from Arabidopsis thaliana. J Klodmann, S Sunderhaus, M Nimtz, L Jänsch, H.-P Braun, Plant Cell. 22Klodmann, J., Sunderhaus, S., Nimtz, M., Jänsch, L., and Braun, H.-P. (2010). Internal architecture of mito- chondrial complex I from Arabidopsis thaliana. Plant Cell 22, 797-810.

. J Kreuzwieser, J Hauberg, K A Howell, A Carroll, H Rennenberg, A H Millar, J Whelan, Kreuzwieser, J., Hauberg, J., Howell, K. A., Carroll, A., Rennenberg, H., Mil- lar, A. H., and Whelan, J. (2009).

Differential response of gray poplar leaves and roots underpins stress adaptation during hypoxia. Plant Physiol. 149Differential response of gray poplar leaves and roots underpins stress adaptation during hypoxia. Plant Physiol. 149, 461-473.

Respiration during photosynthesis. S Krömer, Annu. Rev. Plant Physiol. Plant Mol. Biol. 46Krömer, S. (1995). Respiration dur- ing photosynthesis. Annu. Rev. Plant Physiol. Plant Mol. Biol. 46, 45-70.

Proteomic approach to identify novel mitochondrial proteins in Arabidopsis. V Kruft, H Eubel, L Jänsch, W Werhahn, H.-P Braun, Plant Physiol. 127Kruft, V., Eubel, H., Jänsch, L., Werhahn, W., and Braun, H.- P. (2001). Proteomic approach to identify novel mitochondrial proteins in Arabidopsis. Plant Physiol. 127, 1694-1710.

Development of a kinetic metabolic model: application to Catharanthus roseus hairy root. M Leduc, C Tikhomiroff, M Cloutier, M Perrier, M Jolicoeur, Bioprocess Biosyst. Eng. 28Leduc, M., Tikhomiroff, C., Cloutier, M., Perrier, M., and Jolicoeur, M. (2006). Development of a kinetic metabolic model: application to Catharanthus roseus hairy root. Bio- process Biosyst. Eng. 28, 295-313.

Diurnal changes in mitochondrial function reveal daily optimization of light and dark respiratory metabolism in Arabidopsis. C P Lee, H Eubel, A H Millar, Mol. Cell. Proteomics. 9Lee, C. P., Eubel, H., and Millar, A. H. (2010). Diurnal changes in mito- chondrial function reveal daily opti- mization of light and dark respiratory metabolism in Arabidopsis. Mol. Cell. Proteomics 9, 2125-2139.

Heterogeneity of the mitochondrial proteome for photosynthetic and non-photosynthetic Arabidopsis metabolism. C P Lee, H Eubel, N O&apos;toole, A H Millar, Mol. Cell. Proteomics. 7Lee, C. P., Eubel, H., O'Toole, N., and Millar, A. H. (2008). Heterogeneity of the mitochondrial proteome for pho- tosynthetic and non-photosynthetic Arabidopsis metabolism. Mol. Cell. Proteomics 7, 1297-1316.

Combining proteomics of root and shoot mitochondria and transcript analysis to define constitutive and variable components in plant mitochondria. C P Lee, H Eubel, N O&apos;toole, A H Millar, Phytochemistry. 72Lee, C. P., Eubel, H., O'Toole, N., and Millar, A. H. (2011). Combin- ing proteomics of root and shoot mitochondria and transcript analy- sis to define constitutive and variable components in plant mitochondria. Phytochemistry 72, 1092-1108.

Metabolic recovery of Arabidopsis thaliana roots following cessation of oxidative stress. M Lehmann, M Laxa, L Sweetlove, A Fernie, T Obata, Metabolomics. 8Lehmann, M., Laxa, M., Sweetlove, L., Fernie, A., and Obata, T. (2012). Metabolic recovery of Arabidopsis thaliana roots following cessation of oxidative stress. Metabolomics 8, 143-153.

The metabolic response of Arabidopsis roots to oxidative stress is distinct from that of heterotrophic cells in culture and highlights a complex relationship between the levels of transcripts, metabolites, and flux. M Lehmann, M Schwarzländer, T Obata, S Sirikantaramas, M Burow, C E Olsen, T Tohge, M D Fricker, B L Møller, A R Fernie, L J Sweetlove, M Laxa, Mol. Plant. 2Lehmann, M., Schwarzländer, M., Obata, T., Sirikantaramas, S., Burow, M., Olsen, C. E., Tohge, T., Fricker, M. D., Møller, B. L., Fernie, A. R., Sweetlove, L. J., and Laxa, M. (2009). The metabolic response of Arabidopsis roots to oxidative stress is distinct from that of heterotrophic cells in culture and highlights a com- plex relationship between the levels of transcripts, metabolites, and flux. Mol. Plant 2, 390-406.

Purification and preliminary characterization of mitochondrial complex I (NADH:Ubiquinone reductase) from broad bean (Vicia faba L.). S Leterme, M Boutry, Plant Physiol. 102Leterme, S., and Boutry, M. (1993). Purification and preliminary char- acterization of mitochondrial com- plex I (NADH:Ubiquinone reduc- tase) from broad bean (Vicia faba L.). Plant Physiol. 102, 435-443.

Intracellular metabolite transporters in plants. N Linka, A P M Weber, Mol. Plant. 3Linka, N., and Weber, A. P. M. (2010). Intracellular metabolite transporters in plants. Mol. Plant 3, 21-53.

Gas chromatography mass spectrometry-based metabolite profiling in plants. J Lisec, N Schauer, J Kopka, L Willmitzer, A R Fernie, Nat. Protoc. 1Lisec, J., Schauer, N., Kopka, J., Willmitzer, L., and Fernie, A. R. (2006). Gas chromatography mass spectrometry-based metabolite pro- filing in plants. Nat. Protoc. 1, 387-396.

A comparison of RNA-Seq and high-density exon array for detecting differential gene expression between closely related species. S Liu, L Lin, P Jiang, D Wang, Y Xing, Nucleic Acids Res. 39Liu, S., Lin, L., Jiang, P., Wang, D., and Xing, Y. (2011). A compari- son of RNA-Seq and high-density exon array for detecting differen- tial gene expression between closely related species. Nucleic Acids Res. 39, 578-588.

Analysis of metabolic flux phenotypes for two Arabidopsis mutants with severe impairment in seed storage lipid synthesis. J Lonien, J Schwender, Plant Physiol. 151Lonien, J., and Schwender, J. (2009). Analysis of metabolic flux pheno- types for two Arabidopsis mutants with severe impairment in seed stor- age lipid synthesis. Plant Physiol. 151, 1617-1634.

Sugar-induced increases in trehalose 6-phosphate are correlated with redox activation of ADPglucose pyrophosphorylase and higher rates of starch synthesis in Arabidopsis thaliana. J E Lunn, R Feil, J H M Hendriks, Y Gibon, R Morcuende, D Osuna, W.-R Scheible, P Carillo, M.-R Hajirezaei, M Stitt, Biochem. J. 397Lunn, J. E., Feil, R., Hendriks, J. H. M., Gibon, Y., Morcuende, R., Osuna, D., Scheible, W.-R., Carillo, P., Hajirezaei, M.-R., and Stitt, M. (2006). Sugar-induced increases in trehalose 6-phosphate are correlated with redox activation of ADPglu- cose pyrophosphorylase and higher rates of starch synthesis in Ara- bidopsis thaliana. Biochem. J. 397, 139-148.

Simultaneous determination of multiple intracellular metabolites in glycolysis, pentose phosphate pathway and tricarboxylic acid cycle by liquid chromatography-mass spectrometry. B Luo, K Groenke, R Takors, C Wandrey, M Oldiges, J. Chromatogr. A. 1147Luo, B., Groenke, K., Takors, R., Wan- drey, C., and Oldiges, M. (2007). Simultaneous determination of mul- tiple intracellular metabolites in gly- colysis, pentose phosphate path- way and tricarboxylic acid cycle by liquid chromatography-mass spec- trometry. J. Chromatogr. A 1147, 153-164.

Mitochondrial and nuclear localization of a novel pea thioredoxin: identification of its mitochondrial target proteins. M C Martí, E Olmos, J J Calvete, I Díaz, S Barranco-Medina, J Whelan, J J Lázaro, F Sevilla, Jiménez , A , Plant Physiol. 150Martí, M. C., Olmos, E., Calvete, J. J., Díaz, I., Barranco-Medina, S., Whe- lan, J., Lázaro, J. J., Sevilla, F., and Jiménez, A. (2009). Mitochondrial and nuclear localization of a novel pea thioredoxin: identification of its mitochondrial target proteins. Plant Physiol. 150, 646-657.

Next-generation transcriptome assembly. J A Martin, Wang , Z , Nat. Rev. Genet. 12Martin, J. A., and Wang, Z. (2011). Next-generation transcriptome as- sembly. Nat. Rev. Genet. 12, 671-682.

Remodeled respiration in ndufs4 with low phosphorylation efficiency suppresses Arabidopsis germination and growth and alters control of metabolism at night. E H Meyer, T Tomaz, A J Carroll, G Estavillo, E Delannoy, S K Tanz, I D Small, B J Pogson, A H Millar, Plant Physiol. 151Meyer, E. H., Tomaz, T., Carroll, A. J., Estavillo, G., Delannoy, E., Tanz, S. K., Small, I. D., Pogson, B. J., and Millar, A. H. (2009). Remodeled respiration in ndufs4 with low phosphorylation efficiency suppresses Arabidopsis ger- mination and growth and alters con- trol of metabolism at night. Plant Physiol. 151, 603-619.

The plant mitochondrial proteome. A H Millar, J L Heazlewood, B K Kristensen, H.-P Braun, I M Møller, Trends Plant Sci. 10Millar, A. H., Heazlewood, J. L., Kris- tensen, B. K., Braun, H.-P., and Møller, I. M. (2005). The plant mito- chondrial proteome. Trends Plant Sci. 10, 36-43.

Analysis of the Arabidopsis mitochondrial proteome. A H Millar, L J Sweetlove, P Giegé, C J Leaver, Plant Physiol. 127Millar, A. H., Sweetlove, L. J., Giegé, P., and Leaver, C. J. (2001). Anal- ysis of the Arabidopsis mitochon- drial proteome. Plant Physiol. 127, 1711-1727.

Organization and regulation of mitochondrial respiration in plants. A H Millar, J Whelan, K L Soole, D A Day, Annu. Rev. Plant Biol. 62Millar, A. H., Whelan, J., Soole, K. L., and Day, D. A. (2011). Organization and regulation of mitochondrial res- piration in plants. Annu. Rev. Plant Biol. 62, 79-104.

Reconstruction of Arabidopsis metabolic network models accounting for subcellular compartmentalization and tissue-specificity. S Mintz-Oron, S Meir, S Malitsky, E Ruppin, A Aharoni, T Shlomi, Mintz-Oron, S., Meir, S., Malitsky, S., Ruppin, E., Aharoni, A., and Shlomi, T. (2012). Reconstruction of Ara- bidopsis metabolic network models accounting for subcellular compart- mentalization and tissue-specificity.

. Proc. Natl. Acad. Sci. U.S.A. 109Proc. Natl. Acad. Sci. U.S.A. 109, 339-344.

ROS signalling -specificity is required. I M Møller, L J Sweetlove, Trends Plant Sci. 15Møller, I. M., and Sweetlove, L. J. (2010). ROS signalling -specificity is required. Trends Plant Sci. 15, 370-374.

Decrease in manganese superoxide dismutase leads to reduced root growth and affects tricarboxylic acid cycle flux and mitochondrial redox homeostasis. M J Morgan, M Lehmann, M Schwarzlander, C J Baxter, A Sienkiewicz-Porzucek, T C R Williams, N Schauer, A R Fernie, M D Fricker, R G Ratcliffe, L J Sweetlove, I Finkemeier, Plant Physiol. 147Morgan, M. J., Lehmann, M., Schwarz- lander, M., Baxter, C. J., Sienkiewicz- Porzucek, A., Williams, T. C. R., Schauer, N., Fernie, A. R., Fricker, M. D., Ratcliffe, R. G., Sweetlove, L. J., and Finkemeier, I. (2008). Decrease in manganese superoxide dismutase leads to reduced root growth and affects tricarboxylic acid cycle flux and mitochondrial redox homeostasis. Plant Physiol. 147, 101-114.

Linking high-resolution metabolic flux phenotypes and transcriptional regulation in yeast modulated by the global regulator Gcn4p. J F Moxley, M C Jewett, M R Antoniewicz, S G Villas-Boas, H Alper, R T Wheeler, L Tong, A G Hinnebusch, T Ideker, J Nielsen, G Stephanopoulos, Proc. Natl. Acad. Sci. U.S.A. 106Moxley, J. F., Jewett, M. C., Antoniewicz, M. R., Villas-Boas, S. G., Alper, H., Wheeler, R. T., Tong, L., Hinneb- usch, A. G., Ideker, T., Nielsen, J., and Stephanopoulos, G. (2009). Linking high-resolution metabolic flux phe- notypes and transcriptional regula- tion in yeast modulated by the global regulator Gcn4p. Proc. Natl. Acad. Sci. U.S.A. 106, 6477-6482.

Defining core metabolic and transcriptomic responses to oxygen availability in rice embryos and young seedlings. R Narsai, K A Howell, A Carroll, A Ivanova, A H Millar, J Whelan, Plant Physiol. 151Narsai, R., Howell, K. A., Carroll, A., Ivanova, A., Millar, A. H., and Whelan, J. (2009). Defining core metabolic and transcriptomic responses to oxygen availability in rice embryos and young seedlings. Plant Physiol. 151, 306-322.

Interaction between photosynthesis and respiration in illuminated leaves. K Noguchi, Yoshida , K , Mitochondrion. 8Noguchi, K., and Yoshida, K. (2008). Interaction between photosynthesis and respiration in illuminated leaves. Mitochondrion 8, 87-99.

Targeting mitochondrial metabolism and machinery as a means to enhance photosynthesis. A Nunes-Nesi, W L Araújo, A R Fernie, Plant Physiol. 155Nunes-Nesi, A., Araújo, W. L., and Fernie, A. R. (2011). Targeting mito- chondrial metabolism and machin- ery as a means to enhance photosyn- thesis. Plant Physiol. 155, 101-107.

Deficiency of mitochondrial fumarase activity in tomato plants impairs photosynthesis via an effect on stomatal function. A Nunes-Nesi, F Carrari, Y Gibon, R Sulpice, A Lytovchenko, J Fisahn, J Graham, R G Ratcliffe, L J Sweetlove, A R Fernie, Plant J. 50Nunes-Nesi, A., Carrari, F., Gibon, Y., Sulpice, R., Lytovchenko, A., Fisahn, J., Graham, J., Ratcliffe, R. G., Sweetlove, L. J., and Fernie, A. R. (2007). Deficiency of mitochondrial fumarase activity in tomato plants impairs photosynthesis via an effect on stomatal function. Plant J. 50, 1093-1106.

Enhanced photosynthetic performance and growth as a consequence of decreasing mitochondrial malate dehydrogenase activity in transgenic tomato plants. A Nunes-Nesi, F Carrari, A Lytovchenko, A M O Smith, M E Loureiro, R G Ratcliffe, L J Sweetlove, A R Fernie, Plant Physiol. 137Nunes-Nesi, A., Carrari, F., Lytovchenko, A., Smith, A. M. O., Loureiro, M. E., Ratcliffe, R. G., Sweetlove, L. J., and Fernie, A. R. (2005). Enhanced photosynthetic performance and growth as a conse- quence of decreasing mitochondrial malate dehydrogenase activity in transgenic tomato plants. Plant Physiol. 137, 611-622.

The enigmatic contribution of mitochondrial function in photosynthesis. A Nunes-Nesi, R Sulpice, Y Gibon, A R Fernie, J. Exp. Bot. 59Nunes-Nesi, A., Sulpice, R., Gibon, Y., and Fernie, A. R. (2008). The enig- matic contribution of mitochondrial function in photosynthesis. J. Exp. Bot. 59, 1675-1684.

Systematic functional analysis of the yeast genome. S G Oliver, M K Winson, D B Kell, F Baganz, Trends Biotechnol. 16Oliver, S. G., Winson, M. K., Kell, D. B., and Baganz, F. (1998). Sys- tematic functional analysis of the yeast genome. Trends Biotechnol. 16, 373-378.

Life's complexity pyramid. Z N Oltvai, A.-L Barabási, Science. 298Oltvai, Z. N., and Barabási, A.-L. (2002). Life's complexity pyramid. Science 298, 763-764.

Profiling primary metabolites of tomato fruit with gas chromatography/mass spectrometry. S Osorio, P T Do, A R Fernie, Methods Mol. Biol. 860Osorio, S., Do, P. T., and Fernie, A. R. (2012). Profiling primary metabolites of tomato fruit with gas chromatog- raphy/mass spectrometry. Methods Mol. Biol. 860, 101-109.

Evolution, structure and function of mitochondrial carriers: a review with new insights. F Palmieri, C L Pierri, A De Grassi, A Nunes-Nesi, A R Fernie, Plant J. 66Palmieri, F., Pierri, C. L., De Grassi, A., Nunes-Nesi, A., and Fernie, A. R. (2011). Evolution, structure and function of mitochondrial carriers: a review with new insights. Plant J. 66, 161-181.

Disruption of a nuclear gene encoding a mitochondrial gamma carbonic anhydrase reduces complex I and supercomplex I + III2 levels and alters mitochondrial physiology in Arabidopsis. M Perales, H Eubel, J Heinemeyer, A Colaneri, E Zabaleta, H P Braun, J. Mol. Biol. 350Perales, M., Eubel, H., Heinemeyer, J., Colaneri, A., Zabaleta, E., and Braun, H. P. (2005). Disruption of a nuclear gene encoding a mitochon- drial gamma carbonic anhydrase reduces complex I and supercomplex I + III2 levels and alters mitochon- drial physiology in Arabidopsis. J. Mol. Biol. 350, 263-277.

A genomescale metabolic model of Arabidopsis and some of its properties. M G Poolman, L Miguet, L J Sweetlove, D A Fell, Plant Physiol. 151Poolman, M. G., Miguet, L., Sweetlove, L. J., and Fell, D. A. (2009). A genome- scale metabolic model of Arabidopsis and some of its properties. Plant Physiol. 151, 1570-1581.

Genomescale microbial in silico models: the constraints-based approach. N D Price, J A Papin, C H Schilling, B O Palsson, Trends Biotechnol. 21Price, N. D., Papin, J. A., Schilling, C. H., and Palsson, B. O. (2003). Genome- scale microbial in silico models: the constraints-based approach. Trends Biotechnol. 21, 162-169.

Beneficial interactions of mitochondrial metabolism with photosynthetic carbon assimilation. A S Raghavendra, K Padmasree, Trends Plant Sci. 8Raghavendra, A. S., and Padmasree, K. (2003). Beneficial interactions of mitochondrial metabolism with photosynthetic carbon assimilation. Trends Plant Sci. 8, 546-553.

The composition of plant mitochondrial supercomplexes changes with oxygen availability. S J Ramírez-Aguilar, M Keuthe, M Rocha, V V Fedyaev, K Kramp, K J Gupta, A G Rasmusson, W X Schulze, J T Van Dongen, J. Biol. Chem. 286Ramírez-Aguilar, S. J., Keuthe, M., Rocha, M., Fedyaev, V. V., Kramp, K., Gupta, K. J., Rasmusson, A. G., Schulze, W. X., and Van Don- gen, J. T. (2011). The composition of plant mitochondrial supercomplexes changes with oxygen availability. J. Biol. Chem. 286, 43045-43053.

Measuring multiple fluxes through plant metabolic networks. R G Ratcliffe, Y Shachar-Hill, Plant J. 45Ratcliffe, R. G., and Shachar-Hill, Y. (2006). Measuring multiple fluxes through plant metabolic networks. Plant J. 45, 490-511.

Metabolic profiling allows comprehensive phenotyping of genetically or environmentally modified plant systems. U Roessner, A Luedemann, D Brust, O Fiehn, T Linke, L Willmitzer, A R Fernie, Plant Cell. 13Roessner, U., Luedemann, A., Brust, D., Fiehn, O., Linke, T., Willmitzer, L., and Fernie, A. R. (2001). Metabolic profiling allows comprehensive phe- notyping of genetically or environ- mentally modified plant systems. Plant Cell 13, 11-29.

Kinetic modelling of plant metabolic pathways. J M Rohwer, J. Exp. Bot. 63Rohwer, J. M. (2012). Kinetic modelling of plant metabolic pathways. J. Exp. Bot. 63, 2275-2292.

Analysis of sucrose accumulation in the sugar cane culm on the basis of in vitro kinetic data. J M Rohwer, F C Botha, Biochem. J. 358Rohwer, J. M., and Botha, F. C. (2001). Analysis of sucrose accumulation in the sugar cane culm on the basis of in vitro kinetic data. Biochem. J. 358, 437-445.

Zea mays RS1563: a comprehensive genome-scale metabolic reconstruction of maize metabolism. R Saha, P F Suthers, C D Maranas, 10.1371/journal.pone.0021784PLoS ONE. 6Saha, R., Suthers, P. F., and Maranas, C. D. (2011). Zea mays RS1563: a comprehensive genome-scale metabolic reconstruction of maize metabolism. PLoS ONE 6, e21784. doi: 10.1371/journal.pone.0021784

Integrative functional genomics of salt acclimatization in the model legume Lotus japonicus. D H Sanchez, F Lippold, H Redestig, M A Hannah, A Erban, U Kramer, J Kopka, M K Udvardi, Plant J. 53Sanchez, D. H., Lippold, F., Redes- tig, H., Hannah, M. A., Erban, A., Kramer, U., Kopka, J., and Udvardi, M. K. (2008). Integrative functional genomics of salt acclimatization in the model legume Lotus japonicus. Plant J. 53, 973-987.

Comparative metabolomics of drought acclimation in model and forage legumes. D H Sanchez, F Schwabe, A Erban, M K Udvardi, J Kopka, Plant Cell Environ. 35Sanchez, D. H., Schwabe, F., Erban, A., Udvardi, M. K., and Kopka, J. (2012). Comparative metabolomics of drought acclimation in model and forage legumes. Plant Cell Environ. 35, 136-149.

Simulating plant metabolic pathways with enzyme-kinetic models. K Schallau, B H Junker, Plant Physiol. 152Schallau, K., and Junker, B. H. (2010). Simulating plant metabolic pathways with enzyme-kinetic models. Plant Physiol. 152, 1763-1771.

Plant metabolomics: towards biological function and mechanism. N Schauer, A R Fernie, Trends Plant Sci. 11Schauer, N., and Fernie, A. R. (2006). Plant metabolomics: towards biolog- ical function and mechanism. Trends Plant Sci. 11, 508-516.

Comprehensive metabolic profiling and phenotyping of interspecific introgression lines for tomato improvement. N Schauer, Y Semel, U Roessner, A Gur, I Balbo, F Carrari, T Pleban, A Perez-Melis, C Bruedigam, J Kopka, L Willmitzer, D Zamir, A R Fernie, Nat. Biotechnol. 24Schauer, N., Semel, Y., Roessner, U., Gur, A., Balbo, I., Carrari, F., Ple- ban, T., Perez-Melis, A., Bruedigam, C., Kopka, J., Willmitzer, L., Zamir, D., and Fernie, A. R. (2006). Compre- hensive metabolic profiling and phe- notyping of interspecific introgres- sion lines for tomato improvement. Nat. Biotechnol. 24, 447-454.

The impact of impaired mitochondrial function on retrograde signalling: a meta-analysis of transcriptomic responses. M Schwarzländer, A.-C König, L J Sweetlove, I Finkemeier, J. Exp. Bot. 63Schwarzländer, M., König, A.-C., Sweet- love, L. J., and Finkemeier, I. (2012). The impact of impaired mitochon- drial function on retrograde sig- nalling: a meta-analysis of transcrip- tomic responses. J. Exp. Bot. 63, 1735-1750.

Mitochondrial metabolism in developing embryos of Brassica napus. J Schwender, Y Shachar-Hill, J B Ohlrogge, J. Biol. Chem. 281Schwender, J., Shachar-Hill, Y., and Ohlrogge, J. B. (2006). Mitochondrial metabolism in developing embryos of Brassica napus. J. Biol. Chem. 281, 34040-34047.

Metabolomics for plant stress response. V Shulaev, D Cortes, G Miller, R Mittler, Physiol. Plant. 132Shulaev, V., Cortes, D., Miller, G., and Mittler, R. (2008). Metabolomics for plant stress response. Physiol. Plant. 132, 199-208.

Modification of OsSUT1 gene expression modulates the salt response of rice Oryza sativa cv. Taipei 309. M R Siahpoosh, D H Sanchez, A Schlereth, G N Scofield, R T Furbank, J T Van Dongen, J Kopka, Plant Sci. 182Siahpoosh, M. R., Sanchez, D. H., Schlereth, A., Scofield, G. N., Fur- bank, R. T., Van Dongen, J. T., and Kopka, J. (2012). Modification of OsSUT1 gene expression modu- lates the salt response of rice Oryza sativa cv. Taipei 309. Plant Sci. 182, 101-111.

Mild reductions in mitochondrial citrate synthase activity result in a compromised nitrate assimilation and reduced leaf pigmentation but have no effect on photosynthetic performance or growth. A Sienkiewicz-Porzucek, A Nunes-Nesi, R Sulpice, J Lisec, D C Centeno, P Carillo, A Leisse, E Urbanczyk-Wochniak, A R Fernie, Plant Physiol. 147Sienkiewicz-Porzucek, A., Nunes-Nesi, A., Sulpice, R., Lisec, J., Centeno, D. C., Carillo, P., Leisse, A., Urbanczyk- Wochniak, E., and Fernie, A. R. (2008). Mild reductions in mito- chondrial citrate synthase activity result in a compromised nitrate assimilation and reduced leaf pig- mentation but have no effect on pho- tosynthetic performance or growth. Plant Physiol. 147, 115-127.

Mild reductions in mitochondrial NAD-dependent isocitrate dehydrogenase activity result in altered nitrate assimilation and pigmentation but do not impact growth. A Sienkiewicz-Porzucek, R Sulpice, S Osorio, I Krahnert, A Leisse, E Urbanczyk-Wochniak, M Hodges, A R Fernie, A Nunes-Nesi, Mol. Plant. 3Sienkiewicz-Porzucek, A., Sulpice, R., Osorio, S., Krahnert, I., Leisse, A., Urbanczyk-Wochniak, E., Hodges, M., Fernie, A. R., and Nunes- Nesi, A. (2010). Mild reductions in mitochondrial NAD-dependent isoc- itrate dehydrogenase activity result in altered nitrate assimilation and pig- mentation but do not impact growth. Mol. Plant 3, 156-173.

Toward a systems approach to understanding plant cell walls. C Somerville, S Bauer, G Brininstool, M Facette, T Hamann, J Milne, E Osborne, A Paredez, S Persson, T Raab, S Vorwerk, H Youngs, Science. 306Somerville, C., Bauer, S., Brininstool, G., Facette, M., Hamann, T., Milne, J., Osborne, E., Paredez, A., Pers- son, S., Raab, T., Vorwerk, S., and Youngs, H. (2004). Toward a systems approach to understanding plant cell walls. Science 306, 2206-2211.

From structure to dynamics of metabolic pathways: application to the plant mitochondrial TCA cycle. R Steuer, A N Nesi, A R Fernie, T Gross, B Blasius, J Selbig, Bioinformatics. 23Steuer, R., Nesi, A. N., Fernie, A. R., Gross, T., Blasius, B., and Selbig, J. (2007). From structure to dynamics of metabolic pathways: application to the plant mitochondrial TCA cycle. Bioinformatics 23, 1378-1385.

Reduced expression of succinyl-coenzyme A ligase can be compensated for by upregulation of the γ-aminobutyrate shunt in illuminated tomato leaves. C Studart-Guimarães, A Fait, A Nunes-Nesi, F Carrari, B Usadel, A R Fernie, Plant Physiol. 145Studart-Guimarães, C., Fait, A., Nunes- Nesi, A., Carrari, F., Usadel, B., and Fernie, A. R. (2007). Reduced expression of succinyl-coenzyme A ligase can be compensated for by up- regulation of the γ-aminobutyrate shunt in illuminated tomato leaves. Plant Physiol. 145, 626-639.

Starch as a major integrator in the regulation of plant growth. R Sulpice, E.-T Pyl, H Ishihara, S Trenkamp, M Steinfath, H Witucka-Wall, Y Gibon, B Usadel, F Poree, M C Piques, M Von Korff, M C Steinhauser, J J B Keurentjes, M Guenther, M Hoehne, J Selbig, A R Fernie, T Altmann, M Stitt, Proc. Natl. Acad. Sci. U.S.A. 106Sulpice, R., Pyl, E.-T., Ishihara, H., Trenkamp, S., Steinfath, M., Witucka- Wall, H., Gibon, Y., Usadel, B., Poree, F., Piques, M. C., Von Korff, M., Stein- hauser, M. C., Keurentjes, J. J. B., Guenther, M., Hoehne, M., Selbig, J., Fernie, A. R., Altmann, T., and Stitt, M. (2009). Starch as a major integrator in the regulation of plant growth. Proc. Natl. Acad. Sci. U.S.A. 106, 10348-10353.

Mild reductions in cytosolic NADP-dependent isocitrate dehydrogenase activity result in lower amino acid contents and pigmentation without impacting growth. R Sulpice, A Sienkiewicz-Porzucek, S Osorio, I Krahnert, M Stitt, A R Fernie, A Nunes-Nesi, Amino Acids. 39Sulpice, R., Sienkiewicz-Porzucek, A., Osorio, S., Krahnert, I., Stitt, M., Fernie, A. R., and Nunes-Nesi, A. (2010a). Mild reductions in cytoso- lic NADP-dependent isocitrate dehy- drogenase activity result in lower amino acid contents and pigmen- tation without impacting growth. Amino Acids 39, 1055-1066.

Network analysis of enzyme activities and metabolite levels and their relationship to biomass in a large panel of Arabidopsis accessions. R Sulpice, S Trenkamp, M Steinfath, B Usadel, Y Gibon, H Witucka-Wall, E.-T Pyl, H Tschoep, M C Steinhauser, M Guenther, M Hoehne, J M Rohwer, T Altmann, A R Fernie, M Stitt, Plant Cell. 22Sulpice, R., Trenkamp, S., Steinfath, M., Usadel, B., Gibon, Y., Witucka-Wall, H., Pyl, E.-T., Tschoep, H., Stein- hauser, M. C., Guenther, M., Hoehne, M., Rohwer, J. M., Altmann, T., Fer- nie, A. R., and Stitt, M. (2010b). Network analysis of enzyme activities and metabolite levels and their rela- tionship to biomass in a large panel of Arabidopsis accessions. Plant Cell 22, 2872-2893.

Not just a circle: flux modes in the plant TCA cycle. L J Sweetlove, K F M Beard, A Nunes-Nesi, A R Fernie, R G Ratcliffe, Trends Plant Sci. 15Sweetlove, L. J., Beard, K. F. M., Nunes- Nesi, A., Fernie, A. R., and Ratcliffe, R. G. (2010). Not just a circle: flux modes in the plant TCA cycle. Trends Plant Sci. 15, 462-470.

The mitochondrion: an integration point of cellular metabolism and signalling. L J Sweetlove, A Fait, A Nunes-Nesi, T Williams, A R Fernie, Crit. Rev. Plant Sci. 26Sweetlove, L. J., Fait, A., Nunes-Nesi, A., Williams, T., and Fernie, A. R. (2007). The mitochondrion: an inte- gration point of cellular metabolism and signalling. Crit. Rev. Plant Sci. 26, 17-43.

Regulation of metabolic networks: understanding metabolic complexity in the systems biology era. L J Sweetlove, A R Fernie, New Phytol. 168Sweetlove, L. J., and Fernie, A. R. (2005). Regulation of metabolic networks: understanding metabolic complex- ity in the systems biology era. New Phytol. 168, 9-24.

The impact of oxidative stress on Arabidopsis mitochondria. L J Sweetlove, J L Heazlewood, V Herald, R Holtzapffel, D A Day, C J Leaver, A H Millar, Plant J. 32Sweetlove, L. J., Heazlewood, J. L., Her- ald, V., Holtzapffel, R., Day, D. A., Leaver, C. J., and Millar, A. H. (2002). The impact of oxidative stress on Ara- bidopsis mitochondria. Plant J. 32, 891-904.

Mitochondrial uncoupling protein is required for efficient photosynthesis. L J Sweetlove, A Lytovchenko, M Morgan, A Nunes-Nesi, N L Taylor, C J Baxter, I Eickmeier, A R Fernie, Proc. Natl. Acad. Sci. U.S.A. 103Sweetlove, L. J., Lytovchenko, A., Mor- gan, M., Nunes-Nesi, A., Taylor, N. L., Baxter, C. J., Eickmeier, I., and Fernie, A. R. (2006). Mitochondrial uncou- pling protein is required for efficient photosynthesis. Proc. Natl. Acad. Sci. U.S.A. 103, 19587-19592.

The Arabidopsis thaliana 2-D gel mitochondrial proteome: refining the value of reference maps for assessing protein abundance, contaminants and post-translational modifications. N L Taylor, J L Heazlewood, A H Millar, Proteomics. 11Taylor, N. L., Heazlewood, J. L., and Millar, A. H. (2011). The Arabidopsis thaliana 2-D gel mito- chondrial proteome: refining the value of reference maps for assess- ing protein abundance, contaminants and post-translational modifications. Proteomics 11, 1720-1733.

The hydroxypyruvate-reducing system in Arabidopsis: multiple enzymes for the same end. S Timm, A Florian, K Jahnke, A Nunes-Nesi, A R Fernie, H Bauwe, Plant Physiol. 155Timm, S., Florian, A., Jahnke, K., Nunes-Nesi, A., Fernie, A. R., and Bauwe, H. (2011). The hydroxypyruvate-reducing system in Arabidopsis: multiple enzymes for the same end. Plant Physiol. 155, 694-705.

A cytosolic pathway for the conversion of hydroxypyruvate to glycerate during photorespiration in Arabidopsis. S Timm, A Nunes-Nesi, T Pärnik, K Morgenthal, S Wienkoop, O Keerberg, W Weckwerth, L A Kleczkowski, A R Fernie, H Bauwe, Plant Cell. 20Timm, S., Nunes-Nesi, A., Pärnik, T., Morgenthal, K., Wienkoop, S., Keerberg, O., Weckwerth, W., Kleczkowski, L. A., Fernie, A. R., and Bauwe, H. (2008). A cytosolic path- way for the conversion of hydrox- ypyruvate to glycerate during pho- torespiration in Arabidopsis. Plant Cell 20, 2848-2859.

Mitochondrial malate dehydrogenase lowers leaf respiration and alters photorespiration and plant growth in Arabidopsis. T Tomaz, M Bagard, I Pracharoenwattana, P Linden, C P Lee, A J Carroll, E Stroeher, S M Smith, P Gardestrom, A H Millar, Plant Physiol. 154Tomaz, T., Bagard, M., Pracharoen- wattana, I., Linden, P., Lee, C. P., Carroll, A. J., Stroeher, E., Smith, S. M., Gardestrom, P., and Millar, A. H. (2010). Mitochondrial malate dehydrogenase lowers leaf respiration and alters photorespiration and plant growth in Arabidopsis. Plant Physiol. 154, 1143-1157.

Effect of slow growth on metabolism of Escherichia coli, as revealed by global metabolite pool. H Tweeddale, L Notley-Mcrobb, T Ferenci, J. Bacteriol. 180Metabolome") analysisTweeddale, H., Notley-Mcrobb, L., and Ferenci, T. (1998). Effect of slow growth on metabolism of Escherichia coli, as revealed by global metabo- lite pool ("Metabolome") analysis. J. Bacteriol. 180, 5109-5116.

Parallel analysis of transcript and metabolic profiles: a new approach in systems biology. E Urbanczyk-Wochniak, A Luedemann, J Kopka, J Selbig, U Roessner-Tunali, L Willmitzer, A R Fernie, EMBO Rep. 4Urbanczyk-Wochniak, E., Luedemann, A., Kopka, J., Selbig, J., Roessner- Tunali, U., Willmitzer, L., and Fernie, A. R. (2003). Parallel analysis of tran- script and metabolic profiles: a new approach in systems biology. EMBO Rep. 4, 989-993.

Conversion of MapMan to allow the analysis of transcript data from Solanaceous species: effects of genetic and environmental alterations in energy metabolism in the leaf. E Urbanczyk-Wochniak, B Usadel, O Thimm, A Nunes-Nesi, F Carrari, M Davy, O Blasing, M Kowalczyk, D Weicht, A Polinceusz, S Meyer, M Stitt, A R Fernie, Plant Mol. Biol. 60Urbanczyk-Wochniak, E., Usadel, B., Thimm, O., Nunes-Nesi, A., Carrari, F., Davy, M., Blasing, O., Kowal- czyk, M., Weicht, D., Polinceusz, A., Meyer, S., Stitt, M., and Fer- nie, A. R. (2006). Conversion of MapMan to allow the analysis of tran- script data from Solanaceous species: effects of genetic and environmen- tal alterations in energy metabolism in the leaf. Plant Mol. Biol. 60, 773-792.

Kinetic model of sucrose accumulation in maturing sugarcane culm tissue. L Uys, F C Botha, J H S Hofmeyr, J M Rohwer, O Van Aken, B Zhang, C Carrie, V Uggalla, E Paynter, E Giraud, J Whelan, Phytochemistry. 68Mol. PlantUys, L., Botha, F. C., Hofmeyr, J. H. S., and Rohwer, J. M. (2007). Kinetic model of sucrose accumu- lation in maturing sugarcane culm tissue. Phytochemistry 68, 2375-2392. van Aken, O., Zhang, B., Carrie, C., Uggalla, V., Paynter, E., Giraud, E., and Whelan, J. (2009). Defining the mitochondrial stress response in Arabidopsis thaliana. Mol. Plant 2, 1310-1324.

Metabolomics in systems biology. W Weckwerth, Annu. Rev. Plant Biol. 54Weckwerth, W. (2003). Metabolomics in systems biology. Annu. Rev. Plant Biol. 54, 669-689.

13C metabolic flux analysis. W Wiechert, Metab. Eng. 3Wiechert, W. (2001). 13C metabolic flux analysis. Metab. Eng. 3, 195-206.

From stationary to instationary metabolic flux analysis. W Wiechert, K Nöh, Adv. Biochem. Eng. Biotechnol. 92Wiechert, W., and Nöh, K. (2005). From stationary to instationary metabolic flux analysis. Adv. Biochem. Eng. Biotechnol. 92, 145-172.

A genome-scale metabolic model accurately predicts fluxes in central carbon metabolism under stress conditions. T C R Williams, M G Poolman, A J M Howden, M Schwarzlander, D A Fell, R G Ratcliffe, L J Sweetlove, Plant Physiol. 154Williams, T. C. R., Poolman, M. G., Howden, A. J. M., Schwarzlander, M., Fell, D. A., Ratcliffe, R. G., and Sweet- love, L. J. (2010). A genome-scale metabolic model accurately predicts fluxes in central carbon metabolism under stress conditions. Plant Physiol. 154, 311-323.

The AAA-ATPase p97 is essential for outer mitochondrial membrane protein turnover. S Xu, G Peng, Y Wang, S Fang, M Karbowski, Mol. Biol. Cell. 22Xu, S., Peng, G., Wang, Y., Fang, S., and Karbowski, M. (2011). The AAA-ATPase p97 is essential for outer mitochondrial membrane pro- tein turnover. Mol. Biol. Cell 22, 291-300.

Mapping photoautotrophic metabolism with isotopically nonstationary 13C flux analysis. J D Young, A A Shastri, G Stephanopoulos, J A Morgan, Metab. Eng. 13Young, J. D., Shastri, A. A., Stephanopoulos, G., and Morgan, J. A. (2011). Mapping pho- toautotrophic metabolism with isotopically nonstationary 13C flux analysis. Metab. Eng. 13, 656-665.

An elementary metabolite unit (EMU) based method of isotopically nonstationary flux analysis. J D Young, J L Walther, M R Antoniewicz, H Yoo, G Stephanopoulos, Biotechnol. Bioeng. 99Young, J. D., Walther, J. L., Antoniewicz, M. R., Yoo, H., and Stephanopoulos, G. (2008). An elementary metabolite unit (EMU) based method of iso- topically nonstationary flux analysis. Biotechnol. Bioeng. 99, 686-699.

Analysis of Arabidopsis with highly reduced levels of malate and fumarate sheds light on the role of these organic acids as storage carbon molecules. M B Zell, H Fahnenstich, A Maier, M Saigo, E V Voznesenskaya, G E Edwards, C Andreo, F Schleifenbaum, C Zell, M F Drincovich, V G Maurino, Plant Physiol. 152Zell, M. B., Fahnenstich, H., Maier, A., Saigo, M., Voznesenskaya, E. V., Edwards, G. E., Andreo, C., Schleifen- baum, F., Zell, C., Drincovich, M. F., and Maurino, V. G. (2010). Analy- sis of Arabidopsis with highly reduced levels of malate and fumarate sheds light on the role of these organic acids as storage carbon molecules. Plant Physiol. 152, 1251-1262.