# SURVEY AND SUMMARY Regulatory roles of RNA modifications in breast cancer

CorpusID: 237555138 - [https://www.semanticscholar.org/paper/266a902ceb71a7637608844159df34a9f8cfb99b](https://www.semanticscholar.org/paper/266a902ceb71a7637608844159df34a9f8cfb99b)

Fields: Medicine, Biology

## (s8) A-to-I editing in breast cancer
Number of References: 11

(p8.0) A-to-I editing is known to be the major source of mRNA variability in breast cancer (171), and it is controlled by type-I interferon response and high ADAR DNA copy number (172). Hence, >76 000 RNA editing sites were identified by using a bioluminescent reporter system and upon interferon treatment of the breast cancer MCF7 cell line (173). Breast tumors harbor amplified levels of ADAR (174) and inhibiting its activity has been shown to induce apoptosis. Indeed, A-to-I mRNA editing regulated breast cancer cell proliferation by modulating the stability and expression of genes involved in cell cycle and DNA damage response being ATM, MDM2, MDM4, CENPN and XPO1 among the affected transcripts (175). In another study, increased A-to-I mRNA editing was found on the mRNA of ATM, GINS4 and POLH affecting their stability and/ or expression in primary breast tumors when compared to the noncancerous tissues (171). In addition, Gumireddy et al. reported that A-to-I mRNA editing of ‚ê• -amino butyric acid receptor alpha 3 (GABRA3), one of the subunits of GABA (A) receptor, changed its role from a metastasis promoter to a tumor suppressor by suppressing the ERK pathway activation (176). In an attempt to assess the role of A-to-I RNA editing on proteomic diversity, Peng et al. showed that A-to-I RNA editing contributed to protein heterogeneity in breast cancer (177). Hence, nine unique RNA editing sites with variant peptide evidence were identified such as COPA I164V and IGFBP R78G which were detected across 11 out of 36 BRCA mass spectrometry datasets (177). These peptide variants were shown to be presented as self-antigens by human leukocyte antigen (HLA) molecules, and thus recognized by the immune system. For instance, CD8 + effector T cells were evidenced in tumors in response to peptide variants generated from cyclin I (178). A-to-I RNA editing has also been linked to resistance to chemotherapy. Specifically, RNA editing at 26 sites of 3 UTR modulated the expression of dihydrofolate reductase resulting in highly proliferative breast cancer cells which were resistant to methotrexate (179). To sum up, existing data suggests an important role of A-to-I RNA editing in genes associated with breast cancer-relevant pathways and treatment outcomes, suggesting an important role of ADAR1 function in breast tumorigenesis.
## (s9) C-to-U editing in breast cancer
Number of References: 6

(p9.0) APOBEC-dependent somatic mutation confers increased susceptibility for breast cancer (180) and is found to be enriched in the HER2 subtype (181,182). Overexpression of cytidine deaminase APOBEC3A in HEK293T cells caused mRNA alterations in several tumor associated genes including PTEN, KMT2A, ATM, BRCA1 and BRCA2 (183); however, their significance in breast tumorigenesis is yet to be evaluated. mRNA expression of APOBEC3B was evaluated in ductal carcinoma in situ (DCIS), invasive breast cancer (IBC) and normal breast cells (184). Normal breast cells expressed reduced levels of APOBEC3B, while increased expression was evident in DCIS and IBC. Recently, APOBEC3-mediated C-to-U RNA editing in breast cancer was shown to be associated with improved patient survival and enhanced immune activity (185). In another study, using a bioinformatics approach, C-to-U RNA editing levels were estimated in 1040 primary breast tumor tissues and 93 adjacent normal tissues. More than 5000 APOBEC3-mediated RNA editing sites were identified using TCGA sequencing data. For 440 sites, editing was found on 411 transcripts among which most prevalent were on GATA Zinc Finger Domain Containing 2B (GATAD2B), Serpin Family A Member 1 (SERPINA1) and Adenosine Monophosphate Deaminase 3 (AMPD3) (185). Moreover, Apobec-1 complementation factor (A1CF), another mRNA editing enzyme, promoted proliferation of basal-like breast cancer cells by targeting interleukin-6 (186). Overall, the existing data on C-to-U editing suggests important roles in breast cancer progression and further research may identify targets with therapeutic importance.
## (s10) Other RNA modifications related to breast cancer
Number of References: 12

(p10.0) Growing evidence in recent years has pointed out that tRNAs and their derivatives are dysregulated in breast and other cancers (187). However, the functional role of tRNA modifications in tumorigenesis is still elusive. As mentioned before, several modifications are shared by the distinct RNA species but others are unique to tRNAs, as is the case of cm5U34 and mcm5U34 at the wobble position. These marks have been shown to influence the translational efficiency and the accuracy of the reading frame (188). In addition, the catalytic enzymes for mcm5s2 deposition are upregulated in breast cancer sustaining metastasis (189). They enhanced the translation of DEK and LEF1, two oncogenes, enabling breast cancer cells to migrate and invade other tissues (189). Another modification that occurs at the wobble anticodon position is queuosine which is dependent on the gut microbiome (190). Thus, the microbiome product queuine is the substrate for the enzymes tRNA guanine transglycosylases (TGTs). In eukaryotes, the active TGT is a heterodimer formed of a catalytically active queuine tRNA-ribosyltransferase subunit 1 (QTRT1) and a catalytically inactive QTRT2 subunit (191). Knockout of QTRT1 in MCF7 and MDA-MB-231 cells produced changes in the functions of genes involved in cell proliferation, junction formation and migration (190). These results have been validated in mouse models, thus enforcing the significant role that this modification is playing in breast cancer development (190).

(p10.1) TRMT12, a tRNA methyltransferase that mediates posttranscriptional modifications on tRNAPhe has been reported to be consistently amplified and overexpressed in cell lines and breast cancer patient samples (192). In yeast, it catalyses the formation of wybutosine (yW) on the 37th residue of tRNAPhe, modification that helps in stabilization of the codon-anticodon interaction and in maintenance of the reading frame (192). Thus, disruption at the level of this modification can lead to translational errors and is therefore of great interest to establish the role of these enzymes, together with its effect on tRNAs, in tumorigenesis of breast cells.

(p10.2) It is noteworthy that many of the methyltransferases that catalyze the methylation of different types of RNAs have been shown to be deregulated in breast cancer. Although the lack of any association between the deposited marks and the expression of these proteins made us filter out these studies, there are some that are worth mentioning. For instance, tRNA methyltransferase 2 homolog A (TRMT2A) which catalyzes the methylation of the U5 of tRNAs and regulates cell cycle has been associated with increased recurrence risk in HER2+ breast cancer patients (193). Human tRNA methyltransferase 9-like (hTRM9L) protein, an another tRNA methyltransferase, is downregulated in breast cancer and was shown to suppress tumor growth in vivo by decreasing proliferation, cell cycle arrest in G0/G1, upregulation of LIN9, and blocking the hypoxia response (194) in colorectal cancer. Whether this mechanism is the same in breast cancer and if it is mediated though methylation remains to be further investigated.
