# Oocyte-associated transcription factors in reprogramming after somatic cell nuclear transfer: a review

CorpusID: 83562471 - [https://www.semanticscholar.org/paper/60c04edd2cf072a5c8cffb1dbf5a25b9da84ba59](https://www.semanticscholar.org/paper/60c04edd2cf072a5c8cffb1dbf5a25b9da84ba59)

Fields: Medicine, Biology

## (s7) Transcriptomics may dissect the detailed gene regulatory mechanisms in cellular reprogramming
Number of References: 10

(p7.0) The most advanced single-cell sequencing technique makes it possible to dissect the detail of genetic regulatory mechanisms in a valuable resource, e.g., progressively developmental embryo. The transcriptome analysis of the different developmental stages from oocyte to morula in both mouse and human reveal a sequential order of transcriptional changes in pathways of cell cycle, gene regulation, translation and metabolism, acting in a stepwise fashion from cleavage to morula [92]. In cattle, RNA sequencing analysis of transcripts in GV and M II oocytes, and in 4-, 8-, 16-cell and blastocyst stage embryos indicated that the largest proportion of gene activation was found in 8-cell embryos [93]. Cross-species comparisons between mouse and human showed that significant difference in developmental specificity and timing existed [92]. It has been reported that the maternal glucocorticoid receptor transcript and protein participated in the maternal reprogramming of zebrafish development [94]. Ces5/Ooep was suggested to be a maternal-effect gene directly associated with the oocyte cytoplasmic developmental patterns [95]. Transcription factor gene MSX1 played important roles in bovine preimplantation embryo development. Injection of long double-stranded RNA and small interfering RNA to oocytes at GV stage reduced MSX1 mRNA expression, which significantly affected oocyte maturation and the subsequent embryo development [96]. In Xenopus, the conserved long non-coding RNAs were expressed in a developmental stage-specific fashion [97]. It has been demonstrated that miRNAs played regulatory roles during early embryo development [98]. The Gli-like transcription factor Glis1 was enriched in unfertilized oocytes and embryos at the 1-cell stage. Glis1 was observed to promote multiple pro-reprogramming pathways, including Myc, Nanog, Lin28, Wnt, Essrb and the mesenchymal-epithelial transition [99]. Glis1, as a mater-nal transcription factor effectively promoted the direct reprogramming of somatic cells during iPSC generation [99]. With deep analysis of transcriptomics in oocytes and early developmental stage embryos, more maternal transcription factors inducing cellular reprogramming will be identified. Figure 1 illustrates the possible roles of transcriptional factors in oocyte maturation, fertilization, nuclear transfer reprogramming and embryo development.
