# Urbanization and Waterborne Pathogen Emergence in Low-Income Countries: Where and How to Conduct Surveys?

CorpusID: 210330772
 
tags: #Medicine, #Environmental_Science, #Geography

URL: [https://www.semanticscholar.org/paper/77e0602c01953be5670241add983ba03b66c7890](https://www.semanticscholar.org/paper/77e0602c01953be5670241add983ba03b66c7890)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Urbanization and Waterborne Pathogen Emergence in Low-Income Countries: Where and How to Conduct Surveys?


Alexandra Bastaraud abastaraud@pasteur.mg 
Laboratoire d'Hygiène des Aliments et de l'Environnement, Institut Pasteur de Madagascar
BP 1274, Antananarivo 101Madagascar

Philippe Cecchi philippe.cecchi@ird.fr 
MARBEC (IRD, IFREMER, UM2 and CNRS), University Montpellier
34095MontpellierFrance

INSERM-IRD
Centre de Recherche Océanologique (CRO)
Université de Marseille II
Abidjan BPV 18, Ivory Coast 4 IRD UMR 912 SESSTIM13000MarseilleFrance

Pascal Handschumacher p.handschumacher@unistra.fr 
Mathias Altmann mathias.altmann@u-bordeaux.fr 
ISPED Université Victor Segalen Bordeaux II
146 rue Leo Saignat33076Bordeaux cedexFrance

Ronan Jambou 
Département de Parasitologie et des insectes vecteurs
Institut Pasteur Paris
75015ParisFrance


International Journal of Environmental Research and Public Health


Urbanization and Waterborne Pathogen Emergence in Low-Income Countries: Where and How to Conduct Surveys?
10.3390/ijerph17020480Received: 25 November 2019; Accepted: 20 December 2019; Published: 11 January 2020Reviewwaterborne diseasesdrug resistanceurbanizationsurface waterplasticsmetagenomic
A major forthcoming sanitary issue concerns the apparition and spreading of drug-resistant microorganisms, potentially threatening millions of humans. In low-income countries, polluted urban runoff and open sewage channels are major sources of microbes. These microbes join natural microbial communities in aquatic ecosystems already impacted by various chemicals, including antibiotics. These composite microbial communities must adapt to survive in such hostile conditions, sometimes promoting the selection of antibiotic-resistant microbial strains by gene transfer. The low probability of exchanges between planktonic microorganisms within the water column may be significantly improved if their contact was facilitated by particular meeting places. This could be specifically the case within biofilms that develop on the surface of the myriads of floating macroplastics increasingly polluting urban tropical surface waters. Moreover, as uncultivable bacterial strains could be involved, analyses of the microbial communities in their whole have to be performed. This means that new-omic technologies must be routinely implemented in low-and middle-income countries to detect the appearance of resistance genes in microbial ecosystems, especially when considering the new 'plastic context. '  We summarize the related current knowledge in this short review paper to anticipate new strategies for monitoring and surveying microbial communities.

## An Urgent Need to Investigate Environmental, Human, and Animal Microbiota Interactions

The United Nations predicts that half of the population in Africa and Asia will be urban by 2030 and that the African population will double (to 2.4 billion) by 2050. This global increase in population sustains a growing rate of five million residents per month in towns in low-and middle-income countries. This increase is essentially due to newborns but also to migration from rural areas. Urbanization presents opportunities and challenges for poverty reduction [1], but this rapid urbanization coupled with inadequate urban planning will disturb the functional organization of the cities [2]. The increase in populations in Africa induces "slumization" in towns with informal 


## Uncontrolled Urbanization Creates Large Reservoirs of Environmental/Human Composite Microorganism Communities in Surface Water Favoring Pathogen Evolution

Enteropathogens, such as enterotoxigenic Escherichia coli (ETEC), rotavirus, Vibrio cholerae, Campylobacter, and cryptosporidium, are major agents of diarrhea. They are good models for analyzing the propagation of pathogens in urban settings because their epidemiology is shaped by their ability to survive and to be transported from one household to another. Sustained by 


## Uncontrolled Urbanization Creates Large Reservoirs of Environmental/Human Composite Microorganism Communities in Surface Water Favoring Pathogen Evolution

Enteropathogens, such as enterotoxigenic Escherichia coli (ETEC), rotavirus, Vibrio cholerae, Campylobacter, and cryptosporidium, are major agents of diarrhea. They are good models for analyzing the propagation of pathogens in urban settings because their epidemiology is shaped by their ability to survive and to be transported from one household to another. Sustained by environmental factors, a single transient episode of contamination is often sufficient to start an epidemic, as seen during the 2011 cholera outbreak in Haiti [28]. Therefore, what are the major determinants of pathogen propagation in suburbs?


### Environmental Factors Modulate Human Pathogen Dynamics

The growth, diversity, and dynamics of pathogenic populations are shaped by environmental factors such as rainfall, temperature, nutrient loading, UV light, water flow, pH, and availability of carbon sources [29]. A good model to assess these effects is V. cholerae, as its proliferation is modulated by water temperature, pH, salinity, and plankton blooms [30]. These environmental parameters strongly control both the abundance and diversity of aquatic communities, which, in turn, control pathogen proliferation. High temperatures, sunlight, and nutrient inputs favor the growth of phytoplankton and bacteria (e.g., cyanobacteria). They also control top-down relationships linking microbial communities (such as Vibrio sp.) with their environmental reservoir like shellfish and zooplankton [29][30][31][32]. This was well described in reservoirs in Burkina Faso where alkaline pH and phytoplankton biomass sustain the proliferation of V. cholerae [33]. This was also described in wastewater ponds in Australia [34], where waterborne pathogens are controlled by a range of chemical or physical factors, including salinity, pH, turbulence-induced resuspension, and macronutrients. Viruses are also concerning as they can aggregate and adsorb to particles, sustaining their persistence in surface waters.

According to the physical parameters of the media, pathogens can modulate their metabolism to survive. Microorganisms can become uncultivable but persist for a long time, which impacts the techniques and the timing available to analyze the samples. Indeed, during nutrient starvation, iron limitation, or changes in salinity and pH, V. cholerae survives by losing its flagellum and becoming a spore-like uncultivable bacterium [31]. Escherichia coli, or Campylobacter jejuni, can also enter the viable but noncultivable (VBNC) state (also called dormancy) with low metabolism, allowing their survival under poor environmental conditions and resuscitation when conditions become more favorable. Resuscitation windows vary from a few days to several years (e.g., Vibrio vulnificus, 3 days; Vibrio fluvialis, 6 years [35]). Similarly, bacteria from the Firmicutes family can form endospores that can survive for several years in soil, plants, or sediments and are resistant to chlorination. They can persist or be transported by environmental reservoirs [9], impacting the quality of drinking water extracted from surface water resources long after contamination.


### Selection of Drug-Resistant Microorganism Can Occur in Surface Water

Antibiotics are spread in the environment via wastewater from farms and hospitals [10,[36][37][38]. The treatment of plants and aquaculture has recently become another major contributor of antibiotics. They promote the selection of resistance genes and the development of antibiotic-resistant bacteria [39,40]. The long-term irrigation of soils with untreated wastewater leads to an accumulation of these drugs at high levels in the soil. Freshwater and marine ecosystems have become reservoirs of resistance genes and antimicrobial-resistant bacteria, which are easily transferred to human pathogens [41,42]. In this setup, the acquisition of multidrug resistance enhances the propagation and emergence of pathogens [43]. Studies conducted in Dhaka (India) illustrate this point as well; two multidrug-resistant bacteria were found that were associated with water runoff, i.e., a Pseudomonas aeruginosa resistant to all antibiotics except ceftazidime and an E. coli sensitive only to ceftazidime and cotrimoxazole [44]. In this area, E. coli was thus found as the causative agent in 63% of the diarrhea cases in children. However, 73% of them were resistant to at least one of the 10 antibiotics tested, and 36% were multidrug-resistant. In the same line, the emergence of multidrug-resistant "El Tor" strains in Nigeria was also related to the presence of antimicrobial drugs in the biotope [45].

Aside from pathogens, drug resistance has also been described in environmental bacteria. The level of resistance varies with the place of collection of the microorganisms as the concentration of drugs accumulated in the environment also varies [46,47]. In surface water, antibiotics can usually inhibit 25% to 76% of wild-type environmental bacteria [48]. However, when studies are conducted on river sediments, swine feces, lagoon water, liquid manure, or farmed soil, the resistance of environmental bacteria can reach 60%, 92%, 100%, and 30%, respectively [48]. For these uncultivable environmental bacteria, antibiotic resistance genes (ARGs) can be searched using metagenomics, as illustrated in tannery wastewater [49], in which various ARGs have been identified at high abundance. Over 70 types of insertion sequences were detected in each sludge sample, among which 20% were sulfonamide-resistant sul1 genes. Class 1 integrase genes were prevalent in the whole tannery wastewater treatment plant. Tetracycline resistance genes (particularly tet33) were highly prevalent in anaerobic sludge but not in aerobic sludge [49]. This important finding draws attention to the role of various chemicals in the induction of drug resistance.

Aside from classic resistant pathways, some environmental bacteria can also degrade drugs through specific pathways that are not primarily involved in resistance but that belong to "the hidden resistome" [50]. This term covers all the pathways that can inhibit drug effects through a "mineralization" process of the drug. More than 90% of seawater bacteria are resistant to more than one antibiotic, and 20% are resistant to at least five [51]. Little is known about the potential effects of exogenous antibiotics on the diversity and functioning of bacterial communities in aquatic ecosystems, but ARG could be transferred to environmental bacteria. They have been suspected to shape the microbial community composition in freshwater reservoirs [52], promoting the occurrence of resistant strains (Actinobacteria and Firmicutes).

Moreover, lots of studies highlight the capacity of environmental bacteria to produce natural antibacterial substances that can be produced by marine heterotrophic bacteria, which may inhibit or kill other bacteria [53][54][55][56][57]. The antibacterial activity of two Ludwigia species (invasive aquatic weeds) against a series of bacteria, including pathogens, has been demonstrated [58]. The production of antibiotics as allelochemicals by phytoplankton and cyanobacteria has also been documented [59][60][61][62][63]. However, antibiotic-induced interactions among marine microorganisms have mostly been highlighted within marine biofilms [64]. These natural antimicrobials are mostly considered signaling molecules within species and not as veritable chemical weapons against other organisms [65]. Supporting this view, antibiotics are often produced at subinhibitory concentrations, as the metabolic cost of this production is relatively high. Overall, and due to this natural chemical war, many environmental bacteria have a capacity for fast evolutionary development of tolerance against antimicrobials [65].

At last, heavy metals are ubiquitous contaminants spread by runoff and sewage water into aquatic ecosystems, where they exert selective pressures on aquatic microbial communities. Heavy metals persist within sediment, where they increase the half-life of antibiotics [66]. Even at low concentrations, heavy metals can contribute to the emergence and spread of antimicrobial-resistant strains through coselection of genetic elements encoding both heavy metal and antibiotic resistance genes [67][68][69][70][71]. This was recently confirmed by the selection of antibiotic-resistant enterococci by very low heavy-metal concentrations [72].

Environmental bacteria are thus an unlimited source of resistance genes that might be transferred to pathogenic organisms. This transfer of genes from environmental to pathogenic bacteria has already been demonstrated. High densities of bacteriophages are released into water via defecation [73], while for soil bacteriophages, lysogeny is the prevalent reproductive strategy supporting horizontal gene transfer via transduction [74]. In Salmonella enterica, Staphylococcus aureus, E. coli O157:H7, or Vibrio cholerae, phages promote the secretion of toxins, effectors or regulatory proteins, adhesins, and serum resistance factors [75]. In this context, the domestic use of raw water might expose the human microbiota to new enzymatic structures from environmental microorganisms through lateral transmission. This transfer was already described in humans [76], sustained by integrons [77,78].


### Urban Farming and Untreated Water Consumption Are Sources of Pathogen Contamination for Humans

Enteric pathogens are not native to freshwater. They are imported from various sources, including animal and human feces [9]. Informal settlement areas around water surfaces generate high runoff and significant fecal pollution with a range of 4 to 8 log orders of bacteria [5]. When surface water is used for watering, urban farming and consumption of vegetables such as lettuce are thus major factors of human contamination [6,7,79]. The city of Kumasi in Ghana where reuse of wastewater has become usual is a good study case as water was found contaminated by fecal bacteria, such as Salmonella, viruses, and Cryptosporidium [80]. In this city, the prevalence of intestinal protozoan infection in primary school children reached 43% (mainly Giardia lamblia, Entamoeba histolytica/E. dispar, and Cryptosporidium parvum) [81]. This study also illustrated that the assessment of enteric pathogens in feces of children living in an area can be used as a sentinel marker of contamination of water. The mass distribution of anti-parasite drugs in schools should thus be preceded by stool collection and examination. The prevalence of each pathogen could be investigated, as well as their burden illustrated, by the speed of recontamination after treatment. Molecular techniques such as multiplex Q-PCR could be implemented for automatic analysis.


## Where to Survey the Emergence of Pathogens?

Surveys conducted in dispensaries with humans experiencing diarrhea are key strategies to detect new pathogens. Indeed, new pathogens will usually cause symptoms due to a lack of immunity. Systematic sampling of stools could be easily implemented in children from poor suburbs of the city, as well as attending dispensaries with diarrhea. However, the size of sampling should be large enough to allow detection of rare pathogens. This last point was demonstrated by studies designed to monitor variations of pathogens related to public health interventions [82,83].

Studies could also be conducted in the environment, but sampling sites must be carefully defined. Surface water should be a key area for analysis [84][85][86][87][88]. However, due to large variations in the concentration of microorganisms, studies conducted in environmental water bodies are not sufficiently sensitive for detecting the circulation of pathogens before an epidemic. The collection of water in distribution networks is sometimes more efficient because it provides information on the contamination of the water resource from its source to the tap, including the pipes, which are usually good supports for biofilms. Network contaminations can be due to old pipes allowing the diffusion of sewage water to pipes, providing drinkable water. This is frequent when the landscape is complex, with, for example, important elevation changes, such as in Antananarivo in Madagascar [84]. This strategy is frequently used by public water supply companies, as in Antananarivo (Bastaraud et al. in press). Analyses conducted on steels can reveal abundant contamination with highly resistant strains sometimes harboring specific pathways involving glutathione metabolism, the SoxRS-, OxyR-, or RpoS-system [85].

As previously described, interactions between microorganisms in biofilms are also important for genetic exchange, collaborative survival in water, propagation of pathogens, or protection against adverse environmental conditions. The co-adhesion of Pseudomonas aeruginosa and E. coli O157:H7 was, for example, demonstrated to modify growth rates when bacteria are subjected to various osmotic pressures, temperatures, heavy metal concentrations, or salt stress [86,87]. Biofilms also promote the acquisition of virulence [88,89], and identification of the places to sample is thus crucial. Environment microorganisms and macroorganisms can also host pathogens and promote biofilms. For example, Legionella spp. can survive in free-living amoebas (e.g., Naegleria spp. and Acanthamoeba spp.) and benefit from their protection against desiccation, elevated temperature, and disinfectants [9]. The role of aquatic vertebrates and arthropods remains poorly investigated, but they may trigger interactions through biofilm formation on their shells. Aquatic vertebrates also promote the large-scale dispersion of microbes, thus sustaining the survival and even proliferation of rare microorganisms [90]. Certain bivalves, such as the invasive zebra mussel (Dreissena polymorpha), can host various bacterial and protozoan pathogens. Indeed, Vibrio cholerae can attach to a range of aquatic organisms, including shellfish, plants, algae, and zooplankton, in salty environments and persist for long periods [29], triggering intermittent outbreaks in human communities.

Finally, garbage is also an important site for complex microbial community development. In West Africa, the coastal lagoon network is particularly developed from Sierra Leone to Angola. The main towns of these countries are settled along these lagoons with runoff, garbage, and sewage discharging directly into the lagoons. The large amounts of plastics that can be seen floating on the surface of these lagoons demonstrate these impacts. Indeed, millions of tons of plastics are rejected in aquatic ecosystems yearly [60], with more than 4.4 million originating from Africa in 2010 [91]. Whatever the origin of plastic debris, they have a lifespan of decades, if not centuries [92]. Plastics float and disperse due to winds and currents, and can travel over very long distances. During these travels, they will be degraded and fragmented, creating micro-and nanoparticles. The impacts of the degradation of plastic debris on biodiversity are still poorly known [93][94][95], but plastics can be easily colonized by microorganisms and biofilms [96,97]. They can play a major role in the transfer of organisms from one place to another [98][99][100][101][102][103]. Phytoplankton and zooplankton can develop on these supports and promote the production of bacteria and viruses, which are subsequently locally diverted to the water column [67]. Water pieces locked in floating bottles or plastic containers are also well adapted for colonization by microorganisms. Bacteria living on plastics, organic particles, and surrounding seawaters have been described [68]. The spreading of fish pathogens has also recently been described [104]. Plastic-specific bacteria constitute a distinct set of microorganisms that are very different from the surrounding water [105][106][107][108], with the quantity of bacteria 500 times more elevated than in the surrounding water [109]. Human pathogens such as Vibrio cholerae and E. coli can also stick to such supports [110,111]. Kirstein et al. (2016) reported that 13% of the debris they studied (by cultivation) contained Vibrio spp. We also confirmed this finding in the Abidjan lagoon (Vakou et al. in preparation). Conversely, Debroas et al. [112] performed metagenomic analysis and recorded only a 0.14% prevalence of Vibrio on the material they studied. They pointed out the opportunism of these bacteria ('hitchhikers') that use debris as a simple support unlike the vast majority of other taxa that showed a true affinity (biodegradation) for the polymers. In the marine environment (North Sea), Oberbeckman et al. [108] obtained comparable results and indicated that different communities comprising both prokaryotes and eukaryotes developed according to the nature of the plastic (polymer composition), confirming the observations of Carpenter and Smith (1972) [113]. It was suggested that organic aggregates may facilitate the survival of aquatic pathogens by providing both a 'refuge' [114] and a 'resource-rich microhabitat' [115].

Overall, and regardless of their size, floating plastics behave as ideal bioreactors where bacteria settle and proliferate. To extensively investigate the potentially pathogenic bacterial strains, present in urban tropical surface water masses, plastics must be sampled, and their biofilms must be imperatively studied in parallel with the classically monitored microbial communities of the water column and sediments. Metagenomic will be the best approach to study complex microbiota on these supports.


## How to Identify Complex Microbiota


### Different Technical Approaches

To survey the emergence of pathogens in environmental water, the detection of bacteria, viruses, and parasites is required. This detection refers to two major problems: How and when?

Classic bacteriological techniques can be used to detect contamination of surface water, i.e., (i) the fecal bacteria index (FIB), which consists of measurements of Escherichia coli, intestinal enterococci, and Clostridium perfringens densities [9]; (ii) microbial source tracking (MST), which can confirm fecal contamination; and (iii) host-specific fecal DNA markers, which can be used to differentiate human and nonhuman sources [116]. However, these methods do not help forecast the emergence of new pathogens. In the same line, the emergence of antibiotic resistance can be first detected by phenotypic methods using antibiotics in culture. Proteins involved in drug resistance can be detected from isolated colonies by MALDI-TOF MS-MS. However, this cannot be applied to noncultivable microorganisms, like most environmental bacteria. Detection of genetic structures coding for "drug resistance" means that associated proteins are thus required, which can be achieved by metagenomic and whole genome sequencing.

By analyzing DNA (i.e., with next-generation sequencing (NGS) techniques), complex microbial populations can be analyzed in the same round. This leads to microbial diversity, genetic adaptation, or interactions data [117,118]. However, there are still many technical pitfalls for deploying these innovative methods in low-income countries: (i) the preservation of environmental RNA or the extraction of high-molecular-weight DNA is not straightforward [119], (ii) cDNA analysis (i.e., for viral RNA), which necessitates biomass collection, total RNA extraction, storage, and cDNA synthesis [120], (iii) the maintaining of a quantitative estimation to avoid losing the representativity of the sampling.


### How to Use Metagenomics?

Most environmental microorganisms are uncultivable, and culture-based methods cannot be used. Analysis of DNA is however always possible, allowing direct identification of microorganisms in sludge or in wastewater [121]. These studies are permitted by the drastic decrease in the price of next-generation sequencing (NGS), which is as low as 100 euros per bacterial whole genome. Metagenomic analysis consists of DNA sequencing, (i) without targeting specific DNA sequences (whole genome sequencing (WGS)), or (ii) targeting the 16S gene for bacteria and 18S gene for eukaryotes. Specific genes can also be targeted as those involved in drug resistance.

NGS generates a large number of small sequences (usually 100-200 base pairs), which must be compared to an international database. The quality and completeness of this database are thus key elements. For 16S RNA identification, the good quality of the international databases leads to reliable results. However, for environmental or new pathogen species, databases are still limited, impairing the identification. Similarly, whole-genome sequencing can be used to explore genetic diversity and to identify new enzymatic structures from the environment. However, this approach requires fully annotated genomes or, at least, previous identification of the specific genetic signatures of drug resistance "cassettes" [122]. Finally, the quantity and quality of the DNA available, as well as the deepness of sequencing (i.e., the number of times a single base will be sequenced) will also limit the sensitivity of this approach, especially for complex metagenomes mixed with DNA from contaminants issued from water treatment plants. The deeper the analysis, the more expensive it is.

For environmental biotopes, a few hundred publications are already available using this approach, especially in fresh water contexts [118]. In these samples, some species were described as cosmopolitan, i.e., Actinobacteria [123], but many new species (especially viruses) were described [29,124]. This diversity concerns microbial eukaryotes as well (nematodes, protists, fungi), found in marine sediments, which can play pivotal roles in maintaining ecosystem function [125]. Archaea have also been described in sediments [126,127]. Investigations in ponds [128] and oceans [129] have also highlighted an unknown and rapid turnover of viruses and microorganisms. New viruses (nucleocytoplasmic large DNA viruses) were discovered to play a crucial ecological role in the sea by accelerating the turnover of their unicellular hosts or by causing diseases in animals. Their abundance was extremely high, with up to 10 4 -10 5 genomes mL −1 in the photic zone.

They also used 18S rDNA to investigate eukaryotic microbiota in freshwater [130] and seawater samples [131]. Cytochrome oxidase 1 barcodes were used to identify macroinvertebrates in benthic samples [132,133]. The shotgun sequencing approach was more recently used to investigate microbial and viral diversity in sea water [134,135]. Whole genome sequencing (WGS) was also used to reveal new structures involved in metabolic pathways or mutant proteins [136][137][138][139][140], or nucleocytoplasmic large DNA viruses [141].


### Technical Approaches and Pitfalls in Genomic Analysis

The first step of metagenomic approaches is to obtain good-quality DNA or cDNA, which needs (i) biomass collection, (ii) DNA/RNA preservation, (iii) total RNA extraction, and cDNA synthesis [119]. Preservation of RNA from environmental biotopes or extraction of high-molecular-weight DNA are not straightforward [142] and require quantitative approaches, to ensure the representativity of the material [129]. The high-molecular-weight DNA will be used as a template for sequencing libraries of different sizes (200/300 bp to 3/5 kb) to facilitate genome assembly. When the DNA quantity is not sufficient in the sample, new strategies of whole genome amplification can be used. However, the quality of the data obtained in this way is usually very low. Up to 90% of the sequences generated can be from the self-amplification of primers. At the same time, under-represented DNA cannot be fully sequenced according to the deepness of the sequencing step, and the choice of the polymerase used for amplification will impact the quality and representativity of data [143].

Using the WGS approach, reconstitution of whole genomes from rough data obtained from a mix of organisms such as biofilm is challenging, as a lot of contigs will contain sequences not matching the genome guides [144]. For this analysis step, several software needed to be applied, i.e., sequence trimmers, duplicate remover, and assemblers (such as SOAPdenovo) [145,146]. Protein-coding genes (open read frames, ORFs) are predicted using a gene-finding algorithm such as MetaGeneMark [147][148][149][150][151][152][153][154][155]. Functional annotation can be performed using a BLASTP search, first against the reference genome and then against the database of nr protein sequences.

In addition to this whole genome approach, PCR/sequencing techniques targeting 16S [156] or 18S tRNA can be used to simplify the analysis. Most publications used the V3-V4 region of the rRNA gene, which contains very-well-conserved sequences bordering hypervariable regions. The high-throughput sequencing of PCR products is performed, and the results are trimmed. Trimming of the reads also requires several steps: (i) paired-end reads are filtered for quality control (using QIIME v1.7, for example [157]), (ii) tags attached to the sequences are compared against the Gold reference database to detect chimera sequences and to remove them [158], (iii) sequences with effective tags are assigned to the same operational taxonomic units (OTU) at 97% similarity using software such as UPARSE [159], and (iv) representative sequences are annotated against the small ribosomal subunit rRNA SILVA database [160]. The relative abundance of a given taxon in a biofilm is calculated as the percentage of the number of sequences assigned to this taxon divided by the total number of sequences assigned to all the taxa in the community. For sequences absent from databases, the so-called de novo strategy can be used. It consists of comparing the sequences from a dataset and then grouping the sequences by similarity into clusters that elect a consensus sequence. The consensus sequence can, in turn, be annotated by a database, defining an unknown species. Once the taxonomic assignment has been completed, the metagenomic profiles of species present in each sample will be established.

Compared to WGS, this approach offers a higher selectivity and generates a small set of data, which is easier to handle to explore species. However, the capacity of PCR to amplify the 16S gene depends on the primers used. The diversity detected is thus more limited than with WGS and the identification of new or divergent microorganisms is difficult due to the lack of accurate databases and/or adapted primers. The coverage of sequencing (i.e., the number of times a specific base is determined) will also impact the detection of variants or rare microorganisms. Preliminary studies must be conducted to determine the coverage needed to reach a plateau in the genus or species diversity detected. Some nontranscribable sequences, such as ITS, can also be used for barcoding, especially for eukaryotes [161], but this approach underestimates the diversity of the species [162] to a few key species rather than the whole genus [163]. Specific analysis of antibiotic resistance genes (ARG) can be performed in the same line as 16S using NGS. The NCBI nonredundant database (NCBInr) is used to predict their phylogenetic origin. Putative mobile genetic elements (MGEs) are then searched for and screened on the PFAM and TIGRFAMS databases [148,149]. The abundance and diversity of ARG families in the metagenomic raw sequence data sets are then analyzed by screening with the resistance-gene-specific profile hidden Markov model (HMM) with packages such as the HMMER package [150] and/or with the same data-processing workflow as for soil microbiota [151]. Predicted protein sequences can be compared to CARD [152], ARDB [153], and ResFinder databases [154] using BLASTP to ARG.

Mobile genetic elements (e.g., ICEs or transposons) are also important to detect, which can be performed using BLASTp through the ICEberg database [155]. Inspection of microsynteny (small scale of synteny) can also be performed on regions surrounding resistance genes to detect mobile elements (e.g., transposase-encoding genes) or their traces. In addition, prophages and clustered regularly interspaced short palindromic repeats (CRISPR) can be identified using PHAST and CRISPRfinder, as previously described [139]. To further assess potential ARG mobility, the assembled metagenomic contigs are aligned to plasmid genome sequences (available in the NCBI RefSeq database). Antibiotic-resistant ORFs are considered colocalized with an MGE if they shared a contig with an MGE ORF.


### Questioning the Relevance of the Results of Metagenomic Analysis

After metagenomic analysis, questions about the relevance of the microorganism identified must be considered, as well as the differentiation of dead or living organisms [164].

When technical problems are solved to assure that the data and the sequences obtained are of good quality, the basic questions of ecology reappear. In a particular biotope, what will be the relevance of a microorganism detected in a specific site and at a precise time? What will be the turnover of the microorganisms and the risk of missing them? These questions should be addressed before determining the sampling strategy. Indeed, microbial populations seem very spatially structured in biotopes even when no physical barriers separate them. The sampling strategy can induce huge biases in the results. Diverse parameters can drive phenotypic and genotypic frequency variations in microbial communities and influence the extent and structure of microbial diversity [122,139]. The collection of representative samples thus requires careful consideration of the environmental context and a clear definition of the objectives of the sampling. The place of sampling in the biotope impacts the results, for example, in wastewater pipes [165]. Biofilms in the top and bottom of pipes were found to contain different compositions and abundances of sulfide-oxidizing and sulfate-reducing bacteria. In the same line, microbiota were found to be different in dairy lagoon wastewaters even if they were in open contact [166]. The time and season of sampling are also very important, especially in relation to climate perturbations. Local clustering was shown in the open area [167] or in ponds in relation to rainfall and storms [60] or after typhoons [168]. All environmental parameters drive phenotypic and genotypic frequency variations in microbial communities and thus control the extent and structure of microbial diversity [122,169].

Metagenomics is thus a powerful approach to explore biodiversity in environmental water. However, questions on the relevance of data still exist, and the strategy of sampling will be the most important factor leading to representativity of the results.


## Conclusions

Rapid, inadequate, unplanned urbanization and high population densities are associated with poor environmental and sanitary contexts. In low-income countries, urban runoffs or open sewage channels are major sources of fecal contamination of the environment. Surface waters and urban aquatic ecosystems are thus collecting and concentrating pathogen populations, which secondarily mix with environmental microorganisms. For these pathogens, genetic drift, chemical pressure, and lateral genetic transfer facilitate acquisition of virulence, long-term survival, and multidrug resistance genes. This contributes to the emergence or re-emergence of waterborne pathogens. Virus emergence from rural (and forest) areas like Ebola or Lassa are well documented by the mass-media, whereas no attention is paid to bacteria evolving to be highly resistant in crowed suburbs of tropical cities. The use of urban surface water for bathing, farming, fishing, or as an unimproved drinking-water source means that pathogens and contaminants are discharged in water resources and mix with environmental aquatic microbiota, before coming back to humans. In the same time, between 2000 and 2010, the consumption of antibiotic drugs increased by 35% [22] with a huge increase in drug resistance and deaths. The emergency is to preserve the efficacy of existing drugs for future generations [170]. Modern analytical processes, i.e., metagenomic approaches, have to be implemented to comprehensively study this complex biotope as many of these microorganisms are not cultivable and escape classical microbiology approaches. In the same line, the places to carry out sampling to study bacteria drift cannot be simply water and sediments as classically done but must take into account new habitats associated with floating macroaggregates, especially plastics. The pervasiveness of plastics is said to be the ultimate geological proxy for characterizing our new Anthropocene era. The ubiquity of plastics reshapes aquatic habitats, deeply impacting microbial communities and their interactions. Resistance gene transfer pathways might thus be noticeably enhanced, with huge potential epidemiological consequences. 

## Figure 1 .
1Plastics and garbage in an open sewer network; (A-D) Mahajanga, Madagascar, 2014, (A,B) open sewer network, (C) standpipe in a flooded area, (D) garbage pen in an area liable to flooding; (E,F) Ébrié Lagoon in Abidjan, floating plastics.

## Figure 1 .
1Plastics and garbage in an open sewer network; (A-D) Mahajanga, Madagascar, 2014, (A,B) open sewer network, (C) standpipe in a flooded area, (D) garbage pen in an area liable to flooding; (E,F) Ébrié Lagoon in Abidjan, floating plastics.

## Author
Contributions: Conceptualization, R.J. and P.C.; Writing-Original Draft Preparation, A.B.; Writing-Review & Editing, A.B., P.C., P.H., M.A. and R.J.; Supervision, R.J. All authors have read and agreed to the published version of the manuscript. Funding: This research was funded by the MALINEA project (an ACF/GRET/Institut Pasteur International Network program on malnutrition and infections) funded by the Fond de Solidarité Prioritaire of the French Foreign Office. It also benefited from the support of the Key Initiative Sea and Coast of the Montpellier University of Excellence (KIM Sea & Coast, MUSE). The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication. 35. Li, L.; Mendis, N.; Trigui, H.; Oliver, J.D.; Faucher, S.P. The importance of the viable but non-culturable state in human bacterial pathogens. Front. Microbiol. 2014, 5, 258. [CrossRef] 36. Durso, L.M.; Miller, D.N.; Henry, C.G. Impact of vegetative treatment systems on multiple measures of antibiotic resistance in agricultural Wastewater. Int. J. Environ. Res. Public Health 2018, 15, 1295. [CrossRef] [PubMed] 37. Obayiuwana, A.; Ogunjobi, A.; Yang, M.; Ibekwe, M. Characterization of bacterial communities and their antibiotic resistance profiles in wastewaters obtained from pharmaceutical facilities in Lagos and Ogun States, Nigeria. Int. J. Environ. Res. Public Health 2018, 15, 1365. [CrossRef] [PubMed] 38. Cahill, N.; O'Connor, L.; Mahon, B.; Varley, A.; McGrath, E.; Ryan, P.; Cormican, M.; Brehony, C.; Jolley, K.A.; Maiden, M.C.; et al. Hospital effluent: A reservoir for carbapenemase producing Enterobacterales? Sci. Total Environ. 2019, 672, 618-624. [CrossRef] [PubMed] 39. Rizzo, L.; Manaia, C.; Merlin, C.; Schwartz, T.; Dagot, C.; Ploy, M.C.; Michael, I.; Fatta-Kassinos, D. Urban wastewater treatment plants as hotspots for antibiotic resistant bacteria and genes spread into the environment: A review. Sci. Total Environ. 2013, 447, 345-360. [CrossRef] [PubMed] 40. Brown, P.C.; Borowska, E.; Schwartz, T.; Horn, H. Impact of the particulate matter from wastewater discharge on the abundance of antibiotic resistance genes and facultative pathogenic bacteria in downstream river sediments. Sci. Total Environ. 2019, 649, 1171-1178. [CrossRef] 41. Cabello, F.C.; Godfrey, H.P.; Tomova, A.; Ivanova, L.; Dölz, H.J.; Millanao, A.; Buschmann, A.H. Antimicrobial use in aquaculture re-examined: Its relevance to antimicrobial resistance and to animal and human health. Environ. Microbiol. 2013, 15, 1917-1942. [CrossRef] 42. Cabello, F.C.; Godfrey, H.P.; Buschmann, A.H.; Dölz, H.J. Aquaculture as yet another environmental gateway to the development and globalisation of antimicrobial resistance. Lancet Infect. Dis. 2016, 16, e127-e133. [CrossRef] 43. Muniesa, M.; Imamovic, L.; Jofre, J. Bacteriophages and genetic mobilization in sewage and faecally polluted environments. Microb. Biotechnol. 2011, 4, 725-734. [CrossRef] 44. Talukdar, P.K.; Rahman, M.; Rahman, M.; Nabi, A.; Islam, Z.; Hoque, M.M.; Endtz, H.P.; Islam, M.A. Antimicrobial resistance, virulence factors and genetic diversity of Escherichia coli isolates from household water supply in Dhaka, Bangladesh. PLoS ONE 2013, 8, e61090. [CrossRef] 45. Marin, M.A.; Thompson, C.C.; Freitas, F.S.; Fonseca1, E.L.; Aboderin, A.O.; Zailani, S.B.; Quartey, N.K.E.; Okeke, I.N.; Vicente, A.C.P. Cholera outbreaks in Nigeria are associated with multidrug resistant atypical El Tor and non-O1/non-O139 Vibrio cholerae. PLoS Negl. Trop. Dis. 2013, 7, e2049. [CrossRef] 46. Linares, J.F.; Gustafsson, I.; Baquero, F.; Martinez, J.L. Antibiotics as intermicrobial signaling agents instead of weapons. Proc. Natl. Acad. Sci. USA 2006, 103, 19484-19489. [CrossRef] [PubMed] 47. Calabrese, E.J.; Hoffmann, G.R.; Stanek, E.J.; Nascarella, M.A. Hormesis in high-throughput screening of antibacterial compounds in E coli. Hum. Exp. Toxicol. 2010, 29, 667-677. [CrossRef] [PubMed] 48. Tello, A.; Austin, B.; Telfer, T.C. Selective pressure of antibiotic pollution on bacteria of importance to public health. Environ. Health Perspect. 2012, 120, 1100-1106. [CrossRef] [PubMed] 49. Wang, Z.; Zhang, X.-X.; Huang, K.; Miao, Y.; Shi, P.; Liu, B.; Long, C.; Li, A. Metagenomic profiling of antibiotic resistance genes and mobile genetic elements in a tannery wastewater treatment plant. PLoS ONE 2013, 8, e76079. [CrossRef] [PubMed] 50. D'Costa, V.M.; McGrann, K.M.; Hughes, D.W.; Wright, G.D. Sampling the Antibiotic Resistome. Science 2006, 311, 374-377. [CrossRef] [PubMed] 51. Martinez, J.L. Recent advances on antibiotic resistance genes. In Recent Advances in Marine Biotechnology. Molecular Genetics of Marine Organisms, Vol 10; Fingerman, M., Nagabhushanam, R., Eds.; Taylor & Francis: Oxford, UK, 2003; pp. 13-32. 52. Huerta, B.; Marti, E.; Gros, M.; López, P.; Pompêo, M.; Armengol, J.; Barceló, D.; Balcázar, J.L.; Rodríguez-Mozaz, S.; Marcé, R. Exploring the links between antibiotic occurrence, antibiotic resistance, and bacterial communities in water supply reservoirs. Sci. Total Environ. 2013, 456, 161-170. [CrossRef] 53. Rosenfeld, W.D.; Zobell, C.E. Antibiotic production by marine microorganisms. J Bacteriol. 1947, 54, 393-398. [CrossRef] 54. Saz, A.K.; Watson, S.; Brown, S.R.; Lowery, D. Antimicrobial activity of marine waters. I. Macromolecular nature of antistaphylococcal factor. Limnol. Oceanogr. 1963, 8, 63-67. [CrossRef]
© 2020 by the authors. Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (http://creativecommons.org/licenses/by/4.0/).
Acknowledgments:We thank Renaud Piarroux for his expert advice concerning this text, particularly for the sections related to Vibrio.Conflicts of Interest:The authors declare no conflict of interest.
Urban Environments in Low-Income and Lower Middle-Income Countries: Policy Performance Indicators at the Subnational Level. C Christopher, R Estol-Peixoto, E Hartjes, A Rampton, P Ritger, H Waukau, Christopher, C.; Estol-Peixoto, R.; Hartjes, E.; Rampton, A.; Ritger, P.; Waukau, H. Urban Environments in Low-Income and Lower Middle-Income Countries: Policy Performance Indicators at the Subnational Level;

Sanitation and Hygiene: 2017; Update and SDG Baselines; World Health Organization (WHO) and the United Nations Children's Fund (UNICEF). Geneva, SwitzerlandUNICEF. Progress on Drinking WaterUNICEF. Progress on Drinking Water, Sanitation and Hygiene: 2017; Update and SDG Baselines; World Health Organization (WHO) and the United Nations Children's Fund (UNICEF): Geneva, Switzerland, 2017.

World Urbanization Prospects, the 2015 Revision. United Nations Department of Economics and Social Affairs. Population DivisionUnited Nations. World Urbanization Prospects, the 2015 Revision; United Nations Department of Economics and Social Affairs, Population Division: New York, NY, USA, 2015.

Environmental threats impacts on drinking-water microbial quality in low-income urban water supplies, Case of Madagascar. A Bastaraud, J M Rakotondramanga, J Mahazosaotra, N Ravaonindrina, R Jambou, 10.3390/w1010145010Bastaraud, A.; Rakotondramanga, J.M.; Mahazosaotra, J.; Ravaonindrina, N.; Jambou, R. Environmental threats impacts on drinking-water microbial quality in low-income urban water supplies, Case of Madagascar. Waters 2018, 10, 1450. [CrossRef]

Point and non-point microbial source pollution: A case study of Delhi. P Jamwal, A Mittal, J Mouchel, 10.1016/j.pce.2008.09.005Phys. Chem. Earth Parts A. 36Jamwal, P.; Mittal, A.; Mouchel, J. Point and non-point microbial source pollution: A case study of Delhi. Phys. Chem. Earth Parts A/B/C 2011, 36, 490-499. [CrossRef]

Enterotoxigenic Escherichia coli in developing countries: Epidemiology, Microbiology, Clinical features, Treatment, and Prevention. F Qadri, A M Svennerholm, A S G Faruque, R B Sack, 10.1128/CMR.18.3.465-483.2005Clin. Microbiol Rev. 18PubMedQadri, F.; Svennerholm, A.M.; Faruque, A.S.G.; Sack, R.B. Enterotoxigenic Escherichia coli in developing countries: Epidemiology, Microbiology, Clinical features, Treatment, and Prevention. Clin. Microbiol Rev. 2005, 18, 465-483. [CrossRef] [PubMed]

Impact of rapid urbanization on the rates of infection by Vibrio cholerae O1 and enterotoxigenic Escherichia coli in Dhaka. F Chowdhury, M A Rahman, Y A Begum, A I Khan, A S G Faruque, N C Saha, N I Baby, M A Malek, A R Kumar, A M Svennerholm, 10.1371/journal.pntd.0000999Bangldesh. PLoS. Negl. Trop. Dis. 5e999. [CrossRef. PubMedChowdhury, F.; Rahman, M.A.; Begum, Y.A.; Khan, A.I.; Faruque, A.S.G.; Saha, N.C.; Baby, N.I.; Malek, M.A.; Kumar, A.R.; Svennerholm, A.M.; et al. Impact of rapid urbanization on the rates of infection by Vibrio cholerae O1 and enterotoxigenic Escherichia coli in Dhaka, Bangldesh. PLoS. Negl. Trop. Dis. 2011, 5, e999. [CrossRef] [PubMed]

Quantification of microbial risks to human health caused by waterborne viruses and bacteria in an urban slum. A Y Katukiza, M Ronteltap, P Van Der Steen, J W A Foppen, P N L Lens, 10.1111/jam.12368J. Appl. Microbiol. 116PubMedKatukiza, A.Y.; Ronteltap, M.; van der Steen, P.; Foppen, J.W.A.; Lens, P.N.L. Quantification of microbial risks to human health caused by waterborne viruses and bacteria in an urban slum. J. Appl. Microbiol. 2014, 116, 447-463. [CrossRef] [PubMed]

Ecology and Evolution of Waterborne Pathogens and Indicator Organisms. The National Academies PressWashington, DC, USANational Research CouncilIn Indicators for Waterborne PathogensNational Research Council. Ecology and Evolution of Waterborne Pathogens and Indicator Organisms. In Indicators for Waterborne Pathogens; The National Academies Press: Washington, DC, USA, 2004; pp. 109-163.

International dissemination of epidemic Vibrio cholerae by cargo ship ballast and other non potable waters. S A Mccarthy, F M Khambaty, 10.1128/AEM.60.7.2597-2601.1994Appl. Environ. Microbiol. 60McCarthy, S.A.; Khambaty, F.M. International dissemination of epidemic Vibrio cholerae by cargo ship ballast and other non potable waters. Appl. Environ. Microbiol. 1994, 60, 2597-2601. [CrossRef]

Global spread of microorganisms by ships. G M Ruiz, T K Rawlings, F C Dobbs, L A Drake, T Mullady, A Huq, R R Colwell, 10.1038/35040695Nature. 408Ruiz, G.M.; Rawlings, T.K.; Dobbs, F.C.; Drake, L.A.; Mullady, T.; Huq, A.; Colwell, R.R. Global spread of microorganisms by ships. Nature 2000, 408, 49-50. [CrossRef]

Origin and evolution of antibiotic resistance, the common mechanisms of emergence and spread in water bodies. A Lupo, S Coyne, T U Berendonk, 10.3389/fmicb.2012.00018Front. Microbiol. 318Lupo, A.; Coyne, S.; Berendonk, T.U. Origin and evolution of antibiotic resistance, the common mechanisms of emergence and spread in water bodies. Front. Microbiol. 2012, 3, 18. [CrossRef]

Un-Habitat, The Challenge of Slums: Global Report on Human Settlements. 15UN-Habitat, The Challenge of Slums: Global Report on Human Settlements 2003. Manag. Environ. Qual. Int. J. 2003, 15, 337-338.

. P J Landrigan, R Fuller, N J R Acosta, O Adeyi, R Arnold, N N Basu, A B Baldé, R Bertollini, S Bose-O&apos;reilly, J I Boufford, Urbanization and Development Emerging Futures. World Cities ReportLandrigan, P.J.; Fuller, R.; Acosta, N.J.R.; Adeyi, O.; Arnold, R.; Basu, N.N.; Baldé, A.B.; Bertollini, R.; Bose-O'Reilly, S.; Boufford, J.I.; et al. World Cities Report 2016: Urbanization and Development Emerging Futures;

. Un-Habitat, Nairobi, KenyaUN-Habitat: Nairobi, Kenya, 2016.

Antibiotic Pollution in the Environment: A Review. R Gothwal, T Shashidhar, 10.1002/clen.20130098943Gothwal, R.; Shashidhar, T. Antibiotic Pollution in the Environment: A Review. CSAWAC 2015, 43, 463-620. [CrossRef]

Antibiotics in the aquatic environment-A review-Part II. K Kümmerer, 10.1016/j.chemosphere.2008.12.006Chemosphere. 75PubMedKümmerer, K. Antibiotics in the aquatic environment-A review-Part II. Chemosphere 2009, 75, 435-441. [CrossRef] [PubMed]

Antimicrobial drug resistance among clinically relevant bacterial isolates in sub-Saharan Africa: A systematic review. S J Leopold, F Van Leth, H Tarekegn, C Schultsz, 10.1093/jac/dku176J. Antimicrob. Chemother. 69PubMedLeopold, S.J.; van Leth, F.; Tarekegn, H.; Schultsz, C. Antimicrobial drug resistance among clinically relevant bacterial isolates in sub-Saharan Africa: A systematic review. J. Antimicrob. Chemother. 2014, 69, 2337-2353. [CrossRef] [PubMed]

Antimicrobial resistance in Africa: A systematic review. B T Tadesse, E A Ashley, S Ongarello, J Havumaki, M Wijegoonewardena, I J González, S Dittrich, 10.1186/s12879-017-2713-1BMC Infect. Dis. 17616PubMedTadesse, B.T.; Ashley, E.A.; Ongarello, S.; Havumaki, J.; Wijegoonewardena, M.; González, I.J.; Dittrich, S. Antimicrobial resistance in Africa: A systematic review. BMC Infect. Dis. 2017, 17, 616. [CrossRef] [PubMed]

Dynamics of antimicrobial resistance in intestinal Escherichia coli from children in community settings in South Asia and sub-Saharan Africa. D J Ingle, M M Levine, K L Kotloff, K E Holt, R M Robins-Browne, 10.1038/s41564-018-0217-4Nat. Microbiol. 3Ingle, D.J.; Levine, M.M.; Kotloff, K.L.; Holt, K.E.; Robins-Browne, R.M. Dynamics of antimicrobial resistance in intestinal Escherichia coli from children in community settings in South Asia and sub-Saharan Africa. Nat. Microbiol. 2018, 3, 1063-1073. [CrossRef]

Wastewater used for urban agriculture in West Africa as a reservoir for antibacterial resistance dissemination. B P Bougnom, C Zongo, A Mcnally, V Ricci, F.-X Etoac, S Thiele-Bruhn, L J Piddock, 10.1016/j.envres.2018.09.022Environ. Res. 168Bougnom, B.P.; Zongo, C.; McNally, A.; Ricci, V.; Etoac, F.-X.; Thiele-Bruhn, S.; Piddock, L.J.V. Wastewater used for urban agriculture in West Africa as a reservoir for antibacterial resistance dissemination. Environ. Res. 2019, 168, 14-24. [CrossRef]

The structure and diversity of human, animal and environmental resistomes. C Pal, J Bengtsson-Palme, E Kristiansson, D G J Larsson, 10.1186/s40168-016-0199-54MicrobiomePal, C.; Bengtsson-Palme, J.; Kristiansson, E.; Larsson, D.G.J. The structure and diversity of human, animal and environmental resistomes. Microbiome 2016, 4, 54. [CrossRef]

Interconnected microbiomes and resistomes in low-income human habitats. E C Pehrsson, P Tsukayama, S Patel, M Mejía-Bautista, G Sosa-Soto, K M Navarrete, M Calderon, L Cabrera, W Hoyos-Arango, M T Bertoli, 10.1038/nature17672Nature. 533Pehrsson, E.C.; Tsukayama, P.; Patel, S.; Mejía-Bautista, M.; Sosa-Soto, G.; Navarrete, K.M.; Calderon, M.; Cabrera, L.; Hoyos-Arango, W.; Bertoli, M.T.; et al. Interconnected microbiomes and resistomes in low-income human habitats. Nature 2016, 533, 212-216. [CrossRef]

Bacterial communities and potential waterborne pathogens within the typical urban surface waters. D Jin, X Kong, B Cui, S Jin, Y Xie, X Wang, Y Deng, 10.1038/s41598-018-31706-wSci. Rep. 8Jin, D.; Kong, X.; Cui, B.; Jin, S.; Xie, Y.; Wang, X.; Deng, Y. Bacterial communities and potential waterborne pathogens within the typical urban surface waters. Sci. Rep. 2018, 8, 13368. [CrossRef]

Potential pathogen communities in highly polluted river ecosystems: Geographical distribution and environmental influence. Y Yang, Y Hou, M Ma, A Zhan, 10.1007/s13280-019-01184-zAmbio. 49Yang, Y.; Hou, Y.; Ma, M.; Zhan, A. Potential pathogen communities in highly polluted river ecosystems: Geographical distribution and environmental influence. Ambio 2019, 49, 197-207. [CrossRef]

Life in the 'Plastisphere': Microbial communities on plastic marine debris. E R Zettler, T J Mincer, L A Amaral-Zettler, 10.1021/es401288xEnviron. Sci. Technol. 47Zettler, E.R.; Mincer, T.J.; Amaral-Zettler, L.A. Life in the 'Plastisphere': Microbial communities on plastic marine debris. Environ. Sci. Technol. 2013, 47, 7137-7146. [CrossRef]

Using the class 1 integron-integrase gene as a proxy for anthropogenic pollution. M R Gillings, W H Gaze, A Pruden, K Smalla, J M Tiedje, Y.-G Zhu, 10.1038/ismej.2014.226ISME J. 9PubMedGillings, M.R.; Gaze, W.H.; Pruden, A.; Smalla, K.; Tiedje, J.M.; Zhu, Y.-G. Using the class 1 integron-integrase gene as a proxy for anthropogenic pollution. ISME J. 2015, 9, 1269-1279. [CrossRef] [PubMed]

Tackling antibiotic resistance: The environmental framework. T U Berendonk, C M Manaia, C Merlin, D Fatta-Kassinos, E Cytryn, F Walsh, H Bürgmann, G Sørum, M Norström, M.-N Pons, 10.1038/nrmicro3439Nat. Rev. Microbiol. 13PubMedBerendonk, T.U.; Manaia, C.M.; Merlin, C.; Fatta-Kassinos, D.; Cytryn, E.; Walsh, F.; Bürgmann, H.; Sørum, G.; Norström, M.; Pons, M.-N.; et al. Tackling antibiotic resistance: The environmental framework. Nat. Rev. Microbiol. 2015, 13, 310-317. [CrossRef] [PubMed]

Nepalese origin of cholera epidemic in Haiti. R R Frerichs, P S Keim, R Barrais, R Piarroux, 10.1111/j.1469-0691.2012.03841.xE158-E163Clin. Microbiol. Infect. 18PubMedFrerichs, R.R.; Keim, P.S.; Barrais, R.; Piarroux, R. Nepalese origin of cholera epidemic in Haiti. Clin. Microbiol. Infect. 2012, b18, E158-E163. [CrossRef] [PubMed]

Predicting the Distribution of Vibrio spp. in the Chesapeake Bay: A Vibrio cholerae Case Study. G Constantin De Magny, W Long, C W Brown, R R Hood, A Huq, R Murtugudde, R R Colwell, 10.1007/s10393-009-0273-6EcoHealth. 6PubMedConstantin de Magny, G.; Long, W.; Brown, C.W.; Hood, R.R.; Huq, A.; Murtugudde, R.; Colwell, R.R. Predicting the Distribution of Vibrio spp. in the Chesapeake Bay: A Vibrio cholerae Case Study. EcoHealth 2009, 6, 378-389. [CrossRef] [PubMed]

Lakes as source of cholera outbreaks, Democratic Republic of Congo. D Bompangue, P Giraudoux, P Handschumacher, M Piarroux, B Sudre, M Ekwanzala, I Kebela, R Piarroux, 10.3201/eid1405.071260Emerg. Infect. Dis. 14PubMedBompangue, D.; Giraudoux, P.; Handschumacher, P.; Piarroux, M.; Sudre, B.; Ekwanzala, M.; Kebela, I.; Piarroux, R. Lakes as source of cholera outbreaks, Democratic Republic of Congo. Emerg. Infect. Dis. 2008, 14, 798-800. [CrossRef] [PubMed]

Temperature-induced recovery of Vibrio cholerae from the viable but nonculturable state: Growth or resuscitation? Microbiol. J Ravel, I T Knight, C E Monahan, R T Hill, R R Colwell, 10.1099/13500872-141-2-377141PubMedRavel, J.; Knight, I.T.; Monahan, C.E.; Hill, R.T.; Colwell, R.R. Temperature-induced recovery of Vibrio cholerae from the viable but nonculturable state: Growth or resuscitation? Microbiol 1995, 141, 377-383. [CrossRef] [PubMed]

Extreme water-related weather events and waterborne disease. K F Cann, D R Thomas, R L Salmon, A P Wyn-Jones, 10.1017/S0950268812001653Epidemiol. Infect. 141Cann, K.F.; Thomas, D.R.; Salmon, R.L.; Wyn-Jones, A.P. Extreme water-related weather events and waterborne disease. Epidemiol. Infect. 2013, 141, 671-686. [CrossRef]

Occurrence of Vibrio cholerae in water reservoirs of Burkina Faso. S Kaboré, P Cecchi, T Mosser, M Toubiana, O Traoré, A S Ouattara, A S Traoré, N Barro, R R Colwell, P Monfort, 10.1016/j.resmic.2017.08.004Res. Microbiol. 169Kaboré, S.; Cecchi, P.; Mosser, T.; Toubiana, M.; Traoré, O.; Ouattara, A.S.; Traoré, A.S.; Barro, N.; Colwell, R.R.; Monfort, P. Occurrence of Vibrio cholerae in water reservoirs of Burkina Faso. Res. Microbiol. 2018, 169, 1-10. [CrossRef]

Removal of fecal indicators, pathogenic bacteria, Adenovirus, Cryptosporidium and Giardia (oo)cysts in waste stabilization ponds in Northern and Eastern Australia. M Sheludchenko, A Padovan, M Katouli, H Stratton, 10.3390/ijerph13010096Int. J. Environ. Res. Public Health. 1396Sheludchenko, M.; Padovan, A.; Katouli, M.; Stratton, H. Removal of fecal indicators, pathogenic bacteria, Adenovirus, Cryptosporidium and Giardia (oo)cysts in waste stabilization ponds in Northern and Eastern Australia. Int. J. Environ. Res. Public Health 2016, 13, 96. [CrossRef]

Distribution and significance of heterotrophic marine bacteria with antibacterial activity. S Nair, U Simidu, 10.1128/AEM.53.12.2957-2962.1987Appl. Environ. Microbiol. 53Nair, S.; Simidu, U. Distribution and significance of heterotrophic marine bacteria with antibacterial activity. Appl. Environ. Microbiol. 1987, 53, 2957-2962. [CrossRef]

Antagonistic interactions among marine pelagic bacteria. R A Long, F Azam, 10.1128/AEM.67.11.4975-4983.2001Appl. Environ. Microbiol. 67Long, R.A.; Azam, F. Antagonistic interactions among marine pelagic bacteria. Appl. Environ. Microbiol. 2001, 67, 4975-4983. [CrossRef]

. J Davies, D Davies, 10.1128/MMBR.00016-10Origins and evolution of antibiotic resistance. Microbiol. Mol. Biol. Rev. 74PubMedDavies, J.; Davies, D. Origins and evolution of antibiotic resistance. Microbiol. Mol. Biol. Rev. 2010, 74, 417-433. [CrossRef] [PubMed]

Antibacterial properties of extracts of Ludwigia peploides subsp. montevidensis and Ludwigia grandiflora subsp. hexapetala during their cycle of development. I Smida, C Charpy-Roubaud, S Y Cherif, F Torre, G Audran, S Smiti, J Le Petit, 10.1016/j.aquabot.2014.10.011Aquat. Bot. 121Smida, I.; Charpy-Roubaud, C.; Cherif, S.Y.; Torre, F.; Audran, G.; Smiti, S.; Le Petit, J. Antibacterial properties of extracts of Ludwigia peploides subsp. montevidensis and Ludwigia grandiflora subsp. hexapetala during their cycle of development. Aquat. Bot. 2015, 121, 39-45. [CrossRef]

Allelopathic interactions involving benthic phototrophic microorganisms. J L Allen, L Ten-Hage, J Leflaive, 10.1111/1758-2229.12436Environ. Microbiol. Rep. 8PubMedAllen, J.L.; Ten-Hage, L.; Leflaive, J. Allelopathic interactions involving benthic phototrophic microorganisms. Environ. Microbiol. Rep. 2016, 8, 752-762. [CrossRef] [PubMed]

Cyanobacteria and eukaryotic microalgae as potential sources of antibiotics. G N A Senhorinho, G M Ross, J A Scott, 10.2216/14-092.1Phycologia. 54Senhorinho, G.N.A.; Ross, G.M.; Scott, J.A. Cyanobacteria and eukaryotic microalgae as potential sources of antibiotics. Phycologia 2015, 54, 271-282. [CrossRef]

Cyanobacterial allelochemicals but not cyanobacterial cells markedly reduce microbial community diversity. F Dias, J T Antunes, T Ribeiro, J Azevedo, V Vasconcelos, P N Leão, 10.3389/fmicb.2017.014958PubMedDias, F.; Antunes, J.T.; Ribeiro, T.; Azevedo, J.; Vasconcelos, V.; Leão, P.N. Cyanobacterial allelochemicals but not cyanobacterial cells markedly reduce microbial community diversity. Front. Microbiol. 2017, 8, 1495. [CrossRef] [PubMed]

Algal and cyanobacterial secondary metabolites in freshwaters: A comparison of allelopathic compounds and toxins. Freshwater Biol. J Leflaive, L Ten-Hage, 10.1111/j.1365-2427.2006.01689.x52Leflaive, J.; Ten-Hage, L. Algal and cyanobacterial secondary metabolites in freshwaters: A comparison of allelopathic compounds and toxins. Freshwater Biol. 2007, 52, 199-214. [CrossRef]

Antibiotic effects of three strains of chrysophytes (Ochromonas, Poterioochromonas) on freshwater bacterial isolates. J F Blom, J Pernthaler, 10.1111/j.1574-6941.2009.00800.xFEMS Microbiol. Ecol. 71PubMedBlom, J.F.; Pernthaler, J. Antibiotic effects of three strains of chrysophytes (Ochromonas, Poterioochromonas) on freshwater bacterial isolates. FEMS Microbiol. Ecol. 2009, 71, 281-290. [CrossRef] [PubMed]

The second skin: Ecological role of epibiotic biofilms on marine organisms. M Wahl, F Goecke, A Labes, S Dobretsov, F Weinberger, 10.3389/fmicb.2012.00292Front. Microbiol. 2012, 3, 292. [CrossRefWahl, M.; Goecke, F.; Labes, A.; Dobretsov, S.; Weinberger, F. The second skin: Ecological role of epibiotic biofilms on marine organisms. Front. Microbiol. 2012, 3, 292. [CrossRef]

Bacterial competition: Surviving and thriving in the microbial jungle. M E Hibbing, C Fuqua, M R Parsek, S B Peterson, 10.1038/nrmicro2259Nat. Rev. Microbiol. 8Hibbing, M.E.; Fuqua, C.; Parsek, M.R.; Peterson, S.B. Bacterial competition: Surviving and thriving in the microbial jungle. Nat. Rev. Microbiol. 2010, 8, 15-25. [CrossRef]

Adsorption of tetracycline on soil and sediment: Effects of pH and the presence of Cu(II). Z Zhang, K Sun, B Gao, G Zhang, X Liu, Y Zhao, 10.1016/j.jhazmat.2011.04.017J. Hazard. Mater. 190Zhang, Z.; Sun, K.; Gao, B.; Zhang, G.; Liu, X.; Zhao, Y. Adsorption of tetracycline on soil and sediment: Effects of pH and the presence of Cu(II). J. Hazard. Mater. 2011, 190, 856-862. [CrossRef]

Co-selection of antibiotic and metal resistance. C Baker-Austin, M S Wright, R Stepanauskas, J V Mcarthur, 10.1016/j.tim.2006.02.006Trends Microbiol. 14PubMedBaker-Austin, C.; Wright, M.S.; Stepanauskas, R.; McArthur, J.V. Co-selection of antibiotic and metal resistance. Trends Microbiol. 2006, 14, 176-182. [CrossRef] [PubMed]

Antibiotic resistance gene abundances correlate with metal and geochemical conditions in archived Scottish soils. C W Knapp, S M Mccluskey, B K Singh, C D Campbell, G Hudson, D W Graham, 10.1371/journal.pone.0027300PLoS ONE. 6PubMedKnapp, C.W.; McCluskey, S.M.; Singh, B.K.; Campbell, C.D.; Hudson, G.; Graham, D.W. Antibiotic resistance gene abundances correlate with metal and geochemical conditions in archived Scottish soils. PLoS ONE 2011, 6, e27300. [CrossRef] [PubMed]

Heavy metal driven co-selection of antibiotic resistance in soil and water bodies impacted by agriculture and aquaculture. C Seiler, T U Berendonk, 10.3389/fmicb.2012.00399Front. Microbiol. 3399PubMedSeiler, C.; Berendonk, T.U. Heavy metal driven co-selection of antibiotic resistance in soil and water bodies impacted by agriculture and aquaculture. Front. Microbiol. 2012, 3, 399. [CrossRef] [PubMed]

Bacterial metal resistance genes and metal bioavailability in contaminated sediments. S Roosa, R Wattiez, E Prygiel, L Lesven, G Billon, D C Gillan, 10.1016/j.envpol.2014.02.031Environ. Pollut. 189Roosa, S.; Wattiez, R.; Prygiel, E.; Lesven, L.; Billon, G.; Gillan, D.C. Bacterial metal resistance genes and metal bioavailability in contaminated sediments. Environ. Pollut. 2014, 189, 143-151. [CrossRef]

Sub-inhibitory concentrations of heavy metals facilitate the horizontal transfer of plasmid-mediated antibiotic resistance genes in water environment. Y Zhang, A Z Gu, T Cen, X Li, M He, D Li, J Chen, 10.1016/j.envpol.2018.01.032Environ. Pollut. 237Zhang, Y.; Gu, A.Z.; Cen, T.; Li, X.; He, M.; Li, D.; Chen, J. Sub-inhibitory concentrations of heavy metals facilitate the horizontal transfer of plasmid-mediated antibiotic resistance genes in water environment. Environ. Pollut. 2018, 237, 74-82. [CrossRef]

Antibiotic and heavy metal resistance in enterococci from coastal marine sediment. C Vignaroli, S Pasquaroli, A Di Cesare, G Mangiaterra, D Fattorini, F Biavasco, 10.1016/j.envpol.2018.02.073Environ. Pollut. 237Vignaroli, C.; Pasquaroli, S.; Di Cesare, A.; Mangiaterra, G.; Fattorini, D.; Biavasco, F. Antibiotic and heavy metal resistance in enterococci from coastal marine sediment. Environ. Pollut. 2018, 237, 406-413. [CrossRef]

Potential impact of environmental bacteriophages in spreading antibiotic resistance genes. M Muniesa, M Colomer-Lluch, J Jofre, 10.2217/fmb.13.32Future Microbiol. 8Muniesa, M.; Colomer-Lluch, M.; Jofre, J. Potential impact of environmental bacteriophages in spreading antibiotic resistance genes. Future Microbiol. 2013, 8, 739-751. [CrossRef]

Prevalence of lysogeny among soil bacteria and presence of 16S rRNA and trzN genes in viral-community DNA. D Ghosh, K Roy, K E Williamson, D C White, E Wommack, K L Sublette, M Radosevich, 10.1128/AEM.01435-07Appl. Environ. Microbiol. 74Ghosh, D.; Roy, K.; Williamson, K.E.; White, D.C.; Wommack, E.; Sublette, K.L.; Radosevich, M. Prevalence of lysogeny among soil bacteria and presence of 16S rRNA and trzN genes in viral-community DNA. Appl. Environ. Microbiol. 2008, 74, 495-502. [CrossRef]

Importance of prophages to evolution and virulence of bacterial pathogens. L.-C Fortier, O Sekulovic, 10.4161/viru.24498Virulence. 4Fortier, L.-C.; Sekulovic, O. Importance of prophages to evolution and virulence of bacterial pathogens. Virulence 2013, 4, 354-365. [CrossRef]

In vitro analysis of ISEcp1B-mediated mobilization of naturally occurring-Lactamase Gene blaCTX-M of Kluyvera ascorbata. M F Lartigue, L Poirel, D Aubert, P Nordmann, 10.1128/AAC.50.4.1282-1286.2006Antimicrob. Agents Chemother. 50Lartigue, M.F.; Poirel, L.; Aubert, D.; Nordmann, P. In vitro analysis of ISEcp1B-mediated mobilization of naturally occurring-Lactamase Gene blaCTX-M of Kluyvera ascorbata. Antimicrob. Agents Chemother. 2006, 50, 1282-1286. [CrossRef]

The role of integrons in antibiotic resistance gene capture. D A Rowe-Magnus, D Mazel, 10.1078/1438-4221-00197Int. J. Med. Microbiol. 292PubMedRowe-Magnus, D.A.; Mazel, D. The role of integrons in antibiotic resistance gene capture. Int. J. Med. Microbiol. 2002, 292, 115-125. [CrossRef] [PubMed]

Molecular characterization of SPM-1, a novel metallo-β-lactamase isolated in Latin America: Report from the SENTRY antimicrobial surveillance programme. M A Toleman, A M Simm, T A Murphy, A C Gales, D J Biedenbach, R N Jones, T R Walsh, 10.1093/jac/dkf210J. Antimicrob. Chemother. 50PubMedToleman, M.A.; Simm, A.M.; Murphy, T.A.; Gales, A.C.; Biedenbach, D.J.; Jones, R.N.; Walsh, T.R. Molecular characterization of SPM-1, a novel metallo-β-lactamase isolated in Latin America: Report from the SENTRY antimicrobial surveillance programme. J. Antimicrob. Chemother. 2002, 50, 673-679. [CrossRef] [PubMed]

Effects of water quality and post-harvest handling on microbiological contamination of lettuce at urban and peri-urban locations of Ouagadougou. J Dao, K Stenchly, O Traoré, P Amoah, A Buerkert, 10.3390/foods7120206Burkina Faso. 7206PubMedDao, J.; Stenchly, K.; Traoré, O.; Amoah, P.; Buerkert, A. Effects of water quality and post-harvest handling on microbiological contamination of lettuce at urban and peri-urban locations of Ouagadougou, Burkina Faso. Foods 2018, 7, 206. [CrossRef] [PubMed]

Urban Agriculture Practices and Health Problems Among Farmers Operating on a University Campus in Kumasi. N O B Ackerson, E Awuah, Ghana. Field Actions Science Reports, Special Issue. 1University of Science and Technology (KNUST) campus: KumasiAckerson, N.O.B.; Awuah, E. Urban Agriculture Practices and Health Problems Among Farmers Operating on a University Campus in Kumasi, Ghana. Field Actions Science Reports, Special Issue 1; University of Science and Technology (KNUST) campus: Kumasi, Ghana, 2010.

Prevalence of intestinal protozoan infestation among primary school children in Urban and peri-urban communities in Kumasi. W Walana, S C K Tay, P Tetteh, J B Ziem, Ghana. Sci. J. Public Health. 2Walana, W.; Tay, S.C.K.; Tetteh, P.; Ziem, J.B. Prevalence of intestinal protozoan infestation among primary school children in Urban and peri-urban communities in Kumasi, Ghana. Sci. J. Public Health 2014, 2, 52-57.

Mise en Place D'Aménagements Sanitaires ET Impact Sur L'éTat DE Santé Des Enfants D. P Handschumacher, L Ramananodraitsiory, A P Razakarintsalama, Andohatapenaka ET Ampefiloha Ambodirano. Handschumacher, P.; Ramananodraitsiory, L.; Razakarintsalama, A.P. Mise en Place D'Aménagements Sanitaires ET Impact Sur L'éTat DE Santé Des Enfants D'Andohatapenaka ET Ampefiloha Ambodirano, Tananarive (Madagascar);

. Msf-Orstom; Rapport, Orstom, Antananarivo, MadagascarRapport MSF-ORSTOM; ORSTOM: Antananarivo, Madagascar, 1997.

Variability of diarrheal diseases in children under 5 living in an urban setting: Observations in Rufisque. I Sy, P Handschumacher, K Wyss, G Cissé, B Lo, J L Piermay, M Tanner, Senegal. Med. Trop. 70Sy, I.; Handschumacher, P.; Wyss, K.; Cissé, G.; Lo, B.; Piermay, J.L.; Tanner, M. Variability of diarrheal diseases in children under 5 living in an urban setting: Observations in Rufisque, Senegal. Med. Trop. 2010, 70, 57-61.

Urbanization and waterborne pathogens emergence in low-income countries: where and how to conduct surveys?. A Bastaraud, P Cecchi, P Handschumacher, M Altmann, R Jambou, Int. J. Environ. Res. Public Health. in pressBastaraud, A.; Cecchi, P.; Handschumacher, P.; Altmann, M.; Jambou, R. Urbanization and waterborne pathogens emergence in low-income countries: where and how to conduct surveys? Int. J. Environ. Res. Public Health 2019, in press.

Diversity and functions of bacterial community in drinking water biofilms revealed by high-throughput sequencing. Y Chao, Y Mao, Z Wang, T Zhang, 10.1038/srep10044Sci. Rep. 5Chao, Y.; Mao, Y.; Wang, Z.; Zhang, T. Diversity and functions of bacterial community in drinking water biofilms revealed by high-throughput sequencing. Sci. Rep. 2015, 5, 10044. [CrossRef]

Megacities as sources for pathogenic bacteria in rivers and their fate downstream. W.-R Abraham, 10.1155/2011/798292Int. J. Microbiol. Abraham, W.-R. Megacities as sources for pathogenic bacteria in rivers and their fate downstream. Int. J. Microbiol. 2011, 2011, 798292. [CrossRef]

Transmission of Helicobacter pylori and the role of water and biofilms. S L Percival, J G Thomas, 10.2166/wh.2009.070J. Water Health. 7Percival, S.L.; Thomas, J.G. Transmission of Helicobacter pylori and the role of water and biofilms. J. Water Health 2009, 7, 469-477. [CrossRef]

The ecological habitat and transmission of Escherichia coli O157:H7. S M Chekabab, J Paquin-Veillette, C M Dozois, J Harel, 10.1111/1574-6968.12078FEMS Microbiol. Lett. 341Chekabab, S.M.; Paquin-Veillette, J.; Dozois, C.M.; Harel, J. The ecological habitat and transmission of Escherichia coli O157:H7. FEMS Microbiol. Lett. 2013, 341, 1-12. [CrossRef]

Enhanced biofilm formation and increased resistance to antimicrobial agents and bacterial invasion are caused by synergistic interactions in multispecies biofilms. M Burmølle, J S Webb, D Rao, L H Hansen, S J Sørensen, S Kjelleberg, 10.1128/AEM.03022-05Appl. Environ. Microbiol. 72PubMedBurmølle, M.; Webb, J.S.; Rao, D.; Hansen, L.H.; Sørensen, S.J.; Kjelleberg, S. Enhanced biofilm formation and increased resistance to antimicrobial agents and bacterial invasion are caused by synergistic interactions in multispecies biofilms. Appl. Environ. Microbiol. 2006, 72, 3916-3923. [CrossRef] [PubMed]

Sustaining rare marine microorganisms: Macroorganisms as repositories and dispersal agents of microbial diversity. M Troussellier, A Escalas, T Bouvier, D Mouillot, 10.3389/fmicb.2017.00947Front. Microbiol. 8947PubMedTroussellier, M.; Escalas, A.; Bouvier, T.; Mouillot, D. Sustaining rare marine microorganisms: Macroorganisms as repositories and dispersal agents of microbial diversity. Front. Microbiol. 2017, 8, 947. [CrossRef] [PubMed]

Challenges and emerging solutions to the land-based plastic waste issue in Africa. J Jambeck, B D Hardesty, A L Brooks, T Friend, K Telekic, J Fabres, Y Beaudoin, A Bamba, J Francis, A J Ribbink, 10.1016/j.marpol.2017.10.041Mar. Policy. 96Jambeck, J.; Hardesty, B.D.; Brooks, A.L.; Friend, T.; Telekic, K.; Fabres, J.; Beaudoin, Y.; Bamba, A.; Francis, J.; Ribbink, A.J.; et al. Challenges and emerging solutions to the land-based plastic waste issue in Africa. Mar. Policy 2018, 96, 256-263. [CrossRef]

Microplastics as contaminants in the marine environment: A review. M Cole, P Lindeque, C Halsband, T S Galloway, 10.1016/j.marpolbul.2011.09.025Mar. Pollut. Bull. 62PubMedCole, M.; Lindeque, P.; Halsband, C.; Galloway, T.S. Microplastics as contaminants in the marine environment: A review. Mar. Pollut. Bull. 2011, 62, 2588-2597. [CrossRef] [PubMed]

The pollution of the marine environment by plastic debris: A review. J G Derraik, 10.1016/S0025-326X(02)00220-5Mar. Pollut. Bull. 44Derraik, J.G.B. The pollution of the marine environment by plastic debris: A review. Mar. Pollut. Bull. 2002, 44, 842-852. [CrossRef]

Transport and release of chemicals from plastics to the environment and to wildlife. E L Teuten, J M Saquing, D R U Knappe, M A Barlaz, S Jonsson, A Bj€orn, S J Rowland, R C Thompson, S G Tamara, D Ochi, 10.1098/rstb.2008.0284Philos. Trans. R. Soc. Lond. B Biol. Sci. 364PubMedTeuten, E.L.; Saquing, J.M.; Knappe, D.R.U.; Barlaz, M.A.; Jonsson, S.; Bj€orn, A.; Rowland, S.J.; Thompson, R.C.; Tamara, S.G.; Ochi, D.; et al. Transport and release of chemicals from plastics to the environment and to wildlife. Philos. Trans. R. Soc. Lond. B Biol. Sci. 2009, 364, 2027-2045. [CrossRef] [PubMed]

Editorial: Plastic Pollution. Front. F Galgani, C K Pham, J Reisser, 10.3389/fmars.2017.00307Galgani, F.; Pham, C.K.; Reisser, J. Editorial: Plastic Pollution. Front. Mar. Sci. 2017, 4, 307. [CrossRef]

Macrofouling communities and the degradation of plastic bags in the sea: An in situ experiment. N.-C Pauli, J S Petermann, C Lott, M Weber, 10.1098/rsos.170549R. Soc. Open Sci. Pauli, N.-C.; Petermann, J.S.; Lott, C.; Weber, M. Macrofouling communities and the degradation of plastic bags in the sea: An in situ experiment. R. Soc. Open Sci. 2017, 4, 170549. [CrossRef]

Millimeter-sized marine plastics: A new pelagic habitat for microorganisms and invertebrates. J Reisser, J Shaw, G Hallegraeff, M Proietti, D K A Barnes, M Thums, C Wilcox, B D Hardesty, C Pattiaratchi, 10.1371/journal.pone.0100289PLoS ONE. 9Reisser, J.; Shaw, J.; Hallegraeff, G.; Proietti, M.; Barnes, D.K.A.; Thums, M.; Wilcox, C.; Hardesty, B.D.; Pattiaratchi, C. Millimeter-sized marine plastics: A new pelagic habitat for microorganisms and invertebrates. PLoS ONE 2014, 9, e100289. [CrossRef]

Plastic particles in surface waters of the northwestern Atlantic. J B Colton, Jr, F D Knapp, B R Burns, 10.1126/science.185.4150.491Science. 185Colton, J.B., Jr.; Knapp, F.D.; Burns, B.R. Plastic particles in surface waters of the northwestern Atlantic. Science 1974, 185, 491-497. [CrossRef]

Accumulation and fragmentation of plastic debris in global environments. D K A Barnes, F Galgani, R C Thompson, M Barlaz, 10.1098/rstb.2008.0205Phil. Trans. R. Soc. B Biol. Sci. 364PubMedBarnes, D.K.A.; Galgani, F.; Thompson, R.C.; Barlaz, M. Accumulation and fragmentation of plastic debris in global environments. Phil. Trans. R. Soc. B Biol. Sci. 2009, 364, 1985-1998. [CrossRef] [PubMed]

Environmental implications of plastic debris in marine settings -entanglement, ingestion, smothering, hangers-on, hitchhiking and alien invasions. M R Gregory, 10.1098/rstb.2008.0265Phil. Trans. R. Soc. B Biol. Sci. 364PubMedGregory, M.R. Environmental implications of plastic debris in marine settings -entanglement, ingestion, smothering, hangers-on, hitchhiking and alien invasions. Phil. Trans. R. Soc. B Biol. Sci. 2009, 364, 2013-2025. [CrossRef] [PubMed]

Bacterial community profiling of plastic litter in the Belgian part of the North Sea. C A De Tender, L I Devriese, A Haegeman, S Maes, T Ruttink, P Dawyndt, 10.1021/acs.est.5b01093Environ. Sci. Technol. 49PubMedDe Tender, C.A.; Devriese, L.I.; Haegeman, A.; Maes, S.; Ruttink, T.; Dawyndt, P. Bacterial community profiling of plastic litter in the Belgian part of the North Sea. Environ. Sci. Technol. 2015, 49, 9629-9638. [CrossRef] [PubMed]

Leave no traces-Beached marine litter shelters both invasive and native species. E Garcia-Vazquez, A Cani, A Diem, C Ferreira, R Geldhof, L Marquez, E Molloy, S Perché, 10.1016/j.marpolbul.2018.04.037Mar. Pollut. Bull. 131PubMedGarcia-Vazquez, E.; Cani, A.; Diem, A.; Ferreira, C.; Geldhof, R.; Marquez, L.; Molloy, E.; Perché, S. Leave no traces-Beached marine litter shelters both invasive and native species. Mar. Pollut. Bull. 2018, 131, 314-322. [CrossRef] [PubMed]

Plastic-associated harmful microalgal assemblages in marine environment. S Casabianca, S Capellacci, M G Giacobbe, C Dell&apos;aversano, L Tartaglione, F Varriale, R Narizzano, F Risso, P Moretto, A Dagnino, 10.1016/j.envpol.2018.09.110Environ. Pollut. 244Casabianca, S.; Capellacci, S.; Giacobbe, M.G.; Dell'Aversano, C.; Tartaglione, L.; Varriale, F.; Narizzano, R.; Risso, F.; Moretto, P.; Dagnino, A.; et al. Plastic-associated harmful microalgal assemblages in marine environment. Environ. Pollut. 2019, 244, 617-626. [CrossRef]

Microplastics as a vector for the transport of the bacterial fish pathogen species Aeromonas salmonicida. M K Viršek, M N Lovšinb, S Koren, A Kržan, M Peterlin, 10.1016/j.marpolbul.2017.08.024Mar. Pollut. Bull. 125Viršek, M.K.; Lovšinb, M.N.; Koren, S.; Kržan, A.; Peterlin, M. Microplastics as a vector for the transport of the bacterial fish pathogen species Aeromonas salmonicida. Mar. Pollut. Bull. 2017, 125, 301-309. [CrossRef]

Diversity and activity of communities inhabiting plastic debris in the North Pacific gyre. J A Bryant, T M Clemente, D A Viviani, A A Fong, K A Thomas, P Kemp, D M Karl, A E White, E F Delong, 10.1128/mSystems.00024-161Bryant, J.A.; Clemente, T.M.; Viviani, D.A.; Fong, A.A.; Thomas, K.A.; Kemp, P.; Karl, D.M.; White, A.E.; DeLong, E.F. Diversity and activity of communities inhabiting plastic debris in the North Pacific gyre. MSystems 2016, 1, e00024-16. [CrossRef]

Evidence of niche partitioning among bacteria living on plastics, organic particles and surrounding seawaters. C Dussud, A L Meistertzheim, P Conan, M Pujo-Pay, M George, P Fabre, J Coudane, P Higgs, A Elineau, M L Pedrotti, 10.1016/j.envpol.2017.12.027Environ. Pollut. 236Dussud, C.; Meistertzheim, A.L.; Conan, P.; Pujo-Pay, M.; George, M.; Fabre, P.; Coudane, J.; Higgs, P.; Elineau, A.; Pedrotti, M.L.; et al. Evidence of niche partitioning among bacteria living on plastics, organic particles and surrounding seawaters. Environ. Pollut. 2018, 236, 807-816. [CrossRef]

Microplastic is an abundant and distinct microbial habitat in an urban river. A Mccormick, T J Hoellein, S A Mason, J Schluep, J J Kelly, 10.1021/es503610rEnviron. Sci. Technol. 48McCormick, A.; Hoellein, T.J.; Mason, S.A.; Schluep, J.; Kelly, J.J. Microplastic is an abundant and distinct microbial habitat in an urban river. Environ. Sci. Technol. 2014, 48, 11863-11871. [CrossRef]

Spatial and seasonal variation in diversity and structure of microbial biofilms on marine plastics in Northern European waters. S Oberbeckmann, M G J Loeder, G Gerdts, A M Osborn, 10.1111/1574-6941.12409FEMS Microbiol. Ecol. 90Oberbeckmann, S.; Loeder, M.G.J.; Gerdts, G.; Osborn, A.M. Spatial and seasonal variation in diversity and structure of microbial biofilms on marine plastics in Northern European waters. FEMS Microbiol. Ecol. 2014, 90, 478-492. [CrossRef]

Socio-economic impact of microplastics in the 2 Seas, Channel and France Manche Region, an initial risk assessment. M D Van Der Meulen, L Devriese, J Lee, T Maes, J A Van Dalfsen, A Huvet, P Soudant, J Robbens, A D Vethaak, 17Van der Meulen, M.D.; DeVriese, L.; Lee, J.; Maes, T.; Van Dalfsen, J.A.; Huvet, A.; Soudant, P.; Robbens, J.; Vethaak, A.D. Socio-economic impact of microplastics in the 2 Seas, Channel and France Manche Region, an initial risk assessment. Available online: https://archimer.ifremer.fr/doc/00287/39834/38359.pdf (accessed on 17 September 2019).

Characteristics of marine aggregates in shallow-water ecosystems: Implications for disease ecology. M M Lyons, Y.-T Lau, W E Carden, J E Ward, S B Roberts, R Smolowitz, J Vallino, B Allam, 10.1007/s10393-007-0134-0EcoHealth. 4Lyons, M.M.; Lau, Y.-T.; Carden, W.E.; Ward, J.E.; Roberts, S.B.; Smolowitz, R.; Vallino, J.; Allam, B. Characteristics of marine aggregates in shallow-water ecosystems: Implications for disease ecology. EcoHealth 2007, 4, 406-420. [CrossRef]

Dangerous hitchhikers? Evidence for potentially pathogenic Vibrio spp. on microplastic particles. I V Kirstein, S Kirmizi, A Wichels, A Garin-Fernandez, R Erler, M Löder, G Gerdts, 10.1016/j.marenvres.2016.07.004Mar. Environ. Res. 120PubMedKirstein, I.V.; Kirmizi, S.; Wichels, A.; Garin-Fernandez, A.; Erler, R.; Löder, M.; Gerdts, G. Dangerous hitchhikers? Evidence for potentially pathogenic Vibrio spp. on microplastic particles. Mar. Environ. Res. 2016, 120, 1-8. [CrossRef] [PubMed]

Plastics in the North Atlantic garbage patch: A boat-microbe for hitchhikers and plastic degraders. D Debroas, A Mone, A Ter Halle, 10.1016/j.scitotenv.2017.05.059Sci. Total Environ. 599PubMedDebroas, D.; Mone, A.; Ter Halle, A. Plastics in the North Atlantic garbage patch: A boat-microbe for hitchhikers and plastic degraders. Sci. Total Environ. 2017, 599, 1222-1232. [CrossRef] [PubMed]

Plastics on the Sargasso Sea surface. E J Carpenter, K L Smith, 10.1126/science.175.4027.1240175Carpenter, E.J.; Smith, K.L. Plastics on the Sargasso Sea surface. Science 1972, 175, 1240-1241. [CrossRef]

Microbial biofilms in intertidal systems: An overview. A W Decho, 10.1016/S0278-4343(00)00022-4Cont. Shelf Res. 20Decho, A.W. Microbial biofilms in intertidal systems: An overview. Cont. Shelf Res. 2000, 20, 1257-1273. [CrossRef]

Environmental microbe and human pathogen: The ecology and microbiology of Vibrio cholerae. K L Cottingham, D A Chiavelli, R K Taylor, 10.1890/1540-9295(2003)001[0080:EMAHPT]2.0.CO;2Front. Ecol. Environ. 1Cottingham, K.L.; Chiavelli, D.A.; Taylor, R.K. Environmental microbe and human pathogen: The ecology and microbiology of Vibrio cholerae. Front. Ecol. Environ. 2003, 1, 80-86. [CrossRef]

Distribution and abundance of human-specific Bacteroides and relation to traditional indicators in an urban tropical catchment. J P Nshimyimana, E Ekklesia, P Shanahan, L H C Chua, J R Thompson, 10.1111/jam.12455J. Appl. Microbial. 116Nshimyimana, J.P.; Ekklesia, E.; Shanahan, P.; Chua, L.H.C.; Thompson, J.R. Distribution and abundance of human-specific Bacteroides and relation to traditional indicators in an urban tropical catchment. J. Appl. Microbial. 2014, 116, 1369-1383. [CrossRef]

Nitrogen cycle of a typical Suaeda salsa marsh ecosystem in the Yellow River estuary. X Mou, Z Sun, L Wang, C Wang, 10.1016/S1001-0742(10)60530-XJ. Environ. Sci. 23Mou, X.; Sun, Z.; Wang, L.; Wang, C. Nitrogen cycle of a typical Suaeda salsa marsh ecosystem in the Yellow River estuary. J. Environ. Sci. 2011, 23, 958-967. [CrossRef]

Phylogenetic and functional metagenomic profiling for assessing microbial biodiversity in environmental monitoring. V Kisand, A Valente, A Lahm, G Tanet, T Lettieri, 10.1371/journal.pone.0043630PLoS ONE. 7e43630. [CrossRefKisand, V.; Valente, A.; Lahm, A.; Tanet, G.; Lettieri, T. Phylogenetic and functional metagenomic profiling for assessing microbial biodiversity in environmental monitoring. PLoS ONE 2012, 7, e43630. [CrossRef]

Extraction of high molecular weight DNA from microbial mats. B S Bey, E B Fichot, G Dayama, A W Decho, R S Norman, 10.2144/000113486Biotechniques. 49Bey, B.S.; Fichot, E.B.; Dayama, G.; Decho, A.W.; Norman, R.S. Extraction of high molecular weight DNA from microbial mats. Biotechniques 2010, 49, 631-640. [CrossRef]

Chapter Ten-Preparation of microbial community cDNA for metatranscriptomic analysis in marine plankton. F J Stewart, Methods Enzymol. 531PubMedStewart, F.J. Chapter Ten-Preparation of microbial community cDNA for metatranscriptomic analysis in marine plankton. Methods Enzymol. 2013, 531, 187-218. [PubMed]

Analysis of black fungal biofilms occurring at domestic water taps (I): Compositional analysis using tag-encoded FLX amplicon pyrosequencing. G Heinrichs, I Hübner, C K Schmidt, G Sybren De Hoog, G Haase, 10.1007/s11046-013-9618-3Mycopathologia. 175PubMedHeinrichs, G.; Hübner, I.; Schmidt, C.K.; Sybren de Hoog, G.; Haase, G. Analysis of black fungal biofilms occurring at domestic water taps (I): Compositional analysis using tag-encoded FLX amplicon pyrosequencing. Mycopathologia 2013, 175, 387-397. [CrossRef] [PubMed]

Substrate type determines metagenomic profiles from diverse chemical habitats. T C Jeffries, J R Seymour, J A Gilbert, E A Dinsdale, K Newton, S S C Leterme, B Roudnew, R J Smith, L Seuront, J G Mitchell, 10.1371/journal.pone.0025173PLoS ONE. 6PubMedJeffries, T.C.; Seymour, J.R.; Gilbert, J.A.; Dinsdale, E.A.; Newton, K.; Leterme, S.S.C.; Roudnew, B.; Smith, R.J.; Seuront, L.; Mitchell, J.G. Substrate type determines metagenomic profiles from diverse chemical habitats. PLoS ONE 2011, 6, e25173. [CrossRef] [PubMed]

. R Ghai, F Rodriguez-Valera, K D Mcmahon, D Toyama, R Rinke, T Cristina Souza De Oliveira, J Wagner Garcia, F Pellon De Miranda, Henrique-Silva, 10.1371/journal.pone.0023785PLoS ONE. 6F. Metagenomics of the water column in the pristine upper course of the Amazon river. PubMedGhai, R.; Rodriguez-Valera, F.; McMahon, K.D.; Toyama, D.; Rinke, R.; Cristina Souza de Oliveira, T.; Wagner Garcia, J.; Pellon de Miranda, F.; Henrique-Silva, F. Metagenomics of the water column in the pristine upper course of the Amazon river. PLoS ONE 2011, 6, e23785. [CrossRef] [PubMed]

Metagenomic analysis of DNA viruses in a wastewater treatment plant in tropical climate. H Tamaki, R Zhang, F E Angly, S Nakamura, P Y Hong, T Yasunaga, Y Kamagata, W Liu, 10.1111/j.1462-2920.2011.02630.xEnviron. Microbiol. 14PubMedTamaki, H.; Zhang, R.; Angly, F.E.; Nakamura, S.; Hong, P.Y.; Yasunaga, T.; Kamagata, Y.; Liu, W.T. Metagenomic analysis of DNA viruses in a wastewater treatment plant in tropical climate. Environ. Microbiol. 2012, 14, 441-452. [CrossRef] [PubMed]

Metagenetic community analysis of microbial eukaryotes illuminates biogeographic patterns in deep-sea and shallow water sediments. H M Bik, W Sung, P De Ley, J G Baldwin, J Sharma, A Rocha-Olivares, W K Thomas, 10.1111/j.1365-294X.2011.05297.xMol. Ecol. 21PubMedBik, H.M.; Sung, W.; De Ley, P.; Baldwin, J.G.; Sharma, J.; Rocha-Olivares, A.; Thomas, W.K. Metagenetic community analysis of microbial eukaryotes illuminates biogeographic patterns in deep-sea and shallow water sediments. Mol. Ecol. 2012, 21, 1048-1059. [CrossRef] [PubMed]

Metagenomic profiles of free-living archaea, bacteria and small eukaryotes in coastal areas of Sichang island. N Somboonna, A Assawamakin, A Wilantho, S Tangphatsornruang, S Tongsima, 10.1186/1471-2164-13-S7-S29ThailandBMC Genom. 2012, 13, S29. [CrossRefSomboonna, N.; Assawamakin, A.; Wilantho, A.; Tangphatsornruang, S.; Tongsima, S. Metagenomic profiles of free-living archaea, bacteria and small eukaryotes in coastal areas of Sichang island, Thailand. BMC Genom. 2012, 13, S29. [CrossRef]

De novo metagenomic assembly reveals abundant novel major lineage of Archaea in hypersaline microbial communities. P Narasingarao, S Podell, J A Ugalde, C Brochier-Armanet, J B Emerson, J J Brocks, K B Heidelberg, J F Banfield, E E Allen, 10.1038/ismej.2011.78ISME J. 6Narasingarao, P.; Podell, S.; Ugalde, J.A.; Brochier-Armanet, C.; Emerson, J.B.; Brocks, J.J.; Heidelberg, K.B.; Banfield, J.F.; Allen, E.E. De novo metagenomic assembly reveals abundant novel major lineage of Archaea in hypersaline microbial communities. ISME J. 2012, 6, 81-93. [CrossRef]

Viruses in the desert: A metagenomic survey of viral communities in four perennial ponds of the Mauritanian Sahara. . L Fancello, S Trape, C Robert, M Boyer, N Popgeorgiev, D Raoult, C Desnues, 10.1038/ismej.2012.101ISME J. 7Fancello, .L.; Trape, S.; Robert, C.; Boyer, M.; Popgeorgiev, N.; Raoult, D.; Desnues, C. Viruses in the desert: A metagenomic survey of viral communities in four perennial ponds of the Mauritanian Sahara. ISME J. 2013, 7, 359-369. [CrossRef]

Exploring nucleo-cytoplasmic large DNA viruses in Tara Oceans microbial metagenomes. P Hingamp, N Grimsley, S G Acinas, C Clerissi, L Subirana, J Poulain, I Ferrera, H Sarmento, E Villar, G Lima-Mendez, 10.1038/ismej.2013.59ISME J. 7Hingamp, P.; Grimsley, N.; Acinas, S.G.; Clerissi, C.; Subirana, L.; Poulain, J.; Ferrera, I.; Sarmento, H.; Villar, E.; Lima-Mendez, G.; et al. Exploring nucleo-cytoplasmic large DNA viruses in Tara Oceans microbial metagenomes. ISME J. 2013, 7, 1678-1695. [CrossRef]

Diversity in a hidden world: Potential and limitation of next-generation sequencing for surveys of molecular diversity of eukaryotic microorganisms. R Medinger, V Nolte, R V Pandey, S Jost, B Ottenwälder, C Schlötterer, J Boenigk, 10.1111/j.1365-294X.2009.04478.xMol. Ecol. 19PubMedMedinger, R.; Nolte, V.; Pandey, R.V.; Jost, S.; Ottenwälder, B.; Schlötterer, C.; Boenigk, J. Diversity in a hidden world: Potential and limitation of next-generation sequencing for surveys of molecular diversity of eukaryotic microorganisms. Mol. Ecol. 2010, 19, 32-40. [CrossRef] [PubMed]

Multiple marker parallel tag environmental DNA sequencing reveals a highly complex eukaryotic community in marine anoxic water. T Stoeck, D Bass, M Nebel, R Christen, M D M Jones, H.-W Breiner, T A Richards, 10.1111/j.1365-294X.2009.04480.xMol. Ecol. 19PubMedStoeck, T.; Bass, D.; Nebel, M.; Christen, R.; Jones, M.D.M.; Breiner, H.-W.; Richards, T.A. Multiple marker parallel tag environmental DNA sequencing reveals a highly complex eukaryotic community in marine anoxic water. Mol. Ecol. 2010, 19, 21-31. [CrossRef] [PubMed]

Environmental barcoding: A next-generation sequencing approach for biomonitoring applications using river benthos. M Hajibabaei, S Shokralla, X Zhou, G A C Singer, D J Baird, 10.1371/journal.pone.0017497PLoS ONE. 6PubMedHajibabaei, M.; Shokralla, S.; Zhou, X.; Singer, G.A.C.; Baird, D.J. Environmental barcoding: A next-generation sequencing approach for biomonitoring applications using river benthos. PLoS ONE 2011, 6, e17497. [CrossRef] [PubMed]

Biomonitoring 2.0: A new paradigm in ecosystem assessment made possible by next-generation DNA sequencing. D J Baird, M Hajibabaei, 10.1111/j.1365-294X.2012.05519.xMol. Ecol. 21PubMedBaird, D.J.; Hajibabaei, M. Biomonitoring 2.0: A new paradigm in ecosystem assessment made possible by next-generation DNA sequencing. Mol. Ecol. 2012, 21, 2039-2044. [CrossRef] [PubMed]

Distribution of Mycobacterium ulcerans in Buruli ulcer endemic and non-endemic aquatic sites in Ghana. H R Williamson, M E Benbow, K D Nguyen, D C Beachboard, R K Kimbirauskas, M D Mcintosh, C Quaye, E O Ampadu, D Boakye, R W Merritt, 10.1371/journal.pntd.0000205PLoS Negl. Trop. Dis. PubMedWilliamson, H.R.; Benbow, M.E.; Nguyen, K.D.; Beachboard, D.C.; Kimbirauskas, R.K.; McIntosh, M.D.; Quaye, C.; Ampadu, E.O.; Boakye, D.; Merritt, R.W.; et al. Distribution of Mycobacterium ulcerans in Buruli ulcer endemic and non-endemic aquatic sites in Ghana. PLoS Negl. Trop. Dis. 2008, 2, e205. [CrossRef] [PubMed]

The marine viromes of four oceanic regions. F E Angly, B Felts, M Breitbart, P Salamon, R A Edwards, C Carlson, A M Chan, M Haynes, S Kelley, H Liu, 10.1371/journal.pbio.0040368PLoS Biol. PubMedAngly, F.E.; Felts, B.; Breitbart, M.; Salamon, P.; Edwards, R.A.; Carlson, C.; Chan, A.M.; Haynes, M.; Kelley, S.; Liu, H.; et al. The marine viromes of four oceanic regions. PLoS Biol. 2006, 4, e368. [CrossRef] [PubMed]

Broad dissemination of plasmid-mediated quinolone resistance genes in sediments of two urban coastal wetlands. D E Cummings, K F Archer, D J Arriola, P A Baker, K G Faucett, J B Laroya, K L Pfeil, C R Ryan, K R U Ryan, D E Zuill, 10.1021/es1029206Environ. Sci. Technol. 45Cummings, D.E.; Archer, K.F.; Arriola, D.J.; Baker, P.A.; Faucett, K.G.; Laroya, J.B.; Pfeil, K.L.; Ryan, C.R.; Ryan, K.R.U.; Zuill, D.E. Broad dissemination of plasmid-mediated quinolone resistance genes in sediments of two urban coastal wetlands. Environ. Sci. Technol. 2011, 45, 447-454. [CrossRef]

Comparative fecal metagenomics unveils unique functional capacity of the swine gut. R Lamendella, J W Santo Domingo, S Ghosh, J Martinson, D B Oerther, 10.1186/1471-2180-11-103BMC Microbiol. 11103Lamendella, R.; Santo Domingo, J.W.; Ghosh, S.; Martinson, J.; Oerther, D.B. Comparative fecal metagenomics unveils unique functional capacity of the swine gut. BMC Microbiol. 2011, 11, 103. [CrossRef]

Identification and modeling of a novel chloramphenicol resistance protein detected by functional metagenomics in a wetland of Lerma. M López-Pérez, S Mirete, E Jardón-Valadez, J E González-Pastor, Mexico. Int. Microbiol. 16López-Pérez, M.; Mirete, S.; Jardón-Valadez, E.; González-Pastor, J.E. Identification and modeling of a novel chloramphenicol resistance protein detected by functional metagenomics in a wetland of Lerma, Mexico. Int. Microbiol. 2013, 16, 103-111.

Metagenome analyses of corroded concrete wastewater pipe biofilms reveal a complex microbial system. V Gomez-Alvarez, R P Revetta, J W Domingo, 10.1186/1471-2180-12-122BMC Microbiol. 12Gomez-Alvarez, V.; Revetta, R.P.; Santo Domingo, J.W. Metagenome analyses of corroded concrete wastewater pipe biofilms reveal a complex microbial system. BMC Microbiol. 2012, 12, 122. [CrossRef]

Dynamic assessment of the floc morphology, bacterial diversity, and integron content of an activated sludge reactor processing hospital effluent. T Stalder, M Alrhmoun, J.-N Louvet, M Casellas, C Maftah, C Carrion, M.-N Pons, O Pahl, M.-C Ploy, C Dagot, 10.1021/es4008646Environ. Sci. Technol. 47PubMedStalder, T.; Alrhmoun, M.; Louvet, J.-N.; Casellas, M.; Maftah, C.; Carrion, C.; Pons, M.-N.; Pahl, O.; Ploy, M.-C.; Dagot, C. Dynamic assessment of the floc morphology, bacterial diversity, and integron content of an activated sludge reactor processing hospital effluent. Environ. Sci. Technol. 2013, 47, 7909-7917. [CrossRef] [PubMed]

Metagenomic analysis reveals the prevalence of biodegradation genes for organic pollutants in activated sludge. H Fang, L Cai, Y Yu, T Zhang, 10.1016/j.biortech.2012.11.054Bioresour. Technol. 129PubMedFang, H.; Cai, L.; Yu, Y.; Zhang, T. Metagenomic analysis reveals the prevalence of biodegradation genes for organic pollutants in activated sludge. Bioresour. Technol. 2013, 129, 209-218. [CrossRef] [PubMed]

Evaluation of methods to concentrate and purify ocean virus communities through comparative, replicated metagenomics. B L Hurwitz, L Deng, B T Poulos, M B Sullivan, 10.1111/j.1462-2920.2012.02836.xEnviron. Microbiol. 15PubMedHurwitz, B.L.; Deng, L.; Poulos, B.T.; Sullivan, M.B. Evaluation of methods to concentrate and purify ocean virus communities through comparative, replicated metagenomics. Environ. Microbiol. 2013, 15, 1428-1440. [CrossRef] [PubMed]

Individual genome assembly from complex community short-read metagenomic datasets. C Luo, D Tsementzi, N C Kyrpides, K T Konstantinidis, 10.1038/ismej.2011.147ISME J. 6PubMedLuo, C.; Tsementzi, D.; Kyrpides, N.C.; Konstantinidis, K.T. Individual genome assembly from complex community short-read metagenomic datasets. ISME J. 2012, 6, 898-901. [CrossRef] [PubMed]

Comparison of dimensional reduction methods for detecting and visualizing novel patterns in human and marine microbiome. X Jiang, X Hu, W Xu, T He, E K Park, 10.1109/TNB.2013.2263287IEEE Trans. Nanobiosci. 12PubMedJiang, X.; Hu, X.; Xu, W.; He, T.; Park, E.K. Comparison of dimensional reduction methods for detecting and visualizing novel patterns in human and marine microbiome. IEEE Trans. Nanobiosci. 2013, 12, 199-205. [CrossRef] [PubMed]

De novo assembly of human genomes with massively parallel short read sequencing. R Li, H Zhu, J Ruan, W Qian, X Fang, Z Shi, 10.1101/gr.097261.109Genome. Res. 20Li, R.; Zhu, H.; Ruan, J.; Qian, W.; Fang, X.; Shi, Z. De novo assembly of human genomes with massively parallel short read sequencing. Genome. Res. 2010, 20, 265-272. [CrossRef]

Ab initio gene identification in metagenomic sequences. W Zhu, A Lomsadze, M Borodovsky, 10.1093/nar/gkq275Nucleic Acids Res. 38132Zhu, W.; Lomsadze, A.; Borodovsky, M. Ab initio gene identification in metagenomic sequences. Nucleic Acids Res. 2010, 38, e132. [CrossRef]

The Pfam protein families database. A Bateman, L Coin, R Durbin, R D Finn, V Hollich, S Griffiths-Jones, 10.1093/nar/gkh121Nucleic Acids Res. 32Bateman, A.; Coin, L.; Durbin, R.; Finn, R.D.; Hollich, V.; Griffiths-Jones, S. The Pfam protein families database. Nucleic Acids Res. 2004, 32, D138-D141. [CrossRef]

The TIGRFAMs database of protein families. D H Haft, J D Selengut, O White, 10.1093/nar/gkg128Nucleic Acids Res. 31Haft, D.H.; Selengut, J.D.; White, O. The TIGRFAMs database of protein families. Nucleic Acids Res. 2003, 31, 371-373. [CrossRef]

HMMER web server: Interactive sequence similarity searching. R D Finn, J Clements, S R Eddy, 10.1093/nar/gkr367Nucleic Acids Res. 39Finn, R.D.; Clements, J.; Eddy, S.R. HMMER web server: Interactive sequence similarity searching. Nucleic Acids Res. 2011, 39, W29-W37. [CrossRef]

Diverse gene functions in a soil mobilome. W Luo, Z Xu, L Riber, L H Hansen, S J Sørensen, 10.1016/j.soilbio.2016.07.018Soil. Biol. Biochem. 101Luo, W.; Xu, Z.; Riber, L.; Hansen, L.H.; Sørensen, S.J. Diverse gene functions in a soil mobilome. Soil. Biol. Biochem. 2016, 101, 175-183. [CrossRef]

The comprehensive antibiotic resistance database. A G Mcarthur, N Waglechner, F Nizam, A Yan, M A Azad, A J Baylay, K Bhullar, M J Canova, G De Pascale, L Ejim, 10.1128/AAC.00419-13Antimicrob. Agents Chemother. 57PubMedMcArthur, A.G.; Waglechner, N.; Nizam, F.; Yan, A.; Azad, M.A.; Baylay, A.J.; Bhullar, K.; Canova, M.J.; de Pascale, G.; Ejim, L.; et al. The comprehensive antibiotic resistance database. Antimicrob. Agents Chemother. 2013, 57, 3348-3357. [CrossRef] [PubMed]

ARDB-antibiotic resistance genes database. B Liu, M Pop, 10.1093/nar/gkn656Nucleic Acids Res. 37PubMedLiu, B.; Pop, M. ARDB-antibiotic resistance genes database. Nucleic Acids Res. 2009, 37, D443-D447. [CrossRef] [PubMed]

Identification of acquired antimicrobial resistance genes. E Zankari, H Hasman, S Cosentino, M Vestergaard, S Rasmussen, O Lund, 10.1093/jac/dks261J. Antimicrob. Chemother. 67PubMedZankari, E.; Hasman, H.; Cosentino, S.; Vestergaard, M.; Rasmussen, S.; Lund, O. Identification of acquired antimicrobial resistance genes. J. Antimicrob. Chemother. 2012, 67, 2640-2644. [CrossRef] [PubMed]

ICEberg: A web-based resource for integrative and conjugative elements found in Bacteria. D Bi, Z Xu, E M Harrison, C Tai, Y Wei, X He, S Jia, Z Deng, K Rajakumar, H Y Ou, 10.1093/nar/gkr846Nucleic Acids Res. 40PubMedBi, D.; Xu, Z.; Harrison, E.M.; Tai, C.; Wei, Y.; He, X.; Jia, S.; Deng, Z.; Rajakumar, K.; Ou, H.Y. ICEberg: A web-based resource for integrative and conjugative elements found in Bacteria. Nucleic Acids Res. 2012, 40, D621-D626. [CrossRef] [PubMed]

Analysis of resistance genes of clinical Pannonibacter phragmitetus strain 31801 by complete genome sequencing. D S Ming, Q Q Chen, X T Chen, 10.1007/s00203-018-1522-2Arch. Microbiol. Ming, D.S.; Chen, Q.Q.; Chen, X.T. Analysis of resistance genes of clinical Pannonibacter phragmitetus strain 31801 by complete genome sequencing. Arch. Microbiol. 2018, 200, 1101-1109. [CrossRef]

QIIME allows analysis of high-throughput community sequencing data. J G Caporaso, J Kuczynski, J Stombaugh, K Bittinger, F D Bushman, E K Costello, N Fierer, A G Pena, J K Goodrich, J I Gordon, 10.1038/nmeth.f.303Nat. Methods. 7335Caporaso, J.G.; Kuczynski, J.; Stombaugh, J.; Bittinger, K.; Bushman, F.D.; Costello, E.K.; Fierer, N.; Pena, A.G.; Goodrich, J.K.; Gordon, J.I.; et al. QIIME allows analysis of high-throughput community sequencing data. Nat. Methods 2010, 7, 335. [CrossRef]

UCHIME improves sensitivity and speed of chimera detection. R C Edgar, B J Haas, J C Clemente, C Quince, R Knight, 10.1093/bioinformatics/btr381Bioinformatics. 27Edgar, R.C.; Haas, B.J.; Clemente, J.C.; Quince, C.; Knight, R. UCHIME improves sensitivity and speed of chimera detection. Bioinformatics 2011, 27, 2194-2200. [CrossRef]

UPARSE: Highly accurate OTU sequences from microbial amplicon reads. R C Edgar, 10.1038/nmeth.2604Nat. Methods. 10Edgar, R.C. UPARSE: Highly accurate OTU sequences from microbial amplicon reads. Nat. Methods 2013, 10, 996. [CrossRef]

The SILVA ribosomal RNA gene database project: Improved data processing and web-based tools. C Quast, E Pruesse, P Yilmaz, J Gerken, T Schweer, P Yarza, J Peplies, F O Glöckner, 10.1093/nar/gks1219Nucleic Acids Res. 41Quast, C.; Pruesse, E.; Yilmaz, P.; Gerken, J.; Schweer, T.; Yarza, P.; Peplies, J.; Glöckner, F.O. The SILVA ribosomal RNA gene database project: Improved data processing and web-based tools. Nucleic Acids Res. 2012, 41, D590-D596. [CrossRef]

Comparison of 16S rRNA and protein-coding genes as molecular markers for assessing microbial diversity (Bacteria and Archaea) in ecosystems. S Roux, F Enault, G Bronner, D Debroas, 10.1111/j.1574-6941.2011.01190.xFEMS Microbiol. Ecol. 78PubMedRoux, S.; Enault, F.; Bronner, G.; Debroas, D. Comparison of 16S rRNA and protein-coding genes as molecular markers for assessing microbial diversity (Bacteria and Archaea) in ecosystems. FEMS Microbiol. Ecol. 2011, 78, 617-628. [CrossRef] [PubMed]

How and why DNA barcodes underestimate the diversity of microbial eukaryotes. G Piganeau, A Eyre-Walker, S Jancek, N Grimsley, H Moreau, 10.1371/journal.pone.0016342PLoS ONE. 10PubMedPiganeau, G.; Eyre-Walker, A.; Jancek, S.; Grimsley, N.; Moreau, H. How and why DNA barcodes underestimate the diversity of microbial eukaryotes. PLoS ONE 2011, 10, e16342. [CrossRef] [PubMed]

A molecular method to assess Phytophthora diversity in environmental samples. S Scibetta, L Schena, A Chimento, S O Cacciola, D E Cooke, 10.1016/j.mimet.2011.12.012J. Microbiol. Methods. 88PubMedScibetta, S.; Schena, L.; Chimento, A.; Cacciola, S.O.; Cooke, D.E. A molecular method to assess Phytophthora diversity in environmental samples. J. Microbiol. Methods 2012, 88, 356-368. [CrossRef] [PubMed]

Discrimination between live and dead cellsin bacterial communities from environmental water samples analyzed by 454 pyrosequencing. A Nocker, T Richter-Heitmann, R Montijn, F Schuren, R Kort, Int. Microbiol. 13PubMedNocker, A.; Richter-Heitmann, T.; Montijn, R.; Schuren, F.; Kort, R. Discrimination between live and dead cellsin bacterial communities from environmental water samples analyzed by 454 pyrosequencing. Int. Microbiol. 2010, 13, 59-65. [PubMed]

Identification and characterization of a new true lipase isolated through metagenomic approach. A Glogauer, V P Martini, H Faoro, G H Couto, M Müller-Santos, R A Monteiro, D A Mitchell, E M De Souza, F O Pedrosa, N Krieger, 10.1186/1475-2859-10-54Microb. Cell Fact. 10PubMedGlogauer, A.; Martini, V.P.; Faoro, H.; Couto, G.H.; Müller-Santos, M.; Monteiro, R.A.; Mitchell, D.A.; de Souza, E.M.; Pedrosa, F.O.; Krieger, N. Identification and characterization of a new true lipase isolated through metagenomic approach. Microb. Cell Fact. 2011, 10, 54. [CrossRef] [PubMed]

Metagenomics-based analysis of viral communities in dairy lagoon wastewater. F S Alhamlan, M M Ederer, C J Brown, E R Coats, R L Crawford, 10.1016/j.mimet.2012.11.016J. Microbiol. Methods. 92PubMedAlhamlan, F.S.; Ederer, M.M.; Brown, C.J.; Coats, E.R.; Crawford, R.L. Metagenomics-based analysis of viral communities in dairy lagoon wastewater. J. Microbiol. Methods 2013, 92, 183-188. [CrossRef] [PubMed]

The taxonomic and functional diversity of microbes at a temperate coastal site: A 'multi-omic' study of seasonal and diel temporal variation. J A Gilbert, D Field, P Swift, S Thomas, D Cummings, B Temperton, K Weynberg, S Huse, M Hughes, I Joint, 10.1371/journal.pone.0015545PLoS ONE. 5Gilbert, J.A.; Field, D.; Swift, P.; Thomas, S.; Cummings, D.; Temperton, B.; Weynberg, K.; Huse, S.; Hughes, M.; Joint, I.; et al. The taxonomic and functional diversity of microbes at a temperate coastal site: A 'multi-omic' study of seasonal and diel temporal variation. PLoS ONE 2010, 5, e15545. [CrossRef]

Microbial and viral metagenomes of a subtropical freshwater reservoir subject to climatic disturbances. C H Tseng, P W Chiang, F K Shiah, Y L Chen, J R Liou, T C Hsu, S Maheswararajah, I Saeed, S Halgamuge, S L Tang, 10.1038/ismej.2013.118ISME J. 7Tseng, C.H.; Chiang, P.W.; Shiah, F.K.; Chen, Y.L.; Liou, J.R.; Hsu, T.C.; Maheswararajah, S.; Saeed, I.; Halgamuge, S.; Tang, S.L. Microbial and viral metagenomes of a subtropical freshwater reservoir subject to climatic disturbances. ISME J. 2013, 7, 2374-2386. [CrossRef]

Influence of nutrients and currents on the genomic composition of microbes across an upwelling mosaic. L Zeigler Allen, E E Allen, J H Badger, J P Mccrow, I T Paulsen, L D Elbourne, M Thiagarajan, D B Rusch, K H Nealson, S J Williamson, 10.1038/ismej.2011.201ISME J. 6Zeigler Allen, L.; Allen, E.E.; Badger, J.H.; McCrow, J.P.; Paulsen, I.T.; Elbourne, L.D.; Thiagarajan, M.; Rusch, D.B.; Nealson, K.H.; Williamson, S.J.; et al. Influence of nutrients and currents on the genomic composition of microbes across an upwelling mosaic. ISME J. 2012, 6, 1403-1414. [CrossRef]

Global antibiotic consumption 2000 to 2010: An analysis of national pharmaceutical sales data. T P Van Boeckel, S Gandra, A Ashok, Q Caudron, B T Grenfell, S A Levin, R Laxminarayan, 10.1016/S1473-3099(14)70780-7Lancet. Infect. Dis. 14Van Boeckel, T.P.; Gandra, S.; Ashok, A.; Caudron, Q.; Grenfell, B.T.; Levin, S.A.; Laxminarayan, R. Global antibiotic consumption 2000 to 2010: An analysis of national pharmaceutical sales data. Lancet. Infect. Dis. 2014, 14, 742-750. [CrossRef]

The geological cycle of plastics and their use as a stratigraphic indicator of the Anthropocene. J Czalasiewicz, C N Waters, Ivar, J A Sul, P L Corcoran, A D Barnosky, A Cearreta, M Edgeworth, A Gałuszka, C Jeandel, R Leinfelder, 10.1016/j.ancene.2016.01.002Anthropocene. 13CZalasiewicz, J.; Waters, C.N.; Ivar do Sul, J.A.; Corcoran, P.L.; Barnosky, A.D.; Cearreta, A.; Edgeworth, M.; Gałuszka, A.; Jeandel, C.; Leinfelder, R.; et al. The geological cycle of plastics and their use as a stratigraphic indicator of the Anthropocene. Anthropocene 2016, 13, 4-17. [CrossRef]