# The biological pathways of Alzheimer disease: a review Supplementary material

CorpusID: 230564948
 
tags: #Medicine, #Biology

URL: [https://www.semanticscholar.org/paper/f7b236ff541221952b1f9d563e2ef05f24c25445](https://www.semanticscholar.org/paper/f7b236ff541221952b1f9d563e2ef05f24c25445)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

The biological pathways of Alzheimer disease: a review Supplementary material
Published: 16 December 2020

Marco Calabr√≤ 
Department of Biomedical and Dental Sciences and Morphofunctional Imaging
University of Messina
Italy

Carmela Rinaldi 
Department of Biomedical and Dental Sciences and Morphofunctional Imaging
University of Messina
Italy

Giuseppe Santoro 
Department of Biomedical and Dental Sciences and Morphofunctional Imaging
University of Messina
Italy

Concetta Crisafulli ccrisafulli@unime.it 
Department of Biomedical and Dental Sciences and Morphofunctional Imaging
University of Messina
Italy

The biological pathways of Alzheimer disease: a review Supplementary material

AIMS Neuroscience
81Published: 16 December 202010.3934/Neuroscience.2021005Received: 24 August 2020 Accepted: 23 November 2020Review * Correspondence:


## AIMS Neuroscience

Volume 8, Issue 1, 86-132. 31.53% cholesterol transport 9.91% protein-lipid complex subunit organization 9.91% positive regulation of supramolecular fiber organization 8.11% muscle cell proliferation 6.31% regulation of peptidyl-tyrosine phosphorylation 5.41% regulation of transcription from RNA polymerase II promoter in response to hypoxia 3.60% regulation of cell size 3.60% blood vessel endothelial cell migration 2.70% positive regulation of growth 2.70% negative regulation of immune response 1.80% glial cell differentiation 1 Note: The figure shows the enriched Cluster 1, derived from the initial selection of genes. Cluster 1 depicts the interacting genes and the type of interaction as shown in the figure legend. The Bar Chart associated with cluster 1 reports the percentage of genes belonging to a specific biological process (from Gene Ontology database) included in the pathway under analysis. It also shows the associations levels of biological processes with the cluster under analysis. The pie chart shows the overall degree of association of main biological processes involved with the cluster under investigation. To note, both the bar and pie chart report the top 25 biological processes. The table summarizes the details regarding the pathway analysis and shows all of the associated biological processes linked with the cluster under investigation. The figure shows the enriched Cluster 2, derived from the initial selection of genes, after exclusion of the genes involved in cluster 1. Cluster 2 depicts the interacting genes and the type of interaction as shown in the figure legend. The Bar Chart associated with cluster 2 reports the percentage of genes belonging to a specific biological process (from Gene Ontology database) included in the pathway under analysis. It also shows the associations levels of biological processes with the cluster under analysis. The pie chart shows the overall degree of association of main biological processes involved with the cluster under investigation. Cluster 2 genes seems to be mainly implicated with fat regulative processes. To note GO:0045599 -negative regulation of fat cell differentiation is a sub process of GO:0045598 -regulation of fat cell differentiation. Only main biological processes are represented in the pie chart. The table summarizes the details regarding the pathway analysis and shows all of the associated biological processes linked with the cluster under investigation.


## AIMS Neuroscience


Cluster 1. Informative Data..80% Notch signaling pathway 
1.80% regulation of nitric-oxide synthase activity 
1.80% negative regulation of amyloid precursor protein catabolic process 
0.90% tissue regeneration 
0.90% regulation of humoral immune response 
0.90% animal organ regeneration 
0.90% regeneration 
0.90% positive regulation of small GTPase mediated signal transduction 
0.90% artery morphogenesis 
0.90% positive regulation of hormone secretion 
0.90% negative regulation of innate immune response 
0.90% cardiac septum development 
0.90% vasoconstriction 
AIMS Neuroscience 
Volume 8, Issue 1, 86-132. 

Gene Ontology Identification Number 
Raw pValue 
Corrected pValue* 

Term 
Group 
Term 
Group 
% Associated Genes Nr. Genes 

GO:0042098 
T cell proliferation 
5.31E-07 8.28E-15 2.55E-05 1.99E-13 
4.02 
10.00 

GO:0050870 
positive regulation of T cell activation 
6.43E-08 8.28E-15 4.05E-06 1.99E-13 
4.33 
11.00 

GO:0050730 
regulation of peptidyl-tyrosine phosphorylation 
1.15E-10 8.28E-15 1.00E-08 1.99E-13 
4.52 
15.00 

GO:0042102 
positive regulation of T cell proliferation 
4.36E-05 8.28E-15 3.49E-04 1.99E-13 
4.76 
6.00 

GO:0050731 
positive regulation of peptidyl-tyrosine phosphorylation 
2.96E-09 8.28E-15 2.25E-07 1.99E-13 
5.02 
12.00 

GO:0042531 
positive regulation of tyrosine phosphorylation of STAT 
protein 

2.98E-06 8.28E-15 1.01E-04 1.99E-13 
7.59 
6.00 

GO:0046427 
positive regulation of JAK-STAT cascade 
5.98E-08 8.28E-15 3.83E-06 1.99E-13 
7.62 
8.00 

GO:0045927 
positive regulation of growth 
9.82E-10 1.66E-10 8.06E-08 3.82E-09 
4.29 
14.00 

GO:0035265 
organ growth 
1.05E-06 1.66E-10 4.43E-05 3.82E-09 
4.35 
9.00 

GO:0048660 
regulation of smooth muscle cell proliferation 
2.30E-06 1.66E-10 8.50E-05 3.82E-09 
4.73 
8.00 

GO:0033002 
muscle cell proliferation 
3.25E-09 1.66E-10 2.40E-07 3.82E-09 
4.98 
12.00 

GO:0048661 
positive regulation of smooth muscle cell proliferation 
2.24E-05 1.66E-10 3.36E-04 3.82E-09 
5.36 
6.00 

GO:0046622 
positive regulation of organ growth 
9.36E-07 1.66E-10 4.12E-05 3.82E-09 
9.23 
6.00 

GO:0055021 
regulation of cardiac muscle tissue growth 
1.03E-07 1.66E-10 5.84E-06 3.82E-09 
9.33 
7.00 

GO:0055023 
positive regulation of cardiac muscle tissue growth 
3.50E-06 1.66E-10 1.05E-04 3.82E-09 
10.87 
5.00 

Continued on next page 
AIMS Neuroscience 
Volume 8, Issue 1, 86-132. 

Gene Ontology Identification Number 
Raw pValue 
Corrected pValue* 

Term 
Group 
Term 
Group 
% Associated Genes Nr. Genes 

GO:0050728 
negative regulation of inflammatory response 
1.67E-06 7.72E-10 6.53E-05 1.70E-08 
4.94 
8.00 

GO:0031348 
negative regulation of defense response 
2.96E-09 7.72E-10 2.25E-07 1.70E-08 
5.02 
12.00 

GO:0050777 
negative regulation of immune response 
1.60E-09 7.72E-10 1.25E-07 1.70E-08 
6.18 
11.00 

GO:0030307 
positive regulation of cell growth 
1.29E-06 9.82E-10 5.14E-05 2.06E-08 
4.25 
9.00 

GO:0048639 
positive regulation of developmental growth 
1.62E-07 9.82E-10 8.77E-06 2.06E-08 
4.57 
10.00 

GO:0046165 
alcohol biosynthetic process 
5.48E-06 1.18E-09 1.43E-04 2.36E-08 
4.21 
8.00 

GO:0046890 
regulation of lipid biosynthetic process 
2.91E-08 1.18E-09 2.00E-06 2.36E-08 
4.68 
11.00 

GO:0045834 
positive regulation of lipid metabolic process 
3.93E-07 1.18E-09 1.96E-05 2.36E-08 
4.89 
9.00 

GO:0019217 
regulation of fatty acid metabolic process 
3.93E-06 1.18E-09 1.10E-04 2.36E-08 
5.47 
7.00 

GO:0019218 
regulation of steroid metabolic process 
6.51E-07 1.18E-09 3.00E-05 2.36E-08 
5.59 
8.00 

GO:0022600 
digestive system process 
3.01E-06 1.18E-09 9.94E-05 2.36E-08 
5.69 
7.00 

GO:0008203 
cholesterol metabolic process 
3.01E-09 1.18E-09 2.26E-07 2.36E-08 
5.82 
11.00 

GO:1905952 
regulation of lipid localization 
9.92E-09 1.18E-09 7.05E-07 2.36E-08 
6.13 
10.00 

GO:0046889 
positive regulation of lipid biosynthetic process 
9.74E-07 1.18E-09 4.19E-05 2.36E-08 
6.73 
7.00 

GO:0031102 
neuron projection regeneration 
2.10E-05 1.18E-09 3.37E-04 2.36E-08 
7.58 
5.00 

GO:0034381 
plasma lipoprotein particle clearance 
2.37E-06 1.18E-09 8.53E-05 2.36E-08 
7.89 
6.00 

GO:0060193 
positive regulation of lipase activity 
3.47E-08 1.18E-09 2.36E-06 2.36E-08 
8.16 
8.00 

Continued on next page 
AIMS Neuroscience 
Volume 8, Issue 1, 86-132. 

Gene Ontology Identification Number 
Raw pValue 
Corrected pValue* 

Term 
Group 
Term 
Group 
% Associated Genes Nr. Genes 

GO:0060191 
regulation of lipase activity 
5.45E-11 1.18E-09 4.85E-09 2.36E-08 
8.46 
11.00 

GO:1905953 
negative regulation of lipid localization 
9.36E-06 1.18E-09 2.15E-04 2.36E-08 
8.93 
5.00 

GO:0030301 
cholesterol transport 
1.56E-11 1.18E-09 1.41E-09 2.36E-08 
9.48 
11.00 

GO:0070328 
triglyceride homeostasis 
1.69E-07 1.18E-09 8.96E-06 2.36E-08 
12.24 
6.00 

GO:0033344 
cholesterol efflux 
8.66E-10 1.18E-09 7.18E-08 2.36E-08 
12.90 
8.00 

GO:0090207 
regulation of triglyceride metabolic process 
7.56E-08 1.18E-09 4.46E-06 2.36E-08 
13.95 
6.00 

GO:0032374 
regulation of cholesterol transport 
4.32E-10 1.18E-09 3.71E-08 2.36E-08 
14.04 
8.00 

GO:0034375 
high-density lipoprotein particle remodeling 
7.39E-08 1.18E-09 4.44E-06 2.36E-08 
22.73 
5.00 

GO:0051004 
regulation of lipoprotein lipase activity 
7.42E-11 1.18E-09 6.53E-09 2.36E-08 
25.00 
7.00 

GO:0043691 
reverse cholesterol transport 
1.80E-09 1.18E-09 1.39E-07 2.36E-08 
25.00 
6.00 

GO:0034380 
high-density lipoprotein particle assembly 
4.39E-08 1.18E-09 2.90E-06 2.36E-08 
25.00 
5.00 

GO:0051006 
positive regulation of lipoprotein lipase activity 
8.67E-09 1.18E-09 6.24E-07 2.36E-08 
33.33 
5.00 

GO:0010873 
positive regulation of cholesterol esterification 
1.36E-09 1.18E-09 1.07E-07 2.36E-08 
45.45 
5.00 

GO:0010810 
regulation of cell-substrate adhesion 
1.24E-07 3.89E-09 6.95E-06 7.40E-08 
4.06 
11.00 

GO:0031032 
actomyosin structure organization 
3.38E-07 3.89E-09 1.72E-05 7.40E-08 
4.22 
10.00 

GO:0010811 
positive regulation of cell-substrate adhesion 
1.27E-05 3.89E-09 2.54E-04 7.40E-08 
4.58 
7.00 

GO:1902905 
positive regulation of supramolecular fiber organization 
7.96E-09 3.89E-09 5.81E-07 7.40E-08 
4.60 
12.00 

Continued on next page 
AIMS Neuroscience 
Volume 8, Issue 1, 86-132. 

Gene Ontology Identification Number 
Raw pValue 
Corrected pValue* 

Term 
Group 
Term 
Group 
% Associated Genes Nr. Genes 

GO:0034446 
substrate adhesion-dependent cell spreading 
2.02E-05 3.89E-09 3.44E-04 7.40E-08 
5.45 
6.00 

GO:0043149 
stress fiber assembly 
3.54E-06 3.89E-09 1.03E-04 7.40E-08 
5.56 
7.00 

GO:0110020 
regulation of actomyosin structure organization 
2.85E-06 3.89E-09 9.99E-05 7.40E-08 
5.74 
7.00 

GO:0001954 
positive regulation of cell-matrix adhesion 
3.92E-05 3.89E-09 3.53E-04 7.40E-08 
6.67 
5.00 

GO:0002040 
sprouting angiogenesis 
4.14E-06 5.75E-09 1.12E-04 1.04E-07 
5.43 
7.00 

GO:0043534 
blood vessel endothelial cell migration 
6.86E-08 5.75E-09 4.26E-06 1.04E-07 
6.00 
9.00 

GO:1903670 
regulation of sprouting angiogenesis 
5.68E-05 5.75E-09 2.84E-04 1.04E-07 
6.17 
5.00 

GO:0002042 
cell migration involved in sprouting angiogenesis 
3.01E-05 5.75E-09 3.91E-04 1.04E-07 
7.04 
5.00 

GO:0016101 
diterpenoid metabolic process 
2.85E-06 6.38E-09 9.99E-05 1.08E-07 
5.74 
7.00 

GO:0006695 
cholesterol biosynthetic process 
7.24E-06 6.38E-09 1.74E-04 1.08E-07 
6.52 
6.00 

GO:0015914 
phospholipid transport 
3.45E-06 6.38E-09 1.07E-04 1.08E-07 
7.41 
6.00 

GO:0006641 
triglyceride metabolic process 
1.02E-09 6.38E-09 8.26E-08 1.08E-07 
7.75 
10.00 

GO:0097006 
regulation of plasma lipoprotein particle levels 
5.88E-10 6.38E-09 5.00E-08 1.08E-07 
8.20 
10.00 

GO:0046503 
glycerolipid catabolic process 
9.33E-08 6.38E-09 5.41E-06 1.08E-07 
9.46 
7.00 

GO:0042632 
cholesterol homeostasis 
1.11E-09 6.38E-09 8.87E-08 1.08E-07 
9.57 
9.00 

GO:0071825 
protein-lipid complex subunit organization 
3.25E-12 6.38E-09 2.95E-10 1.08E-07 
13.70 
10.00 

GO:0019433 
triglyceride catabolic process 
7.56E-08 6.38E-09 4.46E-06 1.08E-07 
13.95 
6.00 

Continued on next page 
AIMS Neuroscience 
Volume 8, Issue 1, 86-132. 

Gene Ontology Identification Number 
Raw pValue 
Corrected pValue* 

Term 
Group 
Term 
Group 
% Associated Genes Nr. Genes 

GO:0034369 
plasma lipoprotein particle remodeling 
6.22E-10 6.38E-09 5.22E-08 1.08E-07 
18.92 
7.00 

GO:0034378 
chylomicron assembly 
2.44E-08 6.38E-09 1.71E-06 1.08E-07 
27.78 
5.00 

GO:0007219 
Notch signaling pathway 
4.10E-08 4.10E-08 2.74E-06 6.55E-07 
4.53 
11.00 

GO:0008593 
regulation of Notch signaling pathway 
4.56E-05 4.10E-08 3.19E-04 6.55E-07 
4.72 
6.00 

GO:0031099 
regeneration 
4.84E-08 4.84E-08 3.15E-06 7.26E-07 
4.45 
11.00 

GO:0010001 
glial cell differentiation 
6.97E-08 6.97E-08 4.25E-06 9.76E-07 
4.30 
11.00 

GO:0048708 
astrocyte differentiation 
1.17E-05 6.97E-08 2.58E-04 9.76E-07 
6.00 
6.00 

GO:0051057 
positive regulation of small GTPase mediated signal 
transduction 

4.22E-07 4.22E-07 2.07E-05 5.49E-06 
7.61 
7.00 

GO:1902003 
regulation of amyloid-beta formation 
5.43E-07 7.45E-07 2.55E-05 8.93E-06 
15.63 
5.00 

GO:1902992 
negative regulation of amyloid precursor protein catabolic 
process 

7.39E-08 7.45E-07 4.44E-06 8.93E-06 
22.73 
5.00 

GO:0002920 
regulation of humoral immune response 
1.09E-06 1.09E-06 4.46E-05 1.20E-05 
5.23 
8.00 

GO:0090288 
negative regulation of cellular response to growth factor 
stimulus 

3.03E-05 2.62E-06 3.63E-04 2.09E-05 
4.00 
7.00 

GO:0090101 
negative regulation of transmembrane receptor protein 
serine/threonine kinase signaling pathway 

7.11E-06 2.62E-06 1.78E-04 2.09E-05 
5.00 
7.00 

GO:0030512 
negative regulation of transforming growth factor beta 
receptor signaling pathway 

6.39E-05 2.62E-06 2.56E-04 2.09E-05 
6.02 
5.00 

Continued on next page 
AIMS Neuroscience 
Volume 8, Issue 1, 86-132. 

Gene Ontology Identification Number 
Raw pValue 
Corrected pValue* 

Term 
Group 
Term 
Group 
% Associated Genes Nr. Genes 

GO:0019985 
translesion synthesis 
3.06E-07 2.62E-06 1.59E-05 2.09E-05 
11.11 
6.00 

GO:0006296 
nucleotide-excision repair, DNA incision, 5'-to lesion 
3.13E-06 2.62E-06 1.00E-04 2.09E-05 
11.11 
5.00 

GO:0061418 
regulation of transcription from RNA polymerase II promoter 
in response to hypoxia 

1.49E-07 2.62E-06 8.20E-06 2.09E-05 
12.50 
6.00 

GO:0008361 
regulation of cell size 
7.27E-07 2.11E-06 3.27E-05 2.11E-05 
4.55 
9.00 

GO:0001952 
regulation of cell-matrix adhesion 
1.84E-06 2.11E-06 6.97E-05 2.11E-05 
4.88 
8.00 

GO:0061387 
regulation of extent of cell growth 
3.47E-05 2.11E-06 3.47E-04 2.11E-05 
4.96 
6.00 

GO:0048041 
focal adhesion assembly 
2.36E-05 2.11E-06 3.30E-04 2.11E-05 
5.31 
6.00 

GO:0048844 
artery morphogenesis 
2.37E-06 2.37E-06 8.53E-05 2.13E-05 
7.89 
6.00 

GO:0045824 
negative regulation of innate immune response 
1.32E-05 1.32E-05 2.50E-04 7.90E-05 
8.33 
5.00 

GO:0046887 
positive regulation of hormone secretion 
1.22E-05 1.22E-05 2.56E-04 8.53E-05 
4.61 
7.00 

GO:0051341 
regulation of oxidoreductase activity 
3.31E-05 3.31E-05 3.64E-04 1.66E-04 
5.00 
6.00 

GO:0050999 
regulation of nitric-oxide synthase activity 
1.95E-05 3.31E-05 3.52E-04 1.66E-04 
7.69 
5.00 

GO:0042246 
tissue regeneration 
4.74E-05 4.74E-05 2.84E-04 1.90E-04 
6.41 
5.00 

GO:0003279 
cardiac septum development 
6.68E-05 6.68E-05 2.00E-04 2.00E-04 
4.41 
6.00 

Continued on next page 
Volume 8, Issue 1, 86-132. 

Gene Ontology Identification Number 
Raw pValue 
Corrected pValue* 

Term 
Group 
Term 
Group 
% Associated Genes Nr. Genes 

GO:0031100 
animal organ regeneration 
3.76E-04 3.76E-04 3.76E-04 3.76E-04 
4.13 
5.00 

GO:0042310 
vasoconstriction 
2.22E-04 2.22E-04 4.44E-04 4.44E-04 
4.63 
5.00 




Cluster 2. Informative Data.Cluster 2. Informative Data.Volume 8, Issue 1, 86-132. 

40.00% regulation of fat cell differentiation 

20.00% negative regulation of endothelial cell 
migration 
20.00% regulation of tyrosine phosphorylation of 
STAT protein 
20.00% cellular response to dexamethasone stimulus 
AIMS Neuroscience 
Volume 8, Issue 1, 86-132. 

Gene Ontology Identification Number 
Raw pValue 
Corrected pValue* 

Term 
Group 
Term 
Group 
% Associated Genes Nr. Genes 

GO:0045598 
regulation of fat cell differentiation 
1.61E-07 8.06E-07 1.61E-07 6.45E-07 
4.46 
7.00 

GO:0045599 
negative regulation of fat cell differentiation 
4.09E-07 1.64E-06 1.61E-07 6.45E-07 
8.62 
5.00 

GO:0010596 
negative regulation of endothelial cell migration 
9.83E-07 2.95E-06 9.83E-07 2.95E-06 
7.25 
5.00 

GO:0071549 
cellular response to dexamethasone stimulus 
3.18E-06 6.36E-06 3.18E-06 6.36E-06 
10.26 
4.00 

GO:0042509 
regulation of tyrosine phosphorylation of STAT protein 
1.15E-04 1.15E-04 1.15E-04 1.15E-04 
4.17 
4.00 


Cluster 1. Informative Data.
Cluster 2. Informative Data.
Cluster 3. Informative Data.
Cluster 4. Informative Data.
Cluster 5. Informative Data.
Cluster 6. Informative Data.
¬© 2021 the Author(s), licensee AIMS Press. This is an open access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0)
AIMS NeuroscienceVolume 8, Issue 1, 86-132. The figure shows the enriched Cluster 3, derived from the initial selection of genes, after exclusion of previous clusters. Cluster 3 depicts the interacting genes and the type of interaction as shown in the figure legend. The Bar Chart associated with cluster 3 reports the percentage of genes belonging to a specific biological process (from Gene Ontology database) included in the pathway under analysis. It also shows the associations levels of biological processes with the cluster under analysis. The pie chart shows the overall degree of association of main biological processes involved with the cluster under investigation. The table summarizes the details regarding the pathway analysis and shows all of the associated biological processes linked with the cluster under investigation.   The figure shows the enriched Cluster 5, derived from the initial selection of genes, after exclusion of previous clusters. Cluster 5 depicts the interacting genes and the type of interaction as shown in the figure legend. The Bar Chart associated with cluster 5 reports the percentage of genes belonging to a specific biological process (from Gene Ontology database) included in the pathway under analysis. It also shows the associations levels of biological processes with the cluster under analysis. The pie chart shows the overall degree of association of main biological processes involved with the cluster under investigation. The table summarizes the details regarding the pathway analysis and shows all of the associated biological processes linked with the cluster under investigation. The figure shows the enriched Cluster 6, derived from the initial selection of genes, after exclusion of previous clusters. Cluster 6 depicts the interacting genes and the type of interaction as shown in the figure legend. The Bar Chart associated with cluster 6 reports the percentage of genes belonging to a specific biological process (from Gene Ontology database) included in the pathway under analysis. It also shows the associations levels of biological processes with the cluster under analysis. The pie chart shows the overall degree of association of main biological processes involved with the cluster under investigation. To note, only main biological processes are represented in the pie chart. The table summarizes the details regarding the pathway analysis and shows all of the associated biological processes linked with the cluster under investigation.AIMS NeuroscienceAIMS NeuroscienceAIMS Neuroscience
GO:0042254 ribosome biogenesis 7.16E-15 7.16E-15 4. GO:0042254 ribosome biogenesis 7.16E-15 7.16E-15 4.30E-14 2.15E-14

GO:0042274 ribosomal small subunit biogenesis 1.80E-11 7.16E-15 7.19E-11. 2GO:0042274 ribosomal small subunit biogenesis 1.80E-11 7.16E-15 7.19E-11 2.15E-14

. AIMS Neuroscience. 81AIMS Neuroscience Volume 8, Issue 1, 86-132.

overall summary of the biological processes associated (p <0.05 with Bonferroni step down correction) with the clusters enriched from the initial list of genes (see Table1). Enrichment was based on STRING database (STRING-ACTIONS_v10.5) while pathway recognition used Gene Ontology Database (GO_BiologicalProcess-EBI-UniProt-GOA). Both analyses were performed on Cytoscape software [22], through ClueGO and CluePedia plugins [23Supplementary Figure 1. Reports the overall summary of the biological processes associated (p <0.05 with Bonferroni step down correction) with the clusters enriched from the initial list of genes (see Table1). Enrichment was based on STRING database (STRING-ACTIONS_v10.5) while pathway recognition used Gene Ontology Database (GO_BiologicalProcess-EBI-UniProt-GOA). Both analyses were performed on Cytoscape software [22], through ClueGO and CluePedia plugins [23].

Alzheimer Disease. A Kumar, J W Tsao, StatPearls, Treasure Island (FLKumar A, Tsao JW (2018) Alzheimer Disease, in StatPearls, Treasure Island (FL).

Management of Alzheimer's disease. G T Grossberg, A K Desai, J Gerontol A Biol Sci Med Sci. 58Grossberg GT, Desai AK (2003) Management of Alzheimer's disease. J Gerontol A Biol Sci Med Sci 58: 331-353.

Hepatotoxic effects of tacrine administration in patients with Alzheimer's disease. P B Watkins, H J Zimmerman, M J Knapp, JAMA. 271Watkins PB, Zimmerman HJ, Knapp MJ, et al. (1994) Hepatotoxic effects of tacrine administration in patients with Alzheimer's disease. JAMA 271: 992-998.

3-year study of donepezil therapy in Alzheimer's disease: effects of early and continuous therapy. B Winblad, A Wimo, K Engedal, Dement Geriatr Cogn Disord. 21Winblad B, Wimo A, Engedal K, et al. (2006) 3-year study of donepezil therapy in Alzheimer's disease: effects of early and continuous therapy. Dement Geriatr Cogn Disord 21: 353-363.

Donepezil and paroxetine: possible drug interaction. L Carrier, J Am Geriatr Soc. 471037Carrier L (1999) Donepezil and paroxetine: possible drug interaction. J Am Geriatr Soc 47: 1037.

Donepezil in Alzheimer's disease: from conventional trials to pharmacogenetics. R Cacabelos, Neuropsychiatr Dis Treat. 3Cacabelos R (2007) Donepezil in Alzheimer's disease: from conventional trials to pharmacogenetics. Neuropsychiatr Dis Treat 3: 303-333.

Lack of adverse pharmacodynamic drug interactions with rivastigmine and twenty-two classes of medications. G T Grossberg, H B Stahelin, J C Messina, Int J Geriatr Psychiatry. 15Grossberg GT, Stahelin HB, Messina JC, et al. (2000) Lack of adverse pharmacodynamic drug interactions with rivastigmine and twenty-two classes of medications. Int J Geriatr Psychiatry 15: 242-247.

Response of patients with Alzheimer disease to rivastigmine treatment is predicted by the rate of disease progression. M R Farlow, A Hake, J Messina, Arch Neurol. 58Farlow MR, Hake A, Messina J, et al. (2001) Response of patients with Alzheimer disease to rivastigmine treatment is predicted by the rate of disease progression. Arch Neurol 58: 417-422.

Galantamine in AD: a 6-month randomized, placebo-controlled trial with a 6-month extension. The Galantamine USA-1 Study Group. M A Raskind, E R Peskind, T Wessel, Neurology. 54Raskind MA, Peskind ER, Wessel T, et al. (2000) Galantamine in AD: a 6-month randomized, placebo-controlled trial with a 6-month extension. The Galantamine USA-1 Study Group. Neurology 54: 2261-2268.

Memantine in severe dementia: results of the 9M-Best Study (Benefit and efficacy in severely demented patients during treatment with memantine). B Winblad, N Poritis, Int J Geriatr Psychiatry. 14Winblad B, Poritis N (1999) Memantine in severe dementia: results of the 9M-Best Study (Benefit and efficacy in severely demented patients during treatment with memantine). Int J Geriatr Psychiatry 14: 135-146.

Fulminant chemical hepatitis possibly associated with donepezil and sertraline therapy. M M Verrico, D A Nace, A L Towers, J Am Geriatr Soc. 48Verrico MM, Nace DA, Towers AL (2000) Fulminant chemical hepatitis possibly associated with donepezil and sertraline therapy. J Am Geriatr Soc 48: 1659-1663.

Molecular genetic analysis of the APP, PSEN1, and PSEN2 genes in Finnish patients with early-onset Alzheimer disease and frontotemporal lobar degeneration. J Kruger, V Moilanen, K Majamaa, Alzheimer Dis Assoc Disord. 26Kruger J, Moilanen V, Majamaa K, et al. (2012) Molecular genetic analysis of the APP, PSEN1, and PSEN2 genes in Finnish patients with early-onset Alzheimer disease and frontotemporal lobar degeneration. Alzheimer Dis Assoc Disord 26: 272-276.

Immunohistochemical detection of 4-hydroxy-2-nonenal adducts in Alzheimer's disease is associated with inheritance of APOE4. K S Montine, S J Olson, V Amarnath, Am J Pathol. 150Montine KS, Olson SJ, Amarnath V, et al. (1997) Immunohistochemical detection of 4- hydroxy-2-nonenal adducts in Alzheimer's disease is associated with inheritance of APOE4. Am J Pathol 150: 437-443.

Effects of age, sex, and ethnicity on the association between apolipoprotein E genotype and Alzheimer disease. A meta-analysis. APOE and Alzheimer Disease Meta Analysis Consortium. L A Farrer, L A Cupples, J L Haines, JAMA. 278Farrer LA, Cupples LA, Haines JL, et al. (1997) Effects of age, sex, and ethnicity on the association between apolipoprotein E genotype and Alzheimer disease. A meta-analysis. APOE and Alzheimer Disease Meta Analysis Consortium. JAMA 278: 1349-1356.

Whole exome sequencing study identifies novel rare and common Alzheimer's-Associated variants involved in immune response and transcriptional regulation. J C Bis, X Jian, B W Kunkle, Mol Psychiatry. 25Bis JC, Jian X, Kunkle BW, et al. (2020) Whole exome sequencing study identifies novel rare and common Alzheimer's-Associated variants involved in immune response and transcriptional regulation. Mol Psychiatry 25: 1859-1875.

Identification of blood biomarkers for Alzheimer's disease through computational prediction and experimental validation. F Yao, K Zhang, Y Zhang, Front Neurol. 91158Yao F, Zhang K, Zhang Y, et al. (2018) Identification of blood biomarkers for Alzheimer's disease through computational prediction and experimental validation. Front Neurol 9: 1158.

Identification of novel immune-relevant drug target genes for Alzheimer's disease by combining ontology inference with network analysis. Z J Han, W W Xue, L Tao, CNS Neurosci Ther. 24Han ZJ, Xue WW, Tao L, et al. (2018) Identification of novel immune-relevant drug target genes for Alzheimer's disease by combining ontology inference with network analysis. CNS Neurosci Ther 24: 1253-1263.

Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer's disease risk. I E Jansen, J E Savage, K Watanabe, Nat Genet. 51Jansen IE, Savage JE, Watanabe K, et al. (2019) Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer's disease risk. Nat Genet 51: 404-413.

Rare genetic variation implicated in non-Hispanic white families with Alzheimer disease. G W Beecham, B Vardarajan, E Blue, Neurol Genet. 4286Beecham GW, Vardarajan B, Blue E, et al. (2018) Rare genetic variation implicated in non- Hispanic white families with Alzheimer disease. Neurol Genet 4: e286.

Alzheimer's disease in the omics era. G M Sancesario, S Bernardini, Clin Biochem. 59Sancesario GM, Bernardini S (2018) Alzheimer's disease in the omics era. Clin Biochem 59: 9- 16.

PU.1 regulates Alzheimer's diseaseassociated genes in primary human microglia. J Rustenhoven, A M Smith, L C Smyth, Mol Neurodegener. 1344Rustenhoven J, Smith AM, Smyth LC, et al. (2018) PU.1 regulates Alzheimer's disease- associated genes in primary human microglia. Mol Neurodegener 13: 44.

Cytoscape: a software environment for integrated models of biomolecular interaction networks. P Shannon, A Markiel, O Ozier, Genome Res. 13Shannon P, Markiel A, Ozier O, et al. (2003) Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res 13: 2498-2504.

ClueGO: a Cytoscape plug-in to decipher functionally grouped gene ontology and pathway annotation networks. G Bindea, B Mlecnik, H Hackl, Bioinformatics. 25Bindea G, Mlecnik B, Hackl H, et al. (2009) ClueGO: a Cytoscape plug-in to decipher functionally grouped gene ontology and pathway annotation networks. Bioinformatics 25: 1091-1093.