# Clinicopathological and prognostic significance of long non-coding RNA EWSAT1 in human cancers: A review and meta analysis

CorpusID: 247450338
 
tags: #Medicine

URL: [https://www.semanticscholar.org/paper/4599aa0e8a97131668e169bf5132627347692222](https://www.semanticscholar.org/paper/4599aa0e8a97131668e169bf5132627347692222)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Clinicopathological and prognostic significance of long non-coding RNA EWSAT1 in human cancers: A review and meta analysis


Jian Wen 
People's Clinical Medical College
Nanchang University
NanchangJiangxiChina

Haima 2☯ 
Department of Orthopedics
Jiangxi Provincial People's Hospital
NanchangJiangxiChina

Li 
People's Clinical Medical College
Nanchang University
NanchangJiangxiChina

Dongdong 3☯ 
Department of Neurosurgery
Jiangxi Provincial People's Hospital
NanchangJiangxiChina

Li 
People's Clinical Medical College
Nanchang University
NanchangJiangxiChina

Department of Pulmonary and Critical Care Medicine
Jiangxi Provincial People's Hospital
NanchangJiangxiChina

Xieping Dong Id 
People's Clinical Medical College
Nanchang University
NanchangJiangxiChina

Department of Orthopedics
Jiangxi Provincial People's Hospital
NanchangJiangxiChina

Clinicopathological and prognostic significance of long non-coding RNA EWSAT1 in human cancers: A review and meta analysis
RESEARCH ARTICLE ☯ These authors contributed equally to this work. ‡ DL also contributed equally to this work. *
BackgroundEwing sarcoma-associated transcript 1 (lncRNA EWSAT1) is reported to have a close relationship with the overall survival in many cancers. However, the role of its prognosis and correlations with the clinicopathological features in different cancers haven't been explored yet. Herein, we intend to assess the prognostic value and correlations with the clinicopathological features in several cancers.MethodsPubMed, Embase, Web of Science, and The Cochrane Library were searched for literature review from inception to October 25, 2021. Valid data was extracted to make forest and sensitivity analysis plots using Review Manager 5.4 and Stata software. Hazard ratio (HR) or odds ratio (OR) with 95% confidence interval (CI) was used to evaluate the relationship between different expression of EWSAT1 and patients' prognosis and clinicopathological features.Results7 studies were screened for this review, including 550 samples. Meta-analysis showed that high expression of lncRNA EWSAT1 was associated with poor overall survival (OS) (HR = 2.10, 95% CI, 1.60-2.75, p < 0.0001) in cancers reported. In addition, patients in high expression group of EWAST1 tended to have more metastasis (OR = 2.20, 95% CI 1.47-3.31, p = 0.0001), and higher TNM stage (I+II vs. III: OR = 0.34, 95% CI 0.21-0.56, p < 0.0001), but in the same time with higher differentiation (well + moderate vs. Poor: OR = 2.21, 95% CI 1.02-4.76, p = 0.04). Age (OR = 1.47, 95% CI 0.94-2.30, p = 0.09) was not significantly different in patients with aberrant expression of EWSAT1. Citation: Wen J, Li H, Li D, Dong X (2022) Clinicopathological and prognostic significance of long non-coding RNA EWSAT1 in human cancers: A review and meta analysis. PLoS ONE 17(3): e0265264. https://doi.org/10.
a1111111111 a1111111111 a1111111111 a1111111111 a1111111111
Introduction curve (sufficient data to compute the hazard ratio (HR) and 95% CI of survival), histological (differentiation) grade, TNM stage, distant/lymphnode metastasis. The exclusion criteria were as follows: (1) publication letters, case reports, reviews, conference abstracts, etc.; (2) studies without or not enough clinical data; (3) studies unrelated to lncRNA EWSAT1.


## Data extraction

All articles were reviewed by two independent well trained investigators and a third author was referred when there was any disagreement. The following information was collected according to the inclusion criteria: name of first author, publication year, country of origin, tumor type, sample type, sample size, detection method, Hazard ratio (HR) of OS between high and low EWSAT1 expression groups and the corresponding 95% CI (if not provided directly while a Kaplan-Meier curve was available, Engauge Digitizer version 4.1 and Tierney's method were used to extract HR, 95% CI, Variance and O-E), and the quality of each study was assessed by Newcastle-Ottawa Scale (NOS).


## Quality assessment

The quality of the literature was evaluated by Newcastle-Ottawa Scale (NOS) with a total score of 10. The higher the score, the better the quality of the literature.


## Data synthesis and statistical analysis

When there was only a Kaplan-Meier curve available, we would use Engauge Digitizer version 4.1 and Tierney's method to calculate HR, 95% CI, Variance and O-E. Then, Review Manager 5.4 was employed for further data analysis. Q-test and I 2 were applied to evaluate the heterogeneity among the studies. If the heterogeneity was substantial (P < 0.10, I 2 > 50%), the random effect model would be adopted, otherwise the fixed effect model would be used. Forest plots were drawn to display the meta-analysis results and funnel plots were used to assess any prospective bias in the publications. Sensitive analyses were conducted using local sensitivity analysis approach by excluding each eligible study one at a time, with plots drawn by Stata 12.0 (Stata, College Station, TX, USA). Finally, P < 0.05 was considered statistically significant in our analyses above.


## Expressions of EWSAT1 in pancancer

The TCGA PanCancer Atlas refers to a data set of molecular and clinical information from over 10,000 tumors representing 33 types of cancer [20]. Therefore, UCSC Toil RNA-seq Recompute data of TCGA and GTEx database (https://xenabrowser.net/datapages/) was used to explore the expression of EWSAT1 in pancancer (33 types of cancer in TCGA) by R3.6.3 [21]. Wilcoxon rank sum test was applied to detect the differences between normal and tumor groups in pancancer. And ggplot2 package in R was adopted for result visualization. This study involved 550 patient form China (6 studies, n = 527) and Romania (1 study, n = 23), including 4 cancers: cervical cancer (n = 23), colorectal cancer (n = 151), glioma (n = 42), nasopharyngeal carcinoma (n = 108), osteosarcoma (n = 226). The minimum sample size of each study was 23, and the maximum was 176. Publication Time started from 2016 to 2021. The expressions of EWSAT1 were all detected by qRT-PCR with cancer tissues. The cutoff value for dividing high and low expression group of EWSAT1 varied in the studies ( Table 1). The HRs and 95% CIs of OS between high and low expression groups of EWSAT1 were all extracted from Kaplan-Meier curves provided by original articles. 3 out of 7 articles got a 7 score of NOS and 4 got 6. The main essential characteristics of these studies are shown in Table 1. With the EWSAT1 high of 301 patients and low of 226 patients, total 527 patients, 3 articles touched the line of no effect, heterogeneity among studies was not significant (P = 0.86, I 2 = 0.0%), the fixed effect model was adopted. The result showed that the overall effect estimated HR was 2.10 and 95% CI was 1.60-2.75, P<0.00001, which indicated high expression of EWSAT was associated with low OS. Since there was no significant heterogeneity among the studies, subgroup analysis wasn't performed (P = 0.676, I 2 = 0.0%).


## Results


## Characteristics of studies


## Association between aberrant expression of EWSAT1 and OS


## Association between aberrant expression of EWSAT1 and clinicopathological features

In order to further explore the association between aberrant expression of EWSAT1 and the prognosis of patients, several clinicopathological Features was introduced, including: metastasis ( Table 2).

ORs and the 95% CIs were employed to investigate the relationship between different expression of EWSAT1 and clinicopathological features above. The results were shown in Fig  3 and Table 3, which indicated that metastasis (OR: 2.20, 95% CI: 1.47-3.31, p = 0.0001), differentiation (OR: 2.21, 95% CI: 1.02-4.76, p = 0.04) and TNM stage (OR: 0.34, 95% CI: 0.21-0.56, p<0.0001) were associated with the aberrant expression of EWSAT1, high expression of  


## Publication bias and sensitivity analysis

When more than three articles were involved in an analysis, publication bias was estimated by the funnel plot. Results were shown in Fig 4A and 4B. We deemed that no significant publication bias was found in the two analyses. Sensitivity analysis was also performed in the two analyses, using local sensitivity analysis approach by excluding each eligible study one at a time. Results showed that there was no significant change in our conclusions, thereby confirmed the robustness and liability of our conclusions in this meta-analysis (Fig 4C and 4D). 


## PLOS ONE


## Expressions of EWSAT1 in pancancer

The expression of EWSAT1 was explored in pancancer, results were shown in Fig 5. Aberrant expression of EWSAT1 was validated between normal and tumor tissues. Results showed that EWSAT1 was significantly increased in cancers like Skin Cutaneous Melanoma (SKCM), Glioblastoma multiforme (GBM), Ovarian serous cystadenocarcinoma (OV), and etc., while it was  


## Discussion

With the development of RNA sequencing technology, accumulating evidence shows that lncRNAs can regulate the expression of genes though many elaborate mechanism [23]. LncRNAs can react with DNA, RNA and protein directly or indirectly [24][25][26], and ultimately affect the function of body, resulting in dysfunctions, diseases, even cancers [27][28][29][30][31]. More and more studies show lncRNAs can serve as novel biomarkers or targets for diagnosis, treatment, prognosis of cancers due to their important functions in cancer generation, proliferation, progression and metastasis [8,[32][33][34][35][36]. Furthermore, when combined with different clinicopathological features, more accurate condition of every single patient can be got, and correspondingly more individualized treatment can be made [37,38].

In this meta-analysis, we analyzed 6 studies with HR of OS in 4 different cancers, the overall effect showed that high expression of lncRNA EWSAT1 linked to low OS, which indicated EWSAT1 an oncogene in the cancers involved. 2 studies touched the line of no effect, but the sensitive analysis showed our conclusion was robust and reliable. Though in most studies the expression of EWSAT1 was up regulated in cancers, but there also cancers in which it was down regulated (Fig 5). This meant that EWSAT1 probably could act as a protective factor in some circumstances as well. However, constrained to the limited clinical data and studies to date, more data in this area was needed to further clarify this problem.

Moreover, the relationship between 4 clinical features and EWSAT1 expression was also investigated. We found that high expression of EWAST1 tended to have more metastasis and higher TNM stage, which usually indicated late stage of cancers and poor prognosis. This conclusion was consistent with the previous one. However, more patients of well and moderate differentiation cancers were discovered in EWSAT1 high expression group with OR 2.20, 95% CI 1.02-4.76. We would like to contribute this effect to several reasons. Firstly, the analysis included only two small sample studies (total n = 129), the sampling error was large and the result was not that stable and reliable. We call for more studies with large samples to make this conclusion robust and reliable. Secondly, more specialized subgroups were needed to further evaluate the effect. It seemed more reasonable to divide the samples into well, moderate and poor 3 groups according to the degree of differentiation. Unfortunately, no more detailed original data was available. Thirdly, the cutoff values varied in different studies, which would contribute heterogeneity in the result. Generally speaking, median was more recommended as the threshold for converting a continuous variable to a dichotomous variable in the biomedical field [39,40]. Of course, there were also some other ways of grouping according to the data characteristics and actual needs, like mean. Although the cutoff values in the studies included were different, it's still meaningful and of considerable value to combine these studies and make a meta analysis, because the diseases in these studies were essentially different and they were relatively independent. Last but not least, some other factors could also affect the result. For example, it's known that some type of cancers tended to develop into high and moderate degree of differentiation, and our treatments to this type of patients were effective and mature. While the situation of poor differentiation cancers was on the opposite side, and these patients usually with short OS and small quantity that alive, which also could result in bias. In addition, Age (>60y, �60y) was found to have no significant association with the expression of EWSAT1 statistically in this analysis.

Aberrant expression of EWSAT1 was validated in pancancer using data from UCSC Toil RNA-seq Recompute data of TCGA and GTEx database. The aberrant expression in multiple cancers demonstrated that EWSAT1 had a strong association with varies cancers. The results of GBM and OV were in accord with the conclusions of our included studies.

In general, the results of our meta analysis suggest that lncRNA EWSAT1 acts as an oncogene in many cancers, which relate to low OS, more metastasis, higher stage of TNM stage. The expression level of EWSAT1 could be used to predict possible prognosis, imply the progression stage of cancers, indicate a cancer that may exist for early diagnosis, act as a potential biomarker and drug target in treatment. Meanwhile, when combined with several key clinicopathological features, doctors can make the most of a patient's condition, and guide an individualized treatment.

However, some limitations of this meta-analysis should be noticed. For one thing, there's limited data on EWSAT1 so far, only 7 eligible studies with 550 patients included, which may reduce the stringency of the conclusion. This is also suitable for analysis of the clinicopathological features. For another thing, the HRs and 95% CIs extracted by software from Kaplan-Meier curves may lead to extraneous heterogeneity. In addition, the fact that treatments varied in different cancers might affect the OS and add to heterogeneity as well. However, we deemed that it was still comparable between patients in each study and could be combined for meta analysis, for all patients underwent surgery and patients in each study were treated in the same hospital. Here, confined to the limited data and clinical fact, it's temporarily impossible to discuss the influence of aberrant EWSAT1 expression on OS under the same treatments. Lastly, different studies or cancers have somehow different criteria for evaluating parameters and there is only one small study from non-Asian region, which may also add bias.

In summary, high expression of EWSAT1 is associate with poor OS, more metastasis, higher stage of TNM stage in many cancers. Thus, the results of our meta-analysis indicate that EWSAT1 may be selected as a potential prognosis biomarker for multiple cancers.  


## Supporting information

## Fig 2
2presents the association between EWSAT1 aberrant Expression and OS. Of the 7 articles included, 6 articles (Peng Song, etc. 2016; G.-Y. ZHANG, etc. 2017; R. ZHANG, etc. 2018; Fig 1. Flow diagram of this meta-analysis. https://doi.org/10.1371/journal.pone.0265264.g001 HUI YANG, etc. 2020; Jing Liu, etc. 2021; Dewei Shen, etc. 2021) were involved in this analysis, 1 article (Iulia Virginia Iancu, etc. 2017) was excluded for not providing either HR or a Kaplan-Meier curve.


Dewei Shen, etc. 2021; G.-Y. ZHANG, etc. 2017; Iulia Virginia Iancu, etc. 2017; Jing Liu, etc. 2021; R. ZHANG, etc. 2018; n = 400), differentiation (Iulia Virginia Iancu, etc. 2017; R. ZHANG, etc. 2018; n = 129), age (G.-Y. ZHANG, etc. 2017; R. ZHANG, etc. 2018; HUI YANG, etc. 2020; n = 324) and TNM stage (G.-Y. ZHANG, etc. 2017; R. ZHANG, etc. 2018; n = 282) (

## Fig 2 .
2Forest plot for the association of EWSAT1 expression with overall survival. https://doi.org/10.1371/journal.pone.0265264.g002

## Fig 3 .
3Forest plots for the association of EWSAT1 expression with several clinicopathological features. (A) Forest plots for the association of EWSAT1 expression with cancer metastasis, (B) differentiation, (C) age, (D) TNM stage. https://doi.org/10.1371/journal.pone.0265264.g003

## Fig 4 .
4Funnel plots and sensitivity analyses. (A) Funnel plots for studies involved in analysis of HR and OS, (B) OR and metastasis;(C) Sensitivity analysis for studies involved in analysis of HR and OS, (D) OR and metastasis. https://doi.org/10.1371/journal.pone.0265264.g004 significantly decreased in Testicular Germ Cell Tumors (TGCT), Brain Lower Grade Glioma (LGG) and etc.

## Fig 5 .
5Expressions of EWSAT1 in tissues of normal and pancancer. Wilcoxon rank sum test: no significance (ns), p�0.05; � , p< 0.05; �� , p<0.01; ��� , p<0.001. https://doi.org/10.1371/journal.pone.0265264.g005

## S1
Checklist. PRISMA 2009 checklist. (DOC) Author Contributions Conceptualization: Jian Wen. Data curation: Jian Wen, Haima Li, Dongdong Li.


Xieping Dong. Visualization: Jian Wen. Writing -original draft: Jian Wen. Writing -review & editing: Xieping Dong.


In this meta-analysis, 7 studies (Peng Song, etc. 2016; G.-Y. ZHANG, etc. 2017; Iulia Virginia Iancu, etc. 2017; R. ZHANG, etc. 2018; HUI YANG, etc. 2020; Jing Liu, etc. 2021; Dewei Shen, etc. 2021) were included for meta-analysis after screening 49 candidate articles. Fig 1 shows our screening strategy and results, according to the PRISMA guidelines.

## Table 1 .
1Main characteristics of the included studies. https://doi.org/10.1371/journal.pone.0265264.t001 EWSAT1 tended to have more metastasis and high TNM stage, but better differentiation. Besides, in this analysis, age (OR: 1.47, 95% CI: 0.94-2.30, p = 0.09) didn't statistically show significant association with the expression of EWSAT1.Cancer 
First author 
Year Country Sample type Sample size(n) Detection 
method 

Cutoff 
HR(95%CI) of OS NOS 

nasopharyngeal carcinoma [22] 
Peng Song 
2016 
China 
tissue 
108 
qRT-PCR 2.36 Foldchange 
2.25(0.97-5.20) 
6 

osteosarcoma [12] 
G.-Y. ZHANG 
2017 
China 
tissue 
176 
qRT-PCR 
mean 
2.02(1.27-3.21) 
7 

cervical cancer [15] 
Iulia Virginia Iancu 2017 Romania 
tissue 
23 
qRT-PCR 
median 
NA 
6 

colorectal cancer [16] 
R. ZHANG 
2018 
China 
tissue 
106 
qRT-PCR Not mentioned 
1.64(0.97-2.77) 
6 

glioma [19] 
HUI YANG 
2020 
China 
tissue 
42 
qRT-PCR Not mentioned 
2.71(1.21-6.04) 
7 

colorectal cancer [17] 
Jing Liu 
2021 
China 
tissue 
45 
qRT-PCR Not mentioned 
2.49(0.98-6.34) 
6 

osteosarcoma [14] 
Dewei Shen 
2021 
China 
tissue 
50 
qRT-PCR 
median 
2.94(1.13-7.67) 
7 



## Table 2 .
2Distribution characteristics of EWSAT1 in the patients with different clinicopathological features.Article 
Cancer 
Sample size 
metastasis 
EWSAT1 high 
EWSAT1 low 
Total 

G.-Y. ZHANG 
2017 [12] 

osteosarcoma 
176 
present 
55 
36 
91 

absent 
35 
50 
85 

Iulia Virginia Iancu 2017 [15] 
cervical 
23 
present 
3 
7 
10 

absent 
8 
5 
13 

R. ZHANG 2018 [16] 
colorectal 
106 
present 
26 
16 
42 

absent 
24 
40 
64 

Jing Liu 2021 [17] 
colorectal 
45 
present 
17 
4 
21 

absent 
16 
8 
24 

Dewei Shen 2021 [14] 
osteosarcoma 
50 
present 
23 
6 
29 

absent 
9 
12 
21 

Total (Freq): 5 
4 types 
400 
present 
124 
69 
193 

absent 
92 
115 
207 

Article 
Cancer 
Sample size 
Differentiation 
EWSAT1 high 
EWSAT1 low 
Total 

Iulia Virginia Iancu 2017 [15] 
cervical 
23 
well+moderate 
9 
7 
16 

poor 
2 
5 
7 

R. ZHANG 2018 [16] 
colorectal 
106 
well+moderate 
38 
34 
72 

poor 
12 
22 
34 

Total (Freq): 2 
2 types 
129 
well+moderate 
47 
41 
88 

poor 
14 
27 
41 

Article 
Cancer 
Sample size 
Age 
EWSAT1 high 
EWSAT1 low 
Total 

G.-Y. ZHANG 2017 [12] 
osteosarcoma 
176 
<60y 
47 
41 
88 

�60y 
43 
45 
88 

R. ZHANG 2018 [16] 
colorectal 
106 
<60y 
38 
34 
72 

�60y 
12 
22 
34 

HUI YANG 2020 [19] 
glioma 
42 
<60y 
14 
9 
23 

�60y 
9 
10 
19 

Total (Freq): 3 
3 types 
324 
<60y 
99 
84 
183 

�60y 
64 
77 
141 

Article 
Cancer 
Sample size 
TNM stage 
EWSAT1 high 
EWSAT1 low 
Total 

G.-Y. ZHANG 2017 [12] 
osteosarcoma 
176 
Ⅰ+Ⅱ 
31 
52 
83 

Ⅲ 
59 
34 
93 

R. ZHANG 2018 [16] 
colorectal 
106 
Ⅰ+Ⅱ 
25 
42 
67 

Ⅲ 
25 
14 
39 

Total (Freq): 2 
2 types 
282 
Ⅰ+Ⅱ 
56 
94 
150 

Ⅲ 
84 
48 
132 

https://doi.org/10.1371/journal.pone.0265264.t002 


## Table 3 .
3Association of EWSAT1 expression with clinicopathological features.Clinicopathological parameters studies (n) 
Patients(n) 
OR(95%CI) 
P-value 
Heterogeneity (I 2 , P) 
Model 

metastasis (present vs. absent) 
400 
2.20(1.47-3.31) 
0.0001 
47%, 0.11 
Fixed 

Differentiation grade (well+ moderate vs. poor) 
129 
2.21(1.02-4.76) 
0.04 
0%, 0.67 
Fixed 

Age (<60y vs. �60y) 
324 
1.47(0.94-2.30) 
0.09 
0%, 0.57 
Fixed 

TNM stage (I+II vs. III) 
282 
0.34(0.21-0.56) 
<0.0001 
0%, 0.95 
Fixed 

https://doi.org/10.1371/journal.pone.0265264.t003 


PLOS ONE | https://doi.org/10.1371/journal.pone.0265264 March 14, 2022 4 / 13
PLOS ONE | https://doi.org/10.1371/journal.pone.0265264March 14, 2022  

The global cancer burden and human development: A review. M M Fidler, F Bray, I Soerjomataram, 10.1177/140349481771540028669281Scand J Public Health. 461Fidler MM, Bray F, Soerjomataram I. The global cancer burden and human development: A review. Scand J Public Health. 2018; 46(1):27-36. Epub 2017/07/04. https://doi.org/10.1177/ 1403494817715400 PMID: 28669281.

Tumor-Treating Fields: A Fourth Modality in Cancer Treatment. E J Mun, H M Babiker, U Weinberg, E D Kirson, Von Hoff, D D , 10.1158/1078-0432.CCR-17-111728765323Clin Cancer Res. 242Mun EJ, Babiker HM, Weinberg U, Kirson ED, Von Hoff DD. Tumor-Treating Fields: A Fourth Modality in Cancer Treatment. Clin Cancer Res. 2018; 24(2):266-75. Epub 2017/08/03. https://doi.org/10.1158/ 1078-0432.CCR-17-1117 PMID: 28765323.

Top 10 Challenges in Cancer Immunotherapy. P S Hegde, D S Chen, 10.1016/j.immuni.2019.12.01131940268Immunity. 521Hegde PS, Chen DS. Top 10 Challenges in Cancer Immunotherapy. Immunity. 2020; 52(1):17-35. Epub 2020/01/16. https://doi.org/10.1016/j.immuni.2019.12.011 PMID: 31940268.

Long Noncoding RNA and Cancer: A New Paradigm. A Bhan, M Soleimani, S S Mandal, 10.1158/0008-5472.CAN-16-263428701486Cancer Res. 77158330958PubMed Central PMCIDBhan A, Soleimani M, Mandal SS. Long Noncoding RNA and Cancer: A New Paradigm. Cancer Res. 2017; 77(15):3965-81. Epub 2017/07/14. https://doi.org/10.1158/0008-5472.CAN-16-2634 PMID: 28701486; PubMed Central PMCID: PMC8330958.

New insights into long noncoding RNAs and their roles in glioma. Z Peng, C Liu, M Wu, 10.1186/s12943-018-0812-22018/02/21Mol Cancer. 17161Peng Z, Liu C, Wu M. New insights into long noncoding RNAs and their roles in glioma. Mol Cancer. 2018; 17(1):61. Epub 2018/02/21. https://doi.org/10.1186/s12943-018-0812-2 PMID: 29458374;

. Pubmed Central, Pmcid , 5817731PubMed Central PMCID: PMC5817731.

Long Non-Coding RNA in the Pathogenesis of. Y Chi, D Wang, J Wang, W Yu, J Yang, 10.3390/cells809101531480503Cancers. Cells. 896770362PubMed Central PMCIDChi Y, Wang D, Wang J, Yu W, Yang J. Long Non-Coding RNA in the Pathogenesis of Cancers. Cells. 2019; 8(9). Epub 2019/09/05. https://doi.org/10.3390/cells8091015 PMID: 31480503; PubMed Central PMCID: PMC6770362.

The Application of lncRNAs in Cancer Treatment and Diagnosis. Recent Pat Anticancer Drug Discov. Y Zhang, L Tang, 10.2174/15748928136661802261218192948501013Zhang Y, Tang L. The Application of lncRNAs in Cancer Treatment and Diagnosis. Recent Pat Antican- cer Drug Discov. 2018; 13(3):292-301. Epub 2018/02/28. https://doi.org/10.2174/ 1574892813666180226121819 PMID: 29485010.

Long noncoding RNAs in cancer: From discovery to therapeutic targets. R Choudhari, M J Sedano, A L Harrison, R Subramani, K Y Lin, E I Ramos, 10.1016/bs.acc.2019.08.00332122521Adv Clin Chem. 95Choudhari R, Sedano MJ, Harrison AL, Subramani R, Lin KY, Ramos EI, et al. Long noncoding RNAs in cancer: From discovery to therapeutic targets. Adv Clin Chem. 2020; 95:105-47. Epub 2020/03/04. https://doi.org/10.1016/bs.acc.2019.08.003 PMID: 32122521.

Long noncoding RNA EWSAT1-mediated gene repression facilitates Ewing sarcoma oncogenesis. Marques Howarth, M Simpson, D Ngok, S P Nieves, B Chen, R Siprashvili, Z , 10.1172/JCI7212425401475J Clin Invest. 124124348951PubMed Central PMCIDMarques Howarth M, Simpson D, Ngok SP, Nieves B, Chen R, Siprashvili Z, et al. Long noncoding RNA EWSAT1-mediated gene repression facilitates Ewing sarcoma oncogenesis. J Clin Invest. 2014; 124 (12):5275-90. Epub 2014/11/18. https://doi.org/10.1172/JCI72124 PMID: 25401475; PubMed Central PMCID: PMC4348951.

LncRNA EWSAT1 Regulates the Tumorigenesis of NSCLC as a ceRNA by Modulating miR-330-5p/ITGA5 Axis. S Cui, C L Yang, D Y Chen, 10.1007/s10528-021-10069-42021/05/01Biochem Genet. 59633928467Cui S, Yang CL, Chen DY. LncRNA EWSAT1 Regulates the Tumorigenesis of NSCLC as a ceRNA by Modulating miR-330-5p/ITGA5 Axis. Biochem Genet. 2021; 59(6):1441-56. Epub 2021/05/01. https:// doi.org/10.1007/s10528-021-10069-4 PMID: 33928467.

Calycosin inhibits nasopharyngeal carcinoma cells by influencing EWSAT1 expression to regulate the TRAF6-related pathways. L Kong, X Li, H Wang, G He, A Tang, 10.1016/j.biopha.2018.06.14329966979Biomed Pharmacother. 106Kong L, Li X, Wang H, He G, Tang A. Calycosin inhibits nasopharyngeal carcinoma cells by influencing EWSAT1 expression to regulate the TRAF6-related pathways. Biomed Pharmacother. 2018; 106:342- 8. Epub 2018/07/04. https://doi.org/10.1016/j.biopha.2018.06.143 PMID: 29966979.

Clinical significance of long non-coding RNA EWSAT1 as a novel prognostic biomarker in osteosarcoma. G Y Zhang, J F Zhang, X M Hu, Z P Luo, Y Z Ma, 10.26355/eurrev_201712_1391829243774Eur Rev Med Pharmacol Sci. 2123Zhang GY, Zhang JF, Hu XM, Luo ZP, Ma YZ. Clinical significance of long non-coding RNA EWSAT1 as a novel prognostic biomarker in osteosarcoma. Eur Rev Med Pharmacol Sci. 2017; 21(23):5337-41. Epub 2017/12/16. https://doi.org/10.26355/eurrev_201712_13918 PMID: 29243774.

EWSAT1 Acts in Concert with Exosomes in Osteosarcoma Progression and Tumor-Induced Angiogenesis: The "Double Stacking Effect. S C Tao, J Y Huang, Z Y Wei, Z X Li, S C Guo, 10.1002/adbi.20200015232803878Adv Biosyst. 49Tao SC, Huang JY, Wei ZY, Li ZX, Guo SC. EWSAT1 Acts in Concert with Exosomes in Osteosarcoma Progression and Tumor-Induced Angiogenesis: The "Double Stacking Effect". Adv Biosyst. 2020; 4(9): e2000152. Epub 2020/08/18. https://doi.org/10.1002/adbi.202000152 PMID: 32803878.

Long non-coding RNA EWSAT1 promoted metastasis and actin cytoskeleton changes via miR-24-3p sponging in osteosarcoma. D Shen, Y Liu, Y Liu, T Wang, L Yuan, X Huang, 10.1111/jcmm.1612133225581J Cell Mol Med. 2527812296PubMed Central PMCIDShen D, Liu Y, Liu Y, Wang T, Yuan L, Huang X, et al. Long non-coding RNA EWSAT1 promoted metastasis and actin cytoskeleton changes via miR-24-3p sponging in osteosarcoma. J Cell Mol Med. 2021; 25(2):716-28. Epub 2020/11/24. https://doi.org/10.1111/jcmm.16121 PMID: 33225581; PubMed Central PMCID: PMC7812296.

LINC01101 and LINC00277 expression levels as novel factors in HPV-induced cervical neoplasia. I V Iancu, Anton G Botezatu, A Huica, I Nastase, A Socolov, D G , 10.1111/jcmm.1328828767188J Cell Mol Med. 2112Iancu IV, Anton G, Botezatu A, Huica I, Nastase A, Socolov DG, et al. LINC01101 and LINC00277 expression levels as novel factors in HPV-induced cervical neoplasia. J Cell Mol Med. 2017; 21 (12):3787-94. Epub 2017/08/03. https://doi.org/10.1111/jcmm.13288 PMID: 28767188;

. Pubmed Central, Pmcid , 5706512PubMed Cen- tral PMCID: PMC5706512.

Increased EWSAT1 expression promotes cell proliferation, invasion and epithelial-mesenchymal transition in colorectal cancer. R Zhang, J B Li, X F Yan, Jin K Li, W Y Xu, J , 10.26355/eurrev_201810_1614630402843Eur Rev Med Pharmacol Sci. 2220Zhang R, Li JB, Yan XF, Jin K, Li WY, Xu J, et al. Increased EWSAT1 expression promotes cell proliferation, invasion and epithelial-mesenchymal transition in colorectal cancer. Eur Rev Med Pharmacol Sci. 2018; 22 (20):6801-8. Epub 2018/11/08. https://doi.org/10.26355/eurrev_201810_16146 PMID: 30402843.

LncRNA EWSAT1 Promotes Colorectal Cancer Progression Through Sponging miR-326 to Modulate FBXL20 Expression. J Liu, S Huang, X Liao, Z Chen, L Li, L Yu, 10.2147/OTT.S27289533469313Onco Targets Ther. 14Liu J, Huang S, Liao X, Chen Z, Li L, Yu L, et al. LncRNA EWSAT1 Promotes Colorectal Cancer Pro- gression Through Sponging miR-326 to Modulate FBXL20 Expression. Onco Targets Ther. 2021; 14:367-78. Epub 2021/01/21. https://doi.org/10.2147/OTT.S272895 PMID: 33469313;

. Pubmed Central, Pmcid , 7812937PubMed Cen- tral PMCID: PMC7812937.

LncRNA EWSAT1 promotes ovarian cancer progression through targeting miR-330-5p expression. X Fu, L Zhang, Dan L Wang, K Xu, Y , 28979684PMC5622253Am J Transl Res. 99PubMed CentralFu X, Zhang L, Dan L, Wang K, Xu Y. LncRNA EWSAT1 promotes ovarian cancer progression through targeting miR-330-5p expression. Am J Transl Res. 2017; 9(9):4094-103. Epub 2017/10/06. PMID: 28979684; PubMed Central PMCID: PMC5622253.

Long non-coding RNA EWSAT1 contributes to the proliferation and invasion of glioma by sponging miR-152-3p. H Yang, W Chen, G Jiang, J Yang, W Wang, H Li, 10.3892/ol.2020.1171632724428Oncol Lett. 2027377177PubMed Central PMCIDYang H, Chen W, Jiang G, Yang J, Wang W, Li H. Long non-coding RNA EWSAT1 contributes to the proliferation and invasion of glioma by sponging miR-152-3p. Oncol Lett. 2020; 20(2):1846-54. Epub 2020/07/30. https://doi.org/10.3892/ol.2020.11716 PMID: 32724428; PubMed Central PMCID: PMC7377177.

PanCancer insights from The Cancer Genome Atlas: the pathologist's perspective. L A Cooper, E G Demicco, J H Saltz, R T Powell, A Rao, A J Lazar, 10.1002/path.502829288495J Pathol. 24456240356PubMed Central PMCIDCooper LA, Demicco EG, Saltz JH, Powell RT, Rao A, Lazar AJ. PanCancer insights from The Cancer Genome Atlas: the pathologist's perspective. J Pathol. 2018; 244(5):512-24. Epub 2017/12/31. https:// doi.org/10.1002/path.5028 PMID: 29288495; PubMed Central PMCID: PMC6240356.

Toil enables reproducible, open source, big biomedical data analyses. J Vivian, A A Rao, F A Nothaft, C Ketchum, J Armstrong, A Novak, 10.1038/nbt.377228398314Nat Biotechnol. 3545546205PubMed Central PMCIDVivian J, Rao AA, Nothaft FA, Ketchum C, Armstrong J, Novak A, et al. Toil enables reproducible, open source, big biomedical data analyses. Nat Biotechnol. 2017; 35(4):314-6. Epub 2017/04/12. https://doi. org/10.1038/nbt.3772 PMID: 28398314; PubMed Central PMCID: PMC5546205.

Long non-coding RNA EWSAT1 promotes human nasopharyngeal carcinoma cell growth in vitro by targeting miR-326/-330-5p. P Song, S C Yin, 10.18632/aging.10110327816050PMC5182074Aging (Albany NY). 811PubMed CentralSong P, Yin SC. Long non-coding RNA EWSAT1 promotes human nasopharyngeal carcinoma cell growth in vitro by targeting miR-326/-330-5p. Aging (Albany NY). 2016; 8(11):2948-60. Epub 2016/11/ 07. https://doi.org/10.18632/aging.101103 PMID: 27816050; PubMed Central PMCID: PMC5182074.

Transcriptional and Post-transcriptional Gene Regulation by Long Non-coding RNA. I M Dykes, C Emanueli, 10.1016/j.gpb.2016.12.00528529100Genomics Proteomics Bioinformatics. 1535487525PubMed Central PMCIDDykes IM, Emanueli C. Transcriptional and Post-transcriptional Gene Regulation by Long Non-coding RNA. Genomics Proteomics Bioinformatics. 2017; 15(3):177-86. Epub 2017/05/23. https://doi.org/10. 1016/j.gpb.2016.12.005 PMID: 28529100; PubMed Central PMCID: PMC5487525.

Revealing lncRNA Structures and Interactions by Sequencing-Based Approaches. X Qian, J Zhao, P Y Yeung, Q C Zhang, C K Kwok, 10.1016/j.tibs.2018.09.01230459069Trends Biochem Sci. 441Qian X, Zhao J, Yeung PY, Zhang QC, Kwok CK. Revealing lncRNA Structures and Interactions by Sequencing-Based Approaches. Trends Biochem Sci. 2019; 44(1):33-52. Epub 2018/11/22. https:// doi.org/10.1016/j.tibs.2018.09.012 PMID: 30459069.

Noncoding RNA:RNA Regulatory Networks in Cancer. J J Chan, Y Tay, 10.3390/ijms1905131029702599Int J Mol Sci. 1955983611PubMed Central PMCIDChan JJ, Tay Y. Noncoding RNA:RNA Regulatory Networks in Cancer. Int J Mol Sci. 2018; 19(5). Epub 2018/04/28. https://doi.org/10.3390/ijms19051310 PMID: 29702599; PubMed Central PMCID: PMC5983611.

Emerging role of exosome-derived long non-coding RNAs in tumor microenvironment. Z Sun, S Yang, Q Zhou, G Wang, J Song, Z Li, 10.1186/s12943-018-0831-z29678180Mol Cancer. 1715909226PubMed Central PMCIDSun Z, Yang S, Zhou Q, Wang G, Song J, Li Z, et al. Emerging role of exosome-derived long non-coding RNAs in tumor microenvironment. Mol Cancer. 2018; 17(1):82. Epub 2018/04/22. https://doi.org/10. 1186/s12943-018-0831-z PMID: 29678180; PubMed Central PMCID: PMC5909226.

H19 lncRNA: Roles in tumorigenesis. S Ghafouri-Fard, M Esmaeili, M Taheri, 10.1016/j.biopha.2019.10977431855739Biomed Pharmacother. 123Ghafouri-Fard S, Esmaeili M, Taheri M. H19 lncRNA: Roles in tumorigenesis. Biomed Pharmacother. 2020; 123:109774. Epub 2019/12/20. https://doi.org/10.1016/j.biopha.2019.109774 PMID: 31855739.

Role of lncRNA LUCAT1 in cancer. C Xing, S G Sun, Z Q Yue, F Bai, 10.1016/j.biopha.2020.11115833360049Biomed Pharmacother. 134Xing C, Sun SG, Yue ZQ, Bai F. Role of lncRNA LUCAT1 in cancer. Biomed Pharmacother. 2021; 134:111158. Epub 2020/12/29. https://doi.org/10.1016/j.biopha.2020.111158 PMID: 33360049.

Long non-coding SNHG1 in cancer. K Z Thin, J C Tu, S Raveendran, 10.1016/j.cca.2019.03.00230849309Clin Chim Acta. 494Thin KZ, Tu JC, Raveendran S. Long non-coding SNHG1 in cancer. Clin Chim Acta. 2019; 494:38-47. Epub 2019/03/09. https://doi.org/10.1016/j.cca.2019.03.002 PMID: 30849309.

Long Non-Coding RNA and Acute Leukemia. G M Cruz-Miranda, A Hidalgo-Miranda, D A Bárcenas-López, J C Núñez-Enríquez, J Ramírez-Bello, J M Mejía-Aranguré, 10.3390/ijms2003073530744139Int J Mol Sci. 2036387068PubMed Central PMCIDCruz-Miranda GM, Hidalgo-Miranda A, Bárcenas-López DA, Núñez-Enríquez JC, Ramírez-Bello J, Mejía- Aranguré JM, et al. Long Non-Coding RNA and Acute Leukemia. Int J Mol Sci. 2019; 20(3). Epub 2019/02/ 13. https://doi.org/10.3390/ijms20030735 PMID: 30744139; PubMed Central PMCID: PMC6387068.

Long non-coding RNAs and cervical cancer. H Aalijahan, S Ghorbian, 10.1016/j.yexmp.2018.11.01030471246Exp Mol Pathol. 106Aalijahan H, Ghorbian S. Long non-coding RNAs and cervical cancer. Exp Mol Pathol. 2019; 106:7-16. Epub 2018/11/25. https://doi.org/10.1016/j.yexmp.2018.11.010 PMID: 30471246.

Potential of long non-coding RNAs in cancer patients: From biomarkers to therapeutic targets. Chandra Gupta, S , Nandan Tripathi, Y , 10.1002/ijc.3054627925173Int J Cancer. 1409Chandra Gupta S, Nandan Tripathi Y. Potential of long non-coding RNAs in cancer patients: From bio- markers to therapeutic targets. Int J Cancer. 2017; 140(9):1955-67. Epub 2016/12/08. https://doi.org/ 10.1002/ijc.30546 PMID: 27925173.

MEG3: an Oncogenic Long Non-coding RNA in Different Cancers. A Al-Rugeebah, M Alanazi, N R Parine, 10.1007/s12253-019-00614-32019/02/23Pathol Oncol Res. 25330793226Al-Rugeebah A, Alanazi M, Parine NR. MEG3: an Oncogenic Long Non-coding RNA in Different Can- cers. Pathol Oncol Res. 2019; 25(3):859-74. Epub 2019/02/23. https://doi.org/10.1007/s12253-019- 00614-3 PMID: 30793226.

Role of non-coding RNA networks in leukemia progression, metastasis and drug resistance. A A Bhat, S N Younes, S S Raza, L Zarif, S Nisar, I Ahmed, 10.1186/s12943-020-01175-957. Epub 2020/03/14Mol Cancer. 1917069174PubMed Central PMCIDBhat AA, Younes SN, Raza SS, Zarif L, Nisar S, Ahmed I, et al. Role of non-coding RNA networks in leu- kemia progression, metastasis and drug resistance. Mol Cancer. 2020; 19(1):57. Epub 2020/03/14. https://doi.org/10.1186/s12943-020-01175-9 PMID: 32164715; PubMed Central PMCID: PMC7069174.

LncRNA DDX11-AS1: a novel oncogene in human cancer. Y Feng, M Wu, S Hu, X Peng, F Chen, 10.1007/s13577-020-00409-832772230Hum Cell. 334Feng Y, Wu M, Hu S, Peng X, Chen F. LncRNA DDX11-AS1: a novel oncogene in human cancer. Hum Cell. 2020; 33(4):946-53. Epub 2020/08/11. https://doi.org/10.1007/s13577-020-00409-8 PMID: 32772230.

Emerging impact of the long noncoding RNA MIR22HG on proliferation and apoptosis in multiple human cancers. L Zhang, C Li, X Su, 10.1186/s13046-020-01784-833267888J Exp Clin Cancer Res. 3917712612PubMed Central PMCIDZhang L, Li C, Su X. Emerging impact of the long noncoding RNA MIR22HG on proliferation and apo- ptosis in multiple human cancers. J Exp Clin Cancer Res. 2020; 39(1):271. Epub 2020/12/04. https:// doi.org/10.1186/s13046-020-01784-8 PMID: 33267888; PubMed Central PMCID: PMC7712612.

Update on targeted cancer therapies, single or in combination, and their fine tuning for precision medicine. S S Bashraheel, A Domling, S K Goda, 10.1016/j.biopha.2020.11000932106381Biomed Pharmacother. 125Bashraheel SS, Domling A, Goda SK. Update on targeted cancer therapies, single or in combination, and their fine tuning for precision medicine. Biomed Pharmacother. 2020; 125:110009. Epub 2020/02/ 29. https://doi.org/10.1016/j.biopha.2020.110009 PMID: 32106381.

Review of precision cancer medicine: Evolution of the treatment paradigm. A M Tsimberidou, E Fountzilas, M Nikanjam, R Kurzrock, 10.1016/j.ctrv.2020.10201932251926Cancer Treat Rev. 867272286PubMed Central PMCIDTsimberidou AM, Fountzilas E, Nikanjam M, Kurzrock R. Review of precision cancer medicine: Evolu- tion of the treatment paradigm. Cancer Treat Rev. 2020; 86:102019. Epub 2020/04/07. https://doi.org/ 10.1016/j.ctrv.2020.102019 PMID: 32251926; PubMed Central PMCID: PMC7272286.

Best Practices for Using Median Splits, Artificial Categorization, and their Continuous Alternatives. J Decoster, M Gallucci, A-Mr Iselin, 10.5127/jep.008310Journal of Experimental Psychopathology. 22DeCoster J, Gallucci M, Iselin A-MR. Best Practices for Using Median Splits, Artificial Categorization, and their Continuous Alternatives. Journal of Experimental Psychopathology. 2011; 2(2):197-209. https://doi.org/10.5127/jep.008310

Determining the cutoff based on a continuous variable to define two populations with application to vaccines. S Li, M Parnes, I S Chan, 10.1080/10543406.2012.75650223611202J Biopharm Stat. 233Li S, Parnes M, Chan IS. Determining the cutoff based on a continuous variable to define two popula- tions with application to vaccines. J Biopharm Stat. 2013; 23(3):662-80. Epub 2013/04/25. https://doi. org/10.1080/10543406.2012.756502 PMID: 23611202.