# A preliminary survey reveals that common viruses are found at low titers in a wild population of honey bees (Apis mellifera)

CorpusID: 266226315
 
tags: #Medicine, #Environmental_Science, #Biology

URL: [https://www.semanticscholar.org/paper/a7ed93a1a6e86a4c429814efb95de0d4790c2f70](https://www.semanticscholar.org/paper/a7ed93a1a6e86a4c429814efb95de0d4790c2f70)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

A preliminary survey reveals that common viruses are found at low titers in a wild population of honey bees (Apis mellifera)


Myra Dickey 
Department of Entomology
Texas A&M University
College StationTXUSA

Mckaela Whilden 
Department of Entomology
Texas A&M University
College StationTXUSA

Jordan Twombly Ellis 
Department of Entomology
Texas A&M University
College StationTXUSA

Juliana Rangel jrangel@tamu.edu 0000-0002-0586-9245
Department of Entomology
Texas A&M University
College StationTXUSA

A preliminary survey reveals that common viruses are found at low titers in a wild population of honey bees (Apis mellifera)
BD48BAD7DC08E8CF6C81076D426AFAA810.1093/jisesa/iead117Received on 22 May 2023; revised on 22 August 2023; accepted on 29 November 2023black queen cell virus (BQCV)deformed wing virus (DWV)feral honey beeshoney bee-associated virusesunmanaged colonies
A major threat to honey bee (Apis mellifera Linnaeus, Hymenoptera: Apidae) health continues to be parasitism by the mite Varroa destructor, which has been linked to high colony losses worldwide.Besides feeding on developing and adult bees, Varroa is also a prolific vector of honey bee-associated viruses.Because they live in unmanaged conditions, wild honey bee colonies are not treated against Varroa, which has enabled the natural selection of more mite-tolerant bees.To date, few studies have explored the prevalence of viruses in unmanaged colonies.The Welder Wildlife Refuge (WWR) in Texas is a unique site to study the viral landscape of unmanaged honey bees in the United States.The goals of this study were to identify and quantify viruses in wild colonies at the WWR, to examine changes in the prevalence of viruses in these colonies over time, and to compare the presence and titers of viruses between wild colonies at the WWR and those from the nearest managed apiary.We collected bees from colonies at the WWR in 2013, 2016, and 2021, and analyzed selected viruses for their presence and titers via quantitative polymerase chain reaction.In 2021, we also sampled bees from the nearest managed apiary for comparison.We found low average virus titers in all wild colonies sampled, and no difference in virus titers between colonies at the WWR and those from the managed apiary.Our study indicates that virus titers in wild colonies at the WWR are similar to those found in nearby colonies, and that these titers fluctuate over time.

## Introduction

The western honey bee, Apis mellifera Linnaeus (Hymenoptera: Apidae) is arguably the most widely used beneficial insect for the commercial pollination of major agricultural crops, contributing over $200 billion dollars to the global economy annually (Gallai et al. 2009, Calderone, 2012).But despite their importance, many populations of managed honey bees around the world continue to suffer high rates of overwintering mortality, mostly due to a combination of factors that include parasites, pathogens, pesticide exposure, malnutrition, and poor queen quality (Kulhanek et al. 2017, Bruckner et al. 2022; but see Osterman et al. 2021 on the rise of colony numbers in some locations).Of these factors, the most detrimental threat to honey bee health continues to be the ectoparasitic mite Varroa destructor, which has been linked to high colony losses worldwide (Dainat et al. 2012, Mondet et al. 2014, Natsopoulou et al. 2017, Ramsey et al. 2019).

Varroa mites are highly effective vectors of several honey beeassociated viruses, which they can transmit while feeding on the fat body tissue of developing and adult honey bees (Ramsey et al. 2019, Beaurepaire et al. 2020).The 2 most common virus complexes that have been linked to Varroa transmission are the acute bee paralysis virus (ABPV) complex, which includes Israeli acute paralysis virus (IAPV) and Kashmir bee virus (KBV), as well as the deformed wing virus (DWV) complex, which includes 3 major strains (DWV-A, DWV-B, and DWV-C) (de Miranda et al. 2010, Mordecai et al. 2016, McMahon et al. 2018, Posada-Florez et al. 2019).ABPV and DWV are single-stranded RNA viruses that infect honey bees at all developmental stages (de Miranda et al. 2010).Before the arrival of Varroa as a novel parasite of A. mellifera, these viruses were present at low titers in most colonies and were considered relatively harmless (Genersch and Aubert 2010, de Miranda et al. 2013, Amiri et al. 2018).However, after the arrival of Varroa, colony decline (and ultimately death) has usually been associated with both, high mite infestation levels, and high titers of mite-vectored viruses (Martin et al. 2012, van Dooremalen et al. 2012, Tantillo et al. 2015, Grozinger and Flenniken 2019).While the distinctive signs of DWV infection include wing deformities, which result in flightless adults that die shortly upon emerging (de Miranda and Genersh 2010), those symptoms are almost always seen only through Varroa-mediated transmission of DWV in the pupal stages of bee development (Moeckel et al. 2011).There have also been other DWV genotypes identified that have shown reduced symptoms and higher survival rates than the most common strains (Ray et al. 2023).Likewise, high ABPV titers have been linked in some studies with high Varroa mite titers in colonies, although it is still unclear if the mite directly transmits this virus complex.Whenever present, ABPV is portrayed by severe pupal mortality, as well as trembling and paralysis in those bees that make it into adulthood (de Miranda et al. 2010).

Not surprisingly, when a colony's high mite population is not controlled, there is usually a rampant increase in the titers of DWV and other viruses (Martin et al. 2012, van Dooremalen et al. 2012, Tantillo et al. 2015, Grozinger and Flenniken 2019).The titers of honey bee-associated viruses (henceforth referred to as "viruses") in managed colonies can be detectable in the spring, but typically peak in the late summer to early fall, which correlates with the time of year when Varroa infestation is also at its highest (Runckel et al. 2011, Francis et al. 2013, Faurot-Daniels et al. 2020).Despite this reliance of managed beekeeping operations on chemical treatments against Varroa to ensure colony survival, the use of miticides has been shown to increase virus quantities in certain settings (Locke et al. 2012).

Interestingly, there are wild populations of honey bees scattered throughout the world that have survived without human intervention, even after the introduction of this pervasive parasite (Australia: Chapman et al. 2008;Europe: Oleksa et al. 2013;Thompson et al. 2014;Sweden: Thaduri et al. 2018, 2019;Locke et al. 2021;United States: Schiff et al. 1994;Rangel et al. 2016a;b;Seeley, 2017;Seeley 2019;Rangel et al. 2020).Over the last 30 years, a large population of wild colonies located at the Welder Wildlife Refuge (WWR) near Sinton, TX, has been studied to better understand the biology of unmanaged honey bees in the Southern United States (Pinto et al. 2005, Rangel et al. 2016a, 2016b).A survey of this population from 1991 to 2001 showed that the mitochondrial haplotype frequency of these colonies changed drastically from being solely of Western European maternal lineage in the late 1990s to being dominated by Africanized maternal lineages in the late 2000s (Pinto et al. 2005).A follow-up study showed that the same population is now an admixed swarm with a continued greater presence of the Apis mellifera scutellata lineage and low titers of European genetic ancestry (Rangel et al. 2016a).

The success of Africanized honey bees (AHBs) in the Americas has been attributed to a combination of genetic and ecological factors that have conferred them higher fitness compared to some resident populations of European descent (Rubink et al. 1996, Pinto et al. 2004, Schneider et al. 2004).Some of these characteristics include a shorter development period, higher swarming rate, higher absconding rate, lower selectivity when choosing nest sites, lower colony density, smaller nests, and higher defensiveness against predators, pests, and parasites (Winston et al. 1983, Schmidt and Hurley 1995, Caron 2001, Schneider et al. 2004, Utaipanon et al. 2019).These traits, combined with the advantages of living in unmanaged conditions for decades, make AHBs an ideal system for studying the ecology of wild honey bees in the southern United States.Despite the previous findings that the WWR comprises a genetic predominance of AHBs, not much else is known about the presence of pathogens in this unmanaged honey bee population.

The aim of this study was to identify and quantify selected RNA viruses in unmanaged honey bees at the WWR, and to discern any interesting patterns in virus prevalence and titers in that population over time.In addition, we aimed to compare viral loads between wild colonies at the WWR and those from the nearest managed apiary.We hypothesized that, given the difference in management status, wild colonies at the WWR would have lower viral titers than managed colonies.


## Materials and Methods


## Study Site

This study was conducted at the WWR located in San Patricio County, TX (28°07ʹ 18″, −97°26ʹ34″).The WWR is positioned in a transitional zone between 3 different ecoregions: the South Texas plains, the Gulf prairies, and the marshes (Blankenship 2000).The vegetative cover is comprised of live oaks, mesquites, chaparral brushland, and open grassland (Blankenship 2000).The climate is generally humid, with hot summers and cool winters (Blankenship 2000, Baum 2003).Interestingly, Texas experienced a historical "polar vortex" in the winter of 2021 (before our last sampling event) that was accompanied by unusually intense cold, snow, and wind storms (Evett et al. 2021), arguably making it the worst cold spell in the region in over 100 years.


## Sample Collection from Tree Cavities

A thorough survey of wild honey bee colonies at the WWR conducted in 2013 and again in 2016 led to the discovery of 112 tree cavities in a 5.14 km 2 area (Fig. 1a and b).All trees with suitable nesting cavities (most of which were live oaks) were marked with a metal tag and their global positioning system (GPS) coordinates were logged.We conducted a similar survey in 2021 and located some of the previously found trees by matching their tags to their GPS coordinates.We also marked and logged any new cavities harboring active colonies within the study area.We conducted all sample collections during the month of August to limit seasonal variation of virus prevalence or titers between the colonies that had been previously sampled in 2013 and 2016, and those that were newly sampled in 2021.We collected as many workers as possible from all active colonies by knocking on the entrance with a stick and catching any bees flying out of the hive.On average, about 15 bees were collected from the entrance of the most accessible colonies.However, because some of the cavities were not very accessible and were several meters above ground, we had difficulty collecting individuals from them.All collected bees were immediately placed on dry ice and transported back to the laboratory to be stored at −80°C for subsequent DNA and RNA extraction.


## Sample Collection from Managed Colonies

We partnered with a beekeeper located 30 miles away from WWR who maintains approximately 100 colonies (Fig. 1c).The beekeeper's main commercial activities are queen rearing and honey production, and the colonies are rarely moved away from the site.The proprietor implements typical beekeeping procedures such as splitting, treating colonies against Varroa mites using approved chemical products, and requeening when colonies undergo queen supersedure or are failing.We collected approximately 20 workers from the entrance of 20 randomly chosen colonies, placed them on dry ice, and transported them back to the laboratory to be stored at −80°C for subsequent genetic analysis.In addition, there were 2 managed colonies maintained at the WWR that were sampled in 2021 for virus detection and quantification.Those colonies had been started a few years before by installing into a box a typical "package" of purchased bees that contained 2-3 Lb. of workers, along with a queen of European genetic ancestry.The owner of the colonies then took a "hands-off" approach to colony management, and by the time we sampled them in 2021, they had not been manipulated in over a year.


## DNA Extraction and Mitochondrial DNA Analysis

To determine the maternal genetic lineage of sampled colonies, total DNA was extracted from the thorax of one randomly selected worker per colony sampled at the WWR in 2013 and 2016 using a QIAamp DNA Mini Kit (Qiagen Inc., Valencia, CA), following the manufacturer's specifications (see Rangel et al. 2016a for details on how DNA was extracted from those colonies).In addition, in 2021 DNA was extracted from the legs and wings of two randomly selected workers per colony following the Chelex method (Evans et al. 2013).Total DNA was extracted from workers belonging to the wild colonies sampled at the WWR, those "managed" colonies that were maintained at the WWR, and those managed colonies at the nearest managed apiary.We used 2 different DNA extraction methods because the DNA extracted in 2013 and 2016 (but not the DNA extracted in 2021) was done previously for a separate study (Rangel et al. 2016a).All extracted DNA was stored at −20°C until further analysis.We amplified the intergenic region of the CO-I and CO-II gene in the mitochondrial genome via polymerase chain reaction (PCR) by using the primers E2, 5' GGACTAATATCTATACCACG 3', and H2, 5' CCTAAAGATGGAACTGTTC 3', as done previously (Garnery et al. 1992).

We performed single PCR reactions in 50-μL volumes using the OneTaq kit from New England Biolabs (Ipswich, MA).Reactions contained the following reagents: 1X Standard Buffer, 200 μM of dNTPs, 0.2 μM of both forward and reverse primers, 1.25 units of OneTaq DNA polymerase, and 5 μL of 10-fold diluted DNA.The PCR thermocycler conditions were 94°C for 30 s followed by 29 cycles of 94°C for 15 s, 48°C for 20 s, 68°C for 45 s, and a final extension period of 68°C for 10 min.Following PCR, the products were separated on a 1% agarose gel stained with ethidium bromide and visualized under UV light.Once the presence of PCR products was confirmed, they were cleaned using Mag-Bind Total Pure NGS (Omega Bio-Tek, Norcross, GA).Cleaned PCR products were then sent off for sequencing at Eurofins Genomics (Louisville, KY).The forward and reverse reads were aligned to each other and were then used for carrying out nucleotide BLAST algorithms in the National Center for Biotechnology Information (NCBI) database for A. mellifera mitochondrial haplotype (mitotype) identification.


## RNA Extraction and cDNA Synthesis

To determine the presence and prevalence of selected honey beeassociated RNA viruses, RNA was extracted from a pool of 10 bees per colony.However, this was done only for wild colonies sampled at the WWR in 2013, 2016, and 2021 for which we had collected at least 10 bees, as we were unable to sample enough workers from some of the colonies.RNA was likewise extracted from a pool of 10 bees per colony for 20 randomly selected colonies at the managed apiary.For each of those colonies, RNA was extracted and homogenized using a mortar and pestle following the Ribozol (VWR Life Sciences, Radnor, PA) protocol.The concentration and purity of the RNA were measured on a Nanodrop2000 Spectrophotometer (Thermo Scientific, Wilmington, DE).Next, 100 ng of RNA was used to generate cDNA.Genomic DNA was then removed, and cDNA was synthesized for each sample using the iScript clear gDNA synthesis kit (Bio-Rad Laboratories, Hercules, CA).The resulting cDNA was then diluted 10-fold and stored at −20°C until it was used as a template for quantitative PCR (qPCR) amplification to identify each virus and quantify its relative abundance.


## Quantification of Viruses

Each RNA sample was first screened for the presence of 8 common honey bee-associated viruses: ABPV, BQCV, Chronic bee paralysis virus (CBPV), DWV-A, IAPV, KBV, Lake Sinai virus (LSV), and Sacbrood virus (SBV), using the primers outlined in Table 1.We performed 25-μL reactions using a GoTaq kit from Promega (Madison, WI).Each reaction contained 1X GoTaq master mix, 0.2 μM of both forward and reverse primers, and 1 μL of diluted cDNA.Each PCR product was separated using gel electrophoresis on a 3.5% agarose gel stained with ethidium bromide and visualized under UV light.Each colony was then scored as testing positive or negative for each of the 8 viruses.

To quantify virus titers, 2-step qPCR was performed in 96-well plates on all samples that tested positive for any given virus using a Bio-Rad CFX96 machine (Bio-Rad Corp, Hercules, CA).To do this, 1 μL of diluted cDNA was added to 5 μL of SYBR Green master mix (Applied Biosystems, Waltham, MA).Then, 10 μM of forward and reverse virus-specific primers were added.Each sample was performed in triplicate reactions, as well as having simultaneous water and no-reverse transcriptase reactions as negative controls.Amplification was performed using 39 cycles of 94°C for 2 min, 94°C for 30 s, 60°C for 30 s, and 72°C for 30 s, followed by a melting curve from 55°C to 95°C.The final RNA copy number of each virus per sample was calculated by extrapolation to a standard curve made by serial dilutions (1:10) of a viral fragment used as a reference, which was provided to us by the USDA's Honey Bee Laboratory in Beltsville, MD.


## Statistical Analysis

For cases in which the same virus was detected in samples collected in all 3 years (2013, 2016, and 2021) from wild colonies sampled at the WWR, we conducted analysis of variance (ANOVA) tests to determine if there were differences in virus titers across years, followed by pair-wise Tukey honestly significant difference (HSD) tests.We also conducted Shapiro tests to determine if the quantification data for each identified virus were normally distributed.Because the data were not normally distributed, we performed Wilcoxon rank sum tests to determine if there were differences in virus titers between samples collected at the WWR and those collected from managed colonies in 2021.This comparison was only done for samples collected in 2021 because we did not collect samples from managed colonies in 2013 or 2016.Viral titers are reported as the triplicate average log 10 viral copy number per sample (± the standard error of the mean (SEM)), as done previously (Tantillo et al. 2015, Posada-Florez et al. 2019).Virus infection is generally considered to be at low intensity when it ranges from 0 to 10 3 copies, at medium intensity when it ranges from 10 3 to 10 7 copies, or at high intensity when it is above 10 7 copies per sample (Amiri et al. 2015).All descriptive statistics are reported as the mean ± SEM.We set the level of statistical significance for all tests at α = 0.05.


## Results


## Sample Collection

Of the 112 tree cavities surveyed in 2013 and 2016, 28 harbored active colonies in 2013 and 36 harbored active colonies in 2016.

Because some of the workers collected from those colonies were used in previous studies (Rangel et al. 2016a, b), the total number of colonies that we were able to use for virus detection was only 6 in 2013 and 24 in 2016.In 2021, only 16 of the cavities surveyed in previous years harbored active colonies, but we also found 4 additional cavities that had not been sampled before (for a final sample size of n = 20).We tagged those trees and recorded their GPS coordinates.

Those were the only tree cavities that we were able to locate and sample from during our weeklong survey of the refuge.


## Detection, Prevalence, and Quantification of Viruses

Of the 8 viruses analyzed, CBPV, IAPV, and KBV were not detected in any of the tested colonies during any of the years sampled.Of the 6, 24, and 20 wild colonies at the WWR that were tested for viruses in 2013, 2016, and 2021, respectively, 4 (66.7%), 12 (50.0%),and 5 (25.5%) tested positive for at least one virus, respectively (see Supplementary Table S1 for the full list of the viruses that were detected in each colony in a given year).In 2013 and 2021, DWV and BQCV were the most prevalent viruses detected.In 2016, BQCV was the most prevalent virus detected.There was no difference over time in the prevalence of any of the viruses at the WWR (χ 2 = 6, P-value = 0.199).Some colonies were infected with multiple viruses, which is not uncommon (Chen et al. 2004(Chen et al. , 2005)).For example, at least 2 viruses were found in 2 wild colonies sampled in 2013, 5 wild colonies sampled in 2016, and 3 wild colonies sampled in 2021 (see Supplementary Table S2 for the full list).In addition, 7 of the 20 managed colonies (35.0%) sampled from the nearest managed apiary tested positive for at least one virus (Supplementary Table S2).We found no difference in the prevalence of viruses between colonies from the nearby managed apiary and those at the WWR (χ 2 = 0, P = 1.00).

We also found that viral titers in the wild honey bee population varied from year to year, displaying different infection titers depending on the virus and the year.For example, the triplicate sample average (±SEM) titer of DWV in wild bees at the WWR was 8.59 × 10 4 ± 4.81 × 10 4 log 10 copy numbers in 2013 (n = 3 colonies that tested positive), 2.86 × 10 4 ± 3.83 × 10 4 log 10 copy numbers in 2016 (n = 4), and 1.33 × 10 4 ± 1.12 × 10 4 log 10 copy numbers in 2021 (n = 4), all of which are considered to be medium titers (see Figure 2a in Amiri et al. 2015).The average titers of DWV were significantly different across years (ANOVA, F 2,10 = 8.38, P < 0.001).Furthermore, DWV titers in 2013 were statistically higher than those in 2016 (Tukey HSD test, P < 0.001) and 2021 (Tukey HSD test, P < 0.001).There was no difference in average DWV titers between colonies sampled in 2016 and 2021 (Tukey HSD test, P = 0.691).

The triplicate sample average (±SEM) titer of BQCV infection was 1.21 × 10 4 ± 1.58 × 10 4 log 10 copy numbers in 2013 (n = 3 colonies), 1.93 × 10 3 ± 2.87 × 10 3 log 10 copy numbers in 2016 (n = 13), and 5.02 × 10 4 ± 6.88 × 10 4 log 10 copy numbers in 2021 (n = 4), all of which are considered of low to medium titers (Amiri et al. 2015;Fig. 2b).The average titers of BQCV were significantly different across years (ANOVA, F 2,19 = 9.32, P < 0.001).BQCV titers of samples collected in 2013 were statistically higher than those collected in 2016 (Tukey HSD test, P < 0.001); they were also higher in samples from 2021 compared to those collected in 2016 (Tukey HSD test, P < 0.001).Furthermore, there was no difference in average BQCV titers between samples collected in 2013 and those from 2021 (Tukey HSD test, P = 0.819).SBV was only detected at low titers in WWR samples from 2016, with a triplicate sample average (±SEM) titer of 1.25 × 10 3 ± 1.18 × 10 3 copy numbers per sample (n = 2).LSV was detected from samples in 2013 with a triplicate sample average (±SEM) titer of 8.31 × 10 4 ± 5.76 × 10 3 copy numbers in 2013 (n = 1) and a triplicate sample (±SEM) titer of 5.41 × 10 4 ± 2.40 × 10 3 copy numbers per in 2021 (n = 1), both of which are considered medium intensity (see Figure 2a in Amiri et al. 2015).

We found no difference in DWV titers in managed colonies when including versus excluding the 2 managed colonies that were sampled at the WWR (Fig. 2; Wilcoxon rank sum test, W = 48, P = 0.698).Therefore, we pooled the DWV data from both managed locations in our analysis to increase the sample size slightly.DWV was found at relatively medium intensity in samples collected from the pooled samples from managed colonies, with a triplicate sample average (±SEM) titer of 4.84 × 10 3 ± 2.62 × 10 3 copy numbers (n = 4 colonies total).We found no differences in DWV titers between the wild and managed colonies sampled in 2021 (Wilcoxon rank sum test, W = 48, P-value = 0.175).Furthermore, BQCV was found at relatively medium intensity only in samples from the managed apiary at the WWR, with a triplicate sample average (±SEM) titer of 3.87 × 10 4 ± 4.24 × 10 4 copy numbers (n = 2), while no BQCV was detected in any of the colonies tested for viruses from the nearby managed apiary (n = 19 colonies for which there were enough bees to perform the test).There were no differences in BQCV titers between wild and managed colonies sampled at the WWR in 2021 (Wilcoxon rank sum test, W = 45, P = 0.426).We found low titers of ABPV in the nearby managed apiary samples, with a triplicate sample average (±SEM) titer of 2.52 × 10 1 ± 2.50 × 10 1 copy numbers (n = 3).ABPV was not detected in any of the samples collected at the WWR in 2021.Finally, LSV was found at medium intensity at the off-site managed apiary, with a triplicate sample average (±SEM) titer of 1.97 × 10 6 ± 6.56 × 10 5 copy numbers (n = 1).


## Mitochondrial Haplotype Analysis

Mitochondrial haplotypes for all colonies sampled at the WWR in 2013 were previously published in Rangel et al. (2016a).Of the 28 colonies sampled that year, 5 (83%) belonged to the A lineage (corresponding to the Apis mellifera scutellate subspecies) and one (27%) belonged to the C lineage.However, we were able to perform virus detection and quantification from only 6 of the 28 colonies sampled in 2013 (21.4%) because we did not have enough bees from each of the other colonies for analysis.Likewise, we only report haplotypes for 24 of the 36 colonies sampled in 2016 (66.7%), which correspond to those for which we had enough bees to conduct viral identification and quantification.Of the 24 colonies analyzed from 2016, 22 (92%) belonged to the A lineage, one colony (4%) belonged to the O lineage, and one colony (4%) was scored as there being no mitotype match for it in the NCBI database.For bees collected at the WWR in 2021, we only reported the haplotypes of 19 of the 20 colonies sampled because those were the colonies for which we had enough bees to perform viral identification and quantification.Of those 19 colonies analyzed for viruses, 17 (90%) belonged to the A lineage and 2 (10%) belonged to the O lineage (Supplementary Fig. S1).The haplotypes of wild colonies sampled from the WWR in 2016 and 2021 are listed in Table 2. Finally, of the 20 colonies from the nearby managed apiary that were analyzed, 17 (85%) belonged to the lineage and 3 (15%) belonged to the A lineage (Supplementary Fig. S1).Both "managed" colonies (100%) that are maintained on the WWR property belonged to the A lineage.The haplotypes for colonies sampled at the managed apiary, as well as those of the 2 managed colonies at the WWR, are listed in Table 3.


## Discussion

This study comprised 3 main goals.First, we wanted to do a preliminary survey of the prevalence and titers of selected RNA viruses in wild honey bee colonies located at the WWR.Second, we wanted to identify any possible virus infection patterns in colonies sampled from this population across multiple years.And third, we wanted to compare the presence and titers of viruses between wild colonies at the WWR and those from the nearest managed apiary.Our results showed that DWV and BQCV, 2 of the biggest viral threats to managed honey bees (Ullah et al. 2021), were indeed detected in wild colonies at the WWR.However, infection from both viruses is only considered to be of low to medium intensity (Amiri et al. 2015).Overall, virus titers of the 2 most prevalent viruses, DWV and BQCV, were significantly different in wild colonies across years (Fig. 2a and b, respectively).There were no differences in the prevalence of viruses over time at the WWR or between wild colonies and those from the nearby managed apiary.We saw a similar pattern of viral infection between DWV and BQCV, with titers for both viruses being highest in 2013 and then dropping in 2016.However, it should be noted that the number of colonies sampled at the WWR in 2013 that we could use for virus identification was low (n = 6), given the difficulty in collecting bees from remote hive entrances in many instances, and the fact that many of the collected bees had been used in a previous study (Rangel et al. 2016).Therefore, it would be interesting to know if having a larger sample size could have indicated even stronger statistical differences in DWV and BQCV titers over the years.Furthermore, while the titer of DWV infection in wild colonies sampled in 2016 was similar to those sampled in 2021, the titer of BQCV infection was significantly higher in 2021 than in 2016.This result is not surprising, given that viral prevalence and titers are known to fluctuate in managed colonies on a seasonal basis (Traynor et al. 2020), so this pattern could be also present in wild populations.Regardless of these shortcomings, our study provides some of the first information about how viral titers fluctuate in a wild population of honey bees over time.Furthermore, we found no significant differences in the titers of DWV or BQCV between wild colonies at the WWR and managed colonies from the nearest off-site managed apiary.Interestingly, even though some of the wild bees tested positive for DWV and BQCV that year, the continued presence of active colonies in some of the tree cavities that bees had been sampled from in 2013 and 2016 (e.g., cavity # 383) is an indication that some of these colonies have survived without human intervention year after year, despite them having low to medium intensity of some viruses.However, while other studies have shown that wild colonies are expected to survive at a higher rate than managed colonies (Loftus et al. 2016, Seeley 2017), we were unable to sample each tree cavity more than once per year due to the remote location of the WWR.Therefore, we cannot definitively say that an active colony inhabiting a given tree cavity in 2013 was present and sampled from the same cavity in 2016 or 2021.In the future, studies of the turnover rate of colonies in wild settings should assess each tree cavity multiple times per year and perform genetic analyses to determine if the same active colony is present in the same cavity over time.

Overall, we know little about how virus infections fluctuate over time in free-living colonies compared to managed colonies.Comparable to our current results, a recent study in Southern California found that feral and managed colonies had similar titers of viral pathogens (Geffre et al. 2021).In particular, the authors found that wild and managed colonies were infected at medium intensity with DWV and BQCV, while ABPV infection fluctuated between low and medium intensity.Conversely, 2 recent studies (1 in the United Kingdom and 1 in Pennsylvania) found that DWV titers were higher in feral colonies compared to managed colonies (Thompson et al. 2014, Hinshaw et al. 2021).While the titers of DWV that we found in wild colonies at the WWR were slightly higher than those in the managed population, the difference was not statistically significant.Notably, each of the 3 studies mentioned above (Thompson et al. 2014, Geffre et al. 2021, Hinshaw et al. 2021) defined "feral" colonies as those that would have survived in unmanaged conditions for at least 1 full year.This means that some of the colonies that those studies considered to be feral could have come from escaped swarms that had recently escaped from nearby apiaries and thus, they would have only lived in unmanaged conditions for a year or less.On average, a well-established honey bee colony will swarm with its mother queen once a year in the spring, during which time there is usually a break in the brood-rearing cycle until the new queen begins laying eggs (Winston 1987).This interrupts the life cycle of the Varroa mite, therefore decreasing the number of mites in the colony, and potentially also reducing the transmission of viruses (Seeley 2017).This "natural" method to reduce pathogen loads is generally inhibited in managed beekeeping operations because apiary managers instead prefer to use pesticides for Varroa control.In addition, commercially managed hives have been shown to impact viral dynamics (often causing higher virus prevalence) than feral colonies (Owen 2017, Bartlett et al. 2021).Therefore, considering a colony to be "feral" only if they have survived unmanaged for a year is, in our opinion, likely not long enough for natural selection to effectively cull out colonies not well suited to withstand the pressures of living in the wild without any husbandry.

Here we also document the mitotypes of colonies sampled at the WWR in 2016 and 2021 (the mitotypes from colonies sampled in 2013 had previously been reported in Rangel et al. 2016).The vast majority of the colonies sampled (90-92%) belonged to the A (African) mitotype, with the A1e version being the most common (Table 2).Only 1 or 2 colonies (4-10%) were of the O (Middle Eastern) lineage, a mitotype that is not very common in commercial beekeeping operations (Magnus et al. 2011).A recent study of a feral population in Southern California also reported a few colonies (4 out of 15) belonging to the O lineage (Zárate et al. 2022).Combined, these interesting findings suggest that wild populations of honey bees in the southern United States harbor an untapped resource of genetic diversity that should be further explored in the future.It is also interesting to note that, of the 4 colonies from the nearby managed apiary that belonged to the A linage in our study, none tested positive for any viruses.

While the 2 onsite "managed" colonies at the WWR were also of the A lineage and tested positive for DWV and BQCV, those colonies have not been actively taken care of by beekeepers in years, so the bees that live in those boxes could be swarms that moved into those boxes without anybody noticing.Incidentally, our anecdotal data show that, over time, colonies that are found in the same tree cavity in consecutive years do not always have the same mitotype as those sampled in prior years.This pattern was previously documented in the 1980s and early 2000s when the WWR population was primarily of European maternal ancestry and then, after the introduction of the Varroa mite, colonies of the European linage were displaced in favor of those of primarily African maternal ancestry (Pinto et al. 2005).Since the population of unmanaged colonies at the WWR has become primarily of Africanized descent, it seems that the A lineage is the best-suited mitotype for the ecological conditions in this part of the southern United States.Zárate et al. (2022) similarly found that most of the feral colonies (60%) they sampled in Southern California were of the A maternal lineage.

Colonies belonging to the A maternal lineage continue to dominate the landscape at the WWR despite extreme environmental phenomena such as the historically devastating regional drought in 2011 (Combs 2012), and the cold temperatures during the "polar vortex" Table 3. List of managed honey bee colonies that were screened in 2021 for select honey bee-associated viruses, along with their corresponding mitochondrial haplotype (mitotype) identification.Colonies identifications (IDs) starting with D denote those that were sampled at the nearest managed apiary from the WWR, while colony IDs that start with T denote the two colonies that were managed on-site at the WWR
Colony ID Mitotype D1 A1e D2 A1e D3 A1e D5 C2 D6 C2 D7 C2 D8 C2 D9 C2 D11 C2 D12 C2 D13 C2 D14 C2 D15 C2 D16 C2 D17 C2 D18 C2 D19 C2 D20 C2 D21 C2 D22 C2 T1 A1e T2 A1e
in the winter of 2021.Indeed, previous work has shown that locally adapted honey bees are better suited to environmental fluctuations than non-local colonies (Costa et al. 2012, Hatjina et al. 2014).The wild colonies at the WWR are a unique beacon of genetic diversity because they have truly lived in unmanaged conditions for more than 30 years and are located over 30 miles away from the closest managed beekeeping operation (Pinto et al. 2003, 2004, 2005, Rangel et al. 2016a, 2016b).This makes it unlikely that any of the WWR colonies are swarms that recently escaped from nearby apiaries (except for the 2 on-site "managed" colonies, which are not actively taken care of by beekeepers).Furthermore, wild colonies at the WWR have never been treated against Varroa or any other parasites or pathogens.This lack of management has likely allowed for natural selection to act upon this population, which could explain why some colonies were infected with viruses at medium intensity and still survived to this day.This is a main response of pathogen-host interactions which has allowed these honey bee colonies to survive under stress.

Managed colonies are generally kept in controlled conditions (a vastly different lifestyle than that of wild colonies), which can mask or reverse the evolutionary pressures they are faced with due to human intervention.For example, most managed colonies are treated on an annual basis against high Varroa infestations, mostly through the use of pesticides (Jack and Ellis 2021).These constant treatment regimens disrupt natural selection for mite-and/or virus-tolerant colonies (Neumann and Blacquière, 2017).Managed colonies with high Varroa numbers typically also exhibit high titers of viruses (Carreck et al. 2010, Toufailia et al. 2014, Emsen et al. 2015).Managed colonies are also typically located at high densities in small areas, which can increase transmission of Varroa and potentially other pathogens (Seeley and Smith 2015).In contrast, our results suggest that wild colonies at the WWR might have low Varroa infestations, given that they exhibit low to medium viral titers.In addition, the frequent swarming and small colony size of wild Africanized colonies could play a role in keeping mite numbers and viral titers low (Winston et al. 1983).Furthermore, Hinshaw et al. (2021) found that feral colonies have higher expression of immune genes than their managed counterparts.Potentially, wild colonies at the WWR also have higher expression of immune genes, which could make them more tolerant to pests and pathogens.This concept should be tested further in future studies.

While our study provides some of the first information about the titers of viral pathogens in a truly wild honey bee population in the southern United States, there are some caveats in our experimental design that need to be considered.First, we only collected foragers from the hives' entrances because we were not allowed to destructively sample bees from inside the hives.Also, given that some of these tree cavities were either located several meters above ground, were very aggressive when disturbed, or were in a difficult location to collect samples, we were unable to collect many bees per colony.Therefore, for virus quantification, we decided to pool 10 bees per colony to increase the number of total colonies screened, instead of pooling a collective 20-30 individuals, which is recommended by Pirk et al. (2013).However, the probability of detecting pathogens only increases by 4% when adding more individuals to a pool of 10 (Pirk et al. 2013).Nonetheless, if a virus was present at low titers in one of our sampled colonies, we could have gotten a falsely negative result in our analysis.If this was the case, potentially more of the wild colonies we sampled could have been infected with viruses at low titers, which our pooling of only 10 individuals per sample was unable to detect.Having said that, we chose not to pool 20 or more individuals per colony for virus quantification because this would have cut our sample size by half, going from 20 screened colonies to only 8.

Second, because we did not collect enough bees from a hive's entrance, we were unable to perform suitable Varroa mite counts, which require sampling at least 300 adult bees from the middle of the brood nest area, where the highest number of mites on adult bees can be found (Dietemann et al. 2013).Also, since only foragers were collected, and we were unable to sample from the brood nest area, we only obtained one snapshot of a colony's adult population.However, a recent study compared the viral titers of individual foragers to those of the entire colony and found higher titers of BQCV and CBPV in foragers compared to the rest of the colony (Penn et al. 2022).

Finally, we were unable to confirm if the same colony was present in a given tree cavity for multiple years, as we could not go inside the hive to monitor the turnover rate of new queens.Even though some colonies in the same tree cavity did maintain the same mitotype from one year to the next, we cannot definitively say that these were the same colonies surviving over multiple years.For instance, we were unable to sample inside the colony to determine if the same queen was present across multiple years, and were also unable to determine if a colony swarmed and re-occupied an available cavity, which would allow a sampled colony to have the same mitotype as that in a previous year, but it would not necessarily be from the same queen.We did return to the WWR in the spring of 2022 to sample bees for another study and found that only 2 of the colonies sampled in 2021 were either too weak to sample, or had died.So, overall, we only saw a 10% "loss" of colonies in 2022 compared to the number of active colonies that we found in 2021.While we are unsure if those colonies were truly lost or if they relocated someplace else, this low rate of loss within a 6-month period suggests resilience in some of these wild colonies.Future work should investigate colony survivorship and tree cavity turnover at the WWR year-round and over time.

Despite the aforementioned limitations, it is evident that wild colonies at the WWR only suffered from low to medium infection titers of the tested RNA viruses, potentially indicating that these colonies have higher tolerance to pathogen pressure compared to managed colonies.This is especially important when one considers that the average annual loss of managed colonies in the United States has been upwards of 30% year after year (Kulhanek et al. 2017, Bruckner et al. 2023), even though most colonies are heavily treated against parasites, pathogens, and pests.Given the importance of honey bees for human welfare, it is now even more important to study the population genetics and disease ecology of wild honey bee populations to understand how they are able to combat pathogens and thrive without human intervention.

## Fig. 1 .
1
Fig. 1.Images of a) the geographic layout of the Welder Wildlife Refuge (WWR) with the different types of vegetative cover (colored areas) and the tree cavities (black dots) that have been found harboring wild honey bee colonies (taken from Baum et al. 2005); b) a typical entrance of a tree cavity housing a wild honey bee colony at the WWR; and c) experimental colonies sampled at the nearest managed apiary from the WWR.


## Fig. 2 .
2
Fig. 2. Boxplots showing the log 10 of the viral copy number per bee (± the spread of all the data points, with each point shown by a black dot) for a) DWV and b) BQCV found in honey bee workers collected from colonies at the Welder Wildlife Refuge (WWR) in 2013 (salmon), 2016 (green), and 2021 (teel), as well as those collected from managed colonies sampled at the nearest apiary from the WWR in 2021 (purple).The number of colonies that tested positive for each virus in each year (n) is given below each box.There were 3 technical replicates analyzed per year per sample.Horizontal lines in each box represent the median.The "***" symbols above brackets denote a P-value < 0.001 when comparing virus titers between 2 specific years.


## Table 1 .
1
List of forward and reverse primers (and their respective source of reference) used for the detection and quantification via quantitative polymerase chain reactions for honey bee-associated viruses
VirusForward primerReverse primerReferenceABPVACCGACAAAGGGTATGATGCCTTGAGTTTGCGGTGTTCCTBoncristiani et al. 2012BQCVTTTAGAGCGAATTCGGAAACAGGCGTACCGATAAAGATGGABoncristiani et al. 2012CBPVCAAAATCAACGAGCCAATCAAGTGTGAGGATCACCGGAACBoncristiani et al. 2012DWVGAGATTGAAGCGCATGAACATGAATTCAGTGTCGCCCATABoncristiani et al. 2012IAPVGCGGAGAATATAAGGCTCAGCTTGCAAGATAAGAAAGGGGGBoncristiani et al. 2012KBVTGAACGTCGACCTATTGAAAAATCGATTTTCCATCAAATGAGCBoncristiani et al. 2012LSVGTCATCCCAAGAGAACCACTYACCRCACYGACATGAAGAAATGAGGTCStamets et al. 2018SBVGGGTCGAGTGGTACTGGAAAACACAACACTCGTGGGTGACBoncristiani et al. 2012

## Table 2 .
2
List of wild honey bee colonies sampled at the Welder Wildlife Refuge in 2016 and 2021, along with their corresponding mitochondrial haplotype (mitotype) identification
Colony IDMitotype in 2016Mitotype in 202160A26a29a76.2A4p78A183A1e95A4p301A1349A1e356A1e357A26aA4p359A1eA1e366A26370A1e379A1e381A4p383A26aO5384A1eA4p385A16'393A1e394A4p396A1e403O404A16'29a406A16'407A1e418A1e425A1eA4p427A1eA1e428A1e435A1eA1e436No match501A1e517A1e703O5704A26721A1e
AcknowledgmentsWe would like the thank all the students who came with us to sample at the WWR, including Matthew Bradley, Mckaela Whilden, Jackson Wingert, and Caleb Rodriguez.We would also like to thank the director of the WWR, Terry Blankenship who graciously allowed us to sample his managed colonies and provided housing for us.Data availabilityThe original contributions presented in this study are included in the article and/or the Supplementary material.Further inquiries can be directed to the corresponding author.The datasets generated and analyzed for this study can be found at the Texas Data Repository by visiting https://doi.org/10.18738/T8/IAMHI4.FundingThis work was supported in part by a research scholarship from the American Beekeeping Federation and the John A. Jackman scholarship from the Department of Entomology at Texas A&M University obtained by M. D., and an Undergraduate Research Opportunity in Entomology scholarship from the Department of Entomology at Texas A&M University obtained by M. W. Partial funding was also obtained by J. R. from Texas A&M University's Hatch Project (TEX09557).Housing was generously donated by the Welder Wildlife Refuge Foundation.Author contributionsMyra Dickey (Conceptualization[Equal], Data curation [Lead], Formal analysis [Lead], Funding acquisition [Supporting], Investigation[Equal], Methodology[Equal], Project administration [Supporting], Resources [Supporting], Software [Lead], Supervision [Supporting], Validation[Equal],[Equal],Supplementary materialSupplementary material is available at Journal of Insect Science online.
Jordan Twombly Ellis (Conceptualization [Supporting], Data curation [Supporting], Formal analysis [Supporting], Investigation [Supporting], Methodology [Supporting], Software [Supporting], Writing-original draft. ]) Equal, Mckaela Whilden, Writing-original draft Writing-review. Data curation. Investigation [Equal], Methodology [Equal. Project administration [Lead. Writing-original draft [Equal], Writing-review & editing [Equal

Quantitative patterns of vertical transmission of deformed wing virus in honey bees. References Amiri, E Kryger, P Meixner, M D Strand, M K Tarpy, D R Rueppell, O , 10.1371/journal.pone.0195283PLoS One. 133e01952832018

Four categories of viral infection describe the health status of honey bee colonies. E Amiri, M Meixner, S L Nielsen, P Kryger, 10.1371/journal.pone.0140272PLoS One. 10e01402722015

Persistent effects of management history on honeybee colony virus abundances. L J Bartlett, M Boots, B J Brosi, J C De Roode, K S Delaplane, C A Hernandez, L Wilfert, 10.1016/j.jip.2020.107520J Invertebr Pathol. 1791075202021

Spatial and temporal distribution and nest site characteristics of feral honey bee (Hymenoptera: Apidae) colonies in a coastal prairie landscape. K A Baum, W L Rubink, M A Pinto, R N Coulson, 10.1603/0046-225X-34.3.610Environ Entomol. 3432005

Feral africanized honey bee ecology in a coastal prairie landscape. K A Baum, 2003College Station, TXTexas A&M UniversityPh.D. Dissertation

Diversity and global distribution of viruses of the western honey bee. A Beaurepaire, N Piot, V Doublet, K Antunez, E Campbell, P Chantawannakul, N Chejanovsky, A Gajda, M Heerman, D Panziera, 10.3390/insects11040239Apis mellifera. Insects. 20204239

Ecological response of bobcats to fluctuating prey populations on the Welder Wildlife Foundation Refuge. T L Blankenship, 2000College Station, TXTexas A&M UniversityMaster's thesis

Direct effect of acaricides on pathogen loads and gene expression levels in honey bees Apis mellifera. H Boncristiani, R Underwood, R Schwarz, J D Evans, J Pettis, J Insect Phys. 20125

A national survey of managed honey bee colony losses in the USA: results from the Bee Informed Partnership for. S Bruckner, M Wilson, D Aurell, K Rennich, D Steinhauer, N Williams, G R , 10.1080/00218839.2022.2158586J Api Res. 1832017. 2018-19, and 2019-20

A national survey of managed honey bee colony losses in the USA: results from the Bee Informed Partnership for. S Bruckner, M Wilson, D Aurell, K Rennich, D Steinhauer, N Williams, G R , 10.1080/00218839.2022.2158586J. Apicultural Res. 1832017. 2018-19, and 2019-20

Insect pollinated crops, insect pollinators and US agriculture: trend analysis of aggregate data for the period 1992-2009. N W Calderone, 10.1371/journal.pone.0037235PLoS One. 20125e37235

Africanized honey bees in the Americas. D M Caron, 2001AI Root CoMedina, OH, USA

Honey bee colony collapse and changes in viral prevalence associated with Varroa destructor. N L Carreck, B V Ball, S J Martin, 10.3896/ibra.1.49.1.13J Api Res. 12010

Population genetics of commercial and feral honey bees in Western Australia. N C Chapman, J Lim, B P Oldroyd, 10.1603/0022-0493(2008)101[272:pgocaf]2.0.co;2J Econ Entomol. 10122008

Detection of multiple viruses in queens of the honey bee Apis mellifera L. Y Chen, J S Pettis, M F Feldlaufer, 10.1016/j.jip.2005.08.005J Invertebr Pathol. 9022005

Multiple virus infections in the honey bee and genome divergence of honey bee viruses. Y Chen, Y Zhao, J Hammond, H T Hsu, J Evans, M Feldlaufer, 10.1016/j.jip.2004.07.005J Invertebr Pathol. 872-32004

The impact of the 2011 drought and beyond. S Combs, 2012Texas Comptroller of Public Account's OfficeAustin, TX, USA

Differences in colony phenotypes across different origins and locations: evidence for genotype by environment interactions in the Italian honeybee (Apis mellifera ligustica)?. C Costa, M Lodesani, K Bienefeld, 10.1007/s13592-012-0138-9Apidologie. 62012

Dead or alive: deformed wing virus and Varroa destructor reduce the life span of winter honeybees. B Dainat, J D Evans, Y P Chen, L Gauthier, P Neumann, 10.1128/AEM.06537-11Appl Environ Microbiol. 7842012

The acute bee paralysis virus-Kashmir bee virus-Israeli acute paralysis virus complex. J R De Miranda, G Cordoni, G Budge, 10.1016/j.jip.2009.06.014J Invertebr Pathol. 12010Suppl

Deformed wing virus. J R De Miranda, E Genersch, 10.1016/j.jip.2009.06.012J Invertebr Pathol. 12010Suppl

Standard methods for varroa research. V Dietemann, F Nazzi, S J Martin, D L Anderson, B Locke, K S Delaplane, Q Wauquiez, C Tannahill, E Frey, B Ziegelmann, 10.3896/ibra.1.52.1.09J Api Res. 5212013

Lower virus infections in Varroa destructor-infested and uninfested brood and adult honey bees (Apis mellifera) of a low mite population growth colony compared to a high mite population growth colony. B Emsen, M M Hamiduzzaman, P H Goodwin, E Guzman-Novoa, 10.1371/journal.pone.0118885PLoS One. 102e01188852015

Standard methods for molecular research in Apis mellifera. J D Evans, R S Schwarz, Y P Chen, G Budge, R S Cornman, P De La Rua, J R De Miranda, S Foret, L Foster, L Gauthier, 10.3896/IBRA.1.52.4.11J Apicultural Res. 5242013

Solar node and gateway wireless system functions in record breaking polar vortex outbreak of. S R Evett, A I Thompson, H H Schomberg, M A Andrade, Anderson J , 10.1002/agg2.20193Agrosyst Geosci Environ. 2021:4:e20193. February 2021

Longitudinal monitoring of honey bee colonies reveals dynamic nature of virus abundance and indicates a negative impact of Lake Sinai virus 2 on colony health. C Faurot-Daniels, W Glenny, K F Daughenbaugh, A J Mcmenamin, L A Burkle, M L Flenniken, 10.1371/journal.pone.0237544PLoS One. 9e02375442020

Kryger Varroa-virus interaction in collapsing honey bee colonies. R M Francis, S L Nielsen, PLoS One. 83e575402013

Economic valuation of the vulnerability of world agriculture confronted with pollinator decline. N Gallai, J M Salles, J Settele, B E Vaissière, 10.1016/j.ecolecon.2008.06.014Ecolog Econ. 32009

Evolutionary history of the honey bee Apis mellifera inferred from mitochondrial DNA analysis. L Garnery, J-M Cornuet, M Solignac, 10.1111/j.1365-294X.1992.tb00170.xMol Ecol. 19921

Preliminary analysis shows that feral and managed honey bees in southern California have similar levels of viral pathogens. A Geffre, D Travis, J Kohn, J Nieh, 10.1080/00218839.2021.2001209J Api Res. 20213

Emerging and re-emerging viruses of the honey bee (Apis mellifera L). E Genersch, Aubert M , 10.1051/vetres/2010027Vet Res. 6542010

Bee viruses: ecology, pathogenicity, and impacts. C M Grozinger, M L Flenniken, 10.1146/annurev-ento-011118-111942Annu Rev Entomol. 642019

Population dynamics of European honey bee genotypes under different environmental conditions. F Hatjina, C Costa, R Büchler, A Uzunov, M Drazic, J Filipi, L Charistos, L Ruottinen, S Andonov, M D Meixner, 10.3896/ibra.1.53.2.05J Api Res. 5322014

The role of pathogen dynamics and immune gene expression in the survival of feral honey bees. C Hinshaw, K C Evans, C Rosa, M M López-Uribe, Front Ecol Evol. 85052021

Integrated pest management control of Varroa destructor (Acari: Varroidae), the most damaging pest of (Apis mellifera L (Hymenoptera: Apidae)) colonies. C J Jack, J D Ellis, 10.1093/jisesa/ieab058J Insect Sci. 202156

A national survey of managed honey bee 2015-2016 annual colony losses in the USA. K Kulhanek, N Steinhauer, K Rennich, D M Caron, R R Sagili, J S Pettis, J D Ellis, M E Wilson, J T Wilkes, D R Tarpy, 10.1080/00218839.2017.1344496J Api Res. 5642017

Acaricide treatment affects viral dynamics in Varroa destructor-infested honey bee colonies via both host physiology and mite control. B Locke, E Forsgren, I Fries, J R De Miranda, 10.1128/AEM.06094-11Appl Environ Microbiol. 7812012

Adapted tolerance to virus infections in four geographically distinct Varroa destructor-resistant honeybee populations. B Locke, S Thaduri, J G Stephan, M Low, T Blacquière, B Dahle, Le Conte, Y Neumann, P De Miranda, J R , Sci Rep. 20211

How honey bee colonies survive in the wild: testing the importance of small nests and frequent swarming. J C Loftus, M L Smith, T D Seeley, 10.1371/journal.pone.0150362PLoS One. 20163e0150362

Mitochondrial DNA diversity of honey bees, Apis mellifera L (Hymenoptera: Apidae) from queen breeders in the United States. R M Magnus, A D Tripodi, A L Szalanski, J Apic Sci. 20111

Global honey bee viral landscape altered by a parasitic mite. S J Martin, A C Highfield, L Brettell, E M Villalobos, G E Budge, M Powell, S Nikaido, D C Schroeder, 10.1126/science.1220941Science. 20126086

Emerging viruses in bees: from molecules to ecology. D P Mcmahon, L Wilfert, R J Paxton, M J Brown, Adv Virus Res. 1012018

Standard methods for virus research in Apis mellifera. J R De Miranda, L Bailey, B V Ball, P Blanchard, G E Budge, N Chejanovsky, Y-P Chen, L Gauthier, E Genersch, D C De Graaf, 10.3896/IBRA.1.52.4.2256. 201352

Horizontal transmission of deformed wing virus: pathological consequences in adult bees (Apis mellifera) depend on the transmission route. N Moeckel, S Gisder, E Genersch, J Gen Virol. 9222011

On the front line: quantitative virus dynamics in honeybee (Apis mellifera L.) colonies along a new expansion front of the parasite Varroa destructor. F Mondet, J R De Miranda, A Kretzschmar, Le Conte, Y Mercer, A R , 10.1371/journal.ppat.1004323PLoS Pathog. 10e10043232014

Diversity in a honey bee pathogen: first report of a third master variant of the Deformed Wing Virus quasispecies. G J Mordecai, L Wilfert, S J Martin, I M Jones, D C Schroeder, 10.1038/ismej.2015.178ISME J. 102016

The virulent, emerging genotype B of deformed wing virus is closely linked to overwinter honeybee worker loss. M E Natsopoulou, D P Mcmahon, V Doublet, E Frey, P Rosenkranz, R J Paxton, 2017

The Darwin cure for apiculture? Natural selection and managed honeybee health. P Neumann, T Blacquière, 10.1111/eva.12448Evol Appl. 1032017

Rural avenues as a refuge for feral honey bee population. A Oleksa, R Gawroński, A Tofilski, 10.1007/s10841-012-9528-6J Insect Conserv. 1732013

Global trends in the number and diversity of managed pollinator species. J Osterman, M A Aizen, J C Biesmeijer, J Bosch, B G Howlett, D W Inouye, C Jung, D J Martins, R Medel, A Pauw, 2021:322:107653Agric Ecosyst Environ. 

Role of human action in the spread of honey bee (Hymenoptera: Apidae) pathogens. R Owen, 10.1093/jee/tox075J Econ Entomol. 11032017

Colony-titer viral load influences collective foraging in honey bees. H J Penn, M D Simone-Finstrom, L I De Guzman, P G Tokarz, R Dickens, Front Insect Sci. 20222

Identification of Africanized honey bee (Hymenoptera: Apidae) mitochondrial DNA: validation of a rapid polymerase chain reactionbased assay. M A Pinto, J S Johnston, W L Rubink, R N Coulson, J C Patton, W S Sheppard, 10.1603/0013-8746(2003)096[0679:ioahbh]2.0.co;2Ann Entomol Soc Am. 9652003

Temporal pattern of Africanization in a feral honeybee population from Texas inferred from mitochondrial DNA. Evolution. M A Pinto, W L Rubink, R N Coulson, J C Patton, J S Johnston, 10.1111/j.0014-3820.2004.tb00438.x200458

Africanization in the United States: replacement of feral European honeybees (Apis mellifera L) by an African hybrid swarm. M A Pinto, W L Rubink, J C Patton, R N Coulson, J S Johnston, 10.1534/genetics.104.035030Genetics. 17042005

The COLOSS BEEBOOK, Volume I: standard methods for Apis mellifera research. Cww Pirk, J R De Miranda, M Kramer, T Murray, F Nazzi, D Shutler, Jjm Van Der Steen, C Van Dooremalen, 10.3896/IBRA.1.52.4.13J Apicultural Res. V DietemannJ D EllisP Neumann4522013Statistical guidelines for Apis mellifera research

Deformed wing virus type A, a major honey bee pathogen, is vectored by the mite Varroa destructor in a nonpropagative manner. F Posada-Florez, A K Childers, M C Heerman, N I Egekwu, S C Cook, Y Chen, J D Evans, E V Ryabov, 10.1038/s41598-019-47447-3Sci Rep. 2019112445

Varroa destructor feeds primarily on honey bee fat body tissue and not hemolymph. S D Ramsey, R Ochoa, G Bauchan, C Gulbronson, J D Mowery, A Cohen, D Lim, J Joklik, J M Cicero, J D Ellis, 10.1073/pnas.1818371116Proc Natl Acad Sci U S A. 11652019

Prevalence of Nosema species in a feral honey bee population: a 20-year survey. J Rangel, K Baum, W L Rubink, R N Coulson, J S Johnston, B E Traver, 10.1007/s13592-015-0401-yApidologie. 4742016a

Africanization of a feral honey bee (Apis mellifera) population in South Texas: does a decade make a difference?. J Rangel, M Giresi, M A Pinto, K A Baum, W L Rubink, R N Coulson, J S Johnston, 10.1002/ece3.1974Ecol Evol. 62016b

Genetic diversity of wild and managed honey bees (Apis mellifera) in Southwestern Pennsylvania, and prevalence of the microsporidian gut pathogens Nosema ceranae and N. apis. J Rangel, B Traver, M Stoner, A Hatter, B Trevelline, C Garza, T Shepherd, T D Seeley, J Wenzel, 10.1007/s13592-020-00762-5Apidologie. 20205

Signatures of adaptive decreased virulence of deformed wing virus in an isolated population of wild honey bees (Apis mellifera). A M Ray, E C Gordon, T D Seeley, J L Rasgon, C M Grozinger, Proc R Soc B: Biol Sci. 2902023

Subtropical Apis mellifera (Hymenoptera: Apidae) swarming dynamics and Africanization rates in northeastern Mexico and southern Texas. W L Rubink, P Luevano-Martinez, E A Sugden, W T Wilson, A M Collins, 10.1093/aesa/89.2.243Ann Entomol Soc Am. 8921996

Temporal analysis of the honey bee microbiome reveals four novel viruses and seasonal prevalence of known Nosema, and Crithidia. C Runckel, M L Flenniken, J C Engel, J G Ruby, D Ganem, R Andino, J L Derisi, 10.1371/journal.pone.0020656PLoS One. 6e206562011

Genetic diversity of feral honey bee (Hymenoptera: Apidae) populations in the southern United States. N M Schiff, W S Sheppard, G M Loper, H Shimanuki, 10.1093/aesa/87.6.842Ann Entomol Soc Am. 8761994

Selection of nest cavities by Africanized and European honey bees. J O Schmidt, R Hurley, 10.1051/apido:19950603Apidologie. 61995

The African honey bee: factors contributing to a successful biological invasion. S Schneider, G Degrandi-Hoffman, D R Smith, Annu Rev Entomol. 12004

Life-history traits of wild honey bee colonies living in forests around Ithaca. T D Seeley, 10.1007/s13592-017-0519-1Apidologie. 20176

The lives of bees: the untold story of the honey bee in the wild. T D Seeley, 2019Princeton University PressPrinceton, NJ, USA

Crowding honeybee colonies in apiaries can increase their vulnerability to the deadly ectoparasite Varroa destructor. T D Seeley, M L Smith, 10.1007/s13592-015-0361-2Apidologie. 20156

Extracts of polypore mushroom mycelia reduce viruses in honey bees. P E Stamets, N L Naeger, J D Evans, J O Han, B K Hopkins, D Lopez, H M Moershel, R Nally, D Sumerlin, A W Taylor, 10.1038/s41598-018-32194-8Sci Rep. 81139362018

Virus infections of honeybees Apis mellifera. G Tantillo, M Bottaro, Di Pinto, A Martella, V , Di Pinto, P Terio, V , Ital J Food Saf. 201535364

Temporal changes in the viromes of Swedish Varroa-resistant and Varroa-susceptible honeybee populations. S Thaduri, B Locke, F Granberg, J R De Miranda, 10.1371/journal.pone.0206938PLoS One. 13e02069382018

Disentangling host-parasitepathogen interactions in a varroa-resistant honeybee population reveals virus tolerance as an independent, naturally adapted survival mechanism. S Thaduri, J G Stephan, J R De Miranda, Locke B , 10.1038/s41598-019-42741-6Sci Rep. 201916221

Parasite pressures on feral honey bees (Apis mellifera sp). C E Thompson, J C Biesmeijer, T R Allnutt, S Pietravalle, G E Budge, 10.1371/journal.pone.0105164PLoS One. 9e1051642014

Towards integrated control of varroa: effect of variation in hygienic behaviour among honey bee colonies on mite population increase and deformed wing virus incidence. Hma Toufailia, E Amiri, L Scandian, P Kryger, F L Ratnieks, J Api Res. 52014

Varroa destructor: A complex parasite, crippling honey bees worldwide. K S Traynor, F Mondet, J R De Miranda, M Techer, V Kowallik, M A Oddie, P Chantawannakul, A Mcafee, 10.1016/j.pt.2020.04.004Trends Parasitol. 20207

Viral impacts on honey bee populations: a review. A Ullah, I T Gajger, A Majoros, S A Dar, S Khan, A H Shah, M N Khabir, R Hussain, H U Khan, M Hameed, Saudi J Biol Sci. 20211

Assessing the density of honey bee colonies at ecosystem scales. P Utaipanon, T M Schaerf, B P Oldroyd, 10.1111/een.12715Ecol Entomol. 4432019

Winter survival of individual honey bees and honey bee colonies depends on titer of Varroa destructor infestation. C Van Dooremalen, L Gerritsen, B Cornelissen, J J Van Der Steen, F Van Langevelde, T Blacquiere, 10.1371/journal.pone.0036285PLoS One. 74e362852012

The biology of the honey bee. M L Winston, 1987Harvard University PressLondon, England; London, England

Some differences between temperate European and tropical African and South American honeybees. M L Winston, O R Taylor, G W Otis, 10.1080/0005772x.1983.11097902198364Bee World

D Zárate, T G Lima, J D Poole, E Calfee, R S Burton, J R Kohn, 10.1002/ece3.8580Admixture in Africanized honey bees (Apis mellifera) from Panamá to. San Diego, California (USA2022e8580