# Pseudogene Transcripts in Head and Neck Cancer: Literature Review and In Silico Analysis

CorpusID: 237324218 - [https://www.semanticscholar.org/paper/4e19f449a62a70b0977a23d03a74730522fd81d2](https://www.semanticscholar.org/paper/4e19f449a62a70b0977a23d03a74730522fd81d2)

Fields: Medicine, Biology

## (s0) Introduction
(p0.0) Head and neck cancer (HNC) is the eighth most common cancer worldwide, with more than 835,000 new cases and 431,000 deaths due to the disease per year [1]. HNC comprises tumors in the oral cavity, pharynx, and larynx, and nearly 95% of them are squamous cell carcinoma histological type tumors [2]. The classical risk factors for developing HNC are smoking habits and alcohol consumption [3]. The human papillomavirus (HPV), in particular HPV16, is detected in approximately 25% of HNC cases, especially in tumors located at the oropharynx, and serves as a favorable prognostic factor for these patients [4]. Most HNC patients are diagnosed with measurable locally advanced disease, and only about half of these patients achieve complete or partial responses after treatment [5]. Besides surgery, cisplatin (CDDP), combined or not with radiotherapy (RT), has been used in HNC patients' treatment [6], but therapy resistance has been reported [7].
## (s1) Pseudogene Transcripts
(p1.0) The transcription of pseudogenes depends on their genomic location, and it could be processed mainly into antisense RNAs, endogenous small interfering (endo-si)RNAs, or micro (mi)RNA sponges [18], acting as negative or positive gene regulators [19]. Pseudogene antisense RNAs can bind directly into sense RNA from the parental gene and inhibit its translation and protein production [20]. Pseudogene-derived endo-siRNAs are produced by cleaved pseudogene sense or antisense transcripts of double-stranded RNA or by the inverted repeat region of pseudogenes transcribed into hairpin-shaped RNA and sliced by the ribonuclease Dicer [20]. They are then separated into single strands and incorporated into the RNA-induced silencing complex to bind and degrade target mRNAs, inhibiting protein production [20]. Pseudogene transcripts can also act as competing endogenous RNAs (ceRNA) [21]. They may share miRNA response elements with other genes, especially parental ones, and positively regulate their expression by competing for the same pool of miRNAs, acting as miRNAs sponges [20]. In this process, pseudogene transcripts inhibit miRNA-target gene binding, allowing gene expression and modulating biological processes and tumor progression [22,23]. In addition, pseudogene transcripts in sense orientation may compete for miRNAs, RNA-binding proteins, and translational machinery with the parental gene [18].
## (s3) HNC in General
(p3.0) In HNC tumor cells, loss of PTENP1 (phosphatase and tensin homolog pseudogene 1) pseudogene transcript modulated malignant behavior and worst survival of HNC patients, leading to tumor cell proliferation, colony formation, and migration, possibly by interacting with its parental gene PTEN (phosphatase and tensin homolog) [25].

(p3.1) In HNC tumor cells and cell lines (FaDu, Cal-27, SCC4, and SCC9), FKBP9P1 (FKBP prolyl isomerase 9 pseudogene 1) pseudogene transcript abundance was found to correlate with advanced tumor stage and poor prognosis of patients by enhancing tumor cell proliferation, migration, and invasion, possibly by interacting with the PI3K/AKT signaling pathway [26].
## (s4) Oral Cancer
(p4.0) In oral cancer cells, based on bioinformatics methods, FTH1P3 (ferritin heavy-chain 1 pseudogene 3), GTF2IRD2P1 (GTF2I repeat domain-containing 2 pseudogene 1), and PDIA3P (disulfide isomerase family A, member 3 pseudogene) pseudogene transcripts were associated with oral pathogenesis and metastasis by interacting with their targets, MMP1 (matrix metallopeptidase 1), MMP3 (matrix metallopeptidase 3), MMP9 (matrix metallopeptidase 9), PLAU (plasminogen activator urokinase), and IL8 (interleukin 8) genes, involved in tumor cell proliferation, migration, and metastasis [29].

(p4.1) FTH1P3 pseudogene transcript was also associated with oral cancer risk and prognosis by becoming a miRNA sponge for miR-224-5p and thereby modulating the expression of the FZD5 (frizzled class receptor 5) gene, facilitating cell proliferation and colony formation [30]. Moreover, the FTH1P3 transcript was associated with advanced tumor stage and poor prognosis in oral cancer [31]. FTH1P3 could promote proliferation, migration, and invasion of tumor cells, possibly by PI3K/Akt/GSK3β/Wnt/β-catenin signaling [31].

(p4.2) In addition, the PTENP1 transcript was found to act as a ceRNA, protecting the parental gene PTEN from miR-21 binding and therefore inhibiting tumor cell proliferation and colony formation [32].
## (s10) Co-Expression Networks and GO Enrichment Analysis
(p10.0) We built co-expression networks to better understand the relationship between pseudogenes and other genes in the genome. In a co-expression network, genes that have similar expression variation among samples are clustered in the same module, and they are generally thought to be involved in the same biological process [37]. Using RNAseq data from 213 HNC patients available in TGCA, we built a tumor co-expression network using the Weighted Correlation Network Analysis (WGCNA) R software package from gene expression values (log2-converted FPKM) [38]. It was not possible to obtain RNAseq data from six HNC patients. We kept only protein-coding genes from the human genome assembly reference GRCh38.p13 and the ten most deregulated pseudogenes of each HNC location (HNC in general, oral cavity, oropharynx, hypopharynx, and larynx). We then removed genes and pseudogenes not expressed in at least half of the samples and built the network with 19,406 protein-coding genes and 31 pseudogenes identified in the TCGA data analysis. Additional parameters used to build the network with WGCNA were a soft thresholding power of 8 and a minimum number of 30 genes allowed in a module [39]. The modules were then classified in colors.

(p10.1) To gain more insights about the possible functions of the pseudogenes described above, we performed a GO enrichment analysis for all gene modules containing at least one of most deregulated pseudogenes from HNC using the GOATOOLS Python library [40] (false discovery rate (FDR) adjusted p-values < 0.05) and subsequently summarized and visualized using the web tool REVIGO [41].
## (s13) TCGA Data Analysis
(p13.0) Clinicopathological aspects of the selected 219 patients are presented in Supplementary Table S1. We identified 370 pseudogene transcripts associated with HNC, where SPATA31D5P (SPATA31 subfamily D member 5 pseudogene), HERC2P3 (hect domain and RLD2 pseudogene 3), SPATA31C2 (SPATA31 subfamily C member 2), MAGEB6P1 (melanoma antigen family B6 pseudogene 1), SLC25A51P1 (solute carrier family 25 member 51 pseudogene 1), BAGE2 (B melanoma antigen family member 2), DNM1P47 (DNM1 pseudogene 47), SPATA31C1 (SPATA31 subfamily C member 1), ZNF733P (zinc finger protein 733 pseudogene), and OR2W5 (olfactory receptor, family 2, subfamily W, member 5) were found to be the most deregulated in HNC. The ten most deregulated pseudogene transcripts, chromosome location, gene family function, and involvement in cancer can be found in Table 2. The complete list of results can be found in Supplementary Dataset S1.

(p13.1) When we stratified patients' information by tumor location, we found a few differences in pseudogene transcripts' pattern that could be explained by different cancer behaviors [43], but some of the identified pseudogenes were present in more than one tumor location.

(p13.2) In oral cancer (n = 62), SPATA31D5P, NBPF25P (neuroblastoma breakpoint family member 25 pseudogene), HSP90AB2P (heat shock protein 90 alpha class B member 2 pseudogene), NXF4 (nuclear RNA export factor 4 pseudogene), FOLH1B (folate hydrolase 1B), DNM1P47, BNIP3P1 (BCL2/adenovirus E1B 19 interacting protein 3 pseudogene 1), PKD1L2 (polycystin 1-like 2 pseudogene), BAGE2, and ZNF658B (zinc finger protein 658B, pseudogene) were found to be the 10 most deregulated of 154 pseudogene transcripts identified (Table 2 and Supplementary Dataset S1).

(p13.3) In oropharyngeal cancer (n = 51), POTEA (POTE ankyrin domain family, member A), MROH5 (maestro heat-like repeat family member 5), MSL3P1 (male-specific lethal 3 homolog pseudogene 1), HLA-H (major histocompatibility complex class I, H pseudogene), TUBB8P7 (tubulin beta 8 class VIII pseudogene 7), SLC7A5P2 (solute carrier family 7 member 5 pseudogene 2), DPY19L2P1 (DPY19L2 pseudogene 1), TSSC2 (tumor suppressing sub-transferable candidate 2 pseudogene), SPATA31C2, and NXF4 were found to be the 10 most deregulated of 109 pseudogene transcripts identified ( Table 2 and Supplementary Dataset S1).

(p13.4) In hypopharyngeal cancer (n = 8), DPY19L2P3 (DPY19L2 pseudogene 3), SPATA31D5P, GBA3 (glucosidase beta acid 3), PLEKHM1P (pleckstrin homology domain containing, family M member 1 pseudogene), DPY19L2P1, MST1P2 (macrophage-stimulating 1 pseudogene 2), RP11-44F14.1, ADAM21P1 (ADAM metallopeptidase domain 21 pseudogene 1), MAGEB6P1, and OR12D2 (olfactory receptor family 12 subfamily D member 2) were found to be the 10 most deregulated of 21 pseudogene transcripts identified (Table 2 and Supplementary Dataset S1).

(p13.5) In laryngeal cancer (n = 98), HERC2P3, SPATA31D5P, SPATA31C2, SLC25A51P1, MAGEB6P1, SPATA31C1, BAGE2, PNLIPRP2 (pancreatic lipase-related protein 2), ZNF733P and DNM1P47 were found to be the 10 most deregulated of 287 pseudogene transcripts identified (Table 2 and Supplementary Dataset S1).
## (s14) Co-Expression Networks and GO Enrichment Analysis
(p14.0) We performed co-expression network and GO enrichment analyses to map the relationship between pseudogenes and other genes in the genome. After analyzing 19,406 proteincoding genes and 31 pseudogenes, our co-expression networks contained 14,379 genes organized in 36 modules, with an average of 399 genes per module (median: 140 genes; range: 35-3645 genes; Supplementary Dataset S2).

(p14.1) From the 31 most deregulated pseudogenes from HNC and its subtypes, 14 of them (SPATA31D5P, SPATA31C2, DNM1P47, NBPF25P, HSP90AB2P, NXF4, ZNF658B, POTEA, MROH5, HLA-H, DPY19L2P1, DPY19L2P3, GBA3, and PNLIPRP2) were present in our network across 8 modules (light yellow, red, dark magenta, orange, black, salmon, ivory, and dark olive green). Although the other pseudogenes of interest did not show a coexpression profile similar enough to other genes to be assigned to a module according to the parameters used to build this network, they are still important for further studies in HNC based on our previous TCGA data analysis.

(p14.2) We chose to highlight three modules (light yellow, red, and dark magenta) that contain genes well-known to be involved in head and neck carcinogenesis based on two important reviews [76,77]. However, the other five modules also presented relevant gene associations that should be evaluated in further studies.

(p14.3) The light-yellow module contained the greatest number of pseudogenes (NBPF25P, HSP90AB2P, ZNF658B, and DPY19L2P3). Interestingly, this module also contains 12 genes known to participate in head and neck carcinogenesis (CASP8, involved in apoptosis; NOTCH2 and NOTCH3, involved in cell differentiation; TRAF3, involved in antiviral response; RB1 and HRAS, involved in cell cycle control and proliferation; PTEN, involved in apoptosis and cell cycle control; CUL3 and NFE2L2, involved in oxidative stress response; EP300, involved in chromatin remodeling, and the transcription factors IGF1R and TP63) [76,77], suggesting that these genes may directly or indirectly interact with the pseudogenes in this module and play a role in HNC (Figure 2A). In addition, the light-yellow module is enriched in GO terms related to tumor development, such as regulation of cell communication, cellular response to stress, and intracellular transport ( Figure 2B). The potential relationship between the described pseudogenes and genes in this module has not been explored in the literature yet.

(p14.4) The red module contains the pseudogene DNM1P47 and the tumor suppressor TP53 gene, mainly involved in cell cycle control and apoptosis [76,77] (Figure 3A), and is enriched in GO terms associated with regulation of cell proliferation, response to stress, and cell death ( Figure 3B). The potential relationship between the DNM1P47 pseudogene and the TP53 gene has not been explored in the literature yet. terms identified by GOATOOLS Python library and summarized by the web tool REVIGO. Circles colored with darker red indicate GO terms with lower p-values in our enrichment analysis, while the size of the circles indicates the frequency of a GO term in the GO annotation database. GO terms that are highly similar have thicker lines. The dark magenta module contains the known tumor genes HLA-A and HLA-B, involved in immune response [77], very closely associated with the HLA-H pseudogene ( Figure 4A), and is also enriched in GO terms important to cancer biology, such as cell proliferation, regulation of the immune system, regulation of gene expression, and Wnt signaling pathway ( Figure 4B). The relationship of the HLA-H pseudogene and HLA-A and HLA-B genes has been described in the literature, and even related to lung cancer in the Asian population [53]. The SNV rs12333226, capable of modulating HLA-A and HLA-H expression levels, was suggested to exert an effect on lung cancer through these two immune-related genes and the pseudogene [53].  The complete list of genes in light-yellow, red, and dark magenta modules can be found in Supplementary Dataset S2. The other five modules consist of an orange module containing DPY19L2P1, a black module containing NXF4, MROH5, and SPATA31D5P, a salmon module containing PNLIPRP2, an ivory module containing SPATA31C2, and a dark olive-green module containing POTEA and GBA3 pseudogenes. The list of genes interacting with those pseudogenes and the enriched GO terms for these modules can be found in Supplementary Dataset S2.
## (s15) Survival Analysis
(p15.0) To investigate the prognostic value of the pseudogenes identified by the TCGA database in HNC, we assessed the online database The Kaplan-Meier Plotter. In this analysis, only the prognostic value of the ten most deregulated pseudogenes identified in HNC general samples was assessed (SPATA31D5P, HERC2P3, SPATA31C2, MAGEB6P1, SLC25A51P1, BAGE2, DNM1P47, SPATA31C1, ZNF733P and OR2W5) because the online database does not allow to stratify the patients by tumor location.

(p15.1) We observed that lower expression of SPATA31D5P, SPATA31C2, BAGE2, SPATA31C1, ZNF733P and OR2W5 pseudogene transcripts was associated with the worst RFS of HNC patients. Patients with lower expression presented 2.56, 2.63, 2.33, 3.57, 3.03, and 3.03 more chances respectively, to relapse, compared to other patients ( Figure 5A).

(p15.2) In addition, we observed that higher expression of SPATA31D5P, ZNF733P and OR2W5 pseudogene transcripts was associated with the worst OS of HNC patients. Patients with higher expression presented 1.39, 1.56, and 1.53 more chances respectively, of dying, compared to other patients. Moreover, lower expression of SPATA31C2, BAGE2 and SPATA31C1 pseudogene transcripts was associated with the worst OS of HNC patients. Patients with lower expression presented 1.43, 1.39, and 1.43 more chances respectively, of dying, compared to other patients ( Figure 5B). MAGEB6P1, SLC25A51P1 and DNM1P47 pseudogene transcripts were not available in The Kaplan-Meier Plotter database, so we could not perform survival analysis. The HERC2P3 pseudogene transcript did not influence RFS or OS in the HNC patients. However, in the literature, HERC2P3 upregulation was associated with cell migration in gastric cancer, an independent prognosis factor [44]. The prognostic values of SPATA31D5P, SPATA31C2, BAGE2, SPATA31C1, ZNF733P and OR2W5 have not been explored in the literature yet.  
## (s16) Conclusions
(p16.0) The biology of cancer is complex and not fully understood so far. Pseudogenes lost the label of "junk DNA" and are now known to be modulators of gene expression and potential biomarkers for cancer risk and prognosis [12]. Pseudogene transcripts regulate gene expression by directly binding at the target mRNA or by functioning as miRNA sponges, impeding miRNA-target mRNA binding [20]. Thus, pseudogene transcripts can function as negative or positive gene regulators [20].
