# Association of DNA repair genes polymorphisms and mutations with increased risk of head and neck cancer: a review

CorpusID: 11101581
 
tags: #Medicine

URL: [https://www.semanticscholar.org/paper/78127fcce031cd816e66607b3b24847a666b4342](https://www.semanticscholar.org/paper/78127fcce031cd816e66607b3b24847a666b4342)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Association of DNA repair genes polymorphisms and mutations with increased risk of head and neck cancer: a review
2017

Agata Dylawerska 
Department of Medical Physics
Radiobiology Lab
The Greater Poland Cancer Centre
Garbary 15 Str61-866PoznanPoland

Wojciech Barczak 
Department of Medical Physics
Radiobiology Lab
The Greater Poland Cancer Centre
Garbary 15 Str61-866PoznanPoland

Department of Head and Neck Surgery
The Greater Poland Cancer Centre
Poznan University of Medical Sciences
Garbary 15 Str61-866PoznanPoland

Anna Wegner 
Department of Head and Neck Surgery
The Greater Poland Cancer Centre
Poznan University of Medical Sciences
Garbary 15 Str61-866PoznanPoland

Wojciech Golusinski 
Department of Head and Neck Surgery
The Greater Poland Cancer Centre
Poznan University of Medical Sciences
Garbary 15 Str61-866PoznanPoland

· Wiktoria 
Maria Suchorska 
Department of Medical Physics
Radiobiology Lab
The Greater Poland Cancer Centre
Garbary 15 Str61-866PoznanPoland

Department of Electroradiology
Poznan University of Medical Sciences
Garbary 15 Str61-866PoznanPoland

Wojciech Barczak 
Wojciech Barczak@wco Pl 
Association of DNA repair genes polymorphisms and mutations with increased risk of head and neck cancer: a review

(0123456789) 1 3 Medical Oncology
34197201710.1007/s12032-017-1057-4Received: 22 October 2017 / Accepted: 10 November 2017 / Published online: 15 November 2017REVIEW ARTICLEHead and neck cancer · DNA repair · Polymorphisms · Cancer susceptibility
DNA repair mechanisms allow maintain genomic stability and proper functioning within the cells. Any aberrations may cause an increased risk of many diseases such as cancer. The most crucial risk factors for head and neck squamous cell carcinoma are behavioral factors, predominantly chronic exposure to tobacco, alcohol addiction, and infection with human papillomavirus or Epstein-Barr virus. These agents can induce DNA damage; therefore, cells must activate appropriate mechanisms in order to function correctly. Cancer cells are marked with genomic instability, which is associated with a greater tendency for the accumulation of a DNA damage and increased chemo-and radioresistance. Multiple studies have assessed the correlation of increased head and neck cancer (HNC) risk with polymorphism in the DNA repair genes. However, they suggest that interaction of DNA repair genes mutations with susceptibility to HNC depends on a patient's race and risk factors, especially tobacco smoking. Further identification of these sequence variations must be performed. In this review, we discuss the current knowledge about the DNA repair genes mutations and polymorphisms associated with the high risk of head and neck treatment.

## Introduction

Head and neck cancers (HNC) comprise a group of heterogeneous tumors located in multiple anatomical sites and cellular origins within the head and neck region. Almost 90% of these cancers are squamous cell carcinomas (head and neck squamous cell carcinomas, HNSCC). They include neoplasms of the oral cavity, nasal cavity, pharynx, and larynx. HNC often spread to the lymph nodes of the neck, usually giving the first symptom of the disease [1,2]. HNSCC remains a major health problem worldwide; it is classified as the sixth most common cancer with around 500,000 new diagnoses each year [3]. In Poland almost 60,000 patients are diagnosed with this disease yearly and nearly 4000 deaths attribute to this type of cancer [4]. The most common symptoms of the HNSCC include: mass in the neck, bleeding from the mouth, sore throat, difficulty food swallowing, cough, and trouble breathing or speaking [5].

A first-line treatment is surgery, as a supportive therapy is frequently used radiation and/or chemotherapy as an adjuvant [6]. HNSCC is strongly associated with certain environmental and lifestyle risk factors such as alcohol consumption, tobacco smoking, UV light, and particular strains of viruses including human papillomavirus (HPV) and Epstein-Barr virus (EBV) [7,8]. These environmental agents may induce an abnormal DNA damage response (DDR), which can result in cell death, instability of the chromosomes, and unregulated proliferation [9][10][11]. DNA repair genes play an important role in the maintenance of the genomic integrity and protection of cells from DNA damage. Replication of the damaged DNA contributes to mutation thus causing disease. Therefore, the alteration of DNA repair genes could increase the risk of HNC [12]. The most important pathways include single-strand damage repair and double-strand break (DSB) repair. The aim of this article is to review the polymorphisms and mutations of genes involved in DNA repair systems and role in cancerogenesis of HNC.

197 Page 2 of 8 DNA damage and repair: mechanisms for maintaining DNA integrity Cells respond to DNA lesions by activation of composite DDR pathways that consist of: cell cycle arrest, transcriptional and posttranslational activation of genes involved in DNA repair and inducing programmed cell death. In case of a great amount of DNA damage accumulated within a cell or incapability to repair all the lesions, there are three possible states, that cell can enter: senescence, apoptosis, or unregulated cell division, which can lead to the formation of a tumor. The efficiency of the DDR pathways' activation is determined by the nuclear levels of the DNA repair proteins [13,14]. Incapability of appropriate response to DNA lesions and/or DNA repair leads to genomic instability. The recent studies show that head and neck carcinogenesis is associated with abnormalities in DNA repair, apoptosis, carcinogen metabolism, and cell cycle control [15]. Cells are unable to function properly, if the DNA lesions apply to essential information in the genome. Depending on the type of damage inflicted on the DNA's double helical structure, a variety of repair strategies have evolved to restore lost information. There are two main pathways of DNA repair: single-strand damage and double-strand breaks ( Fig. 1) [13]. Single-strand damage repair is substantially a valid system that requires the presence of an intact DNA strand as a template. It consists of three main types of mechanisms: base excision repair (BER), nucleotide excision repair (NER), and mismatch repair (MMR). BER is activated, when the DNA lesion is localized within the single nitrogenous base. Base modifications are the most common type of endogenous DNA damage, accounting for thousands of lesions per mammalian genome daily. DNA alterations include alkylative and oxidative base products, abasic sites, strand breaks, and misincorporated nucleotides. BER pathway evolved to cope with the high level of spontaneous decay products that are formed in DNA, as well as those damages created upon reactions with natural endogenous substances such as reactive oxygen species (ROS) [16]. This mechanism encompasses following steps: (1) detection and removal of a damaged base by DNA glycosidase, resulting in creation The detected lesions are then corrected with the use of two main different mechanisms, depending on the type of damage: single-strand damage repair (SSD) or double-strand breaks system (DSBs). The first one consists of three distinct pathways: BER, NER, and MMR. BER works when DNA alterations include alkylative and oxidative base products, as well as those afforded by radiation. UV light causes damage, which is repaired by NER system, whereas replicative errors are removed by MMR mechanism. DSBs are associated with oxygen radicals, UV light, radiation, and anti-tumor agents. An adequate system is activated depending on the phase of replication process. NHEJ repair is enabled in G1 stage, whereas in G2/M stage HR works of an apurinic or apyrimidinic site (AP site); (2) cutting the damaged DNA backbone at the AP site by AP endonuclease;

(3) removal of the damaged region by lyase or phosphodiesterase; (4) synthesis of the new strand by DNA polymerase using the complementary strand as a template; (5) re-enactment of a phosphodiester bond by DNA ligase [17]. NER leads to removal of a larger DNA fragment comparing to BER. This system repairs DNA lesions, which consist of a helix-distorting damage, such as pyrimidine dimerization caused by UV light. In the first phase proteins XPA (DNA damage recognition and repair factor) and XPC (XPC complex subunit, DNA damage recognition and repair factor) incorporate to the damaged region. Then, two endonucleases-ERCC4 (ERCC excision repair 4, TFIIH core complex helicase subunit) and ERCC5 (ERCC excision repair 5, TFIIH core complex helicase subunit), cut the DNA strand on both sides of the damage. Impaired fragment is then degraded by DNA helicases (ERCC2 (ERCC excision repair 2, TFIIH core complex helicase subunit) and ERCC3 (ERCC excision repair 3, TFIIH core complex helicase subunit)). Finally, as in BER mechanism, the gap is filled with the use of the DNA polymerase (in the case of eukaryotic cells with DNA polymerase δ or ε) and the phosphodiester bond is recreated by DNA ligase [18]. MMR system's function is to recognize and repair incorrect insertion, deletion, and mis-incorporation of nitrogenous bases that can occur throughout the DNA replication process. The activation of this mechanism depends on the valid functioning of an enzymatic complex that include: MutS protein, which forms a dimer, that recognizes an incorrect complementary between nucleotides and binds to the damaged DNA: MuH, which attaches at hemimethylated sites along the impaired fragment, and MutL protein, which activates the MutH peptide and also acts as a mediator between MutS2 and MutH. At the final step, DNA polymerase and ligase catalyze the synthesis of a new DNA fragment [19]. A crucial part of this system is the MutS-homolog 2 (MSH2) gene. It encodes a protein which recognizes DNA mismatches by forming two functional heterodimers: MSH2-MSH6 and MSH2-MSH3. The first one recognizes single-base mismatches and short insertion-deletion loops, whereas MSH2-MSH3 complex is able to detect larger loops in DNA molecule [20,21]. Due to the irreversibility of the changes, DSBs are the most adverse and fatal types of damage. Repair of these impairments can be divided into three main steps: detection, signaling, correction. An adequate system is activated depending on the stage of the cell cycle. Non-homologic end joining (NHEJ) mechanism is enabled in G1 phase, whereas in G2/M phase homologic repair (HR) is activated [13]. During the NHEJ pathway, broken ends are bound by Ku70/80 heterodimer, which results in activation of DNA-PKcs. After the ends have been joined together, the XRCC4 (X-ray repair crosscomplementing 4)/ligase IV complex completes the final ligation step, and the lesion is repaired [22]. NHEJ system does not require the presence of DNA template and is operative during all the steps of cell cycle [23,24].

When the HR mechanism works, resecting of the impaired DNA endings by MRE11 (MRE11 homolog, double-strand break repair nuclease) or Exo1 (exonuclease 1) takes place. It leads to formation of a structure called 3′overhang, which is stabilized by RPA (replication protein A) and then loaded on the homologous DSB region by the strand exchange protein RAD51 (RAD51 recombinase) and by BRCA2 (breast cancer 2, DNA repair associated), resulting in the formation of the Holliday junctions intermediate. In mammalian cells, the HR system seems to be less practical compared with NHEJ, but defects in the HR mechanism to enhance cellular radiosensitivity. Since most cells arrest the cell cycle at the G2 phase in response to ionic radiation (IR) exposure and the HR mechanism mainly works in Late-S and G2 phases, inhibiting this would be a good strategy for the new cancer treatment [25].

These processes may function before, during or after the DNA replication. They are able to recognize abnormalities in the DNA structure and to recreate the valid structure of a molecule. A large number of mutations can result in uncontrolled cell division, which contributes to the development of a cancer, apoptosis, increasing the rate of cell aging, or the formation of hereditary disorders [13].


## Role of DNA repair in carcinogenesis and therapy

Unfortunately, the current treatment does not provide sufficient results compared to the resulting side effects, which have a negative impact on patient's quality of life (QOL). Nowadays, cancer therapeutics can be a major challenge, when it comes to seeking personalized targeted medicine that is both effective and selective to the malignancy [26]. Characteristic feature of a cancer cells is genomic instability which is associated with a greater tendency to accumulation of a DNA damage. Chemotherapy or radiation therapy leads to the DNA lesions of a cancer cells, thus causing their death [27]. However, a part of them is able to survive in genotoxic stress afforded by therapy due to increased DNA repair [28]. They have an impaired DDR pathway so they can only depend on a sole backup pathway for their survival. The use of inhibitors directed against the essential components of this process could be a promising strategy to eradicate cancer cells. Aberrant epigenetic changes in DNA damage repair genes may also serve as therapeutic targets [29,30]. Numerous studies are currently performed in order to relate the activation of a DNA repair genes in cancer cells to the low effectiveness of therapy.


## Mutations and polymorphisms of DNA repair genes in HNC

DNA repair systems play an important role in protection against carcinogenic agents. Sequence variation in genes regulating these mechanisms may impair their functions and consequently, result in increased cancer susceptibility. Multiple studies have assessed the correlation of HNC increased risk with polymorphism in the DNA repair genes.

Single-strand damage repair: genes polymorphisms BER genes play have been considered as a good candidate causing increased susceptibility for HNC, because of their important role in DNA repair; that is, removal of a lesions caused by ROS and other electrophiles, which are most common types of damage [31]. Mahjabeen et al. [32] demonstrated that APEX1 (apurinic/apyrimidinic endodeoxyribonuclease 1) mutations are associated with increased risk of HNC in the Pakistani population. They studied 50 cases of HNSCC among smokers and non-smokers. At the transcript level, three novel mutations (13T>G, Ser129Arg and Val-131Gly) of APEX1 were observed. The homozygous and heterozygous genotypes of variants mentioned above seem to be significantly involved in the development of HNC. Moreover, increased expression of APEX1 correlated with tumor size, clinical stage, and positive lymph node metastasis [32]. However, sequence variations in XRCC1 (X-ray repair cross-complementing 1) gene appear to be correlated with HNSCC susceptibility depending on population. Arg194Trp and Arg399Gln polymorphisms were studied in 55 cases of HNSCC in Turkish population, as well as Arg194Trp, Arg280His, and Arg399Gln variations were analyzed in 6719 Chinese patients [33,34]. In both studies, mutations of XRCC1 gene are not related to HNC development. However, Choudhury's [35] findings suggest that interaction of tobacco and polymorphisms of XRCC1 and XRCC2 (X-ray repair cross-complementing 2) genes increase the risk of HNSCC. Furthermore, cross talk between these two DNA repair genes might modulate susceptibility toward HNSCC. He investigated the intercommunication of different variants of XRCC1 (Arg399Gln) and XRCC2 (Arg188His) as well as tobacco exposure in the progression of HNSCC in northeast Indian population. The population-based case-control study included 110 HNSCC patients [35]. Other studies analyzed the SNPs XPC A499 V, XPD K751Q, XRCC1 R399Q, and XRCC3 (X-ray repair cross-complementing 3) T241 M among Caucasian population. The aim was to determine if these changes are potential risk factors and indicators of survival. Following results were obtained: (1) XPC A499 V was associated with increased risk of HNSCC, especially laryngeal carcinoma; (2) XPD K751Q was correlated with higher risk of oral SCC among women; (3) wild-type individuals of XRCC3 T241M exhibited an earlier age of onset; (4) XRCC1 R399Q homozygous mutant genotype determined HPV positivity. Moreover, XPD homozygous mutant subjects showed the shortest survival time, which has raised after full-dose radiotherapy. Furthermore, variations of putative risk alleles seemed to act synergistically, raising the risk of HNSCC. Results described above indicate that SNPs of the DNA repair genes (XPC, XPD, XRCC1, and XRCC3) may affect risk and survival of HNSCC [36]. Other studies present similar findings. XPD Asp312Asn and APEX1 Asp148Glu polymorphisms increase the risk for HNC in association with smoking and/or tobacco chewing in a population of northeast India. Das et al. (2015) investigated the effect of the following DNA repair gene variations: XPD Asp312Asn (G>A), APEX1 Asp148Glu (T>G), and MUTYH (mutY DNA glycosylase) Tyr165Cys (G>A). The study included 80 HNC patients and 92 healthy controls. Genotyping with the use of amplification refractory mutation system-PCR (ARMS-PCR) was performed to analyze XPD Asp312Asn (G>A) alteration and PCR using confronting two-pair primers (PCR-CTPP) for APEX1 Asp-148Glu (T>G) and MUTYH Tyr165Cys (G>A) mutations. The XPD Asp/Asn genotype increased the risk for HNC by twofold. Interaction between APEX1 Asp/Asp and XPD Asp/Asn as well as MUTYH Tyr/Tyr and XPD Asp/Asn genotypes further increased the risk by 2.9-folds. The risk was further increased in heavy smokers with the XPD Asp/ Asn genotype and heavy tobacco chewers with XPD Asn/ Asn genotype by 7.7-fold and 10-fold [37].

Abnormalities involving MMR genes occur in microsatellite instability and the accumulation of mutations in proto-oncogenes or tumor suppressor genes. These factors may result in cancer development. Furthermore, it has been observed that decreased expression of MSH2 gene leads to increased frequency of microsatellite instability [38,39]. Pereira et al. [40] investigated the expression of the MSH2 DNA repair protein in HNSCC collected from 55 cases in Brazil. Patients with locoregional metastatic malignance and lower MSH2 protein levels presented worse survival. It contributes to a higher clinic aggressiveness of HNSCC [40]. Moreover, there is a study showing that inherited MLH1 (mutL homolog 1) c.-93G>A, MSH2 c.211 + 9C>G, MSH3 c.3133G>A, and EXO1 c.1765G>A anomalies of DNA MMR pathway are essential determinants and important patient outcomes' predictors of HNSCC, notably among smokers [41]. Alternate studies also point that single nucleotide polymorphism (SNP) in hMLH1 promoter is combined with increased risk of tobacco-related oral squamous cell carcinoma (OSCC) in Asian Indians. 242 patients with tobacco-related cancer and 205 healthy controls were genotyped in order to analyze -93 A>G (rs 1800734) variation in hMLH1 promoter. The number of AA genotype was substantially lower in patients comparing to the controls (21.49 vs. 47.8%). In contrast, the frequency of GG genotype was much higher in patients as compared to the healthy controls (41.32 vs. 13.66%). These results indicate that variant G allele is associated with increased risk of OSCC in comparison with the wild-type A allele [42].


## Double-strand breaks: genes polymorphisms

Since DSBs are the most lethal type of DNA damage, even a single one is able to kill a cell or disturb its functioning [43]. DSBs are mostly corrected by either HR or by NHEJ pathway in mammalian cells [44]. Genetic variations of DSBs genes may lead to adverse differences of repair capacity in the general population and healthy individuals [45]. Recently, RAD51C (RAD51 paralog C) gene has been identified as human cancer gene associated with higher risk for inherited breast and ovarian malignancies [46]. It is one of the five analogues of highly conserved RAD51 recombinase which functions as a principal protein in the HR [47,48]. Scheckenbach et al. [49] analyzed 121 patients with HNSCCs from Germany for germline alterations in RAD51C. All exons and the adjoining introns of this gene were sequenced. Results revealed five distinct heterozygous variations in seven patients (5.8%). One female patient has been identified as a carrier of germline mutation that disrupted the canonical splice acceptor site of exon 5 (c. 706-2A>G). All sequence mutations were single nucleotide modifications. These changes are associated with reduced or absent (c.706-2A>G) function, thus suggesting that germline polymorphisms in genes of the FA pathway could contribute to higher risk of HNSCC development [49].

Kayani et al. [50] found that RAD51 (135G/C, 172 G/T) and XRCC3 (Thr241Met) polymorphisms may be effective biomarkers for genetic susceptibility to HNC. Each variant was genotyped using the polymerase chain reactionrestriction fragment length polymerase (PCR-RFLP) technique. They investigated 200 pathologically confirmed HNC individuals as well as 150 healthy patients. Homozygous variant CC genotype of RAD51 135G/C was associated with a 2.5-fold increased HNC risk, while second variant-RAD51 172G/T, heterozygous genotype was correlated with a 1.68-fold comparing to the controls. The most meaningful result was the Thr241Met alteration of XRCC3 where they observed 16-fold increased HNC risk [50]. However, genetic alterations of the MRN protein complex, which play an important role in DSBs repair and the DNA damage checkpoint activation, do not contribute to a high susceptibility to HNC. RAD50 (RAD50 double-strand break repair  [50] protein) and MRE11 genes were analyzed in 358 patients with three different types of HNC from Poland. PCR-SSCP (polymerase chain reaction single-strand conformation polymorphism) method was used to evaluate the coding sequence and exon-intron boundaries of exons 3, 4, 5, 7, 21, and 25 of the RAD50 gene and exons 5,8,9,10,14,15,17, and 19 of the MRE11 gene. Only four SNPs were found within these sequences (see Table 1) [51]. Regrettably, the potential role of the MRE11 gene in human cancers is not well documented. Only Bartkova et al. [52] have described two germline variations: a missense mutation p.R202G and a truncating mutation p.R633X. These studies certify MRE11 gene as a potential candidate associated with high risk of breast cancer in a subset of non-BRCA1/2 families in Denmark [52]. Specific germline polymorphisms predisposing to HNSCC have only been classified in P53 and INK4a/p16 genes. They are concerned with a wide variety of human cancers [53,54]. More studies must be performed in order to identify HNSCC patients for germline mutations in FA-associated genes. It could provide further insights into "common disease, rare allele" hypothesis [55] and might also apply to HNSCC patients with relatively young age and/or without known risk factors solving the cause of epidemiological phenomenon. Current knowledge about DNA repair genes polymorphisms or mutations is presented in Table 1.


## Conclusions

Activation of DNA repair genes in cancer cells is a severe problem. It leads to the invulnerability to therapy thus decreasing efficiency of the treatment. When the DNA's structure is impaired, cell undergoes a cell cycle arrest. If DNA repair genes are activated and show a high expression, all the lesions can be corrected and the cell can finish a replication process, and if not it is directed to apoptosis pathway or senescence. The most crucial risk factors for HNSCC are behavioral factors, predominantly chronic exposure to tobacco, alcohol addiction, and infection with HPV or EBV. HNSCC cancer carcinogenesis has been related to anomalies in DNA repair, apoptosis, carcinogen metabolism, and cell cycle control. Heretofore, single-strand repair genes alterations are well documented, when DSBs mechanism still need more accurate studies. However, latest findings suggest that DNA repair genes' mutations-related susceptibility to HNC depends on a patient's race. Further identification of these sequence variations and population-specific differences as well as gene-gene interactions may help in high-risk screening of humans exposed to environmental carcinogens and cancer predisposition in different ethnic groups.

## Fig. 1
1Environmental factors may create DNA damage.

## Table 1 DNA
1repair genes 
polymorphisms/mutations or 
their products' sequence in head 
and neck cancer 

Gene 
Mechanism Mutation/polymorphism 
High risk factor Population 
References 

APEX1 BER 
13T>G, Ser129Arg, Val131Gly 
Yes 
Pakistani 
[32] 
XRCC1 BER 
Arg194Trp, Arg399Gln 
No 
Turkish 
[33] 
XRCC1 BER 
Arg194Trp, Arg280His, Arg399Gln No 
Chinese 
[34] 
XRCC1 BER 
Arg399Gln 
Yes 
Indian 
[35] 
XRCC2 BER 
Arg188His 
Yes 
Indian 
[35] 
XPC 
NER 
A499V 
Yes 
Caucasian 
[36] 
XPD 
NER 
K751Q 
Yes 
Caucasian 
[36] 
XRCC1 BER 
R399Q 
Yes 
Caucasian 
[36] 
XRCC3 HR 
T241M 
Yes 
Caucasian 
[36] 
XPD 
NER 
Asp312Asn 
Yes 
Indian 
[37] 
APEX1 BER 
Asp148Glu 
Yes 
Indian 
[37] 
MUTYH BER 
Tyr165Cys 
Yes 
Indian 
[37] 
MSH2 
MMR 
No mutation, low expression 
Yes 
Brazilian 
[40] 
MLH1 
MMR 
c.-93G>A 
Yes 
Brazilian 
[41] 
MSH2 
MMR 
c.211 + 9C>G 
Yes 
Brazilian 
[41] 
MSH3 
MMR 
c.3133G>A 
Yes 
Brazilian 
[41] 
EXO1 
MMR 
c.1765G>A 
Yes 
Brazilian 
[41] 
hMLH1 MMR 
-93 A>G (promoter) 
Yes 
Asian Indians [42] 
RAD51C HR 
c. 706-2A>G 
Yes 
German 
[49] 
RAD50 HR 
c.379G>A (ex. 4), c.943G>T (ex. 
7), c.3876C>T (ex. 25) 

No 
Polish 
[51] 

MRE11 HR 
c.1783-86delAG (int. 16) 
No 
Polish 
[51] 
MRE11 HR 
p.R202G and p.R633X 
Yes 
Danish 
[52] 
RAD51 HR 
135G/C, 172 G/T 
Yes 
Pakistani 
[50] 
XRCC3 HR 
Thr241Met 
Yes 
Pakistani 


Head and neck cancer. A Argiris, M V Karamous, D Raben, R L Ferris, 10.1016/S0140-6736(08)60728-X1016/S0140-6736(08)60728-XLancet. 3719625Argiris A, Karamous MV, Raben D, Ferris RL. Head and neck cancer. Lancet. 2008;371(9625):1695-709. https://doi. org/10.1016/S0140-6736(08)60728-X.

The molecular biology of head and neck cancer. C R Leemans, B J Braakhuis, R H Brakenhoff, 10.1038/nrc2982Nat Rev Cancer. 11Leemans CR, Braakhuis BJ, Brakenhoff RH. The molecular biol- ogy of head and neck cancer. Nat Rev Cancer. 2011;11:9-22. https://doi.org/10.1038/nrc2982.

Cancer statistics. CA Cancer. R Sigel, E Ward, O Brawley, A Jemal, 10.3322/caac.20121J Clin. 61Sigel R, Ward E, Brawley O, Jemal A. Cancer statistics. CA Can- cer J Clin. 2011;61:212-36. https://doi.org/10.3322/caac.20121.

Obecny stan zwalczania nowotworów w Polsce. Polskie Towarzystwo Onkologiczne, Polskie Towarzystwo Onkologiczne (2014) Obecny stan zwalcza- nia nowotworów w Polsce, pp 17-20. https://pto.med.pl/content/ download/7709/84400/file/Obecny_Stan.pdf. Accessed 23 Oct 2016.

The head and neck symptom checklist(s): an instrument to evaluate nutrition impact symptoms effect on energy intake and weight loss. C Kubrak, K Olson, V E Baracos, 10.1007/s00520-013-1870-zSupport Care Cancer. 2111Kubrak C, Olson K, Baracos VE. The head and neck symp- tom checklist(s): an instrument to evaluate nutrition impact symptoms effect on energy intake and weight loss. Support Care Cancer. 2013;21(11):3127-36. https://doi.org/10.1007/ s00520-013-1870-z.

Pathology of head and neck cancers I: epithelial and related tumors. G K Haines, Radosevich JASpringerDordrechtHead and neck cancer: current perspectives, advances and challengesHaines GK. Pathology of head and neck cancers I: epithelial and related tumors. In: Radosevich JA, editor. Head and neck can- cer: current perspectives, advances and challenges. Dordrecht: Springer; 2013. p. 257-87.

The molecular pathogenesis of head and neck cancer. J D Klein, J R Grandis, Cancer Biol Ther. 91Klein JD, Grandis JR. The molecular pathogenesis of head and neck cancer. Cancer Biol Ther. 2010;9(1):1-7.

HPV & head and neck cancer: adescriptive update. P K Goon, M A Stanley, J Ebmeyer, L Steinsträsser, T Upile, W Jerjes, M Bernal-Sprekelsen, M Görner, H H Sudhoff, 10.1186/1758-3284-1-36Head Neck Oncol. 1136Goon PK, Stanley MA, Ebmeyer J, Steinsträsser L, Upile T, Jerjes W, Bernal-Sprekelsen M, Görner M, Sudhoff HH. HPV & head and neck cancer: adescriptive update. Head Neck Oncol. 2009;1(1):36. https://doi.org/10.1186/1758-3284-1-36.

Tobacco smoke carcinogens, DNA damage and p53 mutations in smoking-associated cancers. G P Pfeifer, M F Denissenko, M Olivier, N Tretyakova, S S Hecht, P Hainaut, Oncogene. 21Pfeifer GP, Denissenko MF, Olivier M, Tretyakova N, Hecht SS, Hainaut P. Tobacco smoke carcinogens, DNA damage and p53 mutations in smoking-associated cancers. Oncogene. 2002;21:7435-51.

Immediate consequences of cigarette smoking: rapid formation of polycyclic aromatic hydrocarbon diol epoxides. Y Zhong, S G Carmella, P Upadhyaya, J B Hochalter, D Rauch, A Oliver, J Jensen, D Hatsukami, J Wang, C Zimmerman, S S Hecht, 10.1021/tx100345xChem Res Toxicol. 24Zhong Y, Carmella SG, Upadhyaya P, Hochalter JB, Rauch D, Oliver A, Jensen J, Hatsukami D, Wang J, Zimmerman C, Hecht SS. Immediate consequences of cigarette smoking: rapid forma- tion of polycyclic aromatic hydrocarbon diol epoxides. Chem Res Toxicol. 2011;24:246-52. https://doi.org/10.1021/tx100345x.

Genome maintenance mechanisms for preventing cancer. J H Hoeijmakers, 10.1038/35077232Nature. 411Hoeijmakers JH. Genome maintenance mechanisms for preventing cancer. Nature. 2001;411:366-74. https://doi. org/10.1038/35077232.

. H P Yu, X Y Zhang, X L Wang, L Y Shi, Y Y Li, F Li, Y H Su, Y J Wang, B Lu, X Sun, W H Lu, S Q Xu, DNA repair gene. 1Yu HP, Zhang XY, Wang XL, Shi LY, Li YY, Li F, Su YH, Wang YJ, Lu B, Sun X, Lu WH, Xu SQ. DNA repair gene XRCC1

polymorphisms, smoking, and esophageal cancer risk. Cancer Detect Prev. 28polymorphisms, smoking, and esophageal cancer risk. Cancer Detect Prev. 2004;28:194-9.

Hsp90: a New Player in DNA Repair?. R Pennisi, P Ascenzi, A Di Masi, 10.3390/biom5042589Biomolecules. 54Pennisi R, Ascenzi P, di Masi A. Hsp90: a New Player in DNA Repair? Biomolecules. 2015;5(4):2589-618. https://doi. org/10.3390/biom5042589.

The genetics of human longevity. W S Browner, A J Kahn, E Ziv, A P Reiner, J Oshima, R M Cawthon, W C Hsueh, S R Cummings, 10.1016/j.amjmed.2004.06.033Am J Med. 11711Browner WS, Kahn AJ, Ziv E, Reiner AP, Oshima J, Cawthon RM, Hsueh WC, Cummings SR. The genetics of human longev- ity. Am J Med. 2004;117(11):851-60. https://doi.org/10.1016/j. amjmed.2004.06.033.

Genetic aberrations in oral or head and neck squamous cell carcinoma (SCCHN) carcinogen metabolism, DNA repair and cell cycle control. C Scully, J K Field, H B Tanzawa, Oral Oncol. 36Scully C, Field JK, Tanzawa HB. Genetic aberrations in oral or head and neck squamous cell carcinoma (SCCHN) carcinogen metabolism, DNA repair and cell cycle control. Oral Oncol. 2000;36:256-63.

DNA joining in mammalian cells. T Lindahl, C Prigent, D E Barnes, A R Lehmann, M S Satoh, E Roberts, R A Nash, P Robins, G Daly, Cold Spring Harb Symp Quant Biol. 58Lindahl T, Prigent C, Barnes DE, Lehmann AR, Satoh MS, Rob- erts E, Nash RA, Robins P, Daly G. DNA joining in mammalian cells. Cold Spring Harb Symp Quant Biol. 1993;58:619-24.

Overview of base excision repair biochemistry. Y J Kim, D M Wilson, Curr Mol Pharmacol. 5Kim YJ, Wilson DM. Overview of base excision repair biochem- istry. Curr Mol Pharmacol. 2012;5:3-13.

Purification and characterization of Escherichia coli and human nucleotide excision repair enzyme systems. J T Reardon, A Sancar, Methods Enzymol. 408Reardon JT, Sancar A. Purification and characterization of Escher- ichia coli and human nucleotide excision repair enzyme systems. Methods Enzymol. 2006;408:189-213.

Mismatch repair and homeologous recombination. K C Tham, R Kanaar, J H Lebbink, DNA Repair (Amst). 38Tham KC, Kanaar R, Lebbink JH. Mismatch repair and homeolo- gous recombination. DNA Repair (Amst). 2016;38:75-83.

DNA mismatch repair: molecular mechanism, cancer, and ageing. P Hsieh, K Yamane, 10.1016/j.mad.2008.02.012Mech Ageing Dev. 129Hsieh P, Yamane K. DNA mismatch repair: molecular mechanism, cancer, and ageing. Mech Ageing Dev. 2008;129:391-407. https:// doi.org/10.1016/j.mad.2008.02.012.

Analysis of the human MutLα.MutSα complex. G Plotz, A Piiper, M Wormek, S Zeuzem, J Raedle, Biochem Biophys Res Commun. 3403Plotz G, Piiper A, Wormek M, Zeuzem S, Raedle J. Analysis of the human MutLα.MutSα complex. Biochem Biophys Res Com- mun. 2006;340(3):852-9.

Interaction of Ku protein and DNA-dependent protein kinase catalytic subunit with nucleic acids. W S Dynan, S Yoo, Nucleic Acids Res. 26Dynan WS, Yoo S. Interaction of Ku protein and DNA-dependent protein kinase catalytic subunit with nucleic acids. Nucleic Acids Res. 1998;26:1551-9.

The mechanism of double-strand DNA break repair by the nonhomologous DNA end-joining pathway. M R Lieber, 10.1146/annurev.biochem.052308.093131Annu Rev Biochem. 79Lieber MR. The mechanism of double-strand DNA break repair by the nonhomologous DNA end-joining pathway. Annu Rev Biochem. 2010;79:181-211. https://doi.org/10.1146/annurev. biochem.052308.093131.

The repair and signaling responses to DNA double-strand breaks. A A Goodarzi, P A Jeggo, 10.1016/B978-0-12-407676-1.00001-9Adv Genet. 82Goodarzi AA, Jeggo PA. The repair and signaling responses to DNA double-strand breaks. Adv Genet. 2013;82:1-45. https://doi. org/10.1016/B978-0-12-407676-1.00001-9.

Cancer predisposing mutations in BRCT domains. Di Masi, A Gullotta, F Cappadonna, V Leboffe, L Ascenzi, P , 10.1002/iub.472IUBMB Life. 63Di Masi A, Gullotta F, Cappadonna V, Leboffe L, Ascenzi P. Cancer predisposing mutations in BRCT domains. IUBMB Life. 2011;63:503-12. https://doi.org/10.1002/iub.472.

DNA damage signalling and repair inhibitors: the long-sought-after Achilles' heel of cancer. D Velic, A M Couturier, M T Ferreira, A Rodrigue, G G Poirier, F Fleury, J Y Masson, 10.3390/biom5043204Biomolecules. 54Velic D, Couturier AM, Ferreira MT, Rodrigue A, Poirier GG, Fleury F, Masson JY. DNA damage signalling and repair inhibi- tors: the long-sought-after Achilles' heel of cancer. Biomolecules. 2015;5(4):3204-59. https://doi.org/10.3390/biom5043204.

Targeting the DNA damage response in cancer. M J O&apos;connor, 10.1016/j.molcel.2015.10.040Mol Cell. 60O'Connor MJ. Targeting the DNA damage response in can- cer. Mol Cell. 2015;60:547-60. https://doi.org/10.1016/j. molcel.2015.10.040.

DNA repair pathways to regulate response to chemoradiotherapy in patients with locally advanced head and neck cancer. B Cirauqui, M Margelí, V Quiroga, A Quer, N Karachaliou, I Chaib, J L Ramírez, A Muñoz, C Pollán, I Planas, A Drozdowsky, R Rosell, Tumour Biol. 3710Cirauqui B, Margelí M, Quiroga V, Quer A, Karachaliou N, Chaib I, Ramírez JL, Muñoz A, Pollán C, Planas I, Drozdowsky A, Rosell R. DNA repair pathways to regulate response to chemora- diotherapy in patients with locally advanced head and neck cancer. Tumour Biol. 2016;37(10):13435-43.

The clinical value of aberrant epigenetic changes of DNA damage repair genes in human cancer. D Gao, J G Herman, M Guo, 10.18632/oncotarget.7949Oncotarget. 724Gao D, Herman JG, Guo M. The clinical value of aberrant epi- genetic changes of DNA damage repair genes in human cancer. Oncotarget. 2016;7(24):37331-46. https://doi.org/10.18632/ oncotarget.7949.

DNA repair: dynamic defenders against cancer and aging. J O Fuss, P K Cooper, 10.1371/journal.pbio.0040203PLoS Biol. 46203Fuss JO, Cooper PK. DNA repair: dynamic defenders against can- cer and aging. PLoS Biol. 2006;4(6):203. https://doi.org/10.1371/ journal.pbio.0040203.

APE1 overexpression in XRCC1-deficient cells complements the defective repair of oxidative single strand breaks but increases genomic instability. M Sossou, C Flohr-Beckhaus, I Schulz, F Daboussi, B Epe, J P Radicella, Nucleic Acids Res. 33Sossou M, Flohr-Beckhaus C, Schulz I, Daboussi F, Epe B, Radicella JP. APE1 overexpression in XRCC1-deficient cells complements the defective repair of oxidative single strand breaks but increases genomic instability. Nucleic Acids Res. 2005;33:298-306.

Genetic and expressional variations of APEX1 are associated with increased risk of head and neck cancer. I Mahjabeen, R M Baig, M Sabir, M A Kayani, 10.1093/mutage/ges074Mutagenesis. 282Mahjabeen I, Baig RM, Sabir M, Kayani MA. Genetic and expressional variations of APEX1 are associated with increased risk of head and neck cancer. Mutagenesis. 2013;28(2):213-8. https://doi.org/10.1093/mutage/ges074.

Detection of XRCC1 gene polymorphisms in Turkish head and neck squamous cell carcinoma patients: a comparative analysis with different populations. P Mutlu, M Mutlu, S Yalcin, G Unsoy, A Yaylaci, G Saylam, I Akin, H Korkmaz, U Gunduz, J BUON. 202Mutlu P, Mutlu M, Yalcin S, Unsoy G, Yaylaci A, Saylam G, Akin I, Korkmaz H, Gunduz U. Detection of XRCC1 gene poly- morphisms in Turkish head and neck squamous cell carcinoma patients: a comparative analysis with different populations. J BUON. 2015;20(2):540-7.

DNA repair gene XRCC1 polymorphisms and head and neck cancer risk: an updated meta-analysis including 16344 subjects. Y Lou, W J Peng, D S Cao, J Xie, H H Li, Z X Jiang, 10.1371/journal.pone.0074059PLoS ONE. 8974059Lou Y, Peng WJ, Cao DS, Xie J, Li HH, Jiang ZX. DNA repair gene XRCC1 polymorphisms and head and neck can- cer risk: an updated meta-analysis including 16344 subjects. PLoS ONE. 2013;8(9):e74059. https://doi.org/10.1371/journal. pone.0074059.

Combined effect of tobacco and DNA repair genes polymorphisms of XRCC1 and XRCC2 influence high risk of head and neck squamous cell carcinoma in northeast Indian population. J H Choudhury, B Choudhury, S Kundu, S K Ghosh, 10.1007/s12032-014-0067-8Med Oncol. 31867Choudhury JH, Choudhury B, Kundu S, Ghosh SK. Combined effect of tobacco and DNA repair genes polymorphisms of XRCC1 and XRCC2 influence high risk of head and neck squa- mous cell carcinoma in northeast Indian population. Med Oncol. 2014;31(8):67. https://doi.org/10.1007/s12032-014-0067-8.

DNA repair genes XPC, XPD, XRCC1, and XRCC3 are associated with risk and survival of squamous cell carcinoma of the head and neck. L Farnebo, A Stjernström, M Fredrikson, Ansell A Garvin, S Thunell, L K , 10.1016/j.dnarep.2015.05.003DNA Repair (Amst). 31Farnebo L, Stjernström A, Fredrikson M, Ansell A, Garvin S, Thunell LK. DNA repair genes XPC, XPD, XRCC1, and XRCC3 are associated with risk and survival of squamous cell carcinoma of the head and neck. DNA Repair (Amst). 2015;31:64-72. https://doi.org/10.1016/j.dnarep.2015.05.003.

XPD, APE1, and MUTYH polymorphisms increase head and neck cancer risk: effect of gene-gene and gene-environment interactions. S Das, A Bhowmik, A Bhattacharjee, B Choudhury, M Naiding, A K Laskar, S K Ghosh, Y Choudhury, 10.1007/s13277-015-3472-5Tumour Biol. 3610Das S, Bhowmik A, Bhattacharjee A, Choudhury B, Naid- ing M, Laskar AK, Ghosh SK, Choudhury Y. XPD, APE1, and MUTYH polymorphisms increase head and neck cancer risk: effect of gene-gene and gene-environment interactions. Tumour Biol. 2015;36(10):7569-79. https://doi.org/10.1007/ s13277-015-3472-5.

Correlation of mismatch repair genes immunohistochemistry and microsatellite instability status in HNPCC-associated tumours. A Ruszkiewicz, G Bennett, J Moore, J Manavis, B Rudzki, L Shen, G Suthers, Pathology. 346Ruszkiewicz A, Bennett G, Moore J, Manavis J, Rudzki B, Shen L, Suthers G. Correlation of mismatch repair genes immunohis- tochemistry and microsatellite instability status in HNPCC-asso- ciated tumours. Pathology. 2002;34(6):541-7.

The utility of immunohistochemical detection of DNA mismatch repair gene proteins. J Shia, N A Ellis, D S Klimstra, Virchows Arch. 4455Shia J, Ellis NA, Klimstra DS. The utility of immunohistochemi- cal detection of DNA mismatch repair gene proteins. Virchows Arch. 2004;445(5):431-41.

Low expression of MSH2 DNA repair protein is associated with poor prognosis in head and neck squamous cell carcinoma. C S Pereira, M V Oliveira, L O Barros, G A Bandeira, S H Santos, J R Basile, A L Guimarães, De Paula, A M , 10.1590/1679-775720130206J Appl Oral Sci. 215Pereira CS, Oliveira MV, Barros LO, Bandeira GA, Santos SH, Basile JR, Guimarães AL, De Paula AM. Low expression of MSH2 DNA repair protein is associated with poor progno- sis in head and neck squamous cell carcinoma. J Appl Oral Sci. 2013;21(5):416-21. https://doi.org/10.1590/1679-775720130206.

Association between genetic polymorphisms in DNA mismatch repair-related genes with risk and prognosis of head and neck squamous cell carcinoma. G A Nogueira, G J Lourenço, C B Oliveira, F A Marson, L Lopes-Aguiar, E F Costa, T R Lima, V T Liutti, F Leal, V C Santos, J A Rinck-Junior, C S Lima, 10.1002/ijc.29435Int J Cancer. 1374Nogueira GA, Lourenço GJ, Oliveira CB, Marson FA, Lopes- Aguiar L, Costa EF, Lima TR, Liutti VT, Leal F, Santos VC, Rinck-Junior JA, Lima CS. Association between genetic poly- morphisms in DNA mismatch repair-related genes with risk and prognosis of head and neck squamous cell carcinoma. Int J Can- cer. 2015;137(4):810-8. https://doi.org/10.1002/ijc.29435.

Single nucleotide polymorphism in hMLH1 promoter and risk of tobacco-related oral carcinoma in high-risk Asian Indians. R Jha, P Gaur, S C Sharma, S N Das, 10.1016/j.gene.2013.05.014Gene. 5262Jha R, Gaur P, Sharma SC, Das SN. Single nucleotide polymor- phism in hMLH1 promoter and risk of tobacco-related oral car- cinoma in high-risk Asian Indians. Gene. 2013;526(2):223-7. https://doi.org/10.1016/j.gene.2013.05.014.

The DNA-damage response in human biology and disease. S P Jackson, J Bartek, 10.1038/nature08467Nature. 461Jackson SP, Bartek J. The DNA-damage response in human biology and disease. Nature. 2009;461:1071-8. https://doi. org/10.1038/nature08467.

Sensing and repairing DNA double-strand breaks. Carcinogenesis. S Jackson, 23Jackson S. Sensing and repairing DNA double-strand breaks. Car- cinogenesis. 2002;23:687-96.

Genetic variation in a DNA double strand break repair gene in saudi population: a comparative study with worldwide ethnic groups. M Y Areeshi, Asian Pac J Cancer Prev. 14Areeshi MY. Genetic variation in a DNA double strand break repair gene in saudi population: a comparative study with world- wide ethnic groups. Asian Pac J Cancer Prev. 2013;14:7091-4.

Germline mutations in breast and ovarian cancer pedigrees establish RAD51C as a human cancer susceptibility gene. A Meindl, H Hellebrand, C Wiek, V Erven, B Wappenschmidt, D Niederacher, M Freund, P Lichtner, L Hartmann, H Schaal, J Ramser, E Honisch, C Kubisch, H E Wichmann, K Kast, H Deissler, C Engel, B Müller-Myhsok, K Neveling, M Kiechle, C G Mathew, D Schindler, R K Schmutzler, H Hanenberg, 10.1038/ng.569Nat Genet. 425Meindl A, Hellebrand H, Wiek C, Erven V, Wappenschmidt B, Niederacher D, Freund M, Lichtner P, Hartmann L, Schaal H, Ramser J, Honisch E, Kubisch C, Wichmann HE, Kast K, Deissler H, Engel C, Müller-Myhsok B, Neveling K, Kiechle M, Mathew CG, Schindler D, Schmutzler RK, Hanenberg H. Germline muta- tions in breast and ovarian cancer pedigrees establish RAD51C as a human cancer susceptibility gene. Nat Genet. 2010;42(5):410-4. https://doi.org/10.1038/ng.569.

Human Rad51 protein promotes ATP-dependent homologous pairing and strand transfer reactions in vitro. P Baumann, F E Benson, S C West, Cell. 874Baumann P, Benson FE, West SC. Human Rad51 protein promotes ATP-dependent homologous pairing and strand transfer reactions in vitro. Cell. 1996;87(4):757-66.

The RAD51 gene family, genetic instability and cancer. J Thacker, Cancer Lett. 2192Thacker J. The RAD51 gene family, genetic instability and cancer. Cancer Lett. 2005;219(2):125-35.

RAD51C-a new human cancer susceptibility gene for sporadic squamous cell carcinoma of the head and neck. K Scheckenbach, S E Baldus, V Balz, M Freund, P Pakropa, C Sproll, K L Schäfer, M Wagenmann, J Schipper, H Hanenberg, 10.1016/j.oraloncology.2013.11.007HNSCCScheckenbach K, Baldus SE, Balz V, Freund M, Pakropa P, Sproll C, Schäfer KL, Wagenmann M, Schipper J, Hanenberg H. RAD51C-a new human cancer susceptibility gene for spo- radic squamous cell carcinoma of the head and neck (HNSCC).

. 10.1016/j.oraloncology.2013.11.007Oral Oncol. 503Oral Oncol. 2014;50(3):196-9. https://doi.org/10.1016/j. oraloncology.2013.11.007.

Association of RAD 51 135 G/C, 172 G/T and XRCC3 Thr241Met gene polymorphisms with increased risk of head andneck cancer. M A Kayani, S Khan, R M Baig, I Mahjabeen, Asian Pac J Cancer Prev. 1523Kayani MA, Khan S, Baig RM, Mahjabeen I. Association of RAD 51 135 G/C, 172 G/T and XRCC3 Thr241Met gene polymor- phisms with increased risk of head andneck cancer. Asian Pac J Cancer Prev. 2014;15(23):10457-62.

The MRN protein complex genes: MRE11 and RAD50 and susceptibility to head and neck cancers. I Ziółkowska-Suchanek, M Mosor, M Wierzbicka, M Rydzanicz, M Baranowska, J Nowak, 10.1186/1476-4598-12-113Molecular Cancer. 12113Ziółkowska-Suchanek I, Mosor M, Wierzbicka M, Rydzan- icz M, Baranowska M, Nowak J. The MRN protein complex genes: MRE11 and RAD50 and susceptibility to head and neck cancers. Molecular Cancer. 2013;12:113. https://doi. org/10.1186/1476-4598-12-113.

Aberrations of the MRE11-RAD50-NBS1 DNA damage sensor complex in human breast cancer: MRE11 as a candidate familial cancerpredisposing gene. J Bartkova, J Tommiska, L Oplustilova, K Aaltonen, A Tamminen, T Heikkinen, M Mistrik, K Aittomäki, C Blomqvist, P Heikkilä, J Lukas, H Nevanlinna, J Bartek, 10.1016/j.molonc.2008.09.007Mol Oncol. 2Bartkova J, Tommiska J, Oplustilova L, Aaltonen K, Tamminen A, Heikkinen T, Mistrik M, Aittomäki K, Blomqvist C, Heik- kilä P, Lukas J, Nevanlinna H, Bartek J. Aberrations of the MRE11-RAD50-NBS1 DNA damage sensor complex in human breast cancer: MRE11 as a candidate familial cancerpredispos- ing gene. Mol Oncol. 2008;2:296-316. https://doi.org/10.1016/j. molonc.2008.09.007.

P53 gene mutations: case study of a clinical marker for solid tumors. M C Liu, E P Gelmann, Semin Oncol. 293Liu MC, Gelmann EP. P53 gene mutations: case study of a clinical marker for solid tumors. Semin Oncol. 2002;29(3):246-57.

Head and neck squamous cell carcinoma in FAMMM syndrome. V Vinarsky, R L Fine, A Assaad, Y Qian, J A Chabot, G H Su, H Frucht, Head Neck. 3111Vinarsky V, Fine RL, Assaad A, Qian Y, Chabot JA, Su GH, Frucht H. Head and neck squamous cell carcinoma in FAMMM syndrome. Head Neck. 2009;31(11):1524-7.

Ten genes for inherited breast cancer. T Walsh, M C King, 10.1002/hed.21050Cancer Cell. 112Walsh T, King MC. Ten genes for inherited breast cancer. Cancer Cell. 2007;11(2):103-5. https://doi.org/10.1002/hed.21050.