# An Overview of Genetic Transformation of Soybean

CorpusID: 16096302
 
tags: #Biology, #Agricultural_And_Food_Sciences

URL: [https://www.semanticscholar.org/paper/5cc8291d1cc070656ca1861350c50b610c2c6d10](https://www.semanticscholar.org/paper/5cc8291d1cc070656ca1861350c50b610c2c6d10)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

An Overview of Genetic Transformation of Soybean


Hyeyoung Lee 
So-Yon Park 
Zhanyuan J Zhang 
An Overview of Genetic Transformation of Soybean
10.5772/51076Chapter 23 Additional information is available at the end of the chapter


## Introduction

Soybean (Glycine max (L.) Merrill) is a model legume crop, widely grown in the world for human consumption or animal fodder. Moreover, soybeans have gained worldwide research interest in many public laboratories and industrial sectors. Soybean seeds contain protein, oil, carbohydrates, dietary fibers, vitamins, and minerals. For the last few decades the majority of research laboratories have been investigating genetic traits to improve the yield of protein or oil in soybean seeds through genetic engineering, thereby achieving improved quantity and quality of soybean seeds. Until now, most of the transformation experiments have implemented a single functional gene not multiple genes. Those agronomically and economically important traits affect the enhancement of grain quantity and quality [1]. However, the majority of agronomic and genetic traits such as complex metabolic, biological, and pharmaceutical pathways are polygenetic traits and are produced in a complex pathway. Therefore, those traits are encoded and regulated by a number of genes. In an attempt to study and manipulate those pathways, the transfer of multigene or large inserts into plants have been developed by multigene engineering technology and have also been involved in metabolic engineering. Several examples of multigene or large insert transfers have been reported such as the application of carotenogenic genes in rice, canola, and maize [2][3][4], and of polyunsaturated fatty acid and vitamin E genes in soybean and Arabidopsis [5][6][7]. Therefore, reliable systems for transforming large DNA fragments into plants make it feasible to introduce a natural gene cluster or a series of previously unlinked foreign genes into a single locus.

Over the last two decades, the transfer of DNA into plant cells has been achieved by using several methods. In soybeans, the most frequently employed plant genetic engineering methods are Agrobacterium-mediated transformation and particle bombardment. Both systems have successfully been used in genetic transformation of soybean. Since the initial reports of fertile transgenic soybean production [8][9], various efforts have been made to improve the transformation efficiency and to produce transgenic soybean. Particularly, the preferred and reproducible transformation is the use of the cotyledonary node as a plant material, which is based on Agrobacterium-mediated gene transfer [10][11][12]. Nevertheless, new methods have been developed for more efficient soybean transformation. There still remain, however, many challenges for genotype-and tissue-specific independent transformation of soybean. This review provides an overview of historical efforts in developing and advancing soybean regeneration and transformation systems. In addition, recent advances and challenges in soybean transformation are discussed.


## Different approaches for soybean transformation

In soybean transformation, two major methods are now widely utilized: Agrobacterium-mediated transformation of different explant tissues and particle bombardment. The Agrobacterium-mediated method, as a simple protocol, does not require any specific or expensive equipment. Moreover, this method usually produces single or low copy numbers of insertions with relatively rare rearrangement [13]. On the other hand, bombardment technique directly introduces desired genes into the target plant cell with small tungsten or gold particles [9]. The success of this approach critically depends upon the ability of the target tissue to proliferate as well as proper pre-cultures to make a target plant.


### Cotyledonary-node-based transformation

The routine regeneration system was first reported by using the mature cotyledonary-node [14]. The multiple adventitious buds and shoots from explant tissues were proliferated and regenerated on culture media containing cytokinin by organogenesis. The transgenic soybean plants have been successfully and reproducibly produced using mature or immature cotyledon explants via Agrobacterium-mediated transformation. Hinchee et al. [8] for the first time reported the production of fertile transgenic soybean plants using mature cotyledonary-node by Agrobacterium-mediated transformation, but transformation efficiency was very low. The system employed the neomycin phosphotransferase II (NPT II) gene as a selectable marker and combined kanamycin as a selective agent. However, this selection was addressed with a problem of regeneration of non-transgenic or chimeric shoots at the shoot formation stage. Moreover, the system was highly genotype-dependent. To overcome the high genotypedependency and high chimerism problems by the NPT II selection and develop a new selection system for soybean transformation, Zhang et al. [10] developed the selection system employing herbicide bialaphos resistance (bar) gene as a selectable marker coupled with glufosinate as a selective agent. This system enabled to transform many soybean genotypes with stable transgene inheritance, albeit transformation efficiency remained to be improved. Meanwhile, to solve the escape problem caused by kanamycin selection, Clemete et al. [15] deployed the herbicide glyphosate as a selective agent, leading to high stringent selection and good transgene inheritance. It was discovered later that addition of various thiol compounds in the co-cultivation medium significantly increased the transformation efficiency [11,[16][17]. These thiol compounds, as antioxidants, reduce the oxidative burst that caused tissue browning or necrosis and also promote organogenesis and shoot growth from buds [18].

Recently, an alternative cotyledonary explant derived from mature soybean seed for Agrobacterium transformation has been reported by Paz et al. [19]. The term half-seed explants were used as an experiment material and fertile transgenic plants were attained.

In fact, several laboratories have contributed to enhanced soybean transformation using a cotyledonary-node explant. To overcome the low transfer of Agrobacterium into plant cell, the infection media were first amended with the phenolic compound, 4'-Hydroxy-3',5'-dimethoxyacetophenone (acetosyringone), to induce expression of the virulence (Vir) genes [20][21]. To increase the infection sites, Trick and Finer [22] evaluated cotyledonary node transformation efficiency using a developed sonication assisted Agrobacterium-mediated transformation (SAAT) protocol. Although this treatment was not able to obtain fertile transgenic plants, the increase of Agrobacterium transfer was shown. Olhoft et al. [16][17] discovered that thiol compounds enhanced Agrobacterium infection in soybean. At the same time, however, these compounds caused counter-selection effect when glufosinate was used as a selective agent under previously published selection conditions. To solve this problem, Olhoft et al. [11] developed Hygromycin phosphotransferase (HPT II) selection system using hygromycin B as a selective agent. This has led to a substantial increase in transformation frequency. Transformation efficiency with thiol compounds was increased 5-fold by using refined glufosinate selection [12].

Since the transformation process by use of kanamycin or hygromycin B as selection agent has been proven to be genotype-dependent, the most widely used selection system has been the combination of bar gene with the herbicide phosphinothricin (glufosinate) [10,12]. In this selection system, the concentration of agent glufosinate greatly affects the transformation frequency [12], so the appropriate selection schemes can be varied among genotypes, seed vigor and other in vitro culture conditions.  


### Immature embryos-based transformation

The regeneration using immature embryos via somatic embryogenesis was first reported by Christianson et al. [23]. The immature embryos excised from soybean pods were suspended on semi-solid media or liquid media containing high concentration of auxin, 2,4-Dichlorophenoxyacetic acid (2,4-D), and the whole plantlets were recovered [24][25]. After immature embryos were developed as an alternative plant material, transgenic plants were first obtained from this explant tissue via particle bombardment [26]. This system has been exclusively used to produce transgenic soybean such as glyphosate tolerant, hygromycin resistance, and Bacillus thuringiensis (BT) transgenic soybean [27][28][29]. As the formation of proliferative embryogenic tissue depends on genotype, the use of immature embryos for transformation has been limited to few genotypes cultivars including "Jack" and "Williams 82."

The use of particle bombardment with immature embryos tends to be highly variable, and multiple copies of the introduced DNAs are commons. Moreover, this problem has compounded with aged embryogenic suspension cultures from which a high percentage of regenerated plants lost their fertility [29]. In spite of this limitation, the embryogenic cultures have several advantages, one of which is its relatively high transformation efficiency and less chimeric plants recovered.


### Embryogenic shoot tips-based transformation

The embryonic shoot tip explant is another source of explant which has been used for soybean transformation. McCabe et al., [30] first reported the stable transformation using meristemic cell, shoot apex, by particle acceleration. The shoot derived from these meristems via organogenesis has been produced to form multiple shoots prior to mature plants. However, all of the primary transgenic plants were chimeric. Martinell et al., [31] described the successful method using meristemic shoot tip from germinated seedling by Agrobacteriummediated transformation. This system has provided rapid and efficient soybean transformation. Liu et al. [32] also reported the regeneration system using embryonic shoot tips by shoot organogenesis. The explants have been shown the high regeneration and the transformation efficiencies using Agrobacterium-mediated with up to 15.8%.


### Immature cotyledonary-nodes

The regeneration capacity of immature cotyledonary-node was found by Parrott et al [33]. Based on this regeneration system, first transgenic soybean plants have been developed by Agrobacterium tumefaciens [34]. This system was tested using two different Agrobacterium strains, LBA4404 and EHA101 and deploying kanamycin selection. The system utilized auxin 1-Naphthaleneacetic acid (NAA) for plant regeneration. Although these systems allowed development of transgenic plants from the explants, no fertile transgenic plants were recovered. Recently, Ko et al [35] described the efficient transformation system using immature cotyledonary-nodes by Agrobacterium-mediated transformation, but transformation efficiency was still very low.


### Hypocotyl based transformation

Another type of explant tissue, hypocotyl, was also investigated with 13 different soybean genotypes. Most of the genotypes initiated shoots from this type of explant [36]. This method was reported to be genotype-independent regeneration protocol via organogenesis and utilized the acropetal end of a hypocotyl section from a 7-day old seedling. Despite inducing adventitious shoots from the explant, most recovered shoot did not matured in the soil. Wang et al [37] reported successful production of fertile transgenic plants using hypocotylbased Agrobacterium-mediated transformation. To improve the transformation system, two different chemicals, cytokinin hormone 6-Benzylaminopurine (BAP) and silver nitrate, were added to the shoot formation media. In spite of the term "hypocotyl" used in the above transformation system, the true tissues responsible for regeneration are actually the preexisting meristem tissues located at the nodal area of the cotyledon, essentially the same source of tissue as cotyledonary-nodes [17] except that cotyledons were removed [45,46].


### Leaf tissue-based transformation

The reproducible regeneration methods for whole plants from primary leaf tissue or epicotyls were first reported by Wright et al [38]. The multiple shoots from those explants were continually initiated and proliferated with cytokinin BAP hormone. Rajasckaren et al [39] described regeneration of several varieties of soybean by embryogenesis from epicotyls and primary leaf tissues, thereby inducing fertile plants from those explants. Kan et al [40] first tested transformation efficiency using epicotyls and leaf tissues by Agrobacterium tumefaciens.

To find out proper transformation condition for those explants, they investigated different Agrobacterium strains, EHA101 and LBA4404, but also different treatments on inoculation stage, sucrose and mannose.


## Agrobacterium-mediated transformation of soybean


### Agrobacterium-mediated transformation mechanism

Agrobacterium is a unique organism to generate transgenic plants and in natural conditions [41]. It allows introduction of a single stranded copy of the bacterial transferred DNA (T-DNA) into a host cell and integration of the genomic DNA of interest, resulting in genetic manipulation of the host. Since the development of disarmed tumour-inducing (Ti) plasmid [42][43], Agrobacterium has been used to transform various major crops for genetic modification [44][45][46].

Agrobacterium recognizes wounded host plant cells which produce penolic compounds such as acetosyringone as inducers of vir gene expression [47], and attach to the plant cells to export the T-DNA after virulence (Vir) protein activation. Acetosyringone is now routinely used for improving transformation efficiency. After vir gene activation, a single stranded T-DNA copy (T-strand) is transferred into the plant by type IV secretion system (T4SS) which is related to VirB complex [48]. The VirB complex is composed of at least 12 proteins (VirB1-11 and VirD4) which form a multisubunit envelope-spanning structure [49]. Various

Agrobacterium proteins, such as VirD2-T-DNA, VirE2, VirE3, VirF, and VirD5, pass though VirB complex to transfer into plant cells [50][51]. VirE2 and VirD2 interact with cytosolic T-DNA in the plant cells and form a complex which is later imported into the nucleus when it is bound to VIP1 plant protein [52][53][54][55]. Recently, Gelvin et al., hypothesized that T-complex (T-DNA, VirE2, VirD2 and VIP1) is imported into the nucleus through actin cytoskeleton and thus myosin may be involved in Agrobacterium-mediated transformation [56]. However, the specific mechanism of T-DNA movement through myosin is still unknown.

The T-complex is imported into the nucleus by the phosphorylation of VirE2 Interacting Protein 1 (VIP1), induced by mitogen-activated protein kinase (MAPK), such as MPK3 [55].

After T-complex is imported into the host nucleus, VirE2 and VIP1 need to be degraded before T-DNA integration by a subunit of the SCF (SKP-CUL1-F-box protein) ubiquitin E3 ligase complex. Not only Agrobacterium protein VirF but also protein VBF can mark VIP1 protein for the degradation. Furthermore, binding of VIP1-binding F-box (VBF) to T-complex can induce the degradation of VIP1 and VirE2 by the 26S proteosome, and at the end Tstrand is integrated into plant genomic DNA and expressed in the host plants [57][58].


### History of Agrobacterium-mediated soybean transformation research

Among various transformation technologies, Agrobacterium-mediated transformation method has shown to be effective for the production of transgenic soybeans because of straightforward methodology, familiarity to researchers, minimal equipment cost, and reliable insertion of a single transgene or a low copy number [13]. Till now, a number of reports have been published related to the optimum condition to achieve a high yield of soybean transformation; such as Agrobacterium inoculation conditions, regeneration media components, etc. For Agrobacterium-mediated transformation methods, the susceptibility of soybean to Agrobacterium and various Agrobacterium strains have been tested to improve the transformation efficiency (Table 1). Also, Agrobacterium strains and growth conditions which affect the soybean transformation efficiency have been published [8,[59][60][61][62]. After Pederson et al., [46] and Owens et al., [59] showed the susceptibility of certain soybean genotypes against tumor induction, Agrobacterium biology study has been advanced to enhance transformation efficiency. In addition to Agrobacterium biology study, chemical contents for inoculation have been studied such as varying acetosyringone and syringaldehyde concentrations [63]. For high inoculation efficiency, Mauro et al., [64] tested various Agrobacterium biotypes (nopaline, agropine and octopine) to identify the most effective Agrobacterium biotype for soybean transformation.

After Hinchee et al., [8] developed Agrobacterium-mediated soybean transformation methods, many Agrobacterium strains have been tested and employed, such as EHA101, EHA105, LBA4404 and AGL1. Parrott et al., [33] showed that EHA101 was highly potent to transform immature soybean cotyledons, especially PI283332, and had higher recovery of transformed plants over LBA4404. Dang and Wei [65] tested transformation efficiency using embryonic tips instead of cotyledonary explants and somatic embryos, and when embryogenic tips were infected for 20 hours, hypervirulent strain KYRT1 showed increased efficiency over EHA105 and LBA4404.

Recently, A. tumefaciens KAT23 (AT96-6) which has an ability to efficiently transfer the T-DNA into soybean, was isolated from peach root. After 20 stains were confirmed by common bean and soybean transformation, Yukawa et al. [66] tested their potential availability as legume super virulent A. tumefaciens in various soybean cultivars (Peking, Suzuyutaka, Fayette, Enrei, Mikawashima, WaseMidori, Jack, Leculus, Morocco, Serena, Kentucky Wonder and Minidoka). Without modifying vectors or vir function, they showed that KAT23 (AT96-6) has a high potential to function as a common strain to increase soybean transformation efficiency. Therefore, this study identified a novel soybean super virulent A. tumefaciens strain which transferred not only the T-DNA of the Ti-plasmid but also introduced T-DNA of the binary vector efficiently. These results indicate that KAT23 (AT96-6) has the ability to transform soybeans at high efficiency.

There has been a significant improvement in soybean transformation over the past two decades. However, the efficiency of soybean transformation is not great enough for practical needs and shows high variation. Thus, considering the potential application of soybean transformation, the importance of Agrobacterium can't be over-emphasized. 


## Strain of


## New directions of soybean genetic engineering, skills and vectors

To date, the Agrobacterium-and biolistic-mediated transformation methods remain the very successful methods in soybean transformation, whereas other available transformation technologies have not been practical in soybean, which include electroporation-mediated transformation [72], PEG/liposome-mediated transformation [73], silicon carbide-mediated transformation [74], microinjection [75] and chloroplast-mediated transformation [76]. Of these two, Agrobacterium-mediated transformation has become more adapted in public laboratories worldwide. On the other hand, there are unintended insertions such as unwanted antibiotic markers and promoters, which can be inserted during transformation. This problem has raised potential biosafety issues related to environmental concerns and human health risks. To overcome these potential risks, methods of developing marker free transgenic plants have been developed, such as cotransformation [77], transposon-mediated transformation [78] and site-specific recombination [79].

Among the various methods, co-transformation system is one of the most commonly used methods to produce marker free transgenic plants. In co-transformation systems, a marker gene and genes of interest are placed on separate DNA molecules and introduced into plant genomes. Then, the non-selectable genes segregate from the marker gene in the progeny generations. Most strains of A. tumefaciens have the ability to contain more than one T-DNA, and crown gall tumors were often co-transformed with multiple T-DNAs [42]. As a result, there are two possibilities; Multiple T-DNAs were delivered into plant cells either from a mixture of strains ('mixture methods') or from a single strain ('single-strain methods'). Depicker et al. [80] described that a single strain method was higher in efficiency than a mixture method. For a single-strain method of co-transformation, Kamori et al., [77] tested cotransformation method to develop a suitable superbinary vector system. Using the unique plasmids which carried two T-DNA segments marker free rice and tobacco were produced and evaluated. LBA4404, a derivative of an octopine strain, were used for these co-transformation methods and they hypothesized that LBA4404 may be an important factor contributing to the high frequency of unlinked loci.

To improve plant genetic traits, many soybean research labs have developed tools for soybean functional genomics, such as several libraries containing large inserts of bacterial artificial chromosome (BAC) and plant transformation competent binary plasmids clone (BIBAC) (81). In functional genomic research, bacterial artificial chromosome (BAC) is a single copy artificial chromosome vector and is based on the E. coli fertility (F-factor) plasmid. They are not only stable in host cell, but also are used for large scale gene cloning and discovery [82]. However, some BAC libraries that are desirable for functional genomics are often not amenable for transformation directly into plants because of their large subclones. Therefore, binary bacterial artificial chromosome (BIBAC) libraries have been developed for Agrobacterium-mediated plant transformation and gene functional complementation. The BI-BAC library is based on BAC vector and has both an F-factor plasmid for replication origin of E.coli and an Ri plasmid for replication origin of Agrobacterium rhizogenes. The vector also has a sacB gene as a positive selection for E. coli and a selectable marker gene for plant. Since BIBAC vectors were reported, these vectors have been used for plant transformation in some model plant species including tobacco, canola, tomato, and rice [83][84][85][86]. Although the transformation efficiency was very low, the BIBAC vectors have been successfully employed to transfer large inserts into those crops as a single locus via Agrobacterium-mediated transformation. The introduced T-DNA was stably maintained and inherited through several generations and no gene silencing was observed [85]. However, soybean transformation us-ing a BIBAC vector has not been achieved to date. Moreover, plant transformation with DNA fragments below 20 kb is routine whereas the stable plant transformation with DNA fragments larger than 50 kb is challenging.


## Zinc finger nucleases (ZFNs) and transcription activator-like effectors (TALEs)

Although many methods have been developed, soybean is considered a recalcitrant plant to transform compared to Arabidopsis and rice. Since full genome sequencing data has been rapidly updated in soybean, soybean transformation technology is becoming an essential approach for genomic research. For the phenotypic analysis of genes, knock-out or gene-silencing plants are used to study gene function. However in soybean, making bulk knock-out mutants through conventional mutagenesis approaches is not immediately feasible because of low transformation efficiency. Thus, development of innovative gene targeting methods is necessary to make knock-out plants in soybean.

Zinc finger nucleases (ZFNs) and meganucleases cut specific DNA target sequences in vivo and thus are powerful tools for genome modification. In particular zinc finger domain, which predominantly recognize nucleotide triplets, have been widely used in this research. Importantly, ZFNs modification has been reported in soybean (Glycine max), maize (Zea mays), tobacco (Nicotiana tabacum) and Arabidopsis [87][88][89][90]. Unfortunately, Ramirez et al. [91] found a major disadvantage of ZFNs; they observed that the modular assembly method of engineering zinc-finger arrays has a higher failure rate than previously reported.

To overcome the ZFNs's weakness, in late 2009, a novel DNA binding domain was identified which was a member of the large transcription activator-like (TAL) effector family [92][93]. Transcription activator-like effectors (TALEs) are produced by plant pathogens in the genus Xanthomonas as virulence factors and TAL effector-mediated gene induction leads to plant developmental changes [94][95]. The type III secretion system is used by Xanthomonas to introduce virulence factors into plant cells [96]. Once inside the plant cell, transcription activator-like (TAL) effectors (TALEs) enter the nucleus, bind effector-specific DNA sequences, and transcriptionally activate gene expression [97][98]. For genomic engineering, two methods of TALEs were developed: TALE nucleases (or TALENs) and TALE transcription factors (or TALE-TFs). Both TALENs and TALE-TFs contain as many as 30 tandem repeats of a 33-to 35-amino-acid-sequence motif ( Figure 2). The amino acids in positions 12 and 13 in each 33-to 35-amino-acid-sequence motif have the repeat variable di-residue (RVD). Using this specific ability, two pair (left and right TALENs) of repeats with different RVDs are designed by PCR and bound in the target DNA sequence [92][93]. Fok1 combined with TALE nucleases (or TALENs) make double-strand breaks (DSBs) at specific locations in the genome. These DSBs are repaired by homologous recombination (HR) or non-homologous end-joining (NHEJ) pathways. During DSBs repair, errors in genome via insertion, deletion, or chromosomal rearrangement could be induced by HR and NHEJ ( Figure 2). Unlike TALENs, TALE-TFs require only a single TALE construct for activity induction when com-bined with VP64 activator (derived from the herpex simplex virus activation domain). VP64 binds with RNA polymerase and causes transcriptional activation of the gene of interest [99]. For making transgenic plants, the TALEs technique can be combined with the Agrobacterium-mediated transformation method and it is assumed that gene targeting knock-out will be applied in soybean research. 


## Abbreviation

Acetosyringone: 4'-Hydroxy-3',5'-dimethoxyacetophenone 

## Figure 1 .
1Scheme for genetic transformation of soybean (Glycine max (L.) Merrill) cotyledonary nodes.

## Figure 2 .
2Summary of Transcription activator-like effectors (TALEs) nuclease.


A.tumefacients 
Soybean genotype 
Selection 
Reference 
Marker Agent 

A208 
Peking, Maple Prest 
npt II 
kanamycin 
(8) 

AGL1 
Bert 
bar 
phosphinothricin 
(67) 

EHA101 
Williams 82 
bar 
glufosinate 
(12) 

EHA101 
Williams, Williams 79, Peking, Thorne bar 
glufosinate or 

bialaphos 

(68) 

EHA101 
Thorne, Williams, Williams 79, Williams 

82 

bar 
glufosinate 
(19) 

EHA105 
AC Colibri 
npt II 
kanamycin 
(69) 

EHA105 
Hefeng 25, Dongnong 42, Heinong 37, 

Jilin 39, Jiyu 58 

hpt 
hygromycin 
(70) 

EHA105 
A3237 
bar 
glufosinate 
(10) 

LBA4404 
Jungery 
bar 
phosphinothricin 
(71) 

LBA4404, EHA105 Bert 
hpt 
hygromycin 
(11) 

Table 1. Summary of cotyledonary-node transformation system. 




BAC: bacterial artificial chromosome DSBs: double-strand breaks HPT II: Hygromycin phosphotransferase A Comprehensive Survey of International Soybean Research -Genetics, Physiology, Agronomy and Nitrogen Relationships HR: homologous recombination NAA: Naphthaleneacetic acid NHEJ: non-homologous end-joining NPT II: neomycin phosphotransferase II RVD: repeat variable di-residue Ti plasmid: tumour-inducing (Ti) plasmid T4SS: type IV secretion system TALEs: transcription activator-like effectors VBF: VIP1-binding F-box VIP1: VirE2 Interacting Protein 1 Vir protein: virulence protein ZFNs: Zinc finger nucleases Author details Hyeyoung Lee, So-Yon Park and Zhanyuan J. Zhang * *Address all correspondence to: zhangzh@missouri.edu Plant Transformation Core Facility, University of Missouri, Columbia, USAbar: bialaphos resistance 

BAP: 6-Benzylaminopurine 

2,4-D: 2,4-Dichlorophenoxyacetic acid 

BIBAC: binary bacterial artificial chromosome 

BT: Bacillus thuringiensis 



Gene Discovery and Product Development for Grain Quality Traits. B Mazur, E Krebbers, S Tingey, Science. 2855426Mazur, B., Krebbers, E., & Tingey, S. (1999). Gene Discovery and Product Develop- ment for Grain Quality Traits. Science, 285(5426), 372-375.

Engineering the provitamin A (beta-carotene) biosynthetic pathway into (carotenoid-free) rice endosperm. (Translated from eng). X Ye, Science. 5451in engYe, X., et al. (2000). Engineering the provitamin A (beta-carotene) biosynthetic path- way into (carotenoid-free) rice endosperm. (Translated from eng). Science, 287(5451), 303-305, (in eng).

Improving the nutritional value of Golden Rice through increased pro-vitamin A content. J A Paine, Nat Biotechnol. 234in engPaine, J. A., et al. (2005). Improving the nutritional value of Golden Rice through in- creased pro-vitamin A content. (Translated from eng). Nat Biotechnol, 23(4), 482-487, (in eng).

Pathway engineering of Brassica napus seeds using multiple key enzyme genes involved in ketocarotenoid formation. (Translated from eng). M Fujisawa, J Exp Bot. 604in engFujisawa, M., et al. (2009). Pathway engineering of Brassica napus seeds using multi- ple key enzyme genes involved in ketocarotenoid formation. (Translated from eng). J Exp Bot, 60(4), 1319-1332, (in eng).

. 10.5772/51076An Overview of Genetic Transformation of Soybean. An Overview of Genetic Transformation of Soybean http://dx.doi.org/10.5772/51076

. A J Kinney, COMPANY EIDPDNA WO/2004/071467Kinney, A. J., et al. (2004). COMPANY EIDPDNA WO/2004/071467.

Production of very long chain polyunsaturated omega-3 and omega-6 fatty acids in plants. B Qi, Nat Biotechnol. 226in engQi, B., et al. (2004). Production of very long chain polyunsaturated omega-3 and ome- ga-6 fatty acids in plants. (Translated from eng). Nat Biotechnol, 22(6), 739-745, (in eng).

Metabolically engineered oilseed crops with enhanced seed tocopherol. B Karunanandaa, Metab Eng. 75-6in engKarunanandaa, B., et al. (2005). Metabolically engineered oilseed crops with en- hanced seed tocopherol. (Translated from eng). Metab Eng, 7(5-6), 384-400, (in eng).

Production of Transgenic Soybean Plants Using Agrobacterium-Mediated DNA Transfer. M A W Hinchee, Nat Biotech. 68Hinchee, M. A. W., et al. (1988). Production of Transgenic Soybean Plants Using Agrobacterium-Mediated DNA Transfer. Nat Biotech, 6(8), 915-922.

Stable Transformation of Soybean Callus by DNA-Coated Gold Particles. P Christou, D E Mc Cabe, W F Swain, Plant Physiology. 873Christou, P., Mc Cabe, D. E., & Swain, W. F. (1988). Stable Transformation of Soybean Callus by DNA-Coated Gold Particles. Plant Physiology, 87(3), 671-674.

The use of glufosinate as a selective agent in Agrobacterium-mediated transformation of soybean. Plant Cell, Tissue and Organ Culture. Z Zhang, A Xing, P Staswick, T E Clemente, 56Zhang, Z., Xing, A., Staswick, P., & Clemente, T. E. (1999). The use of glufosinate as a selective agent in Agrobacterium-mediated transformation of soybean. Plant Cell, Tis- sue and Organ Culture , 56(1), 37-46.

Efficient soybean transformation using hygromycin B selection in the cotyledonary-node method. P Olhoft, L Flagel, C Donovan, D Somers, Planta. 2165Olhoft, P., Flagel, L., Donovan, C., & Somers, D. (2003). Efficient soybean transforma- tion using hygromycin B selection in the cotyledonary-node method. Planta, 216(5), 723-735.

Refined glufosinate selection in Agrobacterium-mediated transformation of soybean. P Zeng, D A Vadnais, Z Zhang, J C Polacco, Plant Cell Reports. 227Glycine max (L.) MerrillZeng, P., Vadnais, D. A., Zhang, Z., & Polacco, J. C. (2004). Refined glufosinate selec- tion in Agrobacterium-mediated transformation of soybean [Glycine max (L.) Mer- rill]. Plant Cell Reports, 22(7), 478-482.

Recent advances in the transformation of plants. G Hansen, M S Wright, Trends in Plant Science. 46Hansen, G., & Wright, M. S. (1999). Recent advances in the transformation of plants. Trends in Plant Science, 4(6), 226-231.

Plant regeneration by organogenesis in Glycine max. M S Wright, S M Koehler, M A Hinchee, M G Carnes, Plant Cell Reports. 52Wright, M. S., Koehler, S. M., Hinchee, M. A., & Carnes, M. G. (1986). Plant regenera- tion by organogenesis in Glycine max. Plant Cell Reports, 5(2), 150-154.

Progeny Analysis of Glyphosate Selected Transgenic Soybeans Derived from-Mediated Transformation. T E Clemente, Crop Sci. 403Clemente, T. E., et al. (2000). Progeny Analysis of Glyphosate Selected Transgenic Soybeans Derived from-Mediated Transformation. Crop Sci., 40(3), 797-803.

The role of thiol compounds in increasing Agrobacterium-mediated transformation of soybean cotyledonary-node cells. P O Olhoft, K L Lin, J G Galbraith, N N Nielsen, D S Somers, Plant Cell Reports. 208Olhoft, P. O., Lin, K. L., Galbraith, J. G., Nielsen, N. N., & Somers, D. S. (2001). The role of thiol compounds in increasing Agrobacterium-mediated transformation of soybean cotyledonary-node cells. Plant Cell Reports, 20(8), 731-737.

L-Cysteine increases Agrobacterium-mediated T-DNA delivery into soybean cotyledonary-node cells. P O Olhoft, D S Somers, Plant Cell Reports. 208Olhoft, P. O., & Somers, D. S. (2001). L-Cysteine increases Agrobacterium-mediated T-DNA delivery into soybean cotyledonary-node cells. Plant Cell Reports, 20(8), 706-711.

Biological functions of antioxidants in plant transformation. Y Dan, In Vitro Cellular & Developmental Biology-Plant. 443Dan, Y. (2008). Biological functions of antioxidants in plant transformation. In Vitro Cellular & Developmental Biology-Plant, 44(3), 149-161.

Improved cotyledonary node method using an alternative explant derived from mature seed for efficient <i>Agrobacterium</i>-mediated soybean transformation. M Paz, J Martinez, A Kalvig, T Fonger, K Wang, Plant Cell Reports. 253Paz, M., Martinez, J., Kalvig, A., Fonger, T., & Wang, K. (2006). Improved cotyledo- nary node method using an alternative explant derived from mature seed for effi- cient <i>Agrobacterium</i>-mediated soybean transformation. Plant Cell Reports, 25(3), 206-213.

Identification of the signal molecules produced by wounded plant cells that activate T-DNA transfer in Agrobacterium tumefaciens. S E Stachel, E Messens, M Van Montagu, P Zambryski, Nature. 3186047Stachel, S. E., Messens, E., Van Montagu, M., & Zambryski, P. (1985). Identification of the signal molecules produced by wounded plant cells that activate T-DNA transfer in Agrobacterium tumefaciens. Nature, 318(6047), 624-629.

Transformation of Soybean Cells Using Mixed Strains of Agrobacterium tumefaciens and Phenolic Compounds. (Translated from eng). L D Owens, A C Smigocki, Plant Physiol. 883in engOwens, L. D., & Smigocki, A. C. (1988). Transformation of Soybean Cells Using Mixed Strains of Agrobacterium tumefaciens and Phenolic Compounds. (Translated from eng). Plant Physiol, 88(3), 570-573, (in eng).

SAAT: sonication-assisted Agrobacterium-mediated transformation. H N Trick, J J Finer, Transgenic Research. 65Trick, H. N., & Finer, J. J. (1997). SAAT: sonication-assisted Agrobacterium-mediated transformation. Transgenic Research, 6(5), 329-336.

A Morphogenetically Competent Soybean Suspension Culture. M L Christianson, D A Warnick, P S Carlson, Science. 2224624Christianson, M. L., Warnick, D. A., & Carlson, P. S. (1983). A Morphogenetically Competent Soybean Suspension Culture. Science, 222(4624), 632-634.

Plant regeneration from embryo-derived tissue cultures of soybeans. J Ranch, L Oglesby, A Zielinski, R Horsch, In Vitro Cellular & Developmental Biology -Plant. 2111Ranch, J., Oglesby, L., Zielinski, A., & Horsch, R. (1985). Plant regeneration from em- bryo-derived tissue cultures of soybeans. In Vitro Cellular & Developmental Biology - Plant, 21(11), 653-658.

Development of an embryogenic suspension culture of soybean (Glycine max Merrill.). Plant Cell, Tissue and Organ Culture. J J Finer, A Nagasawa, 15Finer, J. J., & Nagasawa, A. (1988). Development of an embryogenic suspension cul- ture of soybean (Glycine max Merrill.). Plant Cell, Tissue and Organ Culture , 15(2), 125-136.

Transformation of soybean via particle bombardment of embryogenic suspension culture tissue. J Finer, M Mc Mullen, In Vitro Cellular & Developmental Biology -Plant. 274Finer, J., & Mc Mullen, M. (1991). Transformation of soybean via particle bombard- ment of embryogenic suspension culture tissue. In Vitro Cellular & Developmental Biol- ogy -Plant, 27(4), 175-182.

Development, Identification, and Characterization of a Glyphosate-Tolerant Soybean Line. S R Padgette, Crop Sci. 355Padgette, S. R., et al. (2009). Development, Identification, and Characterization of a Glyphosate-Tolerant Soybean Line. Crop Sci., 35(5), 1451-1461.

Transformation of 12 different plasmids into soybean via particle bombardment. M Z Hadi, M D Mc Mullen, J J Finer, Plant Cell Reports. 157Hadi, M. Z., Mc Mullen, M. D., & Finer, J. J. (1996). Transformation of 12 different plasmids into soybean via particle bombardment. Plant Cell Reports, 15(7), 500-505.

Genetic Transformation, Recovery, and Characterization of Fertile Soybean Transgenic for a Synthetic Bacillus thuringiensis cryIAc Gene. C N Stewart, Jr, Plant Physiology. 1121Stewart, C. N., Jr., et al. (1996). Genetic Transformation, Recovery, and Characteriza- tion of Fertile Soybean Transgenic for a Synthetic Bacillus thuringiensis cryIAc Gene. Plant Physiology, 112(1), 121-129.

Stable Transformation of Soybean (Glycine Max) by Particle Acceleration. D E Mc Cabe, W F Swain, B J Martinell, P Christou, Nat Biotech. 68Mc Cabe, D. E., Swain, W. F., Martinell, B. J., & Christou, P. (1988). Stable Transfor- mation of Soybean (Glycine Max) by Particle Acceleration. Nat Biotech, 6(8), 923-926.

. B J Martinell, M Horeb, L S Julson, L Mills, C A Emler, Y Huang, D E Mc Cabe, E J Williams, Patent No. US 6384301Martinell, B. J., Horeb, M., Julson, L. S., Mills, L., Emler, C. A., Huang, Y., Mc Cabe, D. E., & Williams, E. J. (2002). Patent No. US 6384301.

Efficient Agrobacterium tumefaciens-mediated transformation of soybeans using an embryonic tip regeneration system. H.-K Liu, C Yang, Z.-M Wei, Planta. 2196Liu, H.-K., Yang, C., & Wei, Z.-M. (2004). Efficient Agrobacterium tumefaciens-medi- ated transformation of soybeans using an embryonic tip regeneration system. Planta, 219(6), 1042-1049.

Effect of genotype on somatic embryogenesis from immature cotyledons of soybean. W Parrott, E Williams, D Hildebrand, G Collins, Plant Cell, Tissue and Organ Culture. 161Parrott, W., Williams, E., Hildebrand, D., & Collins, G. (1989). Effect of genotype on somatic embryogenesis from immature cotyledons of soybean. Plant Cell, Tissue and Organ Culture, 16(1), 15-21.

Recovery of primary transformants of soybean. W A Parrott, L M Hoffman, D F Hildebrand, E G Williams, G B Collins, Plant Cell Reports. 78Parrott, W. A., Hoffman, L. M., Hildebrand, D. F., Williams, E. G., & Collins, G. B. (1989). Recovery of primary transformants of soybean. Plant Cell Reports, 7(8), 615-617.

Two critical factors are required for efficient transformation of multiple soybean cultivars: &lt. T Ko-S, S Lee, S Krasnyanski, S S Korban, i&gtKo-S, T., Lee, S., Krasnyanski, S., & Korban, S. S. (2003). Two critical factors are re- quired for efficient transformation of multiple soybean cultivars: <i>

Agrobacte-rium</i> strain and orientation of immature cotyledonary explant. TAG Theoretical and Applied Genetics. 1073Agrobacte- rium</i> strain and orientation of immature cotyledonary explant. TAG Theoretical and Applied Genetics , 107(3), 439-447.

Organogenic regeneration of soybean from hypocotyl explants. Y Dan, N Reichert, In Vitro Cellular & Developmental Biology-Plant. 341Dan, Y., & Reichert, N. (1998). Organogenic regeneration of soybean from hypocotyl explants. In Vitro Cellular & Developmental Biology-Plant, 34(1), 14-21.

Hypocotyl-based Agrobacterium-mediated transformation of soybean (Glycine max) and application for RNA interference. G Wang, Y Xu, Plant Cell Reports. 277Wang, G., & Xu, Y. (2008). Hypocotyl-based Agrobacterium-mediated transforma- tion of soybean (Glycine max) and application for RNA interference. Plant Cell Re- ports, 27(7), 1177-1184.

Initiation and propagation of Glycine max L. Merr.: Plants from tissue-cultured epicotyls. M S Wright, M H Williams, P E Pierson, M G Carnes, Plant Cell, Tissue and Organ Culture. 81Wright, M. S., Williams, M. H., Pierson, P. E., & Carnes, M. G. (1987). Initiation and propagation of Glycine max L. Merr.: Plants from tissue-cultured epicotyls. Plant Cell, Tissue and Organ Culture, 8(1), 83-90.

Somatic embryogenesis from cultured epicotyls and primary leaves of soybean [ <i>Glycine max&lt. K Rajasekaran, J Pellow, /i> (L.) MerrillRajasekaran, K., & Pellow, J. (1997). Somatic embryogenesis from cultured epicotyls and primary leaves of soybean [ <i>Glycine max</i> (L.) Merrill].

. In Vitro Cellular & Developmental Biology-Plant. 332In Vitro Cellular & Developmental Biology-Plant, 33(2), 88-91.

. R Khan, Ag Sp, Khan, R. (2006). AG SP.

The host range of crown gall. M De Cleene, J De Ley, The Botanical Review. 424De Cleene, M., & De Ley, J. (1976). The host range of crown gall. The Botanical Review, 42(4), 389-466.

Agrobacterium and plant genetic engineering. P J J Hooykaas, R A Schilperoort, Plant Molecular Biology. 191Hooykaas, P. J. J., & Schilperoort, R. A. (1992). Agrobacterium and plant genetic engi- neering. Plant Molecular Biology, 19(1), 15-38.

Plant transformation technology. C Newell, Molecular Biotechnology. 161Newell, C. (2000). Plant transformation technology. Molecular Biotechnology, 16(1), 53-65.

High efficiency transformation of maize (Zea mays L.) mediated by Agrobacterium tumefaciens. Y Ishida, Nat Biotech. 146Ishida, Y., et al. (1996). High efficiency transformation of maize (Zea mays L.) medi- ated by Agrobacterium tumefaciens. Nat Biotech, 14(6), 745-750.

Transformation of rice mediated by Agrobacterium tumefaciens. Y Hiei, T Komari, T Kubo, Plant Molecular Biology. 351Hiei, Y., Komari, T., & Kubo, T. (1997). Transformation of rice mediated by Agrobac- terium tumefaciens. Plant Molecular Biology, 35(1), 205-218.

Induction and in vitro culture of soybean crown gall tumors. H C Pedersen, J Christiansen, R Wyndaele, Plant Cell Reports. 24Pedersen, H. C., Christiansen, J., & Wyndaele, R. (1983). Induction and in vitro cul- ture of soybean crown gall tumors. Plant Cell Reports, 2(4), 201-204.

The genetic and transcriptional organization of the vir region of the A6 Ti plasmid of Agrobacterium tumefaciens. (Translated from eng). S E Stachel, E W Nester, EMBO J. 57in engStachel, S. E., & Nester, E. W. (1986). The genetic and transcriptional organization of the vir region of the A6 Ti plasmid of Agrobacterium tumefaciens. (Translated from eng). EMBO J, 5(7), 1445-1454, (in eng).

The versatile bacterial type IV secretion systems. E Cascales, P J Christie, Nat Rev Microbiol. 12in engCascales, E., & Christie, P. J. (2003). The versatile bacterial type IV secretion systems. (Translated from eng). Nat Rev Microbiol, 1(2), 137-149, (in eng).

Biogenesis, architecture, and function of bacterial type IV secretion systems. (Translated from eng). P J Christie, K Atmakuri, V Krishnamoorthy, S Jakubowski, E Cascales, Annu Rev Microbiol. 59in engChristie, P. J., Atmakuri, K., Krishnamoorthy, V., Jakubowski, S., & Cascales, E. (2005). Biogenesis, architecture, and function of bacterial type IV secretion systems. (Translated from eng). Annu Rev Microbiol, 59, 451-485, (in eng).

VirB/D4 -dependent protein translocation from Agrobacterium into plant cells. A C Vergunst, Science. 5493in engVergunst, A. C., et al. (2000). VirB/D4 -dependent protein translocation from Agro- bacterium into plant cells. (Translated from eng). Science, 290(5493), 979-982, (in eng).

Positive charge is an important feature of the C-terminal transport signal of the VirB/D4 -translocated proteins of Agrobacterium. (Translated from eng). A C Vergunst, Proc Natl Acad Sci U S A. 1023in engVergunst, A. C., et al. (2005). Positive charge is an important feature of the C-terminal transport signal of the VirB/D4 -translocated proteins of Agrobacterium. (Translated from eng). Proc Natl Acad Sci U S A, 102(3), 832-837, (in eng).

Agrobacterium-mediated plant transformation: the biology behind the "gene-jockeying" tool. S B Gelvin, Microbiol Mol Biol Rev. 671table of contents. in engGelvin, S. B. (2003). Agrobacterium-mediated plant transformation: the biology be- hind the "gene-jockeying" tool. (Translated from eng). Microbiol Mol Biol Rev, 67(1), 16-37, table of contents (in eng).

The Agrobacterium tumefaciens virE2 gene product is a single-stranded-DNA-binding protein that associates with T-DNA. (Translated from eng). P J Christie, J E Ward, S C Winans, E W Nester, J Bacteriol. 1706in engChristie, P. J., Ward, J. E., Winans, S. C., & Nester, E. W. (1988). The Agrobacterium tumefaciens virE2 gene product is a single-stranded-DNA-binding protein that asso- ciates with T-DNA. (Translated from eng). J Bacteriol, 170(6), 2659-2667, (in eng).

Nuclear localization of Agrobacterium VirE2 protein in plant cells. V Citovsky, J Zupan, D Warnick, P Zambryski, Science. 5065in engCitovsky, V., Zupan, J., Warnick, D., & Zambryski, P. (1992). Nuclear localization of Agrobacterium VirE2 protein in plant cells. (Translated from eng). Science, 256(5065), 1802-1805, (in eng).

Trojan horse strategy in Agrobacterium transformation: abusing MAPK defense signaling. A Djamei, A Pitzschke, H Nakagami, I Rajh, H Hirt, Science. 5849in engDjamei, A., Pitzschke, A., Nakagami, H., Rajh, I., & Hirt, H. (2007). Trojan horse strat- egy in Agrobacterium transformation: abusing MAPK defense signaling. (Translated from eng) . Science, 318(5849), 453-456, (in eng).

Traversing the cell: Agrobacterium T-DNAs journey to the host genome. (Translated from English). S B Gelvin, Frontiers in Plant Science. 3in EnglishGelvin, S. B. (2012). Traversing the cell: Agrobacterium T-DNAs journey to the host genome. (Translated from English). Frontiers in Plant Science, 3, (in English).

Agrobacterium induces expression of a host F-box protein required for tumorigenicity. A Zaltsman, A Krichevsky, A Loyter, V Citovsky, Cell Host Microbe. 73in engZaltsman, A., Krichevsky, A., Loyter, A., & Citovsky, V. (2010). Agrobacterium indu- ces expression of a host F-box protein required for tumorigenicity. (Translated from eng). Cell Host Microbe, 7(3), 197-209, (in eng).

Involvement of targeted proteolysis in plant genetic transformation by Agrobacterium. (Translated from eng). T Tzfira, M Vaidya, V Citovsky, Nature. 7004in engTzfira, T., Vaidya, M., & Citovsky, V. (2004). Involvement of targeted proteolysis in plant genetic transformation by Agrobacterium. (Translated from eng). Nature, 431(7004), 87-92, (in eng).

Genotypic variability of soybean response to agrobacterium strains harboring the ti or ri plasmids. L D Owens, D E Cress, Plant Physiol. 771in engOwens, L. D., & Cress, D. E. (1985). Genotypic variability of soybean response to agrobacterium strains harboring the ti or ri plasmids. (Translated from eng). Plant Physiol, 77(1), 87-94, (in eng).

. 10.5772/51076An Overview of Genetic Transformation of Soybean. An Overview of Genetic Transformation of Soybean http://dx.doi.org/10.5772/51076

Agrobacterium tumefaciens Susceptibility and Plant Regeneration of 10 Soybean Genotypes in Maturity Groups 00 to II. B W Delzer, D A Somers, J H Orf, Crop Sci. 302Delzer, B. W., Somers, D. A., & Orf, J. H. . Agrobacterium tumefaciens Susceptibility and Plant Regeneration of 10 Soybean Genotypes in Maturity Groups 00 to II. Crop Sci., 30(2), 320-322.

Soybean Transgenic Crops VI. P M Olhoft, D A Somers, Biotechnology in Agriculture and Forestry. Pua E-C & Davey MRBerlin, HeidelbergSpringer61Olhoft, P. M., & Somers, D. A. (2007). Soybean Transgenic Crops VI. Biotechnology in Ag- riculture and Forestry, eds Pua E-C & Davey MR, Springer, Berlin, Heidelberg, 61, 3-27.

Recent Advances in Legume Transformation. D A Somers, D A Samac, P M Olhoft, Plant Physiology. 1313Somers, D. A., Samac, D. A., & Olhoft, P. M. (2003). Recent Advances in Legume Transformation. Plant Physiology, 131(3), 892-899.

Cytokinin gene fused with a strong promoter enhances shoot organogenesis and zeatin levels in transformed plant cells. (Translated from eng). A C Smigocki, L D Owens, Proc Natl Acad Sci U S A. 8514in engSmigocki, A. C., & Owens, L. D. (1988). Cytokinin gene fused with a strong promoter enhances shoot organogenesis and zeatin levels in transformed plant cells. (Translat- ed from eng). Proc Natl Acad Sci U S A, 85(14), 5131-5135, (in eng).

Inheritance of Soybean Susceptibility to Agrobacterium tumefaciens and Its Relationship to Transformation. A O Mauro, T W Pfeiffer, G B Collins, Crop Sci. 354Mauro, A. O., Pfeiffer, T. W., & Collins, G. B. (2009). Inheritance of Soybean Suscepti- bility to Agrobacterium tumefaciens and Its Relationship to Transformation. Crop Sci., 35(4), 1152-1156.

An optimized Agrobacterium-mediated transformation for soybean for expression of binary insect resistance genes. W Dang, Z-M Wei, Plant Science. 1734Dang, W., & Wei, Z-m. (2007). An optimized Agrobacterium-mediated transforma- tion for soybean for expression of binary insect resistance genes. Plant Science, 173(4), 381-389.

Characterization and host range determination of soybean super virulent Agrobacterium tumefaciens KAT23. K Yukawa, H Kaku, H Tanaka, Y Koga-Ban, M Fukuda, Bioscience, biotechnology, and biochemistry. 717Yukawa, K., Kaku, H., Tanaka, H., Koga-Ban, Y., & Fukuda, M. (2007). Characteriza- tion and host range determination of soybean super virulent Agrobacterium tumefa- ciens KAT23. Bioscience, biotechnology, and biochemistry , 71(7), 1676-1682.

L-Cysteine increasesAgrobacterium-mediated T-DNA delivery into soybean cotyledonary-node cells. P O Olhoft, D S Somers, Plant Cell Reports. 208Olhoft, P. O., & Somers, D. S. (2001). L-Cysteine increasesAgrobacterium-mediated T-DNA delivery into soybean cotyledonary-node cells. Plant Cell Reports, 20(8), 706-711.

Assessment of conditions affecting Agrobacterium-mediated soybean transformation using the cotyledonary node explant. M Paz, Euphytica. 1362Paz, M., et al. (2004). Assessment of conditions affecting Agrobacterium-mediated soybean transformation using the cotyledonary node explant. Euphytica, 136(2), 167-179.

Susceptibility to Agrobacterium tumefaciens and cotyledonary node transformation in short-season soybean. P A Donaldson, D H Simmonds, Plant Cell Reports. 195Donaldson, P. A., & Simmonds, D. H. (2000). Susceptibility to Agrobacterium tume- faciens and cotyledonary node transformation in short-season soybean. Plant Cell Re- ports, 19(5), 478-484.

The effect of co-cultivation and selection parameters on Agrobacterium-mediated transformation of Chinese soybean varieties. S.-J Liu, Z.-M Wei, J.-Q Huang, Plant Cell Reports. 273Liu, S.-J., Wei, Z.-M., & Huang, J.-Q. (2008). The effect of co-cultivation and selection parameters on Agrobacterium-mediated transformation of Chinese soybean variet- ies. Plant Cell Reports, 27(3), 489-498.

A multi-needle-assisted transformation of soybean cotyledonary node cells. R.-G Xue, H.-F Xie, B Zhang, Biotechnology Letters. 2819Xue, R.-G., Xie, H.-F., & Zhang, B. (2006). A multi-needle-assisted transformation of soybean cotyledonary node cells. Biotechnology Letters, 28(19), 1551-1557.

Transgenic barley (Hordeum vulgare L.) by electroporation of protoplasts. M Salmenkallio-Marttila, Plant Cell Reports. 153Salmenkallio-Marttila, M., et al. (1995). Transgenic barley (Hordeum vulgare L.) by electroporation of protoplasts. Plant Cell Reports, 15(3), 301-304.

Plant protoplasts: status and biotechnological perspectives. M R Davey, P Anthony, J B Power, K C Lowe, Biotechnology Advances. 232Davey, M. R., Anthony, P., Power, J. B., & Lowe, K. C. (2005). Plant protoplasts: sta- tus and biotechnological perspectives. Biotechnology Advances, 23(2), 131-171.

Silicon carbide fiber-mediated stable transformation of plant cells. H F Kaeppler, D A Somers, H W Rines, A F Cockburn, TAG Theoretical and Applied Genetics. 845Kaeppler, H. F., Somers, D. A., Rines, H. W., & Cockburn, A. F. (1992). Silicon carbide fiber-mediated stable transformation of plant cells. TAG Theoretical and Applied Genet- ics, 84(5), 560-566.

Integration of foreign DNA following microinjection of tobacco mesophyll protoplasts. A Crossway, Molecular and General Genetics MGG. 2022Crossway, A., et al. (1986). Integration of foreign DNA following microinjection of to- bacco mesophyll protoplasts. Molecular and General Genetics MGG, 202(2), 179-185.

Chloroplast transformation in Chlamydomonas with high velocity microprojectiles. J Boynton, Science. 2404858Boynton, J., et al. (1988). Chloroplast transformation in Chlamydomonas with high velocity microprojectiles. Science, 240(4858), 1534-1538.

Vectors carrying two separate T-DNAs for co-transformation of higher plants mediated by Agrobacterium tumefaciens and segregation of transformants free from selection markers. T Komari, Y Hiei, Y Saito, N Murai, T Kumashiro, The Plant Journal. 101Komari, T., Hiei, Y., Saito, Y., Murai, N., & Kumashiro, T. (1996). Vectors carrying two separate T-DNAs for co-transformation of higher plants mediated by Agrobacte- rium tumefaciens and segregation of transformants free from selection markers. The Plant Journal, 10(1), 165-174.

Transposition-Based Plant Transformation. H Yan, C M Rommens, Plant Physiology. 1432Yan, H., & Rommens, C. M. (2007). Transposition-Based Plant Transformation. Plant Physiology, 143(2), 570-578.

Excision of a selective marker in transgenic rice using a novel Cre/ <i>loxP</i> system controlled by a floral specific promoter. X Bai, Q Wang, C Chu, Transgenic Research. 176Bai, X., Wang, Q., & Chu, C. (2008). Excision of a selective marker in transgenic rice using a novel Cre/ <i>loxP</i> system controlled by a floral specific pro- moter. Transgenic Research, 17(6), 1035-1043.

Frequencies of simultaneous transformation with different T-DNAs and their relevance to the <i>Agrobacterium</i>/plant cell interaction. A Depicker, L Herman, A Jacobs, J Schell, M Montagu, Molecular and General Genetics MGG. 2013Depicker, A., Herman, L., Jacobs, A., Schell, J., & Montagu, M. (1985). Frequencies of simultaneous transformation with different T-DNAs and their relevance to the <i>Agrobacterium</i>/plant cell interaction. Molecular and General Genetics MGG, 201(3), 477-484.

A BAC-and BIBAC-based physical map of the soybean genome. C Wu, Genome Res. 142in engWu, C., et al. (2004). A BAC-and BIBAC-based physical map of the soybean genome. (Translated from eng). Genome Res, 14(2), 319-326, (in eng).

BAC as tools for genome sequencing. H Zhang-B, C Wu, Plant Physiology and Biochemistry. 393-4Zhang-B, H., & Wu, C. (2001). BAC as tools for genome sequencing. Plant Physiology and Biochemistry, 39(3-4), 195-209.

Stable transfer of intact high molecular weight DNA into plant chromosomes. C M Hamilton, A Frary, C Lewis, S D Tanksley, Translated from engHamilton, C. M., Frary, A., Lewis, C., & Tanksley, S. D. (1996). Stable transfer of in- tact high molecular weight DNA into plant chromosomes. (Translated from eng).

. Proc Natl Acad Sci U S A. 9318in engProc Natl Acad Sci U S A, 93(18), 9975-9979, (in eng).

The S locus glycoprotein and the S receptor kinase are sufficient for self-pollen rejection in Brassica. (Translated from eng). Y Cui, Y M Bi, N Brugiere, M Arnoldo, S J Rothstein, Proc Natl Acad Sci. 977in engCui, Y., Bi, Y. M., Brugiere, N., Arnoldo, M., & Rothstein, S. J. (2000). The S locus gly- coprotein and the S receptor kinase are sufficient for self-pollen rejection in Brassica. (Translated from eng). Proc Natl Acad Sci U S A, 97(7), 3713-3717, (in eng).

Efficiency and stability of high molecular weight DNA transformation: an analysis in tomato. A Frary, C M Hamilton, Transgenic Res. 102in engFrary, A., & Hamilton, C. M. (2001). Efficiency and stability of high molecular weight DNA transformation: an analysis in tomato. (Translated from eng). Transgenic Res, 10(2), 121-132, (in eng).

Development of transformation system of rice based on binary bacterial artificial chromosome (BIBAC) vector. (Translated from eng). R F He, Yi Chuan Xue Bao. 333in engHe, R. F., et al. (2006). Development of transformation system of rice based on binary bacterial artificial chromosome (BIBAC) vector. (Translated from eng). Yi Chuan Xue Bao, 33(3), 269-276, (in eng).

. 10.5772/51076An Overview of Genetic Transformation of Soybean. An Overview of Genetic Transformation of Soybean http://dx.doi.org/10.5772/51076

Precise genome modification in the crop species Zea mays using zinc-finger nucleases. V K Shukla, Nature. 7245Shukla, V. K., et al. (2009). Precise genome modification in the crop species Zea mays using zinc-finger nucleases. Nature, 459(7245), 437-441.

High-frequency modification of plant genes using engineered zinc-finger nucleases. J A Townsend, Nature. 7245Townsend, J. A., et al. (2009). High-frequency modification of plant genes using engi- neered zinc-finger nucleases. Nature, 459(7245), 442-445.

Targeted Mutagenesis of Duplicated Genes in Soybean with Zinc-Finger Nucleases. S J Curtin, Plant Physiology. 1562Curtin, S. J., et al. (2011). Targeted Mutagenesis of Duplicated Genes in Soybean with Zinc-Finger Nucleases. Plant Physiology, 156(2), 466-473.

High frequency targeted mutagenesis in Arabidopsis thaliana using zinc finger nucleases. F Zhang, Proceedings of the National Academy of Sciences. 10726Zhang, F., et al. (2010). High frequency targeted mutagenesis in Arabidopsis thaliana using zinc finger nucleases. Proceedings of the National Academy of Sciences, 107(26), 12028-12033.

Unexpected failure rates for modular assembly of engineered zinc fingers. C L Ramirez, Nat Meth. 55Ramirez, C. L., et al. (2008). Unexpected failure rates for modular assembly of engi- neered zinc fingers. Nat Meth, 5(5), 374-375.

Breaking the Code of DNA Binding Specificity of TAL-Type III Effectors. J Boch, Science. 3265959Boch, J., et al. (2009). Breaking the Code of DNA Binding Specificity of TAL-Type III Effectors. Science, 326(5959), 1509-1512.

A Simple Cipher Governs DNA Recognition by TAL Effectors. M J Moscou, A J Bogdanove, Science. 32659591501Moscou, M. J., & Bogdanove, A. J. (2009). A Simple Cipher Governs DNA Recogni- tion by TAL Effectors. Science, 326(5959), 1501.

TAL effectors: finding plant genes for disease and defense. A J Bogdanove, S Schornack, T Lahaye, Current Opinion in Plant Biology. 134Bogdanove, A. J., Schornack, S., & Lahaye, T. (2010). TAL effectors: finding plant genes for disease and defense. Current Opinion in Plant Biology, 13(4), 394-401.

How Xanthomonas type III effectors manipulate the host plant. S Kay, U Bonas, Current Opinion in Microbiology. 121Kay, S., & Bonas, U. (2009). How Xanthomonas type III effectors manipulate the host plant. Current Opinion in Microbiology, 12(1), 37-43.

Xanthomonas AvrBs3 Family-Type III Effectors: Discovery and Function. J Boch, U Bonas, Annual Review of Phytopathology. 481Boch, J., & Bonas, U. (2010). Xanthomonas AvrBs3 Family-Type III Effectors: Discov- ery and Function. Annual Review of Phytopathology, 48(1), 419-436.

Dimerization of the bacterial effector protein AvrBs3 in the plant cell cytoplasm prior to nuclear import. D Grlebeck, B Szurek, U Bonas, The Plant Journal. 422Grlebeck, D., Szurek, B., & Bonas, U. (2005). Dimerization of the bacterial effector protein AvrBs3 in the plant cell cytoplasm prior to nuclear import. The Plant Journal, 42(2), 175-187.

A single plant resistance gene promoter engineered to recognize multiple TAL effectors from disparate pathogens. P Rmer, S Recht, T Lahaye, Proceedings of the National Academy of Sciences. 10648Rmer, P., Recht, S., & Lahaye, T. (2009). A single plant resistance gene promoter en- gineered to recognize multiple TAL effectors from disparate pathogens. Proceedings of the National Academy of Sciences, 106(48), 20526-20531.

A transcription activator-like effector toolbox for genome engineering. N E Sanjana, Nat. Protocols. 71Sanjana, N. E., et al. (2012). A transcription activator-like effector toolbox for genome engineering. Nat. Protocols, 7(1), 171-192.