# Local externalization of phosphatidylserine mediates developmental synaptic pruning by microglia Review Timeline

CorpusID: 218466981
 
tags: #Medicine, #Chemistry, #Biology

URL: [https://www.semanticscholar.org/paper/d1284f627b345b52cdad70d816c3fb41bab4017a](https://www.semanticscholar.org/paper/d1284f627b345b52cdad70d816c3fb41bab4017a)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Local externalization of phosphatidylserine mediates developmental synaptic pruning by microglia Review Timeline


Nicole Scott-Hewitt 
Fabio Perrucci 
Raffaella Morini 
Marco Erreni 
Matthew Mahoney 
Agata Witkowska 
Alanna Carey 
Elisa Faggiani 
Lisa Schuetz 
Sydney Mason 
Matteo Tamborini 
Matteo Bizzotto 
Lorena Passoni 
Fabia Filipello 
Reinhard Jahn 
Beth Stevens beth.stevens@childrens.harvard.edu 
Michela Matteoli 
michela.matteoli@hunimed.euMichela Matteoli 
Beth Stevens 
Local externalization of phosphatidylserine mediates developmental synaptic pruning by microglia Review Timeline
10.15252/embj.2020105380Submission Date: 22nd Apr 20 Editorial Decision: 14th May 20 Revision Received: 26th May 20 Editorial Decision: 2nd Jun 20 Revision Received: 5th Jun 20 Accepted: 15th Jun 20 Editor: Karin DumstreiCorresponding author(s):


In recent years, it has become clear that microglia play important roles in synapse elimination during development, which is required for appropriate refinement of neural circuits. In this article, the authors sought to address two important outstanding questions: What neuronal cues do microglia recognize that direct their engulfment of appropriate synapses? And how do the pathways that have been identified thus far work together during the course of development? These are critical questions for our understanding of nervous system development. This article does show that presentation of PS can influence microglial phagocytic activity in vitro and that PS is present at synapses at times and places when synaptic remodeling is occurring in vivo. Furthermore, PS+ material is engulfed by microglia during synaptic refinement, which is reduced in mice lacking C1q, a known component involved in microglial engulfment of synapses during development. However, the authors do not definitively show that PS serves as a neuronal cue that directs microglial engulfment of appropriate synapses. It also remains unclear how the two mechanisms proposed for its recognition: Trem2 in vitro, and perhaps in their hippocampal experiments, and C1q in the LGN, come together to provide a framework for understanding how these pathways work together. This later question is a challenging one to answer, and the authors should amend their claim in the introduction that "Our data... provide a framework for the convergence of several previously described mechanisms of microglial-mediated developmental synaptic pruning", though the relationship of the data in this paper to that question bears inclusion in the discussion. Major comments: - Fig. 1. This study clearly shows that PS+ liposomes are efficiently taken up by microglia. However, the effect of TREM2 deficiency is only about 50%, as is the effect of AnnexinV on liposome uptake. These results raise the question of to what degree the uptake is cargo-driven phagocytosis, as distinct from liposome uptake through fluid phase uptake, which is also a consequence of normal process motility. This experiment needs to be better controlled. They describe a bead uptake measurement, but the data are not provided. Moreover, there are a number of PS receptors that stimulate phagocytosis on microglia, e.g. CD36, Axl, MERTK etc., and it is imputed that Trem2 is mediating the observed phagocytic uptake. For this conclusion to be accepted the experiment needs to be better controlled and their interpretation tempered. --It is not clear to me what the individual data points represent. I assume that in this and other figures the authors quantify and calculate statistical significance based on the number of cells or image fields instead of experimental replicates. They state that 3 independent experiments were performed, and they artificially increase the experimental N by using repeated measures from the same animal, constituting pseudoreplication. --In Fig 2F, similar to what the authors report later in vivo, there seems to be many examples of ANXV localization not with BSN or SHANK2. Perhaps it would be helpful to quantify the percent of ANXV that is co-localized with these synaptic markers to get a sense of the extent to which PS is synaptic. The images are at too low magnification to be able to really objectively evaluate the extent of colocalization. --The authors clearly establish that PS is present on synapses during developmental time points when refinement occurs and that microglia do engulf PS+ material. However, I am not convinced that PS specifically labels synapses that should be removed. What if PS is simply present on immature synapses? Since it stated in the methods that the authors collected data from regions within the LGN that will ultimately be refined to contain ipsilateral and contralateral inputs, a stronger analysis would be to look at the percent of PS+ ipsilateral inputs within the contralateral region, relative to PS+ contralateral inputs. And also the percent of PS+ contralateral inputs relative to ipsilateral inputs within the ipsilateral region. Even this would not definitively demonstrate that PS labels synapses that should be engulfed, however. It would be suggestive if TTX injection into one eye led to preferential increases in PS exposure on synapses from that eye. I recognize that this is a difficult point to address, but is essential to the author's model that PS serves to label synapses that should undergo engulfment. --Based on the author's data, it is not clear whether microglia preferentially engulf PS+ synapses relative to PS-ones, which is essential for their argument that PS serves a cue to promote engulfment of particular synaptic targets. The authors could address this using their data in Fig 6D-E by quantifying the percent of engulfed synaptic material that is also PS+. If PS is a cue to direct engulfment, then a substantial percentage of engulfed material should be PS+. Or the authors could compare the percentage of PS+ inputs outside of microglia vs the PS+ inputs inside of microglia, relative to the percentage of PS-inputs outside of microglia vs PS-inputs inside of microglia. There should be a higher proportion of PS+ inputs inside microglia compared to PS-inputs if it does indeed serve as a directive 'eat me' cue.

--A similar analysis should be performed in Fig 7 to show that C1q impairs the uptake specifically of PS+ synaptic inputs. In further support of the author's model that PS is a cue that allows C1q binding, then they could quantify the percent of C1q+ synapses that are also PS+. If PS is required for C1q association with synapses, then this should be close to 100%. It is presently an overinterpretation of the data presented in Fig 7 to say "All together, these data... suggest that synaptic PS exposure may act as the signal leading to C1q deposition during pruning" Minor comments: -It would be helpful to make all graphs consistent with one another to be able to more clearly compare experiments performed across figures, for example showing each replicate as a dot or the summary data or both, having error bars shown consistently above or above and below, and having n's reported or not on the graph itself.

--It would be helpful for the authors to include the Trem2 KO data with Annexin V in the figure legend of Fig 1 E, H into the figure itself. These data are important for the author's overall conclusions and warrant inclusion in the figure.

--It would strengthen the case that Trem2 acted as a PS receptor if the authors could show in Fig  1G that Trem2 KO microglia preferentially reduced phagocytosis of the PS99 liposomes, while having less effect on the PS20 liposome uptake. The authors could also support this by repeating the experiment in Fig 1H with Trem2 KO microglia. In this case, AnnexinV should have no effect on uptake in Trem2 KO microglia if it is specifically recognizing PS as the authors suggest. --The authors do describe this in more detail in the methods but the statement in the legend for Fig  1 that "amount of lysosomes internalized by microglia and internalized through CD68 colocalization" could be clarified. --The n's in Fig 2B and 2D seem to be switched on the bars of the graph relative to the legend. --From the experiments in Fig 2, is it known whether co-culturing with microglia is required for the presence of ANXV+ labeling in neurons? It would be interesting to know whether, in addition to removing the synapses, whether the presence of microglia is also required for induction of PS exposure at synapses. --In Figure 2, the authors find no difference in PS association with pre-vs post-synaptic elements, and yet they find a clear preferential association with presynaptic components in vivo. Could the authors speculate on why this might be the case? --In the legend for Fig 2, it states that spines were analyzed in HN neurons alone, but this does not appear to be present on the graphs in Fig 2B-D. If these data exist, it would be helpful to include. Otherwise, the authors cannot claim in the text that ANXV had no effect alone. --In the methods related to Fig 2, it states that 15 min pretreatment of Annexin V was effective block PS 24 hours later which is surprising. Certainly the differences are clear, but did you validate that at this time point PS continues to be blocked? --In Fig 6H, if PS is serving as a cue to promote internalization of inputs, why would there be a difference in the percent of engulfed ipsilateral vs contralateral inputs that were PS+? I recognize that ipsilateral inputs are more likely to be engulfed, and thus outside of microglia, should exhibit higher PS levels (as in Fig 6G), but once they're engulfed, based on your model, wouldn't you expect both input types be equally PS+? --In Fig 7, does the ePS in the title refer to PSVue+? ---As the authors state in the discussion, there are many receptors, even beyond Trem2 and C1q that can mediate engulfment of substrates that are PS+. Could the authors speculate on why these other receptors fail to compensate for the removal of excess PS+ substrates in C1q KO mice?

Referee #2:

This manuscript by Scott-Hewitt & Perrucci et al. aims to better understand the mechanisms through which synapses are eliminated by microglia. Using high-resolution imaging analyses in vivo and in vitro-supplemented by electrophysiology-they establish that phosphatidylserine (PS) tags synapses and acts in microglia-mediated synapse engulfment and elimination. They show that PS labeling of synapses is developmentally regulated, implicated in synapse pruning during eye-specific segregation, and, in the hippocampus, is at play in C1q-dependent microglial engulfment in vivo. They also provide evidence that TREM2, which has been shown by other groups to mediate synapse elimination, may accomplish this by binding its known ligand PS. They conclude that PS is a general "eat me" signal that links several pathways that mediate microglial synaptic pruning. They use well-established methods for assessing microglial engulfment, synapse elimination, with physiological correlates for observed structural changes. These findings are of wide interest as they reveal how a common cellular "eat me" signal could be a common signaling mechanism for elimination of synapses-a process critical for understanding neural circuit development, function, and perhaps dysfunction. There are two major and several minor concerns that if addressed would greatly improve the impact of the present study.

Major Concerns: 1.The authors beautifully show that PS+ particles are engulfed by microglia, that PS co-localizes with synapses, and that blocking PS decreases engulfment. A key question remains: are the engulfed PS+ puncta shown in the dLGN synapses? Under typical circumstances, one would probably ask the authors to add evidence of synaptic markers and colocalization with PS within microglia as proof that the particles are not cellular debris. During these times, however, I will recommend the authors point out this caveat in their conclusions and discussion.

2.Likewise, though it is known that TREM2 binds PS and may be involved in synapse elimination, the authors provide some but incomplete evidence that these two properties are linked in developmental synapse elimination in vivo. In the discussion, the authors could highlight generally their thoughts on experiments that might more conclusively tie these together and also the degree to which these multiple phagocytic receptor pathways may all contribute to synapse elimination (for example, in the in vitro Trem2 KO studies there is considerable residual engulfment). Given this, I would also recommend softening the language "we identify a role of synaptic PS exposure in BOTH C1q and TREM2-mediated microglial elimination."

Minor concerns: 1.Given that most of the data is dependent on imaging analysis, it is important to avoid pseudoreplication. Individual observations of a cell in tissue culture experiment or a section of a brain should not be treated as replicates for statistical purposes. Seeing the data points is very helpful, but for statistical analysis would recommend different handling (i.e. nested testing, error propagation, or mixed modeling). This applies to the analyses in Fig. 1 F, G, H, Fig. 2 A 2.Recommend that results in Fig. 4 C and Fig. 6 B, C, F, G, H be tested using paired t-tests because the underlying data is matched within a sample.

3.Please elaborate in the methods how the samples were blinded (i.e. what does when "when appropriate" mean and was blinding applied to all analyses), especially to underscore how the team limited sampling bias in the colocalization experiments in vivo. Please also provide more information on the algorithms you used for image segmentation in Ilastik. Was the same classification model used for all images or was a new model trained to segment each experiment or image?

Since the analyses are largely volume-based: are there differences in cell volume or CD68 volume in, for example, the Trem2-/-cells as compared with controls? Point-to-point rebuttal Reviewer#1 1) the authors should amend their claim in the introduction that "Our data... provide a framework for the convergence of several previously described mechanisms of microglial-mediated developmental synaptic pruning", though the relationship of the data in this paper to that question bears inclusion in the discussion.

As required by the Reviewer we have amended our claim in the Introduction (page 4). We have also clarified that PS could be involved in several known mechanisms described, however whether these mechanisms converge or provide temporally or spatially specific cues remains unknown (addressed further throughout the Introduction and Discussion).

2) Fig. 1. This study clearly shows that PS+ liposomes are efficiently taken up by microglia. However, the effect of TREM2 deficiency is only about 50%, as is the effect of AnnexinV on liposome uptake. These results raise the question of to what degree the uptake is cargo-driven phagocytosis, as distinct from liposome uptake through fluid phase uptake, which is also a consequence of normal process motility. This experiment needs to be better controlled. They describe a bead uptake measurement, but the data are not provided. Moreover, there are a number of PS receptors that stimulate phagocytosis on microglia, e.g. CD36, Axl, MERTK etc., and it is imputed that Trem2 is mediating the observed phagocytic uptake. For this conclusion to be accepted the experiment needs to be better controlled and their interpretation tempered. Fig. 1 data showing that in fact Trem2 KO microglia preferentially reduces phagocytosis of the PS99 with respect to the PS50 liposomes. Also, we show that, in Trem2 KO microglia, preincubation with annexin does not reduce the phagocytosis of liposomes. These data are consistent with the occurrence of a phagocytic process mediated by TREM2 and dependent on exposed PS, accompanied by a non-specific process of fluid phase uptake. These data have been described in the revised manuscript (page 5). The possible involvement of additional microglia receptors (including MERTK, BAI1, and GPR56) has been now mentioned in Discussion (pages 16-17). The bead uptake measurement that the Reviewer is referring to was already shown in Suppl . FIG 1 B. This figure shows the lack of difference between the ability of WT and KO microglia, with or without ANX treatment, in phagocytosing beads (thus in the absence of surface PS).


## As Reviewer 1 noted, TREM2 deficiency only reduces about 50% of PS+ liposomes. Based also on minor point of Reviewer 1 (It would strengthen the case that Trem2 acted as a PS receptor if the authors could show in Fig 1G that Trem2 KO microglia preferentially reduced phagocytosis of the PS99 liposomes, while having less effect on the PS20 liposome uptake. The authors could also support this by repeating the experiment in Fig 1H with Trem2 KO microglia. In this case, AnnexinV should have no effect on uptake in Trem2 KO microglia if it is specifically recognizing PS as the authors suggest): We added to

3) It is not clear to me what the individual data points represent. I assume that in this and other figures the authors quantify and calculate statistical significance based on the number of cells or image fields instead of experimental replicates. They state that 3 independent experiments were performed, and they artificially increase the experimental N by using repeated measures from the same animal, constituting pseudoreplication.  Fig 2F, similar to what the authors report later in vivo, there seems to be many examples of ANXV localization not with BSN or SHANK2. Perhaps it would be helpful to quantify the percent of ANXV that is co-localized with these synaptic markers to get a sense of the extent to which PS is synaptic. The images are at too low magnification to be able to really objectively evaluate the extent of colocalization.


## 4) In


## The percentage of annexin colocalized with synaptic markers in culture is around 10% (with peaks up to 20%), see figure for

Reviewer. This is expected, as neurons in culture may undergo degeneration and thus result as PS+. However the functional data obtained in co-cultures (electrophysiology, synapse count) indicate that cloaking PS is enough to largely prevent synapse elimination in culture. It is likely that additional cues (besides PS) provide the specificity for synapse elimination and this has been now discussed (pages 16-17). In order to provide a more reliable answer to the point raised by the Reviewer, we moved to brain sections, where nonspecific cell death is expected to be lower, and calculated the percentage of PSVue colocalized with synaptic markers. This turned out to be around 20% (with peaks up to 30%) We do not believe these data need to be added to the text, but we are willing to do that, if needed. Furthermore, a higher magnification for Fig. 2I has now been provided.

5) The authors clearly establish that PS is present on synapses during developmental time points when refinement occurs and that microglia do engulf PS+ material. However, I am not convinced that PS specifically labels synapses that should be removed. What if PS is simply present on immature synapses?

Although we have evidence that, differently from mature synapses, i) filopodia are not eliminated when hippocampal neurons are co-cultured together with microglia and ii) preincubation with annexin does not impact the process (see figure for Reviewer), we cannot exclude that PS is also present on immature synapses (some that will not be eliminated). We modified the discussion (pages 15) to mention that we cannot rule out that PS exposure occurs on immature synapses, and this is something that should be further investigated. 6) Since it stated in the methods that the authors collected data from regions within the LGN that will ultimately be refined to contain ipsilateral and contralateral inputs, a stronger analysis would be to look at the percent of PS+ ipsilateral inputs within the contralateral region, relative to PS+ contralateral inputs. And also the percent of PS+ contralateral inputs relative to ipsilateral inputs within the ipsilateral region. Even this would not definitively demonstrate that PS labels synapses that should be engulfed, however. It would be suggestive if TTX injection into one eye led to preferential increases in PS exposure on synapses from that eye. I recognize that this is a difficult point to address, but is essential to the author's model that PS serves to label synapses that should undergo engulfment.

At P5, the time point we analyzed, there is still considerable overlap between the ipsilateral and contralateral inputs, as the dLGN is actively undergoing eye segregation. Since the ipsi patch is not yet well defined, we see inputs from both eyes localized throughout the LGN. In the analysis, we tried to get representative images that contained both inputs. We could repeat these experiments in P10 animals, where there is a more well-defined ipsi patch, to test whether the remaining PS signal we see is differentially expressed in ipsi vs. contra; however this may be difficult to interpret as eye segregation is mostly completed and microglia engulfment is significantly decreased. We are interested in performing experiments to determine the effect of activity on neuronal PS exposure but we feel this is beyond the scope of the current study as it will require several approaches to manipulate activity in one eye, as TTX injection could have other potential side effects on cell health that make interpretation difficult. For example, in preliminary pilot experiments, we observed an increase in cell death following eye injection, and as PSVue also labels dying cells, differentiating the signal from local synaptic PS vs. apoptotic cells may be difficult. We are continuing to pursue how activity changes may affect PS exposure and engulfment, however, using genetic approaches. Unfortunately these experiments are not ready for publication at this time. Concerning quantification of PS+ synapses inside microglia: Unfortunately it has been technically challenging in the field to identify and validate engulfed synaptic proteins by immunohistochemistry, in particular presynaptic proteins, in vivo (fixed brain tissue). An on-going project in the lab has been to identify and validate synaptic antibodies to quantify engulfed synaptic proteins utilizing FACS in live microglia. We have identified potential candidates that have now been validated using KO controls, however this has not yet been tested in fixed tissue through IHC engulfment analysis. There is a recent publication (Werneburg et al, 2020) demonstrating vGluT2 engulfment analysis in microglia within models of Multiple Sclerosis across several species, however we would need to further validate in our hands at developmental time points. Given these and other technical challenges, we have modified our claims and addressed these caveats and issues in the Discussion, on page 16-17.


## 7)

Based on the author's data, it is not clear whether microglia preferentially engulf PS+ synapses relative to PS-ones, which is essential for their argument that PS serves a cue to promote engulfment of particular synaptic targets. The authors could address this using their data in Fig 6D-E by quantifying the percent of engulfed synaptic material that is also PS+. If PS is a cue to direct engulfment, then a substantial percentage of engulfed material should be PS+. Or the authors could compare the percentage of PS+ inputs outside of microglia vs the PS+ inputs inside of microglia, relative to the percentage of PS-inputs outside of microglia vs PS-inputs inside of microglia. There should be a higher proportion of PS+ inputs inside microglia compared to PS-inputs if it does indeed serve as a directive 'eat me' cue.

Without performing live imaging to follow synapses that do or do not expose PS we do not think it is possible to quantify PS-bound to synaptic elements within microglia in fixed brain tissue given the dynamics of this process and technical issues related to antibodies (as discussed above). Because we are not looking across a time course, we cannot know which synapses exposed PS and were engulfed and which did not. There is also no way to quantify total "synapses engulfed". Further, because astrocytes have been shown to contribute to developmental synaptic pruning, it is likely that they also engulf PS positive inputs. We have some preliminary data demonstrating that astrocytes also engulf PS-labeled material at P5 in the dLGN, so trying to measure microglia PS+ vs -in fixed tissues would not reflect that aspect. Fig 7 to show that C1q impairs the uptake specifically of PS+ synaptic inputs. In further support of the author's model that PS is a cue that allows C1q binding, then they could quantify the percent of C1q+ synapses that are also PS+. If PS is required for C1q association with synapses, then this should be close to 100%. It is presently an over-interpretation of the data presented in Fig 7 to say "All together, these data... suggest that synaptic PS exposure may act as the signal leading to C1q deposition during pruning"


## 8) A similar analysis should be performed in

Thank you for this point and suggestion. We performed several analyses of our data and included the new data, which can be found in revised Fig 7. We calculated the percentage of vGluT2 inputs that colocalized with C1q across each time point (P2, 5, and 10), and found that this did not change (Fig 7B). We further calculated the percentage of C1q+ vGluT2 inputs that were also positive for PS, which peaked at P5 at ~8% (Fig 7C). Based on these new results, we updated the text to clarify our findings in Results (page 11-12).

Minor comments: -It would be helpful to make all graphs consistent with one another to be able to more clearly compare experiments performed across figures, for example showing each replicate as a dot or the summary data or both, having error bars shown consistently above or above and below, and having n's reported or not on the graph itself.

This has been done. This has been done.

-It would strengthen the case that Trem2 acted as a PS receptor if the authors could show in Fig 1G that Trem2 KO microglia preferentially reduced phagocytosis of the PS99 liposomes, while having less effect on the PS20 liposome uptake. The authors could also support this by repeating the experiment in Fig 1H with Trem2 KO microglia. In this case, AnnexinV should have no effect on uptake in Trem2 KO microglia if it is specifically recognizing PS as the authors suggest.

This has been done (see revised Fig. 1).

-The authors do describe this in more detail in the methods but the statement in the legend for Fig 1 that "amount of lysosomes internalized by microglia and internalized through CD68 co-localization" could be clarified.


## This has been done (methods, page 22).

-The n's in Fig 2B and 2D seem to be switched on the bars of the graph relative to the legend.

This has been corrected.

-From the experiments in Fig 2, is it known whether co-culturing with microglia is required for the presence of ANXV+ labeling in neurons? It would be interesting to know whether, in addition to removing the synapses, whether the presence of microglia is also required for induction of PS exposure at synapses.


## We have the evidence that this could in fact be the case (see figure for Reviewer)

[ Figure for referees not shown.]. These data need to be confirmed. Also, we are investigating whether Trem2 KO microglia are playing a similar role. We believe therefore that these observations should be the starting point for a future study.

-In Figure 2, the authors find no difference in PS association with pre-vs post-synaptic elements, and yet they find a clear preferential association with presynaptic components in vivo. Could the authors speculate on why this might be the case?

We do not know exactly why this occurs. It could be that primary cultures are not sufficiently differentiated to display such a specificity.

-In the legend for Fig 2, it states that spines were analyzed in HN neurons alone, but this does not appear to be present on the graphs in Fig 2B-D. If these data exist, it would be helpful to include. Otherwise, the authors cannot claim in the text that ANXV had no effect alone. Morini et al, Immunity 2018, FIG. 3G).


## There was an error in legend we have now corrected. We changed HN neurons alone with HN +annexin as in the graph. The spine analysis in the condition HN alone is shown in supplemental fig 1C and is in line with previous papers (Filipello &

-In the methods related to Fig 2, it states that 15 min pretreatment of Annexin V was effective block PS 24 hours later which is surprising. Certainly the differences are clear, but did you validate that at this time point PS continues to be blocked?

We do not have a time course of the process, however we provide a novel picture showing the presence of annexin at dendritic spines 24 hours later, while being engulfed by microglia. We have added this new figure as panel 2F and described at page 6.

-In Fig 6H, if PS is serving as a cue to promote internalization of inputs, why would there be a difference in the percent of engulfed ipsilateral vs contralateral inputs that were PS+? I recognize that ipsilateral inputs are more likely to be engulfed, and thus outside of microglia, should exhibit higher PS levels (as in Fig 6G), but once they're engulfed, based on your model, wouldn't you expect both input types be equally PS+?

Even for the engulfed ipsi inputs, we only see that ~40% are co-localized with PS probe PSVue. We can only speculate on why that is: it may be possible that some of the inputs are engulfed before the probe is able to bind, and all did in fact have exposed PS at the time of engulfment. Alternatively, following injection, there may be further PS exposure that occurs which is not labeled as the probe has already bound, and thus inputs that are engulfed afterward are not labeled. It may also be that there are additional cues (loss of the 'dont eat me signal' CD47, for example), which could occur independent of PS exposure. As we observed a higher percentage of the ipsi inputs that co-localized with the PS probe, we interpret this finding as PS exposure occurring more frequently on these inputs, leading to increased likelihood of PSVue binding, engulfment, and subsequent co-localization when internalized. We clarified these points within the text in the Discussion (page 15-16).

- In Fig 7, does the ePS in the title refer to PSVue+?

This has been changed.

-As the authors state in the discussion, there are many receptors, even beyond Trem2 and C1q that can mediate engulfment of substrates that are PS+. Could the authors speculate on why these other receptors fail to compensate for the removal of excess PS+ substrates in C1q KO mice We address this on pages 16-17.


## Reviewer#2:

Major Concerns: 1.The authors beautifully show that PS+ particles are engulfed by microglia, that PS co-localizes with synapses, and that blocking PS decreases engulfment. A key question remains: are the engulfed PS+ puncta shown in the dLGN synapses? Under typical circumstances, one would probably ask the authors to add evidence of synaptic markers and colocalization with PS within microglia as proof that the particles are not cellular debris. During these times, however, I will recommend the authors point out this caveat in their conclusions and discussion. discussion (page 16-17). Please see our response to reviewer #1 points 6 and 7. Regarding quantification of PS+ synapses inside microglia: Unfortunately it has been technically challenging to identify and validate engulfed synaptic proteins by immunohistochemistry (vs genetic or fluorescent tracer), in particular presynaptic proteins, in vivo. An on-going project in the lab has been to identify and validate engulfed synaptic proteins utilizing FACS in live microglia. We have identified potential candidates that have now been validated using KO controls, however this has not yet been tested in fixed tissue through IHC engulfment analysis. There is a recent publication (Werneburg et al, 2020) demonstrating vGluT2 engulfment analysis in microglia, however we would need to further validate in our hands that we observed reproducible results.


## This is an excellent point and these caveats and alternate interpretations are now discussed in the

2.Likewise, though it is known that TREM2 binds PS and may be involved in synapse elimination, the authors provide some but incomplete evidence that these two properties are linked in developmental synapse elimination in vivo. In the discussion, the authors could highlight generally their thoughts on experiments that might more conclusively tie these together and also the degree to which these multiple phagocytic receptor pathways may all contribute to synapse elimination (for example, in the in vitro Trem2 KO studies there is considerable residual engulfment). Given this, I would also recommend softening the language "we identify a role of synaptic PS exposure in BOTH C1q and TREM2-mediated microglial elimination."

We incorporated the Reviewer's suggestion within the discussion on pages 16-17.

Minor concerns: 1.Given that most of the data is dependent on imaging analysis, it is important to avoid pseudoreplication. Individual observations of a cell in tissue culture experiment or a section of a brain should not be treated as replicates for statistical purposes. Seeing the data points is very helpful, but for statistical analysis would recommend different handling (i.e. nested testing, error propagation, or mixed modeling). This applies to the analyses in Fig. 1 F, G, H, Fig. 2 A, B, C, D, Fig. 3 C, D, F. Please comment on how the tests were applied (and the meaning of N) in the methods. ''N''. Fig. 3 has been changed and data are now shown as "number of mice" (N). Concerning Fig. 1 and 2 (page 22 and 29).We have also clarified and checked all the statistical analysis as suggested by the Reviewer. Fig. 4 C and Fig. 6 B, C, F, G, H be tested using paired t-tests because the underlying data is matched within a sample.


## As pointed out to Reviewer 1, in all figure legends the number of observations has been indicated with ''n'' and the number of biological replicates (mice) with


## , the evaluation is done on a single cell basis. The number of recorded cells (n) is indicated, while clearly indicating the number of cultures (N) these observations derive from. This is now clarified in the Methods section


## 2.Recommend that results in


## Thank you for pointing this out. Paired t-tests were performed as recommended and figures have been updated

3.Please elaborate in the methods how the samples were blinded (i.e. what does when "when appropriate" mean and was blinding applied to all analyses), especially to underscore how the team limited sampling bias in the colocalization experiments in vivo. Please also provide more information on the algorithms you used for image segmentation in Ilastik. Was the same classification model used for all images or was a new model trained to segment each experiment or image?

This has been done.

4. Since the analyses are largely volume-based: are there differences in cell volume or CD68 volume in, for example, the Trem2-/-cells as compared with controls?

These data have been added as Suppl Fig. 1A and described in the text (page 5) 2nd Jun 2020 1st Revision -Editorial Decision Dear Michela and Beth, Thank you for submitting your revised manuscript to The EMBO Journal. I have now had a chance to take a careful look at everything and I appreciate the introduced changes.

I am therefore very happy to accept the MS for publication here. Before doing so there are just a few editorial points to address. Please also correct the callout to the figures in the text -Can you please confirm that there is an image in S3 PSVue panel -I would like to encourage you to provide the data points from the graphs in excel format as source data. I think this will make everything more transparent also as there were some questions raised regarding the stats. Please assemble one excel file per figure and label that as Source Data Fig X  etc. The files will be published online with the article as supplementary "Source Data" files.

-We include a synopsis of the paper (see http://emboj.embopress.org/). Please provide me with a general summary statement and 3-5 bullet points that capture the key findings of the paper.

-We also need a summary figure for the synopsis. The size should be 550 wide by 400 high (pixels). You can also use something from the figures if that is easier.

-I have asked our publisher to do their pre-publication checks on the paper. They will send me the file within the next few days. Please wait to upload the revised version until you have received their comments.

-please also upload a point-by-point response when you re-submit the revised version. Please check that the title and abstract of the manuscript are brief, yet explicit, even to nonspecialists.

When assembling figures, please refer to our figure preparation guideline in order to ensure proper formatting and readability in print as well as on screen: http://bit.ly/EMBOPressFigurePreparationGuideline IMPORTANT: When you send the revision we will require -a point-by-point response to the referees' comments, with a detailed description of the changes made (as a word file).

-a word file of the manuscript text.

-individual production quality figure files (one file per figure) -a complete author checklist, which you can download from our author guidelines (https://www.embopress.org/page/journal/14602075/authorguide). -Expanded View files (replacing Supplementary Information) Please see out instructions to authors https://www.embopress.org/page/journal/14602075/authorguide#expandedview Please remember: Digital image enhancement is acceptable practice, as long as it accurately represents the original data and conforms to community standards. If a figure has been subjected to significant electronic manipulation, this must be noted in the figure legend or in the 'Materials and Methods' section. The editors reserve the right to request original versions of figures and the original images that were used to assemble the figure.

Further information is available in our Guide For Authors: https://www.embopress.org/page/journal/14602075/authorguide The revision must be submitted online within 90 days; please click on the link below to submit the revision online before 31st Aug 2020. -I would like to encourage you to provide the data points from the graphs in excel format as source data. I think this will make everything more transparent also as there were some questions raised regarding the stats. Please assemble one excel file per figure and label that as Source Data Fig X etc. The files will be published online with the article as supplementary "Source Data" files.

We are now providing data in excel format as source data -We include a synopsis of the paper (see http://emboj.embopress.org/). Please provide me with a general summary statement and 3-5 bullet points that capture the key findings of the paper.


## Please find below a general summary statement and 4 bullet points

Microglia help refine developing neural circuits through the elimination of supernumerary synapses. Here we show that exposed phosphatidylserine on pre-and postsynaptic membranes functions as an 'eat-me' signal contributing to microglial-mediated synapse pruning. Phosphatidylserine exposure at both hippocampal and retinogeniculate synapses coincides with the onset of synapse elimination and PS engulfment by microglia.

-Microglia-mediated synapse elimination is dependent on TREM2 and exposed phosphatidylserine in vitro.

-Exposed phosphatidylserine is developmentally regulated across periods of pruning in both hippocampus and visual system. -In vivo developmental phosphatidylserine exposure is not caspase 3-dependent.

-Loss of C1q leads to elevated phosphatidylserine-positive presynaptic inputs and reduced microglia engulfment.

-We also need a summary figure for the synopsis. The size should be 550 wide by 400 high (pixels 15th Jun 2020 2nd Revision -Editorial Decision Dear Beth and Michela, Thank you for submitting your revised manuscript to The EMBO Journal. I have now had a chance to take a careful look at everything and all looks good. I am therefore very pleased to accept the manuscript for publication here. @Beth will you please link your ORCID ID to your account -please see below for how to do so. We can unfortunately not do this for you because of authentication issues. -In order to link your ORCID iD to your account in our manuscript tracking system, please do the following:

1. Click the 'Modify Profile' link at the bottom of your homepage in our system. 2. On the next page you will see a box half-way down the page titled ORCID*. Below this box is red text reading 'To Register/Link to ORCID, click here'. Please follow that link: you will be taken to ORCID where you can log in to your account (or create an account if you don't have one) 3. You will then be asked to authorise Wiley to access your ORCID information. Once you have approved the linking, you will be brought back to our manuscript system.

We regret that we cannot do this linking on your behalf for security reasons. We also cannot add your ORCID iD number manually to our system because there is no way for us to authenticate this iD number with ORCID.

-Please note that it is EMBO Journal policy for the transcript of the editorial process (containing referee reports and your response letter) to be published as an online supplement to each paper. If you do NOT want this, you will need to inform the Editorial Office via email immediately. More information is available here: http://emboj.embopress.org/about#Transparent_Process -Your manuscript will be processed for publication in the journal by EMBO Press. Manuscripts in the PDF and electronic editions of The EMBO Journal will be copy edited, and you will be provided with page proofs prior to publication. Please note that supplementary information is not included in the proofs.

-Should you be planning a Press Release on your article, please get in contact with embojournal@wiley.com as early as possible, in order to coordinate publication and release dates. Do the data meet the assumptions of the tests (e.g., normal distribution)? Describe any methods used to assess it.

Is there an estimate of variation within each group of data?

Is the variance similar between the groups that are being statistically compared?

At least 3 animal were included for each condition No data were excluded from analysis.

Animals were selected randomly before any treatment and samples were manipulated blind to genotype and treatment. Confocal images were quantified in an automated unbiased method by using imaging softwares.


## Manuscript Number: EMBOJ-2020-105380R

Yes, to the best of our knowledge. Distribution of data was assessed by D'agostino-Pearson omnibus normality test or Shapiro-Wilk normality test by using GraphPad Prism software. Then parametric tests were used for normally distributed data and non-parametric tests were used in cases of not normally distributed data, as clearly stated in figure legends.

D'Agostino & Pearson normality test and Shapiro-Wilk normality test to test normal distribtion. Parametric or non-parametric statistical tests were adopted depending on sample distribution.

Yes, for each group of data, mean +-SEM are provided in figure legends.

Yes, as you may appreciate by SEM given in the figures.

Male and female mice were included in all experiments and randomly selected for in vivo and ex vivo experiments.

All experiments and analysis were performed blind to genotype and condition All experiments and analysis were performed blind to genotype 1. Data the data were obtained and processed according to the field's best practice and are presented to reflect the results of the experiments in an accurate and unbiased manner. figure panels include only data points, measurements or observations that can be compared to each other in a scientifically meaningful way.

The data shown in figures should satisfy the following conditions:

Source Data should be included to report the data underlying graphs. Please follow the guidelines set out in the author ship guidelines on Data Presentation.

Please fill out these boxes Ãª (Do not worry if you cannot see all your text once you press return) a specification of the experimental system investigated (eg cell line, species name).

There is no sample size calculation stated in the text. However our sample sizes are similar or larger than similar studies in the field and all statistical tests were selected as appropriate to the sample sizes.

graphs include clearly labeled error bars for independent experiments and sample sizes. Unless justified, error bars should not be shown for technical replicates. if n< 5, the individual data points from each experiment should be plotted and any statistical test employed should be justified the exact sample size (n) for each experimental group/condition, given as a number, not a range;

Each figure caption should contain the following information, for each panel where they are relevant:


## Captions


## C-Reagents


## B-Statistics and general methods

the assay(s) and method(s) used to carry out the reported observations and measurements an explicit mention of the biological and chemical entity(ies) that are being measured. an explicit mention of the biological and chemical entity(ies) that are altered/varied/perturbed in a controlled manner.

a statement of how many times the experiment shown was independently replicated in the laboratory.

Any descriptions too long for the figure legend should be included in the methods section and/or with the source data.

In the pink boxes below, please ensure that the answers to the following questions are reported in the manuscript itself. Every question should be answered. If the question is not relevant to your research, please write NA (non applicable). We encourage you to include a specific subsection in the methods section for statistics, reagents, animal models and human subjects.

definitions of statistical methods and measures:

a description of the sample collection allowing the reader to understand whether the samples represent technical or biological replicates (including how many animals, litters, cultures, etc.).


## EMBO PRESS


## A-Figures


## Reporting Checklist For Life Sciences Articles (Rev. June 2017)

This checklist is used to ensure good reporting standards and to improve the reproducibility of published results. These guidelines are consistent with the Principles and Guidelines for Reporting Preclinical Research issued by the NIH in 2014. Please follow the journal's authorship guidelines in preparing your manuscript.


, B, C, D, Fig. 3 C, D, F. Please comment on how the tests were applied (and the meaning of N) in the methods.


each mouse, several brain sections have been analyzed. In all figure legends the number of observations has been indicated with ''n'' and the number of biological replicates (mice) with ''N''.Fig. 3has been changed and data are now shown as "number of mice" (N). RegardingFig. 2, the situation is rather different. Electrophysiology recordings are always shown with each recorded cell appearing as a single observation (n). Therefore, in line with the entire scientific literature, we maintained the number of cells (n), while clearly indicating the number of cultures (N) these observations derive from. The same apply toFig. 1, where the evaluation of liposome uptake is done on a single cell basis. The type of analysis (performed on the volume of each single cell) inFig. 1 has been better explained in the text (methods, page 22 and 29).

## -
It would be helpful for the authors to include the Trem2 KO data with Annexin V in the figure legend of Fig 1 E, H into the figure itself. These data are important for the author's overall conclusions and warrant inclusion in the figure.

## -## As Reviewer 1 noted, TREM2 deficiency only reduces about 50% of PS+ liposomes. Based also on minor point of Reviewer 1 (It would strengthen the case that Trem2 acted as a PS receptor if the authors could show in Fig 1G that Trem2 KO microglia preferentially reduced phagocytosis of the PS99 liposomes, while having less effect on the PS20 liposome uptake. The authors could also support this by repeating the experiment in Fig 1H with Trem2 KO microglia. In this case, AnnexinV should have no effect on uptake in Trem2 KO microglia if it is specifically recognizing PS as the authors suggest): We added to

3) It is not clear to me what the individual data points represent. I assume that in this and other figures the authors quantify and calculate statistical significance based on the number of cells or image fields instead of experimental replicates. They state that 3 independent experiments were performed, and they artificially increase the experimental N by using repeated measures from the same animal, constituting pseudoreplication.  Fig 2F, similar to what the authors report later in vivo, there seems to be many examples of ANXV localization not with BSN or SHANK2. Perhaps it would be helpful to quantify the percent of ANXV that is co-localized with these synaptic markers to get a sense of the extent to which PS is synaptic. The images are at too low magnification to be able to really objectively evaluate the extent of colocalization.


## 4) In


## The percentage of annexin colocalized with synaptic markers in culture is around 10% (with peaks up to 20%), see figure for

Reviewer. This is expected, as neurons in culture may undergo degeneration and thus result as PS+. However the functional data obtained in co-cultures (electrophysiology, synapse count) indicate that cloaking PS is enough to largely prevent synapse elimination in culture. It is likely that additional cues (besides PS) provide the specificity for synapse elimination and this has been now discussed (pages 16-17). In order to provide a more reliable answer to the point raised by the Reviewer, we moved to brain sections, where nonspecific cell death is expected to be lower, and calculated the percentage of PSVue colocalized with synaptic markers. This turned out to be around 20% (with peaks up to 30%) We do not believe these data need to be added to the text, but we are willing to do that, if needed. Furthermore, a higher magnification for Fig. 2I has now been provided.

5) The authors clearly establish that PS is present on synapses during developmental time points when refinement occurs and that microglia do engulf PS+ material. However, I am not convinced that PS specifically labels synapses that should be removed. What if PS is simply present on immature synapses?

Although we have evidence that, differently from mature synapses, i) filopodia are not eliminated when hippocampal neurons are co-cultured together with microglia and ii) preincubation with annexin does not impact the process (see figure for Reviewer), we cannot exclude that PS is also present on immature synapses (some that will not be eliminated). We modified the discussion (pages 15) to mention that we cannot rule out that PS exposure occurs on immature synapses, and this is something that should be further investigated. 6) Since it stated in the methods that the authors collected data from regions within the LGN that will ultimately be refined to contain ipsilateral and contralateral inputs, a stronger analysis would be to look at the percent of PS+ ipsilateral inputs within the contralateral region, relative to PS+ contralateral inputs. And also the percent of PS+ contralateral inputs relative to ipsilateral inputs within the ipsilateral region. Even this would not definitively demonstrate that PS labels synapses that should be engulfed, however. It would be suggestive if TTX injection into one eye led to preferential increases in PS exposure on synapses from that eye. I recognize that this is a difficult point to address, but is essential to the author's model that PS serves to label synapses that should undergo engulfment.

At P5, the time point we analyzed, there is still considerable overlap between the ipsilateral and contralateral inputs, as the dLGN is actively undergoing eye segregation. Since the ipsi patch is not yet well defined, we see inputs from both eyes localized throughout the LGN. In the analysis, we tried to get representative images that contained both inputs. We could repeat these experiments in P10 animals, where there is a more well-defined ipsi patch, to test whether the remaining PS signal we see is differentially expressed in ipsi vs. contra; however this may be difficult to interpret as eye segregation is mostly completed and microglia engulfment is significantly decreased. We are interested in performing experiments to determine the effect of activity on neuronal PS exposure but we feel this is beyond the scope of the current study as it will require several approaches to manipulate activity in one eye, as TTX injection could have other potential side effects on cell health that make interpretation difficult. For example, in preliminary pilot experiments, we observed an increase in cell death following eye injection, and as PSVue also labels dying cells, differentiating the signal from local synaptic PS vs. apoptotic cells may be difficult. We are continuing to pursue how activity changes may affect PS exposure and engulfment, however, using genetic approaches. Unfortunately these experiments are not ready for publication at this time. Concerning quantification of PS+ synapses inside microglia: Unfortunately it has been technically challenging in the field to identify and validate engulfed synaptic proteins by immunohistochemistry, in particular presynaptic proteins, in vivo (fixed brain tissue). An on-going project in the lab has been to identify and validate synaptic antibodies to quantify engulfed synaptic proteins utilizing FACS in live microglia. We have identified potential candidates that have now been validated using KO controls, however this has not yet been tested in fixed tissue through IHC engulfment analysis. There is a recent publication (Werneburg et al, 2020) demonstrating vGluT2 engulfment analysis in microglia within models of Multiple Sclerosis across several species, however we would need to further validate in our hands at developmental time points. Given these and other technical challenges, we have modified our claims and addressed these caveats and issues in the Discussion, on page 16-17.


## 7)

Based on the author's data, it is not clear whether microglia preferentially engulf PS+ synapses relative to PS-ones, which is essential for their argument that PS serves a cue to promote engulfment of particular synaptic targets. The authors could address this using their data in Fig 6D-E by quantifying the percent of engulfed synaptic material that is also PS+. If PS is a cue to direct engulfment, then a substantial percentage of engulfed material should be PS+. Or the authors could compare the percentage of PS+ inputs outside of microglia vs the PS+ inputs inside of microglia, relative to the percentage of PS-inputs outside of microglia vs PS-inputs inside of microglia. There should be a higher proportion of PS+ inputs inside microglia compared to PS-inputs if it does indeed serve as a directive 'eat me' cue.

Without performing live imaging to follow synapses that do or do not expose PS we do not think it is possible to quantify PS-bound to synaptic elements within microglia in fixed brain tissue given the dynamics of this process and technical issues related to antibodies (as discussed above). Because we are not looking across a time course, we cannot know which synapses exposed PS and were engulfed and which did not. There is also no way to quantify total "synapses engulfed". Further, because astrocytes have been shown to contribute to developmental synaptic pruning, it is likely that they also engulf PS positive inputs. We have some preliminary data demonstrating that astrocytes also engulf PS-labeled material at P5 in the dLGN, so trying to measure microglia PS+ vs -in fixed tissues would not reflect that aspect. Fig 7 to show that C1q impairs the uptake specifically of PS+ synaptic inputs. In further support of the author's model that PS is a cue that allows C1q binding, then they could quantify the percent of C1q+ synapses that are also PS+. If PS is required for C1q association with synapses, then this should be close to 100%. It is presently an over-interpretation of the data presented in Fig 7 to say "All together, these data... suggest that synaptic PS exposure may act as the signal leading to C1q deposition during pruning"


## 8) A similar analysis should be performed in

Thank you for this point and suggestion. We performed several analyses of our data and included the new data, which can be found in revised Fig 7. We calculated the percentage of vGluT2 inputs that colocalized with C1q across each time point (P2, 5, and 10), and found that this did not change (Fig 7B). We further calculated the percentage of C1q+ vGluT2 inputs that were also positive for PS, which peaked at P5 at ~8% (Fig 7C). Based on these new results, we updated the text to clarify our findings in Results (page 11-12).

Minor comments: -It would be helpful to make all graphs consistent with one another to be able to more clearly compare experiments performed across figures, for example showing each replicate as a dot or the summary data or both, having error bars shown consistently above or above and below, and having n's reported or not on the graph itself.

This has been done. This has been done.

-It would strengthen the case that Trem2 acted as a PS receptor if the authors could show in Fig 1G that Trem2 KO microglia preferentially reduced phagocytosis of the PS99 liposomes, while having less effect on the PS20 liposome uptake. The authors could also support this by repeating the experiment in Fig 1H with Trem2 KO microglia. In this case, AnnexinV should have no effect on uptake in Trem2 KO microglia if it is specifically recognizing PS as the authors suggest.

This has been done (see revised Fig. 1).

-The authors do describe this in more detail in the methods but the statement in the legend for Fig 1 that "amount of lysosomes internalized by microglia and internalized through CD68 co-localization" could be clarified.


## This has been done (methods, page 22).

-The n's in Fig 2B and 2D seem to be switched on the bars of the graph relative to the legend.

This has been corrected.

-From the experiments in Fig 2, is it known whether co-culturing with microglia is required for the presence of ANXV+ labeling in neurons? It would be interesting to know whether, in addition to removing the synapses, whether the presence of microglia is also required for induction of PS exposure at synapses.


## We have the evidence that this could in fact be the case (see figure for Reviewer)

[ Figure for referees not shown.]. These data need to be confirmed. Also, we are investigating whether Trem2 KO microglia are playing a similar role. We believe therefore that these observations should be the starting point for a future study.

-In Figure 2, the authors find no difference in PS association with pre-vs post-synaptic elements, and yet they find a clear preferential association with presynaptic components in vivo. Could the authors speculate on why this might be the case?

We do not know exactly why this occurs. It could be that primary cultures are not sufficiently differentiated to display such a specificity.

-In the legend for Fig 2, it states that spines were analyzed in HN neurons alone, but this does not appear to be present on the graphs in Fig 2B-D. If these data exist, it would be helpful to include. Otherwise, the authors cannot claim in the text that ANXV had no effect alone. Morini et al, Immunity 2018, FIG. 3G).


## There was an error in legend we have now corrected. We changed HN neurons alone with HN +annexin as in the graph. The spine analysis in the condition HN alone is shown in supplemental fig 1C and is in line with previous papers (Filipello &

-In the methods related to Fig 2, it states that 15 min pretreatment of Annexin V was effective block PS 24 hours later which is surprising. Certainly the differences are clear, but did you validate that at this time point PS continues to be blocked?

We do not have a time course of the process, however we provide a novel picture showing the presence of annexin at dendritic spines 24 hours later, while being engulfed by microglia. We have added this new figure as panel 2F and described at page 6.

-In Fig 6H, if PS is serving as a cue to promote internalization of inputs, why would there be a difference in the percent of engulfed ipsilateral vs contralateral inputs that were PS+? I recognize that ipsilateral inputs are more likely to be engulfed, and thus outside of microglia, should exhibit higher PS levels (as in Fig 6G), but once they're engulfed, based on your model, wouldn't you expect both input types be equally PS+?

Even for the engulfed ipsi inputs, we only see that ~40% are co-localized with PS probe PSVue. We can only speculate on why that is: it may be possible that some of the inputs are engulfed before the probe is able to bind, and all did in fact have exposed PS at the time of engulfment. Alternatively, following injection, there may be further PS exposure that occurs which is not labeled as the probe has already bound, and thus inputs that are engulfed afterward are not labeled. It may also be that there are additional cues (loss of the 'dont eat me signal' CD47, for example), which could occur independent of PS exposure. As we observed a higher percentage of the ipsi inputs that co-localized with the PS probe, we interpret this finding as PS exposure occurring more frequently on these inputs, leading to increased likelihood of PSVue binding, engulfment, and subsequent co-localization when internalized. We clarified these points within the text in the Discussion (page 15-16).

- In Fig 7, does the ePS in the title refer to PSVue+?

This has been changed.

-As the authors state in the discussion, there are many receptors, even beyond Trem2 and C1q that can mediate engulfment of substrates that are PS+. Could the authors speculate on why these other receptors fail to compensate for the removal of excess PS+ substrates in C1q KO mice We address this on pages 16-17.


## Reviewer#2:

Major Concerns: 1.The authors beautifully show that PS+ particles are engulfed by microglia, that PS co-localizes with synapses, and that blocking PS decreases engulfment. A key question remains: are the engulfed PS+ puncta shown in the dLGN synapses? Under typical circumstances, one would probably ask the authors to add evidence of synaptic markers and colocalization with PS within microglia as proof that the particles are not cellular debris. During these times, however, I will recommend the authors point out this caveat in their conclusions and discussion. discussion (page 16-17). Please see our response to reviewer #1 points 6 and 7. Regarding quantification of PS+ synapses inside microglia: Unfortunately it has been technically challenging to identify and validate engulfed synaptic proteins by immunohistochemistry (vs genetic or fluorescent tracer), in particular presynaptic proteins, in vivo. An on-going project in the lab has been to identify and validate engulfed synaptic proteins utilizing FACS in live microglia. We have identified potential candidates that have now been validated using KO controls, however this has not yet been tested in fixed tissue through IHC engulfment analysis. There is a recent publication (Werneburg et al, 2020) demonstrating vGluT2 engulfment analysis in microglia, however we would need to further validate in our hands that we observed reproducible results.


## This is an excellent point and these caveats and alternate interpretations are now discussed in the

2.Likewise, though it is known that TREM2 binds PS and may be involved in synapse elimination, the authors provide some but incomplete evidence that these two properties are linked in developmental synapse elimination in vivo. In the discussion, the authors could highlight generally their thoughts on experiments that might more conclusively tie these together and also the degree to which these multiple phagocytic receptor pathways may all contribute to synapse elimination (for example, in the in vitro Trem2 KO studies there is considerable residual engulfment). Given this, I would also recommend softening the language "we identify a role of synaptic PS exposure in BOTH C1q and TREM2-mediated microglial elimination."

We incorporated the Reviewer's suggestion within the discussion on pages 16-17.

Minor concerns: 1.Given that most of the data is dependent on imaging analysis, it is important to avoid pseudoreplication. Individual observations of a cell in tissue culture experiment or a section of a brain should not be treated as replicates for statistical purposes. Seeing the data points is very helpful, but for statistical analysis would recommend different handling (i.e. nested testing, error propagation, or mixed modeling). This applies to the analyses in Fig. 1 F, G, H, Fig. 2 A, B, C, D, Fig. 3 C, D, F. Please comment on how the tests were applied (and the meaning of N) in the methods. ''N''. Fig. 3 has been changed and data are now shown as "number of mice" (N). Concerning Fig. 1 and 2 (page 22 and 29).We have also clarified and checked all the statistical analysis as suggested by the Reviewer. Fig. 4 C and Fig. 6 B, C, F, G, H be tested using paired t-tests because the underlying data is matched within a sample.


## As pointed out to Reviewer 1, in all figure legends the number of observations has been indicated with ''n'' and the number of biological replicates (mice) with


## , the evaluation is done on a single cell basis. The number of recorded cells (n) is indicated, while clearly indicating the number of cultures (N) these observations derive from. This is now clarified in the Methods section


## 2.Recommend that results in


## Thank you for pointing this out. Paired t-tests were performed as recommended and figures have been updated

3.Please elaborate in the methods how the samples were blinded (i.e. what does when "when appropriate" mean and was blinding applied to all analyses), especially to underscore how the team limited sampling bias in the colocalization experiments in vivo. Please also provide more information on the algorithms you used for image segmentation in Ilastik. Was the same classification model used for all images or was a new model trained to segment each experiment or image?

This has been done.

4. Since the analyses are largely volume-based: are there differences in cell volume or CD68 volume in, for example, the Trem2-/-cells as compared with controls?

These data have been added as Suppl Fig. 1A and described in the text (page 5) 2nd Jun 2020 1st Revision -Editorial Decision Dear Michela and Beth, Thank you for submitting your revised manuscript to The EMBO Journal. I have now had a chance to take a careful look at everything and I appreciate the introduced changes.

I am therefore very happy to accept the MS for publication here. Before doing so there are just a few editorial points to address. Please also correct the callout to the figures in the text -Can you please confirm that there is an image in S3 PSVue panel -I would like to encourage you to provide the data points from the graphs in excel format as source data. I think this will make everything more transparent also as there were some questions raised regarding the stats. Please assemble one excel file per figure and label that as Source Data Fig X  etc. The files will be published online with the article as supplementary "Source Data" files.

-We include a synopsis of the paper (see http://emboj.embopress.org/). Please provide me with a general summary statement and 3-5 bullet points that capture the key findings of the paper.

-We also need a summary figure for the synopsis. The size should be 550 wide by 400 high (pixels). You can also use something from the figures if that is easier.

-I have asked our publisher to do their pre-publication checks on the paper. They will send me the file within the next few days. Please wait to upload the revised version until you have received their comments.

-please also upload a point-by-point response when you re-submit the revised version. Please check that the title and abstract of the manuscript are brief, yet explicit, even to nonspecialists.

When assembling figures, please refer to our figure preparation guideline in order to ensure proper formatting and readability in print as well as on screen: http://bit.ly/EMBOPressFigurePreparationGuideline IMPORTANT: When you send the revision we will require -a point-by-point response to the referees' comments, with a detailed description of the changes made (as a word file).

-a word file of the manuscript text.

-individual production quality figure files (one file per figure) -a complete author checklist, which you can download from our author guidelines (https://www.embopress.org/page/journal/14602075/authorguide). -Expanded View files (replacing Supplementary Information) Please see out instructions to authors https://www.embopress.org/page/journal/14602075/authorguide#expandedview Please remember: Digital image enhancement is acceptable practice, as long as it accurately represents the original data and conforms to community standards. If a figure has been subjected to significant electronic manipulation, this must be noted in the figure legend or in the 'Materials and Methods' section. The editors reserve the right to request original versions of figures and the original images that were used to assemble the figure.

Further information is available in our Guide For Authors: https://www.embopress.org/page/journal/14602075/authorguide The revision must be submitted online within 90 days; please click on the link below to submit the revision online before 31st Aug 2020. -I would like to encourage you to provide the data points from the graphs in excel format as source data. I think this will make everything more transparent also as there were some questions raised regarding the stats. Please assemble one excel file per figure and label that as Source Data Fig X etc. The files will be published online with the article as supplementary "Source Data" files.

We are now providing data in excel format as source data -We include a synopsis of the paper (see http://emboj.embopress.org/). Please provide me with a general summary statement and 3-5 bullet points that capture the key findings of the paper.


## Please find below a general summary statement and 4 bullet points

Microglia help refine developing neural circuits through the elimination of supernumerary synapses. Here we show that exposed phosphatidylserine on pre-and postsynaptic membranes functions as an 'eat-me' signal contributing to microglial-mediated synapse pruning. Phosphatidylserine exposure at both hippocampal and retinogeniculate synapses coincides with the onset of synapse elimination and PS engulfment by microglia.

-Microglia-mediated synapse elimination is dependent on TREM2 and exposed phosphatidylserine in vitro.

-Exposed phosphatidylserine is developmentally regulated across periods of pruning in both hippocampus and visual system. -In vivo developmental phosphatidylserine exposure is not caspase 3-dependent.

-Loss of C1q leads to elevated phosphatidylserine-positive presynaptic inputs and reduced microglia engulfment.

-We also need a summary figure for the synopsis. The size should be 550 wide by 400 high (pixels 15th Jun 2020 2nd Revision -Editorial Decision Dear Beth and Michela, Thank you for submitting your revised manuscript to The EMBO Journal. I have now had a chance to take a careful look at everything and all looks good. I am therefore very pleased to accept the manuscript for publication here. @Beth will you please link your ORCID ID to your account -please see below for how to do so. We can unfortunately not do this for you because of authentication issues. -In order to link your ORCID iD to your account in our manuscript tracking system, please do the following:

1. Click the 'Modify Profile' link at the bottom of your homepage in our system. 2. On the next page you will see a box half-way down the page titled ORCID*. Below this box is red text reading 'To Register/Link to ORCID, click here'. Please follow that link: you will be taken to ORCID where you can log in to your account (or create an account if you don't have one) 3. You will then be asked to authorise Wiley to access your ORCID information. Once you have approved the linking, you will be brought back to our manuscript system.

We regret that we cannot do this linking on your behalf for security reasons. We also cannot add your ORCID iD number manually to our system because there is no way for us to authenticate this iD number with ORCID.

-Please note that it is EMBO Journal policy for the transcript of the editorial process (containing referee reports and your response letter) to be published as an online supplement to each paper. If you do NOT want this, you will need to inform the Editorial Office via email immediately. More information is available here: http://emboj.embopress.org/about#Transparent_Process -Your manuscript will be processed for publication in the journal by EMBO Press. Manuscripts in the PDF and electronic editions of The EMBO Journal will be copy edited, and you will be provided with page proofs prior to publication. Please note that supplementary information is not included in the proofs.

-Should you be planning a Press Release on your article, please get in contact with embojournal@wiley.com as early as possible, in order to coordinate publication and release dates. Do the data meet the assumptions of the tests (e.g., normal distribution)? Describe any methods used to assess it.

Is there an estimate of variation within each group of data?

Is the variance similar between the groups that are being statistically compared?

At least 3 animal were included for each condition No data were excluded from analysis.

Animals were selected randomly before any treatment and samples were manipulated blind to genotype and treatment. Confocal images were quantified in an automated unbiased method by using imaging softwares.


## Manuscript Number: EMBOJ-2020-105380R

Yes, to the best of our knowledge. Distribution of data was assessed by D'agostino-Pearson omnibus normality test or Shapiro-Wilk normality test by using GraphPad Prism software. Then parametric tests were used for normally distributed data and non-parametric tests were used in cases of not normally distributed data, as clearly stated in figure legends.

D'Agostino & Pearson normality test and Shapiro-Wilk normality test to test normal distribtion. Parametric or non-parametric statistical tests were adopted depending on sample distribution.

Yes, for each group of data, mean +-SEM are provided in figure legends.

Yes, as you may appreciate by SEM given in the figures.

Male and female mice were included in all experiments and randomly selected for in vivo and ex vivo experiments.

All experiments and analysis were performed blind to genotype and condition All experiments and analysis were performed blind to genotype 1. Data the data were obtained and processed according to the field's best practice and are presented to reflect the results of the experiments in an accurate and unbiased manner. figure panels include only data points, measurements or observations that can be compared to each other in a scientifically meaningful way.

The data shown in figures should satisfy the following conditions:

Source Data should be included to report the data underlying graphs. Please follow the guidelines set out in the author ship guidelines on Data Presentation.

Please fill out these boxes Ãª (Do not worry if you cannot see all your text once you press return) a specification of the experimental system investigated (eg cell line, species name).

There is no sample size calculation stated in the text. However our sample sizes are similar or larger than similar studies in the field and all statistical tests were selected as appropriate to the sample sizes.

graphs include clearly labeled error bars for independent experiments and sample sizes. Unless justified, error bars should not be shown for technical replicates. if n< 5, the individual data points from each experiment should be plotted and any statistical test employed should be justified the exact sample size (n) for each experimental group/condition, given as a number, not a range;

Each figure caption should contain the following information, for each panel where they are relevant:


## Captions


## C-Reagents


## B-Statistics and general methods

the assay(s) and method(s) used to carry out the reported observations and measurements an explicit mention of the biological and chemical entity(ies) that are being measured. an explicit mention of the biological and chemical entity(ies) that are altered/varied/perturbed in a controlled manner.

a statement of how many times the experiment shown was independently replicated in the laboratory.

Any descriptions too long for the figure legend should be included in the methods section and/or with the source data.

In the pink boxes below, please ensure that the answers to the following questions are reported in the manuscript itself. Every question should be answered. If the question is not relevant to your research, please write NA (non applicable). We encourage you to include a specific subsection in the methods section for statistics, reagents, animal models and human subjects.

definitions of statistical methods and measures:

a description of the sample collection allowing the reader to understand whether the samples represent technical or biological replicates (including how many animals, litters, cultures, etc.).


## EMBO PRESS


## A-Figures


## Reporting Checklist For Life Sciences Articles (Rev. June 2017)

This checklist is used to ensure good reporting standards and to improve the reproducibility of published results. These guidelines are consistent with the Principles and Guidelines for Reporting Preclinical Research issued by the NIH in 2014. Please follow the journal's authorship guidelines in preparing your manuscript.


, B, C, D, Fig. 3 C, D, F. Please comment on how the tests were applied (and the meaning of N) in the methods.


each mouse, several brain sections have been analyzed. In all figure legends the number of observations has been indicated with ''n'' and the number of biological replicates (mice) with ''N''.Fig. 3has been changed and data are now shown as "number of mice" (N). RegardingFig. 2, the situation is rather different. Electrophysiology recordings are always shown with each recorded cell appearing as a single observation (n). Therefore, in line with the entire scientific literature, we maintained the number of cells (n), while clearly indicating the number of cultures (N) these observations derive from. The same apply toFig. 1, where the evaluation of liposome uptake is done on a single cell basis. The type of analysis (performed on the volume of each single cell) inFig. 1 has been better explained in the text (methods, page 22 and 29).

## -
It would be helpful for the authors to include the Trem2 KO data with Annexin V in the figure legend of Fig 1 E, H into the figure itself. These data are important for the author's overall conclusions and warrant inclusion in the figure.

## -
In author contribution section please distinguish between Michela and Matthew Mahoney (MMah and MMat). Author contributions are missing for Lorena Passoni -we are missing the ORCID for Beth -There is a callout to Fig 1G, but no such panel. -Please double check that there are figure callouts for Fig 4D and Fig 6I -Please remove the figure legends from the figures and add the legends at the end of the manuscript. -Supplemental Figures should be re-labelled as Expanded View figures -see our guide to authors.


https://emboj.msubmit.net/cgi-bin/main.plex ------------------------------------------------Humanitas Mirasole s.p.a. Sede legale e operativa: Capitale sociale â¬ 13.840.000 i.v. SocietÃ  soggetta Istituto Clinico Humanitas Partita IVA e Cod.Fisc. have been provided in the title page (microglia, synapse pruning, phosphatidylserine, TREM2, C1q) -In author contribution section please distinguish between Michela and Matthew Mahoney (MMah and MMat). Author contributions are missing for Lorena Passoni. These changes have been done -we are missing the ORCID for Beth Beth's ORCID: 0000-0003-4226-1201 -There is a callout to Fig 1G, but no such panel. We have deleted the callout from the text -Please double check that there are figure callouts for Fig 4D and Fig 6I We have highlighted the text callout for Fig 4D. No Fig 6I exists (Fig6H is the last panel), however there is a Fig7I, which is also highlighted just in case. -Please remove the figure legends from the figures and add the legends at the end of the manuscript. This has been done -Supplemental Figures should be re-labelled as Expanded View figures -see our guide to authors. Please also correct the callout to the figures in the text This has been done -Can you please confirm that there is an image in S3 PSVue panel Yes, both the active caspase3 and PSVue panels have images. No staining in the retina was observed for either channel.


----------------------------------------------


). You can also use something from the figures if that is easier. We have provided a summary figures for the synopsis.Humanitas Mirasole s.p.a. 

Sede legale e operativa: 
Capitale sociale â¬ 13.840.000 i.v. 
SocietÃ  soggetta 
Istituto Clinico Humanitas 
Partita IVA e Cod.Fisc. 10125410158 
a direzione e coordinamento 
Via Manzoni 56 -20089 Rozzano (Milano) 
Iscr. Tribunale Milano 10125410158 
di Humanitas s.p.a. 
Telefono 02.8224.1 -Fax 02.8224.2299 
(giÃ  310153) -7785 -3 
www.humanitas.it 
R.E.A. Milano 1352145 




** Click here to be directed to your login page: http://emboj.msubmit.net common tests, such as t-test(please specify whether paired vs. unpaired), simple Ï2 tests, Wilcoxon and Mann-Whitney tests, can be unambiguously identified by name only, but more complex techniques should be described in the methods section; are tests one-sided or two-sided? are there adjustments for multiple comparisons? exact statistical test results, e.g., P values = x but not P values < x; definition of 'center values' as median or average; definition of error bars as s.d. or s.e.m. 1.a. How was the sample size chosen to ensure adequate power to detect a pre-specified effect size? 1.b. For animal studies, include a statement about sample size estimate even if no statistical methods were used. 2. Describe inclusion/exclusion criteria if samples or animals were excluded from the analysis. Were the criteria preestablished? 3. Were any steps taken to minimize the effects of subjective bias when allocating animals/samples to treatment (e.g. randomization procedure)? If yes, please describe. For animal studies, include a statement about randomization even if no randomization was used. 4.a. Were any steps taken to minimize the effects of subjective bias during group allocation or/and when assessing results (e.g. blinding of the investigator)? If yes please describe. 4.b. For animal studies, include a statement about blinding even if no blinding was done 5. For every figure, are statistical tests justified as appropriate?
USEFUL LINKS FOR COMPLETING THIS FORM
To show that antibodies were profiled for use in the system under study (assay and species), provide a citation, catalog number and/or clone number, supplementary information or reference to an antibody validation profile. e.g., Antibodypedia (see link list at top right). 1see link list at top rightTo show that antibodies were profiled for use in the system under study (assay and species), provide a citation, catalog number and/or clone number, supplementary information or reference to an antibody validation profile. e.g., Antibodypedia (see link list at top right), 1DegreeBio (see link list at top right).

Identify the source of cell lines and report if they were recently authenticated (e.g., by STR profiling) and tested for mycoplasma contamination. Identify the source of cell lines and report if they were recently authenticated (e.g., by STR profiling) and tested for mycoplasma contamination.

. * , please see the table at the top right of the document* for all hyperlinks, please see the table at the top right of the document

Report species, strain, gender, age of animals and genetic modification status where applicable. Please detail housing and husbandry conditions and the source of animals. Report species, strain, gender, age of animals and genetic modification status where applicable. Please detail housing and husbandry conditions and the source of animals.

For experiments involving live vertebrates, include a statement of compliance with ethical regulations and identify the committee(s) approving the experiments. For experiments involving live vertebrates, include a statement of compliance with ethical regulations and identify the committee(s) approving the experiments.

), e1000412, 2010) to ensure that other relevant aspects of animal studies are adequately reported. See author guidelines, under 'Reporting Guidelines'. See also: NIH (see link list at top right) and MRC (see link list at top right) recommendations. We recommend consulting the ARRIVE guidelines (see link list at top right). 8Please confirm complianceWe recommend consulting the ARRIVE guidelines (see link list at top right) (PLoS Biol. 8(6), e1000412, 2010) to ensure that other relevant aspects of animal studies are adequately reported. See author guidelines, under 'Reporting Guidelines'. See also: NIH (see link list at top right) and MRC (see link list at top right) recommendations. Please confirm compliance.

For publication of patient photos, include a statement confirming that consent to publish was obtained. 14. Report any restrictions on the availability (and/or on the use) of human data or samples. For publication of patient photos, include a statement confirming that consent to publish was obtained. 14. Report any restrictions on the availability (and/or on the use) of human data or samples.

Report the clinical trial registration number (at ClinicalTrials.gov or equivalent). where applicableReport the clinical trial registration number (at ClinicalTrials.gov or equivalent), where applicable.

For phase II and III randomized controlled trials, please refer to the CONSORT flow diagram (see link list at top right) and submit the CONSORT checklist (see link list at top right) with your submission. See author guidelines, under 'Reporting Guidelines. Please confirm you have submitted this listFor phase II and III randomized controlled trials, please refer to the CONSORT flow diagram (see link list at top right) and submit the CONSORT checklist (see link list at top right) with your submission. See author guidelines, under 'Reporting Guidelines'. Please confirm you have submitted this list.

Provide a "Data Availability" section at the end of the Materials & Methods, listing the accession codes for data generated in this study and deposited in a public database (e.g. RNA-Seq data: Gene Expression Omnibus GSE39462, Proteomics data: PRIDE PXD000208 etc. 18For tumor marker prognostic studies, we recommend that you follow the REMARK reporting guidelines (see link list at top right). Please refer to our author guidelines for 'Data Deposition'For tumor marker prognostic studies, we recommend that you follow the REMARK reporting guidelines (see link list at top right). See author guidelines, under 'Reporting Guidelines'. Please confirm you have followed these guidelines. 18: Provide a "Data Availability" section at the end of the Materials & Methods, listing the accession codes for data generated in this study and deposited in a public database (e.g. RNA-Seq data: Gene Expression Omnibus GSE39462, Proteomics data: PRIDE PXD000208 etc.) Please refer to our author guidelines for 'Data Deposition'.

Data deposition in a public repository is mandatory for: a. Protein, DNA and RNA sequences. Data deposition in a public repository is mandatory for: a. Protein, DNA and RNA sequences

Deposition is strongly recommended for any datasets that are central and integral to the study; please consider the journal's data policy. If no structured public repository exists for a given data type, we encourage the provision of datasets in the manuscript as a Supplementary Document (see author guidelines under 'Expanded View' or in unstructured repositories such as Dryad (see link list at top right) or Figshare. see link list at top rightDeposition is strongly recommended for any datasets that are central and integral to the study; please consider the journal's data policy. If no structured public repository exists for a given data type, we encourage the provision of datasets in the manuscript as a Supplementary Document (see author guidelines under 'Expanded View' or in unstructured repositories such as Dryad (see link list at top right) or Figshare (see link list at top right).

Access to human clinical and genomic datasets should be provided with as few restrictions as possible while respecting ethical obligations to the patients and relevant medical and legal issues. If practically possible and compatible with the individual consent agreement used in the study, such data should be deposited in one of the major public accesscontrolled repositories such as dbGAP (see link list at top right) or EGA. see link list at top rightAccess to human clinical and genomic datasets should be provided with as few restrictions as possible while respecting ethical obligations to the patients and relevant medical and legal issues. If practically possible and compatible with the individual consent agreement used in the study, such data should be deposited in one of the major public access- controlled repositories such as dbGAP (see link list at top right) or EGA (see link list at top right).

Computational models that are central and integral to a study should be shared without restrictions and provided in a machine-readable form. The relevant accession numbers or links should be provided. When possible, standardized format (SBML, CellML) should be used instead of scripts (e.g. MATLAB). Authors are strongly encouraged to follow the MIRIAM guidelines (see link list at top right) and deposit their model in a public database such as Biomodels (see link list at top right) or JWS Online (see link list at top right). If computer source code is provided with the paper, it should be deposited in a public repository or included in supplementary informationComputational models that are central and integral to a study should be shared without restrictions and provided in a machine-readable form. The relevant accession numbers or links should be provided. When possible, standardized format (SBML, CellML) should be used instead of scripts (e.g. MATLAB). Authors are strongly encouraged to follow the MIRIAM guidelines (see link list at top right) and deposit their model in a public database such as Biomodels (see link list at top right) or JWS Online (see link list at top right). If computer source code is provided with the paper, it should be deposited in a public repository or included in supplementary information.

Could your study fall under dual use research restrictions? Please check biosecurity documents (see link list at top right) and list of select agents and toxins (APHIS/CDC) (see link list at top right). According to our biosecurity guidelines, provide a statement only if it could. Tetrodotoxin citrate (Cat. No. 1069. provided by Tocris Bioscience, was adopted upon internalCould your study fall under dual use research restrictions? Please check biosecurity documents (see link list at top right) and list of select agents and toxins (APHIS/CDC) (see link list at top right). According to our biosecurity guidelines, provide a statement only if it could. Tetrodotoxin citrate (Cat. No. 1069), provided by Tocris Bioscience, was adopted upon internal

P10, P18 and P30 male and female littermate animals were used for each experiment. Trem2-/-and littermate control WT mice (backcrossed 12 generations to the C57BL/6 background) were obtained from Marco Colonna (Turnbull et al, 2006) and the two strains were bred in parallel. (Boston Children's Hospital) All mice were used at the ages specified in the experimental procedures outlined below and, unless stated otherwise, male and female mice were included in all experiments. Wildtype C57BL/6 mice were obtained from Charles River, unless otherwise specified below. Caspase3 KO mice were obtained from Jackson Labs (Jax #006233) and Caspase3+/â x Caspase3+/â crosses were used to generate wildtype and Caspase3 KO littermates. N/A (Humanitas) Mice were housed in the SPF animal facility of Humanitas Clinical and Research Center in individually ventilated cages. C57BL/6 Trem2-/-mice. Bioxell-Cosmo PharmaceuticalMilan, Italy; Botto (Imperial College, LondonCorreale et alAnimals were group-housed in Optimice cages and maintained in the temperature range and environmental conditions recommended by AAALAC. (Humanitas) All experiments followed the guidelines. established by the European Directive 2010/63/EU and the Italian D.Lg. 26/2014. (Boston Children's Hospital) All procedures were approved by the Boston Children's Hospital institutional animal care and use committee in accordance with NIH guidelines for the humane treatment of animalsN/A (Humanitas) Mice were housed in the SPF animal facility of Humanitas Clinical and Research Center in individually ventilated cages. C57BL/6 Trem2-/-mice, generated as previously described (Turnbull et al, 2006), were provided by Bioxell-Cosmo Pharmaceutical (Milan, Italy) (Correale et al, 2013). P10, P18 and P30 male and female littermate animals were used for each experiment. Trem2-/-and littermate control WT mice (backcrossed 12 generations to the C57BL/6 background) were obtained from Marco Colonna (Turnbull et al, 2006) and the two strains were bred in parallel. (Boston Children's Hospital) All mice were used at the ages specified in the experimental procedures outlined below and, unless stated otherwise, male and female mice were included in all experiments. Wildtype C57BL/6 mice were obtained from Charles River, unless otherwise specified below. Caspase3 KO mice were obtained from Jackson Labs (Jax #006233) and Caspase3+/â x Caspase3+/â crosses were used to generate wildtype and Caspase3 KO littermates. C1qaKO mice are on a C57BL/6J background and were a kind gift from M. Botto (Imperial College, London). C1qa+/â x C1qa+/â crosses were used to generate wildtype and C1qKO littermates. Animals were group-housed in Optimice cages and maintained in the temperature range and environmental conditions recommended by AAALAC. (Humanitas) All experiments followed the guidelines established by the European Directive 2010/63/EU and the Italian D.Lg. 26/2014. (Boston Children's Hospital) All procedures were approved by the Boston Children's Hospital institutional animal care and use committee in accordance with NIH guidelines for the humane treatment of animals.

Anti-Iba-1 Polyclonal Antibody. Unconjugated, WakoAnti-Iba-1 Polyclonal Antibody, Unconjugated, Wako, Cat# 019-19741;

Synaptic Density Protein 95, clone 6G6-1C9 antibody Merck Millipore Cat# MAB1596; Guinea pig Anti-VGLUT 1. Anti-Post, Anti-Post Synaptic Density Protein 95, clone 6G6-1C9 antibody Merck Millipore Cat# MAB1596; Guinea pig Anti-VGLUT 1

Synaptic Systems Cat# 135 304; Rabbit Anti-Mouse P2Y12 Polyclonal Antibody, AnaSpec; EGT Group Cat# 55043A; Rat Anti-Mouse CD68 Monoclonal antibody, Clone FA-11 Bio-Rad / AbD Serotec Cat# MCA1957T; Anti-Shank2 antibody Synaptic Systems Cat# 162 202; Anti-beta3 Tubulin Monoclonal Antibody, Unconjugated Promega Cat# G7121; Anti-Bassoon Polyclonal rabbit antibody, Synaptic Systems, cat# 141 002; anti-PSD-95, Thermofisher, Cat# 516900; guinea pig anti-vGluT2, Millipore cat# AB2251. Polyclonal Antibody, Rabbit anti-Homer2, Cedarlane Labs, cat #160203(SyPolyclonal Antibody, Synaptic Systems Cat# 135 304; Rabbit Anti-Mouse P2Y12 Polyclonal Antibody, AnaSpec; EGT Group Cat# 55043A; Rat Anti-Mouse CD68 Monoclonal antibody, Clone FA-11 Bio- Rad / AbD Serotec Cat# MCA1957T; Anti-Shank2 antibody Synaptic Systems Cat# 162 202; Anti- beta3 Tubulin Monoclonal Antibody, Unconjugated Promega Cat# G7121; Anti-Bassoon Polyclonal rabbit antibody, Synaptic Systems, cat# 141 002; anti-PSD-95, Thermofisher, Cat# 516900; guinea pig anti-vGluT2, Millipore cat# AB2251; Rabbit anti-Homer2, Cedarlane Labs, cat #160203(Sy);

Enzo Life Sciences cat# ADI-VAM-PS003-F; rabbit anti-C1q, Abcam cat# ab182451. mouse anti-Bassoon, Enzo Life Sciences cat# ADI-VAM-PS003-F; rabbit anti-C1q, Abcam cat# ab182451.