# Survey of Campylobacter jejuni in retail chicken meat products by application of a quantitative PCR protocol

CorpusID: 23299206
 
tags: #Medicine, #Environmental_Science, #Biology

URL: [https://www.semanticscholar.org/paper/9bcfd161bf9e1a4c080513383e93102c0ddf6c66](https://www.semanticscholar.org/paper/9bcfd161bf9e1a4c080513383e93102c0ddf6c66)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Survey of Campylobacter jejuni in retail chicken meat products by application of a quantitative PCR protocol


Kalliopi Rantsiou kalliopi.rantsiou@unito.it 
Dipartimento di Valorizzazione e Protezione delle Risorse Agroforestali
Facoltà di Agraria
Università di Torino
Italy

Cristina Lamberti 
Dipartimento di Biologia Animale e dell'Uomo
Università di Torino
Italy

Università di Torino
Via Leonardo da Vinci 4410095Grugliasco, TurinItaly

Luca Cocolin 
Dipartimento di Valorizzazione e Protezione delle Risorse Agroforestali
Facoltà di Agraria
Università di Torino
Italy

Di P R A Va 
Survey of Campylobacter jejuni in retail chicken meat products by application of a quantitative PCR protocol
10.1016/j.ijfoodmicro.2010.02.002(2) The integrity of the work and identification of the author, copyright owner, and publisher must be preserved in any copy. (3) You must attribute this AAM in the following format: Creative Commons BY--NC--ND license (http://creativecommons.org/licenses/by--nc--nd/4.0/deed.en), *Author for correspondenceCampylobacter jejuniquantitative PCRdirect analysispoultry
AbtractCampylobacter-contaminated food products are currently the cause of the highest number of gastroenteritis cases in developed countries. Apart for biosafety measures at the primary production level, no other official control measures are currently in place for its control. This is partly due to the lack of quantitative data regarding the prevelance and contamination level of different food products by Campylobacter spp. that does not allow for quantitative risk assessment. PCR-based methods, applied without prior enrichment, in food samples circumvent limitations associated with the quantification of foodborne pathogens by traditional, culture-dependent methods. In this study, we report the development of a protocol, based on the amplification of the rpoB gene of Campylobacter jejuni, by quantitative PCR (qPCR), directly in food samples. The quantification limit of the protocol was determined to be in the order of 10 colony forming units (cfu)/g or ml of food sample. The optimized protocol was applied for the survey of C. jejuni in naturally contaminated poultry samples. In parallel, traditional sampling was also performed. A high percentage of samples (87%) resulted to be positive by qPCR, while no C. jejuni was detected by traditional analysis. Furthermore, important differences were observed in the detection by qPCR between samples before and after enrichment.

## Introduction

Campylobacter spp. is currently the bacterial foodborne pathogen causing the highest number of gastrointestinal disease in developed countries, according to the World Health Organization (www.who.int). The main vehicle, responsible for human infections, is broiler meat, either through consumption of inadequately cooked chicken meat or through cross-contamination of ready to eat foods. C. jejuni is the most important species with the highest rate of isolation from human cases of gastroenteritis. In Europe in 2007, the proportion of positive poultry meat samples ranged from 4.3 % to 67.1 % and C. jejuni accounted for the majority of the isolates (Anonymous, 2009). A number of factors make it a problematic agent to control: (i) it is a zoonotic agent, that is asymptomatic in broilers (as well as other birds), (ii) available sampling and testing methods for its detection and quantification give variable results, thus surveillance data from different sources may not be comparable, (iii) compared to other foodborne pathogens, little is known regarding its physiology and its ability to survive common food processing conditions. Furthermore, the previous belief that thermophilic campylobacters are sensitive to conditions they encounter outside their host seems not to be supported by the epidemiological data (Humphrey et al., 2007).

The application of molecular methods to rapidly and inequivocally detect and identify foodborne pathogens in foodstuffs is offering a valid alternative to traditional microbiological testing. The challenge nowdays is the application of these methods, primarily based on amplification of a target DNA (or RNA) sequence by PCR, without previous enrichment or culturing, in order to quantify foodborne pathogens directly in foods. Such an approach has already been employed for other pathogens like Listeria monocytogenes (Rantsiou et al., 2008). Methods to detect and quantify C. jejuni in foods until recently have been applied after an enrichment step (Josefsen et al., 2004;Nogva et al., 2000;Oliveira et al., 2005;Sails et al., 2003;Yang et al., 2003), while direct quantification of C. jejuni in chicken rinses has been performed (Debretsion et al., 2007).

In this paper we focused on the development of a protocol for the detection and quantification of C. jejuni in foods. The protocol is based on the amplification, by quantitative PCR, of the rpoB gene, encoding the β-subunit of the RNA polymerase of C. jejuni. The ultimate goal was to apply the protocol to naturally contaminated poultry samples obtained from the retail market in Italy.


## Materials and methods


### Strains and media.

Strains of C. jejuni belonging to the Laboratory of Microbiology of the University of Ghent, Belgium (strain LMG8842) and to the Collection of Institute Pasteur, France (strain CIP70.2T), as well as 3 strains, with codes 119, 592 and 221/05, isolated from bovine feaces the first and poultry the latter two, were employed in this study. In addition, Campylobacter coli DSMZ 4689, Campylobacter upsaliensis DSMZ 5365 and Campylobacter lari DSMZ 11375 were also used. All strains of Campylobacter spp. were propagated either in Brain Heart Infusion (BHI) supplemented with 5% horse blood (BHIB) (Oxoid, Milan, Italy) or in Campylobacter blood-free selective medium, CCDA (Oxoid) in microaerophilic conditions (CampyGen, Oxoid) at 37 ºC.

Moreover, strains of food related bacteria were selected and used in this study in order to determine the specificity of the assay. Listeria seeligeri and Listeria welshimeri. At least 2 strains for each species were tested. The strains belonged to the culture collection of the University of Turin, Italy (Rantsiou et al., 2008) and were rutinelly propagated in Brain Heart Infusion (BHI) medium (Oxoid, Milan, Italy).


### DNA extraction.

For strains grown in pure culture, DNA was extracted as previously described by Rantsiou et al. (2008). For the extraction of DNA from food samples the Master-Pure™ Complete DNA and RNA Purification Kit (Epicentre, Madison, WI, USA) was used according to the manufacturer's instructions and to previous application for DNA extractions from foodstuffs (Rantsiou et al., 2008).


### Quantitative PCR amplification.

Quantitative PCR was performed by using the SYBR Green chemistry. To amplify part of the rpoB gene of C. jejuni, primers Cj_rpoB1 (5'-GAGTAAGCTTGGTAAGATTAAAG-3') and

Cjs_rpoB2 ( 95 ºC for 30 seconds, 62 ºC for 30 seconds and 72ºC for 30 seconds, followed by the fluorescence reading. The cycle was repeated 50 times and was preceded by a denaturation step at 95 ºC for 10 minutes. In each run, a positive and negative control was used. The positive control was represented by DNA extracted from a pure culture of C. jejuni LM8842, standardized to 100 ng/µl. In a total reaction volume of 25 µl, 1 µl of DNA was added. For the negative control, 1 µl of sterile, PCR-grade DNA (Sigma-Aldrich, Italy) was added. A melting curve analysis was carried out at the end of each run.


### Construction of calibration curves.

Calibration curves were constructed in milk, poultry skin and poultry meat. For the construction of calibration curves in food samples, the strain of C. jejuni LM8842 was used. The strain was streaked in BHIB plates and after 24 hours of incubation, the cells were scraped off the agar surface with 1 ml of Ringer's solution (Oxoid, Milan, Italy) and the use of a plastic, sterile spreader. The cell suspension was serially diluted in Ringer's solution and counted on BHIB plates incubated at 37°C for 24-48 hours.

Each dilution was inoculated in 10 g (for poultry skin or poultry meat) or 10 ml (for milk) of food sample. Then, 40 ml of Bolton broth (Oxoid, Milan, Italy), with supplement (Oxoid, Milan, Italy) but without blood, were added. Solid samples (poultry skin and meat) were homogenized, using a stomacher (PBI, Milan, Italy) for 2 minutes at maximum speed, while liquid samples (milk) mixed well. Subsequently, one ml was recovered and mixed with 9 ml of Ringer's solution and 1 ml of the diluted sample was used for DNA extraction as described above (DNA at t=0). The DNA extraction was also performed on the samples after 24 hours of incubation at 37 ºC in Bolton Broth. Also in this case, 1 ml of a ten-fold dilution in Ringer's solution was processed (DNA at t=24). One µl of the t=0 DNA (in triplicate) was used in qPCR amplifications and calibration curves were constructed plotting the threshold cycle against the colony forming units (cfu)/g or ml. Similarly, the t=24 DNA was used in amplification to determine the detection limit after overnight enrichment. The efficiency of the reactions was calculated according to Rutledge and Cote (Rutledge and Cote, 2003). Standard curves were contructed at least three times from three independent experiments.


### Sampling.

Fourty eight samples of individually packaged poultry were purchased from super markets in the Piedmont region, in the Northwest part of Italy. The packaging was removed under a laminar flow hood. From each package, ten g of either meat (for samples without skin) or skin were aseptically cut and mixed with 40 ml of Bolton broth (Oxoid, Milan, Italy), with supplement (Oxoid, Milan, Italy) but without blood, and homogenized as above. One ml of the homogenate was mixed with 9 ml of Ringer's solution and 1 ml of this mix was used for DNA extraction (t=0). At the same time, the homogenate, the 1 to 10 and 1 to 100 dilutions of it were plated in CCDA medium. Plates were incubated at 37 ºC for 48 hours in microaerophilic conditions by using the Anaerogen system (Oxoid, Milan, Italy). The homogenate was also incubated for 18 hours at 37 ºC and DNA extraction was carried out from a tenfold dilution in Ringer's solution, while a loopfull of the enriched homogenate was streaked in CCDA and incubated at 37 ºC for 48 hours (t=24). Campylobacter spp.

suspected colonies from the CCDA plates were isolated in BHIB and subjected to qPCR amplification as described above.


## Results and Discussion

The method of detection and quantification of C. jejuni presented in this paper is based on the amplification of partial rpoB gene. Primers, intended to be specific for C.

jejuni, were designed based on all sequences of Campylobacter spp. available in GeneBank. By multiple sequence alingment (ClustalW2 algorithm, EBI, UK, available at: http://www.ebi.ac.uk/Tools/clustalw2/index.html), regions were identified that in silico were specific. Subsequently, the specificity of the amplification was optimized in vitro using as target, DNA extracted from pure cultures of different Campylobacter spp. Furthermore, a set of bacteria commonly found in foods was also used. Several conditions of amplifications, focusing mainly on the temperature of annealing and concentration of the primers were tested, in order to obtain amplification signals only when C. jejuni DNA was added as template in the reaction. The conditions described in the materials and methods resulted to be specific for C. jejuni. Sequences of the rpoB gene of phylogenetically close species, namely

Helicobacter and Arcobacter spp., were also used in an in silico analysis to check for possible annealing sites of the primers used in this study. Sequence heterogeneity was high and the primers described here did not align to Helicobacter or Arcobacter sequences.

In order to quantify, in terms of cfu/g or ml, C. jejuni in foods, appropriate calibration curves were constructed. For the DNA extraction, a determining factor for the success of such applications in foodstuffs, a commercial kit (Epicentre) was used which has been previously proven adequate for qPCR in foods (Rantsiou et al., 2008). Its applicability was further confirmed in this study, during the optimization phase. Three food matrices were chosen, based on available information from the literature regarding C. jejuni contamination (Anonymous, 2009;Heuvelink et al., 2009;Horrocks et al., 2008). These were: milk and poultry (meat and skin). The calibration curves are presented in Figure 1. The efficiencies calculated for the three matrices were: 100% for the milk, 79% for the poultry meat and 83% for the poultry skin. The R 2 values were 0.96, 0.90 and 0.99 respectively. As can be seen in Figure 1, for all three matrices a linearity range that spanned 5 logarithms was obtained and the quantification limit was in the order of 10 cfu/g or ml. The results obtained, in terms of efficiency, R 2 and limit of quantification were comparable when three independent experiments were performed (data not shown). After the 18 hours overnight enrichment (t=24) at 37 ºC, also matrices inoculated with concentrations of 1-10 cfu/g or ml could be detected.

The qPCR based protocol was applied, in parallel to traditional microbiological analysis, to detect and quantify C. jejuni in poultry samples. Fourty-eight samples were taken from large super markets in the Piedmont region in the Northwest of Italy.

Samples were parts of chicken and turkey that were pre-packaged and stored at 4 ºC.

A qPCR analysis was performed on DNA extracted at the time of sampling (t=0), as well as after an overnight enrichment (t=24). In parallel, traditional microbiological analysis was performed, both at t=0 and at t=24. The results obtained by the two approaches are summarized in Table 1 Campylobacter spp. colonies, randomly selected from the CCDA plates, were subjected to DNA and qPCR analysis, none of them resulted to be C. jejuni. Most likely, the colonies belonged to other Campylobacter spp. that are capable of growing on this medium. It is possible that the lack of correlation observed between the results obtained by traditional method and qPCR is due to the very low concentration of C.

jejuni and the presence of cells in the samples that were injured or stressed and therefore did not develop on the selective medium.

Surprisingly, at t=24, the percentage of positive samples, obtained by qPCR, decreased compared to the situation at t=0. Thirteen of the 48 samples (27%) gave a positive signal by qPCR. This observation may be attributed either to the presence of DNA of dead cells of C. jejuni in the samples at t=0, or to the presence of stressed and/or injured cells of C. jejuni that are not able to propagate in the selective (and stringent) environment of the enrichment broth used. The latter explanation is supported also by the data of the traditional analysis: none of the Campylobacter populations counted at t=0 was detected at t=24. Such a behaviour has already been reported for Campylobacter in Bolton enrichment broth (Habib et al., 2008). These authors also concluded that lower Campylobacter concentrations had lower likelihood of giving a positive result after enrichment compared to direct plating (Habib et al., 2008). It should also be noted that the conditions used for enrichment in this study (incubation at 37 ºC, Bolton Broth without blood), have recently been shown to have no negative effect on the recovery of campylobacters (Williams et al., 2009).

Furthermore, in a previous study, Bolton enrichment broth without blood detected more samples positive for thermophilic Campylobacter spp. than did Preston enrichment broth (Paulsen et al., 2005). However, it is possible that in these conditions, competing microflora has better chances of outnumbering Campylobacter.

In fact, it has been previously reported that non-Campylobacter species, after enrichment in Bolton broth, can generate in some cases colonies very similar to those produced by Campylobacter on selective plating medium (Baylis et al., 2000;Habib et al., 2008, Jasson et al., 2009). Only one sample (P7 , Table 1), was negative at t=0

and positive at t=24. It is probable that a C. jejuni population below the quantification limit was present in the sample and it was able to produce a signal in qPCR after enrichment.

The use of the qPCR protocol developed in this study accurately detected C. jejuni in a high percentage of poultry samples, while the traditional analysis failed to do so.

Additionally, the results obtained highlighted important aspects of the physiology of Results at t=0 are reported as cfu/g as determined by the direct counts on CCDA plates (Campylobacter spp. suspected colonies in the traditional analysis) or based on the C(t) values obtained for the sample and the appropriate calibration curve. If the C(t) value falls out of the quantification limit of the calibration curve, the sample is reported as positive but without a quantification value. 2 Results at t=24 are reported as positive or negative. In the traditional analysis, positive refers to the presence of suspected colonies while in the qPCR analysis it refers to a fluorescence signal obtained during the amplification. * Randomly selected and isolated Campylobacter spp. suspected colonies from the CCDA plates were not confirmed as C. jejuni by qPCR.  corresponding to each C. jejuni dilution, were loaded in triplicate in a qPCR reaction.

For each C. jejuni dilution, the mean and standard deviation of the C(t) values obtained were calculated and used to contsruct the calibration curve. Three independent calibration curves were constructed for each food matrix but only one is shown here. Figure 1a: calibration curve in milk. Figure 1b: calibration curve in poultry meat. Figure 1c: calibration curve in poultry skin.  


. By direct qPCR analysis on the DNA extracted at t=0, 42 of the 48 (87%) samples resulted positi ve for the presence of C.jejuni. Apart from one sample, for which the concentration of C. jejuni was determined to be 24 cfu/g, all other positive samples had concentrations below the quantification limit. Melting curve analysis at the end of each run, resulted in melting temperatures for the positive samples, similar to those of C. jejuni used as control in the qPCR (data not shown). By traditional analysis, for 33 of 48 samples the Campylobacter spp. populations were below the detection limit, while for the remaining 15 samples, counts were in the order of 10 2 -10 3 cfu/g. When suspected

## Figure legends
Figure legends

## Figure 1 .
1Calibration curves of C. jejuni, serially diluted and inoculated in different food matrices. DNA samples, extracted as described in the materials and methods,

## Figure 1a .Figure 1b .
1a1bFigure 1a.

## Figure 1c .
1cFigure 1c.


These were: Lactobacillus sakei,Staphylococcus xylosus, Lactococcus lactis, Streptococcus thermophilus, 

Enterococcus faecium, Pseudomonas aeruginosa, Serratia marcescens, Bacillus 

cereus, Escherichia coli, Listeria monocytogenes, Listeria innocua, Listeria ivanovvi, 



## Table 1 .
1Results of microbiological and molecular analysis on poultry samples 

Results t=0 1 
Results t=24 2 
Sample 
Code 

Type of Sample 
Traditional 
analysis 
(cfu/g) 

qPCR 
analysis 
(cfu/g) 

Traditional 
analysis 

qPCR 
analysis 

C1 
Meat (chicken) 
<50 
Positive 
Negative 
Negative 
PT1 
Meat (chicken breast) 
<50 
Positive 
Negative 
Negative 
PT2 
Meat (chicken breast) 
<50 
Negative 
Negative 
Negative 
C2 
Meat (chicken) 
6x10 2 
Positive 
Negative 
Positive 
C3 
Meat (chicken) 
<50 
Positive 
Negative 
Negative 
GC 
Meat (turkey) 
<50 
Positive 
Negative 
Negative 
1C 
Meat (chicken) 
<50 
Positive 
Negative 
Negative 
2C 
Meat (chicken) 
<50 
Positive 
Negative 
Negative 
3C 
Meat (chicken) 
<50 
Positive 
Negative 
Negative 
4C 
Meat (chicken) 
2x10 2* 
Positive 
Negative 
Positive 
5C 
Meat (chicken) 
7x10 2* 
Positive 
Negative 
Negative 
6C 
Meat (chicken) 
3,5x10 2* 
Negative 
Negative 
Negative 
10 PT A 
Meat (chicken breast) 
<50 
Positive 
Negative 
Negative 
10 PT B 
Meat (chicken breast) 

<50 

Positive 
Negative 
Negative 
PT3 A 
Meat (chicken breast) 
<50 
Positive 
Negative 
Negative 
C5 
Meat (chicken) 
<50 
Positive 
Negative 
Positive 
AC1 
Meat (chicken wing) 
<50 
Positive 
Negative 
Negative 
C6 
Meat (chicken) 
1x10 3* 
Positive 
Negative 
Negative 
Ca 
Meat (chicken) 
<50 
Positive 
Negative 
Negative 
Cc 
Meat (chicken) 
<50 
Positive 
Negative 
Negative 
Cb 
Meat (chicken) 
<50 
24 
Negative 
Positive 
C7 
Meat (chicken) 
<50 
Positive 
Negative 
Negative 
GC2 
Meat (turkey) 
<50 
Positive 
Negative 
Positive 
C8 
Meat (chicken) 
<50 
Positive 
Negative 
Negative 
CS1 
Meat (chicken leg) 
<50 
Positive 
Negative 
Positive 
CA 
Meat (chicken) 
<50 
Positive 
Negative 
Negative 
CB 
Meat (chicken) 
<50 
Positive 
Negative 
Negative 
CS2 
Meat (chicken leg) 
<50 
Negative 
Negative 
Negative 
CS3 
Meat (chicken leg) 
<50 
Positive 
Negative 
Negative 
CS4 
Meat (chicken leg) 
<50 
Positive 
Negative 
Positive 
P8 
Skin (chicken) 
2x10 3* 
Positive 
Negative 
Negative 
P1 
Skin (chicken) 
<50 
Negative 
Negative 
Negative 
P2 
Skin (chicken) 
<50 
Positive 
Negative 
Negative 
P3 
Skin (chicken) 
<50 
Positive 
Negative 
Negative 
GP 
Skin (turkey) 
<50 
Negative 
Negative 
Negative 
7SP 
Skin (chicken leg) 
<50 
Positive 
Negative 
Negative 
8SP 
Skin (chicken leg) 
<50 
Positive 
Negative 
Positive 
P5 
Skin (chicken) 
7x10 2* 
Positive 
Negative 
Negative 
AP1 
Skin (chicken wings) 
3,5x10 2* 
Positive 
Negative 
Negative 
P6 
Skin (chicken) 
2x10 2* 
Positive 
Negative 
Negative 
Pa 
Skin (chicken) 
4x10 3* 
Positive 
Negative 
Positive 
Pb 
Skin (chicken) 
5x10 2* 
Positive 
Negative 
Positive 
Pc 
Skin (chicken) 
4,5x10 2* 
Positive 
Negative 
Positive 
P7 
Skin (chicken) 
2x10 3* 
Negative 
Negative 
Positive 
GP1 
Skin (turkey) 
3x10 3* 
Positive 
Negative 
Negative 
PA 
Skin (chicken) 
<50 
Positive 
Negative 
Negative 
PB 
Skin (chicken) 
<50 
Positive 
Negative 
Negative 
PS2 
Skin (chicken leg) 
6x10 2* 
Positive 
Negative 
Positive 

1 
C. jejuni in food and in enrichment media, commonly used for its isolation, that require further investigation.
AcknowledgementsThis study was funded by the European Commission within the VI Framework Program, contract no. 007081, 'Pathogen Combat: control and prevention of emerging and future pathogens at cellular and molecular level throughout the food chain'.We wish to thank Prof. Uyttendaele and Dr. Rajkovic of the University of Ghent, Belgium as well as Prof. Biavati and Dr. Gaggia of the University of Bologna, Italy for kindly providing us Campylobacter spp. strains from their collections.
The Community report on trends and sources of zoonoses and zoonotic agents on the European Union in. The EFSA Journal. 223AnonymousAnonymous 2009. The Community report on trends and sources of zoonoses and zoonotic agents on the European Union in 2007. The EFSA Journal 223.

Comparison of three enrichment media for the isolation of Campylobacter spp. from foods. C L Baylis, S Macphee, K W Martin, T J Humphrey, R P Betts, Journal of Applied Microbiology. 89Baylis, C. L., MacPhee, S., Martin, K. W., Humphrey, T. J. and Betts, R. P. 2000. Comparison of three enrichment media for the isolation of Campylobacter spp. from foods. Journal of Applied Microbiology 89, 884-891.

. A Debretsion, T Habtemariam, S Wilson, D Nganwa, T Yehualaeshet, Debretsion, A., Habtemariam, T., Wilson, S., Nganwa, D. and Yehualaeshet, T. 2007.

Real-time PCR assay for rapid detection and quantification of Campylobacter jejuni on chicken rinses from poultry processing plant. Molecular and Cellular Probes. 21Real-time PCR assay for rapid detection and quantification of Campylobacter jejuni on chicken rinses from poultry processing plant. Molecular and Cellular Probes 21, 177-181.

. I Habib, I Sampers, M Uyttendaele, D Berkvens, L De Zutter, Habib, I., Sampers, I., Uyttendaele, M., Berkvens, D. and De Zutter, L. 2008.

Baseline data from a Belgium-wide survey of Campylobacter species contamination in chicken meat preparations and considerations for a reliable monitoring program. Applied and Environmental Microbiology. 74Baseline data from a Belgium-wide survey of Campylobacter species contamination in chicken meat preparations and considerations for a reliable monitoring program. Applied and Environmental Microbiology 74, 5483-5489.

. A E Heuvelink, C Van Heerwaaeden, A Zwartkruis-Nahuis, J J H C Tilburg, Heuvelink, A. E., van Heerwaaeden, C., Zwartkruis-Nahuis, A., Tilburg, J. J. H. C.,

Two outbreaks of campylobacteriosis associated with the consumption of raw cow's milk. M Hanna Bos, F G C Heilmann, A Hofhuis, T Hoekstra, E De Boer, International Journal of Food Microbiology. 134Hanna Bos, M., Heilmann, F. G. C., Hofhuis, A., Hoekstra, T. and de Boer, E. 2009. Two outbreaks of campylobacteriosis associated with the consumption of raw cow's milk. International Journal of Food Microbiology 134, 70-74.

Incidence and ecology of Campylobacter jejuni and coli in animals. S M Horrocks, R C Anderson, D J Nisbet, S C , R , Anaerobe. 15Horrocks, S. M., Anderson, R. C., Nisbet, D. J. and S.C., R. 2008. Incidence and ecology of Campylobacter jejuni and coli in animals. Anaerobe 15, 18-25.

Campylobacters as zoonotic pathogens: A food production perspective. T Humphrey, S O&apos;brien, M Madsen, International Journal of Food Microbiology. 117Humphrey, T., O'Brien, S. and Madsen, M. 2007. Campylobacters as zoonotic pathogens: A food production perspective. International Journal of Food Microbiology 117, 237-257.

Characterization of Escherichia coli from raw poultry in Belgium and impact on the detection of Campylobacter jejuni using Bolton. V Jasson, I Sampers, M Botteldoorn, F López-Gálvez, L Baert, S Denayer, A Rajkovic, I Habib, L De Zutter, J Debevere, M Uyttendaele, International Journal of Food Microbiology. In pressJasson, V., Sampers, I., Botteldoorn, M., López-Gálvez, F., Baert, L., Denayer, S., Rajkovic, A., Habib, I., De Zutter, L., Debevere, J. and Uyttendaele, M., 2009. Characterization of Escherichia coli from raw poultry in Belgium and impact on the detection of Campylobacter jejuni using Bolton. International Journal of Food Microbiology, In press.

Enrichment followed by quantitative PCR both for rapid detection and as a tool for quantitative risk assessment of food-borne thermotolerant Campylobacters. M H Josefsen, N R Jacobsen, J Hoorfar, Applied and Environmental Microbiology. 70Josefsen, M. H., Jacobsen, N. R. and Hoorfar, J. 2004. Enrichment followed by quantitative PCR both for rapid detection and as a tool for quantitative risk assessment of food-borne thermotolerant Campylobacters. Applied and Environmental Microbiology 70, 3588-3592.

Application of the 5'-nuclease PCR assay in evaluation and development of methods for quantitative detection of Campylobacter jejuni. H K Nogva, A Bergh, A Holck, K Rudi, Applied and Environmental Microbiology. 66Nogva, H. K., Bergh, A., Holck, A. and Rudi, K. 2000. Application of the 5'-nuclease PCR assay in evaluation and development of methods for quantitative detection of Campylobacter jejuni. Applied and Environmental Microbiology 66, 4029-4036.

Detection of Campylobacter jejuni in naturally contaminated chicken skin by melting peak analysis of amplicons in real-time PCR. T C R M Oliveira, S Barbut, M W Griffiths, International Journal of Food Microbiology. 104Oliveira, T. C. R. M., Barbut, S. and Griffiths, M. W. 2005. Detection of Campylobacter jejuni in naturally contaminated chicken skin by melting peak analysis of amplicons in real-time PCR. International Journal of Food Microbiology 104, 105- 111.

Comparison of three methods for detecting Campylobacter spp. in chilled or frozen meat. P Paulsen, P Kanzler, F Hilbert, S Mayrhofer, S Baumgartner, F J M Smulders, International Journal of Food Microbiology. 103Paulsen, P., Kanzler, P., Hilbert, F., Mayrhofer, S., Baumgartner, S., Smulders, F.J.M., 2005. Comparison of three methods for detecting Campylobacter spp. in chilled or frozen meat. International Journal of Food Microbiology, 103, 229-233.

Detection, quantification and vitality of Listeria monocytogenes in food as determined by quantitative PCR. K Rantsiou, V Alessandria, R Urso, P Dolci, L Cocolin, International Journal of Food Microbiology. 121Rantsiou, K., Alessandria, V., Urso, R., Dolci, P. and Cocolin, L. 2008. Detection, quantification and vitality of Listeria monocytogenes in food as determined by quantitative PCR. International Journal of Food Microbiology 121, 99-105.

Mathematics of quantitative PCR and the applications of standard curves. R G Rutledge, C Cote, Nucleic Acids Research. 3193Rutledge, R. G. and Cote, C. 2003. Mathematics of quantitative PCR and the applications of standard curves. Nucleic Acids Research 31, e93.

A real-time PCR assay for the detection of Campylobacter jejuni in foods after enrichment culture. A D Sails, A J Fox, F J Bolton, D R A Wareing, D L A Greenway, Applied and Environmental Microbiology. 69Sails, A. D., Fox, A. J., Bolton, F. J., Wareing, D. R. A. and Greenway, D. L. A. 2003. A real-time PCR assay for the detection of Campylobacter jejuni in foods after enrichment culture. Applied and Environmental Microbiology 69, 1383-1390.

Enrichment culture for the isolation of Campylobacter spp: Effects of incubation conditions and the inclusion of blood in selective broths. L K Williams, F Jorgensen, R Grogono-Thomas, T J Humphrey, International Journal of Food Microbiology. 130Williams, L. K., Jorgensen, F., Grogono-Thomas, R. and Humphrey, T. J. 2009. Enrichment culture for the isolation of Campylobacter spp: Effects of incubation conditions and the inclusion of blood in selective broths. International Journal of Food Microbiology 130, 131-134.

Application of real-time PCR for quantitative detection of Campylobacter jejuni in poultry, milk and environmental water. C Yang, Y Jiang, K Huang, C Zhu, Y Yin, FEMS Immunology and Medical Microbiology. 38Yang, C., Jiang, Y., Huang, K., Zhu, C. and Yin, Y. 2003. Application of real-time PCR for quantitative detection of Campylobacter jejuni in poultry, milk and environmental water. FEMS Immunology and Medical Microbiology 38, 265-271.