# GigaScience A large-scale metagenomic survey dataset of the post-weaning piglet gut lumen --Manuscript Draft-- Manuscript Number: GIGA-D-20-00347R2 Full Title: A large-scale metagenomic survey dataset of the post-weaning piglet gut lumen

CorpusID: 235321086
 
tags: #Medicine, #Environmental_Science, #Biology

URL: [https://www.semanticscholar.org/paper/c960410e6392b9dc52e6aa7a6b74e52f665232b0](https://www.semanticscholar.org/paper/c960410e6392b9dc52e6aa7a6b74e52f665232b0)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

GigaScience A large-scale metagenomic survey dataset of the post-weaning piglet gut lumen --Manuscript Draft-- Manuscript Number: GIGA-D-20-00347R2 Full Title: A large-scale metagenomic survey dataset of the post-weaning piglet gut lumen


DrSteven P Djordjevic 
University of Technology Sydney Ultimo
AUSTRALIA

Aaron Darling 
University of Technology Sydney Ultimo
AUSTRALIA

Daniela Gaio 
University of Technology Sydney Ultimo
AUSTRALIA

Daniela Gaio 
University of Technology Sydney Ultimo
AUSTRALIA

Matthew Z Demaere 
University of Technology Sydney Ultimo
AUSTRALIA

Kay Anantanawat 
University of Technology Sydney Ultimo
AUSTRALIA

Graeme J Eamens 
University of Technology Sydney Ultimo
AUSTRALIA

Michael Liu 
University of Technology Sydney Ultimo
AUSTRALIA

Tiziana Zingali 
University of Technology Sydney Ultimo
AUSTRALIA

Linda Falconer 
University of Technology Sydney Ultimo
AUSTRALIA

Toni A Chapman 
University of Technology Sydney Ultimo
AUSTRALIA

Steven P Djordjevic 
University of Technology Sydney Ultimo
AUSTRALIA

Aaron E Darling 
University of Technology Sydney Ultimo
AUSTRALIA

GigaScience A large-scale metagenomic survey dataset of the post-weaning piglet gut lumen --Manuscript Draft-- Manuscript Number: GIGA-D-20-00347R2 Full Title: A large-scale metagenomic survey dataset of the post-weaning piglet gut lumen
Article Type: Data Note Funding Information: Australian Research Council (LP150100912) Corresponding Author: Corresponding Author Secondary Information: Corresponding Author's Institution: University of Technology Sydney Corresponding Author's Secondary Institution: First Author: First Author Secondary Information: Order of Authors: Order of Authors Secondary Information: Response to Reviewers: Reviewer reports: Reviewer #2: The authors have revised their Research paper into a Data Note.
We report on the largest shotgun metagenomic analysis of the pig gut lumen microbiome to date. By processing over 800 faecal time-series samples from 126 piglets and 42 sows, we generated over 8Tbp of metagenomic shotgun sequence data. This study was conducted to generate a publicly available databank of the faecal metagenome of weaner piglets aged between 3 and 9 weeks old, treated with different probiotic formulations and intramuscular antibiotic treatment. Here we describe the animal trial procedures and the generation of our metagenomic dataset.AbstractWe report on the largest shotgun metagenomic analysis of the pig gut lumen microbiome to date. By processing over 800 faecal time-series samples from 126 piglets and 42 sows, we generated over 8Tbp of metagenomic shotgun sequence data.This study was conducted to generate a publicly available databank of the faecal metagenome of weaner piglets aged between 3 and 9 weeks old, treated with different probiotic formulations and intramuscular antibiotic treatment. Here we describe the animal trial procedures and the generation of our metagenomic dataset.Data descriptionThe dataset includes 911 samples, comprising a total of 27 billion raw sequence reads.Preliminary analysis of the dataset consisted in the extraction of 16S rRNA gene containing reads with SortMeRNA 1 and their classification with the RDP classifier 2 .In terms of taxonomic diversity, most OTUs (75.71%) were assigned to the Firmicutes phylum. The next most abundant bacterial phyla were: Bacteroidetes (13.21%), Actinobacteria (5.10%), Proteobacteria (3.36%), and Spirochates (0.69%). A visualization of the microbial composition, obtained with Krona 3 , is shown of the post-weaning piglets (Figure 1A) along with the beta diversity of the mothers and the piglets during the first 5 weeks post-weaning (Figure 1B). Interactive Krona maps areManuscriptClick here to access/download;Manuscript;metapigs_base_DataNote_ms.do

this dataset can find all relevant details on experimental design, experimental approach and primary data processing in this manuscript.

-The authors have removed most of the concerning analysis details. Few comments; 1) Data description / OTUs. Are OTUs used or "ASVs". I understand sortMeRNA was used that includes qiime v1.x but current methods use a DaDa2 similar approach moving away from OTUs (as is qiime v2.x for over a long period already)?

RE: Although we are aware of the advances that ASV inference methods have led to in the analysis of 16S amplicon sequencing data, there are no rigorous performant methods, that we are aware of, to obtain ASVs from shotgun metagenomic reads. We therefore left the existing analysis unchanged.

2) Data description: to largely describe (overall picture) a simple alpha-diversity plot and beta-diversity ordination plot would be an easy/quick way that brings more meaningful insight in describing the overall samples and individual variation observed. Now all data of the various groups is averaged per piglet or mother in a bardiagram/Krona plot which does not make sense since there is a large diversity within those groups of treatments.

RE: With regards to the diversity, an analysis of diversity (included in our first submission) suggested that differences in microbial community composition (alpha and beta diversity) between treatment groups was mild, while more prominent shifts of diversity were detected between samples from distinct time points (age of the piglets). The revised manuscript now includes a PCoA to describe the beta diversity of samples. We removed the Krona plot of the mothers, and merged the Krona plot of the piglets to a panel in a combined figure with the PCoA (beta diversity). The PCoA plots highlight the strong effect of time/aging, and the (dis)similarity between the mothers and piglets at distinct time points in the trial. In the revised submission we also report alpha diversity indices.


## 3) Supplement Fig2

; make more clear that frequency is N samples. Also add bin size in legend for both sub figures. RE: Done. 4) Supplement Fig4; would this be a possible result of using cfu which does not take into account dead/alive ratio's? That discussion/mentioning seems missing in the current text.

RE: Yes, that is true. This has now been added to the manuscript. 


## Pig trial and sample collection

Animal studies were conducted at the Elizabeth Macarthur Agricultural Institute (EMAI) NSW, Australia and were approved by the EMAI Ethics Committee (Approval M16/04). The trial animals comprised 4-week old male weaner pigs (n=126) derived from a commercial swine farm and transferred to the study facility in January 2017. These were cross-bred animals of "Landrace", "Duroc" and "Large White" breeds and had been weaned at approximately 3 weeks of age (Supplementary  Figure 1). Each room had nine pens, consisting of a set of six and a set of three pens, designated a-f and g-i respectively, with the two sets of pens being physically separate, i.e. animals could come in contact with each other through the pen's bars within each set of pens, but not between sets. The rooms were physically separated by concrete walls and contamination between rooms was minimized by using separate equipment (boots, gloves, coveralls) for each room. In addition, under-floor drainage was flushed twice weekly and the flushed faeces/urine was retained in under-floor channels that ran the length of the facility, so that Rooms 1, 2 were separate from Rooms 3, 4 and flushing was in the direction 1 to 2 and 3 to 4.

The pigs were fed ad libitum a commercial pig grower mix of 17.95% protein free of antibiotics, via self-feeders. On the day of arrival (day 1) 30, 18, 18, and 60 pigs were allocated randomly to Rooms 1, 2, 3 and 4 respectively in groups of 6, 6, 6 and 6-7 pigs per pen respectively (Supplementary Figure 1A). Pigs were initially weighed on day 2, and some pigs were moved between pens to achieve an initial mean pig weight per treatment of approximately 6.5 kg (range: 6.48-6.70; mean±SD:

6.53±0.08). Pigs were weighed weekly throughout the trial, and behaviour and faecal consistency scores were taken daily over the 6-week period of the trial (Supplementary Table 2). Developmental and commercial probiotic paste preparations ColiGuard® and D-Scour™ from International Animal Health, were used in some treatment groups.

The animals were acclimatised for 2 days before the following treatments were   Table 2), were obtained throughout this study. At the end of the trial period, all samples were transported from EMAI to the University of Technology Sydney (UTS) for further processing. The experimental workflow is schematically represented in Figure 3.


## Positive controls

As a positive control "mock community" for this study, four Gram positive (Bacillus  


## DNA extraction

Piglet and sow faecal samples, mock community samples, negative controls and probiotic samples (D-Scour™ and ColiGuard® paste) were allocated to a randomized block design to control for batch effects in DNA extraction and library preparation.

The faecal samples were thawed on ice first, followed by the probiotics and mock community samples. MetaPolyzyme (Sigma-Aldrich) treatment was performed according to the manufacturer's instructions except for the dilution factor, which we allowed to be 4.6 times higher. Immediately after incubation, DNA extraction was performed with the MagAttract PowerMicrobiome DNA/RNA EP kit (Qiagen)

according to the manufacturer's instructions. Quantification of DNA was performed using PicoGreen (Thermofisher) and measurements were performed with a plate reader (Tecan, Life Sciences) using 50 and 80 gain settings. All samples were diluted to 10 ng/µL.


## Library preparation

Sample index barcode design using a previously introduced method 4 yielded a set of 96 x 8nt sequences with a 0.5 mean GC content and none of the barcodes containing 3 or more identical bases in a row. Nine hundred sixty different combinations of i5 and i7 primers were used to create a uniquely barcoded library for each sample. The detailed sample-to-barcode assignment is given in Supplementary Table 3. Library preparation was carried out using a modification of the Nextera Flex protocol to produce low bias, called Hackflex, that allows the production of low cost shotgun 


## Normalization and sequencing

The master pool was sequenced on an Illumina MiSeq v2 300 cycle nano flow cell (Illumina, USA). Read counts were obtained and used to normalise libraries. The liquid handling robot OT-One (Opentrons) was programmed to re-pool libraries based on read counts obtained from the previous MiSeq run. The code used to achieve the normalization is available through our Github repository.

The read count distribution after normalisation is displayed in Supplementary   Figure 2. The normalized and purified pooled library was sequenced on an Illumina

NovaSeq 6000 S4 flow cell at the Ramaciotti Centre for Genomics (Sydney, NSW, Australia), generating a total of 27 billion read pairs from 911 samples.


## Sequence data processing

Adapter trimming (parameters: k=23 hdist=1 tpe tbo mink=11), PhiX DNA removal 


## Comparison of the expected and the observed taxonomic profile of the positive controls

All the mock community members, in seven of the eight technical replicates, were detected by MetaPhlAn2 (version 2.7.7) (Supplementary Figure 3) Figure 3). An additional 25 taxa were detected, of which 18 and 7

were identified at the species and at the genus level, respectively. Contaminants were present at a higher concentration in three technical replicates (R3, R7, R8) with the most frequent contaminant (Methanobrevibacter spp.) being present in 5 of the 8 replicates (Supplementary Figure 5).  (Supplementary Figure 3). ColiGuard® contained a total of 20 contaminants, of which 16 and 4 were identified at the species and the genus level, respectively. Contaminants were present at a higher level in two technical replicates (R5, R7), with R7 displaying the most diverse and highest contamination rate (R7: 14 taxa; total contaminating reads: 2.67%; R5: 9 taxa; total contaminating reads: 0.30%).

(Supplementary Figure 5).


## Technical controls in metagenomic studies and methodological limitations

Taxonomic assignment of the raw reads from the positive controls was performed with MetaPhlAn2 8 which relies on a ca. 1M unique clade-specific markers derived from 17,000 reference genomes. Such a database to map against the positive controls suffices as these organisms are cultivable, and for this reason they are widely studied hence the sequences are known. This is not the case for real-world samples where mapping against a database (the completeness of which relies on studied and often cultivable organisms) would narrow the view on the true diversity within the sample.

Positive controls with well-studied members and known ratios within the samples have proven to be a valuable approach to assess consistency among technical replicates across batches and to detect possible biases derived from the DNA extraction method.

Systematic taxonomic bias in microbiome studies, resulting from differences in cell wall structures between Gram positive and Gram negative bacteria, have previously been reported; bead beating and sample treatment with enzymatic cocktails can modestly reduce this bias 9-12 . Although we implemented such steps in our workflow, it seems that, from the read abundance of our mock community, which contained three In terms of contamination we concluded that: a) contamination in our study was not batch specific; b) a problem of sample cross-contamination may have occurred at the DNA extraction step between neighbouring wells. During the bead-beating step of DNA extraction, the deep-well plate is sealed with a rubber sealing mat, rotated and placed in a plate shaker for the bead beating to take place. As leakage was observed around the wells despite the presence of the sealing mat, we consider that sample cross-contamination is most likely to occur during this step.


## Taxonomic profiling of samples

All 


## Alpha and beta diversity

The abundance profile of all samples, based on the 16S rRNA reads that passed 


## Potential uses

This dataset can be utilised to assess a broad range of ecological questions pertaining to host-associated microbial communities of the post-weaning piglet. These include the assessment of: 1. the compositional and functional core faecal microbiome of the postweaning piglet, 2. the microbial changes that piglets undergo between the first and the 5 th week after weaning, 3. the degree of strain-host specificity, 4. the variability of microbiomes within or between host species, 5. the variability of microbiomes between different cross-breeds and small age differences of the hosts, 6. the degree of strain transfer from mothers to piglets, 7. the effects of two probiotic treatments and of intramuscular antibiotic treatment on the post-weaning pig faecal microbiome, 8.

species co-occurrence and co-exclusion, 9. the repertoire of antimicrobial resistance genes and how it is impacted by antibiotic and probiotic treatment, 10. the extent of within-host and population evolution of microbes over a 5-week period.


## Data availability

The sequencing reads from each sequencing library have been deposited at NCBI     Scholarships. NSW DPI approved the paper before submission for publication.


## Competing interests

D-Scour™ was sourced from International Animal Health Products (IAHP).

ColiGuard® was developed in a research project with NSW DPI, IAHP and AusIndustry Commonwealth government funding.


## Author contributions

Pig  


of D-Scour™ paste daily for 14 d; Room 3 -oral 1 g/pig of ColiGuard® paste daily for 14 d; Room 4 -intramuscular (IM) injection of antibiotic administered at 0.1 mL per pig daily from a 200 mg/mL solution for a total treatment duration of 5 d. On the day following the final neomycin treatment (day 8), 36 pigs were moved from Room 4 to Room 2 (n=18, 6 in each pen, pens g-i), and to Room 3 (n=18, 6 in each pen, pens g-i) (Supplementary Figure 1B). The following day (day 9), oral administration of D-Scour™ (1 g/pig) and of ColiGuard® (1 g/pig) commenced for pigs in Room 2 pens g-i and in Room 3 pens g-i, respectively, and continued for a period of 14 days. Assignment of the 36 neomycin-treated pigs to the treatment groups neomycin+D-Scour™ (n=18; Room 2 pens g-i) and neomycin+ColiGuard® (n=18; Room 3 pens g-i), was carried out by distributing them so that the mean weight of the animals distributed across pens and rooms was similar. By this time point, each occupied pen in the trial housed six pigs. (Supplementary Figure 1B) From that time, twelve piglets from the original 126 were no longer present, as they had been euthanised as pre-treatment controls at the start of the trial. Faecal samples were collected from all piglets once per week and from a subset (n=48 pigs; 8 from each of the six cohorts) twice per week over the 6-week study period (Figure 2). From each piglet, faeces were collected per rectum with new disposable gloves; where minimal or no faeces could be collected on a collection day, sampling was performed the following morning. Samples were placed in 50 mL Falcon tubes and stored at 4 o C within 30 mins of sample collection for a minimum of 30 mins and a maximum period of 6 h.


Faecal sample processingSamples (3g/pig) were mixed with 15 mL PBS (200 mg/mL), in sterile stomacher bags and homogenized with a Bio-Rad stomacher. The homogenised samples were divided in replicates: one replicate was stored directly at -80 o C and one replicate was supplemented with glycerol (20% v/v) (Sigma-Aldrich) then stored at -80 o C. In addition, single time-point faecal samples from the dams of the trial pigs (n=42) were obtained from the commercial facility of origin and were pre-processed at EMAI as described above. Thus, a total of 911 unique samples, between one and ten samples per subject (mean: 4.8; median: 3) (Supplementary

## subtilis strain 168 ,
168Enterococcus faecium, Staphylococcus aureus ATCC25923, Staphylococcus epidermidis ATCC35983) and three Gram negative (Enterobacter hormaechei CP_032842, Escherchia coli K-12 MG1655, Pseudomonas aeruginosa PAO1) bacterial strains from -80℃ stocks were cultured at 37 o C for 16 h in LB (Luria-Bertani) then centrifuged at 14,000 rpm for 10 mins. From the resulting pellets, 1 g was transferred to 1 mL of LB and homogenised and a 1:10 dilution of this was made for each bacterial culture. Ten microliters of bacterial suspension from each of the cultures was used to determine the number of colony forming units (CFU) in the original suspension in the following manner: by further diluting tenfold in LB and by plating onto 1.6% LB agar plates and incubated overnight. The remaining suspensions (990 μL from each bacterial culture) were pooled into a sterile tube, then aliquoted into Eppendorf tubes in 500 μL volumes/tube. As a washing step, Eppendorf tubes were centrifuged at 14,000 rpm for 10 mins, 500 μL PBS was added to the pellet and subsequently resuspended. These tubes constituted the mock community samples and were stored at -80 o C. Expected proportions of the mock community members were determined from the estimated colony forming units (CFU) multiplied by the genome size and were as follows: 8.7:13.0:7.7:16.7:38.9:14.5:0.4 for S. aureus, B. subtilis, E.

## libraries 4 .
4For each sample, 10 nanograms of input gDNA in 10 ul ultrapure water (Invitrogen) was mixed with 10 ul of 1:50 diluted BLT beads, 25 ul of 2x laboratorymade tagmentation buffer 20 mM Tris (pH 7.6) (Chem-Supply), 20 mM MgCl (Sigma), and 50% (v/v) Dimethylformamide (DMF) (Sigma); the final volume for each tagmentation reaction was 45 ul. Following, 10 ul of 0.2% of sodium dodecyl sulphate (SDS; Sigma) was added into each sample to stop tagmentation. Beads were then washed three times using 100 ul of washing solution which was filtered prior to use (0.22 μm MF-Millipore™ membrane). The washing solution consisted of 10% polyethylene glycol (PEG) 8000 (Sigma), 0.25M NaCl (Chem-Supply) in Tris-EDTA buffer (TE) (Sigma). Library amplification was carried out using the PrimeSTAR GXL DNA Polymerase kit (Takara), according to the manufacturer protocol. Each PCR reaction contained 10 ul of 5x GXL buffer, 4 ul of 25 mM dNTPs, 2 ul of PrimeStar GXL polymerase, and 19 ul of nuclease free water. The PCR mix was added into washed BLT beads. Then, 5 ul of each custom synthesized 96-well plate Illumina Adapter Oligos i5 and i7 (i7: IDT plate#: 11680765; i5: IDT plate#: 11680754) was added to a final concentration of 0.555 uM to each reaction. Each sample's PCR reaction had a final volume of 45 ul. The following conditions were used: 3 min at 68ºC, 3 min at 98ºC, 12 cycles of [45 sec at 98ºC -30 sec at 62ºC -2 min at 68ºC], 1 min at 68ºC and hold at 10ºC. Following the amplification step, samples were centrifuged at 280 x g for 1 min and stored between 1 and 5 days at 4°C.Size selection and purificationSamples from the same 96-well plates were pooled into one tube by taking 5 μL from each library. This generated 10 pooled samples, one for each plate. A master pool was created by pooling 5 μL from the pool of each plate into a single pool. Forty microliters from each of the 10 plate pools and 40 μL from the master pool underwent library size selection and purification using equal volumes of SPRIselect beads (Beckman Coulter) and ultrapure water (Invitrogen). Sample cleaning with SPRIbeads was performed as described previously4 . A purified master pool comprising samples from all plates, and purified pools of individual plates to check for platespecific anomalies, were diluted to 4 nM and fragment size distribution was assessed using the High Sensitivity DNA kit on the Bioanalyzer (Agilent Technologies, USA).

## Figure 4 )
4. Taxonomic assignment of the mock community samples reported one contaminating species in one of the eight replicates: Lactobacillus salivarius (mean: 0.008)(Supplementary Figure 5).


Gram negative and four Gram positive strains, a bias towards Gram negative taxa may still be present. However, this cannot be conclusively determined by our study as theexpected amount of cells was derived from the colony forming units (CFUs) method, which does not account for dead cells. Those cells would nonetheless contribute to gDNA to the sample 13-16 . Knudsen et al (2016) compared various DNA isolation methods with distinct sample types and reported a reduced bias when using an adapted version of the QIAamp Fast DNA Stool Mini Kit (Qiagen) 12 .


raw reads were analysed with SortMeRNA 1 (version 4.0.0) to extract reads containing 16S rRNA genes. Extraction was performed by mapping reads against the silva-bac-16s-id90.fasta database with -fastx --blast 1 -num_alignments 1 parameters settings (script: sortmerna.sh). Over 60 million reads (n=60,584,650) contained 16S rRNA genes passing the E-value threshold for filtering (E-value ≤ 0.0001). These reads occupy between 36.4% and 37.1% of each sample (script: sortmerna_counts.R). Reads were further filtered based on E-value cutoff (E-value ≤ 1x10 -30 ), sequence identity (identity ≥ 80%), and alignment length (length ≥ 100 bp). Over half of the reads (n=32,419,310) passed the filtering (script: sortmerna_filter.sh) and were classified using the RDP classifier 2 (version 2.13), a naïve Bayesian classifier, that classifies 16S rRNA sequences into the new higher-order taxonomy proposed in Bergey's Taxonomic Outline of the Prokaryotes (2 nd ed., release 5.0, Springer-Verlag, New York, NY, 2004) (script: RDP_Krona.sh). Most abundant Phyla in the piglet population (n=126) were: Firmicutes (75.14%), Bacteroidetes (13.70%), Actinobacteria (5.31%), Proteobacteria (3.17%), Spirochaetes (0.72%), Synergistetes (0.40%). Most abundant Phyla in the mothers (n=42) were: Firmicutes (84.75%), Proteobacteria (6.30%), Bacteroidetes (5.44%), Actinobacteria (1.85%), Verrucomicrobia (0.36%), Synergistetes (0.23%). The RDP classifier estimates the confidence of an assignment using the number of times a genus is selected out of 100 bootstrap trials. Assignments at the phylum level had a mean confidence of 0.93 (scale 0-1; median=1.00) (script: RDP_analyze.R).

## FiguresFigure 1 .
1Microbial composition and diversity of the porcine microbiome. Taxonomic profiling based on the analysis of reads containing bacterial 16S rRNA genes extracted from shotgun metagenomic data. A) The chart was generated using Krona 3 , which displays hierarchically organized nodes of the taxonomic tree based on their relative abundance. Distinct colours represent separate domains of life. B) Principal coordinate analysis (PCoA) plots of the pig faecal microbiomes explaining 19.3% of variance (Axis.1) and 12.5% of variance (Axis.2). Samples are colored by day of collection from the first week post-weaning (t0) to the 5th week post-weaning (t10). Panels are split by cohort: Mothers (tM; bright pink; n=42);

## Figure 2 .
2Timeline. Timeline of the animal trial indicating the start and the length of the treatment for each cohort, the sample collection points, and the piglets' age during the trial. Piglets were allowed 2 days of acclimatisation after the arrival on the site of the trial and before the start of the treatments (pink: placebo paste; yellow: probiotic D-Scour™ formulation; green: probiotic ColiGuard® formulation; blue: antibiotic neomycin intramuscular injection). Large triangles (dark blue) indicate main days of sampling where all piglets were sampled (n=126). Small triangles (light blue) indicate sampling points from a subset of the piglets (8 per cohort; n=48).

## Figure 3 .Supplementary Figure 1 :Supplementary Figure 2 :Supplementary Figure 3 :Supplementary Figure 4 :Supplementary Figure 5 :
312345Workflow.A schematic representation of the experimental workflow from sample collection (yellow), through sample processing and sequencing (orange), to the preliminary data analysis (blue).Additional files Piglets placements across rooms and pens. Read count distribution. Taxonomic assignment of reads from positive control samples. Expected and observed relative abundance of mock community members. Heatmap reporting the contaminating species found within the technical replicates of the positive controls.Supplementary


Trial: TC, LF, DG, TZ, GJE, AED, SPD DNA extraction: DG, ML Library prep, robot pooling: DG, ML, KA, AED Sequencing data processing: MZD,

## Table 1 ).
1The pig facility consisted of four environmentally controlled rooms (Rooms 1-4) 

with air conditioning, concrete slatted block flooring with underground drainage and 

open rung steel pens (Supplementary 


Streptococcus salivarius subspecies thermophilus, with an additional 20 mg of garlic extract (Allium sativum). The probiotic ColiGuard is a probiotic formulation developed for the treatment of entero-toxigenic Escherichia coli (ETEC) in weaner pigs, developed in collaboration between the NSW DPI and International Animal Health Products, containing undefined concentrations of Lactobacillus plantarum and Lactobacillus salivarius.faecium, S. epidermidis, P. aeruginosa, E. cloacae, and E.coli respectively. 

The two probiotic formulations used in this study were used as two additional 

positive controls. D-Scour™ is a commercially available probiotic formulation for 

livestock, with each gram containing 180 million CFU of the following: Lactobacillus 

acidophilus, Lactobacillus delbrueckii subspecies bulgaricus, Lactobacillus 

plantarum, Lactobacillus rhamnosus, Bifidobacterium bifidum, Enterococcus faecium, 




all samples was obtained with MULTIQC 5 . The presence of PCR duplicates was assessed by feeding read pairs to dedupe.sh (http://jgi.doe.gov/data-and-tools/bbtools) (parameters: ac=f). Nextflow 6 (version 18.10.1) was used to manage processing of the data on the HPC.(parameters: k=31 hdist=1), and quality filtering (parameters: ftm=0 qtrim=r 

trimq=20), were performed using bbduk.sh (http://jgi.doe.gov/data-and-tools/bbtools; 

bbmap version 38.22). Piglet samples (n=825) had a median count of 32,949,208 clean 

paired reads (mean=35,557,149) (script: readcounts.R). Quality assessment of raw 

reads 
was 
carried 
out 
using 
FASTQC 

(http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) and a combined report of 




. One sample failed to sequence, reporting zero counts for any species. The observed mean relative Isolate E. cloacae C15117, used in this study for the make-up of the mock community, was recently found to be most closely related to the Enterobacter hormaechei phylogenomic group C type strain DSM 16687 and therefore re-identified as Enterobacter hormaechei subsp. oharae 7 . For this reason, taxonomic assignment by MetaPhlAn2 attributed the reads to E. cloacae instead. The expected proportions of the mock community members were derived from the CFU by the genome size. Basedabundances were as follows: B. subtilis (mean±SD: 2.92±0.994), E. cloacae 

(mean±SD: 38.0±6.404), E. faecium (mean±SD: 0.97±0.081), E.coli (mean±SD: 

10.12±1.480), E.coli unclassified (mean±SD: 7.83± 1.755), P. aeruginosa (mean±SD: 

26.72±3.026), S. aureus (mean±SD: 9.90±3.613), S. epidermidis (mean±SD: 

3.54±1.435). on the expected and the observed relative abundance, we found, with the exception of 

S. aureus (exp: 8.7% obs: 9.9%), three Gram positive members to be under-

represented (B. subtilis: exp: 13.0% obs: 2.9%; E. faecium: exp: 7.7% obs: 1.0%; S. 

epidermidis: exp: 16.7% obs: 3.5%) and, with the exception of P. aeruginosa (exp: 

38.9% obs: 26.8%), two Gram negative members to be over-represented (E. cloacae: 

exp: 14.5% obs: 38.0%; E.coli: exp: 0.4% obs: 7.8-10.1%) (Supplementary 


The probiotic D-Scour™ is expected to contain, per gram, a total of 180 million CFU of Lactobacillus acidophilus, Lactobacillus delbrueckii subspecies bulgaricus, Enterococcus faecium, Streptococcus salivarius subspecies thermophilus in unknown proportions. From taxonomic analysis with MetaPhlAn2, we can conclude that 6 of the 7 expected species were determined to be present in the replicates in the following mean relative abundances: Bifidobacterium bifidum: mean±SD: 40.01±12.558;Lactobacillus plantarum, Lactobacillus rhamnosus, Bifidobacterium bifidum, 

Enterococcus faecium: mean±SD: 30.98±13.472; Lactobacillus delbrueckii: 

mean±SD: 11.56±7.148; Lactobacillus plantarum: mean±SD: 6.23±7.863; 

Lactobacillus rhamnosus: mean±SD: 2.08±1.226; Streptococcus thermophilus: 

mean±SD: 4.28±1.523. Lactobacillus acidophilus was not detected and Lactobacillus 

helveticus was detected instead (Lactobacillus helveticus: mean±SD: 4.75±2.431) 

(Supplementary 


Taxonomic analysis of the technical replicates of the probiotic ColiGuard® also showed a species profile consistent with the expected profile, with Lactobacillus salivarius and Lactobacillus plantarum in a 9:1 ratio (Lactobacillus salivarius:mean±SD: 93.52±1.617; Lactobacillus plantarum: mean±SD: 6.10±1.134) across the 

replicates 


Short Read Archive under project PRJNA526405. All supplementary figures and tables are provided as additional files. The scripts for the automated robot pooling (robot_pooling.py), for the sequence data processing (initial.nf), and for the data (https://github.com/GaioTransposon/metapigs_base; tag: GigaScience). For the data analysis scripts, the R language (version 3.6.3) and the following packages were used:analysis 
can 
be 
found 
in 
our 
Github 
repository 

readr (v1.4), readxl (v1.3.1), tidyr (v1.1.2), tidyverse (v1.3.0), ggplot2 (v3.3.3), dplyr 

(v1.0.3), gridExtra (v2.3), pheatmap (v1.0.12), cowplot (v1.1.1), splitstackshape 

(v1.4.8), pairwiseCI (v0.1-27). 



## Table 1 :Table 2 :Table 3 :
123Animal details. Metadata. Barcodes. This work was supported by the Australian Research Council, linkage grant LP150100912. This project was funded by the Australian Centre for Genomic Epidemiological Microbiology (Ausgem), a collaborative partnership between the NSW Department of Primary Industries and the University of Technology Sydney. TZ and DG are recipients of UTS International Research and UTS President'sSupplementary Supplementary 


Axis.1 [19.3%] "OT.One Liquid Handling Robot" by OpenTrons is licensed under CC BY 4.0Axis.2 [12.7%] 

collection_date • • 

• • 
• • 
• • 
• • 
• • 
• • 

t0 
t2 
t4 
t6 
t8 
t10 
tM 

A 
B 

Figure 
Click here to 
access/download;Figure;metapigs_base_DataNote_Figu 

Neomycin + ColiGuard 

Neomycin + D-Scour 

Neomycin 

ColiGuard 

Control 

D-Scour 

Piglets' age (weeks) 
3-4 
4-5 
5-6 
6-7 
7-8 
8-9 
Week 1 
Week 2 
Week 3 
Week 4 
Week 5 

Treatment Cohorts 

Sample 
collecMon 
subset 

all 

Normalization 

pooling 

Positive controls: 

mock community 
D-Scour 
ColiGuard 

DNA extraction 

Hackflex 
Library prep 

pool equal 
volumes 

MiSeq 

Pig Trial 

NovaSeq 6000 S4 

adapter 
trimming 
PhiX DNA 
removal 
quality 
filtering 
bbduk.sh 
nextflow 

FastQC; MultiQC 

quality assessment 

MetaPhlAn2 

map reads against 
database 

plot observed 
relative 
abundance 

data deposited at: 
PRJNA526405 

1x 

"Sequencer" by UGA 
CAES/Extension is 
licensed under CC 
BY-NC 2.0 

911 samples 

SortMeRNA 

extraction of 16S rRNA 
gene containing reads 

positive controls 

RDP classifier 

classification of 16S 
rRNA gene containing 
reads 

Krona 

microbial community 
visualization 

filtered reads 
PhyloSeq 

α and β diversity 

AcknowledgementsWe would like to thank Shayne Fell for the on site help in the pig trial.Products for providing access to the probiotic supplements and to the Australian Centre for Genomic Epidemiological Microbiology (Ausgem) for financially supporting this study. We thank the peer reviewers whose comments improved the quality of this manuscript.
SortMeRNA: fast and accurate filtering of ribosomal RNAs in metatranscriptomic data. E Kopylova, L Noé, H Touzet, Bioinformatics. 28Kopylova, E., Noé, L. & Touzet, H. SortMeRNA: fast and accurate filtering of ribosomal RNAs in metatranscriptomic data. Bioinformatics 28, 3211-3217 (2012).

Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy. Q Wang, G M Garrity, J M Tiedje, J R Cole, Appl. Environ. Microbiol. 73Wang, Q., Garrity, G. M., Tiedje, J. M. & Cole, J. R. Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy. Appl. Environ. Microbiol. 73, 5261-5267 (2007).

Interactive metagenomic visualization in a Web browser. B D Ondov, N H Bergman, A M Phillippy, BMC Bioinformatics. 12Ondov, B. D., Bergman, N. H. & Phillippy, A. M. Interactive metagenomic visualization in a Web browser. BMC Bioinformatics 12, 1-10 (2011).

Hackflex: low cost Illumina sequencing library construction for high sample counts. D Gaio, 779215Gaio, D. et al. Hackflex: low cost Illumina sequencing library construction for high sample counts. bioRxiv 779215 (2019).

MultiQC: summarize analysis results for multiple tools and samples in a single report. P Ewels, M Magnusson, S Lundin, M Käller, Bioinformatics. 32Ewels, P., Magnusson, M., Lundin, S. & Käller, M. MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics 32, 3047-3048 (2016).

P A Ewels, Community curated bioinformatics pipelines. Ewels, P. A. et al. nf-core : Community curated bioinformatics pipelines. (2019).

High contiguity genome sequence of a multidrugresistant hospital isolate of Enterobacter hormaechei. L G Monahan, Gut Pathog. 113Monahan, L. G. et al. High contiguity genome sequence of a multidrug- resistant hospital isolate of Enterobacter hormaechei. Gut Pathog. 11, 3 (2019).

MetaPhlAn2 for enhanced metagenomic taxonomic profiling. D T Truong, Nat. Methods. 12Truong, D. T. et al. MetaPhlAn2 for enhanced metagenomic taxonomic profiling. Nat. Methods 12, 902-903 (2015).

Determination of microbial diversity in environmental samples: pitfalls of PCR-based rRNA analysis. F Von Wintzingerode, U B Göbel, E Stackebrandt, FEMS Microbiol. Rev. 21von Wintzingerode, F., Göbel, U. B. & Stackebrandt, E. Determination of microbial diversity in environmental samples: pitfalls of PCR-based rRNA analysis. FEMS Microbiol. Rev. 21, 213-229 (1997).

Biases from different DNA extraction methods in intestine microbiome research based on 16S rDNA sequencing: a case in the koi carp. Z Han, J Sun, A Lv, A Wang, Cyprinus carpio var. Koi. MicrobiologyOpen. 8626Han, Z., Sun, J., Lv, A. & Wang, A. Biases from different DNA extraction methods in intestine microbiome research based on 16S rDNA sequencing: a case in the koi carp, Cyprinus carpio var. Koi. MicrobiologyOpen 8, e00626 (2019).

Complete, closed bacterial genomes from microbiomes using nanopore sequencing. E L Moss, D G Maghini, A S Bhatt, 10.1038/s41587-020-0422-6Nat. Biotechnol. Moss, E. L., Maghini, D. G. & Bhatt, A. S. Complete, closed bacterial genomes from microbiomes using nanopore sequencing. Nat. Biotechnol. (2020) doi:10.1038/s41587-020-0422-6.

Impact of sample type and DNA isolation procedure on genomic inference of microbiome composition. B E Knudsen, MSystems. 1Knudsen, B. E. et al. Impact of sample type and DNA isolation procedure on genomic inference of microbiome composition. MSystems 1, (2016).

Consistent and correctable bias in metagenomic sequencing experiments. M R Mclaren, A D Willis, B J Callahan, Elife. 846923McLaren, M. R., Willis, A. D. & Callahan, B. J. Consistent and correctable bias in metagenomic sequencing experiments. Elife 8, e46923 (2019).

Equivolumetric Protocol Generates Library Sizes Proportional to Total Microbial Load in 16S Amplicon Sequencing. G N F Cruz, A P Christoff, L F De Oliveira, Front. Microbiol. 12425Cruz, G. N. F., Christoff, A. P. & de Oliveira, L. F. V. Equivolumetric Protocol Generates Library Sizes Proportional to Total Microbial Load in 16S Amplicon Sequencing. Front. Microbiol. 12, 425 (2021).

Absolute bacterial cell enumeration using flow cytometry. F Ou, C Mcgoverin, S Swift, F Vanholsbeeck, J. Appl. Microbiol. 123Ou, F., McGoverin, C., Swift, S. & Vanholsbeeck, F. Absolute bacterial cell enumeration using flow cytometry. J. Appl. Microbiol. 123, 464-477 (2017).

Flow cytometric viability assessment of lactic acid bacteria starter cultures produced by fluidized bed drying. G Bensch, Appl. Microbiol. Biotechnol. 98Bensch, G. et al. Flow cytometric viability assessment of lactic acid bacteria starter cultures produced by fluidized bed drying. Appl. Microbiol. Biotechnol. 98, 4897-4909 (2014).

phyloseq: an R package for reproducible interactive analysis and graphics of microbiome census data. P J Mcmurdie, S Holmes, PloS One. 8McMurdie, P. J. & Holmes, S. phyloseq: an R package for reproducible interactive analysis and graphics of microbiome census data. PloS One 8, (2013).