# Genome-Wide Survey Reveals the Microsatellite Characteristics and Phylogenetic Relationships of Harpadon nehereus

CorpusID: 239888430
 
tags: #Medicine, #Environmental_Science, #Biology

URL: [https://www.semanticscholar.org/paper/f64ff15379e5f75523ecabdfc5a15316c32a5b2d](https://www.semanticscholar.org/paper/f64ff15379e5f75523ecabdfc5a15316c32a5b2d)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Genome-Wide Survey Reveals the Microsatellite Characteristics and Phylogenetic Relationships of Harpadon nehereus
2021

Tianyan Yang 
316022ZhoushanChina

Xinxin Huang 
316022ZhoushanChina

Zijun Ning 
316022ZhoushanChina

Tianxiang Gao 
316022ZhoushanChina

Genome-Wide Survey Reveals the Microsatellite Characteristics and Phylogenetic Relationships of Harpadon nehereus

Curr. Issues Mol. Biol
43202110.3390/cimb43030091Received: 3 September 2021 Accepted: 22 September 2021Article Citation: Yang, T.; Huang, X.; Ning, Z.; Gao, T. Genome-Wide Survey Reveals the Microsatellite Characteristics and Phylogenetic Relationships of Harpadon nehereus. Academic Editor: Dumitru A. Iacobas


## Introduction

Harpadon nehereus (Hamilton, 1822), also known as Bombay duck or nomei fish, is generally distributed in the estuarine and nearshore shallow waters of the Indian Ocean and the Western Pacific Ocean [1]. As a familiar marine lizardfish, H. nehereus is welcomed by consumers for its delicious taste and high nutritional value. It is admeasured that the protein content reaches up to 70% of the dry weight and the calcium content ranges from 1500 to 2500 mg/100 g [2]. Nowadays, it is one of the most important commercial fisheries along the coast of southeast China, Pakistan, India and the Bay of Bengal region [3][4][5][6]. In China, the potential amount of H. nehereus in the East China Sea was estimated to surpass 5000 tons, which indicated that H. nehereus occupied a significant position in the coastal marine ecosystem of China. Nevertheless, in recent years, a recession of H. nehereus resources has emerged along with the increasing proportion of annual capture yield [6][7][8]. Therefore, it is necessary to strengthen protection of this fishery in a sustainable way, and further ensure rational exploitation and continued utilization of H. nehereus resources.

Investigation of genetic background is an essential precondition for fishery resource management [9]. However, most studies on H. nehereus have been conducted focusing on stock assessment, morphological traits, biological habits and nutrition processing [10][11][12][13], and limited studies involve the field of genetics. For example, these studies include population genetic structure and genetic variation analysis based on simple sequence repeat (SSR), sequence-related amplified polymorphism (SRAP) and mitochondrial DNA (mtDNA) markers [14][15][16][17], the molecular phylogenetic relationship revealed by mtDNA Cyt b, 16S rRNA and the complete mtDNA genome [17][18][19], and the expression analysis of calcium cycling genes by transcriptome sequencing [20]. With the accomplishment of the Human Genome Project (HGP) early in 2001, a new era of genomics has arrived in the field of genetics research. Thus far, more and more genome sequencings of economic fishes such as Larimichthys crocea, Cynoglossus semilaevis, Salmo salar, Oncorhynchus mykiss and Gadus morhua have been carried out along with the rapid development of high-throughput sequencing technology [21]. However, the genomic resource for H. nehereus is still extremely scarce and its full draft genome remains unclear. Therefore, in this study, we conducted a genomic survey of H. nehereus via next-generation sequencing (NGS) technology for the first time to investigate its genomic profile, excavate the SSR markers and further identify the taxonomic position and phylogenetic relationship of H. nehereus. The results might provide new insight into genetic resource conservation and developmental utilization for H. nehereus.


## Materials and Methods


### Sample Collecting

A specimen of a male H. nehereus (body length 16.1 cm and body weight 26.9 g) was obtained from coastal waters by a trawl net in autumn 2020. Approximately 2-3 g fresh musculature below the dorsal fin was snipped and soaked in liquid nitrogen as soon as the fish was captured. The sample was immediately brought to the Fisheries Ecology and Biodiversity Laboratory (FEBL) of Zhejiang Ocean University and stored in a −80 • C ultralow temperature freezer. All animal experiments were conducted according to the guideline and approval of the Ethics Committee for Animal Experimentation of Zhejiang Ocean University, Zhoushan, China. The project identification code was ZJOU-ECAE20210128 and approval date was 6 January 2021.


### DNA Extraction, Library Construction and Illumina Sequencing

The genomic DNA was isolated using a standard phenol-chloroform method [22]. The integrity of DNA was examined by 1% agarose gel electrophoresis. The quality of DNA was assessed using a NanoPhotometer spectrophotometer (Implen, Munich, Germany), Qubit 2.0 fluorometer (Invitrogen, Waltham, MA, USA) and Agilent 2100 bioanalyzer (Agilent, Santa Clara, CA, USA). The qualified DNA sample was randomly interrupted by an ultrasonic crusher (Covaris M220, Woburn, MA, USA), and the DNA fragments (300-350 bp) were used to construct two paired-end DNA libraries followed by terminal repair, adding A-tail and sequencing adaptors, purification and PCR amplification. The constructed library was then sequenced by Wuhan Gooalgene Technology Co., Ltd., Wuhan, China (https://www.gooalgene.com/) (accessed on 18 March 2021) based on the Illumina Nova platform with a read length of 2 × 150 bp. The sequencing data were deposited in the short-read archive (SRA) database (http://www.ncbi.nlm.nih.gov/sra/) (accessed on 15 June 2021) under the accession number PRJNA738314.


### Sequence Quality Control, Assembly and K-mer Analysis

Raw data were first converted to single-sample FASTQ files through base calling and then filtered by SOAPnuke v1.5 [23] with the following criteria: (1) removing reads with splice junctions, (2) discarding duplicated reads caused by PCR reaction or other reasons, (3) deleting reads with N base (unable to determine base information) ratio greater than 10% and low-quality bases (base quality ≤ 5) more than 50% of the total length. Clean data were de novo assembled into contigs and scaffolds by SOAPdenovo v2.01 software [24]. The K-mer analysis was performed to estimate genome size, heterozygosity and repetitive sequences of the genome using Jellyfish v2.2.10 (http://www.genome. umd.edu/jellyfish.html) (accessed on 9 April 2021) and a de Bruijn graph assembler GenomeScope (http://qb.cshl.edu/genomescope/) (accessed on 21 April 2021). Genome size could be expressed as the formula: K-mer number/peak depth [25]. Non-overlapping sliding windows of 10 kb along the assembled sequence were used to calculate the GC content and average depth [26,27].


### Microsatellite Identification and Phylogenetic Analysis

Perl script MISA (MIcroSAtellite identification tool) was used to identify simple sequence repeats (SSRs) in the genome of H. nehereus [28]. The parameters were set for the detection of di-, tri-, tetra-, penta-and hexanucleotide SSR motifs with a minimum of 6, 5, 5, 5 and 5 repeats, respectively. The distribution and frequency of SSRs were computed and mapped by Microsoft Excel. The complete mitochondrial DNA (mtDNA) sequence of H. nehereus was assembled by MITObim version 1.9.1 (https://github.com/ chrishah/MITObim) (accessed on 20 May 2021) and annotated using the online tool MITOS (http://mitos2.bioinf.uni-leipzig.de/index.py) (accessed on 26 May 2021). The mtDNA sequence was corrected manually comparing with published sequences of H. nehereus (GenBank accession numbers: JX534239 and MH204885). The nucleotide composition and genetic relationships were analyzed by MEGA 11 [29]. Phylogenetic analysis was performed using fifty-eight complete mtDNA genomes of related species downloaded from GenBank, with Ijimaia dofleini (GenBank accession number: AP002917) selected as an outgroup. The maximum likelihood (ML) method was used to construct the phylogenetic tree based on concatenated amino acids of the 13 protein-coding genes (PCGs).


## Results


### Sequencing Data Statistics and Sequence Quality Evaluation

After sequence filtering and correction, a total of 68.74 Gb high-quality data were generated by the Illumina Nova platform with high-throughput paired-end sequencing in this study. The Q20 (base quality > 20) value, Q30 (base quality > 30) value and GC content were 96.64%, 91.35% and 41.78%, respectively, with the approximate sequencing depth of 50×. Ten thousand pairs of reads were randomly selected from the filtered high-quality data and compared with the Nucleotide Sequence Database (NT) from the National Center for Biotechnology Information (NCBI) database using the Basic Local Alignment Search Tool (BLAST). The results showed the library was successfully compared to the related species of H. nehereus, which proved that there was no obvious exogenous contamination during the library construction. The proportion of single bases presented the separation of AT content and GC content. In addition, the N content was close to zero ( Figure 1a). All of this suggested that the sequencing quality was good.


### Microsatellite Identification and Phylogenetic Analysis

Perl script MISA (MIcroSAtellite identification tool) was used to identify simple sequence repeats (SSRs) in the genome of H. nehereus [28]. The parameters were set for the detection of di-, tri-, tetra-, penta-and hexanucleotide SSR motifs with a minimum of 6, 5, 5, 5 and 5 repeats, respectively. The distribution and frequency of SSRs were computed and mapped by Microsoft Excel. The complete mitochondrial DNA (mtDNA) sequence of H. nehereus was assembled by MITObim version 1.9.1 (https://github.com/chrishah/MI-TObim) (accessed on 20 May 2021) and annotated using the online tool MITOS (http://mi-tos2.bioinf.uni-leipzig.de/index.py) (accessed on 26 May 2021). The mtDNA sequence was corrected manually comparing with published sequences of H. nehereus (GenBank accession numbers: JX534239 and MH204885). The nucleotide composition and genetic relationships were analyzed by MEGA 11 [29]. Phylogenetic analysis was performed using fifty-eight complete mtDNA genomes of related species downloaded from GenBank, with Ijimaia dofleini (GenBank accession number: AP002917) selected as an outgroup. The maximum likelihood (ML) method was used to construct the phylogenetic tree based on concatenated amino acids of the 13 protein-coding genes (PCGs).


## Results


### Sequencing Data Statistics and Sequence Quality Evaluation

After sequence filtering and correction, a total of 68.74 Gb high-quality data were generated by the Illumina Nova platform with high-throughput paired-end sequencing in this study. The Q20 (base quality >20) value, Q30 (base quality >30) value and GC content were 96.64%, 91.35% and 41.78%, respectively, with the approximate sequencing depth of 50×. Ten thousand pairs of reads were randomly selected from the filtered highquality data and compared with the Nucleotide Sequence Database (NT) from the National Center for Biotechnology Information (NCBI) database using the Basic Local Alignment Search Tool (BLAST). The results showed the library was successfully compared to the related species of H. nehereus, which proved that there was no obvious exogenous contamination during the library construction. The proportion of single bases presented the separation of AT content and GC content. In addition, the N content was close to zero ( Figure 1a). All of this suggested that the sequencing quality was good.  


### Genome Assembly, Heterozygosity and Repeat Prediction

The de Bruijn graph-based assembler SOAPdenovo was applied to generate a total length of 1.35 Gb contigs with an N50 length of 596 bp. A total number of 2,539,084 sequences (the max length was 92,118 bp) comprised approximately 1.36 Gb scaffolds with an N50 length of 1568 kb (Table 1). K-mer analysis (K = 17) was used to estimate the genomic characteristics. The K-mer distribution map is shown in Figure 1b. The K-mer number obtained from the sequencing data was 54,514,460,352, with a K-mer depth of 39. The revised genome size of the diploid fish H. nehereus was 1315 Mb after eliminating the effects of erroneous K-mers. The heterozygosity ratio and repeat sequence ratio were 0.67% and 52.49%, respectively. 


### The Distribution and Characteristics of SSR Loci

A total of 1,027,651 SSR-containing motif repeats were identified, with a total SSR length of 27,355,367 bp ( Table 2). Among all these SSRs, the mono-, di-, tri-and tetranucleotide repeats contributed nearly 99% of SSRs in H. nehereus. The dinucleotide repeat was the most abundant SSR marker, accounting for 54.35% of the total SSR markers, which was followed by mononucleotide (25.96%) and tetranucleotide (10.26%) repeats. The percentage of hexaucleotide repeats was the lowest of all (0.53%). The frequency of different repeat motifs is presented in Figure 2. The frequency of A/T repeats was the highest within the four types of mononucleotide repeat. Among the dinucleotide microsatellite motifs, the CA/TG repeat motif was the most abundant, which accounted for 19.64%, followed by AC/GT at 18.66%. The AAT/ATT (1.05%) repeat motif was the most frequent among all types of repeats. Additionally, among the tetranucleotide repeat motifs, the common motifs were AGAC/GTCT and ACAG/CTGT, accounting for 0.63% and 0.71%, respectively. repeat motifs is presented in Figure 2. The frequency of A/T repeats was the highest within the four types of mononucleotide repeat. Among the dinucleotide microsatellite motifs, the CA/TG repeat motif was the most abundant, which accounted for 19.64%, followed by AC/GT at 18.66%. The AAT/ATT (1.05%) repeat motif was the most frequent among all types of repeats. Additionally, among the tetranucleotide repeat motifs, the common motifs were AGAC/GTCT and ACAG/CTGT, accounting for 0.63% and 0.71%, respectively.  Figure 2(a -f) present frequency of the mono-, di-, tri-, tetra-, penta-and hexanucleotide microsatellite motifs.  


### Mitochondrial DNA Assembly and Phylogenetic Relationships of H. nehereus

The total length of the complete mitogenome of H. nehereus was 16,536 bp. Just like other published bony fishes' mtDNA genomes [30], the closed circular molecule included the 13 PCGs, two ribosomal RNA genes, 22 transfer RNA genes and a control region (Figure 3). Most mitochondrial genes were encoded on the H-strand, excluding ND6 and eight tRNAs (Gln, Ala, Asn, Cys, Tyr, Ser-UCN, Glu and Pro) that were located on the L-strand. The nucleotide percentages were as follows: A = 26.75%, T = 26.74%, G = 16.73% and C = 29.78%. The proportion of A + T (53.49%) was higher than that of G + C (46.51%), indicating an AT bias. The phylogenetic tree was constructed using the ML method based on combined amino acids translated from the 13 PCGs (Figure 4). The topological structure showed that all individuals were divided into two clades. Species of the families Myctophidae and Neoscopelidae formed one clade, and the rest were clustered into the other one. H. nehereus and H. microchir were the closest relatives, and they clustered with the genus Saurida. Members of the suborder Alepisauroidei gathered together and had close relationships with Chlorophthalmus agassizi, C. nigromarginatus and Ipnops sp.    


## Discussion

The rapid development of high-throughput sequencing technology not only greatly reduces the sequencing cost, but also significantly shortens the sequencing cycle and promotes the research process of whole genome sequencing (WGS) and genetic mapping [31,32]. Since the first fish genome (known as "torafugu" Fugu rubripes) was published in 2002 [33], WGS has been explosively applied into more and more fish species, ranging from the model fish medaka [34] and zebrafish [35] to many commercially important fishes. Particularly in recent years, the implementation of "The China Aquatic 10-100-1000 Genomics Program" and the "Fish 10K Project" have brought the sequencing of fish genome into a new stage [36,37]. Genomic approaches have shown their great powers in conservation and utilization of fishery resources, exploration of species evolution, molecular breeding, research and development of marine drugs and disease control [38,39]. In this study, we reported the genome survey of an important economic lizardfish, Bombay duck, using the WGS method for the first time. In addition, the microsatellite and mtDNA markers were also identified and characterized for genetic diversity and population structure studies in the future.

According to the K-mer analysis, the genome size of H. nehereus was estimated to be 1315 Mb, which was smaller than the predicted genome size based on the DNA content [40]. This possibly resulted from the non-specific fluorescent dye binding to nongenomic nucleotides. Until now, the published teleost genome size has ranged from 322.5 Mb (Fugu rubripes) [33] to 40 Gb (Protopterus annectens) [41]. Most of the commercial marine species have a genome size of less than 1 Gb, such as Larimichthys crocea [42], Paralichthys olivaceus [43], Lateolabrax maculatus [44] and Thamnaconus septentrionalis [45]. Additionally, compared with Benthosema glaciale (accession number: PRJEB12469), the only reported genome of lanternfishes, it was almost twice as much as that of this species (approximately 676 Mb). It was inferred that the genome size of H. nehereus was relatively large, as a result of the higher repetitive sequence ratio (52.49%). Sequence bias would occur in Illumina sequencing libraries if the GC content was out of the range of 25-65%, and thus seriously affect genome assembly [46]. The GC content of the H. nehereus genome was 41.78%, which was within the acceptable range for genome assembly.

Heterozygosity is another important factor affecting genome assembly and subsequent analysis. As both sister chromatids would be separately assembled in high-heterozygosity regions, it might cause the imprecise assessment of genome size. The heterozygosity ratio of H. nehereus observed in the present study was 0.67%, which was higher than that of Scatophagus argus (female 0.37% and male 0.38%) [47], Sebastiscus marmoratus (0.17%) [48] and Acanthogobius ommaturus (0.17%) [49], but smaller than that of Sillago sihama (0.92%) [50]. Genome assembly becomes more difficult when the heterozygosity ratio is higher than 0.5% [25]. Therefore, the higher heterozygosity ratio might impact the accuracy of genome size estimation.

As a powerful and popular tool to unravel the higher-level relationships of teleosts, complete mtDNA has been more and more widely applied in the studies of evolutionary biology and phylogenetics [30,51]. The genome-wide data include both nuclear DNA information and mitochondrial DNA sequences, hence we excavated the complete mitogenome of H. nehereus to further decipher its taxonomic status and systematic evolution. Rosen discovered the distinctive features of the second and third pharyngobranchials in Aulopiformes fishes, and separated them from Myctophiformes for the first time [52]. Subsequently, Nelson agreed with Rosen's taxonomy in the reprinted "Fishes of the World (2nd Edition)" on the basis of morphological structures of gill rakers, and he regarded the Myctophiformes (Myctophidae + Neoscopelidae) and Aulopiformes (Auiopoidei + Alepisauroidei) as two different orders [53]. In our study, the cladogram constructed using a concatenated set of amino acid sequences from the 13 PCGs supported the abovementioned taxonomical criteria, and revealed a monophyly of Myctophiformes that was corroborated by morphological characteristics and molecular evidence [54][55][56][57]. The evolutionary tree also showed that H. nehereus and H. microchir belong to the family Synodontidae located in the same clade of Aulopiformes and had the closest relationships with each other. The genus Harpadon was inferred as a sister group to Saurida with a strong bootstrap value, which was consistent with the results of a previous study using 16S rRNA and COI genes [19,58].


## Conclusions

This was the first report of the genome survey analysis of a commercially important marine lizardfish (Bombay duck) based on the Illumina platform. The genome size of H. nehereus was 1315 Mb, with 2,539,084 scaffolds and an N50 length of 1568 bp. The candidate SSR and mitochondrial markers developed from H. nehereus genome data could provide novel insight into population genetics and germplasm resource conservation in the future. Considering that the higher level of repetitive DNA and heterozygosity might make it more challenging to generate highly accurate de novo genome assembly, the strategy of Illumina combined with PacBio and Hi-C techniques should be applied to obtain a higher-quality genome of H. nehereus, and more efforts are still to be made in the further research on the genome of this species.  


## Institutional Review Board Statement:

The study was conducted according to the guidelines of the Declaration of Helsinki, and approved by the Ethics Committee of Zhejiang Ocean University (protocol code ZJOU-ECAE20210128 and date of approval 6 January 2021).

## Figure 1 .
1The base distribution and K-mer (K = 17) analysis of H. nehereus. (a) On the x-axis, the read-1 base contents are on the left side of the dotted line and the read-2 base contents are on the right.

## Figure 1 .
1The base distribution and K-mer (K = 17) analysis of H. nehereus. (a) On the x-axis, the read-1 base contents are on the left side of the dotted line and the read-2 base contents are on the right. Different colors represent different base types. The y-axis represents the sequencing depth. (b) The x-axis means K-mer depth and the y-axis represents the frequency for the corresponding depth.

## Figure 2 .
2The distribution and frequency of microsatellite motifs in H. nehereus.

## Figure 2 .
2The distribution and frequency of microsatellite motifs in H. nehereus. (a-f) present frequency of the mono-, di-, tri-, tetra-, penta-and hexanucleotide microsatellite motifs.

## 2021, 1 ,
1FOR PEERREVIEW  6    one. H. nehereus and H. microchir were the closest relatives, and they clustered with the genus Saurida. Members of the suborder Alepisauroidei gathered together and had close relationships with Chlorophthalmus agassizi, C. nigromarginatus and Ipnops sp.

## Figure 3 .
3The complete mitogenome structure of H. nehereus.

## Figure 3 .Figure 4 .
34The complete mitogenome structure of H. nehereus. The phylogenetic tree inferred from mitochondrial genomes of related species.

## Figure 4 .
4The phylogenetic tree inferred from mitochondrial genomes of related species.

## Author
Contributions: Conceptualization, T.G. and T.Y.; investigation, Z.N.; writing-original draft preparation, T.Y.; writing-review and editing, X.H. All authors have read and agreed to the published version of the manuscript.

## Funding:
This research was funded by the National Science Foundation of China (No. 41776171) and Scientific Research Projects of the Zhejiang Department of Education (No. Y201942611).

## Table 1 .
1Assembly statistics for stitched contigs and scaffolds of H. nehereus.Sample 
Total Length 
(bp) 

Total 
Number 

Total Number 
(≥2 kb) 

Max Length 
(bp) 

N50 
(bp) 
N90 (bp) 

H. nehereus 

contigs 
1,352,668,147 
3,766,078 
73,281 
30,284 
596 
138 

scaffolds 
1,363,443,545 
2,539,084 
133,089 
92,118 
1568 
162 



## Table 2 .
2The statistics of SSRs in the H. nehereus genome based on repeat types.SSR Type 
Number 
Percent (%) 
Sequence Number 
Total SSR Length (bp) 

p1 
266,761 
25.96 
208,219 
3,369,363 
p2 
558,543 
54.35 
400,985 
17,745,246 
p3 
66,270 
6.45 
62,046 
1,638,864 
p4 
105,426 
10.26 
97,016 
3,527,488 
p5 
25,182 
2.45 
23,958 
883,000 
p6 
5469 
0.53 
5374 
191,406 
Total 
1,027,651 
100 
797,598 
27,355,367 


## Table 2 .
2The statistics of SSRs in the H. nehereus genome based on repeat types.SSR Type 
Number 
Percent (%) 
Sequence 
Number 

Total SSR Length 
(bp) 
p1 
266,761 
25.96 
208,219 
3,369,363 
p2 
558,543 
54.35 
400,985 
17,745,246 
p3 
66,270 
6.45 
62,046 
1,638,864 
p4 
105,426 
10.26 
97,016 
3,527,488 


Informed Consent Statement: Not applicable.Data Availability Statement: All data presented during this study are included in this article.Conflicts of Interest:The authors declare no conflict of interest.
. J S Nelson, T C Grande, M V H Wilson, Fishes of the World. John & Wiley & Sons5th ed.Nelson, J.S.; Grande, T.C.; Wilson, M.V.H. Fishes of the World, 5th ed.; John & Wiley & Sons: Hoboken, NJ, USA, 2016.

Marine lizardfish (Harpadon nehereus) meal concentrate in preparation of ready-to-eat protein and calcium rich extruded snacks. P Chakraborty, S Sahoo, D K Bhattacharyya, M Ghosh, 10.1007/s13197-019-04066-0J. Food Sci. Technol. 57Chakraborty, P.; Sahoo, S.; Bhattacharyya, D.K.; Ghosh, M. Marine lizardfish (Harpadon nehereus) meal concentrate in preparation of ready-to-eat protein and calcium rich extruded snacks. J. Food Sci. Technol. 2020, 57, 338-349. [CrossRef]

Study of Main Biology Character and Analysis of Resource Status on the Harpodon nehereus. H Z Luo, Chinese). Zhoushan, ChinaZhejiang Ocean UniversityMaster's ThesisLuo, H.Z. Study of Main Biology Character and Analysis of Resource Status on the Harpodon nehereus. Master's Thesis, Zhejiang Ocean University, Zhoushan, China, 2012. (In Chinese).

Estimation of maximum sustainable yield of Bombay Duck, Harpodon nehereus fishery in Pakistan using the CEDA and ASPIC packages. M A Kalhoro, Q Liu, K H Memon, M S Chang, A N Jatt, Pakistan J. Zool. 45Kalhoro, M.A.; Liu, Q.; Memon, K.H.; Chang, M.S.; Jatt, A.N. Estimation of maximum sustainable yield of Bombay Duck, Harpodon nehereus fishery in Pakistan using the CEDA and ASPIC packages. Pakistan J. Zool. 2013, 45, 1757-1764.

A review on fishery, biology and stock parameters of Bombay duck Harpodon nehereus (Hamilton, 1822) occurring in India. A K Jaiswar, S K Chakraborty, J. Indian Fish. Assoc. 43Jaiswar, A.K.; Chakraborty, S.K. A review on fishery, biology and stock parameters of Bombay duck Harpodon nehereus (Hamilton, 1822) occurring in India. J. Indian Fish. Assoc. 2016, 43, 67-96.

Population dynamics of Bombay duck Harpodon nehereus (Hamilton, 1822) of the Bay of Bengal along Bangladesh coast. M Sarker, M Humayun, M A Rahman, M S Uddin, 10.3329/bjz.v45i2.35705Bangl. J. Zool. 45Sarker, M.; Humayun, M.; Rahman, M.A.; Uddin, M.S. Population dynamics of Bombay duck Harpodon nehereus (Hamilton, 1822) of the Bay of Bengal along Bangladesh coast. Bangl. J. Zool. 2018, 45, 101-110. [CrossRef]

Growth characteristics and resources sustainable utilization of Harpodon nehereus. L Chen, B N Shui, W X Dong, Technol. Manag. 3In ChineseChen, L.; Shui, B.N.; Dong, W.X. Growth characteristics and resources sustainable utilization of Harpodon nehereus. Technol. Manag. 2012, 3, 68-70. (In Chinese)

Implication towards growth overfishing of Harpadon nehereus in bag net fishing. D Sudheesan, C M Roshith, R K Manna, S K Das, S K Koushlesh, T N Chanu, D Bhakta, Proceedings of the 29th All India Congress of Zoology, ICAR-CIFRI. the 29th All India Congress of Zoology, ICAR-CIFRIKolkata, IndiaSudheesan, D.; Roshith, C.M.; Manna, R.K.; Das, S.K.; Koushlesh, S.K.; Chanu, T.N.; Bhakta, D. Implication towards growth overfishing of Harpadon nehereus in bag net fishing. In Proceedings of the 29th All India Congress of Zoology, ICAR-CIFRI, Kolkata, India, 9-11 June 2017.

Biochemical genetics and fishery management: An historical perspective. F M Utter, 10.1111/j.1095-8649.1991.tb05063.xJ. Fish Biol. 39Utter, F.M. Biochemical genetics and fishery management: An historical perspective. J. Fish Biol. 2010, 39, 1-20. [CrossRef]

Stock assessment of Bombay duck Harpodon nehereus (Ham.) off Maharashtra coast. A Kurian, K N Kurup, Indian J. Fish. 39Kurian, A.; Kurup, K.N. Stock assessment of Bombay duck Harpodon nehereus (Ham.) off Maharashtra coast. Indian J. Fish. 1992, 39, 243-248.

Improvement of cooking quality and gel formation capacity of Bombay duck (Harpodon nehereus) fish meat. C Rupsankar, 10.1007/s13197-010-0092-3J. Food Sci. Technol. 47Rupsankar, C. Improvement of cooking quality and gel formation capacity of Bombay duck (Harpodon nehereus) fish meat. J. Food Sci. Technol. 2010, 47, 534-540. [CrossRef]

The expression of the body shape in fish species Harpadon nehereus (Hamilton, 1822) in the waters of Juata Laut, Tarakan city. M Firdaus, T D Lelono, R Saleh, G Bintoro, G Salim, North Kalimantan. AACL Bioflux. 11Firdaus, M.; Lelono, T.D.; Saleh, R.; Bintoro, G.; Salim, G. The expression of the body shape in fish species Harpadon nehereus (Hamilton, 1822) in the waters of Juata Laut, Tarakan city, North Kalimantan. AACL Bioflux 2018, 11, 613-624.

Nomei fish (Harpadon nehereus, Ham. 1822) reproduction biology in Tarakan waters. A Taqwa, A I Burhanuddin, A Niartiningsih, M Nessa, IOP Conference Series: Earth and Environmental Science. Bristol, UK, 2020IOP Publishing473Taqwa, A.; Burhanuddin, A.I.; Niartiningsih, A.; Nessa, M.N. Nomei fish (Harpadon nehereus, Ham. 1822) reproduction biology in Tarakan waters. In IOP Conference Series: Earth and Environmental Science; IOP Publishing: Bristol, UK, 2020; Volume 473, pp. 1-6.

. Curr. Issues Mol. Biol. 431291Curr. Issues Mol. Biol. 2021, 43 1291

Development and characterization of microsatellite markers for the lizardfish known as the Bombay duck, Harpadon nehereus (Synodontidae). T J Xu, D Q Sun, H Y Li, R X Wang, 10.4238/vol10-3gmr1045Genet. Mol. Res. 10PubMedXu, T.J.; Sun, D.Q.; Li, H.Y.; Wang, R.X. Development and characterization of microsatellite markers for the lizardfish known as the Bombay duck, Harpadon nehereus (Synodontidae). Genet. Mol. Res. 2011, 10, 1701-1706. [CrossRef] [PubMed]

Genetic diversity and population structure in Harpadon nehereus based on sequence-related amplified polymorphism markers. Z H Zhu, H Y Li, Y Qin, R X Wang, 10.4238/2014.August.7.13Genet. Mol. Res. 13Zhu, Z.H.; Li, H.Y.; Qin, Y.; Wang, R.X. Genetic diversity and population structure in Harpadon nehereus based on sequence-related amplified polymorphism markers. Genet. Mol. Res. 2014, 13, 5974-5981. [CrossRef]

The genetic structure of the Bombay duck (Harpadon nehereus) based on mitochondrial Cyt b gene. Y J Guo, T Y Yang, W Meng, Z Q Hang, T X Gao, Acta Hydrobiol. Sin. 43In ChineseGuo, Y.J.; Yang, T.Y.; Meng, W.; Hang, Z.Q.; Gao, T.X. The genetic structure of the Bombay duck (Harpadon nehereus) based on mitochondrial Cyt b gene. Acta Hydrobiol. Sin. 2019, 43, 945-952. (In Chinese)

Polymorphic loci analysis of 16S ribosomal RNA gene of economically important marine lizardfish Bombay duck (Harpadon nehereus). S Saha, Z Ferdous, H Jahan, A M Khandaker, M R Shahjahan, R A Begum, 10.3329/bjz.v47i1.42020Bangl. J. Zool. 47Saha, S.; Ferdous, Z.; Jahan, H.; Khandaker, A.M.; Shahjahan, M.R.; Begum, R.A. Polymorphic loci analysis of 16S ribosomal RNA gene of economically important marine lizardfish Bombay duck (Harpadon nehereus). Bangl. J. Zool. 2019, 47, 49-57. [CrossRef]

The complete mitochondrial genome of the larval Bombay duck Harpodon nehereus (Aulopiformes, Synodontidae) from Yangtze estuary and the phylogenetic relationship of Synodontidae species. H Zhang, W W Xian, 10.1080/23802359.2018.1476070Mitochondrial DNA. 3PubMedZhang, H.; Xian, W.W. The complete mitochondrial genome of the larval Bombay duck Harpodon nehereus (Aulopiformes, Synodontidae) from Yangtze estuary and the phylogenetic relationship of Synodontidae species. Mitochondrial DNA B 2018, 3, 657-658. [CrossRef] [PubMed]

Molecular phylogeny of Harpadon nehereus and its close relatives based on mitochondrial Cyt b gene. T Y Yang, Y L Jiang, Y J Guo, L Q Zhu, Y J Lin, Y J Zheng, T X Gao, Trans. Oceanol. Limn. 6In ChineseYang, T.Y.; Jiang, Y.L.; Guo, Y.J.; Zhu, L.Q.; Lin, Y.J.; Zheng, Y.J.; Gao, T.X. Molecular phylogeny of Harpadon nehereus and its close relatives based on mitochondrial Cyt b gene. Trans. Oceanol. Limn. 2020, 6, 77-85. (In Chinese)

Comparative study the expression of calcium cycling genes in Bombay duck (Harpadon nehereus) and beltfish (Trichiurus lepturus) with different swimming activities. H Zhang, G Audira, Y Li, W W Xian, M M Varikkodan, C D Hsiao, 10.1016/j.gdata.2017.03.00312PubMedZhang, H.; Audira, G.; Li, Y.; Xian, W.W.; Varikkodan, M.M.; Hsiao, C.D. Comparative study the expression of calcium cycling genes in Bombay duck (Harpadon nehereus) and beltfish (Trichiurus lepturus) with different swimming activities. Genom. Data 2017, 12, 58-61. [CrossRef] [PubMed]

Research advances in whole-genome sequencing of representative fish species. H Y Chen, Y Y Chen, L I Rong, H Xiao, S Y Chen, J. Biol. 34In ChineseChen, H.Y.; Chen, Y.Y.; Rong, L.I.; Xiao, H.; Chen, S.Y. Research advances in whole-genome sequencing of representative fish species. J. Biol. 2017, 34, 73-77. (In Chinese)

Molecular Cloning: A Laboratory Manual. J Sambrook, E F Fritsch, T Maniatis, Cold Spring Harbor Laboratory PressNew York, NY, USASambrook, J.; Fritsch, E.F.; Maniatis, T. Molecular Cloning: A Laboratory Manual; Cold Spring Harbor Laboratory Press: New York, NY, USA, 1982.

A MapReduce acceleration-supported software for integrated quality control and preprocessing of high-throughput sequencing data. Y X Chen, Y S Chen, C M Shi, Z B Huang, Y Zhang, S K Li, Y Li, J Ye, C Yu, Z Li, 10.1093/gigascience/gix1207OxfordPubMedChen, Y.X.; Chen, Y.S.; Shi, C.M.; Huang, Z.B.; Zhang, Y.; Li, S.K.; Li, Y.; Ye, J.; Yu, C.; Li, Z.; et al. SOAPnuke: A MapReduce acceleration-supported software for integrated quality control and preprocessing of high-throughput sequencing data. Oxford Open 2018, 7, 1-6. [CrossRef] [PubMed]

SOAPdenovo2: An empirically improved memory-efficient short-read de novo assembler. R B Luo, B H Liu, Y L Xie, Z Y Li, W H Huang, J Y Yuan, G Z He, Y X Chen, Q Pan, Y J Liu, 10.1186/2047-217X-1-181PubMedLuo, R.B.; Liu, B.H.; Xie, Y.L.; Li, Z.Y.; Huang, W.H.; Yuan, J.Y.; He, G.Z.; Chen, Y.X.; Pan, Q.; Liu, Y.J.; et al. SOAPdenovo2: An empirically improved memory-efficient short-read de novo assembler. GigaScience 2012, 1, 1-6. [CrossRef] [PubMed]

A fast, lock-free approach for efficient parallel counting of occurrences of k-mers. G Marcais, C Kingsford, 10.1093/bioinformatics/btr011Bioinformatics. 27PubMedMarcais, G.; Kingsford, C. A fast, lock-free approach for efficient parallel counting of occurrences of k-mers. Bioinformatics 2011, 27, 764-770. [CrossRef] [PubMed]

De novo assembly of human genomes with massively parallel short read sequencing. R Q Li, H M Zhu, J Ruan, W B Qian, X D Fang, Z B Shi, Y R Li, S T Li, G Shan, K Kristiansen, 10.1101/gr.097261.109Genome Res. 20Li, R.Q.; Zhu, H.M.; Ruan, J.; Qian, W.B.; Fang, X.D.; Shi, Z.B.; Li, Y.R.; Li, S.T.; Shan, G.; Kristiansen, K.; et al. De novo assembly of human genomes with massively parallel short read sequencing. Genome Res. 2010, 20, 265-272. [CrossRef]

Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads. R Kajitani, K Toshimoto, H Noguchi, A Toyoda, Y Ogura, M Okuno, M Yabana, M Harada, E Nagayasu, H Nagayasu, 10.1101/gr.170720.113Genome Res. 24Kajitani, R.; Toshimoto, K.; Noguchi, H.; Toyoda, A.; Ogura, Y.; Okuno, M.; Yabana, M.; Harada, M.; Nagayasu, E.; Nagayasu, H.; et al. Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads. Genome Res. 2014, 24, 1384-1395. [CrossRef]

Exploiting EST databases for the development and characterization of gene-derived SSR-markers in barley (Hordeum vulgare L.). T Thiel, W Michalek, R K Varshney, A Graner, 10.1007/s00122-002-1031-0Theor. Appl. Genet. 106Thiel, T.; Michalek, W.; Varshney, R.K.; Graner, A. Exploiting EST databases for the development and characterization of gene-derived SSR-markers in barley (Hordeum vulgare L.). Theor. Appl. Genet. 2003, 106, 411-422. [CrossRef]

Molecular evolutionary genetics analysis version 11. K Tamura, G Stecher, S Kumar, Mega11, 10.1093/molbev/msab120Mol. Biol. Evol. 38Tamura, K.; Stecher, G.; Kumar, S. MEGA11: Molecular evolutionary genetics analysis version 11. Mol. Biol. Evol. 2021, 38, 3022-3027. [CrossRef]

Mitogenomic exploration of higher teleostean phylogenies: A case study for moderate-scale evolutionary genomics with 38 newly determined complete mitochondrial DNA sequences. M Miya, A Kawaguchi, M Nishida, 10.1093/oxfordjournals.molbev.a003741Mol. Biol. Evol. 18PubMedMiya, M.; Kawaguchi, A.; Nishida, M. Mitogenomic exploration of higher teleostean phylogenies: A case study for moderate-scale evolutionary genomics with 38 newly determined complete mitochondrial DNA sequences. Mol. Biol. Evol. 2001, 18, 1993-2009. [CrossRef] [PubMed]

High-throughput sequencing technology and its application. Q L Zhu, S Liu, P Gao, F S Luan, J. Northeast Agric. Univ. (Engl. Ed.). 21Zhu, Q.L.; Liu, S.; Gao, P.; Luan, F.S. High-throughput sequencing technology and its application. J. Northeast Agric. Univ. (Engl. Ed.) 2014, 21, 84-96.

High-throughput sequencing technologies. J Reuter, D V Spacek, M Snyder, 10.1016/j.molcel.2015.05.004Mol. Cell. 58PubMedReuter, J.; Spacek, D.V.; Snyder, M. High-throughput sequencing technologies. Mol. Cell 2015, 58, 586-597. [CrossRef] [PubMed]

Whole-genome shotgun assembly and analysis of the genome of Fugu rubripes. S Aparicio, J Chapman, E Stupka, N Putnam, J M Chia, P Dehal, A Christoffels, S Rash, A Hoon, A Smit, 10.1126/science.1072104Science. 297Aparicio, S.; Chapman, J.; Stupka, E.; Putnam, N.; Chia, J.M.; Dehal, P.; Christoffels, A.; Rash, S.; Hoon, A.; Smit, A.; et al. Whole-genome shotgun assembly and analysis of the genome of Fugu rubripes. Science 2002, 297, 1301-1310. [CrossRef]

The medaka draft genome and insights into vertebrate genome evolution. M Kasahara, K Naruse, S Sasaki, Y Nakatani, W Qu, B Ahsan, T Yamada, Y Nagayasu, K Doi, Y Kasai, 10.1038/nature05846Nature. 44PubMedKasahara, M.; Naruse, K.; Sasaki, S.; Nakatani, Y.; Qu, W.; Ahsan, B.; Yamada, T.; Nagayasu, Y.; Doi, K.; Kasai, Y.; et al. The medaka draft genome and insights into vertebrate genome evolution. Nature 2007, 44, 714-719. [CrossRef] [PubMed]

The zebrafish reference genome sequence and its relationship to the human genome. K Howe, M D Clark, C F Torroja, J Torrance, C Berthelot, M Muffato, J E Collins, S Humphray, K Mclaren, L Matthews, 10.1038/nature12111Nature. 496Howe, K.; Clark, M.D.; Torroja, C.F.; Torrance, J.; Berthelot, C.; Muffato, M.; Collins, J.E.; Humphray, S.; McLaren, K.; Matthews, L.; et al. The zebrafish reference genome sequence and its relationship to the human genome. Nature 2013, 496, 498-503. [CrossRef]

China is initiating the aquatic 10-100-1000 genomics program. Y J Liu, P Xu, J M Xu, Y Huang, Y X Liu, H Fang, Y C Hu, X X You, C Bian, M Sun, 10.1007/s11427-016-0273-9Sci. China Life Sci. 60Liu, Y.J.; Xu, P.; Xu, J.M.; Huang, Y.; Liu, Y.X.; Fang, H.; Hu, Y.C.; You, X.X.; Bian, C.; Sun, M.; et al. China is initiating the aquatic 10-100-1000 genomics program. Sci. China Life Sci. 2017, 60, 329-332. [CrossRef]

Initial data release and announcement of the 10,000 Fish Genomes Project (Fish10K). G Y Fan, Y Song, D L Yang, X Y Huang, S Y Zhang, M Q Zhang, X W Yang, Y Chang, H Zhang, Y X Li, 10.1093/gigascience/giaa0802020Fan, G.Y.; Song, Y.; Yang, D.L.; Huang, X.Y.; Zhang, S.Y.; Zhang, M.Q.; Yang, X.W.; Chang, Y.; Zhang, H.; Li, Y.X.; et al. Initial data release and announcement of the 10,000 Fish Genomes Project (Fish10K). GigaScience 2020, 9, 1-8. [CrossRef]

Fish genomics: Casting the net wide. F Wardle, F Mueller, 10.1093/bfgp/elt055Brief. Funct. Genomics. 13Wardle, F.; Mueller, F. Fish genomics: Casting the net wide. Brief. Funct. Genomics 2014, 13, 79-81. [CrossRef]

Fish genomic research: Decade review and prospect. S L Chen, W T Xu, Y Liu, J. Fish. China. 43In ChineseChen, S.L.; Xu, W.T.; Liu, Y. Fish genomic research: Decade review and prospect. J. Fish. China 2019, 43, 1-14. (In Chinese)

Nuclean DNA content and genome size of trout and human. J Dolezel, J Bartos, H Voglmayr, J Greilhuber, Cytom. Part A. 51Dolezel, J.; Bartos, J.; Voglmayr, H.; Greilhuber, J. Nuclean DNA content and genome size of trout and human. Cytom. Part A 2003, 51, 127-128.

African lungfish genome sheds light on the vertebrate water-to-land transition. K Wang, J Wang, C L Zhu, L D Yang, Y D Ren, J Ruan, G Y Fan, J Hu, W J Xu, X P Bi, 10.1016/j.cell.2021.01.047Cell. 184Wang, K.; Wang, J.; Zhu, C.L.; Yang, L.D.; Ren, Y.D.; Ruan, J.; Fan, G.Y.; Hu, J.; Xu, W.J.; Bi, X.P.; et al. African lungfish genome sheds light on the vertebrate water-to-land transition. Cell 2021, 184, 1362-1376. [CrossRef]

The draft genome of the large yellow croaker reveals well-developed innate immunity. C W Wu, D Zhang, M Y Kan, Z M Lv, A Y Zhu, Y Q Su, D Z Zhou, J S Zhang, Z Zhang, M Y Xu, 10.1038/ncomms6227Nat. Commun. 5PubMedWu, C.W.; Zhang, D.; Kan, M.Y.; Lv, Z.M.; Zhu, A.Y.; Su, Y.Q.; Zhou, D.Z.; Zhang, J.S.; Zhang, Z.; Xu, M.Y.; et al. The draft genome of the large yellow croaker reveals well-developed innate immunity. Nat. Commun. 2014, 5, 5227. [CrossRef] [PubMed]

The genome and transcriptome of Japanese flounder provide insights into flatfish asymmetry. C W Shao, B L Bao, Z Y Xie, X Y Chen, B Li, X D Jia, Q L Yao, G Ortí, W H Li, X H Li, 10.1038/ng.3732Nat. Genet. 49PubMedShao, C.W.; Bao, B.L.; Xie, Z.Y.; Chen, X.Y.; Li, B.; Jia, X.D.; Yao, Q.L.; Ortí, G.; Li, W.H.; Li, X.H.; et al. The genome and transcriptome of Japanese flounder provide insights into flatfish asymmetry. Nat. Genet. 2017, 49, 119-124. [CrossRef] [PubMed]

Chromosome-level genome assembly of the spotted sea bass. C W Shao, C Li, N Wang, Y T Qin, W T Xu, Q Liu, Q Zhou, Y Zhao, X H Li, S S Liu, 10.1093/gigascience/giy114Lateolabrax maculatus. 7PubMedShao, C.W.; Li, C.; Wang, N.; Qin, Y.T.; Xu, W.T.; Liu, Q.; Zhou, Q.; Zhao, Y.; Li, X.H.; Liu, S.S.; et al. Chromosome-level genome assembly of the spotted sea bass, Lateolabrax maculatus. GigaScience 2018, 7, 1-7. [CrossRef] [PubMed]

Chromosome-level genome assembly of the greenfin horse-faced filefish (Thamnaconus septentrionalis) using Oxford Nanopore PromethION sequencing and Hi-C technology. L Bian, F H Li, J L Ge, P F Wang, Q Chang, S N Zhang, J Li, C L Liu, K Liu, X T Liu, 10.1111/1755-0998.13183Mol. Ecol. Resour. 20PubMedBian, L.; Li, F.H.; Ge, J.L.; Wang, P.F.; Chang, Q.; Zhang, S.N.; Li, J.; Liu, C.L.; Liu, K.; Liu, X.T.; et al. Chromosome-level genome assembly of the greenfin horse-faced filefish (Thamnaconus septentrionalis) using Oxford Nanopore PromethION sequencing and Hi-C technology. Mol. Ecol. Resour. 2020, 20, 1069-1079. [CrossRef] [PubMed]

Analyzing and minimizing PCR amplification bias in Illumina sequencing libraries. D Aird, M G Ross, W S Chen, M Danielsson, T Fennell, C Russ, D B Jaffe, C Nusbaum, A Gnirke, 10.1186/gb-2011-12-2-r18Genome Biol. 12R18. [CrossRef. PubMedAird, D.; Ross, M.G.; Chen, W.S.; Danielsson, M.; Fennell, T.; Russ, C.; Jaffe, D.B.; Nusbaum, C.; Gnirke, A. Analyzing and minimizing PCR amplification bias in Illumina sequencing libraries. Genome Biol. 2011, 12, R18. [CrossRef] [PubMed]

Genome survey of male and female spotted scat. Y Q Huang, D N Jiang, M Li, U F Mustapha, C X Tian, H P Chen, Y Huang, S P Deng, T L Wu, C H Zhu, 10.3390/ani91211179Scatophagus argus. PubMedHuang, Y.Q.; Jiang, D.N.; Li, M.; Mustapha, U.F.; Tian, C.X.; Chen, H.P.; Huang, Y.; Deng, S.P.; Wu, T.L.; Zhu, C.H.; et al. Genome survey of male and female spotted scat (Scatophagus argus). Animals 2019, 9, 1117. [CrossRef] [PubMed]

Whole genome survey analysis and microsatellite motif identification of Sebastiscus marmoratus. S Y Xu, N Song, S J Xiao, T X Gao, 10.1042/BSR20192252Biosci. Rep. 2020Xu, S.Y.; Song, N.; Xiao, S.J.; Gao, T.X. Whole genome survey analysis and microsatellite motif identification of Sebastiscus marmoratus. Biosci. Rep. 2020, 40, 1-7. [CrossRef]

Genomic characteristics and profile of microsatellite primers for Acanthogobius ommaturus by genome survey sequencing. B J Chen, Z C Sun, F R Lou, T X Gao, N Song, 10.1042/BSR20201295Biosci. Rep. 40Chen, B.J.; Sun, Z.C.; Lou, F.R.; Gao, T.X.; Song, N. Genomic characteristics and profile of microsatellite primers for Acanthogobius ommaturus by genome survey sequencing. Biosci. Rep. 2020, 40, 1-8. [CrossRef]

A first insight into a draft genome of silver sillago (Sillago sihama) via genome survey sequencing. Z Y Li, C X Tian, Y Huang, X H Lin, Y R Wang, D N Jiang, C H Zhu, H P Chen, G L Li, 10.3390/ani91007569PubMedLi, Z.Y.; Tian, C.X.; Huang, Y.; Lin, X.H.; Wang, Y.R.; Jiang, D.N.; Zhu, C.H.; Chen, H.P.; Li, G.L. A first insight into a draft genome of silver sillago (Sillago sihama) via genome survey sequencing. Animals 2019, 9, 756. [CrossRef] [PubMed]

Major patterns of higher teleostean phylogenies: A new perspective based on 100 complete mitochondrial DNA sequences. M Miya, H Takeshima, H Endo, N B Ishiguro, J G Inoue, T Mukai, T P Satoh, M Yamaguchi, A Kawaguchi, K Mabuchi, 10.1016/S1055-7903(02)00332-9Mol. Phylogenet. Evol. 26Miya, M.; Takeshima, H.; Endo, H.; Ishiguro, N.B.; Inoue, J.G.; Mukai, T.; Satoh, T.P.; Yamaguchi, M.; Kawaguchi, A.; Mabuchi, K.; et al. Major patterns of higher teleostean phylogenies: A new perspective based on 100 complete mitochondrial DNA sequences. Mol. Phylogenet. Evol. 2003, 26, 121-138. [CrossRef]

Interrelationships of higher euteleostean fishes. D E Rosen, Zool. J. Linn. Soc. 53Rosen, D.E. Interrelationships of higher euteleostean fishes. Zool. J. Linn. Soc. 1973, 53, 397-513.

. J S Nelson, Fishes of the World. John & Wiley & Sons2nd ed.Nelson, J.S. Fishes of the World, 2nd ed.; John & Wiley & Sons: Hoboken, NJ, USA, 1984.

Basal ctenosquamate relationships and the interrelationships of the Myctophiform (Scopelomorph) fishes. In Interrelationships of Fishes. M L J Stiassny, Stiassny, M.L.J. Basal ctenosquamate relationships and the interrelationships of the Myctophiform (Scopelomorph) fishes. In Interrelationships of Fishes;

. M L J Stiassny, L R Parenti, G D Johnson, Eds, Academic PressCambridge, UKStiassny, M.L.J., Parenti, L.R., Johnson, G.D., Eds.; Academic Press: Cambridge, UK, 1996; pp. 405-426.

Mitogenomic sequences and evidence from unique gene rearrangements corroborate evolutionary relationships of Myctophiformes (Neoteleostei). J Y Poulsen, I Byrkjedal, E Willassen, D Rees, H Takeshima, T P Satoh, G Shinohara, M Nishida, M Miya, 10.1186/1471-2148-13-111BMC Evol. Biol. 13Poulsen, J.Y.; Byrkjedal, I.; Willassen, E.; Rees, D.; Takeshima, H.; Satoh, T.P.; Shinohara, G.; Nishida, M.; Miya, M. Mitoge- nomic sequences and evidence from unique gene rearrangements corroborate evolutionary relationships of Myctophiformes (Neoteleostei). BMC Evol. Biol. 2013, 13, 1-21. [CrossRef]

Seven-locus molecular phylogeny of Myctophiformes. J S Denton, TeleosteiDenton, J.S.S. Seven-locus molecular phylogeny of Myctophiformes (Teleostei;

Scopelomorpha) highlights the utility of the order for studies of deep-sea evolution. 10.1016/j.ympev.2014.02.009Mol. Phylogenet. Evol. 76PubMedScopelomorpha) highlights the utility of the order for studies of deep-sea evolution. Mol. Phylogenet. Evol. 2014, 76, 270-292. [CrossRef] [PubMed]

The evolution of specialized dentition in the deep-sea lanternfishes (Myctophiformes). R P Martin, M P Davis, 10.1002/jmor.21120J. Morphol. 2020PubMedMartin, R.P.; Davis, M.P. The evolution of specialized dentition in the deep-sea lanternfishes (Myctophiformes). J. Morphol. 2020, 281, 536-555. [CrossRef] [PubMed]

DNA barcoding of nomei fish (Synodontidae: Harpadon sp.) in Tarakan Island. E D Nugroho, D Nawir, M Amin, U Lestari, Indonesia. AACL Bioflux. 10Nugroho, E.D.; Nawir, D.; Amin, M.; Lestari, U. DNA barcoding of nomei fish (Synodontidae: Harpadon sp.) in Tarakan Island, Indonesia. AACL Bioflux 2017, 10, 1466-1474.