# Molecular Sciences High Molecular Weight Kininogen: A Review of the Structural Literature

CorpusID: 245211283
 
tags: #Medicine, #Chemistry, #Materials_Science

URL: [https://www.semanticscholar.org/paper/45a2bb2bb59b58215cd41c564a67f2ca26a095dc](https://www.semanticscholar.org/paper/45a2bb2bb59b58215cd41c564a67f2ca26a095dc)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Molecular Sciences High Molecular Weight Kininogen: A Review of the Structural Literature
Published: 13 December 2021

Michał B Ponczek 
International Journal
Department of General Biochemistry
Faculty of Biology and Environmental Protection
University of Lodz
Pomorska 141/14390-236LodzPoland

N Uversky 
International Journal
Department of General Biochemistry
Faculty of Biology and Environmental Protection
University of Lodz
Pomorska 141/14390-236LodzPoland

Mohammad Hassan Baig 
International Journal
Department of General Biochemistry
Faculty of Biology and Environmental Protection
University of Lodz
Pomorska 141/14390-236LodzPoland

Molecular Sciences High Molecular Weight Kininogen: A Review of the Structural Literature
Published: 13 December 2021Received: 29 October 2021 Accepted: 11 December 2021Review Citation: Ponczek, M.B. High Molecular Weight Kininogen: A Review of the Structural Literature. Int. J. Mol. Sci. 2021, 22, 13370. https://doi.org/10.3390/ijms 222413370 Academic Editors: Vladimir


## Introduction

Kininogens or Fitzgerald factors are multidomain glycoproteins found in the blood of most vertebrates [1][2][3][4]. The orthological proteins (kininogen-1, UniProt ID: P01042 for human) are produced by splicing of the KNG1 gene transcript. High molecular weight kininogen (HK) is one of the members of the kallikrein-Kinin system (KKS). Together with coagulation factor (f)XI, the KKS form the contact activation system of blood coagulation (CAS) or more commonly known as the intrinsic pathway of coagulation contributing to thrombin generation primarily under pathologic conditions. CAS can operate independently from the tissue factor (TF) extrinsic pathway ( Figure 1). As a class, kininogens have been studied for decades, nonetheless, the atomic arrangement of full length kininogen or its fragments are still unknown for all species with no coordinates in Protein Data Bank (PDB). A protein Basic Local Alignment Search Tool (BlastP) applied with human kininogen amino acid sequence against PDB returns only proteins containing homologous cystatin domains with identity around 30% (Table 1).  Moreover, entering the keyword "kininogen" in the database search engine returns only structures containing polypeptide fragments referred to as "inserts of kininogen peptides", "peptides derived from kininogen" or bradykinin (BK) derived peptides (Table 2). The purpose of this review article is to summarize the finding of structural studies that involve kininogens and emphasize how a good-quality spatial structure would allow for a better understanding of their role in disease and as therapeutic targets.


## HK Biochemical and Evolutionary Characterization

The human high molecular weight kininogen (huHK) is a 626 amino acid glycoprotein (devoid of signal peptide). The molecular mass is only 70 kDa, but because it undergoes numerous post-translational modifications, the real mass is much higher and it run~120 KDa on SDS-PAGE under non-reducing conditions [1][2][3][4]. huHK circulates in plasma at a concentration of approximately 70 µg/mL. The protein is mostly expressed by hepatocytes and secreted into the blood. huHK theoretical isoelectric point based on amino acid sequence was calculated using Compute pI/Mw tool (https://web.expasy.org/compute_pi/, accessed on 29 October 2021) to be 6.23, however the experimental reported pI is 4.9 ± 0.2 because of high sialic acid content. The sialic acid constitutes almost 42% of HK molecular mass content. A highly purified human kininogen has been established to be about 8.6 mol/mol (50 kDa) [5]. This explains the discrepancy between the theoretical mass from the amino acid sequence and the experimental values of HK weight.

In plasma, huHK circulates as non-covalent complex with PK [6][7][8]. Mouse plasma studies as well as huHK case reports demonstrated an important carrier function of HK where HK deficiency was accompanied with low PK levels [9]. Similar to its interaction with PK, huHK also forms non-covalent complex with fXI (a PK homolog, produced by duplication of the PK gene) [4,[10][11][12][13][14]. There is enough huHK in the plasma to cover the entire range of PK and fXI. In contrast to PK, the carrier properties of HK are not required for maintaining fXI levels [10]. Kininogens, mostly HK, have many different biological activities, including: the role in blood coagulation by helping to locate suitably plasma prekallikrein (PK) and fXI next to fXII on negatively charged surfaces, inhibition of blood platelets aggregation induced by thrombin and plasmin, and the liberation of vasoactive peptide BK ( Figure 1). BK manifests important physiological effects: smooth muscle contraction, induction of hypotension, natriuresis and diuresis, and decrease in blood glucose level. It is also a mediator of inflammation and causes and increase in vascular permeability, release of mediators of inflammation like prostaglandins, stimulation of nociceptors and has a direct and indirect cardioprotective effect [4].

HK is organized into distinct domains designated D1 through D6 that can be listed as follows with the amino acid positions numbering: platelets aggregation induced by thrombin and plasmin, and the liberation of vasoactive peptide BK (Figure 1). BK manifests important physiological effects: smooth muscle contraction, induction of hypotension, natriuresis and diuresis, and decrease in blood glucose level. It is also a mediator of inflammation and causes and increase in vascular permeability, release of mediators of inflammation like prostaglandins, stimulation of nociceptors and has a direct and indirect cardioprotective effect [4] HK is organized into distinct domains designated D1 through D6 that can be listed as follows with the amino acid positions numbering:   Some features and functions are recognized for these domains. Domains 1-3 belong to a family of cysteine protease inhibitors (Cystatins, MEROPS inhibitor family I25, clan IH) [6][7][8][9] and sequence homology can be simply searched based on human HK sequence using online databases of protein domains like PFAM (http://pfam.xfam.org/, accessed on 19 October 2021) [15][16][17][18][19][20][21][22][23][24], PROSITE (https://prosite.expasy.org/, accessed on 19 October 2021) [25][26][27][28][29][30][31][32][33][34][35] or SMART (http://smart.embl-heidelberg.de/, accessed on 19 October 2021) [36][37][38][39]. As cystatins-like domains their inhibitory action was confirmed experimentally with inhibition of atrial natriuretic factor by D1 and inhibition of calpain and papain (cysteine or thiol proteases) by D2 and D3, respectively. HK is cleaved at two positions in D4 (the smallest domain of HK) by plasma kallikrein (protease form of PK) to release the nonapeptide bradykinin (RPPGFSPFR) and the two-chain HKa. BK boosts vasodilation and vascular permeability and can develop soft tissue swelling that can be life threatening as that observed in hereditary angioedema [4]. D5 and 6 are vital for the contact pathway of blood coagulation. D5 is rich in histidine motifs facilitating binding to the negatively charged surfaces anchoring PK and fXI in close proximity to fXII. D6 contains overlapping stretches of amino acids that interacts in the complex formation with the coagulation fXI and PK [40][41][42][43].

Human HKa run as a two bands on SDS-PAGE under reducing conditions corresponding to a heavy, spanning amino acid residues 1-362 (D1-D3 and the first six amino acids of the short Domain 4), and a light chain consisted 255 amino acids (D5 and D6). huHK is heavily glycosylated with 3 N-linked carbohydrates on the heavy chain and 9 O-linked carbohydrates in the light chain of huHK. Many of the carbohydrate derivatives in huHK are sialic acid polymers [4,5]. Messenger RNA alternative splicing transcript of the KNG1 gene is translated into a smaller protein variant (70 kDa)-low molecular weight kininogen (LK). LK has identical domains 1-4 as huHK albeit Domain 5 is shortened by 38 amino acids in length and designated D5L while D6 is lacking. Digestion of LK by tissue kallikrein (TK) liberates a 10 amino acid peptide KRPPGFSPFR-Lysyl-Bradykinin (LBK) which is subsequently split to BK by the enzyme arginine aminopeptidase. LK is not involved in blood clotting but can still inhibit the aggregation of blood platelets. Domain interaction of HK has been reviewed extensively in Winter et al. (2020) publication [4].

Thanks to genome and transcriptome sequencing advancements, it is nowadays known that a similar situation with KNG1 gene messenger RNA alternatively splicing occurs not only in mammals, but also in reptiles, birds and amphibians, where HK and LK share D1 through D4 domains, but have varied D5 domains. D6 was found in HK from all tetrapods, suggesting HK carries PK in most terrestrial vertebrates. HK D5 is more mutable and in amphibians, crocodilians, birds, and turtles it is shorter than in mammals and is absent in lizards and snakes (Supplementary Figure S1A,B). Cartilaginous or ray-finned fish kininogens are more like mammalian LK than to HK where D5 or D6 domains are absent. Although kininogens are found in lobe-finned fishes, the coelacanth and the lungfish have a D6 domain with short histidine-rich motifs corresponding to a more D5 domain configuration. The appearance of HK in the lobe-finned fishes indicates an adaptation to blood clotting activation upon contact with silica-rich soil or mud impurities, coinciding with fXII appearance in the lungfishes, but fXII disappearance in birds and cetaceans where contact with the soil is much less frequent may indicate that contact activation is not very vital. Moreover, bird HK has shorter D5 domains (Domains 5 and 6 are important for the contact), but it seems it is still needed for kinin generation, as well as in cetaceans, where HK seems normal (Supplementary Figure S1), however, fXII and PK are not expressed as corresponding DNA fragments are degraded, pseudogene and non-gene junk DNA fragment, respectively [13]. Evolutionary and functional links between orthological HK sequences can be enriched after solving the spatial structure for the human protein to provide an adequate template for homology modeling.


## The Importance of Kininogen in Civilization and Infectious Diseases

HK significance, beyond physiological BK production, is participation in thrombotic disorders as it participates as cofactor for the contact activation system on many negatively charged surfaces (e.g., silica, nucleic acids, polyphosphates of different origins, misfolded proteins, collagen, bacteria, or viruses). Thrombosis and embolism are significant problems connected to civilization diseases related to the circulatory system where the role of HK should not be neglected. Thrombotic complications and coagulation disorders appear frequently secondary to infections with various pathogens where contact activation is triggered. BK released from HK during either contact activation or as part of the KKS contribute to the pathology of many inflammatory diseases like hereditary angioedema. Coronavirus SARS-CoV-2 the virus causing COVID-19 pandemic is known to bind Angiotensin-Converting Enzyme 2 (ACE2). Such binding increases BK release which promotes inflammation in the lungs, causing cough and fever, as well as further activation of the coagulation and the complement system. Emerging thrombotic complications associated with COVID-19 infection and reports of post-vaccination thrombotic reactions indicate the need to investigate the relationship within the hemostatic system, including contact factors [44,45]. Understanding the structure of HK is important for a better understanding of the changes and processes it undergoes not only in physiological state, but pathological situations like thrombotic disorders accompanying various civilization and infectious diseases, with the activation of the immune system, inflammation, and coagulation.


## Literature Describing the Structure of Kininogen

Although images of huHK were made many years ago by transmission electron microscopy (TEM) using various negative staining techniques, they do not shed much light on the actual shape of the kininogen [3,13,14]. In the oldest work by Weisel et al. (1994) [3], where TEM images of human kininogen were taken, the samples were applied to a substrate from mica with tungsten shading and an 80 kV Philips 400 microscope was used to obtain pictures at 60,000× magnification. Using the scale provided by the authors, 3 domains of the kininogen seem to have a total of 130-180 Å, while one domain was 50-90 Å. In this work, together with HK images, IgG antibodies (mass~150 kDa, length 100-150 Å) were visualized in some figures. Based on these data (Supplementary Figure S2), it can be calculated that the length of the antibody is about 150 Å, and the kininogen associated with this antibody is~180 Å long. Perhaps what the authors of this study showed were not individual kininogen molecules, but rather their aggregates, where a single domain is actually one kininogen molecule.

In 2001, Herwald et al. (2001) [44] used 5 µL of sample with a concentration of 5-19 µg per mL in 50 mM Tris. HCl, pH 7.4 buffer plus 150 mM NaCl to carry negative stain imaging on a glow discharged carbon-coated copper grids using JEOL 1200 EX 60 kV transmission electron microscope. In this work, according to the scale given in some figures, the cation-free (EDTA treated) or Mg 2+ treated protein molecules have estimated dimensions of 95 ± 30 Å (width) and 150 ± 30 Å (length) (Supplementary Figure S3A-C) with two larger domains (55 ± 20 Å) and three smaller domains (35 ± 20 Å) of 130-150 Å, and the substructure (subdomain) were visible. In the presence of Zn 2+ ions, the protein was more spherical and compact, with the dimensions reduced to 90 ± 30 Å (width) and 110 ± 30 Å (length) (Supplementary Figure S3D).

In 2009, Oehmcke, Mörgelin, and Herwald reported TEM images of HK [45], and according to the image scale (Supplementary Figure S4), the molecule should have only about 45 Å diameter. The internal structures corresponding to five to six domains were visible in those images. The dimensions of a given domain are only 15-20 Å. The authors did not describe the TEM methodology in the paper but referred to an earlier work by the group [44], and they did not provide numerical dimensions. In this study, the scale appears to underestimate the results and is not in agreement with previous reports.

In conclusion, based on the available literature with published TEM results of kininogen, there is no clarity and consistency as to the size of this molecule and its domains. This underscores the need for microscopic examination using more modern techniques than those from more than a decade ago.

The failure to publish the structure of HK may be due to the difficulty of its crystallization even in parts and inability to use the NMR technique, as the protein is too large. Unfortunately, all classic structure prediction methods, based on homology, threading, and de novo (ab initio), have failed. In the first two cases, it was due to the lack of appropriate, similar enough templates, and in the last, it was due to the huge size of this protein. did not describe the TEM methodology in the paper but referred to an earlier work by the group [44], and they did not provide numerical dimensions. In this study, the scale appears to underestimate the results and is not in agreement with previous reports.

In conclusion, based on the available literature with published TEM results of kininogen, there is no clarity and consistency as to the size of this molecule and its domains. This underscores the need for microscopic examination using more modern techniques than those from more than a decade ago.

The failure to publish the structure of HK may be due to the difficulty of its crystallization even in parts and inability to use the NMR technique, as the protein is too large. Unfortunately, all classic structure prediction methods, based on homology, threading, and de novo (ab initio), have failed. In the first two cases, it was due to the lack of appropriate, similar enough templates, and in the last, it was due to the huge size of this protein.

Using new tools such as AlphaFold Protein Structure Database, a structure for HK by DeepMind calculations de novo (https://www.alphafold.ebi.ac.uk/entry/P01042) could be predicted ( Figure 3) [46][47][48][49]. Unfortunately, despite the solution of cystatin domains (D1-D3) with good model confidence, the other domains of the protein have very low confidence and appear as almost unfolded floating chains, and no glycans were present ( Figure  3). A potential solution for this large 120 kDa protein containing many sugar residues is the use of cryogenic electron microscopy (cryo-EM). This will omit the bottle neck in traditional crystallography, as the formation of crystals is no longer needed. In contrast, a sample of low concentration protein solution is frozen to cryogenic temperatures by liquid ethane and fixed in an environment of thin film of vitreous ice to take electron microscope pictures of plenty protein molecules in different random orientations. Such flat two-dimensional images are classified and processed by powerful computers to generate a threedimensional structure [52][53][54][55][56]. Nowadays, available cryogenic microscopes with sensitive detectors and appropriate software are available and enable the resolution of some pro- A potential solution for this large 120 kDa protein containing many sugar residues is the use of cryogenic electron microscopy (cryo-EM). This will omit the bottle neck in traditional crystallography, as the formation of crystals is no longer needed. In contrast, a sample of low concentration protein solution is frozen to cryogenic temperatures by liquid ethane and fixed in an environment of thin film of vitreous ice to take electron microscope pictures of plenty protein molecules in different random orientations. Such flat two-dimensional images are classified and processed by powerful computers to generate a three-dimensional structure [52][53][54][55][56]. Nowadays, available cryogenic microscopes with sensitive detectors and appropriate software are available and enable the resolution of some proteins at 1.22 Å using a 300 kV beam of electrons [52,53,55]. More common and accessible dissemination of this technique should allow the solution of good-quality kininogen structures.


## Conclusions

Kininogens are important proteins playing a role in normal human physiology and pathology. Kininogens are part of the intrinsic coagulation system and the kallikrein-kinin systems and the source of the vasoactive nonapeptide BK. BK in turn has role in normal physiology as well as in disease. Despite the knowledge of the primary structure and the domain organization, still, there are no known spatial 3D structures of the whole proteins or its individual domains. Developed and improved visualization techniques providing near and/or atomic-level resolution provide the light at the end of the tunnel to resolve these structures. The application of cryo-EM to determine the spatial structure of kininogens will draw new frontiers in understanding the function of these proteins and open new pathways for drug development. 

## Figure 1 .
1Blood coagulation cascade. Roman numerals denote individual coagulation factors, the letter "a" stands for the active form, other abbreviations are as in the text.


D1: 1-113, D2: 114-234, D3: 235-357, D4: 358-383, D5: 384-502, and D6: 503-626. The domains 1, 2, 3, 5 and 6 are roughly 120 amino acids in length and Domain 4 is only 26 amino acids long. Domains 1 and 6 are connected by a single disulfide bridge involving Cys 10 and Cys 596 [4] (Figure 2).

## Figure 2 .
2(A) Human plasma kininogen showing alternative splicing products of KNG1 gene messenger RNA-HK and LK. Both forms of protein have similar D1, D2, D3, and D4 domains, but different D5 domains. Domain 6 is absent in LK and the molecular weight of this form is lower than

## Figure 2 .
2(A) Human plasma kininogen showing alternative splicing products of KNG1 gene messenger RNA-HK and LK. Both forms of protein have similar D1, D2, D3, and D4 domains, but different D5 domains. Domain 6 is absent in LK and the molecular weight of this form is lower than that of HK. (B) Human HK amino acid sequence according to UniProt P01042 with the indication of domains. N-terminus and C-terminus marked by corresponding letters. Disulfide (SS) bonds between the respective domains are marked.


Using new tools such as AlphaFold Protein Structure Database, a structure for HK by DeepMind calculations de novo (https://www.alphafold.ebi.ac.uk/entry/P01042, accessed on 19 October 2021) could be predicted (Figure 3) [46-49]. Unfortunately, despite the solution of cystatin domains (D1-D3) with good model confidence, the other domains of the protein have very low confidence and appear as almost unfolded floating chains, and no glycans were present (Figure 3). Int. J. Mol. Sci. 2021, 22, x FOR PEER REVIEW 7 of 10

## Figure 3 .
3AlphaFold prediction of human HK. Colors of domains according toFigure 2. The structure contains a signal peptide colored brown. The figure was prepared in UCSF Chimera 1.15 based on downloaded AlPhaFold PDB coordinates (accessed date 29 October c2021)[50,51].

## Figure 3 .
3AlphaFold prediction of human HK. Colors of domains according toFigure 2. The structure contains a signal peptide colored brown. The figure was prepared in UCSF Chimera 1.15 based on downloaded AlPhaFold PDB coordinates (accessed date 29 October 2021)[50,51].

## Table 1 .
1Blastp results of human HK amino acid sequence against Protein Data Bank proteins.Table 1. Cont. Percent identity is less than 30%. Cystatin domain homology is detected. None is HK fragment. 2 PDB according to https://www.rcsb.org/, https://www.ebi.ac.uk/pdbe/ and https://pdbj.org/, accessed on 29 October 2021.Description 
Percent Identity 1 
PDB Accession and Chain 2 

Cleaved human fetuin-b in 
complex with crayfish astacin 
[Homo sapiens] 

26.20 
6SAZ B 

Crystal structure of L68V 
mutant of human cystatin C 
[Homo sapiens] 

29.73 
3PS8 A 

Human Cystatin C; Dimeric 
Form With 3d Domain 
Swapping [Homo sapiens] 

30.63 
4N6M A 

Crystal structure of human 
cystatin E/M [Homo sapiens] 
29.73 
1G96 A 


## Table 2 .
2The best ten of 39 hits for searching "kininogen" keyword against RCSB PDB: https: //www.rcsb.org/, accessed on 29 October 2021.ID 
Title 
Released 

4ECC 
Chimeric GST Containing Inserts of Kininogen Peptides 
16 May 2012 
4ECB 
Chimeric GST Containing Inserts of Kininogen Peptides 
16 May 2012 
6BFP 
Bovine trypsin bound to potent inhibitor 
31 October 2018 
4JD9 
Contact pathway inhibitor from a sand fly 
16 October 2013 

6F3Y 

Backbone structure of Des-Arg10-Kallidin (DAKD) peptide bound to 
human Bradykinin 1 Receptor (B1R) determined by DNP-enhanced 
MAS SSNMR 

10 January 2018 

6F3X 

Backbone structure of Des-Arg10-Kallidin (DAKD) peptide in frozen 
DDM/CHS detergent micelle solution determined by DNP-enhanced 
MAS SSNMR 

10 January 2018 

6F3W 
Backbone structure of free bradykinin (BK) in DDM/CHS detergent 
micelle determined by MAS SSNMR 
10 January 2018 

6O1S Structure of human plasma kallikrein protease domain with inhibitor 
6 March 2019 
6O1G 
Full length human plasma kallikrein with inhibitor 
6 March 2019 
6F27 
NMR solution structure of non-bound [des-Arg10]-kallidin (DAKD) 10 January 2018 




Supplementary Materials: The following are available online at https://www.mdpi.com/article/10 .3390/ijms222413370/s1. Funding: This research received no external funding. Review Board Statement: Not applicable. Informed Consent Statement: Not applicable. Data Availability Statement: Not applicable.Institutional 
Acknowledgments: Molecular graphics image (Figure 3) was produced using the UCSF Chimera package from the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco (supported by NIH P41 RR001081). The author would like to thank the editor and the anonymous reviewers, whose comments helped improve the quality of the final manuscript, and Bassem M Mohammed from Saint Louis University, Department of Biochemistry & Molecular Biology, USA for very valuable comments regarding the improvement of the final manuscript.Conflicts of Interest:The authors declare no conflict of interest.
Characterization of Human High Molecular Weight Kininogen. Procoagulant Activity Associated with the Light Chain of Kinin-Free High Molecular Weight Kininogen. R E Thompson, R Mandle, A P Kaplan, 10.1084/jem.147.2.488J. Exp. Med. 147PubMedThompson, R.E.; Mandle, R.; Kaplan, A.P. Characterization of Human High Molecular Weight Kininogen. Procoagulant Activity Associated with the Light Chain of Kinin-Free High Molecular Weight Kininogen. J. Exp. Med. 1978, 147, 488-499. [CrossRef] [PubMed]

Human High Molecular Weight Kininogen. Studies of Structure-Function Relationships and of Proteolysis of the Molecule Occurring during Contact Activation of Plasma. D M Kerbiriou, J H Griffin, 10.1016/S0021-9258(19)86421-9J. Biol. Chem. 254Kerbiriou, D.M.; Griffin, J.H. Human High Molecular Weight Kininogen. Studies of Structure-Function Relationships and of Proteolysis of the Molecule Occurring during Contact Activation of Plasma. J. Biol. Chem. 1979, 254, 12020-12027. [CrossRef]

The Shape of High Molecular Weight Kininogen. Organization into Structural Domains, Changes with Activation, and Interactions with Prekallikrein, as Determined by Electron Microscopy. J W Weisel, C Nagaswami, J L Woodhead, R A D Cadena, J D Page, R W Colman, 10.1016/S0021-9258(17)36995-8J. Biol. Chem. 269Weisel, J.W.; Nagaswami, C.; Woodhead, J.L.; Cadena, R.A.D.; Page, J.D.; Colman, R.W. The Shape of High Molecular Weight Kininogen. Organization into Structural Domains, Changes with Activation, and Interactions with Prekallikrein, as Determined by Electron Microscopy. J. Biol. Chem. 1994, 269, 10100-10106. [CrossRef]

Clotting Factors: Clinical Biochemistry and Their Roles as Plasma Enzymes. W E Winter, D N Greene, S G Beal, J A Isom, H Manning, G Wilkerson, N Harris, Adv. Clin. Chem. 94PubMedWinter, W.E.; Greene, D.N.; Beal, S.G.; Isom, J.A.; Manning, H.; Wilkerson, G.; Harris, N. Clotting Factors: Clinical Biochemistry and Their Roles as Plasma Enzymes. Adv. Clin. Chem. 2020, 94, 31-84. [PubMed]

The Sialic Acid Content and Isoelectric Point of Human Kininogen. J C Londesborough, U Hamberg, 10.1042/bj1450401Biochem. J. 145PubMedLondesborough, J.C.; Hamberg, U. The Sialic Acid Content and Isoelectric Point of Human Kininogen. Biochem. J. 1975, 145, 401-403. [CrossRef] [PubMed]

Identification of Prekallikrein and High Molecular Weight Kininogen as a Complex in Human Plasma. R J Mandle, R W Colman, A P Kaplan, 10.1073/pnas.73.11.4179Proc. Natl. Acad. Sci. Natl. Acad. SciUSA73Mandle, R.J.; Colman, R.W.; Kaplan, A.P. Identification of Prekallikrein and High Molecular Weight Kininogen as a Complex in Human Plasma. Proc. Natl. Acad. Sci. USA 1976, 73, 4179-4183. [CrossRef]

. Int. J. Mol. Sci. 2213370Int. J. Mol. Sci. 2021, 22, 13370

High Molecular Weight Kininogen-Binding Site of Prekallikrein Probed by Monoclonal Antibodies. J Hock, R Vogel, R P Linke, W Müller-Esterl, 10.1016/S0021-9258(19)38500-XJ. Biol. Chem. 265Hock, J.; Vogel, R.; Linke, R.P.; Müller-Esterl, W. High Molecular Weight Kininogen-Binding Site of Prekallikrein Probed by Monoclonal Antibodies. J. Biol. Chem. 1990, 265, 12005-12011. [CrossRef]

Immunochemical Studies of Human High Molecular Weight Kininogen and of Its Complexes with Plasma Prekallikrein or Kallikrein. D M Kerbiriou, B N Bouma, J H Griffin, 10.1016/S0021-9258(19)85618-1J. Biol. Chem. 255Kerbiriou, D.M.; Bouma, B.N.; Griffin, J.H. Immunochemical Studies of Human High Molecular Weight Kininogen and of Its Complexes with Plasma Prekallikrein or Kallikrein. J. Biol. Chem. 1980, 255, 3952-3958. [CrossRef]

A Comparison of the Effects of Factor XII Deficiency and Prekallikrein Deficiency on Thrombus Formation. Y Kokoye, I Ivanov, Q Cheng, A Matafonov, S K Dickeson, S Mason, D J Sexton, T Renné, K Mccrae, E P Feener, 10.1016/j.thromres.2016.02.020Thromb. Res. 140Kokoye, Y.; Ivanov, I.; Cheng, Q.; Matafonov, A.; Dickeson, S.K.; Mason, S.; Sexton, D.J.; Renné, T.; McCrae, K.; Feener, E.P.; et al. A Comparison of the Effects of Factor XII Deficiency and Prekallikrein Deficiency on Thrombus Formation. Thromb. Res. 2016, 140, 118-124. [CrossRef]

An Update on Factor XI Structure and Function. B M Mohammed, A Matafonov, I Ivanov, M.-F Sun, Q Cheng, S K Dickeson, C Li, D Sun, I M Verhamme, J Emsley, 10.1016/j.thromres.2017.10.008Thromb. Res. 161Mohammed, B.M.; Matafonov, A.; Ivanov, I.; Sun, M.-F.; Cheng, Q.; Dickeson, S.K.; Li, C.; Sun, D.; Verhamme, I.M.; Emsley, J.; et al. An Update on Factor XI Structure and Function. Thromb. Res. 2018, 161, 94-105. [CrossRef]

Studies of Binding of Prekallikrein and Factor XI to High Molecular Weight Kininogen and Its Light Chain. R E Thompson, R Mandle, A P Kaplan, 10.1073/pnas.76.10.4862Proc. Natl. Acad. Sci. Natl. Acad. SciUSA76Thompson, R.E.; Mandle, R.; Kaplan, A.P. Studies of Binding of Prekallikrein and Factor XI to High Molecular Weight Kininogen and Its Light Chain. Proc. Natl. Acad. Sci. USA 1979, 76, 4862-4866. [CrossRef]

Evolution of the Contact Phase of Vertebrate Blood Coagulation. M B Ponczek, D Gailani, R F Doolittle, 10.1111/j.1538-7836.2008.03143.xJ. Thromb. Haemost. 6Ponczek, M.B.; Gailani, D.; Doolittle, R.F. Evolution of the Contact Phase of Vertebrate Blood Coagulation. J. Thromb. Haemost. 2008, 6, 1876-1883. [CrossRef]

The Evolution of Factor XI and the Kallikrein-Kinin System. M B Ponczek, A Shamanaev, A Laplace, S K Dickeson, P Srivastava, M.-F Sun, A Gruber, C Kastrup, J Emsley, D Gailani, 10.1182/bloodadvances.2020002456Blood Adv. 2020Ponczek, M.B.; Shamanaev, A.; LaPlace, A.; Dickeson, S.K.; Srivastava, P.; Sun, M.-F.; Gruber, A.; Kastrup, C.; Emsley, J.; Gailani, D. The Evolution of Factor XI and the Kallikrein-Kinin System. Blood Adv. 2020, 4, 6135-6147. [CrossRef]

Association of Factor XI and High Molecular Weight Kininogen in Human Plasma. R E Thompson, R Mandle, A P Kaplan, 10.1172/JCI108898J. Clin. Investig. 60Thompson, R.E.; Mandle, R.; Kaplan, A.P. Association of Factor XI and High Molecular Weight Kininogen in Human Plasma. J. Clin. Investig. 1977, 60, 1376-1380. [CrossRef]

R D Finn, J Mistry, B Schuster-Böckler, S Griffiths-Jones, V Hollich, T Lassmann, S Moxon, M Marshall, A Khanna, R Durbin, 10.1093/nar/gkj149Clans, Web Tools and Services. 34Finn, R.D.; Mistry, J.; Schuster-Böckler, B.; Griffiths-Jones, S.; Hollich, V.; Lassmann, T.; Moxon, S.; Marshall, M.; Khanna, A.; Durbin, R.; et al. Pfam: Clans, Web Tools and Services. Nucleic Acids Res. 2006, 34, D247-D251. [CrossRef]

J Mistry, S Chuguransky, L Williams, M Qureshi, G A Salazar, E L L Sonnhammer, S C E Tosatto, L Paladin, S Raj, L J Richardson, 10.1093/nar/gkaa913The Protein Families Database in 2021. 49Mistry, J.; Chuguransky, S.; Williams, L.; Qureshi, M.; Salazar, G.A.; Sonnhammer, E.L.L.; Tosatto, S.C.E.; Paladin, L.; Raj, S.; Richardson, L.J.; et al. Pfam: The Protein Families Database in 2021. Nucleic Acids Res. 2021, 49, D412-D419. [CrossRef]

The Pfam Protein Families Database. A Bateman, E Birney, R Durbin, S R Eddy, K L Howe, E L L Sonnhammer, 10.1093/nar/28.1.263Nucleic Acids Res. 28Bateman, A.; Birney, E.; Durbin, R.; Eddy, S.R.; Howe, K.L.; Sonnhammer, E.L.L. The Pfam Protein Families Database. Nucleic Acids Res. 2000, 28, 263-266. [CrossRef]

The Pfam Protein Families Database. R D Finn, J Mistry, J Tate, P Coggill, A Heger, J E Pollington, O L Gavin, P Gunasekaran, G Ceric, S K Forslund, 10.1093/nar/gkp985Nucleic Acids Res. 38Finn, R.D.; Mistry, J.; Tate, J.; Coggill, P.; Heger, A.; Pollington, J.E.; Gavin, O.L.; Gunasekaran, P.; Ceric, G.; Forslund, S.K.; et al. The Pfam Protein Families Database. Nucleic Acids Res. 2009, 38, D211-D222. [CrossRef]

The Pfam Protein Families Database. A Bateman, L Coin, R Durbin, R D Finn, V Hollich, S Griffiths-Jones, A Khanna, M Marshall, S Moxon, E L L Sonnhammer, 10.1093/nar/gkh121Nucleic Acids Res. 32Bateman, A.; Coin, L.; Durbin, R.; Finn, R.D.; Hollich, V.; Griffiths-Jones, S.; Khanna, A.; Marshall, M.; Moxon, S.; Sonnhammer, E.L.L.; et al. The Pfam Protein Families Database. Nucleic Acids Res. 2004, 32, D138-D141. [CrossRef]

The Pfam Protein Families Database. M Punta, P C Coggill, R Y Eberhardt, J Mistry, J Tate, C Boursnell, N Pang, S K Forslund, G Ceric, J Clements, 10.1093/nar/gkr1065Nucleic Acids Res. 40Punta, M.; Coggill, P.C.; Eberhardt, R.Y.; Mistry, J.; Tate, J.; Boursnell, C.; Pang, N.; Forslund, S.K.; Ceric, G.; Clements, J.; et al. The Pfam Protein Families Database. Nucleic Acids Res. 2012, 40, D290-D301. [CrossRef]

The Pfam Protein Families Database. R D Finn, J Tate, J Mistry, P C Coggill, S J Sammut, H.-R Hotz, G Ceric, K Forslund, S R Eddy, E L L Sonnhammer, 10.1093/nar/gkm960Nucleic Acids Res. 36Finn, R.D.; Tate, J.; Mistry, J.; Coggill, P.C.; Sammut, S.J.; Hotz, H.-R.; Ceric, G.; Forslund, K.; Eddy, S.R.; Sonnhammer, E.L.L.; et al. The Pfam Protein Families Database. Nucleic Acids Res. 2008, 36, D281-D288. [CrossRef]

Identifying Protein Domains with the Pfam Database. P Coggill, R D Finn, A Bateman, 10.1002/0471250953.bi0205s23Curr. Protoc. Bioinform. 1PubMedCoggill, P.; Finn, R.D.; Bateman, A. Identifying Protein Domains with the Pfam Database. Curr. Protoc. Bioinform. 2008, 1, 2-5. [CrossRef] [PubMed]

Pfam 10 Years on: 10 000 Families and Still Growing. S J Sammut, R D Finn, A Bateman, 10.1093/bib/bbn010Brief. Bioinform. 9PubMedSammut, S.J.; Finn, R.D.; Bateman, A. Pfam 10 Years on: 10 000 Families and Still Growing. Brief. Bioinform. 2008, 9, 210-219. [CrossRef] [PubMed]

Pfam Protein Families Database in 2019. S El-Gebali, J Mistry, A Bateman, S R Eddy, A Luciani, S C Potter, M Qureshi, L J Richardson, G A Salazar, A Smart, 10.1093/nar/gky995Nucleic Acids Res. 47El-Gebali, S.; Mistry, J.; Bateman, A.; Eddy, S.R.; Luciani, A.; Potter, S.C.; Qureshi, M.; Richardson, L.J.; Salazar, G.A.; Smart, A.; et al. Pfam Protein Families Database in 2019. Nucleic Acids Res. 2019, 47, D427-D432. [CrossRef]

PROSITE, a Protein Domain Database for Functional Characterization and Annotation. C J A Sigrist, L Cerutti, E De Castro, P S Langendijk-Genevaux, V Bulliard, A Bairoch, N Hulo, 10.1093/nar/gkp885Nucleic Acids Res. 38PubMedSigrist, C.J.A.; Cerutti, L.; de Castro, E.; Langendijk-Genevaux, P.S.; Bulliard, V.; Bairoch, A.; Hulo, N. PROSITE, a Protein Domain Database for Functional Characterization and Annotation. Nucleic Acids Res. 2009, 38, D161-D166. [CrossRef] [PubMed]

PROSITE: A Documented Database Using Patterns and Profiles as Motif Descriptors. C J A Sigrist, L Cerutti, N Hulo, A Gattiker, L Falquet, M Pagni, A Bairoch, P Bucher, 10.1093/bib/3.3.265Brief. Bioinform. 3PubMedSigrist, C.J.A.; Cerutti, L.; Hulo, N.; Gattiker, A.; Falquet, L.; Pagni, M.; Bairoch, A.; Bucher, P. PROSITE: A Documented Database Using Patterns and Profiles as Motif Descriptors. Brief. Bioinform. 2002, 3, 265-274. [CrossRef] [PubMed]

The PROSITE Database. N Hulo, A Bairoch, V Bulliard, L Cerutti, E De Castro, P S Langendijk-Genevaux, M Pagni, C J Sigrist, 10.1093/nar/gkj063D227-D230Nucleic Acids Res. 34Hulo, N.; Bairoch, A.; Bulliard, V.; Cerutti, L.; de Castro, E.; Langendijk-Genevaux, P.S.; Pagni, M.; Sigrist, C.J.A. The PROSITE Database. Nucleic Acids Res. 2006, 34, D227-D230. [CrossRef]

The 20 Years of PROSITE. N Hulo, A Bairoch, V Bulliard, L Cerutti, B A Cuche, E De Castro, C Lachaize, P S Langendijk-Genevaux, C J Sigrist, 10.1093/nar/gkm977Nucleic Acids Res. 36PubMedHulo, N.; Bairoch, A.; Bulliard, V.; Cerutti, L.; Cuche, B.A.; de castro, E.; Lachaize, C.; Langendijk-Genevaux, P.S.; Sigrist, C.J.A. The 20 Years of PROSITE. Nucleic Acids Res. 2008, 36, D245-D249. [CrossRef] [PubMed]

Recent Improvements to the PROSITE Database. N Hulo, C J A Sigrist, V Le Saux, P S Langendijk-Genevaux, L Bordoli, A Gattiker, E De Castro, P Bucher, A Bairoch, 10.1093/nar/gkh044Nucleic Acids Res. 32Hulo, N.; Sigrist, C.J.A.; le Saux, V.; Langendijk-Genevaux, P.S.; Bordoli, L.; Gattiker, A.; de Castro, E.; Bucher, P.; Bairoch, A. Recent Improvements to the PROSITE Database. Nucleic Acids Res. 2004, 32, D134-D137. [CrossRef]

Xenarios, I. New and Continuing Developments at PROSITE. C J A Sigrist, E De Castro, L Cerutti, B A Cuche, N Hulo, A Bridge, L Bougueleret, 10.1093/nar/gks1067Nucleic Acids Res. 41Sigrist, C.J.A.; de Castro, E.; Cerutti, L.; Cuche, B.A.; Hulo, N.; Bridge, A.; Bougueleret, L.; Xenarios, I. New and Continuing Developments at PROSITE. Nucleic Acids Res. 2013, 41, D344-D347. [CrossRef]

The PROSITE Database, Its Status in 1999. K Hofmann, P Bucher, L Falquet, A Bairoch, 10.1093/nar/27.1.215Nucleic Acids Res. 27PubMedHofmann, K.; Bucher, P.; Falquet, L.; Bairoch, A. The PROSITE Database, Its Status in 1999. Nucleic Acids Res. 1999, 27, 215-219. [CrossRef] [PubMed]

The PROSITE Database, Its Status in 2002. L Falquet, M Pagni, P Bucher, N Hulo, C J A Sigrist, K Hofmann, A Bairoch, 10.1093/nar/30.1.235Nucleic Acids Res. 30PubMedFalquet, L.; Pagni, M.; Bucher, P.; Hulo, N.; Sigrist, C.J.A.; Hofmann, K.; Bairoch, A. The PROSITE Database, Its Status in 2002. Nucleic Acids Res. 2002, 30, 235-238. [CrossRef] [PubMed]

PfsearchV3: A Code Acceleration and Heuristic to Search PROSITE Profiles. T Schuepbach, M Pagni, A Bridge, L Bougueleret, I Xenarios, L Cerutti, 10.1093/bioinformatics/btt129Bioinformatics. 29PubMedSchuepbach, T.; Pagni, M.; Bridge, A.; Bougueleret, L.; Xenarios, I.; Cerutti, L. PfsearchV3: A Code Acceleration and Heuristic to Search PROSITE Profiles. Bioinformatics 2013, 29, 1215-1217. [CrossRef] [PubMed]

C J A Sigrist, E De Castro, P S Langendijk-Genevaux, V Le Saux, A Bairoch, N Hulo, Prorule, 10.1093/bioinformatics/bti614A New Database Containing Functional and Structural Information on PROSITE Profiles. 21PubMedSigrist, C.J.A.; de Castro, E.; Langendijk-Genevaux, P.S.; le Saux, V.; Bairoch, A.; Hulo, N. ProRule: A New Database Containing Functional and Structural Information on PROSITE Profiles. Bioinformatics 2005, 21, 4060-4066. [CrossRef] [PubMed]

. Int. J. Mol. Sci. 2213370Int. J. Mol. Sci. 2021, 22, 13370

ScanProsite: Detection of PROSITE Signature Matches and ProRule-Associated Functional and Structural Residues in Proteins. E De Castro, C J A Sigrist, A Gattiker, V Bulliard, P S Langendijk-Genevaux, E Gasteiger, A Bairoch, N Hulo, 10.1093/nar/gkl124Nucleic Acids Res. 34PubMedde Castro, E.; Sigrist, C.J.A.; Gattiker, A.; Bulliard, V.; Langendijk-Genevaux, P.S.; Gasteiger, E.; Bairoch, A.; Hulo, N. ScanProsite: Detection of PROSITE Signature Matches and ProRule-Associated Functional and Structural Residues in Proteins. Nucleic Acids Res. 2006, 34, W362-W365. [CrossRef] [PubMed]

Recent Updates to the Protein Domain Annotation Resource. I Letunic, T Doerks, P Bork, 10.1093/nar/gkr931Nucleic Acids Res. 7Letunic, I.; Doerks, T.; Bork, P. SMART 7: Recent Updates to the Protein Domain Annotation Resource. Nucleic Acids Res. 2012, 40, D302-D305. [CrossRef]

I Letunic, T Doerks, P Bork, Smart, 10.1093/nar/gku949Recent Updates, New Developments and Status in 2015. 43Letunic, I.; Doerks, T.; Bork, P. SMART: Recent Updates, New Developments and Status in 2015. Nucleic Acids Res. 2015, 43, D257-D260. [CrossRef]

SMART: A Web-Based Tool for the Study of Genetically Mobile Domains. J Schultz, R R Copley, T Doerks, C P Ponting, P Bork, 10.1093/nar/28.1.231Nucleic Acids Res. 28Schultz, J.; Copley, R.R.; Doerks, T.; Ponting, C.P.; Bork, P. SMART: A Web-Based Tool for the Study of Genetically Mobile Domains. Nucleic Acids Res. 2000, 28, 231-234. [CrossRef]

20 Years of the SMART Protein Domain Annotation Resource. I Letunic, P Bork, 10.1093/nar/gkx922Nucleic Acids Res. 46Letunic, I.; Bork, P. 20 Years of the SMART Protein Domain Annotation Resource. Nucleic Acids Res. 2018, 46, D493-D496. [CrossRef]

Evolution of Proteins of the Cystatin Superfamily. N D Rawlings, A J Barrett, 10.1007/BF02102453J. Mol. Evol. 30Rawlings, N.D.; Barrett, A.J. Evolution of Proteins of the Cystatin Superfamily. J. Mol. Evol. 1990, 30, 60-71. [CrossRef]

Phylogenetic Analysis of Vertebrate Kininogen Genes. L Zhou, J Li-Ling, H Huang, F Ma, Q Li, 10.1016/j.ygeno.2007.10.007Genomics. 91PubMedZhou, L.; Li-Ling, J.; Huang, H.; Ma, F.; Li, Q. Phylogenetic Analysis of Vertebrate Kininogen Genes. Genomics 2008, 91, 129-141. [CrossRef] [PubMed]

Phylogenomic Analysis of the Cystatin Superfamily in Eukaryotes and Prokaryotes. D Kordiš, V Turk, 10.1186/1471-2148-9-266BMC Evol. Biol. 9Kordiš, D.; Turk, V. Phylogenomic Analysis of the Cystatin Superfamily in Eukaryotes and Prokaryotes. BMC Evol. Biol. 2009, 9, 266. [CrossRef]

Kininogens: More than Cysteine Protease Inhibitors and Kinin Precursors. G Lalmanach, C Naudin, F Lecaille, H Fritz, 10.1016/j.biochi.2010.03.011Biochimie. 92Lalmanach, G.; Naudin, C.; Lecaille, F.; Fritz, H. Kininogens: More than Cysteine Protease Inhibitors and Kinin Precursors. Biochimie 2010, 92, 1568-1579. [CrossRef]

Zinc-Dependent Conformational Changes in Domain D5 of High Molecular Mass Kininogen Modulate Contact Activation. H Herwald, M Mörgelin, H G Svensson, U Sjöbring, 10.1046/j.1432-1033.2001.01888.xEur. J. Biochem. 268PubMedHerwald, H.; Mörgelin, M.; Svensson, H.G.; Sjöbring, U. Zinc-Dependent Conformational Changes in Domain D5 of High Molecular Mass Kininogen Modulate Contact Activation. Eur. J. Biochem. 2001, 268, 396-404. [CrossRef] [PubMed]

Activation of the Human Contact System on Neutrophil Extracellular Traps. S Oehmcke, M Mörgelin, H Herwald, 10.1159/000203700J. Innate Immun. 1Oehmcke, S.; Mörgelin, M.; Herwald, H. Activation of the Human Contact System on Neutrophil Extracellular Traps. J. Innate Immun. 2009, 1, 225-230. [CrossRef]

AlphaFold at CASP13. M Alquraishi, 10.1093/bioinformatics/btz422Bioinformatics. 35Alquraishi, M. AlphaFold at CASP13. Bioinformatics 2019, 35, 4862-4865. [CrossRef]

Could AlphaFold Revolutionize Chemical Therapeutics?. A B Tong, J D Burch, D Mckay, C Bustamante, M A Crackower, H Wu, 10.1038/s41594-021-00670-xNat. Struct. Mol. Biol. 28PubMedTong, A.B.; Burch, J.D.; McKay, D.; Bustamante, C.; Crackower, M.A.; Wu, H. Could AlphaFold Revolutionize Chemical Therapeutics? Nat. Struct. Mol. Biol. 2021, 28, 771-772. [CrossRef] [PubMed]

What Does AlphaFold Mean for Drug Discovery?. A Mullard, 10.1038/d41573-021-00161-0Nat. Rev. Drug Discov. 20Mullard, A. What Does AlphaFold Mean for Drug Discovery? Nat. Rev. Drug Discov. 2021, 20, 725-727. [CrossRef]

Highly Accurate Protein Structure Prediction with AlphaFold. J Jumper, R Evans, A Pritzel, T Green, M Figurnov, O Ronneberger, K Tunyasuvunakool, R Bates, A Žídek, A Potapenko, 10.1038/s41586-021-03819-2Nature. 596Jumper, J.; Evans, R.; Pritzel, A.; Green, T.; Figurnov, M.; Ronneberger, O.; Tunyasuvunakool, K.; Bates, R.; Žídek, A.; Potapenko, A.; et al. Highly Accurate Protein Structure Prediction with AlphaFold. Nature 2021, 596, 583-589. [CrossRef]

UCSF Chimera-A Visualization System for Exploratory Research and Analysis. E F Pettersen, T D Goddard, C C Huang, G S Couch, D M Greenblatt, E C Meng, T E Ferrin, 10.1002/jcc.20084J. Comput. Chem. 25Pettersen, E.F.; Goddard, T.D.; Huang, C.C.; Couch, G.S.; Greenblatt, D.M.; Meng, E.C.; Ferrin, T.E. UCSF Chimera-A Visualiza- tion System for Exploratory Research and Analysis. J. Comput. Chem. 2004, 25, 1605-1612. [CrossRef]

UCSF Chimera-A visualization system for exploratory research and analysis. A Waterhouse, M Bertoni, S Bienert, G Studer, G Tauriello, R Gumienny, F T Heer, T A P De Beer, C Rempfer, L Bordoli, J. Biol. Chem. 25Waterhouse, A.; Bertoni, M.; Bienert, S.; Studer, G.; Tauriello, G.; Gumienny, R.; Heer, F.T.; de Beer, T.A.P.; Rempfer, C.; Bordoli, L.; et al. UCSF Chimera-A visualization system for exploratory research and analysis. J. Biol. Chem. 2004, 25, 1605-1612.

L A Earl, V Falconieri, J L Milne, S Subramaniam, Cryo-Em, 10.1016/j.sbi.2017.06.002Beyond the Microscope. 46PubMedEarl, L.A.; Falconieri, V.; Milne, J.L.; Subramaniam, S. Cryo-EM: Beyond the Microscope. Curr. Opin. Struct. Biol. 2017, 46, 71-78. [CrossRef] [PubMed]

Single-Particle Cryo-EM at Atomic Resolution. T Nakane, A Kotecha, A Sente, G Mcmullan, S Masiulis, P M G E Brown, I T Grigoras, L Malinauskaite, T Malinauskas, J Miehling, 10.1038/s41586-020-2829-0Nature. 587Nakane, T.; Kotecha, A.; Sente, A.; McMullan, G.; Masiulis, S.; Brown, P.M.G.E.; Grigoras, I.T.; Malinauskaite, L.; Malinauskas, T.; Miehling, J.; et al. Single-Particle Cryo-EM at Atomic Resolution. Nature 2020, 587, 152-156. [CrossRef]

Single-Particle Cryo-EM-How Did It Get Here and Where Will It Go. Y Cheng, 10.1126/science.aat4346Science. 361Cheng, Y. Single-Particle Cryo-EM-How Did It Get Here and Where Will It Go. Science 2018, 361, 876-880. [CrossRef]

Reveals Exquisite Molecular Structures-At High Cost. A Cheaper Microscope Could Bring the Resolution Revolution to the Masses. E Hand, Cryo-Em, 10.1126/science.367.6476.354Science. 367Hand, E. Cryo-EM Reveals Exquisite Molecular Structures-At High Cost. A Cheaper Microscope Could Bring the Resolution Revolution to the Masses. Science 2020, 367, 354-358. [CrossRef]

Cryogenic Transmission Electron Microscopy: Aqueous Suspensions of Nanoscale Objects. N D Burrows, R L Penn, 10.1017/S1431927613013354Microsc. Microanal. 19PubMedBurrows, N.D.; Penn, R.L. Cryogenic Transmission Electron Microscopy: Aqueous Suspensions of Nanoscale Objects. Microsc. Microanal. 2013, 19, 1542-1553. [CrossRef] [PubMed]