# Utilization of CRISPR-Mediated Tools for Studying Functional Genomics in Hematological Malignancies: An Overview on the Current Perspectives, Challenges, and Clinical Implications

CorpusID: 246362613
 
tags: #Medicine, #Biology

URL: [https://www.semanticscholar.org/paper/b260c059f8df1567d8e5025be40910a20703ae9d](https://www.semanticscholar.org/paper/b260c059f8df1567d8e5025be40910a20703ae9d)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | True |
| By Annotator      | (Not Annotated) |

---

Utilization of CRISPR-Mediated Tools for Studying Functional Genomics in Hematological Malignancies: An Overview on the Current Perspectives, Challenges, and Clinical Implications
January 2022

Maheswaran Solayappan 
Regenerative Medicine Sciences Cluster
Advanced Medical and Dental Institute
Universiti Sains Malaysia
PenangMalaysia

Department of Biotechnology
Faculty of Applied Sciences
AIMST University
BedongMalaysia

Adam Azlan 
Regenerative Medicine Sciences Cluster
Advanced Medical and Dental Institute
Universiti Sains Malaysia
PenangMalaysia

Zi Kang 
Khor 
Regenerative Medicine Sciences Cluster
Advanced Medical and Dental Institute
Universiti Sains Malaysia
PenangMalaysia

MotYee Yik 
Regenerative Medicine Sciences Cluster
Advanced Medical and Dental Institute
Universiti Sains Malaysia
PenangMalaysia

Matiullah Khan 
Department of Pathology
Faculty of Medicine
AIMST University
BedongMalaysia

Huazhong Agricultural University
China

Xuying Zhang
ICAR Indian Institute of Soybean Research
India

University of Veterinary Medicine Hannover
Germany

Narazah Mohd Yusoff 
Regenerative Medicine Sciences Cluster
Advanced Medical and Dental Institute
Universiti Sains Malaysia
PenangMalaysia

Emmanuel Jairaj Moses emmanuel_jm@usm.my 
Regenerative Medicine Sciences Cluster
Advanced Medical and Dental Institute
Universiti Sains Malaysia
PenangMalaysia

Shengsong Xie 
Milind B Ratnaparkhe 
Emmanuel Jairaj Moses 
Utilization of CRISPR-Mediated Tools for Studying Functional Genomics in Hematological Malignancies: An Overview on the Current Perspectives, Challenges, and Clinical Implications

Frontiers in Genetics | www.frontiersin.org
12767298January 202210.3389/fgene.2021.767298Received: 30 August 2021 Accepted: 17 November 2021 Published: 28 January 2022Edited by: Reviewed by: *Correspondence: Specialty section: This article was submitted to Genomic Assay Technology, a section of the journal Frontiers in Genetics Citation: Solayappan M, Azlan A, Khor KZ, Yik MY, Khan M, Yusoff NM and Moses EJ (2022) Utilization of CRISPR-Mediated Tools for Studying Functional Genomics in Hematological Malignancies: An Overview on the Current Perspectives, Challenges, and Clinical Implications. Front. Genet. 12:767298. 1 REVIEWclustered regularly interspaced short palindromic repeats (CRISPR)gene editinghematological malignanciesfunctional genomicstherapeutic targets
Hematological malignancies (HM) are a group of neoplastic diseases that are usually heterogenous in nature due to the complex underlying genetic aberrations in which collaborating mutations enable cells to evade checkpoints that normally safeguard it against DNA damage and other disruptions of healthy cell growth. Research regarding chromosomal structural rearrangements and alterations, gene mutations, and functionality are currently being carried out to understand the genomics of these abnormalities. It is also becoming more evident that cross talk between the functional changes in transcription and proteins gives the characteristics of the disease although specific mutations may induce unique phenotypes. Functional genomics is vital in this aspect as it measures the complete genetic change in cancerous cells and seeks to integrate the dynamic changes in these networks to elucidate various cancer phenotypes. The advent of CRISPR technology has indeed provided a superfluity of benefits to mankind, as this versatile technology enables DNA editing in the genome. The CRISPR-Cas9 system is a precise genome editing tool, and it has revolutionized methodologies in the field of hematology. Currently, there are various CRISPR systems that are used to perform robust site-specific gene editing to study HM. Furthermore, experimental approaches that are based on CRISPR technology have created promising tools for developing effective hematological therapeutics. Therefore, this review will focus on diverse applications of CRISPR-based gene-editing tools in HM and its potential future trajectory. Collectively, this review will demonstrate the key roles of different CRISPR systems that are being used in HM, and the literature will be a representation of a critical step toward further understanding the biology of HM and the development of potential therapeutic approaches.

## INTRODUCTION

Hematological malignancies (HM) are cancers that begin in the cells of blood-forming tissues such as the bone marrow or generally in the cells immune system. In the revised World Health Organization (WHO) classification, hematological malignancies are divided according to morphology, immunophenotype, and genetic and clinical features (Taylor et al., 2017). Further divisions can be made which are myeloid malignancies that are categorized into the following groups: myeloproliferative neoplasms, myelodysplastic/ myeloproliferative neoplasms, myelodysplastic syndromes, acute myeloid leukemias, acute leukemias of ambiguous lineage, and precursor lymphoid neoplasms. Then there are lymphoid malignancies that are categorized into mature B-cell neoplasms, mature T-and NK-cell neoplasms, Hodgkin's lymphoma, histiocytic and dendritic cell neoplasms, and posttransplantation lymphoproliferative disorders.

The current advancement in the field of molecular biology has spurred the way toward understanding a disease in an extensive manner and especially deepens our knowledge of hematological malignancies. Many advanced molecular biology and bioinformatics tools such as PCR, NGS, and karyotyping are currently utilized to understand the pathogenesis of hematological malignancies. Nevertheless, these methods only allow us to map genes but not fully understand the function. Therefore, it is imperative for researchers to utilize functional genomics to decipher and unravel new mechanisms that were previously unknown.

Advancement in molecular biology has now paved the way for us to understand the roles of genes that are involved directly or indirectly in the molecular and cellular mechanisms underlying hematological malignancies. The way forward now is to take the optimal charge through unraveling the roles of specific genes by manipulating their functions through gene editing. Currently, there are many techniques available to study gene function including site-directed mutagenesis and siRNA-mediated gene silencing (Gavrilov and Saltzman, 2012).

Nevertheless, these techniques are regarded to be less efficient and specific as compared to the various gene editing techniques (Zhang and McCarty, 2017) which will be reviewed in the next section.


## GENE EDITING TECHNIQUES

Gene editing is the ability to make extremely precise changes in the DNA sequence of a living organism and essentially customizing its genetic makeup. Gene editing is achieved by using enzymes, predominantly nucleases, that have been synthesized to target a specific DNA sequence. This is when they introduce cuts into the DNA strands, allowing the removal of existing DNA and the insertion of a replacement DNA. It is a type of genetic engineering tool to insert, delete, or replace a DNA in the genome of an organism using "molecular scissors" (Saha et al., 2019). These nucleases create site-specific double-strand breaks (DSBs) at desired locations in the genome (Vítor et al., 2020). The induced double-strand breaks are then repaired through non-homologous endjoining (NHEJ) or homologous recombination (HR) which results in targeted mutations also known as "edits" (Salsman et al., 2017). There have been three important families of bioengineered nucleases being used such as zinc-finger nucleases (ZFNs), transcription activator-like effector-based nucleases (TALENs), and CRISPR-Cas systems .


## Zinc-Finger Nucleases

The discovery of zinc-finger nucleases (ZFN) in the 1980s added value to the gene editing approach by being a precision tool in genome editing: it carried a site-specific recognition pattern in editing the desired gene. The ZFNs are composed of two parts, namely, Fokl nuclease that is fused with zinc-finger DNA-binding domains. The zinc-finger DNA-binding domain has a unique characteristic of recognizing a 3-base pair site on DNA, and it can be combined to recognize longer sequences (Wu et al., 2007). Furthermore, the ZFNs act as dimers by upsurging the length of the DNA recognition site and increasing the specificity. Cys 2 -His 2 ZFNs are fusions between a custom-designed Cys 2 -His 2 zincfinger protein and the cleavage domain of the FokI restriction endonuclease. Cys 2 -His 2 ZFPs bind DNA by inserting an α-helix into the major groove of the double helix (Wu et al., 2007) ZFNs function as dimers, with each monomer recognizing a specific "half site" sequence, typically nine to 18 base pairs of DNA, via the zinc-finger DNA-binding domain. One major concern associated with the use of ZFNs for genome editing is offtarget mutations (Chen et al., 2014). As a result, several approaches have been undertaken to enhance their specificity; among the most successful of these has been the creation of obligate heterodimeric ZFN architectures that rely on charge-charge repulsion to prevent unwanted homodimerization of the FokI cleavage domain, thereby minimizing the potential for ZFNs to dimerize at off-target sites. Additionally, protein-engineering methods have been used to enhance the cleavage efficiency of the FokI cleavage domain (Chen et al., 2014). The main hurdle in using ZFNs was the 3-base pair site on DNA requirement that made the design more challenging (Gupta and Musunuru, 2014). The upper hand was the guanine-rich target sites that appeared to be more efficient at editing when compared to the non-guaninerich sites. Furthermore, the ZFN interaction with DNA is modular, and each ZF interacts with DNA independently that hampered the editing efficiency. Therefore, scientists needed to address these issues if they wanted to have more efficiently edited genome. According to a study which adopted the ZFN approach for genome editing in human pluripotent stem cells, the scientists observed multiple off-target genomic sites based on highsequence similarity to the on-target site and found a single off-target mutation in the 184 clones assessed (Hockemeyer et al., 2009). Therefore, scientists should be aware of the likelihood that ZFNs that are designed for a purpose may experience undesired off-target effects at a low rate. There are ways to reduce off-target effects, by using a pair of ZFNs that have different FokI domains that are obligate heterodimers. Through this approach, we will be able to prevent a single ZFN from binding to two adjacent off-target sites and in turn generating a DSB. The second approach would be the introduction of purified ZFN proteins into cells. Although it is efficient at inducing DNA edits, ZFNs are cumbersome and laborious to assemble. Currently, the ZFN approach is still at the early stage and there are some difficulties that need to be addressed and sorted off before it is clinically used to treat human diseases. ZFN-based strategies for gene editing of human cells may provide a viable option to treat human disease in the future.


## TALENs (TALE Nucleases)

Transcription activator-like effector nuclease (TALEN) has rapidly emerged as an alternative to ZFNs for genome editing, and they are bacterial effector proteins. In 2009, the code used by TALE proteins to recognize DNA was uncovered. TALE DNA-binding domains can be constructed using a variety of methods, with the most straightforward approach being the Golden Gate assembly (Maeder et al., 2013). TALENs comprise a non-specific DNA cleavage domain fused to a customizable sequence-specific DNA-binding domain to generate DSBs. This DNA-binding domain consists of a highly conserved repeat sequence from transcription activator-like effector (TALE), which is a protein originally discovered in the phytopathogenic Xanthomonas bacteria that naturally alters the transcription of genes in host plant cells. The binding of TALE to DNA is mediated by a central region that contains an array of 33 to 35 amino acid sequence motifs. The amino acid sequence of each repeat is structurally similar, except for two hypervariable amino acids (the repeat variable di-residues or RVDs) at positions 12 and 13. DNA-binding specificity is determined by RVDs, with ND specifically binding to C nucleotides, HN to A or G nucleotides, NH to G nucleotides, and NP to all nucleotides. There is a one-to-one correspondence between RVDs and contiguous nucleotides in the target site, constituting a strikingly simple TALE-DNA recognition cipher.

Functional endonuclease FokI is factitiously fused to DNAbinding domains to create site-specific DSBs and thereby stimulate DNA recombination to achieve TALEN-induced targeted genomic modification. To cleave the two strands of the targeted DNA, the FokI cleavage domain must be dimerized. Hence, like zinc fingers, such a TALEN module is designed in pairs to bind opposing DNA target loci, with proper spacing (12-30 bp) between the two binding sites. However, compared to zinc-finger proteins, there is no need to redesign the linkage between repeats constituting long arrays of TALEs, whose function is to target individual genomic sites. Following pioneering studies on zinc-finger proteins, multiple effector domains have become accessible to support the fusion of TALE repeats for different genomic modification purposes, including nucleases, transcriptional activators, and site-specific recombinases. Although their simpler cipher codes provide better simplicity in design than triplet-confined zinc-finger proteins, one of the primary technical hurdles for cloning repeat TALE arrays is the design of identical repeat sequences on a large scale. To address this limitation, a few strategies have been established to facilitate the fast assembly of custom TALE arrays, including "Golden Gate" molecular cloning, high-throughput solid phase assembly, and connection-independent cloning techniques. More recent advances in TALEN assembly, though, have focused on the development of methods that can enhance their performance, including specificity profiling to uncover non-conventional RVDs that improve TALEN activity, directed evolution as means to refine TALE specificity, and even fusing TALE domains to homing endonuclease variants to generate chimeric nucleases with extended targeting specificity (Liu et al., 2014). TALENs attach FokI to arrays of DNA-binding modules, originally from plant pathogens, that each targets a single base pair. TALENs are smaller than Cas9 but larger than ZFNs. The modules have high DNA-binding affinity but include repeated sequences that create cloning challenges. According to a study conducted by Bethany K. R. and Randall S. P. in 2015, they have stated that TALENs are not efficient at making biallelic modifications, and it requires further cloning steps or alternative animal breeding step to produce animals with the intended biallelic mutations.


## CRISPR

Although recently developed programmable editing tools such as zinc-finger nucleases and transcription activator-like effector nucleases have significantly improved the capacity for precise genome modification, these techniques have limitations. The new kid on the block, CRISPR (clustered regularly interspaced short palindromic repeats) technology represents a significant improvement over these, reaching a new level of targeting, efficiency, and ease of use. CRISPR nucleases lead the geneediting platform because they are the most powerful and direct to use gene-editing tools available now. Based on their cas gene content, CRISPR-Cas systems can be classified into six major types (I-VI) (Koonin et al., 2017). The systems are classified into two general parts, namely, clustered regularly interspaced short palindromic repeats (CRISPR) arrays and CRISPR-associated (Cas) proteins. The CRISPR immunological signature represents the memory of the previous infections encoded within individual spacers separated by conserved repeats. Cas proteins carry out the adaptive immune functions and are highly diverse, resulting in CRISPR-Cas systems currently being grouped into two classes, six types, and over 30 subtypes (Wimmer and Beisel, 2020). The speedy unearthing and continuous development has led to the availability of choices; hence, choosing the right nuclease for an experiment is a mandatory part of this approach. The CRISPR system allows for site-specific genomic targeting in virtually any organism (Guilinger et al., 2014). It also offers platform that is an efficient way of making precise genetic changes to the human genome. This can be employed for disruption, addition, and correction of genes, thereby enabling a new class of genetic therapies that can be applied to hematological disorders.


## Cas 9 System

The CRISPR-Cas9 system, which has a role in adaptive immunity in bacteria, is the most recent addition to the genome-editing toolbox. The primal CRISPR gene editor is from S. pyogenes Cas9, and it still remains to be the most popular CRISPR nuclease for the gene editing approach. The SpCas9 is a colossal CRISPR nuclease bearing 1,368 amino acids in size and possesses a moderately relaxed protospacer adjacent motif (PAM; 5′-NGG) as compared to its orthologs. SpCas9 happens to be the most well-characterized of all CRISPR systems due to it being the first to be repurposed as a gene editor. Thus, making it readily available in virtually any format (plasmid, mRNA, and protein) and in-line with other genome engineering applications. In bacteria, the type-II CRISPR system provides protection against DNA from invading viruses and plasmids via RNAguided DNA cleavage by Cas proteins (Guilinger et al., 2014). Short segments of foreign DNA are integrated within the CRISPR locus and transcribed into CRISPR RNA (crRNA), which then anneal to trans-activating crRNA (tracrRNA) to direct sequencespecific degradation of pathogenic DNA by the Cas9 protein. This system has since been simplified for genome engineering and now consists of only the Cas9 nuclease and a single guide RNA (gRNA) that contains the essential crRNA and tracrRNA elements (Maeder et al., 2013).

There are many naturally-occuring Cas9 ortologs available for gene-editing purposes. There are three types of SpCas9, namely, SpCas9 with relaxed PAM (SpCas9-NG), SpCas9 nickase (Cas9n), and nuclease-dead SpCas9 (dCas9). In order to differentiate these ortologs from SpCas9, there are two main characteristics which are size and PAM specificity. The size really matters for hard-to-transfect cell lines which are vectors that easily exceed 10 kb as well as AAV applications where AAV can only package ∼4.5 kb. Compact Cas9 ortologs include those isolated from S. aureus (SaCas9) and C. jejuni (CjCas9), both of which can be packaged in AAV. SpCas9-NG is engineered with valine to disrupt the interaction between Cas9 and the second G of the 5′-NGG PAM. There is additional modification done to this SpCas9-NG based on molecular modeling to restore the cleavage activity of this "relaxed" Cas9 variant (Nishimasu et al., 2018). The second type, SpCas9 nickase, is engineered with a single alanine substitution that transforms Cas9 into a "nickase" that can only cut one strand of target DNA. In order to create a double-stranded break in gene editing, a pair of Cas9 nickase is used as it gives fewer off-target DSBs than SpCas9 (Tsai and Joung, 2016). The dCas9 has two alanine substitutions to eliminate Cas9's nuclease activity but its DNA targeting ability is not affected. This synthetic construct that is generated by tethering transcriptional activation domains to a dCas9 can be directed to the promoters of endogenous target genes by single guide RNAs (sgRNAs) to activate transcription. It can also be used to target various effector domains can be fused to dCas9 to enable programmable transcriptional and epigenetic control (Li et al., 2019).

PAM specificity, on the other hand, dictates the available 'target space' for each ortolog or how many guide RNA (gRNA) binding sites there are in a given genome for a given PAM. Ortologs with simple PAMs can interact with more genomics sites compared to those with stringent PAMs. For example, there are more instances of 5′-NGG in any given genome as opposed to 5′-NGGNG which is the PAM site for St3Cas9 (Streptococcus thermophilus). Relaxed PAMs are good for more flexibility in the gRNA design, but it should be avoided because it may lead to off-target activity. When the condition allows for increased gRNA-binding sites, it will lead to higher potential off-target activity. However, like any emerging technology, the CRISPR-based assay system has minor potential pitfalls, including promiscuous off-target activity by Cas9. To address this issue, various preventative strategies have been employed, such as introducing purified Cas9 straight into target cells, using Cas9 nickase (Cas9n), decreasing sgRNA sequences by 2-3 nt, and exploiting additional two guanines at the 5ˊterminus of gRNA immediately juxtaposed to the targetcomplementary region (Concordet and Giovannangeli, 2017).


## Cas10 System

Classification of CRISPR-Cas10 variants that would capture their evolutionary relationships to the maximum possible extent is essential for comparative genomic and functional characterization of this theoretically and practically important system of adaptive immunity (Chen et al., 2014). The Cas10 enzyme is classified into Class 1 CRISPR-Cas systems which consist of the three types, type I, III, and IV systems as well as 22 subtypes. Along with Cas10 in Class 1, it also includes Cas3 and Cas8-like (csf1) enzyme which are considered signature genes in types I, III, and IV, respectively (Vangah et al., 2020). Type III systems, which are identified by their signature Cas10 gene, are further divided into subtypes: III-A/D, which encodes the Cas10-Csm complex, and III-B/C, which encodes the Cas10-Cmr complex. Type III effector complexes employ a uniquely elaborate targeting mechanism (Pyenson et al., 2017;Tamulaitis et al., 2017) in which active transcription of the target sequence is required for CRISPR immunity (Deng et al., 2013). The large subunit of type III systems, Cas10, is a protein containing two RNA recognition motif domains, whereby one domain shows a high significant similarity to the palm domain, the catalytic domain of a broad variety of RNA and DNA polymerases and nucleotide cyclases. This palm domain is predicted to be an active enzyme, whereas the second RRM domain is inactivated form (Koonin et al., 2017).


## Cas13 System

DNA targeting CRISPR enzymes, such as Cas9 and Cas12a, have enabled many new possibilities for manipulating and studying DNA. Recent computational efforts to identify new CRISPR systems uncovered a novel type of RNA targeting enzyme, Cas13. The diverse Cas13 family contains at least four known subtypes, including Cas13a, Cas13b, Cas13c, and Cas13d (Elliott et al., 2021). One of the most straightforward applications of Cas13 in vivo is targeted RNA knockdown using mammalian codon optimized Cas13 and guide expression vectors. Knockdown of RNA relies on cleavage of the targeted transcripts by the endogenous RNase activity of the dual HEPN domains of the protein, the efficiency of which varies between different orthologs and subtypes of Cas13. The Cas13a is an RNA-guided, single-component enzyme that possesses two higher eukaryotes and prokaryotes nucleotide-binding (HEPN) domains that target single-stranded RNA (ssRNA). It encompasses a functionally distinct nuclease that is responsible for catalyzing crRNA maturation to form a Cas13a:crRNA complex that is competent for target RNA binding. The binding of the said complex to a complementary ssRNA is termed as activator-RNA. This binding activates the HEPN-nuclease for both target and general ssRNase activity. The action of the HEPN-nuclease is repressed by fractional occlusion of the HEPN active site until binding to the activator-RNA occurs. This effectively makes the activator-RNA the allosteric switch for RNase activity (Narimani et al., 2019). While the mechanism appears conserved between homologs, Cas13a enzymes can be functionally separated into two distinct subtypes based on their processing activity and HEPN-nuclease nucleotide preference . As a result, guide design and restrictions on targeting depend on the system used . In a study conducted by Cox et al. (2017), they expanded the RNA targeting ability of Cas13 to direct ADAR2 deaminase for RNA base (adenosine to inosine) modifications in human cells to recover functional proteins and halt disease progression. The application of Cas13a for nucleic acid detection and targeting are active Cas13a HEPN nuclease, which will turn over multiple ssRNA substrates, a biochemical behavior that can be leveraged for signal amplification in target detection by coupling Cas13a activation to specific ssRNA reporter cleavage, resulting in liberation of a quenched fluorophore. Expanding on this, the specific high-sensitivity enzymatic reporter unlocking (SHERLOCK) platform was developed as a tool for nucleic acid detection (Narimani et al., 2019). The method starts with RNA sequence amplification via RPA or reverse transcriptase RPA (rt-RPA), before incubating the sample with Cas13a and reporter probes, and fluorescence is then measured. SHERLOCK can detect nucleic acids in patient biofluids down to low atto molar concentrations, allowing the detection of tumor mutation in cell-free DNA (cfDNA). In mock cfDNA samples, SHERLOCK can detect two cancer mutations, EGFR L858R and BRAF V600E, under low allelic fraction with single-base mismatch sensitivity. In addition to in vitro RNA target detection, catalytically inactive Cas13a retains its RNA-binding activity such that it can be coupled to a fluorescent probe to enable live cell RNA tracking. This provides an alternative method to recognize and visualize RNA (Pandolfi, 2001).


## CRISPRainbow System

In the past, there was a vacuum in studying and imaging multiple genomic loci in living cells at a time. This created a limitation for us in exploring the biological chromodynamics or chromosome dynamic. CRISPRainbow is a system for labeling DNA in human cells based on nuclease-dead (d) Cas9 combined with engineered single guide RNA (sgRNA) scaffolds that bind to fluorescent proteins. In this applied science technique, it allows simultaneous imaging of up to seven chromosomal loci in an individual human cell and observes large differences in the chromodynamic properties of different chromosomal loci at a time. This system is a valuable tool for studying the transfiguration of the genome in real time basis, and it is used to label the DNA and then track the movement of DNA in live cells (Ma et al., 2016).


## Summary on Gene Editing Methods

In summary, the CRISPR system allows for site-specific genomic targeting in virtually any organism (Guilinger et al., 2014). It also offers platform that is an efficient way of making precise genetic changes to the human genome. This can be employed for disruption, addition, and correction of genes. Thus, the remaining parts of this review will focus on the utilization of CRISPR tools for understanding the functional genomics of various hematological malignancies, its challenges, as well as the clinical implications.


## APPLICATION OF VARIOUS CRISPR SYSTEM IN STUDYING THE FUNCTIONAL GENOMICS OF AML

AML is a multifaceted genetic disease caused by an interwoven process in which numerous collaborating mutations that allow a cell to evade the checkpoints that normally safeguard it against DNA damage and other disruptions of healthy cell growth (Babaei et al., 2013). It is a type of blood cancer which causes excessive proliferation of abnormal immature leukemic cells known as blasts. Uncontrolled growth occurs through mutations in the FLT3, NPM1, CEBPA, RAS, and c-KIT genes among others. A key gene involved in AML pathogenesis is the FMS-like tyrosine kinase 3, a receptor-type tyrosine-protein kinase (FLT3), positioned at chromosome 13q12 encoding a class III receptor tyrosine kinase that regulates hematopoiesis. This receptor is activated by the binding of the related tyrosine kinase three ligand (FL) to its extracellular domain which induces homodimer formation in the plasma membrane and autophosphorylation (Rivera-Torres et al., 2020). The activated receptor kinase phosphorylates multiple cytoplasmic effector molecules in cellular pathways involved in apoptosis, proliferation, and differentiation of hematopoietic cells. Protein kinase activation can be induced by somatic mutations, a common mechanism of tumorigenesis led by the constitutive activation of the receptor resulting in acute myeloid leukemia and acute lymphoblastic leukemia. Advancement in the treatment of leukemia over the last 40 years has impacted and transformed a uniformly fatal disease into one that is somewhat manageable. There are, however, several subtypes of pediatric and adult leukemia that evade treatment and continue poor prognosis; many of these involve FLT3 mutations. For example, the FLT3 ITD associated with a single point mutation in the tyrosine kinase domain is known to induce resistance to tyrosine kinase inhibitors (TKI) treatment (Rivera-Torres et al., 2020).

Many common variants have been associated with hematological traits, but identification of causal genes and pathways have proven to be very challenging. Since acute myeloid leukemia (AML) is a malignant, cancerous development in hematopoietic stem cell lines attributed by several gene mutations, CRISPR serves as an excellent tool to study the underlying molecular and genetic reasons for AML progression. Researchers such as (Rivera-Torres et al., 2020) utilized CRISPR-based site-directed mutation termed as CRISPR-directed mutagenesis (CDM) in deriving plasmid vectors capable of expressing and recapitulating FLT3 sequences mutations as seen in AML patients. The system enabled the in vitro study of both the phenotypical cellular changes induced by such FLT3 mutations and the different levels of response to anti-AML drugs, including the potential for discovering new therapeutic targets for AML treatments. A number of researchers have utilized CRISPR systems to investigate the genetic targets behind differentiation blockages commonly seen in causing the development of AML. For instance, Wang et al., identified RNA-binding protein ZFP36L2 as a crucial mechanistic regulator of cell differentiation in AML using cell surface antigen-guided-CRISPR systems (Li et al., 2019;Wang et al., 2021a). Genetic suppression of ZFP36L2 enables myeloid leukemic cells to transit out from an undifferentiated state and resume cellular differentiations. Brabetz et al., gained insights into AML progressions by using CRISPR to functionally study and "edit" the IDH2 genes to both create and correctively repair the mutation, IDH2 R140Q, frequently spotted in AML and are known to impede cellular differentiation and self-repair mechanisms (Brabetz et al., 2017).

Authors such as Ho et al., applied the gene-editing process of CRISPR/Cas systems in knocking out interleukin-1 receptor accessory protein (IL1RAP) genes in human leukemia stem cells in an attempt to abolish relapses of leukemia in AML patients by designing a bone marrow scaffold-mediated delivery of the therapeutic CRISPR system (Ho et al., 2021). Certain studies have utilized CRISPR-mediated gene knockouts in identifying the significant role of KAT7 genes for the survival of AML cells (Au et al., 2021). Tiansu and coworkers attempted to study the development of AML by carrying out a sequential CRISPR/Cas9 gene editing approach involving step-wise introductions of different mutations in induced pluripotent stem cells (iPSC). The technique offered better comprehension of the evolution of AML conditions, highlighted the corresponding changes at transcriptome and chromosome levels between the different stages of diseases progression as well as illuminated potential early disease markers to improve AML prognosis (Wang et al., 2021b).

Combining large-scale or genome-wide genetic analysis with CRISPR and gene targeting yields screening results several folds more accurate with greater gene knockout efficiency and minimal off-target sequence hybridizations. Authors such as Tzelepis et al., incorporated genome-wide CRISPR screening in distinguishing more than 100 essential AML-gene triggers, among which KAT2A was chosen as a promising therapeutic target against AML (Tzelepis et al., 2016). The results revealed that inhibition of KAT2A genes consequently leads to myeloid differentiation and apoptosis, demonstrating its potent anti-AML properties. Wallace et al., identified essential microRNAs which regulate the growth and onset of AML such as miR-155 and miR-150 using large-scale gene knockouts via CRISPR/Cas gene editing (Wallace et al., 2016a). Likewise, Lin et al., also employed CRISPR-mediated wide-scale screenings to analyze the genetic dependencies of AML in orthotopic xenograft models, in which several vital AML-linked genetic vulnerabilities were identified such as the SLC5A3 and MARCH5 genes . In vitro CRISPR screens were also utilized to identify genes which sensitizes AML cells to double-negative T cells (DNTs) therapies for effective patient treatments (Soares et al., 2021). Moreover, researchers have also studied the genetic mechanisms that undermine the effectiveness of BCL2 inhibitors in AML treatments using CRISPR-based genome-wide screening (Romine et al., 2021). CRISPR gene knockout screens were also performed to recognize mitochondrial genes necessary for the growth of AML cells. Mitochondrial carrier homolog 2 (MTCH2) was identified as a potent regulator for AML in histone acetylation and subsequent cell differentiation (Khan et al., 2020).

An attempt to alter the abnormal gene function in AMLrelated disease led researchers to explore therapies that possibly ablate leukemia stem cells (LSCs), thus increasing the chances of eliminating this cancer in patients. A latest study conducted by Tzu-Chieh Ho et al., showcased scaffold-mediated CRISPR-Cas9 delivery system for targeting the gene interleukin-1 receptor accessory protein (IL1RAP) in human LSCs (Ho et al., 2021). This approach was mediated using a bio-reducible lipidoidencapsulated Cas9/single guide RNA (sgRNA) ribonucleoprotein-induced efficient gene editing. The CRISPR-Cas9 system was an effective strategy for attenuating LSC growth to improve AML therapy. In another approach, a group of researchers from Icahn School of Medicine, Mount Sinia, US have used CRISPR technology in developing a model to study AML at different stages of metastasis. This has paved a way for the group to identify possible biomarkers for early-stage leukemia detection. Hence, CRISPR has not only open the door toward treatment but has also given us an avenue to have huge amount of information pertaining to a disease, in this case it is AML. The information also unraveled other details pertaining to blood cancer myelodysplastic syndrome and clonal hematopoiesis which is often a pre-leukemic condition (Rees, 2021). This has given us a chance to study the molecular mechanisms that underlie the progression of the disease by tracking the evolution of human leukemia. In an another study done by a group of researchers, they have used CRISPR to unveil a new gene that is involved in the regulation of molecular mechanism of leukemia stem cells, the cells responsible for propagating the disease and for therapy resistance. Stau2 has been previously studied in the brain and the nervous system but until now was not known to have a role in cancer and other hematological malignancies (Bajaj et al., 2020).

Acute myeloid leukemia (AML) remains a devastating disease in need of new therapies to improve patient survival. Targeted adoptive T-cell therapies have achieved impressive clinical outcomes in some B-cell leukemia and lymphomas but not in AML. Double-negative T cells (DNTs) effectively kill blast cells from the majority of AML patients and are now being tested in clinical trials. However, AML blasts obtained from ∼30% of patients show resistance to DNT-mediated cytotoxicity; the markers or mechanisms underlying this resistance have not been elucidated. Here, a targeted clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPRassociated protein 9 (Cas9) screen was utilized to identify genes that caused susceptibility of AML cells to DNT therapy. Inactivation of the Spt-Ada-Gcn5-acetyltransferase (SAGA) deubiquitinating complex components sensitized AML cells to DNT-mediated cytotoxicity. In contrast, CD64 inactivation resulted in resistance to DNT-mediated cytotoxicity. Importantly, the level of CD64 expression correlated strongly with the sensitivity of AML cells to DNT treatment. Furthermore, the ectopic expression of CD64 overcame AML resistance to DNTs in vitro and in vivo. Altogether, the data demonstrated the utility of CRISPR/Cas9 screens to uncover mechanisms underlying the sensitivity to DNT therapy and suggests CD64 as a predictive marker for response in AML patients (Hsu et al., 2013).


## APPLICATION OF VARIOUS CRISPR SYSTEMS IN STUDYING THE FUNCTIONAL GENOMICS OF ALL

ALL can be characterized by looking at the development of precursor lymphoid into B or T cell. This type of leukemia is usually rare but requires immediate treatment as ALL leukemogenesis is rapid and aggressive (Hsu et al., 2013). Accumulation of lymphoid blast could be observed in patient bearing this disease which is usually translated by enlarged lymph nodes (Kebriaei et al., 2002;Inaba et al., 2013;Litzow and Ferrando, 2015).

The CRISPR/Cas9 knockout system was utilized in the manipulation of ALL's genomics and in studying the differentiation of ALL into erythrocytes. A study conducted by Xie and group reported that GATA-1 motif flanking 5 kb upstream ARHGEF12 was impaired due to the CC to TT SNPs in hematopoietic cells of ALL patients suggesting that GATA-1 transcriptional activation of ARHGEF12 is crucial in driving erythrogenesis in which ALL patients lacked. The luciferase reporter assay confirmed GATA-1 interaction onto the ARHGEF12 flanking region when compared to the impaired motif brought about by the CC to TT SNPs. To further ascertain that this switch was responsible in driving erythropoiesis, the group employed CRISPR/Cas9 knockout where guide RNA was designed to be specifically targeting the ARHGEF12 gene directly. Result showed a stunted production of red blood cells and immature erythrocytes accumulation (Terwilliger and Abdul-Hay, 2017).

Furthermore, blockade of transcriptional machinery via the use of CRISPR system was also demonstrated when the MYBbinding site located proximally upstream of LMO2 was targeted using Cas9. In this study, a second binding motif of MYB was found to occur on LMO proximal site, this occurs due to the translocation event of the LMO2 gene which shift the MYBbinding site to a second MYB-binding site thus duplicating the MYB motif resulting in an increase in LMO2 activation by MYB. Interesting enough, Cas9 cleavage of the duplicated MYB motif resulted in a significant reduction in LMO2 expression .

ETV6/RUNX1 fusion oncogene commonly found in ALL was also successfully targeted using CRISPR/Cas9. In this case, guide RNA was designed to target the specific ETV6 and RUNX1 fusion region. Subsequent qPCR reported a reduction in the mRNA of ETV6/RUNX1 fusion construct following CRISPR/ Cas9-mediated knockout of the fusion region. Furthermore, functional studies of the ETV6/RUNX1 abrogated REH cells, and ALL cell line showed a reduction in proliferative potential which also translates to a reduction in chemoresistance where REH cells were shown to be more apoptotic following chemodrug treatment (Rahman et al., 2017). Moreover, it was also reported that the BCR-ABL fusion gene was successfully targeted in Philadelphia positive ALL cells. Small guide RNA was designed to target a specific region of the fusion protein; this group used the nickase system of CRISPR/Cas9 where cleavage occur between the exon-exon junction, and this resulted in total abolishment of the fusion protein trailed by rapid apoptosis of ALL cells (Montaño et al., 2020). Derivative of the Cas9 knockout system, the dCas9 repressor, was also used in studying the leukemogenesis of ALL. A study sought to look into the function of ARID5B in ALL resistance toward treatment. The dCas9-KRAB repressor system was used to repress ARID5B, and the knockdown showed a significant increase in IC50 following MTX (chemodrug) treatment, suggesting that ARID5B could probably be downregulated in ALL for chemoresistance. This was interesting as patients with relapse showed significant downregulation of ARID5B (Tan et al., 2020). This study reflects the usage of the CRISPR system in studying the functional genomics of disease progression.

Additionally, ALL metabolism dysregulation was also studied in further understanding the mechanism of ALL leukemogenesis, and it was reported that the PAX5 transcription activator was deactivated in which this trailed increase in glucose uptake of ALL thus supporting ALL development. Using the CRISPR activation system (VP64), the same group induced PAX5 expression which in turn would activate PAX5 repressed gene; this attenuated glucose uptake thus restoring the gate keeping function of PAX5 as a metabolic switch .


## APPLICATION OF VARIOUS CRISPR SYSTEMS IN FUNCTIONAL GENOMICS OF CML

This type of leukemia can be characterized by looking at accumulation of granulocyte within the bone marrow. Trailing myeloid precursor aberration neutrophilic and basophilic cell accumulation could also be observed within the peripheral blood. Synonymous to CML is the BCR-ABL fusion oncoprotein resulting from the Philadelphia chromosome, which is a direct result of translocation between chromosome 9 and 22 (Trela et al., 2014;Chan et al., 2017;Chopade and Akard, 2018;Hsieh et al., 2021).

The use of CRISPR in CML is mainly on aberating the BCR-ABL fusion tyrosine kinase. BCR-ABL fusion resulted in a surface protein with constitutive tyrosine kinase activity which is one of the main factors in CML leukemogenesis (Russo et al., 2020;O'Dwyer et al., 2002). Although imatinib mesylate (IM), a first of its kind tyrosine kinase inhibitor, proved to be effective in abrogating CML, patients still suffer relapse prompting scientist to directly target the fusion construct in treatment of CML (Guilhot, 2004;Barnes et al., 2005;Chan et al., 2017).

Frontiers in Genetics | www.frontiersin.org January 2022 | Volume 12 | Article 767298

An interesting study sought to abolish the region of fusion by specifically targeting the ABL exon 6. In this case, the CRISPR knockout system was used where sgRNA were designed to nick a region of about 100 bp on ABL locus within the genome. The CRISPR nickase system was proven to be effective as expression of the BCR-ABL mRNA reduced to only 2.6%, with a total abolishment of protein translation. This also leads to an increase in apoptosis when compared to wild type CML cells. Furthermore, the group also targeted leukemic stem cells (LSCs) using the same system and found out that mice with LSCs-CRISPR-containing engraftment showed a reduction in CML oncogenic properties and restoration of multipotency of LSC with unbiased development toward only myeloid lineage (Sharma et al., 2010).

Recently, the CRISPR/Cas9 system was fused with an E. coli exonuclease 3; this resulted in an efficient cleavage of large genomic locus which is more efficient than CRISPR nickase. In CML, this technology was used to target the BCR-ABL gene where an efficient deletion of the target site was observed, and this was also followed by an increase in apoptosis in CRISPR-EXO3-targeted CML cell line (Vuelta et al., 2020). The same group also used the technology to target BCR-ABL in vivo where animal models with CRSPR-EXO electroporation showed an increase in survivability and tumor size reduction (Vuelta et al., 2020).

Another example on the use of the CRISPR system in CML can be seen in a study by Cheng and colleagues where Cas9 was designed to target specifically MBD2. MBD2 protein associates with a highly methylated region resulting in gene suppression of crucial factors in proper cellular development. This protein was found to be upregulated in the CML crisis phase. Targeting the MBD2 gene via Cas9 resulted in a significant ablation in proliferative potential of CML cell lines (Cheng et al., 2018). Furthermore, in vivo mice model also showed reduction in tumor volume; thus, this study proved the applicability of Cas9 in targeting methylation-associated factors in leukemic aberration (Lainšček et al., 2020).

Recently, the CRISPR/Cas9 system has been used to correct the ASXL1 homozygous nonsense mutations that are present in the CML cell line KBM5, which lacks ASXL1 protein expression. This resulted in protein re-expression with normal functionality restored including downregulation of Polycomb repressive complex two target genes (Kim et al., 2014).

The CRISPR/Cas9 system was also used to directly target miRNA locus in CML. Recent research by Arya et al. showed that CRISPR knockout of miR182-5p sensitized CML cells toward TKI treatment. miR182-5p was found to be highly expressed in resistant CML and that it was also associated with improper blast differentiation. Cas9-mediated knockout of miR182-5p relieved leukemic properties allowing for CML to be effectively treated (Arya et al., 2018).


## APPLICATION OF VARIOUS CRISPR SYSTEMS IN FUNCTIONAL GENOMICS OF CLL

Chronic lymphocytic leukemia is a type of malignancy involving the accumulation of CD5 + B cell within the brain, marrow, and lymphoid tissue. This type of leukemia was found to be one of the most occurring type of leukemia, and depending on the degree of mutation-driven proliferation, treatment of this disease varies in which in some patients treatment required is minimal, while in others immediate treatment is required, which reflect the heterogeneity of this disease (Julio Delgado et al., 2020;Milne et al., 2020). Mutation that are usually looked into to determine the degree of the disease is the IGHV mutation, which could form fusion with the BCL family due to translocation, and the use of the B-cell receptor as a target, therefore, favor CLL management (Kipps et al., 2017).

The use of CRISPR in CLL recently focused on driver mutation contributing to CLL leukemogenesis. One of the gene of interest looked upon is the NOTCH1 gene as it was found to affect CLL homing into the spleen and the brain, which is a major contributor to bad prognosis when the gene is mutated. Using CRISPR/Cas9 targeting, the NOTCH1-PEST protein domain in Mec-1 a CLL cell line showed low levels of DUSP22 expression which significantly correlates with the spleen and the brain homing suggesting that NOTCH1 mutation occurs specifically on the PEST domain which could act as a major driver in CLL leukemogenesis with bad prognosis (Arruga et al., 2017).

In a more recent study, NOTCH1 dysregulation accompanying FBXW7 mutation in CLL was found to be more prominent in patients with worse outcome. FBXW7 is a negative switch of NOTCH1 where physical interaction between these two would deactivate NOTCH1. Furthermore, the group also found that mutation in the WD40 domain of the FBXW7 is common in CLL patients. The use of Cas9 to abrogate the WD40 domain further support that this mutation is a contributing driver in CLL progression into worse prognosis as an increase in NOTCH1 activation was observed despite physical interaction (Close et al., 2019).

In order to further understand CLL progression, a group chose to look into specific CLL mutation, namely, CLL with del (11q) with either TP53 or ATM mutation. Using CRISPR/Cas9, the group induced deletion of the 11q arm and designed small guide RNA against TP53 and ATM. Results showed that del (11q) with TP53 mutation confer leukemic survivability suggesting that these two mutational events co-occurrence potentiate CLL progression. However biallelic mutation of ATM and TP53 result in reduction in CLL engraftment in vivo suggesting that mutual targeting of both ATM and TP53 is favorable in aberrating CLL. Further study also confirms the reliability of this method as TP53 and ATM mutation never co-occurred in patient with CLL (Quijada-Álamo et al., 2021a). Targeting of CLL with del (11q) and ATM mutation was also proven to be more effective with BCR or PARP inhibitors, thus further supporting that direct targeting of both TP53 and ATM is favorable in CLL aberration (Quijada-Álamo et al., 2020).


## APPLICATION OF VARIOUS CRISPR SYSTEMS IN FUNCTIONAL GENOMICS OF OTHER HEMATOLOGICAL MALIGNANCIES

CRISPR has also been utilized to study the functional genomics of other types of hematological malignancies other than AML, ALL, Frontiers in Genetics | www.frontiersin.org January 2022 | Volume 12 | Article 767298 8 CML, and CLL. Other common hematological malignancies are multiple myeloma, Hodgkin, and non-Hodgkin lymphomas. Researchers usually conduct whole genome sequencing (WGS) to identify crucial genes. Next, they utilized CRISPR to screen the genes identified with WGS to uncover the function of the genes and how it contributes to the malignancies. CRISPR is also used to uncover how the malignancies adapt and develop resistance to cancer treatments. It is hoped that by studying the functional genomics, new treatment targets against these malignancies can be identified and new treatment options can be developed, tabulated in Table.1.

DLBCL is the most common non-Hodgkin lymphoma worldwide and is a fast growing cancer which affects the B lymphocytes (Diffuse Large B-Cell Lymphoma, 2021). DLBCL have been categorized into three subgroups: activated B cell-like [ABC], germinal-center B cell-like [GCB], and unclassified based on their gene expression profile (Chapuy et al., 2018). Large scale genetic profiling studies have also shown that there are many recurrent altered genes in DLBCL (Reddy et al., 2017;Schmitz et al., 2018); however, the function of the genes and its contribution to lymphomagenesis is not well understood. Therefore, CRISPR has been utilized in multiple studies to better understand how the genetic alterations interact and leads to tumorigenesis. In a study by Reddy et al. the researchers utilized CRISPR to identify crucial oncogenes in DLBCL and then compared it to the expression profile of 1001 DLBCL patients. Based on the analysis, they determined a list of crucial oncogenes that act as genetic drivers for DLBCL (Reddy et al., 2017). In the study, 35 driver genes were identified as functional oncogenes as their knockout with CRISPR resulted in reduced viability of DLBCL cells. Nine out of the genes were then found to be important in specific subtypes. In the ABC DLBCL subtype, EBF1, IRF4, CARD11, MYD88, and IKBKB knockout produced significant reduction in cell viability, whereas in GCB DLBCL, the knockout of ZBTB7A, XPO1, TGFBR2, and PTPN6 was more selectively lethal to the subtype. Based on the results obtained, the researchers postulate that these genes can be utilized to classify the different subtypes of DLBCL and also be specific treatment targets for future treatment of different subtypes of DLBCL. In another recent study by Nie et al. the researchers conducted genome-wide CRISPR screens on the DLBCL cell line: RC-K8 and discovered the synthetic lethal interaction between CREBBP and EP300 genes (Nie et al., 2021). In this study, a genome-wide CRISPR-Cas9 loss-of-function screening determined CREBBP and EP300 as crucial genes. It was determined that DLBCL cells that have deficiency in both CREBBP and EP300 are sensitive to histone acetyltransferases inhibition, and this can be utilized as a treatment target in the future.

Phelan et al. tried to determine the cause of ibrutinib sensitivity in certain DLBCL cell lines. Based on their genomewide loss-of-function CRISPR-Cas9 screens on three ibrutinibsensitive ABC cell lines, one ibrutinib-insensitive ABC cell line, and four ibrutinib-insensitive GCB cell lines, they uncovered a new B-cell receptor signaling in ibrutinib-responsive cell lines which is coordinated by a multiprotein supercomplex formed by MYD88, TLR9, and the BCR (My-T-BCR supercomplex) . There are also in vivo researches utilizing CRISPR to elucidate the function of genes in B-cell lymphomas. Felce et al. conducted research on murine A20 lymphoma cell line which is B-cell lymphoma derived from mice as a model for B-cell lymphoma. The researchers utilized CRISPR to knockdown the Foxp1 expression in the cells which upregulated cell surface I-Ab (MHC-II) expression without impairing cell viability in vitro. The reduction of Foxp1 restores immune surveillance. This shows that Foxp1 helps the lymphoma cells evade the immune system. Based on the results, it is hoped that targeting Foxp1 in B-cell lymphomas will enhance the effects of other immunotherapies. By utilizing CRISPR in tandem with other molecular techniques, researchers were able to detect and determine the function of genes crucial in DLBCL (Felce et al., 2020).

Burkitt's lymphoma (BL) is another type of non-Hodgkin lymphoma. It is divided into three subtypes which are sporadic, endemic, and immunodeficiency associated (Lenze et al., 2011). Panea et al. conducted whole genome sequencing (WGS) and transcriptome sequencing on 101 patient samples to identify the genomic basis for Burkitt's lymphoma. CRISPR was done to functionally annotate the role of oncogenic drivers that were identified from WGS. The genes IGLL5, BACH2, SIN3A, and DNMT1 were determined to be involved in the tumorigenesis of BL. In this study, they also identified ID3 as the most frequently recurrently silenced gene in all subtypes of BL. Further in vitro and in vivo study uncovered the role of ID3 in deregulation of TCF3 and TCF4 which affects the cell proliferation in BL cells (Panea et al., 2019). Katigbak et al. conducted CRISPR on an in vivo model of Burkitt's lymphoma: Eμ-Myc genetically engineered mouse model (GEMM). The results of this study shows that both Sp3 and Phip behave as tumor suppressors in Eμ-Myc driven lymphomas (Katigbak et al., 2016). Both these studies utilized CRISPR to identified and validate tumor suppressor genes in vitro and in vivo from BL genome sequencing data.

Multiple myeloma (MM) is the cancer of the plasma cells and is relatively rare cancer that contributes around 1.8% of all diagnosed cancer in United States (Myeloma, 2021). There are limited treatment options for MM, and relapse and developing resistance against treatment drugs is a worrying trend in MM (Anderson et al., 2009). Therefore, CRISPR has been utilized to identify crucial genes that can be targeted as a treatment option and also to determine genes that confer resistance to MM to allow for personalized treatment of MM patients. Khaled et al., 2021 utilized CRISPR to knockout crucial gene in primary MM cell to reduce its viability. After conducting a bioinformatic analysis on the gene expression data of MM cells, they identified V-set pre B-cell surrogate light chain 1 (VPREB1) gene as the knockout target gene. CRISPR-Cas9-mediated knockout of VPREB1 on primary MM cells was done, and it resulted in reduction of myeloma cell proliferation (Khaled et al., 2021). The results show that VPREB1 is a crucial gene that affects proliferation in MM cells and can be a possible target for treatment of MM. In another study, Tagde et al. tried to uncover the mechanism by which MYC oncoprotein is upregulated in multiple myeloma. MYC is a crucial oncoprotein for MM cells and plays an important role on the progression and survival of MM. However, there is no  comprehensive research that shows how MYC is upregulated in MM cells. CRISPR was utilized to silence mucin 1 C-terminal subunit (MUC1-C) which led to reduction in MYC expression levels. The result was further validated by looking at the expression levels in primary cells from MM patients. It was discovered that MUC1-C activates MYC gene through a TCF4-mediated mechanism (Tagde et al., 2016). By utilizing CRISPR to silence genes, the pathway on how MYC expression is upregulated in MM cells was discovered. In another study by Shi et al., the researchers tried to uncover the pathway of bortezomib resistance in MM cells. They started by doing CRISPR targeting the 19,052 human genes in the MM cell line: RPMI8226 and selecting the cells in lethal doses of bortezomib. The surviving cells were propagated, and the genome was sequenced to identify inactivated genes that led to resistance. Furthermore, CRISPR was done to validate the function of the genes, whereby proteasome regulatory subunit PSMC6 was the only gene validated to reproducibly confer bortezomib resistance (Shi et al., 2017). The researchers therefore determined that PSMC6 is crucial in bortezomib resistance in MM cells. Further research showed that the ability of bortezomib to inhibit chymotrypsin-like proteasome activity was severely affected in cells deficient in PSM6. Bohl et al. performed combined whole-exome sequencing (WES) on patient samples and relapsed patient samples. Based on the comparison between the samples, 170 relapse-specific mutations were identified and CRISPR was performed against them to determine their function. Based on the CRISPR results, 15 of them are functionally linked to drug resistance. In this study, it was discovered that there were specific genes associated with resistance against each type of drug. Resistance against lenalidomide was associated with E3 ligase complex genes, dexamethasone was associated with PCDHA5 and ANKMY2, bortezomib was linked with RB1, and CDK2NC and melphalan were linked with TP53 (Bohl et al., 2021). It was also discovered that inactivation of genes such as ATM, FANCA, BRCC3, and RAD54B, which are involved in DNA damage repair, made MM cells more sensitive to chemotherapy. The findings from this study suggest that gene alteration is linked to sensitivity or resistance to chemo drugs which can be utilized in future to advice treatment options for MM patients. Ramkumar et al. wanted to identify pathways that control the expression of B-cell maturation antigen (BCMA) since in one of the ways MM develops resistance against immunotherapy is via loss of surface antigen expression. In the study, CRISPR interference and CRISPR activation were used to identify genes that led to increase in BCMA on the MM cell surface. They discovered that the knockdown of HDAC7 and SEC61A (part of SEC61 complex) increased the levels of BCMA (Ramkumar et al., 2020). They then verified their results by using drugs to inhibit HDACs protein and the Sec61 complex, whereby an increase in BCMA levels was observed. The cells with elevated BCMA were also more susceptible to the immunotherapy drug: BCMA-targeted antibody-drug conjugate (ADC), HDP-101. These studies show how CRISPR has been utilized to uncover underlying genomic changes in MM cells that confers them resistance against treatment. The same information can then be used to develop new treatment options for patients.

CRISPR has also been used to uncover the functional genomics in rare hematological malignancies. Ng et al. conducted a study on T-and NK-cell lymphomas (TCL) which have poor clinical outcomes. The goal of the study was to identify targetable vulnerabilities in TCL using genome-wide CRISPR screening. They discovered that potential treatment targets JAK2 or IKZF1 which can be targeted with available inhibitors (Ng et al., 2018). In TP53-wild-type TCLs, they discovered MDM2 and MDMX as vulnerable targets, whereby their interaction with p53 can be blocked with ALRN-6924. This study identified treatment targets in TCL where there are available drugs. In another study, Zhang et al. conducted CRISPR on anaplastic lymphoma kinase (ALK)-positive anaplastic large-cell lymphoma (ALK + ALCL) to uncover programmed cell death protein 1 (PD-L1) regulation (Zhang et al., 2019). They discovered that PD-L1 is induced by STAT3 and GRB2/SOS1. STAT3 and GRB2/SOS1 through action of IRF4 and BATF3 transcription factors act on enhancer region of PD-L1 gene to induce its expression. By uncovering the pathway of PD-L1 induction, new treatment targets can be identified to allow for improved immunotherapy strategies. Patil et al. conducted a study to uncover the toxicity of cereblon-modulating agents (CMs) on primary effusion lymphoma (PEL). CMs are drugs used to treat PEL, and the downstream mechanisms of toxicity are not well studied. The researchers conducted genome-wide CRISPR selection screening against CMs with increasing toxicity in PEL: lenalidomide (LEN), pomalidomide (POM), and CC-122 (Patil et al., 2019). Based on the results they determined that inactivation of the E2 ubiquitin-conjugating enzyme UBE2G1 conferred resistance against LEN and POM, while inactivation of CRBN provided resistance to CC-122. The genes can be utilized in future as biomarkers to determine treatment options for patients. Despite improved responses with frontline therapies in lymphoma relapsed and refractory cases remain a significant challenge. Genome-editing technology has become a potential therapeutic option in lymphoid malignancies although most applications of CRISPR-Cas9 to date have been limited to bench studies. Recently, an exciting study unveiled the target gene for the therapeutic activity of NUTLIN3A, a novel smallmolecule antagonist of MDM2 that promotes TP53 activation. Using mouse models lacking TP53 target genes, the authors demonstrated that BBC3 (PUMA) is responsible for the resistance of NUTLIN3A in lymphomas. Furthermore, CRISPR-Cas9-mediated silencing of the BBC3 gene confirmed that BBC3 expression might predict NUTLIN3A treatment outcomes in patients. More recently, a study reported the application of a CRISPR-Cas9 system to disrupt CXCR4 expression in mantle cell lymphomas (MCL), a highly aggressive subset of B-cell non-Hodgkin lymphomas (NHLs). Performing lentiviral-based CRISPR-Cas9-mediated silencing of CXCR4, the researchers found that reactive oxygen-mediated CXCR4 expression is a key signal inducing autophagy, which contributes to the survival of bortezomib-resistant MCL cells .


## CHALLENGES IN USING CRISPR TOOLS IN THE STUDY OF FUNCTIONAL GENOMICS

While CRISPR-Cas9 technology has emerged as a promising tool in interrogation of gene functions, it has been recognized that several technical challenges greatly mar the progression of this field. Some of the more perplexing issues have been encountered through the use of CRISPR in pooled screening, occurrence of offtargets, false positive and false negative results, low efficiency of HDR events, use of high passage number cell lines, and limitations to the types of genes that can be studied.

One of the major disadvantages in CRISPR, which has been identified early in the development of this technology, is the potential to generate off-target double-strand cuts. Although the typical sgRNA which consists of 22bp is able to provide high enough diversity for it to remain unique even within the human genome of over three billion bp, many studies have demonstrated that a degree of base mismatch is well tolerated by the Cas9 system, leading to the cleavage of alternative sites (Hsu et al., 2013;Lin et al., 2014). Furthermore, it has also been shown that if these mismatches are to occur further away from the PAM sequence, the likelihood of binding and cleavage would be higher (Hsu et al., 2014). Moreover, the presence of minor DNA or RNA bulge resulting from insertions or deletions in the genome are also tolerated (Lin et al., 2014). Some solutions to this problem has been devised including development of predictive scores in guide RNA design software (Tycko et al., 2016) and also incorporating the use of enhanced specific Cas9 (eSpCas9) and high fidelity Cas9 (SpCas9-HF1) (Kleinstiver et al., 2016;Slaymaker et al., 2016). Alternatively, Cas9 can also be delivered in a protein form to provide immediate activity and degrade shortly thereafter to prevent binding and cleavage of other sites (Kim et al., 2014). A recent study has further demonstrated that the precision of CRISPR editing can be significantly enhanced with the incorporation of a hairpin sequence at the 5' end of the sgRNA (Kocak et al., 2019).

Another major hindrance lie in the shortcomings encountered when pooled library screens are performed, whereby paracrine signaling from wild-type cells may mask the effects of knockout (KO) cells (Ford et al., 2019). This is exemplified in growth factor KO cultures where continued compensatory secretion by adjacent unaffected cells is able to prevent the emergence of the true phenotype. Therefore, pooled library screens of heterogenous cultures may fail to identify the full set of genes responsible for particular phenotypes. Additionally, another disadvantage of pooled screening is that the range of phenotypes that can be read is restricted, typically only to survival and proliferation studies. Some initial efforts made in this area have been able to partially address the abovementioned limitations. FACS, for example, has been shown capable of enriching cultures of positively transformed cells via expression of fluorescence proteins and also cell surface markers, which have been demonstrated even in patientderived xenografts (Hulton et al., 2020). This would allow for more precise investigations of cells by exclusion of crosstalk by neighboring wild-type cells.

Another major concern is that genes which are essential for survival cannot be interrogated via complete knockout (Ford et al., 2019). In such cases, it would be suitable to employ CRISPR interference for knockdown studies. On the other hand, it would be appropriate to utilize KO approaches to investigate genes which are capable of maintaining function at low expression levels considering the phenotype could be misinterpreted with knockdown studies. These circumstances would need to be identified prior and addressed with the use of appropriate CRISPR systems to avoid the emergence of perplexing outcomes.

Furthermore, CRISPR systems have been known to generate false positive and false negative results. False positives can potentially arise when genes with high copy number or genes amplified in aneuploid cells are targeted. In this instance, the double-strand breaks that occur at numerous sites cause the cells to undergo apoptosis, which leads to false positive results and hence misidentification of cancer survival genes. This has been shown by several groups which studied sgRNAs on amplified genes in malignant cell lines (Aguirre et al., 2016;Munoz et al., 2016;Sheel and Xue, 2016). A straightforward way of identifying this scenario is when apoptosis of cells occurs independent of gene of interest (GOI) transcriptional arrest. This scenario has been exemplified in knockout studies of the high copy number BCR-ABL fusion gene found in Philadelphia chromosome-related CML (Wang et al., 2015). A possible solution to overcoming this problem would be using CRISPRi to repress transcription of the GOI without introducing doublestrand breaks to the genome. In terms of false negatives, these scenarios are most often seen with the use of sgRNAs that exhibit low relative activity. To overcome this predicament, investigators could employ sgRNAs which have been prior validated by other research groups in wet lab experiments. With the advent of machine learning, it is also possible to design sgRNA with higher activity based on published sequences.

In the use of homology-directed repair (HDR) to introduce foreign GOIs into the human genome, low efficiency has been a major obstacle. Knocking-in genes with CRISPR/Cas9 is able to circumvent the detrimental effects on cellular phenotype caused by random integration with lentiviral systems. This approach has many potential uses including the possibility of creating disease models by incorporating mutant genes and also determining the functions of promoters and repressors in knock-in studies. Studies aimed at increasing HDR events have investigated approaches such as using blocking mutations through the incorporation of silent mutations in the PAM and target gene sequence to prevent re-cutting by the Cas9 nuclease once the donor DNA has been ligated (Paquet et al., 2016). Another strategy is via the engineering of a fusion fork head protein homolog1 transcription factor (Fkh1p) and LexA DNA-binding protein to generate LexA-Fkh1p, which is capable of efficiently recruiting exogenous DNA to the cleavage site (Roy et al., 2018). Furthermore, as it has been elucidated that HDR takes place mainly during the S and G2 phase of the cell cycle, a study was conducted to cell-cycle-tailor the expression of Cas9 (Gutschner et al., 2016). This delay in expression until the initiation of the S and G2 phases was achieved via fusion of Cas9 to the N-terminal of human geminin and was able to increase HDR events up to 87%.

Currently, researchers are also highly reliant on cell lines to conduct pooled screenings. Doubts have been cast on the ability of cell line studies to correlation with actual human diseases especially involving functional genomics as cultures maintained for prolonged durations accumulate point mutations, acquire epigenetic modifications, and also develop chromosomal aberrations (Rebuzzini et al., 2015). The ideal scenario would be to conduct CRISPR screens on primary cells isolated from human tissues/samples while in low passage culture (Hulton et al., 2020). Optimum growth media should also be selected for maintenance of native cellular physiology as it has been known that cells grown on different media tended to display varying gene expression patterns.

Adequately addressing the abovementioned issues will greatly enhance the utility of CRISPR and broaden applications to other emerging areas. Moreover, confounding results occurring from off-targets, false positives, false negatives, and failure of phenotype development could be further prevented to make CRISPR a more robust system in the interrogation of functional genomics. Additionally, it is to be noted that regardless of the hematological malignancies, the potential applications of the CRISPR on gene expression studies remain the same.


## CLINICAL IMPLICATIONS OF CRISPR APPLICATIONS IN HEMATOLOGICAL MALIGNANCIES

Advancements in CRISPR/Cas9 technology have greatly impacted basic research in hematological malignancies which have immense potential to be translated into clinical applications in the near future. Among the most promising are in the areas of drug discovery, identification of treatment resistance, disease modeling, and genetic manipulation to improve other cancer therapies such as CAR-T cells and editing HSCs for autologous hematopoietic reconstitution.

CRISPR has been extensively explored for discovery of drug targets as potential treatments through genetic screens on cancer cell lines. This approach, among others, has the capacity of identifying essential genes for survival, thereby establishing actionable targets for therapeutic development. A study conducted on AML cell lines, which employed genome-wide CRISPR screening, has identified that survival of the cancer cells is dependent on the mRNA decapping enzyme scavenger (DCPS) (Yamauchi et al., 2018). Further study on inhibitors of DCPS indicated strong anti-leukemic effects by causing missplicing of pre-mRNA. Transcription factors (TF) have also been known to be dysregulated in cancers and can be a main vulnerability in its pathogenesis. PAX5 and IKZF1 are among the most commonly altered TFs in B-ALL, affecting approximately 80% of patients (Mullighan et al., 2007). CRISPR screens on these TFs were able to identify downstream effectors including CB2, TXNIP, and NR3C1, which are targetable by inhibitors . This approach, aside from identification of specific genetic vulnerabilities present in cancer cells, is also applicable for delineating the roles of genomic aberrations in affecting cellular fitness which may further provide clues to underlying causes in drug resistance development and how these polymorphic variants drive the growth of cancer (Shalem et al., 2014;Wang et al., 2014;Hart et al., 2015).

In identification of drug resistance genes, a genome-scale CRISPR KO screen has been explored in BCR-ABL1 CML cell lines (Lewis et al., 2020). This approach was found to identify novel drug resistance mechanisms which are related to MAPK signaling and apoptosis via the intrinsic pathway. The information generated from such screens could potentially be used to design combinatorial therapies to simultaneously target the resistance and vulnerable survival genes, thereby reestablishing treatment susceptibility.

The absence of cellular models which closely recapitulates disease state in leukemic patients has been a major hurdle in the drug discovery process. Among the most burgeoning issues are the heterogeneity of mutations and non-uniform leukemic cell distribution that co-exist in vivo. In these terms, gene editing with the CRISPR system has presented a simple and versatile solution to leukemic disease modeling with the capability of introducing multiple deletions and insertions to simultaneously manipulate expression of several genes and alleles (Lucas et al., 2017). Furthermore, downstream identification of targets for leukemia treatment could also be streamlined with this strategy. This has been exemplified in the generation of a CLL line with complete biallelic loss of the ATM gene function to mimic this adverse prognosis state found in approximately 1/3 of patients (Quijada-Álamo et al., 2020). Furthermore, drug screening revealed a previously unknown susceptibility of del (11q)/ATM-mutant to inhibition of PARP and BCR. This discovery has unraveled that this high-risk cytogenetic abnormality is treatable with a combination of olaparib and ibrutinib, which confirms the feasibility in utilizing CRISPR in modeling and therapeutic discovery for hematological malignancies.

CRISPR is also able to improve on other cancer therapies including chimeric antigen receptor (CAR) T cells. At present, CAR-T cells have been investigated to treat ALL, CLL, and non-Hodgkin lymphoma through the ectopic expression of CD22, CD20, CD19, and dual B targeting (Paczesny et al., 2018). However, this treatment option is currently limited by low persistence and function of T cells, cytokine release syndrome, treatment-associated toxicity, and labor/cost-intensive preparation processes. To overcome these issues, CRISPR has been utilized to augment CAR-T cell therapy by knocking out endogenous TCR-β which leads to higher efficiency of anticancer activity (Legut et al., 2018). Additionally, it was also found that CRISPR can be used to disrupt T-cell surface receptor inhibitors including PD1 to block immunosuppressive signaling (Xia et al., 2019). Further studies are also underway to develop CRISPR engineered universal allogeneic T cells by knockout of MHC I and TCR, which may allow for upscale productions to reduce associate costs, duration of manufacturing, and the need for highly skilled technical personnel (Ren et al., 2017;.

The mortality rates in the early nineties were more than 80% but have improved gradually to 59% due to multiple reasons. In recent years, autologous HSC transplantation has gained traction with the advent of gene editing technologies which enables the rectification of disease-causing mutations in the patients' own cells, thereby circumventing complications of allogenic transplants, which includes the lack of suitable donors and the development of graft-versus-host disease. However, earlier studies with viral vectors which integrate randomly into the host genome have raised concerns over activation of oncogenes, leading to secondary leukemia (Zhang and McCarty, 2017). CRISPR/Cas9 is, therefore, able to offer a solution to this predicament with its capability to deliver nonintegrative components for site-specific genome editing of ex vivo HSC modifications. Additionally, this system is also able to couple with conventional gene therapy to guide donor DNA to safe harboring sites for HDR gene integration. These possibilities open new avenues for researchers to explore not only just silencing and activation of expression but also gene replacement as potential therapies for leukemia. As applications of the former methods are obvious, the latter approach will be highly valued in replacing essential genes involved in complex translocations which gave rise to the hematological malignancy. Early studies on correcting the ASXL1 mutation in the CML cell line KBM5 has already confirmed the restoration of gene function and prolonger survival of xenograft mouse models (Valletta et al., 2015). Furthermore, a publication by Sánchez-Rivera and Jacks in 2015 reported a CRISPR-Cas9 AML mouse model system which has been developed by the Ebert group. It is a lentivirus-mediated ex vivo editing of single or multiple genes in a primary mouse HSPCs. This highlights the potential of the CRISPR system for ex vivo somatic genome editing of hematopoietic cells, which can be further exploited for various human hematological malignancies.

The applications of CRISPR in hematological malignancies spans across fundamental research to clinical investigations and have made rapid progress in the revelation of vulnerable targets, improvements on other existing therapies, and cellular modifications. In the near future, it is expected that this technology will serve as a practical solution both individually and in combinational therapies to improve the remission rates of leukemic patients.


## SUMMARY AND FUTURE PERSPECTIVES

This review has clearly demonstrated that CRISPR has served well in deciphering and understanding the genetic basis of hematological malignancies and has bloomed to become the new gamechanger in the field of genome editing by overcoming all the limitations that were displayed by other editing techniques earlier. In spite of all the merits posed by this so-called prime technique, CRISPR also come with certain limitations like its earlier predecessors.

Currently, CRISPR systems can be delivered in different modes whereby, gene editing can be achieved through transient or stable delivery system. Viral-based transfection of CRISPR is the most efficient method for producing stablymodified cells (Lino et al., 2018). Nevertheless, there has been a recent paradigm shift where non-viral methods are becoming the main homestay given biosafety and ethical issues when utilizing viral vectors. Non-viral vector systems include systems such as lipid nanoparticles, cell-penetrating peptides (CPPs), DNA "nanoclews," and gold nanoparticles. Having said that, the efficiency of the delivery still remains a challenge in hematological-related malignancies. There has been special technique developed to shuttle the cargo of the CRISPR system into the cell via rapid cell mechanical deformation (Peyravian et al., 2021). This has, to a certain extent, alleviated the limitation posed earlier and has allowed high delivery efficiency in hematological malignancies.

The issue of specificity in the CRISPR system is a major concern since Cas9 binds to unintended genomic sites for cleavage, termed as off-target effects. The target efficiency of the CRISPR system is only determined through 20 nucleotide sequences of guide RNA (gRNA) and PAM sites adjacent to target loci. If there are more than three mismatches between target sequences and 20 nucleotides of gRNA, it can result in off-target effects. Researchers have proposed two types of off-target effects, the first types of off-target effects likely to occur due to the sequence homology of the target loci and the next types of offtarget sites occur in the genome other than the target site (Naeem et al., 2020). The effect causes multiple cellular issues at the genomic level and in turn leads to deletions, alterations in the respective gene, and sometimes could lead to lethal genetic mutations. There are different methods established to validate the off-target effects and the most unique of them all is the Genome-wide Unbiased Identification of DSBs Enabled by Sequencing (GUIDE-seq). This provides an unbiased and genome-wide method for detecting CRISPR RNA-guided DSBs in cells . This groundbreaking study has increased the practical viability of the off-target detection. With standardization of all the methods that has been drawn FIGURE 1 | Timeline describing the research progress of CRISPR applications in hematological malignancies. Key studies contributing to major breakthroughs in the field are highlighted. Frontiers in Genetics | www.frontiersin.org January 2022 | Volume 12 | Article 767298 16 over the years, it makes the gene editing tool known as CRISPR to be a possible therapeutic approach in field of hematological malignancy.

The molecular scissors known as "CRISPR" provides complete silencing, and knockdown of gene can still be a valuable means depending on the type of malignancy. This method of gene silencing usually involves an extended time frame to generate a stable cell line. Taking into consideration on the other gene editing tools available with the likes of ZFN, TALENs, and meganucleases, it can be said that CRISPR is the most efficient, stable, and effective. CRISPR is appropriate for delineating gene functions, and genome-editing technology is the apparent option for creating a true genetic null allele, introducing a point mutation, and correcting a specific mutation.

In conclusion, through current advancement in terms of gene editing, we can say that CRISPR has taken the research world by storm, evidently allowing us to make changes to the once impossible. CRISPR has made its way onto the freezers of labs all around the spectrum, easily accessible for researchers to use it. It is also becoming a mainstream methodology to study cancer biology given its versatility. It has now matured its way from experimental approach toward customized treatment involving cancer patients. CRISPR gene editing tools have also sparked significant advancements in enhancing our knowledge regarding hematological malignancies, which in essence presents us with potential therapeutic applications which holds much promise for alternative treatments for patients with this group of malignancies. Development of CRISPR utilization in studying the functional genomics of haematologic malignancies is depicted in Figure 1 and a general summary for studying functional genomics via CRISPR based tools is presented in Figure 2.  


## AUTHOR CONTRIBUTIONS

## FIGURE 2 |
2Functional Genomic analysis of hematological malignancies using CRISPR applications.


Conceptualization: EM and MS; writing-original draft preparation: EM, MS, AA, KK, and MY; writing-review and editing: EM and MS; MK and NY; supervision: EM and MK. All authors have read and agreed to the published version of the manuscript.

## FUNDING
Studies related to this topic were funded and supported by the Fundamental Research Grant Scheme (FRGS) 203/CIPPT/ 6711672 of the Ministry of Higher Education (MOHE), Malaysia, and the Fundamental Research Grant Scheme (FRGS), FRGS/1/2019/SKK15/AIMST/02/2.

## TABLE 1 |
1Recent findings on CRISPR usage in targeting leukemia(Ergo 2015(Ergo  -2021.Type of 
cas/Tools 

Gene target 
Leukemic type 
Target method 
Outcome 
References 

Cas9/Genomic 
cleavage 

USO1 
B-ALL with MLL 
translocation 

Knockout of gene 
Deregulated mTOR signaling and 
reduced colony-forming potential 

Jaiswal et al. (2021) 

Cas9/Genomic 
cleavage 

LMO2 promoter 
T-ALL 
Knockout of transcription 
factor binding site 

Blocks binding of crucial TFs reducing 
LMO2 oncogene expression 

Xie et al. (2020) 

Cas9/Genomic 
cleavage 

BCR-ABL 
BCR-ABL 
positive ALL 

Ablation of the BCR-ABL 
breakpoint and SH2 kinase 
domain 

Delayed leukemic onset post 
transplantation in mice models 

Montaño et al. (2020) 

Cas9/Genomic site 
nicking 

EZH2 
T-ALL 
Knockout of gene at exon 2 
Sensitizes ALL cells toward chemodrug 
treatment in the primary model 

León et al. (2020) 

Cas9/Genomic 
cleavage 

LDHA 
T-ALL 
Knockout of gene 
Arrested cell growth and suppression of 
the oncogene C-MYC in the primary 
model 

Yu et al. (2020) 

Cas9/Genomic 
cleavage 

microRNA-21 
CML 
Knockout of miRNA locus 
Inhibition of cellular proliferative potential 
and increased CML sensitivity toward 
Imatinib treatment 

Zhang et al. (2021) 

Cas9/Genomic 
cleavage 

BCR-ABL 
CML 
Knockout of BCR-ABL 
junction 

Inhibition of proliferation in vitro and in 
vivo xenografted transplant models 

Chen et al. (2020) 

Cas9/Genomic 
cleavage 

RUNX1 
CML 
RUNX1 gene knockout 
Sensitizes CML toward glucocorticoids 
and mTOR, BCL and VEGFR inhibitors, 
and increased ex vivo CAR-T cells 
targeting sensitivity 

Adnan Awad et al. 
(2021) 

Cas9/Genomic 
cleavage 

BCR-ABL 
CML 
BCR/ABL gene knockout 
Improved delivery and targeting of 
CRISPR system via delivery by PEG 
nanoparticle, improved survival of 
mouse models 

Liu et al. (2018) 

Cas9/Genomic 
cleavage 

HDAC1/2 
CML 
HDAC1/2 gene knockout 
Induced cellular apoptosis post 
knockout of imatinib-resistant patient-
derived CML 

Chen et al. (2019) 

Cas9/Genomic 
Knock in 

ASLX1 
CML 
Wild-type ASLX1 was used at 
template for HDRF-mediated 
insertion 

Reduced CML model cell growth and 
induce differentiation increased overall 
survival of mice xenotransplanted with 
knocked in CML model lines 

Valletta et al. (2015) 

Cas9/Genomic 
Knock in 

TOP2α 
CML 
Insertion of GAG//GTAA AC 
→GAG//GTAA GT to exon 19/ 
intron 19.5 splice site of 
TOP2α 

Increased etoposide induced DNA 
damage otherwise desensitized in CML 
with suboptimal TOP2α 

Hernandez and 
Carvajal-Moreno, 
(2021) 

Cas9/Genomic 
cleavage 

APOBEC3 
CLL 
Direct gene knockout and cis 
regulatory element deletion 

APOBEC3 expression occurs in synergy 
with NFATc1 enhancer binding and BCR 
pathway post BTKi treatment leading to 
genomic instability 

Wang et al. (2021c) 

Cas9/Genomic 
cleavage 

BIRC/del (11q) 
CLL 
Direct gene knockout 
Uncovered mechanism on BIRC/del 
(11q) on CLL leukemogenesis where 
primary model with both BIRC1 
mutation and 11q deletion was 
sensitized to venetoclax 

Quijada-Álamo et al. 
(2021b) 

Cas9/Genomic 
cleavage 

TP53 
CLL 
Direct p53 gene knockout 
P53 knockout reveals multiple p53 
dependent miRNA crucial for CLL 
development 

Blume, (2015) 

Cas9/gRNA 
transcription factor 
pool cleavage 

147 CLL 
Transcription factor 
library and PAX5 

CLL 
Transcription factor gene 
knockout 

PAX5 was found to be predominantly 
contributing to CLL progression in 
primary samples where reduced CLL 
cell growth was observed 

Ott et al. (2018) 

Cas9/Genomic 
cleavage 

Notch2 
CLL 
Direct gene knockout 
Notch2-deleted cells deactivated Wnt 
signaling leading to the impairment in 
tumor survival 

Mangolini et al. (2018) 

(Continued on following page) 

Frontiers in Genetics | www.frontiersin.org 
January 2022 | Volume 12 | Article 767298 

## TABLE 1
1| (Continued) Recent findings on CRISPR usage in targeting leukemia (Ergo 2015-2021). 

Type of 
cas/Tools 

Gene target 
Leukemic type 
Target method 
Outcome 
References 

Cas9/Genomic 
cleavage 

Genome wide 
ABC DLBCL 
Whole genome knockout 
In ABC DLBC, EBF1, IRF4, CARD11, 
MYD88, and IKBKB knockout reduced 
viability 

Reddy et al. (2017) 

GCB DLBCL 
In GCB DLBCL, the knockout of 
ZBTB7A, XPO1, TGFBR2, and PTPN6 
reduced viability 

Cas9/Genomic 
cleavage 

Genome wide 
GCB DLBCL 
Whole genome knockout 
Discovered the synthetic lethal 
interaction between CREBBP and 
EP300 genes in DLBCL. 

Nie et al. (2021) 

Cas9/Genomic 
cleavage 

Genome wide 
ABC DLBCL 
Whole genome knockout 
Discovered a new form of BCR signaling 
coordinated by multiprotein 
supercomplex formed by MYD88, 
TLR9, and BCR (My-T-BCR) in ibrutinib-
responsive DLBCL. 

Phelan et al. (2018) 

GCB DLBCL 

Cas9/Genomic 
cleavage 

Foxp1 
Mice model of B-cell 
lymphoma 

Knockout of gene 
Knockdown of Foxp1 causes 
upregulated cell surface I-Ab (MHC-II) 
expression and restores immune 
surveillance 

Felce et al. (2020) 

Cas9/Genomic 
cleavage 

Genome wide 
BL 
Whole genome knockout 
The genes IGLL5, BACH2, SIN3A, and 
DNMT1 were determined to be involved 
in tumorigenesis of BL. ID3 is the most 
frequently silenced gene in all subtypes 
of BL. 

Panea et al. (2019) 

Cas9/Genomic 
cleavage 

Sp3 and Phip 
Eμ-Myc genetically 
engineered mouse 
model of BL 

Knockout of genes 
Both Sp3 and Phip act as tumor 
suppressors in Eμ-Myc driven 
lymphomas 

Katigbak et al. (2016) 

Cas9/Genomic 
cleavage 

VPREB1 
MM 
Knockout of gene 
Knockout of VPREB1 on primary 
MM cells resulted in reduction of 
myeloma cell proliferation 

Khaled et al. (2021) 

Cas9/Genomic 
cleavage 

MUC1-C 
MM 
Knockout of gene 
It was discovered that MUC1-C 
activates MYC gene through a TCF4-
mediated mechanism 

Tagde et al. (2016) 

Cas9/Genomic 
cleavage 

Genome wide 
MM 
Whole genome knockout 
Proteasome regulatory subunit PSMC6 
confers bortezomib resistance 

Shi et al. (2017) 

Cas9/Genomic 
cleavage 

Genome wide 
MM 
Whole genome knockout 
Drug resistance in MM was associated 
with E3 ligase complex genes, PCDHA5, 
ANKMY2, RB1, CDK2NC, and TP53 

Bohl et al. (2021) 

Inactivation of ATM, FANCA BRCC3, 
and RAD54B made MM cells more 
sensitive to chemotherapy 

dCas9/Genomic 
interference and 
activation 

Genome wide 
MM 
Whole genome 
Knockdown of HDAC7 and SEC61A 
increased the levels of BCMA 

Ramkumar et al. 
(2020) 

Cas9/Genomic 
cleavage 

Genome wide 
TCL 
Whole genome knockout 
Discovered JAK2 and IKZF1as potential 
treatment targets. In TP53-wild-type 
TCLs, MDM2, and MDMX can be 
blocked with ALRN-6924 

Ng et al. (2018) 

Cas9/Genomic 
cleavage 

600 genes 
associated with 
PD-L1 

ALCL 
Knockout of gene 
Discovered that PD-L1 is induced by 
STAT3 and GRB2/SOS1 

Zhang et al. (2019) 

Cas9/Genomic 
cleavage 

Genome wide 
PEL 
Whole genome knockout 
Inactivation of UBE2G1 conferred 
resistance against LEN and POM, while 
inactivation of CRBN provided 
resistance to CC-122 

Patil et al. (2019) 

(Continued on following page) 

Frontiers in Genetics | www.frontiersin.org 
January 2022 | Volume 12 | Article 767298 

## TABLE 1 |
1(Continued) Recent findings on CRISPR usage in targeting leukemia(Ergo 2015(Ergo  -2021.Type of 
cas/Tools 

Gene target 
Leukemic type 
Target method 
Outcome 
References 

Cas9/Genomic 
cleavage 

Genome wide 
AML 
Whole genome screening 
492 AML-specific genes, including 
DOT1L, BCL2, and MEN1. KAT2A 
inhibition demonstrated anti-AML 
activity by inducing myeloid 
differentiation and apoptosis, 
suppressed the growth of AMLs 

Tzelepis et al. (2016) 

Cas9/Genomic 
cleavage 

IL1RAP (interleukin-1 
receptor accessory 
protein) 

AML 
Knockout of gene 
IL1RAP knockout reduced LSC colony-
forming capacity and leukemic burden 

Ho et al. (2021) 

Cas9/Genomic 
cleavage 

RBM39 (RNA-
Binding Motif 
Protein 39) 

AML 
Knockout of gene 
Effects of RBM39 loss resulted in 
lethality of spliceosomal mutant AML, 
providing a strategy for treatment 
of AML. 

Wang et al. (2019) 

Cas9/Genomic 
cleavage 

miR-155 
AML 
Whole genome screen 
CRISPR-Cas9 global loss-of-function 
screen to simultaneously test the 
functions of individual miRNAs and 
protein-coding genes. miR-155 was 
promoting cellular fitness, which was 
confirmed with 2 distinct miR-155 
targeting by CRISPR-Cas9 lentiviral 
constructs 

Wallace et al. (2016b) 


Conflict of Interest:The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.
RNA Targeting with CRISPR-Cas13. O O Abudayyeh, J S Gootenberg, P Essletzbichler, S Han, J Joung, J J Belanto, 10.1038/nature24049Nature. 5507675Abudayyeh, O. O., Gootenberg, J. S., Essletzbichler, P., Han, S., Joung, J., Belanto, J. J., et al. (2017). RNA Targeting with CRISPR-Cas13. Nature 550 (7675), 280-284. doi:10.1038/nature24049

Adnan Awad, S Dufva, O Ianevski, A Ghimire, B Koski, J Maliniemi, P , 10.1038/s41375-020-01011-5RUNX1 Mutations in Blast-phase Chronic Myeloid Leukemia Associate with Distinct Phenotypes, Transcriptional Profiles, and Drug Responses. 35Adnan Awad, S., Dufva, O., Ianevski, A., Ghimire, B., Koski, J., Maliniemi, P., et al. (2021). RUNX1 Mutations in Blast-phase Chronic Myeloid Leukemia Associate with Distinct Phenotypes, Transcriptional Profiles, and Drug Responses. Leukemia 35 (4), 1087-1099. doi:10.1038/s41375-020-01011-5

Genomic Copy Number Dictates a Gene-independent Cell Response to CRISPR/Cas9 Targeting. A J Aguirre, R M Meyers, B A Weir, F Vazquez, C.-Z Zhang, U Ben-David, 10.1158/2159-8290.cd-16-0154Cancer Discov. 68Aguirre, A. J., Meyers, R. M., Weir, B. A., Vazquez, F., Zhang, C.-Z., Ben-David, U., et al. (2016). Genomic Copy Number Dictates a Gene-independent Cell Response to CRISPR/Cas9 Targeting. Cancer Discov. 6 (8), 914-929. doi:10.1158/2159-8290.cd-16-0154

Multiple Myeloma. K C Anderson, M Alsina, W Bensinger, J S Biermann, A Chanan-Khan, A D Cohen, 10.6004/jnccn.2009.0061J. Natl. Compr. Canc Netw. 79Anderson, K. C., Alsina, M., Bensinger, W., Biermann, J. S., Chanan-Khan, A., Cohen, A. D., et al. (2009). Multiple Myeloma. J. Natl. Compr. Canc Netw. 7 (9), 908-942. doi:10.6004/jnccn.2009.0061

Mutations in NOTCH1 PEST Domain Orchestrate CCL19-Driven Homing of Chronic Lymphocytic Leukemia Cells by Modulating the Tumor Suppressor Gene DUSP22. F Arruga, B Gizdic, C Bologna, S Cignetto, R Buonincontri, S Serra, 10.1038/leu.2016.383doi:10.1038/ leu.2016.383Leukemia. 319Arruga, F., Gizdic, B., Bologna, C., Cignetto, S., Buonincontri, R., Serra, S., et al. (2017). Mutations in NOTCH1 PEST Domain Orchestrate CCL19-Driven Homing of Chronic Lymphocytic Leukemia Cells by Modulating the Tumor Suppressor Gene DUSP22. Leukemia 31 (9), 1882-1893. doi:10.1038/ leu.2016.383

MiRNA182 Regulates Percentage of Myeloid and Erythroid Cells in Chronic Myeloid Leukemia. D Arya, S P Sachithanandan, C Ross, D Palakodeti, S Li, Krishna , S , 10.1038/cddis.2016.471e2547. doi:10.1038/ cddis.2016.471Cell Death Dis. 81Arya, D., Sachithanandan, S. P., Ross, C., Palakodeti, D., Li, S., and Krishna, S. (2018). MiRNA182 Regulates Percentage of Myeloid and Erythroid Cells in Chronic Myeloid Leukemia. Cell Death Dis 8 (1), e2547. doi:10.1038/ cddis.2016.471

KAT7 Is a Genetic Vulnerability of Acute Myeloid Leukemias Driven by MLL Rearrangements. Y Z Au, M Gu, E De Braekeleer, M Gozdecka, D Aspris, Y Tarumoto, 10.1038/s41375-020-1001-zLeukemia. 354Au, Y. Z., Gu, M., De Braekeleer, E., Gozdecka, M., Aspris, D., Tarumoto, Y., et al. (2021). KAT7 Is a Genetic Vulnerability of Acute Myeloid Leukemias Driven by MLL Rearrangements. Leukemia 35 (4), 1012-1022. doi:10.1038/s41375-020- 1001-z

Detecting Recurrent Gene Mutation in Interaction Network Context Using Multi-Scale Graph Diffusion. S Babaei, M Hulsman, M Reinders, J D Ridder, 10.1186/1471-2105-14-29BMC Bioinformatics. 1429Babaei, S., Hulsman, M., Reinders, M., and Ridder, J. d. (2013). Detecting Recurrent Gene Mutation in Interaction Network Context Using Multi-Scale Graph Diffusion. BMC Bioinformatics 14, 29. doi:10.1186/1471-2105-14-29

An In Vivo Genome-wide CRISPR Screen Identifies the RNA-Binding Protein Staufen2 as a Key Regulator of Myeloid Leukemia. J Bajaj, M Hamilton, Y Shima, K Chambers, K Spinler, E L Van Nostrand, 10.1038/s43018-020-0054-2Nat. Cancer. 1410Bajaj, J., Hamilton, M., Shima, Y., Chambers, K., Spinler, K., Van Nostrand, E. L., et al. (2020). An In Vivo Genome-wide CRISPR Screen Identifies the RNA-Binding Protein Staufen2 as a Key Regulator of Myeloid Leukemia. Nat. Cancer 1, 410. doi:10.1038/s43018-020-0054-2

Bcr-Abl Expression Levels Determine the Rate of Development of Resistance to Imatinib Mesylate in Chronic Myeloid Leukemia. D J Barnes, D Palaiologou, E Panousopoulou, B Schultheis, A S M Yong, A Wong, 10.1158/0008-5472.can-05-0076Cancer Res. 65Barnes, D. J., Palaiologou, D., Panousopoulou, E., Schultheis, B., Yong, A. S. M., Wong, A., et al. (2005). Bcr-Abl Expression Levels Determine the Rate of Development of Resistance to Imatinib Mesylate in Chronic Myeloid Leukemia. Cancer Res. 65 (19), 8912-8919. doi:10.1158/0008-5472.can-05- 0076

p53-dependent Non-coding RNA Networks in Chronic Lymphocytic Leukemia. C J Blume, 10.1038/leu.2015.119Leukemia. 29Blume, C. J. (2015). p53-dependent Non-coding RNA Networks in Chronic Lymphocytic Leukemia. Leukemia 29, 2015, doi:10.1038/leu.2015.119

Comprehensive CRISPR-Cas9 Screens Identify Genetic Determinants of Drug Responsiveness in Multiple Myeloma | Blood Advances. S R Bohl, L K Schmalbrock, I Bauhuf, T Meyer, D Anna, M Szyka, 10.1182/bloodadvances.2020003541Am. Soc. Hematol. 52391Bohl, S. R., Schmalbrock, L. K., Bauhuf, I., Meyer, T., Anna, D., Szyka, M., et al. (2021). Comprehensive CRISPR-Cas9 Screens Identify Genetic Determinants of Drug Responsiveness in Multiple Myeloma | Blood Advances. Am. Soc. Hematol. 5, 2391. doi:10.1182/bloodadvances.2020003541

RNA-guided CRISPR-Cas9 System-Mediated Engineering of Acute Myeloid Leukemia Mutations. O Brabetz, V Alla, L Angenendt, C Schliemann, W E Berdel, M.-F Arteaga, 10.1016/j.omtn.2016.12.012Mol. Ther. -Nucleic Acids. 6Brabetz, O., Alla, V., Angenendt, L., Schliemann, C., Berdel, W. E., Arteaga, M.-F., et al. (2017). RNA-guided CRISPR-Cas9 System-Mediated Engineering of Acute Myeloid Leukemia Mutations. Mol. Ther. -Nucleic Acids 6, 243-248. doi:10.1016/j.omtn.2016.12.012

. L N Chan, Z Chen, D Braas, J.-W Lee, G Xiao, H Geng, Chan, L. N., Chen, Z., Braas, D., Lee, J.-W., Xiao, G., Geng, H., et al. (2017).

Metabolic Gatekeeper Function of B-Lymphoid Transcription Factors. 10.1038/nature21076Nature. 5427642Metabolic Gatekeeper Function of B-Lymphoid Transcription Factors. Nature 542 (7642), 479-483. doi:10.1038/nature21076

Molecular Subtypes of Diffuse Large B Cell Lymphoma Are Associated with Distinct Pathogenic Mechanisms and Outcomes. B Chapuy, C Stewart, A J Dunford, J Kim, A Kamburov, R A Redd, 10.1038/s41591-018-0016-8Nat. Med. 245Chapuy, B., Stewart, C., Dunford, A. J., Kim, J., Kamburov, A., Redd, R. A., et al. (2018). Molecular Subtypes of Diffuse Large B Cell Lymphoma Are Associated with Distinct Pathogenic Mechanisms and Outcomes. Nat. Med. 24 (5), 679-690. doi:10.1038/s41591-018-0016-8

Induced DNA Demethylation by Targeting Ten-Eleven Translocation 2 to the Human ICAM-1 Promoter. H Chen, H G Kazemier, M L De Groote, M H J Ruiters, G.-L Xu, M G Rots, 10.1093/nar/gkt1019Nucleic Acids Res. 423Chen, H., Kazemier, H. G., de Groote, M. L., Ruiters, M. H. J., Xu, G.-L., and Rots, M. G. (2014). Induced DNA Demethylation by Targeting Ten-Eleven Translocation 2 to the Human ICAM-1 Promoter. Nucleic Acids Res. 42 (3), 1563-1574. doi:10.1093/nar/gkt1019

HDAC1,2 Knock-Out and HDACi Induced Cell Apoptosis in Imatinib-Resistant K562 Cells. S.-H Chen, J.-M Chow, Y.-Y Hsieh, C.-Y Lin, K.-W Hsu, W.-S Hsieh, 10.3390/ijms20092271Ijms. 2092271Chen, S.-H., Chow, J.-M., Hsieh, Y.-Y., Lin, C.-Y., Hsu, K.-W., Hsieh, W.-S., et al. (2019). HDAC1,2 Knock-Out and HDACi Induced Cell Apoptosis in Imatinib- Resistant K562 Cells. Ijms 20 (9), 2271. doi:10.3390/ijms20092271

ABL Genomic Editing Sufficiently Abolishes Oncogenesis of Human Chronic Myeloid Leukemia Cells In Vitro and In Vivo. S.-H Chen, Y.-Y Hsieh, H.-E Tzeng, C.-Y Lin, K.-W Hsu, Y.-S Chiang, 10.3390/cancers12061399Cancers. 1261399Chen, S.-H., Hsieh, Y.-Y., Tzeng, H.-E., Lin, C.-Y., Hsu, K.-W., Chiang, Y.-S., et al. (2020). ABL Genomic Editing Sufficiently Abolishes Oncogenesis of Human Chronic Myeloid Leukemia Cells In Vitro and In Vivo. Cancers 12 (6), 1399. doi:10.3390/cancers12061399

Frontiers in Genetics | www.frontiersin.org. 12767298Frontiers in Genetics | www.frontiersin.org January 2022 | Volume 12 | Article 767298

Deletion of MBD2 Inhibits Proliferation of Chronic Myeloid Leukaemia Blast Phase Cells. L Cheng, Y Tang, X Chen, L Zhao, S Liu, Y Ma, 10.1080/15384047.2018.1450113doi:10.1080/ 15384047.2018.1450113Cancer Biol. Ther. 198Cheng, L., Tang, Y., Chen, X., Zhao, L., Liu, S., Ma, Y., et al. (2018). Deletion of MBD2 Inhibits Proliferation of Chronic Myeloid Leukaemia Blast Phase Cells. Cancer Biol. Ther. 19 (8), 676-686. doi:10.1080/ 15384047.2018.1450113

Improving Outcomes in Chronic Myeloid Leukemia over Time in the Era of Tyrosine Kinase Inhibitors. P Chopade, L P Akard, 10.1016/j.clml.2018.06.029Review. 18710Chopade, P., and Akard, L. P. (2018). Improving Outcomes in Chronic Myeloid Leukemia over Time in the Era of Tyrosine Kinase Inhibitors. Review 18, 710. doi:10.1016/j.clml.2018.06.029

FBXW7 Mutations Reduce Binding of NOTCH1, Leading to Cleaved NOTCH1 Accumulation and Target Gene Activation in CLL. V Close, W Close, S J Kugler, M Reichenzeller, D Y Yosifov, J Bloehdorn, 10.1182/blood-2018-09-874529Blood. 1338Close, V., Close, W., Kugler, S. J., Reichenzeller, M., Yosifov, D. Y., Bloehdorn, J., et al. (2019). FBXW7 Mutations Reduce Binding of NOTCH1, Leading to Cleaved NOTCH1 Accumulation and Target Gene Activation in CLL. Blood 133 (8), 830-839. doi:10.1182/blood-2018-09-874529

CRISPR-cas Systems for Genome Engineering and Investigation. J P Concordet, C Giovannangeli, 10.1016/j.ymeth.2017.06.0061-2. doi:10.1016/ j.ymeth.2017.06.006Methods. 121Concordet, J. P., and Giovannangeli, C. (2017). CRISPR-cas Systems for Genome Engineering and Investigation. Methods 121-122, 1-2. doi:10.1016/ j.ymeth.2017.06.006

RNA Editing with CRISPR-Cas13. D B T Cox, J S Gootenberg, O O Abudayyeh, B Franklin, M J Kellner, J Joung, 10.1126/science.aaq0180Science. 3586366Cox, D. B. T., Gootenberg, J. S., Abudayyeh, O. O., Franklin, B., Kellner, M. J., Joung, J., et al. (2017). RNA Editing with CRISPR-Cas13. Science 358 (6366), 1019-1027. doi:10.1126/science.aaq0180

L Deng, R A Garrett, S A Shah, X Peng, Q She, A Novel Interference Mechanism by a Type IIIB CRISPR-Cmr Module inSulfolobus. Deng, L., Garrett, R. A., Shah, S. A., Peng, X., and She, Q. (2013). A Novel Interference Mechanism by a Type IIIB CRISPR-Cmr Module inSulfolobus.

. 10.1111/mmi.12152Mol. Microbiol. 875Mol. Microbiol. 87 (5), 1088-1099. doi:10.1111/mmi.12152

Lymphoma Research Foundation. Diffuse Large, B-Cell Lymphoma, InternetDiffuse Large B-Cell Lymphoma (2021). Lymphoma Research Foundation. [Internet]. [cited 2021 Jun 2]. Available at: https://lymphoma.org/ aboutlymphoma/nhl/dlbcl/.

Mini Review: Genome and Transcriptome Editing Using CRISPR-Cas Systems for Haematological Malignancy Gene Therapy. E K Elliott, L M Haupt, L R Griffiths, 10.1007/s11248-020-00232-9doi:10.1007/ s11248-020-00232-9Transgenic Res. 302Elliott, E. K., Haupt, L. M., and Griffiths, L. R. (2021). Mini Review: Genome and Transcriptome Editing Using CRISPR-Cas Systems for Haematological Malignancy Gene Therapy. Transgenic Res. 30 (2), 129-141. doi:10.1007/ s11248-020-00232-9

CRISPR/Cas9-Mediated Foxp1 Silencing Restores Immune Surveillance in an Immunocompetent A20 Lymphoma Model. S L Felce, A P Anderson, S Maguire, D M Gascoyne, R N Armstrong, K K Wong, 10.3389/fonc.2020.00448Front. Oncol. 10448Felce, S. L., Anderson, A. P., Maguire, S., Gascoyne, D. M., Armstrong, R. N., Wong, K. K., et al. (2020). CRISPR/Cas9-Mediated Foxp1 Silencing Restores Immune Surveillance in an Immunocompetent A20 Lymphoma Model. Front. Oncol. 10, 448. doi:10.3389/fonc.2020.00448

Functional Genomics via CRISPR-Cas. K Ford, D Mcdonald, P Mali, 10.1016/j.jmb.2018.06.034J. Mol. Biol. 4311Ford, K., McDonald, D., and Mali, P. (2019). Functional Genomics via CRISPR- Cas. J. Mol. Biol. 431 (1), 48-65. doi:10.1016/j.jmb.2018.06.034

Therapeutic siRNA: Principles, Challenges, and Strategies. K Gavrilov, W M Saltzman, Yale J. Biol. Med. 852Gavrilov, K., and Saltzman, W. M. (2012). Therapeutic siRNA: Principles, Challenges, and Strategies. Yale J. Biol. Med. 85 (2), 187-200.

Indications for Imatinib Mesylate Therapy and Clinical Management. F Guilhot, 10.1634/theoncologist.9-3-271The Oncologist. 93Guilhot, F. (2004). Indications for Imatinib Mesylate Therapy and Clinical Management. The Oncologist 9 (3), 271-281. doi:10.1634/theoncologist.9-3-271

Broad Specificity Profiling of TALENs Results in Engineered Nucleases with Improved DNA-Cleavage Specificity. J P Guilinger, V Pattanayak, D Reyon, S Q Tsai, J D Sander, J K Joung, 10.1038/nmeth.2845Nat. Methods. 114Guilinger, J. P., Pattanayak, V., Reyon, D., Tsai, S. Q., Sander, J. D., Joung, J. K., et al. (2014). Broad Specificity Profiling of TALENs Results in Engineered Nucleases with Improved DNA-Cleavage Specificity. Nat. Methods 11 (4), 429-435. doi:10.1038/nmeth.2845

Expanding the Genetic Editing Tool Kit: ZFNs, TALENs, and CRISPR-Cas9. R M Gupta, K Musunuru, 10.1172/jci72992J. Clin. Invest. 12410Gupta, R. M., and Musunuru, K. (2014). Expanding the Genetic Editing Tool Kit: ZFNs, TALENs, and CRISPR-Cas9. J. Clin. Invest. 124 (10), 4154-4161. doi:10.1172/jci72992

Post-translational Regulation of Cas9 during G1 Enhances Homology-Directed Repair. T Gutschner, M Haemmerle, G Genovese, G F Draetta, L Chin, 10.1016/j.celrep.2016.01.019Cell Rep. 146Gutschner, T., Haemmerle, M., Genovese, G., Draetta, G. F., and Chin, L. (2016). Post-translational Regulation of Cas9 during G1 Enhances Homology-Directed Repair. Cell Rep. 14 (6), 1555-1566. doi:10.1016/j.celrep.2016.01.019

High-Resolution CRISPR Screens Reveal Fitness Genes and Genotype-specific Cancer Liabilities. T Hart, M Chandrashekhar, M Aregger, Z Steinhart, K R Brown, G Macleod, 10.1016/j.cell.2015.11.015doi:10.1016/ j.cell.2015.11.015Cell. 1636Hart, T., Chandrashekhar, M., Aregger, M., Steinhart, Z., Brown, K. R., MacLeod, G., et al. (2015). High-Resolution CRISPR Screens Reveal Fitness Genes and Genotype-specific Cancer Liabilities. Cell 163 (6), 1515-1526. doi:10.1016/ j.cell.2015.11.015

CRISPR/Cas9 Genome Editing of the Human Topoisomerase IIα Intron 19 5′ Splice Site Circumvents Etoposide Resistance in Human Leukemia K562 Cells. V A Hernandez, J Moreno, 10.1124/molpharm.120.000173Mol. Pharmacol. 99226Hernandez, V. A., and Carvajal-Moreno, J. (2021). CRISPR/Cas9 Genome Editing of the Human Topoisomerase IIα Intron 19 5′ Splice Site Circumvents Etoposide Resistance in Human Leukemia K562 Cells. Mol. Pharmacol. 99, 226. doi:10.1124/molpharm.120.000173

Scaffold-mediated CRISPR-Cas9 Delivery System for. T-C Ho, H S Kim, Y Chen, Y Li, M W Lamere, C Chen, 10.1126/sciadv.abg3217Acute Myeloid Leukemia Therapy. Sci. Adv. 73217Ho, T-C., Kim, H. S., Chen, Y., Li, Y., LaMere, M. W., Chen, C., et al. (2021). Scaffold-mediated CRISPR-Cas9 Delivery System for Acute Myeloid Leukemia Therapy. Sci. Adv. 7, eabg3217. doi:10.1126/sciadv.abg3217

Efficient Targeting of Expressed and Silent Genes in Human ESCs and iPSCs Using Zinc-finger Nucleases. D Hockemeyer, F Soldner, C Beard, Q Gao, M Mitalipova, R C Dekelver, 10.1038/nbt.1562Nat. Biotechnol. 279Hockemeyer, D., Soldner, F., Beard, C., Gao, Q., Mitalipova, M., DeKelver, R. C., et al. (2009). Efficient Targeting of Expressed and Silent Genes in Human ESCs and iPSCs Using Zinc-finger Nucleases. Nat. Biotechnol. 27 (9), 851-857. doi:10.1038/nbt.1562

Improving Outcomes in Chronic Myeloid Leukemia through Harnessing the Immunological Landscape. Y.-C Hsieh, K Kirschner, M Copland, 10.1038/s41375-021-01238-wLeukemia. 355Hsieh, Y.-C., Kirschner, K., and Copland, M. (2021). Improving Outcomes in Chronic Myeloid Leukemia through Harnessing the Immunological Landscape. Leukemia 35 (5), 1229-1242. doi:10.1038/s41375-021-01238-w

Development and Applications of CRISPR-Cas9 for Genome Engineering. P D Hsu, E S Lander, F Zhang, 10.1016/j.cell.2014.05.010doi:10.1016/ j.cell.2014.05.010Cell. 1576Hsu, P. D., Lander, E. S., and Zhang, F. (2014). Development and Applications of CRISPR-Cas9 for Genome Engineering. Cell 157 (6), 1262-1278. doi:10.1016/ j.cell.2014.05.010

DNA Targeting Specificity of RNA-Guided Cas9 Nucleases. P D Hsu, D A Scott, J A Weinstein, F A Ran, S Konermann, V Agarwala, 10.1038/nbt.2647Nat. Biotechnol. 319Hsu, P. D., Scott, D. A., Weinstein, J. A., Ran, F. A., Konermann, S., Agarwala, V., et al. (2013). DNA Targeting Specificity of RNA-Guided Cas9 Nucleases. Nat. Biotechnol. 31 (9), 827-832. doi:10.1038/nbt.2647

Applications of CRISPR-Cas Enzymes in Cancer Therapeutics and Detection. C.-H Huang, K.-C Lee, J A Doudna, 10.1016/j.trecan.2018.05.006Trends Cancer. 47Huang, C.-H., Lee, K.-C., and Doudna, J. A. (2018). Applications of CRISPR-Cas Enzymes in Cancer Therapeutics and Detection. Trends Cancer 4 (7), 499-512. doi:10.1016/j.trecan.2018.05.006

Direct Genome Editing of Patient-Derived Xenografts Using CRISPR-Cas9 Enables Rapid In Vivo Functional Genomics. C H Hulton, E A Costa, N S Shah, A Quintanal-Villalonga, G Heller, E De Stanchina, 10.1038/s43018-020-0040-8Nat. Cancer. 13Hulton, C. H., Costa, E. A., Shah, N. S., Quintanal-Villalonga, A., Heller, G., de Stanchina, E., et al. (2020). Direct Genome Editing of Patient-Derived Xenografts Using CRISPR-Cas9 Enables Rapid In Vivo Functional Genomics. Nat. Cancer 1 (3), 359-369. doi:10.1038/s43018-020-0040-8

Acute Lymphoblastic Leukaemia. H Inaba, M Greaves, C G Mullighan, 10.1016/s0140-6736(12)62187-4The Lancet. 3819881Inaba, H., Greaves, M., and Mullighan, C. G. (2013). Acute Lymphoblastic Leukaemia. The Lancet 381 (9881), 1943-1955. doi:10.1016/s0140-6736(12) 62187-4

Focused CRISPR-Cas9 Genetic Screening Reveals USO1 as a Vulnerability in B-Cell Acute Lymphoblastic. A K Jaiswal, H Truong, T M Tran, T L Lin, D Casero, M O Alberti, 10.1038/s41598-021-92448-wLeukemia. Sci. Rep. 11113158Jaiswal, A. K., Truong, H., Tran, T. M., Lin, T. L., Casero, D., Alberti, M. O., et al. (2021). Focused CRISPR-Cas9 Genetic Screening Reveals USO1 as a Vulnerability in B-Cell Acute Lymphoblastic Leukemia. Sci. Rep. 11 (1), 13158. doi:10.1038/s41598-021-92448-w

Chronic Lymphocytic Leukemia: from Molecular Pathogenesis to. Julio Delgado, J Ferran Nadeu, F Dolors Colomer, D , Elias Campo, E , 10.3324/haematol.2019.236000doi:10.3324/ haematol.2019.236000Novel Therapeutic Strategies. haematol. 1059Julio Delgado, J., Ferran Nadeu, F., Dolors Colomer, D., and Elias Campo, E. (2020). Chronic Lymphocytic Leukemia: from Molecular Pathogenesis to Novel Therapeutic Strategies. haematol 105 (9), 2205-2217. doi:10.3324/ haematol.2019.236000

A CRISPR/Cas9 Functional Screen Identifies Rare Tumor Suppressors. A Katigbak, R Cencic, F Robert, P Sénécha, C Scuoppo, J Pelletier, 10.1038/srep38968Sci. Rep. 6138968Katigbak, A., Cencic, R., Robert, F., Sénécha, P., Scuoppo, C., and Pelletier, J. (2016). A CRISPR/Cas9 Functional Screen Identifies Rare Tumor Suppressors. Sci. Rep. 6 (1), 38968. doi:10.1038/srep38968

Acute Lymphoblastic Leukaemia: Diagnosis and Classification. P Kebriaei, J Anastasi, R A Larson, 10.1053/beha.2002.0224Best Pract. Res. Clin. Haematol. 154Kebriaei, P., Anastasi, J., and Larson, R. A. (2002). Acute Lymphoblastic Leukaemia: Diagnosis and Classification. Best Pract. Res. Clin. Haematol. 15 (4), 597-621. doi:10.1053/beha.2002.0224

M Khaled, A S Moustafa, N El-Khazragy, M I Ahmed, M A Abd Elkhalek, E M Salahy, CRISPR/Cas9 Mediated Knock-Out of VPREB1. Khaled, M., Moustafa, A. S., El-Khazragy, N., Ahmed, M. I., Abd Elkhalek, M. A., and El Salahy, E. M. (2021). CRISPR/Cas9 Mediated Knock-Out of VPREB1

Gene Induces a Cytotoxic Effect in Myeloma Cells. 10.1371/journal.pone.0245349PLoS ONE. 161Gene Induces a Cytotoxic Effect in Myeloma Cells. PLoS ONE 16 (1), e0245349. doi:10.1371/journal.pone.0245349

Mitochondrial Carrier Homolog 2 Is Necessary for AML Survival. D H Khan, M Mullokandov, Y Wu, V Voisin, M Gronda, R Hurren, 10.1182/blood.2019000106Blood. 1361Khan, D. H., Mullokandov, M., Wu, Y., Voisin, V., Gronda, M., Hurren, R., et al. (2020). Mitochondrial Carrier Homolog 2 Is Necessary for AML Survival. Blood 136 (1), 81-92. doi:10.1182/blood.2019000106

Identifying Genome-wide Off-Target Sites of CRISPR RNA-Guided Nucleases and Deaminases with Digenome-Seq. D Kim, B.-C Kang, J.-S Kim, Kim, D., Kang, B.-C., and Kim, J.-S. (2021). Identifying Genome-wide Off-Target Sites of CRISPR RNA-Guided Nucleases and Deaminases with Digenome-Seq.

. 10.1038/s41596-020-00453-6Nat. Protoc. 162Nat. Protoc. 16 (2), 1170-1192. doi:10.1038/s41596-020-00453-6

Highly Efficient RNA-Guided Genome Editing in Human Cells via Delivery of Purified Cas9 Ribonucleoproteins. S Kim, D Kim, S W Cho, J Kim, J.-S Kim, 10.1101/gr.171322.113doi:10.1101/ gr.171322.113Genome Res. 246Kim, S., Kim, D., Cho, S. W., Kim, J., and Kim, J.-S. (2014). Highly Efficient RNA- Guided Genome Editing in Human Cells via Delivery of Purified Cas9 Ribonucleoproteins. Genome Res. 24 (6), 1012-1019. doi:10.1101/ gr.171322.113

Chronic Lymphocytic Leukaemia. T J Kipps, F K Stevenson, C J Wu, C M Croce, G Packham, W G Wierda, 10.1038/nrdp.2016.96Nat. Rev. Dis. Primers. 3116096Kipps, T. J., Stevenson, F. K., Wu, C. J., Croce, C. M., Packham, G., Wierda, W. G., et al. (2017). Chronic Lymphocytic Leukaemia. Nat. Rev. Dis. Primers 3 (1), 16096. doi:10.1038/nrdp.2016.96

High-fidelity CRISPR-Cas9 Nucleases with No Detectable Genome-wide Off-Target Effects. B P Kleinstiver, V Pattanayak, M S Prew, S Q Tsai, N T Nguyen, Z Zheng, 10.1038/nature16526doi:10.1038/ nature16526Nature. 5297587Kleinstiver, B. P., Pattanayak, V., Prew, M. S., Tsai, S. Q., Nguyen, N. T., Zheng, Z., et al. (2016). High-fidelity CRISPR-Cas9 Nucleases with No Detectable Genome-wide Off-Target Effects. Nature 529 (7587), 490-495. doi:10.1038/ nature16526

Increasing the Specificity of CRISPR Systems with Engineered RNA Secondary Structures. D D Kocak, E A Josephs, V Bhandarkar, S S Adkar, J B Kwon, C A Gersbach, 10.1038/s41587-019-0095-1Nat. Biotechnol. 376Kocak, D. D., Josephs, E. A., Bhandarkar, V., Adkar, S. S., Kwon, J. B., and Gersbach, C. A. (2019). Increasing the Specificity of CRISPR Systems with Engineered RNA Secondary Structures. Nat. Biotechnol. 37 (6), 657-666. doi:10.1038/s41587-019-0095-1

Diversity, Classification and Evolution of CRISPR-Cas Systems. E V Koonin, K S Makarova, F Zhang, 10.1016/j.mib.2017.05.008Curr. Opin. Microbiol. 37Koonin, E. V., Makarova, K. S., and Zhang, F. (2017). Diversity, Classification and Evolution of CRISPR-Cas Systems. Curr. Opin. Microbiol. 37, 67-78. doi:10.1016/j.mib.2017.05.008

P09.06'Anenhanced CRISPR Tool for Treating Chronic Myelogenous Leukemia. D Lainšček, V Forstnerič, S Malenšek, M Skrbinek, M Sever, R Jerala, J. Immunotherapy Cancer. Available. Lainšček, D., Forstnerič, V., Malenšek, S., Skrbinek, M., Sever, M., and Jerala, R. (2020). P09.06'Anenhanced CRISPR Tool for Treating Chronic Myelogenous Leukemia'. J. Immunotherapy Cancer. Available at: https://jitc.bmj.com/ content/8/Suppl_2/A54.2.

CRISPR-mediated TCR Replacement Generates superior Anticancer Transgenic T Cells. M Legut, G Dolton, A A Mian, O G Ottmann, A K Sewell, 10.1182/blood-2017-05-787598Blood. 1313Legut, M., Dolton, G., Mian, A. A., Ottmann, O. G., and Sewell, A. K. (2018). CRISPR-mediated TCR Replacement Generates superior Anticancer Transgenic T Cells. Blood 131 (3), 311-322. doi:10.1182/blood-2017-05-787598

The Different Epidemiologic Subtypes of Burkitt Lymphoma Share a Homogenous Micro RNA Profile Distinct from Diffuse Large B-Cell Lymphoma. D Lenze, L Leoncini, M Hummel, S Volinia, C G Liu, T Amato, 10.1038/leu.2011.156Leukemia. 2512Lenze, D., Leoncini, L., Hummel, M., Volinia, S., Liu, C. G., Amato, T., et al. (2011). The Different Epidemiologic Subtypes of Burkitt Lymphoma Share a Homogenous Micro RNA Profile Distinct from Diffuse Large B-Cell Lymphoma. Leukemia 25 (12), 1869-1876. doi:10.1038/leu.2011.156

EZH2-Deficient T-Cell Acute Lymphoblastic Leukemia Is Sensitized to CHK1 Inhibition through Enhanced Replication Stress. T E León, -D&apos; Rapoz, T Silva, C Bertoli, S Rahman, M Magnussen, B Philip, 10.1158/2159-8290.cd-19-0789Cancer Discov. 107León, T. E., Rapoz-D'Silva, T., Bertoli, C., Rahman, S., Magnussen, M., Philip, B., et al. (2020). EZH2-Deficient T-Cell Acute Lymphoblastic Leukemia Is Sensitized to CHK1 Inhibition through Enhanced Replication Stress. Cancer Discov. 10 (7), 998-1017. doi:10.1158/2159-8290.cd-19-0789

A Genome-scale CRISPR Knock-out Screen in Chronic Myeloid Leukemia Identifies Novel Drug Resistance Mechanisms along with Intrinsic Apoptosis and MAPK Signaling. M Lewis, V Prouzet-Mauléon, F Lichou, E Richard, R Iggo, B Turcq, 10.1002/cam4.3231doi:10.1002/ cam4.3231Cancer Med. 918Lewis, M., Prouzet-Mauléon, V., Lichou, F., Richard, E., Iggo, R., Turcq, B., et al. (2020). A Genome-scale CRISPR Knock-out Screen in Chronic Myeloid Leukemia Identifies Novel Drug Resistance Mechanisms along with Intrinsic Apoptosis and MAPK Signaling. Cancer Med. 9 (18), 6739-6751. doi:10.1002/ cam4.3231

Applications of Genome Editing Technology in the Targeted Therapy of Human Diseases: Mechanisms, Advances and Prospects. H Li, Y Yang, W Hong, M Huang, M Wu, X Zhao, 10.1038/s41392-019-0089-ySig Transduct Target. Ther. 51Li, H., Yang, Y., Hong, W., Huang, M., Wu, M., and Zhao, X. (2020). Applications of Genome Editing Technology in the Targeted Therapy of Human Diseases: Mechanisms, Advances and Prospects. Sig Transduct Target. Ther. 5 (1), 1-23. doi:10.1038/s41392-019-0089-y

Targeted Transcriptional Activation in Plants Using a Potent Dead Cas9-Derived Synthetic Gene Activator. Z Li, F Wang, J F Li, 10.1002/cpmb.89Curr. Protoc. Mol. Biol. 127189Li, Z., Wang, F., and Li, J. F. (2019). Targeted Transcriptional Activation in Plants Using a Potent Dead Cas9-Derived Synthetic Gene Activator. Curr. Protoc. Mol. Biol. 127 (1), e89. doi:10.1002/cpmb.89

An In Vivo CRISPR Screening Platform for Prioritizing Therapeutic Targets in AML. Cancer Discov. S Lin, C Larrue, N K Scheidegger, B K A Seong, N V Dharia, M Kuljanin, citedLin, S., Larrue, C., Scheidegger, N. K., Seong, B. K. A., Dharia, N. V., Kuljanin, M., et al. (2021). An In Vivo CRISPR Screening Platform for Prioritizing Therapeutic Targets in AML. Cancer Discov. [Internet]. 2021 Jan 1 [cited

CRISPR/Cas9 Systems Have Off-Target Activity with Insertions or Deletions between Target DNA and Guide RNA Sequences. Y Lin, T J Cradick, M T Brown, H Deshmukh, P Ranjan, N Sarode, 10.1093/nar/gku402Nucleic Acids Res. 4211Lin, Y., Cradick, T. J., Brown, M. T., Deshmukh, H., Ranjan, P., Sarode, N., et al. (2014). CRISPR/Cas9 Systems Have Off-Target Activity with Insertions or Deletions between Target DNA and Guide RNA Sequences. Nucleic Acids Res. 42 (11), 7473-7485. doi:10.1093/nar/gku402

Delivering CRISPR: a Review of the Challenges and Approaches. C A Lino, J C Harper, J P Carney, J A Timlin, 10.1080/10717544.2018.1474964Drug Deliv. 251Lino, C. A., Harper, J. C., Carney, J. P., and Timlin, J. A. (2018). Delivering CRISPR: a Review of the Challenges and Approaches. Drug Deliv. 25 (1), 1234-1257. doi:10.1080/10717544.2018.1474964

How I Treat T-Cell Acute Lymphoblastic Leukemia in Adults. M R Litzow, A A Ferrando, 10.1182/blood-2014-10-551895doi:10.1182/ blood-2014-10-551895Blood. 1267Litzow, M. R., and Ferrando, A. A. (2015). How I Treat T-Cell Acute Lymphoblastic Leukemia in Adults. Blood 126 (7), 833-841. doi:10.1182/ blood-2014-10-551895

Cell-Penetrating Peptide-Mediated Delivery of TALEN Proteins via Bioconjugation for Genome Engineering. J Liu, T Gaj, J T Patterson, S J Sirk, Iii Barbas, C F , 10.1371/journal.pone.0085755doi:10.1371/ journal.pone.0085755PLOS ONE. 9185755Liu, J., Gaj, T., Patterson, J. T., Sirk, S. J., and Barbas III, C. F. (2014). Cell- Penetrating Peptide-Mediated Delivery of TALEN Proteins via Bioconjugation for Genome Engineering. PLOS ONE 9 (1), e85755. doi:10.1371/ journal.pone.0085755

CRISPR/Cas9 Genome Editing: Fueling the Revolution in Cancer Immunotherapy. X Liu, Y Zhao, 10.1016/j.retram.2018.04.003Curr. Res. Translational Med. 662Liu, X., and Zhao, Y. (2018). CRISPR/Cas9 Genome Editing: Fueling the Revolution in Cancer Immunotherapy. Curr. Res. Translational Med. 66 (2), 39-42. doi:10.1016/j.retram.2018.04.003

Systemic Delivery of CRISPR/Cas9 with PEG-PLGA Nanoparticles for Chronic Myeloid Leukemia Targeted Therapy. Y Liu, G Zhao, C.-F Xu, Y.-L Luo, Z.-D Lu, Wang , J , 10.1039/c8bm00263kdoi:10.1039/ c8bm00263kBiomater. Sci. 66Liu, Y., Zhao, G., Xu, C.-F., Luo, Y.-L., Lu, Z.-D., and Wang, J. (2018). Systemic Delivery of CRISPR/Cas9 with PEG-PLGA Nanoparticles for Chronic Myeloid Leukemia Targeted Therapy. Biomater. Sci. 6 (6), 1592-1603. doi:10.1039/ c8bm00263k

Utility of CRISPR/Cas9 Systems in Hematology Research. D Lucas, H A O&apos;leary, B L Ebert, C A Cowan, C S Tremblay, 10.1016/j.exphem.2017.06.006Exp. Hematol. 54Lucas, D., O'Leary, H. A., Ebert, B. L., Cowan, C. A., and Tremblay, C. S. (2017). Utility of CRISPR/Cas9 Systems in Hematology Research. Exp. Hematol. 54, 1-3. doi:10.1016/j.exphem.2017.06.006

. H Ma, L.-C Tu, A Naseri, M Huisman, S Zhang, D Grunwald, Ma, H., Tu, L.-C., Naseri, A., Huisman, M., Zhang, S., Grunwald, D., et al. (2016).

Multiplexed Labeling of Genomic Loci with dCas9 and Engineered sgRNAs Using CRISPRainbow. 10.1038/nbt.3526doi:10.1038/ nbt.3526Nat. Biotechnol. 345Multiplexed Labeling of Genomic Loci with dCas9 and Engineered sgRNAs Using CRISPRainbow. Nat. Biotechnol. 34 (5), 528-530. doi:10.1038/ nbt.3526

Robust, Synergistic Regulation of Human Gene Expression Using TALE Activators. M L Maeder, S J Linder, D Reyon, J F Angstman, Y Fu, J D Sander, 10.1038/nmeth.2366Nat. Methods. 103Maeder, M. L., Linder, S. J., Reyon, D., Angstman, J. F., Fu, Y., Sander, J. D., et al. (2013). Robust, Synergistic Regulation of Human Gene Expression Using TALE Activators. Nat. Methods 10 (3), 243-245. doi:10.1038/nmeth.2366

Notch2 Controls Non-autonomous Wnt-Signalling in Chronic Lymphocytic Leukaemia. M Mangolini, F Götte, A Moore, T Ammon, M Oelsner, G Lutzny-Geier, 10.1038/s41467-018-06069-5Nat. Commun. 913839Mangolini, M., Götte, F., Moore, A., Ammon, T., Oelsner, M., Lutzny-Geier, G., et al. (2018). Notch2 Controls Non-autonomous Wnt-Signalling in Chronic Lymphocytic Leukaemia. Nat. Commun. 9 (1), 3839. doi:10.1038/s41467-018- 06069-5

Chronic Lymphocytic Leukaemia in 2020: the Future Has Arrived. K Milne, B Sturrock, T Chevassut, 10.1007/s11912-020-0893-0doi:10.1007/ s11912-020-0893-0Curr. Oncol. Rep. 22436Milne, K., Sturrock, B., and Chevassut, T. (2020). Chronic Lymphocytic Leukaemia in 2020: the Future Has Arrived. Curr. Oncol. Rep. 22 (4), 36. doi:10.1007/ s11912-020-0893-0

ETV6/RUNX1 Fusion Gene Abrogation Decreases the Oncogenicity of Tumour Cells in a Preclinical Model of. A Montaño, J L Ordoñez, V Alonso-Pérez, J Hernández-Sánchez, S Santos, T González, 10.3390/cells9010215Acute Lymphoblastic Leukaemia. Cells. 91215Montaño, A., Ordoñez, J. L., Alonso-Pérez, V., Hernández-Sánchez, J., Santos, S., González, T., et al. (2020). ETV6/RUNX1 Fusion Gene Abrogation Decreases the Oncogenicity of Tumour Cells in a Preclinical Model of Acute Lymphoblastic Leukaemia. Cells 9 (1), 215. doi:10.3390/cells9010215

Genome-wide Analysis of Genetic Alterations in Acute Lymphoblastic Leukaemia. C G Mullighan, S Goorha, I Radtke, C B Miller, E Coustan-Smith, J D Dalton, 10.1038/nature05690doi:10.1038/ nature05690Nature. 4467137Mullighan, C. G., Goorha, S., Radtke, I., Miller, C. B., Coustan-Smith, E., Dalton, J. D., et al. (2007). Genome-wide Analysis of Genetic Alterations in Acute Lymphoblastic Leukaemia. Nature 446 (7137), 758-764. doi:10.1038/ nature05690

CRISPR Screens Provide a Comprehensive Assessment of Cancer Vulnerabilities but Generate False-Positive Hits for Highly Amplified Genomic Regions. D M Munoz, P J Cassiani, L Li, E Billy, J M Korn, M D Jones, 10.1158/2159-8290.cd-16-0178Cancer Discov. 68Munoz, D. M., Cassiani, P. J., Li, L., Billy, E., Korn, J. M., Jones, M. D., et al. (2016). CRISPR Screens Provide a Comprehensive Assessment of Cancer Vulnerabilities but Generate False-Positive Hits for Highly Amplified Genomic Regions. Cancer Discov. 6 (8), 900-913. doi:10.1158/2159-8290.cd- 16-0178

Cancer Stat Facts. Myeloma, InternetMyeloma (2021). Cancer Stat Facts [Internet]. [cited 2021 Nov 2]. Available at: https://seer.cancer.gov/statfacts/html/mulmy.html.

Latest Developed Strategies to Minimize the Off-Target Effects in CRISPR-Cas-Mediated Genome Editing. M Naeem, S Majeed, M Z Hoque, Ahmad , I , 10.3390/cells9071608Cells. 97Naeem, M., Majeed, S., Hoque, M. Z., and Ahmad, I. (2020). Latest Developed Strategies to Minimize the Off-Target Effects in CRISPR-Cas-Mediated Genome Editing. Cells 9 (7), E1608. doi:10.3390/cells9071608

Knockout of BIRC5 Gene by CRISPR/Cas9 Induces Apoptosis and Inhibits Cell Proliferation in Leukemic Cell Lines, HL60 and KG1. M Narimani, M Sharifi, Jalili , A , 10.2147/blctt.s230383doi:10.2147/ blctt.s230383Blood Lymphat Cancer. 9Narimani, M., Sharifi, M., and Jalili, A. (2019). Knockout of BIRC5 Gene by CRISPR/Cas9 Induces Apoptosis and Inhibits Cell Proliferation in Leukemic Cell Lines, HL60 and KG1. Blood Lymphat Cancer 9, 53-61. doi:10.2147/ blctt.s230383

Targetable Vulnerabilities in T-and NK-Cell Lymphomas Identified through Preclinical Models. S Y Ng, N Yoshida, A L Christie, M Ghandi, N V Dharia, J Dempster, 10.1038/s41467-018-04356-9Nat. Commun. 912024Ng, S. Y., Yoshida, N., Christie, A. L., Ghandi, M., Dharia, N. V., Dempster, J., et al. (2018). Targetable Vulnerabilities in T-and NK-Cell Lymphomas Identified through Preclinical Models. Nat. Commun. 9 (1), 2024. doi:10.1038/s41467- 018-04356-9

Genome-wide CRISPR Screens Reveal Synthetic Lethal Interaction between CREBBP and EP300 in Diffuse Large B-Cell Lymphoma. M Nie, L Du, W Ren, J Joung, X Ye, X Shi, 10.1038/s41419-021-03695-8Cel Death Dis. 125419Nie, M., Du, L., Ren, W., Joung, J., Ye, X., Shi, X., et al. (2021). Genome-wide CRISPR Screens Reveal Synthetic Lethal Interaction between CREBBP and EP300 in Diffuse Large B-Cell Lymphoma. Cel Death Dis 12 (5), 419. doi:10.1038/s41419-021-03695-8

Engineered CRISPR-Cas9 Nuclease with Expanded Targeting Space. H Nishimasu, X Shi, S Ishiguro, L Gao, S Hirano, S Okazaki, 10.1126/science.aas9129Science. 3611259Nishimasu, H., Shi, X., Ishiguro, S., Gao, L., Hirano, S., Okazaki, S., et al. (2018). Engineered CRISPR-Cas9 Nuclease with Expanded Targeting Space. Science 361, 1259. doi:10.1126/science.aas9129

The Impact of Clonal Evolution on Response to Imatinib Mesylate (STI571) in Accelerated Phase CML. M E O&apos;dwyer, M J Mauro, G Kurilik, M Mori, S Balleisen, S Olson, 10.1182/blood-2002-03-0777doi:10.1182/ blood-2002-03-0777Blood. 1005O'Dwyer, M. E., Mauro, M. J., Kurilik, G., Mori, M., Balleisen, S., Olson, S., et al. (2002). The Impact of Clonal Evolution on Response to Imatinib Mesylate (STI571) in Accelerated Phase CML. Blood 100 (5), 1628-1633. doi:10.1182/ blood-2002-03-0777

Enhancer Architecture and Essential Core Regulatory Circuitry of Chronic Lymphocytic Leukemia. C J Ott, A J Federation, L S Schwartz, S Kasar, J L Klitgaard, R Lenci, 10.1016/j.ccell.2018.11.001Cancer Cell. 346Ott, C. J., Federation, A. J., Schwartz, L. S., Kasar, S., Klitgaard, J. L., Lenci, R., et al. (2018). Enhancer Architecture and Essential Core Regulatory Circuitry of Chronic Lymphocytic Leukemia. Cancer Cell 34 (6), 982-995. e7. doi:10.1016/j.ccell.2018.11.001

Introduction to a Review Series on Emerging Immunotherapies for Hematologic Diseases. S Paczesny, S Z Pavletic, C M Bollard, 10.1182/blood-2018-04-840793Blood. 13124Paczesny, S., Pavletic, S. Z., and Bollard, C. M. (2018). Introduction to a Review Series on Emerging Immunotherapies for Hematologic Diseases. Blood 131 (24), 2617-2620. doi:10.1182/blood-2018-04-840793

Oncogenes and Tumor Suppressors in the Molecular Pathogenesis of Acute Promyelocytic Leukemia. P P Pandolfi, 10.1093/hmg/10.7.769Hum. Mol. Genet. 107Pandolfi, P. P. (2001). Oncogenes and Tumor Suppressors in the Molecular Pathogenesis of Acute Promyelocytic Leukemia. Hum. Mol. Genet. 10 (7), 769-775. doi:10.1093/hmg/10.7.769

The Whole-Genome Landscape of Burkitt Lymphoma Subtypes. R I Panea, C L Love, J R Shingleton, A Reddy, J A Bailey, A M Moormann, 10.1182/blood.2019001880Blood. 134Panea, R. I., Love, C. L., Shingleton, J. R., Reddy, A., Bailey, J. A., Moormann, A. M., et al. (2019). The Whole-Genome Landscape of Burkitt Lymphoma Subtypes. Blood 134 (19), 1598-1607. doi:10.1182/blood.2019001880

Efficient Introduction of Specific Homozygous and Heterozygous Mutations Using CRISPR/Cas9. D Paquet, D Kwart, A Chen, A Sproul, S Jacob, S Teo, 10.1038/nature17664Nature. 5337601Paquet, D., Kwart, D., Chen, A., Sproul, A., Jacob, S., Teo, S., et al. (2016). Efficient Introduction of Specific Homozygous and Heterozygous Mutations Using CRISPR/Cas9. Nature 533 (7601), 125-129. doi:10.1038/nature17664

Genome-wide CRISPR Screens Reveal Genetic Mediators of Cereblon Modulator Toxicity in Primary Effusion Lymphoma. A Patil, M Manzano, E Gottwein, 10.1182/bloodadvances.2019031732doi:10.1182/ bloodadvances.2019031732Blood Adv. 314Patil, A., Manzano, M., and Gottwein, E. (2019). Genome-wide CRISPR Screens Reveal Genetic Mediators of Cereblon Modulator Toxicity in Primary Effusion Lymphoma. Blood Adv. 3 (14), 2105-2117. doi:10.1182/ bloodadvances.2019031732

CRISPR-associated (CAS) Effectors Delivery via Microfluidic Cell-Deformation Chip. N Peyravian, M Malekzadeh Kebria, J Kiani, P Brouki Milan, M Mozafari, 10.3390/ma14123164Materials. 14123164Peyravian, N., Malekzadeh Kebria, M., Kiani, J., Brouki Milan, P., and Mozafari, M. (2021). CRISPR-associated (CAS) Effectors Delivery via Microfluidic Cell- Deformation Chip. Materials 14 (12), 3164. doi:10.3390/ma14123164

A Multiprotein Supercomplex Controlling Oncogenic Signalling in Lymphoma. J D Phelan, R M Young, D E Webster, S Roulland, G W Wright, M Kasbekar, 10.1038/s41586-018-0290-0Nature. 5607718Phelan, J. D., Young, R. M., Webster, D. E., Roulland, S., Wright, G. W., Kasbekar, M., et al. (2018). A Multiprotein Supercomplex Controlling Oncogenic Signalling in Lymphoma. Nature 560 (7718), 387-391. doi:10.1038/s41586- 018-0290-0

Broad Targeting Specificity during Bacterial Type III CRISPR-Cas Immunity Constrains Viral Escape. N C Pyenson, K Gayvert, A Varble, O Elemento, L A Marraffini, 10.1016/j.chom.2017.07.016e3. doi:10.1016/ j.chom.2017.07.016Cell Host & Microbe. 223Pyenson, N. C., Gayvert, K., Varble, A., Elemento, O., and Marraffini, L. A. (2017). Broad Targeting Specificity during Bacterial Type III CRISPR-Cas Immunity Constrains Viral Escape. Cell Host & Microbe 22 (3), 343-353. e3. doi:10.1016/ j.chom.2017.07.016

Biological significance of monoallelic and biallelic BIRC3 loss in del(11q) chronic lymphocytic leukemia progression. M Quijada-Álamo, M Hernández-Sánchez, A-E Rodríguez-Vicente, C Pérez-Carretero, A Rodríguez-Sánchez, M Martín-Izquierdo, 10.1038/s41408-021-00520-5Blood Cancer J. 117Quijada-Álamo, M., Hernández-Sánchez, M., Rodríguez-Vicente, A-E., Pérez- Carretero, C., Rodríguez-Sánchez, A., Martín-Izquierdo, M., et al. (2021). Biological significance of monoallelic and biallelic BIRC3 loss in del(11q) chronic lymphocytic leukemia progression. Blood Cancer J. 11 (7), 1-11. doi:10.1038/s41408-021-00520-5

Dissecting the role of TP53 alterations in del(11q) chronic lymphocytic leukemia. M Quijada-Álamo, C Pérez-Carretero, M Hernández-Sánchez, A-E Rodríguez-Vicente, A-B Herrero, J-M Hernández-Sánchez, e304. Quijada-Álamo. 11Quijada-Álamo, M., Pérez-Carretero, C., Hernández-Sánchez, M., Rodríguez- Vicente, A-E., Herrero, A-B., Hernández-Sánchez, J-M., et al. (2021). Dissecting the role of TP53 alterations in del(11q) chronic lymphocytic leukemia. Clin. Transl Med. 11 (2), e304. Quijada-Álamo, M., Hernández-Sánchez, M., Alonso-Pérez, V., Rodríguez- Vicente, A. E., García-Tuñón, I., Martín-Izquierdo, M., et al. (2020).

Frontiers in Genetics | www.frontiersin.org. 12767298Frontiers in Genetics | www.frontiersin.org January 2022 | Volume 12 | Article 767298

CRISPR/Cas9-generated models uncover therapeutic vulnerabilities of del(11q) CLL cells to dual BCR and PARP inhibition. 10.1038/s41375-020-0714-3Leukemia. 346CRISPR/Cas9-generated models uncover therapeutic vulnerabilities of del(11q) CLL cells to dual BCR and PARP inhibition. Leukemia 34 (6), 1599-1612. doi:10.1038/s41375-020-0714-3

Activation of the LMO2 Oncogene through a Somatically Acquired Neomorphic Promoter in T-Cell Acute Lymphoblastic Leukemia. S Rahman, M Magnussen, T E León, N Farah, Z Li, B J Abraham, 10.1182/blood-2016-09-742148Blood. 12924Rahman, S., Magnussen, M., León, T. E., Farah, N., Li, Z., Abraham, B. J., et al. (2017). Activation of the LMO2 Oncogene through a Somatically Acquired Neomorphic Promoter in T-Cell Acute Lymphoblastic Leukemia. Blood 129 (24), 3221-3226. doi:10.1182/blood-2016-09-742148

CRISPR-based Screens Uncover Determinants of Immunotherapy Response in Multiple Myeloma. P Ramkumar, A B Abarientos, R Tian, M Seyler, J T Leong, M Chen, 10.1182/bloodadvances.2019001346doi:10.1182/ bloodadvances.2019001346Blood Adv. 413Ramkumar, P., Abarientos, A. B., Tian, R., Seyler, M., Leong, J. T., Chen, M., et al. (2020). CRISPR-based Screens Uncover Determinants of Immunotherapy Response in Multiple Myeloma. Blood Adv. 4 (13), 2899-2911. doi:10.1182/ bloodadvances.2019001346

Chromosomal Abnormalities in Embryonic and Somatic Stem Cells. P Rebuzzini, M Zuccotti, C A Redi, S Garagna, 10.1159/000441645Cytogenet. Genome Res. 147Rebuzzini, P., Zuccotti, M., Redi, C. A., and Garagna, S. (2015). Chromosomal Abnormalities in Embryonic and Somatic Stem Cells. Cytogenet. Genome Res. 147, 1. doi:10.1159/000441645

Genetic and Functional Drivers of Diffuse Large B Cell Lymphoma. A Reddy, J Zhang, N S Davis, A B Moffitt, C L Love, A Waldrop, 10.1016/j.cell.2017.09.027Cell. 1712Reddy, A., Zhang, J., Davis, N. S., Moffitt, A. B., Love, C. L., Waldrop, A., et al. (2017). Genetic and Functional Drivers of Diffuse Large B Cell Lymphoma. Cell 171 (2), 481-494. e15. doi:10.1016/j.cell.2017.09.027

CRISPR Used to Model Acute Myeloid Leukaemia Progression. V Rees, Drug Target ReviewRees, V. (2021). CRISPR Used to Model Acute Myeloid Leukaemia Progression [Internet]. Drug Target Review. [cited 2021 Nov 1]. Available at: https://www. drugtargetreview.com/news/83381/crispr-used-to-model-acute-myeloid- leukaemia-progression/.

Multiplex Genome Editing to Generate Universal CAR T Cells Resistant to PD1 Inhibition. J Ren, X Liu, C Fang, S Jiang, C H June, Y Zhao, 10.1158/1078-0432.ccr-16-1300Clin. Cancer Res. 239Ren, J., Liu, X., Fang, C., Jiang, S., June, C. H., and Zhao, Y. (2017). Multiplex Genome Editing to Generate Universal CAR T Cells Resistant to PD1 Inhibition. Clin. Cancer Res. 23 (9), 2255-2266. doi:10.1158/1078-0432.ccr- 16-1300

Modeling Pediatric AML FLT3 Mutations Using CRISPR/Cas12a-Mediated Gene Editing. N Rivera-Torres, K Banas, E B Kmiec, 10.1080/10428194.2020.1805740Leuk. Lymphoma. 6113Rivera-Torres, N., Banas, K., and Kmiec, E. B. (2020). Modeling Pediatric AML FLT3 Mutations Using CRISPR/Cas12a-Mediated Gene Editing. Leuk. Lymphoma 61 (13), 3078-3088. doi:10.1080/10428194.2020.1805740

Monocytic Differentiation and AHR Signaling as Primary Nodes of BET Inhibitor Response in Acute Myeloid Leukemia. K A Romine, T Nechiporuk, D Bottomly, S Jeng, S K Mcweeney, A Kaempf, 10.1158/2643-3230.bcd-21-0012Blood Cancer Discov. 25Romine, K. A., Nechiporuk, T., Bottomly, D., Jeng, S., McWeeney, S. K., Kaempf, A., et al. (2021). Monocytic Differentiation and AHR Signaling as Primary Nodes of BET Inhibitor Response in Acute Myeloid Leukemia. Blood Cancer Discov. 2 (5), 518-531. doi:10.1158/2643-3230.bcd-21-0012

. K R Roy, J D Smith, S C Vonesch, G Lin, C S Tu, A R Lederer, Roy, K. R., Smith, J. D., Vonesch, S. C., Lin, G., Tu, C. S., Lederer, A. R., et al. (2018).

Multiplexed Precision Genome Editing with Trackable Genomic Barcodes in Yeast. 10.1038/nbt.4137Nat. Biotechnol. 366Multiplexed Precision Genome Editing with Trackable Genomic Barcodes in Yeast. Nat. Biotechnol. 36 (6), 512-520. doi:10.1038/nbt.4137

Chronic Myeloid Leukemia Prognosis and Therapy: Criticisms and Perspectives. D Russo, J V Garcia-Gutierrez, S Soverini, M Baccarani, 10.3390/jcm9061709J. Clin. Med. 96Russo, D., Garcia-Gutierrez, J. V., Soverini, S., and Baccarani, M. (2020). Chronic Myeloid Leukemia Prognosis and Therapy: Criticisms and Perspectives. J. Clin. Med. 9 (6), E1709. doi:10.3390/jcm9061709

Programmable Molecular Scissors: Applications of a New Tool for. S K Saha, F K Saikot, M S Rahman, M A H M Jamal, S M K Rahman, S M R Islam, 10.1016/j.omtn.2018.11.016Genome Editing in Biotech. Mol. Ther. -Nucleic Acids. 14Saha, S. K., Saikot, F. K., Rahman, M. S., Jamal, M. A. H. M., Rahman, S. M. K., Islam, S. M. R., et al. (2019). Programmable Molecular Scissors: Applications of a New Tool for Genome Editing in Biotech. Mol. Ther. -Nucleic Acids 14, 212-238. doi:10.1016/j.omtn.2018.11.016

CRISPR/Cas9 Gene Editing: From Basic Mechanisms to Improved Strategies for Enhanced Genome Engineering In Vivo. J Salsman, J Y Masson, A Orthwein, G Dellaire, 10.2174/1566523217666171122094629doi:10.2174/ 1566523217666171122094629Curr. Gene Ther. 174Salsman, J., Masson, J. Y., Orthwein, A., and Dellaire, G. (2017). CRISPR/Cas9 Gene Editing: From Basic Mechanisms to Improved Strategies for Enhanced Genome Engineering In Vivo. Curr. Gene Ther. 17 (4), 263-274. doi:10.2174/ 1566523217666171122094629

Genetics and Pathogenesis of Diffuse Large B-Cell Lymphoma. R Schmitz, G W Wright, D W Huang, C A Johnson, J D Phelan, J Q Wang, 10.1056/NEJMoa1801445New Engl. J. Med. 3781396Schmitz, R., Wright, G. W., Huang, D. W., Johnson, C. A., Phelan, J. D., Wang, J. Q., et al. (2018). Genetics and Pathogenesis of Diffuse Large B-Cell Lymphoma. New Engl. J. Med. 378, 1396. doi:10.1056/NEJMoa1801445

Genome-scale CRISPR-Cas9 Knockout Screening in Human Cells. O Shalem, N E Sanjana, E Hartenian, X Shi, D A Scott, T S Mikkelsen, 10.1126/science.1247005Science. 3436166Shalem, O., Sanjana, N. E., Hartenian, E., Shi, X., Scott, D. A., Mikkelsen, T. S., et al. (2014). Genome-scale CRISPR-Cas9 Knockout Screening in Human Cells. Science 343 (6166), 84-87. doi:10.1126/science.1247005

Response to Imatinib Mesylate in Chronic Myeloid Leukemia Patients with Variant BCR-ABL Fusion Transcripts. P Sharma, L Kumar, S Mohanty, V Kochupillai, 10.1007/s00277-009-0822-7Ann. Hematol. 893Sharma, P., Kumar, L., Mohanty, S., and Kochupillai, V. (2010). Response to Imatinib Mesylate in Chronic Myeloid Leukemia Patients with Variant BCR- ABL Fusion Transcripts. Ann. Hematol. 89 (3), 241-247. doi:10.1007/s00277- 009-0822-7

Genomic Amplifications Cause False Positives in CRISPR Screens. A Sheel, W Xue, 10.1158/2159-8290.CD-16-0665Cancer Discov. 6824Sheel, A., and Xue, W. (2016). Genomic Amplifications Cause False Positives in CRISPR Screens. Cancer Discov. 6, 824. doi:10.1158/2159-8290.CD-16-0665

CRISPR Genome-wide Screening Identifies Dependence on the Proteasome Subunit PSMC6 for Bortezomib Sensitivity in Multiple Myeloma. C.-X Shi, K M Kortüm, Y X Zhu, L A Bruins, P Jedlowski, P G Votruba, 10.1158/1535-7163.mct-17-0130Mol. Cancer Ther. 1612Shi, C.-X., Kortüm, K. M., Zhu, Y. X., Bruins, L. A., Jedlowski, P., Votruba, P. G., et al. (2017). CRISPR Genome-wide Screening Identifies Dependence on the Proteasome Subunit PSMC6 for Bortezomib Sensitivity in Multiple Myeloma. Mol. Cancer Ther. 16 (12), 2862-2870. doi:10.1158/1535- 7163.mct-17-0130

Rationally Engineered Cas9 Nucleases with Improved Specificity. I M Slaymaker, L Gao, B Zetsche, D A Scott, W X Yan, F Zhang, 10.1126/science.aad5227Science. 3516268Slaymaker, I. M., Gao, L., Zetsche, B., Scott, D. A., Yan, W. X., and Zhang, F. (2016). Rationally Engineered Cas9 Nucleases with Improved Specificity. Science 351 (6268), 84-88. doi:10.1126/science.aad5227

CRISPR Screen Identifies Genes that Sensitize AML Cells to Double-Negative T-Cell Therapy. F Soares, B Chen, J B Lee, M Ahmed, D Ly, E Tin, 10.1182/blood.2019004108Blood. 13716Soares, F., Chen, B., Lee, J. B., Ahmed, M., Ly, D., Tin, E., et al. (2021). CRISPR Screen Identifies Genes that Sensitize AML Cells to Double-Negative T-Cell Therapy. Blood 137 (16), 2171-2181. doi:10.1182/blood.2019004108

. A Tagde, H Rajabi, A Bouillez, M Alam, R Gali, S Bailey, Tagde, A., Rajabi, H., Bouillez, A., Alam, M., Gali, R., Bailey, S., et al. (2016).

. Myc Muc1-C Drives, In Multiple, Myeloma, 10.1182/blood-2015-07-659151Blood. 12721MUC1-C Drives MYC in Multiple Myeloma. Blood 127 (21), 2587-2597. doi:10.1182/blood-2015-07-659151

Type III CRISPR-Cas Immunity: Major Differences Brushed Aside. G Tamulaitis, Č Venclovas, V Siksnys, 10.1016/j.tim.2016.09.012Trends Microbiol. 251Tamulaitis, G., Venclovas, Č., and Siksnys, V. (2017). Type III CRISPR-Cas Immunity: Major Differences Brushed Aside. Trends Microbiol. 25 (1), 49-61. doi:10.1016/j.tim.2016.09.012

CRISPR/ Cas9-mediated Gene Deletion Efficiently Retards the Progression of Philadelphia-positive Acute Lymphoblastic Leukemia in a P210 BCR-Abl1t315i Mutation Mouse Model. Y.-T Tan, L Ye, F Xie, J Wang, M Müschen, S.-J Chen, 10.3324/haematol.2019.229013Haematologica. 1055Tan, Y.-T., Ye, L., Xie, F., Wang, J., Müschen, M., Chen, S.-J., et al. (2020). CRISPR/ Cas9-mediated Gene Deletion Efficiently Retards the Progression of Philadelphia-positive Acute Lymphoblastic Leukemia in a P210 BCR- Abl1t315i Mutation Mouse Model. Haematologica 105 (5), e232-e236. doi:10.3324/haematol.2019.229013

Diagnosis and Classification of Hematologic Malignancies on the Basis of Genetics. J Taylor, W Xiao, O Wahab, 10.1182/blood-2017-02-734541Blood. 1304Taylor, J., Xiao, W., and Abdel-Wahab, O. (2017). Diagnosis and Classification of Hematologic Malignancies on the Basis of Genetics. Blood 130 (4), 410-423. doi:10.1182/blood-2017-02-734541

Acute Lymphoblastic Leukemia: a Comprehensive Review and 2017 Update. T Terwilliger, M Hay, 10.1038/bcj.2017.53Blood Cancer J. 76577Terwilliger, T., and Abdul-Hay, M. (2017). Acute Lymphoblastic Leukemia: a Comprehensive Review and 2017 Update. Blood Cancer J. 7 (6), e577. doi:10.1038/bcj.2017.53

Therapy of Chronic Myeloid Leukemia: Twilight of the Imatinib Era? ISRN Oncol. E Trela, S Glowacki, J Błasiak, 10.1155/2014/596483596483Trela, E., Glowacki, S., and Błasiak, J. (2014). Therapy of Chronic Myeloid Leukemia: Twilight of the Imatinib Era? ISRN Oncol. 2014, 596483. doi:10.1155/2014/596483

Defining and Improving the Genome-wide Specificities of CRISPR-Cas9 Nucleases. S Q Tsai, J K Joung, 10.1038/nrg.2016.28Nat. Rev. Genet. 175Tsai, S. Q., and Joung, J. K. (2016). Defining and Improving the Genome-wide Specificities of CRISPR-Cas9 Nucleases. Nat. Rev. Genet. 17 (5), 300-312. doi:10.1038/nrg.2016.28

Methods for Optimizing CRISPR-Cas9 Genome Editing Specificity. J Tycko, V E Myer, P D Hsu, 10.1016/j.molcel.2016.07.004doi:10.1016/ j.molcel.2016.07.004Mol. Cel. 633Tycko, J., Myer, V. E., and Hsu, P. D. (2016). Methods for Optimizing CRISPR- Cas9 Genome Editing Specificity. Mol. Cel 63 (3), 355-370. doi:10.1016/ j.molcel.2016.07.004

A CRISPR Dropout Screen Identifies Genetic Vulnerabilities and Therapeutic Targets in Acute Myeloid Leukemia. K Tzelepis, H Koike-Yusa, E De Braekeleer, Y Li, O M Dovey, A Mupo, Cell. 17Tzelepis, K., Koike-Yusa, H., De Braekeleer, E., Li, Y., Dovey, O. M., and Mupo, A. (2016). A CRISPR Dropout Screen Identifies Genetic Vulnerabilities and Therapeutic Targets in Acute Myeloid Leukemia. Cell 17, 1193, 2016 . Available at: https://www.sciencedirect.com/science/article/pii/ S2211124716313353.

ASXL1 Mutation Correction by CRISPR/Cas9 Restores Gene Function in Leukemia Cells and Increases Survival in Mouse Xenografts. S Valletta, H Dolatshad, M Bartenstein, B H Yip, E Bello, S Gordon, 10.18632/oncotarget.6392Oncotarget. 642Valletta, S., Dolatshad, H., Bartenstein, M., Yip, B. H., Bello, E., Gordon, S., et al. (2015). ASXL1 Mutation Correction by CRISPR/Cas9 Restores Gene Function in Leukemia Cells and Increases Survival in Mouse Xenografts. Oncotarget 6 (42), 44061-44071. doi:10.18632/oncotarget.6392

Correction to: CRISPR-Based Diagnosis of Infectious and Noninfectious Diseases. S J Vangah, C Katalani, H A Boone, A Hajizade, A Sijercic, Ahmadian , G , 10.1186/s12575-020-00136-2Biol. Proced. Online. 22124Vangah, S. J., Katalani, C., Boone, H. A., Hajizade, A., Sijercic, A., and Ahmadian, G. (2020). Correction to: CRISPR-Based Diagnosis of Infectious and Noninfectious Diseases. Biol. Proced. Online 22 (1), 24. doi:10.1186/s12575- 020-00136-2

Studying DNA Double-Strand Break Repair: An Ever-Growing Toolbox. A C Vítor, P Huertas, G Legube, S F De Almeida, 10.3389/fmolb.2020.00024Front. Mol. Biosci. 724Vítor, A. C., Huertas, P., Legube, G., and de Almeida, S. F. (2020). Studying DNA Double-Strand Break Repair: An Ever-Growing Toolbox. Front. Mol. Biosci. 7, 24. doi:10.3389/fmolb.2020.00024

E Vuelta, J L Ordoñez, V Alonso-Pérez, L Méndez, P Hernández-Carabias, R Saldaña, CRISPR/Cas9 Technology Abolishes the BCR/ABL1. Vuelta, E., Ordoñez, J. L., Alonso-Pérez, V., Méndez, L., Hernández-Carabias, P., Saldaña, R., et al. (2020). CRISPR/Cas9 Technology Abolishes the BCR/ABL1

https:/www.biorxiv.org/content/10.1101/2020.08.05.237610v1Chronic Myeloid Leukemia and Restores normal Hematopoiesis. InternetOncogene Effect in Chronic Myeloid Leukemia and Restores normal Hematopoiesis [Internet]. Available at: https://www.biorxiv.org/content/10. 1101/2020.08.05.237610v1.

Genome-Wide CRISPR-Cas9 Screen Identifies MicroRNAs that Regulate Myeloid Leukemia Cell Growth. J Wallace, R Hu, T L Mosbruger, T J Dahlem, W Z Stephens, D S Rao, 10.1371/journal.pone.0153689Plos one. 11Wallace, J., Hu, R., Mosbruger, T. L., Dahlem, T. J., Stephens, W. Z., Rao, D. S., et al. (2016). Genome-Wide CRISPR-Cas9 Screen Identifies MicroRNAs that Regulate Myeloid Leukemia Cell Growth. Plos one 11, e0153689. doi:10.1371/journal.pone.0153689

Genome-Wide CRISPR-Cas9 Screen Identifies MicroRNAs that Regulate Myeloid Leukemia Cell Growth. J Wallace, R Hu, T L Mosbruger, T J Dahlem, W Z Stephens, D S Rao, 10.1371/journal.pone.0153689PLOS ONE. 114153689Wallace, J., Hu, R., Mosbruger, T. L., Dahlem, T. J., Stephens, W. Z., Rao, D. S., et al. (2016). Genome-Wide CRISPR-Cas9 Screen Identifies MicroRNAs that Regulate Myeloid Leukemia Cell Growth. PLOS ONE 11 (4), e0153689. doi:10.1371/journal.pone.0153689

Targeting an RNA-Binding Protein Network in Acute Myeloid Leukemia. E Wang, S X Lu, A Pastore, X Chen, J Imig, S Chun-Wei Lee, 10.1016/j.ccell.2019.01.010e7. doi:10.1016/ j.ccell.2019.01.010Cancer Cell. 353Wang, E., Lu, S. X., Pastore, A., Chen, X., Imig, J., Chun-Wei Lee, S., et al. (2019). Targeting an RNA-Binding Protein Network in Acute Myeloid Leukemia. Cancer Cell 35 (3), 369-384. e7. doi:10.1016/ j.ccell.2019.01.010

Surface Antigen-Guided CRISPR Screens Identify Regulators of Myeloid Leukemia Differentiation. E Wang, H Zhou, B Nadorp, G Cayanan, X Chen, A H Yeaton, 10.1016/j.stem.2020.12.005e6. doi:10.1016/ j.stem.2020.12.005Cell Stem Cell. 284Wang, E., Zhou, H., Nadorp, B., Cayanan, G., Chen, X., Yeaton, A. H., et al. (2021). Surface Antigen-Guided CRISPR Screens Identify Regulators of Myeloid Leukemia Differentiation. Cell Stem Cell 28 (4), 718-731. e6. doi:10.1016/ j.stem.2020.12.005

Sequential CRISPR Gene Editing in Human iPSCs Charts the Clonal Evolution of Myeloid Leukemia and Identifies Early Disease Frontiers in Genetics | www. T Wang, A R Pine, A G Kotini, H Yuan, L Zamparo, D T Starczynowski, 12767298Wang, T., Pine, A. R., Kotini, A. G., Yuan, H., Zamparo, L., Starczynowski, D. T., et al. (2021). Sequential CRISPR Gene Editing in Human iPSCs Charts the Clonal Evolution of Myeloid Leukemia and Identifies Early Disease Frontiers in Genetics | www.frontiersin.org January 2022 | Volume 12 | Article 767298

. Targets, 10.1016/j.stem.2021.01.011e7. doi:10.1016/ j.stem.2021.01.011Cell Stem Cell. 286Targets. Cell Stem Cell 28 (6), 1074-1089. e7. doi:10.1016/ j.stem.2021.01.011

Genetic Screens in Human Cells Using the CRISPR-Cas9 System. T Wang, J J Wei, D M Sabatini, E S Lander, 10.1126/science.1246981Science. 3436166Wang, T., Wei, J. J., Sabatini, D. M., and Lander, E. S. (2014). Genetic Screens in Human Cells Using the CRISPR-Cas9 System. Science 343 (6166), 80-84. doi:10.1126/science.1246981

. Z Wang, H Yan, J C Boysen, C R Secreto, J Zhong, J Zhou, Wang, Z., Yan, H., Boysen, J. C., Secreto, C. R., Zhong, J., Zhou, J., et al. (2021).

B Cell Receptor Signaling Drives APOBEC3 Expression via Direct Enhancer Regulation in Chronic Lymphocytic Leukemia B Cells. 10.1101/2021.07.27.454050doi:10.1101/ 2021.07.27.454050Biorxiv. B Cell Receptor Signaling Drives APOBEC3 Expression via Direct Enhancer Regulation in Chronic Lymphocytic Leukemia B Cells. Biorxiv. doi:10.1101/ 2021.07.27.454050

Chronic Myeloid Leukemia with Variation of Translocation at (Ph). Z Wang, W Zen, F Meng, X Xin, L Luo, H Sun, ins (22;9Wang, Z., Zen, W., Meng, F., Xin, X., Luo, L., Sun, H., et al. (2015). Chronic Myeloid Leukemia with Variation of Translocation at (Ph) [ins (22;9) (Q11;

. Int. J. Clin. Exp. Pathol. 810Case Reportq21q34)]: a Case Report. Int. J. Clin. Exp. Pathol. 8 (10), 13707-13710.

CRISPR-cas Systems and the Paradox of Self-Targeting Spacers. F Wimmer, C L Beisel, 10.3389/fmicb.2019.03078Front. Microbiol. 103078Wimmer, F., and Beisel, C. L. (2020). CRISPR-cas Systems and the Paradox of Self- Targeting Spacers. Front. Microbiol. 10, 3078. doi:10.3389/fmicb.2019.03078

Custom-designed Zinc finger Nucleases: What Is Next?. J Wu, K Kandavelou, S Chandrasegaran, 10.1007/s00018-007-7206-8Cell. Mol. Life Sci. 6422Wu, J., Kandavelou, K., and Chandrasegaran, S. (2007). Custom-designed Zinc finger Nucleases: What Is Next? Cell. Mol. Life Sci. 64 (22), 2933-2944. doi:10.1007/s00018-007-7206-8

Applications and Advances of CRISPR-Cas9 in Cancer Immunotherapy. A.-L Xia, Q.-F He, J.-C Wang, J Zhu, Y.-Q Sha, B Sun, Xia, A.-L., He, Q.-F., Wang, J.-C., Zhu, J., Sha, Y.-Q., Sun, B., et al. (2019). Applications and Advances of CRISPR-Cas9 in Cancer Immunotherapy.

. 10.1136/jmedgenet-2018-105422J. Med. Genet. 561J. Med. Genet. 56 (1), 4-9. doi:10.1136/jmedgenet-2018-105422

ARHGEF12 Regulates Erythropoiesis and Is Involved in Erythroid Regeneration after Chemotherapy in Acute Lymphoblastic Leukemia Patients. Y Xie, L Gao, C Xu, L Chu, L Gao, R Wu, 10.3324/haematol.2018.210286Haematologica. 1054Xie, Y., Gao, L., Xu, C., Chu, L., Gao, L., Wu, R., et al. (2020). ARHGEF12 Regulates Erythropoiesis and Is Involved in Erythroid Regeneration after Chemotherapy in Acute Lymphoblastic Leukemia Patients. Haematologica 105 (4), 925-936. doi:10.3324/haematol.2018.210286

ARID5B Influences Antimetabolite Drug Sensitivity and Prognosis of Acute Lymphoblastic Leukemia. H Xu, X Zhao, D Bhojwani, E Shuyu, C Goodings, H Zhang, 10.1158/1078-0432Clin. Cancer Res. 26Xu, H., Zhao, X., Bhojwani, D., Shuyu, E., Goodings, C., Zhang, H., et al. (2020). ARID5B Influences Antimetabolite Drug Sensitivity and Prognosis of Acute Lymphoblastic Leukemia. Clin. Cancer Res. 26. doi:10.1158/1078- 0432

Genome-wide CRISPR-Cas9 Screen Identifies Leukemiaspecific Dependence on a Pre-mRNA Metabolic Pathway Regulated by DCPS. T Yamauchi, T Masuda, M C Canver, M Seiler, Y Semba, M Shboul, 10.1016/j.ccell.2018.01.012Cancer Cell. 333Yamauchi, T., Masuda, T., Canver, M. C., Seiler, M., Semba, Y., Shboul, M., et al. (2018). Genome-wide CRISPR-Cas9 Screen Identifies Leukemia- specific Dependence on a Pre-mRNA Metabolic Pathway Regulated by DCPS. Cancer Cell 33 (3), 386-400. e5. doi:10.1016/j.ccell.2018.01.012

Targeting Lactate Dehydrogenase A ( LDHA ) Exerts Antileukemic Effects on T-cell Acute Lymphoblastic Leukemia. H Yu, Y Yin, Y Yi, Z Cheng, W Kuang, R Li, 10.1002/cac2.12080doi:10.1002/ cac2.12080Cancer Commun. 4010Yu, H., Yin, Y., Yi, Y., Cheng, Z., Kuang, W., Li, R., et al. (2020). Targeting Lactate Dehydrogenase A ( LDHA ) Exerts Antileukemic Effects on T-cell Acute Lymphoblastic Leukemia. Cancer Commun. 40 (10), 501-517. doi:10.1002/ cac2.12080

. H Zhang, Z Chen, S S Neelapu, J Romaguera, N Mccarty, Zhang, H., Chen, Z., Neelapu, S. S., Romaguera, J., and McCarty, N. (2016).

Hedgehog Inhibitors Selectively Target Cell Migration and Adhesion of Mantle Cell Lymphoma in Bone Marrow Microenvironment. 10.18632/oncotarget.7320Oncotarget. 712Hedgehog Inhibitors Selectively Target Cell Migration and Adhesion of Mantle Cell Lymphoma in Bone Marrow Microenvironment. Oncotarget 7 (12), 14350-14365. doi:10.18632/oncotarget.7320

CRISPR Editing in Biological and Biomedical Investigation. H Zhang, N Mccarty, 10.1002/jcb.26111J. Cel. Biochem. 11812Zhang, H., and McCarty, N. (2017). CRISPR Editing in Biological and Biomedical Investigation. J. Cel. Biochem. 118 (12), 4152-4162. doi:10.1002/jcb.26111

A Novel Model of Controlling PD-L1 Expression in ALK+ Anaplastic Large Cell Lymphoma Revealed by CRISPR Screening. J.-P Zhang, Z Song, H.-B Wang, L Lang, Y.-Z Yang, W Xiao, 10.1182/blood.2019001043Blood. 1342Zhang, J.-P., Song, Z., Wang, H.-B., Lang, L., Yang, Y.-Z., Xiao, W., et al. (2019). A Novel Model of Controlling PD-L1 Expression in ALK+ Anaplastic Large Cell Lymphoma Revealed by CRISPR Screening. Blood 134 (2), 171-185. doi:10.1182/blood.2019001043

. Y Zhang, L Y Wang, J Z Li, P F Jiang, J D Hu, Chen , B Y , Zhang, Y., Wang, L. Y., Li, J. Z., Jiang, P. F., Hu, J. D., and Chen, B. Y. (2021).

CRISPR/Cas9-mediated microRNA-21 Knockout Increased Imatinib Sensitivity in Chronic Myeloid Leukemia Cell. 10.3760/cma.j.issn.0253-2727.2021.03.011Zhonghua Xue Ye Xue Za Zhi. 423CRISPR/Cas9-mediated microRNA-21 Knockout Increased Imatinib Sensitivity in Chronic Myeloid Leukemia Cell. Zhonghua Xue Ye Xue Za Zhi 42 (3), 243-249. doi:10.3760/cma.j.issn.0253-2727.2021.03.011