# Overview of Hydroxychloroquine and Remdesivir on severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)

CorpusID: 251347821
 
tags: #Medicine

URL: [https://www.semanticscholar.org/paper/a7c4d5a90344679fe1bd2dfc4930299b04cca48e](https://www.semanticscholar.org/paper/a7c4d5a90344679fe1bd2dfc4930299b04cca48e)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Overview of Hydroxychloroquine and Remdesivir on severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)


Arup K Kabi 
Pal || Maynak 
Raghuram Gujjarappa 
| Chandi 
C Malakar 
Mithun Roy mithunroy.iisc@gmail.com 
Chandi C Correspondence 
Mithun Malakar 
Roy 

Department of Chemistry
Department of Chemistry
National Institute of Technology Manipur
ImphalManipurIndia


National Institute of Technology Manipur
795004Langol, ImphalManipurIndia

Overview of Hydroxychloroquine and Remdesivir on severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)
10.1002/jhet.4541R E V I E W Funding information Ministry of Education, New Delhi; Board of Research in Nuclear Science (BRNS), Mumbai, Grant/Award Number: 37 (2)/14/18/2017; NIT Manipur; Science and Engineering Research Board (SERB), Grant/Award Numbers: CRG/2020/004509, ECR/2016/000337; Science and Engineering Research Board (SERB), New Delhi, Grant/Award Number: ECR-2016-000839/CS
The novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) caused the ongoing pandemic named COVID-19 which causes a serious emergency on public health hazards of international concern. In the face of a critical medical emergency, repositioning of drugs is one of the most authentic options to design an adequate treatment for infected patients immediately. In this strategy, Remdesivir (Veklury), Hydroxychloroquine appears to be the drug of choice and garnered unprecedented attention as potential therapeutic agents against the pandemic realized worldwide due to SARS-CoV-2 infection.These are the breathtaking instances of possible repositioning of drugs, whose pharmacokinetics and optimal dosage are familiar. In this review, we provide an overview of these medications, their synthesis, and the possible mechanism of action against SARS-CoV-2.Arup K. Kabi and Maynak Pal contributed equally.

# | INTRODUCTION

The virus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) appeared on December 2019 and spread swiftly to worldwide, posing an immense threat to the world economy and human health [1][2][3][4][5][6]. According to the literature, the previous report of coronavirus triggered the severe acute respiratory syndrome coronavirus (SARS-CoV) outburst, in addition, severe acute diarrhea syndrome coronavirus ravaged livestock creation by causing fatal diseases in pigs. The above-mentioned epidemics were originated in China by a coronaviruses of bat origin [7][8][9][10][11][12][13][14][15][16][17]. In 2019, Fan and workers retraced the initial case of SARS which was reported in November 2012 in Foshan city of Guangdong Province in China [18]. In particular, SARS-CoV-2, a novel betacoronavirus, (previously designated 2019-nCoV) is the root of this global health crisis [19][20][21]. SARS-CoV-2 is found to contain a spherical morphology along with projection of spike on the surface. It was verified that SARS-CoV-2 has high identity of sequence with that of bat SARS-like coronavirus and SARS-CoV [22]. Remarkably, SARS-CoV-2 has higher transmissibility but decreased pathogenicity from one human to another human related with SARS-CoV [23][24][25][26][27][28][29][30]. As of April 14, 2022, over 500,186,525 cases have been confirmed, including 6,190,349 deaths in 235 countries according to the WHO [31]. The coronavirus pandemic  has become the greatest global health crisis of recent time after the second world war. Together the World Bank and the World Health Organization (WHO) in 2019 estimated the impact of the pandemic at 2.2%-2.8% of global GDP with a fear of socio-economic imbalance.

Such massive numbers of infected people and human casualties urge for an imperative demand of very efficient, accessible, and inexpensive medications to control and decline the pandemic. Thus, it is crucial to recognize vital antiviral molecules these could suppress the infection at the solitary phase and terminate its advancement over the globe. The health calamity has imposed the relocation of approved drugs in different therapeutic areas [32] and it has been proposed to be the most dynamic way to find a treatment for this virus [33][34][35]. Chloroquine (CQ) and Hydroxychloroquine (HCQ) have emerged as the most-effective therapeutic option and labeled as potential "game-changers" in popular press for current state of pandemic COVID-19 [35]. Here in this review, the organization of SARS-CoV-2, its pathogenesis, synthesis, and chemotherapeutic role of Remdesivir, CQ or HCQ in COVID-19 with the perspective of in silico approach of possible prevention of SARS-CoV-2 pathogenesis are discussed.


# | SARS-COV-AND PATHOGENESIS


## | The virus

SARS-CoV-2 associated to b-group of the coronavirus family responsible for acute respiratory distress syndrome in human beings like other SARS viruses and transmitted through human contacts through respiratory droplets ( Figure 1). Therefore, SARS-CoV-2 is extremely fatal to the patients with pre-existed respiratory diseases. The COVID-19 virus size, determined from electron micrograph was 70-90 nm [36].


## | Genome organization

The virus, SARS-CoV-2 corresponds to the family of coronavirus which consists a large single-stranded, RNA genome of positive-sense with molecular weight of around 29.2 kbp ( Figure 1). Their genomes are typically made of a 3 0 -poly-A tail at the 3 0 -end and a 5 0methylguanosine cap at the 5 0 -end. These genes are highly conserved. Genome analysis has shown that the genetic sequence of SARS-CoVs consists of open reading frames (ORFs) in variable numbers (6)(7)(8)(9)(10)(11). The translation of the two important polyproteins (pp1a and pp1b), which is necessary to encode 16 non-structural proteins (NSP), is carried out by the first ORF (ORF1a/b) and other accessory and structural proteins are encoded by the remaining ORFs. The four other essential structural proteins, comprising spike (S) glycoprotein, membrane (M) protein, matrix, nucleocapsid (N) protein and small envelope (E) protein, and also several accessory proteins interfering with the host-innate immune response is encoded by remaining viral genome [37]. Although, SARS-CoV-2 genes are subjected to undergo several mutations in different provinces, there are some similarities in the genomic structure with SARS-CoV, especially the S-glycoprotein genome is similar to SARS-CoV along with the receptor-binding domain (RBD) this feature became important for human-to-human viral infection innate immune response [38].
3 | STRUCTURAL PROTEINS OF SARS-COV-2 3.1 | Spike protein
The viral envelope consists of three protein components among which the Spike-glycoprotein is of utmost importance. Recognition of host cells by the spike protein is the entry point of SARS-CoV-2 to the host cells. The S protein is a large transmembrane protein consisting of N-exo, Cendotrimeric assembly forming the impression of the spikes. The S protein consists of a small C terminal end of 71 residues and another large unit of 1162-1452 residues. Each monomeric unit, before glycosylation, has a molecular weight of 128-160 KDa. The S protein can be easily glycosylated via the modification of the N terminal end. The S protein has two protein subunits S1 and S2. The subunit S1 consists of the receptor-binding site. The SARS-CoV-2 S protein provides an open binding site that allows better attachment and structural modification possibilities to the virus. The S proteins contain 30-50 cysteine residues whose positions are fixed with various types of coronaviruses, although their activity is not well known till now. The SARS-CoV-2 cooperate with the angiotensin-converting enzyme 2 (ACE2) receptor as they consist of 75% similarly in the surface amino acids and 50% similarity with receptor binding motifs ( Figure 2).

The previous reports suggest that there are 14 key sites for binding with SARS-CoV binding sites with human angiotensin-converting enzyme 2 as follows: Y491, G488, T487, T486, Y484 N479, Y475, N473, L472, Y442, Y440, Y436, R426, and T402. In SARS-CoV-2, eight positions are strictly conserved, and others are substituted conservatively. The modified parts are T487 SARS-CoV N501 SARS-CoV-2 , Y484 SARS-CoV Q498 SARS-CoV-2 , N479 SARS-CoV Q493 SARS-CoV-2 , L472 SARS-CoV F486 SARS-CoV-2 , Y442 SARS-CoV L455 SARS-CoV-2 , and R426 SARS-CoV N439 SARS-CoV-2 . The virus recognizes the host cells through the non-covalent interaction between these sites, and subsequently, the envelope layer is dissolved and the viral genome is inserted in the cells [39]. 


## | Membrane protein


## | Envelope protein

Envelope (E) protein is the most important protein in the SARS-CoV-2 and it is the very small protein of molecular weight 8.4-12 kDa (76-109 amino acid residue). The arrangements in the E proteins are very diverse across the different types of corona viruses. It is constructed of a short hydrophilic N-terminus of 8-12 amino acids and a large hydrophobic region of 21-29 amino acid residues, 2-4 cysteine's and hydrophilic C-terminal tail of 39-76 residues. The hydrophobic domain forms a hairpin-like structure, facilitating the insertion of E proteins in the endoplasmic reticulum-Golgi pocket. The E proteins take part in the trafficking of the new viral genome and other proteins to the endoplasmic reticulum-Golgi pocket and bind with M protein to promote the assembly process of new viruses [41].


## | Nucleocapsid (N) protein

The N protein is a phosphor-protein of a molecular weight of 43-50 kDa and the protein binds as beads-on-astring manner with the viral RNA genome. It consists of different domains in which three conserved domains are separated by two variable spacers. The domains are rich in arginine and lysine residues which favors the binding of N proteins with the viral RNA. The N and C terminal sites of N proteins are susceptible to bind with the RNA. The bound N protein-RNA nucleocapsid is the active viral genome [41].


# | PATHOGENESIS AND HOST-IMMUNE RESPONSES

The pathogenesis of SARS-CoVs is almost similar to other SARS viruses where the spike protein plays a pivotal role in recognizing the ACE2 receptor of the host alveoli epithelial cells (Figure 3).

After the receptor recognition and receptor mediated endocytosis, the viral genome encapsulated nucleocapsid protein released into host cellular cytoplasm media. Then occurs the translation of ORF1a and ORF1b genes for production of two polyproteins (PPs), pp1a and pp1b, which has the primary operation to modulate host ribosomes for the translation process of viral genome. Both pp1a and pp1b play a major role for the generation of the replication transcription complex. The polyproteins are activated by protease and produces 16 NSPs which carries out their specific functions including (i) NSP1 and NSP2 suppresses the host gene expression, (ii) NSP3, NSP5 forms of a multidomain complex (M protease), plays important character in replication, (iii) The role of transmembrane (TM) proteins is carried out by NSP4 and NSP6, (iv) The role of a primase is carried out by NSP7 and NSP8, (v) NSP9 acts as a RNA-binding protein which has an important role during viral infection, (vi) The role of cofactor is played by NSP10, for the activation of the replicative enzyme, (vii) RNA-dependent RNA polymerase activity is exhibited by NSP12, (viii) helicase activity exhibited by NSP13, exoribonuclease activity exhibited by NSP14, (ix) endoribonuclease activity pertains to NSP15, and (x) NSP16 has a crucial role in methyltransferase activity. The replication and transcription process of the viral genome is regulated by all NSPs and they perform a pivotal character in the processes. The structural proteins such as M proteins, E proteins, and S proteins are synthesized and for developing the structure of viral envelope, embedded inside the endoplasmic reticulum (ER)-Golgi intermediate compartment (ERGIC) complex. Simultaneously the replicated (+)-genome (RNA) gets bound to nucleocapsid (N) protein which initiates formation of the ribonucleoprotein (RNP) complex. Additionally, the new viral genome outer cover is formulated through structural S proteins, M proteins, and E proteins and eventually, the virus particle reveals from the compartment endoplasmicreticulum-Golgi intermediate compartment (ERGIC) by forming a bud-like structure through exocytosis. These mature virions come out of the cell by formation of vesicle and discharge the virus particles into the extracellular region by fusing with the plasma membrane [42][43][44][45][46][47][48][49][50].

The release of replicas of SARS-CoV-2 activates inflammatory mediators which stimulate macrophages and release specific cytokines (Interleukin-1, Interleukin-6, and TNF-a). This causes vasodilatation and endothelial contraction leading to alveolar edema, hypoxemia and damages the lung cells causing acute distress respiratory syndrome [51].


## | Immuno-or chemo-therapeutic option for SARS-CoV-2

Till now there is no efficient therapeutic option for SARS-CoV-2. However, symptomatic treatment strategies are recommended for clinical intervention. There are several antiviral agents that are found potentially important in the medication for SARS-CoV-2. Among them CQ and HCQ have shown immense potential to treat pneumonia associated with COVID-19 in recent clinical studies [52,53]. In China, the addition of CQ has been made to the list of trial drug molecules for the cure of COVID-19 [54,55]. Wang et al. recently described that two drugs, Remdesivir (GS-5734), a nucleoside analog of the prodrug established by Gilead Sciences (USA) and CQ phosphate, is described to suppress the SARS-CoV-2 infection in vitro [56,57]. A recent report exhibited that, the clinical situation of the first patient infected by SARS-CoV-2 in the United States has been upgraded by the treatment method involving Remdesivir [58] for which it gained to be emergency use authorization from the FDA on May 1, 2020 [59].


### | Interferon (Type-I)

Type I Interferon's are the antiviral cytokines which involve inducing a large spectrum of proteins that can inhibit viral replication in targeted cells. Earlier studies revealed that IFN-b was effective against SARS virus. Synergic antiviral effects of leukocytic IFN-α with ribavirin and IFN-β with ribavirin against SARS-CoV has also been illustrated in vitro [60].

F I G U R E 3 Pathogenesis of severe acute respiratory syndrome coronavirus 2 in host pneumonocyte cells [42][43][44][45][46][47][48][49][50] 


# | ANTIVIRAL AGENTS

The CQ (N 4 -[7-Chloro-4-quinolinyl]-N 1 ,N 1 -diethyl-1,4-pentanediamine) medication was significantly reduced due to infrequent consumption in clinical practice for treating malaria as a derivative of CQ was established to be much reduced ($40%) toxicity than CQ in animals by instigating a hydroxyl group (-OH) onto CQ ring and it has been termed as HCQ sulfate [61]. The greater doses of HCQ (up to 600 mg/day) has been implemented for decades toward the cure of autoimmune diseases like rheumatoid arthritis (RA) and systemic lupus erythematosus (SLE) [62]. The mode of action of HCQ and CQ falls in to the category of a weak base and immunomodulator, as well as oral consumption of HCQ and CQ in human beings is very much reasonable. In animals, the two medicines share parallel tissue disposition patterns with large concentrations in the kidney, liver, lungs and spleen with a reaching level of 200-700 times superior compared to plasma [63]. It was stated that HCQ sulfate in optimal dosage (6-6.5 mg/kg per day) could produce serum levels of 1.4-1.5 μM in humans [64]. Consequently, SARS-CoV-2 infection can be prevented by the concentration of HCQ in the above tissues over a safe dosage. Clinical examinations revealed that the severely ill patients affected with SARS-CoV-2 are found to have cytokines in high concentration in the plasma, which suggests that cytokine storm was accompanying with disease intensity [65]. Apart from its antiviral action, HCQ is a benign and effective antiinflammatory drug which has been greatly used to treat autoimmune diseases and in reducing the significant production of pro-inflammatory factors of cytokines. In this regard, Remdesivir, HCQ became the most effective, simplest and cheapest drugs to treat and prevent infectious global pandemic coronavirus disease 2019 (COVID-19) caused by the SARS-CoV-2.

Remdesivir inhibited RNA dependent RNA polymerase (RdRp) during viral pathogenesis in host cells. Remdesivir was tested in vitro against SARS-CoV-2 and entered into phase 3 clinical trial [66]. Favipiravir also found to inhibit RdRp and a randomized trial has been started. Ribavirin tested in vitro against SARS-CoV-2, it generally inhibits the viral RNA synthesis and mRNA capping [67]. Randomized trial of Ribavirin has been initiated. Lopinavir and Ritonavir inhibit the 3C protease like 3Clpro. Both antiviral drugs have entered into the phase 3 clinical trials [68]. HIV protease inhibitors like Darunavir, Cobicistat, and ASC09F have successfully entered into the phase 3 clinical trial [69]. Arbidol (Umifenovir) which inhibit generally viral fusion in host cells, are also selected for phase 3 clinical trials [70].


## | The advantages of HCQ over other antimalarial drugs for the treatment of COVID-19

Due to the swift spread of the novel SARS-CoV-2, the WHO acknowledged the epidemic of COVID-19 as a pandemic on March 12, 2020 [71]. Thus, there is a crucial necessity for an efficacious medicine toward treatment of symptomatic patients as well as to reduce the extent of virus carriage in order to bound the spreading in the community. Several drugs such as CQ, nafamostat, nitazoxanide, penciclovir, ribavirin and two well-accepted comprehensive antiviral drugs such as favipiravir (T-705) and Remdesivir (GS5734) have been recommended as medications that could diminish viral spreading [56,57]. Drugs, such as interferon, ribavirin, corticosteroids, and lopinavir-ritonavir have also been employed in patients having MERS or SARS (Middle East Respiratory Syndrome) though the efficacy of some drugs remains contentious [72]. Regrettably, toward the cure of SARS-CoV-2 infection, until now, no drugs have approved by WHO. In this regard, among these drugs to cure the COVID-19, replacement of old medications to utilize as antiviral treatment is a fascinating approach because of their well-known knowledge on drug interactions, safety profile, posology, and side effects [32][33][34][35]. Delightfully, numerous in-vitro studies have revealed that SARS-CoV-2 transmission can be inhibited by both CQ and HCQ, through alkalinization of the intracellular phagolysosome, which was responsible for the uncoating and virion fusion and consequently the viral spread [54][55][56][57]73].

In precise, CQ is extensively used as immunomodulator with anti-malarial activity [74][75][76][77][78]. Recently, an in vitro study revealed that, SARS-CoV-2 growth was found to be inhibited by CQ and these results have been reinforced by clinical trials on approximately 100 patients infected by SARS-CoV-2 [56,57,79,80]. In addition, it is more toxic than HCQ [54,55] and an overdose of CQ can be the reason of acute poisoning and death [81].

CQ can inhibit the viral cell signaling process by inhibiting phosphorylation. The immunomodulatory and antimalarial drug HCQ has seized greater awareness among the myriad therapeutics progressed as possible repurposing candidates in viral clearance in patients with SARS-CoV-2. Moreover, it was demonstrated that, HCQ has in vitro anti-SARS-CoV action in the former SARS outbreak [82] and this encourages that it may be a promising pharmacological agent to treat COVID-19 infection. Very recently, several in vitro studies exposed that HCQ have better anti-SARS-CoV-2 activity than CQ [54,55]. Additionally, verification through EC 50 values, which were consistently lesser for HCQ when compared to CQ, indicating the more potent antiviral activity of HCQ [73]. In another, results of some studies proposed that the viral loading in patients with COVID-19 could be minimized by combined medication of HCQ with azithromycin (Table 1) [83][84][85][86][87]. In addition, a study on 30 patients with SARS-CoV-2 infection by Chen et al. [85]; when HCQ was associated as regular treatment, nasopharyngeal viral carriage on the 7th day seems to be unaltered. However, elucidating the results of this survey; providing associated antivirals, which might have assisted as confounders.

Again, in another trial on 62 number of patients, Chen and co-workers reported, the patients required short period of time for clinical recovery (cough, temperature) on administering HCQ for treatment than placebo [86].

The MOA of CQ and HCQ has not yet been entirely clarified. Although it was revealed from the previous studies, the coronavirus may get inhibited through a progressive step by the medication of CQ and HCQ. First, the change in pH at the cell membrane surface occurs by the intake of drugs and resulting in an inhibitory action on virus fusion to the cell membrane. Moreover, the replication of nucleic acid can be prevented by this drug. Nevertheless, glycosylation of viral proteins, virus assembly, virus release, new virus particle transport, and related processes to accomplish the antiviral action of this drug [88].

In the survey by Wang and co-workers [56,57], after a 48-h incubation time with an EC 50 value of 1.13 μM of CQ, appeared to have a repressive effect on SARS-CoV-2 infection. These findings are comparable with the study of Yao et al. [73], where EC 50 value was reported to be 5.47 μM by an in vitro CQ study. In this study, the EC 50 values were reduced on longer incubation times for both HCQ and CQ. This recommends that the drug's antiviral activity get influenced by incubation time. In addition, it was reported that HCQ and CQ can accumulate in cells [89]. Possibly, drug will have sufficient time to accumulate in higher intracellular concentrations on providing longer incubation time and resulting a good potency on antiviral activity [90].

Based on HCQ higher prophylactic activity as well as antiviral action along with acceptable safety profile in comparison with CQ [91], the US FDA (United States Food and Drug Administration) operated its emergency authority only the second time ever to approve the administration of a drug for an unapproved symptom [92]. HCQ has been approved to treat COVID-19 in conditions when the clinical trials are inaccessible or infeasible [93].

Additionally, it has been reported in some of the studies that, mechanical ventilation risk in hospitalized patients with COVID-19 does not get lowered with medication of HCQ either with or without azithromycin [87,[94][95][96]. Basically, azithromycin has been appeared with potent activity in vitro against Ebola and Zika viruses [97][98][99] and to restrict severe infections of respiratory tract when administered to viral infected patients [100].

According to a study, there were a total of 70 deaths out of 368 patients appraised with. The lowest death rate has been reported for patients who are not treated with HCQ when compared to HCQ and azithromycin associates [94].

India had revealed a stumble direction of SARS-CoV-2 transmission, with 43,039,023 cases diagnosed as of April 11, 2022 [95]. As no major side-effects of HCQ were found in studies in India, the Indian Council of Medical Research under the Ministry of Health and Family Welfare, has endorsed chemoprophylaxis with HCQ as preventive treatment for COVID-19 [96]. However, all drugs have the risks of adverse effects and there might be other risks when HCQ sulfate is used for treating COVID-19 earlier. Moreover, risks related with these drugs include low blood sugar, particularly among people with anemia, diabetes and other blood problems, retina (layer of eye tissue) damage that can cause vision problems, serious heart rhythm problems, worsening of seizures, and other neurology (brain) problems. Also, problems can arise because of interactions between HCQ sulfate and other drugs which results in serious health issues. As mentioned above, the conflicting findings of HCQ administration were reported recently where the mortality rates are higher in comparison with the initial data available for the SARS-CoV-2 treatment [101].

Hence, it is advised to patients to inform their health care provider regarding the medications which are already administered for other purposes [93].


# | SYNTHESIS OF CQ AND HCQ

CQ and HCQ belong to a similar molecular family of substituted 4-aminoquinolines. HCQ differs from CQ by the existence of a hydroxyl functionality (-OH) at the terminal of the side chain. In particular, CQ is known as an amine acidotropic derivative of quinine which was prepared in Germany by Bayer in 1934 and appeared around 70 years ago as a potent surrogate for natural quinine [102]. Additionally, quinine is a compound originate in the bark of Cinchona trees which are found in Peru and was the earlier drug of choice to treat malaria [103]. During the past decades, CQ was the prescribed medicine for the prophylaxis and cure of malaria disease and also remain among the most recommended drug worldwide [104]. Very recently, it has been described as an antiviral drug with broad-spectrum of potential [105][106][107].

HCQ is one among the oldest prescribed drugs still used in clinical practice for the treatment of malaria due to its relatively low-cost and tolerance. HCQ sulfate is an orally administered drug which is another form of HCQ. These drugs have been acknowledged to be effectual in autoimmune diseases like RA and SLE as well as a pharmacologic agent to reduce heart rate [108][109][110]. In 1946, Alexander Surrey and Henry Hammer fabricated HCQ sulfate by introducing a hydroxyl group into one of the N-ethyl groups of CQ phosphate which may dramatically decrease the latter's toxicity [111].


## | Synthesis of CQ
CQ (N 4 -[7-Chloro-4-quinolinyl]-N 1 ,N 1 -diethyl-1,4-pentane- diamine)
is synthesized by reacting 4,7-dichloroquinoline with 4-diethylamino1-methylbutylamine at 180 C (Scheme 1) [112,113].

In order to appreciate the preparation of CQ, the essential 4,7-dichloroquinoline (1) is assembled in diverse routes from 3-chloroaniline (4). One of such protocol focused on the reaction between 3-chloroaniline (4) and ethoxymethylenmalonic ester (5) to obtain the (3-choroanilino)-methylenemalonic ester (6), which afterwards experiences the hetero-cyclization reaction at hightemperature to afford the ethyl ester of 7-chloro-4-hydroxyquinolin-3-carboxylic acid (7). After that, the hydrolysis of compound 7 using sodium hydroxide (NaOH) delivers the 7-chloro-4-hydroxyquinoline-3-carboxylic acid (8), which on heating at 250-270 C gets decarboxylated to form 7-chloro-4-hydroxyquinoline (9). The product 9 on treating in the presence of phosphorus oxychloride (POCl 3 ) produces the 4,7-dichloroquinoline (1) (Scheme 2) [114,115].

In another, 3-chloroaniline (4) with diethyl oxaloacetate (10) were used for the preparation of 4,7-dichloroquinoline (1). The reaction of 4 and 10 in participation of acetic acid and heating to 250 C resulted in the consistent enamine (11). It encounters hetero-cyclization to realize the ethyl ester of 7-chloro-4-hydroxyquinolin-2-carboxylic acid (7) accompanied with a slight amount of 5-chloro-4-hydroxyquinolin-2-carboxylic acid (12), which is isolated using crystallization process from the mixture of main product in presence acetic acid. After that, alkaline hydrolysis of 7 results in 7-chloro-4-hydroxyquinoline-2-carboxylic acid (8) and successive high temperature decarboxylation of the formed acid 8 gives 7-chloro-4-hydroxyquinoline (9). Consequently, the reaction of 7-chloro-4-hydroxyquinoline (9) with phosphorus oxychloride (POCl 3 ) gives 4,7-dichloroquinoline (1) (Scheme 3) [111].

Additionally, the proposed alternatives for synthesizing 4,7-dichloroquinoline (1) compose of the reaction between 3-chloroaniline (4) and the ethyl ester of formylacetic acid (13) to prepare the corresponding enamine (14). Then, the enamine (14) experiences cyclization reactions upon heating to deliver the 7-chloro-4-hydroxyquinoline (9) and reaction using phosphorus oxychloride dispatches the 4,7-dichloroquinoline (1) (Scheme 4) [116].

The essential component for the synthesis of CQ is 4-diethylamino-1-methylbutylamine (2), which was also made in various synthetic pathways. Alkylating ethyl acetoacetate (15) using 2-diethylaminoethylchloride (16) delivers 2-diethylaminoethylacetoacetic acid ester (17), which produces 1-diethylamino-4-pentanone (18) followed by acidic hydrolysis using hydrochloric acid and rapid decarboxylation process. Reductive amination of 18 with H 2 and NH 3 in presence of Raney nickel gives 4-diethylamino-1-methylbutylamine (2) (Scheme 5) [117].

The compound 3-acetylbutyrolactone (20) has been utilized as a starting component for the synthesis of 4-diethylamino-1-methylbutylamine (2). The compound 3-acetylbutyrolactone (20) is made by reacting ethyl acetoacetate (15) with ethylenoxide (19). Then, acidic hydrolysis using HBr of the ester S C H E M E 1 Synthesis of Chloroquine (3) [112,113] S C H E M E 2 Synthesis of 4,7-dichloroquineoline (1) [114,115] group in 3-acetylbutyrolactone (20) followed by subsequent decarboxylation process afforded 1-bromo-4-pentanone (21). Next, reacting 21 with diethyl amine gives 1-diethylamino-4-pentanone (18), followed by reductive amination with H 2 and NH 3 employing Raney nickel catalyst results 4-diethyl-1-methylbutylamine (2) (22), is prepared by the Scheme similar to that of the construction of CQ molecule. The reaction between 1-chloro-4-pentanone (23) and 2-ethylaminoethanol (24) delivers the corresponding S C H E M E 3 Synthesis of 4,7-dichloroquineoline (1) using 3-chloroaniline (4) and diethyl oxaloacetate (10) [111] S C H E M E 4 Synthesis of 4,7-dichloroquineoline (1) using 3-chloroaniline (4) and ethyl ester of formylacetic acid (13) [116] S C H E M E 5 Synthesis of 4-diethylamino-1-methylbutylamine (2) [117] amino ketone 25. The reductive amination of 25 with H 2 and NH 3 using Raney nickel as a catalyst results in 4-[ethyl(2-hydroxyethyl)amino]-1-methylbutylamine (26). Next, the reaction of component 18 with 4,7-dichloroquinoline (1) by the use of phenol and potassium iodide (KI) at the temperature ranging from 125 to 130 C for 18 h gives the desired HCQ (22) (Scheme 7) [119].

The compounds 4,7-dichloroquinoline (1) and 4-[ethyl(2-hydroxyethyl)amino]-1-methylbutylamine (26) in a molar ratio of 1:1.1 were positioned into a highpressure reactor. The internal pressure of the reactor is then regulated within the range of 5 to 20 bars and preferably 10 to 15 bars by nitrogen pressure. The compound 4,7-dichloroquinoline is made to dissolve completely by stirring the reactor at 80 C for 30 min, accompanied by supplemental stirring at a temperature of 100 to 120 C for 4 to 6 h. This development allows the excellent production of HCQ with high yield by significantly decreasing the reaction temperature and time using pressure without making use of any catalyst and solvent [120].


# | SYNTHESIS OF REMDESIVIR (VEKLURY, GS-5734)

Remdesivir has doubtlessly become a significant scaffold as most promising drug for COVID-19 [58,121]. The S C H E M E 6 Synthesis of 4-diethylamino-1-methylbutylamine (2) by using ethyl acetoacetate (15) with ethylenoxide (19) [118] S C H E M E 7 Synthesis of Hydroxychloroquine (22) [119] insistence of global pandemic of COVID-19 has impelled the governments of India, the European Union, United States, Italy, Japan, etc. to accept the Remdesivir as a precise medication [122]. Recent clinical studies revealed it can efficiently abridge the retrieval period of patients and diminish the chance of death by 62% for critical hospitalized patients [123], and so it can significantly reduce the number of casualties and no of cases that benefits doctors and hospitals during the pandemic. Gilead Sciences, in 2020 conducted almost 2 million Remdesivir medication courses and they intend to do better by 2021 [124]. However, Remdesivir is prepared as a Pstereogenic prodrug of a parent nucleoside (GS441524) by employing an amino acid ester- (27) and aryloxysubstituted phosphoryl group (28) with the hydroxyl functionality of the nucleoside (30) [125,126]. Mainly, the establishment of the phosphorus-linked stereogeniccenter is the crucial stage for the synthesis of Remdesivir. The synthetic methods of Remdesivir (31) (first generation) are displayed in Scheme 8. By using stoichiometric amounts of N-methyl imidazole along with GS441524 and P-racemic phosphoryl chloride, the desired enantiopure Sp-phosphoramidate is synthesized, followed by chiral preparative HPLC segregation of the two diastereoisomer products [125].

The second generation of preparation procedure of Remdesivir is showcased in Scheme 9 which embraced another method including separation from the accompanying composed combination of two diastereoisomer intermediates followed by reaction of the enantiopure phosphorylating agent via nucleophilic substitutions [126].

Recently, Wang and co-workers showcased an asymmetric synthetic route of Remdesivir using a bicyclic imidazole-catalyzed DyKAT, followed by reaction between protected nucleoside and P-racemic phosphoryl chloride via coupling reaction (Scheme 10) [127].


# | PERSPECTIVE OF CQ AND HCQ AS INHIBITORS OF SPIKE PROTEIN: IN SILICO APPROACH

In the last few decades, computational chemistry has emerged as a helpful aspect for the aid of understanding chemical phenomena. Molecular docking is a crucial aspect for visualization of the interaction between host proteins and foreign guest molecules, which gives us insights to understand the pharmacophore for a specific target protein. In our current approach, the specific target protein is the spike protein of the SARS-CoV-2 and the pharmacophores to be CQ and HCQ. As per the literature, it is evident that the spike protein is liable toward binding of the virus with the receptor ACE2 protein. The active RBD of SARS-CoV-2 was obtained from protein data bank (PDB ID: 6VW1) [39]. The structures of CQ and HCQ were optimized by DFT calculation using S C H E M E 8 First generation synthesis of Remdesivir (31) [125] Gaussian09 software [128]. Prior to the molecular docking calculation, excess water molecules were deleted and polar hydrogen was added in the target protein (6VW1) followed by computation of Kollman charges. CQ and HCQ were docked in the RBD of spike protein of SARS-CoV-2 in Autodock 4 [129] docking software with a grid of 75 Â 110 Â 70 (x dimension Â y dimension Â z dimension) consisting with 598,959 total grid points, with a grid center of À35.612, 31.244, 7.128 (x, y, z coordinates). The docking calculation was done in Genetic algorithm with 10 runs, population size of 150 and 25,000,000 evaluations with rate of gene mutation 0.02 and rate of crossover 0.8. For visualization of docked pose and calculation of interactions, Discovery Studio 09 software was used. The docked pose (Figure 4) indicated us that CQ and HCQ has the potential ability to bind with the RBD of spike protein with binding constants of À4.63 and À3.85 Kcal/mol, inhibition constants (Ki) of 1.52 and 0.4 mM respectively. The docked poses were analyzed by BIOVIA Discovery Studio 9 molecular graphics program software [130]. The docked pose is stabilized by extensive hydrogen bonding between CQ/HCQ and the RBD.

CQ is subjected to hydrogen bonding with ASN 501 of RBD and CQ hydrogen along with π-donor S C H E M E 9 Second generation synthesis of Remdesivir (31) [126] S C H E M E 1 0 Catalytic asymmetric synthesis of Remdesivir (31) [127] hydrogen bonding between GLY 496 hydrogen and π-orbitals of CQ. HCQ is associated with hydrogen  Table 2). The binding energy details, and the hydrogen bonding interactions are depicted in Table 3 and Figure 2. From the inhibition constant and binding energy, it is deductible that CQ is a better option than HCQ in terms of spike protein inhibition.


# | SUMMARY

In conclusion, COVID-19 has created a great risk to the safety and human health; hence, to administer the F I G U R E 4 Docked pose of Chloroquine (A) and Hydroxychloroquine (B) on the receptor binding domain of the SARS-CoV-2 spike protein [130] T A B L E 2 Hydrogen bonding in docked pose of Chloroquine and Hydroxychloroquine [128,129]  proliferation of the epidemic and faster control of the mortality is the prime concern. The successful option of antiviral therapy and vaccines such as Pfizer-BioNTech, Moderna, CoronaVac, Covaxin, Sputnik V, Oxford-AstraZeneca etc. are currently under evaluation and development. However, based on the current and previously published evidence, scientists all over the globe predict that Remdesivir as well as HCQ has prime efficacy to fight this widespread disease. In this perspective, we systematically summarize the synthesis, mechanism of action of Remdesivir, CQ, and HCQ as well as infection source, transmission route, pathogenesis, clinical characteristics, prevention, and treatment of SARS-CoV-2. This review is written in the hope of helping the public and deal with the novel coronavirus and providing the references for follow-up research, therapy, and prohibition. Moreover, it may support readers to have the recent comprehension of this new infectious disease. 


## ACKNOWLEDGMENTS

## F
I G U R E 1 (A) Electron micrograph of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) (image credit: Content source(s): CDC/Dr. Murphy F). This material comes with an identification number #4814 from the Centers for Disease Control and Prevention's Public Health Image Library. (B) Schematic presentation of SARS-CoV-2 showing spike (S) glycoprotein protein, membrane glyco (M) protein, envelope (E) protein, and nucleocapsid (N) protein (adapted from: Peiris et al.). (C) Large single-stranded, RNA genome (positive-sense) of SARS-CoV-2 [36]


The most abundant protein present in the coronavirus is the membrane (M) protein. Before glycosylation, the molecular weight of M glycoprotein has been reported as 25-30 kDa (221-262 amino acid residue) in the SARS-CoV virus. The M glycoprotein is multispanning and a small N terminal part is situated in the exterior of the virus. The other three transmembrane sequences and a large Cterminal domain constitute the whole M protein. The M protein is involved in the development of the envelope and governs the shape of new virions. While new virions are created, the M protein binds with the S, E, N, and M proteins and creates the envelope for new virions[40].

## F
I G U R E 2 (A) The spike (S) glycoprotein protein of severe acute respiratory syndrome coronavirus 2 (SARS CoV-2). The red colored residue indicated the receptor binding domain (PDB Id: 6VXX). (B-D) Interacting amino acids of receptor-binding domain of SARS-CoV-2 and binding domain of angiotensin-converting enzyme 2 receptor of the host cells at different angles (adapted from Yan et al. [38 ])


bonding as TYS 449 (H) -HCQ (N1), GLY 502 (H) -HCQ (O), HCQ 29 (H) -GLY 496 (O), HCQ 29 (H) -GLN 498 (O), TYR 449 (H) -HCQ (O) (


T A B L E 1 Clinical trials of Hydroxychloroquine (HCQ) in patients with COVID-19[83][84][85][86][87] Ref 

Study type (no. of patients) 
Duration 
Primary outcomes 
ICU patients (n/N) 

Mortality 
(n/N) 
Treatment 

Control group 
(no. of patients) Clinical outcomes 

Adverse 
events (n/N) 

49 
Prospective open-label, non-
randomized trial (n = 42) 

10 days 
Viral load (nasopharyngeal 
swab): absence or presence 
of SARS-CoV-2 at Day 6 

0/36 
0/36 

HCQ (200 mg every 8 h) alone 
(n = 14) or with azithromycin 
(500 mg on Day 1, 250 mg on 
Days 2-5; n = 6) 

Yes (n = 16) 
NR (not reported) 
NR 

49 
Prospective observational study 
(n = 80) 

10 days 
Disease progression: need for 
oxygen or ICU admission 

3/80 
1/80 

HCQ (200 mg every 8 h) and 
azithromycin (500 mg on Day 1, 
250 mg on Days 2-5) 

No 
Viral load, hospital length of 
stay 

7/80 

49 
Randomized Controlled Trial (RCT) 
(n = 30) 

7 days 
Viral load (nasopharyngeal 
swab): presence of SARS-
CoV-2 at Day 7 

0/30 
0/30 

HCQ (200 mg every 12 h) 
Yes (n = 15) 
NR 
4/15 

49 
RCT (n = 62) 
5 days 
Time to clinical recovery 
0/62 
0/62 

HCQ (200 mg every 12 h) 
Yes (n = 31) 
Pulmonary recovery, adverse 
events 

2/31 

49 
Prospective observational study 
(n = 11) 

10 days 
Viral load (nasopharyngeal 
swab): presence of SARS-
CoV-2 on Days 5-6 

2/11 
1/11 

HCQ (200 mg every 8 h) and 
azithromycin (500 mg on Day 1, 
250 mg on Days 2-5) 

No 
NR 
1/11 



T A B L E 3 Binding energy of the docked poses of Chloroquine and Hydroxychloroquine on receptor binding domain of spike protein[128,129] Compound 
Type of bond 
Donor 
Acceptor 
Bond distance (Å) 

Chloroquine 
Hydrogen bond 
ASN 501 (H) 
CQ (N) 
2.61 

Chloroquine 
Π-donor Hydrogen bond 
GLY 496 (H) 
CQ (π-orbitals) 
2.86 

Hydroxychloroquine 
Hydrogen bond 
TYR 449 (H) 
HCQ (N) 
1.93 

Hydroxychloroquine 
Hydrogen bond 
GLY 502 (H) 
HCQ (O) 
1.81 

Hydroxychloroquine 
Hydrogen bond 
HCQ 29 (H) 
GLY 496 (O) 
2.30 

Hydroxychloroquine 
Hydrogen bond 
HCQ 29 (H) 
GLN 498 (O) 
3.09 

Hydroxychloroquine 
Π-donor Hydrogen bond 
TYR 449 (H) 
HCQ (O) 
2.77 

Compound 

Binding energy 
(kcal/mol) 
Intermolecular energy (kcal/mol) 

Electrostatic 
energy (kcal/mol) 

Torsional 
energy (kcal/mol) 

Inhibition 
constant (mM) 

Van der Waals + hydrogen 
bonding + desolvation 
energy (kcal/mol) 

Total internal 
energy (kcal/mol) 

Unbound 
energy (kcal/mol) 

Chloroquine 
À4.63 
À7.02 
0.1 
2.39 

0.404 
À7.11 
À1.5 
À1.5 

Hydroxychloroquine 
À3.85 
À6.83 
À0.05 
2.78 

1.52 
À6.78 
À0.71 
À0.71 



Chandi C. Malakar appreciate Science and Engineering Research Board (SERB), New Delhi and NIT Manipur for financial support in the form of research grant (ECR/2016/000337 and CRG/2020/004509). Mithun Roy gratefully thank the Board of Research in Nuclear Science (BRNS), Mumbai (37(2)/14/18/2017-BRNS) and the Science and Engineering Research Board, Government of India, New Delhi for financial support [ECR-2016-000839/CS]. Chandi C. Malakar and Mithun Roy greatly appreciate Technical Education Quality Improvement Program, Phase-III (TEQIP-III) for financial assistance as Minor Research Grant. Arup K. Kabi and Raghuram Gujjarappa grateful to Ministry of Education, New Delhi for fellowship support. Maynak Pal is thankful to BRNS for scholarship.DATA AVAILABILITY STATEMENTData sharing not applicable to this article as no datasets were generated or analysed during the current study.ORCIDRaghuram Gujjarappa https://orcid.org/0000-0002-2586-3190 Chandi C. Malakar https://orcid.org/0000-0002-9478-017X

. W Guan, Z Ni, Y Hu, W Liang, C Ou, J He, L Liu, H Shan, C Lei, D S C Hui, B Du, L Li, G Zeng, K.-Y Yuen, R Chen, C Tang, T Wang, P Chen, J Xiang, S Li, J.-L , W. Guan, Z. Ni, Y. Hu, W. Liang, C. Ou, J. He, L. Liu, H. Shan, C. Lei, D. S. C. Hui, B. Du, L. Li, G. Zeng, K.-Y. Yuen, R. Chen, C. Tang, T. Wang, P. Chen, J. Xiang, S. Li, J.-L.

. Z Wang, Y Liang, L Peng, Y Wei, Y.-H Liu, P Hu, J.-M Peng, J Wang, Z Liu, G Chen, Z Li, S Zheng, J Qiu, C Luo, S Ye, N Zhu, Zhong, N. Engl, J. Med. 3821708Wang, Z. Liang, Y. Peng, L. Wei, Y. Liu, Y.-H. Hu, P. Peng, J.-M. Wang, J. Liu, Z. Chen, G. Li, Z. Zheng, S. Qiu, J. Luo, C. Ye, S. Zhu, N. Zhong, N. Engl, J. Med. 2020, 382, 1708.

. C C Lai, T P Shih, W C Ko, H J Tang, P R Hsueh, Int. J. Antimicrob. Agents. 55105924C. C. Lai, T. P. Shih, W. C. Ko, H. J. Tang, P. R. Hsueh, Int. J. Antimicrob. Agents 2020, 55, 105924.

. L S Wang, Y R Wang, D W Ye, Q Q Liu, Int. J. Antimicrob. Agents. 55105948L. S. Wang, Y. R. Wang, D. W. Ye, Q. Q. Liu, Int. J. Antimicrob. Agents 2020, 55, 105948.

. T M A El-Aziz, J D Stockand, Infect. Genet. Evol. 83104327T. M. A. El-Aziz, J. D. Stockand, Infect. Genet. Evol. 2020, 83, 104327.

. N Baildya, N N Ghosh, A P Chattopadhyay, J. Mol. Struct. N. Baildya, N. N. Ghosh, A. P. Chattopadhyay, J. Mol. Struct. 2020, 1219, 128595.

. S Sanjuan-Reyes, L M Omez-Oliv An, H Islas-Flores, 127973S. SanJuan-Reyes, L. M. G omez-Oliv an, H. Islas-Flores, Che- mosphere 2020, 263, 127973.

. P Zhou, H Fan, T Lan, X L Yang, W F Shi, W Zhang, Y Zhu, Y W Zhang, Q M Xie, S Mani, X S Zheng, B Li, J M Li, H Guo, G Q Pei, X P An, J. W. Chen, L. Zhou, K. J. Mai, Z. X. Wu, D. Li, D. E. Anderson, L. BP. Zhou, H. Fan, T. Lan, X. L. Yang, W. F. Shi, W. Zhang, Y. Zhu, Y. W. Zhang, Q. M. Xie, S. Mani, X. S. Zheng, B. Li, J. M. Li, H. Guo, G. Q. Pei, X. P. An, J. W. Chen, L. Zhou, K. J. Mai, Z. X. Wu, D. Li, D. E. Anderson, L. B.

. S Y Zhang, Z Q Li, T T Mi, F He, P J Cong, R Guo, Y Huang, X L Luo, J Liu, Y Chen, Q Huang, X L L Sun, Y Y Zhang, S Z Wang, Y S Xing, Y Chen, J Sun, P Li, L F Daszak, Z L Wang, Y G Shi, J Y Tong, Ma, Nature. 556255Zhang, S. Y. Li, Z. Q. Mi, T. T. He, F. Cong, P. J. Guo, R. Huang, Y. Luo, X. L. Liu, J. Chen, Y. Huang, Q. Sun, X. L. L. Zhang, Y. Y. Wang, S. Z. Xing, Y. S. Chen, Y. Sun, J. Li, P. Daszak, L. F. Wang, Z. L. Shi, Y. G. Tong, J. Y. Ma, Nature 2018, 556, 255.

. C Drosten, S Günther, W Preiser, S Van Der Werf, H R Brodt, S Becker, H Rabenau, M Panning, L Kolesnikova, R A M Fouchier, A Berger, A M Burguière, J Cinatl, M Eickmann, N Escriou, K Grywna, S Kramme, J , C. Drosten, S. Günther, W. Preiser, S. van der Werf, H. R. Brodt, S. Becker, H. Rabenau, M. Panning, L. Kolesnikova, R. A. M. Fouchier, A. Berger, A. M. Burguière, J. Cinatl, M. Eickmann, N. Escriou, K. Grywna, S. Kramme, J. C.

. S Manuguerra, V Müller, M Rickerts, S Stürmer, H D Vieth, A D M E Klenk, H Osterhaus, H W Schmitz, Doerr, N. Engl. J. Med. 348Manuguerra, S. Müller, V. Rickerts, M. Stürmer, S. Vieth, H. D. Klenk, A. D. M. E. Osterhaus, H. Schmitz, H. W. Doerr, N. Engl. J. Med. 2003, 348, 1967.

. Y Guan, B J Zheng, Y Q He, X L Liu, Z X Zhuang, C L Cheung, S W Luo, P H Li, L J Zhang, Y J Guan, K M Butt, K L Wong, K W Chan, W Lim, K F Shortridge, K Y Yuen, J S M Peiris, L L M Poon, Science. 302276Y. Guan, B. J. Zheng, Y. Q. He, X. L. Liu, Z. X. Zhuang, C. L. Cheung, S. W. Luo, P. H. Li, L. J. Zhang, Y. J. Guan, K. M. Butt, K. L. Wong, K. W. Chan, W. Lim, K. F. Shortridge, K. Y. Yuen, J. S. M. Peiris, L. L. M. Poon, Science 2003, 302, 276.

S K P Lau, P C Y Woo, K S M Li, Y Huang, H W Tsoi, B H L Wong, S S Y Wong, S Y Leung, K H Chan, K Y Yuen, Proc. Natl. Acad. Sci. Natl. Acad. SciUSA10214040S. K. P. Lau, P. C. Y. Woo, K. S. M. Li, Y. Huang, H. W. Tsoi, B. H. L. Wong, S. S. Y. Wong, S. Y. Leung, K. H. Chan, K. Y. Yuen, Proc. Natl. Acad. Sci. USA 2005, 102, 14040.

. W Li, Z Shi, M Yu, W Ren, C Smith, J H Epstein, H Wang, G Crameri, Z Hu, H Zhang, J Zhang, J Mceachern, H Field, P Daszak, B T Eaton, S Zhang, L F Wang, 676W. Li, Z. Shi, M. Yu, W. Ren, C. Smith, J. H. Epstein, H. Wang, G. Crameri, Z. Hu, H. Zhang, J. Zhang, J. McEachern, H. Field, P. Daszak, B. T. Eaton, S. Zhang, L. F. Wang, Sci- ence 2005, 310, 676.

. X Y Ge, J L Li, X L Yang, A A Chmura, G Zhu, J H Epstein, J K Mazet, B Hu, W Zhang, C Peng, Y J Zhang, C M Luo, B Tan, N Wang, Y Zhu, G Crameri, S Y Zhang, L F Wang, P Daszak, Z L Shi, Nature. 535X. Y. Ge, J. L. Li, X. L. Yang, A. A. Chmura, G. Zhu, J. H. Epstein, J. K. Mazet, B. Hu, W. Zhang, C. Peng, Y. J. Zhang, C. M. Luo, B. Tan, N. Wang, Y. Zhu, G. Crameri, S. Y. Zhang, L. F. Wang, P. Daszak, Z. L. Shi, Nature 2013, 503, 535.

. B He, Y Zhang, L Xu, W Yang, F Yang, Y Feng, L Xia, J Zhou, W Zhen, Y Feng, H Guo, H Zhang, C Tu, J. Virol. 887070B. He, Y. Zhang, L. Xu, W. Yang, F. Yang, Y. Feng, L. Xia, J. Zhou, W. Zhen, Y. Feng, H. Guo, H. Zhang, C. Tu, J. Virol. 2014, 88, 7070.

. X L Yang, B Hu, B Wang, M N Wang, Q Zhang, W Zhang, L J Wu, X Y Ge, Y Z Zhang, P Daszak, L F Wang, Z L Shi, J. Virol. 3253X. L. Yang, B. Hu, B. Wang, M. N. Wang, Q. Zhang, W. Zhang, L. J. Wu, X. Y. Ge, Y. Z. Zhang, P. Daszak, L. F. Wang, Z. L. Shi, J. Virol. 2016, 90, 3253.

. Z Wu, L Yang, X Ren, J Zhang, F Yang, S Zhang, Q Jin, J. Infect. Dis. 213579Z. Wu, L. Yang, X. Ren, J. Zhang, F. Yang, S. Zhang, Q. Jin, J. Infect. Dis 2016, 213, 579.

. L Wang, S Fu, Y Cao, H Zhang, Y Feng, W Yang, K Nie, X Ma, G Liang, Emerg. Microbes. Infect. 614L. Wang, S. Fu, Y. Cao, H. Zhang, Y. Feng, W. Yang, K. Nie, X. Ma, G. Liang, Emerg. Microbes. Infect 2017, 6, e14.

. B Hu, L P Zeng, X L Yang, X Y Ge, W Zhang, B Li, J Z Xie, X R Shen, Y Z Zhang, N Wang, D S Luo, X S , B. Hu, L. P. Zeng, X. L. Yang, X. Y. Ge, W. Zhang, B. Li, J. Z. Xie, X. R. Shen, Y. Z. Zhang, N. Wang, D. S. Luo, X. S.

. M N Zheng, P Wang, L F Daszak, J Wang, Z L Cui, Shi, PLoS Pathog. 13Zheng, M. N. Wang, P. Daszak, L. F. Wang, J. Cui, Z. L. Shi, PLoS Pathog. 2017, 13, e1006698.

. Y Fan, K Zhao, Z L Shi, P Zhou, 11210Y. Fan, K. Zhao, Z. L. Shi, P. Zhou, Viruses 2019, 11, 210.

. N Zhu, D Zhang, W Wang, X Li, B Yang, J Song, X Zhao, B Huang, W Shi, R Lu, P Niu, F Zhan, X Ma, D Wang, W Xu, G Wu, F Gao, W A Tan, N. Engl. J. Med. 2020727N. Zhu, D. Zhang, W. Wang, X. Li, B. Yang, J. Song, X. Zhao, B. Huang, W. Shi, R. Lu, P. Niu, F. Zhan, X. Ma, D. Wang, W. Xu, G. Wu, F. Gao, W. A. Tan, N. Engl. J. Med. 2020, 382, 727.

. P Zhou, X L Yang, X G Wang, B Hu, L Zhang, W Zhang, H R Si, Y Zhu, B Li, C L Huang, H D Chen, J Chen, Y Luo, H Guo, R D Jiang, M Q Liu, Y Chen, X R Shen, X Wang, X S Zheng, K Zhao, Q J Chen, F Deng, L L Liu, B Yan, F X Zhan, Y Y Wang, G F Xiao, Z L Shi, Nature. 2020270P. Zhou, X. L. Yang, X. G. Wang, B. Hu, L. Zhang, W. Zhang, H. R. Si, Y. Zhu, B. Li, C. L. Huang, H. D. Chen, J. Chen, Y. Luo, H. Guo, R. D. Jiang, M. Q. Liu, Y. Chen, X. R. Shen, X. Wang, X. S. Zheng, K. Zhao, Q. J. Chen, F. Deng, L. L. Liu, B. Yan, F. X. Zhan, Y. Y. Wang, G. F. Xiao, Z. L. Shi, Nature 2020, 579, 270.

. K G Andersen, A Rambaut, W I Lipkin, E C Holmes, R F Garry, Nat. Med. 450K. G. Andersen, A. Rambaut, W. I. Lipkin, E. C. Holmes, R. F. Garry, Nat. Med. 2020, 26, 450.

. X Tian, C Li, A Huang, S Xia, S Lu, Z Shi, L Lu, S Jiang, Z Yang, Y Wu, T Ying, Emerg. Microbes. Infect. 2020382X. Tian, C. Li, A. Huang, S. Xia, S. Lu, Z. Shi, L. Lu, S. Jiang, Z. Yang, Y. Wu, T. Ying, Emerg. Microbes. Infect. 2020, 9, 382.

. H Li, Y Zhou, V Zhang, H Wang, Q Zhao, J Liu, Antimicrob. Agents. Chemother. 64H. Li, Y. Zhou, V. Zhang, H. Wang, Q. Zhao, J. Liu, Antimi- crob. Agents. Chemother. 2020, 64, e00483-20.

El-hoshoudy. A , J. Mol. Liq. 2020113968A. N. El-hoshoudy, J. Mol. Liq. 2020, 318, 113968.

. Y Abo-Zeid, N S M Ismail, G R Mclean, N M Hamdy, Eur. J. Pharm. Sci. 2020105465Y. Abo-zeid, N. S. M. Ismail, G. R. McLean, N. M. Hamdy, Eur. J. Pharm. Sci. 2020, 153, 105465.

. S Mohanty, M H A I Rashid, M Mridul, C Mohanty, S Swayamsiddha, Metab. Syndr. Clin. Res. Rev. 141027S. Mohanty, M. H. A. I. Rashid, M. Mridul, C. Mohanty, S. Swayamsiddha, Metab. Syndr. Clin. Res. Rev. 2020, 14, 1027.

. H Rahmani, E Davoudi-Monfared, A Nourian, H Khalili, N Hajizadeh, N Z Jalalabadi, M R Fazeli, M Ghazaeian, M S Yekaninejad, Int. Immunopharmacol. 88106903H. Rahmani, E. Davoudi-Monfared, A. Nourian, H. Khalili, N. Hajizadeh, N. Z. Jalalabadi, M. R. Fazeli, M. Ghazaeian, M. S. Yekaninejad, Int. Immunopharmacol. 2020, 88, 106903.

. A E C Antunes, G Vinderola, D Xavier-Santos, K Sivieri, Food. Res. Int. 136109577A. E. C. Antunes, G. Vinderola, D. Xavier-Santos, K. Sivieri, Food. Res. Int. 2020, 136, 109577.

. S Ismail, S Ahmad, S S Azam, J. Mol. Liq. 113612S. Ismail, S. Ahmad, S. S. Azam, J. Mol. Liq. 2020, 314, 113612.

. M Kumar, K Taki, R Gahlot, A Sharma, K Dhangar, Sci. Total Environ. 734139278M. Kumar, K. Taki, R. Gahlot, A. Sharma, K. Dhangar, Sci. Total Environ. 2020, 734, 139278.

. World Health Organization. World Health Organization, https://www.who.int/ emergencies/diseases/novel-coronavirus-2019 (accessed: April 14 2022).

. A Mullard, Rev. Drug Discov. 11505A. Mullard, Rev. Drug Discov. 2012, 11, 505.

. P Colson, J M Rolain, D Raoult, Int. J. Antimicrob. Agents. 55105923P. Colson, J. M. Rolain, D. Raoult, Int. J. Antimicrob. Agents 2020, 55, 105923.

. P Colson, J M Rolain, J C Lagier, P Brouqui, D Raoult, Int. J. Antimicrob. Agents. 55105932P. Colson, J. M. Rolain, J. C. Lagier, P. Brouqui, D. Raoult, Int. J. Antimicrob. Agents 2020, 55, 105932.

. A H J Kim, J A Sparks, J W Liew, M S Putman, F Berenbaum, A Duarte-García, E R Graef, P Korsten, S E Sattui, E Sirotich, M F Ugarte-Gil, K Webb, R A Grainger, Ann. Intern. Med. 2020819A. H. J. Kim, J. A. Sparks, J. W. Liew, M. S. Putman, F. Berenbaum, A. Duarte-García, E. R. Graef, P. Korsten, S. E. Sattui, E. Sirotich, M. F. Ugarte-Gil, K. Webb, R. A. Grainger, Ann. Intern. Med. 2020, 172, 819.

. K Stadler, V Masignani, M Eickmann, S Becker, S Abrignani, H.-D Klenk, R Rappuoli, Osong. Public Health Res. Perspect. 209. b)J. M. Kim, Y. S. Chung, H. J. Jo, N. J. Lee, M. S. Kim, S. H. Woo, S. Park, J. W. Kim, H. M. Kim, M. G. Han13Nat. Rev. Microbiol.K. Stadler, V. Masignani, M. Eickmann, S. Becker, S. Abrignani, H.-D. Klenk, R. Rappuoli, Nat. Rev. Microbiol. 2003, 1, 209. b)J. M. Kim, Y. S. Chung, H. J. Jo, N. J. Lee, M. S. Kim, S. H. Woo, S. Park, J. W. Kim, H. M. Kim, M. G. Han, Osong. Public Health Res. Perspect. 2020, 11, 3.

. L Mousavizadeh, S Ghasemi, J. Microbiol. Immunol. Infect. 54159L. Mousavizadeh, S. Ghasemi, J. Microbiol. Immunol. Infect. 2021, 54, 159.

. R Yan, Y Zhang, Y Li, L Xia, Y Guo, Q Zhou, Science. 20201444R. Yan, Y. Zhang, Y. Li, L. Xia, Y. Guo, Q. Zhou, Science 2020, 367, 1444.

. J Shang, G Ye, K Shi, Y Wan, C Luo, H Aihara, Q Geng, A Auerbach, F Li, Nature. 2020221J. Shang, G. Ye, K. Shi, Y. Wan, C. Luo, H. Aihara, Q. Geng, A. Auerbach, F. Li, Nature 2020, 581, 221.

. Y Wan, J Shang, R Graham, R S Baric, F Li, J. Virol. 94127Y. Wan, J. Shang, R. Graham, R. S. Baric, F. Li, J. Virol. 2020, 94, e00127.

. Y L Siu, K T Teoh, J Lo, C M Chan, F Kien, N Escriou, S W Tsao, J M Nicholls, R Altmeyer, J S M Peiris, R Bruzzone, B , J. Virol. 8211318Y. L. Siu, K. T. Teoh, J. Lo, C. M. Chan, F. Kien, N. Escriou, S. W. Tsao, J. M. Nicholls, R. Altmeyer, J. S. M. Peiris, R. Bruzzone, B. Nal, J. Virol. 2008, 82, 11318.

. L Alanagreh, F Alzoughool, M Atoum, 2020331L. Alanagreh, F. Alzoughool, M. Atoum, Pathogens 2020, 9, 331.

. C C Stobart, N R Sexton, H Munjal, X Lu, K L Molland, S Tomar, A D Mesecar, M R Denison, J. Virol. 8712611C. C. Stobart, N. R. Sexton, H. Munjal, X. Lu, K. L. Molland, S. Tomar, A. D. Mesecar, M. R. Denison, J. Virol. 2013, 87, 12611.

. Z Song, Y Xu, L Bao, L Zhang, P Yu, Y Qu, H Zhu, W Zhao, C Qin, Viruses. 1159Z. Song, Y. Xu, L. Bao, L. Zhang, P. Yu, Y. Qu, H. Zhu, W. Zhao, C. Qin, Viruses 2019, 11, 59.

. T Hu, C Chen, H Li, Y Dou, M Zhou, D Lu, Q Zong, Y Li, C Yang, Z Zhong, N Singh, H Hu, R Zhang, H Yang, D Su, Protein Sci. 261037T. Hu, C. Chen, H. Li, Y. Dou, M. Zhou, D. Lu, Q. Zong, Y. Li, C. Yang, Z. Zhong, N. Singh, H. Hu, R. Zhang, H. Yang, D. Su, Protein Sci. 2017, 26, 1037.

. J S M Peiris, Y Guan, K Y Yuen, Nat. Med. 1088J. S. M. Peiris, Y. Guan, K. Y. Yuen, Nat. Med. 2004, 10, S88.

. K Narayanan, A Maeda, J Maeda, S Makino, J. Virol. 748127K. Narayanan, A. Maeda, J. Maeda, S. Makino, J. Virol. 2000, 74, 8127.

. E De Wit, N Van Doremalen, D Falzarano, V J Munster, Nat. Rev. Microbiol. 14523E. De Wit, N. Van Doremalen, D. Falzarano, V. J. Munster, Nat. Rev. Microbiol. 2016, 14, 523.

. J L Nieto-Torres, M L Dediego, E Álvarez, J M Jiménez-Guardeño, J A Regla-Nava, M Llorente, L Kremer, S Shuo, L Enjuanes, Virology. 69J. L. Nieto-Torres, M. L. DeDiego, E.Álvarez, J. M. Jiménez- Guardeño, J. A. Regla-Nava, M. Llorente, L. Kremer, S. Shuo, L. Enjuanes, Virology 2011, 415, 69.

. L Du, Y He, Y Zhou, S Liu, B J Zheng, S Jiang, Nat. Rev. Microbiol. 7226L. Du, Y. He, Y. Zhou, S. Liu, B. J. Zheng, S. Jiang, Nat. Rev. Microbiol. 2009, 7, 226.

. T A Singhal, Indian J. Pediatr. 2020281T. A. Singhal, Indian J. Pediatr. 2020, 87, 281.

. J Gao, Z Tian, X Yang, BioScience Trends. 1472J. Gao, Z. Tian, X. Yang, BioScience Trends 2020, 14, 72.

. J Fantini, C D Scala, H Chahinian, N Yahi, Int. J. Antimicrob. Agents. 55105960J. Fantini, C. D. Scala, H. Chahinian, N. Yahi, Int. J. Antimicrob. Agents 2020, 55, 105960.

. J Liu, R Cao, M Xu, X Wang, H Zhang, H Hu, Y Li, Z Hu, W Zhong, M Wang, Cell Discov, 616J. Liu, R. Cao, M. Xu, X. Wang, H. Zhang, H. Hu, Y. Li, Z. Hu, W. Zhong, M. Wang, Cell Discov. 2020, 6, 16.

. M F Marmor, U Kellner, T Y Y Lai, R B Melles, W F Mieler, Ophthalmology. 1231386M. F. Marmor, U. Kellner, T. Y. Y. Lai, R. B. Melles, W. F. Mieler, Ophthalmology 2016, 123, 1386.

. M Wang, R Cao, L Zhang, X Yang, J Liu, M Xu, Z Shi, Z Hu, W Zhong, G Xiao, Cell Res. M. Wang, R. Cao, L. Zhang, X. Yang, J. Liu, M. Xu, Z. Shi, Z. Hu, W. Zhong, G. Xiao, Cell Res. 2020, 30, 269.

. L Dong, S Hu, J Gao, Drug Discov. Ther. 1458L. Dong, S. Hu, J. Gao, Drug Discov. Ther. 2020, 14, 58.

. M L Holshue, C Debolt, S Lindquist, K H Lofy, J Wiesman, H Bruce, C Spitters, K Ericson, S Wilkerson, A Tural, G Diaz, A Cohn, L Fox, A Patel, S I Gerber, L Kim, S Tong, X Lu, S Lindstrom, M A Pallansch, W C Weldon, H M Biggs, T M Uyeki, S K Pillai, N. Engl. J. Med. 2020929M. L. Holshue, C. DeBolt, S. Lindquist, K. H. Lofy, J. Wiesman, H. Bruce, C. Spitters, K. Ericson, S. Wilkerson, A. Tural, G. Diaz, A. Cohn, L. Fox, A. Patel, S. I. Gerber, L. Kim, S. Tong, X. Lu, S. Lindstrom, M. A. Pallansch, W. C. Weldon, H. M. Biggs, T. M. Uyeki, S. K. Pillai, N. Engl. J. Med. 2020, 382, 929.

Food and Drug Administration. Food and Drug Administration. 2020, https://www.fda.gov/ media/137564/download (accessed: September 08 2020).

. B Morgenstern, M Michaelis, P C Baer, H W Doerr, J. Cinatl. Biochem. Biophys. Res. Commun. 326905B. Morgenstern, M. Michaelis, P. C. Baer, H. W. Doerr, J. Cinatl. Biochem. Biophys. Res. Commun. 2005, 326, 905.

. E W Mcchesney, Am. J. Med. 7511E. W. McChesney, Am. J. Med. 1983, 75, 11.

. S J Lee, E Silverman, J M Bargman, Nat. Rev. Nephrol. S. J. Lee, E. Silverman, J. M. Bargman, Nat. Rev. Nephrol. 2011, 7, 718.

. A J Popert, Rheumatology Rehabil. 15235A. J. Popert, Rheumatology Rehabil 1976, 15, 235.

. A L Laaksonen, V Koskiahde, K Juva, . J Scand, 3103A. L. Laaksonen, V. Koskiahde, K. Juva, Scand. J. Rheumatol. 1974, 3, 103.

. C Huang, Y Wang, X Li, L Ren, J Zhao, Y Hu, L Zhang, G Fan, J Xu, X Gu, Z Cheng, T Yu, J Xia, Y Wei, W Wu, X Xie, W Yin, H Li, M Liu, Y Xiao, H Gao, L Guo, J Xie, G Wang, R Jiang, Z Gao, Q Jin, J Wang, B Cao, Lancet. 2020497C. Huang, Y. Wang, X. Li, L. Ren, J. Zhao, Y. Hu, L. Zhang, G. Fan, J. Xu, X. Gu, Z. Cheng, T. Yu, J. Xia, Y. Wei, W. Wu, X. Xie, W. Yin, H. Li, M. Liu, Y. Xiao, H. Gao, L. Guo, J. Xie, G. Wang, R. Jiang, Z. Gao, Q. Jin, J. Wang, B. Cao, Lancet 2020, 395, 497.

. Y Wang, D Zhang, G Du, R Du, J Zhao, Y Jin, S Fu, L Gao, Z Cheng, Q Lu, Y Hu, G Luo, K Wang, Y Lu, H Li, S Wang, S Ruan, C Yang, C Mei, Y Wang, D Ding, F Wu, X Tang, X Ye, Y Ye, B Liu, J Yang, W Yin, A Wang, G Fan, F Zhou, Z Liu, X Gu, J Xu, L Shang, Y Zhang, L Cao, T Guo, Y Wan, H Qin, Y Jiang, T Jaki, F G Hayden, P W Horby, B Cao, C Wang, Lancet. 20201569Y. Wang, D. Zhang, G. Du, R. Du, J. Zhao, Y. Jin, S. Fu, L. Gao, Z. Cheng, Q. Lu, Y. Hu, G. Luo, K. Wang, Y. Lu, H. Li, S. Wang, S. Ruan, C. Yang, C. Mei, Y. Wang, D. Ding, F. Wu, X. Tang, X. Ye, Y. Ye, B. Liu, J. Yang, W. Yin, A. Wang, G. Fan, F. Zhou, Z. Liu, X. Gu, J. Xu, L. Shang, Y. Zhang, L. Cao, T. Guo, Y. Wan, H. Qin, Y. Jiang, T. Jaki, F. G. Hayden, P. W. Horby, B. Cao, C. Wang, Lancet 2020, 395, 1569.

. R Shetty, A Ghosh, S G Honavar, P Khamar, S Sethu, Indian J. Ophthalmol. 693R. Shetty, A. Ghosh, S. G. Honavar, P. Khamar, S. Sethu, Indian J. Ophthalmol. 2020, 68, 693.

. X Liu, X J Wang, J. Genet. Genomics. 2020119X. Liu, X. J. Wang, J. Genet. Genomics. 2020, 47, 119.

. K Dhama, K Sharun, R Tiwari, M Dadar, Y S Malik, K P Singh, W Chaicumpa, Hum. Vaccin. Immunother. 161232K. Dhama, K. Sharun, R. Tiwari, M. Dadar, Y. S. Malik, K. P. Singh, W. Chaicumpa, Hum. Vaccin. Immunother. 2020, 16, 1232.

. J M Sanders, M L Monogue, T Z Jodlowski, J B Cutrell, J. Am. Med. Assoc. 3231824J. M. Sanders, M. L. Monogue, T. Z. Jodlowski, J. B. Cutrell, J. Am. Med. Assoc. 2020, 323, 1824.

who-director-general-s-opening-remarks-at-the-mediabriefing-on-covid. WHO Director-General's opening remarks at the media briefing on COVID-19. 2020. WHO Director-General's opening remarks at the media brief- ing on COVID-19. 2020, https://www.who.int/dg/speeches/ detail/who-director-general-s-opening-remarks-at-the-media- briefing-on-covid-19-11-march-2020 (accessed: September 08 2020).

. A Zumla, J F W Chan, E I Azhar, D S C Hui, K Y Yuen, Nat. Rev. Drug Discov. 5327A. Zumla, J. F. W. Chan, E. I. Azhar, D. S. C. Hui, K. Y. Yuen, Nat. Rev. Drug Discov. 2016, 5, 327.

. X Yao, F Ye, M Zhang, C Cui, B Huang, P Niu, X Liu, L Zhao, E Dong, C Song, S Zhan, R Lu, H Li, W Tan, D Liu, Clin. Infect. Dis. 202032X. Yao, F. Ye, M. Zhang, C. Cui, B. Huang, P. Niu, X. Liu, L. Zhao, E. Dong, C. Song, S. Zhan, R. Lu, H. Li, W. Tan, D. Liu, Clin. Infect. Dis. 2020, 71, 32.

. F Romanelli, K M Smith, A D Hoven, Curr. Pharm. Des. 102643F. Romanelli, K. M. Smith, A. D. Hoven, Curr. Pharm. Des. 2004, 10, 2643.

. E Keyaerts, L Vijgen, P Maes, J Neyts, M V Ranst, Biochem. Biophys. Res. Commun. 323264E. Keyaerts, L. Vijgen, P. Maes, J. Neyts, M. V. Ranst, Bio- chem. Biophys. Res. Commun. 2004, 323, 264.

. M J Vincent, E Bergeron, S Benjannet, B R Erickson, P E Rollin, T G Ksiazek, N G Seidah, S T Nichol, Virol. J. 269M. J. Vincent, E. Bergeron, S. Benjannet, B. R. Erickson, P. E. Rollin, T. G. Ksiazek, N. G. Seidah, S. T. Nichol, Virol. J. 2005, 2, 69.

. E E Ooi, J S W Chew, J P Loh, R C S Chua, Virol. J. 339E. E. Ooi, J. S. W. Chew, J. P. Loh, R. C. S. Chua, Virol. J. 2006, 3, 39.

. C Li, X Zhu, X Ji, N Quanquin, Y Q Deng, M Tian, R Aliyari, X Zuo, L Yuan, S K Afridi, X F Li, J U Jung, K Nielsen-Saines, F X F Qin, C F Qin, Z Xu, G Cheng, 24189C. Li, X. Zhu, X. Ji, N. Quanquin, Y. Q. Deng, M. Tian, R. Aliyari, X. Zuo, L. Yuan, S. K. Afridi, X. F. Li, J. U. Jung, K. Nielsen-Saines, F. X. F. Qin, C. F. Qin, Z. Xu, G. Cheng, EBio- Medicine 2017, 24, 189.

Efficacy of Chloroquine and Lopinavir/ Ritonavir in mild/general novel coronavirus (CoVID-19) infections: a prospective, open-label, multicenter randomized controlled clinical study. J Huang, J. Huang. Efficacy of Chloroquine and Lopinavir/ Ritonavir in mild/general novel coronavirus (CoVID-19) infections: a prospective, open-label, multicenter randomized controlled clinical study. 2020, http://www.chictr.org.cn/showproj.aspx? proj=49263 (accessed: September 08 2020).

Ministry of Science and Technology of China: chloroquine phosphate is effective in the treatment of novel coronavirus pneumonia. Y Li, Y. Li. Ministry of Science and Technology of China: chloro- quine phosphate is effective in the treatment of novel corona- virus pneumonia. 2020, http://news.ynet.com/2020/02/17/ 2388070t70.html (accessed: September 08 2020).

. H Weniger, Bull. World Health. 79906H. Weniger, Bull. World Health. 1979, 79, 906.

. C Biot, W Daher, N Chavain, T Fandeur, J Khalife, D Dive, E De Clercq, J. Med. Chem. 2845C. Biot, W. Daher, N. Chavain, T. Fandeur, J. Khalife, D. Dive, E. De Clercq, J. Med. Chem. 2006, 49, 2845.

. P Gautret, J C Lagier, P Parola, V T Hoang, L Meddeb, M Mailhe, B Doudier, J Courjon, V Giordanengo, V E Vieira, H T Dupont, S Honoré, P Colson, E Chabrière, B L Scola, J M Rolain, P Brouqui, D Raoult, Int. J. Antimicrob. Agents. 56105949P. Gautret, J. C. Lagier, P. Parola, V. T. Hoang, L. Meddeb, M. Mailhe, B. Doudier, J. Courjon, V. Giordanengo, V. E. Vieira, H. T. Dupont, S. Honoré, P. Colson, E. Chabrière, B. L. Scola, J. M. Rolain, P. Brouqui, D. Raoult, Int. J. Antimicrob. Agents 2020, 56, 105949.

. P Gautret, J C Lagier, P Parola, V T Hoang, L Meddeb, J Sevestre, M Mailhe, B Doudier, C Aubry, S Amrane, P Seng, M Hocquart, C Eldin, J Finance, V E Vieira, H T Dupont, S Honoré, A Stein, M Million, P Colson, B L Scola, V Veit, A Jacquier, J C Deharo, M Drancourt, P E Fournier, J M Rolain, P Brouqui, D Raoult, Travel Med. Infect. Dis. 2020101663P. Gautret, J. C. Lagier, P. Parola, V. T. Hoang, L. Meddeb, J. Sevestre, M. Mailhe, B. Doudier, C. Aubry, S. Amrane, P. Seng, M. Hocquart, C. Eldin, J. Finance, V. E. Vieira, H. T. Dupont, S. Honoré, A. Stein, M. Million, P. Colson, B. L. Scola, V. Veit, A. Jacquier, J. C. Deharo, M. Drancourt, P. E. Fournier, J. M. Rolain, P. Brouqui, D. Raoult, Travel Med. Infect. Dis. 2020, 34, 101663.

. C Jun, L Danping, L Li, L Ping, X Qingnian, X Lu, L Yun, H Dan, S Shuli, Z Dandan, Q Zhiping, L Tao, S Yinzhong, L Hongzhou, J. Zhejiang Univ. (Med. Sci.). 2020215C. Jun, L. Danping, L. Li, L. Ping, X. Qingnian, X. Lu, L. Yun, H. Dan, S. Shuli, Z. Dandan, Q. Zhiping, L. Tao, S. Yinzhong, L. Hongzhou, J. Zhejiang Univ. (Med. Sci.). 2020, 49, 215.

. Z Chen, J Hu, Z Zhang, S Jiang, S Han, D Yan, R Zhuang, B Hu, Z Zhang, 10.1101/2020.03.22.200407582020Z. Chen, J. Hu, Z. Zhang, S. Jiang, S. Han, D. Yan, R. Zhuang, B. Hu, Z. Zhang, medRxiv 2020, 1, 1. https://doi.org/ 10.1101/2020.03.22.20040758

. J M Molina, C Delaugerre, J L Goff, B Mela-Lima, D Ponscarme, L Goldwirt, N Castro, Med. Mal. Infect. 50384J. M. Molina, C. Delaugerre, J. L. Goff, B. Mela-Lima, D. Ponscarme, L. Goldwirt, N. de Castro, Med. Mal. Infect. 2020, 50, 384.

. R I Fox, Semin. Arthritis Rheum. 2382R. I. Fox, Semin. Arthritis Rheum. 1993, 23, 82.

. J Quintart, M A Leroy-Houyet, A Trouet, P Baudhuin, J. Cell Biol. 82644J. Quintart, M. A. Leroy-Houyet, A. Trouet, P. Baudhuin, J. Cell Biol. 1979, 82, 644.

. C Ying, E De Clercq, J Neyts, J. Viral Hepat. 779C. Ying, E. De Clercq, J. Neyts, J. Viral Hepat. 2000, 7, 79.

. D E Furst, 511LupusD. E. Furst, Lupus 1996, 5, 11.

. B N Rome, J Avorn, N. Engl. J. Med. 3822282B. N. Rome, J. Avorn, N. Engl. J. Med. 2020, 382, 2282.

Fact Sheet For Health Care Providers Emergency Use Authorization (EUA) of Hydroxychloroquine Sulfate Supplied From The Strategic National Stockpile For Treatment of Covid-19 In Certain Hospitalized Patients. Fact Sheet For Health Care Providers Emergency Use Autho- rization (EUA) of Hydroxychloroquine Sulfate Supplied From The Strategic National Stockpile For Treatment of Covid-19 In Certain Hospitalized Patients. 2020, https:// www.fda.gov/media/136537/download (accessed: September 08 2020).

. J Magagnoli, S Narendran, F Pereira, T Cummings, J W Hardin, S S Sutton, J. Ambati. Adv. Clin. Exp. Med. J. Magagnoli, S. Narendran, F. Pereira, T. Cummings, J. W. Hardin, S. S. Sutton, J. Ambati. Adv. Clin. Exp. Med. 2020, 1, 114.

Our World in Data; Total and daily confirmed COVID-19 cases. tab=table&time=2025-08-02&country= $INDOur World in Data; Total and daily confirmed COVID-19 cases, India. 2020, https://ourworldindata.org/grapher/covid- daily-vs-total-cases?tab=table&time=2025-08-02&country= $IND (accessed: April 11 2022).

National Taskforce for COVID-19. Revised advisory on the use of Hydroxychloroquine (HCQ) as prophylaxis for SARS-CoV-2 infection. National Taskforce for COVID-19. Revised advisory on the use of Hydroxychloroquine (HCQ) as prophylaxis for SARS- CoV-2 infection. 2020, https://www.icmr.gov.in/pdf/covid/ techdoc/V5_Revised_advisory_on_the_use_of_HCQ_SARS_ CoV2_infection.pdf (accessed: September 08 2020).

H Retallack, E D Lullo, C Arias, K A Knopp, M T Laurie, C Sandoval-Espinosa, W R M Leon, R Krencik, E M Ullian, J Spatazz, A A Pollen, C Mandel-Brehm, T J Nowakowski, A R Kriegstein, J L Derisi, Proc. Natl. Acad. Sci. USA. Natl. Acad. Sci. USA1134408H. Retallack, E. D. Lullo, C. Arias, K. A. Knopp, M. T. Laurie, C. Sandoval-Espinosa, W. R. M. Leon, R. Krencik, E. M. Ullian, J. Spatazz, A. A. Pollen, C. Mandel-Brehm, T. J. Nowakowski, A. R. Kriegstein, J. L. DeRisi, Proc. Natl. Acad. Sci. USA. 2016, 113, 4408.

. P B Madrid, R G Panchal, T K Warren, A C Shurtleff, A N Endsley, C E Green, A Kolokoltsov, R Davey, I D Manger, L Gilfillan, S Bavari, M J Tanga, ACS Infect. Dis. 1317P. B. Madrid, R. G. Panchal, T. K. Warren, A. C. Shurtleff, A. N. Endsley, C. E. Green, A. Kolokoltsov, R. Davey, I. D. Manger, L. Gilfillan, S. Bavari, M. J. Tanga, ACS Infect. Dis. 2015, 1, 317.

. E Bosseboeuf, M Aubry, T Nhan, J J De Pina, J M Rolain, D Raoult, D Musso, J. Antivir. Antiretrovir. 106E. Bosseboeuf, M. Aubry, T. Nhan, J. J. de Pina, J. M. Rolain, D. Raoult, D. Musso, J. Antivir. Antiretrovir. 2018, 10, 6.

. L B Bacharier, T W Guilbert, D T Mauger, S Boehmer, A Beigelman, A M Fitzpatrick, D J Jackson, S N Baxi, M Benson, C A D Burnham, M Cabana, M Castro, J , L. B. Bacharier, T. W. Guilbert, D. T. Mauger, S. Boehmer, A. Beigelman, A. M. Fitzpatrick, D. J. Jackson, S. N. Baxi, M. Benson, C. A. D. Burnham, M. Cabana, M. Castro, J. F.

. R Chmiel, M Covar, J M Daines, D A Gaffin, F Gentile, E Holguin, H W Israel, S C Kelly, R F Lazarus, Chmiel, R. Covar, M. Daines, J. M. Gaffin, D. A. Gentile, F. Holguin, E. Israel, H. W. Kelly, S. C. Lazarus, R. F.

. N Lemanskejr, K Ly, W Meade, J Morgan, T Moy, S P Olin, W Peters, J A Phipatanakul, H H Pongracic, K Raissy, W J Ross, C Sheehan, S J Sorkness, W G Szefler, S Teague, F D Thyne, Martinez, J. Am. Med. Assoc. 2034LemanskeJr, N. Ly, K. Meade, W. Morgan, J. Moy, T. Olin, S. P. Peters, W. Phipatanakul, J. A. Pongracic, H. H. Raissy, K. Ross, W. J. Sheehan, C. Sorkness, S. J. Szefler, W. G. Teague, S. Thyne, F. D. Martinez, J. Am. Med. Assoc. 2015, 314, 2034.

. C Axfors, A M Schmitt, P Janiaud, J Van&apos;t Hooft, S Abd-Elsalam, E F Abdo, B S Abella, J Akram, R K , a) C. Axfors, A. M. Schmitt, P. Janiaud, J. Van't Hooft, S. Abd-Elsalam, E. F. Abdo, B. S. Abella, J. Akram, R. K.

. D C Amaravadi, Y M Angus, S Arabi, L R Azhar, A W Baden, L Baker, T Belkhir, M A H Benfield, C P Berrevoets, T C Chen, S H Chen, Cheng, C. Y. Cheng, W. S. Chung, Y. Z. Cohen, L. N. Cowan, O. Dalgard, F. F. deAmaravadi, D. C. Angus, Y. M. Arabi, S. Azhar, L. R. Baden, A. W. Baker, L. Belkhir, T. Benfield, M. A. H. Berrevoets, C. P. Chen, T. C. Chen, S. H. Cheng, C. Y. Cheng, W. S. Chung, Y. Z. Cohen, L. N. Cowan, O. Dalgard, F. F. de

. M V G Almeida E Val, G C De Lacerda, L De Melo, V Derde, A Dubee, A C Elfakir, C M Gordon, T Hernandez-Cardenas, A I M Hills, Y W Hoepelman, B Huang, R Igau, F Jin, K S Jurado-Camacho, P G Khan, B Kremsner, C Y Kreuels, T Kuo, Y C Le, W P Lin, T H Lin, M N Lin, C Lyngbakken, B J Mcarthur, P Mcverry, W M Meza-Meneses, S C Monteiro, A Morpeth, M J Mourad, S Mulligan, S Murthy, S Naggie, A Narayanasamy, L A Nichol, S M Novack, N L O&apos;brien, L Okeke, R Perez, L Perez-Padilla, A Perrin, N E Remigio-Luna, F W Rivera-Martinez, S Rockhold, R Rodriguez-Llamazares, R Rolfe, H Rosa, V S Røsjø, T B Sampaio, M Seto, S Shahzad, J E Soliman, R Stout ; P, C S Bignardi, B M Vengrusa, A C Aquinoa, Neto, Pathog Global Health. Y. Chen, M.-X. Li, G.-D. Lu, H.-M. Shen, J. Zhou121538Int. J. Biol. Sci.Almeida E Val, M. V. G. de Lacerda, G. C. de Melo, L. Derde, V. Dubee, A. Elfakir, A. C. Gordon, C. M. Hernandez- Cardenas, T. Hills, A. I. M. Hoepelman, Y. W. Huang, B. Igau, R. Jin, F. Jurado-Camacho, K. S. Khan, P. G. Kremsner, B. Kreuels, C. Y. Kuo, T. Le, Y. C. Lin, W. P. Lin, T. H. Lin, M. N. Lyngbakken, C. McArthur, B. J. McVerry, P. Meza- Meneses, W. M. Monteiro, S. C. Morpeth, A. Mourad, M. J. Mulligan, S. Murthy, S. Naggie, S. Narayanasamy, A. Nichol, L. A. Novack, S. M. O'Brien, N. L. Okeke, L. Perez, R. Perez- Padilla, L. Perrin, A. Remigio-Luna, N. E. Rivera-Martinez, F. W. Rockhold, S. Rodriguez-Llamazares, R. Rolfe, R. Rosa, H. Røsjø, V. S. Sampaio, T. B. Seto, M. Shahzad, S. Soliman, J. E. Stout, I. Thirion-Romero, A. B. Troxel, T. Y. Tseng, N. A. Turner, R. J. Ulrich, S. R. Walsh, S. A. Webb, J. M. Weehuizen, M. Velinova, H. L. Wong, R. Wrenn, F. G. Zampieri, W. Zhong, D. Moher, S. N. Goodman, J. P. A. Ioannidis, L. G. Hemkens, Nature Commun. 2021, 12, 2349.b) P. R. Bignardi, C. S. Vengrusa, B. M. Aquinoa, A. C. Neto, Pathog Global Health 2021, 115, 139. c) Y. Chen, M.-X. Li, G.-D. Lu, H.-M. Shen, J. Zhou, Int. J. Biol. Sci. 2021, 17, 1538.

. E A R Winzeler ; A, A Parhizgar, Tahghighi, Iran J. Med. Sci. 455115Naturea)E. A. Winzeler, Nature 2008, 455, 751. b) A. R. Parhizgar, A. Tahghighi, Iran J. Med. Sci. 2017, 42, 115.

. C A Devaux, J M Rolain, P Colson, D Raoult, Int. J. Antimicrob. Agents. 55105938C. A. Devaux, J. M. Rolain, P. Colson, D. Raoult, Int. J. Antimicrob. Agents. 2020, 55, 105938.

. N J White, S Pukrittayakamee, T T Hien, M A Faiz, O A Mokuolu, A M Dondorp, Malaria. Lancet. 383723N. J. White, S. Pukrittayakamee, T. T. Hien, M. A. Faiz, O. A. Mokuolu, A. M. Dondorp, Malaria. Lancet 2014, 383, 723.

. A Savarino, L D Trani, I Donatelli, R Cauda, A Cassone, Lancet Infect. Dis. 667A. Savarino, L. D. Trani, I. Donatelli, R. Cauda, A. Cassone, Lancet Infect. Dis. 2006, 6, 67.

. A Cortegiani, G Ingoglia, M Ippolito, A Giarratano, S Einav, J. Crit. Care. 57279A. Cortegiani, G. Ingoglia, M. Ippolito, A. Giarratano, S. Einav, J. Crit. Care. 2020, 57, 279.

. H Akpovwa, Cell Biochem. Funct. 34191H. Akpovwa, Cell Biochem. Funct. 2016, 34, 191.

. M P Hage, M R Al-Badri, S T Azar, Ther , Adv. Endocrinol. Metab. 577M. P. Hage, M. R. Al-Badri, S. T. Azar, Ther. Adv. Endocrinol. Metab. 2014, 5, 77.

. J Poorvashree, D Suneela, Drug Delivery Transl. Res. 7709J. Poorvashree, D. Suneela, Drug Delivery Transl. Res. 2017, 7, 709.

. R A Capel, N Herring, M Kalla, A Yavari, G R Mirams, G Douglas, G Bub, K Channon, D J Paterson, D A Terrar, R A B Burton, Heart Rhythm. 122186R. A. Capel, N. Herring, M. Kalla, A. Yavari, G. R. Mirams, G. Douglas, G. Bub, K. Channon, D. J. Paterson, D. A. Terrar, R. A. B. Burton, Heart Rhythm 2015, 12, 2186.

. A R Surrey, H F Hammer, J. Am. Chem. Soc. 113A. R. Surrey, H. F. Hammer, J. Am. Chem. Soc. 1946, 68, 113.

. H Andersag, S Breitner, H , H. Andersag, S. Breitner, H. Jung. US2233970. 1941.

. N L Drake, H J Creech, J A Garman, S T Haywood, R M Peck, J O Van Hook, E Walton, J. Am. Chem. Soc. 1214N. L. Drake, H. J. Creech, J. A. Garman, S. T. Haywood, R. M. Peck, J. O. Van Hook, E. Walton, J. Am. Chem. Soc. 1946, 68, 1214.

. C C Price, R M Roberts, J. Am. Chem. Soc. 1204C. C. Price, R. M. Roberts, J. Am. Chem. Soc. 1946, 68, 1204.

. C F Gerschickter, L Rice, Us2947664, C. F. Gerschickter, L. Rice. US2947664. 1960.

. E H Northey, P F Dreisbach, E. H. Northey, P. F. Dreisbach. US2478125. 1949.

. W Schuelemann, F Schoenhoefer, A Wingler, De486079, W. Schuelemann, F. Schoenhoefer, A. Wingler. DE486079. 1924.

. R C Elderfield, W J Gensler, F Brody, J D Head, S C Dickerman, L Wiederhold, C B Cramer, H A Hageman, F J Kreyse, J M Griffing, S M Kupchan, B Newman, J T Maynard, J. Am. Chem. Soc. 1579R. C. Elderfield, W. J. Gensler, F. Brody, J. D. Head, S. C. Dickerman, L. Wiederhold, C. B. Cramer, H. A. Hageman, F. J. Kreyse, J. M. Griffing, S. M. Kupchan, B. Newman, J. T. Maynard, J. Am. Chem. Soc. 1946, 68, 1579.

) A R Surrey, H F Hammer, 1814. b) A. R. Surrey. US2546658. 72a) A. R. Surrey, H. F. Hammer, J. Am. Chem. Soc 1950, 72, 1814. b) A. R. Surrey. US2546658. 1951.

. Y S Min, H S Cho, K W Mo, Wo2010027150, Y. S. Min, H. S. Cho, K. W. Mo. WO2010027150. 2010.

An Open Letter from Daniel O'Day, Chairman & CEO, Gilead Sciences, Daniel O'Day. An Open Letter from Daniel O'Day, Chairman & CEO, Gil- ead Sciences, Daniel O'Day. 2020.

Remdesivir EUA letter of authorization. US Food and Drug AdministrationUS Food and Drug Administration. Remdesivir EUA letter of authorization. 2020, https://www.fda.gov/media/137564/ (accessed: December 30 2021).

An Open Letter from Daniel O'Day, Chairman & CEO, Gilead Sciences. An Open Letter from Daniel O'Day, Chairman & CEO, Gil- ead Sciences. 2020, https://www.gilead.com/stories/articles/ an-openletter-from-daniel-oday-june-29

Gilead Sciences Update on Veklury ® (Remdesivir) Manufacturing Network. 2020. Gilead Sciences Update on Veklury ® (Remdesivir) Manufacturing Network. 2020, https://www.gilead.com/ news-andpress/company-statements/gilead-sciences-update- on-vekluryremdesivir-manufacturing-network

. D Siegel, H C Hui, E Doerffler, M O Clarke, K Chun, L Zhang, S Neville, E Carra, W Lew, B Ross, Q Wang, L Wolfe, R Jordan, V Soloveva, J Knox, J Perry, M Perron, K M Stray, O Barauskas, J Y Feng, Y Xu, G Lee, A L Rheingold, A S Ray, R Bannister, R Strickley, S Swaminathan, W A Lee, S Bavari, T Cihlar, M K Lo, T K Warren, R L Mackman, J. Med. Chem. 1648D. Siegel, H. C. Hui, E. Doerffler, M. O. Clarke, K. Chun, L. Zhang, S. Neville, E. Carra, W. Lew, B. Ross, Q. Wang, L. Wolfe, R. Jordan, V. Soloveva, J. Knox, J. Perry, M. Perron, K. M. Stray, O. Barauskas, J. Y. Feng, Y. Xu, G. Lee, A. L. Rheingold, A. S. Ray, R. Bannister, R. Strickley, S. Swaminathan, W. A. Lee, S. Bavari, T. Cihlar, M. K. Lo, T. K. Warren, R. L. Mackman, J. Med. Chem. 2017, 60, 1648.

. T K Warren, R Jordan, M K Lo, A S Ray, R L Mackman, V Soloveva, D Siegel, M Perron, R Bannister, H C Hui, N Larson, R Strickley, J Wells, K S Stuthman, S A Van Tongeren, N L Garza, G Donnelly, A C Shurtleff, C J Retterer, D Gharaibeh, R Zamani, T Kenny, B P Eaton, E Grimes, L S Welch, L Gomba, C L Wilhelmsen, D K , T. K. Warren, R. Jordan, M. K. Lo, A. S. Ray, R. L. Mackman, V. Soloveva, D. Siegel, M. Perron, R. Bannister, H. C. Hui, N. Larson, R. Strickley, J. Wells, K. S. Stuthman, S. A. Van Tongeren, N. L. Garza, G. Donnelly, A. C. Shurtleff, C. J. Retterer, D. Gharaibeh, R. Zamani, T. Kenny, B. P. Eaton, E. Grimes, L. S. Welch, L. Gomba, C. L. Wilhelmsen, D. K.

. J E Nichols, E R Nuss, J R Nagle, G Kugelman, E Palacios, S Doerffler, E Neville, M O Carra, C F Clarke, Nichols, J. E. Nuss, E. R. Nagle, J. R. Kugelman, G. Palacios, E. Doerffler, S. Neville, E. Carra, M. O. Clarke, C. F.

. W A Spiropoulou, S T Lee, T Nichol, S Cihlar, Bavari, Nature. 531381Spiropoulou, W. A. Lee, S. T. Nichol, T. Cihlar, S. Bavari, Nature 2016, 531, 381.

. M Wang, L Zhang, X Huo, Z Zhang, Q Yuan, P Li, J Chen, Y Zou, Z Wu, W Zhang, Angew. Chem., Int. Ed. 59M. Wang, L. Zhang, X. Huo, Z. Zhang, Q. Yuan, P. Li, J. Chen, Y. Zou, Z. Wu, W. Zhang, Angew. Chem., Int. Ed. 2020, 59, 20814.

. M J Frisch, G W Trucks, H B Schlegel, G E Scuseria, M A Robb, J R Cheeseman, G Scalmani, V Barone, G A Petersson, H Nakatsuji, X Li, M Caricato, A Marenich, J Bloino, B G Janesko, R Gomperts, B Mennucci, H P , M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. Marenich, J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P.

. J V Hratchian, A F Ortiz, J L Izmaylov, D Sonnenberg, F Williams-Young, F Ding, F Lipparini, J Egidi, B Goings, A Peng, T Petrone, D Henderson, V G Ranasinghe, J Zakrzewski, N Gao, G Rega, W Zheng, M Liang, M Hada, K Ehara, R Toyota, J Fukuda, M Hasegawa, T Ishida, Y Nakajima, O Honda, H Kitao, T Nakai, K Vreven, J A Throssell, J E MontgomeryJr, F Peralta, M Ogliaro, J J Bearpark, E Heyd, K N Brothers, V N Kudin, T Staroverov, R Keith, J Kobayashi, K Normand, A Raghavachari, J C Rendell, S S Burant, J Iyengar, M Tomasi, J M Cossi, M Millam, C Klene, R Adamo, J W Cammi, R L Ochterski, K Martin, O Morokuma, J B Farkas, D J Foresman, Fox, Gaussian, IncWallingford, CTGaussian 09, Revision A.02Hratchian, J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, T. Vreven, K. Throssell, J. A. Montgomery Jr., J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, J. B. Foresman, D. J. Fox, Gaussian 09, Revision A.02, Gaussian, Inc., Wallingford, CT 2016. https://gaussian.com/ g09citation/ (accessed: September 08 2020).

. G M Morris, R Huey, W Lindstrom, M F Sanner, R K Belew, D S Goodsell, A J Olson, J. Comput. Chem. G. M. Morris, R. Huey, W. Lindstrom, M. F. Sanner, R. K. Belew, D. S. Goodsell, A. J. Olson, J. Comput. Chem. 2009, 30, 2785.

. Dassaultsystèmes Biovia, San Diego, DassaultSystèmes. BIOVIA, DassaultSystèmes, San Diego. DassaultSystèmes, 2015, https://www.3ds.com/products-services/biovia/resource-center/ citations-and-references/ (accessed: September 08 2020).

. A K Kabi, M Pal, R Gujjarappa, C C Malakar, M Roy, 10.1002/jhet.4541J. Heterocycl. Chem. 2022How to cite this article: A. K. Kabi, M. Pal, R. Gujjarappa, C. C. Malakar, M. Roy, J. Heterocycl. Chem. 2022, 1. https://doi.org/10.1002/jhet.4541