# Chromatin-RNA in situ Reverse Transcription Sequencing (CRIST-seq) Approach to Profile the Non-coding RNA Interaction Network Graphical overview Background

CorpusID: 260013544
 
tags: #Medicine, #Biology

URL: [https://www.semanticscholar.org/paper/ba183159e6192051dcebba98e5319aa395d34045](https://www.semanticscholar.org/paper/ba183159e6192051dcebba98e5319aa395d34045)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Chromatin-RNA in situ Reverse Transcription Sequencing (CRIST-seq) Approach to Profile the Non-coding RNA Interaction Network Graphical overview Background
2022

Shilin Zhang 
Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education
First Hospital of Jilin University
Cancer Center130061Changchun, JilinChina

Stanford University Medical School
Palo Alto Health Care System
94304Palo AltoVA, CAUSA

Xue Wen 
Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education
First Hospital of Jilin University
Cancer Center130061Changchun, JilinChina

Lei Zhou 
Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education
First Hospital of Jilin University
Cancer Center130061Changchun, JilinChina

Hui Li 
Wei Li 
Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education
First Hospital of Jilin University
Cancer Center130061Changchun, JilinChina

Andrew R Hoffman arhoffman@stanford.edu 
Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education
First Hospital of Jilin University
Cancer Center130061Changchun, JilinChina

Stanford University Medical School
Palo Alto Health Care System
94304Palo AltoVA, CAUSA

Ji-Fan Hu 
Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education
First Hospital of Jilin University
Cancer Center130061Changchun, JilinChina

Stanford University Medical School
Palo Alto Health Care System
94304Palo AltoVA, CAUSA

Jiuwei Cui 
Key Laboratory of Organ Regeneration and Transplantation of Ministry of Education
First Hospital of Jilin University
Cancer Center130061Changchun, JilinChina

Chromatin-RNA in situ Reverse Transcription Sequencing (CRIST-seq) Approach to Profile the Non-coding RNA Interaction Network Graphical overview Background

J. Cell. Biol
2022Cite as: Zhang, S. et al. (2023). Chromatin-RNA in situ Reverse Transcription Sequencing (CRIST-seq) Approach to Profile the Non-coding RNA Interaction Network. Bio-protocol 13(14): e4718. 1 *For correspondence: This protocol is used in:CRIST-seqNon coding RNARNA-seqRegulatory elementRNA-DNA interactionEpigenetics
Key features• This method provides a new technique for screening ncRNAs and establishing chromatin interaction networks. • The target gene for this method can be any gene of interest and any site in the entire genome.• This method can be further extended to detect RNAs, DNAs, and proteins that interact with target genes.Non-coding RNAs (ncRNAs) are defined as RNAs that do not encode proteins, but many ncRNAs do have the ability to regulate gene expression. These ncRNAs play a critical role in the epigenetic regulation of various physiological and pathological processes through diverse biochemical mechanisms. However, the existing screening methods to identify regulatory ncRNAs only focus on whole-cell expression levels and do not capture every ncRNA that targets certain genes. We describe a new method, chromatin-RNA in situ reverse transcription sequencing (CRIST-seq), that can identify all the ncRNAs that are associated with the regulation of any given gene. In this article, we targeted the ncRNAs that are associated with pluripotent gene Sox2, allowing us to catalog the ncRNA regulation network of pluripotency maintenance. This methodology is universally applicable for the study of epigenetic regulation of any genes by making simple changes on the CRISPR-dCas9 gRNAs. Cite as: Zhang, S. et al. (2023). Chromatin-RNA in situ Reverse Transcription Sequencing (CRIST-seq) Approach to Profile the Non-coding RNA Interaction Network. Bio-protocol 13(14): e4718.


B. Lenti-dCas9 construction (Figure 1A) 1. Design gRNAs using Broad Institute website (https://portals.broadinstitute.org/gppx/crispick/public). 2. Synthesize pU6-gRNA1-pH1-gRNA2 cassette using JH3915-JH3986, JH3987-JH3988, and JH3989-J441 primers (1 cycle at 95 °C for 5 min, 32 cycles at 95 °C for 20 s, 62 °C for 30 s, and 72 °C for 15 s, and 1 cycle at 72 °C for 10 min). 3. Use Age I and Not I restriction enzymes (37 ℃ for 30 min) and T4 DNA ligase (22 ℃ for 10 min) to insert Sox2 gRNAs or gCTs downstream from U6 promotor in the lenti-dCas9 vector. 4. Use DH5α competent cells for transformation. 5. Confirm the correct cloning by sequencing and extract the plasmid with GeneAll Hybrid-Q Plasmid Rapidprep. Cite as: Zhang, S. et al. (2023). Chromatin-RNA in situ Reverse Transcription Sequencing (CRIST-seq) Approach to Profile the Non-coding RNA Interaction Network. Bio-protocol 13(14): e4718. DOI: 10.21769/BioProtoc.4718.

## Figure 1 .Figure 2 .
12Plasmid maps and sample preliminary detection. (A) Map of lenti-dCas9 plasmid. Sox2-gRNA1 and Sox2-gRNA2 were inserted into the vector using Age I and Not I. (B) Preliminary detection of gRNA specific binding sites in chromatin-RNA in situ reverse transcription (CRIST) samples by PCR (using pSox2-F and pSox2-R primers). The sequence of gRNA specific binding sites cannot be detected in anti-IgG sample but can be detected in anti-Flag sample and input control, which demonstrates the specificity of the method. The anti-IgG here served as background control. (C) Sample specific testing was performed by real-time PCR using specific primers derived from the pSox2 targeting site, 5′-Ct control site, and off-target site. pSox2: targeting site in the Sox2 promoter where the Cas9 gRNAs are designed; 5′-Ct: fragment that is 14.6 kb away from the pSox2 target site and is used as the control site. Cas9 vector: cells that were treated with the Cas9 control vector that lacks the gRNAs; Cas9-gRNA: cells that were targeted by both Cas9 and Sox2 gRNAs; Cas9-gCT: cells that were treated with the random control gRNA vector. Off-target: CRIST control site that is 33.8 kb upstream of the housekeeping gene GAPDH. All data shown are mean ± SEM from three independent experiments by normalization over the IgG control. ( * * ) P < 0.01 as compared with the Cas9 Vector and Cas9-gCT controls.C. Lentiviral packaging and transfection Sequencing analysis. (A) Analysis of ncRNA sequencing results. Preliminary screening for non-coding RNAs (ncRNAs) that are highly expressed in iPSC and much less expressed in FIB, which are related to pluripotency maintenance. Data regarding the RNAs that were changed greater than two-fold were combined with the CRIST-seq data using a VENN program. A cut-off threshold of peak enrichment FPKM > 50 was arbitrarily set to select CRIST-seq RNAs for VENN analysis.(B) Cross matching sequencing results with RNA-seq. (C) CRIST-seq identifies the top 42 Sox2 promoter-interacting RNAs. The Sox2 interacting RNAs are listed in order of the enrichment fold of the top 42 CRIST-seq data.

## Table 1 .
1List of primers or productsNote: Feeder cells can enhance the growth and reproduction of iPSCs, maintain pluripotency, and inhibit differentiation. In this assay, the feeding layer is made with FIB cells. After being treated with mitomycin, FIBs lose their mitotic ability and will die during digestion and inoculation, which has no impact on subsequent experiments. a. Coat a 100 mm plate with 2 mL of 0.1% gelatin and place in an incubator at 37 °C for 2 h. b. Wash the plate twice with PBS and discard PBS before use. c. Seed FIB cells (served as feeder layer) on the gelatin-coated plate and grow to ~70% confluency. d. Add mitomycin C (final concentration of 10 μg/mL) and incubate for 2 h to block mitosis. e. Wash the plate twice with PBS. f. Grow iPSCs (1 × 10 6 ) on feeder layer and culture in knockout growth media (see Recipe 1) at 37 °C and 5% CO2.Primer/Product 
5′→3′ sequence 
pSox2-gRNA1 
GGGGTTGAGGACACGTGCTG 
pSox2-gRNA2 
GAGCCAATATTCCGTAGCAT 
gCT1 
GTTCCCTGCAAGAGTGCCCA 
gCT2 
GCACTACCAGAGCTAACTCA 
JH3915 
TAGTAATGAGTTTAAACAAGGTCGGGCAGGAAGAGGGCCT 
JH3986 
CCAGCACGTGTCCTCAACCCCGGTGTTTCGTCCTTTCCACAAG 
JH3987 
GGGGTTGAGGACACGTGCTGGTTTTAGAGCTAGAAATAGCAAGTT 
JH3988 
ACATGCTACGGAATATTGGCTCGGATCCAAGGTGTCTCATACAG 
JH3989 
GAGCCAATATTCCGTAGCATGTTTTAGAGCTAGAAATAGCAAGTT 
J441 
CAACTTCTCGGGGACTGTGGGCGAT 
pSox2-F 
GAGCCAATATTCCGTAGCATG 
pSox2-R 
CGCTGGGGAACCTTTGTATC 
5′CT-F 
GAGCCAATATTCCGTAGCATG 
5′CT-R 
CGCTGGGGAACCTTTGTATC 
Off-target-F 
AGCCATCCTGTCCTCCGCCTG 
Off-target-R 
CTGCACGGAAGGTCACGATG 

Solutions 

1. Hypotonic buffer (see Recipes) 
2. Sonication buffer (see Recipes) 
3. ChIP dilution buffer (see Recipes) 
4. Protein A/G beads binding & washing buffer (see Recipes) 
5. Protein A/G beads elution buffer (see Recipes) 
6. 2× M-280 beads binding & washing buffer (see Recipes) 
7. M-280 beads elution buffer (see Recipes) 

Recipes 

1. Knockout growth media 
KnockOut DMEM supplemented with 15% KnockOut SR 
1% MEM-NEAA 
2 mM glutamine 
1% penicillin-streptomycin 
200 μM β-mercaptoethanol 
ESGRO Mouse LIF 

2. Normal growth media 
DMEM supplemented with 10% FBS and 1% penicillin-streptomycin 

3. Hypotonic buffer 
10 mM HEPES 
1.5 mM MgCl2 
10 mM KCl 
0.4% NP-40 (add before use) 

4. Sonication buffer 
50 mM Tris-HCl (pH 7.5) 
5 mM EDTA 


Cite as: Zhang, S. et al. (2023). Chromatin-RNA in situ Reverse Transcription Sequencing (CRIST-seq) Approach to Profile the Non-coding RNA Interaction Network. Bio-protocol 13(14): e4718. DOI: 10.21769/BioProtoc.4718.
AcknowledgmentsI. Preliminary detection of CRIST samplesNote: In order to identify ncRNAs binding to the Sox2 gene promoter, we selected two sites from the Sox2 gene promoter to design gRNAs. Therefore, the ds-cDNAs obtained by the above method must contain the binding site of gRNAs on the Sox2 gene promoter, so that the ncRNA combined with it can be precipitated together, and the sample quality can be preliminarily determined as qualified and used for subsequent experiments.Resuspend purified samples in 30 μL of ultrapure water and use pSox2 primers(Table 1)Figure 1B). Cas9 enrichment signals were quantitated by real-time PCR using specific primers derived from the pSox2 targeting site, 5′-Ct control site, and off-target site (Figure 1C). After confirming the specificity of the Cas9 gRNA, the Cas9 Sox2-gRNA iPSCs were then used for CRIST-seq assay.J. Synthesis of double-stranded cDNAFor long-term stability, the single-strand cDNAs should be converted into double-stranded cDNAs. 1. Synthesize ds-cDNAs at 16 °C for 2.5 h with 20 μL of purified samples, 2 μL of synthesis enzyme mix, 4 μL of 10× reaction buffer, and 14 μL of RNase-free water. Use NEBNext mRNA Second Strand Synthesis Module. 2. Purify DNAs again use phenol/chloroform/isoamyl alcohol (25:24:1) as previously described.K. Construction of CRIST-seq libraryCompeting interestsThe authors declare no conflict of interest.
6-well cell culture plates (Corning, catalog number: 3516) 2. 100 mm cell culture dishes (BD Falcon. 3530036-well cell culture plates (Corning, catalog number: 3516) 2. 100 mm cell culture dishes (BD Falcon, catalog number: 353003)

mm Petri dishes (BD Falcon. 351005mm Petri dishes (BD Falcon, catalog number: 351005)

mL microcentrifuge tubes. Pierce697151.5 mL microcentrifuge tubes (Pierce, catalog number: 69715)

2 mL 8-tube stripes. 803Bio-Rad0.2 mL 8-tube stripes (Bio-Rad, catalog number: TBC0803)

. Chloroform, Fisher ChemicalChloroform (Fisher Chemical, catalog number: C298-500)

. Isopropanol, Thermo Scientific610080040Isopropanol (Thermo Scientific, catalog number: AC610080040)

. Ethanol, Thermo Scientific10617864Ethanol (Thermo Scientific, catalog number: 10617864)

. Phenol/Chloroform/Isoamyl Alcohol. 251Fisher BioreagentsPhenol/Chloroform/Isoamyl Alcohol (25:24:1) (Fisher Bioreagents, catalog number: BP1752I-100)

Glycogen, mg/mL) (Invitrogen, catalog number: AM9501). Glycogen (5 mg/mL) (Invitrogen, catalog number: AM9501)

RNase inhibitor (Invitrogen, catalog number: AM2694). RNase inhibitor (Invitrogen, catalog number: AM2694)

Random hexamers (Invitrogen, catalog number: N8080127) 15. dNTP mix (2.5 mM) (Invitrogen, catalog number: R72501). Random hexamers (Invitrogen, catalog number: N8080127) 15. dNTP mix (2.5 mM) (Invitrogen, catalog number: R72501)

. -Q Plasmid Hybrid, Rapidprep, GeneAllHybrid-Q Plasmid Rapidprep (GeneAll, catalog number: 100-102)

FastDigest Restriction endonucleases Age I (BshT I), Not I, and DpnI (Thermo Scientific, catalog numbers: FD1464, FD0596, and FD1703). FastDigest Restriction endonucleases Age I (BshT I), Not I, and DpnI (Thermo Scientific, catalog numbers: FD1464, FD0596, and FD1703)

10× FastDigest buffer. Thermo Scientific6410× FastDigest buffer (Thermo Scientific, catalog number: B64)

T4 Dna Ligase, U/μL) (Thermo Scientific, catalog number: EL0011). T4 DNA ligase (5 U/μL) (Thermo Scientific, catalog number: EL0011)

PEG-it virus precipitation solution (System Biosciences. PEG-it virus precipitation solution (System Biosciences, catalog number: LV825A-1)

. Pei (polysciences, PEI (Polysciences, catalog number: 24765-1)

. ( Polybrene, Sigma, 1003Polybrene (Sigma, catalog number: TR-1003)

Formaldehyde, 37% by weight. Fisher ChemicalFormaldehyde, 37% by weight (Fisher Chemical, catalog number: F79-500)

. Np-40, Thermo Scientific85124NP-40 (Thermo Scientific, catalog number: 85124)

. Hepes (sigma, 3375HEPES (Sigma, catalog number: H3375)

. M Edta (thermo, Scientific, 1021M EDTA (Thermo Scientific, catalog number: R1021)

10% SDS (Sigma, catalog number: L3771). 10% SDS (Sigma, catalog number: L3771)

Halt protease inhibitor cocktail (100×) (Thermo Scientific. 87785Halt protease inhibitor cocktail (100×) (Thermo Scientific, catalog number: 87785)

. Inhibitor Pmsf Protease, Thermo Scientific36978PMSF protease inhibitor (Thermo Scientific, catalog number: 36978)

Thermo Scientific, catalog number: J20605AP). 20Tween 20 (Thermo Scientific, catalog number: J20605AP)

. K ( Proteinase, Invitrogen, 25530049Proteinase K (Invitrogen, catalog number: 25530049)

Dynabeads M-280 streptavidin (Invitrogen. 60210Dynabeads M-280 streptavidin (Invitrogen, catalog number: 60210)

. ( Formamide, Sigma, 9037Formamide (Sigma, catalog number: F9037)

Anti-Flag antibody (Sigma, catalog number: F1804). Anti-Flag antibody (Sigma, catalog number: F1804)

Anti-IgG antibody (Abcam, catalog number: ab171870). Anti-IgG antibody (Abcam, catalog number: ab171870)

PureProteome Protein A/G mix magnetic beads (Millipore, catalog number: LSKMAGAG10). PureProteome Protein A/G mix magnetic beads (Millipore, catalog number: LSKMAGAG10)

RNase A (20 mg/mL) (Invitrogen, catalog number: 12091021). RNase A (20 mg/mL) (Invitrogen, catalog number: 12091021)

HEK293T cells (ATCC, catalog number: CRL-3219). stored in liquid nitrogenHEK293T cells (ATCC, catalog number: CRL-3219), stored in liquid nitrogen

Mouse FIB cells (muscle-derived fibroblasts), cultured from a 129 mouse fetus (Zhai et al., 2015) and stored in liquid nitrogen. Mouse FIB cells (muscle-derived fibroblasts), cultured from a 129 mouse fetus (Zhai et al., 2015) and stored in liquid nitrogen

Mouse induced pluripotent stem cells (iPSCs) were reprogrammed by Pou5f1-Sox2-Klf4-Myc (OSKM) cocktail factors in our lab using FIB cells. M Chen, Mouse induced pluripotent stem cells (iPSCs) were reprogrammed by Pou5f1-Sox2-Klf4-Myc (OSKM) cocktail factors in our lab using FIB cells (M. Chen et al., 2012;

. H Zhang, H. Zhang et al., 2013; Zhai et al., 2015;

. X Chen, X. Chen et al., 2016), and stored in liquid nitrogen

. Dmem ( Knockout, Gibco, 10829018KnockOut DMEM (Gibco, catalog number: 10829018)

. Sr ( Knockout, Gibco, 10828010KnockOut SR (Gibco, catalog number: 10828010)

. Dmem (gibco, DMEM (Gibco, catalog number: 11995-065)

. Fbs (gibco, 31605FBS (Gibco, catalog number: A31605)

. Mem-Neaa ( Gibco, 11140050MEM-NEAA (Gibco, catalog number: 11140050)

. ( Glutamine, Sigma, 7513Glutamine (Sigma, catalog number: G7513)

. Penicillin-Streptomycin, 4458SigmaPenicillin-streptomycin (Sigma, catalog number: P4458)

. Lif ( Esgro Mouse, Millipore, 1107ESGRO Mouse LIF (Millipore, catalog number: ESG1107)

Puromycin (Invivogen, catalog number: ant-pr-1). Puromycin (Invivogen, catalog number: ant-pr-1)

Mitomycin C solution (1 mg/mL) (Nacalai Tesque. Mitomycin C solution (1 mg/mL) (Nacalai Tesque, catalog number: 20898-21)

2% gelatin solution (Sigma, catalog number: G1393). 2% gelatin solution (Sigma, catalog number: G1393)

. Pbs (gibco, 10010023PBS (Gibco, catalog number: 10010023)

. -Mem ( Opti, Gibco, 51985034Opti-MEM (Gibco, catalog number: 51985034)

Centrifuge (Eppendorf, model: 5415D). Centrifuge (Eppendorf, model: 5415D)

Vibra-Cell Ultrasonic liquid processors. Sonics, model130Vibra-Cell Ultrasonic liquid processors (Sonics, model: VCX-130)

. ( Thermomixer, Eppendorf, Thermomixer (Eppendorf, catalog number: vwrCA21516-176)

. DNA spectrophotometer. 1000DNA spectrophotometer (NanoDrop, model: ND-1000)

Laboratory inverted microscope (Carl Zeiss, model: Axiovert 40 CFL). Laboratory inverted microscope (Carl Zeiss, model: Axiovert 40 CFL)

Ph Meter (mettler, Toledo, SevenEasy S20). pH meter (Mettler Toledo, model: SevenEasy S20)

PCR instrument (Eppendorf, model: Mastercycler pro PCR System). PCR instrument (Eppendorf, model: Mastercycler pro PCR System)

. model: EPS 300Electrophoresis power supply (Pharmacia Biotech. Electrophoresis power supply (Pharmacia Biotech, model: EPS 300)

Labquake shaker (Lab Industries. Labquake shaker (Lab Industries, catalog number: 400-110)

Gel documentation system. 1000Axygen, modelGel documentation system (Axygen, model: GD-1000)

. Tophat, Center for Computational Biology at Johns Hopkins UniversityTopHat (Center for Computational Biology at Johns Hopkins University, http://tophat.cbcb.umd.edu)

. Cufflinks, Cole Trapnell's lab at the University of WashingtonCufflinks (Cole Trapnell's lab at the University of Washington, http://cole-trapnell-lab.github.io/cufflinks/)

. Ucsc Genome Browser, UCSC Genomics InstituteUCSC Genome Browser (UCSC Genomics Institute, https://genome.ucsc.edu)

Grow 293T cells to ~70% confluency for lentiviral packaging. 2. Change the normal growth media to Opti-MEM. 1RIPSeeker software (University of Toronto1h before packagingRIPSeeker software (University of Toronto, 1. Grow 293T cells to ~70% confluency for lentiviral packaging. 2. Change the normal growth media to Opti-MEM 1h before packaging.

Put 5 μg of PEI and 100 μL of Opti-MEM into a tube. Add 0.8 μg of psPAX2, 0.4 μg of PMD2.G, and 0.8 μg of lenti-dCas9 (carrying pSox2-gRNAs or gCTsPut 5 μg of PEI and 100 μL of Opti-MEM into a tube. Add 0.8 μg of psPAX2, 0.4 μg of PMD2.G, and 0.8 μg of lenti-dCas9 (carrying pSox2-gRNAs or gCTs).

Mix and let stand for 20 min; then, add them to the culture media of 293T cells. Mix and let stand for 20 min; then, add them to the culture media of 293T cells.

Collect virus supernatants at 24, 48, and 72 h, add PEG-it to the collected medium, incubate overnight at 4 °C, and spin at 1,500× g for 30 min. Collect virus supernatants at 24, 48, and 72 h, add PEG-it to the collected medium, incubate overnight at 4 °C, and spin at 1,500× g for 30 min.

Discard the supernatant and resuspend viral particles in 100 μL of PBS. Discard the supernatant and resuspend viral particles in 100 μL of PBS.

Grow iPSCs and fibroblasts (FIB) in 100 mm plates to ~50% confluency. Grow iPSCs and fibroblasts (FIB) in 100 mm plates to ~50% confluency.

Mix 20 μL of lentivirus solution with polybrene (final concentration of 5-10 ng/μL) and add it to culture medium. Mix 20 μL of lentivirus solution with polybrene (final concentration of 5-10 ng/μL) and add it to culture medium.

Change medium after 24 h and use puromycin (0.5-2 μg/mL) to select puromycin-resistant cells. Change medium after 24 h and use puromycin (0.5-2 μg/mL) to select puromycin-resistant cells.

RNA reverse transcription Note: In order to protect unstable RNAs, we performed intranuclear in situ reverse transcription prior to the subsequent steps and labeled RNAs with biotin-dCTP during reverse transcription. D Intranuclear, D. Intranuclear RNA reverse transcription Note: In order to protect unstable RNAs, we performed intranuclear in situ reverse transcription prior to the subsequent steps and labeled RNAs with biotin-dCTP during reverse transcription.

resuspend with 1 mL of PBS, and add 54 μL of 37% formaldehyde (final concentration of 2%) for 10 min at room temperature. Quench the formaldehyde with 60 μL of 2 M glycine. Collect iPSCs and fibroblasts (~1 × 10 7 ). incubate for 5 min at room temperatureCollect iPSCs and fibroblasts (~1 × 10 7 ), resuspend with 1 mL of PBS, and add 54 μL of 37% formaldehyde (final concentration of 2%) for 10 min at room temperature. Quench the formaldehyde with 60 μL of 2 M glycine (with a final concentration of 0.125 M) and incubate for 5 min at room temperature.

Wash both iPSCs and fibroblasts with PBS twice, resuspend with 800 μL of hypotonic buffer (see Recipe 3), and incubate for 5 min on ice to lyse the cell membranes. Wash both iPSCs and fibroblasts with PBS twice, resuspend with 800 μL of hypotonic buffer (see Recipe 3), and incubate for 5 min on ice to lyse the cell membranes.

Centrifuge samples for 10 min at 1,500× g and wash twice with hypotonic buffer (without NP-40). Centrifuge samples for 10 min at 1,500× g and wash twice with hypotonic buffer (without NP-40).

Conduct intranuclear RNA reverse transcription at 65 °C for 30 min in a 20 μL reaction with biotin-dCTP (1 μL of 50 μM random hexamer, 1 μL of 10 mM dNTP, 1 μL of 0.4 mM biotin-dCTP, 1 μL of Maxima reverse transcriptase, 0.5 μL of RNase inhibitors, 1 μL of 0.1 M DTT, 4 μL of 5× cDNA synthesis buffer, and RNase-free water to 20 μL). Use 4 μL of 0.5 M EDTA to stop the reactionConduct intranuclear RNA reverse transcription at 65 °C for 30 min in a 20 μL reaction with biotin-dCTP (1 μL of 50 μM random hexamer, 1 μL of 10 mM dNTP, 1 μL of 0.4 mM biotin-dCTP, 1 μL of Maxima reverse transcriptase, 0.5 μL of RNase inhibitors, 1 μL of 0.1 M DTT, 4 μL of 5× cDNA synthesis buffer, and RNase-free water to 20 μL). Use 4 μL of 0.5 M EDTA to stop the reaction.

Centrifuge samples at 1,600× g for 10 min at 4 °C. Wash twice with ice-cold PBS. Centrifuge samples at 1,600× g for 10 min at 4 °C. Wash twice with ice-cold PBS.

Note: Using too much liquid may lead to splashing, and using too little liquid may produce foam. 2. Clamp the sample tubes and the ultrasonic probe together and place them on ice. E , Ultrasonic disintegration 1. Resuspend cell pellets with 300 μL of sonication buffer (see Recipe 4)E. Ultrasonic disintegration 1. Resuspend cell pellets with 300 μL of sonication buffer (see Recipe 4). Note: Using too much liquid may lead to splashing, and using too little liquid may produce foam. 2. Clamp the sample tubes and the ultrasonic probe together and place them on ice.

Put the 2 mm ultrasonic probe 1 cm below the liquid level. Note: In order to avoid foaming, the probe should always be kept below the liquid level. Put the 2 mm ultrasonic probe 1 cm below the liquid level. Note: In order to avoid foaming, the probe should always be kept below the liquid level

Shear DNAs into segments of 200-1,000 base pairs for 15 min (10 s on and 20 s off) at an amplitude of 40%. Shear DNAs into segments of 200-1,000 base pairs for 15 min (10 s on and 20 s off) at an amplitude of 40%.

Use the two equal portions for immunoprecipitation (using anti-Flag and anti-IgG antibody respectively) and the 2 μL portion for input control. Divide 300 μL of sonicated DNAs into three portions (149, 149, and 2 μL)Divide 300 μL of sonicated DNAs into three portions (149, 149, and 2 μL). Use the two equal portions for immunoprecipitation (using anti-Flag and anti-IgG antibody respectively) and the 2 μL portion for input control.

Immunoprecipitation of biotin-cDNA and dCas9 1. To reduce non-specific background, use Protein A/G mix magnetic beads (5 μL/ChIP) to pre-clear the sonicated samples. F , Put 5 μL of protein A/G beads in a new tube and wash twice with beads binding & washing buffer. see Recipe 6F. Immunoprecipitation of biotin-cDNA and dCas9 1. To reduce non-specific background, use Protein A/G mix magnetic beads (5 μL/ChIP) to pre-clear the sonicated samples. a. Put 5 μL of protein A/G beads in a new tube and wash twice with beads binding & washing buffer (see Recipe 6).

Dilute sonicated samples (10-fold) with ChIP dilution buffer (see Recipe 5), mix with 5 μL of washed protein A/G beads, and incubate at 4 °C with rotation for 30 min. Dilute sonicated samples (10-fold) with ChIP dilution buffer (see Recipe 5), mix with 5 μL of washed protein A/G beads, and incubate at 4 °C with rotation for 30 min.

Discard protein A/G magnetic bead pellet in a magnetic rack and transfer sonicated samples to new tubes. c. Discard protein A/G magnetic bead pellet in a magnetic rack and transfer sonicated samples to new tubes.

Add anti-Flag and anti-IgG antibodies (2-5 μg/ChIP) to experimental and control groups, respectively, and incubate overnight at 4 °C with rotation. Add anti-Flag and anti-IgG antibodies (2-5 μg/ChIP) to experimental and control groups, respectively, and incubate overnight at 4 °C with rotation.

To enrich the immunoprecipitation samples, add 10 μL of pre-cleared protein A/G magnetic beads to samples and incubate at room temperature with rotation for 30 min. Wash the beads three times with Protein A/G Beads binding & washing buffer. To enrich the immunoprecipitation samples, add 10 μL of pre-cleared protein A/G magnetic beads to samples and incubate at room temperature with rotation for 30 min. Wash the beads three times with Protein A/G Beads binding & washing buffer.

Discard the supernatant and elute the immunoprecipitation samples twice with 100 μL of elution buffer. see Recipe 7Discard the supernatant and elute the immunoprecipitation samples twice with 100 μL of elution buffer (see Recipe 7).

Streptavidin-biotin capture. G , G. Streptavidin-biotin capture

Put M-280 streptavidin beads in a new tube and wash twice with beads binding & washing buffer. see Recipe 8Put M-280 streptavidin beads in a new tube and wash twice with beads binding & washing buffer (see Recipe 8).

Transfer 200 μL of eluent samples to the washed beads, add 50 μL of 5 M NaCl (final concentration of 1 M), and incubate for 30 min with rotation at room temperature. Transfer 200 μL of eluent samples to the washed beads, add 50 μL of 5 M NaCl (final concentration of 1 M), and incubate for 30 min with rotation at room temperature.

Discard the supernatant, elute the samples with 100 μL of elution buffer (see Recipe 9) twice, and collect 200 μL of eluate. Discard the supernatant, elute the samples with 100 μL of elution buffer (see Recipe 9) twice, and collect 200 μL of eluate.

To facilitate sequencing, use DpnI to cut cDNAs into fragments of approximately 300 bps: add 1 μL of Fastdigest DpnI and 3 μL of 10× FastDigest Buffer to 26 μL of purified cDNAs and incubate at 37 °C for 20 min. To facilitate sequencing, use DpnI to cut cDNAs into fragments of approximately 300 bps: add 1 μL of Fastdigest DpnI and 3 μL of 10× FastDigest Buffer to 26 μL of purified cDNAs and incubate at 37 °C for 20 min.

24:1) and resuspend in 22 μL of ultrapure water. Follow the protocol of NEBNext ChIP-Seq Library Prep Master Mix Set for Illumina to prepare samples for next-generation sequencing. /isoamyl alcohol. 25Note: Follow the protocol step by step from 1.1-1.8A.Purify DNAs with phenol/chloroform/isoamyl alcohol (25:24:1) and resuspend in 22 μL of ultrapure water. Follow the protocol of NEBNext ChIP-Seq Library Prep Master Mix Set for Illumina to prepare samples for next-generation sequencing. Note: Follow the protocol step by step from 1.1-1.8A.

Use Index primers in NEBNext Multiplex Oligos for Illumina Index Primers Set 1 & Set 2 to amplify samples and get the cDNA libraries (1 cycle at 98 °C for 30 min, 15 cycles at 98 °C for 10 s and 65 °C for 75 s, and 1 cycle at 65 °C for 5 min. then hold at 4 °CUse Index primers in NEBNext Multiplex Oligos for Illumina Index Primers Set 1 & Set 2 to amplify samples and get the cDNA libraries (1 cycle at 98 °C for 30 min, 15 cycles at 98 °C for 10 s and 65 °C for 75 s, and 1 cycle at 65 °C for 5 min, then hold at 4 °C).

Sequence the cDNA library and use index primers as sequencing tags. Sequence the cDNA library and use index primers as sequencing tags.

Filter raw data and the low-quality data using FASTX software. Du, v0.0.13Filter raw data and the low-quality data using FASTX software (v0.0.13) (Du et al., 2018).

Use TopHat software (version 2.0.9) to map clean reads to the mouse mm10 genome. Trapnell, Use TopHat software (version 2.0.9) to map clean reads to the mouse mm10 genome (Trapnell et al., 2009).

Quantitate the mapped reads as "fragments per kilobase of transcript per million fragments mapped. Trapnell, FPKM) using Cufflinks (version 2.1.1Quantitate the mapped reads as "fragments per kilobase of transcript per million fragments mapped" (FPKM) using Cufflinks (version 2.1.1) (Trapnell et al., 2010).

2013) and adjust over the peaks overlapping with the IgG control enriched regions. Li, Call and annotate the peak using RIPSeeker softwareCall and annotate the peak using RIPSeeker software (Li et al., 2013) and adjust over the peaks overlapping with the IgG control enriched regions.

Normalize the CRIST-seq signal intensities over the nontargeting Cas9 gCT control using the DiffBind package. Ross-Innes, Stark, fold change difference ≥ 2 and p-value < 0.05, with false discovery rate (FDR) < 0.1Normalize the CRIST-seq signal intensities over the nontargeting Cas9 gCT control using the DiffBind package (Ross-Innes, Stark et al. 2012) [fold change difference ≥ 2 and p-value < 0.05, with false discovery rate (FDR) < 0.1].

Fund of Jilin Provincial Science and Technology Department. Fund of Changchun City Science and Technology Bureau (21ZGY28), China Guanghua Fund and the Youth Fund of First Hospital of Jilin University (2020-CXM-01, JDYYGH2019004, JDYY 102019002, JDYY 102019043). California Institute of Regenerative Medicine (CIRM) grantNatural Science Fund of Jilin Provincial Finance Department (JLSWSRCZX2020-023, JLSWSRCZX2020-100), the Youth Fund of Jilin Provincial Health Commission. RT2-01942), and the Department of Veterans Affairs (BX002905)2020YFA0707704), the Innovative Program of National Natural Science Foundation of China (82050003), National Natural Science Foundation of China (31871297, 82273191, 81874052, 32000431, 81900327, 82001670, 82101675, 82900701), Fund of Jilin Provincial Science and Technology Department (20210101311JC, 20190303146SF, 20200602032ZP, 20200201390JC), Provincial Science Fund of Jilin Province Development and Reform Commission (2021C10 and 2020C038-4), Natural Science Fund of Jilin Provincial Finance Department (JLSWSRCZX2020-023, JLSWSRCZX2020-100), the Youth Fund of Jilin Provincial Health Commission (2016Q035), Fund of Changchun City Science and Technology Bureau (21ZGY28), China Guanghua Fund and the Youth Fund of First Hospital of Jilin University (2020-CXM-01, JDYYGH2019004, JDYY 102019002, JDYY 102019043), California Institute of Regenerative Medicine (CIRM) grant (RT2-01942), and the Department of Veterans Affairs (BX002905).

This protocol is derived from the original research paper. Y Wang, 10.1083/jcb.202009134This protocol is derived from the original research paper (Y. Wang et al., 2022; DOI: 10.1083/jcb.202009134).

Promotion of the induction of cell pluripotency through metabolic remodeling by thyroid hormone triiodothyronineactivated PI3K/AKT signal pathway. M Chen, H Zhang, J Wu, L Xu, D Xu, J Sun, Y He, X Zhou, Z Wang, L Wu, Biomaterials. 3322Chen, M., Zhang, H., Wu, J., Xu, L., Xu, D., Sun, J., He, Y., Zhou, X., Wang, Z., Wu, L., et al. (2012). Promotion of the induction of cell pluripotency through metabolic remodeling by thyroid hormone triiodothyronine- activated PI3K/AKT signal pathway. Biomaterials 33(22): 5514-5523.

Valproic Acid Enhances iPSC Induction From Human Bone Marrow-Derived Cells Through the Suppression of Reprogramming-Induced Senescence. X Chen, Y Zhai, D Yu, J Cui, J F Hu, W Li, J Cell Physiol. 2318Chen, X., Zhai, Y., Yu, D., Cui, J., Hu, J. F. and Li, W. (2016). Valproic Acid Enhances iPSC Induction From Human Bone Marrow-Derived Cells Through the Suppression of Reprogramming-Induced Senescence. J Cell Physiol 231(8): 1719-1727.

Combined RNA-seq and RAT-seq mapping of long noncoding RNAs in pluripotent reprogramming. Z Du, L Jia, Y Wang, C Wang, X Wen, J Chen, Y Zhu, D Yu, L Zhou, N Chen, Sci Data. 5180255Du, Z., Jia, L., Wang, Y., Wang, C., Wen, X., Chen, J., Zhu, Y., Yu, D., Zhou, L., Chen, N., et al. (2018). Combined RNA-seq and RAT-seq mapping of long noncoding RNAs in pluripotent reprogramming. Sci Data 5: 180255.

Chromatin lncRNA Platr10 controls stem cell pluripotency by coordinating an intrachromosomal regulatory network. Z Du, X Wen, Y Wang, L Jia, S Zhang, Y Liu, L Zhou, H Li, W Yang, C Wang, Du, Z., Wen, X., Wang, Y., Jia, L., Zhang, S., Liu, Y., Zhou, L., Li, H., Yang, W., Wang, C., et al. (2021). Chromatin lncRNA Platr10 controls stem cell pluripotency by coordinating an intrachromosomal regulatory network.

. Genome Biol. 221233Genome Biol 22(1): 233.

Linc-RoR promotes c-Myc expression through hnRNP I and AUF1. J Huang, A Zhang, T T Ho, Z Zhang, N Zhou, X Ding, X Zhang, M Xu, Y Y Mo, Nucleic Acids Res. 447Huang, J., Zhang, A., Ho, T. T., Zhang, Z., Zhou, N., Ding, X., Zhang, X., Xu, M. and Mo, Y. Y. (2016). Linc-RoR promotes c-Myc expression through hnRNP I and AUF1. Nucleic Acids Res 44(7): 3059-3069.

Oplr16 serves as a novel chromatin factor to control stem cell fate by modulating pluripot ency-specific chromosomal looping and TET2-mediated DNA demethylation. L Jia, Y Wang, C Wang, Z Du, S Zhang, X Wen, L Zhou, H Li, H Chen, D Li, Nucleic Acids Res. 487Jia, L., Wang, Y., Wang, C., Du, Z., Zhang, S., Wen, X., Zhou, L., Li, H., Chen, H., Li, D., et al. (2020). Oplr16 serves as a novel chromatin factor to control stem cell fate by modulating pluripot ency-specific chromosomal looping and TET2-mediated DNA demethylation. Nucleic Acids Res 48(7): 3935-3948.

RIPSeeker: a statistical package for identifying proteinassociated transcripts from RIP-seq experiments. Y Li, D Y Zhao, J F Greenblatt, Z Zhang, Nucleic Acids Res. 41894Li, Y., Zhao, D. Y., Greenblatt, J. F. and Zhang, Z. (2013). RIPSeeker: a statistical package for identifying protein- associated transcripts from RIP-seq experiments. Nucleic Acids Res 41(8): e94.

Single-nuclei RNA-seq on human retinal tissue provides improved transcriptome profiling. Q Liang, R Dharmat, L Owen, A Shakoor, Y Li, S Kim, A Vitale, I Kim, D Morgan, S Liang, Nat Commun. 1015743Liang, Q., Dharmat, R., Owen, L., Shakoor, A., Li, Y., Kim, S., Vitale, A., Kim, I., Morgan, D., Liang, S., et al. (2019). Single-nuclei RNA-seq on human retinal tissue provides improved transcriptome profiling. Nat Commun 10(1): 5743.

RNA-seq reveals novel CircRNAs involved in breast cancer progression and patient therapy response. C Nicot, Mol Cancer. 19176Nicot, C. (2020). RNA-seq reveals novel CircRNAs involved in breast cancer progression and patient therapy response. Mol Cancer 19(1): 76.

Differential oestrogen receptor binding is associated with clinical outcome in breast cancer. C S Ross-Innes, R Stark, A E Teschendorff, K A Holmes, H R Ali, M J Dunning, G D Brown, O Gojis, I O Ellis, A R Green, Nature. 4817381Ross-Innes, C. S., Stark, R., Teschendorff, A. E., Holmes, K. A., Ali, H. R., Dunning, M. J., Brown, G. D., Gojis, O., Ellis, I. O., Green, A. R., et al. (2012). Differential oestrogen receptor binding is associated with clinical outcome in breast cancer. Nature 481(7381): 389-393.

Transcriptomic and epigenetic mechanisms underlying myeloid diversity in the lung. E Sajti, V M Link, Z Ouyang, N J Spann, E Westin, C E Romanoski, G J Fonseca, L S Prince, C K Glass, Nat Immunol. 212Sajti, E., Link, V. M., Ouyang, Z., Spann, N. J., Westin, E., Romanoski, C. E ., Fonseca, G. J., Prince, L. S. and Glass, C. K. (2020). Transcriptomic and epigenetic mechanisms underlying myeloid diversity in the lung. Nat Immunol 21(2): 221-231.

Systematic Mapping of RNA-Chromatin Interactions In Vivo. B Sridhar, M Rivas-Astroza, T C Nguyen, W Chen, Z Yan, X Cao, L Hebert, S Zhong, Curr Biol. 274Sridhar, B., Rivas-Astroza, M., Nguyen, T. C., Chen, W., Yan, Z., Cao, X., Hebert, L. and Zhong, S. (2017). Systematic Mapping of RNA-Chromatin Interactions In Vivo. Curr Biol 27(4): 602-609.

Chromatin-RNA in situ Reverse Transcription Sequencing (CRIST-seq) Approach to Profile the Non-coding RNA Interaction Network. S Zhang, 10.21769/BioProtoc.4718.13Bio-protocol. 1314Cite as: Zhang, S. et al. (2023). Chromatin-RNA in situ Reverse Transcription Sequencing (CRIST-seq) Approach to Profile the Non-coding RNA Interaction Network. Bio-protocol 13(14): e4718. DOI: 10.21769/BioProtoc.4718. 13 Published: Jul 20, 2023

TopHat: discovering splice junctions with RNA-Seq. C Trapnell, L Pachter, S L Salzberg, Bioinformatics. 259Trapnell, C., Pachter, L. and Salzberg, S. L. (2009). TopHat: discovering splice junctions with RNA-Seq. Bioinformatics 25(9): 1105-1111.

Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. C Trapnell, B A Williams, G Pertea, A Mortazavi, G Kwan, M J Van Baren, S L Salzberg, B J Wold, L Pachter, Nat Biotechnol. 285Trapnell, C., Williams, B. A., Pertea, G., Mortazavi, A., Kwan, G., van Baren, M. J., Salzberg, S. L., Wold, B. J. and Pachter, L. (2010). Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. Nat Biotechnol 28(5): 511-515.

Genomewide interaction target profiling reveals a novel Peblr20-eRNA activation pathway to control stem cell pluripotency. C Wang, L Jia, Y Wang, Z Du, L Zhou, X Wen, H Li, S Zhang, H Chen, N Chen, Theranostics. 101Wang, C., Jia, L., Wang, Y., Du, Z., Zhou, L., Wen, X., Li, H., Zhang, S., Chen, H., Chen, N., et al. (2020). Genome- wide interaction target profiling reveals a novel Peblr20-eRNA activation pathway to control stem cell pluripotency. Theranostics 10(1): 353-370.

Pluripotency exit is guided by the Peln1-mediated disruption of intrachromosomal architecture. Y Wang, L Jia, C Wang, Z Du, S Zhang, L Zhou, X Wen, H Li, H Chen, Y Nie, J Cell Biol. 2214202009134Wang, Y., Jia, L., Wang, C., Du, Z., Zhang, S., Zhou, L., Wen, X., Li, H., Chen, H., Nie, Y., et al. (2022). Pluripotency exit is guided by the Peln1-mediated disruption of intrachromosomal architecture. J Cell Biol 221(4): e202009134.

Histone deacetylase inhibitor valproic acid promotes the induction of pluripotency in mouse fibroblasts by suppressing reprogramminginduced senescence stress. Y Zhai, X Chen, D Yu, T Li, J Cui, G Wang, J F Hu, W Li, Exp Cell Res. 3371Zhai, Y., Chen, X., Yu, D., Li, T., Cui, J., Wang, G., Hu, J. F. and Li, W. (2015). Histone deacetylase inhibitor valproic acid promotes the induction of pluripotency in mouse fibroblasts by suppressing reprogramming- induced senescence stress. Exp Cell Res 337(1): 61-67.

Intrachromosomal looping is required for activation of endogenous pluripotency genes during reprogramming. H Zhang, W Jiao, L Sun, J Fan, M Chen, H Wang, X Xu, A Shen, T Li, B Niu, Cell Stem Cell. 131Zhang, H., Jiao, W., Sun, L., Fan, J., Chen, M., Wang, H., Xu, X., Shen, A., Li, T., Niu, B., et al. (2013). Intrachromosomal looping is required for activation of endogenous pluripotency genes during reprogramming. Cell Stem Cell 13(1): 30-35.

Profiling the long noncoding RNA interaction network in the regulatory elements of target genes by chromatin in situ reverse transcription sequencing. S Zhang, Y Wang, L Jia, X Wen, Z Du, C Wang, Y Hao, D Yu, L Zhou, N Chen, Genome Res. 299Zhang, S., Wang, Y., Jia, L., Wen, X., Du, Z., Wang, C., Hao, Y., Yu, D., Zhou, L., Chen, N., et al. (2019). Profiling the long noncoding RNA interaction network in the regulatory elements of target genes by chromatin in situ reverse transcription sequencing. Genome Res 29(9): 1521-1532.

Osblr8 orchestrates intrachromosomal loop structure required for maintaining stem cell pluripotency. Y Zhu, Z Yan, Z Du, S Zhang, C Fu, Y Meng, X Wen, Y Wang, A R Hoffman, J F Hu, Int J Biol Sci. 1611Zhu, Y., Yan, Z., Du, Z., Zhang, S., Fu, C., Meng, Y., Wen, X., Wang, Y., Hoffman, A. R., Hu, J. F., et al. (2020). Osblr8 orchestrates intrachromosomal loop structure required for maintaining stem cell pluripotency. Int J Biol Sci 16(11): 1861-1875.

LncRNA Osilr9 coordinates promoter DNA demethylation and the intrachromosomal loop structure required for maintaining stem cell pluripotency. Y Zhu, Z Yan, C Fu, X Wen, L Jia, L Zhou, Z Du, C Wang, Y Wang, J Chen, Mol Ther. 316Zhu, Y., Yan, Z., Fu, C., Wen, X., Jia, L., Zhou, L., Du, Z., Wang, C., Wang, Y., Chen, J., et al. (2023). LncRNA Osilr9 coordinates promoter DNA demethylation and the intrachromosomal loop structure required for maintaining stem cell pluripotency. Mol Ther 31(6): 1791-1806.