# Protecting DNA from errors and damage: an overview of DNA repair mechanisms in plants compared to mammals

CorpusID: 31229576
 
tags: #Medicine, #Environmental_Science, #Biology

URL: [https://www.semanticscholar.org/paper/ad33cf05fe5608bbc171dcc28cd919acb29e3398](https://www.semanticscholar.org/paper/ad33cf05fe5608bbc171dcc28cd919acb29e3398)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | True |
| By Annotator      | (Not Annotated) |

---

Protecting DNA from errors and damage: an overview of DNA repair mechanisms in plants compared to mammals


Claudia P Spampinato 
Protecting DNA from errors and damage: an overview of DNA repair mechanisms in plants compared to mammals
10.1007/s00018-016-2436-2Received: 8 August 2016 / Revised: 1 December 2016 / Accepted: 5 December 2016 / Published online: 20 December 2016 Ó Springer International Publishing 2016REVIEWDNA repairPhotolyasesBERNERMMRHRNHEJ
The genome integrity of all organisms is constantly threatened by replication errors and DNA damage arising from endogenous and exogenous sources. Such base pair anomalies must be accurately repaired to prevent mutagenesis and/or lethality. Thus, it is not surprising that cells have evolved multiple and partially overlapping DNA repair pathways to correct specific types of DNA errors and lesions. Great progress in unraveling these repair mechanisms at the molecular level has been made by several talented researchers, among them Tomas Lindahl, Aziz Sancar, and Paul Modrich, all three Nobel laureates in Chemistry for 2015. Much of this knowledge comes from studies performed in bacteria, yeast, and mammals and has impacted research in plant systems. Two plant features should be mentioned. Plants differ from higher eukaryotes in that they lack a reserve germline and cannot avoid environmental stresses. Therefore, plants have evolved different strategies to sustain genome fidelity through generations and continuous exposure to genotoxic stresses. These strategies include the presence of unique or multiple paralogous genes with partially overlapping DNA repair activities. Yet, in spite (or because) of these differences, plants, especially Arabidopsis thaliana, can be used as a model organism for functional studies. Some advantages of this model system are worth mentioning: short life cycle, availability of both homozygous and heterozygous lines for many genes, plant transformation techniques, tissue culture methods and reporter systems for gene expression and function studies. Here, I provide a current understanding of DNA repair genes in plants, with a special focus on A. thaliana. It is expected that this review will be a valuable resource for future functional studies in the DNA repair field, both in plants and animals.

## Introduction

All organisms are constantly exposed to environmental stresses as well as genotoxic products from endogenous metabolic processes that induce, either directly or indirectly, DNA damage [1]. Another source of DNA damage includes base-base mismatches produced by the incorporation of a non-complementary Watson-Crick base and unpaired nucleotides caused by misalignment of the two DNA strands, both generated during DNA synthesis. These DNA biosynthetic errors differ from typical DNA damage in that they involve undamaged bases and exist only transiently provided that the DNA remains double-stranded [2]. To counteract these genome integrity threats, all organisms have evolved the DNA damage response (DDR) [3,4]. The DDR is a pathway that transduces the DNA damage signal into activation of various pathways that leads to DNA repair, cell cycle checkpoint and programmed cell death. Plant DDR also includes endoreduplication. Endoreduplication involves replication of the nuclear DNA without cytokinesis [5]. Here, I only focus on DNA repair mechanisms in mammals and plants. Plants are distinctly different from mammals in that they lack a reserved germ line and are sessile. Gametes arise from meristem cells that & Claudia P. Spampinato spampinato@cefobi-conicet.gov.ar have been exposed to the continuous effect of environmental and endogenous mutagens and have divided many times. Thus, plants have evolved particularly efficient DNA repair mechanisms. Advancements in this field have been provided by the Arabidopsis genome sequence [6]. Plants have orthologs of most of the genes involved in mammalian DNA repair pathways [7][8][9]. However, several interesting features may be noted: the presence of unique genes, the presence of multiple gene copies and the absence of well-characterized genes in plant genomes [9,10]. Such differences suggest plant-specific DNA repair pathways. Provided here is a review of major findings over the last decade related to mammal and plant DNA repair mechanisms, with a special focus on DNA repair genes from A. thaliana.


## Photoreactivation

Photoreactivation is the direct reversal of major lesions induced by ultraviolet (UV) light on two adjacent pyrimidine bases, such as cyclobutane pyrimidine dimers (CPDs) and pyrimidine (6-4) pyrimidones (6-4 photoproducts or 6-4 PPs). The history of the discovery of this repair system has been recently reviewed [11]. Photoreactivation relies on single light-activated flavoenzymes called photolyases. Photolyases belong to the photolyase/cryptochrome family and have been classified into three related subfamilies known as class I CPD photolyase, class II CPD photolyase, and 6-4 photolyase, based on their sequence homology and substrate specificity [12][13][14]. Multiple sequence alignment analysis of members from the three subfamilies indicates a highly functional conservation of a photolyase-homologous region (PHR) domain that binds the chromophore FAD (flavin adenine dinucleotide) [15,16]. Class I CPD photolyases have been reported in diverse microbial organisms such as archaea, eubacteria, and fungi; class II CPD photolyases have been found not only in archaea, eubacteria, some viruses, but also in higher organisms such as animals and plants; and 6-4 photolyases have been identified in cyanobacteria and certain eukarya [15,17]. Humans and other placental animals have lost photolyase activities. All photolyases use the energy of visible or blue light (k = 320-500 nm) for the catalytic cleavage of the photoproducts and contain the fully reduced FADHas the catalytically active cofactor [13,14]. All photolyases characterized to date also contain a second chromophore, such as methenyltetrahydrofolate found in the majority of the enzymes or 7,8-didemethyl-8-hydroxy-5-deazariboflavin, FAD or FMN (flavin mononucleotide) in others that functions as a photoantenna to enhance light absorption [14,[18][19][20]. Upon blue light absorption, the excitation energy is efficiently transferred from the photoantenna to FADHby Förster resonance energy. The resulting FADH -* donates an electron to the DNA lesion with the generation of two canonical pyrimidines [12,14,[19][20][21].


## Photoreactivation in plants

Arabidopsis thaliana contains two specific class II CPD photolyases [PHR1, also named UVR2 (At1g12370) and PHR2 (At2g47590)] and one 6-4 photolyase [UVR3 (At3g15620)]. The FAD-binding domains of plant PHR2 proteins are smaller and less conserved than in the other subfamilies [15]. Several articles provide a summary of the molecular cloning, the tissue-specific and light-dependent regulation and the characterization of mutants of CPD photolyase and 6-4 photolyase genes from various plant species (e.g., A. thaliana, Oriza sativa, Sinapis alba) [22][23][24]. Recent evidence suggests that AtPHR1 expression induced by UV-B is primarily regulated by the UVR8 (UV RESISTANCE LOCUS8)-dependent pathway but is also mediated by a UVR8-independent pathway [25]. The latter one was found to be correlated with UV-B-induced CPD levels [25].

The crystal structure of a UVR2 homolog has recently been determined in rice and compared with prokaryotic class I CPD photolyases [26]. Results indicate that both enzymes recognize CPD damage in similar active sites but create a different conformational distortion in the DNA duplex [26]. The crystallographic structure and molecular mechanism of A. thaliana 6-4 photolyase have also been determined [27,28]. After light excitation, there is initially a forward electron transfer followed by a cyclic proton transfer involving an active-site histidine residue [28]. This proton transfer is the determinant step in the repair and determines the overall quantum yield (A = number of dimers repaired per number of photons absorbed) [28]. For comparison, the reported A for CPD and 6-4PP repair are 0.82 and 0.1, respectively, and the whole catalysis reaction takes less than a nanosecond for CPD repair or tens of nanoseconds for 6-4PP repair [29]. The complete photocycle in real-time from the initial 6-4 PP to several intermediates and subsequent conversion to two thymine bases catalyzed by the A. thaliana (6-4) photolyase has been recently well reviewed [21,29].


## Base excision repair

Base excision repair (BER) recognizes and repairs lesions such as deaminated, oxidized and alkylated bases, abasic (apurinic and/or apyrimidinic, AP) sites and single-strand breaks (SSBs). BER occurs in several steps (Table 1A; see references [30][31][32] for recent reviews in mammalian BER).

(1) Lesion recognition and removal by DNA glycosylases. DNA glycosylases hydrolyze the N-glycosidic bond of the damaged base, leaving an AP site. Different DNA glycosylases exist, each one with broad or narrow substrate specificities. Enzymes can be classified as monofunctional or bifunctional. Monofunctional glycosylases perform base excision only, whereas bifunctional glycosylases also have lyase activity. Biochemical mechanisms of DNA glycosylases have been the subject of recent reviews [33,34]. (2) Cleavage of the sugar-phosphate backbone at the AP site. Cleavage is catalyzed either by an AP endonuclease or by the intrinsic AP lyase activity of a bifunctional DNA glycosylase. AP endonucleases and lyases generate different types of DNA termini. AP endonucleases release a 3 0 OH and a 5 0 deoxyribose-phosphate moiety (5 0 dRP), while AP lyases release a 3 0 phospho a,b-unsaturated aldehyde (3 0 PUA) or 3 0 phosphate by bor bd-elimination reactions, respectively, and a 5 0 phosphate. (3) Processing of unconventional ends to conventional 3 0 OH and 5 0 phosphate moieties. This step involves different enzymes depending on whether the modified terminus is processed from the 5 0 or 3 0 end. The intrinsic dRP lyase activity of DNA polymerase b removes the 5 0 dRP; the 3 0 phosphodiesterase activity of the AP endonuclease eliminates the 3 0 PUA, whereas polynucleotide kinase 3 0 phosphatase (PNKP) processes the 3 0 phosphate moiety. Additional SSB end cleaning enzymes are aprataxin and tyrosyl DNA phosphodiesterases (TDP) that process abortive ligation or topoisomerase reactions, respectively [35].  Step Schematic representation Gene a AGI locus code b,c


## i) Lesion recognition and removal OGG1

At1g21710 
FPG At1g52500 NTH1 At2g31450 NTH2 At1g05900 DME At5g04560 ROS1 At2g36490 UNG At3g18630 DML3 At4g34060 MBD4L At3g07930 ii) Cleavage

## PARP2 At4g02390

Pol δ See Table 2 Pol ε See Table 2  LIG1 See Table 2 OH P a Genes encoding bifunctional glycosylases are listed first, followed by monofunctional glycosylases b AGI Arabidopsis genome initiative c AGI locus code for components involved in DNA metabolic pathways is shown in Table 2 d Bifunctional DNA glycosylases with associated b-elimination activity are not detailed Protecting DNA from errors and damage 1695 in filling single-nucleotide gaps and can perform gap filling in long-patch BER [36]. Poly ADP-ribose polymerase 1 (PARP1) and X-ray cross-complementation group 1 (XRCC1) also have roles in BER. PARP1 protects SSBs from the formation of more deleterious double-strand breaks (DSBs) [35] and XRCC1 is involved in the organization of BER-proficient multiprotein complexes [37]. Finally, the nick is sealed by a DNA ligase.


## BER in plants

Most of the BER proteins found in animal cells have been described in plants [38,39]. Tables 1B and 2 show an overview of described factors involved in BER mechanism in A. thaliana. Plants contain several DNA glycosylases: 8-oxoguanine DNA glycosylase (OGG1), formamidopyrimidine DNA glycosylase (FPG), NTH, and uracil DNA glycosylase (UNG) [38][39][40][41][42][43][44]. The enzymes OGG1 and FPG recognize oxidized purines; NTH, oxidized pyrimidines and UNG, U:G mismatches. Both NTH homologs (NTH1 and NTH2) were co-localized to nucleoids within Arabidopsis chloroplasts, thus indicating the existence of a BER pathway in these organelles to deal with photo-oxidative stress [44]. Plant genomes, in contrast to mammals, also encode specific 5-methyl-cytosine (5-meC) glycosylases that are involved in DNA demethylation [DEMETER (DME) and Repressor of Silencing 1 (ROS1)] [45][46][47][48]. These proteins show preference for 5-meC as substrate but can also process T:G mispairs. It has been reported that ROS1 has a very low turnover [46], with the recognition of pyrimidine modifications being the rate-limiting step [49]. An additional activity against T:G mispairs is displayed by MBD4L (methyl binding 4 DNA glycosylase-like) [50]. This enzyme was demonstrated to act on both U:G and T:G mispairs and to excise both U and T more efficiently at a CpG context. Roles of OGG1, ROS1 and MBDL4 in Arabidopsis tolerance to oxidative DNA damage have also been described using mutant or overexpression plants [51][52][53]. Other downstream proteins of the BER pathway have been characterized in plants. The Arabidopsis genome contains three genes encoding AP endonuclease-like proteins: APE1L (abasic endonuclease-1-like), APE2 (abasic endonuclease 2) and ARP (abasic endonuclease-redox protein) [54]. All three enzymes were found to exhibit AP endonuclease activity in vitro [55], with ARP representing the major activity in Arabidopsis cell extracts [56]. Based on the available DNA sequence data, a wheat homolog of AtAPE1L has been cloned and characterized [57]. Orthologues of human PNKP (named AtZDP), TDP1, XRCC1 and DNA ligase I were also functionally characterized in Arabidopsis [42,55,56,[58][59][60][61]. Finally, it has been demonstrated that AP sites may be processed in plants through both SN-and LP-BER [41] and that DNA ligase I restores the continuity of the repaired DNA strand during both SN-and LP-BER [56]. Recent studies identified and characterized a gene named SAV6 (shade avoidance mutant), a homolog of human FEN1 [62]. Compared to the animal FEN1, SAV6 shows both flap and gap endonuclease activities, but lacks exonuclease activity.


## Nucleotide excision repair

Nucleotide excision repair (NER) is responsible for the processing of bulky helix-distorting damage, such as CPDs and 6-4PPs induced by UV irradiation. Deficiencies in NER are associated with several human autosomal recessive disorders, namely xeroderma pigmentosum (XP), cockayne syndrome (CS) and trichothiodistrophy (TTD) among others. XP, CS and TTD have provided the names of some of the genes involved in the pathway (XPA through XPG, XPV, CSA, CSB and TTDA). NER is a multistep process which comprises four steps (Table 3A).

(1) Damage recognition. Two different modes of damage recognition coexist: global genome NER (GG-NER) and transcription-coupled NER (TC-NER). GG-NER detects damage occurring throughout the genome and is specifically initiated by a heterotrimeric XPC-HR23B-CEN2 complex (XPC and HR23A/HR23B are the human homologs of yeast and plant RAD4 and RAD23 proteins, respectively), with the assistance in some cases of DDB (damaged DNA-binding) protein complex. DDB, a heterodimeric complex comprising DDB1 and DDB2 subunits, specifically binds CPDs and 6-4PPs with moderate or high affinity, respectively, and stimulates significantly XPC binding to UV-damaged lesions. GG- NER in mammalian cells has been covered in recent reviews [63,64]. TC-NER is responsible for the recognition of lesions in the transcribed strand of active genes and is initiated by RNA polymerase stalled at a lesion, with the help of specific factors CSA, CSB, and XPA binding protein 2 (XAB2) whose function is not clear. Many aspects of mammalian TC-NER can be found in some recent reviews [64][65][66].

(2) Formation of a stable preincision complex around the damage site. After DNA recognition, GG-NER and TC-NER converge into the same pathway. The transcription factor II H (TFIIH) is recruited to the lesion. TFIIH is a multiprotein complex composed of ten subunits arranged in two functional subcomplexes: the core subcomplex containing the DNA helicase XPB, p62, p52, p44, p34 and p8 (TTDA) and the CAK [cyclin dependent kinase (CDK)-activating kinase] subcomplex containing CDK7, cyclin H (CYCH) and the assembly factor ménage-à-trois-1 (MAT1); both subcomplexes are bridged by the DNA helicase XPD [67,68]. The result of the TFIIH activity is the partial unwinding of the DNA duplex that leads to the recruitment of XPA, RPA (replication protein A), and the endonuclease XPG. Then, XPA interacts and recruits the second endonuclease, ERCC1 (excision repair cross complementing 1)-XPF. (3) Excision of the damaged nucleotide. The damaged nucleotide is removed by dual incisions 5 0 and 3 0 to the lesion by endonucleases XPF and XPG, respectively, generating a 24-32 oligonucleotide single-strand fragment containing the damaged site. There Table 3 Overview of steps (A) and A. thaliana required factors (B) involved in NER


## A B

Step Schematic representation Gene AGI locus See Table 2 a AGI Arabidopsis genome initiative b AGI locus code for components involved in DNA metabolic pathways is shown in Tables 2 and 4 Protecting DNA from errors and damage 1697 is a defined order of the two incisions. The 5 0 incision by the heterodimer ERCC1-XPF is performed first; repair synthesis is initiated followed by the 3 0 incision by XPG [69]. (4) Completion of DNA synthesis followed by ligation. The repair synthesis is carried out by DNA polymerases d, e, or j depending on chromatin accessibility of the damaged site [70]. NER is completed by nick sealing by DNA ligase I or IIIa.
code a,b i) Damage recognition RAD4 At5g16630 RAD23A At1g16190 RAD23B At1g79650 RAD23C At3g02540 RAD23D At5g38470 CEN2 At4g37010 DDB1A At4g05420 DDB1B At4g21100 DDB2 At5g58760 CSA At1g27840 At1g19750 CHR8 At2g18760 CHR24 At5g63950 ii) Formation of a stable preincision XPB1 At5g41370 complex around the damage site XPB2 At5g41360 UVH6 At1g03190 TFIIH1 At1g55750 At1g61420 GTF2H2 At1g05055 TFIIH3 At1g18340 TFIIH4 At4g17020 TTDA At1g12400 At1g62886 CDKD;1 At1g73690 CDKD;2 At1g66750 CDKD;3 At1g18040 CYCH;1 At5g27620 MAT1 At4g30820 iii) Excision of the damaged nucleotide UVH3 At3g28030 UVH1 At5g41150 ERCC1 At3g05210 RPA See

## NER in plants

This pathway has been primarily studied in Arabidopsis and rice [7]. In Arabidopsis, four genes, namely RAD23, DDB1, CSA and XPB have been duplicated (Table 3B) [7,71]. Interestingly, RAD23 was also found to be duplicated in rice, Populus and Sorghum and many times in Arabidopsis [7,71]. Although Arabidopsis mutants affecting the expression of individual RAD23 genes exhibited pleiotropic developmental defects (rad23b) or no obvious phenotype (rad23a, rad23c, and rad23d), higherorder mutant combinations indicated that the RAD23 family is essential in Arabidopsis [72]. The two DDB1 homologs, DDB1A and DDB1B, were expressed throughout plant development with DDB1B showing the lowest levels [73,74] but playing a crucial role because ddb1b null alleles appear lethal ( [73] and references therein).

Overlapping expression patterns were also observed for the two CSA homologs. Both homologs share significant sequence identity and likely form heterotetramers in plants [75]. Decreased levels of several of these genes, namely DDB1A, DDB1B, CSA or XPB1 affected UV plant tolerance [75][76][77][78][79] The interaction of XPC-CEN2 (AtRAD4-AtCEN2) and validation of some in silico predicted TFIIH subunit transcripts were also reported in plants [7,[83][84][85][86][87]. Human TFIIH components p62, p44, p34 and p52 correspond to TFIIH1, TFIIH2 (AtGTF2H2), TFIIH3 and TFIIH4 in A. thaliana, while CDK7 is encoded by three CDKDs (Table 3B). Homologs of the human RPA complex, heterotrimer composed of three associated subunits RPA1, RPA2 and RPA3, were also functionally characterized in Arabidopsis [88]. However, in contrast to other eukaryotes, multiple genes encode the RPA subunits in A. thaliana (Table 4) [88,89]. A phylogenetic tree based on RPA1 protein sequences revealed three evolutionary groups [90]. Group A is composed of RPA1A; group B, of RPA1B and RPA1D and group C, of RPA1C and RPA1E proteins. These authors propose that group C includes proteins involved in DNA damage repair [90]. No apparent XPA homolog exists in plants [10].


## Mismatch repair

The mismatch repair (MMR) system is best known for its role in the recognition and correction of single base-base mismatches and unpaired nucleotides that arise through replication errors, deamination of 5-methylcytosine, and recombination between divergent sequences. Eukaryotic MMR has been studied in great detail and has been covered in several recent reviews [2,[91][92][93][94]. Also, an overview of the history of eukaryotic MMR has been recently reported [95]. The pathway involves several steps (Table 5A). (1) Lesion recognition by MutS proteins. In eukaryotes, MutS proteins function as heterodimers composed of related, but distinct MutS homologs (MSH) subunits. To date, eight MSH subunits were discovered [96], of which MSH2, MSH3 and MSH6 are involved in nuclear MMR in human cells. These MSHs subunits assemble as MSH2-MSH6 (MutSa) and MSH2-MSH3 (MutSb). MutSa recognizes base-base mismatches and short insertion/deletion loops (IDLs) while MutSb mediates the repair of IDLs up to 16 nucleotides [97]. The molecular basis of substrate specificities has been investigated. A series of crystal structures of human MutSa in complex with different DNA substrates showed an extensive interaction of a conserved phenylalanine residue in the MSH6 subunit with the DNA mispair [98]. MSH3 lacks the phenylalanine residue that makes protein-mispair contact. Instead, MutSb binds the sugar-phosphate backbone of the IDL [99]. DNA is severely bent and the unpaired bases are flipped out and solvent exposed as revealed by the crystal structure of human MutSb complexed with DNA containing IDLs of varying size [99]. In addition to crystal reports, abilities of MutSa and MutSb to process base-base mismatches and IDLs were explored. The results indicate that MutSa and MutSb differ in DNA substrate recognition and that the MutSb-IDL interaction is greatly stimulated by an excess . The presence of these linkers has been associated with large asymmetric conformational changes following sequential ATP binding and hydrolysis [107]. (3) Strand discrimination. Identification and discrimination of the newly synthesized DNA strand in eukaryotes are attributed to strand breaks involved in the leading-and lagging-strand synthesis. Replication factor C (RFC) specifically recognizes 3 0 termini at the replication fork and loads PCNA asymmetrically onto these sites, with the same side facing towards the strand discontinuity [108]. PCNA loaded at these breaks is required for MutLa endonuclease activation, apparently through protein-protein interaction [109]. Once activated, MutLa incises the strand containing the preexisting strand break due to the loading polarity of PCNA [109]. MutLa endonuclease can also be activated on DNA that contains a MutSb recognizable lesion but, while MutSa can interact independently with MutLa and PCNA, interactions of MutSb with MutLa and PCNA were found to be mutually exclusive [102]. These results suggest important differences in the repair mechanisms of single base-base mismatches and IDLs. (4) Strand excision and re-synthesis. Multiply incised molecules are substrates for the 5 0 -3 0 activity of MutSaactivated exonuclease I (Exo1) [110]. Once the mismatch is excised, MutSa and RPA regulate Exo1 activity that leads to excision termination [111]. The repair reaction is completed by a correct DNA re-synthesis by DNA polymerase d, followed by ligation. In the absence of Exo1, MMR can still be detected suggesting the existence of one or more Exo1-independent modes of mismatch repair. The mechanism most likely involves a strand displacement synthesis by DNA polymerases d that is strongly stimulated by RPA  Table 4 POL δ See Table 2 a AGI Arabidopsis genome initiative b AGI locus code for components involved in DNA metabolic pathways is shown in Tables 2 and 4 Protecting DNA from errors and damage 1699 [112]. Exo1-dependent and independent MMR has been the subject of a recent review [113].


## MMR in plants

Arabidopsis and other plants encode homologs of MutS and MutL proteins found in other eukaryotic organisms, but also contain a unique MSH polypeptide (MSH7) [114,115] (Table 5B). MSH7 forms a heterodimer with MSH2 and the protein complex is designated MutSc. Quantitative analysis of transcript levels of representative genes (MSH2 and PMS1) of the pathway indicated that these genes show a higher expression in calli than in seedlings from A. thaliana, thus confirming the essential genome maintenance function performed by MMR system in rapidly dividing tissues [116,117]. Further knowledge of   Table 2 iv) Resolution of the D-loop
RAD50 At2g31970 NBS1 At3g02680 COM1 At3g52115 RECQ4A At1g10930 RAD51 At5g20850 RAD51B At2g28560 RAD51C At2g45280 RAD51D At1g07745 XRCC2 At5g64520 XRCC3 At5g54750 BRCA2A At4g00020 BRCA2B At5g01630 RAD54At3g19210SDSA SRS2 At4g25120 FANCM At1g35530 DSBR EME1A At2g21800 EME1B At2g22140 MUS81 At4g30870 GEN1 At1g01880 SEND1 At3g48900 HJ dissolution RECQ4A At1g10930
TOP3α At5g63920


## RMI1 At5g63540

a AGI Arabidopsis genome initiative b AGI locus code for components involved in DNA metabolic pathways is shown in Table 2 the role of a given plant MMR protein came from in vivo studies. Using a yeast system, we have found that AtMutSc is able to specifically recognize G/T, A/C, T/C, G/A, T/T, or A/A mismatches and/or neighboring sequences containing T [118] and that the expression of AtMutLa affected the repair of IDLs in microsatellite sequences [119]. The functionality of plant MutS and MutL homologs was also assessed using A. thaliana (reviewed in [114]), tomato [120], rice [121] and Nicotiana [122] mutant plants.

Suppression of MSH2, PMS1 or MLH1 was associated with microsatellite instability, an increase in homeologous recombination relative to wild-type plants and a broad spectrum of mutant phenotypes in their progenies. Inactivation of MSH7 led a modest increase in the frequency of recombination between Solanum lycopersicum and a distantly related species (Solanum lycopersicoides) [120], while in A. thaliana, the inactivation of MSH7 did not affect the rates of somatic homologous or homeologous recombination, but influenced the rates of meiotic recombination [123]. Finally, somewhat comparable to mammalian proteins, we demonstrated that msh2 mutant plants showed an altered expression pattern of cell cycle marker genes after induced DNA damage in comparison to wild-type plants and we experimentally validated MSH6 as an E2F target gene [124].


## Double-strand break repair

Double-strand break (DSB) repair uses two main strategies: a homology-dependent, error-free homologous recombination repair (HR); a potentially error-prone, classical, Kudependent non-homologous end-joining (C-NHEJ) and a recently identified error-prone repair pathway, named alternative non-homologous end-joining pathway (A-NHEJ). The choice between these pathways depends on the phase of the cell cycle and the initiation of DNA end resection [125][126][127].


## Homologous recombination

HR is favored during the S/G2 phase of the cell cycle, when sister chromatids are available as homologous templates [128,129]. This mechanism was the subject of several recent reviews [130][131][132]. In this pathway, four steps are required (Table 6A). (1) DNA end resection and nucleoprotein filament formation. Initial processing of the ends involves the trimeric complex MRE11-RAD50-NBS1 (MRN) and the CtBP (C-terminal binding protein) interacting protein (CtIP) [133,134]. Further resection involves the 5 0 -3 0 exonuclease EXO1 or the combined helicase/ nuclease activities of the BLM/DNA2 (Bloom Syndrome RecQ Like Helicase/DNA replication helicase/nuclease2) [135]. The 3 0 single-strand DNA (ssDNA) tails generated are initially bound by ssDNA-binding protein RPA [136]. Subsequently, RPA is replaced by another ssDNA-binding protein, RAD51, which forms a nucleoprotein filament on the DNA. The replacement of RPA by RAD51 requires several mediator proteins, such as BRCA2 (protein encoded by the breast cancer-associated gene 2) and RAD54 [137][138][139]. Human cells have two RAD54 homologs, RAD54 and RAD54B, which display similar biochemical activities [140]. Evidence also suggests that RAD51 paralogs, such as XRCC2, XRCC3, RAD51B, RAD51C, and RAD51D, function in promoting and/or stabilizing RAD51 nucleofilaments [131,141]. However, unlike RAD51 that self-assembles, RAD51 paralogs form at least four different heterodimeric and heterotetrameric complexes. (2) DNA homology search and strand invasion. The RAD51 nucleoprotein filament begins searching for a homologous DNA sequence and then invades the intact double-stranded DNA molecule to form a heteroduplex DNA structure termed the displacement loop (D-loop). D-loop formation is stimulated by RAD54 [131,132]. Besides ssDNA binding, RAD51 also interacts with dsDNA and associates with the newly formed heteroduplex DNA. (3) DNA heteroduplex extension. The invading strand in the D-loop structure is then extended by several components of the replication machinery, namely DNA polymerase d, PCNA and RFC [142]. (4) Processing of the D-loop. HR can take several different steps: synthesis-dependent strand annealing (SDSA), Holliday junction (HJ) resolution (the DSBR model) and HJ dissolution [130,132]. SDSA involves the displacement of the newly synthesized strand by DNA helicase(s), followed by strand annealing, DNA synthesis and ligation. DSBR is characterized by the capture of the second end of the DSB, formation of a double HJ (dHJ) structure and resolution by specialized resolvases (e.g., GEN1, MUS81-EME1, SLX1-SLX4). Alternatively, the dHJ can be dissolved by the BTB complex consisting of BLM helicase, DNA topoisomerase IIIa and BLAP75. Regulation of HJ processing enzymes has been covered in recent reviews [143,144]. Depending on the pathway employed for HJ processing, different recombination outcomes are generated. SDSA and HJ dissolution lead to the formation of non-crossover while HJ resolution contributes to the formation of non-crossover or crossover. The formation of non-crossover recombinants is promoted in mitotic cells, in contrast with that occurring in meiotic cells.

An alternative process that involves recombination between regions of homology at both sides of the break can also occur. This process, known as single-strand annealing (SSA), requires HR proteins involved in DNA end resection and annealing and also RPA, but is independent of RAD51 [130][131][132]145]. After DNA end resection, the 3 0 ssDNA tails generated can reveal complementary sequences on both sides of the break. These sequences anneal forming a branched structure. The 3 0 tails are then removed and the nicks are ligated. This process inevitably results in a deletion at the repair junction.

Classical non-homologous end-joining C-NHEJ functions throughout the cell cycle but is dominant in G1 and G2 [129]. Several recent reviews have focused on C-NHEJ [146][147][148][149][150]. Three different steps are involved in this pathway. (1) DNA end recognition and tethering. The initial step is the recognition and binding of DSBs by the Ku protein (heterodimer of KU70-KU80 in eukaryotes) in a sequence-independent manner. Recent evidence suggests that only two Ku molecules are present at a DSB in vivo, presumably one at each end of the DSB [151]. The Ku-DNA complex then interacts with the DNAdependent protein kinase (DNA-PK) catalytic subunit (DNA-PKcs) forming a very stable and functional DNA-PK complex. This complex remains tethered to the end and prevents end access. The DNA-PK activated protein undergoes extensive autophosphorylation in vivo [152]. This results in a conformational change that opens the central DNA-binding cavity and releases the autophosphorylated DNA-PKcs from DNA ends [152]. Structures and functions of the Ku protein and the DNA-PK have been recently reviewed [153][154][155]. (2) DNA end processing. Depending on the nature of the DNA damage, one or more enzyme activities are involved in DNA end processing. For example, the 5 0 -dRP/AP lyase activity of the Ku protein excises abasic or 5 0 -dRP/AP sites [156,157], the tyrosyl DNA phosphodiesterase activity of TDP2 hydrolyzes 5 0 phosphotyrosyl-DNA bonds [158], and the 3 0 -DNA phosphatase and 5 0 -DNA kinase activities of the polynucleotide kinase/phosphatase (PNKP) generate ligatable 5 0 -phosphates and 3 0 -hydroxyl ends [159,160], among others. In many cases, recruitment of these proteins depends on their direct interaction with Ku [154]. (3) Ligation of the DSB. The final step is carried out by DNA ligase IV, which functions in complex with its cofactors XRCC4 and XLF (XRCC4 like factor, also known as Cernunnos) [150]. Recently, a new protein named PAXX (PAralog of XRCC4 and XLF) has been reported to promote Ku-dependent DNA ligation in vitro [161].


## Alternative non-homologous end-joining pathway

A-NHEJ increases as cells enter S-phase [162]. The molecular mechanism of A-NHEJ has recently been reviewed [163][164][165]. A-NHEJ occurs in the absence of key C-NHEJ factors and can involve alignment of microhomologous sequences; thus, this pathway is also known as microhomology-mediated end-joining (MMEJ). A-NHEJ shares with C-NHEJ similar reaction steps. (1) DNA end recognition and tethering. DSBs are efficiently bound by PARP1 under Ku defective conditions [166]. In fact Ku and PARP1 compete for DNA end binding, and Ku also limits poly ADP-ribose (PAR) synthesis and ssDNA production in response to DSBs [166,167]. Upon DSB binding, PARP1 gets activated and subsequently autopoly(ADP-ribosyl)ated. Auto-modification of PARP1 results in a decreased affinity for nucleosomes and in a relaxed local chromatin structure, which facilitates chromatin remodeling for DNA repair [168,169]. Overviews of the involvement of PARP1 in DNA repair have been published [170][171][172]. (2) DNA end processing. PARP1mediated PAR synthesis then triggers recruitment of the MRN complex to initiate incompatible end resection [173,174]. (3) Ligation of DNA ends. MRN interacts with the DNA ligase IIIa/XRCC1 complex and stimulates the joining of DNA ends [175]. Joining junctions involve microhomologies and deletions [175]. A second pathway independent of microhomologies that relies on DNA ligase I has also been suggested [176].


## DSB repair in plants

Knowledge of recombination mechanisms in plants has significantly advanced in recent years, especially because DSB repair emerged as an important tool to achieve controlled modifications of plant genomes (for recent reviews see [177][178][179][180][181][182][183]). Most of the HR proteins have been identified and characterized in A. thaliana (Table 6B). Plants with decreased transcript levels of genes involved in the initial processing of the ends, namely MRE11 [22], RAD50 [22], NBS1 [184] and COM1 (a homolog of the human CtIP, [185]) displayed hypersensitivity (measured as a reduction in root growth, fresh weight or development of true leaves) to some genotoxic compounds [mitomycin (MMC) or methylmethanesulfonate (MMS), agents that produce inter-strand DNA crosslinks or that methylates DNA, respectively]. Interaction between NBS1 and MRE11 has been reported for A. thaliana, maize and rice [184,186]. The NBS1 region involved in the interaction was further characterized in A. thaliana and defined towards the C-terminus [184]. Homologs of BLM, DNA2 and RPA are also conserved in plants. The loss of RECQ4A (a homolog of mammalian BLM helicase, [187][188][189], JHS1 (a homolog of human DNA2, [190] or RPA1C and RPA1E

[89] led to hypersensitivity (measured as a reduction in fresh weight or root growth) in response to a particular DNA damaging agent [MMS; cisplatin (CPT, an agent that mainly induces intra-strand DNA crosslinks); hydroxyurea (an agent that inhibits the ribonucleotide reductase and consequently reduces the dNTP pool and stalls replication forks); zeocin (an agent that induces DSBs) or ionizing radiation]. The recq4A mutant also exhibited an increased frequency of HR [187,189] suggesting a role for the RECQ4A protein in HR regulation. Suppression of HR is dependent on both the N-terminal region and the helicase activity of the protein [189]. A hyper-recombination phenotype was also observed for the jhs1 mutant, probably due to an increased expression of genes involved in DNA damage repair [190].

Several studies have also addressed the functions of RAD51 and RAD51 paralogues in plants [191][192][193][194]. The rad51, rad51c or xrcc3 A. thaliana mutants showed a deficiency in SDSA [192]. Mutants impaired for the other three RAD51 paralogues (RAD51B, RAD51D, XRCC2) were also defective for spontaneous HR [191,193,195], although to differing extents [191,193]. The xrcc2 mutant was the most affected. A further reduction in the recombination rate has been observed in rad51b/xrcc2 double mutant and rad51b/rad51d/xrcc2 triple mutant line [193]. The triple mutant also exhibited a greater sensitivity to the DSB-inducing agent bleomycin (BLE) than the single and double mutants, suggesting a partial functional redundancy [194]. The redundancy of the two AtBRCA2 genes has been also investigated [196][197][198]. Both Atbrca2a and Atbrca2b mutants showed hypersensitivity against CPT and ionizing irradiation, further stimulated in the Atbrca2a/Atbrca2b double-mutant line [196]. Another report indicated that the double mutant also showed hypersensitivity to MMC and a defect in somatic HR frequency [197].

The role of other factors involved in the process of strand exchange or in the stabilization of recombination intermediates has been elucidated using mutant plants. Loss of RAD54 reduced SDSA and loss of FANCM reduced both SDSA and SSA [192]. Proteins involved in the resolution of the D-loop were characterized in vitro or in vivo. Recombinantly expressed SRS2 protein showed a functional 3 0 to 5 0 DNA helicase activity that was able to unwind nicked and partial HJ [199]. Biochemical properties of MUS81-EME1 complexes and GEN1 were also determined after cloning, heterologous expression and purification [200,201]. Two functional EME1 (EME1A and EME1B) and two functional GEN1 [GEN1 (XPG-like endonuclease) and SEND1 (single-strand DNA endonu-clease1)] homologs have been identified in A. thaliana. Both complexes, MUS81-EME1A and MUS81-EME1B, were able to cleave nicked HJs and, with a reduced efficiency, intact HJs [201]. Both GEN1 paralogs preferentially cleaved HJs near the junction point; slight differences in sequence preferences have been detected [200]. The role of MUS81 was also defined in vivo. The mus81 mutant line showed a strong reduction in dry weight to MMC, CPT or MMS treatment [188]. Also, altered sensitivity (measured as a reduction in fresh weight) to MMC and CPT was observed in recq4A, top3A and rmi1 single mutant lines [202]. These three lines also showed an enhanced frequency of uninduced somatic HR [202].

In addition to the HR mechanism, KU-dependent and independent pathways have been also identified in plants. Arabidopsis encodes KU70 (At1g16970), KU80 (At1g48050), PARP1 (At2g31320) and PARP2 (At4g02390) [22,203]. A. thaliana impaired in KU80 showed severe root developmental defects after c irradiation [204,205] and higher levels of DSBs [195]. As with the ku80 mutant plant, the xrcc1 mutant line also exhibited a hypersensitivity to c irradiation [205]. Furthermore, analysis of DSB repair kinetics of ku80 and xrcc1 single mutant lines and ku80/xrcc1 double mutant line indicated a more severe repair defect in the double mutant than in single mutant plants [205]. These observations thus confirm the existence of both KU-dependent and XRCC1-dependent pathways in plants [205]. Besides XRCC1, PARP is also involved in A-NHEJ. In contrast to humans where PARP1 is the major contributor to the PARP activity in response to DNA damage, PARP2 is the primary enzyme responsible for poly(ADP-ribosyl)ation under genotoxic stress in A. thaliana [203]. Plants impaired in both PARP1 and PARP2 displayed a strong reduction in root length and fresh weight after treatment with MMS [206]. However, growth of parp1 and parp2 single mutants was not more sensitive to MMS than wild-type seedlings [206]. Analysis of in vitro endjoining assays indicated that the parp1/parp2 double mutant plant had twofold less end-joining products compared with the wild-type, while the ku70 and ku80 single mutant lines had almost fourfold more products than the wild-type [206]. These observations suggest a regulation of non-homologous recombination pathways. Studies in rice also reported that ku70, ku80, lig4 or xrcc3 mutants showed hypersensitivity to genotoxic agents and/or altered DSB repair [207,208]. On the other hand, overexpression of OsRecQI4 (the AtRecQ4A counterpart in rice), OsExo1 and MtTpd2a enhanced DSB processing in rice and Medicago truncatula, respectively [209,210]. No apparent DNA-PK, XLF, DNA ligase III or DNA polymerase b homologs exist in plants [24,204,205,211].


## Concluding remarks

Significant progress in plant DNA repair has been made in recent years. Sequence homology-based analysis allowed identification of many plant factors, not only in model organisms but also in some crop species. Most of them were also functionally validated. Thus, it is clear that DNA repair pathways are well conserved between plants and mammals. However, some activities have been particularly intensified in plants, probably related to a better adaptation to variable environmental conditions, from which plants cannot escape, and/or to maintain genome stability over multiple generations. Although only limited information on DNA repair mechanisms from crop plants is available, the current advances in A. thaliana can well be translated to gain insights into the challenges of obtaining new varieties with novel traits. protein 1A and regulate the response to UV irradiation. Plant 

## ( 4 )
4Gap-filling and nick sealing. The short-patch or single nucleotide (SN) BER involves the incorporation of only a single nucleotide by the DNA polymerase b. The long-patch (LP) BER requires DNA synthesis mediated by DNA polymerases d and e and assisted by proliferating cell nuclear antigen (PCNA). The newly synthesized DNA displaces the downstream 5 0 DNA end to form a flap structure containing 2-13 nucleotides. The displaced strand is then processed by a flap endonuclease (FEN). The choice of the pathway depends on the specificity of the DNA glycosylase and the proliferation status of the cell. Recent findings suggest that DNA polymerase k can substitute for DNA polymerase b


Khateeb W, Schroeder D (2009) Overexpression of Arabidopsis damaged DNA binding protein 1A (DDB1A) enhances UV tolerance. Plant Mol Biol 70:371-383 79. Ganpudi A, Schroeder D (2013) Genetic interactions of Arabidopsis thaliana damaged DNA binding protein 1B (DDB1B) with DDB1A, DET1, and COP1. G3 (Bethesda) 3:493-503 80. Koga A, Ishibashi T, Kimura S, Uchiyama Y, Sakaguchi K (2006) Characterization of T-DNA insertion mutants and RNAi silenced plants of Arabidopsis thaliana UV-damaged DNA binding protein 2 (AtUV-DDB2). Plant Mol Biol 61:227-240 81. Ly V, Hatherell A, Kim E, Chan A, Belmonte M, Schroeder D (2013) Interactions between Arabidopsis DNA repair genes UVH6, DDB1A, and DDB2 during abiotic stress tolerance and floral development. Plant Sci 213:88-97 82. Shaked H, Avivi-Ragolsky N, Levy A (2006) Involvement of the Arabidopsis SWI2/SNF2 chromatin remodeling gene family in DNA damage response and recombination. Genetics 173:985-994 83. Liang L, Flury S, Kalck V, Hohn B, Molinier J (2006) CEN-TRIN2 interacts with the Arabidopsis homolog of the human XPC protein (AtRAD4) and contributes to efficient synthesisdependent repair of bulky DNA lesions. Plant Mol Biol 61:345-356 84. Vonarx E, Tabone E, Osmond M, Anderson H, Kunz B (2006) Arabidopsis homologue of human transcription factor IIH/nucleotide excision repair factor p44 can function intranscription and DNA repair and interacts with AtXPD. Plant J 46:512-521 85. Gentile A, Ditt R, Dias F, Da Silva M, Dornelas M, Menossi M (2009) Characterization of ScMat1, a putative TFIIH subunit from sugarcane. Plant Cell Rep 28:663-672 86. Gutierrez C (2009) The Arabidopsis cell division cycle. Arabidopsis Book 7:e0120 87. Van Leene J, Hollunder J, Eeckhout D, Persiau G, Van De Slijke E, Stals H, Van Isterdael G, Verkest A, Neirynck S, Buffel Y, De Bodt S, Maere S, Laukens K, Pharazyn A, Ferreira P, Eloy N, Renne C, Meyer C, Faure J, Steinbrenner J, Beynon J, Larkin J, Van de Peer Y, Hilson P, Kuiper M, De Veylder L, Van Onckelen H, Inzé D, Witters E, De Jaeger G (2010) Targeted interactomics reveals a complex core cell cycle machinery in Arabidopsis thaliana. Mol Syst Biol 6:397 88. Eschbach V, Kobbe D (2014) Different replication protein A complexes of Arabidopsis thaliana have different DNA-binding properties as a function of heterotrimer composition. Plant Cell Physiol 55:1460-1472 89. Aklilu B, Soderquist R, Culligan K (2014) Genetic analysis of the Replication Protein A large subunit family in Arabidopsis reveals unique and overlapping roles in DNA repair, meiosis and DNA replication. Nucleic Acids Res 42:3104-3108 90. Aklilu B, Culligan K (2016) Molecular evolution and functional diversification of Replication Protein A1 in plants. Front Plant Sci 7:33 91. Peña-Diaz J, Jiricny J (2012) Mammalian mismatch repair: error-free or error-prone? Trends Biochem Sci 37:206-214 92. Kunkel T, Erie D (2015) Eukaryotic mismatch repair in relation to DNA replication. Annu Rev Genet 49:291-313

## Table 1
1Overview of steps (A) and A. thaliana required factors (B) involved in BERA 


## Table 2
2Components involved in DNA metabolic pathways in A. thalianaGene 
EXO1 
PCNA 
POL d 
POL e 
RFC 
LIG1 

AGI locus code 
At1g29630 
At1g07370 
At2g29570 

At1g09815 

At2g42120 

At5g63960 

At1g08260 

At2g27120 

At5g22110 

At1g21690 

At1g63160 

At1g77470 

At5g22010 

At5g27740 

At1g08130 

At1g49250 


## Table 4
4iv) Completion of DNA synthesis 
PCNA 
See Table 2 
followed by ligation 
RFC 
See Table 2 
Pol δ 
See Table 2 
Pol ε 
See Table 2 
LIG1 



. A difference in sensitivity upon exposure of UV radiation was also observed for plants defective in DDB2, UVH6 (defective in the human XPD homolog) or CSB homologs (CHR8 and CHR24) [76, 77, 80-82]. It should be mentioned that some plant homologs of the human XP genes were initially named after the UV hypersensitive (uvh) phenotype of the mutants. On the other hand, overexpression of DDB1A conferred increased resistance to UV-induced DNA damage[78].

## Table 4
4RPA proteins in A. thalianaGene 
RPA1 
RPA2 
RPA3 

AGI locus code At2g06510 (A) 

At5g08020 (B) 

At5g45400 (C) 

At5g61000 (D) 

At4g19130 (E) 

At2g24490 (A) 

At3g02920 (B) 

At3g52630 (A) 

At4g18590 (B) 


## Table 5
5Overview of steps (A) and A. thaliana required factors (B) involved in MMR 

A 

Step 

i) Lesion 

ii) DNA r 

iii) 

iv) 

B 

Schematic 
representation 
Gene 
AGI locus 
code a,b 

recognition 
MSH2 
At3g18524 
MSH3 
At4g25540 
MSH6 
At4g02070 
MSH7 
At3g24495 
epair complex assembly 
MLH1 
At4g09140 

PMS2 
At4g02460 
Strand discrimination 
RFC 
See Table 2 
PCNA 
See Table 2 
Strand excision and resynthesis 
EXO1 
See Table 2 
RPA 
See 

## Table 6## Introduction

All organisms are constantly exposed to environmental stresses as well as genotoxic products from endogenous metabolic processes that induce, either directly or indirectly, DNA damage [1]. Another source of DNA damage includes base-base mismatches produced by the incorporation of a non-complementary Watson-Crick base and unpaired nucleotides caused by misalignment of the two DNA strands, both generated during DNA synthesis. These DNA biosynthetic errors differ from typical DNA damage in that they involve undamaged bases and exist only transiently provided that the DNA remains double-stranded [2]. To counteract these genome integrity threats, all organisms have evolved the DNA damage response (DDR) [3,4]. The DDR is a pathway that transduces the DNA damage signal into activation of various pathways that leads to DNA repair, cell cycle checkpoint and programmed cell death. Plant DDR also includes endoreduplication. Endoreduplication involves replication of the nuclear DNA without cytokinesis [5]. Here, I only focus on DNA repair mechanisms in mammals and plants. Plants are distinctly different from mammals in that they lack a reserved germ line and are sessile. Gametes arise from meristem cells that & Claudia P. Spampinato spampinato@cefobi-conicet.gov.ar have been exposed to the continuous effect of environmental and endogenous mutagens and have divided many times. Thus, plants have evolved particularly efficient DNA repair mechanisms. Advancements in this field have been provided by the Arabidopsis genome sequence [6]. Plants have orthologs of most of the genes involved in mammalian DNA repair pathways [7][8][9]. However, several interesting features may be noted: the presence of unique genes, the presence of multiple gene copies and the absence of well-characterized genes in plant genomes [9,10]. Such differences suggest plant-specific DNA repair pathways. Provided here is a review of major findings over the last decade related to mammal and plant DNA repair mechanisms, with a special focus on DNA repair genes from A. thaliana.


## Photoreactivation

Photoreactivation is the direct reversal of major lesions induced by ultraviolet (UV) light on two adjacent pyrimidine bases, such as cyclobutane pyrimidine dimers (CPDs) and pyrimidine (6-4) pyrimidones (6-4 photoproducts or 6-4 PPs). The history of the discovery of this repair system has been recently reviewed [11]. Photoreactivation relies on single light-activated flavoenzymes called photolyases. Photolyases belong to the photolyase/cryptochrome family and have been classified into three related subfamilies known as class I CPD photolyase, class II CPD photolyase, and 6-4 photolyase, based on their sequence homology and substrate specificity [12][13][14]. Multiple sequence alignment analysis of members from the three subfamilies indicates a highly functional conservation of a photolyase-homologous region (PHR) domain that binds the chromophore FAD (flavin adenine dinucleotide) [15,16]. Class I CPD photolyases have been reported in diverse microbial organisms such as archaea, eubacteria, and fungi; class II CPD photolyases have been found not only in archaea, eubacteria, some viruses, but also in higher organisms such as animals and plants; and 6-4 photolyases have been identified in cyanobacteria and certain eukarya [15,17]. Humans and other placental animals have lost photolyase activities. All photolyases use the energy of visible or blue light (k = 320-500 nm) for the catalytic cleavage of the photoproducts and contain the fully reduced FADHas the catalytically active cofactor [13,14]. All photolyases characterized to date also contain a second chromophore, such as methenyltetrahydrofolate found in the majority of the enzymes or 7,8-didemethyl-8-hydroxy-5-deazariboflavin, FAD or FMN (flavin mononucleotide) in others that functions as a photoantenna to enhance light absorption [14,[18][19][20]. Upon blue light absorption, the excitation energy is efficiently transferred from the photoantenna to FADHby Förster resonance energy. The resulting FADH -* donates an electron to the DNA lesion with the generation of two canonical pyrimidines [12,14,[19][20][21].


## Photoreactivation in plants

Arabidopsis thaliana contains two specific class II CPD photolyases [PHR1, also named UVR2 (At1g12370) and PHR2 (At2g47590)] and one 6-4 photolyase [UVR3 (At3g15620)]. The FAD-binding domains of plant PHR2 proteins are smaller and less conserved than in the other subfamilies [15]. Several articles provide a summary of the molecular cloning, the tissue-specific and light-dependent regulation and the characterization of mutants of CPD photolyase and 6-4 photolyase genes from various plant species (e.g., A. thaliana, Oriza sativa, Sinapis alba) [22][23][24]. Recent evidence suggests that AtPHR1 expression induced by UV-B is primarily regulated by the UVR8 (UV RESISTANCE LOCUS8)-dependent pathway but is also mediated by a UVR8-independent pathway [25]. The latter one was found to be correlated with UV-B-induced CPD levels [25].

The crystal structure of a UVR2 homolog has recently been determined in rice and compared with prokaryotic class I CPD photolyases [26]. Results indicate that both enzymes recognize CPD damage in similar active sites but create a different conformational distortion in the DNA duplex [26]. The crystallographic structure and molecular mechanism of A. thaliana 6-4 photolyase have also been determined [27,28]. After light excitation, there is initially a forward electron transfer followed by a cyclic proton transfer involving an active-site histidine residue [28]. This proton transfer is the determinant step in the repair and determines the overall quantum yield (A = number of dimers repaired per number of photons absorbed) [28]. For comparison, the reported A for CPD and 6-4PP repair are 0.82 and 0.1, respectively, and the whole catalysis reaction takes less than a nanosecond for CPD repair or tens of nanoseconds for 6-4PP repair [29]. The complete photocycle in real-time from the initial 6-4 PP to several intermediates and subsequent conversion to two thymine bases catalyzed by the A. thaliana (6-4) photolyase has been recently well reviewed [21,29].


## Base excision repair

Base excision repair (BER) recognizes and repairs lesions such as deaminated, oxidized and alkylated bases, abasic (apurinic and/or apyrimidinic, AP) sites and single-strand breaks (SSBs). BER occurs in several steps (Table 1A; see references [30][31][32] for recent reviews in mammalian BER).

(1) Lesion recognition and removal by DNA glycosylases. DNA glycosylases hydrolyze the N-glycosidic bond of the damaged base, leaving an AP site. Different DNA glycosylases exist, each one with broad or narrow substrate specificities. Enzymes can be classified as monofunctional or bifunctional. Monofunctional glycosylases perform base excision only, whereas bifunctional glycosylases also have lyase activity. Biochemical mechanisms of DNA glycosylases have been the subject of recent reviews [33,34]. (2) Cleavage of the sugar-phosphate backbone at the AP site. Cleavage is catalyzed either by an AP endonuclease or by the intrinsic AP lyase activity of a bifunctional DNA glycosylase. AP endonucleases and lyases generate different types of DNA termini. AP endonucleases release a 3 0 OH and a 5 0 deoxyribose-phosphate moiety (5 0 dRP), while AP lyases release a 3 0 phospho a,b-unsaturated aldehyde (3 0 PUA) or 3 0 phosphate by bor bd-elimination reactions, respectively, and a 5 0 phosphate. (3) Processing of unconventional ends to conventional 3 0 OH and 5 0 phosphate moieties. This step involves different enzymes depending on whether the modified terminus is processed from the 5 0 or 3 0 end. The intrinsic dRP lyase activity of DNA polymerase b removes the 5 0 dRP; the 3 0 phosphodiesterase activity of the AP endonuclease eliminates the 3 0 PUA, whereas polynucleotide kinase 3 0 phosphatase (PNKP) processes the 3 0 phosphate moiety. Additional SSB end cleaning enzymes are aprataxin and tyrosyl DNA phosphodiesterases (TDP) that process abortive ligation or topoisomerase reactions, respectively [35].  Step Schematic representation Gene a AGI locus code b,c


## i) Lesion recognition and removal OGG1

At1g21710 
FPG At1g52500 NTH1 At2g31450 NTH2 At1g05900 DME At5g04560 ROS1 At2g36490 UNG At3g18630 DML3 At4g34060 MBD4L At3g07930 ii) Cleavage

## PARP2 At4g02390

Pol δ See Table 2 Pol ε See Table 2  LIG1 See Table 2 OH P a Genes encoding bifunctional glycosylases are listed first, followed by monofunctional glycosylases b AGI Arabidopsis genome initiative c AGI locus code for components involved in DNA metabolic pathways is shown in Table 2 d Bifunctional DNA glycosylases with associated b-elimination activity are not detailed Protecting DNA from errors and damage 1695 in filling single-nucleotide gaps and can perform gap filling in long-patch BER [36]. Poly ADP-ribose polymerase 1 (PARP1) and X-ray cross-complementation group 1 (XRCC1) also have roles in BER. PARP1 protects SSBs from the formation of more deleterious double-strand breaks (DSBs) [35] and XRCC1 is involved in the organization of BER-proficient multiprotein complexes [37]. Finally, the nick is sealed by a DNA ligase.


## BER in plants

Most of the BER proteins found in animal cells have been described in plants [38,39]. Tables 1B and 2 show an overview of described factors involved in BER mechanism in A. thaliana. Plants contain several DNA glycosylases: 8-oxoguanine DNA glycosylase (OGG1), formamidopyrimidine DNA glycosylase (FPG), NTH, and uracil DNA glycosylase (UNG) [38][39][40][41][42][43][44]. The enzymes OGG1 and FPG recognize oxidized purines; NTH, oxidized pyrimidines and UNG, U:G mismatches. Both NTH homologs (NTH1 and NTH2) were co-localized to nucleoids within Arabidopsis chloroplasts, thus indicating the existence of a BER pathway in these organelles to deal with photo-oxidative stress [44]. Plant genomes, in contrast to mammals, also encode specific 5-methyl-cytosine (5-meC) glycosylases that are involved in DNA demethylation [DEMETER (DME) and Repressor of Silencing 1 (ROS1)] [45][46][47][48]. These proteins show preference for 5-meC as substrate but can also process T:G mispairs. It has been reported that ROS1 has a very low turnover [46], with the recognition of pyrimidine modifications being the rate-limiting step [49]. An additional activity against T:G mispairs is displayed by MBD4L (methyl binding 4 DNA glycosylase-like) [50]. This enzyme was demonstrated to act on both U:G and T:G mispairs and to excise both U and T more efficiently at a CpG context. Roles of OGG1, ROS1 and MBDL4 in Arabidopsis tolerance to oxidative DNA damage have also been described using mutant or overexpression plants [51][52][53]. Other downstream proteins of the BER pathway have been characterized in plants. The Arabidopsis genome contains three genes encoding AP endonuclease-like proteins: APE1L (abasic endonuclease-1-like), APE2 (abasic endonuclease 2) and ARP (abasic endonuclease-redox protein) [54]. All three enzymes were found to exhibit AP endonuclease activity in vitro [55], with ARP representing the major activity in Arabidopsis cell extracts [56]. Based on the available DNA sequence data, a wheat homolog of AtAPE1L has been cloned and characterized [57]. Orthologues of human PNKP (named AtZDP), TDP1, XRCC1 and DNA ligase I were also functionally characterized in Arabidopsis [42,55,56,[58][59][60][61]. Finally, it has been demonstrated that AP sites may be processed in plants through both SN-and LP-BER [41] and that DNA ligase I restores the continuity of the repaired DNA strand during both SN-and LP-BER [56]. Recent studies identified and characterized a gene named SAV6 (shade avoidance mutant), a homolog of human FEN1 [62]. Compared to the animal FEN1, SAV6 shows both flap and gap endonuclease activities, but lacks exonuclease activity.


## Nucleotide excision repair

Nucleotide excision repair (NER) is responsible for the processing of bulky helix-distorting damage, such as CPDs and 6-4PPs induced by UV irradiation. Deficiencies in NER are associated with several human autosomal recessive disorders, namely xeroderma pigmentosum (XP), cockayne syndrome (CS) and trichothiodistrophy (TTD) among others. XP, CS and TTD have provided the names of some of the genes involved in the pathway (XPA through XPG, XPV, CSA, CSB and TTDA). NER is a multistep process which comprises four steps (Table 3A).

(1) Damage recognition. Two different modes of damage recognition coexist: global genome NER (GG-NER) and transcription-coupled NER (TC-NER). GG-NER detects damage occurring throughout the genome and is specifically initiated by a heterotrimeric XPC-HR23B-CEN2 complex (XPC and HR23A/HR23B are the human homologs of yeast and plant RAD4 and RAD23 proteins, respectively), with the assistance in some cases of DDB (damaged DNA-binding) protein complex. DDB, a heterodimeric complex comprising DDB1 and DDB2 subunits, specifically binds CPDs and 6-4PPs with moderate or high affinity, respectively, and stimulates significantly XPC binding to UV-damaged lesions. GG- NER in mammalian cells has been covered in recent reviews [63,64]. TC-NER is responsible for the recognition of lesions in the transcribed strand of active genes and is initiated by RNA polymerase stalled at a lesion, with the help of specific factors CSA, CSB, and XPA binding protein 2 (XAB2) whose function is not clear. Many aspects of mammalian TC-NER can be found in some recent reviews [64][65][66].

(2) Formation of a stable preincision complex around the damage site. After DNA recognition, GG-NER and TC-NER converge into the same pathway. The transcription factor II H (TFIIH) is recruited to the lesion. TFIIH is a multiprotein complex composed of ten subunits arranged in two functional subcomplexes: the core subcomplex containing the DNA helicase XPB, p62, p52, p44, p34 and p8 (TTDA) and the CAK [cyclin dependent kinase (CDK)-activating kinase] subcomplex containing CDK7, cyclin H (CYCH) and the assembly factor ménage-à-trois-1 (MAT1); both subcomplexes are bridged by the DNA helicase XPD [67,68]. The result of the TFIIH activity is the partial unwinding of the DNA duplex that leads to the recruitment of XPA, RPA (replication protein A), and the endonuclease XPG. Then, XPA interacts and recruits the second endonuclease, ERCC1 (excision repair cross complementing 1)-XPF. (3) Excision of the damaged nucleotide. The damaged nucleotide is removed by dual incisions 5 0 and 3 0 to the lesion by endonucleases XPF and XPG, respectively, generating a 24-32 oligonucleotide single-strand fragment containing the damaged site. There Table 3 Overview of steps (A) and A. thaliana required factors (B) involved in NER


## A B

Step Schematic representation Gene AGI locus See Table 2 a AGI Arabidopsis genome initiative b AGI locus code for components involved in DNA metabolic pathways is shown in Tables 2 and 4 Protecting DNA from errors and damage 1697 is a defined order of the two incisions. The 5 0 incision by the heterodimer ERCC1-XPF is performed first; repair synthesis is initiated followed by the 3 0 incision by XPG [69]. (4) Completion of DNA synthesis followed by ligation. The repair synthesis is carried out by DNA polymerases d, e, or j depending on chromatin accessibility of the damaged site [70]. NER is completed by nick sealing by DNA ligase I or IIIa.
code a,b i) Damage recognition RAD4 At5g16630 RAD23A At1g16190 RAD23B At1g79650 RAD23C At3g02540 RAD23D At5g38470 CEN2 At4g37010 DDB1A At4g05420 DDB1B At4g21100 DDB2 At5g58760 CSA At1g27840 At1g19750 CHR8 At2g18760 CHR24 At5g63950 ii) Formation of a stable preincision XPB1 At5g41370 complex around the damage site XPB2 At5g41360 UVH6 At1g03190 TFIIH1 At1g55750 At1g61420 GTF2H2 At1g05055 TFIIH3 At1g18340 TFIIH4 At4g17020 TTDA At1g12400 At1g62886 CDKD;1 At1g73690 CDKD;2 At1g66750 CDKD;3 At1g18040 CYCH;1 At5g27620 MAT1 At4g30820 iii) Excision of the damaged nucleotide UVH3 At3g28030 UVH1 At5g41150 ERCC1 At3g05210 RPA See

## NER in plants

This pathway has been primarily studied in Arabidopsis and rice [7]. In Arabidopsis, four genes, namely RAD23, DDB1, CSA and XPB have been duplicated (Table 3B) [7,71]. Interestingly, RAD23 was also found to be duplicated in rice, Populus and Sorghum and many times in Arabidopsis [7,71]. Although Arabidopsis mutants affecting the expression of individual RAD23 genes exhibited pleiotropic developmental defects (rad23b) or no obvious phenotype (rad23a, rad23c, and rad23d), higherorder mutant combinations indicated that the RAD23 family is essential in Arabidopsis [72]. The two DDB1 homologs, DDB1A and DDB1B, were expressed throughout plant development with DDB1B showing the lowest levels [73,74] but playing a crucial role because ddb1b null alleles appear lethal ( [73] and references therein).

Overlapping expression patterns were also observed for the two CSA homologs. Both homologs share significant sequence identity and likely form heterotetramers in plants [75]. Decreased levels of several of these genes, namely DDB1A, DDB1B, CSA or XPB1 affected UV plant tolerance [75][76][77][78][79] The interaction of XPC-CEN2 (AtRAD4-AtCEN2) and validation of some in silico predicted TFIIH subunit transcripts were also reported in plants [7,[83][84][85][86][87]. Human TFIIH components p62, p44, p34 and p52 correspond to TFIIH1, TFIIH2 (AtGTF2H2), TFIIH3 and TFIIH4 in A. thaliana, while CDK7 is encoded by three CDKDs (Table 3B). Homologs of the human RPA complex, heterotrimer composed of three associated subunits RPA1, RPA2 and RPA3, were also functionally characterized in Arabidopsis [88]. However, in contrast to other eukaryotes, multiple genes encode the RPA subunits in A. thaliana (Table 4) [88,89]. A phylogenetic tree based on RPA1 protein sequences revealed three evolutionary groups [90]. Group A is composed of RPA1A; group B, of RPA1B and RPA1D and group C, of RPA1C and RPA1E proteins. These authors propose that group C includes proteins involved in DNA damage repair [90]. No apparent XPA homolog exists in plants [10].


## Mismatch repair

The mismatch repair (MMR) system is best known for its role in the recognition and correction of single base-base mismatches and unpaired nucleotides that arise through replication errors, deamination of 5-methylcytosine, and recombination between divergent sequences. Eukaryotic MMR has been studied in great detail and has been covered in several recent reviews [2,[91][92][93][94]. Also, an overview of the history of eukaryotic MMR has been recently reported [95]. The pathway involves several steps (Table 5A). (1) Lesion recognition by MutS proteins. In eukaryotes, MutS proteins function as heterodimers composed of related, but distinct MutS homologs (MSH) subunits. To date, eight MSH subunits were discovered [96], of which MSH2, MSH3 and MSH6 are involved in nuclear MMR in human cells. These MSHs subunits assemble as MSH2-MSH6 (MutSa) and MSH2-MSH3 (MutSb). MutSa recognizes base-base mismatches and short insertion/deletion loops (IDLs) while MutSb mediates the repair of IDLs up to 16 nucleotides [97]. The molecular basis of substrate specificities has been investigated. A series of crystal structures of human MutSa in complex with different DNA substrates showed an extensive interaction of a conserved phenylalanine residue in the MSH6 subunit with the DNA mispair [98]. MSH3 lacks the phenylalanine residue that makes protein-mispair contact. Instead, MutSb binds the sugar-phosphate backbone of the IDL [99]. DNA is severely bent and the unpaired bases are flipped out and solvent exposed as revealed by the crystal structure of human MutSb complexed with DNA containing IDLs of varying size [99]. In addition to crystal reports, abilities of MutSa and MutSb to process base-base mismatches and IDLs were explored. The results indicate that MutSa and MutSb differ in DNA substrate recognition and that the MutSb-IDL interaction is greatly stimulated by an excess . The presence of these linkers has been associated with large asymmetric conformational changes following sequential ATP binding and hydrolysis [107]. (3) Strand discrimination. Identification and discrimination of the newly synthesized DNA strand in eukaryotes are attributed to strand breaks involved in the leading-and lagging-strand synthesis. Replication factor C (RFC) specifically recognizes 3 0 termini at the replication fork and loads PCNA asymmetrically onto these sites, with the same side facing towards the strand discontinuity [108]. PCNA loaded at these breaks is required for MutLa endonuclease activation, apparently through protein-protein interaction [109]. Once activated, MutLa incises the strand containing the preexisting strand break due to the loading polarity of PCNA [109]. MutLa endonuclease can also be activated on DNA that contains a MutSb recognizable lesion but, while MutSa can interact independently with MutLa and PCNA, interactions of MutSb with MutLa and PCNA were found to be mutually exclusive [102]. These results suggest important differences in the repair mechanisms of single base-base mismatches and IDLs. (4) Strand excision and re-synthesis. Multiply incised molecules are substrates for the 5 0 -3 0 activity of MutSaactivated exonuclease I (Exo1) [110]. Once the mismatch is excised, MutSa and RPA regulate Exo1 activity that leads to excision termination [111]. The repair reaction is completed by a correct DNA re-synthesis by DNA polymerase d, followed by ligation. In the absence of Exo1, MMR can still be detected suggesting the existence of one or more Exo1-independent modes of mismatch repair. The mechanism most likely involves a strand displacement synthesis by DNA polymerases d that is strongly stimulated by RPA  Table 4 POL δ See Table 2 a AGI Arabidopsis genome initiative b AGI locus code for components involved in DNA metabolic pathways is shown in Tables 2 and 4 Protecting DNA from errors and damage 1699 [112]. Exo1-dependent and independent MMR has been the subject of a recent review [113].


## MMR in plants

Arabidopsis and other plants encode homologs of MutS and MutL proteins found in other eukaryotic organisms, but also contain a unique MSH polypeptide (MSH7) [114,115] (Table 5B). MSH7 forms a heterodimer with MSH2 and the protein complex is designated MutSc. Quantitative analysis of transcript levels of representative genes (MSH2 and PMS1) of the pathway indicated that these genes show a higher expression in calli than in seedlings from A. thaliana, thus confirming the essential genome maintenance function performed by MMR system in rapidly dividing tissues [116,117]. Further knowledge of   Table 2 iv) Resolution of the D-loop
RAD50 At2g31970 NBS1 At3g02680 COM1 At3g52115 RECQ4A At1g10930 RAD51 At5g20850 RAD51B At2g28560 RAD51C At2g45280 RAD51D At1g07745 XRCC2 At5g64520 XRCC3 At5g54750 BRCA2A At4g00020 BRCA2B At5g01630 RAD54At3g19210SDSA SRS2 At4g25120 FANCM At1g35530 DSBR EME1A At2g21800 EME1B At2g22140 MUS81 At4g30870 GEN1 At1g01880 SEND1 At3g48900 HJ dissolution RECQ4A At1g10930
TOP3α At5g63920


## RMI1 At5g63540

a AGI Arabidopsis genome initiative b AGI locus code for components involved in DNA metabolic pathways is shown in Table 2 the role of a given plant MMR protein came from in vivo studies. Using a yeast system, we have found that AtMutSc is able to specifically recognize G/T, A/C, T/C, G/A, T/T, or A/A mismatches and/or neighboring sequences containing T [118] and that the expression of AtMutLa affected the repair of IDLs in microsatellite sequences [119]. The functionality of plant MutS and MutL homologs was also assessed using A. thaliana (reviewed in [114]), tomato [120], rice [121] and Nicotiana [122] mutant plants.

Suppression of MSH2, PMS1 or MLH1 was associated with microsatellite instability, an increase in homeologous recombination relative to wild-type plants and a broad spectrum of mutant phenotypes in their progenies. Inactivation of MSH7 led a modest increase in the frequency of recombination between Solanum lycopersicum and a distantly related species (Solanum lycopersicoides) [120], while in A. thaliana, the inactivation of MSH7 did not affect the rates of somatic homologous or homeologous recombination, but influenced the rates of meiotic recombination [123]. Finally, somewhat comparable to mammalian proteins, we demonstrated that msh2 mutant plants showed an altered expression pattern of cell cycle marker genes after induced DNA damage in comparison to wild-type plants and we experimentally validated MSH6 as an E2F target gene [124].


## Double-strand break repair

Double-strand break (DSB) repair uses two main strategies: a homology-dependent, error-free homologous recombination repair (HR); a potentially error-prone, classical, Kudependent non-homologous end-joining (C-NHEJ) and a recently identified error-prone repair pathway, named alternative non-homologous end-joining pathway (A-NHEJ). The choice between these pathways depends on the phase of the cell cycle and the initiation of DNA end resection [125][126][127].


## Homologous recombination

HR is favored during the S/G2 phase of the cell cycle, when sister chromatids are available as homologous templates [128,129]. This mechanism was the subject of several recent reviews [130][131][132]. In this pathway, four steps are required (Table 6A). (1) DNA end resection and nucleoprotein filament formation. Initial processing of the ends involves the trimeric complex MRE11-RAD50-NBS1 (MRN) and the CtBP (C-terminal binding protein) interacting protein (CtIP) [133,134]. Further resection involves the 5 0 -3 0 exonuclease EXO1 or the combined helicase/ nuclease activities of the BLM/DNA2 (Bloom Syndrome RecQ Like Helicase/DNA replication helicase/nuclease2) [135]. The 3 0 single-strand DNA (ssDNA) tails generated are initially bound by ssDNA-binding protein RPA [136]. Subsequently, RPA is replaced by another ssDNA-binding protein, RAD51, which forms a nucleoprotein filament on the DNA. The replacement of RPA by RAD51 requires several mediator proteins, such as BRCA2 (protein encoded by the breast cancer-associated gene 2) and RAD54 [137][138][139]. Human cells have two RAD54 homologs, RAD54 and RAD54B, which display similar biochemical activities [140]. Evidence also suggests that RAD51 paralogs, such as XRCC2, XRCC3, RAD51B, RAD51C, and RAD51D, function in promoting and/or stabilizing RAD51 nucleofilaments [131,141]. However, unlike RAD51 that self-assembles, RAD51 paralogs form at least four different heterodimeric and heterotetrameric complexes. (2) DNA homology search and strand invasion. The RAD51 nucleoprotein filament begins searching for a homologous DNA sequence and then invades the intact double-stranded DNA molecule to form a heteroduplex DNA structure termed the displacement loop (D-loop). D-loop formation is stimulated by RAD54 [131,132]. Besides ssDNA binding, RAD51 also interacts with dsDNA and associates with the newly formed heteroduplex DNA. (3) DNA heteroduplex extension. The invading strand in the D-loop structure is then extended by several components of the replication machinery, namely DNA polymerase d, PCNA and RFC [142]. (4) Processing of the D-loop. HR can take several different steps: synthesis-dependent strand annealing (SDSA), Holliday junction (HJ) resolution (the DSBR model) and HJ dissolution [130,132]. SDSA involves the displacement of the newly synthesized strand by DNA helicase(s), followed by strand annealing, DNA synthesis and ligation. DSBR is characterized by the capture of the second end of the DSB, formation of a double HJ (dHJ) structure and resolution by specialized resolvases (e.g., GEN1, MUS81-EME1, SLX1-SLX4). Alternatively, the dHJ can be dissolved by the BTB complex consisting of BLM helicase, DNA topoisomerase IIIa and BLAP75. Regulation of HJ processing enzymes has been covered in recent reviews [143,144]. Depending on the pathway employed for HJ processing, different recombination outcomes are generated. SDSA and HJ dissolution lead to the formation of non-crossover while HJ resolution contributes to the formation of non-crossover or crossover. The formation of non-crossover recombinants is promoted in mitotic cells, in contrast with that occurring in meiotic cells.

An alternative process that involves recombination between regions of homology at both sides of the break can also occur. This process, known as single-strand annealing (SSA), requires HR proteins involved in DNA end resection and annealing and also RPA, but is independent of RAD51 [130][131][132]145]. After DNA end resection, the 3 0 ssDNA tails generated can reveal complementary sequences on both sides of the break. These sequences anneal forming a branched structure. The 3 0 tails are then removed and the nicks are ligated. This process inevitably results in a deletion at the repair junction.

Classical non-homologous end-joining C-NHEJ functions throughout the cell cycle but is dominant in G1 and G2 [129]. Several recent reviews have focused on C-NHEJ [146][147][148][149][150]. Three different steps are involved in this pathway. (1) DNA end recognition and tethering. The initial step is the recognition and binding of DSBs by the Ku protein (heterodimer of KU70-KU80 in eukaryotes) in a sequence-independent manner. Recent evidence suggests that only two Ku molecules are present at a DSB in vivo, presumably one at each end of the DSB [151]. The Ku-DNA complex then interacts with the DNAdependent protein kinase (DNA-PK) catalytic subunit (DNA-PKcs) forming a very stable and functional DNA-PK complex. This complex remains tethered to the end and prevents end access. The DNA-PK activated protein undergoes extensive autophosphorylation in vivo [152]. This results in a conformational change that opens the central DNA-binding cavity and releases the autophosphorylated DNA-PKcs from DNA ends [152]. Structures and functions of the Ku protein and the DNA-PK have been recently reviewed [153][154][155]. (2) DNA end processing. Depending on the nature of the DNA damage, one or more enzyme activities are involved in DNA end processing. For example, the 5 0 -dRP/AP lyase activity of the Ku protein excises abasic or 5 0 -dRP/AP sites [156,157], the tyrosyl DNA phosphodiesterase activity of TDP2 hydrolyzes 5 0 phosphotyrosyl-DNA bonds [158], and the 3 0 -DNA phosphatase and 5 0 -DNA kinase activities of the polynucleotide kinase/phosphatase (PNKP) generate ligatable 5 0 -phosphates and 3 0 -hydroxyl ends [159,160], among others. In many cases, recruitment of these proteins depends on their direct interaction with Ku [154]. (3) Ligation of the DSB. The final step is carried out by DNA ligase IV, which functions in complex with its cofactors XRCC4 and XLF (XRCC4 like factor, also known as Cernunnos) [150]. Recently, a new protein named PAXX (PAralog of XRCC4 and XLF) has been reported to promote Ku-dependent DNA ligation in vitro [161].


## Alternative non-homologous end-joining pathway

A-NHEJ increases as cells enter S-phase [162]. The molecular mechanism of A-NHEJ has recently been reviewed [163][164][165]. A-NHEJ occurs in the absence of key C-NHEJ factors and can involve alignment of microhomologous sequences; thus, this pathway is also known as microhomology-mediated end-joining (MMEJ). A-NHEJ shares with C-NHEJ similar reaction steps. (1) DNA end recognition and tethering. DSBs are efficiently bound by PARP1 under Ku defective conditions [166]. In fact Ku and PARP1 compete for DNA end binding, and Ku also limits poly ADP-ribose (PAR) synthesis and ssDNA production in response to DSBs [166,167]. Upon DSB binding, PARP1 gets activated and subsequently autopoly(ADP-ribosyl)ated. Auto-modification of PARP1 results in a decreased affinity for nucleosomes and in a relaxed local chromatin structure, which facilitates chromatin remodeling for DNA repair [168,169]. Overviews of the involvement of PARP1 in DNA repair have been published [170][171][172]. (2) DNA end processing. PARP1mediated PAR synthesis then triggers recruitment of the MRN complex to initiate incompatible end resection [173,174]. (3) Ligation of DNA ends. MRN interacts with the DNA ligase IIIa/XRCC1 complex and stimulates the joining of DNA ends [175]. Joining junctions involve microhomologies and deletions [175]. A second pathway independent of microhomologies that relies on DNA ligase I has also been suggested [176].


## DSB repair in plants

Knowledge of recombination mechanisms in plants has significantly advanced in recent years, especially because DSB repair emerged as an important tool to achieve controlled modifications of plant genomes (for recent reviews see [177][178][179][180][181][182][183]). Most of the HR proteins have been identified and characterized in A. thaliana (Table 6B). Plants with decreased transcript levels of genes involved in the initial processing of the ends, namely MRE11 [22], RAD50 [22], NBS1 [184] and COM1 (a homolog of the human CtIP, [185]) displayed hypersensitivity (measured as a reduction in root growth, fresh weight or development of true leaves) to some genotoxic compounds [mitomycin (MMC) or methylmethanesulfonate (MMS), agents that produce inter-strand DNA crosslinks or that methylates DNA, respectively]. Interaction between NBS1 and MRE11 has been reported for A. thaliana, maize and rice [184,186]. The NBS1 region involved in the interaction was further characterized in A. thaliana and defined towards the C-terminus [184]. Homologs of BLM, DNA2 and RPA are also conserved in plants. The loss of RECQ4A (a homolog of mammalian BLM helicase, [187][188][189], JHS1 (a homolog of human DNA2, [190] or RPA1C and RPA1E

[89] led to hypersensitivity (measured as a reduction in fresh weight or root growth) in response to a particular DNA damaging agent [MMS; cisplatin (CPT, an agent that mainly induces intra-strand DNA crosslinks); hydroxyurea (an agent that inhibits the ribonucleotide reductase and consequently reduces the dNTP pool and stalls replication forks); zeocin (an agent that induces DSBs) or ionizing radiation]. The recq4A mutant also exhibited an increased frequency of HR [187,189] suggesting a role for the RECQ4A protein in HR regulation. Suppression of HR is dependent on both the N-terminal region and the helicase activity of the protein [189]. A hyper-recombination phenotype was also observed for the jhs1 mutant, probably due to an increased expression of genes involved in DNA damage repair [190].

Several studies have also addressed the functions of RAD51 and RAD51 paralogues in plants [191][192][193][194]. The rad51, rad51c or xrcc3 A. thaliana mutants showed a deficiency in SDSA [192]. Mutants impaired for the other three RAD51 paralogues (RAD51B, RAD51D, XRCC2) were also defective for spontaneous HR [191,193,195], although to differing extents [191,193]. The xrcc2 mutant was the most affected. A further reduction in the recombination rate has been observed in rad51b/xrcc2 double mutant and rad51b/rad51d/xrcc2 triple mutant line [193]. The triple mutant also exhibited a greater sensitivity to the DSB-inducing agent bleomycin (BLE) than the single and double mutants, suggesting a partial functional redundancy [194]. The redundancy of the two AtBRCA2 genes has been also investigated [196][197][198]. Both Atbrca2a and Atbrca2b mutants showed hypersensitivity against CPT and ionizing irradiation, further stimulated in the Atbrca2a/Atbrca2b double-mutant line [196]. Another report indicated that the double mutant also showed hypersensitivity to MMC and a defect in somatic HR frequency [197].

The role of other factors involved in the process of strand exchange or in the stabilization of recombination intermediates has been elucidated using mutant plants. Loss of RAD54 reduced SDSA and loss of FANCM reduced both SDSA and SSA [192]. Proteins involved in the resolution of the D-loop were characterized in vitro or in vivo. Recombinantly expressed SRS2 protein showed a functional 3 0 to 5 0 DNA helicase activity that was able to unwind nicked and partial HJ [199]. Biochemical properties of MUS81-EME1 complexes and GEN1 were also determined after cloning, heterologous expression and purification [200,201]. Two functional EME1 (EME1A and EME1B) and two functional GEN1 [GEN1 (XPG-like endonuclease) and SEND1 (single-strand DNA endonu-clease1)] homologs have been identified in A. thaliana. Both complexes, MUS81-EME1A and MUS81-EME1B, were able to cleave nicked HJs and, with a reduced efficiency, intact HJs [201]. Both GEN1 paralogs preferentially cleaved HJs near the junction point; slight differences in sequence preferences have been detected [200]. The role of MUS81 was also defined in vivo. The mus81 mutant line showed a strong reduction in dry weight to MMC, CPT or MMS treatment [188]. Also, altered sensitivity (measured as a reduction in fresh weight) to MMC and CPT was observed in recq4A, top3A and rmi1 single mutant lines [202]. These three lines also showed an enhanced frequency of uninduced somatic HR [202].

In addition to the HR mechanism, KU-dependent and independent pathways have been also identified in plants. Arabidopsis encodes KU70 (At1g16970), KU80 (At1g48050), PARP1 (At2g31320) and PARP2 (At4g02390) [22,203]. A. thaliana impaired in KU80 showed severe root developmental defects after c irradiation [204,205] and higher levels of DSBs [195]. As with the ku80 mutant plant, the xrcc1 mutant line also exhibited a hypersensitivity to c irradiation [205]. Furthermore, analysis of DSB repair kinetics of ku80 and xrcc1 single mutant lines and ku80/xrcc1 double mutant line indicated a more severe repair defect in the double mutant than in single mutant plants [205]. These observations thus confirm the existence of both KU-dependent and XRCC1-dependent pathways in plants [205]. Besides XRCC1, PARP is also involved in A-NHEJ. In contrast to humans where PARP1 is the major contributor to the PARP activity in response to DNA damage, PARP2 is the primary enzyme responsible for poly(ADP-ribosyl)ation under genotoxic stress in A. thaliana [203]. Plants impaired in both PARP1 and PARP2 displayed a strong reduction in root length and fresh weight after treatment with MMS [206]. However, growth of parp1 and parp2 single mutants was not more sensitive to MMS than wild-type seedlings [206]. Analysis of in vitro endjoining assays indicated that the parp1/parp2 double mutant plant had twofold less end-joining products compared with the wild-type, while the ku70 and ku80 single mutant lines had almost fourfold more products than the wild-type [206]. These observations suggest a regulation of non-homologous recombination pathways. Studies in rice also reported that ku70, ku80, lig4 or xrcc3 mutants showed hypersensitivity to genotoxic agents and/or altered DSB repair [207,208]. On the other hand, overexpression of OsRecQI4 (the AtRecQ4A counterpart in rice), OsExo1 and MtTpd2a enhanced DSB processing in rice and Medicago truncatula, respectively [209,210]. No apparent DNA-PK, XLF, DNA ligase III or DNA polymerase b homologs exist in plants [24,204,205,211].


## Concluding remarks

Significant progress in plant DNA repair has been made in recent years. Sequence homology-based analysis allowed identification of many plant factors, not only in model organisms but also in some crop species. Most of them were also functionally validated. Thus, it is clear that DNA repair pathways are well conserved between plants and mammals. However, some activities have been particularly intensified in plants, probably related to a better adaptation to variable environmental conditions, from which plants cannot escape, and/or to maintain genome stability over multiple generations. Although only limited information on DNA repair mechanisms from crop plants is available, the current advances in A. thaliana can well be translated to gain insights into the challenges of obtaining new varieties with novel traits. protein 1A and regulate the response to UV irradiation. Plant 

## ( 4 )
4Gap-filling and nick sealing. The short-patch or single nucleotide (SN) BER involves the incorporation of only a single nucleotide by the DNA polymerase b. The long-patch (LP) BER requires DNA synthesis mediated by DNA polymerases d and e and assisted by proliferating cell nuclear antigen (PCNA). The newly synthesized DNA displaces the downstream 5 0 DNA end to form a flap structure containing 2-13 nucleotides. The displaced strand is then processed by a flap endonuclease (FEN). The choice of the pathway depends on the specificity of the DNA glycosylase and the proliferation status of the cell. Recent findings suggest that DNA polymerase k can substitute for DNA polymerase b


Khateeb W, Schroeder D (2009) Overexpression of Arabidopsis damaged DNA binding protein 1A (DDB1A) enhances UV tolerance. Plant Mol Biol 70:371-383 79. Ganpudi A, Schroeder D (2013) Genetic interactions of Arabidopsis thaliana damaged DNA binding protein 1B (DDB1B) with DDB1A, DET1, and COP1. G3 (Bethesda) 3:493-503 80. Koga A, Ishibashi T, Kimura S, Uchiyama Y, Sakaguchi K (2006) Characterization of T-DNA insertion mutants and RNAi silenced plants of Arabidopsis thaliana UV-damaged DNA binding protein 2 (AtUV-DDB2). Plant Mol Biol 61:227-240 81. Ly V, Hatherell A, Kim E, Chan A, Belmonte M, Schroeder D (2013) Interactions between Arabidopsis DNA repair genes UVH6, DDB1A, and DDB2 during abiotic stress tolerance and floral development. Plant Sci 213:88-97 82. Shaked H, Avivi-Ragolsky N, Levy A (2006) Involvement of the Arabidopsis SWI2/SNF2 chromatin remodeling gene family in DNA damage response and recombination. Genetics 173:985-994 83. Liang L, Flury S, Kalck V, Hohn B, Molinier J (2006) CEN-TRIN2 interacts with the Arabidopsis homolog of the human XPC protein (AtRAD4) and contributes to efficient synthesisdependent repair of bulky DNA lesions. Plant Mol Biol 61:345-356 84. Vonarx E, Tabone E, Osmond M, Anderson H, Kunz B (2006) Arabidopsis homologue of human transcription factor IIH/nucleotide excision repair factor p44 can function intranscription and DNA repair and interacts with AtXPD. Plant J 46:512-521 85. Gentile A, Ditt R, Dias F, Da Silva M, Dornelas M, Menossi M (2009) Characterization of ScMat1, a putative TFIIH subunit from sugarcane. Plant Cell Rep 28:663-672 86. Gutierrez C (2009) The Arabidopsis cell division cycle. Arabidopsis Book 7:e0120 87. Van Leene J, Hollunder J, Eeckhout D, Persiau G, Van De Slijke E, Stals H, Van Isterdael G, Verkest A, Neirynck S, Buffel Y, De Bodt S, Maere S, Laukens K, Pharazyn A, Ferreira P, Eloy N, Renne C, Meyer C, Faure J, Steinbrenner J, Beynon J, Larkin J, Van de Peer Y, Hilson P, Kuiper M, De Veylder L, Van Onckelen H, Inzé D, Witters E, De Jaeger G (2010) Targeted interactomics reveals a complex core cell cycle machinery in Arabidopsis thaliana. Mol Syst Biol 6:397 88. Eschbach V, Kobbe D (2014) Different replication protein A complexes of Arabidopsis thaliana have different DNA-binding properties as a function of heterotrimer composition. Plant Cell Physiol 55:1460-1472 89. Aklilu B, Soderquist R, Culligan K (2014) Genetic analysis of the Replication Protein A large subunit family in Arabidopsis reveals unique and overlapping roles in DNA repair, meiosis and DNA replication. Nucleic Acids Res 42:3104-3108 90. Aklilu B, Culligan K (2016) Molecular evolution and functional diversification of Replication Protein A1 in plants. Front Plant Sci 7:33 91. Peña-Diaz J, Jiricny J (2012) Mammalian mismatch repair: error-free or error-prone? Trends Biochem Sci 37:206-214 92. Kunkel T, Erie D (2015) Eukaryotic mismatch repair in relation to DNA replication. Annu Rev Genet 49:291-313

## Table 1
1Overview of steps (A) and A. thaliana required factors (B) involved in BERA 


## Table 2
2Components involved in DNA metabolic pathways in A. thalianaGene 
EXO1 
PCNA 
POL d 
POL e 
RFC 
LIG1 

AGI locus code 
At1g29630 
At1g07370 
At2g29570 

At1g09815 

At2g42120 

At5g63960 

At1g08260 

At2g27120 

At5g22110 

At1g21690 

At1g63160 

At1g77470 

At5g22010 

At5g27740 

At1g08130 

At1g49250 


## Table 4
4iv) Completion of DNA synthesis 
PCNA 
See Table 2 
followed by ligation 
RFC 
See Table 2 
Pol δ 
See Table 2 
Pol ε 
See Table 2 
LIG1 



. A difference in sensitivity upon exposure of UV radiation was also observed for plants defective in DDB2, UVH6 (defective in the human XPD homolog) or CSB homologs (CHR8 and CHR24) [76, 77, 80-82]. It should be mentioned that some plant homologs of the human XP genes were initially named after the UV hypersensitive (uvh) phenotype of the mutants. On the other hand, overexpression of DDB1A conferred increased resistance to UV-induced DNA damage[78].

## Table 4
4RPA proteins in A. thalianaGene 
RPA1 
RPA2 
RPA3 

AGI locus code At2g06510 (A) 

At5g08020 (B) 

At5g45400 (C) 

At5g61000 (D) 

At4g19130 (E) 

At2g24490 (A) 

At3g02920 (B) 

At3g52630 (A) 

At4g18590 (B) 


## Table 5
5Overview of steps (A) and A. thaliana required factors (B) involved in MMR 

A 

Step 

i) Lesion 

ii) DNA r 

iii) 

iv) 

B 

Schematic 
representation 
Gene 
AGI locus 
code a,b 

recognition 
MSH2 
At3g18524 
MSH3 
At4g25540 
MSH6 
At4g02070 
MSH7 
At3g24495 
epair complex assembly 
MLH1 
At4g09140 

PMS2 
At4g02460 
Strand discrimination 
RFC 
See Table 2 
PCNA 
See Table 2 
Strand excision and resynthesis 
EXO1 
See Table 2 
RPA 
See 

## Table 6
6Overview of steps (A) and A. thaliana required factors (B) involved in HR 

A 
B 

Step 
Schematic 
representation 
Gene 
AGI locus 
code a,b 

i) DNA end resection and 
MRE11 
At5g54260 
nucleoprotein filament formation 

Acknowledgements I apologize to those whose work is not mentioned here. I acknowledge research support from Agencia Nacional de Promoción Científica y Tecnológica (PICT 2014-3127). C.P.S. is a member of the Researcher Career of CONICET.
DNA base damage by reactive oxygen species, oxidizing agents, and UV radiation. J Cadet, J Wagner, Cold Spring Harb Perspect Biol. 512559Cadet J, Wagner J (2013) DNA base damage by reactive oxygen species, oxidizing agents, and UV radiation. Cold Spring Harb Perspect Biol 5:a012559

Postreplicative mismatch repair. J Jiricny, Cold Spring Harb Perspect Biol. 512633Jiricny J (2013) Postreplicative mismatch repair. Cold Spring Harb Perspect Biol 5:a012633

Mechanisms used by plants to cope with DNA damage. Z Hu, T Cools, De Veylder, L , Annu Rev Plant Biol. 67Hu Z, Cools T, De Veylder L (2016) Mechanisms used by plants to cope with DNA damage. Annu Rev Plant Biol 67:439-462

DNA damage response in plants: conserved and variable response compared to animals. K Yoshiyama, K Sakaguchi, S Kimura, Biology. 2Yoshiyama K, Sakaguchi K, Kimura S (2013) DNA damage response in plants: conserved and variable response compared to animals. Biology 2:1338-1356

Endocycles: a recurrent evolutionary innovation for post-mitotic cell growth. B Edgar, N Zielke, C Gutierrez, Nat Rev Mol Cell Biol. 15Edgar B, Zielke N, Gutierrez C (2014) Endocycles: a recurrent evolutionary innovation for post-mitotic cell growth. Nat Rev Mol Cell Biol 15:197-210

Analysis of the genome sequence of the flowering plant Arabidopsis thaliana. The Arabidopsis Genome Initiative. 408NatureThe Arabidopsis Genome Initiative (2000) Analysis of the genome sequence of the flowering plant Arabidopsis thaliana. Nature 408:796-815

DNA repair and recombination in higher plants: insights from comparative genomics of Arabidopsis and rice. S Singh, S Roy, S Choudhury, D Sengupta, BMC Genom. 11443Singh S, Roy S, Choudhury S, Sengupta D (2010) DNA repair and recombination in higher plants: insights from comparative genomics of Arabidopsis and rice. BMC Genom 11:443

Research on plants for the understanding of diseases of nuclear and mitochondrial origin. C Spampinato, D Gómez-Casati, ID 836196J Biomed Biotech. Spampinato C, Gómez-Casati D (2012) Research on plants for the understanding of diseases of nuclear and mitochondrial origin. J Biomed Biotech 2012:ID 836196

Recognition and repair pathways of damaged DNA in higher plants. Selected Topics in DNA Repair. S Biedermann, S Mooney, H Hellmann, San DiegoUniversity of CaliforniaBiedermann S, Mooney S, Hellmann H (2011) Recognition and repair pathways of damaged DNA in higher plants. Selected Topics in DNA Repair. University of California, San Diego

DNA repair in plants. S Kimura, K Sakaguchi, Chem Rev. 106Kimura S, Sakaguchi K (2006) DNA repair in plants. Chem Rev 106:753-766

A history of the DNA repair and mutagenesis field I. The discovery of enzymatic photoreactivation. E Friedberg, DNA Repair (Amst). 33Friedberg E (2015) A history of the DNA repair and mutagen- esis field I. The discovery of enzymatic photoreactivation. DNA Repair (Amst) 33:35-42

DNA repair in mammalian cells: direct DNA damage reversal: elegant solutions for nasty problems. A Eker, C Quayle, I Chaves, G Van Der Horst, Cell Mol Life Sci. 66Eker A, Quayle C, Chaves I, van der Horst G (2009) DNA repair in mammalian cells: direct DNA damage reversal: elegant solutions for nasty problems. Cell Mol Life Sci 66:968-980

Light-driven DNA repair by photolyases. L Essen, T Klar, Cell Mol Life Sci. 63Essen L, Klar T (2006) Light-driven DNA repair by photolyases. Cell Mol Life Sci 63:1266-1277

Structure and function of photolyase and in vivo enzymology: 50th anniversary. A Sancar, J Biol Chem. 283Sancar A (2008) Structure and function of photolyase and in vivo enzymology: 50th anniversary. J Biol Chem 283:32153-32157

Evolutionary history of the photolyase/cryptochrome superfamily in eukaryotes. Q Mei, V Dvornyk, PLoS One. 10135940Mei Q, Dvornyk V (2015) Evolutionary history of the pho- tolyase/cryptochrome superfamily in eukaryotes. PLoS One 10:e0135940

Light-induced activation of class II cyclobutane pyrimidine dimer photolyases. A Okafuji, T Biskup, K Hitomi, E Getzoff, G Kaiser, A Batschauer, A Bacher, J Hidema, M Teranishi, K Yamamoto, E Schleicher, S Weber, DNA Repair (Amst). 9Okafuji A, Biskup T, Hitomi K, Getzoff E, Kaiser G, Batschauer A, Bacher A, Hidema J, Teranishi M, Yamamoto K, Schleicher E, Weber S (2010) Light-induced activation of class II cyclobutane pyrimidine dimer photolyases. DNA Repair (Amst) 9:495-505

Evolution of mutation rates: phylogenomic analysis of the photolyase/cryptochrome family. J Lucas-Lledó, M Lynch, Mol Biol Evol. 26Lucas-Lledó J, Lynch M (2009) Evolution of mutation rates: phylogenomic analysis of the photolyase/cryptochrome family. Mol Biol Evol 26:1143-1153

Structural and evolutionary aspects of antenna chromophore usage by class II photolyases. S Kiontke, P Gnau, R Haselsberger, A Batschauer, L Essen, J Biol Chem. 289Kiontke S, Gnau P, Haselsberger R, Batschauer A, Essen L (2014) Structural and evolutionary aspects of antenna chro- mophore usage by class II photolyases. J Biol Chem 289:19659-19669

Molecular mechanisms of ultraviolet radiation-induced DNA damage and repair. R Rastogi, X Richa, A Kumar, M Tyagi, R Sinha, J Nucleic Acids. 592980Rastogi R, Richa X, Kumar A, Tyagi M, Sinha R (2010) Molecular mechanisms of ultraviolet radiation-induced DNA damage and repair. J Nucleic Acids 2010:592980

Physiological aspects of UVexcitation of DNA. Richa Sinha, R Häder, D , Top Curr Chem. 356Richa Sinha R, Häder D (2015) Physiological aspects of UV- excitation of DNA. Top Curr Chem 356:203-248

Electron transfer mechanisms of DNA repair by photolyase. D Zhong, Annu Rev Phys Chem. 66Zhong D (2015) Electron transfer mechanisms of DNA repair by photolyase. Annu Rev Phys Chem 66:691-715

Gene regulation in response to DNA damage. A Mannuss, O Trapp, H Puchta, Biochim Biophys Acta. 1819Mannuss A, Trapp O, Puchta H (2012) Gene regulation in response to DNA damage. Biochim Biophys Acta 1819:154-165

Ultraviolet-defense mechanisms in higher plants. T Ueda, C Nakamura, Biotechnol Biotechnol Equip. 25Ueda T, Nakamura C (2011) Ultraviolet-defense mechanisms in higher plants. Biotechnol Biotechnol Equip 25:2177-2182

DNA damage and repair in plants-from models to crops. V Manova, D Gruszka, Front Plant Sci. 6885Manova V, Gruszka D (2015) DNA damage and repair in plants-from models to crops. Front Plant Sci 6:art 885

UV-B-induced CPD photolyase gene expression is regulated by UVR8-dependent and -independent pathways in Arabidopsis. N Li, M Teranishi, H Yamaguchi, T Matsushita, M Watahiki, T Tsuge, S Li, J Hidema, Plant Cell Physiol. 56Li N, Teranishi M, Yamaguchi H, Matsushita T, Watahiki M, Tsuge T, Li S, Hidema J (2015) UV-B-induced CPD photolyase gene expression is regulated by UVR8-dependent and -inde- pendent pathways in Arabidopsis. Plant Cell Physiol 56:2014-2023

Eukaryotic class II cyclobutane pyrimidine dimer photolyase structure reveals basis for improved ultraviolet tolerance in plants. K Hitomi, A Arvai, J Yamamoto, C Hitomi, M Teranishi, T Hirouchi, K Yamamoto, S Iwai, J Tainer, J Hidema, E Getzoff, J Biol Chem. 287Hitomi K, Arvai A, Yamamoto J, Hitomi C, Teranishi M, Hir- ouchi T, Yamamoto K, Iwai S, Tainer J, Hidema J, Getzoff E (2012) Eukaryotic class II cyclobutane pyrimidine dimer pho- tolyase structure reveals basis for improved ultraviolet tolerance in plants. J Biol Chem 287:12060-12069

Functional motifs in the (6-4) photolyase crystal structure make a comparative framework for DNA repair photolyases and clock cryptochromes. K Hitomi, L Ditacchio, A Arvai, J Yamamoto, S Kim, T Todo, J Tainer, S Iwai, S Panda, E Getzoff, Proc Natl Acad Sci. 106Hitomi K, DiTacchio L, Arvai A, Yamamoto J, Kim S, Todo T, Tainer J, Iwai S, Panda S, Getzoff E (2009) Functional motifs in the (6-4) photolyase crystal structure make a comparative framework for DNA repair photolyases and clock cryp- tochromes. Proc Natl Acad Sci USA 106:6962-6967

Dynamics and mechanism of repair of UV-induced (6-4) photoproduct by photolyase. J Li, Z Liu, C Tan, X Guo, L Wang, A Sancar, D Zhong, Nature. 466Li J, Liu Z, Tan C, Guo X, Wang L, Sancar A, Zhong D (2010) Dynamics and mechanism of repair of UV-induced (6-4) pho- toproduct by photolyase. Nature 466:887-890

Dynamics and mechanisms of DNA repair by photolyase. Z Liu, L Wang, D Zhong, Phys Chem Chem Phys. 17Liu Z, Wang L, Zhong D (2015) Dynamics and mechanisms of DNA repair by photolyase. Phys Chem Chem Phys 17:11933-11949

Overview of base excision repair biochemistry. Y Kim, D I Wilson, Curr Mol Pharmacol. 5Kim Y, Wilson DI (2012) Overview of base excision repair biochemistry. Curr Mol Pharmacol 5:3-13

Base excision repair. H Krokan, M Bjørås, Cold Spring Harb Perspect Biol. 512583Krokan H, Bjørås M (2013) Base excision repair. Cold Spring Harb Perspect Biol 5:a012583

Base excision repair and lesion-dependent subpathways for repair of oxidative DNA damage. D Svilar, E Goellner, K Almeida, R Sobol, Antioxid Redox Signal. 14Svilar D, Goellner E, Almeida K, Sobol R (2011) Base excision repair and lesion-dependent subpathways for repair of oxidative DNA damage. Antioxid Redox Signal 14:2491-2507

Recent advances in the structural mechanisms of DNA glycosylases. S Brooks, S Adhikary, E Rubinson, B Eichman, Biochim Biophys Acta. 1834Brooks S, Adhikary S, Rubinson E, Eichman B (2013) Recent advances in the structural mechanisms of DNA glycosylases. Biochim Biophys Acta 1834:247-271

DNA glycosylases: in DNA repair and beyond. A Jacobs, P Schär, Chromosoma. 121Jacobs A, Schär P (2012) DNA glycosylases: in DNA repair and beyond. Chromosoma 121:1-20

Mammalian base excision repair: the forgotten archangel. G Dianov, U Hübscher, Nucleic Acids Res. 41Dianov G, Hübscher U (2013) Mammalian base excision repair: the forgotten archangel. Nucleic Acids Res 41:3483-3490

Structure-function studies of DNA polymerase k. K Bebenek, L Pedersen, T Kunkel, Biochemistry. 53Bebenek K, Pedersen L, Kunkel T (2014) Structure-function studies of DNA polymerase k. Biochemistry 53:2781-2792

XRCC1coordinates disparate responses and multiprotein repair complexes depending on the nature and context of the DNA damage. A Hanssen-Bauer, K Solvang-Garten, O Sundheim, J Peña-Diaz, S Andersen, G Slupphaug, H Krokan, D I Wilson, M Akbari, M Otterlei, Environ Mol Mutagen. 52Hanssen-Bauer A, Solvang-Garten K, Sundheim O, Peña-Diaz J, Andersen S, Slupphaug G, Krokan H, Wilson DI, Akbari M, Otterlei M (2011) XRCC1coordinates disparate responses and multiprotein repair complexes depending on the nature and context of the DNA damage. Environ Mol Mutagen 52:623-635

Genotoxic stress and DNA repair in plants: emerging functions and tools for improving crop productivity. A Balestrazzi, M Confalonieri, A Macovei, M Donà, D Carbonera, Plant Cell Rep. 30Balestrazzi A, Confalonieri M, Macovei A, Donà M, Carbonera D (2011) Genotoxic stress and DNA repair in plants: emerging functions and tools for improving crop productivity. Plant Cell Rep 30:287-295

Repair and tolerance of oxidative DNA damage in plants. T Roldán-Arjona, R Ariza, Mutat Res. 681Roldán-Arjona T, Ariza R (2009) Repair and tolerance of oxidative DNA damage in plants. Mutat Res 681:169-179

Arabidopsis uracil DNA glycosylase (ung) is required for base excision repair of uracil and increases plant sensitivity to 5-fluorouracil. D Córdoba-Cañero, E Dubois, R Ariza, M-P Doutriaux, T Roldán-Arjona, J Biol Chem. 285Córdoba-Cañero D, Dubois E, Ariza R, Doutriaux M-P, Roldán- Arjona T (2010) Arabidopsis uracil DNA glycosylase (ung) is required for base excision repair of uracil and increases plant sensitivity to 5-fluorouracil. J Biol Chem 285:7475-7483

Single-nucleotide and long-patch base excision repair of DNA damage in plants. D Cordoba-Cañero, T Morales-Ruiz, T Roldán-Arjona, R Ariza, Plant J. 60Cordoba-Cañero D, Morales-Ruiz T, Roldán-Arjona T, Ariza R (2009) Single-nucleotide and long-patch base excision repair of DNA damage in plants. Plant J 60:716-728

Arabidopsis ZDP DNA 3 0 -phosphatase and ARP endonuclease function in 8-oxoG repair initiated by FPG and OGG1 DNA glycosylases. D Córdoba-Cañero, T Roldán-Arjona, R Ariza, Plant J. 79Córdoba-Cañero D, Roldán-Arjona T, Ariza R (2014) Ara- bidopsis ZDP DNA 3 0 -phosphatase and ARP endonuclease function in 8-oxoG repair initiated by FPG and OGG1 DNA glycosylases. Plant J 79:824-834

Structural and biochemical studies of a plant formamidopyrimidine-DNA glycosylase reveal why eukaryotic Fpg glycosylases do not excise 8-oxoguanine. S Duclos, P Aller, P Jaruga, M Dizdaroglu, S Wallace, S Doublié, DNA Repair (Amst). 11Duclos S, Aller P, Jaruga P, Dizdaroglu M, Wallace S, Doublié S (2012) Structural and biochemical studies of a plant for- mamidopyrimidine-DNA glycosylase reveal why eukaryotic Fpg glycosylases do not excise 8-oxoguanine. DNA Repair (Amst) 11:714-725

Evidence for base excision repair of oxidative DNA damage in chloroplasts of Arabidopsis thaliana. B Gutman, K Niyogi, J Biol Chem. 284Gutman B, Niyogi K (2009) Evidence for base excision repair of oxidative DNA damage in chloroplasts of Arabidopsis thaliana. J Biol Chem 284:17006-17012

DEME-TER and REPRESSOR OF SILENCING 1 encode 5-methylcytosine DNA glycosylases. T Morales-Ruiz, A Ortega-Galisteo, M Ponferrada-Marín, M Martínez-Macías, R Ariza, T Roldán-Arjona, Proc Natl Acad Sci. 103Morales-Ruiz T, Ortega-Galisteo A, Ponferrada-Marín M, Martínez-Macías M, Ariza R, Roldán-Arjona T (2006) DEME- TER and REPRESSOR OF SILENCING 1 encode 5-methylcytosine DNA glycosylases. Proc Natl Acad Sci USA 103:6853-6858

ROS1 5-methylcytosine DNA glycosylase is a slow-turnover catalyst that initiates DNA demethylation in a distributive fashion. M Ponferrada-Marin, T Roldan-Arjona, R Ariza, Nucleic Acids Res. 37Ponferrada-Marin M, Roldan-Arjona T, Ariza R (2009) ROS1 5-methylcytosine DNA glycosylase is a slow-turnover catalyst that initiates DNA demethylation in a distributive fashion. Nucleic Acids Res 37:4264-4274

5-methylcytosine recognition by Arabidopsis thaliana DNA glycosylases DEMETER and DML3. S Brooks, R Fischer, J Huh, B Eichman, Biochemistry. 53Brooks S, Fischer R, Huh J, Eichman B (2014) 5-methylcytosine recognition by Arabidopsis thaliana DNA glycosylases DEMETER and DML3. Biochemistry 53:2525-2532

Early steps of active DNA demethylation initiated by ROS1 glycosylase require three putative helix-invading residues. J Parrilla-Doblas, M Ponferrada-Marin, T Roldan-Arjona, R Ariza, Nucleic Acids Res. 41Parrilla-Doblas J, Ponferrada-Marin M, Roldan-Arjona T, Ariza R (2013) Early steps of active DNA demethylation initiated by ROS1 glycosylase require three putative helix-invading resi- dues. Nucleic Acids Res 41:8654-8664

The carboxy-terminal domain of ROS1 is essential for 5-methylcytosine DNA glycosylase activity. S Hong, H Hashimoto, Y Kow, X Zhang, X Cheng, J Mol Biol. 426Hong S, Hashimoto H, Kow Y, Zhang X, Cheng X (2014) The carboxy-terminal domain of ROS1 is essential for 5-methylcy- tosine DNA glycosylase activity. J Mol Biol 426:3703-3712

Molecular characterization of a putative plant homolog of MBD4 DNA glycosylase. A Ramiro-Merina, R Ariza, T Roldán-Arjona, DNA Repair (Amst). 12Ramiro-Merina A, Ariza R, Roldán-Arjona T (2013) Molecular characterization of a putative plant homolog of MBD4 DNA glycosylase. DNA Repair (Amst) 12:890-898

Overexpression of AtOGG1, a DNA glycosylase/AP lyase, enhances seed longevity and abiotic stress tolerance in Arabidopsis. H Chen, P Chu, Y Zhou, Y Li, J Liu, Y Ding, E Tsang, L Jiang, K Wu, S Huang, J Exp Bot. 63Chen H, Chu P, Zhou Y, Li Y, Liu J, Ding Y, Tsang E, Jiang L, Wu K, Huang S (2012) Overexpression of AtOGG1, a DNA glycosylase/AP lyase, enhances seed longevity and abiotic stress tolerance in Arabidopsis. J Exp Bot 63:4107-4121

Expression and function of AtMBD4L, the single gene encoding the nuclear DNA glycosylase MBD4L in Arabidopsis. F Nota, D Cambiagno, P Ribone, M Alvarez, Plant Sci. 235Nota F, Cambiagno D, Ribone P, Alvarez M (2015) Expression and function of AtMBD4L, the single gene encoding the nuclear DNA glycosylase MBD4L in Arabidopsis. Plant Sci 235:122-129

DDM1 and ROS1 have a role in UV-B induced-and oxidative DNA damage in A. thaliana. J Qüesta, J Fina, P Casati, Front Plant Sci. 4420Qüesta J, Fina J, Casati P (2013) DDM1 and ROS1 have a role in UV-B induced-and oxidative DNA damage in A. thaliana. Front Plant Sci 4:420

Requirement for abasic endonuclease gene homologues in Arabidopsis seed development. T Murphy, M Belmonte, S Shu, A Britt, J Hatteroth, PLoS One. 44297Murphy T, Belmonte M, Shu S, Britt A, Hatteroth J (2009) Requirement for abasic endonuclease gene homologues in Arabidopsis seed development. PLoS One 4:e4297

An AP endonuclease functions in active DNA demethylation and gene imprinting in Arabidopsis. Y Li, D Córdoba-Cañero, W Qian, X Zhu, K Tang, H Zhang, R Ariza, T Roldán-Arjona, J-K Zhu, PLoS Genet. 111004905Li Y, Córdoba-Cañero D, Qian W, Zhu X, Tang K, Zhang H, Ariza R, Roldán-Arjona T, Zhu J-K (2015) An AP endonuclease functions in active DNA demethylation and gene imprinting in Arabidopsis. PLoS Genet 11:e1004905

Arabidopsis ARP endonuclease functions in a branched base excision DNA repair pathway completed by LIG1. D Cordoba-Cañero, T Roldán-Arjona, R Ariza, Plant J. 68Cordoba-Cañero D, Roldán-Arjona T, Ariza R (2011) Ara- bidopsis ARP endonuclease functions in a branched base excision DNA repair pathway completed by LIG1. Plant J 68:693-702

Cloning and characterization of a wheat homologue of apurinic/apyrimidinic endonuclease Ape1L. B Joldybayeva, P Prorok, I Grin, D Zharkov, A Ishenko, B Tudek, A Bissenbaev, M Saparbaev, PLoS One. 992963Joldybayeva B, Prorok P, Grin I, Zharkov D, Ishenko A, Tudek B, Bissenbaev A, Saparbaev M (2014) Cloning and characterization of a wheat homologue of apurinic/apyrimidinic endonuclease Ape1L. PLoS One 9:e92963

The DNA repair protein XRCC1 functions in the plant DNA demethylation pathway by stimulating cytosine methylation (5-mec) excision, gap tailoring, and DNA ligation. M Martínez-Macías, D Córdoba-Cañero, R Ariza, T Roldán-Arjona, J Biol Chem. 288Martínez-Macías M, Córdoba-Cañero D, Ariza R, Roldán-Ar- jona T (2013) The DNA repair protein XRCC1 functions in the plant DNA demethylation pathway by stimulating cytosine methylation (5-mec) excision, gap tailoring, and DNA ligation. J Biol Chem 288:5496-5505

A DNA 3 0 phosphatase functions in active DNA demethylation in Arabidopsis. M Martínez-Macías, W Qian, D Miki, O Pontes, Y Liu, K Tang, R Liu, T Morales-Ruiz, R Ariza, T Roldán-Arjona, J-K Zhu, Mol Cell. 45Martínez-Macías M, Qian W, Miki D, Pontes O, Liu Y, Tang K, Liu R, Morales-Ruiz T, Ariza R, Roldán-Arjona T, Zhu J-K (2012) A DNA 3 0 phosphatase functions in active DNA demethylation in Arabidopsis. Mol Cell 45:357-370

Structure-function studies of a plant tyrosyl-DNA phosphodiesterase provide novel insights into DNA repair mechanisms of Arabidopsis thaliana. H Kim, Na S Lee, S-Y Jeong, Y-M Hwang, H-J Hur, J Park, S-H Woo, J-C Kim, S , Biochem J. 443Kim H, Na S, Lee S-Y, Jeong Y-M, Hwang H-J, Hur J, Park S-H, Woo J-C, Kim S (2012) Structure-function studies of a plant tyrosyl-DNA phosphodiesterase provide novel insights into DNA repair mechanisms of Arabidopsis thaliana. Biochem J 443:49-56

DNA ligase 1 deficient plants display severe growth defects and delayed repair of both DNA single and double strand breaks. W Waterworth, J Kozak, C Provost, C Bray, K Angelis, C West, BMC Plant Biol. 979Waterworth W, Kozak J, Provost C, Bray C, Angelis K, West C (2009) DNA ligase 1 deficient plants display severe growth defects and delayed repair of both DNA single and double strand breaks. BMC Plant Biol 9:79

Shade avoidance 6 encodes an Arabidopsis flap endonuclease required for maintenance of genome integrity and development. Y Zhang, C Wen, S Liu, L Zheng, B Shen, Y Tao, Nucleic Acids Res. 44Zhang Y, Wen C, Liu S, Zheng L, Shen B, Tao Y (2016) Shade avoidance 6 encodes an Arabidopsis flap endonuclease required for maintenance of genome integrity and development. Nucleic Acids Res 44:1271-1284

Orchestral maneuvers at the damaged sites in nucleotide excision repair. S Alekseev, F Coin, Cell Mol Life Sci. 72Alekseev S, Coin F (2015) Orchestral maneuvers at the damaged sites in nucleotide excision repair. Cell Mol Life Sci 72:2177-2186

Nucleotide excision repair in eukaryotes. O Schärer, Cold Spring Harb Perspect Biol. 512609Schärer O (2013) Nucleotide excision repair in eukaryotes. Cold Spring Harb Perspect Biol 5:a012609

The complex choreography of transcription-coupled repair. G Spivak, A Ganesan, DNA Repair (Amst). 19Spivak G, Ganesan A (2014) The complex choreography of transcription-coupled repair. DNA Repair (Amst) 19:64-70

Mammalian transcription coupled excision repair. W Vermeulen, M Fousteri, Cold Spring Harb Perspect Biol. 512625Vermeulen W, Fousteri M (2013) Mammalian transcription coupled excision repair. Cold Spring Harb Perspect Biol 5:a012625

Functional insights into the core-TFIIH from a comparative survey. F Bedez, B Linard, X Brochet, R Ripp, J Thompson, D Moras, O Lecompte, O Poch, Genomics. 101Bedez F, Linard B, Brochet X, Ripp R, Thompson J, Moras D, Lecompte O, Poch O (2013) Functional insights into the core- TFIIH from a comparative survey. Genomics 101:178-186

XPB and XPD helicases in TFIIH orchestrate DNA duplex opening and damage verification to coordinate repair with transcription and cell cycle via CAK kinase. J Fuss, J Tainer, DNA Repair (Amst). 10Fuss J, Tainer J (2011) XPB and XPD helicases in TFIIH orchestrate DNA duplex opening and damage verification to coordinate repair with transcription and cell cycle via CAK kinase. DNA Repair (Amst) 10:697-713

Regulation of endonuclease activity in human nucleotide excision repair. A Fagbemi, B Orelli, O Schärer, DNA Repair (Amst). 10Fagbemi A, Orelli B, Schärer O (2011) Regulation of endonu- clease activity in human nucleotide excision repair. DNA Repair (Amst) 10:722-729

Three DNA polymerases, recruited by different mechanisms, carry out NER repair synthesis in human cells. T Ogi, S Limsirichaikul, R Overmeer, M Volker, K Takenaka, R Cloney, Y Nakazawa, A Nimi, N Jaspers, L Mullenders, S Y Fousteri, M Lehamann, A , Mol Cell. 37Ogi T, Limsirichaikul S, Overmeer R, Volker M, Takenaka K, Cloney R, Nakazawa Y, Nimi A, Jaspers N, Mullenders L, S Y, Fousteri M, Lehamann A (2010) Three DNA polymerases, recruited by different mechanisms, carry out NER repair syn- thesis in human cells. Mol Cell 37:714-727

UV damaged DNA repair and tolerance in plants. Selected Topics in DNA Repair. Ganpudi A, Schroeder DIntech, CroatiaGanpudi A, Schroeder D (eds) (2011) UV damaged DNA repair and tolerance in plants. Selected Topics in DNA Repair, Intech, Croatia

The RAD23 family provides an essential connection between the 26S proteasome and ubiquitylated proteins in Arabidopsis. L Farmer, A Book, K Lee, Y Lin, H Fu, R Vierstra, Plant Cell. 22Farmer L, Book A, Lee K, Lin Y, Fu H, Vierstra R (2010) The RAD23 family provides an essential connection between the 26S proteasome and ubiquitylated proteins in Arabidopsis. Plant Cell 22:124-142

DDB2, DDB1A and DET1 exhibit complex interactions during Arabidopsis development. Al Khateeb, W Schroeder, D , Genetics. 176Al Khateeb W, Schroeder D (2007) DDB2, DDB1A and DET1 exhibit complex interactions during Arabidopsis development. Genetics 176:231-242

Arabidopsis DDB1a and DDB1b are critical for embryo development. A Bernhardt, S Mooney, H Hellmann, Planta. 232Bernhardt A, Mooney S, Hellmann H (2010) Arabidopsis DDB1a and DDB1b are critical for embryo development. Planta 232:555-566

Arabidopsis cockayne syndrome A-like proteins 1A and 1B form a complex with CULLIN4 and damage DNA binding Protecting DNA from errors and damage 1705. C Zhang, H Guo, J Zhang, G Guo, K Schumaker, Y Guo, Zhang C, Guo H, Zhang J, Guo G, Schumaker K, Guo Y (2010) Arabidopsis cockayne syndrome A-like proteins 1A and 1B form a complex with CULLIN4 and damage DNA binding Protecting DNA from errors and damage 1705

New insights into the mechanism of DNA mismatch repair. G Reyes, T Schmidt, R Kolodner, H Hombauer, Chromosoma. 124Reyes G, Schmidt T, Kolodner R, Hombauer H (2015) New insights into the mechanism of DNA mismatch repair. Chro- mosoma 124:443-462

Protein-protein interactions in DNA mismatch repair. P Friedhoff, P Li, J Gotthardt, DNA Repair (Amst). 38Friedhoff P, Li P, Gotthardt J (2016) Protein-protein interactions in DNA mismatch repair. DNA Repair (Amst) 38:50-57

A personal historical view of DNA mismatch repair with an emphasis on eukaryotic DNA mismatch repair. R Kolodner, DNA Repair (Amst). 38Kolodner R (2016) A personal historical view of DNA mismatch repair with an emphasis on eukaryotic DNA mismatch repair. DNA Repair (Amst) 38:3-13

Conservation and diversity of MutS proteins. P Sachadyn, Mutat Res. 694Sachadyn P (2010) Conservation and diversity of MutS proteins. Mutat Res 694:20-30

Distinct nucleotide binding/hydrolysis properties and molar ratio of MutSa and MutSb determine their differential mismatch binding activities. L Tian, L Gu, G-M Li, J Biol Chem. 284Tian L, Gu L, Li G-M (2009) Distinct nucleotide binding/hy- drolysis properties and molar ratio of MutSa and MutSb determine their differential mismatch binding activities. J Biol Chem 284:11557-11562

Structure of the human MutSa DNA lesion recognition complex. J Warren, T Pohlhaus, A Changela, R Iyer, P Modrich, L Beese, Mol Cell. 26Warren J, Pohlhaus T, Changela A, Iyer R, Modrich P, Beese L (2007) Structure of the human MutSa DNA lesion recognition complex. Mol Cell 26:579-592

Mechanism of mismatch recognition revealed by human MutSb bound to unpaired DNA loops. S Gupta, M Gellert, W Yang, Nat Struct Mol Biol. 19Gupta S, Gellert M, Yang W (2011) Mechanism of mismatch recognition revealed by human MutSb bound to unpaired DNA loops. Nat Struct Mol Biol 19:72-78

The conserved molecular machinery in DNA mismatch repair enzyme structures. F Groothuizen, T Sixma, DNA Repair (Amst). 38Groothuizen F, Sixma T (2016) The conserved molecular machinery in DNA mismatch repair enzyme structures. DNA Repair (Amst) 38:14-23

The nucleotide binding dynamics of human MSH2-MSH3 are lesion dependent. B Owen, W Lang, C Mcmurray, Nat Struct Mol Biol. 16Owen B, Lang W, McMurray C (2009) The nucleotide binding dynamics of human MSH2-MSH3 are lesion dependent. Nat Struct Mol Biol 16:550-557

MutLaand proliferating cell nuclear antigen share binding sites on MutSb. R Iyer, A Pluciennik, J Genschel, M Tsai, L Beese, P Modrich, J Biol Chem. 285Iyer R, Pluciennik A, Genschel J, Tsai M, Beese L, Modrich P (2010) MutLaand proliferating cell nuclear antigen share bind- ing sites on MutSb. J Biol Chem 285:11730-11739

Structural, molecular and cellular functions of MSH2 and MSH6 during DNA mismatch repair, damage signaling and other noncanonical activities. M Edelbrock, S Kaliyaperumal, K Williams, Mutat Res. 743Edelbrock M, Kaliyaperumal S, Williams K (2013) Structural, molecular and cellular functions of MSH2 and MSH6 during DNA mismatch repair, damage signaling and other noncanoni- cal activities. Mutat Res 743-744:53-66

Cell cycle regulation of human DNA repair and chromatin remodeling genes. R Mjelle, S Hegre, P Aas, G Slupphaug, F Drabløs, P Saetrom, H Krokan, DNA Repair (Amst). 30Mjelle R, Hegre S, Aas P, Slupphaug G, Drabløs F, Saetrom P, Krokan H (2015) Cell cycle regulation of human DNA repair and chromatin remodeling genes. DNA Repair (Amst) 30:53-67

Endonuclease activities of MutLa and its homologs in DNA mismatch repair. L Kadyrova, F Kadyrov, DNA Repair (Amst). 38Kadyrova L, Kadyrov F (2016) Endonuclease activities of MutLa and its homologs in DNA mismatch repair. DNA Repair (Amst) 38:42-49

Insights from a decade of biophysical studies on MutL: roles in strand discrimination and mismatch removal. A Guarné, J Charbonnier, Prog Biophys Mol Biol. 117Guarné A, Charbonnier J (2016) Insights from a decade of biophysical studies on MutL: roles in strand discrimination and mismatch removal. Prog Biophys Mol Biol 117:149-156

Direct visualization of asymmetric adenine-nucleotide-induced conformational changes in MutLa. E Sacho, F Kadyrov, P Modrich, T Kunkel, D Erie, Mol Cell. 29Sacho E, Kadyrov F, Modrich P, Kunkel T, Erie D (2008) Direct visualization of asymmetric adenine-nucleotide-induced con- formational changes in MutLa. Mol Cell 29:112-121

Analysis of the role of PCNA-DNA contacts during clamp loading. R Mcnally, G Bowman, E Goedken, O&apos; Donnell, M Kuriyan, J , BMC Struct Biol. 103McNally R, Bowman G, Goedken E, O'Donnell M, Kuriyan J (2010) Analysis of the role of PCNA-DNA contacts during clamp loading. BMC Struct Biol 10:3

PCNA function in the activation and strand direction of MutLaendonuclease in mismatch repair. A Pluciennik, L Dzantiev, R Iyer, N Constantin, F Kadyrov, P Modrich, Proc Natl Acad Sci. 107Pluciennik A, Dzantiev L, Iyer R, Constantin N, Kadyrov F, Modrich P (2010) PCNA function in the activation and strand direction of MutLaendonuclease in mismatch repair. Proc Natl Acad Sci USA 107:16066-16071

Hydrolytic function of Exo1 in mammalian mismatch repair. H Shao, C Baitinger, E Soderblom, V Burdett, P Modrich, Nucleic Acids Res. 42Shao H, Baitinger C, Soderblom E, Burdett V, Modrich P (2014) Hydrolytic function of Exo1 in mammalian mismatch repair. Nucleic Acids Res 42:7104-7112

Functions of MutLalpha, replication protein A (RPA), and HMGB1 in 5 0 -directed mismatch repair. J Genschel, P Modrich, J Biol Chem. 284Genschel J, Modrich P (2009) Functions of MutLalpha, repli- cation protein A (RPA), and HMGB1 in 5 0 -directed mismatch repair. J Biol Chem 284:21536-21544

A possible mechanism for exonuclease 1-independent eukaryotic mismatch repair. F Kadyrov, J Genschel, Y Fang, E Penland, W Edelmann, P Modrich, Proc Natl Acad Sci. 106Kadyrov F, Genschel J, Fang Y, Penland E, Edelmann W, Modrich P (2009) A possible mechanism for exonuclease 1-in- dependent eukaryotic mismatch repair. Proc Natl Acad Sci USA 106:8495-8500

Exonuclease 1-dependent and independent mismatch repair. E Goellner, C Putnam, R Kolodner, DNA Repair (Amst). 32Goellner E, Putnam C, Kolodner R (2015) Exonuclease 1-de- pendent and independent mismatch repair. DNA Repair (Amst) 32:24-32

From bacteria to plants: a compendium of mismatch repair assays. C Spampinato, R Gomez, C Galles, L Lario, Mutat Res. 682Spampinato C, Gomez R, Galles C, Lario L (2009) From bac- teria to plants: a compendium of mismatch repair assays. Mutat Res 682:110-128

Characterization and comparative sequence analysis of the DNA mismatch repair MSH2 and MSH7 genes from tomato. S Tam, S Samipak, A Britt, R Chetelat, Genetica. 137Tam S, Samipak S, Britt A, Chetelat R (2009) Characterization and comparative sequence analysis of the DNA mismatch repair MSH2 and MSH7 genes from tomato. Genetica 137:341-354

PMS1 from Arabidopsis thaliana: optimization of protein overexpression in Escherichia coli. C Galles, R Gomez, C Spampinato, Mol Biol Rep. 38Galles C, Gomez R, Spampinato C (2011) PMS1 from Ara- bidopsis thaliana: optimization of protein overexpression in Escherichia coli. Mol Biol Rep 38:1063-1070

High-level production of MSH2 from Arabidopsis thaliana: a DNA mismatch repair system key subunit. R Gomez, C Galles, C Spampinato, Mol Biotechnol. 47Gomez R, Galles C, Spampinato C (2011) High-level production of MSH2 from Arabidopsis thaliana: a DNA mismatch repair system key subunit. Mol Biotechnol 47:120-129

Mismatch recognition function of Arabidopsis thaliana MutSc. R Gomez, C Spampinato, DNA Repair (Amst). 12Gomez R, Spampinato C (2013) Mismatch recognition function of Arabidopsis thaliana MutSc. DNA Repair (Amst) 12:257-264

Yeast mutator phenotype enforced by Arabidopsis PMS1 expression. C Galles, C Spampinato, Mol Biol Rep. 40Galles C, Spampinato C (2013) Yeast mutator phenotype enforced by Arabidopsis PMS1 expression. Mol Biol Rep 40:2107-2114

Effects of suppressing the DNA mismatch repair system on homeologous recombination in tomato. S Tam, J Hays, R Chetelat, Theor Appl Genet. 123Tam S, Hays J, Chetelat R (2011) Effects of suppressing the DNA mismatch repair system on homeologous recombination in tomato. Theor Appl Genet 123:1445-1458

Rapid generation of rice mutants via the dominant negative suppression of the mismatch repair protein OsPMS1. J Xu, M Li, L Chen, G Wu, H Li, Theor Appl Genet. 125Xu J, Li M, Chen L, Wu G, Li H (2012) Rapid generation of rice mutants via the dominant negative suppression of the mismatch repair protein OsPMS1. Theor Appl Genet 125:975-986

Genomic stability in Nicotiana plants upon silencing of the mismatch repair gene MSH2. I Van Marcke, G Angenon, Plant Biotechnol Rep. 7Van Marcke I, Angenon G (2013) Genomic stability in Nico- tiana plants upon silencing of the mismatch repair gene MSH2. Plant Biotechnol Rep 7:467-480

Role of AtMSH7 in UV-B-induced DNA damage recognition and recombination. L Lario, P Botta, P Casati, C Spampinato, J Exp Bot. 66Lario L, Botta P, Casati P, Spampinato C (2015) Role of AtMSH7 in UV-B-induced DNA damage recognition and recombination. J Exp Bot 66:3019-3026

Regulation of plant MSH2 and MSH6 genes in the UV-Binduced DNA damage response. L Lario, E Ramirez-Parra, C Gutierrez, P Casati, C Spampinato, J Exp Bot. 62Lario L, Ramirez-Parra E, Gutierrez C, Casati P, Spampinato C (2011) Regulation of plant MSH2 and MSH6 genes in the UV-B- induced DNA damage response. J Exp Bot 62:2925-2937

Repair pathway choices and consequences at the double-strand break. R Ceccaldi, B Rondinelli, D &apos;andrea, A , Trends Cell Biol. 26Ceccaldi R, Rondinelli B, D'Andrea A (2016) Repair pathway choices and consequences at the double-strand break. Trends Cell Biol 26:52-64

Initiation of DNA double strand break repair: signaling and single-stranded resection dictate the choice between homologous recombination, non-homologous end-joining and alternative end-joining. A Grabarz, A Barascu, J Guirouilh-Barbat, B Lopez, Am J Cancer Res. 2Grabarz A, Barascu A, Guirouilh-Barbat J, Lopez B (2012) Initiation of DNA double strand break repair: signaling and single-stranded resection dictate the choice between homolo- gous recombination, non-homologous end-joining and alternative end-joining. Am J Cancer Res 2:249-268

DNA DSB repair pathway choice: an orchestrated handover mechanism. A Kakarougkas, P Jeggo, Br J Radiol. 87Kakarougkas A, Jeggo P (2014) DNA DSB repair pathway choice: an orchestrated handover mechanism. Br J Radiol 87:20130685

A cell cycle-dependent regulatory circuit composed of 53BP1-RIF1 and BRCA1-CtIP controls DNA repair pathway choice. C Escribano-Diaz, A Orthwein, A Fradet-Turcotte, M Xing, J Young, J Tkác, M Cook, A Rosebrock, M Munro, M Canny, D Xu, D Durocher, Mol Cell. 49Escribano-Diaz C, Orthwein A, Fradet-Turcotte A, Xing M, Young J, Tkác J, Cook M, Rosebrock A, Munro M, Canny M, Xu D, Durocher D (2013) A cell cycle-dependent regulatory circuit composed of 53BP1-RIF1 and BRCA1-CtIP controls DNA repair pathway choice. Mol Cell 49:872-883

Quantitative live cell imaging reveals a gradual shift between DNA repair mechanisms and a maximal use of HR in mid S phase. K Karanam, R Kafri, A Loewer, G Lahav, Mol Cell. 47Karanam K, Kafri R, Loewer A, Lahav G (2012) Quantitative live cell imaging reveals a gradual shift between DNA repair mechanisms and a maximal use of HR in mid S phase. Mol Cell 47:320-329

Repair of strand breaks by homologous recombination. M Jasin, R Rothstein, Cold Spring Harb Perspect Biol. 512740Jasin M, Rothstein R (2013) Repair of strand breaks by homologous recombination. Cold Spring Harb Perspect Biol 5:a012740

Homologous recombination and its regulation. L Krejci, V Altmannova, M Spirek, X Zhao, Nucleic Acids Res. 40Krejci L, Altmannova V, Spirek M, Zhao X (2012) Homologous recombination and its regulation. Nucleic Acids Res 40:5795-5818

An overview of the molecular mechanisms of recombinational DNA repair. S Kowalczykowski, Cold Spring Harb Perspect Biol. 716410Kowalczykowski S (2015) An overview of the molecular mechanisms of recombinational DNA repair. Cold Spring Harb Perspect Biol 7:a016410

Human CtIP promotes DNA end resection. A Sartori, C Lukas, J Coates, M Mistrik, S Fu, J Bartek, R Baer, J Lukas, S Jackson, Nature. 450Sartori A, Lukas C, Coates J, Mistrik M, Fu S, Bartek J, Baer R, Lukas J, Jackson S (2007) Human CtIP promotes DNA end resection. Nature 450:509-514

Ctp1/CtIP and the MRN complex collaborate in the initial steps of homologous recombination. S Takeda, K Nakamura, Y Taniguchi, T Paull, Mol Cell. 28Takeda S, Nakamura K, Taniguchi Y, Paull T (2007) Ctp1/CtIP and the MRN complex collaborate in the initial steps of homologous recombination. Mol Cell 28:351-352

BLM-DNA2-RPA-MRN and EXO1-BLM-RPA-MRN constitute two DNA end resection machineries for human DNA break repair. A Nimonkar, J Genschel, E Kinoshita, P Polaczek, J Campbell, C Wyman, P Modrich, S Kowalczykowski, Genes Dev. 25Nimonkar A, Genschel J, Kinoshita E, Polaczek P, Campbell J, Wyman C, Modrich P, Kowalczykowski S (2011) BLM-DNA2- RPA-MRN and EXO1-BLM-RPA-MRN constitute two DNA end resection machineries for human DNA break repair. Genes Dev 25:350-362

Human CtIP mediates cell cycle control of DNA end resection and double strand break repair. P Huertas, S Jackson, J Biol Chem. 264Huertas P, Jackson S (2009) Human CtIP mediates cell cycle control of DNA end resection and double strand break repair. J Biol Chem 264:9558-9565

Two classes of BRC repeats in BRCA2 promote RAD51 nucleoprotein filament function by distinct mechanisms. A Carreira, S Kowalczykowski, Proc Natl Acad Sci. 108Carreira A, Kowalczykowski S (2011) Two classes of BRC repeats in BRCA2 promote RAD51 nucleoprotein filament function by distinct mechanisms. Proc Natl Acad Sci USA 108:10448-10453

BRCA2 diffuses as oligomeric clusters with RAD51 and changes mobility after DNA damage in live cells. M Reuter, A Zelensky, I Smal, E Meijering, W Van Cappellen, H De Gruiter, G Van Belle, M Van Royen, A Houtsmuller, J Essers, R Kanaar, C Wyman, J Cell Biol. 207Reuter M, Zelensky A, Smal I, Meijering E, van Cappellen W, de Gruiter H, van Belle G, van Royen M, Houtsmuller A, Essers J, Kanaar R, Wyman C (2014) BRCA2 diffuses as oligomeric clusters with RAD51 and changes mobility after DNA damage in live cells. J Cell Biol 207:599-613

ATPdependent and independent functions of Rad54 in genome maintenance. S Agarwal, W Van Cappellen, A Guénolé, B Eppink, S Linsen, E Meijering, A Houtsmuller, R Kanaar, J Essers, J Cell Biol. 192Agarwal S, van Cappellen W, Guénolé A, Eppink B, Linsen S, Meijering E, Houtsmuller A, Kanaar R, Essers J (2011) ATP- dependent and independent functions of Rad54 in genome maintenance. J Cell Biol 192:735-750

Functions of the Snf2/Swi2 family Rad54 motor protein in homologous recombination. S Ceballos, W Heyer, Biochim Biophys Acta. 1809Ceballos S, Heyer W (2011) Functions of the Snf2/Swi2 family Rad54 motor protein in homologous recombination. Biochim Biophys Acta 1809:509-523

RAD51 paralogs: roles in DNA damage signalling, recombinational repair and tumorigenesis. N Suwaki, K Klare, M Tarsounas, Semin Cell Dev Biol. 22Suwaki N, Klare K, Tarsounas M (2011) RAD51 paralogs: roles in DNA damage signalling, recombinational repair and tumori- genesis. Semin Cell Dev Biol 22:898-905

Reconstitution of recombination-associated DNA synthesis with human proteins. J Sneeden, S Grossi, I Tappin, J Hurwitz, W Heyer, Nucleic Acids Res. 41Sneeden J, Grossi S, Tappin I, Hurwitz J, Heyer W (2013) Reconstitution of recombination-associated DNA synthesis with human proteins. Nucleic Acids Res 41:4913-4925

Hold your horSSEs: controlling structure-selective endonucleases MUS81 and Yen1/GEN1. M Blanco, J Matos, Front Genet. 6253Blanco M, Matos J (2015) Hold your horSSEs: controlling structure-selective endonucleases MUS81 and Yen1/GEN1. Front Genet 6:253

Holliday junction resolution: regulation in space and time. J Matos, S West, DNA Repair (Amst). 19Matos J, West S (2014) Holliday junction resolution: regulation in space and time. DNA Repair (Amst) 19:176-181

Is homologous recombination really an error-free process. J Guirouilh-Barbat, S Lambert, P Bertrand, B Lopez, Front Genet. 5175Guirouilh-Barbat J, Lambert S, Bertrand P, Lopez B (2014) Is homologous recombination really an error-free process? Front Genet 5:175

Is non-homologous end-joining really an inherently error-prone process?. M Bétermier, P Bertrand, B Lopez, PLoS Genet. 101004086Bétermier M, Bertrand P, Lopez B (2014) Is non-homologous end-joining really an inherently error-prone process? PLoS Genet 10:e1004086

Repair of double-strand breaks by end joining. K Chiruvella, Z Liang, T Wilson, Cold Spring Harb Perspect Biol. 512757Chiruvella K, Liang Z, Wilson T (2013) Repair of double-strand breaks by end joining. Cold Spring Harb Perspect Biol 5:a012757

Non-homologous end joining: emerging themes and unanswered questions. S Radhakrishnan, N Jette, S Lees-Miller, DNA Repair (Amst). 17Radhakrishnan S, Jette N, Lees-Miller S (2014) Non-homolo- gous end joining: emerging themes and unanswered questions. DNA Repair (Amst) 17:2-8

Nonhomologous end joining: a good solution for bad ends. C Waters, N Strande, D Wyatt, J Pryor, D Ramsden, DNA Repair (Amst). 17Waters C, Strande N, Wyatt D, Pryor J, Ramsden D (2014) Nonhomologous end joining: a good solution for bad ends. DNA Repair (Amst) 17:39-51

Structural insights into NHEJ: building up an integrated picture of the dynamic DSB repair super complex, one component and interaction at a time. G Williams, M Hammel, S Radhakrishnan, D Ramsden, S Lees-Miller, J Tainer, DNA Repair (Amst). 17Williams G, Hammel M, Radhakrishnan S, Ramsden D, Lees- Miller S, Tainer J (2014) Structural insights into NHEJ: building up an integrated picture of the dynamic DSB repair super complex, one component and interaction at a time. DNA Repair (Amst) 17:110-120

A new method for highresolution imaging of Ku foci to decipher mechanisms of DNA double-strand break repair. S Britton, J Coates, S Jackson, J Cell Biol. 202Britton S, Coates J, Jackson S (2013) A new method for high- resolution imaging of Ku foci to decipher mechanisms of DNA double-strand break repair. J Cell Biol 202:579-595

A structural model for regulation of NHEJ by DNA-PKcs autophosphorylation. T Dobbs, J Tainer, S Lees-Miller, DNA Repair (Amst). 9Dobbs T, Tainer J, Lees-Miller S (2010) A structural model for regulation of NHEJ by DNA-PKcs autophosphorylation. DNA Repair (Amst) 9:1307-1314

Protecting DNA from errors and damage 1707. Protecting DNA from errors and damage 1707

DNA-PK: a dynamic enzyme in a versatile DSB repair pathway. A Davis, B Chen, D Chen, DNA Repair (Amst). 17Davis A, Chen B, Chen D (2014) DNA-PK: a dynamic enzyme in a versatile DSB repair pathway. DNA Repair (Amst) 17:21-29

The Ku heterodimer: function in DNA repair and beyond. V Fell, C Schild-Poulter, Mutat Res Rev Mutat Res. 763Fell V, Schild-Poulter C (2015) The Ku heterodimer: function in DNA repair and beyond. Mutat Res Rev Mutat Res 763:15-29

The DNA-dependent protein kinase: a multifunctional protein kinase with roles in DNA double strand break repair and mitosis. N Jette, S Lees-Miller, Prog Biophys Mol Biol. 117Jette N, Lees-Miller S (2015) The DNA-dependent protein kinase: a multifunctional protein kinase with roles in DNA double strand break repair and mitosis. Prog Biophys Mol Biol 117:194-205

Ku is a 5 0 -dRP/AP lyase that excises nucleotide damage near broken ends. S Roberts, N Strande, M Burkhalter, C Strom, J Havener, P Hasty, D Ramsden, Nature. 464Roberts S, Strande N, Burkhalter M, Strom C, Havener J, Hasty P, Ramsden D (2010) Ku is a 5 0 -dRP/AP lyase that excises nucleotide damage near broken ends. Nature 464:1214-1217

Specificity of the dRP/AP lyase of Ku promotes nonhomologous end joining (NHEJ) fidelity at damaged ends. N Strande, S Roberts, S Oh, E Hendrickson, D Ramsden, J Biol Chem. 287Strande N, Roberts S, Oh S, Hendrickson E, Ramsden D (2012) Specificity of the dRP/AP lyase of Ku promotes nonhomologous end joining (NHEJ) fidelity at damaged ends. J Biol Chem 287:13686-13693

. Y Pommier, S Huang, R Gao, B Das, J Murai, C Marchand, Tyrosyl-DNA-phosphodiesterases (TDP1 and TDP2Pommier Y, Huang S, Gao R, Das B, Murai J, Marchand C (2014) Tyrosyl-DNA-phosphodiesterases (TDP1 and TDP2).

. DNA Repair (Amst). 19DNA Repair (Amst) 19:114-129

Structural basis for the phosphatase activity of polynucleotide kinase/phosphatase on single and double-stranded DNA substrates. N Coquelle, Z Havali-Shahriari, N Bernstein, R Green, J Glover, Proc Natl Acad Sci. 108Coquelle N, Havali-Shahriari Z, Bernstein N, Green R, Glover J (2011) Structural basis for the phosphatase activity of polynu- cleotide kinase/phosphatase on single and double-stranded DNA substrates. Proc Natl Acad Sci USA 108:21022-21027

The structural basis for substrate recognition by mammalian polynucleotide kinase 3 0 phosphatase. F Garces, L Pearl, A Oliver, Mol Cell. 44Garces F, Pearl L, Oliver A (2011) The structural basis for substrate recognition by mammalian polynucleotide kinase 3 0 phosphatase. Mol Cell 44:385-396

PAXX, a paralog of XRCC4 and XLF, interacts with Ku to promote DNA double-strand break repair. T Ochi, A Blackford, J Coates, S Jhujh, S Mehmood, N Tamura, J Travers, Q Wu, V Draviam, C Robinson, T Blundell, S Jackson, Science. 347Ochi T, Blackford A, Coates J, Jhujh S, Mehmood S, Tamura N, Travers J, Wu Q, Draviam V, Robinson C, Blundell T, Jackson S (2015) PAXX, a paralog of XRCC4 and XLF, interacts with Ku to promote DNA double-strand break repair. Science 347:185-188

Microhomology-mediated end joining and homologous recombination share the initial end resection step to repair DNA double-strand breaks in mammalian cells. L Truong, Y Li, L Shi, P Hwang, J He, H Wang, N Razavian, M Berns, X Wu, Proc Natl Acad Sci. 110Truong L, Li Y, Shi L, Hwang P, He J, Wang H, Razavian N, Berns M, Wu X (2013) Microhomology-mediated end joining and homologous recombination share the initial end resection step to repair DNA double-strand breaks in mammalian cells. Proc Natl Acad Sci USA 110:7720-7725

Alternative end-joining mechanisms: a historical perspective. A Decottignies, Front Genet. 448Decottignies A (2013) Alternative end-joining mechanisms: a historical perspective. Front Genet 4:48

Alternative end-joining pathway(s): bricolage at DNA breaks. P Frit, N Barboule, Y Yuan, D Gomez, P Calsou, DNA Repair (Amst). 17Frit P, Barboule N, Yuan Y, Gomez D, Calsou P (2014) Alternative end-joining pathway(s): bricolage at DNA breaks. DNA Repair (Amst) 17:81-97

Microhomology-mediated end joining: a back-up survival mechanism or dedicated pathway?. A Sfeir, L Symington, Trends Biochem Sci. 40Sfeir A, Symington L (2015) Microhomology-mediated end joining: a back-up survival mechanism or dedicated pathway? Trends Biochem Sci 40:701-714

Ku counteracts mobilization of PARP1 and MRN in chromatin damaged with DNA double-strand breaks. Q Cheng, N Barboule, P Frit, D Gomez, O Bombarde, B Couderc, G Ren, B Salles, P Calsou, Nucleic Acids Res. 39Cheng Q, Barboule N, Frit P, Gomez D, Bombarde O, Couderc B, Ren G, Salles B, Calsou P (2011) Ku counteracts mobiliza- tion of PARP1 and MRN in chromatin damaged with DNA double-strand breaks. Nucleic Acids Res 39:9605-9619

PARP-1 and Ku compete for repair of DNA double strand breaks by distinct NHEJ pathways. M Wang, W Wu, W Wu, B Rosidi, L Zhang, H Wang, G Iliakis, Nucleic Acids Res. 34Wang M, Wu W, Wu W, Rosidi B, Zhang L, Wang H, Iliakis G (2006) PARP-1 and Ku compete for repair of DNA double strand breaks by distinct NHEJ pathways. Nucleic Acids Res 34:6170-6182

Automodification switches PARP-1 function from chromatin architectural protein to histone chaperone. U Muthurajan, M Hepler, A Hieb, N Clark, M Kramer, T Yao, K Luger, Proc Natl Acad Sci. 111Muthurajan U, Hepler M, Hieb A, Clark N, Kramer M, Yao T, Luger K (2014) Automodification switches PARP-1 function from chromatin architectural protein to histone chaperone. Proc Natl Acad Sci USA 111:12752-12757

Dynamics of DNA damage response proteins at DNA breaks: a focus on protein modifications. S Polo, S Jackson, Genes Dev. 25Polo S, Jackson S (2011) Dynamics of DNA damage response proteins at DNA breaks: a focus on protein modifications. Genes Dev 25:409-433

Poly(ADP-ribose) polymerases in double-strand break repair: focus on PARP1, PARP2 and PARP3. C Beck, I Robert, B Reina-San-Martin, V Schreiber, F Dantzer, Exp Cell Res. 329Beck C, Robert I, Reina-San-Martin B, Schreiber V, Dantzer F (2014) Poly(ADP-ribose) polymerases in double-strand break repair: focus on PARP1, PARP2 and PARP3. Exp Cell Res 329:18-25

Functional aspects of PARP1 in DNA repair and transcription. H Ko, E Ren, Biomolecules. 2Ko H, Ren E (2012) Functional aspects of PARP1 in DNA repair and transcription. Biomolecules 2:524-548

Touching base with PARPs: moonlighting in the repair of UV lesions and double-strand breaks. A Pines, L Mullenders, H Van Attikum, M Luijsterburg, Trends Biochem Sci. 38172. Pines A, Mullenders L, van Attikum H, Luijsterburg M (2013) Touching base with PARPs: moonlighting in the repair of UV lesions and double-strand breaks. Trends Biochem Sci 38:321-330

PARP is activated at stalled forks to mediate Mre11-dependent replication restart and recombination. H Bryant, E Petermann, N Schultz, A Jemth, O Loseva, N Issaeva, F Johansson, S Fernandez, P Mcglynn, T Helleday, EMBO J. 28Bryant H, Petermann E, Schultz N, Jemth A, Loseva O, Issaeva N, Johansson F, Fernandez S, McGlynn P, Helleday T (2009) PARP is activated at stalled forks to mediate Mre11-dependent replication restart and recombination. EMBO J 28:2601-2615

PARP1-dependent kinetics of recruitment of MRE11 and NBS1 proteins to multiple DNA damage sites. J Haince, D Mcdonald, A Rodrigue, U Dery, J Masson, M Hendzel, G Poirier, J Biol Chem. 283Haince J, McDonald D, Rodrigue A, Dery U, Masson J, Hendzel M, Poirier G (2008) PARP1-dependent kinetics of recruitment of MRE11 and NBS1 proteins to multiple DNA damage sites. J Biol Chem 283:1197-1208

Human Mre11/human Rad50/Nbs1 and DNA ligase IIIalpha/XRCC1 protein complexes act together in an alternative nonhomologous end joining pathway. J Della-Maria, Y Zhou, M Tsai, J Kuhnlein, J Carney, T Paull, A Tomkinson, J Biol Chem. 286Della-Maria J, Zhou Y, Tsai M, Kuhnlein J, Carney J, Paull T, Tomkinson A (2011) Human Mre11/human Rad50/Nbs1 and DNA ligase IIIalpha/XRCC1 protein complexes act together in an alternative nonhomologous end joining pathway. J Biol Chem 286:33845-33853

DNA ligase III promotes alternative nonhomologous end-joining during chromosomal translocation formation. D Simsek, E Brunet, S-W Wong, S Katyal, Y Gao, P Mckinnon, J Lou, L Zhang, J Li, E Rebar, P Gregory, M Holmes, M Jasin, PLoS Genet. 71002080Simsek D, Brunet E, Wong S-W, Katyal S, Gao Y, McKinnon P, Lou J, Zhang L, Li J, Rebar E, Gregory P, Holmes M, Jasin M (2011) DNA ligase III promotes alternative nonhomologous end-joining during chromosomal translocation formation. PLoS Genet 7:e1002080

Enabling plant synthetic biology through genome engineering. N Baltes, D Voytas, Trends Biotechnol. 33Baltes N, Voytas D (2015) Enabling plant synthetic biology through genome engineering. Trends Biotechnol 33:120-131

DNA recombination in somatic plant cells: mechanisms and evolutionary consequences. A Knoll, F Fauser, H Puchta, Chromosome Res. 22Knoll A, Fauser F, Puchta H (2014) DNA recombination in somatic plant cells: mechanisms and evolutionary consequences. Chromosome Res 22:191-201

ZFN, TALEN, and CRISPR/Cas-based methods for genome engineering. T Gaj, C Gersbach, Cr Barbas, Trends Biotechnol. 31Gaj T, Gersbach C, Cr Barbas (2013) ZFN, TALEN, and CRISPR/Cas-based methods for genome engineering. Trends Biotechnol 31:397-405

Plant genome engineering with sequencespecific nucleases. D Voytas, Annu Rev Plant Biol. 64Voytas D (2013) Plant genome engineering with sequence- specific nucleases. Annu Rev Plant Biol 64:327-350

Gene targeting and transgene stacking using intra genomic homologous recombination in plants. S Kumar, P Barone, M Smith, Plant Methods. 1211Kumar S, Barone P, Smith M (2016) Gene targeting and transgene stacking using intra genomic homologous recombi- nation in plants. Plant Methods 12:11

Synthetic nucleases for genome engineering in plants: prospects for a bright future. H Puchta, F Fauser, Plant J. 78Puchta H, Fauser F (2014) Synthetic nucleases for genome engineering in plants: prospects for a bright future. Plant J 78:727-741

Plant genome editing by novel tools: tALEN and other sequence specific nucleases. T Sprink, J Metje, F Hartung, Curr Opin Biotechnol. 32Sprink T, Metje J, Hartung F (2015) Plant genome editing by novel tools: tALEN and other sequence specific nucleases. Curr Opin Biotechnol 32:47-53

NBS1 is involved in DNA repair and plays a synergistic role with ATM in mediating meiotic homologous recombination in plants. W Waterworth, C Altun, S Armstrong, N Roberts, P Dean, K Young, C Weil, C Bray, C West, Plant J. 52Waterworth W, Altun C, Armstrong S, Roberts N, Dean P, Young K, Weil C, Bray C, West C (2007) NBS1 is involved in DNA repair and plays a synergistic role with ATM in mediating meiotic homologous recombination in plants. Plant J 52:41-52

A novel plant gene essential for meiosis is related to the human CtIP and the yeast COM1/SAE2 gene. C Uanschou, T Siwiec, A Pedrosa-Harand, C Kerzendorfer, E Sanchez-Moran, M Novatchkova, S Akimcheva, A Woglar, F Klein, P Schlögelhofer, EMBO J. 26Uanschou C, Siwiec T, Pedrosa-Harand A, Kerzendorfer C, Sanchez-Moran E, Novatchkova M, Akimcheva S, Woglar A, Klein F, Schlögelhofer P (2007) A novel plant gene essential for meiosis is related to the human CtIP and the yeast COM1/SAE2 gene. EMBO J 26:5061-5070

Characterization of the plant homolog of Nijmegen breakage syndrome 1: involvement in DNA repair and recombination. N Akutsu, K Iijima, T Hinata, H Tauchi, Biochem Biophys Res Commun. 353Akutsu N, Iijima K, Hinata T, Tauchi H (2007) Characterization of the plant homolog of Nijmegen breakage syndrome 1: involvement in DNA repair and recombination. Biochem Bio- phys Res Commun 353:394-398

Two closely related RecQ helicases have antagonistic roles in homologous recombination and DNA repair in Arabidopsis thaliana. F Hartung, S Suer, H Puchta, Proc Natl Acad Sci. 104Hartung F, Suer S, Puchta H (2007) Two closely related RecQ helicases have antagonistic roles in homologous recombination and DNA repair in Arabidopsis thaliana. Proc Natl Acad Sci USA 104:18836-18841

RAD5A, RECQ4A, and MUS81 have specific functions in homologous recombination and define different pathways of DNA repair in Arabidopsis thaliana. A Mannuss, S Dukowic-Schulze, S Suer, F Hartung, M Pacher, H Puchta, Plant Cell. 22Mannuss A, Dukowic-Schulze S, Suer S, Hartung F, Pacher M, Puchta H (2010) RAD5A, RECQ4A, and MUS81 have specific functions in homologous recombination and define different pathways of DNA repair in Arabidopsis thaliana. Plant Cell 22:3318-3330

Defining the roles of the N-terminal region and the helicase activity of RECQ4A in DNA repair and homologous recombination in Arabidopsis. S Schröpfer, D Kobbe, F Hartung, A Knoll, H Puchta, Nucleic Acids Res. 42Schröpfer S, Kobbe D, Hartung F, Knoll A, Puchta H (2014) Defining the roles of the N-terminal region and the helicase activity of RECQ4A in DNA repair and homologous recombi- nation in Arabidopsis. Nucleic Acids Res 42:1684-1697

A DNA2 homolog is required for DNA damage repair, cell cycle regulation, and meristem maintenance in plants. N Jia, X Liu, H Gao, Plant Physiol. 171Jia N, Liu X, Gao H (2016) A DNA2 homolog is required for DNA damage repair, cell cycle regulation, and meristem maintenance in plants. Plant Physiol 171:318-333

Effects of XRCC2 and RAD51B mutations on somatic and meiotic recombination in Arabidopsis thaliana. O Da Ines, F Degroote, S Amiard, C Goubely, M Gallego, C White, Plant J. 74Da Ines O, Degroote F, Amiard S, Goubely C, Gallego M, White C (2013) Effects of XRCC2 and RAD51B mutations on somatic and meiotic recombination in Arabidopsis thaliana. Plant J 74:959-970

The requirement for recombination factors differs considerably between different pathways of homologous double-strand break repair in somatic plant cells. N Roth, J Klimesch, S Dukowic-Schulze, M Pacher, A Mannuss, H Puchta, Plant J. 72Roth N, Klimesch J, Dukowic-Schulze S, Pacher M, Mannuss A, Puchta H (2012) The requirement for recombination factors differs considerably between different pathways of homologous double-strand break repair in somatic plant cells. Plant J 72:781-790

Roles of XRCC2, RAD51B and RAD51D in RAD51-independent SSA recombination. H Serra, O Da Ines, F Degroote, M Gallego, C White, PLoS Genet. 91003971Serra H, Da Ines O, Degroote F, Gallego M, White C (2013) Roles of XRCC2, RAD51B and RAD51D in RAD51-indepen- dent SSA recombination. PLoS Genet 9:e1003971

The Arabidopsis RAD51 paralogs RAD51B, RAD51D and XRCC2 play partially redundant roles in somatic DNA repair and gene regulation. Y Wang, R Xiao, H Wang, Z Cheng, W Li, G Zhu, Y Wang, H Ma, New Phytol. 201Wang Y, Xiao R, Wang H, Cheng Z, Li W, Zhu G, Wang Y, Ma H (2014) The Arabidopsis RAD51 paralogs RAD51B, RAD51D and XRCC2 play partially redundant roles in somatic DNA repair and gene regulation. New Phytol 201:292-304

Genome stability of Arabidopsis atm, ku80 and rad51b mutants: somatic and transgenerational responses to stress. Y Yao, A Bilichak, V Titov, A Golubov, I Kovalchuk, Plant Cell Physiol. 54Yao Y, Bilichak A, Titov V, Golubov A, Kovalchuk I (2013) Genome stability of Arabidopsis atm, ku80 and rad51b mutants: somatic and transgenerational responses to stress. Plant Cell Physiol 54:982-989

Inefficient double-strand DNA break repair is associated with increased fasciation in Arabidopsis BRCA2 mutants. K Abe, K Osakabe, Y Ishikawa, A Tagiri, H Yamanouchi, T Takyuu, T Yoshioka, T Ito, M Kobayashi, K Shinozaki, H Ichikawa, S Toki, J Exp Bot. 60Abe K, Osakabe K, Ishikawa Y, Tagiri A, Yamanouchi H, Takyuu T, Yoshioka T, Ito T, Kobayashi M, Shinozaki K, Ichikawa H, Toki S (2009) Inefficient double-strand DNA break repair is associated with increased fasciation in Arabidopsis BRCA2 mutants. J Exp Bot 60:2751-2761

BRCA2 is a mediator of RAD51-and DMC1-facilitated homologous recombination in Arabidopsis thaliana. K Seeliger, S Dukowic-Schulze, R Wurz-Wildersinn, M Pacher, H Puchta, New Phytol. 193Seeliger K, Dukowic-Schulze S, Wurz-Wildersinn R, Pacher M, Puchta H (2012) BRCA2 is a mediator of RAD51-and DMC1- facilitated homologous recombination in Arabidopsis thaliana. New Phytol 193:364-375

Homologs of breast cancer genes in plants. O Trapp, K Seeliger, H Puchta, Front Plant Sci. 219Trapp O, Seeliger K, Puchta H (2011) Homologs of breast cancer genes in plants. Front Plant Sci 2:19

A SRS2 homolog from Arabidopsis thaliana disrupts recombinogenic DNA intermediates and facilitates single strand annealing. S Blanck, D Kobbe, F Hartung, K Fengler, M Focke, H Puchta, Nucleic Acids Res. 37Blanck S, Kobbe D, Hartung F, Fengler K, Focke M, Puchta H (2009) A SRS2 homolog from Arabidopsis thaliana disrupts recombinogenic DNA intermediates and facilitates single strand annealing. Nucleic Acids Res 37:7163-7176

AtGEN1 and AtSEND1, two paralogs in Arabidopsis, possess holliday junction resolvase activity. M Bauknecht, D Kobbe, Plant Physiol. 166Bauknecht M, Kobbe D (2014) AtGEN1 and AtSEND1, two paralogs in Arabidopsis, possess holliday junction resolvase activity. Plant Physiol 166:202-216

Two distinct MUS81-EME1 complexes from Arabidopsis process Holliday junctions. V Geuting, D Kobbe, F Hartung, J Dürr, M Focke, H Puchta, Plant Physiol. 150Geuting V, Kobbe D, Hartung F, Dürr J, Focke M, Puchta H (2009) Two distinct MUS81-EME1 complexes from Arabidop- sis process Holliday junctions. Plant Physiol 150:1062-1071

Topoisomerase 3alpha and RMI1 suppress somatic crossovers and are essential for resolution of meiotic recombination intermediates in Arabidopsis thaliana. F Hartung, S Suer, A Knoll, R Wurz-Wildersinn, H Puchta, PLoS Genet. 41000285Hartung F, Suer S, Knoll A, Wurz-Wildersinn R, Puchta H (2008) Topoisomerase 3alpha and RMI1 suppress somatic crossovers and are essential for resolution of meiotic recombi- nation intermediates in Arabidopsis thaliana. PLoS Genet 4:e1000285

PARP2 is the predominant Poly(ADP-Ribose) polymerase in Arabidopsis DNA damage and immune responses. J Song, B Keppler, R Wise, A Bent, PLoS Genet. 111005200Song J, Keppler B, Wise R, Bent A (2015) PARP2 is the pre- dominant Poly(ADP-Ribose) polymerase in Arabidopsis DNA damage and immune responses. PLoS Genet 11:e1005200

Kinetic analysis of DNA double-strand break repair pathways in Arabidopsis. C Charbonnel, E Allain, M Gallego, C White, DNA Repair (Amst). 10Charbonnel C, Allain E, Gallego M, White C (2011) Kinetic analysis of DNA double-strand break repair pathways in Ara- bidopsis. DNA Repair (Amst) 10:611-619

Xrcc1-dependent and Ku-dependent DNA double-strand break repair kinetics in Arabidopsis plants. C Charbonnel, M Gallego, C White, Plant J. 64Charbonnel C, Gallego M, White C (2010) Xrcc1-dependent and Ku-dependent DNA double-strand break repair kinetics in Arabidopsis plants. Plant J 64:280-290

Poly(ADP-ribose)polymerases are involved in microhomology mediated back-up non-homologous end joining in Arabidopsis thaliana. Q Jia, A Den Dulk-Ras, H Shen, P Hooykaas, S De Pater, Plant Mol Biol. 82Jia Q, den Dulk-Ras A, Shen H, Hooykaas P, de Pater S (2013) Poly(ADP-ribose)polymerases are involved in microhomology mediated back-up non-homologous end joining in Arabidopsis thaliana. Plant Mol Biol 82:339-351

Suppression of Ku70/80 or Lig4 leads to decreased stable transformation and enhanced homologous recombination in rice. A Nishizawa-Yokoi, S Nonaka, H Saika, Y Kwon, K Osakabe, S Toki, New Phytol. 196Nishizawa-Yokoi A, Nonaka S, Saika H, Kwon Y, Osakabe K, Toki S (2012) Suppression of Ku70/80 or Lig4 leads to decreased stable transformation and enhanced homologous recombination in rice. New Phytol 196:1048-1059

XRCC3 is essential for proper double-strand break repair and homologous recombination in rice meiosis. B Zhang, M Wang, D Tang, Y Li, M Xu, M Gu, Z Cheng, H Yu, J Exp Bot. 66Zhang B, Wang M, Tang D, Li Y, Xu M, Gu M, Cheng Z, Yu H (2015) XRCC3 is essential for proper double-strand break repair and homologous recombination in rice meiosis. J Exp Bot 66:5713-5725

Enhanced osmotic stress tolerance in Medicago truncatula plants overexpressing the DNA repair gene MtTdp2a (tyrosyl-DNA phosphodiesterase 2). M Confalonieri, M Faè, A Balestrazzi, M Donà, A Macovei, A Valassi, G Giraffa, D Carbonera, Plant Cell Tiss Organ Cult. 116Confalonieri M, Faè M, Balestrazzi A, Donà M, Macovei A, Valassi A, Giraffa G, Carbonera D (2014) Enhanced osmotic stress tolerance in Medicago truncatula plants overexpressing the DNA repair gene MtTdp2a (tyrosyl-DNA phosphodiesterase 2). Plant Cell Tiss Organ Cult 116:187-203

Overexpression of OsRecQl4 and/or OsExo1 enhances DSB-induced homologous recombination in rice. Y Kwon, K Abe, K Osakabe, M Endo, A Nishizawa-Yokoi, H Saika, H Shimada, S Toki, Plant Cell Physiol. 53Kwon Y, Abe K, Osakabe K, Endo M, Nishizawa-Yokoi A, Saika H, Shimada H, Toki S (2012) Overexpression of OsRecQl4 and/or OsExo1 enhances DSB-induced homologous recombination in rice. Plant Cell Physiol 53:2142-2152

Recent progress in research on DNA damage responsesin animals and plants. K Yoshiyama, Genes Genet Syst. 90Yoshiyama K (2016) Recent progress in research on DNA damage responsesin animals and plants. Genes Genet Syst 90:185-186