# Prospects & Overviews Review essays

CorpusID: 3780728
 
tags: #Medicine, #Biology

URL: [https://www.semanticscholar.org/paper/383416eae2a40bb92e2aed66b5e5e38629846a77](https://www.semanticscholar.org/paper/383416eae2a40bb92e2aed66b5e5e38629846a77)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

Prospects & Overviews Review essays


Claire Rougeulle claire.rougeulle@univ-paris-diderot.fr 
Epigenetics and Cell Fate
Sorbonne Paris Cit e
Universit e Paris Diderot
ParisFrance

Prospects & Overviews Review essays

Bioessays
3810.1002/bies.201600121*Corresponding author:d evelopmentdifferentiationheterochromatin landscapenon-coding RNApluripotencystem cellsX chromosome inactivation RNATGC, trophoblast giant cellTS, trophoblast stem cellXa, active X chromosomeXCR, X chromosome reactivationXi, inactive X chromosomeXp, paternal X chromosome
Establishment of X chromosome inactivation and epigenomic features of the inactive X depend on cellular contexts C eline Vallot, Jean-François Ouimette and Claire Rougeulle Ã X chromosome inactivation (XCI) is an essential epigenetic process that ensures X-linked gene dosage equilibrium between sexes in mammals. XCI is dynamically regulated during development in a manner that is intimately linked to differentiation. Numerous studies, which we review here, have explored the dynamics of X inactivation and reactivation in the context of development, differentiation and diseases, and the phenotypic and molecular link between the inactive status, and the cellular context. Here, we also assess whether XCI is a uniform mechanism in mammals by analyzing epigenetic signatures of the inactive X (Xi) in different species and cellular contexts. It appears that the timing of XCI and the epigenetic signature of the inactive X greatly vary between species. Surprisingly, even within a given species, various Xi configurations are found across cellular states. We discuss possible mechanisms underlying these variations, and how they might influence the fate of the Xi.Abbreviations: (h/m)ESC, (human/mouse) embryonic stem cell; (r/i)XCI, (random/imprinted) X chromosome inactivation; ICM, inner cell mass; lncRNA, long non-coding

## Introduction

Dosage disequilibrium introduced by the heteromorphic nature of the X and Y sex chromosomes is compensated by mechanisms that specifically target the X chromosome and modulate its transcriptional competency. The first hint of male/female chromatin difference in mammals was cytological evidence of a nuclear body specific to female somatic cells, referred to as the Barr body [1], and identified as a condensed X chromosome [2]. The formation of this sex chromatin was proposed by Mary Lyon to arise early during development and to result in the transcriptional silencing of one the two Xs, in females only [3]. Balancing X-linked gene products in females through X chromosome inactivation (XCI) is essential for proper development, and failure to do so blocks embryogenesis shortly after implantation [4,5]. In adults, presence of two active X (Xa) chromosomes in somatic cells has been observed mainly in deleterious cell states such as cancer cells [6,7], and loss of X-linked gene dosage compensation is believed to cause aggressive hematological cancers [8].

XCI is triggered by chromosome-wide coating of the long non-coding RNA (lncRNA) Xist and characterized by profound changes of the epigenetic landscape of the inactive X chromosome (Xi) with respect to its active counterpart. Molecular events downstream of XIST accumulation and leading to stable silencing include eviction of RNA polymerases, accumulation of specific histones modifications and variants, changes in DNA methylation profiles, switch of replication timing, and reallocation of the Xi to specific nuclear compartments. XCI is a striking example of facultative heterochromatin formation. In this case, the facultative nature of the heterochromatin, as opposed to the constitutive one, refers primarily to the fact that one X chromosome is packaged into condensed chromatin while its homolog is organized into open, transcriptionally active euchromatin. As XCI is random in most species, the parental origin of the inactive X can vary from cell-to-cell, so that the maternal X can be inactive in one cell and active in the other. In addition, although the inactive status is remarkably stable throughout adult life, it fluctuates during the early life cycle, where the X chromosome undergoes waves of inactivation and reactivation. In fact, XCI depends on a highly wired and dynamic network that connects the process of inactivation to the cellular state. The epigenetic landscape of the inactive chromatin may also vary from one developmental stage to another, and from one species to the next, influencing the stability of XCI and the probability of X-linked genes to escape from this inactivation. Here, we will discuss the developmental regulation of XCI and how this process is connected to cellular states, both within and across organisms. We will also review the variety of epigenetic signatures of the inactive X chromosome, and how they relate to cellular contexts and to the stability of the inactive state in eutherians as well as non-eutherian species.

X chromosome inactivation and reactivation are dynamically regulated in physiological and pathological contexts XCI initiates early in development, concomitantly to cellular differentiation Dosage compensation of sex chromosomes is absolutely required for mammalian development to progress properly. The presence of two active X chromosomes appears to be tolerated in some contexts, which may vary from one species to the other, but these usually correspond to undifferentiated or poorly differentiated states. The first hint that XCI was coupled to cell differentiation came from biochemical analysis of X-linked enzymatic activities in early mouse embryonic stages, which highlighted different XCI kinetics in extraembryonic versus embryonic tissues that follows the differentiation timing of these lineages [9].

Further analysis of early cleavage stages revealed that XCI could initiate prior to implantation, at least in the mouse (Fig. 1A). This inactivation, which begins around the four-cell stage, is imprinted: the paternal X (Xp) is preferentially inactivated. Inactivation of the Xp persists up to the early blastocyst stage and is preserved in throphectoderm (TE), and primitive endoderm (PrE) derivatives [10][11][12][13][14]. Imprinted XCI is, however, reversed in the pluripotent cells of early to midblastocysts, leading to both X chromosomes being active in those cells [9,[15][16][17][18]. As cells of the inner cell mass (ICM) progress to the epiblast stage and enter differentiation, random XCI (rXCI) is rapidly triggered, and leads to the inactivation of either the maternal or paternal X chromosome. This second wave of inactivation is completed in most cells of the E6.5 epiblast, and possible timing differences depend on the cell lineage [13,19,20]. Once established, rXCI is stably maintained in somatic cells.

In contrast to the mouse, rabbit, and human preimplantation development proceeds in the absence of XCI (Fig. 1A). Assessing the allelic expression of candidate X-linked genes by RNA-FISH on rabbit embryos, Okamoto et al. showed that XCI initiates at the blastocyst stage, and may progress through a transient state where both Xs are inactivated, a situation that is rapidly resolved. In humans, both X chromosomes are also active from the zygotic genome activation up to peri-implantation stages, when rXCI is believed to be initiated [21]. This confirms earlier studies showing that human, and also cat and macaque embryos, display the first cytological signs of an inactive X (Xi) at the late blastocysts to epiblast stage [22,23]. Intriguingly however, in humans XIST is expressed and accumulates on X chromosomes several divisions prior to the onset of XCI, so that XIST coating and XCI are transiently uncoupled in this species.

Thus, although the different mammals appear to display variable tolerance for X chromosome activity at pre-implantation stages, XCI is likely robustly and stably established soon after implantation, concomitantly to cellular differentiation. Further investigation in a broader range of species is, however, necessary to assess the consistency of this rule.

To what extent is XCI coupled to differentiation in vitro?

Ex vivo cellular models have been very powerful to decipher mechanisms of XCI initiation, and cell lines derived from embryonic contexts further confirmed the coupling of XCI to cell differentiation. The first cell line used was embryonal carcinoma (EC) female cells, derived from teratocarcinomas, which bear characteristics of blastocyst cells [24,25]. In these cells, the activity of X-linked enzymes is reduced twofold after differentiation, an indication that cell differentiation is accompanied by XCI. Mouse pluripotent, embryonic stem cells (ESC), however, rapidly became the model of reference for the study of rXCI (Fig. 1B). Female mESC display two active X chromosomes and recapitulate rXCI when differentiation is triggered, similarly to the cells of the ICM from where they are derived. Multiple metastable states of pluripotency have been described, namely ground/na€ ıve and primed, depending on the embryonic stage from which cells are derived, and related to their capability to give rise to blastocyst chimeras [26]. ESCs in the mouse are found in ground/na€ ıve states. In contrast, mouse epiblast stem cells (EpiSC), derived from epiblast stage embryos, where XCI has already been triggered, are in a XaXi status, and are considered to be in the primed state ( Fig. 1B) [27]. Based on studies in the mouse, ground pluripotency has been defined by the presence of two active X chromosomes, while primed cells have undergone XCI.

In contrast to the mouse, most human ESC (hESC) are found in the primed state, in which XCI has already been initiated (Fig. 1B). Their transcriptomic profiles and the signaling pathways ensuring their maintenance indeed positioned them closer to mouse EpiSC than to mouse ESCs [27][28][29]. Female hESCs, however, display a surprising XCI instability, which has not been reported so far for mEpiSC. Female hESCs undergo a process of erosion of XCI, characterized by the loss of some of the hallmarks of the inactive state. Erosion of XCI also leads to a certain degree of X chromosome reactivation (XCR) [30,31], which could argue in favor of the necessity for double dose of certain X-linked genes to sustain pluripotency in humans.

Numerous studies have tackled the issue of stably propagating hESC in the ground/na€ ıve state, which could favor retention of the pre-XCI status in female cells. Precise assessment of XCI status in human pluripotent cells is, however, complicated by the possible confusion between preinactive and eroded state, and further work is needed to characterize the pre-inactive state in humans and to establish robust protocols for the maintenance of pre-XCI female hESC. However, in the few cases in which pre-inactive hESC were reported, their differentiation was accompanied by inactivation of one of the two Xs [32]. ESCs are thus, a powerful model to study XCI and to assess its link to cell differentiation. No doubt that the availability of ESC lines from other mammalian species will prove extremely useful to further explore this connection and to unravel the underlying mechanisms across mammals.


## X chromosome reactivation

The progression to a differentiated state is accompanied by the inactivation of one X chromosome in female. Hence reprogramming toward pluripotency generally leads to the reactivation of the inactive X, both in vivo and ex vivo [33]. The first occurrence of developmental XCR takes place in the zygote, after the fusion of the highly specialized germ cells, the spermatozoa and oocyte, and concomitantly to zygotic genome activation (Fig. 1A). Indeed, the paternal X is inactive in the sperm, an effect that results from meiotic silencing of un-synapsed sex chromosomes that occurs during spermatogenesis (meiotic sex chromosome inactivation, MSCI). However, Figure 1. Contexts for X chromosome in-and reactivation in vivo and ex vivo. A: Developmental timing of X chromosome inactivation and reactivation in human, mouse, and rabbit embryos of different stages (active and inactive X chromosomes are depicted in green and red, respectively). XIST RNA expression is schematized by the colored background. Imprinted XCI (iXCI) is observed in preimplantation stages and extraembryonic tissues in the mouse, while random XCI (rXCI) is observed at later stages of mouse development, and in human and rabbit. The X chromosome is reactivated in primordial germ cells (green halo in embryo and fetus). The X chromosome is inactive in the sperm due to meiotic sex chromosome inactivation (MSCI). B: Patterns of XCI in mouse (top) and human (bottom) ex vivo cellular models. C: Genetic and epigenetic instability leading to abnormal XCI figures in cancer cells. the paternal X undergoes a transient phase of activity in the zygote before imprinted XCI is established [34][35][36]. A second wave of XCR is observed in the early mouse blastocyst, when the paternal X is reactivated prior to the establishment of rXCI, but this might be specific to some mammalian species undergoing imprinted XCI, such as rodents, and possibly bovines (Fig. 1A). XCR also occurs later in a subset of epiblast cells that dedifferentiate to give rise to primordial germ cell (PGCs), before meiosis and prior to sex specific differentiation of the germline (Fig. 1A) [37][38][39][40][41].

XCR can also be induced ex vivo during reprogramming of differentiated cells toward pluripotency. Nuclear transfer, cell fusion, or ectopic expression of reprogramming factors is accompanied by the reactivation of the inactive X, at least in mouse cells [42][43][44][45][46]. Whether Xi reactivation occurs when human differentiated cells are reprogrammed is still a matter of debate, but this might be linked to the difficulty of deriving and maintaining human pluripotent stem cells with two active X chromosomes. This emphasizes the importance of understanding the mechanisms governing both pluripotency and XCI reactivation in human, as further clinical applications of disease modeling in female human induced pluripotent stem cells might be influenced by this incomplete reprogramming of the Xi and/or by erosion of XCI [30,47].


## X chromosome inactivation may be altered in pathological contexts

While XCI is extremely stable in normal differentiated cells, abnormal XCI patterns are a hallmark of some pathological contexts, including aging and cancers. For example, female cells without a Barr body have been observed in breast and ovarian tumors [7], and X chromosome aneuploidies were reported in several tumor types [48]. Although genetic instability due to mitotic segregation errors commonly accounts for the loss of the Xi and duplication of the Xa, recent work has identified major epigenomic remodeling of the Xi in some cancer cells associated with abnormal dosage of some X-linked genes ( Fig. 1C) [49]. Impairing Xist expression in the hematopoietic lineage results in the development of aggressive hematological cancers in females only [8], further suggesting that loss of XCI could play a causative role in tumorigenesis. In cancers also, the presence of more than one Xa is linked to the differentiation status of the tumor cells, because the loss of the Xi has been observed in aggressive, poorly differentiated, tumor cells [7,50,51]. The instability of X chromosome activity in cancer cells might be related to the fact that the X chromosome harbors many genes (tumor suppressors, chromatin remodelers) that are related to cancer progression. Abnormal X-linked gene dosage might thus, confer a selective advantage to the cell, a situation that is reminiscent of human pluripotent stem cells in which erosion of XCI spontaneously occurs in culture.

X chromosome activity is thus, connected to the differentiation status of the cell, and only undifferentiated or poorly differentiated cells in physiological, or pathological contexts cope with the presence of two active X chromosomes. A recent study proposed that XCI is required for differentiation to proceed, because double dosage of X-linked genes prevents the cells from exiting pluripotency [52]. This study, conducted in mouse, identified signaling pathways critical for differentiation that are blocked by the presence of two Xa. This would ensure that differentiation would progress only after XCI has been achieved. A tight coordination between dosage compensation and early developmental transition is further secured by direct control of XCI regulators by the pluripotency network. Whether X-linked gene dosage also controls differentiation in human is currently unknown. There is, however, no evidence so far that XCI profiles of female hESCs influence their differentiation potential; cells with eroded XCI, in which X-linked genes reactivation is irreversible [30], appear to differentiate properly. In that case, however, X chromosome dosage compensation might be achieved through a different mechanism, as recently reported in human preimplantation embryos [53].

What are the molecular mechanisms coupling XCI to the cellular state?

Multiple cis-and trans-acting factors cooperates to regulate Xist and XCI Regulatory mechanisms of XCI initiation and maintenance have mainly been studied in the mouse. The complex interplay that controls the expression of Xist, the master coordinator of the process, is integrated at a region physically mapped on the X chromosome, referred to as the X inactivation center (Xic). In the mouse, this region is topologically organized in two broad domains (Topologically Associated Domains, TAD) [54], which segregate factors that promote XCI from repressors of the process (Fig. 2). The boundary between the two Xic TADs lies between Xist promoter and that of Tsix, an antisense transcript whose expression is anti-correlated with Xist, and which blocks Xist accumulation in cis [55,56]. Within the Xist TAD lie the noncoding loci Jpx and Ftx, as well as the protein-coding gene Rnf12, which altogether function as activators of Xist [57][58][59][60][61] (Furlan and Rougeulle, unpublished results). Conversely, the Tsix TAD encompasses loci such as Xite and Linx, which regulate Tsix and thus, further act as blocker of XCI [54,62,63]. Pluripotency-associated transcription factors are involved in the transcriptional regulation of those two TADs at multiple levels to ultimately repress Xist in pluripotent stem cells. OCT3/4, NANOG, SOX2, and REX1 have been suggested to directly repress Xist expression by binding within the Xist locus or its regulatory regions [64,65] while REX1, KLF4, and c-MYC act indirectly through activation of Tsix [66]. Rnf12/Rlim is also regulated by pluripotency factors [67], and the ubiquitin ligase that it encodes in turn targets REX1 protein for degradation, contributing to the activation of Xist upon differentiation [65,68]. Altogether, the multiple interactions of pluripotency factors and XCI regulators ensure a tight connection of XCI to differentiation, at least in the mouse.

The extent to which this holds true in other species remains to be investigated. It appears from the analysis of XIST expression profiles in human and rabbit blastocysts, and in human pluripotent cells, that the link between XIST expression and pluripotency might not be straightforward in non-rodent species [21,[69][70][71][72]. More generally, very little is known about XIST regulation in species other than mouse. TSIX, for instance, is poorly conserved, and is unlikely to fulfill extensive repressive function on XIST [73]. The possibility exists that additional factors participate in the regulation of XIST or XCI in a species-specific manner. The lncRNA XACT might be such a candidate in the human. XACT is transcribed from a genomic region located more than 40 Mb downstream of the XIC, in the form of a very long non-coding transcript that remains associated to the chromosome from which it is expressed [74]. In contrast to XIST, XACT coats active X chromosomes, and its expression is restricted to early human embryonic contexts preceding, or concomitant with, the establishment of XCI. XACT is downregulated upon differentiation of hPSC and re-repressed upon reprogramming. Studies of XACT expression in the context of XCI erosion in hPSC suggested that XACT could serve as a modulator of XIST localization and/or activity [31]. XACT might thus, to some extent, connect XCI to specific developmental contexts and to pluripotency in the human.


## Xist-mediated silencing initiates in defined developmental contexts

Xist is essential for triggering XCI, yet, it can only do so in certain cellular environments. Elegant experiments using inducible Xist cDNA transgenes indeed defined developmental contexts favorable for the establishment of Xist-mediated silencing. In pluripotent ESCs, for instance, overexpression of Xist causes long-range transcriptional silencing, but this repression is reversible, and depends on continuous expression of Xist. The repression becomes stable and maintained independently of Xist when differentiation is triggered. In contrast, forcing Xist expression in cells differentiated for more than 48-72 hours fails to induce silencing. The early stages of ESC differentiation thus, provide a window of opportunity for the establishment of stable silencing induced by Xist [75].

Similar experiments performed in vivo indicated that Xist is able to initiate silencing from the blastocyst stage up to mid gestation (E9.5-12.5), from which Xist competency to cause inactivation is gradually lost [76]. A notable exception was found specifically in adult hematopoietic progenitor cells, in which ectopic XCI can be initiated. This is not the case in the stem or the mature cell compartment [76]. These results suggest the existence of competency factors for Xist-mediated silencing that are downregulated as cells commit to differentiation in most somatic lineages, except in hematopoietic progenitors. SATB1 was identified as such a competency factor in a mouse T-cell lymphoma model [77]. Its overexpression in cultured mouse embryonic fibroblasts restored the ability of Xist to induce silencing in this differentiated context. SATB1 is, however, unlikely to be the only factor for XCI initiation, because its mutation is compatible with female development [78]. Other proteins that bind to Xist RNA and are necessary for various aspects of gene silencing have been systematically identified, and some of these Xist interactors could also function as competency factors [79][80][81][82][83]. These proteins may contribute to provide the proper context for Xist to be functional, but they are also directly involved in shaping the inactive X chromosome and its particular heterochromatin epigenetic signature.

Are all inactive X chromosomes equivalent?

The X chromosome in females can thus, be found in an active or an inactive form, and the transition between states is closely related to development and to the cellular context. In addition, the inactive status in itself is far from being fixed and constant; although it is generally characterized by the accumulation of repressive epigenetic marks, the Xi epigenetic signature can vary, both across and more strikingly within species, in a context-dependent manner.


## Imprinted and random XCI lead to distinct Xi signature in the mouse

In mice, the molecular bases of imprinted inactivation are set during pre-implantation embryogenesis, and have been studied in vivo in early mouse embryos [16]. Epigenetic marks of inactivation accumulate as early as the four-cell stage: loss of H3K4 methylation and H3K9 acetylation immediately follows the accumulation of Xist, and the exclusion of the RNA Polymerase II on the future inactive X. Accumulation of members of the PRC2 complex, such as EED and EZH2, and the resulting H3K27me3 enrichment, takes place subsequently, at the morula stage (16-cell), concomitantly to that of the histone variant macro-H2A (mH2A). Enrichment on the Xi of H3K9me2 is only detected from the blastocyst stage (32-cell).

The kinetics of rXCI initiation has been inferred from in vitro studies using ES cell differentiation. Similarly to imprinted XCI, the process is initiated with Xist coating, immediately followed by loss of RNA Polymerase II and of active histone marks from the future inactive X [84][85][86]. Dynamics of downstream histone modifications during rXCI, however, differ from iXCI; PRC1 and PRC2 complexes are rapidly recruited, and H3K27me3 and H2AK119Ub accumulate early in the process, concomitantly to the loss of histone acetylation, at around day 2 of differentiation [87]. H3K9me2 enrichment is also observed early in differentiation [88] and simultaneously with H3K27me3 [89,90]. In contrast, association of the histone variant macro-H2A only occurs several days after Polycomb recruitment. Are these differences in the timing of epigenetic transformation of the Xi linked to differences in the kinetics and stability of gene silencing? Studies in pre-implantation embryos have shown that if most X-linked genes are efficiently silenced upon iXCI, some genes are only partially inactivated (Atp6ap2 and Mecp2) or escape inactivation (Huwe1) up to the blastocyst stage [91]. These genes are ultimately subjected to iXCI, as observed later on in extraembryonic tissues, suggesting that iXCI might not be complete in the pre-implantation embryo. This delayed inactivation might be related to the slow kinetics of repressive histone marks enrichment during iXCI compared with rXCI. The partial nature of the iXCI process at this developmental stage might also facilitate its reversal in cells of the ICM, prior to rXCI.

The inactive state of the Xp is, however, passed on and maintained, at least for the period of in utero life, in cells of the extra-embryonic lineage. The maintenance and stability of iXCI in this post-implantation context has mainly been addressed in vitro, through the analysis of cells derived from the trophectoderm (TS, trophoblast stem cells, and differentiated derivatives such as TGC, trophoblast giant cells) and from the primitive endoderm (XEN cells) -and in rare cases, from in vivo analyses [92][93][94]. These studies revealed peculiar features of imprinted Xi chromatin and lower stability of iXCI. Enrichment of H3K27me3 on the Xi has been observed in all cells from the extra-embryonic lineages [93][94][95], yet, to a lower extent compared to a randomly inactivated X: approximately 76-80% of X-linked genes display female-specific enrichment of H3K27me3 in XEN and TS cells, while this number reaches 93% in liver cells [93]. This is in agreement with the observation that levels of H3K27me3 are globally lower in TS and XEN cells compared with post-implantation embryonic stages [96]. Extra-embryonic tissues also display reduced levels of DNA methylation, especially at promoters of X-linked genes [97,98]. Dnmt1 deficient mice show reactivation of transgenes on the Xi in the embryonic lineage, but not in the extra-embryonic annexes [98], which further suggests that DNA methylation does not play a major role in the maintenance of iXCI. Other epigenetic marks further distinguish the imprinted Xi from its random counterpart. Notably, while the Xi is globally depleted from H3K4me2 and H4 acetylation in somatic cells [16,84,85,99], TGCs, which are derivatives of TS cells, display only partial or no hypoacetylation of H4, and only partial hypomethylation of H3K4 of the Xi [92]. Moreover, these cells do not show enrichment in another Xi characteristic repressive mark, H4K20me1.

It remains to be fully understood to what extent this peculiar epigenetic Xi signature in extra-embryonic lineages is directly linked to partial lack of XCI and/or frequent reactivation events from the Xi. Indeed, more genes escape XCI in placental progenitor cells (around 13% in TS cells) than in various adult tissues [93,[100][101][102]. Relaxation of XCI has been observed in up to 40% of TGCs for X-linked genes (Huwe1 and Atrx), which do not escape XCI in other lineages [92]. This confirms earlier results showing spontaneous reactivation of X-linked GFP and LacZ transgenes in cells from the trophectoderm lineage [19,103]. These findings taken together highlight the plasticity of imprinted XCI, which might be linked to reduced epigenetic lockdown of the Xi silenced state in this lineage [104]. It must be noted, however, that in some cases, the epigenetic status of X-linked genes might not fully correlate with their transcriptional state: the gene G6pd displays low promoter DNA methylation in TGCs, yet, it remains tightly regulated and solely expressed from the Xa [92].


## Heterochromatin signature of the Xi varies across cellular states

Pioneer immunofluorescence (IF) studies on human metaphase chromosomes have defined two major subtypes of facultative heterochromatin on the human Xi: one resembling constitutive heterochromatin, characterized by H3K9me3, HP1 [105][106][107][108], and H4K20me3 enrichment [105] and a second distinguished by H3K27me3, macroH2A, and XIST RNA accumulation [105]. These signatures were shown to be mutually exclusive, each being associated with different cytogenetic bands on the Xi in hTERT-RPE1 cells [105]. Further analysis of these epigenomic marks on metaphase spreads from additional cell lines; however, highlighted a variability in this organization, with substantial cell-to-cell, and line-toline differences in H3K27me3 and H3K9me3 distribution on the Xi [109]. The banding pattern of H3K27me3 appears to be a consistent feature of all cell lines examined; however, only one region, Xq22-23, remained commonly enriched in H3K27me3 in all cells and across all cell lines. In contrast, H3K9me3 was rather found restricted to Xi centromeres and telomeres. Although confined to the organization of metaphase chromosomes, these initial observations already suggested the existence of an epigenetic plasticity of the Xi in somatic cells, and implied that the X chromosome inactive state might not be propagated with a single epigenetic signature.

We recently addressed the diversity of the Xi heterochromatin landscape across various human cellular contexts, by ChIP-seq analysis of H3K9me3 and H3K27me3 distribution in pluripotent stem cells, as well as in primary and immortalized differentiated cell lines [31]. While the inactive X is overall enriched in H3K9me3 and H3K27me3 in all cases, integrated bioinformatics analysis revealed a specificity of the Xi chromatin organization for each context (Fig. 3, left panels). First, the distribution along the Xi of each of the two marks varies from one state to the other. Some regions, however, display a common pattern in all conditions. In particular, ChIP-seq analysis of multiple datasets confirms the observation made by IF on metaphase chromosome [109]: that is that the Xq22-23 region, located between 103 and 116.5-Mb (hg19 assembly), is consistently enriched in H3K27me3. Intriguingly, Figure 3. Heterochromatin signature of the inactive X chromosome in human. For each cellular context ("pluripotent," "primary differentiated," and "immortalized differentiated"), the left panel displays chromosomal density maps of peaks detected for H3K27me3 (green tracks) and H3K9me3 (blue tracks) in one human cell line representative of each category: XaXi H9 for "pluripotent," IMR90 for "primary differentiated," and RPE-hTERT for "immortalized differentiated." The right panel displays a plot of the cumulative H3K27me3 (green) and H3K9me3 (blue) occupancy (i.e. sum of peak size) in bp versus the size for each human chromosome. The red line corresponds to the linear regression between the two variables. Datasets are from GSE62562 and DRP000928. this domain encompasses several loci potentially involved in the initiation or maintenance of XCI. DXZ4 is a macrosatellite that produces non-coding transcripts and displays unusual chromatin features: the allele on the Xi is enriched in euchromatin marks and bound by CTCF, while the Xa copy is packaged into heterochromatin [110]. The DXZ4 locus has recently been proposed to play a role in the organization of the Xi chromosome into megadomains [111,112]. The locus producing the XACT lncRNA also maps to within this domain [74], raising the hypothesis that H3K27me3-mediated repression of the region might be important for proper XCI.

In addition, the relative organization of H3K27me3 and H3K9me3 vary across cellular states, and these two marks are anti-correlated specifically in pluripotent cells and immortalized differentiated cells. This is in agreement with what was previously found by immunofluorescence on hTERT-RPE1, an immortalized cell line [105]. In surprising contrast, we found overlap between H3K27me3 and H3K9me3 enrichment zones in all the primary differentiated cell lines that were included in our analysis. Notably, in this context H3K9me3 was no longer organized into domains (Fig. 3, "Xi in primary differentiated cells," left panel), but rather into focal loci dispersed all over the chromosome.

Finally, the relative enrichment in H3K9me3 and H3K27me3 on the Xi compared with autosomes also differs between cellular states (Fig. 3, right panels). In pluripotent cells, the Xi is the only chromosome to be highly enriched in both H3K9me3 and H3K27me3. In contrast, it is only H3K27me3 that distinguishes the Xi from other chromosomes in primary differentiated cells; while in immortalized cells the Xi is selectively marked by H3K9me3. Taken together, the combination of H3K9me3 and H3K27me3 distribution distinguishes the Xi from a pluripotent to a differentiated cell type, and between primary versus immortalized cell types. The consequences of such plasticity remain unclear. In the case of hESCs, where XCI has been shown to be highly unstable [32,69], we provided evidence that the heterochromatin organization of the Xi might underlie gene-specific predisposition to reactivation, in which transcriptional reactivation of the Xi is limited to H3K27me3 domains [31]. Following this line of reasoning, the overlapping distribution of H3K9me3 and H3K27me3 in primary differentiated cells might protect the Xi from reactivation, and ensure the stable maintenance of the inactive state across cell divisions, although it remains to be shown whether these marks coexist on the same nucleosome.

We also explored the variability of the Xi chromatin landscape in the mouse by analyzing published H3K27me3 ChIP-seq and ChIP-chip datasets (Fig. 4). This preliminary investigation revealed striking differences in the relative distribution of this mark on the X and autosomes, compared with the human. In pluripotent cells, for instance, many autosomes display substantial levels of H3K27me3, while the Xs emerge as the less-enriched chromosomes (Fig. 4B, upper panels). Most chromosomes overall maintain their methylation levels in differentiated cells, but the X undergoes a massive transition to become the most enriched chromosome, specifically in female cells, reflecting the inactivation of one of the two homologues. The active X, as seen in male cells, remains mostly depleted in H3K27me3, and this might be linked to the up-regulation of the Xa, which takes place to compensate for X-autosome dosage imbalance [113].

In primary differentiated female cells, H3K27me3 is distributed into large domains over the Xi, and the enrichment of the Xi in H3K27me3 is around twice that of autosomes, just as in human cells. The presence and relative distribution of H3K9me3 remain to be addressed through similar ChIP-seq analysis, and so far few data sets characterize H3K9me3 distribution. Evidence suggests that H3K9me2 accumulates on the Xi in the mouse [88,90,114]. One study provided ChIPchip datasets for H3K27me3 and H3K9me2 in mouse ES cells and cells differentiated for 14 days [115]. We reanalyzed the data to compare the relative positioning of H3K9me2 and H3K27me3 on the X chromosome (Fig. 4C), and found anti-correlated distribution of the two marks on the Xi in differentiated cells. Accordingly, a recent study analyzing H3K9 di-and tri-methylation, and H3K27me3 in mouse embryonic fibroblasts also revealed anti-correlated distribution of the K9me2/3 and K27me3 marks on the Xi [116]. These results contrast with what we observed in human primary differentiated cells, where these marks overlap significantly on the Xi.

Drivers of this epigenomic variability might lie in the mechanisms targeting these repressive modifications to the Xi. First of all, what triggers H3K9me2/3 accumulation on the Xi is still unclear. We have shown, however, that H3K9me3 maintenance on the Xi is independent of XIST and of H3K27me3, at least in hESCs [31]. In contrast, it is well described that Xist is the main trigger for H3K27me3 deposition and maintenance on the Xi, and the genomic regions contacted by Xist RNA correlate with those occupied by H3K27me3, at least in the mouse [117]. Such link has not been investigated in depth in human. In addition, we know that during early human development, XIST accumulation on X chromosomes in male and female embryos does not trigger H3K27me3 deposition [21]. What uncouples XIST from H3K27me3 is unknown, and it will be of great interest to assess the involvement the lncRNA XACT in this process, as XIST and XACT accumulate simultaneously on the X chromosome at these stages (unpublished data).

The divergence in Xist sequences across species [118] could impact on its associated factors and on its activity. While Xist interactome is beginning to be deciphered in the mouse [79,80,82], it is yet unknown whether Xist partners are the same in other species. Recently, JARID2 has been implicated in bridging PRC2 complex with Xist RNA, and thus, in targeting H3K27me3 to the Xi in the mouse [119]. Changes in JARID2/ XIST interaction in human could explain why XIST RNA coating does not lead to immediate H3K27me3 deposition present in early embryos.

Dynamics of expression of factors involved in the deposition of H3K27me3 and/or H3K9me2/3 may as well contribute in setting up the Xi landscape, and should be revisited in that context. This concerns H3K9 and H327 histone methyltransferases (HMT), as well as co-factors. CDYL in particular is a chromodomain-containing transcriptional corepressor whose recruitment to the Xi depends on both H3K9me2 and H3K27me3. CDYL interacts with the HMT G9A, which may be responsible for triggering H3K9me2 in the vicinity, or across H3K27me3 marked regions [120]. Species differences in the organization of the Xi chromatin might also be promoted by difference in the genomic content of the X chromosome. Repeats in particular, which differ between mouse and human [121], could influence the global epigenomic landscape. On mouse autosomes, H3K27me3 broad domains correspond to gene-rich regions enriched in SINE repeats, and alternate with H3K9me3/H4K20me3marked regions enriched in LINE/LTR repeats [122]. Such organization appears to be maintained on the mouse Xi [116]. In contrast, in hESC we did not detect preferential accumulation of either H3K9me3 or H3K27me3 over specific repeat-or generich regions (data not shown).

It is important to bear in mind that the H3K9-K27 methylation differences observed in the various cellular contexts and across species are likely only glimpses of the epigenomic differences that exist on the various Xi. Additional epigenetic marks, such as DNA methylation and other histone modifications, should be investigated and integrated to fully grasp the variability of the epigenetic organization of the Xi across cellular contexts, and the influence of this epigenetic signature on the stability of XCI. This in-depth characterization of Xi epigenetic signatures, as well as the higher order chromatin organization will be necessary to further characterize the tenants of this potential heterogeneity observed across species and cellular states.  Fig. 3 for each mouse chromosome, representing cumulative H3K27me3 occupancy versus chromosome size in bp. C: Superposition of H3K9me2 and H3K27me3 ChIP-chip profiles in LF2 mESC, and LF2 differentiating cells at day 10 [115]. Log2 ratios were compared using Pearson correlation test and the associated p-value was calculated using random permutations of the data sets. Datasets are from GSE23943, GSM1754897, GSM769032, and GSE15883. 


## Conclusions and outlook

The connection between the activity status of the X chromosomes and the level of differentiation was among the first observations made in the exciting field of XCI. Since these early days, we have advanced greatly in our description of the process and our understanding of the molecular network that sustain this intimate relationshipat least in the mouse, which has, till now, been the leading model species in the field. Many questions remain unanswered, however. The necessity of such coupling is, for instance, not so clear: why are two active Xs tolerated only in poorly differentiated contexts? This is of importance not only because we, researchers, aim to disentangle fundamental issues in biology. It will also likely impact our comprehension of pathological states linked to dedifferentiation. In addition, the variation in the strategies and kinetics of XCI between species that is being brought to our attention raises questions as to the extent to which the discoveries that have been made in the mouse in that matter hold true in other mammals.

XCI is considered as the paradigm of epigenetic regulations, in which the status of the inactive X chromosome is stably transmitted to daughter cells upon cell division, for the duration of the individual's life, in mammals. We are now facing the possibility that this might be a bit of an illusion, and that the nature of the Xi might also change depending on cellular contexts. For sure, a simple epigenetic signature does not characterize the inactive status of the X chromosome, and differences have been found in the kinetics and the quality of the marks that form the Xi facultative heterochromatin between species. Even within a given species, the mouse, the Xi differs depending on whether it has been inactivated in an imprinted or a random manner, prior to or after implantation. Such a difference is not so unexpected, however, because a transient phase of reactivation during which the marks are erased separates those two states, allowing for new remodeling to take place. More striking is recent evidence of a certain plasticity of the Xi described in humans, in which variation in the distribution of at least some chromatin marks occurs between cell lines and between cellular states that are, at least in theory, not disconnected by a reactivation phase. Transmission of the inactive state across cell divisions might thus, not be as faithful as initially postulated, and while the inactive state is maintained, flexibility is tolerated in the internal organization of the Xi chromatin. Determining what governs these changes and their consequences will require further investigation.


869-880, ß 2016 The Authors BioEssays Published by WILEY Periodicals, Inc.

## Figure 2 .
2Molecular coupling of XCI and pluripotency in the mouse. The X inactivation center, which contains protein coding (in gray) and non-coding (in bronze) genes, is partitioned in two topologically associated domains (TAD) that integrate negative (Tsix TAD) and positive (Xist TAD) regulatory inputs for the expression of Xist. Pluripotency-associated transcription factors (TF) activate the expression of Tsix and repress the expression of Xist and Rnf12. RNF12, an E3 ubiquitin ligase, has been shown to promote Xist expression by regulating cellular levels of REX1. 880, ß 2016 The Authors BioEssays Published by WILEY Periodicals, Inc.


880, ß 2016 The Authors BioEssays Published by WILEY Periodicals, Inc.


880, ß 2016 The Authors BioEssays Published by WILEY Periodicals, Inc.

## Figure 4 .
4Heterochromatin signature of the inactive X chromosome in mouse. A: ChIP-seq profiles (log2 enrichment over input) and chromosomal density maps of peaks detected for H3K27me3 (green tracks) for male and female ES cells, and male and female differentiated cells. B: Same plots as in


880, ß 2016 The Authors BioEssays Published by WILEY Periodicals, Inc.


C.Vallot  et al. Prospects & Overviews .... Bioessays 38: 869-880, ß 2016 The Authors BioEssays Published by WILEY Periodicals, Inc.870 




C.Vallot  et al. Prospects & Overviews .... Bioessays 38: 869-880, ß 2016 The Authors BioEssays Published by WILEY Periodicals, Inc.872 




C.Vallot  et al. Prospects & Overviews .... Bioessays 38: 869-880, ß 2016 The Authors BioEssays Published by WILEY Periodicals, Inc.876 


Bioessays 38: 869-880, ß 2016 The Authors BioEssays Published by WILEY Periodicals, Inc.
AcknowledgmentsThe research leading to this review received funding from the Agence Nationale pour la Recherche (ANR-14-CE10-0017-01), from the "Ligue Contre le Cancer," and from the European Research Council/ EpiGeneSys FP7 257082 Network of Excellence.The authors have declare no conflict of interest.
A morphological distinction between neurones of the male and female, and the behaviour of the nucleolar satellite during accelerated nucleoprotein synthesis. M L Barr, E G Bertram, Nature. 163676Barr ML, Bertram EG. 1949. A morphological distinction between neurones of the male and female, and the behaviour of the nucleolar satellite during accelerated nucleoprotein synthesis. Nature 163: 676.

Formation of the sex chromatin by a single X-chromosome in liver cells of Rattus norvegicus. S Ohno, W D Kaplan, R Kinosita, Exp Cell Res. 18Ohno S, Kaplan WD, Kinosita R. 1959. Formation of the sex chromatin by a single X-chromosome in liver cells of Rattus norvegicus. Exp Cell Res 18: 415-8.

Gene action in the X-chromosome of the mouse (Mus musculus L.). M F Lyon, Nature. 190Lyon MF. 1961. Gene action in the X-chromosome of the mouse (Mus musculus L.). Nature 190: 372-3.

Detrimental effects of two active X chromosomes on early mouse development. N Takagi, K Abe, Development. 109Takagi N, Abe K. 1990. Detrimental effects of two active X chromosomes on early mouse development. Development 109: 189- 201.

Xistdeficient mice are defective in dosage compensation but not spermatogenesis. Y Marahrens, B Panning, J Dausman, W Strauss, Genes Dev. 11Marahrens Y, Panning B, Dausman J, Strauss W, et al. 1997. Xist- deficient mice are defective in dosage compensation but not spermatogenesis. Genes Dev 11: 156-66.

The sex chromatin in benign tumours and related conditions in man. K L Moore, M L Barr, Br J Cancer. 9Moore KL, Barr ML. 1955. The sex chromatin in benign tumours and related conditions in man. Br J Cancer 9: 246-52.

The disappearing Barr body in breast and ovarian cancers. G J Pageau, L L Hall, S Ganesan, D M Livingston, Nat Rev Cancer. 7Pageau GJ, Hall LL, Ganesan S, Livingston DM, et al. 2007. The disappearing Barr body in breast and ovarian cancers. Nat Rev Cancer 7: 628-33.

Xist RNA is a potent suppressor of hematologic cancer in mice. E Yildirim, J E Kirby, D E Brown, F E Mercier, Cell. 152Yildirim E, Kirby JE, Brown DE, Mercier FE, et al. 2013. Xist RNA is a potent suppressor of hematologic cancer in mice. Cell 152: 727-42.

Sequential X chromosome inactivation coupled with cellular differentiation in early mouse embryos. M Monk, M I Harper, Nature. 281Monk M, Harper MI. 1979. Sequential X chromosome inactivation coupled with cellular differentiation in early mouse embryos. Nature 281: 311-3.

Preferential paternal X inactivation in extraembryonic tissues of early mouse embryos. M I Harper, M Fosten, M Monk, J Embryol Exp Morphol. 67Harper MI, Fosten M, Monk M. 1982. Preferential paternal X inactivation in extraembryonic tissues of early mouse embryos. J Embryol Exp Morphol 67: 127-35.

X chromosome inactivation revealed by the X-linked lacZ transgene activity in periimplantation mouse embryos. M Sugimoto, S S Tan, N Takagi, Int J Dev Biol. 44Sugimoto M, Tan SS, Takagi N. 2000. X chromosome inactivation revealed by the X-linked lacZ transgene activity in periimplantation mouse embryos. Int J Dev Biol 44: 177-82.

Preferential inactivation of the paternally derived X chromosome in the extraembryonic membranes of the mouse. N Takagi, M Sasaki, Nature. 256Takagi N, Sasaki M. 1975. Preferential inactivation of the paternally derived X chromosome in the extraembryonic membranes of the mouse. Nature 256: 640-2.

X-chromosome inactivation occurs at different times in different tissues of the post-implantation mouse embryo. S S Tan, E A Williams, P P Tam, Nat Genet. 3Tan SS, Williams EA, Tam PP. 1993. X-chromosome inactivation occurs at different times in different tissues of the post-implantation mouse embryo. Nat Genet 3: 170-4.

Preferential expression of the Xm in the mouse yolk sac. J D West, W I Frels, V M Chapman, Cell. 12West JD, Frels WI, Chapman VM. 1977. Preferential expression of the Xm in the mouse yolk sac. Cell 12: 873-82.

Reactivation of the paternal X chromosome in early mouse embryos. W Mak, T B Nesterova, M De Napoles, R Appanah, Science. 303Mak W, Nesterova TB, de Napoles M, Appanah R, et al. 2004. Reactivation of the paternal X chromosome in early mouse embryos. Science 303: 666-9.

Epigenetic dynamics of imprinted X inactivation during early mouse development. I Okamoto, A P Otte, C D Allis, D Reinberg, Science. 303Okamoto I, Otte AP, Allis CD, Reinberg D, et al. 2004. Epigenetic dynamics of imprinted X inactivation during early mouse development. Science 303: 644-9.

Both X chromosomes function before visible X-chromosome inactivation in female mouse embryos. C J Epstein, S Smith, B Travis, G Tucker, Nature. 274Epstein CJ, Smith S, Travis B, Tucker G. 1978. Both X chromosomes function before visible X-chromosome inactivation in female mouse embryos. Nature 274: 500-3.

Correlation between X-chromosome inactivation and cell differentiation in female preimplantation mouse embryos. O Sugawara, N Takagi, M Sasaki, Cytogenet Cell Genet. 39Sugawara O, Takagi N, Sasaki M. 1985. Correlation between X-chromosome inactivation and cell differentiation in female preimplan- tation mouse embryos. Cytogenet Cell Genet 39: 210-9.

Expression of an X-linked HMG-lacZ transgene in mouse embryos: implication of chromosomal imprinting and lineage-specific X-chromosome activity. P P Tam, E A Williams, S S Tan, Dev Genet. 15Tam PP, Williams EA, Tan SS. 1994. Expression of an X-linked HMG- lacZ transgene in mouse embryos: implication of chromosomal imprinting and lineage-specific X-chromosome activity. Dev Genet 15: 491-503.

Timing of X-chromosome inactivation in postimplantation mouse embryos. S Rastan, J Embryol Exp Morphol. 71Rastan S. 1982. Timing of X-chromosome inactivation in postimplan- tation mouse embryos. J Embryol Exp Morphol 71: 11-24.

Eutherian mammals use diverse strategies to initiate X-chromosome inactivation during development. I Okamoto, C Patrat, D Thepot, N Peynot, Nature. 472Okamoto I, Patrat C, Thepot D, Peynot N, et al. 2011. Eutherian mammals use diverse strategies to initiate X-chromosome inactivation during development. Nature 472: 370-4.

Sex chromatin in early cat embryos. C R Austin, E C Amoroso, Exp Cell Res. 13Austin CR, Amoroso EC. 1957. Sex chromatin in early cat embryos. Exp Cell Res 13: 419-21.

The occurrence of sex chromatin in early human and macaque embryos. W W Park, J Anat. 91Park WW. 1957. The occurrence of sex chromatin in early human and macaque embryos. J Anat 91: 369-73.

Embryonic stem (ES) cells and embryonal carcinoma (EC) cells: opposite sides of the same coin. P W Andrews, M M Matin, A R Bahrami, I Damjanov, Biochem Soc Trans. 33Andrews PW, Matin MM, Bahrami AR, Damjanov I, et al. 2005. Embryonic stem (ES) cells and embryonal carcinoma (EC) cells: opposite sides of the same coin. Biochem Soc Trans 33: 1526-30.

X-chromosome inactivation during differentiation of female teratocarcinoma stem cells in vitro. G R Martin, C J Epstein, B Travis, G Tucker, Nature. 271Martin GR, Epstein CJ, Travis B, Tucker G, et al. 1978. X-chromosome inactivation during differentiation of female teratocarcinoma stem cells in vitro. Nature 271: 329-33.

Naive and primed pluripotent states. J Nichols, A Smith, Cell Stem Cell. 4Nichols J, Smith A. 2009. Naive and primed pluripotent states. Cell Stem Cell 4: 487-92.

New cell lines from mouse epiblast share defining features with human embryonic stem cells. P J Tesar, J G Chenoweth, F A Brook, T J Davies, Nature. 448Tesar PJ, Chenoweth JG, Brook FA, Davies TJ, et al. 2007. New cell lines from mouse epiblast share defining features with human embryonic stem cells. Nature 448: 196-9.

. C Vallot, C. Vallot et al.

. . Prospects &amp; Overviews, Prospects & Overviews ....

. Bioessays. 38The Authors BioEssays Published by WILEY Periodicals, IncBioessays 38: 869-880, ß 2016 The Authors BioEssays Published by WILEY Periodicals, Inc.

Derivation of pluripotent epiblast stem cells from mammalian embryos. I G Brons, L E Smithers, M W Trotter, P Rugg-Gunn, Nature. 448Brons IG, Smithers LE, Trotter MW, Rugg-Gunn P, et al. 2007. Derivation of pluripotent epiblast stem cells from mammalian embryos. Nature 448: 191-5.

Human embryonic stem cells with biological and epigenetic characteristics similar to those of mouse ESCs. J Hanna, A W Cheng, K Saha, J Kim, Proc Natl Acad Sci. 107Hanna J, Cheng AW, Saha K, Kim J, et al. 2010. Human embryonic stem cells with biological and epigenetic characteristics similar to those of mouse ESCs. Proc Natl Acad Sci USA 107: 9222-7.

Erosion of dosage compensation impacts human iPSC disease modeling. S Mekhoubad, C Bock, A S De Boer, E Kiskinis, Cell Stem Cell. 10Mekhoubad S, Bock C, de Boer AS, Kiskinis E, et al. 2012. Erosion of dosage compensation impacts human iPSC disease modeling. Cell Stem Cell 10: 595-609.

Erosion of X chromosome inactivation in human pluripotent cells initiates with XACT coating and depends on a specific heterochromatin landscape. C Vallot, J F Ouimette, M Makhlouf, O Feraud, Cell Stem Cell. 16Vallot C, Ouimette JF, Makhlouf M, Feraud O, et al. 2015. Erosion of X chromosome inactivation in human pluripotent cells initiates with XACT coating and depends on a specific heterochromatin landscape. Cell Stem Cell 16: 533-46.

X-chromosome inactivation and epigenetic fluidity in human embryonic stem cells. S S Silva, R K Rowntree, S Mekhoubad, J T Lee, Proc Natl Acad Sci. 105Silva SS, Rowntree RK, Mekhoubad S, Lee JT. 2008. X-chromosome inactivation and epigenetic fluidity in human embryonic stem cells. Proc Natl Acad Sci USA 105: 4820-5.

Reactivation of the inactive X chromosome in development and reprogramming. T Ohhata, A Wutz, Cell Mol Life Sci. 70Ohhata T, Wutz A. 2013. Reactivation of the inactive X chromosome in development and reprogramming. Cell Mol Life Sci 70: 2443-61.

The morphology and ultrastructure of the sex vesicle in the mouse. A J Solari, Exp Cell Res. 36Solari AJ. 1964. The morphology and ultrastructure of the sex vesicle in the mouse. Exp Cell Res 36: 160-8.

Meiotic sex chromosome inactivation. J M Turner, Development. 134Turner JM. 2007. Meiotic sex chromosome inactivation. Development 134: 1823-31.

Evidence for de novo imprinted X-chromosome inactivation independent of meiotic inactivation in mice. I Okamoto, D Arnaud, Le Baccon, P Otte, A P , Nature. 438Okamoto I, Arnaud D, Le Baccon P, Otte AP, et al. 2005. Evidence for de novo imprinted X-chromosome inactivation independent of meiotic inactivation in mice. Nature 438: 369-73.

X chromosome activity in mouse XX primordial germ cells. S M Chuva De Sousa Lopes, K Hayashi, T C Shovlin, W Mifsud, PLoS Genet. 430Chuva de Sousa Lopes SM, Hayashi K, Shovlin TC, Mifsud W, et al. 2008. X chromosome activity in mouse XX primordial germ cells. PLoS Genet 4: e30.

X chromosome reactivation initiates in nascent primordial germ cells in mice. M Sugimoto, Abe K , PLoS Genet. 3116Sugimoto M, Abe K. 2007. X chromosome reactivation initiates in nascent primordial germ cells in mice. PLoS Genet 3: e116.

The mammalian germline as a pluripotency cycle. H G Leitch, A Smith, Development. 140Leitch HG, Smith A. 2013. The mammalian germline as a pluripotency cycle. Development 140: 2495-501.

Early loss of Xist RNA expression and inactive X chromosome associated chromatin modification in developing primordial germ cells. M De Napoles, T Nesterova, N Brockdorff, PLoS ONE. 2860de Napoles M, Nesterova T, Brockdorff N. 2007. Early loss of Xist RNA expression and inactive X chromosome associated chromatin modifi- cation in developing primordial germ cells. PLoS ONE 2: e860.

Epigenetic reprogramming in mouse primordial germ cells. P Hajkova, S Erhardt, N Lane, T Haaf, Mech Dev. 117Hajkova P, Erhardt S, Lane N, Haaf T, et al. 2002. Epigenetic reprogramming in mouse primordial germ cells. Mech Dev 117: 15-23.

X-Chromosome inactivation in cloned mouse embryos. K Eggan, H Akutsu, K Hochedlinger, W Rideout, Science. 290Eggan K, Akutsu H, Hochedlinger K, Rideout W, 3rd, et al. 2000. X-Chromosome inactivation in cloned mouse embryos. Science 290: 1578-81.

Reversal of X-inactivation in female mouse somatic cells hybridized with murine teratocarcinoma stem cells in vitro. N Takagi, M A Yoshida, O Sugawara, M Sasaki, Cell. 34Takagi N, Yoshida MA, Sugawara O, Sasaki M. 1983. Reversal of X-inactivation in female mouse somatic cells hybridized with murine teratocarcinoma stem cells in vitro. Cell 34: 1053-62.

X chromosome reactivation and regulation in cloned embryos. L D Nolen, S Gao, Z Han, M R Mann, Dev Biol. 279Nolen LD, Gao S, Han Z, Mann MR, et al. 2005. X chromosome reactivation and regulation in cloned embryos. Dev Biol 279: 525-40.

Directly reprogrammed fibroblasts show global epigenetic remodeling and widespread tissue contribution. N Maherali, R Sridharan, W Xie, J Utikal, Cell Stem Cell. 1Maherali N, Sridharan R, Xie W, Utikal J, et al. 2007. Directly reprogrammed fibroblasts show global epigenetic remodeling and widespread tissue contribution. Cell Stem Cell 1: 55-70.

Defining molecular cornerstones during fibroblast to iPS cell reprogramming in mouse. M Stadtfeld, N Maherali, D T Breault, K Hochedlinger, Cell Stem Cell. 2Stadtfeld M, Maherali N, Breault DT, Hochedlinger K. 2008. Defining molecular cornerstones during fibroblast to iPS cell reprogramming in mouse. Cell Stem Cell 2: 230-40.

Female human iPSCs retain an inactive X chromosome. J Tchieu, E Kuoy, M H Chin, H Trinh, Cell Stem Cell. 7Tchieu J, Kuoy E, Chin MH, Trinh H, et al. 2010. Female human iPSCs retain an inactive X chromosome. Cell Stem Cell 7: 329-42.

ConteXt of change-X inactivation and disease. R Agrelo, A Wutz, EMBO Mol Med. 2Agrelo R, Wutz A. 2010. ConteXt of change-X inactivation and disease. EMBO Mol Med 2: 6-15.

The inactive X chromosome is epigenetically unstable and transcriptionally labile in breast cancer. R Chaligne, T Popova, M A Mendoza-Parra, M A Saleem, Genome Res. 25Chaligne R, Popova T, Mendoza-Parra MA, Saleem MA, et al. 2015. The inactive X chromosome is epigenetically unstable and transcrip- tionally labile in breast cancer. Genome Res 25: 488-503.

Barr body distribution and estrogen receptor protein in mammary carcinoma. P P Rosen, A Savino, C Menendez-Botet, J A Urban, Ann Clin Lab Sci. 7Rosen PP, Savino A, Menendez-Botet C, Urban JA, et al. 1977. Barr body distribution and estrogen receptor protein in mammary carcinoma. Ann Clin Lab Sci 7: 491-9.

Steroid hormone receptors and sex chromatin frequency in breast cancer. M Smethurst, N P Bishun, D Fernandez, J Allen, J Endocrinol Invest. 4Smethurst M, Bishun NP, Fernandez D, Allen J, et al. 1981. Steroid hormone receptors and sex chromatin frequency in breast cancer. J Endocrinol Invest 4: 455-7.

The two active X chromosomes in female ESCs block exit from the pluripotent state by modulating the ESC signaling network. E G Schulz, J Meisig, T Nakamura, I Okamoto, Cell Stem Cell. 14Schulz EG, Meisig J, Nakamura T, Okamoto I, et al. 2014. The two active X chromosomes in female ESCs block exit from the pluripotent state by modulating the ESC signaling network. Cell Stem Cell 14: 203-16.

Single-cell RNA-seq reveals lineage and X chromosome dynamics in human preimplantation embryos. S Petropoulos, D Edsgard, B Reinius, Q Deng, Cell. 165Petropoulos S, Edsgard D, Reinius B, Deng Q, et al. 2016. Single-cell RNA-seq reveals lineage and X chromosome dynamics in human preimplantation embryos. Cell 165: 1012-26.

Spatial partitioning of the regulatory landscape of the X-inactivation centre. E P Nora, B R Lajoie, E G Schulz, L Giorgetti, Nature. 485Nora EP, Lajoie BR, Schulz EG, Giorgetti L, et al. 2012. Spatial partitioning of the regulatory landscape of the X-inactivation centre. Nature 485: 381-5.

Targeted mutagenesis of Tsix leads to nonrandom X inactivation. J T Lee, N Lu, Cell. 99Lee JT, Lu N. 1999. Targeted mutagenesis of Tsix leads to nonrandom X inactivation. Cell 99: 47-57.

Tsix, a gene antisense to Xist at the X-inactivation centre. J T Lee, L S Davidow, D Warshawsky, Nat Genet. 21Lee JT, Davidow LS, Warshawsky D. 1999. Tsix, a gene antisense to Xist at the X-inactivation centre. Nat Genet 21: 400-4.

RNF12 is an X-Encoded dose-dependent activator of X chromosome inactivation. I Jonkers, T S Barakat, E M Achame, K Monkhorst, Cell. 139Jonkers I, Barakat TS, Achame EM, Monkhorst K, et al. 2009. RNF12 is an X-Encoded dose-dependent activator of X chromosome inactiva- tion. Cell 139: 999-1011.

The long noncoding RNA, Jpx, is a molecular switch for X chromosome inactivation. D Tian, S Sun, J T Lee, Cell. 143Tian D, Sun S, Lee JT. 2010. The long noncoding RNA, Jpx, is a molecular switch for X chromosome inactivation. Cell 143: 390-403.

RNF12 activates Xist and is essential for X chromosome inactivation. T S Barakat, N Gunhanlar, C G Pardo, E M Achame, PLoS Genet. 71002001Barakat TS, Gunhanlar N, Pardo CG, Achame EM, et al. 2011. RNF12 activates Xist and is essential for X chromosome inactivation. PLoS Genet 7: e1002001.

. S Sun, Del Rosario, B C Szanto, A Ogawa, Y , Jpx RNA activates Xist by evicting CTCF. Cell. 153Sun S, Del Rosario BC, Szanto A, Ogawa Y, et al. 2013. Jpx RNA activates Xist by evicting CTCF. Cell 153: 1537-51.

Ftx is a noncoding RNA which affects Xist expression and chromatin structure within the X-inactivation center region. C Chureau, S Chantalat, A Romito, A Galvani, Hum Mol Genet. 20Chureau C, Chantalat S, Romito A, Galvani A, et al. 2011. Ftx is a non- coding RNA which affects Xist expression and chromatin structure within the X-inactivation center region. Hum Mol Genet 20: 705-18.

Xite, X-inactivation intergenic transcription elements that regulate the probability of choice. Y Ogawa, J T Lee, Mol Cell. 11Ogawa Y, Lee JT. 2003. Xite, X-inactivation intergenic transcription elements that regulate the probability of choice. Mol Cell 11: 731-43.

Identification of developmentally specific enhancers for Tsix in the regulation of X chromosome inactivation. N Stavropoulos, R K Rowntree, J T Lee, Mol Cell Biol. 25Stavropoulos N, Rowntree RK, Lee JT. 2005. Identification of developmentally specific enhancers for Tsix in the regulation of X chromosome inactivation. Mol Cell Biol 25: 2757-69.

Molecular coupling of Xist regulation and pluripotency. P Navarro, I Chambers, V Karwacki-Neisius, C Chureau, Science. 321Navarro P, Chambers I, Karwacki-Neisius V, Chureau C, et al. 2008. Molecular coupling of Xist regulation and pluripotency. Science 321: 1693-5.

RNF12 initiates X-chromosome inactivation by targeting REX1 for degradation. C Gontan, E M Achame, J Demmers, T S Barakat, Nature. 485Gontan C, Achame EM, Demmers J, Barakat TS, et al. 2012. RNF12 initiates X-chromosome inactivation by targeting REX1 for degradation. Nature 485: 386-90.

Molecular coupling of Tsix regulation and pluripotency. P Navarro, A Oldfield, J Legoupi, N Festuccia, Nature. 468Navarro P, Oldfield A, Legoupi J, Festuccia N, et al. 2010. Molecular coupling of Tsix regulation and pluripotency. Nature 468: 457-60.

The X-inactivation trans-activator Rnf12 is negatively regulated by pluripotency factors in embryonic stem cells. P Navarro, M Moffat, N P Mullin, I Chambers, Hum Genet. 130Navarro P, Moffat M, Mullin NP, Chambers I. 2011. The X-inactivation trans-activator Rnf12 is negatively regulated by pluripotency factors in embryonic stem cells. Hum Genet 130: 255-64.

A prominent and conserved role for YY1 in Xist transcriptional activation. M Makhlouf, J F Ouimette, A Oldfield, P Navarro, Nat Commun. 54878Makhlouf M, Ouimette JF, Oldfield A, Navarro P, et al. 2014. A prominent and conserved role for YY1 in Xist transcriptional activation. Nat Commun 5: 4878.

X-inactivation status varies in human embryonic stem cell lines. L M Hoffman, L Hall, J L Batten, H Young, Stem Cells. 23Hoffman LM, Hall L, Batten JL, Young H, et al. 2005. X-inactivation status varies in human embryonic stem cell lines. Stem Cells 23: 1468-78.

X-inactivation reveals epigenetic anomalies in most hESC but identifies sublines that initiate as expected. L L Hall, M Byron, J Butler, K A Becker, J Cell Physiol. 216Hall LL, Byron M, Butler J, Becker KA, et al. 2008. X-inactivation reveals epigenetic anomalies in most hESC but identifies sublines that initiate as expected. J Cell Physiol 216: 445-52.

Derivation conditions impact X-inactivation status in female human induced pluripotent stem cells. K Tomoda, K Takahashi, K Leung, A Okada, Cell Stem Cell. 11Tomoda K, Takahashi K, Leung K, Okada A, et al. 2012. Derivation conditions impact X-inactivation status in female human induced pluripotent stem cells. Cell Stem Cell 11: 91-9.

Linking X chromosome inactivation to pluripotency: necessity or fate?. M Makhlouf, C Rougeulle, Trends Mol Med. 17Makhlouf M, Rougeulle C. 2011. Linking X chromosome inactivation to pluripotency: necessity or fate? Trends Mol Med 17: 329-36.

Identification of regulatory elements flanking human XIST reveals species differences. S C Chang, C J Brown, BMC Mol Biol. 1120Chang SC, Brown CJ. 2010. Identification of regulatory elements flanking human XIST reveals species differences. BMC Mol Biol 11: 20.

XACT, a long noncoding transcript coating the active X in human pluripotent cells. C Vallot, C Huret, Y Lesecque, A Resh, Nat Genet. 45Vallot C, Huret C, Lesecque Y, Resh A, et al. 2013. XACT, a long non- coding transcript coating the active X in human pluripotent cells. Nat Genet 45: 239-41.

A shift from reversible to irreversible X inactivation is triggered during ES cell differentiation. A Wutz, R Jaenisch, Mol Cell. 5Wutz A, Jaenisch R. 2000. A shift from reversible to irreversible X inactivation is triggered during ES cell differentiation. Mol Cell 5: 695-705.

Hematopoietic precursor cells transiently reestablish permissiveness for X inactivation. F Savarese, K Flahndorfer, R Jaenisch, M Busslinger, Mol Cell Biol. 26Savarese F, Flahndorfer K, Jaenisch R, Busslinger M, et al. 2006. Hematopoietic precursor cells transiently reestablish permissiveness for X inactivation. Mol Cell Biol 26: 7167-77.

SATB1 defines the developmental context for gene silencing by Xist in lymphoma and embryonic cells. R Agrelo, A Souabni, M Novatchkova, C Haslinger, Dev Cell. 16Agrelo R, Souabni A, Novatchkova M, Haslinger C, et al. 2009. SATB1 defines the developmental context for gene silencing by Xist in lymphoma and embryonic cells. Dev Cell 16: 507-16.

Satb1 and Satb2 are dispensable for X chromosome inactivation in mice. R Nechanitzky, A Davila, F Savarese, S Fietze, Dev Cell. 23Nechanitzky R, Davila A, Savarese F, Fietze S, et al. 2012. Satb1 and Satb2 are dispensable for X chromosome inactivation in mice. Dev Cell 23: 866-71.

Systematic discovery of Xist RNA binding proteins. C Chu, Q C Zhang, S T Da Rocha, R A Flynn, Cell. 161Chu C, Zhang QC, da Rocha ST, Flynn RA, et al. 2015. Systematic discovery of Xist RNA binding proteins. Cell 161: 404-16.

The Xist lncRNA interacts directly with SHARP to silence transcription through HDAC3. C A Mchugh, C K Chen, A Chow, C F Surka, Nature. 521McHugh CA, Chen CK, Chow A, Surka CF, et al. 2015. The Xist lncRNA interacts directly with SHARP to silence transcription through HDAC3. Nature 521: 232-6.

A pooled shRNA screen identifies Rbm15, Spen, and Wtap as factors required for Xist RNA-mediated silencing. B Moindrot, A Cerase, H Coker, O Masui, Cell Rep. 12Moindrot B, Cerase A, Coker H, Masui O, et al. 2015. A pooled shRNA screen identifies Rbm15, Spen, and Wtap as factors required for Xist RNA-mediated silencing. Cell Rep 12: 562-72.

Chromosomes. A comprehensive Xist interactome reveals cohesin repulsion and an RNA-directed chromosome conformation. A Minajigi, J E Froberg, C Wei, H Sunwoo, Science. 349Minajigi A, Froberg JE, Wei C, Sunwoo H, et al. 2015. Chromosomes. A comprehensive Xist interactome reveals cohesin repulsion and an RNA-directed chromosome conformation. Science 349: aab2276:1-12.

Identification of Spen as a crucial factor for Xist function through forward genetic screening in haploid embryonic stem cells. A Monfort, G Di Minin, A Postlmayr, R Freimann, Cell Rep. 12Monfort A, Di Minin G, Postlmayr A, Freimann R, et al. 2015. Identification of Spen as a crucial factor for Xist function through forward genetic screening in haploid embryonic stem cells. Cell Rep 12: 554-61.

. C Pr Ospects &amp; Overviews, Vallot, Pr ospects & Overviews C. Vallot et al.

. Bioessays. 38The Authors BioEssays Published by WILEY Periodicals, IncBioessays 38: 869-880, ß 2016 The Authors BioEssays Published by WILEY Periodicals, Inc.

A novel role for Xist RNA in the formation of a repressive nuclear compartment into which genes are recruited when silenced. J Chaumeil, Le Baccon, P Wutz, A Heard, E , Genes Dev. 20Chaumeil J, Le Baccon P, Wutz A, Heard E. 2006. A novel role for Xist RNA in the formation of a repressive nuclear compartment into which genes are recruited when silenced. Genes Dev 20: 2223-37.

Integrated kinetics of X chromosome inactivation in differentiating embryonic stem cells. J Chaumeil, I Okamoto, M Guggiari, E Heard, Cytogenet Genome Res. 99Chaumeil J, Okamoto I, Guggiari M, Heard E. 2002. Integrated kinetics of X chromosome inactivation in differentiating embryonic stem cells. Cytogenet Genome Res 99: 75-84.

The inactive X chromosome in female mammals is distinguished by a lack of histone H4 acetylation, a cytogenetic marker for gene expression. P Jeppesen, B M Turner, Cell. 74Jeppesen P, Turner BM. 1993. The inactive X chromosome in female mammals is distinguished by a lack of histone H4 acetylation, a cytogenetic marker for gene expression. Cell 74: 281-9.

Evolutionary diversity and developmental regulation of X-chromosome inactivation. M Escamilla-Del-Arenal, S T Da Rocha, E Heard, Hum Genet. 130Escamilla-Del-Arenal M, da Rocha ST, Heard E. 2011. Evolutionary diversity and developmental regulation of X-chromosome inactivation. Hum Genet 130: 307-27.

Methylation of histone H3 at Lys-9 is an early mark on the X chromosome during X inactivation. E Heard, C Rougeulle, D Arnaud, P Avner, Cell. 107Heard E, Rougeulle C, Arnaud D, Avner P, et al. 2001. Methylation of histone H3 at Lys-9 is an early mark on the X chromosome during X inactivation. Cell 107: 727-38.

Histone H3 lysine 9 methylation occurs rapidly at the onset of random X chromosome inactivation. J E Mermoud, B Popova, A H Peters, T Jenuwein, Curr Biol. 12Mermoud JE, Popova B, Peters AH, Jenuwein T, et al. 2002. Histone H3 lysine 9 methylation occurs rapidly at the onset of random X chromosome inactivation. Curr Biol 12: 247-51.

Differential histone H3 Lys-9 and Lys-27 methylation profiles on the X chromosome. C Rougeulle, J Chaumeil, K Sarma, C D Allis, Mol Cell Biol. 24Rougeulle C, Chaumeil J, Sarma K, Allis CD, et al. 2004. Differential histone H3 Lys-9 and Lys-27 methylation profiles on the X chromosome. Mol Cell Biol 24: 5475-84.

Dynamic changes in paternal X-chromosome activity during imprinted X-chromosome inactivation in mice. C Patrat, I Okamoto, P Diabangouaya, V Vialon, Proc Natl Acad Sci. 106Patrat C, Okamoto I, Diabangouaya P, Vialon V, et al. 2009. Dynamic changes in paternal X-chromosome activity during imprinted X-chromosome inactivation in mice. Proc Natl Acad Sci USA 106: 5198-203.

Unusual chromatin status and organization of the inactive X chromosome in murine trophoblast giant cells. C Corbel, P Diabangouaya, A V Gendrel, J C Chow, Development. 140Corbel C, Diabangouaya P, Gendrel AV, Chow JC, et al. 2013. Unusual chromatin status and organization of the inactive X chromo- some in murine trophoblast giant cells. Development 140: 861-72.

Lineagespecific regulation of imprinted X inactivation in extraembryonic endoderm stem cells. S Merzouk, J L Deuve, A Dubois, P Navarro, Epigenetics Chromatin. 711Merzouk S, Deuve JL, Dubois A, Navarro P, et al. 2014. Lineage- specific regulation of imprinted X inactivation in extraembryonic endoderm stem cells. Epigenetics Chromatin 7: 11.

Imprinted X-inactivation in extra-embryonic endoderm cell lines from mouse blastocysts. T Kunath, D Arnaud, G D Uy, I Okamoto, Development. 132Kunath T, Arnaud D, Uy GD, Okamoto I, et al. 2005. Imprinted X-inactivation in extra-embryonic endoderm cell lines from mouse blastocysts. Development 132: 1649-61.

Mitotically stable association of polycomb group proteins Eed and Enx1 with the inactive X chromosome in trophoblast stem cells. W Mak, J Baxter, J Silva, A E Newall, Curr Biol. 12Mak W, Baxter J, Silva J, Newall AE, et al. 2002. Mitotically stable association of polycomb group proteins Eed and Enx1 with the inactive X chromosome in trophoblast stem cells. Curr Biol 12: 1016-20.

Distinct histone modifications in stem cell lines and tissue lineages from the early mouse embryo. P J Rugg-Gunn, B J Cox, A Ralston, J Rossant, Proc Natl Acad Sci. 107Rugg-Gunn PJ, Cox BJ, Ralston A, Rossant J. 2010. Distinct histone modifications in stem cell lines and tissue lineages from the early mouse embryo. Proc Natl Acad Sci USA 107: 10783-90.

Differences in the DNA of the inactive X chromosomes of fetal and extraembryonic tissues of mice. P G Kratzer, V M Chapman, H Lambert, R E Evans, Cell. 33Kratzer PG, Chapman VM, Lambert H, Evans RE, et al. 1983. Differences in the DNA of the inactive X chromosomes of fetal and extraembryonic tissues of mice. Cell 33: 37-42.

X inactivation in the mouse embryo deficient for Dnmt1: distinct effect of hypomethylation on imprinted and random X inactivation. T Sado, S-S Tan, Mhf, P Tam, Dev Biol. 225Sado T, Tan S-S MHF, Tam P, et al. 2000. X inactivation in the mouse embryo deficient for Dnmt1: distinct effect of hypomethylation on imprinted and random X inactivation. Dev Biol 225: 294-303.

X inactivation and histone H4 acetylation in embryonic stem cells. A M Keohane, L P Neil, N D Belyaev, J S Lavender, Dev Biol. 180Keohane AM, O'Neil LP, Belyaev ND, Lavender JS, et al. 1996. X inactivation and histone H4 acetylation in embryonic stem cells. Dev Biol 180: 618-30.

Escape from X inactivation varies in mouse tissues. J B Berletch, W Ma, F Yang, J Shendure, PLoS Genet. 111005079Berletch JB, Ma W, Yang F, Shendure J, et al. 2015. Escape from X inactivation varies in mouse tissues. PLoS Genet 11: e1005079.

Site-specific silencing of regulatory elements as a mechanism of X inactivation. J M Calabrese, W Sun, L Song, J W Mugford, Cell. 151Calabrese JM, Sun W, Song L, Mugford JW, et al. 2012. Site-specific silencing of regulatory elements as a mechanism of X inactivation. Cell 151: 951-63.

Global survey of escape from X inactivation by RNA-sequencing in mouse. F Yang, T Babak, J Shendure, C M Disteche, Genome Res. 20Yang F, Babak T, Shendure J, Disteche CM. 2010. Global survey of escape from X inactivation by RNA-sequencing in mouse. Genome Res 20: 614-22.

An X-linked GFP transgene reveals unexpected paternal X-chromosome activity in trophoblastic giant cells of the mouse placenta. A K Hadjantonakis, L L Cox, P P Tam, A Nagy, Genesis. 29Hadjantonakis AK, Cox LL, Tam PP, Nagy A. 2001. An X-linked GFP transgene reveals unexpected paternal X-chromosome activity in trophoblastic giant cells of the mouse placenta. Genesis 29: 133-40.

A rapid passage through a two-active-X-chromosome state accompanies the switch of imprinted X-inactivation patterns in mouse trophoblast stem cells. J Prudhomme, A Dubois, P Navarro, D Arnaud, Epigenetics Chromatin. 852Prudhomme J, Dubois A, Navarro P, Arnaud D, et al. 2015. A rapid passage through a two-active-X-chromosome state accompanies the switch of imprinted X-inactivation patterns in mouse trophoblast stem cells. Epigenetics Chromatin 8: 52.

Multiple spatially distinct types of facultative heterochromatin on the human inactive X chromosome. B P Chadwick, H F Willard, Proc Natl Acad Sci. 101Chadwick BP, Willard HF. 2004. Multiple spatially distinct types of facultative heterochromatin on the human inactive X chromosome. Proc Natl Acad Sci USA 101: 17450-5.

Chromatin of the Barr body: histone and non-histone proteins associated with or excluded from the inactive X chromosome. B P Chadwick, H F Willard, Hum Mol Genet. 12Chadwick BP, Willard HF. 2003. Chromatin of the Barr body: histone and non-histone proteins associated with or excluded from the inactive X chromosome. Hum Mol Genet 12: 2167-78.

Differentially methylated forms of histone H3 show unique association patterns with inactive human X chromosomes. B A Boggs, P Cheung, E Heard, D L Spector, Nat Genet. 30Boggs BA, Cheung P, Heard E, Spector DL, et al. 2002. Differentially methylated forms of histone H3 show unique association patterns with inactive human X chromosomes. Nat Genet 30: 73-6.

Heterochromatin, HP1 and methylation at lysine 9 of histone H3 in animals. I G Cowell, R Aucott, S K Mahadevaiah, P S Burgoyne, Chromosoma. 111Cowell IG, Aucott R, Mahadevaiah SK, Burgoyne PS, et al. 2002. Heterochromatin, HP1 and methylation at lysine 9 of histone H3 in animals. Chromosoma 111: 22-36.

Variation in Xi chromatin organization and correlation of the H3K27me3 chromatin territories to transcribed sequences by microarray analysis. B P Chadwick, Chromosoma. 116Chadwick BP. 2007. Variation in Xi chromatin organization and correlation of the H3K27me3 chromatin territories to transcribed sequences by microarray analysis. Chromosoma 116: 147-57.

The macrosatellite DXZ4 mediates CTCF-dependent long-range intrachromosomal interactions on the human inactive X chromosome. A H Horakova, S C Moseley, C R Mclaughlin, D C Tremblay, Hum Mol Genet. 21Horakova AH, Moseley SC, McLaughlin CR, Tremblay DC, et al. 2012. The macrosatellite DXZ4 mediates CTCF-dependent long-range intrachromosomal interactions on the human inactive X chromosome. Hum Mol Genet 21: 4367-77.

A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. S S Rao, M H Huntley, N C Durand, E K Stamenova, Cell. 159Rao SS, Huntley MH, Durand NC, Stamenova EK, et al. 2014. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell 159: 1665-80.

Bipartite structure of the inactive mouse X chromosome. X Deng, W Ma, V Ramani, A Hill, Genome Biol. 16152Deng X, Ma W, Ramani V, Hill A, et al. 2015. Bipartite structure of the inactive mouse X chromosome. Genome Biol 16: 152.

Evidence for compensatory upregulation of expressed X-linked genes in mammals, Caenorhabditis elegans and Drosophila melanogaster. X Deng, J B Hiatt, D K Nguyen, S Ercan, Nat Genet. 43Deng X, Hiatt JB, Nguyen DK, Ercan S, et al. 2011. Evidence for compensatory upregulation of expressed X-linked genes in mammals, Caenorhabditis elegans and Drosophila melanogaster. Nat Genet 43: 1179-85.

Sensing X chromosome pairs before X inactivation via a novel X-pairing region of the Xic. S Augui, G J Filion, S Huart, Nora E , Science. 318Augui S, Filion GJ, Huart S, Nora E, et al. 2007. Sensing X chromosome pairs before X inactivation via a novel X-pairing region of the Xic. Science 318: 1632-6.

Highresolution analysis of epigenetic changes associated with X inactivation. H Marks, J C Chow, S Denissov, K J Francoijs, Genome Res. 19Marks H, Chow JC, Denissov S, Francoijs KJ, et al. 2009. High- resolution analysis of epigenetic changes associated with X inactivation. Genome Res 19: 1361-73.

Setdb1-mediated H3K9 methylation is enriched on the inactive X and plays a role in its epigenetic silencing. A Keniry, L J Gearing, N Jansz, J Liu, Epigenetics Chromatin. 916Keniry A, Gearing LJ, Jansz N, Liu J, et al. 2016. Setdb1-mediated H3K9 methylation is enriched on the inactive X and plays a role in its epigenetic silencing. Epigenetics Chromatin 9: 16.

The Xist lncRNA exploits three-dimensional genome architecture to spread across the X chromosome. J M Engreitz, A Pandya-Jones, P Mcdonel, A Shishkin, Science. 3411237973Engreitz JM, Pandya-Jones A, McDonel P, Shishkin A, et al. 2013. The Xist lncRNA exploits three-dimensional genome architecture to spread across the X chromosome. Science 341: 1237973.

A dual origin of the Xist gene from a protein-coding gene and a set of transposable elements. E A Elisaphenko, N N Kolesnikov, A I Shevchenko, I B Rogozin, PLoS ONE. 32521Elisaphenko EA, Kolesnikov NN, Shevchenko AI, Rogozin IB, et al. 2008. A dual origin of the Xist gene from a protein-coding gene and a set of transposable elements. PLoS ONE 3: e2521.

Jarid2 Is Implicated in the Initial Xist-Induced Targeting of PRC2 to the Inactive X Chromosome. S T Da Rocha, V Boeva, M Escamilla-Del-Arenal, K Ancelin, Mol Cell. 53da Rocha ST, Boeva V, Escamilla-Del-Arenal M, Ancelin K, et al. 2014. Jarid2 Is Implicated in the Initial Xist-Induced Targeting of PRC2 to the Inactive X Chromosome. Mol Cell 53: 301-16.

Cdyl, a new partner of the inactive X chromosome and potential reader of H3K27me3 and H3K9me2. M Escamilla-Del-Arenal, S T Da Rocha, C G Spruijt, O Masui, Mol Cell Biol. 33Escamilla-Del-Arenal M, da Rocha ST, Spruijt CG, Masui O, et al. 2013. Cdyl, a new partner of the inactive X chromosome and potential reader of H3K27me3 and H3K9me2. Mol Cell Biol 33: 5005-20.

Enlightening the contribution of the dark matter to the X chromosome inactivation process in mammals. M Casanova, Liyakat Ali, T M Rougeulle, C , 10.1016/j.semcdb.2016.05.003Semin Cell Dev Biol. Casanova M, Liyakat Ali TM, Rougeulle C. 2016. Enlightening the contribution of the dark matter to the X chromosome inactivation process in mammals. Semin Cell Dev Biol, http://dx.doi.org/10.1016/j. semcdb.2016.05.003.

H3K27me3 forms BLOCs over silent genes and intergenic regions and specifies a histone banding pattern on a mouse autosomal chromosome. F M Pauler, M A Sloane, R Huang, K Regha, Genome Res. 19Pauler FM, Sloane MA, Huang R, Regha K, et al. 2009. H3K27me3 forms BLOCs over silent genes and intergenic regions and specifies a histone banding pattern on a mouse autosomal chromosome. Genome Res 19: 221-33.

. C Vallot, C. Vallot et al.

. . Prospects &amp; Overviews, Prospects & Overviews ....