# A Review of QTL Mapping in Cotton: Molecular Markers, Mapping Populations and Statistical Methods

CorpusID: 90560371
 
tags: #Agricultural_And_Food_Sciences, #Biology

URL: [https://www.semanticscholar.org/paper/794d18d191459374980d99da7f53fca14be74122](https://www.semanticscholar.org/paper/794d18d191459374980d99da7f53fca14be74122)
 
| Is Survey?        | Result          |
| ----------------- | --------------- |
| By Classifier     | False |
| By Annotator      | (Not Annotated) |

---

A Review of QTL Mapping in Cotton: Molecular Markers, Mapping Populations and Statistical Methods
2017

Ashok Kumar Meena 
University of Agricultural Sciences Dharwad
580005India

M Ramesh 
PDR
HyderabadIndia

C H Nagaraju 
University of Agricultural and Horticultural Sciences
ShivamoggaIndia

Bheru Lal Kumhar 
Agriculture University
KotaIndia

A Review of QTL Mapping in Cotton: Molecular Markers, Mapping Populations and Statistical Methods

Int.J.Curr.Microbiol.App.Sci
66201710.20546/ijcmas.2017.606.3643057 Review Article *Corresponding author A B S T R A C T


## Introduction

The primary breeding goal for the worldwide cotton scientists is how to genetically improve both yield and fibre quality. Previous research reports showed that yield, yield contributing and fibre quality traits of interest were negatively associated and controlled by multiple environmental sensitive quantitative genes. Current genetic information and plant breeding methods cannot lead to improvement of such negative association and controlling multiple environmental sensitive quantitative genes for yield and fibre quality. In conventional breeding aim is to develop both high yield and superior quality fibre properties but the quality of fibre can be determined only after harvesting and testing of the fibre. As a result, it is difficult, expensive and time consuming to develop cotton cultivars with high yield and superior quality fibre by these methods. Acceleration of the conventional breeding method has become possible by using biotechnological tool called molecular markers. Construction of genetic linkage maps has been recognized The upland cotton (Gossypium hirsutum L.) accounts for about 95% of world cotton production. Improving upland cotton cultivars has been the focus of worldwide cotton breeding programs. In cotton, seed cotton yield, yield contributing and fibre quality traits are under the control of polygenes or quantitative trait locus (QTL), for these traits QTL analysis holds a great promises these are the genomic regions that links the information between phenotypic (trait measurement) and genetic data (molecular markers) and explain the genetic basis of variation in complex traits. The development of appropriate molecular markers in the background of suitable mapping population and construction of genetic linkage maps and QTL identification using statistical programs are earnest for QTL mapping. Present review provides an updates on comparative QTL analysis to obtain a better insight into the genome-wide distribution of QTL and to identify consistent QTL for marker assisted breeding and marker-assisted QTL manipulation to the genetic improvement of quantitative traits in cotton.

as an essential tool for plant molecular breeding using molecular markers or DNA markers because they have the properties of neutrality, lack of epitasis and are simply inherited Mendelian characters. Therefore in marker assisted selection (MAS) the use of DNA markers which is highly associated with traits of importance will be an important approach in reaching breeding goal. Various types of DNA markers viz., RFLP, the PCR based DNA markers such as AFLP, RAPD, SSR, STS and EST-SSR have been widely used in cotton linkage (i.e. Lacape et al., 2003;Zhang et al., 2003) and SNP markers. Recent availability of cotton genome reference sequences for G. raimondii (Paterson et al., 2012), draft sequences for G. arboreum (Li et al., 2014), G. raimondii  and draft sequences for G. arboreum and G. herbaceum (Katageri et al.,2014) and millions of SNPs were generated in different crops such as Soybean (Lam et al., 2010), Arabidopsis (Zhang et al., 2009), Rice (Subbaiyan et al., 2012;Xu et al., 2012) and other crops (Sim et al., 2012;Sharpe et al., 2013;Delourme et al., 2013), help to cotton scientists for genome based identification efforts and mapping the QTLs. High throughput genome-scale nextgeneration sequencing (NGS) technologies provide new strategies for sequence-based SNP genotyping. As a result, genotypic data and phenotypic data are widely used in construction of linkage groups and QTL tagging. The F 2 , backcross and recombinant inbred (RI) populations have been most popularly used for QTLs mapping. Each population has some advantages and disadvantages (Paterson, 1996).

In cotton crops most traits of economical importance, including seed cotton yield, yield contributing and fibre quality traits are controlled by many genes and are known as quantitative traits (also -polygenic‖ -multifactorial‖ or -complex‖ traits). The term QTL was first coined by Geldermann (1975). The regions within genomes that contain genes associated with a particular quantitative trait are known as QTLs. Conceptually, a QTL can be a single gene, or it may be a cluster of linked genes that affect trait. The procedures for finding and locating the QTLs and analyzing their magnitude of genetic effects and interactions with environment are called QTL mapping. The development of molecular markers and the use of these markers in QTL analysis is increasingly becoming a common approach for evaluating the inheritance and feasibility of accelerating gains from selection for complex quantitative traits in crop plants. Yield contributing and fibre quality traits for which QTL analysis holds great promise.

QTL mapping requires (1) selection of appropriate molecular marker (s) and generation molecular data with adequate number of uniformly-spaced polymorphic markers; (2) Development of appropriate mapping population and phenotyping the population for the trait (s) of interest; (3) Construction of genetic linkage map and identification of QTLs for the trait (s) of interest using statistical programs. Details on molecular markers, mapping population, statistical methods, linkage maps and QTL mapping of agronomics and fibre quality related traits are reviewed here under.


## Molecular markers

Until recent advances in molecular genetics, breeders have been improving phenotype through evaluation and selection, which were resource-consuming. Currently, two main types of molecular markers, biochemical markers and DNA based markers are available for genetic studies. Distinguish the molecular markers from morphological markers (1) Distinguish the genotypes at the any part of plants;

(2) these markers behave in a co-dominant manner, allowing the heterozygotes to be differentiated from homozygotes; (4) phenotypic neutrality: deleterious effects are not usually associated with different alleles; (5) alleles at many loci are co-dominant, thus all possible genotypes can be distinguished and (6) few epistatic or pleiotropic effects are observed. All these advantages make molecular genetic markers very important tools in various genetic analyses and crop improvement strategies.

A DNA marker is considered as good or powerful if it is easy to detect, amenable for automation, highly polymorphic and distributed across genome at random. These molecular markers include: (i) hybridizationbased markers such as restriction fragment length polymorphism (RFLP) (ii) PCR-based markers: random amplification of polymorphic DNA (RAPD), amplified fragment length polymorphism (AFLP) and microsatellite or simple sequence repeat (SSR) and (iii) sequence-based markers: single nucleotide polymorphism (SNP). The majority of these molecular markers has been developed either from genomic DNA libraries (e.g. RFLPs and SSRs) or from random PCR amplification of genomic DNA (e.g. RAPDs) or both (e.g. AFLPs). Different types of molecular markers commonly used in cotton breeding programs are presented in table 1, and their application in cotton improvements I as follows:


## Restriction Fragment Length Polymorphisms (RFLPs)

It is hybridization based techniques in which organisms are differentiated by analysis of patterns derived from cleavage of their DNA by restriction enzymes. The main steps involve isolation of DNA, digestion with restriction enzymes, separation of restricted fragments by agarose gel electrophoresis, transfer of fragments to nylon membrane, hybridization with probe and scoring of polymorphism by autoradiography. In various species of cotton, RFLP markers have been used to study the genetic diversity, population genetics, evolution and phylogenetic relationships (Yu et al., 1997). Brubaker et al., 1999;Meredith et al., 2000 andUlloa et al., 2002 are published genetic mapping of cotton using RFLPs and it was reported that in cotton 64 % RFLPs are codominant in nature (Reinisch et al., 1994). Genetic diversity in upland cotton has also been examined using RFLP markers (Brubaker and wendel et al., 1994). Molecular map of the cotton genome was first constructed using 705 RFLP loci and partitioned into 41 linkage groups (Reinisch et al., 1994). Wright et al., 1998, reported utility of RFLP markers in marker assisted selection (MAS) and RFLP linked to resistance allele for pathogen of bacterial blight was validated. RFLP markers are very complex and time and cost intensive technique which restrict it's uses and leads to development of less complicated techniques known as PCR base markers (Agarwal et al., 2008).However, at present, RFLPs are not popular in cotton genome studies because of low ability to detect polymorphism in cotton compared to other plant taxa (Brubaker et al., 2000).


## Random amplified polymorphic DNA (RAPD)

In RAPDs, DNA fragments are amplified by the PCR reaction using random primers (usually of 10 bp) (Khanam et al., 2012). Polymorphism is obtained because of sequence variation in the genome for primer binding sites, making RAPDs as dominant marker. RAPD marker system is easy to carry out, needs no prior sequence information, requires very less amount of DNA and is amenable to automation. However, the technique suffers with low reproducibity (Rafalski et al., 1997).RAPD techniques have been used for many purposes in cotton including assessment of, diversity, genome mapping, phylogentic studies (Rahman et al., 2002;Zhong et al., 2002;Rahman et al., 2008 andRana andBhat 2004), genetic variations in population (Chalmers et al., 1992), DNA fingerprinting (Multani et al., 1995) and determining the relationship between the genotypes of different and same species . In cotton RAPDs were used to distinguish the cotton varieties resistant to jassids, aphids and mites (Geng et al., 1995). RAPD marker (R-6592) for the male sterility gene has been identified in cotton (Lan et al., 1999). RAPD techniques were used to evaluate the genetic relationship among cotton genotypes (Shu et al., 2001), to identify the QTLs for stomatal conductance (Ulloa and Meredith 2000) and to construct linkage mapping in cotton.


## Amplified Fragment Length Polymorphism (AFLP)

It is a technique which combines reliability of RLFP with the ease of RAPD (Vos et al., 1995). The process involves three simple steps: (i) restriction of genomic DNA and ligation of oligonucleotide adaptors (ii) pre and selective amplification of restriction fragments and (iii) gel analysis of amplified fragments. The polymorphic fragments are detected as present or absent making it a dominant marker system. The technique can be automated and allows the simultaneous analysis of many genetic loci per experiments. AFLP produces more polymorphic loci per primer than RFLPs, SSRs or RAPDs (Maughan et al., 1996).AFLP is an effective tool for the observation of genetic diversity (Murtaza et al., 2006), fingerprinting studies and tagging of agronomic, seed and fibre quality traits (Zhong et al., 2002;Rakshit et al., 2010 andBadigannavar andMyers 2010). AFLP is a great valued technique for gene mapping studies due to their high abundance and random distribution throughout the genome (Voset al., 1995). A linkage map of cotton was developed using the AFLP and RAPD markers (Altaf et al., 1997). AFLP markers have also been used for analyzing the genetic diversity (Abdalla et al., 2001 andRana andBhat 2004) and map saturation in cotton (Zhang et al., 2005 andLacape et al., 2003).


## Inter Simple Sequence Repeats (ISSR)

It allows the detection of polymorphism in inter SSR loci using primer (16-25 bp long) complimentary to a single SSR and anneal at either the 3' or 5' end (Khanam et al., 2012) which can be di, tri, tetra or pentanucleotide (Reddy et al., 2002). The technique of ISSR markers combines many benefits of AFLPs and SSRs with universality of RAPDs (Bornet and Branchard et al., 2001). Generally the sequence of ISSR primers is larger as compare to RAPD primers, allowing higher annealing temperature which results in greater reproducibility of bands than RAPDs (Reddy et al., 2002, Culley andWolf et al., 2000). Amplification of ISSRs also revealed larger fragments number per primer than RAPDs (Wang and Yi et al., 2002).

Many earlier studies reported that ISSR markers were more informative than RAPDs for genetic diversity evaluation in different crop species (Nagaoka and Ogihara et al., 1997;Galv´an et al., 2003). The applications of ISSRs for different purposes depend on the diversity and frequencies of SSR within the particular genomes (Shi et al., 2010). It is quickly being utilized by the research community in different areas of plant improvement like in gene tagging, analysis of genetic diversity and estimation of SSR motif [Blair et al., 1999;Bornet et al., 2002 andSica et al., 2005]. ISSRs have been reported as quite useful markers for revealing polymorphism in cotton genotypes (Liu and Wendel 2001).


## Microsatellites or Simple Sequence Repeats (SSR)

These are di-, tri-, tetra-or pentatandom repeats of nucleotide, scattered abundantly in both noncoding and coding regions of a genome (Kalia et al., 2011;Khanam et al., 2012). Microsatellites are created from sphere where variants of repetitive DNA sequence are previously over represented (Tautz et al., 1986). The loci of these markers are highly transferable about 50% across species (Saha et al., 2004). For SSRs analysis forward and reverse primers are employed in PCR reaction that anneal to the template DNA at the 5' and 3' ends. Short repetitive DNA sequences furnish the basis for multi allelic, co-dominant PCR based molecular marker and found more polymorphic as compare to other DNA markers (Preetha andRaveendr et al., 2008 andKhanam et al., 2012). Due to their greater polymorphism, SSRs are considered as an important marker system in fingerprinting, analysis of genetic diversity, molecular mapping and marker assisted selection (Reddy et al., 2002). Several methods have been pursued to develop SSR markers in cottons, including analysis of SSR-enriched small insert genomic DNA libraries (Richard and Beckman et al., 1995;Udall et al., 2006;Ince et al., 2010 andKalia et al.,2011), SSR mining from ESTs (Shaheen et al., 2009) and large-insert BAC derivation by end sequence analysis (Reddy et al., 2002). Cotton researchers have explored simple sequence repeats (SSRs) for studies of the phylogenetic and diversity analysis (Lacape et al., 2007) genetic mapping Lacape et al., 2009;Park et al., 2005;Xiao et al., 2009;Yu et al., 2011;Yu et al., 2013 andGore et al., 2014), association mapping (Kantartzi et al., 2008).


## Single Nucleotide Polymorphism (SNPs)

To understand the shift to single nucleotide polymorphism (SNP) markers, we must first look into the limitations of SSR markers. First, there are limited numbers of SSR motifs in the genome which becomes a constraint when trying to saturate a region with markers or when trying to identify gene-based markers. In addition, one of the main advantages of SSRs is high information content from multiple alleles per locus and also presents difficulties when merging SSR data from different platforms and curating allele sizes in databases. In addition, gelbased SSRs are labor intensive and automated fragment sizing systems have limited scope for multiplexing. Therefore, SSR genotyping quickly hits a point where the low throughput and higher cost becomes a limiting factor which is in contrast to recent SNP genotyping techniques. The main advantages of SNP markers relate to their ease of data management along with their flexibility, speed and cost-effectiveness. Bi-allelic SNP markers are straight forward to merge data across groups and create large databases of marker information, since there are only two alleles per locus and different genotyping platforms will provide the same allele calls once proper data QC has been performed. A major factor in the advantages of SNP markers for flexibility, speed and costeffectiveness is the range of genotyping platforms available to address a variety of needs for different marker densities and costs per sample. Variations of single nucleotide (A, T, C, and G) in sequence of individual genome are known as single nucleotide polymorphism or SNPs (Agarwal et al., 2008). These may occur in the non-coding, coding and intergenic regions of the genome, so allowing the detection of the genes due to the variations in the sequences of nucleotides (Agarwal et al., 2008, Ayeh 2008) and these are either non synonymous or synonymous within the coding regions of the genome. Synonymous changes can alter mRNA splicing that result the changes in the phenotype of an individual (Richard and Beckman 1995). SNP markers are important tool for linkage mapping, map based cloning and marker assisted selection due to the high level of polymorphism. The co-dominant nature of SNPs makes these markers able to distinguish the heterozygous and homozygous alleles (Shaheen et al., 2009).In cotton, many research have been conducted to observe diversity, characterization and mapping of SNPs in the nucleotide sequence of Gossypium genome , Deynze et al., 2009. Recently, an international collaborative effort has developed 70K SNP chip based on Illumina Infinium genotyping assay (Unpublished data; http://www.cottongen.org/node/ 1287616). This high-throughput genotyping assay will be a resource that will be used globally by public and private breeders, geneticists and other researchers to enhance cotton genetic analysis, breeding, genome sequence assembly and many other uses.


## Mapping population

To study genotypes diversity, finger printing, gene tagging, map construction and QTLs identification all these requires appropriate mapping population and is very critical for the success of QTL mapping project. These populations are developed by crossing between two inbred parents with clear contrasting difference in their phenotypic traits of interest. In auto gamous species, QTL mapping studies make use of F 2 or segregating generation derived families, backcross (BC), recombinant inbred lines (RILs), near isogenic lines (NILs) and double haploids (DH). The primary mapping populations for QTLs mapping is F 2 , backcross (BC), recombinant inbred lines (RILs) and double haploid (DH) populations. Both F 2 and BC populations are the simplest types of mapping populations because they are easy to construct and require only a short time to produce. F 2 is more powerful for detecting QTLs with additive effects and can also be used to estimate the degree of dominance for detected QTLs. In cotton several studies used F 2 as mapping population (Reinisch et al., 1994;Jiang et al., 1998;Jiang et al., 2000;Kohel et al., 2001;Saranga et al., 2001;Rong et al., 2007 andYu et al., 2007). When dominance is present, backcrosses give biased estimates of the effects because additive and dominant effects are completely confounded. However, both F 2 and BC populations have three limitations. First, development of these populations require relatively few meioses such that even markers that are far from the QTLs remain strongly associated with it. Such longdistance associations hamper precise localization of the QTLs. Second, F 2 and BC populations are temporary populations as they are highly heterozygous and cannot be propagated indefinitely through seeds (i.e., these populations can't be evaluated several times in different environmental conditions, years, locations, etc.). Finally, epistatic interactions could hardly be studied in both F 2 and BC populations. RILs are derived from an F 2 population by generations of selfing (bulk or single seed descent) Beckman, 1990 andCrouch, 2008). RILs are advanced homozygous lines that have undergone several rounds of inbreeding (Darvasi and Soller, 1995). Such multiple generations of mating increases the potential number of recombination events and improves map resolution (i.e., sufficient meioses have occurred to reduce disequilibrium between moderately linked markers). In cotton a considerable number of studies have used RILs as mapping population for mapping yield and fibre quality related and other traits (Park et al., 2005;Shen et al., 2007;Wang et al., 2006;Abdurakhmonov et al., 2007;Wu et al., 2008;Zhang et al., 2009;Lacape et al., 2009Lacape et al., , 2010Gore et al., 2014andYu et al., 2013.DH populations have also been used for QTL mapping in several species (Bao et al., 2002;Mahmood et al., 2003;Behn et al., 2005;Semagn et al., 2006;Semagn et al., 2007 andCrouch, 2008).

The DH production methodology improves breeding efficiency by generating inbred lines with 100 per cent purity and genetic uniformity in just two generations. DH lines make it easy to carry genetic studies and shorten the breeding time significantly. DH populations are quicker to generate than RILs and NILs but the production of DHs is only possible for species with a well-established protocol for haploid production. RILs, NILs and DHs are permanent populations because they are homozygous or ‗true-breeding' lines that can be multiplied and reproduced without any occurrence of genetic change. Seeds from RILs, NILs and DHs can be transferred between different laboratories for mapping to ensure that all collaborators examine identical material (Young, 1994 andLekgari, 2010). So that genetic result from phenotyping, genotyping and QTL mapping can be accumulated across laboratories. In spite of the availability of various papers on genetic mapping, specific studies relating to the ideal number of individuals in a given population required to establish accurate genetic maps have yet been inconclusive. Simulation studies performed using a sample size ranging from 50 to 1000 individuals of F 2 , BC, RILs and DHs populations have shown that the type and size of mapping populations can exert an influence on the accuracy of genetic maps.


## Statistical methods for QTL analysis and mapping

QTL analysis looks for co-segregation between the quantitative trait and marker allele in a segregating population. Undoubtedly, the development of statistical methods has played an important role for the detection of the association between DNA markers and quantitative characters. The first report of an association between a morphological marker and a quantitative trait was reported by Sax (1923).QTL mapping programs can be roughly classified into different groups according to the number of markers or genetic models and analytical approaches applied. According to the number of markers, models can be classified as single-QTL models and multiple-locus models (Liu, 1998). According to the analytical technology, the methods can be grouped into one-way analysis of variance (ANOVA) or simple t-test, simple linear regression, multiple linear regression, nonlinear regression, log-linear regression, likelihood functions, MCMC (Markoff Chain Monte Carlo) and mixed linear models (Wang et al., 1999).

Briefly, the statistical analyses of associations between phenotype and genotype in a population to detect QTLs include singlemarker mapping (Luo and Kearsey, 1989), simple interval mapping (SIM) (Lander and Botstein, 1989) and composite interval mapping (CIM) (Zeng, 1994), multiple interval mapping (MIM) (Jiang and Zeng 1995;Ronin et al., 1995) as follow:


## Single Marker Analysis (SMA)

The simplest method for QTL mapping is single-marker mapping, includes t-test, ANOVA and simple linear regression, which assess the segregation of a phenotype with respect to a marker genotype (Soller and Brody, 1976). According to this principle progeny is classified by marker genotype and phenotypic mean between classes is compared (t-test or ANOVA). A significant difference indicates that a marker is linked to a QTL. The difference between the phenotypic mean provides an estimate of the QTL effect. This approach can indicate which markers linked to potential QTLs are significantly associated with the quantitative trait investigated. In short, QTL location is indicated only by looking at which markers give the greatest differences between genotype group averages. Depending on the density of markers, the apparent QTL effect at a given marker may be smaller than the true QTL effect as a result of recombination between the marker and the QTL. The advantage of this method is that it is a simple procedure that can be accomplished by a standard statistical analysis software package, such as SAS and Minitab. In contrast the main weakness of singlemarker tests is the failure to provide an accurate estimate of QTL location or recombination frequency between the marker and the QTL because the evaluation of individual markers is independent and without reference to their position or order (Doerge and Churchill, 1996).


## Simple Interval Mapping (SIM)

Lander and Botstein, (1989) developed interval mapping, which overcomes the three disadvantages of analysis of variance at marker loci. Interval mapping is currently the most popular approach for QTL mapping in experimental crosses. The method makes use of a genetic map of the typed markers and like analysis of variance, it also assume assumes the presence of a single QTL. Each location in the genome is posited, one at a time, as the location of the putative QTL.


## Simple Interval Mapping (SIM)

Lander and Botstein, (1989) developed interval mapping, which overcomes the three disadvantages of analysis of variance at marker loci. Interval mapping is currently the most popular approach for QTL mapping in experimental crosses. The method makes use of a genetic map of the typed markers and, like analysis of variance, it also assumes the presence of a single QTL. Each location in the genome is posited, one at a time, as the location of the putative QTL. Interval mapping has several advantages over analysis of variance at the marker loci. (1) It provides a curve which indicates the evidence for QTL location.

(2) It allows for the inference of QTLs to positions between markers. (3) It provides improved estimates of QTL effects.

And perhaps most important, appropriately performed interval mapping makes proper allowance for incomplete marker genotype data. The key disadvantage to interval mapping, in comparison to analysis of variance, is that it requires some increase in computation time and the use of specially designed software. The principle behind interval mapping is to test a model for the presence of a QTL at many positions between two mapped marker loci. The model is fit and its goodness is tested using the method of maximum likelihood. If it is assumed that a QTL is located between two markers, the 2locus marker genotypes contain mixtures of QTL genotypes each. Maximum likelihood involves searching for QTL parameters that give the best approximation for quantitative trait distributions that are observed for each marker class. Models are evaluated by computing the likelihood of the observed distributions with and without fitting a QTL effect. The LOD (logarithm of the odds) score is the log of the ratio between the null hypothesis (no QTL) and the alternative hypothesis (QTL at the testing position). Large LOD scores correspond to greater evidence for the presence of a QTL. The best estimate of the location of the QTLs is given by the chromosomal location that corresponds to the highest significant likelihood ratio. The LOD score is calculated at each position of the genome.    Lander and Bostein, 1989;Lincoln et al.,(1992) 


## PlabQTL (Version 1.2) DOS

A program characterizing loci that affect the variation of quantitative traits. Utz and Melchinger (2003) 5. QGene (Version 4.0) Any computer An entirely rebuilt Java application that will run on any computer. Nelson (1997) 6. QTL Cartographer (Version 2.5)

For Windows UNIX, DOS, Windows,


## Mac OS

A program to map quantitative traits using a map of molecular markers.

Bastenet al., (1994);   In case of many missing genotypes and large gaps on the map, the missing data are replaced by probabilities estimated from the nearest flanking markers (Broman, 2001). Until now, many software packages based on interval mapping were developed for QTL mapping, such as MAPMAKER/QTL (Lincoln et al., 1992) and Q Gene (Nelson, 1997) (Table 2).


## Composite Interval Mapping (CIM)

There are two problems with SIM; one is that the effects of additional QTL will contribute to sampling variance. The other is that combined effects of two linked QTLs will cause biased estimates. The ideal solution would be to fit a model that contains the effects of all QTL. Jansen (1993), Zeng (1993) and Zeng (1994) independently proposed combining SIM with multiple regression analysis in mapping, which is termed as "composite interval mapping" (CIM). Like SIM, CIM evaluates the possibility of a target QTL at multiple analysis points across each intermarker interval. However, at each point, it also includes the effect of one or more background markers that are often referred as cofactors.

The purpose of using cofactors is to minimize the effects of QTLs in the remainder of the genome when attempting to identify a QTL in a particular region.

The inclusion of cofactors in the analysis helps in one of two ways, depending on whether the background markers and the target interval are linked. If they are not linked, inclusion of the background markers makes the analysis more sensitive to the presence of a QTL in the target interval. If they are linked, inclusion of the background marker may help to separate the target QTL from other linked QTL on the far side of the background marker (Zeng, 1993;Zeng, 1994).


## Multiple Interval Mapping (MIM)

MIM (Kao et al., 1999) is the extension of interval mapping to map multiple QTLs simultaneously, just as multiple regression extends analysis of variance. MIM allows one to infer the location of QTLs to positions between markers, makes proper allowance for missing genotype data and can allow interactions between QTLs. The idea of MIM is to fit multiple putative QTL effects and associated epistatic effects directly in a model to facilitate the search, test and estimation of positions, effects and interactions of multiple QTLs (Semagn et al., 2010). MIM consists of four components: (1) an evaluation procedure designed to analyze the likelihood of the data given in a genetic model (number, positions and epistatic terms of QTL); (2) a search strategy optimized to select the best genetic model (among those sampled) in the parameter space; (3) an estimation procedure for all parameters of the genetic architecture of the quantitative traits (number, positions, effects and epistasis of QTL; genetic variances and covariance's explained by QTL effects); and (4) a prediction procedure to estimate or predict the genotypic values of individuals and their offspring based on the selected genetic model and estimated genetic parameter values . When compared with methods such as SIM and CIM, therefore, MIM tends to be more powerful and precise in detecting QTLs. The MIM model is based on Cockerham's model for interpreting genetic parameters and the method of maximum likelihood for estimating genetic parameters ( (Kao et al., (1999) and Satagopan et al., (1996)) used a Bayesian approach relying on a Markov chain Monte Carlo simulation to map multiple QTLs.

All the different QTL mapping methods described above share a common assumption that the phenotype follows a normal distribution with equal variance in both parents. Based on these statistical principles different QTL mapping programs have been developed. Details of each statistical program are given in table 3.


## Genetic linkage maps in cotton

Genetic mapping refers to the determination of the relative position and distances between markers along chromosomes. Genetic map distances between two markers are defined as the mean number of recombination events, involving a given chromatid, in that region per meiosis. Genetic linkage maps are fundamental for the localization of genes conferring biotic and abiotic stress tolerance. Linkage maps of organisms are constructed to map genomic regions controlling qualitative and quantitative traits, to exercise indirect selection for several agronomic, fibre quality traits and to isolate the genes involved based on their map position. Genetic maps based on molecular markers have several advantages over classical maps (Subudhi and Nguyen, 2000).In cotton, the first molecular linkage map was constructed by Reinisch et al., (1994) using RFLPs molecular markers in F 2 mapping population. Till date, fifty linkage mapping studies done using intra or interspecific mapping populations (F 2 /RILs/BIL) and SSRs/RAPD/AFLPs/RFLPs/SRAP markers have been found. The range of markers mapped in different studies mapped 19 to 1306 SSR/RAPD/RFLP/AFLP markers. However, Rong et al., (2004) developed consensus genetic maps using different molecular linkage studies and mapped 2584 markers. This low number of markers maps in different studies indicates the presence of low number of polymorphic markers (Zhang et al., 2009;Wang et al., 2007;Wang et al., 2006;Rong et al., 2007 andYu et al., 2007). In cotton there is only one study available on developing a linkage map using F 2 interspecific maps and SNP markers (Hulse-Kemp et al., 2015b). So, they were able to map so many markers on cotton chromosomes with an average 0.23 cM distance between the markers, which is the very finest map available in cotton. In the present study also it was attempted to map SNP markers using recombinant inbred lines derived from interspecific cross. 63K SNP chips were used in the present study which were also used by Hulse-Kemp et al., (2015b).


## QTL mapping for yield, yield contributing and fibre quality trait in cotton

The regions in genomes to have genes linked with a quantitative trait are known as quantitative trait loci, QTLs (Collard et al., 2005) and the process of developing linkage maps and performing QTL analysis is referred to as QTL mapping (Paterson et al., 1996). QTL analysis stands on the principal of identifying a connection among phenotype and genotype of markers. Over last twenty years, twenty studies on QTL mapping for various traits have been carried out (Table 4). Out of these, seven mapping populations use F 2 and only seven mapping populations used recombinant inbred lines. All these previous studies on QTLs mapping were based on RFLP/RAPD/AFLP/ SRAPs/ REMAPs /SSRs/ EST-SSRs markers genotyping. In all these studies, the number of QTLs identified ranged from two to one hundred and eighty for yield, yield contributing and fibre quality traits.


## Marker-Assisted Selection (MAS)

Marker assisted selection (MAS) is a procedure by which a phenotype is selected on the basis of genotype of a marker (Collard et al., 2005). Selecting the plants in the segregating population that have the suitable gene combinations is the important component of plant breeding (Weeden et al., 1993). Once the markers tightly linked to the genes that are to be detected, breeders may use particular DNA marker to identify the plants carry the genes. The effectiveness and cost of MAS are influenced by the marker technique; therefore, it must be selected carefully (Young et al., 1996). During the past two decades, RAPDs techniques have been used for MAS for getting the glanded plants and glandless seeds in the interspecific population of G. sturtianumand other species (Mergeai et al., 1998). It was exposed that DNA markers connected to the major QTL (QTLFS1) for fibre strength could be utilized in MAS to increase fibre strength of commercial varieties in segregating populations (Zhang et al., 2003). Identified tightly linked QTL used for marker assisted breeding and marker-assisted QTL manipulation to the genetic improvement of quantitative traits in cotton. specific tools for genome fingerprinting. Pl. Mol. Biol. Reporter, 19(3): 209-215. Bornet, B., Muller, C., Paulus, F. and Branchard, M., 2002, highly informative nature of inter simple sequence repeat (ISSR) sequences amplified using tri-and tetra-nucleotide primers from DNA of cauliflower (Brassica oleracea var. botrytis L.). Genome, 45 (5) difference between cotton cultivars (G. hirsutum) using the RAPD method. Jiangsu J. Agri. Sci., 11(4): 21-24. Gore, M., David, A., Fang, D., Poland, J. A., Zhang, J., Percy, R. G., Cantrell, R. G., Thyssen, G. and Lipka, A. E., 2014, Linkage map construction and quantitative trait locus analysis of agronomic and fiber quality traits in cotton. Pl. Genom., Guo, W., Cai, P. and Wang, C., 2007, A microsatellite-based, generich linkage map reveals genome structure, function and evolution in Gossypium. Genet, 176 (1) 


: 527-541. He, D. H., Lin, Z. X. and Zhang, X. L., 2008, Dissection of genetic variance of fibre quality in advanced generations from an interspecific cross of Gossypium hirsutum and G. barbadense. Pl. Breed, 127(3): 286-294. He, D. H., Lin, Z. X. and Zhang, X. L.2007., -QTL mapping for economic traits based on a dense genetic map of cotton with PCR-based markers using the interspecific cross of Gossypium hirsutum × Gossypium barbadense,‖


International Journal of Current Microbiology and Applied SciencesJournal homepage: http://www.ijcmas.comISSN: 2319-7706 Volume 6 Number 6 (2017) pp. 3057-3080 



## Table . 1
.Different types of molecular markers, their advantages and disadvantagesSl. 
No. 

Feature 
RFLP 
RAPD 
AFLP 
SSR 
SNP 

1. 

Genomic abundance 
High 
High 
High 
Moderate to high 
Very high 

2. 

Genomic coverage 
Low 
copy 
coding 
region 
Whole genome 
Whole genome 
Whole genome 
Whole genome 

3. 

Expression/inheritance 
Codominant 
Dominant 
Dominant / co-
dominant 
Co-dominant 
Co-dominant 

4. 

Number of loci 
Small (<1,000) 
Small (<1,000) 
Moderate (1,000s) 
High 
(1,000s -10,000s) 

Very 
high 
(>100,000) 

5. 

Type of polymorphism 
Single base changes, 
indels 

Single base changes, 
indels 

Single 
base 
changes, indels 

Changes in length of 
repeats 

Single 
base 
changes, indels 

6. 

Reproducibility/reliability 
High 
Low 
High 
High 
High 

7. 

Genotyping throughput 
Low 
Low 
High 
High 
High 

8. 

Ease of use 
Not easy 
Easy 
Moderate 
Easy 
Easy 

9. 

Ease of automation 
Low 
Moderate 
Moderate to high High 
High 

10. 

Primary application 
Genetics 
Diversity 
Diversity 
and 
genetics 
All purposes 
All purposes 

11. 

Type of probes/primers 
Low copy DNA or 
cDNA clones 

10 
bp 
random 
nucleotides 
Specific sequence Specific sequence 
Allele-specific PCR 
primers 

12. 

Cloning and/or sequencing 
Yes 
No 
No 
Yes 
Yes 

13. 

PCR-based 
Usually no 
Yes 
Yes 
Yes 
Yes 

14. 

Radioactive detection 
Usually yes 
No 
Yes or no 
Usually no 
No 

15. 

Effective multiplex ratio 
Low 
Moderate 
High 
High 
Moderate to high 

16. 

Amount of DNA required 
Large (5 -50 μg) 
Small (0.01 -0.1 
μg) 

Moderate (0.5 -
1.0 μg) 
Small (0.05 -0.12 μg) Small (≥ 0.05 μg) 

17. 

Quality of DNA required 
High 
Moderate 
High 
Moderate to high 
High 

18. 

Technically demanding 
Moderate 
Low 
Moderate 
Low 
High 

19. 

Time demanding 
High 
Low 
Moderate 
Low 
Low 

20. 

Development/start-up cost 
Moderate to high 
Low 
Moderate 
Moderate to high 
High 


## Table . 2
.Software availability for genetic map constructionSl. 
No. 
Software name 

Plat 
form 
operating 
system 

Experimental designs 
Availability 
Reference 

1. 
AntMap 
DOS and UNIX F 2 intercross, F 2 backcross, RIL (self), 
DH 

http://cse.naro.affrc.go.jp/iwatah/ 
antmap/index.htmlhttp://cse.naro 
.affrc.go.jp/iwatah/antmap/index. 
html 

Iwata and Ninomiya 
(2006) 

2. 
CarthaGe'ne 
C++ 
F 2 intercross, F 2 backcross, RIL 
http://www.inra.fr/mia/T/Cartha 
Gene/ 

Schiex 
and 
Gaspin(1997) 
3. 
DGMAP 
UNIX 
Various including F 2 backcross 
No longer available 
Newell et al., (1995) 
4. 
JoinMap 
MS-Windows 
BC 1 , F 2 intercross, RILs (self),DH, 
DH 1 , DH 2 , HAP, HAP 1 CP 

http://www.kyazma.nl/ 
Stam (1993) 

5. 
MadMapper 
Python scripting 
language 

Specializes in RILs but flexible scoring 
scheme can be employed for many 
other design types 

http://cgpdb.ucdavis.edu/XLinka 
ge/MadMapper/ 

Kozik and Michelmore 
(2006) 

6. 
MAPMAKER/E 
XP 

DOS and UNIX F 2 intercross, F 2 backcross, RIL (self), 
F 3 intercross (self), RIL (sib) 

http://www.broadinstitute.org/ftp 
/distribution/software/mapmaker 
3/ 

Lander et al., (1987) 

7. 
Map Manager 
QTX 

Microsoft 
Windows 
and 
Mac OS 

Advanced 
intercross, 
advanced 
backcross, RILs 

http://www.mapmanager.org/ 
Manly and Olson 
(1999) 

8. 
MSTMAP 
C++ and Linux BC 1 , DH, HAP, RIL 
http://www.138.23.191.145/mst 
map/ 

Wu and Huang (2008) 

9. 
RECORD 
DOS 
BC 1 , F 2 , F 3 , RIL 
http://www.plantbreeding.wur.nl 
/UK/software_record.html 

Van et al., (2005) 

10. 
THREaD 
Mapper 

Wed-Browser 
enable tool 

F 2 intercross, F 2 backcross, RIL (self), 
DH 

http://cbr.jic.ac.uk/dicks/softwar 
e/threadmapper/index.html 

Cheema et al., (2008) 
3067 



## Table . 3
.The commonly used QTL mapping statistical programsSl. 
No. 
Name 
Platform (operating system) 
Description 
References 

1. Map Manager QTX (Version b29) Windows, Mac OS 

A graphic, interactive program to 
map QTL using intercrosses, 
backcrosses or recombinant inbred 
strains in experimental plants or 
animals 

Manly and Olson (1999) 

2. Mapmaker/ QTL (Version 1.1) 
UNIX, VMS, DOS, Mac OS 

A package containing a program for 
genetic linkage analysis and a 
program 
for mapping genes 
underlying complex traits. 



## Table . 4
.Details of QTLs identified trait-wise in cottonNB: number of bolls per plant, BW: boll weight, SI: seed index, LP: lint percent, LI: lint index, SI: seed index, SCY: seed cotton yield per plant, LY: lint yield per plant, FL: 
fiber length, FS: fiber strength, FE: fiber elongation, FU: fiber uniformity ratio, FY: fiber yellowness, FF: fiber fineness, FMT: fiber maturity, PH: plant height, FBL: fruit branch 
length, FBN: fruit branch number, FBA: fruit branch angle, FLU: fiber length uniformity, SFC: short fiber content, FR: fiber reflectance, SW: seed weight, NS: number of seeds 
per bolls, UQ: upper quartile length, SF: short fiber content, FT: fiber tenacity, IF: immature fiber content, SFI: short fiber index, NSB: number of seeds per boll 

Sl. No. Traits 
Descriptor 
Population 
Marker (number and Type) 
QTLs No. Reference 
Type 
Size 
1 
Fiber quality 
FS, FL, FF 
F 2 
171 
216 RFLP, 139 RAPDs 
13 
Kohelet al., 2001 

FS 
F 2 
186 
217 SSRs, 800 RAPDs UBC and 1040 
OPERON 
2 
Zhang et al., 2003 

LY, LP, SW, NS, UQ, SF, FL, FE, FT, FF and IF 
F 2 
120 
144 AFLPs, RFLPs and 150 SSRs 
28 
Mei et al., 2004 
FS,FE, FF, FU and FL 
F 2 
200 
448 RFLP 
28 
Zhang et al., 2011 
FS, FE, FL, FU, LP and FF 
F 2 
117 
290 SSRs and 9 AFLPs 
16 
Zhang et al., 2005 
FF 
BC3F 2 
3,662 262 RFLPs 
41 
Drayeet al., 2005 
FL, FLU and SFC 
BC3F 2 
3,662 262 RFLPs 
45 
Cheeet al., 2005 
FS, FL, FF, FE 
RILs 
95 SSRs, 72 CSR 
13 
Parket al., 2005 
FL, FS, FF and FE 
F 2 
-
1378 SSRs 
39 
Shenet al., 2005 
FS, FL, FF, FMT, FE and SFI 
RIL's 
180 
4106 SSRs, AFLPs, RAPDs and SRAPs 
48 
Wang et al., 2006 

FS, FE, FU, FL and FF 
RIL's 
270 
7508 SSRs, 384 SRAPs and 740 IT-ISJs 
13 
Zhang et al., 2009 
FE, FL, FS, FF and FU 
CP 
172 
16052 SSRs 
63 
Zhang et al., 2012 

2 
Fiber 
and 
agronomical 

SCY, LY, LP, BW, SI, FMT, PER, WF,WT, FF, FL, 
FE, FS 
RIL's 
188 
141 SSRs 
36 
Wu et al., 2009 

3 
Yield and fiber 
SCY, LI, SI, LY, no. of seeds per boll, FS, FL and FF F 2 
69 
834 SSRs, 437 SRAPs, 107 RAPDs, 16 
REMAPs 
57 
He et al.,2008 

FS, FL, FF, FE, LP, SI, NB, SCY and LY 
RIL's 
258 
2131 SSRs 
53 
Shenet al., 2007 

LI, SI, LY, SCY, NSB and FS 
F 2 
69 
834 SSRs, 437 SRAPs, 107 RAPDs and 16 
REMAPs 
52 
He et al., 2007 

NB, BW, SI, LP, LI, SCY, LY, FL, FS, FF, FE and 
FU 

4WC 
and 
inbred lines 
280 
6123 SSRs and EST-SSRs 
31 
Qin et al., 2008 

SCY, LY, NB, BW, LP, SI, LI and FBN 
RIL's and IF 2 180 
2675 EST-SSRs 
111 
Liu et al., 2012 

PH, FBN, BW, LP, LI, SI, LY, FL, FS, FE, FF and FU 
G. hirsutum 
accessions 
81 
121 SSRs 
180 
Zhang et al., 2013 

SCY, LY,LI, BW, FL, FS, FU 
BILs 
146 
2,041 SSRs 
67 
Yu et al., 2013 
LI, LY, PH, FL, FF, FS, FU 
RILs 
98 
2,183 
28 
Gore et al., 2014 



: 890-896. Broman, K. W., 2001, Review of statistical methods for QTL mapping in experimental crosses. Laboratory Animals, 30(7): 44-52. Brubaker, C. L. and Wendel, J. F., 1994, Re-RFLPs). Amer. J. Bot., 81(10): 1309-1326. Brubaker, C. L., Paterson, A. H. and Wendel, J. F., 1999, Comparative genetic mapping of allotetraploid cotton and its diploid progenitors. Genome, 42(2): 184-203. Chalmers, K. J., Waugh, R., Sprent, J. I., Simons, A. J. and Powell, W., 1992, Detection of genetic variation between and within populations of Gliricidiasepium and G. maculata using RAPD markers. Heredity, 69: 465-472. Chee, P. W., Draye, X. and Jiang C. X., 2005, Molecular dissection of phenotypic variation between Gossypium hirsutum and Gossypium barbadense (cotton) by a backcross-self approach: III. Fiber length. Theor. Appl. Genet., 111(4): 772-781. Cheema JJS, Ellis THN, Dicks J. 2008, Exploring genome wide genetic linkage manifolds. In: Proceedings of the Genome Informatics Conference, Hinxton, Abstract, p. 36. THREAD Mapper site (http://cbr.jic.ac.uk/dicks/ software/threadmapper).evaluating the origin of domesticated 
cotton 
(Gossypium 
hirsutum; 
Malvaceae) using nuclear restriction 
fragment 
length 
polymorphisms 
(

Microsatellite markers associated with lint percentage trait in cotton, Gossypium hirsutum. I Y Abdurakhmonov, Z T Buriev, S Saha, Euphytica. 1562Abdurakhmonov, I. Y., Buriev, Z. T., Saha, S., 2007, Microsatellite markers associated with lint percentage trait in cotton, Gossypium hirsutum. Euphytica, 156 (2): 141-156.

Advances in molecular marker techniques and their applications in plant sciences. M Agarwal, N Shrivastava, H Padh, Pl. Cell Reports. 274Agarwal, M., Shrivastava, N. and Padh, H., 2008, Advances in molecular marker techniques and their applications in plant sciences. Pl. Cell Reports, 27 (4): 617-631.

Molecular and morphological genetics of a trispecies F2 population of cotton. M K Altaf, J M C D Stewart, M K Wajahatullah, J Zhang, R G Cantrell, Proceedings of the Beltwide Cotton Conferences. the Beltwide Cotton ConferencesNew Orleans, La, USA1Altaf, M. K., Stewart, J. M. C. D., Wajahatullah, M. K., Zhang, J. and Cantrell, R. G., 1997, Molecular and morphological genetics of a trispecies F2 population of cotton, In Proceedings of the Beltwide Cotton Conferences,, New Orleans, La, USA, vol. 1: 448-452.

Genetic advance and heritability as a selection index for improvement of yield and quality in cotton. D T H An, R Ravikesavan, K Iyanar, J. Cotton Res. Develop. 221An, D. T. H., Ravikesavan, R., and Iyanar, K., 2008, Genetic advance and heritability as a selection index for improvement of yield and quality in cotton. J. Cotton Res. Develop., 22(1): 14-18.

Expressed sequence tags (ESTs) and single nucleotide polymorphisms. K O Ayeh, SNPsAyeh, K. O., 2008, Expressed sequence tags (ESTs) and single nucleotide polymorphisms (SNPs):

. African J. Biotech. 74African J. Biotech., 7(4): 331-341.

Genetic analysis of AFLP markers associated with seed quality traits in upland cotton (Gossypium hirsutum). A Badigannavar, G Myers, Beltwide Cotton Conferences. New Orleans, La, USABadigannavar, A. and Myers, G., 2010, Genetic analysis of AFLP markers associated with seed quality traits in upland cotton (Gossypium hirsutum). In Beltwide Cotton Conferences, New Orleans, La, USA.

QTL for rice grain quality based on a DH population derived from parents with similar apparent amylose content. J S Bao, Y R Wu, B Hu, P Wu, H R Cui, Q Y Shu, Euphytica. 1283Bao, J. S., Wu, Y. R., Hu, B., Wu, P., Cui, H. R. and Shu, Q. Y., 2002, QTL for rice grain quality based on a DH population derived from parents with similar apparent amylose content. Euphytica, 128(3): 317-324.

Zmap -a QTL cartographer. C J Basten, B S Weir, Z B Zeng, C Smith, J S Gavora, B Benkel, J Chesnais, W Fairfull, J P Gibson, B W Kennedy, E B Burnside, Congress on Genetics Applied to Livestock Production. Guelph, ON., Canada; Guelph5Proceedings. Computing strategies and softwareBasten, C. J., Weir, B. S. and Zeng, Z. B., 1994, Zmap -a QTL cartographer. In: Smith, C.; Gavora, J. S., Benkel, B., Chesnais, J., Fairfull, W., Gibson, J. P., Kennedy, B.W. and Burnside, E. B., eds. World Congress on Genetics Applied to Livestock Production, 5th, 7th -12th August, 1994, Guelph, ON., Canada. Proceedings. Computing strategies and software. Guelph, Organizing Committee, 1994, pp. 22: 65-66.

Molecular mapping of QTLs for non-parasitic leaf spot resistance and comparison of half-sib DH populations in spring barley. A Behn, L Hartl, G Schweizer, M Baumer, Euphytica. 1413Behn, A., Hartl, L., Schweizer, G. and Baumer, M., 2005, Molecular mapping of QTLs for non-parasitic leaf spot resistance and comparison of half-sib DH populations in spring barley. Euphytica, 141(3): 291-299.

Inter-simple sequence repeat (ISSR) amplification for analysis of microsatellite motif frequency and fingerprinting in rice. M W Blair, O Panaud, S R Mccouch, Oryza sativa. Blair, M. W., Panaud, O. and McCouch, S. R., 1999, Inter-simple sequence repeat (ISSR) amplification for analysis of microsatellite motif frequency and fingerprinting in rice (Oryza sativa L.).

. Theor. Appl. Genetics. 985Theor. Appl. Genetics, 98(5): 780-792.

Nonanchored inter simple sequence repeat (ISSR) markers: reproducible and. B Bornet, M Branchard, Bornet, B. and Branchard, M., 2001, Nonanchored inter simple sequence repeat (ISSR) markers: reproducible and

An introduction to markers, quantitative trait loci (QTL) mapping and marker assisted selection for crop improvement: The basic concepts. B C Y Collard, M Z Z Jahufer, J B Brouwer, E C K Pang, Euphytica. 142Collard, B. C. Y., Jahufer, M. Z. Z., Brouwer, J. B. and Pang, E. C. K., 2005, An introduction to markers, quantitative trait loci (QTL) mapping and marker assisted selection for crop improvement: The basic concepts. Euphytica, 142: 169-196.

Population genetic structure of the cleistogamous plant species Viola pubescensAiton (Violaceae), as indicated by allozyme and ISSR molecular markers. T M Culley, A D Wolfe, Heredity. 865Culley, T. M. and Wolfe, A. D., 2000, Population genetic structure of the cleistogamous plant species Viola pubescensAiton (Violaceae), as indicated by allozyme and ISSR molecular markers. Heredity, 86(5): 545-556.

advanced intercross lines, an experimental population for fine genetic mapping. A Darvasi, M Soller, 141GenetDarvasi, A. and M. Soller., 1995, advanced intercross lines, an experimental population for fine genetic mapping. Genet, 141: 1199-1207.

High density SNP-based genetic map development and linkage disequilibrium assessment in Brassica napus L. R Delourme, C Falentin, B F Fomeju, M Boillot, G Lassalle, BMC Genom. 14120Delourme, R., Falentin, C., Fomeju, B. F., Boillot, M. and Lassalle G., 2013, High density SNP-based genetic map development and linkage disequilibrium assessment in Brassica napus L. BMC Genom., 14: 120.

Sampling nucleotide diversity in cotton. A V Deynze, K Stoffel, M Lee, T A Wilkins, A Kozik, R G Cantrell, J Z Yu, R J Kohel, D M Stelly, BMC Pl. Biol. 9125Deynze, A. V., Stoffel, K., Lee, M., Wilkins, T. A., Kozik, A., Cantrell, R. G., Yu, J. Z., Kohel, R. J. and Stelly, D. M., 2009, Sampling nucleotide diversity in cotton. BMC Pl. Biol., 9: 125.

Permutation tests for multiple loci affecting a quantitative character. R W Doerge, G A Churchill, Genet. 142Doerge, R. W. and Churchill, G. A., 1996, Permutation tests for multiple loci affecting a quantitative character. Genet, 142: 285-294.

Inter simple sequence repeat (ISSR) markers as a tool for the assessment of both genetic diversity and gene pool origin in common bean. M Z Galv´an, B Bornet, P A Balatti, M Branchard, Phaseolus vulgaris L.Galv´an, M. Z., Bornet, B., Balatti, P. A. and Branchard, M., 2003, Inter simple sequence repeat (ISSR) markers as a tool for the assessment of both genetic diversity and gene pool origin in common bean (Phaseolus vulgaris L.).

. Euphytica. 1323Euphytica, 132(3): 297-301.

Investigations on inheritance of quantitative characters in animals by gene markers I. H Geldermann, Methods. Theor. Appl. Genet. 46Geldermann, H., 1975, Investigations on inheritance of quantitative characters in animals by gene markers I. Methods. Theor. Appl. Genet., 46: 319-330.

. C D Geng, Z Z Gong, J Q Huang, Z C Zhang, 153Identification of EuphyticaGeng, C. D., Gong, Z. Z., Huang, J. Q. and Zhang, Z. C., 1995, Identification of Euphytica, 153(1-2):181-197.

Development of a 63K SNP array for cotton and highdensity mapping of intra and inter. A M Hulse-Kemp, L Jana, P Joerg, A Hamid, B Ramesh, D F David, F James, G Marc, L L Steve Hague, K J Hinze, P K Kochan, J A S Riggs, A U Joshua, U Mauricio, S W Shirley, Z Qian-Hao, K B Sumit, B Archana, J B John, L B Robert, C Michel, A G Michael, B H David, S I Md, N Johnie, D C J Jenkins, L Jean-Marc, Danny , J L Richard, G P Alan, E Pepper, J A Poland, M R Krishan, V S Samir, S Kumar, S Andrew, M Jen, F W Taylor, M Y Scott, Z Xiuting, T Cindy, M W Lawley, A V W Ganal, M S David, specific populations of gossypium spp. G3: Genes|Genomes|Genetics. doi: 10. 1534/g3. 115. 018416Hulse-Kemp, A. M., Jana, L., Joerg, P., Hamid, A., Ramesh, B., David, D. F., James, F., Marc, G., Steve Hague, L. L., Hinze, K. J., Kochan, P. K., Riggs, J. A. S., Joshua, A. U., Mauricio, U., Shirley, S. W., Qian-Hao, Z., Sumit, K. B., Archana, B., John. J. B., Robert, L. B., Michel, C., Michael, A. G., David, B. H., Md, S. I., Johnie, N., Jenkins, D. C. J., Jean-Marc, L, Danny, J. L., Richard, G. P., Alan, E., Pepper, J. A., Poland, Krishan, M. R., Samir, V. S., Sunil Kumar, S., Andrew, S., Jen, M., Taylor, F. W., Scott, M. Y., Xiuting, Z., Cindy, T., Lawley, M. W., Ganal, A. V. W. and David, M. S., 2015b, Development of a 63K SNP array for cotton and high- density mapping of intra and inter- specific populations of gossypium spp. G3: Genes|Genomes|Genetics. doi: 10. 1534/g3. 115. 018416.

CAPS-microsatellites: use of CAPS method to convert non polymorphic microsatellites into useful markers. A G Ince, M Karaca, A N Onus, Mol. Breed. 253Ince, A. G., Karaca, M. and Onus, A. N. 2010, CAPS-microsatellites: use of CAPS method to convert non polymorphic microsatellites into useful markers. Mol. Breed., 25(3): 491-499.

AntMap: constructing genetic linkage maps using and ant colony optimization algorithm. H Iwata, S Ninomiya, Breed Sci. 56Iwata H, Ninomiya S. 2006, AntMap: constructing genetic linkage maps using and ant colony optimization algorithm. Breed Sci; 56:371-7.

multiple trait analysis of genetic mapping for quantitative trait loci. C J Jiang, Z B Zeng, Genet. 1403Jiang, C. J. and Zeng, Z. B., 1995, multiple trait analysis of genetic mapping for quantitative trait loci. Genet, 140(3): 1111-1127.

Multilocus interactions restrict gene introgression in interspecific populations of polyploidy Gossypium (cotton). C X Jiang, P W Chee, X Draye, P L Morrell, C W Smith, A H Paterson, Evolution. 543Jiang, C. X., Chee, P. W., Draye, X., Morrell, P. L., Smith, C. W. and Paterson, A. H., 2000, Multilocus interactions restrict gene introgression in interspecific populations of polyploidy Gossypium (cotton). Evolution, 54(3): 798-814.

Polyploid formation created unique avenues for response to selection in Gossypium (cotton). C X Jiang, R J Wright, K M El-Zik, A H Paterson, Proc. Nat. Acad. Sci. USA. 958Jiang, C. X., Wright, R. J., El-Zik, K. M. and Paterson, A. H., 1998, Polyploid formation created unique avenues for response to selection in Gossypium (cotton). Proc. Nat. Acad. Sci. USA., 95(8): 4419-4424.

Microsatellite markers: an overview of the recent progress in plants. R K Kalia, M K Rai, S Kalia, R Singh, A K Dhawan, Euphytica. 1773Kalia, R. K., Rai, M. K., Kalia, S., Singh, R. and Dhawan, A. K. 2011, Microsatellite markers: an overview of the recent progress in plants, Euphytica, 177(3): 309-334.

Association analysis of fibre traits in Gossypium arboreum accessions. S K Kantartzi, J Stewart, D Mc, Plant breed. 127Kantartzi, S. K., Stewart, J., Mc, D., 2008, Association analysis of fibre traits in Gossypium arboreum accessions. Plant breed, 127: 173-179.

multiple interval mapping for quantitative trait loci. C H Kao, Z B Zeng, R D Teasdale, 152GenetKao, C. H., Zeng, Z. B. and Teasdale, R. D., 1999, multiple interval mapping for quantitative trait loci. Genet, 152: 1203- 1216.

Whole Genome Sequencing and Analysis of 'A' Genome of a Indian Diploid Cottons Gossypium arboreum and Gossypium herbaceum. I S Katageri, V B Lachagari, A Malarvizhi, M Milner Kumar, R Gupta, S Lekkala, B Kuriakose, A R Reddy, G Thomas, S Santhosh, H M Vamadevaiah, B M Khadi, D P Biradar, Nimhans, India Bangalore, S A Khanam, J L Sham, Bennetzen, M A M Aly, NextGen genomics and Bioinformatics Technologies (NGBT) Conference. Analysis of molecular marker-based characterization and genetic variation in date palm. Phoenix Dactylifera L.Katageri, I. S.,Lachagari, V. B., Malarvizhi, A., Milner Kumar, M., Gupta, R.,Lekkala, S., Kuriakose, B., Reddy, A. R., Thomas, G.,Santhosh, S.,Vamadevaiah, H. M.,Khadi, B. M. and Biradar, D. P., 2014, Whole Genome Sequencing and Analysis of 'A' Genome of a Indian Diploid Cottons Gossypium arboreum and Gossypium herbaceum, NextGen genomics and Bioinformatics Technologies (NGBT) Conference, November 17th-18th, NIMHANS, Bangalore, INDIA. Khanam, S. A., Sham, J. L., Bennetzen, and Aly, M. A. M., 2012, Analysis of molecular marker-based characterization and genetic variation in date palm (Phoenix Dactylifera L.).

. Aust. J. Crop Sci. 68Aust. J. Crop Sci., 6(8): 1236-1244.

Molecular mapping and characterization of traits controlling fiber quality in cotton. R J Kohel, J Yu, Y H Park, G R Lazo, Euphytica. 1212Kohel, R. J., Yu, J., Park Y. H. and Lazo, G. R., 2001, Molecular mapping and characterization of traits controlling fiber quality in cotton. Euphytica, 121(2): 163-172.

MadMapper and CheckMatrix-python scripts to infer orders of genetic markers and for visualization and validation of genetic maps and haplotypes. A Kozik, R Michelmore, Proceedings of the Plant and Animal Genome XIV Conference. the Plant and Animal Genome XIV ConferenceSan DiegoAbstract P957/CP013. date last accessedKozik A, Michelmore R. 2006, MadMapper and CheckMatrix-python scripts to infer orders of genetic markers and for visualization and validation of genetic maps and haplotypes. In: Proceedings of the Plant and Animal Genome XIV Conference, San Diego, Abstract P957/CP013- http://www.intl- pag.org/14/abstracts/PAG14_C013.html (13 July 2009, date last accessed).

Gossypium hirsutum x G. barbadense, RIL population: towards a unified consensus linkage map of tetraploid cotton. J M Lacape, J Jacobs, T Arioli, R Derijcker, N C Forestier, D Llewellyn, J Jean, E Thomas, C Viot, Theor. Appl Genet. 119Lacape, J. M., Jacobs, J., Arioli, T., Derijcker, R., Forestier, N. C., Llewellyn, D., Jean, J., Thomas, E. and Viot, C., 2009, a new interspecific, Gossypium hirsutum x G. barbadense, RIL population: towards a unified consensus linkage map of tetraploid cotton. Theor. Appl Genet., 119: 281-292.

Meta-analysis of cotton fiber quality QTLs across diverse environments in a Gossypium hirsutum × G. barbadense RIL population. J M Lacape, D Llewellyn, J Jacobs, BMC Pl. Biol. 132Lacape, J. M., Llewellyn, D. and Jacobs, J., 2010, Meta-analysis of cotton fiber quality QTLs across diverse environments in a Gossypium hirsutum × G. barbadense RIL population, BMC Pl. Biol., p. 132.

A combined RFLP-SSR-AFLP map of tetraploid cotton based on a Gossypium hirsutum × Gossypium barbadense backcross population. J M Lacape, T B Nguyen, S Thibivilliers, Genom. 464Lacape, J. M., Nguyen, T. B. and Thibivilliers, S., 2003, A combined RFLP-SSR-AFLP map of tetraploid cotton based on a Gossypium hirsutum × Gossypium barbadense backcross population. Genom., 46(4): 612-626.

J M Lacape, D Dessauw, M Rajab, J L Noyer, B Hau, Microsatellite diversity in tetraploidGossypium germplasm: assembling a highly informative genotyping set of cotton SSRs. 19Lacape, J.M., Dessauw, D., Rajab, M., Noyer, J. L. and Hau, B., 2007, -Microsatellite diversity in tetraploidGossypium germplasm: assembling a highly informative genotyping set of cotton SSRs,‖ Molecul. Breed. 19(1):45-58.

Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection. H M Lam, X Xu, X Liu, W Chen, G Yang, F L Wong, M W Li, W He, N Qin, B Wang, J Li, M Jian, J Wang, G Shao, J Wang, S S Sun, G Zhang, Nat. Genet. 42Lam, H. M., Xu, X., Liu, X., Chen, W., Yang, G., Wong, F. L., Li, M. W., He, W., Qin, N., Wang, B., Li, J., Jian, M., Wang, J., Shao, G., Wang, J., Sun, S. S. and Zhang, G., 2010, Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection, Nat. Genet., 42: 1053-1059.

Identification of a RAPD marker linked to male fertility restoration gene in cotton (Gossypium hirsutum L.). T H Lan, C G Cook, A H Paterson, J. Agri. Genom. 1Lan, T. H., Cook, C. G. and Paterson, A. H., 1999, Identification of a RAPD marker linked to male fertility restoration gene in cotton (Gossypium hirsutum L.). J. Agri. Genom., 1: 1-5.

Construction of multilocus genetic maps in humans. E Lander, P Green, ProcNatlAcad Sci USA. 84Lander E, Green P. 1987, Construction of multilocus genetic maps in humans. ProcNatlAcad Sci USA; 84: 2363-7.

mapping mendelian factors underlaying quantitative traits using RFLP linkage maps. E S Lander, D Botstein, 121GenetLander, E. S. and Botstein, D., 1989, mapping mendelian factors underlaying quantitative traits using RFLP linkage maps. Genet, 121: 185-199.

Genetic mapping of quantitative trait loci associated with bioenergy traits, and the assessment of genetic variability in sweet sorghum (Sorghum bicolor L. Moench). A Lekgari, LincolnUniv. NebraskaPh D ThesisLekgari, A., 2010, Genetic mapping of quantitative trait loci associated with bioenergy traits, and the assessment of genetic variability in sweet sorghum (Sorghum bicolor L. Moench). Ph D Thesis, Univ. Nebraska, Lincoln.

. F G Li, G Y Fan, K B Wang, F M Sun, Y L Yuan, G L Song, Q Li, Z Y Ma, C R Lu, C S Zou, W B Chen, X M Liang, H H Shang, W Q Liu, C C Shi, G H Xiao, C Y Gou, W W Ye, X Xu, X Y Zhang, H L Wei, Z F Li, G Y Zhang, J Y Wang, K Liu, R J Kohel, R G Percy, J Z Yu, Y X Zhu, J Wang, S X Yu, Nat. Genet. 466Genome sequence of the cultivated cotton Gossypium arboreumLi, F. G., Fan, G. Y., Wang, K. B., Sun, F. M., Yuan, Y. L., Song, G. L., Li, Q., Ma, Z. Y., Lu, C. R., Zou, C. S., Chen, W. B., Liang, X. M., Shang, H. H., Liu, W. Q., Shi, C. C., Xiao, G. H., Gou, C. Y., Ye, W. W., Xu, X., Zhang, X. Y., Wei, H. L., Li, Z. F., Zhang, G. Y., Wang, J. Y., Liu, K., Kohel, R. J., Percy, R. G., Yu, J. Z., Zhu, Y. X., Wang, J. and Yu. S. X., 2014, Genome sequence of the cultivated cotton Gossypium arboreum. Nat. Genet., 46(6): 567-572.

Mapping genes controlling quantitative traits with MAPMAKER/QTL. Version 1. 1: A tutorial and reference manual. S Lincoln, M Daly, E Lander, 46Whitehead Institute TechnicalReport2nd edition. Cambridge, M A.Lincoln, S., Daly, M. and Lander, E., 1992, Mapping genes controlling quantitative traits with MAPMAKER/QTL. Version 1. 1: A tutorial and reference manual. 2nd edition. Cambridge, M A., Whitehead Institute Technical Report, p. 46.

Statistical Genomics: Linkage, mapping and QTL analysis. B Liu, CRC pressBoca RatonLiu B (1998). Statistical Genomics: Linkage, mapping and QTL analysis. CRC press, Boca Raton.

Intersimple sequence repeat (ISSR) polymorphisms as a genetic marker system in cotton. B Liu, J F Wendel, Mol. Ecol. Notes. 1Liu, B. and Wendel, J. F., 2001, Intersimple sequence repeat (ISSR) polymorphisms as a genetic marker system in cotton. Mol. Ecol. Notes, 1: 205-208.

Quantitative trait loci mapping for yield and its components by using two immortalized populations of a heterotic hybrid in Gossypium hirsutum L. R Liu, B Wang, W Guo, Molecular Breeding. 292Liu, R., Wang, B. and Guo, W., 2012, Quantitative trait loci mapping for yield and its components by using two immortalized populations of a heterotic hybrid in Gossypium hirsutum L. Molecular Breeding, 29(2): 297-311.

Maximum likelihood estimation linkage between a marker gene and a quantitative trait locus. Z W Luo, M J Kearsey, Heredity. 63Luo, Z. W. and Kearsey, M. J., 1989, Maximum likelihood estimation linkage between a marker gene and a quantitative trait locus.Heredity, 63: 401-408.

RFLP linkage analysis and mapping genes controlling the fatty acid profile of Brassica juncea using reciprocal DH populations. T Mahmood, U Ekuere, F Yeh, A G Good, G R Stringam, Theor. Appl. Genet. 1072Mahmood, T., Ekuere, U., Yeh, F., Good, A. G. and Stringam, G. R., 2003, RFLP linkage analysis and mapping genes controlling the fatty acid profile of Brassica juncea using reciprocal DH populations. Theor. Appl. Genet., 107(2): 283-290.

Overview of QTL mapping software and introduction to Map Manager QTL. K F Manly, J M Olson, Mammalian Genome. 10Manly, K. F. and Olson, J. M., 1999, Overview of QTL mapping software and introduction to Map Manager QTL. Mammalian Genome, 10: 327-334.

Amplified fragment length polymorphism in soyabean: species diversity, inheritance and near -isogenic line analysis. P J Maughan, M A Seghai, G R Maroof, Buss, G M Huestis, Theor. Appl. Genet. 93Maughan, P. J., Seghai, M. A., Maroof, G. R., Buss. And Huestis, G. M., 1996, Amplified fragment length polymorphism in soyabean: species diversity, inheritance and near -isogenic line analysis. Theor. Appl. Genet., 93: 392-401.

Genetic mapping and QTL analysis of fiber-related traits in cotton (Gossypium). M Mei, N H Syed, W Gao, ‖ Theoretical and Applied Genetics. 108Mei, M., Syed, N. H. and Gao W., 2004, -Genetic mapping and QTL analysis of fiber-related traits in cotton (Gossypium),‖ Theoretical and Applied Genetics, 108 2:280-291.

Production of high gossypol cotton plants with low gossypol seed from trispecific hybrids including Gossypium sturtianum. G Mergeai, J P Baudoin, B I Vroh, proceedings of the World Cotton Research Conference. the World Cotton Research Conference2Mergeai, G., Baudoin, J. P. and Vroh, B. I., 1998, Production of high gossypol cotton plants with low gossypol seed from trispecific hybrids including Gossypium sturtianum. In proceedings of the World Cotton Research Conference, vol. 2: 206-210.

Genetic variability, coefficient of variance, heritability and genetic advance of some Gossypium hirsutum L. accessions. Z A Muhammad, S M Muhammad, B Hidayatullah, W S Abdul, P Faizhussain, R C Abdul, B S Karim, J. Agril. Sci. 72Muhammad, Z. A., Muhammad, S. M., Hidayatullah, B., Abdul, W. S., FaizHussain, P., Abdul, R. C. and Karim, B. S., 2015., Genetic variability, coefficient of variance, heritability and genetic advance of some Gossypium hirsutum L. accessions. J. Agril. Sci., 7 (2).

Genetic fingerprinting of Australian cotton cultivars with RAPD markers. D S Multani, B R Lyon, Genom. 385Multani, D. S. and Lyon, B. R., 1995, Genetic fingerprinting of Australian cotton cultivars with RAPD markers. Genom, 38(5): 1005-1008.

Cotton genetic diversity study by AFLP markers. N Murtaza, Electron. J. Biot. 94Murtaza, N., 2006, Cotton genetic diversity study by AFLP markers. Electron. J. Biot., 9(4): 456-460.

Applicability of inter-simple sequence repeat polymorphisms in wheat for use as DNA markers in comparison to RFLP and RAPD markers. T Nagaoka, Y Ogihara, Theor. Appl. Genet. 945Nagaoka, T. and Ogihara, Y., 1997, Applicability of inter-simple sequence repeat polymorphisms in wheat for use as DNA markers in comparison to RFLP and RAPD markers. Theor. Appl. Genet., 94(5): 597-602.

QGENE: software for marker-based genomic analysis and breeding. J C Nelson, Mol. Breed. 3Nelson, J. C., 1997, QGENE: software for marker-based genomic analysis and breeding. Mol. Breed., 3: 239-245.

Construction of genetic maps using distance geometry. W R Newell, R Mott, S Beck, Genomics. 30Newell WR, Mott R, Beck S, 1995, Construction of genetic maps using distance geometry. Genomics; 30:59- 70.

Genetic mapping of new cotton fiber loci using EST-derived microsatellites in an interspecific recombinant inbred line. Y H Park, M S Alabady, B Sickler, RILPark, Y. H., Alabady, M. S. and Sickler, B., 2005, Genetic mapping of new cotton fiber loci using EST-derived microsatellites in an interspecific recombinant inbred line (RIL)

mapping genes responsible for differences in phenotype. A H Paterson, R G Landes, C San Diego, Acad. PressAustin, Tex, USAPaterson A. H., Landes, R. G. and San Diego, C., 1996, mapping genes responsible for differences in phenotype. Acad. Press, Austin, Tex, USA, 41-54.

. A H Paterson, J F Wendel, H Gundlach, H Guo, J Jenkins, D Jin, D Llewellyn, K C Showmaker, S Shu, J Udall, M J Yoo, R Byers, W Chen, A Doron-Faigenboim, M V Duke, L Gong, J Grimwood, C Grover, K Grupp, G Hu, T H Lee, J Li, L Lin, T Liu, B S Marler, J T Page, A W Roberts, E Romanel, W S Sanders, E Szadkowski, X Tan, H Tang, C Xu, J Wang, Z Wang, D Zhang, L Zhang, H Ashrafi, F Bedon, J E Bowers, C L Brubaker, P W Chee, S Das, A R Gingle, C H Haigler, D Harker, L V Hoffmann, R Hovav, D C Jones, C Lemke, S Mansoor, M Rahman, L N Rainville, A Rambani, U K Reddy, J K Rong, Y Saranga, B E Scheffler, J A Scheffler, D M Stelly, B A Triplett, A Van Deynze, M F Vaslin, V N Waghmare, S A Walford, R J Wright, E A Zaki, T Zhang, E S Dennis, K F Mayer, D G Peterson, D S Rokhsar, X Wang, J Schmutz, Nature. 4927429423Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibresPaterson, A. H., Wendel, J. F., Gundlach, H., Guo, H., Jenkins, J., Jin, D., Llewellyn, D., Showmaker, K. C., Shu, S., Udall, J., Yoo, M. J., Byers, R., Chen, W., Doron-Faigenboim, A., Duke, M. V., Gong, L., Grimwood, J., Grover, C., Grupp, K., Hu, G., Lee, T. H., Li, J., Lin, L., Liu, T., Marler, B. S., Page, J. T., Roberts, A. W., Romanel, E., Sanders, W. S., Szadkowski, E., Tan, X., Tang, H., Xu, C., Wang, J., Wang, Z., Zhang, D., Zhang, L., Ashrafi, H., Bedon, F., Bowers, J. E., Brubaker, C. L., Chee, P. W., Das, S., Gingle, A. R., Haigler, C. H., Harker, D., Hoffmann, L. V., Hovav, R., Jones, D. C., Lemke, C., Mansoor, S., Rahman, M., Rainville, L. N., Rambani, A., Reddy, U. K., Rong, J. K., Saranga, Y., Scheffler, B. E., Scheffler, J. A., Stelly, D. M., Triplett, B. A., Van Deynze, A., Vaslin, M. F., Waghmare, V. N., Walford, S. A., Wright, R. J., Zaki, E. A., Zhang, T., Dennis, E. S., Mayer, K. F., Peterson, D. G., Rokhsar, D. S., Wang, X. and Schmutz, J., 2012, "Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres. " Nature, 492 (7429): 423-

Molecular marker technology in cotton. S Preetha, T S Raveendren, Biotech. Mol. Biol. Rev. 32Preetha, S. and Raveendren, T. S. 2008, Molecular marker technology in cotton. Biotech. Mol. Biol. Rev., 3(2): 32-45.

H Qin, W Guo, Y M Zhang, T Zhang, QTL mapping of yield and fiber traits based on a four-way cross population in Gossypium hirsutum L. 117Qin, H., Guo, W., Zhang, Y. M. and Zhang, T., 2008, -QTL mapping of yield and fiber traits based on a four-way cross population in Gossypium hirsutum L,‖ Theoretical and Applied Genetics, 117(6):883-894.

randomly amplified polymorphic DNA (RAPD) analysis. J A Rafalski, DNA markers: Protocols, Applications and Overviews. G. C. Anolles and P. M. GresshoffWiley-Liss, Inc. USA364Rafalski, J. A., 1997, randomly amplified polymorphic DNA (RAPD) analysis. In: DNA markers: Protocols, Applications and Overviews (eds. G. C. Anolles and P. M. Gresshoff), Wiley-Liss, Inc. USA, pp. 364.

Estimation of genetic divergence among elite cotton cultivars-genotypes by DNA finger printing technology. M Rahman, D Hussain, Y Zafar, Crop Sci. 42Rahman, M., Hussain, D. and Zafar, Y., 2002, Estimation of genetic divergence among elite cotton cultivars-genotypes by DNA finger printing technology. Crop Sci., 42: 2137-2144.

AFLP and RAPD markers for genetic diversity and cultivar identification in cotton. M K Rana, K V Bhat, J. Pl. Biochem. Biotech. 131Rana, M. K. and Bhat, K. V., 2004, a comparison of AFLP and RAPD markers for genetic diversity and cultivar identification in cotton. J. Pl. Biochem. Biotech, 13(1): 19-24.

Inter simple sequence repeat (ISSR) polymorphism and its application in plant breeding. M P Reddy, N Sarla, E A Siddiq, Reddy, M. P., Sarla, N. and Siddiq, E. A., 2002, Inter simple sequence repeat (ISSR) polymorphism and its application in plant breeding.

. Euphytica. 1281Euphytica, 128(1): 9-17.

a detailed RFLP map of cotton, Gossypium hirsutum × Gossypium barbadense: chromosome organization and evolution in a disomic polyploid genome. J Reinisch, J M Dong, C L Brubaker, D M Stelly, J F Wendel, A H Paterson, Genet. 1383Reinisch, J., Dong, J. M., Brubaker, C. L., Stelly, D. M., Wendel, J. F. and Paterson, A. H., 1994, a detailed RFLP map of cotton, Gossypium hirsutum × Gossypium barbadense: chromosome organization and evolution in a disomic polyploid genome. Genet, 138(3): 829- 884.

How neutral are synonymous codon mutations. I Richard, J S Beckmann, Nat. genet. 103259Richard, I. and Beckmann, J. S., 1995, How neutral are synonymous codon mutations. Nat. genet, 10(3): 259.

Meta-analysis of polyploid cotton QTL shows unequal contributions of subgenomes to a complex network of genes and gene clusters implicated in lint fiber development. J Rong, F A Feltus, V N Waghmare, G J Pierce, P W Chee, X Draye, Y Saranga, R J Wright, T A Wilkins, O L May, C W Smith, J R Gannaway, J F Wendel, A H Paterson, 176GenetRong, J. Feltus, F. A., Waghmare, V. N., Pierce, G. J., Chee, P. W., Draye, X., Saranga, Y., Wright, R. J., Wilkins, T. A., May, O. L., Smith, C. W., Gannaway, J. R., Wendel, J. F. and Paterson, A. H., 2007, Meta-analysis of polyploid cotton QTL shows unequal contributions of subgenomes to a complex network of genes and gene clusters implicated in lint fiber development. Genet, 176(4): 2577- 2588.

A 3347-locus genetic recombination map of sequence-tagged sites reveals features of genome organization. J Rong, C Abbey, J E Bowers, C L Brubaker, C Chang, P W Chee, T A Delmonte, X Ding, J J Garza, B S Marler, C Park, G J Pierce, K M Rainey, V K Rastogi, S R Schulze, N L Trolinder, J F Wendel, T A Wilkins, T D Williams-Coplin, R A Wing, R J Wright, X Zhao, L Zhu, A H Paterson, 166transmission and evolution of cotton (Gossypium). GeneticsRong, J., Abbey, C., Bowers, J. E., Brubaker, C. L., Chang, C., Chee, P. W., Delmonte, T. A., Ding, X., Garza, J. J., Marler, B. S., Park, C., Pierce, G. J., Rainey, K. M., Rastogi, V. K., Schulze, S. R., Trolinder, N. L., Wendel, J. F., Wilkins, T. A., Williams-Coplin, T. D., Wing, R. A., Wright, R. J., Zhao, X., Zhu, L. and Paterson, A. H., 2004, A 3347-locus genetic recombination map of sequence-tagged sites reveals features of genome organization, transmission and evolution of cotton (Gossypium). Genetics, 166: 389-417.

Linkage between loci of quantitative traits and marker loci: multi-trait analysis with a single marker. Y I Ronin, V M Kirzhner, A B Korol, Theor. Appl. Genet. 906Ronin, Y. I., Kirzhner, V. M. and Korol, A. B., 1995, Linkage between loci of quantitative traits and marker loci: multi-trait analysis with a single marker. Theor. Appl. Genet., 90(6): 776-786.

breeding and genetics: effect of chromosome substitutions from Gossypium barbadense L. 3-79 into G. hirsutumL. TM-1 on agronomic and fiber traits. S Saha, J Wu, J N Jenkins, J. Cotton Sci. 83162Saha, S., Wu, J. and Jenkins, J. N., 2004, breeding and genetics: effect of chromosome substitutions from Gossypium barbadense L. 3-79 into G. hirsutumL. TM-1 on agronomic and fiber traits. J. Cotton Sci., 8(3): 162-

Genomic dissection of genotype x environment interactions conferring adaptation of cotton to arid conditions. K M Santosh, India Coimbatore, Y Saranga, M Menz, C X Jiang, R J Wright, D Yakir, A H Paterson, Genom. Res. 1112Tamil Nadu Agricultural UniversityM. S. thesisGenetic and quantitative trait loci analysis for yield and fibre quality traits in cotton (Gossypium sppSantosh, K. M., 2010, Genetic and quantitative trait loci analysis for yield and fibre quality traits in cotton (Gossypium spp.), M. S. thesis, Tamil Nadu Agricultural University, Coimbatore, India. Saranga. Y., Menz, M., Jiang, C. X., Wright, R. J., Yakir, D. and Paterson, A. H., 2001, Genomic dissection of genotype x environment interactions conferring adaptation of cotton to arid conditions. Genom. Res., 11(12):

a bayesian approach to detect quantitative trait loci using Markov chain Monte Carlo. J M Satagopan, B S Yandell, M A Newton, T G Osborn, 144GenetSatagopan, J. M., Yandell, B. S., Newton, M. A. and Osborn, T. G., 1996, a bayesian approach to detect quantitative trait loci using Markov chain Monte Carlo. Genet, 144: 805-816.

1923, the association of size differences with seed coat pattern and pigmentation in Phaseolus vulgaris. K Sax, Genet. 8Sax, K., 1923, the association of size differences with seed coat pattern and pigmentation in Phaseolus vulgaris. Genet, 8: 552-560.

CarthaGene: constructing and joining maximum likelihood genetic maps. T Schiex, C Gaspin, Proceedings of the International Conference on Intelligent Systems for Molecular Biology. Abstract 5, AAAI Press (www.aaai.org). the International Conference on Intelligent Systems for Molecular Biology. Abstract 5, AAAI Press (www.aaai.org)Schiex T, Gaspin C. 1997, CarthaGene: constructing and joining maximum likelihood genetic maps. In: Proceedings of the International Conference on Intelligent Systems for Molecular Biology. Abstract 5, AAAI Press (www.aaai.org), pp. 258-67.

Principles requirements and prospects of genetic mapping in plants. K Semagn, Å Bjørnstad, M N Ndjiondjop, African J. Biotechnol. 525Semagn, K., Bjørnstad, Å. and Ndjiondjop, M. N., 2006, Principles requirements and prospects of genetic mapping in plants. African J. Biotechnol., 5(25): 2569-2587.

2010, the genetic dissection of quantitative traits in crops. K Semagn, A Bjørnstad, Y Xu, Electronic J. Biotechnol. Semagn, K., Bjørnstad, A. and Xu, Y., 2010, the genetic dissection of quantitative traits in crops. Electronic J. Biotechnol., ISSN: 0717-3458.

Quantitative trait loci controlling Fusarium high blight resistance and low deoxynivalenol content in hexaploid wheat population derived from -Arina‖ and NK93604. K Semagn, H Skinnes, Å Bjørnstad, A G Marøy, Y Tarkegne, Crop Sci. 47Semagn, K., Skinnes, H., Bjørnstad, Å., Marøy, A. G. and Tarkegne., Y., 2007, Quantitative trait loci controlling Fusarium high blight resistance and low deoxynivalenol content in hexaploid wheat population derived from -Arina‖ and NK93604. Crop Sci., 47: 294-303.

Single nucleotide polymorphism analysis of MT-SHSP gene of Gossypium arboreum and its relationship with other diploid cotton genomes, G. hirsutum and Arabidopsisthaliana. T Shaheen, M Asif, Y Zafar, Pakistan J. Botany. 411Shaheen, T., Asif, M. and Zafar, Y., 2009, Single nucleotide polymorphism analysis of MT-SHSP gene of Gossypium arboreum and its relationship with other diploid cotton genomes, G. hirsutum and Arabidopsisthaliana. Pakistan J. Botany, 41(1): 177-183.

Ancient orphan crop joins modern era: gene-based SNP discovery and mapping in lentil. A G Sharpe, L Ramsay, L A Sanderson, M J Fedoruk, W E Clarke, R Li, S Kagale, P Vijayan, A Vandenberg, K E Bett, BMC Genom. 1814192Sharpe, A. G., Ramsay, L., Sanderson, L. A., Fedoruk, M. J., Clarke, W. E., Li, R., Kagale, S., Vijayan, P., Vandenberg, A., and Bett, K. E., 2013, Ancient orphan crop joins modern era: gene-based SNP discovery and mapping in lentil. BMC Genom, 18(14): 192.

Genetic mapping of quantitative trait loci for fibre quality and yield trait by RIL approach in upland cotton. X L Shen, W Guo, Q Lu, X Zhu, Y Yuan, T Zhang, Euphytica. 1553Shen, X. L., Guo, W., Lu, Q., Zhu, X., Yuan, Y. and Zhang, T., 2007, Genetic mapping of quantitative trait loci for fibre quality and yield trait by RIL approach in upland cotton. Euphytica, 155 (3): 371-380.

Molecular mapping of QTLs for fiber qualities in three diverse lines in Upland cotton using SSR markers. X Shen, W Guo, X Zhu, Molecular Breeding. 152Shen, X., Guo, W. and Zhu, X., 2005, Molecular mapping of QTLs for fiber qualities in three diverse lines in Upland cotton using SSR markers, Molecular Breeding, 15(2):169-181.

Development of ISSR PCR markers for diversity study in dogwood (Cornus spp. A Shi, S Kantartzi, M Mmbaga, P Chen, Agril. And Biol. J. North America. 1Shi, A., Kantartzi, S., Mmbaga, M. and Chen, P., 2010, Development of ISSR PCR markers for diversity study in dogwood (Cornus spp.). Agril. And Biol. J. North America, 1: 189-194.

. B Shu, K Fenling, Z Y Yao, Z G Mei, Z Q Yuan, W X Gang, Shu, B., K., Fenling, Z. Y., Yao, Z. G., Mei, Z. Q., Yuan, and Gang, W. X., 2001,

. Scientiaagriculturasinica, 34ScientiaAgriculturaSinica, 34: 597-603.

ISSR markers show differentiation among Italian populations of Asparagusacutifolius L. M Sica, G Gamba, S Montieri, L Gaudio, S Aceto, BMC Genet. 617Sica, M., Gamba, G., Montieri, S., Gaudio, L. and Aceto, S., 2005, ISSR markers show differentiation among Italian populations of Asparagusacutifolius L. BMC Genet., 6, article 17.

Development of a large SNP genotyping array and generation of high-density genetic maps in tomato. S C Sim, G Durstewitz, J Plieske, R Wieseke, M W Ganal, doi: 10. 1371/journal. Pone. 0040563PLoS One. 7740563Sim, S. C., Durstewitz, G., Plieske, J., Wieseke, R. and Ganal, M. W., 2012, Development of a large SNP genotyping array and generation of high-density genetic maps in tomato. PLoS One, 7(7): e40563. doi: 10. 1371/journal. Pone. 0040563.

Marker based mapping of quantitative trait loci using replicated progenies. M Soller, J S Beckmann, Theor. Appl. Genet. 802Soller, M. and Beckmann, J. S., 1990, Marker based mapping of quantitative trait loci using replicated progenies. Theor. Appl. Genet., 80(2): 205-208.

Construction of integrated genetic linkage maps by means of a new computer package. P Stam, JoinMap. Plant J. 3Stam P. 1993; Construction of integrated genetic linkage maps by means of a new computer package: JoinMap. Plant J., 3:739-44.

Genome-wide DNA polymorphisms in elite indica rice inbreds discovered by whole-genome sequencing. G K Subbaiyan, D L E Waters, S K Katiyar, A R Sadananda, S Vaddadi, R J Henry, doi: 10. 1111/j. 1467-7652. 2011. 00676. XPl. Biotech. J. Subbaiyan, G. K., Waters, D. L. E., Katiyar, S. K., Sadananda, A. R., Vaddadi, S. and Henry, R. J., 2012, Genome-wide DNA polymorphisms in elite indica rice inbreds discovered by whole-genome sequencing. Pl. Biotech. J., 1-12, doi: 10. 1111/j. 1467-7652. 2011. 00676. X.

Linkage group alignment of sorghum RFLP maps using RIL mapping population. P K Subudhi, H T Nguyen, Genom. 43Subudhi, P. K. and Nguyen, H. T., 2000, Linkage group alignment of sorghum RFLP maps using RIL mapping population. Genom, 43: 240-249.

Cryptic simplicity in DNA is a major source of genetic variation. D Tautz, M Trick, G A Dover, Nat. 3226080Tautz, D., Trick, M. and Dover, G. A. 1986, Cryptic simplicity in DNA is a major source of genetic variation. Nat., 322(6080): 652-656.

. J A Udall, J M Swanson, K Haller, ESTs. Genom. Res. 163Udall, J. A., Swanson, J. M. and Haller, K., 2006, a global assembly of cotton ESTs. Genom. Res., 16(3): 441-450.

Genetic linkage map and QTL analysis of agronomic and fibre traits in an intraspecific population. M Ulloa, W R Meredith, J. Cotton Sci. 43Ulloa, M. and Meredith, W. R., 2000, Genetic linkage map and QTL analysis of agronomic and fibre traits in an intraspecific population. J. Cotton Sci., 4 (3): 161-170.

RFLP genetic linkage maps from four F2. 3 populations and a join map of Gossypium hirsutum L. M Ulloa, W R Meredith, Z W Shappley, A L Kahler, Theor. Appl. Genet. 104Ulloa, M., Meredith, W. R., Shappley, Z. W. and Kahler, A. L., 2002, RFLP genetic linkage maps from four F2. 3 populations and a join map of Gossypium hirsutum L. Theor. Appl. Genet., (104): 200-208.

. H F Utz, A E Melchinger, PLABQTL, a computer program to map QTL, Version 1.2. 2006-06-01Utz, H. F. and Melchinger, A. E., 2003, PLABQTL, a computer program to map QTL, Version 1.2. 2006-06-01.

. Stuttgart, 33Institute of Plant Breeding, Seed Science, and Population Genetics, University of HohenheimStuttgart, Institute of Plant Breeding, Seed Science, and Population Genetics, University of Hohenheim, p. 33.

RECORD: a novel method for ordering loci on a genetic linkage map. O H Van, P Stam, R G F Visser, TheoretAppl Genetics. 112Van, O. H., Stam, P., Visser, R.G.F., 2005, RECORD: a novel method for ordering loci on a genetic linkage map. TheoretAppl Genetics, 112:30-40.

AFLP: a new technique for DNA fingerprinting. P Vos, R Hogers, M Bleeker, Nucleic Acids Res. 2321Vos, P., Hogers, R. and Bleeker, M., 1995, AFLP: a new technique for DNA fingerprinting. Nucleic Acids Res., 23(21): 4407-4414.

Genomic affinity among Gossypium subgenus Sturtia species by RAPD analysis. M K Wajahatullah, J M Stewart, Proceeding of the Belt wide Cotton Conference. eeding of the Belt wide Cotton ConferenceNational Cotton Council; Memphis, Tenn, USA452Wajahatullah, M. K. and Stewart, J. M., 1997, Genomic affinity among Gossypium subgenus Sturtia species by RAPD analysis. In Proceeding of the Belt wide Cotton Conference, National Cotton Council, p. 452, Memphis, Tenn, USA.

QTL analysis and epistasis effects dissection of fiber qualities in an elite cotton hybrid grown in second generation. B Wang, Y Wu, W Guo, X Zhu, N Huang, T Zhang, ‖ Crop Science. 474Wang, B., Wu, Y., Guo, W., Zhu, X., Huang, N. and Zhang, T., 2007, -QTL analysis and epistasis effects dissection of fiber qualities in an elite cotton hybrid grown in second generation,‖ Crop Science, 47(4):1384-1392.

Characterization, development and exploitation of EST derived microsatellites in Gossypium raimondiiUlbrich. C B Wang, W Z Guo, C P Cai, T Z Zhang, Chin. Sci. Bull. 51Wang, C. B., Guo, W. Z., Cai, C. P. and Zhang, T. Z., 2006, Characterization, development and exploitation of EST derived microsatellites in Gossypium raimondiiUlbrich. Chin. Sci. Bull., 51: 557-561.

ISSR markers and their applications in plant genetics. J B Wang, C Yi, 24Wang, J. B. and Yi, C., 2002, ISSR markers and their applications in plant genetics. 24(5): 613-616.

A global analysis of QTLs for expression variations in rice shoots at the early seedling stage. J Wang, H Yu, W Xie, Y Xing, S Yu, C Xu, X Li, J Xiao, Q Zhang, Plant J. 63Wang, J., Yu, H., Xie, W., Xing, Y., Yu, S., Xu, C., Li, X., Xiao, J., and Zhang, Q. (2010). A global analysis of QTLs for expression variations in rice shoots at the early seedling stage. Plant J. 63, 1063-1074.

The draft genome of a diploid cotton Gossypiumraimondii. K Wang, Z Wang, F Li, W Ye, J Wang, G Song, Z Yue, L Cong, H Shang, S Zhu, C Zou, Q Li, Y Yuan, C Lu, H Wei, C Gou, Z Zheng, Y Yin, X Zhang, K Liu, B Wang, C Song, N Shi, R J Kohel, R G Percy, J Z Yu, Y X Zhu, S Yu, Nat. Genet. 4410Wang, K., Wang, Z., Li, F., Ye, W., Wang, J., Song, G., Yue, Z., Cong, L., Shang, H., Zhu, S., Zou, C., Li, Q., Yuan, Y., Lu, C., Wei, H., Gou, C., Zheng, Z., Yin, Y., Zhang, X., Liu, K., Wang, B., Song, C., Shi, N., Kohel, R. J., Percy, R. G., Yu, J. Z., Zhu, Y. X. and Yu, S., 2012, The draft genome of a diploid cotton Gossypiumraimondii. Nat. Genet., 44 (10): 1098-103.

1999, the limits of selection during maize domestication. R L Wang, A Stec, J Hey, L Lukens, J F Doebley, Nature. 398Wang, R. L., Stec, A., Hey, J., Lukens, L. and Doebley, J. F., 1999, the limits of selection during maize domestication. Nature, 398: 236-239.

RNA-Seq: a revolutionary tool for transcriptomics. Z Wang, M Gerstein, M Snyder, Nat. Rev. Genet. 101Wang, Z., Gerstein, M. and Snyder, M., 2009, RNA-Seq: a revolutionary tool for transcriptomics. Nat. Rev. Genet., 10(1): 57-63.

Identifying and mapping genes of economic significance. N F Weeden, G M Timmerman, J Lu, Euphytica. 731-2Weeden, N. F., Timmerman, G. M. and Lu, J., 1993, Identifying and mapping genes of economic significance. Euphytica, 73 (1-2): 191-198.

Dsubgenome bias of Xcmresistance genes in tetraploidGossypium (cotton) suggests that polyploid formation has created novel avenues for evolution. R J Wright, P M Thaxton, K M El-Zik, A H Paterson, Genetics. 1494Wright, R. J., Thaxton, P. M., El-Zik, K. M. and Paterson, A. H., 1998, D- subgenome bias of Xcmresistance genes in tetraploidGossypium (cotton) suggests that polyploid formation has created novel avenues for evolution. Genetics, 149(4): 1987-1996.

Molecular mapping of QTLs for resistance to the greenbugSchizaphisgraminum (Rondani) in Sorghum bicolor (Moench). Y Q Wu, Y H Huang, Theor. Appl. Genet. 117Wu, Y. Q. and Huang, Y. H., 2008, Molecular mapping of QTLs for resistance to the greenbugSchizaphisgraminum (Rondani) in Sorghum bicolor (Moench). Theor. Appl. Genet., 117:117-124.

Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph. Y Wu, P R Bhat, T J Close, PLoS Genet. 41000212Wu, Y., Bhat, P. R., Close, T.J., 2009, Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph. PLoS Genet; 4: e1000212.

New SSR markers for use in cotton (Gossypium spp.) improvement. J Xiao, K Wu, D D Fang, D M Stelly, J Yu, J. Cotton Sci. 13Xiao, J., Wu, K., Fang, D. D., Stelly, D. M. and Yu, J., 2009, New SSR markers for use in cotton (Gossypium spp.) improvement. J. Cotton Sci. 13: 75- 157.

Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes. X Xu, X Liu, S Ge, J D Jensen, F Hu, X Li, Y Dong, R N Gutenkunst, L Fang, L Huang, J Li, W He, G Zhang, X Zheng, F Zhang, Y Li, C Yu, K Kristiansen, X Zhang, J Wang, M Wright, S Mccouch, R Nielsen, J Wang, W Wang, Nat. Biotechnol. 30Xu, X., Liu, X., Ge, S., Jensen, J. D., Hu, F., Li, X., Dong, Y., Gutenkunst, R. N., Fang, L., Huang, L., Li, J., He, W., Zhang, G., Zheng, X., Zhang, F., Li, Y., Yu, C., Kristiansen, K., Zhang, X., Wang, J., Wright, M., McCouch, S., Nielsen, R., Wang, J. and Wang, W., 2012, Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes. Nat. Biotechnol., 30: 105-111.

Markerassisted selection in plant breeding: from publications to practice. Y Xu, J H Crouch, Crop Sci. 482Xu, Y. and Crouch, J. H., 2008, Marker- assisted selection in plant breeding: from publications to practice. Crop Sci., 48(2): 391-407.

constructing a plant genetic linkage map with DNA markers. N D Young, DNA-based markers in plants. Ronald, I. K. V. and Phillips, L.Kluwer, Dordrecht, Boston, LondonYoung, N. D., 1994, constructing a plant genetic linkage map with DNA markers. In: DNA-based markers in plants (eds. Ronald, I. K. V. and Phillips, L.), Kluwer, Dordrecht, Boston, London. pp. 39-57.

QTL mapping and quantitative disease resistance in plants. N D Young, Annu Rev Phytopathol. 34Young, N.D., 1996. QTL mapping and quantitative disease resistance in plants. Annu Rev Phytopathol 34: 479-501

High-density linkage map of cultivated allotetraploid cotton based on SSR, TRAP, SRAP and AFLP markers. J W Yu, S X Yu, C R Lu, W Wang, S L Fan, M Z Song, Z X Lin, X L Zhang, J F Zhang, J. Integr Plant Biol. 49Yu, J. W., Yu, S. X., Lu, C. R., Wang, W., Fan, S. L., Song, M. Z., Lin, Z. X., Zhang, X. L. and Zhang, J. F., 2007, High-density linkage map of cultivated allotetraploid cotton based on SSR, TRAP, SRAP and AFLP markers. J. Integr Plant Biol., 49: 716-724.

. J Z Yu, R J Kohel, D D Fang, J Cho, A Van Deynze, M Ulloa, S M Hoffman, A E Pepper, D M Stelly, J N Jenkins, S Saha, S P Kumpatla, M R Shah, W V Hugie, Percy, Yu, J. Z., Kohel, R. J., Fang, D. D., Cho, J., Van Deynze, A., Ulloa, M., Hoffman, S. M., Pepper, A. E., Stelly, D. M., Jenkins, J. N., Saha, S., Kumpatla, S. P., Shah, M. R., Hugie, W. V. and Percy,

A high density simple sequence repeat and single nucleotide polymorphism genetic map of the tetraploid cotton genome. R G , 3R. G., 2012, A high density simple sequence repeat and single nucleotide polymorphism genetic map of the tetraploid cotton genome. G3, 2 (1): 43- 58.

QTL mapping of leafy heads by genome resequencing in the RIL population of Brassica rapa. X Yu, H Wang, W Zhong, J Bai, P Liu, Y He, PloS One. 81076059Yu, X., Wang. H., Zhong, W., Bai, J., Liu, P. and He, Y., 2013, QTL mapping of leafy heads by genome resequencing in the RIL population of Brassica rapa. PloS One, 8(10): e76059.

Genome structure of cotton revealed by a genome-wide SSR genetic map constructed from a BC1 population between Gossypium hirsutum and G. barbadense. Y Yu, D Yuan, S Liang, X Li, X Wang, BMC Genomics. 1215Yu, Y., Yuan, D., Liang, S., Li, X. and Wang, X., 2011, Genome structure of cotton revealed by a genome-wide SSR genetic map constructed from a BC1 population between Gossypium hirsutum and G. barbadense. BMC Genomics, 12: 15----.

Molecular Mapping of the Cotton Genome. Z H Yu, Y H Park, G R Lazo, R J Kohel, Agron, Abstracts, ASA, Madison, Wis, USAYu, Z. H., Park, Y. H., Lazo, G. R. and Kohel, R. J., 1997, Molecular Mapping of the Cotton Genome. Agron, Abstracts, ASA, Madison, Wis, USA

Theoretical basis of precision mapping of quantitative trait loci. Z B Zeng, Proc. Natl. Acad. Sci. Natl. Acad. SciUSA90Zeng, Z. B., 1993, Theoretical basis of precision mapping of quantitative trait loci. Proc. Natl. Acad. Sci., USA, 90: 10972-10976.

Precision mapping of quantitative trait loci. Z B Zeng, Genet. 136Zeng, Z. B., 1994, Precision mapping of quantitative trait loci. Genet, 136: 1457- 1468.

. Z B Zeng, K Chen-Hung, C J Basten, estimating the genetic architecture of quantitative traitsZeng, Z. B., Chen-Hung K. and Basten, C. J., 1999, estimating the genetic architecture of quantitative traits.

. Genetical Res. 743Genetical Res., 74(3): 279-289.

Molecular tagging of a major QTL for fibre strength in upland cotton and its marker assisted selection. T Z Zhang, Y L Yuan, J Yu, W Z Guo, R J Kohel, Theor. Appl. Genet. 106Zhang, T. Z., Yuan, Y. L., Yu, J., Guo, W. Z. and Kohel, R. J., 2003, Molecular tagging of a major QTL for fibre strength in upland cotton and its marker assisted selection. Theor. Appl. Genet., 106: 262-268.

2013, a consensus linkage map provides insights on genome character and evolution in common carp (Cyprinuscarpio L.). X Zhang, Y Zhang, X Zheng, Y Kuang, Z Zhao, Mar Biotechnol (NY). 15Zhang, X., Zhang, Y., Zheng, X., Kuang, Y., Zhao, Z., 2013, a consensus linkage map provides insights on genome character and evolution in common carp (Cyprinuscarpio L.). Mar Biotechnol (NY) 15: 275-312.

Construction of a genetic linkage map and QTL analysis of fiber-related traits in upland cotton (Gossypium hirsutum L.). Z S Zhang, Y H Xiao, M Luo, Euphytica. 1441-2Zhang, Z. S., Xiao, Y. H. and Luo, M., 2005, Construction of a genetic linkage map and QTL analysis of fiber-related traits in upland cotton (Gossypium hirsutum L.). Euphytica, 144(1-2): 91-99.

Genetic diversity and genetic relationship of Jatrophacurcas between China and Southeast Asian revealed by amplified fragment length polymorphisms. Z Zhang, X Guo, B Liu, L Tang, Chen , F , ‖ African J of Biotech. 1015Zhang, Z., Guo, X., Liu, B., Tang, L., and Chen, F., 2011, -Genetic diversity and genetic relationship of Jatrophacurcas between China and Southeast Asian revealed by amplified fragment length polymorphisms,‖ African J of Biotech, 10 (15) 2825-2832.

Construction of a comprehensive PCR-based marker linkage map and QTL mapping for fiber quality traits in upland cotton. Z Zhang, M Hu, J Zhang, D Liu, J Zheng, K Zhang, W Wang, Q Wan, Gossypium hirsutum L.Zhang, Z., Hu, M., Zhang, J., Liu, D., Zheng, J., Zhang, K., Wang, W. and Wan, Q., 2009, Construction of a comprehensive PCR-based marker linkage map and QTL mapping for fiber quality traits in upland cotton (Gossypium hirsutum L.).

. Mol. Breed. 241Mol. Breed., 24 (1): 49-61.

Assessment of dayneutral backcross populations of cotton using AFLP markers. M Zhong, J Mccarty, J N Cjenkins, S Saha, J. Cotton Sci. 62Zhong, M., McCarty, J. CJenkins, J. N. and Saha, S., 2002, Assessment of day- neutral backcross populations of cotton using AFLP markers. J. Cotton Sci., 6(2): 97-103.

A Review of QTL Mapping in Cotton: Molecular Markers, Mapping Populations and Statistical Methods. Ashok Kumar Meena, M Ramesh, C H Nagaraju, Bheru Lal Kumhar, 10.20546/ijcmas.2017.606.364Int.J.Curr.Microbiol.App.Sci. 66Ashok kumar meena, M. Ramesh, C.H. Nagaraju and Bheru Lal Kumhar. 2017. A Review of QTL Mapping in Cotton: Molecular Markers, Mapping Populations and Statistical Methods. Int.J.Curr.Microbiol.App.Sci. 6(6): 3057-3080. doi: https://doi.org/10.20546/ijcmas.2017.606.364