# Open Access A genome-wide transcriptional activity survey of rice transposable element-related genes

CorpusID: 52828534 - [https://www.semanticscholar.org/paper/f87c08ae8201b616738180c26569496701fbd5a3](https://www.semanticscholar.org/paper/f87c08ae8201b616738180c26569496701fbd5a3)

Fields: Environmental Science, Biology, Medicine

## (s0) Background
Number of References: 124

(p0.0) The completion of the rice (Oryza sativa) genome sequence allowed further functional classification of the coding sequences of this important crop and model of grass species [1,2]. Detailed annotation of the rice genome revealed that nearly a quarter of the rice open reading frame (ORF) coding capacity has features of transposable elements (TEs) and are therefore defined as TE-related genes [3]. Like other genes, these TE-related genes have predicted normal gene structure with protein coding capacity. However, they share significant sequence similarity with known TEs in either or both of the following ways: they have TE signature sequences in The Institute for Genomic Research (TIGR) Oryza Repeat Database [4] or they contain TE-related protein domains [3]. By this definition, TE-related genes can include potentially active TEs (based on the existence of a functional ORF) as well as cellular genes derived from TEs. Many of these TErelated genes encode reverse transcriptases, transposases, or other related proteins [5], and they can be further classified based on protein domain and other sequence features [3,4]. Those TEs overwhelming in number that lack functional ORFs are not considered to be genes [3]. Although there are many TE-related genes, the biologic functions of these genes remain elusive [6].

(p0.1) TEs are considered to be important for the maintenance and diversification of genomes. TEs are usually separated into two classes that differ in the mode of propagation: retrotransposons, or type I elements, which transpose by reverse transcription of an RNA intermediate; and type II elements, which only use a DNA intermediate in movement within the genome. Both classes can be further divided into several superfamilies, each with a unique evolutionary history. Representatives of virtually all superfamilies of TEs have been detected in grass genomes [7][8][9]. Accumulating evidence suggests that TE activities have profound impact on the genome [5], influencing genome size, genome rearrangement, chromatin transcription, and gene evolution [10][11][12][13][14][15]; many of these factors relying specifically on the transposition activity of TEs.

(p0.2) Although most TEs are considered inactive [16,17], there have been isolated reports of TE transposition in rice and other grasses [18]. A common condition promoting transposition is stress, including that which occurs in in vitro cell or tissue culture [19][20][21][22]. Developmental regulation of transposition has also been reported in intact plants [23,24]. Transcription of TE-related genes is required for their own transposition and that of other related TEs, although transcription itself may not be sufficient for transposition [20,25,26]. Analysis of TE-related genes from certain subgroups of the type I class and the Mutator-like superfamily of the type II class suggests that their transcripts are widely present in grasses [27,28]. Most of these transcribed TEs have coding capacity and are therefore considered TE-related genes. A recent study of expressed sequence tags (ESTs) in sugarcane identified 267 active TE-related transcripts [29]. Transcription of TE-related genes was also reported in an unbiased survey of the transcriptional activity of a single rice chromosome using a tiling microarray [30].

(p0.3) Apart from the potentially active TEs among these TE-related genes, domesticated TE-related genes, which acquire new functions for the host, have also been found to exist. Although our current classification for distinguishing TE-related genes from non-TE-related genes is not definitive [31], two recent studies in Arabidopsis identified domesticated TE-related genes contributing to cellular processes [32,33]. Similar examples were also found in animals [34,35]. Such findings in part support the hypothesis that TE-related genes may influence the evolution of their host by providing a source of novel coding capacity.

(p0.4) The potential impact of domesticated TE-related genes on the evolution of genomes requires systematic investigation. One attempt to identify further domesticated TE-related genes is sequence mining [36]. Because a change of position through transcription can be detrimental to the host, transposonderived genes with known host function usually lack mobility. As a consequence, they may be devoid of transposon-specific terminal sequences [32,36]. By employing this criterion in a search, one particular member of the MULE superfamily was identified as a domesticated gene candidate [36]. Transcription is an important feature of domesticated TE-related genes, because it is generally required in cellular functions of the host [32,33]. By surveying transcriptional activity and combining other approaches, we would be able to identify domesticated TE-derived gene candidates.

(p0.5) Another mechanism for the evolution of new genes from TEs is through their ability to acquire and fuse fragments of genes to new genomic locations, as seen in plant Pack-MULE and, more recently, in certain Helitron-like and CACTA elements [13,14,37,38]. However, many of these Pack-MULEs have been suggested to possess pseudogene-like features [39]. Pack-MULE, as a unique group of TE-related genes, is relatively well annotated and is a current focus of interest regarding the origin of genes [37].

(p0.6) Given the paucity of information on TE-related genes, a systematic study of their transcriptional activity in a well characterized genome is required to enhance our understanding of the activity of TE-related genes. That the sequence of the rice genome is now completely annotated makes it a good resource for such a genome-wide survey [3]. Recent advances in microarray technology allow us to study the transcriptional activity of genes in a high-throughput manner. It is therefore possible to conduct a genome-wide survey of the transcriptional activity of rice TE-related genes, especially those more divergent ones for which unique oligomer probes can be designed. Different from simple TEs composing mostly repetitive sequences, many TE-related genes are diverged enough to have short oligomers representing their unique sequence regions. Such an approach has recently been utilized to analyze transcription of TE-related genes in plants and animals [11,30,40]. In addition to TE-related genes, TEs without protein-coding capacity and other tandem repeats may also exhibit transcriptional activity [26,41]. Transcripts derived from tandem repeats in the heterochromatin can give rise to small RNAs, which in turn direct the modification of histones and DNA in TE-related sequences and nearby regions by means of RNA interference [16]. Although transcripts from tandem repeats are important for the genome, their highly repetitive nature prohibits characterization of their unique identities in chromosomal organization on a genome-wide scale [42,43].
## (s6) Transcription of type II TE-related genes
Number of References: 50

(p6.0) Type II TE-related genes are in general more actively transcribed than type I TE-related genes. Different from type I, Degrees of lineage-specific transcription in the LINE superfamily Figure 3 Degrees of lineage-specific transcription in the LINE superfamily. The phylogenetic tree was generated from a multiple alignment of conceptually translated sequences by using neighbor-joining methods and rooted with human L1. Bootstrap values were calculated from 1,000 replicates. Sample numbers are identical to those in Table 2. Shades of gray indicate the magnitude of transcription signals, which are based on microarray hybridization signals without units. Names of previously reported members are shown. *Previously reported members with transcription or transposition. † Previously reported inactivate members. LINE, long interspersed element.  Os07g04110   SS  TS  TR  FL  HP  FP  SC  CR  CS  TSD  TSS  FLD  FLS  HPD  Mutator-like superfamily (MULE) is one of the first groups of identified transposases with a few reported transcriptionally active members in rice [27]. There are 607 autonomous members of this superfamily (Table 1), which has one of the strongest transcription levels, at 35% to 40% in each sample ( Figure 4). The MULEs can be further divided into three branches: MuDR-like, Jittery-like, and TRAP-like [27]. The TRAP-like branch may have recently been amplified, and high similarity among family members has resulted in lack of unique oligo probes with which to examine their expression profiles. Interestingly, we have found at least three clades with clear active transcription in MuDR-like and Jittery-like branches ( Figure 4). The one highly transcribed clade in the MuDR-like branch included MUG1, an evolutionarily conserved MULE sequence found in diverse angiosperms and a candidate for categorization as a domesticated transposaserelated gene [36]. The larger, highly transcribed clade in the Jittery-like branch includes homologs to Arabidopsis genes FAR1 and FHY3, both of which are transposon-derived genes with demonstrated host function as transcription factors downstream of phytochrome A [32,51,52]. There are no reports on any members of the other highly transcribed clade in the Jittery-like branch, which has rampant transcription ( Figure 4, middle).

(p6.1) The CACTA superfamily is a diverse group of high-copy repetitive genes in grasses [53,54]. CACTA transposons with active transcription or even transposition have been reported in rice and other grass genomes [54][55][56][57]. A total of 2,276 intact CACTA transposase-coding genes are identified in rice, making it the largest superfamily in type II TE-related genes ( Table 1). The CACTA superfamily is also highly active, with more than 40% transcribed in each sample. Several clades with active transcription were identified (Additional data file 4). Among them, two clades include over 20 members. No members within these actively transcribed CACTA transposons have previously been characterized.

(p6.2) The hAT-like superfamily is another widespread superfamily in grasses [58]. It is a medium-sized superfamily in rice with 184 autonomous members (Table 1). About 20% of this superfamily is transcribed in a single sample ( Figure 5). Interestingly, we found a small clade of four genes that exhibited relatively uniform and strong transcription across a wide range of samples. A sequence comparison indicates that these genes have high similarity with a recently identified domesticated Arabidopsis transposase DAYSLEEPER, which is a pleiotropic regulator of development through its specific DNA-binding activity [33]. There is one reported hAT-like transposon group in rice, Dart, which is capable of active transposition in plants [24,59]. Sequence analysis indicates that Dart is a recently amplified clade with 30 almost identical members. Although no oligonucleotide probes have been developed to represent individual members, there are a few probes that can detect all or most of them. Clear hybridization signals have been found for these probes in all shoot and cell culture samples. This finding suggests that some or all members of Dart are highly transcribed in a large number of rice samples.

(p6.3) Both PIF/Pong-like and Mariner-like TE-related genes are autonomous partners of nonautonomous miniature inverted repeat transposable elements (MITEs), which are ubiquitous in the rice genome [12]. Low proportions of both families have detectable transcription (<20%) in each sample ( Figure  6 and Additional data file 4). Two transpositionally active PIF/Pong-like elements were recently reported: maize PIF and rice Pong [22,23,60]. Interestingly, the rice homolog of PIF, namely OsPIF1 [60], was not expressed in any samples ( Figure 6). There are six almost identical Pong elements in the rice genome, which are represented by a single probe in the microarray. This probe detected transcription activity in tillering shoot and drought-exposed panicles only ( Figure 6), suggesting rigorous regulation at the transcriptional level for members of this family. We did not detect any transcriptional activity of the Pong element in cultured cells. The Marinerlike superfamily has a much smaller member size [61]; this superfamily includes a small proportion of transcribed genes, similar to that for the PIF/Pong-like superfamily (Additional data file 4).

(p6.4) A recently identified unique type II TE superfamily, Helitronlike, is relatively under-characterized in the rice genome [62]. Strikingly, Helitron-like transposons have the potential to move and shuffle genes or exons in maize [13,14]. In rice, we found only one member with transcriptional activity in all the samples. There is no other Helitron-like transposon among the seven examined ones with transcriptional activity in any samples (Additional data file 5).
## (s7) Transcription of Pack-MULE
Number of References: 16

(p7.0) Genes or exons can be transduplicated by MULEs [9,63], which have recently been suggested to be important facilitators of the evolution of genes in higher plants, and have therefore been termed Pack-MULE [37]. However, a detailed sequence analysis suggests that the products of this process are more likely to be pseudogenes than novel functional genes [39]. To gain better insight into this group, we examined their transcriptional activities using microarray analysis, because transcription is usually a prerequisite for biologic function of 

(p7.1) Genes or exons can be transduplicated by MULEs [9,63], which have recently been suggested to be important facilitators of the evolution of genes in higher plants, and have therefore been termed Pack-MULE [37]. However, a detailed sequence analysis suggests that the products of this process are more likely to be pseudogenes than novel functional genes [39]. To gain better insight into this group, we examined their transcriptional activities using microarray analysis, because transcription is usually a prerequisite for biologic function of 
## (s14) Transcription profiles of TE-related genes in rice
Number of References: 12

(p14.0) TEs account for an overwhelming proportion of plant genomes. To ensure the viability of their host and hence their own survival, the transposition of TEs should be tightly controlled [17]. Transcribed autonomous TEs among TE-related genes have the potential to self-activate or activate transcrip-tion of related nonautonomous TEs. Transcriptional regulation is therefore one major control step used by plants, but it remains insufficiently understood. The recently available rice genome sequence has enabled us to characterize TErelated gene transcription on a genome-wide scale.

(p14.1) Using 70-mer oligonucleotide microarrays covering more than 2,000 rice TE-related genes, we surveyed the transcription profiles under a wide range of organ samples under various conditions. Considering that TE-derived cellular genes are relatively rare, autonomous TEs probably contribute to most of these TE-related genes. Genome profiling revealed that 25% to 30% of the TE-related genes were transcribed in one sample, which was much lower than the corresponding percentage of non-TE-related genes (Figures 1 and  2). Moreover, TE-related genes differed from their non-TErelated counterparts in two additional aspects. First, TErelated genes tended to be transcribed in only a subset of organs or developmental stages, whereas non-TE-related genes had transcription in more samples on average ( Figure 1 and Figure 2). Second, transcribed TE-related genes exhibited weaker transcription overall compared with non-TErelated genes in all of the samples we profiled (Figure 2). It worth noting that our estimation of TE-related gene transcription was biased toward low-copy elements, because it was difficult to distinguish transcripts among recently duplicated high-copy TE-related genes, which share high sequence similarity within clades. It has been reported in Arabidopsis and Drosophila that the activity of TE elements may reduce as the copy number increases [77,78]. Therefore, we expect the transcriptional activity of those high-copy TE-related genes will be lower than for low-copy ones.

(p14.2) Among TE-related genes, a smaller proportion of type I than type II genes were transcribed (Figure 1a), a discrepancy that resulted primarily from the strong transcription of MULE and CACTA superfamilies as well as unclassified type II members. It is interesting to note that all TE-related gene superfamilies with potential to severely expand, including all type I TE-related genes and PIF/Pong-like, Mariner-like and Helitron-like type II TE-related genes, were more tightly controlled at the transcription level. Type I TE-related genes are amplified through a copy-and-paste mechanism [79]. PIF/Pong-like and Mariner-like superfamilies regulate the activity of MITEs, which dominate the rice genome [12]. Members of the Helitron-like superfamily go through a unique rolling cycle replication to rapidly amplify themselves [62].
## (s18) Microarray analysis
Number of References: 10

(p18.0) The rice 70-mer oligonucleotide set was described previously [44]. Briefly, 70-mer oligonucleotides were designed based on a combination of FGENESH predicted genes from an improved shotgun sequence [2] and the available full-length cDNAs and ESTs [87]. Designed 70-mer oligonucleotides correspond to the sequence within the coding region of genes, and the design was corrected for such factors as oligo crosshybridization, uniform TM value, GC content, and hairpin/ stem nucleotide number. All oligonucleotides were remapped to TIGR rice genome annotation version 3.1 genes [3] using BLAST. We requested greater than 90% alignment of a 70-mer oligonucleotide probe to a gene during the remapping. Moreover, only those 70-mer probes without a greater than 80% second-best aligned gene were considered to be free from cross-hybridization. These criteria were selected because a mismatch of 20% removes more than 90% of the hybridization signals, whereas a 10% mismatch retains at least half of the hybridization signals [88].
## (s20) Microarray data processing
Number of References: 16

(p20.0) Microarray spot intensity signals were acquired using Axon GenePix Pro 3.0 software package (Molecular Devices, Sun-nyvale, CA, USA). To identify and remove systematic sources of variation, including dye and spatial effects, spot intensities from the GenePix Pro output files of all repeats of a given sample pair were normalized using limma, a software package for the analysis of gene expression microarray [91]. This normalization process identified and ameliorated spatial, intensity-based, and dye-specific artifacts using multiple step corrections. To determine objectively whether a gene exhibited significant expression in a given sample, we followed a method that relied on negative control spots and data reproducibility [44]. To estimate nonspecific hybridization, a distribution of normalized intensities was obtained from the subset of negative control spots present on each array slide. From this distribution, we chose an intensity cutoff at which less than 10% of the distribution was greater than or equal to this threshold. Expression of a gene was only considered detectable if it was above the threshold in two or more repeats out of the three. These criteria had been demonstrated suitable for oligonucleotide arrays with an error rate range of 1% to 3% false negatives [44]. RT-PCR results and independent analysis using different microarrays and statistical approaches [43] further supported this threshold.

(p20.1) Microarray spot intensity signals were acquired using Axon GenePix Pro 3.0 software package (Molecular Devices, Sun-nyvale, CA, USA). To identify and remove systematic sources of variation, including dye and spatial effects, spot intensities from the GenePix Pro output files of all repeats of a given sample pair were normalized using limma, a software package for the analysis of gene expression microarray [91]. This normalization process identified and ameliorated spatial, intensity-based, and dye-specific artifacts using multiple step corrections. To determine objectively whether a gene exhibited significant expression in a given sample, we followed a method that relied on negative control spots and data reproducibility [44]. To estimate nonspecific hybridization, a distribution of normalized intensities was obtained from the subset of negative control spots present on each array slide. From this distribution, we chose an intensity cutoff at which less than 10% of the distribution was greater than or equal to this threshold. Expression of a gene was only considered detectable if it was above the threshold in two or more repeats out of the three. These criteria had been demonstrated suitable for oligonucleotide arrays with an error rate range of 1% to 3% false negatives [44]. RT-PCR results and independent analysis using different microarrays and statistical approaches [43] further supported this threshold.
## (s22) Cluster analysis
Number of References: 4

(p22.0) Cluster analysis was applied to all TE-related genes and 1,353 randomly selected non-TE-related genes showing expression in at least one sample. Average normalized log-transformed expression intensities were subjected to cluster analysis. For hierarchical clustering, Pearson correlation was used to compute similarities, and a complete linkage clustering algorithm was used. Cluster analysis was performed using the software Cluster [94] and visualized using custom scripts.

(p22.1) Cluster analysis was applied to all TE-related genes and 1,353 randomly selected non-TE-related genes showing expression in at least one sample. Average normalized log-transformed expression intensities were subjected to cluster analysis. For hierarchical clustering, Pearson correlation was used to compute similarities, and a complete linkage clustering algorithm was used. Cluster analysis was performed using the software Cluster [94] and visualized using custom scripts.
## (s24) Histone and DNA methylation
Number of References: 8

(p24.0) ChIP was carried out as described elsewhere [64] using seedling shoots and cultured cells. Histone H3 anti-dimethyl lysine-4 or anti-dimethyl lysine-9 antibodies (Upstate, Avon, NY, USA) were used to precipitate genomic DNA, which was resuspended in water for PCR analysis. The same PCR and gel electrophoresis conditions were used as for RT-PCR analysis.

(p24.1) Methylation of DNA was assessed by McrBC digestion following a previously published protocol [81]. Genomic DNA was isolated from seedling shoots and cultured cells using Qiagen DNeasy plant kit and divided into two equal samples. One sample was digested with McrBC, a methylation-dependent restriction enzyme that cuts the sequence A/G 5 mC (New England Biolabs, Beverly, MA, USA). Both digested and untreated samples were subject to PCR amplification as described previously. Successful amplification after digestion indicates lack of methylation.

(p24.2) ChIP was carried out as described elsewhere [64] using seedling shoots and cultured cells. Histone H3 anti-dimethyl lysine-4 or anti-dimethyl lysine-9 antibodies (Upstate, Avon, NY, USA) were used to precipitate genomic DNA, which was resuspended in water for PCR analysis. The same PCR and gel electrophoresis conditions were used as for RT-PCR analysis.

(p24.3) Methylation of DNA was assessed by McrBC digestion following a previously published protocol [81]. Genomic DNA was isolated from seedling shoots and cultured cells using Qiagen DNeasy plant kit and divided into two equal samples. One sample was digested with McrBC, a methylation-dependent restriction enzyme that cuts the sequence A/G 5 mC (New England Biolabs, Beverly, MA, USA). Both digested and untreated samples were subject to PCR amplification as described previously. Successful amplification after digestion indicates lack of methylation.
## (s33) Background
Number of References: 124

(p33.0) The completion of the rice (Oryza sativa) genome sequence allowed further functional classification of the coding sequences of this important crop and model of grass species [1,2]. Detailed annotation of the rice genome revealed that nearly a quarter of the rice open reading frame (ORF) coding capacity has features of transposable elements (TEs) and are therefore defined as TE-related genes [3]. Like other genes, these TE-related genes have predicted normal gene structure with protein coding capacity. However, they share significant sequence similarity with known TEs in either or both of the following ways: they have TE signature sequences in The Institute for Genomic Research (TIGR) Oryza Repeat Database [4] or they contain TE-related protein domains [3]. By this definition, TE-related genes can include potentially active TEs (based on the existence of a functional ORF) as well as cellular genes derived from TEs. Many of these TErelated genes encode reverse transcriptases, transposases, or other related proteins [5], and they can be further classified based on protein domain and other sequence features [3,4]. Those TEs overwhelming in number that lack functional ORFs are not considered to be genes [3]. Although there are many TE-related genes, the biologic functions of these genes remain elusive [6].

(p33.1) TEs are considered to be important for the maintenance and diversification of genomes. TEs are usually separated into two classes that differ in the mode of propagation: retrotransposons, or type I elements, which transpose by reverse transcription of an RNA intermediate; and type II elements, which only use a DNA intermediate in movement within the genome. Both classes can be further divided into several superfamilies, each with a unique evolutionary history. Representatives of virtually all superfamilies of TEs have been detected in grass genomes [7][8][9]. Accumulating evidence suggests that TE activities have profound impact on the genome [5], influencing genome size, genome rearrangement, chromatin transcription, and gene evolution [10][11][12][13][14][15]; many of these factors relying specifically on the transposition activity of TEs.

(p33.2) Although most TEs are considered inactive [16,17], there have been isolated reports of TE transposition in rice and other grasses [18]. A common condition promoting transposition is stress, including that which occurs in in vitro cell or tissue culture [19][20][21][22]. Developmental regulation of transposition has also been reported in intact plants [23,24]. Transcription of TE-related genes is required for their own transposition and that of other related TEs, although transcription itself may not be sufficient for transposition [20,25,26]. Analysis of TE-related genes from certain subgroups of the type I class and the Mutator-like superfamily of the type II class suggests that their transcripts are widely present in grasses [27,28]. Most of these transcribed TEs have coding capacity and are therefore considered TE-related genes. A recent study of expressed sequence tags (ESTs) in sugarcane identified 267 active TE-related transcripts [29]. Transcription of TE-related genes was also reported in an unbiased survey of the transcriptional activity of a single rice chromosome using a tiling microarray [30].

(p33.3) Apart from the potentially active TEs among these TE-related genes, domesticated TE-related genes, which acquire new functions for the host, have also been found to exist. Although our current classification for distinguishing TE-related genes from non-TE-related genes is not definitive [31], two recent studies in Arabidopsis identified domesticated TE-related genes contributing to cellular processes [32,33]. Similar examples were also found in animals [34,35]. Such findings in part support the hypothesis that TE-related genes may influence the evolution of their host by providing a source of novel coding capacity.

(p33.4) The potential impact of domesticated TE-related genes on the evolution of genomes requires systematic investigation. One attempt to identify further domesticated TE-related genes is sequence mining [36]. Because a change of position through transcription can be detrimental to the host, transposonderived genes with known host function usually lack mobility. As a consequence, they may be devoid of transposon-specific terminal sequences [32,36]. By employing this criterion in a search, one particular member of the MULE superfamily was identified as a domesticated gene candidate [36]. Transcription is an important feature of domesticated TE-related genes, because it is generally required in cellular functions of the host [32,33]. By surveying transcriptional activity and combining other approaches, we would be able to identify domesticated TE-derived gene candidates.

(p33.5) Another mechanism for the evolution of new genes from TEs is through their ability to acquire and fuse fragments of genes to new genomic locations, as seen in plant Pack-MULE and, more recently, in certain Helitron-like and CACTA elements [13,14,37,38]. However, many of these Pack-MULEs have been suggested to possess pseudogene-like features [39]. Pack-MULE, as a unique group of TE-related genes, is relatively well annotated and is a current focus of interest regarding the origin of genes [37].

(p33.6) Given the paucity of information on TE-related genes, a systematic study of their transcriptional activity in a well characterized genome is required to enhance our understanding of the activity of TE-related genes. That the sequence of the rice genome is now completely annotated makes it a good resource for such a genome-wide survey [3]. Recent advances in microarray technology allow us to study the transcriptional activity of genes in a high-throughput manner. It is therefore possible to conduct a genome-wide survey of the transcriptional activity of rice TE-related genes, especially those more divergent ones for which unique oligomer probes can be designed. Different from simple TEs composing mostly repetitive sequences, many TE-related genes are diverged enough to have short oligomers representing their unique sequence regions. Such an approach has recently been utilized to analyze transcription of TE-related genes in plants and animals [11,30,40]. In addition to TE-related genes, TEs without protein-coding capacity and other tandem repeats may also exhibit transcriptional activity [26,41]. Transcripts derived from tandem repeats in the heterochromatin can give rise to small RNAs, which in turn direct the modification of histones and DNA in TE-related sequences and nearby regions by means of RNA interference [16]. Although transcripts from tandem repeats are important for the genome, their highly repetitive nature prohibits characterization of their unique identities in chromosomal organization on a genome-wide scale [42,43].
## (s39) Transcription of type II TE-related genes
Number of References: 50

(p39.0) Type II TE-related genes are in general more actively transcribed than type I TE-related genes. Different from type I, Degrees of lineage-specific transcription in the LINE superfamily Figure 3 Degrees of lineage-specific transcription in the LINE superfamily. The phylogenetic tree was generated from a multiple alignment of conceptually translated sequences by using neighbor-joining methods and rooted with human L1. Bootstrap values were calculated from 1,000 replicates. Sample numbers are identical to those in Table 2. Shades of gray indicate the magnitude of transcription signals, which are based on microarray hybridization signals without units. Names of previously reported members are shown. *Previously reported members with transcription or transposition. † Previously reported inactivate members. LINE, long interspersed element.  Os07g04110   SS  TS  TR  FL  HP  FP  SC  CR  CS  TSD  TSS  FLD  FLS  HPD  Mutator-like superfamily (MULE) is one of the first groups of identified transposases with a few reported transcriptionally active members in rice [27]. There are 607 autonomous members of this superfamily (Table 1), which has one of the strongest transcription levels, at 35% to 40% in each sample ( Figure 4). The MULEs can be further divided into three branches: MuDR-like, Jittery-like, and TRAP-like [27]. The TRAP-like branch may have recently been amplified, and high similarity among family members has resulted in lack of unique oligo probes with which to examine their expression profiles. Interestingly, we have found at least three clades with clear active transcription in MuDR-like and Jittery-like branches ( Figure 4). The one highly transcribed clade in the MuDR-like branch included MUG1, an evolutionarily conserved MULE sequence found in diverse angiosperms and a candidate for categorization as a domesticated transposaserelated gene [36]. The larger, highly transcribed clade in the Jittery-like branch includes homologs to Arabidopsis genes FAR1 and FHY3, both of which are transposon-derived genes with demonstrated host function as transcription factors downstream of phytochrome A [32,51,52]. There are no reports on any members of the other highly transcribed clade in the Jittery-like branch, which has rampant transcription ( Figure 4, middle).

(p39.1) The CACTA superfamily is a diverse group of high-copy repetitive genes in grasses [53,54]. CACTA transposons with active transcription or even transposition have been reported in rice and other grass genomes [54][55][56][57]. A total of 2,276 intact CACTA transposase-coding genes are identified in rice, making it the largest superfamily in type II TE-related genes ( Table 1). The CACTA superfamily is also highly active, with more than 40% transcribed in each sample. Several clades with active transcription were identified (Additional data file 4). Among them, two clades include over 20 members. No members within these actively transcribed CACTA transposons have previously been characterized.

(p39.2) The hAT-like superfamily is another widespread superfamily in grasses [58]. It is a medium-sized superfamily in rice with 184 autonomous members (Table 1). About 20% of this superfamily is transcribed in a single sample ( Figure 5). Interestingly, we found a small clade of four genes that exhibited relatively uniform and strong transcription across a wide range of samples. A sequence comparison indicates that these genes have high similarity with a recently identified domesticated Arabidopsis transposase DAYSLEEPER, which is a pleiotropic regulator of development through its specific DNA-binding activity [33]. There is one reported hAT-like transposon group in rice, Dart, which is capable of active transposition in plants [24,59]. Sequence analysis indicates that Dart is a recently amplified clade with 30 almost identical members. Although no oligonucleotide probes have been developed to represent individual members, there are a few probes that can detect all or most of them. Clear hybridization signals have been found for these probes in all shoot and cell culture samples. This finding suggests that some or all members of Dart are highly transcribed in a large number of rice samples.

(p39.3) Both PIF/Pong-like and Mariner-like TE-related genes are autonomous partners of nonautonomous miniature inverted repeat transposable elements (MITEs), which are ubiquitous in the rice genome [12]. Low proportions of both families have detectable transcription (<20%) in each sample ( Figure  6 and Additional data file 4). Two transpositionally active PIF/Pong-like elements were recently reported: maize PIF and rice Pong [22,23,60]. Interestingly, the rice homolog of PIF, namely OsPIF1 [60], was not expressed in any samples ( Figure 6). There are six almost identical Pong elements in the rice genome, which are represented by a single probe in the microarray. This probe detected transcription activity in tillering shoot and drought-exposed panicles only ( Figure 6), suggesting rigorous regulation at the transcriptional level for members of this family. We did not detect any transcriptional activity of the Pong element in cultured cells. The Marinerlike superfamily has a much smaller member size [61]; this superfamily includes a small proportion of transcribed genes, similar to that for the PIF/Pong-like superfamily (Additional data file 4).

(p39.4) A recently identified unique type II TE superfamily, Helitronlike, is relatively under-characterized in the rice genome [62]. Strikingly, Helitron-like transposons have the potential to move and shuffle genes or exons in maize [13,14]. In rice, we found only one member with transcriptional activity in all the samples. There is no other Helitron-like transposon among the seven examined ones with transcriptional activity in any samples (Additional data file 5).
## (s40) Transcription of Pack-MULE
Number of References: 16

(p40.0) Genes or exons can be transduplicated by MULEs [9,63], which have recently been suggested to be important facilitators of the evolution of genes in higher plants, and have therefore been termed Pack-MULE [37]. However, a detailed sequence analysis suggests that the products of this process are more likely to be pseudogenes than novel functional genes [39]. To gain better insight into this group, we examined their transcriptional activities using microarray analysis, because transcription is usually a prerequisite for biologic function of 

(p40.1) Genes or exons can be transduplicated by MULEs [9,63], which have recently been suggested to be important facilitators of the evolution of genes in higher plants, and have therefore been termed Pack-MULE [37]. However, a detailed sequence analysis suggests that the products of this process are more likely to be pseudogenes than novel functional genes [39]. To gain better insight into this group, we examined their transcriptional activities using microarray analysis, because transcription is usually a prerequisite for biologic function of 
## (s47) Transcription profiles of TE-related genes in rice
Number of References: 12

(p47.0) TEs account for an overwhelming proportion of plant genomes. To ensure the viability of their host and hence their own survival, the transposition of TEs should be tightly controlled [17]. Transcribed autonomous TEs among TE-related genes have the potential to self-activate or activate transcrip-tion of related nonautonomous TEs. Transcriptional regulation is therefore one major control step used by plants, but it remains insufficiently understood. The recently available rice genome sequence has enabled us to characterize TErelated gene transcription on a genome-wide scale.

(p47.1) Using 70-mer oligonucleotide microarrays covering more than 2,000 rice TE-related genes, we surveyed the transcription profiles under a wide range of organ samples under various conditions. Considering that TE-derived cellular genes are relatively rare, autonomous TEs probably contribute to most of these TE-related genes. Genome profiling revealed that 25% to 30% of the TE-related genes were transcribed in one sample, which was much lower than the corresponding percentage of non-TE-related genes (Figures 1 and  2). Moreover, TE-related genes differed from their non-TErelated counterparts in two additional aspects. First, TErelated genes tended to be transcribed in only a subset of organs or developmental stages, whereas non-TE-related genes had transcription in more samples on average ( Figure 1 and Figure 2). Second, transcribed TE-related genes exhibited weaker transcription overall compared with non-TErelated genes in all of the samples we profiled (Figure 2). It worth noting that our estimation of TE-related gene transcription was biased toward low-copy elements, because it was difficult to distinguish transcripts among recently duplicated high-copy TE-related genes, which share high sequence similarity within clades. It has been reported in Arabidopsis and Drosophila that the activity of TE elements may reduce as the copy number increases [77,78]. Therefore, we expect the transcriptional activity of those high-copy TE-related genes will be lower than for low-copy ones.

(p47.2) Among TE-related genes, a smaller proportion of type I than type II genes were transcribed (Figure 1a), a discrepancy that resulted primarily from the strong transcription of MULE and CACTA superfamilies as well as unclassified type II members. It is interesting to note that all TE-related gene superfamilies with potential to severely expand, including all type I TE-related genes and PIF/Pong-like, Mariner-like and Helitron-like type II TE-related genes, were more tightly controlled at the transcription level. Type I TE-related genes are amplified through a copy-and-paste mechanism [79]. PIF/Pong-like and Mariner-like superfamilies regulate the activity of MITEs, which dominate the rice genome [12]. Members of the Helitron-like superfamily go through a unique rolling cycle replication to rapidly amplify themselves [62].
## (s51) Microarray analysis
Number of References: 10

(p51.0) The rice 70-mer oligonucleotide set was described previously [44]. Briefly, 70-mer oligonucleotides were designed based on a combination of FGENESH predicted genes from an improved shotgun sequence [2] and the available full-length cDNAs and ESTs [87]. Designed 70-mer oligonucleotides correspond to the sequence within the coding region of genes, and the design was corrected for such factors as oligo crosshybridization, uniform TM value, GC content, and hairpin/ stem nucleotide number. All oligonucleotides were remapped to TIGR rice genome annotation version 3.1 genes [3] using BLAST. We requested greater than 90% alignment of a 70-mer oligonucleotide probe to a gene during the remapping. Moreover, only those 70-mer probes without a greater than 80% second-best aligned gene were considered to be free from cross-hybridization. These criteria were selected because a mismatch of 20% removes more than 90% of the hybridization signals, whereas a 10% mismatch retains at least half of the hybridization signals [88].
## (s53) Microarray data processing
Number of References: 16

(p53.0) Microarray spot intensity signals were acquired using Axon GenePix Pro 3.0 software package (Molecular Devices, Sun-nyvale, CA, USA). To identify and remove systematic sources of variation, including dye and spatial effects, spot intensities from the GenePix Pro output files of all repeats of a given sample pair were normalized using limma, a software package for the analysis of gene expression microarray [91]. This normalization process identified and ameliorated spatial, intensity-based, and dye-specific artifacts using multiple step corrections. To determine objectively whether a gene exhibited significant expression in a given sample, we followed a method that relied on negative control spots and data reproducibility [44]. To estimate nonspecific hybridization, a distribution of normalized intensities was obtained from the subset of negative control spots present on each array slide. From this distribution, we chose an intensity cutoff at which less than 10% of the distribution was greater than or equal to this threshold. Expression of a gene was only considered detectable if it was above the threshold in two or more repeats out of the three. These criteria had been demonstrated suitable for oligonucleotide arrays with an error rate range of 1% to 3% false negatives [44]. RT-PCR results and independent analysis using different microarrays and statistical approaches [43] further supported this threshold.

(p53.1) Microarray spot intensity signals were acquired using Axon GenePix Pro 3.0 software package (Molecular Devices, Sun-nyvale, CA, USA). To identify and remove systematic sources of variation, including dye and spatial effects, spot intensities from the GenePix Pro output files of all repeats of a given sample pair were normalized using limma, a software package for the analysis of gene expression microarray [91]. This normalization process identified and ameliorated spatial, intensity-based, and dye-specific artifacts using multiple step corrections. To determine objectively whether a gene exhibited significant expression in a given sample, we followed a method that relied on negative control spots and data reproducibility [44]. To estimate nonspecific hybridization, a distribution of normalized intensities was obtained from the subset of negative control spots present on each array slide. From this distribution, we chose an intensity cutoff at which less than 10% of the distribution was greater than or equal to this threshold. Expression of a gene was only considered detectable if it was above the threshold in two or more repeats out of the three. These criteria had been demonstrated suitable for oligonucleotide arrays with an error rate range of 1% to 3% false negatives [44]. RT-PCR results and independent analysis using different microarrays and statistical approaches [43] further supported this threshold.
## (s55) Cluster analysis
Number of References: 4

(p55.0) Cluster analysis was applied to all TE-related genes and 1,353 randomly selected non-TE-related genes showing expression in at least one sample. Average normalized log-transformed expression intensities were subjected to cluster analysis. For hierarchical clustering, Pearson correlation was used to compute similarities, and a complete linkage clustering algorithm was used. Cluster analysis was performed using the software Cluster [94] and visualized using custom scripts.

(p55.1) Cluster analysis was applied to all TE-related genes and 1,353 randomly selected non-TE-related genes showing expression in at least one sample. Average normalized log-transformed expression intensities were subjected to cluster analysis. For hierarchical clustering, Pearson correlation was used to compute similarities, and a complete linkage clustering algorithm was used. Cluster analysis was performed using the software Cluster [94] and visualized using custom scripts.
## (s57) Histone and DNA methylation
Number of References: 8

(p57.0) ChIP was carried out as described elsewhere [64] using seedling shoots and cultured cells. Histone H3 anti-dimethyl lysine-4 or anti-dimethyl lysine-9 antibodies (Upstate, Avon, NY, USA) were used to precipitate genomic DNA, which was resuspended in water for PCR analysis. The same PCR and gel electrophoresis conditions were used as for RT-PCR analysis.

(p57.1) Methylation of DNA was assessed by McrBC digestion following a previously published protocol [81]. Genomic DNA was isolated from seedling shoots and cultured cells using Qiagen DNeasy plant kit and divided into two equal samples. One sample was digested with McrBC, a methylation-dependent restriction enzyme that cuts the sequence A/G 5 mC (New England Biolabs, Beverly, MA, USA). Both digested and untreated samples were subject to PCR amplification as described previously. Successful amplification after digestion indicates lack of methylation.

(p57.2) ChIP was carried out as described elsewhere [64] using seedling shoots and cultured cells. Histone H3 anti-dimethyl lysine-4 or anti-dimethyl lysine-9 antibodies (Upstate, Avon, NY, USA) were used to precipitate genomic DNA, which was resuspended in water for PCR analysis. The same PCR and gel electrophoresis conditions were used as for RT-PCR analysis.

(p57.3) Methylation of DNA was assessed by McrBC digestion following a previously published protocol [81]. Genomic DNA was isolated from seedling shoots and cultured cells using Qiagen DNeasy plant kit and divided into two equal samples. One sample was digested with McrBC, a methylation-dependent restriction enzyme that cuts the sequence A/G 5 mC (New England Biolabs, Beverly, MA, USA). Both digested and untreated samples were subject to PCR amplification as described previously. Successful amplification after digestion indicates lack of methylation.
