# Oocyte-associated transcription factors in reprogramming after somatic cell nuclear transfer: a review

CorpusID: 83562471 - [https://www.semanticscholar.org/paper/60c04edd2cf072a5c8cffb1dbf5a25b9da84ba59](https://www.semanticscholar.org/paper/60c04edd2cf072a5c8cffb1dbf5a25b9da84ba59)

Fields: Biology, Medicine

## (s0) Introduction
Number of References: 14

(p0.0) Fetal genesis is a unidirectional process that is accom-panied by a loss of developmental potential of various cell lineages. Starting as a unicellular zygote, the progression of changes ends in the formation of 220 specialized cell types and a living organism [1]. Specific terms such as unipotency, pluripotency, multipotency and totipotency have been ascribed to specific populations of cells that arise during development [1][2][3]. Epigenetic signatures have been identified with these cell populations and correlated with differentiation potentials [4].

(p0.1) The nucleus of differentiated cells can revert to a less differentiated state by a process known as nuclear reprogramming [5]. Briggs, King [6] and Gurdon [7] demonstrated in an amphibian model during the 1950s and 1960s that genomes of adult cells, or terminally differentiated cells, can generate viable cloned animals upon nuclear transfer (NT) [2,3]. The birth of Dolly in 1997, the first cloned mammal [8], and other animal species to follow, indicates that differentiated cells, even those terminally differentiated, can revert to their original nuclear state (the zygotic state) and support term development of a live birth [9].

(p0.2) Although many cloned animal species have produced normal offspring, the somatic cell nuclear transfer (SCNT) efficiency is still extremely low ranging from 1% to 5% of live births [10,11]. The fact that somatic cells do not fully support pseudo-zygote (nuclear transfer embryos) development suggests that nuclear reprogramming is incomplete and that the donor cell nucleus does not fully revert to a pluripotent state. Although the causes are poorly understood, any reprogramming errors are most likely to be caused by genetic and epigenetic insufficiencies. The more that is learned about nuclear reprogramming, the more apparent is its complexity and the potential for as yet unknown factors. This review highlights the current stateof-the-art of somatic cellular reprogramming used in animal cloning, and discusses various nuclear-associated and transcription factors.
## (s1) Nuclear reprogramming begins from oogenesis
Number of References: 3

(p1.0) The vertebrate egg is the only known animal cell that can be reprogrammed with a somatic cell nucleus and undergoes embryonic development [12]. At birth, a mouse ovary contains clusters with thousands of oocytes. The vast majority of the oocytes have already entered meiosis. At, or around the time of birth, some of the oocytes will have entered the transitory stages of prophase (pachytene, early and late diplotene and dictyate) where they remain until meiosis resumes just before ovulation [13]. At the transition of primordial follicles to primary follicles on the third postnatal day in mice, the surrounding granulosa cells revert to a squamous to cuboidal shape with the diameter of the oocyte increasing to more than 20 μm. Numerous oocyte-specific genes that are critical for folliculogenesis and embryogenesis are initiated during the transition of the primordial follicle to primary follicle [14].
## (s3) Zygote cytoplasts as recipients for somatic cell nuclear transfer
Number of References: 5

(p3.0) Modliński micro-surgically introduced mouse CBA/H T6T6 strain 8-cell stage nuclei into intact zygotes and identified the presence of two T6 marker chromosomes in resulting tetraploid blastocysts [39]. McGrath and Solter reported in their study the inability of mouse blastomere nuclei transferred into enucleated zygotes to support in vitro development [40]. Upon analyzing causes leading to failure of development of reconstituted zygotes, in which karyoplasts containing intact pronuclei were removed, Greda et al. stated that the failure might have been caused by the enucleation protocol [33]. It is known that the nucleolus in the zygote and early embryo is maternally inherited and originates from material sources that are present in the oocyte GV [41]. The nucleoli originating from a somatic cell or even from an embryonic stem cell cannot substitute for the original oocyte nucleolar material [41]. These studies suggest that zygote nuclear factors are critically important for the development of cloned embryos with zygote cytoplasts as recipients.
## (s6) Localization and distribution of transcriptional factors in oocytes
Number of References: 7

(p6.0) The mammalian oocyte has a maternal gene expression profile that is unique. In mice, transcription arrest occurs at GV stage and is maintained until zygotic genome activation takes place approximately 9-10 h after fertilization [79][80][81]. Between cessation and resumption of transcription, the maternal transcription profile is reprogrammed to that observed in the embryos [82,83]. The mechanisms underlying transcription-reprogramming still warrant further investigation. A clearer understanding of the molecular mechanisms responsible for the reprogramming process will undoubtedly lead to a higher efficiency in cloning technology [84,85].
## (s7) Transcriptomics may dissect the detailed gene regulatory mechanisms in cellular reprogramming
Number of References: 10

(p7.0) The most advanced single-cell sequencing technique makes it possible to dissect the detail of genetic regulatory mechanisms in a valuable resource, e.g., progressively developmental embryo. The transcriptome analysis of the different developmental stages from oocyte to morula in both mouse and human reveal a sequential order of transcriptional changes in pathways of cell cycle, gene regulation, translation and metabolism, acting in a stepwise fashion from cleavage to morula [92]. In cattle, RNA sequencing analysis of transcripts in GV and M II oocytes, and in 4-, 8-, 16-cell and blastocyst stage embryos indicated that the largest proportion of gene activation was found in 8-cell embryos [93]. Cross-species comparisons between mouse and human showed that significant difference in developmental specificity and timing existed [92]. It has been reported that the maternal glucocorticoid receptor transcript and protein participated in the maternal reprogramming of zebrafish development [94]. Ces5/Ooep was suggested to be a maternal-effect gene directly associated with the oocyte cytoplasmic developmental patterns [95]. Transcription factor gene MSX1 played important roles in bovine preimplantation embryo development. Injection of long double-stranded RNA and small interfering RNA to oocytes at GV stage reduced MSX1 mRNA expression, which significantly affected oocyte maturation and the subsequent embryo development [96]. In Xenopus, the conserved long non-coding RNAs were expressed in a developmental stage-specific fashion [97]. It has been demonstrated that miRNAs played regulatory roles during early embryo development [98]. The Gli-like transcription factor Glis1 was enriched in unfertilized oocytes and embryos at the 1-cell stage. Glis1 was observed to promote multiple pro-reprogramming pathways, including Myc, Nanog, Lin28, Wnt, Essrb and the mesenchymal-epithelial transition [99]. Glis1, as a mater-nal transcription factor effectively promoted the direct reprogramming of somatic cells during iPSC generation [99]. With deep analysis of transcriptomics in oocytes and early developmental stage embryos, more maternal transcription factors inducing cellular reprogramming will be identified. Figure 1 illustrates the possible roles of transcriptional factors in oocyte maturation, fertilization, nuclear transfer reprogramming and embryo development.
