# Pseudogene Transcripts in Head and Neck Cancer: Literature Review and In Silico Analysis

CorpusID: 237324218 - [https://www.semanticscholar.org/paper/4e19f449a62a70b0977a23d03a74730522fd81d2](https://www.semanticscholar.org/paper/4e19f449a62a70b0977a23d03a74730522fd81d2)

Fields: Biology, Medicine

## (s3) HNC in General
(p3.0) In HNC tumor cells, loss of PTENP1 (phosphatase and tensin homolog pseudogene 1) pseudogene transcript modulated malignant behavior and worst survival of HNC patients, leading to tumor cell proliferation, colony formation, and migration, possibly by interacting with its parental gene PTEN (phosphatase and tensin homolog) [25].

(p3.1) In HNC tumor cells and cell lines (FaDu, Cal-27, SCC4, and SCC9), FKBP9P1 (FKBP prolyl isomerase 9 pseudogene 1) pseudogene transcript abundance was found to correlate with advanced tumor stage and poor prognosis of patients by enhancing tumor cell proliferation, migration, and invasion, possibly by interacting with the PI3K/AKT signaling pathway [26].
## (s13) TCGA Data Analysis
(p13.0) Clinicopathological aspects of the selected 219 patients are presented in Supplementary Table S1. We identified 370 pseudogene transcripts associated with HNC, where SPATA31D5P (SPATA31 subfamily D member 5 pseudogene), HERC2P3 (hect domain and RLD2 pseudogene 3), SPATA31C2 (SPATA31 subfamily C member 2), MAGEB6P1 (melanoma antigen family B6 pseudogene 1), SLC25A51P1 (solute carrier family 25 member 51 pseudogene 1), BAGE2 (B melanoma antigen family member 2), DNM1P47 (DNM1 pseudogene 47), SPATA31C1 (SPATA31 subfamily C member 1), ZNF733P (zinc finger protein 733 pseudogene), and OR2W5 (olfactory receptor, family 2, subfamily W, member 5) were found to be the most deregulated in HNC. The ten most deregulated pseudogene transcripts, chromosome location, gene family function, and involvement in cancer can be found in Table 2. The complete list of results can be found in Supplementary Dataset S1.

(p13.1) When we stratified patients' information by tumor location, we found a few differences in pseudogene transcripts' pattern that could be explained by different cancer behaviors [43], but some of the identified pseudogenes were present in more than one tumor location.

(p13.2) In oral cancer (n = 62), SPATA31D5P, NBPF25P (neuroblastoma breakpoint family member 25 pseudogene), HSP90AB2P (heat shock protein 90 alpha class B member 2 pseudogene), NXF4 (nuclear RNA export factor 4 pseudogene), FOLH1B (folate hydrolase 1B), DNM1P47, BNIP3P1 (BCL2/adenovirus E1B 19 interacting protein 3 pseudogene 1), PKD1L2 (polycystin 1-like 2 pseudogene), BAGE2, and ZNF658B (zinc finger protein 658B, pseudogene) were found to be the 10 most deregulated of 154 pseudogene transcripts identified (Table 2 and Supplementary Dataset S1).

(p13.3) In oropharyngeal cancer (n = 51), POTEA (POTE ankyrin domain family, member A), MROH5 (maestro heat-like repeat family member 5), MSL3P1 (male-specific lethal 3 homolog pseudogene 1), HLA-H (major histocompatibility complex class I, H pseudogene), TUBB8P7 (tubulin beta 8 class VIII pseudogene 7), SLC7A5P2 (solute carrier family 7 member 5 pseudogene 2), DPY19L2P1 (DPY19L2 pseudogene 1), TSSC2 (tumor suppressing sub-transferable candidate 2 pseudogene), SPATA31C2, and NXF4 were found to be the 10 most deregulated of 109 pseudogene transcripts identified ( Table 2 and Supplementary Dataset S1).

(p13.4) In hypopharyngeal cancer (n = 8), DPY19L2P3 (DPY19L2 pseudogene 3), SPATA31D5P, GBA3 (glucosidase beta acid 3), PLEKHM1P (pleckstrin homology domain containing, family M member 1 pseudogene), DPY19L2P1, MST1P2 (macrophage-stimulating 1 pseudogene 2), RP11-44F14.1, ADAM21P1 (ADAM metallopeptidase domain 21 pseudogene 1), MAGEB6P1, and OR12D2 (olfactory receptor family 12 subfamily D member 2) were found to be the 10 most deregulated of 21 pseudogene transcripts identified (Table 2 and Supplementary Dataset S1).

(p13.5) In laryngeal cancer (n = 98), HERC2P3, SPATA31D5P, SPATA31C2, SLC25A51P1, MAGEB6P1, SPATA31C1, BAGE2, PNLIPRP2 (pancreatic lipase-related protein 2), ZNF733P and DNM1P47 were found to be the 10 most deregulated of 287 pseudogene transcripts identified (Table 2 and Supplementary Dataset S1).
## (s14) Co-Expression Networks and GO Enrichment Analysis
(p14.0) We performed co-expression network and GO enrichment analyses to map the relationship between pseudogenes and other genes in the genome. After analyzing 19,406 proteincoding genes and 31 pseudogenes, our co-expression networks contained 14,379 genes organized in 36 modules, with an average of 399 genes per module (median: 140 genes; range: 35-3645 genes; Supplementary Dataset S2).

(p14.1) From the 31 most deregulated pseudogenes from HNC and its subtypes, 14 of them (SPATA31D5P, SPATA31C2, DNM1P47, NBPF25P, HSP90AB2P, NXF4, ZNF658B, POTEA, MROH5, HLA-H, DPY19L2P1, DPY19L2P3, GBA3, and PNLIPRP2) were present in our network across 8 modules (light yellow, red, dark magenta, orange, black, salmon, ivory, and dark olive green). Although the other pseudogenes of interest did not show a coexpression profile similar enough to other genes to be assigned to a module according to the parameters used to build this network, they are still important for further studies in HNC based on our previous TCGA data analysis.

(p14.2) We chose to highlight three modules (light yellow, red, and dark magenta) that contain genes well-known to be involved in head and neck carcinogenesis based on two important reviews [76,77]. However, the other five modules also presented relevant gene associations that should be evaluated in further studies.

(p14.3) The light-yellow module contained the greatest number of pseudogenes (NBPF25P, HSP90AB2P, ZNF658B, and DPY19L2P3). Interestingly, this module also contains 12 genes known to participate in head and neck carcinogenesis (CASP8, involved in apoptosis; NOTCH2 and NOTCH3, involved in cell differentiation; TRAF3, involved in antiviral response; RB1 and HRAS, involved in cell cycle control and proliferation; PTEN, involved in apoptosis and cell cycle control; CUL3 and NFE2L2, involved in oxidative stress response; EP300, involved in chromatin remodeling, and the transcription factors IGF1R and TP63) [76,77], suggesting that these genes may directly or indirectly interact with the pseudogenes in this module and play a role in HNC (Figure 2A). In addition, the light-yellow module is enriched in GO terms related to tumor development, such as regulation of cell communication, cellular response to stress, and intracellular transport ( Figure 2B). The potential relationship between the described pseudogenes and genes in this module has not been explored in the literature yet.
