# SURVEY AND SUMMARY Regulatory roles of RNA modifications in breast cancer

CorpusID: 237555138 - [https://www.semanticscholar.org/paper/266a902ceb71a7637608844159df34a9f8cfb99b](https://www.semanticscholar.org/paper/266a902ceb71a7637608844159df34a9f8cfb99b)

Fields: Medicine, Biology

## (s0) INTRODUCTION
(p0.0) RNA modifications, collectively termed the epitranscriptome, are crucial regulators of temporal and spatial gene expression programs. Currently, over 170 RNA modifications decorating all RNA species and in all three kingdoms of life have been described (1,2). Although modification to RNA has been documented for over 50 years, the functions of most of these modifications are largely unknown. Recent development of more sensitive and specific technologies, such as high-throughput sequencing and improved mass spectrometry, shed light on to the exciting new field of RNA epitranscriptomics.

(p0.1) In general, the fate of modified RNAs is determined by the coordinated actions of writers, erasers and readers that impose, remove and recognize the chemical mark. Some of the writers are stand-alone enzymes, whereas others act as multiprotein writer complexes that also comprise accessory subunits. In addition, some of the RNA modifications are reversible i.e. removed by the erasers, whereas others are irreversible. Most of the known RNA modifications map to abundant RNAs such as transfer RNA (tRNA) and ribosomal RNA (rRNA), tRNA being the most extensively modified RNA type in the cell (3) (Figure 1).
## (s1) Breast cancer
(p1.0) According to a global cancer statistics study, breast cancer is the most commonly diagnosed cancer in women and among the leading cause of cancer-related death in females (27) (Figure 4). Although there is an evident increase in breast cancer incidence and mortality among pre-and postmenopausal women, females of younger age are also at risk every year, with higher emphasis in developing countries (28). Hence, although advanced therapies and early detection have improved the survival rate (29), the causal factors of breast carcinoma still remain elusive.
## (s4) 5-methylcytosine
(p4.0) The methylation of the carbon 5 in cytosine (m 5 C) was originally found in rRNA and tRNA; however, recent studies have also detected m 5 C on mRNA (87,88). In eukaryotes, the large ribosomal subunit contains two m 5 C residues (22) that are essential to maintain ribosomal structure and fidelity during translation, playing a significant role in lifespan and stress resistance (89). On tRNAs, m 5 C protected from angiogenin-mediated endonucleolytic cleavage, and thereby avoided the biogenesis of tRNA-derived small RNA fragments (tRFs) (90,91). Such tRFs play a key role in regulating gene expression programs participating in various physiological processes such as cell stress, cell growth and cell differentiation (92). tRFs also play significant roles in various human diseases, including cancer (93). On mRNA, m 5 C promotes export (87), stabilization (88) and translation (94,95).
## (s7) RNA editing
(p7.0) RNA editing refers to modifications of specific nucleotides in RNA sequence differentiating it from its corresponding DNA sequence that may lead to pathological consequences. The most common form of RNA editing is from adenosine (A) to inosine (I) (161) and is catalyzed by specific enzymes namely adenosine deaminases acting on RNA (ADARs). In mammals, three ADAR genes encode for four different isoforms known as ADAR1p150, ADAR1p110, ADAR2 and ADAR3 (162). A-to-I editing within the protein-coding region of an mRNA can result in an amino acid change in the encoded protein as the resulting inosine is interpreted as guanosine by the translational machinery leading to an amino acid substitution in the protein product (163). In addition, A-to-I can alter regulatory motifs, binding of RNA-binding proteins, or RNA secondary structures influencing pre-mRNA alternative splicing (164)(165)(166). Furthermore, RNA editing within 3 UTRs of target transcripts can alter miRNA targeting an perturb miRNA-mediated regulation of oncogenes and tumor-supressors (167). The enzymes involved in A-I editing on tRNAs are known as Adenosine deaminases that act on tRNAs (ADATs). Deregulated ADATs-mediated RNA editing is associated with diseases such as type 2 diabetes, neurological disorders, mitochondria-related disorders and cancer (168). Less frequent cytidine (C) to uridine (U) mRNA editing is carried out by cytidine deaminases belonging to the family of enzymes known as activation induced cytidine deaminase/apolipoprotein B editing complex (AID/APOBEC) or AADs (169), eleven of which are found in humans (170).
## (s8) A-to-I editing in breast cancer
(p8.0) A-to-I editing is known to be the major source of mRNA variability in breast cancer (171), and it is controlled by type-I interferon response and high ADAR DNA copy number (172). Hence, >76 000 RNA editing sites were identified by using a bioluminescent reporter system and upon interferon treatment of the breast cancer MCF7 cell line (173). Breast tumors harbor amplified levels of ADAR (174) and inhibiting its activity has been shown to induce apoptosis. Indeed, A-to-I mRNA editing regulated breast cancer cell proliferation by modulating the stability and expression of genes involved in cell cycle and DNA damage response being ATM, MDM2, MDM4, CENPN and XPO1 among the affected transcripts (175). In another study, increased A-to-I mRNA editing was found on the mRNA of ATM, GINS4 and POLH affecting their stability and/ or expression in primary breast tumors when compared to the noncancerous tissues (171). In addition, Gumireddy et al. reported that A-to-I mRNA editing of ‚ê• -amino butyric acid receptor alpha 3 (GABRA3), one of the subunits of GABA (A) receptor, changed its role from a metastasis promoter to a tumor suppressor by suppressing the ERK pathway activation (176). In an attempt to assess the role of A-to-I RNA editing on proteomic diversity, Peng et al. showed that A-to-I RNA editing contributed to protein heterogeneity in breast cancer (177). Hence, nine unique RNA editing sites with variant peptide evidence were identified such as COPA I164V and IGFBP R78G which were detected across 11 out of 36 BRCA mass spectrometry datasets (177). These peptide variants were shown to be presented as self-antigens by human leukocyte antigen (HLA) molecules, and thus recognized by the immune system. For instance, CD8 + effector T cells were evidenced in tumors in response to peptide variants generated from cyclin I (178). A-to-I RNA editing has also been linked to resistance to chemotherapy. Specifically, RNA editing at 26 sites of 3 UTR modulated the expression of dihydrofolate reductase resulting in highly proliferative breast cancer cells which were resistant to methotrexate (179). To sum up, existing data suggests an important role of A-to-I RNA editing in genes associated with breast cancer-relevant pathways and treatment outcomes, suggesting an important role of ADAR1 function in breast tumorigenesis.
## (s9) C-to-U editing in breast cancer
(p9.0) APOBEC-dependent somatic mutation confers increased susceptibility for breast cancer (180) and is found to be enriched in the HER2 subtype (181,182). Overexpression of cytidine deaminase APOBEC3A in HEK293T cells caused mRNA alterations in several tumor associated genes including PTEN, KMT2A, ATM, BRCA1 and BRCA2 (183); however, their significance in breast tumorigenesis is yet to be evaluated. mRNA expression of APOBEC3B was evaluated in ductal carcinoma in situ (DCIS), invasive breast cancer (IBC) and normal breast cells (184). Normal breast cells expressed reduced levels of APOBEC3B, while increased expression was evident in DCIS and IBC. Recently, APOBEC3-mediated C-to-U RNA editing in breast cancer was shown to be associated with improved patient survival and enhanced immune activity (185). In another study, using a bioinformatics approach, C-to-U RNA editing levels were estimated in 1040 primary breast tumor tissues and 93 adjacent normal tissues. More than 5000 APOBEC3-mediated RNA editing sites were identified using TCGA sequencing data. For 440 sites, editing was found on 411 transcripts among which most prevalent were on GATA Zinc Finger Domain Containing 2B (GATAD2B), Serpin Family A Member 1 (SERPINA1) and Adenosine Monophosphate Deaminase 3 (AMPD3) (185). Moreover, Apobec-1 complementation factor (A1CF), another mRNA editing enzyme, promoted proliferation of basal-like breast cancer cells by targeting interleukin-6 (186). Overall, the existing data on C-to-U editing suggests important roles in breast cancer progression and further research may identify targets with therapeutic importance.
## (s10) Other RNA modifications related to breast cancer
(p10.0) Growing evidence in recent years has pointed out that tRNAs and their derivatives are dysregulated in breast and other cancers (187). However, the functional role of tRNA modifications in tumorigenesis is still elusive. As mentioned before, several modifications are shared by the distinct RNA species but others are unique to tRNAs, as is the case of cm5U34 and mcm5U34 at the wobble position. These marks have been shown to influence the translational efficiency and the accuracy of the reading frame (188). In addition, the catalytic enzymes for mcm5s2 deposition are upregulated in breast cancer sustaining metastasis (189). They enhanced the translation of DEK and LEF1, two oncogenes, enabling breast cancer cells to migrate and invade other tissues (189). Another modification that occurs at the wobble anticodon position is queuosine which is dependent on the gut microbiome (190). Thus, the microbiome product queuine is the substrate for the enzymes tRNA guanine transglycosylases (TGTs). In eukaryotes, the active TGT is a heterodimer formed of a catalytically active queuine tRNA-ribosyltransferase subunit 1 (QTRT1) and a catalytically inactive QTRT2 subunit (191). Knockout of QTRT1 in MCF7 and MDA-MB-231 cells produced changes in the functions of genes involved in cell proliferation, junction formation and migration (190). These results have been validated in mouse models, thus enforcing the significant role that this modification is playing in breast cancer development (190).

(p10.1) TRMT12, a tRNA methyltransferase that mediates posttranscriptional modifications on tRNAPhe has been reported to be consistently amplified and overexpressed in cell lines and breast cancer patient samples (192). In yeast, it catalyses the formation of wybutosine (yW) on the 37th residue of tRNAPhe, modification that helps in stabilization of the codon-anticodon interaction and in maintenance of the reading frame (192). Thus, disruption at the level of this modification can lead to translational errors and is therefore of great interest to establish the role of these enzymes, together with its effect on tRNAs, in tumorigenesis of breast cells.

(p10.2) It is noteworthy that many of the methyltransferases that catalyze the methylation of different types of RNAs have been shown to be deregulated in breast cancer. Although the lack of any association between the deposited marks and the expression of these proteins made us filter out these studies, there are some that are worth mentioning. For instance, tRNA methyltransferase 2 homolog A (TRMT2A) which catalyzes the methylation of the U5 of tRNAs and regulates cell cycle has been associated with increased recurrence risk in HER2+ breast cancer patients (193). Human tRNA methyltransferase 9-like (hTRM9L) protein, an another tRNA methyltransferase, is downregulated in breast cancer and was shown to suppress tumor growth in vivo by decreasing proliferation, cell cycle arrest in G0/G1, upregulation of LIN9, and blocking the hypoxia response (194) in colorectal cancer. Whether this mechanism is the same in breast cancer and if it is mediated though methylation remains to be further investigated.

(p10.3) Despite being less explored than other RNA marks, many of these modifications convey to be promising clues in deciphering the molecular enigma behind the development of breast cancer. Thus, it will be of increased interest to see how extensive studies of these marks on the various species of RNA can contribute to a more precise picture of breast cancer tumorigenic process.
## (s11) Perspectives
(p11.0) In the last decade, and as a result of the advent of more reliable methodology, the number of epitranscriptomic studies has dramatically increased. The boost and excitement of this emerging field has led to a tremendous progress in identifying the machinery and the role of RNA modifications. Hence, RNA marks have emerged as important regulators of a variety of cellular processes and their roles in human disease, including cancer, have extensively been described. It has been shown that writers, erasers and readers can act as tumor suppressors or as oncogenes depending on the cellular context. Thus, a single enzyme can have opposing roles in distinct cancer types. However, as a consequence of this fast-growing discipline, many erroneous conclusions have been drawn, and a large number of controversial studies has emerged. In addition, most of the works have just focused on few RNA modifications, and thereby the functions of most RNA modifications remain to be characterized.

(p11.1) During these years, RNA modifications have been comprehensively mapped transcriptome-wide by coupling antibody immunoprecipitation or chemical probing with nextgeneration sequencing. Yet, tools for the simultaneous identification of distinct chemical marks in the same RNA are currently not available. Hence, it is unknown how the different RNA modifications interplay to influence cancer development. Aberrant expression of such RNA-modifying machinery has been acknowledged for most aspects of breast and other cancer types. One particular challenge is to attribute a phenotype to a particular chemical mark as there is a crosstalk between rRNA, tRNA and mRNA modifications which involves common modification factors. For example, NSUN2 can impose m 5 C on both mRNA and tRNA species (195). In addition, several previous conclusions might have to be revised as some of these key players are moonlighting proteins which perform multiple autonomous and often unrelated functions. Therefore, it is also challenging to distinguish between the canonical, i.e. directly related to RNA modifications and secondary protein functions. For instance, knockouts of IME4, the yeast homolog of METTL3, display more severe phenotypes than those derived by a catalytic mutant, suggesting that IME4 has methylation-independent functions (196,197). Indeed, it has been shown that in lung cancer, cytoplasmic METTL3 promotes the translation of oncogenes by a mechanism independent of its methyltransferase activity (198).

(p11.2) Because of the inherent reversibility of some of the RNA modifications, inhibitors targeting the RNA-modification machinery are promising therapeutic targets for breast and other tumors. As m 6 A is the most studied RNA modification, it is not surprising that lead biotech companies have developed drugs to target the writer complex, specifically METTL3 (199). Hence, STORM Therapeutics has developed the first catalytic inhibitor of METTL3. Pharmacological inhibition of METTL3 led to strong antitumor effects in vitro and in relevant mouse models of acute myeloid leukemia (AML) (200).

(p11.3) Overall, as highlighted in this review, considerable advance has been made in recent years in the research field of epitranscriptomics which has provided a link between RNA modifications and breast cancer. Yet, the epitranscrip-tome remains vastly unexplored. Thus, it appears that we just perceive the tip of the iceberg and that the number of RNA modifications is much higher than previously estimated. The exhaustive characterization of RNA modifications and the molecular mechanism in which they are involved holds a new avenue for novel cancer therapies.
