# Sequencing Overview of Ewing Sarcoma: A Journey across Genomic, Epigenomic and Transcriptomic Landscapes

CorpusID: 5776686 - [https://www.semanticscholar.org/paper/1df3138d7e87f50e9caa848e207e7d2c35dd8294](https://www.semanticscholar.org/paper/1df3138d7e87f50e9caa848e207e7d2c35dd8294)

Fields: Biology, Medicine

## (s8) DNA Damage Response and Repair: Systemic and Targeted Therapy
Number of References: 24

(p8.0) Chemotherapy is an essential part in the treatment of EWS [154,158]. Over the past decades, the combination of chemotherapeutics, dosage and administration protocol has been adjusted to improve tumor response and reduce toxicity [155,159,160]. The present standard treatment protocol for EWS is based on combination of vincristine, doxorubicin, ifosfamide or cyclophosphamide and etoposide [2,159]. Most of these are DNA damaging drugs. As EWS contains a limited number of secondary mutations, it is likely that these tumors have an intact DNA damage response mechanism. Alkylating and double strand break causing agents consequently activate this mechanism leading to growth arrest and apoptosis of EWS cells. The hypothesis of an intact DNA damage response mechanism correlates with the chemotherapeutic resistance of TP53 mutated EWS tumors, a key gene in this mechanism, since these tumors do not have an intact DNA damage response mechanism [41]. Fusion positive RMS has a limited number of mutations, similarly to EWS, but it is less sensitive to chemotherapy compared to translocation negative RMS [44]. This implies that downstream EWSR1-ETS effects may partly be responsible for the chemotherapy and radiotherapy sensitivity. In accordance, EWSR1-ETS associated DSBs have been identified and radiation induced damage turnover in EWS was reduced compared to osteosarcoma [161,162]. This makes the damage repair pathway a promising candidate to target. Compromising the DNA damage repair pathway via inhibition of poly (ADP ribose) polymerase 1 (PARP1) did indeed lead to inhibited proliferation in EWS cell lines and potentiated the response to temozolomide and irinotecan [19,[161][162][163]. However, in EWS xenografts and patients treated with a PARP1 inhibitor, only the combination with temozolomide or irinotecan was effective [162,164]. In colon cancer xenografts this effect was observed as well [165]. The sensitivity of colon cancer to PARP1 inhibition is hypothesized to be related to a less functional homologous recombination due to cohesin complex aberrancy [75,77,166]. In glioblastoma cells, a correlation between PARP1 sensitivity and the presence or absence of the cohesin complex gene STAG2 was demonstrated [167]. In the EWS PARP1 inhibition studies both STAG2 wild type and mutant cell lines were sensitive to PARP1 inhibition in combination with chemotherapy [163,168]. A specific role for STAG2 in this is therefore unlikely in EWS. Overall, it seems EWSR1-ETS interferes in the DNA damage repair pathway by a yet unexplainable way based on data obtained from genome and transcriptome sequencing studies leading to chemotherapy sensitivity. Identifying the TP53 independent DNA damage response and repair pathway could open novel therapeutic options.
## (s12) Conclusions
Number of References: 4

(p12.0) By sequencing EWS at the genome, epigenome and transcriptome level, an atlas can be created which would help to fundamentally understand EWS and help to identify important nodes as therapeutic candidates. The EWSR1-ETS translocation is the characteristic pathognomonic alteration found in all tumors so far. The fusion protein act as a strong transforming oncogene and, in experimental conditions, the transfection of cells with normal cellular backgrounds rather leads to oncogene-induced apoptosis than to transformation. Recent studies, however, showed that stem cells from young individuals with the necessary permissive background did form tumors, pointing towards the importance of epigenetic controlling in cellular/tissue differentiation in providing the necessary niche for the transformation [19,22,25,38]. Therefore, mapping these genomic and functional genomic alterations can lead to identification of the cell of origin, improvements in prediction of clinical outcome, and discovery of novel therapeutic targets. These prospects have led to numerous, independent investigations using various approaches related to OMICs.
